{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 185,
   "metadata": {},
   "outputs": [],
   "source": [
    "# This python notebook annotate the bioconda recipes using the ids from bio.tools in the external session \n",
    "# in bioconda recipes. Two different input should be provided, the bioconda_recipes_path in your local machine and\n",
    "# the file with the mapping.   \n",
    "\n",
    "\n",
    "import glob2\n",
    "import os\n",
    "\n",
    "import requests\n",
    "import sys\n",
    "import yaml\n",
    "import getopt\n",
    "import re\n",
    "from pandas import DataFrame, read_csv\n",
    "\n",
    "from ruamel.yaml import YAML\n",
    "\n",
    "from yaml.constructor import ConstructorError\n",
    "from yaml.scanner import ScannerError"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 190,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_bioconda_yalm(bioconda_path):\n",
    "    yalm_files = dict()\n",
    "    matches =  glob2.glob(bioconda_path + '/**/*.yaml', with_matches=True)\n",
    "    yaml = YAML(typ='jinja2')\n",
    "    yaml.allow_duplicate_keys = True\n",
    "    yaml.allow_unicode = True\n",
    "    if(len(matches) > 0 ):\n",
    "        for file_name in matches:\n",
    "            if(os.path.isfile(file_name[0])):\n",
    "                #print(file_name[0])\n",
    "                path_list = file_name[0].replace(bioconda_path, \"\").split(os.sep)\n",
    "                software_name = path_list[1]\n",
    "                try:\n",
    "                    with open(file_name[0]) as fp:\n",
    "                         yalm_file = yaml.load(fp)\n",
    "                    yalm_files[software_name] = (file_name[0], yalm_file)\n",
    "                except (ScannerError, ConstructorError, TypeError, AttributeError) as error:\n",
    "                    print(\"ERROR | \" + software_name + \"\\t\" + file_name[0])\n",
    "    return yalm_files          "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 191,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_annotation_mapping(mapping_file):\n",
    "    mapping = read_csv(mapping_file)\n",
    "    return mapping    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 192,
   "metadata": {},
   "outputs": [],
   "source": [
    "def annotate_yaml(yaml_file, biotool):\n",
    "    path_file = yaml_file[0]\n",
    "    yaml_file_content = yaml_file[1]\n",
    "    if yaml_file_content.get('extra') is None:\n",
    "        yaml_file_content['extra'] = {}\n",
    "    else:\n",
    "        print(\"EXTRA has been found: \" + biotool)\n",
    "    yaml_file_content['extra']['biotools']=[biotool]\n",
    "    with open(path_file, 'w') as fp:\n",
    "        yaml.dump(yaml_file_content, fp)\n",
    "    \n",
    "\n",
    "def annotate_yamls(yaml_dictonary, mappings):\n",
    "    for (idx, row) in mappings.iterrows():\n",
    "        if yaml_dictonary.get(row.bioconda) is not None:\n",
    "            annotate_yaml(yaml_dictonary[row.bioconda], row.biotools)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 193,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'2pg_cartesian': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/2pg_cartesian/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/rodrigofaccioli/2pg_cartesian'), ('license_file', 'LICENSE'), ('summary', '2pg cartesian is a framework of optimization algorithms for protein structure prediction.')])), ('package', CommentedMap([('name', '2pg_cartesian'), ('version', '1.0.1')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', 'v1.0.1.tar.gz'), ('url', 'https://github.com/rodrigofaccioli/2pg_cartesian/archive/v1.0.1.tar.gz'), ('md5', 'd97fcd5fff5cb0813d4ea918c57d991f')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc', 'gromacs'])])), ('test', CommentedMap([('commands', ['protpred-Gromacs-Test_random_number'])]))])), 'abawaca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abawaca/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/CK7/abawaca/archive/v<{ version }}.tar.gz'), ('sha256', '57465bb291c3a9af93605ffb11d704324079036205e5ac279601c9e98c467529')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://github.com/CK7/abawaca'), ('license', 'open source'), ('summary', 'abawaca is a binning program for metagenomics')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1038/nature14486')]))])), 'abricate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abricate/0.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'abricate'), ('version', '0.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'abricate_v0.5.tar.gz'), ('url', 'https://github.com/tseemann/abricate/archive/v0.5.tar.gz'), ('md5', '9d2b54511190c3beb6e018675c4d4d0e')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'perl', 'perl-bioperl >=1.6.924', 'perl-file-slurp', 'perl-text-csv', 'perl-list-moreutils', 'perl-file-spec', 'perl-json', 'perl-lwp-simple', 'perl-findbin', 'blast >=2.2.30', 'zlib', 'unzip', 'emboss'])])), ('test', CommentedMap([('commands', ['abricate --version', 'abricate --list', 'unzip --help'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/abricate'), ('license', 'GPL2'), ('license_file', 'LICENSE'), ('summary', 'Mass screening of contigs for antibiotic resistance genes')]))])), 'abruijn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abruijn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'abruijn'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'abruijn-<{ version }}.tar.gz'), ('url', 'https://github.com/fenderglass/ABruijn/archive/<{ version }}.tar.gz'), ('sha256', '199a560a931508ac2ba3aeb66b8af596e9937dd747ee37078ae0f732479a4b8b')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install  --record record.txt')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['abruijn --version'])])), ('about', CommentedMap([('home', 'https://github.com/fenderglass/ABruijn/'), ('license', 'BSD-3-Clause'), ('summary', 'Long read assembly via A-Bruijn graph')]))])), 'abundancebin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abundancebin/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://omics.informatics.indiana.edu/mg/get.php?justdoit=yes&software=<{ name }}.<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['command -v abundancebin', 'abundancebin &> /dev/null || [[ \"$?\" == \"255\" ]]'])])), ('about', CommentedMap([('home', 'hhttp://omics.informatics.indiana.edu/AbundanceBin/'), ('license', 'copyright'), ('summary', 'Abundance-based tool for binning metagenomic sequences'), ('description', 'AbundanceBin is an abundance-based tool for binning metagenomic sequences,\\nsuch that the reads classified in a bin belong to species of identical or\\nvery similar abundances. AbundanceBin also gives estimations of species\\nabundances and their genome sizes — two important characteristic parameters\\nfor a microbial community.\\n')])), ('extra', CommentedMap([('recipe-maintainers', ['keuv-grvl']), ('doi', '10.1007/978-3-642-12683-3_35')]))])), 'abyss': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abyss/2.0.1-k128/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}-k128'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/<{ name|lower }}/releases/<{ version }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '596c56a476b26a294b46e6e36890c136')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'google-sparsehash', 'sqlite', 'openmpi']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'bwa', 'pandoc', 'sqlite', 'openmpi'])])), ('test', CommentedMap([('commands', ['ABYSS --version'])])), ('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler')]))])), 'adam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/adam/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/bigdatagenomics/adam'), ('license', 'Apache 2'), ('summary', 'Genomics analysis platform built on Apache Avro, Apache Spark and Parquet')])), ('package', CommentedMap([('name', 'adam'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'adam-distribution-spark2_2.11-<{ version }}-bin.tar.gz'), ('url', 'https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.22.0/adam-distribution-spark2_2.11-0.22.0-bin.tar.gz'), ('sha256', '31624954eb473f2ec24354af7e6303a47887cf2ca4076bb71f54618aef59e15b'), ('patches', ['adam-shell.patch', 'adam-submit.patch'])])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=8', 'pyspark'])])), ('test', CommentedMap([('commands', ['adam-submit --help', 'adam-submit transform --help'])]))])), 'addrg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/addrg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'addrg'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'addrg-<{ version }}.tar.gz'), ('url', 'https://github.com/holtgrewe/addrg/archive/v<{ version }}.tar.gz'), ('md5', 'bc87406382100ded3e701cc410107ac5')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib >=1.0.0']), ('run', ['libgcc', 'htslib >=1.0.0'])])), ('about', CommentedMap([('home', 'https://github.com/holtgrewe/addrg'), ('license', 'MIT'), ('summary', 'Add read group to BAM files')])), ('test', CommentedMap([('commands', ['{ >&1 addrg || true; } | grep Usage'])]))])), 'aenum': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/aenum/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'aenum'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'aenum-1.4.5.tar.gz'), ('url', 'https://pypi.python.org/packages/dd/70/70cf6b8bb88b77f5f14d12dfc0cc3237ef488bdc516651aeca2a02b69d5f/aenum-1.4.5.tar.gz'), ('md5', '1d83188570b126022a2467a18765decc')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['aenum'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/stoneleaf/aenum'), ('license', 'BSD License'), ('summary', \"Advanced Enumerations (compatible with Python's stdlib Enum), NamedTuples, and NamedConstants\"), ('license_family', 'BSD')]))])), 'afterqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/afterqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'afterqc'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'afterQC.tar.gz'), ('url', 'https://github.com/OpenGene/AfterQC/archive/v<{ version }}.tar.gz'), ('md5', '1ab2e8140ef6d383535edf46232626c9')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['after.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/OpenGene/AfterQC'), ('license', 'MIT'), ('summary', 'Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data. AfterQC can simply go through all fastq files in a folder and then output three folders: good, bad and QC folders, which contains good reads, bad reads and the QC results of each fastq file/pair. Currently it supports processing data from HiSeq 2000/2500/3000/4000, Nextseq 500/550, MiniSeq...and other Illumina 1.8 or newer formats.')]))])), 'age-metasv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/age-metasv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'age-metasv'), ('version', '2015.01.29.3')])), ('source', CommentedMap([('fn', 'age-metasv-344429ccb.tar.gz'), ('url', 'https://github.com/marghoob/AGE/archive/344429ccb.tar.gz'), ('md5', 'ec3fd9c0046b8f4a1d5269e3e77dac86'), ('patches', ['flags.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['age_align'])])), ('about', CommentedMap([('home', 'https://github.com/marghoob/AGE/tree/simple-parseable-output'), ('license', 'CCPL (Creative Commons Public License)'), ('summary', 'optimal alignment of sequences with structural variants (SVs), modifiied for MetaSV integration')]))])), 'agg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/agg/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Illumina/agg/archive/v<{ version }}.tar.gz'), ('md5', '76740cae2ba59de2d8b78b950b1eb499')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['agg  -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/agg'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'gvcf aggregation tool')]))])), 'ale': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ale/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '5fb3b66c495844707c0f6f870b350ce324d2fced.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://github.com/sc932/ALE/archive/5fb3b66c495844707c0f6f870b350ce324d2fced.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'samtools', 'setuptools', 'pymix', 'mpmath']), ('run', ['libgcc', 'python', 'mpmath', 'samtools', 'matplotlib', 'pymix'])])), ('test', CommentedMap([('commands', ['ALE --help', 'synthReadGen --help'])])), ('about', CommentedMap([('home', 'https://github.com/sc932/ALE'), ('license', 'NCSA'), ('summary', 'ALE: Assembly Likelihood Estimator')]))])), 'alignlib-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/alignlib-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'alignlib-lite'), ('version', 0.3)])), ('source', CommentedMap([('fn', 'alignlib-lite-0.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/33/5d/327aed4a90b315f886bcae96ee85149b275e48061b95a51da5f224841fb9/alignlib-lite-0.3.tar.gz'), ('md5', '5a7e3cea5539d9e81c94926c9334edb6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'setuptools', 'boost', 'cython']), ('run', ['libgcc', 'python'])])), ('test', CommentedMap([('imports', ['alignlib_lite'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/alignlib/'), ('license', 'BSD'), ('summary', 'Simple wrapper around alignlib C++ library for sequence alignment')]))])), 'amos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/amos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'amos'), ('version', '3.1.0')])), ('about', CommentedMap([('home', 'http://amos.sourceforge.net/wiki/index.php/AMOS'), ('summary', 'A Modular, Open-Source whole genome assembler'), ('license', 'Artistic License')])), ('source', CommentedMap([('git_url', 'http://git.code.sf.net/p/amos/code'), ('git_rev', '9bd658')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('test', CommentedMap([('commands', ['Minimo -h > /dev/null'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'boost', 'autoconf', 'automake', 'blat', 'mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python']), ('run', ['mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python'])]))])), 'andi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/andi/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/EvolBioInf/andi/releases/download/v<{ version }}/andi-<{ version }}.tar.gz'), ('md5', 'af3430c1d817a7a7f6045b97c0518121')])), ('requirements', CommentedMap([('build', ['gcc', 'libdivsufsort', 'gsl']), ('run', ['libgcc', 'libdivsufsort', 'gsl'])])), ('test', CommentedMap([('commands', ['andi -h'])])), ('about', CommentedMap([('home', 'https://github.com/evolbioinf/andi/'), ('license', 'GPL-3.0'), ('summary', 'Efficient Estimation of Evolutionary Distances')]))])), 'angsd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/angsd/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('about', CommentedMap([('home', 'http://www.popgen.dk/angsd/index.php/ANGSD'), ('license', 'GPLv3, MIT'), ('summary', 'ANGSD: Analysis of next generation Sequencing Data')])), ('package', CommentedMap([('name', 'angsd'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['htslib', 'gcc', 'llvm']), ('run', ['libgcc', 'htslib'])])), ('test', CommentedMap([('commands', ['angsd 2>&1 | grep \"angsd version\"', 'thetaStat 2>&1 | grep \"neutrality test\"'])])), ('source', CommentedMap([('fn', 'angsd-<{ version }}.tar.gz'), ('url', 'https://github.com/ANGSD/angsd/archive/<{ version }}.tar.gz'), ('md5', '89a5dcbff49de6b72e474b6f4bf8d112'), ('patches', ['Makefile.patch'])]))])), 'ansible': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ansible/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ansible'), ('version', '1.9.4')])), ('source', CommentedMap([('fn', 'ansible-1.9.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/a/ansible/ansible-1.9.4.tar.gz')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'paramiko', 'jinja2', 'pyyaml', 'httplib2', 'pycrypto', 'six']), ('run', ['python', 'setuptools', 'paramiko', 'jinja2', 'pyyaml', 'httplib2', 'pycrypto', 'six'])])), ('test', CommentedMap([('imports', ['ansible', 'ansible.constants', 'ansible.errors', 'ansible.playbook', 'ansible.utils']), ('commands', ['ansible -h'])])), ('about', CommentedMap([('home', 'http://www.ansible.com/'), ('license', 'GPLv3'), ('summary', 'Radically simple IT automation')]))])), 'ant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ant/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://apache.mirrors.spacedump.net/<{ name }}/source/apache-<{ name }}-<{ version }}-src.tar.bz2'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['openjdk']), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['ant -version > /dev/null'])])), ('about', CommentedMap([('home', 'http://ant.apache.org'), ('license', 'Apache 2.0'), ('summary', 'Apache Ant is a Java library and command-line tool that help building software.')]))])), 'antarna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'antarna'), ('version', '2.0.1.2')])), ('source', CommentedMap([('fn', 'antarna-2.0.1.2.tar.gz'), ('url', 'https://github.com/RobertKleinkauf/antarna/archive/v2.0.1.2.tar.gz'), ('sha256', '537d8e7cf94adf0d58ea3e381e21d9ff2fedc03a745abd4fbf4e48250cf2b4f3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'uuid', 'networkx', 'scipy']), ('run', ['python', 'numpy', 'uuid', 'networkx', 'scipy', 'viennarna'])])), ('test', CommentedMap([('commands', ['antarna.py -tGC 0.5 MFE -Cstr \"(((((...)))))\"', 'antarna.py -tGC 0.5 DP --accuracy \"((((((....)))))).((((((....)))))) UB 0.5\" --accuracy \"((((((((((((((.....)))))))))))))) UB 0.5\"', 'antarna.py -tGC 0.5 DP -Cstr \"(((((...)))))................\" --diff-accuracy \"................(((((...))))) UB 0.0 B 1.0\" --diff-accuracy \"........(((((...)))))........ UB 1.0 B 0.0\"'])])), ('about', CommentedMap([('home', 'https://github.com/RobertKleinkauf/antarna'), ('license', 'The MIT License (MIT)'), ('license_file', 'LICENSE'), ('summary', 'antaRNA is a python based implementation of ant-colony optimization of the RNA inverse folding problem.')]))])), 'antismash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antismash/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/antismash/antismash/downloads/<{ name|lower  }}-<{ version }}.tar.gz'), ('md5', '98e7a6402f0614acc80d8e87b17ed36b')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'argparse', 'backports.lzma', 'bcbiogff', 'biopython >=1.65', 'blast ==2.2.31', 'clustalw', 'cssselect', 'diamond ==0.8.36', 'ete2', 'fasttree ==2.1.9', 'glimmer ==3.02', 'glimmerhmm ==3.0.4', 'helperlibs', 'hmmer', 'hmmer2', 'openjdk', 'libxml2 ==2.9.4', 'mafft', 'matplotlib', 'meme', 'muscle', 'networkx', 'numpy', 'pandas', 'perl-bioperl', 'perl-pathtools', 'prodigal', 'pyexcelerator', 'pyquery ==1.2.9', 'pysvg', 'scipy', 'scikit-learn ==0.18.2', 'xz ==5.2.2'])])), ('test', CommentedMap([('commands', ['antismash -h'])])), ('about', CommentedMap([('home', 'http://antismash.secondarymetabolites.org/'), ('license', 'GPL3'), ('license_file', 'LICENSE.txt'), ('summary', 'antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters.')]))])), 'anvio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/anvio/2.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'anvio'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'anvio-2.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/1a/f4/e75b895b3348ec7d0d21e220735d0f9dd0ea2e67ef9a6ebb9c92bd1d7593/anvio-2.4.0.tar.gz'), ('md5', '581caa914116a24938938f99e3227b43')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'setuptools', 'cython', 'numpy', 'bottle >=0.12.9', 'pysam ==0.11.1', 'ete3 ==3.0.0b35', 'scipy >=0.13.3', 'scikit-learn >=0.17.1', 'django >=1.9.7', 'h5py >=2.6.0', 'cherrypy >=8,<9', 'requests >=2.10.0', 'psutil >=5.0.1', 'mistune ==0.7.4', 'openblas', 'gsl ==2.2.1', 'six ==1.10.0']), ('run', ['python', 'libgcc', 'cython', 'numpy', 'bottle >=0.12.9', 'pysam ==0.11.1', 'ete3 ==3.0.0b35', 'scipy >=0.13.3', 'scikit-learn >=0.17.1', 'django >=1.9.7', 'h5py >=2.6.0', 'cherrypy >=8,<9', 'requests >=2.10.0', 'psutil >=5.0.1', 'mistune ==0.7.4', 'six ==1.10.0', 'openblas', 'gsl ==2.2.1', 'sqlite', 'diamond', 'blast', 'prodigal', 'mcl', 'muscle', 'hmmer', 'bwa', 'samtools'])])), ('test', CommentedMap([('commands', ['anvi-pan-genome --help', 'anvi-script-reformat-fasta --help', 'anvi-profile --version'])])), ('about', CommentedMap([('home', 'http://merenlab.org/software/anvio/index.html'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An interactive analysis and visualization platform for omics data'), ('license_family', 'GPL3')]))])), 'apoc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/apoc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'apoc'), ('version', '1b16')])), ('source', CommentedMap([('fn', 'apoc_v1b16.tar.gz'), ('url', 'http://cssb.biology.gatech.edu/skolnick/files/APoc/apoc_v1b16.tar.gz'), ('sha256', 'e66f33312e8a10b82ddc25a6ecdb1d178d9349a10e9be20d33b41ef75556dd51'), ('patches', ['apoc_gfortran.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['apoc -h >/dev/null'])])), ('about', CommentedMap([('home', 'http://cssb.biology.gatech.edu/APoc'), ('license', 'This software is freely available to ALL users.'), ('summary', 'Large-scale structural comparison of protein pockets')]))])), 'arb-bio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arb-bio/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}-bio'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{name | lower }}-<{ version }}-source.tgz'), ('url', 'http://download.arb-home.de/release/<{ name|lower }}-<{ version }}/<{ name|lower }}-<{ version }}-source.tgz'), ('sha256', '<{ sha256 }}'), ('patches', ['disable_pkgs.patch', 'no_mafftlinks.patch', 'link_dyn.patch', 'make.patch', 'make_path_fix.patch', 'abs_path_fix.patch', 'gsed.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkg-config', 'openmotif-dev', 'glib 2.51.*', 'gettext !=0.19.8', 'pthread-stubs', 'xorg-libxp', 'xorg-libxmu', 'xorg-libxi', 'xorg-libxaw', 'xorg-libxpm', 'xorg-makedepend', 'xerces-c', 'libtiff', 'libxslt', 'perl !=5.22.0.1-0,!=5.22.2.1-0', 'lynx', 'time', 'sed >=4.4']), ('run', ['libgcc', 'gettext !=0.19.8', 'xorg-libxi', 'xorg-libxmu', 'xorg-libxp', 'xorg-libxaw', 'xorg-libxpm', 'xorg-libxft', 'glib 2.51.*', 'openmotif', 'gnuplot', 'muscle', 'mafft', 'raxml', 'mrbayes', 'phylip', 'phyml 3.2.0', 'fasttree', 'sed >=4.4', 'xfig', 'fig2dev', 'perl !=5.22.0.1-0,!=5.22.2.1-0'])])), ('test', CommentedMap([('requires', ['conda']), ('commands', ['if command -v conda; then conda inspect linkages -p $PREFIX $PKG_NAME; fi', 'if command -v conda; then conda inspect objects -p $PREFIX $PKG_NAME; fi', 'which arb', 'which arb_pt_server'])])), ('about', CommentedMap([('home', 'http://www.arb-home.de'), ('license', 'ARB'), ('licence_file', 'arb_LICENSE.txt'), ('summary', 'ARB 6 Sequence Analysis Suite')]))])), 'arem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arem'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'AREM-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/bc/0d/6db36d917d821d150f37c68839d8911e52b78c867b27372da5be41d4cc4f/AREM-1.0.1.tar.gz'), ('md5', 'fab4d78f1fd12c691d4bd62140b927ea')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['AREM', 'AREM.IO'])])), ('about', CommentedMap([('home', 'http://cbcl.ics.uci.edu/AREM'), ('license', 'Artistic License'), ('summary', 'Aligning Reads by Expectation-Maximization.\\\\nBased on MACS (Model Based Analysis for ChIP-Seq data)'), ('license_family', 'OTHER')]))])), 'argparse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/argparse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'argparse'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'argparse-1.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/18/dd/e617cfc3f6210ae183374cd9f6a26b20514bbb5a792af97949c5aacddf0f/argparse-1.4.0.tar.gz'), ('md5', '08062d2ceb6596fcbc5a7e725b53746f')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('about', CommentedMap([('home', 'https://github.com/ThomasWaldmann/argparse/'), ('license', 'Python Software Foundation License'), ('summary', 'Python command-line parsing library'), ('license_family', 'Proprietary')]))])), 'argutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/argutils/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'argutils'), ('version', '0.3.2')])), ('source', CommentedMap([('fn', 'argutils-0.3.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/a/argutils/argutils-0.3.2.tar.gz'), ('md5', '31a4c3446b8074ab8c38a9fa9aaacf13')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'pyyaml']), ('run', ['python', 'six', 'pyyaml'])])), ('test', CommentedMap([('imports', ['argutils'])])), ('about', CommentedMap([('home', 'https://github.com/eclarke/argutils'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('summary', 'Functions to build matched argument parsers and config files')]))])), 'aria2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/aria2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'aria2'), ('version', '1.23.0')])), ('source', CommentedMap([('fn', 'aria2-1.23.0.tar.gz'), ('url', 'https://github.com/aria2/aria2/releases/download/release-1.23.0/aria2-1.23.0.tar.gz'), ('md5', '0a28844c8db660d6a20d8a34ae4b0f37')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkg-config', 'zlib', 'openssl', 'sqlite', 'libssh2', 'c-ares', 'libxml2']), ('run', ['libgcc', 'zlib', 'openssl', 'sqlite', 'libssh2', 'c-ares', 'libxml2'])])), ('test', CommentedMap([('commands', ['aria2c -v > /dev/null'])])), ('about', CommentedMap([('home', 'https://aria2.github.io/'), ('summary', 'aria2 is a lightweight multi-protocol & multi-source, cross platform download utility operated in command-line. It supports HTTP/HTTPS, FTP, SFTP, BitTorrent and Metalink.'), ('license', 'GPLv2')]))])), 'ariba': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ariba/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ariba'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ariba-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/a/ariba/ariba-<{ version }}.tar.gz'), ('md5', '9d0b336f7adc0d5c82c5f3d1a8358981')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'beautifulsoup4 >=4.1.0', 'dendropy >=4.2.0', 'matplotlib', 'pyfastaq >=3.12.0', 'pysam >=0.9.1', 'pymummer >=0.10.2', 'gcc', 'zlib']), ('run', ['python', 'beautifulsoup4 >=4.1.0', 'dendropy >=4.2.0', 'matplotlib', 'pyfastaq >=3.12.0', 'pysam >=0.9.1', 'pymummer >=0.10.2', 'bowtie2 >=2.3.1', 'cd-hit >=4.6.5', 'samtools >=1.2', 'bcftools >=1.2', 'mummer >=3.23', 'spades >=3.5.0', 'libgcc', 'zlib'])])), ('test', CommentedMap([('imports', ['ariba', 'ariba.tasks']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/ariba'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'ARIBA: Antibiotic Resistance Identification By Assembly')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'art': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/art/3.11.14/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.niehs.nih.gov/research/resources/software/biostatistics/art/'), ('license', 'GPLv2'), ('summary', 'Illumina, 454 and Solid read simulator')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'art'), ('version', '3.11.14')])), ('requirements', CommentedMap([('build', ['gsl ==1.16']), ('run', ['gsl ==1.16'])])), ('source', CommentedMap([('fn', 'artsrcvanillaicecream031114linuxtgz.tgz'), ('sha256', '31f172ac5953ec32e4ae7787b2aa25e62d55b086647ac1e3e6298bbe39e89645'), ('url', 'http://www.niehs.nih.gov/research/resources/assets/docs/artsrcvanillaicecream031114linuxtgz.tgz')])), ('test', CommentedMap([('commands', ['art_454'])]))])), 'arvados-cli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arvados-cli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arvados-cli'), ('version', '0.1.20151207150126')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'arvados-cli-0.1.20151207150126.gem'), ('url', 'https://rubygems.org/downloads/arvados-cli-0.1.20151207150126.gem')])), ('requirements', CommentedMap([('build', ['ruby', 'curl']), ('run', ['ruby', 'curl'])])), ('test', CommentedMap([('commands', [\"arv --help 2>&1 | grep 'ARVADOS'\"])])), ('about', CommentedMap([('home', 'http://doc.arvados.org/sdk/cli/index.html'), ('license', 'Apache v2'), ('summary', 'Command line interface to Arvados, a free and open source platform for big data science')]))])), 'arvados-cwl-runner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arvados-cwl-runner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arvados-cwl-runner'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'arvados-cwl-runner-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/a/arvados-cwl-runner/arvados-cwl-runner-<{ version }}.tar.gz'), ('md5', '47549b214df3cfcb8fc576e7ca6d712b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ruamel.yaml >=0.13.7,<0.15', 'cwltool ==1.0.20170828135420', 'arvados-python-client >=0.1.20170818194607']), ('run', ['python', 'ruamel.yaml >=0.13.7,<0.15', 'cwltool ==1.0.20170828135420', 'arvados-python-client >=0.1.20170818194607'])])), ('test', CommentedMap([('imports', ['arvados_cwl'])])), ('about', CommentedMap([('home', 'https://github.com/curoverse/arvados/tree/master/sdk/cwl'), ('license', 'Apache 2.0'), ('summary', 'Arvados Common Workflow Language runner')]))])), 'arvados-python-client': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arvados-python-client/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arvados-python-client'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'arvados-python-client-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/a/arvados-python-client/arvados-python-client-<{ version }}.tar.gz'), ('md5', '3d2e18595760e35e52bfc3bd8a5388bd')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ciso8601', 'future', 'google-api-python-client <1.7,>=1.6.2', 'httplib2', 'oauth2client <2,>=1.4.6', 'pycurl <7.21.5,>=7.19.5.1', 'python-gflags', 'requests', 'rsa', 'urllib3 >1.12', 'ws4py']), ('run', ['python', 'ciso8601', 'future', 'google-api-python-client <1.7,>=1.6.2', 'httplib2', 'oauth2client <2,>=1.4.6', 'pycurl <7.21.5,>=7.19.5.1', 'python-gflags', 'requests', 'rsa', 'urllib3 >1.12', 'ws4py'])])), ('test', CommentedMap([('imports', ['arvados'])])), ('about', CommentedMap([('home', 'https://github.com/curoverse/arvados/tree/master/sdk/python'), ('license', 'Apache License 2.0'), ('summary', 'Python API for Arvados, an open source platform for managing and analyzing biomedical big data')])), ('extra', CommentedMap([('skip-lints', ['uses_setuptools'])]))])), 'ascat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ascat/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/Crick-CancerGenomics/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', 'True [win]'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils']), ('run', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils'])])), ('test', CommentedMap([('commands', ['$R -e \"library(ASCAT)\"'])])), ('about', CommentedMap([('home', 'https://www.crick.ac.uk/research/a-z-researchers/researchers-v-y/peter-van-loo/software/'), ('license', 'GPL v3'), ('summary', 'ASCAT is a method to derive copy number profiles of tumour cells,\\naccounting for normal cell admixture and tumour aneuploidy (Figure 1).\\nASCAT infers tumour purity (the fraction of tumour cells) and ploidy (the\\namount of DNA per tumour cell, expressed as multiples of haploid genomes)\\nfrom SNP array or massively parallel sequencing data, and calculates\\nwhole-genome allele-specific copy number profiles (the number of copies of\\nboth parental alleles for all SNP loci across the genome).\\n')]))])), 'asciigenome': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/asciigenome/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'asciigenome'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'ASCIIGenome-1.8.0.zip'), ('url', 'https://github.com/dariober/ASCIIGenome/releases/download/v1.8.0/ASCIIGenome-1.8.0.zip'), ('sha256', 'bb6ae616604898045ed7dca6d267c506cff9e78968d149344f79d1444d80bd0f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['ASCIIGenome -h > /dev/null 2>&1 || [[ $? == 1 ]]'])])), ('about', CommentedMap([('home', 'https://github.com/dariober/ASCIIGenome'), ('license', 'MIT'), ('summary', 'Command-line genome browser running from terminal window and solely based on ASCII characters')]))])), 'askocli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/askocli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'askocli'), ('version', '0.4.3')])), ('source', CommentedMap([('fn', 'askocli-0.4.3.tar.gz'), ('url', 'https://pypi.python.org/packages/e3/3a/e281d5400338292995443eb60c847d952b0605a871efb224a22ecc0aea82/askocli-0.4.3.tar.gz'), ('md5', '730e1587650ba41f7d4c2b77ad96a987')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.4.3']), ('run', ['python', 'requests >=2.4.3'])])), ('test', CommentedMap([('imports', ['libaskocli', 'libaskocli.actions']), ('commands', ['askocli -h'])])), ('about', CommentedMap([('home', 'https://github.com/askomics/askocli'), ('license', 'GNU Affero General Public License v3'), ('summary', 'Command line interface for a distant AskOmics'), ('license_family', 'AGPL')]))])), 'assembly-stats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/assembly-stats/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/sanger-pathogens/assembly-stats/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['assembly-stats -v'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/assembly-stats'), ('license', 'GPL-3.0'), ('summary', 'Get assembly statistics from FASTA and FASTQ files')]))])), 'assemblyutility': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/assemblyutility/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/yechengxi/AssemblyUtility/archive/8e476764eb67ebb486e673ecbd8a0fa5d8eae22c.tar.gz'), ('md5', '113518baf23faf4d6af7605373e2d49f')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['AssemblyStatistics | grep \"ProgramFile contigs contigs_filename LenTh cut_off_length\" > /dev/null', 'SelectLongestReads | grep \"ProgramFile sum total_length longest 0 o outfile f fa/fq_file f fa/fq_file\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/yechengxi/AssemblyUtility'), ('summary', 'Tools for DBG2OLC genoome assembler')]))])), 'astalavista': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/astalavista/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'astalavista'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'astalavista-<{ version }}.tgz'), ('url', 'http://sammeth.net/data/public/astalavista-<{ version }}.tgz'), ('sha256', '46c61e00ddf3427f2646046060708ffd05ecc4610509580fdfda53765f6a2eb2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['astalavista --help 2>&1 | grep \"List of available tools\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://sammeth.net/confluence/display/ASTA/Home'), ('license', 'BSD'), ('summary', 'AStalavista is a computer program to extract alternative splicing (AS) events from a given genomic annotation of exon-intron gene coordinates. By comparing all given transcripts, AStalavista detects the variations in their splicing structure and identify all AS events (like exon skipping, alternate donor, etc) by assigning to each of them an AS code.')]))])), 'atac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/atac/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Overview_of_the_ATAC_process'), ('license', 'GPL'), ('summary', 'ATAC is a computational process for comparative mapping between two genome assemblies, or between two different genomes')])), ('source', CommentedMap([('fn', 'ATAC-r2008.tar.bz2'), ('url', 'http://sourceforge.net/projects/kmer/files/ATAC-r2008.tar.bz2'), ('md5', 'e23ef1d41836b29332b70271b7c9eb74')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'perl', 'bzip2']), ('run', ['libgcc', 'perl', 'python'])])), ('test', CommentedMap([('commands', [\"atac.pl 2>&1 | grep 'atac.pl'\"])]))])), 'atropos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/atropos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'atropos'), ('version', '1.1.10')])), ('about', CommentedMap([('home', 'https://atropos.readthedocs.io/'), ('license', 'CC0'), ('summary', 'trim adapters from high-throughput sequencing reads')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/2a/31/af393bbed1a61448deb94c6e615b74d10cf1d6c51e8fa86b927e1cb658c0/atropos-1.1.10.tar.gz'), ('md5', 'd75f93751ccbc86b11986ecce4c2938f')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install'), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython']), ('run', ['python', 'tqdm', 'pysam', 'jinja2'])])), ('test', CommentedMap([('commands', ['atropos | grep \"Atropos version\"'])]))])), 'augustus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/augustus/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://bioinf.uni-greifswald.de/augustus/binaries/old/augustus-3.2.3.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['patches/bam2hints.Makefile.patch', 'patches/filterBam.src.Makefile.patch', 'patches/filterBam.Makefile.patch', 'patches/compileSpliceCands.Makefile.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*', 'libgcc', 'bamtools', 'gsl', 'lp_solve', 'perl', 'perl-module-build', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'bamtools', 'boost <{CONDA_BOOST}}*', 'gsl', 'lp_solve', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'perl-yaml', 'perl-dbi', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('commands', ['augustus --version', 'autoAug.pl --help &>/dev/null', 'bedgraph2wig.pl --help &>/dev/null', 'blat2gbrowse.pl --help 2>&1 | grep \"convert blat file to gbrowse file\" &>/dev/null', 'blat2hints.pl --help', 'cegma2gff.pl --help 2>&1 | grep \"Could not open CEGMA file --help\"', 'checkParamArchive.pl --help 2>&1 | grep \"checkParamArchive.pl param-archive.tar.gz projectDir > utrPossible 2> errors\"', 'cleanDOSfasta.pl --help 2>&1 | grep \"Could not open file --help\"', 'clusterAndSplitGenes.pl --help', 'createAugustusJoblist.pl --help', 'del_from_prfl.pl --help 2>&1 | grep \"Need two arguments\"', 'evalCGP.pl --help', 'exonerate2hints.pl --help', 'echo \"hello\" | exoniphyDb2hints.pl', 'echo \"hello\" | extractTranscriptEnds.pl | grep \">seq1\"', 'filterGenesIn_mRNAname.pl --help', 'filterGenesOut_mRNAname.pl --help', 'filterGenes.pl --help 2>&1 | grep \"usage:filterGenes.pl namefile dbfile\"', 'filterInFrameStopCodons.pl --help 2>&1 | grep \"Could not open protein file\"', 'filterMaf.pl --help 2>&1 | grep \"filter blocks from a MAF alignment that intersect with a given genomic interval\"', 'echo \"hello\" | filter-ppx.pl | grep \"hello\"', 'filterPSL.pl --help', 'filterShrimp.pl --help', 'filterSpliceHints.pl --help', 'gbrowseold2gff3.pl --help 2>&1 | grep \"gbrowseold2gff3.pl format convert a GFF file\"', 'gbSmallDNA2gff.pl --help 2>&1 | grep \"Could not open genbank input file --help\"', 'getAnnoFasta.pl --help 2>&1 | grep \"Makes a fasta file with protein sequences (augustus.aa)\"', 'gff2gbSmallDNA.pl --help 2>&1 | grep \"convert GFF file and sequence fasta file to minimal genbank format\"', 'gffGetmRNA.pl --help', 'gp2othergp.pl --help 2>&1 | grep \"t open --help\"', 'gtf2gff.pl --help 2>&1 | grep \"gtf2gff.pl format convert a GTF file\"', 'hal2maf_split.pl --help 2>&1 | grep \"this script works on top of the \"', 'intron2exex.pl --help | grep \"intron2exex.pl make a fasta file with exon-exon sequences\"', 'join_mult_hints.pl --help', 'joinPeptides.pl --help 2>&1 | grep \"joinPeptides.pl in1.fa in2.fa > out.fa\"', 'maf2conswig.pl --help | grep \"maf2conswig.pl convert maf alignment format\"', 'makeMatchLists.pl --help  2>&1 | grep \"t open --help\"', 'makeUtrTrainingSet.pl --help 2>&1 | grep \"./makeUtrTrainingSet.pl -- make a genbank or gff training file with \"', 'maskNregions.pl --help 2>&1 | grep \"could not open file --help\"', 'moveParameters.pl --help', 'msa2prfl.pl --help 2>&1 | grep \"Converts a Multiple Sequence Alignment\"', 'new_species.pl --help', 'optimize_augustus.pl --help', 'partition_gtf2gb.pl --help 2>&1 | grep \"Partition a gtf file into gene clusters\"', 'pasapolyA2hints.pl --help 2>&1 | grep \"PASA-Output file in fasta-format\"', 'peptides2alternatives.pl --help 2>&1 | grep \"peptides2alternatives.pl peptides.aa augustus-genes.aa\"', 'peptides2hints.pl --help 2>&1 | grep \"peptides2hints.pl psl-file gff-file fa-file src pri\"', 'polyA2hints.pl --help 2>&1 | grep \"transcript end at position p gives rise to a tts hint\"', 'randomSplit.pl --help', 'ratewig.pl --help 2>&1 | grep \"ratewig /path/to/refseqfile/ /path/to/wigfile\"', 'rmRedundantHints.pl --help 2>&1 | grep \"rmRedundantHints.pl list1.hints list2.hints > nonredundant.hints\"', 'runAllSim4.pl --help', 'samMap.pl --help', 'scipiogff2gff.pl --help 2>&1 | grep \"convert a gff from scipio to a gff file\"', 'simpleFastaHeaders.pl --help 2>&1 | grep \"in.fa prefix out.fa mapping.txt\"', 'simplifyFastaHeaders.pl --help 2>&1 | grep \"in.fa nameStem out.fa header.map\"', 'splitMfasta.pl --help 2>&1 | grep \"Unknown option\"', 'split_wiggle.pl --help', 'summarizeACGTcontent.pl --help 2>&1 | grep \"Couldn\"', 'transMap2hints.pl --help 2>&1 | grep \"convert transmap alignments to hints file\"', 'uniquePeptides.pl --help 2>&1 | grep \"uniquePeptides.pl in.fa prefix > out.fa\"', 'utrgff2gbrowse.pl --help', 'wigchoose.pl --help', 'writeResultsPage.pl --help 2>&1 | grep \"writeResultsPage ID species-name db-file grails-out www-out AugustusConfigPath AugustusScriptsPath final-flag\"', 'yaml2gff.1.4.pl --help 2>&1 | grep \"< scipio.yaml > scipio.gff\"'])])), ('about', CommentedMap([('home', 'http://bioinf.uni-greifswald.de/augustus/'), ('license', 'MIT'), ('license_family', 'MIT'), ('summary', 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.')]))])), 'autodock-vina': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/autodock-vina/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://vina.scripps.edu/'), ('license', 'Apache License 2.0'), ('summary', 'AutoDock Vina is an open-source program for doing molecular docking')])), ('package', CommentedMap([('name', 'autodock-vina'), ('version', '1.1.2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'autodock_vina_1_1_2_mac.tgz'), ('url', 'http://vina.scripps.edu/download/autodock_vina_1_1_2_mac.tgz'), ('md5', 'b8165555fa13132e0658c36005e0edf3')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['vina --help'])]))])), 'autolog': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/autolog/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'autolog'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'autolog-0.1.3.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/a9/b8/ad4a519d45a80473a32925375b97b398b0583f4ea76fcffba9e4c6f4e72e/autolog-0.1.3.tar.gz'), ('md5', 'f900d1e7582a38b04d69e45b3a7ebcfa')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'forked-path']), ('run', ['python', 'forked-path'])])), ('test', CommentedMap([('imports', ['autolog'])])), ('about', CommentedMap([('home', 'http://noble.gs.washington.edu/~mmh1/software/autolog/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'quick and easy logging setup')]))])), 'avro-python2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/avro-python2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'avro-python2'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'avro-1.8.2.tar.gz'), ('url', 'https://pypi.python.org/packages/eb/27/143f124a7498f841317a92ced877150c5cb8d28a4109ec39666485925d00/avro-1.8.2.tar.gz'), ('md5', '44ec007d432a2f3c35f87eee01f1e9ec')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['avro'])])), ('about', CommentedMap([('home', 'http://avro.apache.org/'), ('license', 'Apache License 2.0'), ('summary', 'Avro is a serialization and RPC framework.')]))])), 'avro-python3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/avro-python3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'avro-python3'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'avro-python3-1.8.2.tar.gz'), ('url', 'https://pypi.python.org/packages/63/b4/7a81640bb69c844c912987007ba36276c29690d6fde2cd41fec16cd24c3b/avro-python3-1.8.2.tar.gz'), ('md5', '05369070c2b4ba80bb110858dc753dbd')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['avro'])])), ('about', CommentedMap([('home', 'http://avro.apache.org/'), ('license', 'Apache License 2.0'), ('summary', 'Avro is a serialization and RPC framework.')]))])), 'axiome': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/axiome/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'axiome'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'axiome-2.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/24/10/f8e819957de18eb18fe5197af72ecad3372a562b59e49de7d2ec70b2fe76/axiome-2.0.4.tar.gz'), ('md5', '428dc40ca56b71c3bbf0310df939ce7f')])), ('build', CommentedMap([('skip', True), ('entry_points', ['axiome=axiome.axiome:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'npyscreen']), ('run', ['python', 'npyscreen'])])), ('test', CommentedMap([('imports', ['axiome']), ('commands', ['axiome --help'])])), ('about', CommentedMap([('home', 'https://github.com/neufeld/AXIOME2'), ('license', 'MIT License'), ('summary', 'AXIOME2: Automation Extension and Integration of Microbial Ecology'), ('license_family', 'MIT')]))])), 'azure-cli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/azure-cli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'azure-cli'), ('version', '0.10.3')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'azure-cli.0.10.3.tar.gz'), ('url', 'https://github.com/Azure/azure-xplat-cli/releases/download/v0.10.3-August2016/azure-cli.0.10.3.tar.gz')])), ('requirements', CommentedMap([('build', ['nodejs']), ('run', ['nodejs'])])), ('test', CommentedMap([('commands', [\"printf 'n\\n' | azure --help\"])])), ('about', CommentedMap([('home', 'https://github.com/azure/azure-xplat-cli'), ('license', 'Apache v2.0'), ('summary', 'Microsoft Azure Cross Platform Command Line')]))])), 'backports.csv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/backports.csv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'backports.csv'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'backports.csv-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/backports.csv/backports.csv-1.0.1.tar.gz'), ('md5', '68349405f94164612371a536badfd2d6')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('about', CommentedMap([('home', 'https://github.com/ryanhiebert/backports.csv'), ('license', 'UNKNOWN'), ('summary', 'Backport of Python 3 csv module')]))])), 'backports.unittest_mock': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/backports.unittest_mock/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'backports.unittest_mock'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'backports.unittest_mock-1.1.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/83/f9/95f046b034d2f54692e97fddaccd3a9731896917ac47f8d3ce0689241c7b/backports.unittest_mock-1.1.1.tar.gz'), ('md5', '6ee907f7d8e35df16a06268e65d28e5f')])), ('build', CommentedMap([('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'setuptools_scm', 'mock']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['backports', 'backports.unittest_mock']), ('requires', ['pytest >=2.8'])])), ('about', CommentedMap([('home', 'https://github.com/jaraco/backports.unittest_mock'), ('license', 'MIT License'), ('summary', 'backports.unittest_mock')]))])), 'baitfisher': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/baitfisher/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/cmayer/BaitFisher-package/archive/v1.0.tar.gz'), ('md5', '02a6ebdd593d930187e5751d3bcb4375')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['BaitFilter --version'])])), ('about', CommentedMap([('home', 'https://github.com/cmayer/BaitFisher-package'), ('license', 'GPL3'), ('summary', 'The BaitFisher-package is a software package for designing hybrid enrichment probes.')]))])), 'bam-readcount': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bam-readcount/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bam-readcount'), ('version', '0.8')])), ('source', CommentedMap([('fn', 'v0.8.0.tar.gz'), ('url', 'https://github.com/genome/bam-readcount/archive/v0.8.0.tar.gz'), ('md5', '58d72588866f16f658ce4f40a3809af9')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['cmake >=2.8.3', 'gcc', 'llvm', 'zlib', 'ncurses', 'perl', 'python']), ('run', ['python', 'libgcc', 'zlib', 'ncurses', 'perl'])])), ('test', CommentedMap([('commands', ['bam-readcount 2>&1 | grep bam-readcount > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/genome/bam-readcount'), ('license', 'MIT'), ('summary', 'bam-readcount generates metrics at single nucleotide positions.')]))])), 'bambamc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bambamc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bambamc'), ('version', '0.0.50')])), ('source', CommentedMap([('fn', '0.0.50-release-20140430085950.tar.gz'), ('url', 'https://github.com/gt1/bambamc/archive/0.0.50-release-20140430085950.tar.gz'), ('md5', '4de1f030fab1de54387f68e4fbd2cd35')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/include/bambamc/BamBam_Config.h', 'test -f ${PREFIX}/lib/libbambamc.so.0.0.50', 'test -f ${PREFIX}/lib/libbambamc.0.dylib', 'test -f ${PREFIX}/lib/libbambamc.a'])])), ('about', CommentedMap([('home', 'https://github.com/gt1/bambamc'), ('license', 'GPLv3'), ('license_file', 'COPYING'), ('summary', 'lightweight C implementation of name collating BAM file input and BAM file output')]))])), 'bamm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamm'), ('version', '1.7.3')])), ('source', CommentedMap([('fn', '1.7.3.tar.gz'), ('url', 'https://github.com/Ecogenomics/BamM/archive/1.7.3.tar.gz'), ('md5', 'c109f98449b041a55d18b65cb3f9297e')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('extra', CommentedMap([('skip-lints', ['setup_py_install_args'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib 1.3.1', 'cython', 'python >=2.7,<3', 'setuptools', 'zlib', 'automake 1.15', 'autoconf', 'libtool', 'nose']), ('run', ['libgcc', 'python >=2.7,<3', 'htslib 1.3.1', 'zlib', 'numpy', 'samtools ==1.3.1', 'bwa ==0.7.12', 'nose'])])), ('test', CommentedMap([('imports', ['bamm']), ('requires', ['nose']), ('commands', ['bamm --help', 'nosetests'])])), ('about', CommentedMap([('home', 'https://github.com/Ecogenomics/BamM'), ('license', 'LGPL-3.0'), ('summary', 'Metagenomics-focused BAM file manipulation')]))])), 'bamtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamtools/2.3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamtools'), ('version', '2.3.0')])), ('source', CommentedMap([('fn', 'bamtools-2.3.0.tar.gz'), ('url', 'https://github.com/pezmaster31/bamtools/archive/v2.3.0.tar.gz'), ('sha256', '288046e6d5d41afdc5fce8608c5641cf2b8e670644587c1315b90bbe92f039af'), ('patches', ['bamtools_lib.diff'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bamtools --help > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/pezmaster31/bamtools'), ('license', 'MIT'), ('summary', 'C++ API & command-line toolkit for working with BAM data')]))])), 'bamutil': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamutil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamutil'), ('version', '1.0.14')])), ('source', CommentedMap([('fn', 'v1.0.14.tar.gz'), ('url', 'https://github.com/statgen/bamUtil/archive/v1.0.14.tar.gz'), ('sha256', 'f5ec8d5e98a3797742106c3413a4ab1622d8787e38b29b3df4cddb59d77efda5')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bam --help 2>&1 | grep Version', 'bam squeeze --help 2>&1 | grep squeeze'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/BamUtil'), ('license', 'GPLv3'), ('summary', 'Programs that perform operations on SAM/BAM files, all built into a single executable, bam.')]))])), 'basic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/basic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'basic'), ('version', '1.0.1')])), ('source', CommentedMap([('url', 'http://ttic.uchicago.edu/~aakhan/BASIC/BASIC.tar.gz'), ('sha256', 'a4d3b90f494ed1bce82ad4d17180e9ba4ef1d6623ec4d988f21470ed61bab123')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'bowtie2'])])), ('test', CommentedMap([('commands', ['BASIC.py -h'])])), ('about', CommentedMap([('home', 'http://ttic.uchicago.edu/~aakhan/BASIC/'), ('license', 'MIT'), ('summary', 'BASIC is a semi-de novo assembly method to determine the full-length sequence of the BCR in single B cells from scRNA-seq data.\\n')]))])), 'bats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bats/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/sstephenson/bats'), ('license', 'Open Source'), ('summary', 'Bash automated testing system')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'bats'), ('version', '0.4.0')])), ('source', CommentedMap([('fn', 'bats-0.4.0.tar.gz'), ('url', 'https://github.com/sstephenson/bats/archive/v0.4.0.tar.gz')])), ('test', CommentedMap([('commands', ['bats --help'])]))])), 'bayescan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bayescan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bayescan'), ('version', '2.0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'BayeScan2.01.zip'), ('url', 'http://cmpg.unibe.ch/software/BayeScan/files/BayeScan2.01.zip')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bayescan2 | grep BayeScan > /dev/null'])])), ('about', CommentedMap([('home', 'http://cmpg.unibe.ch/software/BayeScan/'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))])), 'bbmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bbmap/37.17/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/bbmap'), ('license', 'UC-LBL license (see package)'), ('summary', 'BBMap is a short read aligner, as well as various other bioinformatic tools')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}_<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0']), ('run', ['openjdk >=7.0'])])), ('test', CommentedMap([('commands', ['bbmap.sh -V 2>&1 | grep \"Executing\" > /dev/null'])])), ('extra', CommentedMap([('notes', 'BBMap is a series of Java programs, but they come with a number of custom\\nwrapper shell scripts. Each of these is symlinked to the conda bin directory\\nduring install.\\n')]))])), 'bcbio-nextgen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-nextgen/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-nextgen'), ('version', '1.0.6a0')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('source', CommentedMap([('fn', 'bcbio-nextgen-880a1fa.tar.gz'), ('url', 'https://github.com/chapmanb/bcbio-nextgen/archive/880a1fa.tar.gz'), ('md5', 'b1a05b0ed66620ad9afbbcf5670bf4c5')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pip']), ('run', ['python', 'argparse [py26]', 'arrow', 'azure', 'beautifulsoup4', 'bioblend', 'biopython', 'boto', 'cython', 'cyvcf2', 'dnapi', 'fabric', 'fadapa', 'geneimpacts', 'gffutils', 'h5py', 'htslib', 'ipyparallel >=4.0,<5.0', 'ipython-cluster-helper >=0.5.6', 'joblib', 'libsodium >=0.4,<1.0', 'logbook', 'matplotlib', 'mock', 'msgpack-python', 'numpy', 'openssl', 'pandas', 'path.py', 'patsy', 'pip', 'py', 'pycrypto', 'pytest', 'pytest-cov', 'pytest-mock', 'progressbar', 'psutil', 'python-dateutil', 'pybedtools', 'pysam >=0.11.0', 'pyvcf', 'pyyaml', 'pyzmq', 'requests', 'scipy', 'seaborn', 'seqcluster', 'sqlalchemy', 'statsmodels', 'tabulate', 'toolz', 'tornado', 'yamllint'])])), ('test', CommentedMap([('imports', ['bcbio.distributed.multi', 'bcbio.pipeline.main', 'bcbio.provenance.do']), ('commands', ['LANG=C.UTF-8 bcbio_nextgen.py -h', 'LANG=C.UTF-8 bcbio_setup_genome.py -h', 'LANG=C.UTF-8 bcbio_prepare_samples.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbio-nextgen'), ('license', 'MIT'), ('summary', 'Validated, scalable, community developed variant calling, RNA-seq and small RNA analysis')]))])), 'bcbio-nextgen-vm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-nextgen-vm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-nextgen-vm'), ('version', '0.1.0a')])), ('build', CommentedMap([('number', 106), ('skip', True)])), ('source', CommentedMap([('fn', 'bcbio-nextgen-vm-4346e83.tar.gz'), ('url', 'https://github.com/chapmanb/bcbio-nextgen-vm/archive/4346e83.tar.gz'), ('md5', 'eedf0fcf4df112c92a556ec9a738a429')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'bcbio-nextgen >=1.0.5', 'ipyparallel >=4.0,<5.0', 'pysam >=0.11.0', 'arvados-cwl-runner', 'cwl2wdl', 'ruamel.yaml >=0.13.0', 'toil >=3.11.0', 'rabix-bunny >=1.0.1', 'nodejs', 'elasticluster', 'nose', 'sevenbridges-python', 'synapseclient', 'dxpy', 'six'])])), ('test', CommentedMap([('imports', ['bcbiovm.version']), ('commands', ['bcbio_vm.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbio-nextgen-vm'), ('license', 'MIT'), ('summary', 'Run bcbio-nextgen genomic sequencing analyses using isolated containers and virtual machines')]))])), 'bcbio-prioritize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-prioritize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-prioritize'), ('version', '0.0.8')])), ('source', CommentedMap([('fn', 'bcbio-prioritize'), ('url', 'https://github.com/chapmanb/bcbio.prioritize/releases/download/v0.0.8/bcbio-prioritize'), ('md5', '1cb61817f74328dd3a0656a64154a5ec')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['bcbio-prioritize version'])])), ('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbio.prioritize'), ('license', 'MIT'), ('summary', 'Prioritize small variants, structural variants and coverage based on biological inputs')]))])), 'bcbio-rnaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-rnaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-rnaseq'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'bcbio-rnaseq'), ('url', 'https://github.com/roryk/bcbio.rnaseq/releases/download/v1.2.0/bcbio-rnaseq'), ('md5', 'e3f150f45026af6d7513b16094bd54e9')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk', 'r-base', 'r-ggplot2', 'r-stringr', 'r-dplyr', 'r-knitr', 'r-pheatmap', 'r-matrixstats', 'bioconductor-deseq2', 'bioconductor-edger', 'r-hexbin', 'bioconductor-vsn', 'bioconductor-degreport', 'bioconductor-biomart', 'bioconductor-tximport', 'r-chbutils', 'r-rmarkdown', 'r-readr', 'r-quantreg', 'bioconductor-clusterprofiler', 'r-sleuth', 'bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db']), ('build', [])])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'https://github.com/roryk/bcbio.rnaseq'), ('license', 'MIT'), ('summary', 'Report generation for bcbio-nextgen RNA-seq runs')]))])), 'bcbio-variation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-variation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-variation'), ('version', '0.2.6')])), ('source', CommentedMap([('fn', 'bcbio.variation-0.2.6-standalone.jar'), ('url', 'https://github.com/chapmanb/bcbio.variation/releases/download/v0.2.6/bcbio.variation-0.2.6-standalone.jar'), ('md5', '92af95325a1fbc4739f43b26a17dbf7f')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['bcbio-variation variant-utils --help'])])), ('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbio.variation'), ('license', 'MIT'), ('summary', 'Toolkit to analyze genomic variation data, built on the GATK with Clojure')]))])), 'bcbio-variation-recall': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio-variation-recall/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio-variation-recall'), ('version', '0.1.7')])), ('source', CommentedMap([('fn', 'bcbio-variation-recall'), ('url', 'https://github.com/chapmanb/bcbio.variation.recall/releases/download/v0.1.7/bcbio-variation-recall'), ('md5', 'b8a02fc75b71aad9a9565252b7700b58')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['bcbio-variation-recall version'])])), ('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbio.variation.recall'), ('license', 'MIT'), ('summary', 'Parallel merging, squaring off and ensemble calling for genomic variants')]))])), 'bcbio_monitor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbio_monitor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbio_monitor'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'bcbio_monitor-1.0.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/bcbio_monitor/bcbio_monitor-1.0.2.tar.gz'), ('md5', '8f84cc4d99cfdf99d8bc78c651a45ed8')])), ('build', CommentedMap([('skip', True), ('entry_points', ['bcbio_monitor = bcbio_monitor:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask 0.10.1', 'gevent 1.0', 'requests', 'py-graphviz', 'paramiko', 'pyyaml']), ('run', ['python', 'flask 0.10.1', 'gevent 1.0', 'requests', 'py-graphviz', 'paramiko', 'pyyaml', 'pytz', 'dateutil'])])), ('test', CommentedMap([('imports', ['bcbio_monitor', 'bcbio_monitor.config', 'bcbio_monitor.graph', 'bcbio_monitor.log', 'bcbio_monitor.parser']), ('commands', ['bcbio_monitor --help'])])), ('about', CommentedMap([('home', 'https://github.com/guillermo-carrasco/bcbio-nextgen-monitor'), ('license', 'MIT'), ('summary', 'bcbio-monitor is an extension of bcbio-nextgen to visualize its progress')]))])), 'bcbiogff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbiogff/0.6.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/chapmanb/bcbb/blob/master/gff'), ('license', 'Biopython License Agreement'), ('summary', 'Read and write Generic Feature Format (GFF) with Biopython integration.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bcbiogff'), ('version', '0.6.2')])), ('requirements', CommentedMap([('build', ['python', 'six']), ('run', ['python', 'six', 'biopython'])])), ('source', CommentedMap([('fn', 'bcbio-gff-0.6.2.tar.gz'), ('sha256', 'c682dc46a90e9fdb124ab5723797a5f71b2e3534542ceff9f6572b64b9814e68'), ('url', 'https://pypi.python.org/packages/source/b/bcbio-gff/bcbio-gff-0.6.2.tar.gz')])), ('test', CommentedMap([('imports', ['BCBio.GFF'])]))])), 'bcbreport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcbreport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcbreport'), ('version', '0.99.29')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/bcbio.coverage'), ('git_tag', '23ecd4ec57e3a1b43e471a8a5400b865484d9bac')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['bcbreport'])])), ('about', CommentedMap([('home', 'https://github.com/lpantano/bcbio.coverage'), ('license', 'MIT License'), ('summary', 'Rmd templates for bcbio-nextgen analysis')]))])), 'bcftools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcftools/1.3.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/bcftools'), ('license', 'MIT'), ('summary', 'BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bcftools'), ('version', '1.3.1')])), ('requirements', CommentedMap([('build', ['zlib ==1.2.8']), ('run', ['zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'bcftools-1.3.1.tar.bz2'), ('sha256', '12c37a4054cbf1980223e2b3a80a7fdb3fd850324a4ba6832e38fdba91f1b924'), ('url', 'https://github.com/samtools/bcftools/releases/download/1.3.1/bcftools-1.3.1.tar.bz2')])), ('test', CommentedMap([('commands', ['bcftools -h > /dev/null', 'bcftools --version', 'bcftools plugin -lv'])]))])), 'bcftools-snvphyl-plugin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcftools-snvphyl-plugin/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/phac-nml/snvphyl-tools'), ('license', 'apache_2.0'), ('summary', 'The SNVPhyl (Single Nucleotide Variant PHYLogenomics) pipeline is a pipeline for identifying Single Nucleotide Variants (SNV) within a collection\\\\ of microbial genomes and constructing a phylogenetic tree. This package is the bcftools C plugin')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'bcftools-snvphyl-plugin'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'bzip2', 'curl', 'xz', 'zlib']), ('run', ['bcftools <{ version }}', 'libgcc', 'bzip2', 'curl', 'xz', 'zlib'])])), ('source', CommentedMap([('fn', 'bcftools-<{ version }}.tar.bz2'), ('url', 'https://github.com/samtools/bcftools/releases/download/<{ version }}/bcftools-<{ version }}.tar.bz2'), ('sha256', '293010736b076cf684d2873928924fcc3d2c231a091084c2ac23a8045c7df982')])), ('test', CommentedMap([('commands', ['bcftools plugin -lv | grep filter_snv_density'])]))])), 'bcl2fastq-nextseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcl2fastq-nextseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bcl2fastq-nextseq'), ('version', '0.1.0')])), ('source', CommentedMap([('git_url', 'https://github.com/brwnj/bcl2fastq.git'), ('git_rev', 'add0bd6669ced224223529d2ae3a8931431f8857')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'pandas', 'matplotlib', 'numpy', 'seaborn']), ('run', ['python', 'click', 'pandas', 'matplotlib', 'numpy', 'seaborn'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 bcl_to_fastq -h &> /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/brwnj/bcl2fastq'), ('license', 'MIT'), ('summary', 'NextSeq .bcl Conversion')]))])), 'bctools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bctools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bctools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'bctools-<{ version }}.tar.gz'), ('url', 'https://github.com/dmaticzka/bctools/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl', 'biopython', 'bedtools', 'pybedtools', 'datamash', 'future'])])), ('test', CommentedMap([('commands', ['convert_bc_to_binary_RY.py --help | grep convert_bc_to_binary_RY > /dev/null', 'coords2clnt.py --help | grep coords2clnt > /dev/null', 'extract_aln_ends.py --help | grep extract_aln_ends > /dev/null', 'extract_bcs.py --help | grep extract_bcs > /dev/null', 'merge_pcr_duplicates.py --help | grep merge_pcr_duplicates > /dev/null', 'remove_tail.py --help | grep remove_tail > /dev/null', 'rm_spurious_events.py --help | grep rm_spurious_events > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/dmaticzka/bctools'), ('license', 'Apache 2.0'), ('summary', 'Tools for handling barcodes in NGS data.')]))])), 'bd2k-python-lib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bd2k-python-lib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bd2k-python-lib'), ('version', '1.14a1.dev37')])), ('source', CommentedMap([('fn', 'bd2k-python-lib-1.14a1.dev37.tar.gz'), ('url', 'https://pypi.python.org/packages/4e/70/ed36173869506cd87c32554aba3ae526189232fd563c6d193a1970ae0f0f/bd2k-python-lib-1.14a1.dev37.tar.gz'), ('md5', '6969b4bf8965713d766cf1ee80a7762b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['bd2k', 'bd2k.util', 'bd2k.util.ec2', 'bd2k.util.ec2.test', 'bd2k.util.test', 'bd2k.util.xml']), ('requires', ['boto', 'lockfile', 'mock', 'pytest'])])), ('about', CommentedMap([('home', 'https://github.com/BD2KGenomics/bd2k-python-lib'), ('license', 'Apache 2.0'), ('summary', 'The BD2K Python module kitchen sink')]))])), 'beagle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'beagle'), ('version', '4.1_21Jan17.6cc.jar')])), ('source', CommentedMap([('fn', 'beagle.21Jan17.6cc.jar'), ('url', 'https://faculty.washington.edu/browning/beagle/beagle.21Jan17.6cc.jar'), ('md5', 'd23fbcef8707b9added46937149c3e76')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"beagle | grep -q 'Browning'\"])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/beagle/beagle.html'), ('license', 'GPLv3'), ('summary', 'Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.')]))])), 'beagle-lib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beagle-lib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'beagle-lib'), ('version', '2.1.2')])), ('source', CommentedMap([('fn', 'beagle_release_2_1_2.tar.gz'), ('url', 'https://github.com/beagle-dev/beagle-lib/archive/beagle_release_2_1_2.tar.gz'), ('md5', '1107614e86f652f8ee45c1c92f2af3d4'), ('patches', ['osx_jni.patch', 'osx_ltdl.patch'])])), ('build', CommentedMap([('number', 7)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'libtool', 'openjdk']), ('run', ['openjdk', 'libgcc', 'libtool'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/lib/libhmsbeagle-jni.so', 'test -f ${PREFIX}/lib/libhmsbeagle-jni.jnilib'])])), ('about', CommentedMap([('home', 'https://github.com/beagle-dev/beagle-lib'), ('license', 'GPLv3+'), ('summary', 'general purpose library for evaluating the likelihood of sequence evolution on trees')]))])), 'beast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beast/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://beast.bio.ed.ac.uk'), ('license', 'GPLv2+'), ('summary', 'BEAST is a cross-platform program for Bayesian analysis of molecular sequences using MCMC')])), ('package', CommentedMap([('name', 'beast'), ('version', '1.8.4')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'BEASTv1.8.4.tgz'), ('url', 'https://github.com/beast-dev/beast-mcmc/releases/download/v1.8.4/BEASTv1.8.4.tgz'), ('md5', 'cb8752340c1f77a22d39ca4fe09687b0')])), ('requirements', CommentedMap([('run', ['openjdk', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['beast -version'])]))])), 'beast2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beast2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'beast2'), ('version', '2.4.5')])), ('source', CommentedMap([('fn', 'BEAST.v2.4.5.Linux.tgz'), ('url', 'https://github.com/CompEvol/beast2/releases/download/v2.4.5/BEAST.v2.4.5.Linux.tgz'), ('md5', 'aa7579069e47a2ef6257071205320665'), ('patches', ['beast_lib.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gettext']), ('run', ['openjdk 8.0* zulu8*', 'beagle-lib', 'fontconfig', 'font-ttf-ubuntu', 'fontconfig', 'freetype', 'xorg-libxtst'])])), ('test', CommentedMap([('commands', ['beast -version', 'beast -beagle_info | grep Flags', 'addonmanager -help', 'appstore -help', 'loganalyser -h', 'logcombiner -help', 'treeannotator -help'])])), ('about', CommentedMap([('home', 'http://www.beast2.org'), ('license', 'LGPLv2.1'), ('license_file', 'LICENSE.txt'), ('summary', 'BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences.')]))])), 'bedops': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedops/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'http://bedops.readthedocs.io'), ('license', 'GPLv2'), ('summary', 'High-performance genomic feature operations.')])), ('package', CommentedMap([('name', 'bedops'), ('version', '2.4.27')])), ('source', CommentedMap([('url', 'https://github.com/bedops/bedops/archive/v2.4.27.tar.gz'), ('fn', 'bedops-2.4.27.tar.gz'), ('sha256', '23a4e1ccae608a80f1cda99e24b7c1a314914adb2ff098753f5aac143746569d')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bedops --help', 'gtf2bed --help'])]))])), 'behst': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/behst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'behst'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'behst-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/hoffmanlab/behst/get/<{ version }}.tar.bz2'), ('sha256', 'f17cd1de03c51528833678e1461acc83d3dbf736ec1900a41e6cd4315d8e91d5')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'pandas', 'bedtools', 'pybedtools', 'r-base', 'r-rcurl', 'r-gprofiler', 'gnu-wget'])])), ('test', CommentedMap([('commands', ['behst -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/hoffmanlab/behst/overview'), ('license', 'GPLv2'), ('summary', \"BEHST reads an input dataset of chromosome regions, and intersects them with the chromatin interactions available in the Hi-C dataset. Of these chromosome regions, BEHST selects those that are presentthe regulatory regions of genes of APPRIS, a dataset of principal isoform annotations. We defined these cis-regulatory regions upon the position of their nearest transcription start site of the APPRIS genes' principal transcripts (obtained through GENCODE), plus an upstream and downstream extension. Afterwards, BEHST takes the genes of the resulting partner loci found in gene regulatory regions, and performs a gene set enrichment analysis on them through g:Profiler. BEHST, finally, outputs the list of the most significant Gene Ontology terms detected by g:Profiler.\")]))])), 'bellmans-gapc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bellmans-gapc/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bibiserv.cebitec.uni-bielefeld.de/gapc'), ('license', 'GPLv3+'), ('summary', \"The Bellman's GAP Compiler (GAP-C) is the novel ADP compiler which translates GAP-L programs into efficient C++ code. It implements several semantic analyses for optimization purposes, error reporting, type checking and automatic table design.\")])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'bellmans-gapc'), ('version', 0.1)])), ('requirements', CommentedMap([('build', ['gcc', 'flex', 'bison', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*'])])), ('source', CommentedMap([('fn', 'GAP-C-git-0.1.tar.gz'), ('sha256', 'a78c39660ebcc7175466c7848f95d33aba277709352a65b6a5c7a2deb5cd10d5'), ('url', 'https://github.com/mmiladi/gapc/archive/git-0.1.tar.gz')])), ('test', CommentedMap([('commands', ['gapc  -h'])]))])), 'berokka': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/berokka/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/tseemann/<{ name|lower }}/archive/1cc46ac511881fafe73a0692afcee7e72d5bee55.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-bioperl', 'blast >=2.3'])])), ('test', CommentedMap([('commands', [\"berokka -h | grep 'canu'\"])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/berokka'), ('license', 'GPL-3.0'), ('summary', 'Trim, circularise and orient long read bacterial genome assemblies.')]))])), 'besst': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/besst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'besst'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'BESST-2.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/0f/d8/e5a4408a6ed452567a160c22222b1a94c08ee498208ca79dec0293bc4ff6/BESST-2.2.3.tar.gz'), ('md5', 'b94e93751a14d9350506dca6203980ed')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9']), ('run', ['python', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9'])])), ('test', CommentedMap([('imports', ['BESST', 'BESST.diploid']), ('commands', ['runBESST -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ksahlin/BESST'), ('license', 'GPLv3'), ('summary', 'Scaffolder for genomic assemblies.')]))])), 'bfc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bfc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bfc'), ('version', 'r181')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'bfc_r181.tar.gz'), ('url', 'https://github.com/lh3/bfc/archive/a73dad248dc56d9d4d22eacbbbc51ac276045168.tar.gz'), ('md5', 'd4bb3b3f995f51017bde170062f676e9')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bfc  2>&1 | grep bfc > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/bfc'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'BFC is a standalone high-performance tool for correcting sequencing errors from Illumina sequencing data.')]))])), 'bgt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bgt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bgt'), ('version', 'r283')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bgt_r283.tar.gz'), ('url', 'https://github.com/lh3/bgt/archive/7c66cc885df74328a191b943140e3db96a3c7de6.tar.gz'), ('md5', '7cff0edf644b05363149a8df1690180b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bgt  2>&1 | grep bgt > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/bgt'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Flexible genotype query among 30,000+ samples whole-genome.')]))])), 'bio-vcf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bio-vcf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bio-vcf'), ('version', '0.9.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'bio-vcf-2063373.tar.gz'), ('url', 'https://github.com/pjotrp/bioruby-vcf/archive/2063373.tar.gz')])), ('requirements', CommentedMap([('build', ['ruby']), ('run', ['ruby'])])), ('test', CommentedMap([('commands', [\"bio-vcf --help | grep 'Usage: bio-vcf'\"])])), ('about', CommentedMap([('home', 'https://github.com/pjotrp/bioruby-vcf'), ('license', 'MIT'), ('summary', 'Smart VCF parser')]))])), 'bio_assembly_refinement': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bio_assembly_refinement/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bio_assembly_refinement'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'bio_assembly_refinement-0.5.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/bio_assembly_refinement/bio_assembly_refinement-0.5.0.tar.gz'), ('md5', '93657bf827b05d033fd33215ace8e197')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfastaq >=3.10.0', 'pymummer']), ('run', ['python', 'pyfastaq >=3.10.0', 'pymummer'])])), ('test', CommentedMap([('imports', ['bio_assembly_refinement']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/nds/bio_assembly_refinement'), ('license', 'GPLv3'), ('summary', 'Assembly refinement tools, mostly useful for (but not limited to) pacbio bacterial assemblies')]))])), 'bio_hansel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bio_hansel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bio_hansel'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/b/bio-hansel/<{ name }}-<{ version }}.tar.gz'), ('<{ hash_type }}', '<{ hash_value }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.12.1', 'pandas >=0.20.1', 'attrs ==17.2.0']), ('run', ['python', 'blast >=2.6', 'jellyfish >=2.0', 'numpy >=1.12.1', 'pandas >=0.20.1', 'attrs ==17.2.0'])])), ('test', CommentedMap([('commands', ['hansel -V'])])), ('about', CommentedMap([('home', 'https://github.com/phac-nml/bio_hansel'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Subtype Salmonella enterica genomes using 33bp k-mer typing schemes.\\\\nIncludes schemes for Heidelberg and Enteritidis subtyping.\\\\nDeveloped by Genevieve Labbe, James Robertson, Peter Kruczkiewicz, Chad R. Laing, Kim Ziebell, Marisa Rankin, Aleisha R. Reimer, Lorelee Tschetter, Gary Van Domselaar, Eduardo N. Taboada, Sadjia Bekal, Kimberley A. MacDonald, Linda Hoang, Linda Chui, Danielle Daignault, Durda Slavic, Frank Pollari, E. Jane Parmley, Philip Mabon, Elissa Giang, Lok Kan Lee, Jonathan Moffat, Joanne MacKinnon, Benjamin M. Hetman, Roger Johnson, John H.E. Nash.\\\\n'), ('license_family', 'GPL3')]))])), 'bioawk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioawk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioawk'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'bioawk-1.0.tar.gz'), ('url', 'https://github.com/lh3/bioawk/archive/v1.0.tar.gz'), ('sha256', '5cbef3f39b085daba45510ff450afcf943cfdfdd483a546c8a509d3075ff51b5')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'bison', 'm4']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['echo \"hw\" | bioawk \\'{print}\\''])])), ('about', CommentedMap([('home', 'https://github.com/lh3/bioawk'), ('license', 'Free software license (https://github.com/lh3/bioawk/blob/master/README.awk#L1)'), ('summary', 'BWK awk modified for biological data')]))])), 'biobambam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biobambam/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'biobambam2-<{ version }}.tar.gz'), ('url', 'https://github.com/gt1/biobambam2/releases/download/2.0.78-release-20170927110023/biobambam2-2.0.78-release-20170927110023-x86_64-etch-linux-gnu.tar.gz'), ('sha256', '<{ sha256hash }}')])), ('build', CommentedMap([('number', 0), ('binary_relocation', False), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['bamsormadup --help', 'bamtofastq --help'])])), ('about', CommentedMap([('home', 'https://github.com/gt1/biobambam2'), ('license', 'GPLv3'), ('summary', 'Tools for early stage alignment file processing')]))])), 'bioblend': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioblend/0.7.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioblend'), ('version', '0.7.0')])), ('source', CommentedMap([('fn', 'bioblend-0.7.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/bioblend/bioblend-0.7.0.tar.gz'), ('md5', '9b91e764de795e81ae7d136218b3b07e')])), ('build', CommentedMap([('skip', False), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'requests-toolbelt', 'boto', 'pyyaml', 'six']), ('run', ['python', 'requests', 'requests-toolbelt', 'boto', 'pyyaml', 'six'])])), ('test', CommentedMap([('imports', ['bioblend', 'bioblend.galaxy'])])), ('about', CommentedMap([('home', 'https://github.com/galaxy/bioblend'), ('license', 'Academic Free License 3.0'), ('summary', 'A Python library for interacting with CloudMan and the Galaxy API')]))])), 'bioconductor-a4core': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4core/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4core'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Core_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Core_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Core_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Core/a4Core_1.24.0_src_all.tar.gz']), ('md5', 'd7f79c350ae0a6175f2ecc9a337ca61f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-glmnet']), ('run', ['bioconductor-biobase', 'r-base', 'r-glmnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Core\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Core.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Core Package')]))])), 'bioconductor-a4preproc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4preproc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4preproc'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Preproc_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Preproc_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Preproc_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Preproc/a4Preproc_1.24.0_src_all.tar.gz']), ('md5', '8de73ad20a28155f34a4636c3f7c6539')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Preproc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Preproc.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Preprocessing Package')]))])), 'bioconductor-absseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-absseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-absseq'), ('version', '1.22.8')])), ('source', CommentedMap([('fn', 'ABSSeq_1.22.8.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ABSSeq_1.22.8.tar.gz', 'https://depot.galaxyproject.org/software/ABSSeq/ABSSeq_1.22.8_src_all.tar.gz']), ('md5', 'bfdb1800f2e7c60dfa7f6b43026ec8f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ABSSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ABSSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Inferring differential expression genes by absolute counts difference between two groups, utilizing Negative binomial distribution and moderating fold-change according to heterogeneity of dispersion across expression level.')]))])), 'bioconductor-acde': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acde'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'acde_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/acde_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/acde/acde_1.6.0_src_all.tar.gz']), ('md5', 'e92ce91f75bab3bb1d79995bec1b42cc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-boot >=1.3']), ('run', ['r-base', 'r-boot >=1.3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'acde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/acde.html'), ('license', 'GPL-3'), ('summary', \"This package provides a multivariate inferential analysis method for detecting differentially expressed genes in gene expression data. It uses artificial components, close to the data's principal components but with an exact interpretation in terms of differential genetic expression, to identify differentially expressed genes while controlling the false discovery rate (FDR). The methods on this package are described in the vignette or in the article 'Multivariate Method for Inferential Identification of Differentially Expressed Genes in Gene Expression Experiments' by J. P. Acosta, L. Lopez-Kleine and S. Restrepo (2015, pending publication).\")]))])), 'bioconductor-acgh': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acgh/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acgh'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'aCGH_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/aCGH_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/aCGH_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/aCGH/aCGH_1.54.0_src_all.tar.gz']), ('md5', '8e5bd1800b2760e46fcab4179c2e920c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aCGH\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/aCGH.html'), ('license', 'GPL-2'), ('summary', 'Functions for reading aCGH data from image analysis output files and clone information files, creation of aCGH S3 objects for storing these data. Basic methods for accessing/replacing, subsetting, printing and plotting aCGH objects.')]))])), 'bioconductor-acme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acme'), ('version', '2.32.0')])), ('source', CommentedMap([('fn', 'ACME_2.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ACME_2.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/ACME_2.32.0.tar.gz', 'https://depot.galaxyproject.org/software/ACME/ACME_2.32.0_src_all.tar.gz']), ('md5', 'f99ea6b94399fd7a10f55ac7e7ec04fa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ACME\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ACME.html'), ('license', 'GPL (>= 2)'), ('summary', 'ACME (Algorithms for Calculating Microarray Enrichment) is a set of tools for analysing tiling array ChIP/chip, DNAse hypersensitivity, or other experiments that result in regions of the genome showing \"enrichment\".  It does not rely on a specific array technology (although the array should be a \"tiling\" array), is very general (can be applied in experiments resulting in regions of enrichment), and is very insensitive to array noise or normalization methods.  It is also very fast and can be applied on whole-genome tiling array experiments quite easily with enough memory.')]))])), 'bioconductor-adsplit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-adsplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-adsplit'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'adSplit_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/adSplit_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/adSplit/adSplit_1.46.0_src_all.tar.gz']), ('md5', '7638432c734c1fe458acbb9b29384f57')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'adSplit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/adSplit.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements clustering of microarray gene expression profiles according to functional annotations. For each term genes are annotated to, splits into two subclasses are computed and a significance of the supporting gene set is determined.')]))])), 'bioconductor-affxparser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affxparser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affxparser'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'affxparser_1.48.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/affxparser_1.48.0.tar.gz'), ('md5', '20ae3f61e3ea25c3baeabf949b1f1165')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affxparser\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affxparser.html'), ('license', 'LGPL (>= 2)'), ('summary', \"Package for parsing Affymetrix files (CDF, CEL, CHP, BPMAP, BAR).  It provides methods for fast and memory efficient parsing of Affymetrix files using the Affymetrix' Fusion SDK.  Both ASCII- and binary-based files are supported.  Currently, there are methods for reading chip definition file (CDF) and a cell intensity file (CEL).  These files can be read either in full or in part.  For example, probe signals from a few probesets can be extracted very quickly from a set of CEL files into a convenient list structure.\")]))])), 'bioconductor-affy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affy'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affy_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affy_1.50.0.tar.gz'), ('md5', 'aed20ee9653de3ef15868b7e6fd901de')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affy.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"The package contains functions for exploratory oligonucleotide array analysis. The dependence on tkWidgets only concerns few convenience functions. 'affy' is fully functional without it.\")]))])), 'bioconductor-affycomp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affycomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affycomp'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'affycomp_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affycomp_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/affycomp_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/affycomp/affycomp_1.52.0_src_all.tar.gz']), ('md5', 'efa67e67701f2083fadbed99bf7d60b9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.3.3', 'r-base']), ('run', ['bioconductor-biobase >=2.3.3', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affycomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affycomp.html'), ('license', 'GPL (>= 2)'), ('summary', 'The package contains functions that can be used to compare expression measures for Affymetrix Oligonucleotide Arrays.')]))])), 'bioconductor-affydata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affydata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affydata'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'affydata_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/release/data/experiment/src/contrib/affydata_1.24.0.tar.gz']), ('md5', '0b6938685c450a56d65dd5628ebed42d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'r-base']), ('run', ['bioconductor-affy', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affydata\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/affydata.html'), ('license', 'GPL-2'), ('summary', 'Example datasets of a slightly large size. They represent real world examples, unlike the artificial examples included in the package affy')]))])), 'bioconductor-affyexpress': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affyexpress/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affyexpress'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'AffyExpress_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AffyExpress_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/AffyExpress_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/AffyExpress/AffyExpress_1.42.0_src_all.tar.gz']), ('md5', '329507ed3f2ee6b7f0b26169ce9bb7d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-affy >=1.23.4', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AffyExpress\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AffyExpress.html'), ('license', 'LGPL'), ('summary', 'The purpose of this package is to provide a comprehensive and easy-to-use tool for quality assessment and to identify differentially expressed genes in the Affymetrix gene expression data.')]))])), 'bioconductor-affyio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affyio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affyio'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'affyio_1.42.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affyio_1.42.0.tar.gz'), ('md5', 'aa52c3eacb3401eaf7bdaefea62e6c06')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affyio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affyio.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Routines for parsing Affymetrix data files based upon file format information. Primary focus is on accessing the CEL and CDF file formats.')]))])), 'bioconductor-affypdnn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affypdnn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affypdnn'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affypdnn_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affypdnn_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/affypdnn_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/affypdnn/affypdnn_1.50.0_src_all.tar.gz']), ('md5', 'b041e6646942df5d9769a8c051fe075d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.5', 'r-base']), ('run', ['bioconductor-affy >=1.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affypdnn\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affypdnn.html'), ('license', 'LGPL'), ('summary', 'The package contains functions to perform the PDNN method described by Li Zhang et al.')]))])), 'bioconductor-agdex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-agdex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-agdex'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'AGDEX_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AGDEX_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/AGDEX_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/AGDEX/AGDEX_1.24.0_src_all.tar.gz']), ('md5', '678b3c153050238576764230c00b1bca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AGDEX\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AGDEX.html'), ('license', 'GPL Version 2 or later'), ('summary', 'A tool to evaluate agreement of differential expression for cross-species genomics')]))])), 'bioconductor-aims': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aims/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aims'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'AIMS_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AIMS_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/AIMS_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/AIMS/AIMS_1.8.0_src_all.tar.gz']), ('md5', '9f64fa78cff2e01db4f04ebb84188e51')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AIMS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AIMS.html'), ('license', 'Artistic-2.0'), ('summary', 'This package contains the AIMS implementation. It contains necessary functions to assign the five intrinsic molecular subtypes (Luminal A, Luminal B, Her2-enriched, Basal-like, Normal-like). Assignments could be done on individual samples as well as on dataset of gene expression data.')]))])), 'bioconductor-aldex2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aldex2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aldex2'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'ALDEx2_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ALDEx2_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/ALDEx2_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/ALDEx2/ALDEx2_1.8.0_src_all.tar.gz']), ('md5', '979f4e257eb75248e8059a88674df0c8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ALDEx2\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ALDEx2.html'), ('license', 'file LICENSE'), ('summary', 'A differential abundance analysis for the comparison of two or more conditions. For example, single-organism and meta-RNA-seq high-throughput sequencing assays, or of selected and unselected values from in-vitro sequence selections. Uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected false discovery rate given the biological and sampling variation using the Wilcox rank test or Welches t-test (aldex.ttest) or the glm and Kruskal Wallis tests (aldex.glm). Reports both P and fdr values calculated by the Benjamini Hochberg correction.')]))])), 'bioconductor-all': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-all/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-all'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'ALL_1.18.0.tar.gz'), ('url', 'https://bioconductor.org/packages/release/data/experiment/src/contrib/ALL_1.18.0.tar.gz'), ('md5', 'a24d014e5de6a039db0180d3cb01dd01')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ALL\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/data/experiment/html/ALL.html'), ('license', 'Artistic-2.0'), ('summary', 'Data of T- and B-cell Acute Lymphocytic Leukemia from the Ritz Laboratory at the DFCI (includes Apr 2004 versions)')]))])), 'bioconductor-alpine': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-alpine/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-alpine'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'alpine_1.0.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-alpine/bioconductor-alpine_1.0.0_src_all.tar.gz'), ('md5', 'f70ca32c5336aca7745ee3875bbbf28f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'alpine\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/alpine.html'), ('license', 'GPL (>=2)'), ('summary', 'Fragment sequence bias modeling and correction for RNA-seq transcript abundance estimation.')]))])), 'bioconductor-amountain': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-amountain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-amountain'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'AMOUNTAIN_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AMOUNTAIN_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/AMOUNTAIN_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/AMOUNTAIN/AMOUNTAIN_1.2.0_src_all.tar.gz']), ('md5', 'd6994f3f83620937b4a6110d505403ed')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AMOUNTAIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AMOUNTAIN.html'), ('license', 'GPL (>= 2)'), ('summary', 'A pure data-driven gene network, weighted gene co-expression network (WGCN) could be constructed only from expression profile. Different layers in such networks may represent different time points, multiple conditions or various species. AMOUNTAIN aims to search active modules in multi-layer WGCN using a continuous optimization approach.')]))])), 'bioconductor-annaffy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annaffy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annaffy'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'annaffy_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/annaffy_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/annaffy_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/annaffy/annaffy_1.48.0_src_all.tar.gz']), ('md5', 'c51219a222b377403fe4bc08a6c57e7f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=0.1.15', 'bioconductor-biobase', 'bioconductor-go.db', 'bioconductor-kegg.db', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi >=0.1.15', 'bioconductor-biobase', 'bioconductor-go.db', 'bioconductor-kegg.db', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annaffy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/annaffy.html'), ('license', 'LGPL'), ('summary', 'Functions for handling data from Bioconductor Affymetrix annotation data packages. Produces compact HTML and text reports including experimental data and URL links to many online databases. Allows searching biological metadata using various criteria.')]))])), 'bioconductor-annmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annmap'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'annmap_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/annmap_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/annmap_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/annmap/annmap_1.18.0_src_all.tar.gz']), ('md5', 'd3406e404798142039623bfcf33fa3ae')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-dbi', 'r-digest', 'r-lattice', 'r-rmysql >=0.6-0']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-dbi', 'r-digest', 'r-lattice', 'r-rmysql >=0.6-0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annmap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/annmap.html'), ('license', 'GPL-2'), ('summary', 'annmap provides annotation mappings for Affymetrix exon arrays and coordinate based queries to support deep sequencing data analysis. Database access is hidden behind the API which provides a set of functions such as genesInRange(), geneToExon(), exonDetails(), etc. Functions to plot gene architecture and BAM file data are also provided. Underlying data are from Ensembl.')]))])), 'bioconductor-annotate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotate'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'annotate_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/annotate_1.50.0.tar.gz'), ('md5', '917d5e8134e7137218dcb1fd0f6a73f2')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-dbi', 'r-xml', 'r-xtable']), ('run', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-dbi', 'r-xml', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annotate\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/annotate.html'), ('license', 'Artistic-2.0'), ('summary', 'Using R enviroments for annotation.')]))])), 'bioconductor-annotationdbi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationdbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationdbi'), ('version', '1.38.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-annotationdbi/bioconductor-annotationdbi_1.38.0_src_all.tar.gz'), ('sha256', '47bc40347c3383ffb992bfd31768a6f69860fc02d9b2231073f504a451aacef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationDbi\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides user interface and database connection code for annotation data packages using SQLite data storage.')]))])), 'bioconductor-annotationforge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationforge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationforge'), ('version', '1.14.2')])), ('source', CommentedMap([('fn', 'AnnotationForge_1.14.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/AnnotationForge_1.14.2.tar.gz'), ('md5', '7a4ce57b476f58367b8807d2a7e96c04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml']), ('run', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationForge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationForge.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides code for generating Annotation packages and their databases.  Packages produced are intended to be used with AnnotationDbi.')]))])), 'bioconductor-annotationfuncs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationfuncs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationfuncs'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'AnnotationFuncs_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AnnotationFuncs_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/AnnotationFuncs_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/AnnotationFuncs/AnnotationFuncs_1.26.0_src_all.tar.gz']), ('md5', '6ff063493468ddf5598f370a583f0799')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationFuncs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AnnotationFuncs.html'), ('license', 'GPL-2'), ('summary', 'Functions for handling translating between different identifieres using the Biocore Data Team data-packages (e.g. org.Bt.eg.db).')]))])), 'bioconductor-annotationhub': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationhub/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationhub'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'AnnotationHub_2.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/AnnotationHub_2.6.0.tar.gz'), ('md5', 'b5dc791eaa7a3a7f1b4ee03ffb1c93f1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationHub\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationHub.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a client for the Bioconductor AnnotationHub web resource. The AnnotationHub web resource provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.')]))])), 'bioconductor-annotationtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationtools'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'annotationTools_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/annotationTools_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/annotationTools_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/annotationTools/annotationTools_1.50.0_src_all.tar.gz']), ('md5', '203f2a9f550931fd499aa0a360655402')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annotationTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/annotationTools.html'), ('license', 'GPL'), ('summary', 'Functions to annotate microarrays, find orthologs, and integrate heterogeneous gene expression profiles using annotation and other molecular biology information available as flat file database (plain text files).')]))])), 'bioconductor-anota': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-anota/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-anota'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'anota_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/anota_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/anota_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/anota/anota_1.24.0_src_all.tar.gz']), ('md5', '76b4cfb1482f0b3f84ec08389bce1599')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'anota\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/anota.html'), ('license', 'GPL-3'), ('summary', 'Genome wide studies of translational control is emerging as a tool to study verious biological conditions. The output from such analysis is both the mRNA level (e.g. cytosolic mRNA level) and the levl of mRNA actively involved in translation (the actively translating mRNA level) for each mRNA. The standard analysis of such data strives towards identifying differential translational between two or more sample classes - i.e. differences in actively translated mRNA levels that are independent of underlying differences in cytosolic mRNA levels. This package allows for such analysis using partial variances and the random variance model. As 10s of thousands of mRNAs are analyzed in parallell the library performs a number of tests to assure that the data set is suitable for such analysis.')]))])), 'bioconductor-antiprofiles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-antiprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-antiprofiles'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'antiProfiles_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/antiProfiles_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/antiProfiles_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/antiProfiles/antiProfiles_1.16.0_src_all.tar.gz']), ('md5', '13a0e36ace84d5e293d39766e61429ae')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0']), ('run', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'antiProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/antiProfiles.html'), ('license', 'Artistic-2.0'), ('summary', 'Implements gene expression anti-profiles as described in Corrada Bravo et al., BMC Bioinformatics 2012, 13:272 doi:10.1186/1471-2105-13-272.')]))])), 'bioconductor-aroma.light': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aroma.light/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aroma.light'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'aroma.light_3.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/aroma.light_3.4.0.tar.gz'), ('md5', '3ef387057a6734dc03e7a8f6aced4803')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0']), ('run', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aroma.light\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/aroma.light.html'), ('license', 'GPL (>= 2)'), ('summary', 'Methods for microarray analysis that take basic data types such as matrices and lists of vectors.  These methods can be used standalone, be utilized in other packages, or be wrapped up in higher-level classes.')]))])), 'bioconductor-arrayquality': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-arrayquality/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-arrayquality'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'arrayQuality_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/arrayQuality_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/arrayQuality_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/arrayQuality/arrayQuality_1.54.0_src_all.tar.gz']), ('md5', '56ea80ed13e2924289b84d317d56f6e3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'arrayQuality\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/arrayQuality.html'), ('license', 'LGPL'), ('summary', 'Functions for performing print-run and array level quality assessment.')]))])), 'bioconductor-arraytools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-arraytools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-arraytools'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'ArrayTools_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ArrayTools_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/ArrayTools_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/ArrayTools/ArrayTools_1.36.0_src_all.tar.gz']), ('md5', '3d1865118b5c0aeecd54bd52816fa39f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'bioconductor-limma', 'r-base', 'r-xtable']), ('run', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'bioconductor-limma', 'r-base', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ArrayTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ArrayTools.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"This package is designed to provide solutions for quality assessment and to detect differentially expressed genes for the Affymetrix GeneChips, including both 3' -arrays and gene 1.0-ST arrays. The package generates comprehensive analysis reports in HTML format. Hyperlinks on the report page will lead to a series of QC plots, processed data, and differentially expressed gene lists. Differentially expressed genes are reported in tabular format with annotations hyperlinked to online biological databases.\")]))])), 'bioconductor-asafe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asafe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asafe'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'ASAFE_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASAFE_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASAFE_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/ASAFE/ASAFE_1.2.0_src_all.tar.gz']), ('md5', '7d710be786263956d727cee14f71d7f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASAFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASAFE.html'), ('license', 'Artistic-2.0'), ('summary', \"Given admixed individuals' bi-allelic SNP genotypes and ancestry pairs (where each ancestry can take one of three values) for multiple SNPs, perform an EM algorithm to deal with the fact that SNP genotypes are unphased with respect to ancestry pairs, in order to estimate ancestry-specific allele frequencies for all SNPs.\")]))])), 'bioconductor-aseb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aseb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aseb'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'ASEB_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASEB_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASEB_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/ASEB/ASEB_1.20.0_src_all.tar.gz']), ('md5', '24c4f3e438eecc9d9b7904a529f7096f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASEB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASEB.html'), ('license', 'GPL (>= 3)'), ('summary', 'ASEB is an R package to predict lysine sites that can be acetylated by a specific KAT-family.')]))])), 'bioconductor-asgsca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asgsca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asgsca'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ASGSCA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASGSCA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASGSCA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ASGSCA/ASGSCA_1.10.0_src_all.tar.gz']), ('md5', '66e1202d89d90a8e4752c5ddb5fb87fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-matrix']), ('run', ['r-base', 'r-mass', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASGSCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASGSCA.html'), ('license', 'GPL-3'), ('summary', 'The package provides tools to model and test the association between multiple genotypes and multiple traits, taking into account the prior biological knowledge. Genes, and clinical pathways are incorporated in the model as latent variables. The method is based on Generalized Structured Component Analysis (GSCA).')]))])), 'bioconductor-aspli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aspli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aspli'), ('version', '1.2.3')])), ('source', CommentedMap([('fn', 'ASpli_1.2.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASpli_1.2.3.tar.gz', 'https://depot.galaxyproject.org/software/ASpli/ASpli_1.2.3_src_all.tar.gz']), ('md5', '8f896b652087ca75b109763ffa670454')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASpli\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASpli.html'), ('license', 'GPL'), ('summary', 'Integrative pipeline for the analysis of alternative splicing using RNA-seq.')]))])), 'bioconductor-atlasrdf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-atlasrdf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-atlasrdf'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'AtlasRDF_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AtlasRDF_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/AtlasRDF/AtlasRDF_1.12.0_src_all.tar.gz']), ('md5', '3bff6e8da872b7ddf1de9962e81ef7b5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-hash', 'r-sparql']), ('run', ['r-base', 'r-hash', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AtlasRDF\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AtlasRDF.html'), ('license', 'Apache License 2.0'), ('summary', 'Query the Gene Expression Atlas RDF data at the European Bioinformatics Institute using genes, experimental factors (such as disease, cell type, compound treatments), pathways and proteins. Also contains a function to perform an enrichment of your gene list across Experimental Factor Ontology (EFO) using the Atlas background set.')]))])), 'bioconductor-bacon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bacon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bacon'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'bacon_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bacon_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/bacon_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/bacon/bacon_1.4.0_src_all.tar.gz']), ('md5', 'f43327ba3b4e0a8871e047ad6fc9ae88')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2']), ('run', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bacon\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bacon.html'), ('license', 'GPL (>= 2)'), ('summary', 'Bacon can be used to remove inflation and bias often observed in epigenome- and transcriptome-wide association studies. To this end bacon constructs an empirical null distribution using a Gibbs Sampling algorithm by fitting a three-component normal mixture on z-scores.')]))])), 'bioconductor-bader': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bader/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bader'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'BADER_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BADER_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/BADER_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/BADER/BADER_1.14.0_src_all.tar.gz']), ('md5', '53074e51b3e2022946bf2c37f7ded3e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BADER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BADER.html'), ('license', 'GPL-2'), ('summary', 'For RNA sequencing count data, BADER fits a Bayesian hierarchical model. The algorithm returns the posterior probability of differential expression for each gene between two groups A and B. The joint posterior distribution of the variables in the model can be returned in the form of posterior samples, which can be used for further down-stream analyses such as gene set enrichment.')]))])), 'bioconductor-ballgown': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ballgown/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ballgown'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'ballgown_2.2.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/ballgown_2.2.0.tar.gz'), ('md5', '717570395edef570a4606858bafc6edb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ballgown\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ballgown.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.')]))])), 'bioconductor-basespacer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-basespacer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-basespacer'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'BaseSpaceR_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BaseSpaceR_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/BaseSpaceR_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/BaseSpaceR/BaseSpaceR_1.20.0_src_all.tar.gz']), ('md5', 'd61df80804ea7140d51b428076533eaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-rjsonio']), ('run', ['r-base', 'r-rcurl', 'r-rjsonio'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BaseSpaceR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BaseSpaceR.html'), ('license', 'Apache License 2.0'), ('summary', \"A rich R interface to Illumina's BaseSpace cloud computing environment, enabling the fast development of data analysis and visualisation tools.\")]))])), 'bioconductor-basic4cseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-basic4cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-basic4cseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Basic4Cseq_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Basic4Cseq_1.6.0.tar.gz'), ('md5', '0a056c1a35f02918bfd165339fd6c00f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Basic4Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Basic4Cseq.html'), ('license', 'LGPL-3'), ('summary', \"Basic4Cseq is an R/Bioconductor package for basic filtering, analysis and subsequent visualization of 4C-seq data. Virtual fragment libraries can be created for any BSGenome package, and filter functions for both reads and fragments and basic quality controls are included. Fragment data in the vicinity of the experiment's viewpoint can be visualized as a coverage plot based on a running median approach and a multi-scale contact profile.\")]))])), 'bioconductor-bayespeak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayespeak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayespeak'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BayesPeak_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BayesPeak_1.24.0.tar.gz'), ('md5', '821ed8cbee83a5b8dfd7530306d9e3ee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'r-base >=2.14', 'gcc', 'llvm']), ('run', ['bioconductor-iranges', 'r-base >=2.14', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BayesPeak\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BayesPeak.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package is an implementation of the BayesPeak algorithm for peak-calling in ChIP-seq data.')]))])), 'bioconductor-bayseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayseq'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'baySeq_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/baySeq_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/baySeq_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/baySeq/baySeq_2.10.0_src_all.tar.gz']), ('md5', 'cff209dfec913f634109386cb2e3a74c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base']), ('run', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'baySeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/baySeq.html'), ('license', 'GPL-3'), ('summary', \"This package identifies differential expression in high-throughput 'count' data, such as that derived from next-generation sequencing machines, calculating estimated posterior likelihoods of differential expression (or more complex hypotheses) via empirical Bayesian methods.\")]))])), 'bioconductor-bbcanalyzer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bbcanalyzer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bbcanalyzer'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'BBCAnalyzer_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BBCAnalyzer_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/BBCAnalyzer_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/BBCAnalyzer/BBCAnalyzer_1.6.0_src_all.tar.gz']), ('md5', 'd60fce04b70d570259a69326ae7d867c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BBCAnalyzer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BBCAnalyzer.html'), ('license', 'LGPL-3'), ('summary', 'BBCAnalyzer is a package for visualizing the relative or absolute number of bases, deletions and insertions at defined positions in sequence alignment data available as bam files in comparison to the reference bases. Markers for the relative base frequencies, the mean quality of the detected bases, known mutations or polymorphisms and variants called in the data may additionally be included in the plots.')]))])), 'bioconductor-bcrank': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bcrank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bcrank'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'BCRANK_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BCRANK_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/BCRANK_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/BCRANK/BCRANK_1.38.0_src_all.tar.gz']), ('md5', '82e9e56be5f987e71722178ef22c0fcc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base']), ('run', ['bioconductor-biostrings', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BCRANK\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BCRANK.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for de novo prediction of transcription factor binding consensus by heuristic search')]))])), 'bioconductor-beadarraysnp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-beadarraysnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-beadarraysnp'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'beadarraySNP_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/beadarraySNP_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/beadarraySNP_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/beadarraySNP/beadarraySNP_1.42.0_src_all.tar.gz']), ('md5', '54396dead6d9f58a76adabd9d938aaf3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base']), ('run', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'beadarraySNP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/beadarraySNP.html'), ('license', 'GPL-2'), ('summary', 'Importing data from Illumina SNP experiments and performing copy number calculations and reports.')]))])), 'bioconductor-beat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-beat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-beat'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'BEAT_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BEAT_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/BEAT_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/BEAT/BEAT_1.14.0_src_all.tar.gz']), ('md5', '4b5c1d10035152be5c2921602d8392b9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BEAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BEAT.html'), ('license', 'LGPL (>= 3.0)'), ('summary', 'Model-based analysis of single-cell methylation data')]))])), 'bioconductor-beclear': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-beclear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-beclear'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'BEclear_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BEclear_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/BEclear_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/BEclear/BEclear_1.8.0_src_all.tar.gz']), ('md5', '494fb264673dbac5804362e40c942c99')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrix', 'r-snowfall']), ('run', ['r-base', 'r-matrix', 'r-snowfall'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BEclear\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BEclear.html'), ('license', 'GPL-2'), ('summary', 'Provides some functions to detect and correct for batch effects in DNA methylation data. The core function \"BEclear\" is based on latent factor models and can also be used to predict missing values in any other matrix containing real numbers.')]))])), 'bioconductor-bgafun': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgafun/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgafun'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'bgafun_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bgafun_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/bgafun_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/bgafun/bgafun_1.38.0_src_all.tar.gz']), ('md5', 'b8553334229959eed8569fd90827cc8d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr']), ('run', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bgafun\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bgafun.html'), ('license', 'Artistic-2.0'), ('summary', 'A method to identify specifity determining residues in protein families using Between Group Analysis')]))])), 'bioconductor-bgeedb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgeedb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgeedb'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'BgeeDB_2.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BgeeDB_2.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/BgeeDB_2.2.0.tar.gz', 'https://depot.galaxyproject.org/software/BgeeDB/BgeeDB_2.2.0_src_all.tar.gz']), ('md5', '40e0c3f4017d1ebe40bb165057d1c81d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BgeeDB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BgeeDB.html'), ('license', 'GPL-2'), ('summary', 'A package for the annotation and gene expression data download from Bgee database, and TopAnat analysis: GO-like enrichment of anatomical terms, mapped to genes by expression patterns.')]))])), 'bioconductor-bgmix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgmix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgmix'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'BGmix_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BGmix_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/BGmix_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/BGmix/BGmix_1.36.0_src_all.tar.gz']), ('md5', 'bdcc2f4d74fcc889d2a7da4392ee230f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-kernsmooth']), ('run', ['r-base', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BGmix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BGmix.html'), ('license', 'GPL-2'), ('summary', 'Fully Bayesian mixture models for differential gene expression')]))])), 'bioconductor-bhc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bhc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bhc'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'BHC_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BHC_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/BHC_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/BHC/BHC_1.28.0_src_all.tar.gz']), ('md5', '8b0542c53177dcd0d4eb0adfce13078f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BHC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BHC.html'), ('license', 'GPL-3'), ('summary', 'The method performs bottom-up hierarchical clustering, using a Dirichlet Process (infinite mixture) to model uncertainty in the data and Bayesian model selection to decide at each step which clusters to merge.  This avoids several limitations of traditional methods, for example how many clusters there should be and how to choose a principled distance metric.  This implementation accepts multinomial (i.e. discrete, with 2+ categories) or time-series data. This version also includes a randomised algorithm which is more efficient for larger data sets.')]))])), 'bioconductor-bicare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bicare/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bicare'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BicARE_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BicARE_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/BicARE_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/BicARE/BicARE_1.34.0_src_all.tar.gz']), ('md5', '4cad470020ab4fe967673def474010b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BicARE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BicARE.html'), ('license', 'GPL-2'), ('summary', 'Biclustering Analysis and Results Exploration')]))])), 'bioconductor-biobase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biobase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biobase'), ('version', '2.34.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-biobase/bioconductor-biobase_2.34.0_src_all.tar.gz'), ('sha256', 'f4ab46c3a02e2c9e9f70b99d751c8749a5b528a53e5117c3290e698a7992494b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.3.2', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.3.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Biobase\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/Biobase.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions that are needed by many other packages or which replace R functions.')]))])), 'bioconductor-biocgenerics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocgenerics'), ('version', '0.22.0')])), ('source', CommentedMap([('fn', 'BiocGenerics_0.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocGenerics_0.22.0.tar.gz'), ('md5', '8de9aa737dfca9b53404a1b32169b007')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by many Bioconductor packages.')]))])), 'bioconductor-biocinstaller': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocinstaller/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocinstaller'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BiocInstaller_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocInstaller_1.24.0.tar.gz'), ('sha256', '74d89e27db12d216cd22d99e12372f3e1ffbad5eb71ab6f446ab7190cbcb7943')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocInstaller\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocInstaller.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is used to install and update Bioconductor, CRAN, and (some) github packages.')]))])), 'bioconductor-biocparallel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocparallel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocparallel'), ('version', '1.6.6')])), ('source', CommentedMap([('fn', 'BiocParallel_1.6.6.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/BiocParallel_1.6.6.tar.gz', 'https://depot.galaxyproject.org/software/BiocParallel/BiocParallel_1.6.6_src_all.tar.gz']), ('md5', '445ec78a20e348d678d2e7b8dc5b4354')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-futile.logger', 'r-snow']), ('run', ['r-base', 'r-futile.logger', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocParallel\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiocParallel.html'), ('license', 'GPL-2 | GPL-3'), ('summary', 'This package provides modified versions and novel implementation of functions for parallel evaluation, tailored to use with Bioconductor objects.')]))])), 'bioconductor-biocstyle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocstyle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocstyle'), ('version', '2.0.3')])), ('source', CommentedMap([('fn', 'BiocStyle_2.0.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/BiocStyle_2.0.3.tar.gz', 'https://depot.galaxyproject.org/software/BiocStyle/BiocStyle_2.0.3_src_all.tar.gz']), ('md5', '426dcf23e33d0d29b10b6d40cae2054e')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocStyle\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiocStyle.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides standard formatting styles for Bioconductor PDF and HTML documents. Package vignettes illustrate use and functionality.')]))])), 'bioconductor-biocviews': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocviews/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocviews'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'biocViews_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/biocViews_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/biocViews_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/biocViews/biocViews_1.44.0_src_all.tar.gz']), ('md5', '6e34b966aee2fd4fc808c40a88fa588a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biocViews\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biocViews.html'), ('license', 'Artistic-2.0'), ('summary', \"Infrastructure to support Bioconductor 'views' used to classify software packages. 'biocViews' are directed acyclic graphs of terms from a controlled vocabulary. There are three major classifications, corresponding to 'software', 'annotation', and 'experiment data' packages.\")]))])), 'bioconductor-biocworkflowtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocworkflowtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocworkflowtools'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'BiocWorkflowTools_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocWorkflowTools_1.2.0.tar.gz'), ('md5', '5539398f8427538d0df59e74fb2148a2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocstyle', 'r-base', 'r-bookdown', 'r-httr', 'r-knitr', 'r-rmarkdown', 'r-stringr']), ('run', ['bioconductor-biocstyle', 'r-base', 'r-bookdown', 'r-httr', 'r-knitr', 'r-rmarkdown', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocWorkflowTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Provides functions to ease the transition between Rmarkdown and LaTeX documents when authoring a Bioconductor Workflow.')]))])), 'bioconductor-biomart': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biomart/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biomart'), ('version', '2.30.0')])), ('source', CommentedMap([('fn', 'biomaRt_2.30.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biomaRt_2.30.0.tar.gz'), ('md5', '060c2b56b64e9de6fd725f90a4da9056')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biomaRt\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biomaRt.html'), ('license', 'Artistic-2.0'), ('summary', 'In recent years a wealth of biological data has become available in public data repositories. Easy access to these valuable data resources and firm integration with data analysis is needed for comprehensive bioinformatics data analysis.  biomaRt provides an interface to a growing collection of databases implementing the BioMart software suite (http://www.biomart.org). The package enables retrieval of large amounts of data in a uniform way without the need to know the underlying database schemas or write complex SQL queries. Examples of BioMart databases are Ensembl, COSMIC, Uniprot, HGNC, Gramene, Wormbase and dbSNP mapped to Ensembl. These major databases give biomaRt users direct access to a diverse set of data and enable a wide range of powerful online queries from gene annotation to database mining.')]))])), 'bioconductor-biomformat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biomformat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biomformat'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'biomformat_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/biomformat_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/biomformat_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/biomformat/biomformat_1.2.0_src_all.tar.gz']), ('md5', '0a418f48c731375953ff7389b9b6f7a0')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix']), ('run', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biomformat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biomformat.html'), ('license', 'GPL-2'), ('summary', 'This is an R package for interfacing with the BIOM format. This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object (which is more complex than a single table), as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.')]))])), 'bioconductor-bionet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bionet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bionet'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BioNet_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BioNet_1.34.0.tar.gz'), ('md5', '3fd2c44a09b7c3914d553ca75b05c153')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BioNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BioNet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. E.g. p-values obtained from the differential expression of the genes from an Affymetrix array are assigned to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from the p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.')]))])), 'bioconductor-biosigner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biosigner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biosigner'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'biosigner_1.1.10.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biosigner_1.1.10.tar.gz'), ('md5', 'c66f001a78bf92af46568ed81cfdf243')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest']), ('run', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biosigner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biosigner.html'), ('license', 'CeCILL'), ('summary', \"Feature selection is critical in omics data analysis to extract restricted and meaningful molecular signatures from complex and high-dimension data, and to build robust classifiers. This package implements a new method to assess the relevance of the variables for the prediction performances of the classifier. The approach can be run in parallel with the PLS-DA, Random Forest, and SVM binary classifiers. The signatures and the corresponding 'restricted' models are returned, enabling future predictions on new datasets. A Galaxy implementation of the package is available within the Workflow4metabolomics.org online infrastructure for computational metabolomics.\")]))])), 'bioconductor-biostrings': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biostrings/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biostrings'), ('version', '2.42.1')])), ('source', CommentedMap([('fn', 'Biostrings_2.42.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/Biostrings_2.42.1.tar.gz'), ('md5', 'f26d6153248ece91637251dd5a7a1da1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.15.6', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.11.1', 'bioconductor-xvector >=0.11.6', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.15.6', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.11.1', 'bioconductor-xvector >=0.11.6', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Biostrings\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/Biostrings.html'), ('license', 'Artistic-2.0'), ('summary', 'Memory efficient string containers, string matching algorithms, and other utilities, for fast manipulation of large biological sequences or sets of sequences.')]))])), 'bioconductor-biovizbase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biovizbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biovizbase'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'biovizBase_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biovizBase_1.20.0.tar.gz'), ('md5', '6d646b803e5aa620b4758e5f488fb6e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biovizBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biovizBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.')]))])), 'bioconductor-biseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'BiSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BiSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/BiSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/BiSeq/BiSeq_1.16.0_src_all.tar.gz']), ('md5', 'c0579533cd705c74f9e8dc513996c713')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiSeq.html'), ('license', 'LGPL-3'), ('summary', 'The BiSeq package provides useful classes and functions to handle and analyze targeted bisulfite sequencing (BS) data such as reduced-representation bisulfite sequencing (RRBS) data. In particular, it implements an algorithm to detect differentially methylated regions (DMRs). The package takes already aligned BS data from one or multiple samples.')]))])), 'bioconductor-bsgenome': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'BSgenome_1.42.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-bsgenome/bioconductor-bsgenome_1.42.0_src_all.tar.gz'), ('md5', 'ee6db342fe767fc8092ce88c0d5d15cc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-biostrings >=2.35.3', 'bioconductor-genomeinfodb >=1.3.19', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.1.33', 'bioconductor-rsamtools', 'bioconductor-rtracklayer >=1.25.8', 'bioconductor-s4vectors >=0.9.36', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-biostrings >=2.35.3', 'bioconductor-genomeinfodb >=1.3.19', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.1.33', 'bioconductor-rsamtools', 'bioconductor-rtracklayer >=1.25.8', 'bioconductor-s4vectors >=0.9.36', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BSgenome.html'), ('license', 'Artistic-2.0'), ('summary', 'Infrastructure shared by all the Biostrings-based genome data packages')]))])), 'bioconductor-bsgenome.celegans.ucsc.ce10': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.celegans.ucsc.ce10/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.celegans.ucsc.ce10'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Celegans.UCSC.ce10_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Celegans.UCSC.ce10_1.4.0.tar.gz'), ('md5', '98a8ca836d6db23e46674552669d7942')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Celegans.UCSC.ce10\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Celegans.UCSC.ce10.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Caenorhabditis elegans (Worm) as provided by UCSC (ce10, Oct. 2010) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.celegans.ucsc.ce6': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.celegans.ucsc.ce6/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.celegans.ucsc.ce6'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Celegans.UCSC.ce6_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Celegans.UCSC.ce6_1.4.0.tar.gz'), ('md5', 'cb86ff861d8f660c2abd8fc1907d84a6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Celegans.UCSC.ce6\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Celegans.UCSC.ce6.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Caenorhabditis elegans (Worm) as provided by UCSC (ce6, May 2008) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.dmelanogaster.ucsc.dm3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.dmelanogaster.ucsc.dm3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.dmelanogaster.ucsc.dm3'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Dmelanogaster.UCSC.dm3_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Dmelanogaster.UCSC.dm3_1.4.0.tar.gz'), ('md5', 'b7ceebf7bfee766596f602f9e808d069')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Dmelanogaster.UCSC.dm3\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Dmelanogaster.UCSC.dm3.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Drosophila melanogaster (Fly) as provided by UCSC (dm3, Apr. 2006) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.ecoli.ncbi.20080805': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.ecoli.ncbi.20080805/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.ecoli.ncbi.20080805'), ('version', '1.3.1000')])), ('source', CommentedMap([('fn', 'BSgenome.Ecoli.NCBI.20080805_1.3.1000.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Ecoli.NCBI.20080805_1.3.1000.tar.gz'), ('md5', 'c653e9cbee3faeb6fd5759b7575f234d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Ecoli.NCBI.20080805\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Ecoli.NCBI.20080805.html'), ('license', 'Artistic-2.0'), ('summary', 'Escherichia coli full genomes for several strains as provided by NCBI on 2008/08/05 and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.hsapiens.ucsc.hg19': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.hsapiens.ucsc.hg19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Hsapiens.UCSC.hg19_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Hsapiens.UCSC.hg19_1.4.0.tar.gz'), ('md5', '672a988b28d8602afb2bd5595db7303b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Hsapiens.UCSC.hg19\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg19.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Homo sapiens (Human) as provided by UCSC (hg19, Feb. 2009) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.hsapiens.ucsc.hg38': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.hsapiens.ucsc.hg38/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.hsapiens.ucsc.hg38'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'BSgenome.Hsapiens.UCSC.hg38_1.4.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/BSgenome.Hsapiens.UCSC.hg38_1.4.1.tar.gz', 'https://depot.galaxyproject.org/software/BSgenome.Hsapiens.UCSC.hg38/BSgenome.Hsapiens.UCSC.hg38_1.4.1_src_all.tar.gz']), ('sha256', '71d7e4557666e2ff76ffc6c637331bfab0c0b59f4963f2b6f5606f12f74580e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.35.16', 'r-base']), ('run', ['bioconductor-bsgenome >=1.35.16', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Hsapiens.UCSC.hg38\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Homo sapiens (Human) as provided by UCSC (hg38, Dec. 2013) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.mmusculus.ucsc.mm10': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.mmusculus.ucsc.mm10/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.mmusculus.ucsc.mm10'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Mmusculus.UCSC.mm10_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Mmusculus.UCSC.mm10_1.4.0.tar.gz'), ('sha256', '6e2eb3bce208cc7766f0e0ef1e14465289d5463b34d94c9a9279256545b5408b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Mmusculus.UCSC.mm10\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Mmusculus.UCSC.mm10.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Mus musculus (Mouse) as provided by UCSC (mm10, Jul. Dec. 2011) and stored in Biostrings objects.')]))])), 'bioconductor-bsgenome.mmusculus.ucsc.mm9': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bsgenome.mmusculus.ucsc.mm9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bsgenome.mmusculus.ucsc.mm9'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'BSgenome.Mmusculus.UCSC.mm9_1.4.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/BSgenome.Mmusculus.UCSC.mm9_1.4.0.tar.gz'), ('md5', '5a30ba2b4481a17e8e6979eb69eabf78')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.33.5', 'r-base']), ('run', ['bioconductor-bsgenome >=1.33.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BSgenome.Mmusculus.UCSC.mm9\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/BSgenome.Mmusculus.UCSC.mm9.html'), ('license', 'Artistic-2.0'), ('summary', 'Full genome sequences for Mus musculus (Mouse) as provided by UCSC (mm9, Jul. 2007) and stored in Biostrings objects.')]))])), 'bioconductor-bubbletree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bubbletree/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bubbletree'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ commit }}.zip'), ('url', 'https://github.com/Bioconductor-mirror/BubbleTree/archive/<{ commit }}.zip'), ('sha256', '36791ce784e7df796599e09900f4bbc171579ae3fca1ba7ef21b0f89e5608ca9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-biocstyle', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-dplyr', 'r-e1071', 'r-ggplot2', 'r-gridextra', 'r-gtable', 'r-gtools', 'r-magrittr', 'r-plyr', 'r-rainbow', 'r-rcolorbrewer', 'r-scales', 'r-writexls']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-biocstyle', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-dplyr', 'r-e1071', 'r-ggplot2', 'r-gridextra', 'r-gtable', 'r-gtools', 'r-magrittr', 'r-plyr', 'r-rainbow', 'r-rcolorbrewer', 'r-scales', 'r-writexls'])])), ('test', CommentedMap([('commands', ['$R -e \"options(rgl.useNULL=TRUE); library(\\'BubbleTree\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BubbleTree.html'), ('license', 'LGPL (>= 3)'), ('summary', 'CNV analysis in groups of tumor samples (Publication Pending).')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'bioconductor-bumphunter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bumphunter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bumphunter'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'bumphunter_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/bumphunter_1.12.0.tar.gz'), ('md5', 'ad8d1badfa41a1bd08d2a736d6e63488')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bumphunter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/bumphunter.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for finding bumps in genomic data')]))])), 'bioconductor-cager': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cager/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cager'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'CAGEr_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAGEr_1.16.0.tar.gz'), ('md5', '7228ea30d29211190ecf8ba9c854ee61')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAGEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAGEr.html'), ('license', 'GPL-3'), ('summary', 'Preprocessing of CAGE sequencing data, identification and normalization of transcription start sites and downstream analysis of transcription start sites clusters (promoters).')]))])), 'bioconductor-calib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-calib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-calib'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'CALIB_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CALIB_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/CALIB_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/CALIB/CALIB_1.42.0_src_all.tar.gz']), ('md5', '505bfa5712ec91529ed491a1ae14fc0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CALIB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CALIB.html'), ('license', 'LGPL'), ('summary', 'This package contains functions for normalizing spotted microarray data, based on a physically motivated calibration model. The model parameters and error distributions are estimated from external control spikes.')]))])), 'bioconductor-camera': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-camera/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-camera'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'CAMERA_1.28.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAMERA_1.28.0.tar.gz'), ('md5', 'd8f8c9bcf40564f440c572ec169f4ec3')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAMERA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAMERA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, EIC correlation based tagging of unknown adducts and fragments')]))])), 'bioconductor-cancermutationanalysis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cancermutationanalysis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cancermutationanalysis'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'CancerMutationAnalysis_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CancerMutationAnalysis_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/CancerMutationAnalysis_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/CancerMutationAnalysis/CancerMutationAnalysis_1.18.0_src_all.tar.gz']), ('md5', '607e9ad684f2a0b260107836197caa1e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CancerMutationAnalysis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CancerMutationAnalysis.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'This package implements gene and gene-set level analysis methods for somatic mutation studies of cancer.  The gene-level methods distinguish between driver genes (which play an active role in tumorigenesis) and passenger genes (which are mutated in tumor samples, but have no role in tumorigenesis) and incorporate a two-stage study design.  The gene-set methods implement a patient-oriented approach, which calculates gene-set scores for each sample, then combines them across samples; a gene-oriented approach which uses the Wilcoxon test is also provided for comparison.')]))])), 'bioconductor-cand': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cand/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cand'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CAnD_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CAnD_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CAnD_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CAnD/CAnD_1.8.0_src_all.tar.gz']), ('md5', '7128428f743b57ee1e02500334a0242b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-reshape']), ('run', ['r-base', 'r-ggplot2', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAnD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CAnD.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions to perform the CAnD test on a set of ancestry proportions. For a particular ancestral subpopulation, a user will supply the estimated ancestry proportion for each sample, and each chromosome or chromosomal segment of interest. A p-value for each chromosome as well as an overall CAnD p-value will be returned for each test. Plotting functions are also available.')]))])), 'bioconductor-cardinal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cardinal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cardinal'), ('version', '1.7.0')])), ('source', CommentedMap([('fn', 'Cardinal_1.7.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-cardinal/bioconductor-cardinal_1.7.0_src_all.tar.gz'), ('sha256', 'eece5d6dcabf347303af12dbf53354315f0511b71e752ba954fa51cf02cd2d9f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-protgenerics', 'r-base', 'r-irlba', 'r-signal', 'r-sp']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-protgenerics', 'r-base', 'r-irlba', 'r-signal', 'r-sp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Cardinal\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/devel/bioc/html/Cardinal.html'), ('license', 'Artistic-2.0'), ('summary', 'Implements statistical & computational tools for analyzing mass spectrometry imaging datasets, including methods for efficient pre-processing, spatial segmentation, and classification.')]))])), 'bioconductor-casper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-casper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-casper'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'casper_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/casper_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/casper_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/casper/casper_2.10.0_src_all.tar.gz']), ('md5', 'fa3ed4940649987086225eafec4412d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'casper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/casper.html'), ('license', 'GPL (>=2)'), ('summary', 'Infer alternative splicing from paired-end RNA-seq data. The model is based on counting paths across exons, rather than pairwise exon connections, and estimates the fragment size and start distributions non-parametrically, which improves estimation precision.')]))])), 'bioconductor-category': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-category/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-category'), ('version', '2.38.0')])), ('source', CommentedMap([('fn', 'Category_2.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/Category_2.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Category_2.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Category/Category_2.38.0_src_all.tar.gz']), ('md5', '4562ae7a6d4d5d52ee883c71ab2a81f8')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-graph', 'bioconductor-gseabase', 'bioconductor-rbgl', 'r-base', 'r-matrix', 'r-rsqlite']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-graph', 'bioconductor-gseabase', 'bioconductor-rbgl', 'r-base', 'r-matrix', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Category\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Category.html'), ('license', 'Artistic-2.0'), ('summary', 'A collection of tools for performing category analysis.')]))])), 'bioconductor-causalr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-causalr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-causalr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CausalR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CausalR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CausalR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CausalR/CausalR_1.8.0_src_all.tar.gz']), ('md5', '7ce1907a1bb84e05edff6e99694ac0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph']), ('run', ['r-base', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CausalR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CausalR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Causal network analysis methods for regulator prediction and network reconstruction from genome scale data.')]))])), 'bioconductor-cellgrowth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellgrowth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellgrowth'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'cellGrowth_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cellGrowth_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/cellGrowth/cellGrowth_1.20.0_src_all.tar.gz']), ('md5', '7d912003d8f1eabd6af944fae4217104')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice', 'r-locfit >=1.5-4']), ('run', ['r-base', 'r-lattice', 'r-locfit >=1.5-4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cellGrowth\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cellGrowth.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides functionalities for the fitting of cell population growth models on experimental OD curves.')]))])), 'bioconductor-cellhts2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellhts2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellhts2'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'cellHTS2_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cellHTS2_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/cellHTS2_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/cellHTS2/cellHTS2_2.40.0_src_all.tar.gz']), ('md5', 'de4a1a4e5d0f38fb15fef4495e72c140')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-category', 'bioconductor-genefilter', 'bioconductor-gseabase', 'bioconductor-prada', 'bioconductor-splots', 'bioconductor-vsn', 'r-base', 'r-hwriter', 'r-locfit', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-category', 'bioconductor-genefilter', 'bioconductor-gseabase', 'bioconductor-prada', 'bioconductor-splots', 'bioconductor-vsn', 'r-base', 'r-hwriter', 'r-locfit', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cellHTS2\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cellHTS2.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides tools for the analysis of high-throughput assays that were performed in microtitre plate formats (including but not limited to 384-well plates). The functionality includes data import and management, normalisation, quality assessment, replicate summarisation and statistical scoring. A webpage that provides a detailed graphical overview over the data and analysis results is produced. In our work, we have applied the package to RNAi screens on fly and human cells, and for screens of yeast libraries. See ?cellHTS2 for a brief introduction.')]))])), 'bioconductor-cellmapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellmapper'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'CellMapper_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CellMapper_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/CellMapper_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/CellMapper/CellMapper_1.2.0_src_all.tar.gz']), ('md5', '318fb0d8cedd6560d880e4a834c56f40')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CellMapper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CellMapper.html'), ('license', 'Artistic-2.0'), ('summary', 'Infers cell type-specific expression based on co-expression similarity with known cell type marker genes. Can make accurate predictions using publicly available expression data, even when a cell type has not been isolated before.')]))])), 'bioconductor-cellnoptr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellnoptr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellnoptr'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'CellNOptR_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CellNOptR_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/CellNOptR_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/CellNOptR/CellNOptR_1.22.0_src_all.tar.gz']), ('md5', 'b98ac4d69b5a3d5dd3bd0d9d445f8646')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-rgraphviz', 'r-base', 'r-ggplot2', 'r-hash', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-rgraphviz', 'r-base', 'r-ggplot2', 'r-hash', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CellNOptR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CellNOptR.html'), ('license', 'GPL-3'), ('summary', 'This package does optimisation of boolean logic networks of signalling pathways based on a previous knowledge network and a set of data upon perturbation of the nodes in the network.')]))])), 'bioconductor-cexor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cexor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cexor'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'CexoR_1.14.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/CexoR_1.14.0.tar.gz'), ('sha256', 'a45fed681bcbe9f6845b6ccd92b312fda8ec221f771861fa6127adb7e6546373')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer']), ('run', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CexoR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CexoR.html'), ('license', 'Artistic-2.0 | GPL-2 + file LICENSE'), ('summary', \"Strand specific peak-pair calling in ChIP-exo replicates. The cumulative Skellam distribution function (package 'skellam') is used to detect significant normalised count differences of opposed sign at each DNA strand (peak-pairs). Irreproducible discovery rate for overlapping peak-pairs across biological replicates is estimated using the package 'idr'.\")]))])), 'bioconductor-cfassay': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cfassay/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cfassay'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'CFAssay_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CFAssay_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/CFAssay_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/CFAssay/CFAssay_1.10.0_src_all.tar.gz']), ('md5', 'cd147e0d36f2f7a511cce1e577592322')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CFAssay\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CFAssay.html'), ('license', 'LGPL'), ('summary', 'The package provides functions for calculation of linear-quadratic cell survival curves and for ANOVA of experimental 2-way designs along with the colony formation assay.')]))])), 'bioconductor-cghbase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghbase'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CGHbase_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHbase_1.32.0.tar.gz'), ('md5', '905e8bac8f42dd1b0eaf22bdb148743a')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHbase.html'), ('license', 'GPL'), ('summary', 'Contains functions and classes that are needed by arrayCGH packages.')]))])), 'bioconductor-cghcall': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghcall/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghcall'), ('version', '2.34.1')])), ('source', CommentedMap([('fn', 'CGHcall_2.34.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHcall_2.34.1.tar.gz'), ('md5', '4e2a7e89d577b984ce8e760d645397e1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHcall\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHcall.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Calls aberrations for array CGH data using a six state mixture model as well as several biological concepts that are ignored by existing algorithms. Visualization of profiles is also provided.')]))])), 'bioconductor-cghmcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghmcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghmcr'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'cghMCR_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cghMCR_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/cghMCR_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/cghMCR/cghMCR_1.34.0_src_all.tar.gz']), ('md5', '6c46dd1d8db2d9bbd8a7481da8d85523')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cghMCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cghMCR.html'), ('license', 'LGPL'), ('summary', 'This package provides functions to identify genomic regions of interests based on segmented copy number data from multiple samples.')]))])), 'bioconductor-cghregions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghregions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghregions'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'CGHregions_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CGHregions_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/CGHregions_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/CGHregions/CGHregions_1.34.0_src_all.tar.gz']), ('md5', '4b2caa536a8bf1f923620f8144dd85e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHregions\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CGHregions.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Dimension Reduction for Array CGH Data with Minimal Information Loss')]))])), 'bioconductor-chicago': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chicago/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chicago'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'Chicago_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Chicago_1.2.0.tar.gz'), ('md5', 'cce9ec0fbb7b589c3862146c77afc260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats']), ('run', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Chicago\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Chicago.html'), ('license', 'Artistic-2.0'), ('summary', 'A pipeline for analysing Capture Hi-C data.')]))])), 'bioconductor-chipcomp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipcomp'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ChIPComp_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ChIPComp_1.0.0.tar.gz'), ('md5', '8f7972f826e866923a2691a3a634b01d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPComp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ChIPComp.html'), ('license', 'GPL'), ('summary', 'ChIPComp detects differentially bound sharp binding sites across multiple conditions considering matching control.')]))])), 'bioconductor-chipqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipqc'), ('version', '1.10.3')])), ('source', CommentedMap([('fn', 'ChIPQC_1.10.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/ChIPQC_1.10.3.tar.gz', 'https://depot.galaxyproject.org/software/ChIPQC/ChIPQC_1.10.3_src_all.tar.gz']), ('sha256', 'd1dd1ce67a24baec546985941cf3f9dabfdbd933932c5d8d9f8bdc663d7c4bd9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/ChIPQC.html'), ('license', 'GPL (>= 3)'), ('summary', 'Quality metrics for ChIPseq data.')]))])), 'bioconductor-chipseeker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipseeker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipseeker'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ChIPseeker_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ChIPseeker_1.10.0.tar.gz'), ('md5', 'f4c1ac00b082e050fc92faa4d21c4b89')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-dose', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'r-base', 'r-dplyr', 'r-ggplot2', 'r-gplots', 'r-gridbase', 'r-gtools', 'r-magrittr', 'r-plotrix', 'r-rcolorbrewer', 'r-upsetr', 'r-boot']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-dose', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'r-base', 'r-dplyr', 'r-ggplot2', 'r-gplots', 'r-gridbase', 'r-gtools', 'r-magrittr', 'r-plotrix', 'r-rcolorbrewer', 'r-upsetr', 'r-boot'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPseeker\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ChIPseeker.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements functions to retrieve the nearest genes around the peak, annotate genomic region of the peak, statstical methods for estimate the significance of overlap among ChIP peak data sets, and incorporate GEO database for user to compare the own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses. Several visualization functions are implemented to summarize the coverage of the peak experiment, average profile and heatmap of peaks binding to TSS regions, genomic annotation, distance to TSS, and overlap of peaks or genes.')]))])), 'bioconductor-chipseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'chipseq_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/chipseq_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/chipseq_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/chipseq/chipseq_1.24.0_src_all.tar.gz']), ('sha256', 'c09252008bb8dc25e3cf6896fff9632a9618f8d485f4ccb58a29e92cd0f5aa84')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'chipseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/chipseq.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for helping process short read data for chipseq experiments')]))])), 'bioconductor-chronos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chronos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chronos'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'CHRONOS_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CHRONOS_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/CHRONOS_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/CHRONOS/CHRONOS_1.4.0_src_all.tar.gz']), ('md5', '5814a1a819c300b14e74fd283dbfde1b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-circlize', 'r-doparallel', 'r-foreach', 'r-openxlsx', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-biomart', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-circlize', 'r-doparallel', 'r-foreach', 'r-openxlsx', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CHRONOS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CHRONOS.html'), ('license', 'GPL-2'), ('summary', 'A package used for efficient unraveling of the inherent dynamic properties of pathways. MicroRNA-mediated subpathway topologies are extracted and evaluated by exploiting the temporal transition and the fold change activity of the linked genes/microRNAs.')]))])), 'bioconductor-clusterprofiler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-clusterprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-clusterprofiler'), ('version', '3.0.5')])), ('source', CommentedMap([('fn', 'clusterProfiler_3.0.5.tar.gz'), ('url', 'https://www.bioconductor.org/packages/3.3/bioc/src/contrib/clusterProfiler_3.0.5.tar.gz'), ('md5', '501b5ff766c9458faffcd3e483d527c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'clusterProfiler\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements methods to analyze and visualize functional profiles (GO and KEGG) of gene and gene clusters.')]))])), 'bioconductor-cn.mops': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cn.mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cn.mops'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'cn.mops_1.18.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.3/bioc/src/contrib/cn.mops_1.18.0.tar.gz'), ('md5', '3b763aedfb1ea1004141194f917432ef')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cn.mops\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cn.mops.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'cn.mops (Copy Number estimation by a Mixture Of PoissonS) is a data processing pipeline for copy number variations and aberrations (CNVs and CNAs) from next generation sequencing (NGS) data. The package supplies functions to convert BAM files into read count matrices or genomic ranges objects, which are the input objects for cn.mops. cn.mops models the depths of coverage across samples at each genomic position. Therefore, it does not suffer from read count biases along chromosomes. Using a Bayesian approach, cn.mops decomposes read variations across samples into integer copy numbers and noise by its mixture components and Poisson distributions, respectively. cn.mops guarantees a low FDR because wrong detections are indicated by high noise and filtered out. cn.mops is very fast and written in C++.')]))])), 'bioconductor-cner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cner'), ('version', '1.10.2')])), ('source', CommentedMap([('fn', 'CNEr_1.10.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/CNEr_1.10.2.tar.gz', 'https://depot.galaxyproject.org/software/CNEr/CNEr_1.10.2_src_all.tar.gz']), ('sha256', '31160b6b3a4711199c315884335080c876cdea21e46c6a6ff689bf0ad48fe28b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4']), ('run', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/CNEr.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Large-scale identification and advanced visualization of sets of conserved noncoding elements.')]))])), 'bioconductor-cntools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cntools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cntools'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CNTools_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CNTools_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/CNTools_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/CNTools/CNTools_1.32.0_src_all.tar.gz']), ('md5', '52790802246df29a86b624aed44eb7c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'r-base']), ('run', ['bioconductor-genefilter', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CNTools.html'), ('license', 'LGPL'), ('summary', 'This package provides tools to convert the output of segmentation analysis using DNAcopy to a matrix structure with overlapping segments as rows and samples as columns so that other computational analyses can be applied to segmented data')]))])), 'bioconductor-complexheatmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-complexheatmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-complexheatmap'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'ComplexHeatmap_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ComplexHeatmap_1.6.0.tar.gz'), ('md5', '326b74a2700c659cef63bb88884bd61f')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer']), ('run', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ComplexHeatmap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html'), ('license', 'GPL (>= 2)'), ('summary', 'Complex heatmaps are efficient to visualize associations between different sources of data sets and reveal potential structures. Here the ComplexHeatmap package provides a highly flexible way to arrange multiple heatmaps and supports self-defined annotation graphics.')]))])), 'bioconductor-consensusclusterplus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-consensusclusterplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-consensusclusterplus'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'ConsensusClusterPlus_1.40.0.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/ConsensusClusterPlus_1.40.0.tar.gz'), ('md5', 'ae9231f43df080d1f37d5c23719051fe')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster']), ('run', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ConsensusClusterPlus\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html'), ('license', 'GPL-2'), ('summary', 'Algorithm for determining cluster count and membership by stability evidence in unsupervised analysis.')]))])), 'bioconductor-copynumber': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-copynumber/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-copynumber'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'copynumber_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/copynumber_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/copynumber_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/copynumber/copynumber_1.16.0_src_all.tar.gz']), ('sha256', '1a33d3766416e077fc12dd2d5c4db3cd256771833305346eca6ac0a55ffd7f93')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'copynumber\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/copynumber.html'), ('license', 'Artistic-2.0'), ('summary', 'Penalized least squares regression is applied to fit piecewise constant curves to copy number data to locate genomic regions of constant copy number. Procedures are available for individual segmentation of each sample, joint segmentation of several samples and joint segmentation of the two data tracks from SNP-arrays. Several plotting functions are available for visualization of the data and the segmentation results.')]))])), 'bioconductor-cqn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cqn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cqn'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'cqn_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cqn_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/cqn_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/cqn/cqn_1.22.0_src_all.tar.gz']), ('md5', '34a908ef7485af2d18948f07ebe9f529')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-preprocesscore', 'r-base', 'r-mclust', 'r-nor1mix', 'r-quantreg']), ('run', ['bioconductor-preprocesscore', 'r-base', 'r-mclust', 'r-nor1mix', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cqn\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cqn.html'), ('license', 'Artistic-2.0'), ('summary', 'A normalization tool for RNA-Seq data, implementing the conditional quantile normalization method.')]))])), 'bioconductor-csaw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-csaw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-csaw'), ('version', '1.6.1')])), ('source', CommentedMap([('fn', 'csaw_1.6.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/csaw_1.6.1.tar.gz'), ('md5', '4a700b2bb3929766ef98be2824f0a431')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-rhtslib', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-rhtslib', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'csaw\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/csaw.html'), ('license', 'GPL-3'), ('summary', 'Detection of differentially bound regions in ChIP-seq data with sliding windows, with methods for normalization and proper FDR control.')]))])), 'bioconductor-cummerbund': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cummerbund/2.8.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cummerbund'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'cummeRbund_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.0/bioc/src/contrib/cummeRbund_2.8.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cummeRbund\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cummeRbund.html'), ('license', 'Artistic-2.0'), ('summary', 'Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.')]))])), 'bioconductor-curatedovariandata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-curatedovariandata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-curatedovariandata'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'curatedOvarianData_1.14.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/experiment/src/contrib/curatedOvarianData_1.14.0.tar.gz'), ('md5', '5338384dfe2f75c93ddffd68eae7e3e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'curatedOvarianData\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/curatedOvarianData.html'), ('license', 'Artistic-2.0'), ('summary', 'The curatedOvarianData package provides data for gene expression analysis in patients with ovarian cancer.')]))])), 'bioconductor-dada2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dada2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dada2'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'dada2-<{ version }}.tar.gz'), ('url', 'https://github.com/benjjneb/dada2/archive/v<{ version }}.tar.gz'), ('sha256', '5a590edd581a37a6b6b6c5f560b3456c0b81d7c319eaf10a76dcb8030277b5f7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dada2\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dada2.html'), ('license', 'LGPL-3'), ('summary', 'The dada2 package provides \"OTU picking\" functionality, but instead of picking OTUs the DADA2 algorithm exactly infers samples sequences. The dada2 pipeline starts from demultiplexed fastq files, and outputs inferred sample sequences and associated abundances after removing substitution and chimeric errors. Taxonomic classification is also available via a native implementation of the RDP classifier method.')]))])), 'bioconductor-deepsnv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deepsnv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deepsnv'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'deepSNV_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/deepSNV_1.20.0.tar.gz'), ('md5', 'a545ff264d5c188ddb770cfe00e9de7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'deepSNV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/deepSNV.html'), ('license', 'GPL-3'), ('summary', 'This package provides provides quantitative variant callers for detecting subclonal mutations in ultra-deep (>=100x coverage) sequencing experiments. The deepSNV algorithm is used for a comparative setup with a control experiment of the same loci and uses a beta-binomial model and a likelihood ratio test to discriminate sequencing errors and subclonal SNVs. The shearwater algorithm computes a Bayes classifier based on a beta-binomial model for variant calling with multiple samples for precisely estimating model parameters such as local error rates and dispersion and prior knowledge, e.g. from variation data bases such as COSMIC.')]))])), 'bioconductor-degreport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degreport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degreport'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'DEGreport_1.8.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/DEGreport_1.8.2.tar.gz', 'https://depot.galaxyproject.org/software/DEGreport/DEGreport_1.8.2_src_all.tar.gz']), ('md5', '4006a779e33f3bb68308ba9f4a7d9f84')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGreport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGreport.html'), ('license', 'GPL (>=2)'), ('summary', 'Creation of a HTML report of differential expression analyses of count data. It integrates some of the code mentioned in DESeq2 and edgeR vignettes, and report a ranked list of genes according to the fold changes mean and variability for each selected gene.')]))])), 'bioconductor-degseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degseq'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'DEGseq_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DEGseq_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/DEGseq_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/DEGseq/DEGseq_1.30.0_src_all.tar.gz']), ('md5', 'f5dbf8f5bbe4de681f37a451f75fd9f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue', 'r-base', 'r-samr']), ('run', ['bioconductor-qvalue', 'r-base', 'r-samr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGseq.html'), ('license', 'LGPL (>=2)'), ('summary', 'DEGseq is an R package to identify differentially expressed genes from RNA-Seq data.')]))])), 'bioconductor-deseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'DESeq_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DESeq_1.24.0.tar.gz'), ('md5', 'e5b94046a8d1a62b20d81cfe720982cb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass']), ('run', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DESeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DESeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution')]))])), 'bioconductor-deseq2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deseq2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deseq2'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'release-3.5.zip'), ('url', 'https://github.com/Bioconductor-mirror/DESeq2/archive/release-3.5.zip'), ('md5', '6c1ee009a51b5b370970ee488ded6c28')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-biocparallel', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-ggplot2', 'r-hmisc', 'r-locfit', 'r-rcpp >=0.11.0', 'r-rcpparmadillo', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-biocparallel', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges', 'bioconductor-genomicfeatures', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment >=1.1.6', 'bioconductor-tximport', 'r-rcpparmadillo', 'r-base', 'r-ggplot2', 'r-hmisc', 'r-locfit', 'r-rcpp >=0.11.0', 'libgcc', 'r-rjson', 'r-hmisc', 'r-getopt', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DESeq2\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/DESeq2.html'), ('license', 'LGPL (>= 3)'), ('summary', 'Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.')]))])), 'bioconductor-dexseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dexseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dexseq'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DEXSeq_<{ version }}.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/DEXSeq_<{ version }}.tar.gz'), ('md5', '37baa66bb10474b43d4ea96594330911')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr', 'python'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEXSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/DEXSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.')]))])), 'bioconductor-dirichletmultinomial': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dirichletmultinomial/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dirichletmultinomial'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'DirichletMultinomial_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DirichletMultinomial_1.16.0.tar.gz'), ('md5', '59f90d37be2dfd1763a6b8a384196ded')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/']), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'blas', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'blas', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DirichletMultinomial\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html'), ('license', 'LGPL-3'), ('summary', 'Dirichlet-multinomial mixture models can be used to describe variability in microbial metagenomic data. This package is an interface to code originally made available by Holmes, Harris, and Quince, 2012, PLoS ONE 7(2): 1-15, as discussed further in the man page for this package, ?DirichletMultinomial.')]))])), 'bioconductor-dnacopy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnacopy/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnacopy'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'DNAcopy_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DNAcopy_1.48.0.tar.gz'), ('sha256', '4094d45c761d2494912f07dc6f8f70f6c37c1831220bebc28404a9d95203fbbf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAcopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DNAcopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the circular binary segmentation (CBS) algorithm to segment DNA copy number data and identify genomic regions with abnormal copy number.')]))])), 'bioconductor-dnashaper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnashaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnashaper'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'DNAshapeR_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DNAshapeR_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/DNAshapeR_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/DNAshapeR/DNAshapeR_1.4.0_src_all.tar.gz']), ('sha256', '946ee0f468f94b4776c74b013b88c3f75091415378f84aebc4864cd569e46e9e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAshapeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DNAshapeR.html'), ('license', 'GPL-2'), ('summary', 'DNAhapeR is an R/BioConductor package for ultra-fast, high-throughput predictions of DNA shape features. The package allows to predict, visualize and encode DNA shape features for statistical learning.')]))])), 'bioconductor-do.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-do.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-do.db'), ('version', 2.9)])), ('source', CommentedMap([('fn', 'DO.db_2.9.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/DO.db_2.9.tar.gz'), ('md5', '63dda6d46d2fe40c52a2e79260a7fb9d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.9.7', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.9.7', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DO.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/DO.db.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of annotation maps describing the entire Disease Ontology assembled using data from DO')]))])), 'bioconductor-doppelgangr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-doppelgangr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-doppelgangr'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'doppelgangR_1.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/doppelgangR_1.4.1.tar.gz'), ('md5', 'f677adff8eee1342a0e2eea5edbef632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'doppelgangR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/doppelgangR.html'), ('license', 'GPL (>=2.0)'), ('summary', 'The main function is doppelgangR(), which takes as minimal input a list of ExpressionSet object, and searches all list pairs for duplicated samples.  The search is based on the genomic data (exprs(eset)), phenotype/clinical data (pData(eset)), and \"smoking guns\" - supposedly unique identifiers found in pData(eset).')]))])), 'bioconductor-dose': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dose/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dose'), ('version', '2.10.7')])), ('source', CommentedMap([('fn', 'DOSE_2.10.7.tar.gz'), ('url', 'https://bioconductor.statistik.tu-dortmund.de/packages/3.3/bioc/src/contrib/DOSE_2.10.7.tar.gz'), ('md5', 'efedb464ed94567791746ecaa8d396aa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-do.db', 'bioconductor-gosemsim', 'bioconductor-qvalue', 'r-base', 'r-ggplot2', 'r-igraph', 'r-plyr', 'r-reshape2', 'r-scales', 'gmp']), ('run', ['bioconductor-annotationdbi', 'bioconductor-do.db', 'bioconductor-gosemsim', 'bioconductor-qvalue', 'r-base', 'r-ggplot2', 'r-igraph', 'r-plyr', 'r-reshape2', 'r-scales', 'gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DOSE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DOSE.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively for measuring semantic similarities among DO terms and gene products. Enrichment analyses including hypergeometric model and gene set enrichment analysis are also implemented for discovering disease associations of high-throughput biological data.')]))])), 'bioconductor-dupradar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dupradar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dupradar'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'dupRadar_1.2.2.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/dupRadar_1.2.2.tar.gz'), ('sha256', '0c6a907efe9035dc6ba386160f9f7fb47b18b39e8a2dcfeecb2743f14ad607d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base']), ('run', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dupRadar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dupRadar.html'), ('license', 'GPL-3'), ('summary', 'Duplication rate quality control for RNA-Seq datasets.')]))])), 'bioconductor-ebarrays': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebarrays/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebarrays'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'EBarrays_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/EBarrays_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/EBarrays_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/EBarrays/EBarrays_2.40.0_src_all.tar.gz']), ('md5', '44c9760016b30cadc193fb9aa8c30089')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice']), ('run', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBarrays\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/EBarrays.html'), ('license', 'GPL (>= 2)'), ('summary', 'EBarrays provides tools for the analysis of replicated/unreplicated microarray data.')]))])), 'bioconductor-ebimage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebimage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebimage'), ('version', '4.13.0')])), ('source', CommentedMap([('fn', 'EBImage_4.13.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/EBImage_4.13.0.tar.gz'), ('md5', '0fbcbf6b8f7051eab79e9f6c98805d40')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.7.1', 'r-base', 'r-abind', 'r-fftwtools', 'r-jpeg', 'r-locfit', 'r-png', 'r-tiff']), ('run', ['bioconductor-biocgenerics >=0.7.1', 'r-base', 'r-abind', 'r-fftwtools', 'r-jpeg', 'r-locfit', 'r-png', 'r-tiff'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBImage\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/EBImage.html'), ('license', 'LGPL'), ('summary', 'EBImage provides general purpose functionality for image processing and analysis. In the context of (high-throughput) microscopy-based cellular assays, EBImage offers tools to segment cells and extract quantitative cellular descriptors. This allows the automation of such tasks using the R programming language and facilitates the use of other tools in the R environment for signal processing, statistical modeling, machine learning and visualization with image data.')]))])), 'bioconductor-ebseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebseq'), ('version', '1.14.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/EBSeq_1.14.0.tar.gz'), ('md5', 'ea44b6345b561cf238845d5690aa69e3')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat']), ('run', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/EBSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Differential Expression analysis at both gene and isoform level using RNA-seq data')]))])), 'bioconductor-edaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edaseq'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'EDASeq_2.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/EDASeq_2.8.0.tar.gz'), ('md5', '07196d42966d048d0873f0659d63f637')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EDASeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/EDASeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Numerical and graphical summaries of RNA-Seq read data. Within-lane normalization procedures to adjust for GC-content effect (or other gene-level effects) on read counts: loess robust local regression, global-scaling, and full-quantile normalization (Risso et al., 2011). Between-lane normalization procedures to adjust for distributional differences between lanes (e.g., sequencing depth): global-scaling and full-quantile normalization (Bullard et al., 2010).')]))])), 'bioconductor-edger': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edger/3.12.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edger'), ('version', '3.12.1')])), ('source', CommentedMap([('fn', 'c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('url', 'https://github.com/Bioconductor-mirror/edgeR/archive/c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('md5', 'a76554192ad964763f2c1040e04b7347')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'edgeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/edgeR.html'), ('license', 'GPL (>=2)'), ('summary', 'Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.')]))])), 'bioconductor-ensembldb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ensembldb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ensembldb'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'ensembldb_1.6.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/ensembldb_1.6.2.tar.gz'), ('sha256', '79b8f50cf2ddb7af0de7479a045f6cc57d763286dcc495bf932bce37e995bbc3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ensembldb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ensembldb.html'), ('license', 'LGPL'), ('summary', 'The package provides functions to create and use transcript centric annotation databases/packages. The annotation for the databases are directly fetched from Ensembl using their Perl API. The functionality and data is similar to that of the TxDb packages from the GenomicFeatures package, but, in addition to retrieve all gene/transcript models and annotations from the database, the ensembldb package provides also a filter framework allowing to retrieve annotations for specific entries like genes encoded on a chromosome region or transcript models of lincRNA genes.')]))])), 'bioconductor-exomecopy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-exomecopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-exomecopy'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'exomeCopy_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/exomeCopy_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/exomeCopy_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/exomeCopy/exomeCopy_1.22.0_src_all.tar.gz']), ('md5', 'd944c56edbb5dec52fa7410e2c3c01b2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'exomeCopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/exomeCopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Detection of copy number variants (CNV) from exome sequencing samples, including unpaired samples.  The package implements a hidden Markov model which uses positional covariates, such as background read depth and GC-content, to simultaneously normalize and segment the samples into regions of constant copy count.')]))])), 'bioconductor-fastseg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-fastseg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-fastseg'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'fastseg_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/fastseg_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/fastseg_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/fastseg/fastseg_1.20.0_src_all.tar.gz']), ('md5', '1840f449546b1d89983e95a6280006c1')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'fastseg\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/fastseg.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"fastseg implements a very fast and efficient segmentation algorithm. It has similar functionality as DNACopy (Olshen and Venkatraman 2004), but is considerably faster and more flexible. fastseg can segment data from DNA microarrays and data from next generation sequencing for example to detect copy number segments. Further it can segment data from RNA microarrays like tiling arrays to identify transcripts. Most generally, it can segment data given as a matrix or as a vector. Various data formats can be used as input to fastseg like expression set objects for microarrays or GRanges for sequencing data. The segmentation criterion of fastseg is based on a statistical test in a Bayesian framework, namely the cyber t-test (Baldi 2001). The speed-up arises from the facts, that sampling is not necessary in for fastseg and that a dynamic programming approach is used for calculation of the segments' first and higher order moments.\")]))])), 'bioconductor-fdb.infiniummethylation.hg19': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-fdb.infiniummethylation.hg19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-fdb.infiniummethylation.hg19'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'FDb.InfiniumMethylation.hg19_2.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/FDb.InfiniumMethylation.hg19_2.2.0.tar.gz', 'https://depot.galaxyproject.org/software/FDb.InfiniumMethylation.hg19/FDb.InfiniumMethylation.hg19_2.2.0_src_all.tar.gz']), ('sha256', '605aa3643588a2f40a942fa760b92662060a0dfedb26b4e4cd6f1a78b703093f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biostrings', 'bioconductor-genomicfeatures >=1.7.22', 'bioconductor-org.hs.eg.db', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biostrings', 'bioconductor-genomicfeatures >=1.7.22', 'bioconductor-org.hs.eg.db', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'FDb.InfiniumMethylation.hg19\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/FDb.InfiniumMethylation.hg19.html'), ('license', 'Artistic-2.0'), ('summary', 'Compiled HumanMethylation27 and HumanMethylation450 annotations')]))])), 'bioconductor-flowai': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowai/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowai'), ('version', '1.2.9')])), ('source', CommentedMap([('fn', 'flowAI_1.2.9.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/flowAI_1.2.9.tar.gz'), ('md5', '1dd66dd6dde1a351bda75a00d6f69a85')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowAI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowAI.html'), ('license', 'GPL'), ('summary', 'The package is able to perform an automatic or interactive quality control on FCS data acquired using flow cytometry instruments. By evaluating three different properties: 1) flow rate, 2) signal acquisition, 3) dynamic range, the quality control enables the detection and removal of anomalies.')]))])), 'bioconductor-flowcl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowcl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowcl'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'flowCL_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowCL_1.14.0.tar.gz'), ('md5', '5d523ab593a8f48d52db0ac70249c946')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql']), ('run', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowCL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowCL.html'), ('license', 'Artistic-2.0'), ('summary', 'Semantic labelling of flow cytometric cell populations.')]))])), 'bioconductor-flowcore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowcore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowcore'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'flowCore_1.42.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowCore_1.42.0.tar.gz'), ('md5', 'a806b81e133f818889df8f16688f2ddd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.22.0', 'bioconductor-graph', 'r-base', 'r-corpcor', 'r-matrixstats', 'r-rcpp', 'r-rrcov', 'r-bh >=1.62.0.1', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.22.0', 'bioconductor-graph', 'r-base', 'r-corpcor', 'r-matrixstats', 'r-rcpp', 'r-rrcov'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowCore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowCore.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides S4 data structures and basic functions to deal with flow cytometry data.')]))])), 'bioconductor-flowdensity': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowdensity/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowdensity'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'flowDensity_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowDensity_1.10.0.tar.gz'), ('md5', 'f1eb297bc6454a4bc4732459a5200a87')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr']), ('run', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowDensity\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowDensity.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides tools for automated sequential gating analogous to the manual gating strategy based on the density of the data.')]))])), 'bioconductor-flowfp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowfp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowfp'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'flowFP_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowFP_1.32.0.tar.gz'), ('md5', '5aae3664847befdfd04c29adfdc19826')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.1.6', 'bioconductor-flowcore', 'bioconductor-flowviz', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.1.6', 'bioconductor-flowcore', 'bioconductor-flowviz', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowFP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowFP.html'), ('license', 'Artistic-2.0'), ('summary', 'Fingerprint generation of flow cytometry data, used to facilitate the application of machine learning and datamining tools for flow cytometry.')]))])), 'bioconductor-flowsom': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowsom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowsom'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'FlowSOM_1.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/FlowSOM_1.8.0.tar.gz'), ('md5', 'df7694a37060723e8b5e2cd35fdf9032')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-consensusclusterplus', 'bioconductor-flowcore', 'bioconductor-flowutils', 'r-base', 'r-igraph', 'r-tsne', 'r-xml']), ('run', ['bioconductor-biocgenerics', 'bioconductor-consensusclusterplus', 'bioconductor-flowcore', 'bioconductor-flowutils', 'r-base', 'r-igraph', 'r-tsne', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'FlowSOM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/FlowSOM.html'), ('license', 'GPL (>= 2)'), ('summary', 'FlowSOM offers visualization options for cytometry data, by using Self-Organizing Map clustering and Minimal Spanning Trees.')]))])), 'bioconductor-flowutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowutils'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'flowUtils_1.40.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowUtils_1.40.0.tar.gz'), ('md5', 'e11862c5493fbe0fa22a208876a72d56')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-flowcore >=1.32.0', 'bioconductor-graph', 'r-base', 'r-corpcor', 'r-runit', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-flowcore >=1.32.0', 'bioconductor-graph', 'r-base', 'r-corpcor', 'r-runit', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowUtils\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowUtils.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides utilities for flow cytometry data.')]))])), 'bioconductor-flowviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowviz'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'flowViz_1.40.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowViz_1.40.0.tar.gz'), ('md5', '59faa4fce623429d5062d17cc7ce5dd7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-flowcore >=1.42.0', 'r-base', 'r-hexbin', 'r-idpmisc', 'r-latticeextra', 'r-rcolorbrewer', 'r-mass', 'r-kernsmooth']), ('run', ['bioconductor-biobase', 'r-mass', 'bioconductor-flowcore >=1.42.0', 'r-kernsmooth', 'r-base', 'r-hexbin', 'r-idpmisc', 'r-latticeextra', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowViz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowViz.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides visualization tools for flow cytometry data.')]))])), 'bioconductor-flowworkspace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowworkspace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowworkspace'), ('version', '3.24.4')])), ('source', CommentedMap([('fn', 'flowWorkspace_3.24.4.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/flowWorkspace_3.24.4.tar.gz'), ('md5', '76ca821e29806d133f651e00bc06b5c0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.39.9', 'bioconductor-ncdfflow >=2.19.5', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-bh >=1.62.0.1', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*']), ('run', ['libgcc', 'bioconductor-flowcore >=1.39.9', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'bioconductor-ncdfflow >=2.19.5', 'bioconductor-flowviz'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowWorkspace\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowWorkspace.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is designed to facilitate comparison of automated gating methods against manual gating done in flowJo. This package allows you to import basic flowJo workspaces into BioConductor and replicate the gating from flowJo using the flowCore functionality. Gating hierarchies, groups of samples, compensation, and transformation are performed so that the output matches the flowJo analysis')]))])), 'bioconductor-fourcseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-fourcseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-fourcseq'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'FourCSeq_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/FourCSeq_1.4.0.tar.gz'), ('md5', '1fa8c7291a9c0806ae9ada8e1f97e7f1')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-deseq2 >=1.9.11', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-summarizedexperiment', 'r-base', 'r-fda', 'r-ggplot2', 'r-gtools', 'r-lsd', 'r-reshape2']), ('run', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-deseq2 >=1.9.11', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-summarizedexperiment', 'r-base', 'r-fda', 'r-ggplot2', 'r-gtools', 'r-lsd', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'FourCSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/FourCSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'FourCSeq is an R package dedicated to the analysis of (multiplexed) 4C sequencing data. The package provides a pipeline to detect specific interactions between DNA elements and identify differential interactions between conditions. The statistical analysis in R starts with individual bam files for each sample as inputs. To obtain these files, the package contains a python script (extdata/python/demultiplex.py) to demultiplex libraries and trim off primer sequences. With a standard alignment software the required bam files can be then be generated.')]))])), 'bioconductor-gaga': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gaga/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gaga'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'gaga_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gaga_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/gaga_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/gaga/gaga_2.22.0_src_all.tar.gz']), ('md5', 'e9f2419240b574fdb7467578ec7250a2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv']), ('run', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gaga\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gaga.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the GaGa model for high-throughput data analysis, including differential expression analysis, supervised gene clustering and classification. Additionally, it performs sequential sample size calculations using the GaGa and LNNGV models (the latter from EBarrays package).')]))])), 'bioconductor-gage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gage'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gage_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/gage_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gage\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/gage.html'), ('license', 'GPL (>=2.0)'), ('summary', 'GAGE is a published method for gene set (enrichment or GSEA) or pathway analysis. GAGE is generally applicable independent of microarray or RNA-Seq data attributes including sample sizes, experimental designs, assay platforms, and other types of heterogeneity, and consistently achieves superior performance over other frequently used methods. In gage package, we provide functions for basic GAGE analysis, result processing and presentation. We have also built pipeline routines for of multiple GAGE analyses in a batch, comparison between parallel analyses, and combined analysis of heterogeneous data from different sources/studies. In addition, we provide demo microarray data and commonly used gene set data based on KEGG pathways and GO terms. These funtions and data are also useful for gene set analysis using other methods.')]))])), 'bioconductor-gagedata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gagedata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gagedata'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'gageData_2.10.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/data/experiment/src/contrib/gageData_2.10.0.tar.gz'), ('sha256', '948b38925f573cbd484fd5dd36e4423c8fcdcf8e01487d1a87bf6461dd0cd99e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gageData\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/gageData.html'), ('license', 'GPL (>=2.0)'), ('summary', 'This is a supportive data package for the software package, gage. However,\\nthe data supplied here are also useful for gene set or pathway analysis or microarray\\ndata analysis in general. In this package, we provide two demo microarray dataset:\\nGSE16873 (a breast cancer dataset from GEO) and BMP6 (originally published as\\nan demo dataset for GAGE, also registered as GSE13604 in GEO). This package also\\nincludes commonly used gene set data based on KEGG pathways and GO terms for major\\nresearch species, including human, mouse, rat and budding yeast. Mapping data\\nbetween common gene IDs for budding yeast are also included.\\n')]))])), 'bioconductor-gcrma': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gcrma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gcrma'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'gcrma_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gcrma_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/gcrma_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/gcrma/gcrma_2.48.0_src_all.tar.gz']), ('md5', 'c3a2f21c6f93458d1a123594e437a210')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gcrma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gcrma.html'), ('license', 'LGPL'), ('summary', 'Background adjustment using sequence information')]))])), 'bioconductor-genebreak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genebreak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genebreak'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'GeneBreak_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/GeneBreak_1.4.0.tar.gz'), ('md5', '5a4fbe3045bf61ca0f20e3a8a6854ead')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cghbase', 'bioconductor-cghcall', 'bioconductor-genomicranges', 'bioconductor-qdnaseq', 'r-base']), ('run', ['bioconductor-cghbase', 'bioconductor-cghcall', 'bioconductor-genomicranges', 'bioconductor-qdnaseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GeneBreak\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GeneBreak.html'), ('license', 'GPL-2'), ('summary', 'Recurrent breakpoint gene detection on copy number aberration profiles.')]))])), 'bioconductor-genefilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genefilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genefilter'), ('version', '1.58.1')])), ('source', CommentedMap([('fn', 'genefilter_1.58.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/genefilter_1.58.1.tar.gz', 'https://depot.galaxyproject.org/software/genefilter/genefilter_1.58.1_src_all.tar.gz']), ('md5', 'bc1a90bdf93d8db994220545cd80f438')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'gcc']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'genefilter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/genefilter.html'), ('license', 'Artistic-2.0'), ('summary', 'Some basic functions for filtering genes')]))])), 'bioconductor-genelendatabase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genelendatabase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genelendatabase'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'geneLenDataBase_1.10.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/data/experiment/src/contrib/geneLenDataBase_1.10.0.tar.gz'), ('md5', '93ccc6d3d06c4407b12c91387cb349b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicfeatures >=1.3.15', 'bioconductor-rtracklayer', 'r-base']), ('run', ['bioconductor-genomicfeatures >=1.3.15', 'bioconductor-rtracklayer', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'geneLenDataBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/geneLenDataBase.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Length of mRNA transcripts for a number of genomes and gene ID formats, largely based on UCSC table browser')]))])), 'bioconductor-geneplotter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geneplotter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geneplotter'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'geneplotter_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/geneplotter_1.50.0.tar.gz'), ('md5', '869b681a8d568ac5e1fe9fc576c8cb65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'geneplotter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/geneplotter.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions for plotting genomic data')]))])), 'bioconductor-genomation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomation'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'genomation_1.6.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-genomation/bioconductor-genomation_1.6.0_src_all.tar.gz'), ('md5', 'ceb645ad51f99c5b9ef09491c942d06a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.26', 'bioconductor-impute', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqpattern', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-rhtslib', 'r-base', 'r-data.table', 'r-ggplot2', 'r-gridbase', 'r-matrixstats', 'r-plotrix', 'r-plyr', 'r-readr', 'r-reshape2', 'r-rcpp', 'r-runit', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.26', 'bioconductor-impute', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqpattern', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-rhtslib', 'r-base', 'r-data.table', 'r-ggplot2', 'r-gridbase', 'r-matrixstats', 'r-plotrix', 'r-plyr', 'r-readr', 'r-reshape2', 'r-rcpp', 'r-runit', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'genomation\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/genomation.html'), ('license', 'Artistic-2.0'), ('summary', 'A package for summary and annotation of genomic intervals. Users can visualize and quantify genomic intervals over pre-defined functional regions, such as promoters, exons, introns, etc. The genomic intervals represent regions with a defined chromosome position, which may be associated with a score, such as aligned reads from HT-seq experiments, TF binding sites, methylation scores, etc. The package can use any tabular genomic feature data as long as it has minimal information on the locations of genomic intervals. In addition, It can use BAM or BigWig files as input.')]))])), 'bioconductor-genomeinfodb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomeinfodb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomeinfodb'), ('version', '1.10.3')])), ('source', CommentedMap([('fn', 'GenomeInfoDb_1.10.3.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomeInfoDb_1.10.3.tar.gz'), ('md5', '88376d3b96da905887491c9b6ef28339')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-iranges >=1.99.26', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-rcurl']), ('run', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-iranges >=1.99.26', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomeInfoDb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomeInfoDb.html'), ('license', 'Artistic-2.0'), ('summary', 'Contains data and functions that define and allow translation between different chromosome sequence naming conventions (e.g., \"chr1\" versus \"1\"), including a function that attempts to place sequence names in their natural, rather than lexicographic, order.')]))])), 'bioconductor-genomeinfodbdata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomeinfodbdata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomeinfodbdata'), ('version', '0.99.0')])), ('source', CommentedMap([('fn', 'GenomeInfoDbData_0.99.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/GenomeInfoDbData_0.99.0.tar.gz'), ('md5', '85977b51061dd02a90153db887040d05')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomeInfoDbData\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/GenomeInfoDbData.html'), ('license', 'Artistic-2.0'), ('summary', 'Files for mapping between NCBI taxonomy ID and species. Used by functions in the GenomeInfoDb package.')]))])), 'bioconductor-genomicalignments': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicalignments/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicalignments'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GenomicAlignments_1.10.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-genomicalignments/bioconductor-genomicalignments_1.10.0_src_all.tar.gz'), ('md5', '6e6af96adcad077abdb5889e5620cd6a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicAlignments\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient containers for storing and manipulating short genomic alignments (typically obtained by aligning short reads to a reference genome). This includes read counting, computing the coverage, junction detection, and working with the nucleotide content of the alignments.')]))])), 'bioconductor-genomicfeatures': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicfeatures/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicfeatures'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicFeatures_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicFeatures_1.26.4.tar.gz'), ('md5', '275a00ad50b7a5ec5ca5389b136491ca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicFeatures\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicFeatures.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and cds of a given organism, from either the UCSC Genome Browser or a BioMart database (more sources will be supported in the future). This information is then stored in a local database that keeps track of the relationship between transcripts, exons, cds and genes. Flexible methods are provided for extracting the desired features in a convenient format.')]))])), 'bioconductor-genomicranges': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicranges'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicRanges_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicRanges_1.26.4.tar.gz'), ('md5', '86c7d60ee21a86abc9f48b6527877bf4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages respectively. Both packages build on top of the GenomicRanges infrastructure.')]))])), 'bioconductor-geoquery': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geoquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geoquery'), ('version', '2.38.4')])), ('source', CommentedMap([('fn', 'GEOquery_2.38.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GEOquery_2.38.4.tar.gz'), ('md5', '54c96cb6d2b4eab9421b7d47180f26d6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GEOquery\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GEOquery.html'), ('license', 'GPL-2'), ('summary', 'The NCBI Gene Expression Omnibus (GEO) is a public repository of microarray data.  Given the rich and varied nature of this resource, it is only natural to want to apply BioConductor tools to these data.  GEOquery is the bridge between GEO and BioConductor.')]))])), 'bioconductor-ggbio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggbio/1.22.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggbio'), ('version', '1.22.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/ggbio_1.22.0.tar.gz'), ('sha256', '5ba61f3a2065de08a4ca888d6da3fee6aec5ca42bc5a47df64bd4574f4d81033')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggbio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggbio.html'), ('license', 'Artistic-2.0'), ('summary', 'The ggbio package extends and specializes the grammar of graphics for biological data. The graphics are designed to answer common scientific questions, in particular those often asked of high throughput genomics data. All core Bioconductor data structures are supported, where appropriate. The package supports detailed views of particular genomic regions, as well as genome-wide overviews. Supported overviews include ideograms and grand linear views. High-level plots include sequence fragment length, edge-linked interval to data view, mismatch pileup, and several splicing summaries.')]))])), 'bioconductor-ggcyto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggcyto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggcyto'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ggcyto_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ggcyto_1.4.0.tar.gz'), ('md5', 'c9dac98b39ce05da851dba598a345b7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggcyto\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggcyto.html'), ('license', 'Artistic-2.0'), ('summary', 'With the dedicated fority method implemented for flowSet, ncdfFlowSet and GatingSet classes, both raw and gated flow cytometry data can be plotted directly with ggplot. ggcyto wrapper and some customed layers also make it easy to add gates and population statistics to the plot.')]))])), 'bioconductor-ggtree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggtree/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggtree'), ('version', '1.4.20')])), ('source', CommentedMap([('fn', 'ggtree_1.4.20.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/ggtree_1.4.20.tar.gz'), ('md5', '0e87401239d610e3655de3c9640f33a7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ape', 'r-ggplot2 >=2.0.0', 'r-jsonlite', 'r-magrittr', 'r-tidyr']), ('run', ['r-base', 'r-ape', 'r-ggplot2 >=2.0.0', 'r-jsonlite', 'r-magrittr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggtree\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggtree.html'), ('license', 'Artistic-2.0'), ('summary', 'ggtree extends the ggplot2 plotting system which implemented the grammar of graphics. ggtree is designed for visualizing phylogenetic tree and different types of associated annotation data.')]))])), 'bioconductor-globalseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globalseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globalseq'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'globalSeq_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globalSeq_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/globalSeq/globalSeq_1.4.2_src_all.tar.gz']), ('md5', 'df77fed2999dca5a318409dd15141493')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globalSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globalSeq.html'), ('license', 'GPL-3'), ('summary', 'The method may be conceptualised as a test of overall significance in regression analysis, where the response variable is overdispersed and the number of explanatory variables exceeds the sample size.')]))])), 'bioconductor-globaltest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globaltest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globaltest'), ('version', '5.30.0')])), ('source', CommentedMap([('fn', 'globaltest_5.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globaltest_5.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/globaltest_5.30.0.tar.gz', 'https://depot.galaxyproject.org/software/globaltest/globaltest_5.30.0_src_all.tar.gz']), ('md5', 'baddf5286a53396d041bd459d54dc4af')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globaltest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globaltest.html'), ('license', 'GPL (>= 2)'), ('summary', 'The global test tests groups of covariates (or features) for association with a response variable. This package implements the test with diagnostic plots and multiple testing utilities, along with several functions to facilitate the use of this test for gene set testing of GO and KEGG terms.')]))])), 'bioconductor-go.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-go.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-go.db'), ('version', '3.4.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-go.db/bioconductor-go.db_3.4.0_src_all.tar.gz'), ('md5', '06e4ea8bd692bda85ae546263f39912d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.35.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.35.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GO.db\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/data/annotation/html/GO.db.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of annotation maps describing the entire Gene Ontology assembled using data from GO')]))])), 'bioconductor-gosemsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gosemsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gosemsim'), ('version', '1.30.3')])), ('source', CommentedMap([('fn', 'GOSemSim_1.30.3.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GOSemSim_1.30.3.tar.gz'), ('md5', 'db34559c6d9a0f092604e976ef26b801')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-go.db', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-annotationdbi', 'bioconductor-go.db', 'r-base', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GOSemSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GOSemSim.html'), ('license', 'Artistic-2.0'), ('summary', 'Implemented five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively for estimating GO semantic similarities. Support many species, including Anopheles, Arabidopsis, Bovine, Canine, Chicken, Chimp, Coelicolor, E coli strain K12 and Sakai, Fly, Gondii, Human, Malaria, Mouse, Pig, Rhesus, Rat, Worm, Xenopus, Yeast, and Zebrafish.')]))])), 'bioconductor-goseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-goseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-goseq'), ('version', '1.26.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-goseq/bioconductor-goseq_1.26.0_src_all.tar.gz'), ('md5', 'da46e940cc1cf1f624a06c908f07f910')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'goseq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/goseq.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Detects Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data')]))])), 'bioconductor-gostats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gostats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gostats'), ('version', '2.38.1')])), ('source', CommentedMap([('fn', 'GOstats_2.38.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/GOstats_2.38.1.tar.gz', 'https://depot.galaxyproject.org/software/GOstats/GOstats_2.38.1_src_all.tar.gz']), ('md5', '684862c904e7b5f4731bd27d09f9ce43')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.13.2', 'bioconductor-annotationdbi >=0.0.89', 'bioconductor-annotationforge', 'bioconductor-biobase >=1.15.29', 'bioconductor-category >=2.3.26', 'bioconductor-go.db >=1.13.0', 'bioconductor-graph >=1.15.15', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-annotate >=1.13.2', 'bioconductor-annotationdbi >=0.0.89', 'bioconductor-annotationforge', 'bioconductor-biobase >=1.15.29', 'bioconductor-category >=2.3.26', 'bioconductor-go.db >=1.13.0', 'bioconductor-graph >=1.15.15', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GOstats\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/GOstats.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of tools for interacting with GO and microarray data. A variety of basic manipulation tools for graphs, hypothesis testing and other simple calculations.')]))])), 'bioconductor-gothic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gothic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gothic'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GOTHiC_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/GOTHiC_1.10.0.tar.gz'), ('md5', 'd53ee31c7fe5d8924ce29eb3497da4ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GOTHiC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GOTHiC.html'), ('license', 'GPL-3'), ('summary', 'This is a Hi-C analysis package using a cumulative binomial test to detect interactions between distal genomic loci that have significantly more reads than expected by chance in Hi-C experiments. It takes mapped paired NGS reads as input and gives back the list of significant interactions for a given bin size in the genome.')]))])), 'bioconductor-graph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-graph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-graph'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'graph_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/graph_1.50.0.tar.gz'), ('md5', 'd1fbfd7dfffce6f46a0e1d288fa5858a'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.11', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.13.11', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'graph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/graph.html'), ('license', 'Artistic-2.0'), ('summary', 'A package that implements some simple graph handling capabilities.')]))])), 'bioconductor-graphite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-graphite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-graphite'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'graphite_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/graphite_1.16.0.tar.gz'), ('md5', '67553b5b50c8fee87aa44d4737d5c40a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-base', 'r-rappdirs']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-base', 'r-rappdirs'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'graphite\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/graphite.html'), ('license', 'AGPL-3'), ('summary', 'Graph objects from pathway topology derived from Biocarta, HumanCyc, KEGG, NCI, Panther, Reactome and SPIKE databases.')]))])), 'bioconductor-gseabase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gseabase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gseabase'), ('version', '1.34.1')])), ('source', CommentedMap([('fn', 'GSEABase_1.34.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GSEABase_1.34.1.tar.gz'), ('md5', '26f0a512fddadcb36162dd80fcf8a2d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSEABase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSEABase.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides classes and methods to support Gene Set Enrichment Analysis (GSEA).')]))])), 'bioconductor-gsva': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gsva/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gsva'), ('version', '1.24.1')])), ('source', CommentedMap([('fn', 'GSVA_1.24.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-gsva/bioconductor-gsva_1.24.1_src_all.tar.gz'), ('md5', '93579e46cd3bc4f3a9db0b9bfa7a4921')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSVA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSVA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Gene Set Variation Analysis (GSVA) is a non-parametric, unsupervised method for estimating variation of gene set enrichment through the samples of a expression data set. GSVA performs a change in coordinate systems, transforming the data from a gene by sample matrix to a gene-set by sample matrix, thereby allowing the evaluation of pathway enrichment for each sample. This new matrix of GSVA enrichment scores facilitates applying standard analytical methods like functional enrichment, survival analysis, clustering, CNV-pathway analysis or cross-tissue pathway analysis, in a pathway-centric manner.')]))])), 'bioconductor-gviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gviz'), ('version', '1.16.5')])), ('source', CommentedMap([('fn', 'Gviz_1.16.5.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/Gviz_1.16.5.tar.gz'), ('md5', '81f61d0d12c823f91eff0a69d14b3943')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer']), ('run', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Gviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Gviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Genomic data analyses requires integrated visualization of known genomic information and new experimental data.  Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.')]))])), 'bioconductor-heatplus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-heatplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-heatplus'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'Heatplus_2.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Heatplus_2.20.0.tar.gz'), ('md5', '7607b82c9960e3991cb850d484eb800a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Heatplus\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Heatplus.html'), ('license', 'GPL (>= 2)'), ('summary', 'Display a rectangular heatmap (intensity plot) of a data matrix. By default, both samples (columns) and features (row) of the matrix are sorted according to a hierarchical clustering, and the corresponding dendrogram is plotted. Optionally, panels with additional information about samples and features can be added to the plot.')]))])), 'bioconductor-hicrep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hicrep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower}}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz', 'https://depot.galaxyproject.org/software/<{ name }}/<{ name }}_<{ version }}_src_all.tar.gz']), ('sha256', 'c0ae75a9c6558031a1cde17fde9a1a571d17d2a15843ad90fb2ebcb27eef1600')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base >=3.4']), ('run', ['r-base >=3.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'hicrep\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/hicrep.html'), ('license', 'GPL (>= 2.0)'), ('summary', 'Hi-C is a powerful technology for studying genome-wide chromatin interactions. However, current methods for assessing Hi-C data reproducibility can produce misleading results because they ignore spatial features in Hi-C data, such as domain structure and distance-dependence. We present a novel reproducibility measure that systematically takes these features into consideration. This measure can assess pairwise differences between Hi-C matrices under a wide range of settings, and can be used to determine optimal sequencing depth. Compared to existing approaches, it consistently shows higher accuracy in distinguishing subtle differences in reproducibility and depicting interrelationships of cell lineages than existing approaches. This R package `hicrep` implements our approach.')]))])), 'bioconductor-hilbertvis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hilbertvis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hilbertvis'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'HilbertVis_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/HilbertVis_1.32.0.tar.gz'), ('md5', '59c578593bf698fecfbd2fad4db832ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'HilbertVis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/HilbertVis.html'), ('license', 'GPL (>= 3)'), ('summary', 'Functions to visualize long vectors of integer data by means of Hilbert curves')]))])), 'bioconductor-hom.hs.inp.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hom.hs.inp.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hom.hs.inp.db'), ('version', '3.1.2')])), ('source', CommentedMap([('fn', 'hom.Hs.inp.db_3.1.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/data/annotation/src/contrib/hom.Hs.inp.db_3.1.2.tar.gz'), ('md5', 'b0708148e9c6656527e004fbaf80c05c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.29.20', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.29.20', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'hom.Hs.inp.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/hom.Hs.inp.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Homology information for Homo Sapiens from Inparanoid assembled using data from public repositories')]))])), 'bioconductor-hpar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hpar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hpar'), ('version', '1.15.0')])), ('source', CommentedMap([('fn', 'hpar_1.15.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/hpar_1.15.0.tar.gz'), ('md5', 'c27602ea84d160c0b71b65c68dea6bf0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'hpar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/hpar.html'), ('license', 'Artistic-2.0'), ('summary', 'A simple interface to and data from the Human Protein Atlas project.')]))])), 'bioconductor-hsmmsinglecell': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hsmmsinglecell/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hsmmsinglecell'), ('version', '0.110.0')])), ('source', CommentedMap([('fn', 'HSMMSingleCell_0.110.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/experiment/src/contrib/HSMMSingleCell_0.110.0.tar.gz'), ('md5', '9e2e48e26b05c6ba8b9cbb1cb224441a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'HSMMSingleCell\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/HSMMSingleCell.html'), ('license', 'Artistic-2.0'), ('summary', 'Skeletal myoblasts undergo a well-characterized sequence of morphological and transcriptional changes during differentiation. In this experiment, primary human skeletal muscle myoblasts (HSMM) were expanded under high mitogen conditions (GM) and then differentiated by switching to low-mitogen media (DM).  RNA-Seq libraries were sequenced from each of several hundred cells taken over a time-course of serum-induced differentiation. Between 49 and 77 cells were captured at each of four time points (0, 24, 48, 72 hours) following serum switch using the Fluidigm C1 microfluidic system. RNA from each cell was isolated and used to construct mRNA-Seq libraries, which were then sequenced to a depth of ~4 million reads per library, resulting in a complete gene expression profile for each cell.')]))])), 'bioconductor-ihw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ihw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ihw'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'IHW_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/IHW_1.2.0.tar.gz'), ('md5', 'f1463042daa169d2c485be2ff22bf333')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-lpsymphony', 'r-base', 'r-fdrtool', 'r-slam']), ('run', ['bioconductor-biocgenerics', 'bioconductor-lpsymphony', 'r-base', 'r-fdrtool', 'r-slam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IHW\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/IHW.html'), ('license', 'Artistic-2.0'), ('summary', 'Independent hypothesis weighting (IHW) is a multiple testing procedure that increases power compared to the method of Benjamini and Hochberg by assigning data-driven weights to each hypothesis. The input to IHW is a two-column table of p-values and covariates. The covariate can be any continuous-valued or categorical variable that is thought to be informative on the statistical properties of each hypothesis test, while it is independent of the p-value under the null hypothesis.')]))])), 'bioconductor-illuminahumanmethylation450kanno.ilmn12.hg19': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-illuminahumanmethylation450kanno.ilmn12.hg19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-illuminahumanmethylation450kanno.ilmn12.hg19'), ('version', '0.6.0')])), ('source', CommentedMap([('fn', 'IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.0.tar.gz', 'https://depot.galaxyproject.org/software/IlluminaHumanMethylation450kanno.ilmn12.hg19/IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.0_src_all.tar.gz']), ('md5', '2f569646ca8adc49863224b1cd076a79')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-minfi >=1.19.15', 'r-base']), ('run', ['bioconductor-minfi >=1.19.15', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IlluminaHumanMethylation450kanno.ilmn12.hg19\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/IlluminaHumanMethylation450kanno.ilmn12.hg19.html'), ('license', 'Artistic-2.0'), ('summary', \"Manifests and annotation for Illumina's 450k array data.\")]))])), 'bioconductor-illuminaio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-illuminaio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-illuminaio'), ('version', '0.14.0')])), ('source', CommentedMap([('fn', 'illuminaio_0.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/illuminaio_0.14.0.tar.gz'), ('md5', '53189e29b2e2a8d6ca3a2a86f9f0c53f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-base64']), ('run', ['r-base', 'r-base64'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'illuminaio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/illuminaio.html'), ('license', 'GPL-2'), ('summary', \"Tools for parsing Illumina's microarray output files, including IDAT.\")]))])), 'bioconductor-illuminamousev2.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-illuminamousev2.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-illuminamousev2.db'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'illuminaMousev2.db_1.26.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/illuminaMousev2.db_1.26.0.tar.gz'), ('md5', 'fba228a71f264f976d8cdb035861974d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.29.20', 'bioconductor-org.mm.eg.db >=3.1.2', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.29.20', 'bioconductor-org.mm.eg.db >=3.1.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'illuminaMousev2.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/illuminaMousev2.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Illumina MouseWG6v2 annotation data (chip illuminaMousev2) assembled using data from public repositories')]))])), 'bioconductor-impute': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-impute/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-impute'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'impute_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/impute_1.46.0.tar.gz'), ('md5', '8d0adfd6536a82a0c39cea3c5783ffe4')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'impute\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/impute.html'), ('license', 'GPL-2'), ('summary', 'Imputation for microarray data (currently KNN only)')]))])), 'bioconductor-interactivedisplaybase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-interactivedisplaybase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-interactivedisplaybase'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'interactiveDisplayBase_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/interactiveDisplayBase_1.12.0.tar.gz'), ('md5', 'ef3fc69145648ea333677e4f8c3d0a0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base', 'r-shiny']), ('run', ['bioconductor-biocgenerics', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'interactiveDisplayBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The interactiveDisplayBase package contains the the basic methods needed to generate interactive Shiny based display methods for Bioconductor objects.')]))])), 'bioconductor-ipo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ipo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ipo'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'IPO_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/IPO_1.0.0.tar.gz'), ('md5', 'd0cf5ff014677ed4ef4fcdb5a8c64bca')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm']), ('run', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IPO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/IPO.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'The outcome of XCMS data processing strongly depends on the parameter settings. IPO (`Isotopologue Parameter Optimization`) is a parameter optimization tool that is applicable for different kinds of samples and liquid chromatography coupled to high resolution mass spectrometry devices, fast and free of labeling steps. IPO uses natural, stable 13C isotopes to calculate a peak picking score. Retention time correction is optimized by minimizing the relative retention time differences within features and grouping parameters are optimized by maximizing the number of features showing exactly one peak from each injection of a pooled sample. The different parameter settings are achieved by design of experiment. The resulting scores are evaluated using response surface models.')]))])), 'bioconductor-iranges': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-iranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-iranges'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'IRanges_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/IRanges_2.8.2.tar.gz'), ('md5', 'ae1e6b78c6b84e4df6bec44e98e2136c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/IRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient low-level and highly reusable S4 classes for storing, manipulating and aggregating over annotated ranges of integers. Implements an algebra of range operations, including efficient algorithms for finding overlaps and nearest neighbors. Defines efficient list-like classes for storing, transforming and aggregating large grouped data, i.e., collections of atomic vectors and DataFrames.')]))])), 'bioconductor-isomirs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-isomirs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-isomirs'), ('version', '1.0.3')])), ('source', CommentedMap([('fn', 'isomiRs_1.0.3.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/isomiRs_1.0.3.tar.gz'), ('md5', '36669b2e85ce69a0f168afe55e11c928')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer']), ('run', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'isomiRs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/isomiRs.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Characterization of miRNAs and isomiRs, clustering and differential expression.')]))])), 'bioconductor-iwtomics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-iwtomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz'), ('md5', 'd7d17984d9f9c4babf027c4403b572c9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicranges', 'r-base', 'r-gtable', 'r-kernsmooth', 'r-fda', 'bioconductor-iranges', 'bioconductor-s4vectors']), ('run', ['bioconductor-genomicranges', 'r-base', 'r-gtable', 'r-kernsmooth', 'r-fda', 'bioconductor-iranges', 'bioconductor-s4vectors'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'<{ name }}\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/IWTomics.html'), ('license', 'GPL (>=2)'), ('summary', 'Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in \"Omics\" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.')]))])), 'bioconductor-jaspar2014': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-jaspar2014/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-jaspar2014'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'JASPAR2014_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/experiment/src/contrib/JASPAR2014_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/JASPAR2014/JASPAR2014_1.12.0_src_all.tar.gz']), ('sha256', 'b3d873e944ce9cfda343c35e694ec90c9dec06119d5ad900fd608a5d0b2fbf8e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.29.19', 'r-base']), ('run', ['bioconductor-biostrings >=2.29.19', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'JASPAR2014\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/experiment/html/JASPAR2014.html'), ('license', 'GPL-2'), ('summary', 'Data package for JASPAR 2014. To search this databases, please use the package TFBSTools.')]))])), 'bioconductor-jmosaics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-jmosaics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-jmosaics'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'jmosaics_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/jmosaics_1.10.0.tar.gz'), ('md5', 'b9a41d743cdfec000437f8ab009ef927')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-mosaics', 'r-base']), ('run', ['bioconductor-mosaics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'jmosaics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/jmosaics.html'), ('license', 'GPL (>= 2)'), ('summary', 'jmosaics detects enriched regions of ChIP-seq data sets jointly.')]))])), 'bioconductor-kcsmart': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-kcsmart/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-kcsmart'), ('version', '2.32.0')])), ('source', CommentedMap([('fn', 'KCsmart_2.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/KCsmart_2.32.0.tar.gz'), ('md5', '36576407fc0fca284eb002de75e4e8a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-multtest', 'bioconductor-siggenes', 'r-base', 'r-kernsmooth']), ('run', ['bioconductor-biocgenerics', 'bioconductor-multtest', 'bioconductor-siggenes', 'r-base', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KCsmart\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/KCsmart.html'), ('license', 'GPL-3'), ('summary', 'Multi sample aCGH analysis package using kernel convolution')]))])), 'bioconductor-kegg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-kegg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-kegg.db'), ('version', '3.2.3')])), ('source', CommentedMap([('fn', 'KEGG.db_3.2.3.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/KEGG.db_3.2.3.tar.gz'), ('md5', '023ac22f57063627c2e62d1ae5e011b0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.34.3', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.34.3', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KEGG.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/KEGG.db.html'), ('license', 'file LICENSE'), ('summary', 'A set of annotation maps for KEGG assembled using data from KEGG')]))])), 'bioconductor-kegggraph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-kegggraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-kegggraph'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'KEGGgraph_1.30.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/KEGGgraph_1.30.0.tar.gz'), ('md5', '885c16a5dd746ce7a276bd538fc7602f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0']), ('run', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KEGGgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/KEGGgraph.html'), ('license', 'GPL (>= 2)'), ('summary', 'KEGGGraph is an interface between KEGG pathway and graph object as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers functionalities including parsing, graph operation, visualization and etc.')]))])), 'bioconductor-keggrest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-keggrest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-keggrest'), ('version', '1.14.1')])), ('source', CommentedMap([('fn', 'KEGGREST_1.14.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/KEGGREST_1.14.1.tar.gz', 'https://depot.galaxyproject.org/software/KEGGREST/KEGGREST_1.14.1_src_all.tar.gz']), ('sha256', 'dea0d1afdd64f4c777b59d5671034cf44b2a2b758a628ade230df88ee7aa8d35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-httr', 'r-png']), ('run', ['bioconductor-biostrings', 'r-base', 'r-httr', 'r-png'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KEGGREST\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/KEGGREST.html'), ('license', 'Artistic-2.0'), ('summary', 'A package that provides a client interface to the KEGG REST server. Based on KEGGSOAP by J. Zhang, R. Gentleman, and Marc Carlson, and KEGG (python package) by Aurelien Mazurie.')]))])), 'bioconductor-limma': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-limma/3.28.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-limma'), ('version', '3.28.2')])), ('source', CommentedMap([('fn', 'limma_3.28.2.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/limma/limma_3.28.2_src_all.tar.gz'), ('sha256', '70acdd49e77b8514afd8ca364c3c47b6fbeef46b7a40881d5312832e90cf9c75')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'limma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/limma.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.\\n')]))])), 'bioconductor-lpsymphony': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-lpsymphony/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-lpsymphony'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'lpsymphony_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/lpsymphony_1.2.0.tar.gz'), ('md5', '043beed308f85f461dfd2760ade9362c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'lpsymphony\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/lpsymphony.html'), ('license', 'EPL'), ('summary', \"This package was derived from Rsymphony_0.1-17 from CRAN. These packages provide an R interface to SYMPHONY, an open-source linear programming solver written in C++. The main difference between this package and Rsymphony is that it includes the solver source code (SYMPHONY version 5.6), while Rsymphony expects to find header and library files on the users' system. Thus the intention of lpsymphony is to provide an easy to install interface to SYMPHONY. For Windows, precompiled DLLs are included in this package.\")]))])), 'bioconductor-m3drop': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-m3drop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-m3drop'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'M3Drop_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/M3Drop_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/M3Drop_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/M3Drop/M3Drop_1.2.0_src_all.tar.gz']), ('md5', 'e2bcdd73fbc61c739d13db83b5ace1db')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod']), ('run', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'M3Drop\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/M3Drop.html'), ('license', 'GPL (>=2)'), ('summary', 'This package fits a Michaelis-Menten model to the pattern of dropouts in single-cell RNASeq data. This model is used as a null to identify significantly variable (i.e. differentially expressed) genes for use in downstream analysis, such as clustering cells.')]))])), 'bioconductor-maanova': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-maanova/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-maanova'), ('version', '1.46.1')])), ('source', CommentedMap([('fn', 'maanova_1.46.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maanova_1.46.1.tar.gz', 'https://depot.galaxyproject.org/software/maanova/maanova_1.46.1_src_all.tar.gz']), ('md5', '1edc4dab038dfeb276a518c37d04783b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maanova\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maanova.html'), ('license', 'GPL (>= 2)'), ('summary', 'Analysis of N-dye Micro Array experiment using mixed model effect. Containing analysis of variance, permutation and bootstrap, cluster and consensus tree.')]))])), 'bioconductor-macat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macat'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'macat_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/macat_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/macat_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/macat/macat_1.50.0_src_all.tar.gz']), ('md5', '1fef5c90534f564c40cc4708be621d11')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'macat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/macat.html'), ('license', 'Artistic-2.0'), ('summary', \"This library contains functions to investigate links between differential gene expression and the chromosomal localization of the genes. MACAT is motivated by the common observation of phenomena involving large chromosomal regions in tumor cells. MACAT is the implementation of a statistical approach for identifying significantly differentially expressed chromosome regions. The functions have been tested on a publicly available data set about acute lymphoblastic leukemia (Yeoh et al.Cancer Cell 2002), which is provided in the library 'stjudem'.\")]))])), 'bioconductor-macorrplot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macorrplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macorrplot'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'maCorrPlot_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maCorrPlot_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/maCorrPlot_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/maCorrPlot/maCorrPlot_1.46.0_src_all.tar.gz']), ('md5', '96c19ed0a305c4da96b1619e4db20fda')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maCorrPlot\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maCorrPlot.html'), ('license', 'GPL (>= 2)'), ('summary', 'Graphically displays correlation in microarray data that is due to insufficient normalization')]))])), 'bioconductor-made4': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-made4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-made4'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'made4_1.44.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/made4_1.44.0.tar.gz'), ('md5', '68c0e762f8fca1b702a66b14df1e7f66')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d']), ('run', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'made4\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/made4.html'), ('license', 'Artistic-2.0'), ('summary', 'Multivariate data analysis and graphical display of microarray data. Functions include between group analysis and coinertia analysis. It contains functions that require ADE4.')]))])), 'bioconductor-makecdfenv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-makecdfenv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-makecdfenv'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'makecdfenv_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/makecdfenv_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/makecdfenv_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/makecdfenv/makecdfenv_1.52.0_src_all.tar.gz']), ('md5', '6471ca53c57eb2c9d3580e7306337790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'makecdfenv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/makecdfenv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package has two functions. One reads a Affymetrix chip description file (CDF) and creates a hash table environment containing the location/probe set membership mapping. The other creates a package that automatically loads that environment.')]))])), 'bioconductor-manta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-manta'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'manta_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/manta_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/manta_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/manta/manta_1.22.0_src_all.tar.gz']), ('md5', '3f77679b1a72c82bd988352ceb00a20d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc']), ('run', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'manta\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/manta.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for robust comparative metatranscriptomics.')]))])), 'bioconductor-mantelcorr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mantelcorr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mantelcorr'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'MantelCorr_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MantelCorr_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/MantelCorr_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/MantelCorr/MantelCorr_1.46.0_src_all.tar.gz']), ('md5', '4d422d6b713bbb38bdee222c0c66a307')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MantelCorr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MantelCorr.html'), ('license', 'GPL (>= 2)'), ('summary', 'Computes Mantel cluster correlations from a (p x n) numeric data matrix (e.g. microarray gene-expression data).')]))])), 'bioconductor-marray': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-marray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-marray'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'marray_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/marray_1.50.0.tar.gz'), ('md5', 'c068e960357aa8d9d84b9f517dd1ef50')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'marray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/marray.html'), ('license', 'LGPL'), ('summary', 'Class definitions for two-color spotted microarray data. Fuctions for data input, diagnostic plots, normalization and quality checking.')]))])), 'bioconductor-masigpro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-masigpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-masigpro'), ('version', '1.49.3')])), ('source', CommentedMap([('fn', 'maSigPro_1.49.0.tar.gz'), ('url', 'http://bioconductor.org/packages/devel/bioc/src/contrib/maSigPro_1.49.3.tar.gz'), ('md5', 'fc8332ffd3752ac8139fbf5037938d2e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maSigPro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/maSigPro.html'), ('license', 'GPL (>= 2)'), ('summary', 'maSigPro is a regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.')]))])), 'bioconductor-massarray': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massarray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massarray'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'MassArray_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MassArray_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/MassArray_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/MassArray/MassArray_1.28.0_src_all.tar.gz']), ('md5', '71153966697991f134350a2c23c14082')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassArray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MassArray.html'), ('license', 'GPL (>=2)'), ('summary', \"This package is designed for the import, quality control, analysis, and visualization of methylation data generated using Sequenom's MassArray platform.  The tools herein contain a highly detailed amplicon prediction for optimal assay design.  Also included are quality control measures of data, such as primer dimer and bisulfite conversion efficiency estimation.  Methylation data are calculated using the same algorithms contained in the EpiTyper software package.  Additionally, automatic SNP-detection can be used to flag potentially confounded data from specific CG sites.  Visualization includes barplots of methylation data as well as UCSC Genome Browser-compatible BED tracks.  Multiple assays can be positionally combined for integrated analysis.\")]))])), 'bioconductor-massir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massir'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'massiR_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/massiR_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/massiR_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/massiR/massiR_1.12.0_src_all.tar.gz']), ('md5', 'b5a116a80b4e7414011c54b257ec5593')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-diptest', 'r-gplots']), ('run', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-diptest', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'massiR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/massiR.html'), ('license', 'GPL-3'), ('summary', 'Predicts the sex of samples in gene expression microarray datasets')]))])), 'bioconductor-massspecwavelet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massspecwavelet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massspecwavelet'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'MassSpecWavelet_1.40.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MassSpecWavelet_1.40.0.tar.gz'), ('md5', '878b0dec9f50294ec4cb487b02de4c65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-waveslim']), ('run', ['r-base', 'r-waveslim'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassSpecWavelet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Processing Mass Spectrometry spectrum by using wavelet based algorithm')]))])), 'bioconductor-mbamethyl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbamethyl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbamethyl'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBAmethyl_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBAmethyl_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBAmethyl_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBAmethyl/MBAmethyl_1.10.0_src_all.tar.gz']), ('md5', 'b960d48ffe66f080d8daa5b4e9baa318')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBAmethyl\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBAmethyl.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a function for reconstructing DNA methylation values from raw measurements. It iteratively implements the group fused lars to smooth related-by-location methylation values and the constrained least squares to remove probe affinity effect across multiple sequences.')]))])), 'bioconductor-mbased': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbased/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbased'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBASED_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBASED_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBASED_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBASED/MBASED_1.10.0_src_all.tar.gz']), ('md5', 'f1f31202aea10cf5921accb1197a3315')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBASED\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBASED.html'), ('license', 'Artistic-2.0'), ('summary', 'The package implements MBASED algorithm for detecting allele-specific gene expression from RNA count data, where allele counts at individual loci (SNVs) are integrated into a gene-specific measure of ASE, and utilizes simulations to appropriately assess the statistical significance of observed ASE.')]))])), 'bioconductor-mbttest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbttest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbttest'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MBttest_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBttest_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBttest_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MBttest/MBttest_1.4.0_src_all.tar.gz']), ('md5', '984d689b8f577d0920dca8985ee4d800')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-gtools']), ('run', ['r-base', 'r-gplots', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBttest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBttest.html'), ('license', 'GPL-3'), ('summary', 'MBttest method was developed from beta t-test method of Baggerly et al(2003). Compared to baySeq (Hard castle and Kelly 2010), DESeq (Anders and Huber 2010) and exact test (Robinson and Smyth 2007, 2008) and the GLM of McCarthy et al(2012), MBttest is of high work efficiency,that is, it has high power, high conservativeness of FDR estimation and high stability. MBttest is suit- able to transcriptomic data, tag data, SAGE data (count data) from small samples or a few replicate libraries. It can be used to identify genes, mRNA isoforms or tags differentially expressed between two conditions.')]))])), 'bioconductor-mdgsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mdgsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mdgsa'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'mdgsa_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mdgsa_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/mdgsa_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/mdgsa/mdgsa_1.8.0_src_all.tar.gz']), ('md5', 'b11c4973665408f0ccdc43835a8f1557')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-kegg.db', 'r-base', 'r-cluster', 'r-dbi', 'r-matrix']), ('run', ['bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-kegg.db', 'r-base', 'r-cluster', 'r-dbi', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mdgsa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mdgsa.html'), ('license', 'GPL'), ('summary', 'Functions to preform a Gene Set Analysis in several genomic dimensions. Including methods for miRNAs.')]))])), 'bioconductor-mdqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mdqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mdqc'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'mdqc_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mdqc_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/mdqc_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/mdqc/mdqc_1.38.0_src_all.tar.gz']), ('md5', 'ab47f4c330b4825b4b73cfb77fa665d5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-cluster', 'r-mass']), ('run', ['r-base', 'r-cluster', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mdqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mdqc.html'), ('license', 'LGPL (>= 2)'), ('summary', \"MDQC is a multivariate quality assessment method for microarrays based on quality control (QC) reports. The Mahalanobis distance of an array's quality attributes is used to measure the similarity of the quality of that array against the quality of the other arrays. Then, arrays with unusually high distances can be flagged as potentially low-quality.\")]))])), 'bioconductor-medips': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-medips/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-medips'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MEDIPS_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MEDIPS_1.24.0.tar.gz'), ('md5', '29d4cdec874a39a6b9c255fc5e634d49')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MEDIPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MEDIPS.html'), ('license', 'GPL (>=2)'), ('summary', 'MEDIPS was developed for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, MEDIPS provides functionalities for the analysis of any kind of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential coverage between groups of samples and saturation and correlation analysis.')]))])), 'bioconductor-mergemaid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergemaid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergemaid'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'MergeMaid_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MergeMaid_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MergeMaid_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MergeMaid/MergeMaid_2.48.0_src_all.tar.gz']), ('md5', 'afc57533d7efaa0c17490ff86f2d4b29')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MergeMaid\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MergeMaid.html'), ('license', 'GPL (>= 2)'), ('summary', \"The functions in this R extension are intended for cross-study comparison of gene expression array data. Required from the user is gene expression matrices, their corresponding gene-id vectors and other useful information, and they could be 'list','matrix', or 'ExpressionSet'. The main function is 'mergeExprs' which transforms the input objects into data in the merged format, such that common genes in different datasets can be easily found. And the function 'intcor' calculate the correlation coefficients. Other functions use the output from 'modelOutcome' to graphically display the results and cross-validate associations of gene expression data with survival.\")]))])), 'bioconductor-mergeomics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergeomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergeomics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'Mergeomics_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mergeomics_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mergeomics_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/Mergeomics/Mergeomics_1.4.0_src_all.tar.gz']), ('md5', '0a40c90a30eb100c29686cc569b0847d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mergeomics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mergeomics.html'), ('license', 'GPL (>= 2)'), ('summary', 'The Mergeomics pipeline serves as a flexible framework for integrating multidimensional omics-disease associations, functional genomics,  canonical pathways and gene-gene interaction networks to generate mechanistic hypotheses. It includes two main parts, 1) Marker set enrichment analysis (MSEA); 2) Weighted Key Driver Analysis (wKDA).')]))])), 'bioconductor-meshdbi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-meshdbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-meshdbi'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'MeSHDbi_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MeSHDbi_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/MeSHDbi_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/MeSHDbi/MeSHDbi_1.12.0_src_all.tar.gz']), ('md5', '03d136edff42166a4b1f0b8f6f326519')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'r-base', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'r-base', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MeSHDbi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MeSHDbi.html'), ('license', 'Artistic-2.0'), ('summary', 'The package is unified implementation of MeSH.db, MeSH.AOR.db, and MeSH.PCR.db and also is interface to construct Gene-MeSH package (MeSH.XXX.eg.db). loadMeSHDbiPkg import sqlite file and generate MeSH.XXX.eg.db.')]))])), 'bioconductor-meshsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-meshsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-meshsim'), ('version', '1.7.0')])), ('source', CommentedMap([('fn', 'MeSHSim_1.7.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MeSHSim_1.7.0.tar.gz', 'https://bioarchive.galaxyproject.org/MeSHSim_1.7.0.tar.gz', 'https://depot.galaxyproject.org/software/MeSHSim/MeSHSim_1.7.0_src_all.tar.gz']), ('md5', '77d3c05e09a0e30980e235d456f68edc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-xml']), ('run', ['r-base', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MeSHSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MeSHSim.html'), ('license', 'GPL-2'), ('summary', 'Provide for measuring semantic similarity over MeSH headings and MEDLINE documents')]))])), 'bioconductor-messina': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-messina/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-messina'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'messina_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/messina_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/messina_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/messina/messina_1.12.0_src_all.tar.gz']), ('md5', '077273969af5ba95bab6a77cbdfe0849')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-foreach >=1.4.1', 'r-ggplot2 >=0.9.3.1', 'r-plyr >=1.8', 'r-rcpp >=0.11.1', 'r-survival >=2.37-4', 'gcc', 'llvm']), ('run', ['r-base', 'r-foreach >=1.4.1', 'r-ggplot2 >=0.9.3.1', 'r-plyr >=1.8', 'r-rcpp >=0.11.1', 'r-survival >=2.37-4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'messina\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/messina.html'), ('license', 'EPL (>= 1.0)'), ('summary', 'Messina is a collection of algorithms for constructing optimally robust single-gene classifiers, and for identifying differential expression in the presence of outliers or unknown sample subgroups.  The methods have application in identifying lead features to develop into clinical tests (both diagnostic and prognostic), and in identifying differential expression when a fraction of samples show unusual patterns of expression.')]))])), 'bioconductor-metacca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metacca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metacca'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'metaCCA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaCCA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaCCA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/metaCCA/metaCCA_1.4.0_src_all.tar.gz']), ('md5', '9623dba2a19fa1d8b9998506c1fd07dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaCCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaCCA.html'), ('license', 'MIT + file LICENSE'), ('summary', 'metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness.')]))])), 'bioconductor-metagenomeseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metagenomeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metagenomeseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metagenomeSeq_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metagenomeSeq_1.16.0.tar.gz'), ('md5', '537eed40d187418841a19e89a70a802b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metagenomeSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'metagenomeSeq is designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. metagenomeSeq is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.')]))])), 'bioconductor-metahdep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metahdep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metahdep'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'metahdep_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metahdep_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/metahdep_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/metahdep/metahdep_1.34.0_src_all.tar.gz']), ('md5', '769b7067870b6c965bc8b49aa60bbaea')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metahdep\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metahdep.html'), ('license', 'GPL-3'), ('summary', 'Tools for meta-analysis in the presence of hierarchical (and/or sampling) dependence, including with gene expression studies')]))])), 'bioconductor-metams': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metams/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metams'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'metaMS_1.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metaMS_1.8.0.tar.gz'), ('md5', 'a1188b28fc2f0df985fe44bc69c2fb2b')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase']), ('run', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaMS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metaMS.html'), ('license', 'GPL (>= 2)'), ('summary', 'MS-based metabolomics data processing and compound annotation pipeline.')]))])), 'bioconductor-metaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metaseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metaSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/metaSeq/metaSeq_1.16.0_src_all.tar.gz']), ('md5', '3212d61d0c3a8572b21d5cd4b28a34e4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow', 'gcc', 'llvm']), ('run', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaSeq.html'), ('license', 'Artistic-2.0'), ('summary', \"The probabilities by one-sided NOISeq are combined by Fisher's method or Stouffer's method\")]))])), 'bioconductor-methped': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methped/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methped'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MethPed_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethPed_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethPed_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MethPed/MethPed_1.4.0_src_all.tar.gz']), ('md5', '56c270065cbe5f49423a0a598b5715ef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-randomforest']), ('run', ['bioconductor-biobase', 'r-base', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethPed\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethPed.html'), ('license', 'GPL-2'), ('summary', 'Classification of pediatric tumors into biologically defined subtypes is challenging and multifaceted approaches are needed. For this aim, we developed a diagnostic classifier based on DNA methylation profiles. We offer MethPed as an easy-to-use toolbox that allows researchers and clinical diagnosticians to test single samples as well as large cohorts for subclass prediction of pediatric brain tumors.  The current version of MethPed can classify the following tumor diagnoses/subgroups: Diffuse Intrinsic Pontine Glioma (DIPG), Ependymoma, Embryonal tumors with multilayered rosettes (ETMR), Glioblastoma (GBM), Medulloblastoma (MB) - Group 3 (MB_Gr3), Group 4 (MB_Gr3), Group WNT (MB_WNT), Group SHH (MB_SHH) and Pilocytic Astrocytoma (PiloAstro).')]))])), 'bioconductor-methtargetedngs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methtargetedngs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methtargetedngs'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'MethTargetedNGS_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethTargetedNGS_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethTargetedNGS_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/MethTargetedNGS/MethTargetedNGS_1.8.0_src_all.tar.gz']), ('md5', '26100031f59e66d819fbd2780f4ef7c4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr']), ('run', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethTargetedNGS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethTargetedNGS.html'), ('license', 'Artistic-2.0'), ('summary', 'Perform step by step methylation analysis of Next Generation Sequencing data.')]))])), 'bioconductor-methylkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylkit'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'methylKit_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylKit_1.0.0.tar.gz'), ('md5', '3612ce6bf0b748d8869a4a16c57265dc'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'gcc', 'llvm']), ('run', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylKit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylKit.html'), ('license', 'Artistic-2.0'), ('summary', 'methylKit is an R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. The package is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods and whole genome bisulfite sequencing. It also has functions to analyze base-pair resolution 5hmC data from experimental protocols such as oxBS-Seq and TAB-Seq. Perl is needed to read SAM files only.')]))])), 'bioconductor-methylmnm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylmnm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylmnm'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'methylMnM_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/methylMnM_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/methylMnM_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/methylMnM/methylMnM_1.14.0_src_all.tar.gz']), ('md5', '12b9fea00269b0db050b38a4705bdc65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'r-base', 'r-statmod']), ('run', ['bioconductor-edger', 'r-base', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylMnM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/methylMnM.html'), ('license', 'GPL-3'), ('summary', 'To give the exactly p-value and q-value of MeDIP-seq and MRE-seq data for different samples comparation.')]))])), 'bioconductor-methylpipe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylpipe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylpipe'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'methylPipe_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/methylPipe_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/methylPipe_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/methylPipe/methylPipe_1.10.0_src_all.tar.gz']), ('md5', 'eb95d03c3d651b420442809d475031e8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylPipe\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/methylPipe.html'), ('license', 'GPL(>=2)'), ('summary', 'Memory efficient analysis of base resolution DNA methylation data in both the CpG and non-CpG sequence context. Integration of DNA methylation data derived from any methodology providing base- or low-resolution data.')]))])), 'bioconductor-methylseekr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylseekr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylseekr'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MethylSeekR_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethylSeekR_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethylSeekR_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/MethylSeekR/MethylSeekR_1.16.0_src_all.tar.gz']), ('md5', 'ba38b395a899e88619f414301225447d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4']), ('run', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethylSeekR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethylSeekR.html'), ('license', 'GPL (>=2)'), ('summary', 'This is a package for the discovery of regulatory regions from Bis-seq data')]))])), 'bioconductor-methylumi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylumi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylumi'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'methylumi_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylumi_2.22.0.tar.gz'), ('md5', '3d169d5e6fbd51d829c691353428478d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylumi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylumi.html'), ('license', 'GPL-2'), ('summary', 'This package provides classes for holding and manipulating Illumina methylation data.  Based on eSet, it can contain MIAME information, sample information, feature information, and multiple matrices of data.  An \"intelligent\" import function, methylumiR can read the Illumina text files and create a MethyLumiSet. methylumIDAT can directly read raw IDAT files from HumanMethylation27 and HumanMethylation450 microarrays.  Normalization, background correction, and quality control features for GoldenGate, Infinium, and Infinium HD arrays are also included.')]))])), 'bioconductor-mgfm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfm'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MGFM_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFM_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFM_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFM/MGFM_1.10.0_src_all.tar.gz']), ('md5', 'ad048930fe4233344085409b2a5b11cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFM.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from Microarray gene expression data sets')]))])), 'bioconductor-mgfr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfr'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'MGFR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFR/MGFR_1.2.0_src_all.tar.gz']), ('md5', '50d79a1e21138cf4f84326811d3da6ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFR.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from RNA-seq data.')]))])), 'bioconductor-mgsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgsa'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'mgsa_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mgsa_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/mgsa_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/mgsa/mgsa_1.24.0_src_all.tar.gz']), ('md5', '1a0757f3de04eff70284e297afbae1dc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots']), ('run', ['r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mgsa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mgsa.html'), ('license', 'Artistic-2.0'), ('summary', 'Model-based Gene Set Analysis (MGSA) is a Bayesian modeling approach for gene set enrichment. The package mgsa implements MGSA and tools to use MGSA together with the Gene Ontology.')]))])), 'bioconductor-mgug4122a.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgug4122a.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgug4122a.db'), ('version', '3.2.3')])), ('source', CommentedMap([('fn', 'mgug4122a.db_3.2.3.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/mgug4122a.db_3.2.3.tar.gz'), ('md5', '5fcdce909ac1c0ccbf06bf6411f6fe52')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.34.3', 'bioconductor-org.mm.eg.db >=3.3.0', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.34.3', 'bioconductor-org.mm.eg.db >=3.3.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mgug4122a.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/mgug4122a.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Agilent \"Mouse Genome,  Whole\" annotation data (chip mgug4122a) assembled using data from public repositories')]))])), 'bioconductor-michip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-michip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-michip'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MiChip_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiChip_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiChip_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MiChip/MiChip_1.30.0_src_all.tar.gz']), ('md5', '0242abbd32d6d5a13c2f384a79cf99b0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiChip\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiChip.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package takes the MiChip miRNA microarray .grp scanner output files and parses these out, providing summary and plotting functions\\tto analyse MiChip hybridizations. A set of hybridizations is packaged into an ExpressionSet allowing it to be used by other\\tBioConductor packages.')]))])), 'bioconductor-microrna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-microrna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-microrna'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'microRNA_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/microRNA_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/microRNA_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/microRNA/microRNA_1.34.0_src_all.tar.gz']), ('md5', '3760e1ec8b1830541621a9f98acb2d9d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.11.32', 'r-base']), ('run', ['bioconductor-biostrings >=2.11.32', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'microRNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/microRNA.html'), ('license', 'Artistic-2.0'), ('summary', 'Different data resources for microRNAs and some functions for manipulating them.')]))])), 'bioconductor-minfi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-minfi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-minfi'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'minfi_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/minfi_1.20.0.tar.gz'), ('md5', '1b5c8e55fd8fca426d6bb2c5072d36b3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme']), ('run', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'minfi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/minfi.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools to analyze & visualize Illumina Infinium methylation arrays.')]))])), 'bioconductor-mipp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mipp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mipp'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'MiPP_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiPP_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiPP_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MiPP/MiPP_1.48.0_src_all.tar.gz']), ('md5', '5d9d5dfbe0a0f6ccba21673466bb9f41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiPP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiPP.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package finds optimal sets of genes that seperate samples into two or more classes.')]))])), 'bioconductor-mirage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirage'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'MiRaGE_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiRaGE_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiRaGE_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/MiRaGE/MiRaGE_1.18.0_src_all.tar.gz']), ('md5', 'ca996ba9edb5d1f21674b67bc63878d7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiRaGE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiRaGE.html'), ('license', 'GPL'), ('summary', 'The package contains functions for inferece of target gene regulation by miRNA, based on only target gene expression profile.')]))])), 'bioconductor-mirnapath': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirnapath/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirnapath'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'miRNApath_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/miRNApath_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/miRNApath_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/miRNApath/miRNApath_1.36.0_src_all.tar.gz']), ('md5', '6486870670b9f7557c0347450b54d336')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'miRNApath\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/miRNApath.html'), ('license', 'LGPL-2.1'), ('summary', 'This package provides pathway enrichment techniques for miRNA expression data. Specifically, the set of methods handles the many-to-many relationship between miRNAs and the multiple genes they are predicted to target (and thus affect.)  It also handles the gene-to-pathway relationships separately. Both steps are designed to preserve the additive effects of miRNAs on genes, many miRNAs affecting one gene, one miRNA affecting multiple genes, or many miRNAs affecting many genes.')]))])), 'bioconductor-mirnatap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirnatap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirnatap'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'miRNAtap_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/miRNAtap_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/miRNAtap_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/miRNAtap/miRNAtap_1.10.0_src_all.tar.gz']), ('md5', '32a93a993be062879cf6e3090dfb79e7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-plyr', 'r-rsqlite', 'r-sqldf', 'r-stringr']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-plyr', 'r-rsqlite', 'r-sqldf', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'miRNAtap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/miRNAtap.html'), ('license', 'GPL-2'), ('summary', 'The package facilitates implementation of workflows requiring miRNA predictions, it allows to integrate ranked miRNA target predictions from multiple sources available online and aggregate them with various methods which improves quality of predictions above any of the single sources. Currently predictions are available for Homo sapiens, Mus musculus and Rattus norvegicus (the last one through homology translation).')]))])), 'bioconductor-mirsynergy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirsynergy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirsynergy'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'Mirsynergy_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mirsynergy_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mirsynergy_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/Mirsynergy/Mirsynergy_1.12.0_src_all.tar.gz']), ('md5', '14f46bc5e653d284295e7baa11857791')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mirsynergy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mirsynergy.html'), ('license', 'GPL-2'), ('summary', 'Detect synergistic miRNA regulatory modules by overlapping neighbourhood expansion.')]))])), 'bioconductor-mlp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mlp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mlp'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MLP_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MLP_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/MLP_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/MLP/MLP_1.24.0_src_all.tar.gz']), ('md5', 'c23abefc67186aa4f9f349ebdce590d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix']), ('run', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MLP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MLP.html'), ('license', 'GPL-3'), ('summary', 'Mean Log P Analysis')]))])), 'bioconductor-mmdiff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mmdiff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mmdiff'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MMDiff_1.10.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/MMDiff_1.10.0.tar.gz'), ('md5', '79a1f6646081d5c09621a80a26945516')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-genomicranges', 'bioconductor-diffbind', 'r-gmd', 'bioconductor-rsamtools', 'bioconductor-iranges', 'bioconductor-biobase']), ('run', ['r-base', 'bioconductor-genomicranges', 'bioconductor-diffbind', 'r-gmd', 'bioconductor-rsamtools', 'bioconductor-iranges', 'bioconductor-biobase'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MMDiff\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MMDiff.html'), ('license', 'Artistic-2.0'), ('summary', 'This package detects statistically significant difference between read enrichment profiles in different ChIP-Seq samples. To take advantage of shape differences it uses Kernel methods (Maximum Mean Discrepancy, MMD).')]))])), 'bioconductor-mmnet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mmnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mmnet'), ('version', '1.13.0')])), ('source', CommentedMap([('fn', 'mmnet_1.13.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mmnet_1.13.0.tar.gz', 'https://bioarchive.galaxyproject.org/mmnet_1.13.0.tar.gz', 'https://depot.galaxyproject.org/software/mmnet/mmnet_1.13.0_src_all.tar.gz']), ('md5', '90508186eaaa876bff44b42fb6e4a46d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mmnet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mmnet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package gives the implementations microbiome metabolic network constructing and analyzing. It introduces a unique metagenomic systems biology approach, mapping metagenomic data to the KEGG global metabolic pathway and constructing a systems-level network. The system-level network and the next topological analysis will be of great help to analysis the various functional properties, including regulation and metabolic functionality of the metagenome.')]))])), 'bioconductor-monocle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-monocle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz'), ('md5', '5d9bca1e5c32ae86a9a970e95867e230')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'monocle\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/monocle.html'), ('license', 'Artistic-2.0'), ('summary', 'Monocle performs differential expression and time-series analysis for single-cell expression experiments. It orders individual cells according to progress through a biological process, without knowing ahead of time which genes define progress through that process. Monocle also performs differential expression analysis, clustering, visualization, and other useful tasks on single cell expression data.  It is designed to work with RNA-Seq and qPCR data, but could be used with other types as well.')]))])), 'bioconductor-mops': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mops'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MoPS_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MoPS_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MoPS_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MoPS/MoPS_1.10.0_src_all.tar.gz']), ('md5', '96dd954498d1a6acd69dfdb9310a6f44')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MoPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MoPS.html'), ('license', 'GPL-3'), ('summary', 'Identification and characterization of periodic fluctuations in time-series data.')]))])), 'bioconductor-mosaics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mosaics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mosaics'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'mosaics_2.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/mosaics_2.10.0.tar.gz'), ('md5', '06465236065b08ebbab99d610f597e7d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-mass', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base', 'r-mass', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mosaics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mosaics.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for fitting MOSAiCS and MOSAiCS-HMM, a statistical framework to analyze one-sample or two-sample ChIP-seq data of transcription factor binding and histone modification.')]))])), 'bioconductor-motifbreakr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifbreakr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifbreakr'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'motifbreakR_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/motifbreakR_1.4.0.tar.gz'), ('md5', '66d72ab927fb1da866a9376416fc2e79')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifbreakR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/motifbreakR.html'), ('license', 'GPL-2'), ('summary', 'We introduce motifbreakR, which allows the biologist to judge in the first place whether the sequence surrounding the polymorphism is a good match, and in the second place how much information is gained or lost in one allele of the polymorphism relative to another. MotifbreakR is both flexible and extensible over previous offerings; giving a choice of algorithms for interrogation of genomes with motifs from public sources that users can choose from; these are 1) a weighted-sum probability matrix, 2) log-probabilities, and 3) weighted by relative entropy. MotifbreakR can predict effects for novel or previously described variants in public databases, making it suitable for tasks beyond the scope of its original design. Lastly, it can be used to interrogate any genome curated within Bioconductor (currently there are 22).')]))])), 'bioconductor-motifdb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MotifDb_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MotifDb_1.16.0.tar.gz'), ('md5', 'b30e09247f857e7bca54681acdd4e849')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotifDb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MotifDb.html'), ('license', 'Artistic-2.0 | file LICENSE'), ('summary', 'More than 2000 annotated position frequency matrices from nine public sources, for multiple organisms.')]))])), 'bioconductor-motifrg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifrg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifrg'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'motifRG_1.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-motifrg/bioconductor-motifrg_1.18.0_src_all.tar.gz'), ('md5', '2f1dab8afccdc3077109dcb514945f3b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifRG\\')\"'])])), ('about', CommentedMap([('home', 'https://www.bioconductor.org/packages/release/bioc/html/motifRG.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for discriminative motif discovery using regression methods')]))])), 'bioconductor-motifstack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifstack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifstack'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'motifStack_1.18.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/motifStack_1.18.0.tar.gz'), ('md5', '0391df16d61699fbe22fa38e809fc2cb')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-motiv', 'r-base', 'r-ade4', 'r-grimport', 'r-scales', 'r-xml']), ('run', ['bioconductor-biostrings', 'bioconductor-motiv', 'r-base', 'r-ade4', 'r-grimport', 'r-scales', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifStack\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/motifStack.html'), ('license', 'GPL (>= 2)'), ('summary', 'The motifStack package is designed for graphic representation of multiple motifs with different similarity scores. It works with both DNA/RNA sequence motif and amino acid sequence motif. In addition, it provides the flexibility for users to customize the graphic parameters such as the font type and symbol colors.')]))])), 'bioconductor-motiv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motiv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motiv'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MotIV_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/MotIV_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MotIV_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MotIV/MotIV_1.30.0_src_all.tar.gz']), ('md5', '19da23b2900e7e9b334084a7bd78c17c')])), ('build', CommentedMap([('string', 'bioconductor_<{CONDA_R}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}'), ('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'gsl <{CONDA_GSL}}*', 'r-lattice', 'gcc', 'llvm']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'r-lattice', 'gsl <{CONDA_GSL}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotIV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MotIV.html'), ('license', 'GPL-2'), ('summary', 'This package makes use of STAMP for comparing a set of motifs to a given database (e.g. JASPAR). It can also be used to visualize motifs, motif distributions, modules and filter motifs.')]))])), 'bioconductor-mpfe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mpfe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mpfe'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'MPFE_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MPFE_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/MPFE_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/MPFE/MPFE_1.12.0_src_all.tar.gz']), ('md5', 'efc12549261886fd1c4ed958bc29d416')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MPFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MPFE.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate distribution of methylation patterns from a table of counts from a bisulphite sequencing experiment given a non-conversion rate and read error rate.')]))])), 'bioconductor-mulcom': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mulcom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mulcom'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'Mulcom_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mulcom_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mulcom_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/Mulcom/Mulcom_1.26.0_src_all.tar.gz']), ('md5', 'c1d015fbeded777ad696baed0263fb78')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-fields']), ('run', ['bioconductor-biobase', 'r-base', 'r-fields'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mulcom\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mulcom.html'), ('license', 'GPL-2'), ('summary', 'Identification of differentially expressed genes and false discovery rate (FDR) calculation by Multiple Comparison test')]))])), 'bioconductor-multidataset': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multidataset/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multidataset'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MultiDataSet_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MultiDataSet_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MultiDataSet_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MultiDataSet/MultiDataSet_1.4.0_src_all.tar.gz']), ('md5', '8c8c27b4d6dfd42a2371d31e52ef5cd1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MultiDataSet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MultiDataSet.html'), ('license', 'file LICENSE'), ('summary', \"Implementation of the BRGE's (Bioinformatic Research Group in Epidemiology from Center for Research in Environmental Epidemiology) MultiDataSet and MethylationSet. MultiDataSet is designed for integrating multi omics data sets and MethylationSet to contain normalized methylation data. These package contains base classes for MEAL and rexposome packages.\")]))])), 'bioconductor-multimed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multimed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multimed'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MultiMed_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MultiMed_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MultiMed_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MultiMed/MultiMed_1.10.0_src_all.tar.gz']), ('md5', 'a8d5867f71e7341f7cb3d84b0b49ebf1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MultiMed\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MultiMed.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Implements permutation method with joint correction for testing multiple mediators')]))])), 'bioconductor-multiscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multiscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multiscan'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'multiscan_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/multiscan_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/multiscan_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/multiscan/multiscan_1.36.0_src_all.tar.gz']), ('md5', '27ed1fe589004fef769d3179b5231f0b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multiscan\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multiscan.html'), ('license', 'GPL (>= 2)'), ('summary', 'Estimates gene expressions from several laser scans of the same microarray')]))])), 'bioconductor-multtest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multtest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multtest'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'multtest_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/multtest_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/multtest_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/multtest/multtest_2.28.0_src_all.tar.gz']), ('md5', 'a82a1564e15fdc5afe6efc4c33a33bbc')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multtest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multtest.html'), ('license', 'LGPL'), ('summary', 'Non-parametric bootstrap and permutation resampling-based multiple testing procedures (including empirical Bayes methods) for controlling the family-wise error rate (FWER), generalized family-wise error rate (gFWER), tail probability of the proportion of false positives (TPPFP), and false discovery rate (FDR).  Several choices of bootstrap-based null distribution are implemented (centered, centered and scaled, quantile-transformed). Single-step and step-wise methods are available. Tests based on a variety of t- and F-statistics (including t-statistics based on regression parameters from linear and survival models as well as those based on correlation parameters) are included.  When probing hypotheses with t-statistics, users may also select a potentially faster null distribution which is multivariate normal with mean zero and variance covariance matrix derived from the vector influence function.  Results are reported in terms of adjusted p-values, confidence regions and test statistic cutoffs. The procedures are directly applicable to identifying differentially expressed genes in DNA microarray experiments.')]))])), 'bioconductor-mvcclass': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvcclass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvcclass'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'MVCClass_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MVCClass_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/MVCClass_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/MVCClass/MVCClass_1.50.0_src_all.tar.gz']), ('md5', 'fde9aa791e789b3eab70420f3081886f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MVCClass\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MVCClass.html'), ('license', 'LGPL'), ('summary', 'Creates classes used in model-view-controller (MVC) design')]))])), 'bioconductor-mvgst': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvgst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvgst'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'mvGST_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mvGST_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/mvGST_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/mvGST/mvGST_1.10.0_src_all.tar.gz']), ('md5', '029ab61ae4876a87d952915b8528071a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mvGST\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mvGST.html'), ('license', 'GPL-3'), ('summary', 'mvGST provides platform-independent tools to identify GO terms (gene sets) that are differentially active (up or down) in multiple contrasts of interest.  Given a matrix of one-sided p-values (rows for genes, columns for contrasts), mvGST uses meta-analytic methods to combine p-values for all genes annotated to each gene set, and then classify each gene set as being significantly more active (1), less active (-1), or not significantly differentially active (0) in each contrast of interest.  With multiple contrasts of interest, each gene set is assigned to a profile (across contrasts) of differential activity.  Tools are also provided for visualizing (in a GO graph) the gene sets classified to a given profile.')]))])), 'bioconductor-myvariant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-myvariant/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-myvariant'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'myvariant_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/myvariant_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/myvariant_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/myvariant/myvariant_1.6.0_src_all.tar.gz']), ('md5', '55a4fa3c2e59ac665ccb38b8fa5b27be')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-variantannotation', 'r-base', 'r-hmisc', 'r-httr', 'r-jsonlite', 'r-magrittr', 'r-plyr']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-variantannotation', 'r-base', 'r-hmisc', 'r-httr', 'r-jsonlite', 'r-magrittr', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'myvariant\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/myvariant.html'), ('license', 'Artistic-2.0'), ('summary', 'MyVariant.info is a comprehensive aggregation of variant annotation resources. myvariant is a wrapper for querying MyVariant.info services')]))])), 'bioconductor-mzid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzid'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'mzID_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mzID_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/mzID_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/mzID/mzID_1.14.0_src_all.tar.gz']), ('md5', '7069a877ea94c63ecfea78f8556a6fd2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml']), ('run', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzID\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mzID.html'), ('license', 'GPL (>= 2)'), ('summary', \"A parser for mzIdentML files implemented using the XML package. The parser tries to be general and able to handle all types of mzIdentML files with the drawback of having less 'pretty' output than a vendor specific parser. Please contact the maintainer with any problems and supply an mzIdentML file so the problems can be fixed quickly.\")]))])), 'bioconductor-mzr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzr'), ('version', '2.6.3')])), ('source', CommentedMap([('fn', 'mzR_2.6.3.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-mzr/bioconductor-mzr_2.6.3_src_all.tar.gz'), ('md5', 'c36c4a9ec2ec82642f5ffae56d9a08ea')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'gcc']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mzR.html'), ('license', 'Artistic-2.0'), ('summary', 'mzR provides a unified API to the common file formats and parsers available for mass spectrometry data. It comes with a wrapper for the ISB random access parser for mass spectrometry mzXML, mzData and mzML files. The package contains the original code written by the ISB, and a subset of the proteowizard library for mzML and mzIdentML. The netCDF reading code has previously been used in XCMS.')]))])), 'bioconductor-ncdfflow': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncdfflow/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncdfflow'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'ncdfFlow_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ncdfFlow_2.22.0.tar.gz'), ('md5', 'b5a69c2d1776bb34deaf6e7a0bff3384')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5', 'gcc', 'llvm']), ('run', ['libgcc', 'bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ncdfFlow\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ncdfFlow.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides HDF5 storage based methods and functions for manipulation of flow cytometry data.')]))])), 'bioconductor-ncigraph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncigraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncigraph'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'NCIgraph_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NCIgraph_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/NCIgraph_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/NCIgraph/NCIgraph_1.24.0_src_all.tar.gz']), ('md5', '494714008fc79bf0e2e945742277c07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NCIgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NCIgraph.html'), ('license', 'GPL-3'), ('summary', 'Provides various methods to load the pathways from the NCI Pathways Database in R graph objects and to re-format them.')]))])), 'bioconductor-nem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nem'), ('version', '2.50.0')])), ('source', CommentedMap([('fn', 'nem_2.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nem_2.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/nem_2.50.0.tar.gz', 'https://depot.galaxyproject.org/software/nem/nem_2.50.0_src_all.tar.gz']), ('md5', '67c37dcdaa983aedffac020cfa646632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod']), ('run', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nem.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package 'nem' allows to reconstruct features of pathways from the nested structure of perturbation effects. It takes as input (1.) a set of pathway components, which were perturbed, and (2.) phenotypic readout of these perturbations (e.g. gene expression, protein expression). The output is a directed graph representing the phenotypic hierarchy.\")]))])), 'bioconductor-netbiov': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-netbiov/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-netbiov'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'netbiov_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/netbiov_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/netbiov_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/netbiov/netbiov_1.10.0_src_all.tar.gz']), ('md5', '3f6ffb9a475561b7f49e79eec5d75214')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=0.7.1']), ('run', ['r-base', 'r-igraph >=0.7.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'netbiov\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/netbiov.html'), ('license', 'GPL (>= 2)'), ('summary', 'A package that provides an effective visualization of large biological networks')]))])), 'bioconductor-netpathminer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-netpathminer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-netpathminer'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'NetPathMiner_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NetPathMiner_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/NetPathMiner/NetPathMiner_1.12.1_src_all.tar.gz']), ('md5', 'b4b7d04540ab6f03cba5dc3fa141e9cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0']), ('run', ['r-base', 'r-igraph >=1.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NetPathMiner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NetPathMiner.html'), ('license', 'GPL (>= 2)'), ('summary', 'NetPathMiner is a general framework for network path mining using genome-scale networks. It constructs networks from KGML, SBML and BioPAX files, providing three network representations, metabolic, reaction and gene representations. NetPathMiner finds active paths and applies machine learning methods to summarize found paths for easy interpretation. It also provides static and interactive visualizations of networks and paths to aid manual investigation.')]))])), 'bioconductor-nnnorm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nnnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nnnorm'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'nnNorm_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nnNorm_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/nnNorm_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/nnNorm/nnNorm_2.40.0_src_all.tar.gz']), ('md5', 'e5d78c1120e7cd09a530637c89811eb8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-nnet']), ('run', ['bioconductor-marray', 'r-base', 'r-nnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nnNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nnNorm.html'), ('license', 'LGPL'), ('summary', 'This package allows to detect and correct for spatial and intensity biases with two-channel microarray data. The normalization method implemented in this package is based on robust neural networks fitting.')]))])), 'bioconductor-noiseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-noiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-noiseq'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'NOISeq_2.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-noiseq/bioconductor-noiseq_2.18.0_src_all.tar.gz'), ('sha256', '050e6ab51f91e9a5574eefb708596f8128f8aa1c7c120c8c3593021580ff8447')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NOISeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/NOISeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Analysis of RNA-seq expression data or other similar kind of data. Exploratory plots to evualuate saturation, count distribution, expression per chromosome, type of detected features, features length, etc. Differential expression between two experimental conditions with no parametric assumptions.')]))])), 'bioconductor-npgsea': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-npgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-npgsea'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'npGSEA_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/npGSEA_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/npGSEA_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/npGSEA/npGSEA_1.12.0_src_all.tar.gz']), ('md5', '15869313306ed711156c582821d4c4bb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'npGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/npGSEA.html'), ('license', 'Artistic-2.0'), ('summary', 'Current gene set enrichment methods rely upon permutations for inference.  These approaches are computationally expensive and have minimum achievable p-values based on the number of permutations, not on the actual observed statistics.  We have derived three parametric approximations to the permutation distributions of two gene set enrichment test statistics.  We are able to reduce the computational burden and granularity issues of permutation testing with our method, which is implemented in this package. npGSEA calculates gene set enrichment statistics and p-values without the computational cost of permutations.  It is applicable in settings where one or many gene sets are of interest.  There are also built-in plotting functions to help users visualize results.')]))])), 'bioconductor-ntw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ntw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ntw'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'NTW_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NTW_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/NTW_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/NTW/NTW_1.26.0_src_all.tar.gz']), ('md5', '5b746a15be789dcc4fe2b859f29bf0bc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mvtnorm']), ('run', ['r-base', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NTW\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NTW.html'), ('license', 'GPL-2'), ('summary', 'This package predicts the gene-gene interaction network and identifies the direct transcriptional targets of the perturbation using an ODE (Ordinary Differential Equation) based method.')]))])), 'bioconductor-nucleosim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucleosim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucleosim'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'nucleoSim_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleoSim_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleoSim_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleoSim/nucleoSim_1.4.0_src_all.tar.gz']), ('md5', '1efb2ee4c162e11775179d859c1b0a8b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleoSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleoSim.html'), ('license', 'Artistic-2.0'), ('summary', 'This package can generate a synthetic map with reads covering the nucleosome regions as well as a synthetic map with forward and reverse reads emulating next-generation sequencing. The user has choice between three different distributions for the read positioning: Normal, Student and Uniform.')]))])), 'bioconductor-nucler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucler'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'nucleR_2.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleR_2.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleR_2.8.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleR/nucleR_2.8.0_src_all.tar.gz']), ('md5', 'c23125444464c780fce47cf09391a6bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleR.html'), ('license', 'LGPL (>= 3)'), ('summary', 'Nucleosome positioning for Tiling Arrays and NGS experiments.')]))])), 'bioconductor-nudge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nudge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nudge'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'nudge_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nudge_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/nudge_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/nudge/nudge_1.42.0_src_all.tar.gz']), ('md5', 'bb1641c652cdb1dd7193933a0073d9bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nudge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nudge.html'), ('license', 'GPL-2'), ('summary', 'Package for normalizing microarray data in single and multiple replicate experiments and fitting a normal-uniform mixture to detect differentially expressed genes in the cases where the two samples are being compared directly or indirectly (via a common reference sample)')]))])), 'bioconductor-occugene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-occugene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-occugene'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'occugene_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/occugene_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/occugene_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/occugene/occugene_1.36.0_src_all.tar.gz']), ('md5', '181f362099ae9c53d87ae7bfe3d37570')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'occugene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/occugene.html'), ('license', 'GPL (>= 2)'), ('summary', 'Statistical tools for building random mutagenesis libraries for prokaryotes. The package has functions for handling the occupancy distribution for a multinomial and for estimating the number of essential genes in random transposon mutagenesis libraries.')]))])), 'bioconductor-ogsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ogsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ogsa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'OGSA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OGSA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/OGSA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/OGSA/OGSA_1.6.0_src_all.tar.gz']), ('md5', '146f36f67ca15d7958556fd3ff67621e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OGSA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OGSA.html'), ('license', 'GPL (== 2)'), ('summary', 'OGSA provides a global estimate of pathway deregulation in cancer subtypes by integrating the estimates of significance for individual pathway members that have been identified by outlier analysis.')]))])), 'bioconductor-oligo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oligo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oligo'), ('version', '1.40.1')])), ('source', CommentedMap([('fn', 'oligo_1.40.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-oligo/bioconductor-oligo_1.40.1_src_all.tar.gz'), ('md5', 'f5d262569456c869ea9c06d4afee0d53')])), ('build', CommentedMap([('script', 'R CMD INSTALL --build .'), ('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ff', 'r-rcurl', 'r-foreach', 'r-rcpp', 'r-digest', 'r-dbi', 'r-rsqlite', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller', 'bioconductor-oligoclasses', 'bioconductor-biostrings', 'bioconductor-affyio', 'bioconductor-affxparser', 'bioconductor-preprocesscore']), ('run', ['r-base', 'r-ff', 'r-rcurl', 'r-foreach', 'r-rcpp', 'r-digest', 'r-dbi', 'r-rsqlite', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller', 'bioconductor-oligoclasses', 'bioconductor-biostrings', 'bioconductor-affyio', 'bioconductor-affxparser', 'bioconductor-preprocesscore'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oligo\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/3.5/bioc/html/oligo.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A package to analyze oligonucleotide arrays (expression/SNP/tiling/exon) at probe-level. It currently supports Affymetrix (CEL files) and NimbleGen arrays (XYS files).')]))])), 'bioconductor-oligoclasses': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oligoclasses/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oligoclasses'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'oligoClasses_1.38.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.5/bioc/src/contrib/oligoClasses_1.38.0.tar.gz'), ('md5', 'dc3ec1a000bbe9dd1ba5e6fc07e7d6e0')])), ('build', CommentedMap([('script', 'R CMD INSTALL --build .'), ('skip', True)])), ('requirements', CommentedMap([('build', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller']), ('run', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oligoClasses\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/3.5/bioc/html/oligoClasses.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains class definitions, validity checks, and initialization methods for classes used by the oligo and crlmm packages.')]))])), 'bioconductor-olin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-olin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-olin'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'OLIN_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OLIN_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/OLIN_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/OLIN/OLIN_1.54.0_src_all.tar.gz']), ('md5', 'b659246e12660920db5522a38109161c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OLIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OLIN.html'), ('license', 'GPL-2'), ('summary', 'Functions for normalisation of two-color microarrays by optimised local regression and for detection of artefacts in microarray data')]))])), 'bioconductor-omicade4': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-omicade4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-omicade4'), ('version', '1.16.1')])), ('source', CommentedMap([('fn', 'omicade4_1.16.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/omicade4_1.16.1.tar.gz', 'https://depot.galaxyproject.org/software/omicade4/omicade4_1.16.1_src_all.tar.gz']), ('md5', 'b0bed984322168dde82c4f14b136f4f3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-made4', 'r-ade4', 'r-base']), ('run', ['bioconductor-made4', 'r-ade4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'omicade4\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/omicade4.html'), ('license', 'GPL-2'), ('summary', 'Multiple co-inertia analysis of omics datasets')]))])), 'bioconductor-omiccircos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-omiccircos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-omiccircos'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'OmicCircos_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OmicCircos_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/OmicCircos_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/OmicCircos/OmicCircos_1.14.0_src_all.tar.gz']), ('md5', '0e450c5c2f144467d6bc43a2d613feb7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicranges', 'r-base']), ('run', ['bioconductor-genomicranges', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OmicCircos\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OmicCircos.html'), ('license', 'GPL-2'), ('summary', 'OmicCircos is an R application and package for generating high-quality circular plots for omics data.')]))])), 'bioconductor-oppar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oppar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oppar'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'oppar_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/oppar_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/oppar_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/oppar/oppar_1.4.0_src_all.tar.gz']), ('md5', 'de85939cfe9df6f69cf4c72792ed4445')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oppar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/oppar.html'), ('license', 'GPL-2'), ('summary', 'The R implementation of mCOPA package published by Wang et al. (2012). Oppar provides methods for Cancer Outlier profile Analysis. Although initially developed to detect outlier genes in cancer studies, methods presented in oppar can be used for outlier profile analysis in general. In addition, tools are provided for gene set enrichment and pathway analysis.')]))])), 'bioconductor-org.bt.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.bt.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.bt.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Bt.eg.db_3.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Bt.eg.db_3.4.1.tar.gz'), ('md5', '74c67eae5d29e86c9eb72047c71dccce')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Bt.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Bt.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Bovine, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.ce.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.ce.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.ce.eg.db'), ('version', '3.2.3')])), ('source', CommentedMap([('fn', 'org.Ce.eg.db_3.2.3.tar.gz'), ('url', 'https://bioc.ism.ac.jp/packages/3.2/data/annotation/src/contrib/org.Ce.eg.db_3.2.3.tar.gz'), ('md5', '7c85c6469114956030ec18f8952238d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Ce.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Ce.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Worm, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.cf.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.cf.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.cf.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Cf.eg.db_3.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Cf.eg.db_3.4.1.tar.gz'), ('md5', '6d0cefa74e5aaf46ce487497ea783b0c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Cf.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Cf.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Canine, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.dm.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.dm.eg.db/3.3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.dm.eg.db'), ('version', '3.3.0')])), ('source', CommentedMap([('fn', 'org.Dm.eg.db_3.3.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Dm.eg.db_3.3.0.tar.gz'), ('md5', 'ea55478a0a914b3da40d0437db06616e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.10', 'r >=2.7.0']), ('run', ['bioconductor-annotationdbi >=1.33.10', 'r >=2.7.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Dm.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Dm.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Fly, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.dr.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.dr.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.dr.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Dr.eg.db_3.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Dr.eg.db_3.4.1.tar.gz'), ('md5', '266073d731a28c3177ffd46495b8d797')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Dr.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Dr.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Zebrafish, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.gg.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.gg.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.gg.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Gg.eg.db_3.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Gg.eg.db_3.4.1.tar.gz'), ('md5', 'c822d4520ae1b9b0b61a4e96e439509a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Gg.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Gg.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Chicken, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.hs.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.hs.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.hs.eg.db'), ('version', '3.3.0')])), ('source', CommentedMap([('fn', 'org.Hs.eg.db_3.3.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/data/annotation/src/contrib/org.Hs.eg.db_3.3.0.tar.gz', 'https://depot.galaxyproject.org/software/org.Hs.eg.db/org.Hs.eg.db_3.3.0_src_all.tar.gz']), ('md5', 'ef4a05f46e7cdb0ba23fcd898a4a1d57')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.10', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.33.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Hs.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/org.Hs.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Human, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.mm.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.mm.eg.db/3.3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.mm.eg.db'), ('version', '3.3.0')])), ('source', CommentedMap([('fn', 'org.Mm.eg.db_3.3.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/data/annotation/src/contrib/org.Mm.eg.db_3.3.0.tar.gz'), ('md5', '26246d219940da6a7fa6f85cc6a6b2b2')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.10', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.33.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Mm.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Mm.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Mouse, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.rn.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.rn.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.rn.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Rn.eg.db_3.4.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-org.rn.eg.db/bioconductor-org.rn.eg.db_3.4.1_src_all.tar.gz'), ('md5', 'b1eade0c815dc9a0d45cc5efb1e16c44')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Rn.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Rn.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Rat, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-org.ss.eg.db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-org.ss.eg.db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-org.ss.eg.db'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'org.Ss.eg.db_3.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Ss.eg.db_3.4.1.tar.gz'), ('md5', '77d3974ff50426fb803c82dec17be0b1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.37.4', 'r-base']), ('run', ['bioconductor-annotationdbi >=1.37.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'org.Ss.eg.db\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/org.Ss.eg.db.html'), ('license', 'Artistic-2.0'), ('summary', 'Genome wide annotation for Pig, primarily based on mapping using Entrez Gene identifiers.')]))])), 'bioconductor-organismdbi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-organismdbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-organismdbi'), ('version', '1.14.1')])), ('source', CommentedMap([('fn', 'OrganismDbi_1.14.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/OrganismDbi_1.14.1.tar.gz'), ('md5', 'd71faed4e6a8c573960bcb48b8f433e4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-genomicfeatures >=1.23.31', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-genomicfeatures >=1.23.31', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OrganismDbi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/OrganismDbi.html'), ('license', 'Artistic-2.0'), ('summary', 'The package enables a simple unified interface to several annotation packages each of which has its own schema by taking advantage of the fact that each of these packages implements a select methods.')]))])), 'bioconductor-osat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-osat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-osat'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'OSAT_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OSAT_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/OSAT_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/OSAT/OSAT_1.24.0_src_all.tar.gz']), ('md5', 'f45375c9d6f49b7049c1c9285b64a3bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OSAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OSAT.html'), ('license', 'Artistic-2.0'), ('summary', 'A sizable genomics study such as microarray often involves the use of multiple batches (groups) of experiment due to practical complication. To minimize batch effects, a careful experiment design should ensure the even distribution of biological groups and confounding factors across batches. OSAT (Optimal Sample Assignment Tool) is developed to facilitate the allocation of collected samples to different batches. With minimum steps, it produces setup that optimizes the even distribution of samples in groups of biological interest into different batches, reducing the confounding or correlation between batches and the biological variables of interest. It can also optimize the even distribution of confounding factors across batches. Our tool can handle challenging instances where incomplete and unbalanced sample collections are involved as well as ideal balanced RCBD. OSAT provides a number of predefined layout for some of the most commonly used genomics platform. Related paper can be find at http://www.biomedcentral.com/1471-2164/13/689 .')]))])), 'bioconductor-oscope': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oscope/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oscope'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Oscope_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Oscope_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/Oscope_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Oscope/Oscope_1.6.0_src_all.tar.gz']), ('md5', 'ce210cd09760b511bddb8c1f12b56eb0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat']), ('run', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Oscope\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Oscope.html'), ('license', 'Artistic-2.0'), ('summary', 'Oscope is a statistical pipeline developed to identifying and recovering the base cycle profiles of oscillating genes in an unsynchronized single cell RNA-seq experiment. The Oscope pipeline includes three modules: a sine model module to search for candidate oscillator pairs; a K-medoids clustering module to cluster candidate oscillators into groups; and an extended nearest insertion module to recover the base cycle order for each oscillator group.')]))])), 'bioconductor-otubase': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-otubase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-otubase'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'OTUbase_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OTUbase_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/OTUbase_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/OTUbase/OTUbase_1.26.0_src_all.tar.gz']), ('md5', 'c5890afc116654b4d243ca6a60e49c35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan']), ('run', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OTUbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OTUbase.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides a platform for Operational Taxonomic Unit based analysis')]))])), 'bioconductor-outlierd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-outlierd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-outlierd'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'OutlierD_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OutlierD_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/OutlierD_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/OutlierD/OutlierD_1.40.0_src_all.tar.gz']), ('md5', '529ece912ce51d273345dda44d968c59')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-quantreg']), ('run', ['bioconductor-biobase', 'r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OutlierD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OutlierD.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package detects outliers using quantile regression on the M-A scatterplots of high-throughput data.')]))])), 'bioconductor-paircompviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-paircompviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-paircompviz'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'paircompviz_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/paircompviz_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/paircompviz_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/paircompviz/paircompviz_1.14.0_src_all.tar.gz']), ('md5', 'b329f0bceb502f17cfd26caf99b9a285')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rgraphviz', 'r-base']), ('run', ['bioconductor-rgraphviz', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'paircompviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/paircompviz.html'), ('license', 'GPL (>=3.0)'), ('summary', 'This package provides visualization of the results from the multiple (i.e. pairwise) comparison tests such as pairwise.t.test, pairwise.prop.test or pairwise.wilcox.test. The groups being compared are visualized as nodes in Hasse diagram. Such approach enables very clear and vivid depiction of which group is significantly greater than which others, especially if comparing a large number of groups.')]))])), 'bioconductor-pannbuilder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pannbuilder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pannbuilder'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'PAnnBuilder_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PAnnBuilder_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/PAnnBuilder_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/PAnnBuilder/PAnnBuilder_1.40.0_src_all.tar.gz']), ('md5', '61a204dada73cf39989cfa7d12e5912b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.3.12', 'bioconductor-biobase >=1.17.0', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.3.12', 'bioconductor-biobase >=1.17.0', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PAnnBuilder\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PAnnBuilder.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'Processing annotation data from public data repositories and building protein-centric annotation data packages.')]))])), 'bioconductor-panp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-panp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-panp'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'panp_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/panp_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/panp_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/panp/panp_1.46.0_src_all.tar.gz']), ('md5', 'e001987527a2b8e67d1066e36d75b293')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base']), ('run', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'panp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/panp.html'), ('license', 'GPL (>= 2)'), ('summary', 'A function to make gene presence/absence calls based on distance from negative strand matching probesets (NSMP) which are derived from Affymetrix annotation. PANP is applied after gene expression values are created, and therefore can be used after any preprocessing method such as MAS5 or GCRMA, or PM-only methods like RMA. NSMP sets have been established for the HGU133A and HGU133-Plus-2.0 chipsets to date.')]))])), 'bioconductor-papi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-papi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-papi'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PAPi_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PAPi_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PAPi_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PAPi/PAPi_1.16.0_src_all.tar.gz']), ('md5', '428a61cedc96ed775381b0c4798e8cb5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-keggrest', 'r-base', 'r-svdialogs']), ('run', ['bioconductor-keggrest', 'r-base', 'r-svdialogs'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PAPi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PAPi.html'), ('license', 'GPL(>= 2)'), ('summary', 'The Pathway Activity Profiling - PAPi - is an R package for predicting the activity of metabolic pathways based solely on a metabolomics data set containing a list of metabolites identified and their respective abundances in different biological samples. PAPi generates hypothesis that improves the final biological interpretation. See Aggio, R.B.M; Ruggiero, K. and Villas-Boas, S.G. (2010) - Pathway Activity Profiling (PAPi): from metabolite profile to metabolic pathway activity. Bioinformatics.')]))])), 'bioconductor-parody': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-parody/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-parody'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'parody_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/parody_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/parody_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/parody/parody_1.34.0_src_all.tar.gz']), ('md5', '9dca708335f65ef8799b7b19247f1700')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'parody\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/parody.html'), ('license', 'Artistic-2.0'), ('summary', 'routines for univariate and multivariate outlier detection with a focus on parametric methods, but support for some methods based on resistant statistics')]))])), 'bioconductor-path2ppi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-path2ppi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-path2ppi'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Path2PPI_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Path2PPI_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Path2PPI/Path2PPI_1.6.0_src_all.tar.gz']), ('md5', 'f1090e5beb965dda232dc020ed82278c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0.1']), ('run', ['r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Path2PPI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Path2PPI.html'), ('license', 'GPL (>= 2)'), ('summary', 'Package to predict protein-protein interaction (PPI) networks in target organisms for which only a view information about PPIs is available. Path2PPI predicts PPI networks based on sets of proteins which can belong to a certain pathway from well-established model organisms. It helps to combine and transfer information of a certain pathway or biological process from several reference organisms to one target organism. Path2PPI only depends on the sequence similarity of the involved proteins.')]))])), 'bioconductor-pathnet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathnet'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PathNet_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PathNet_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PathNet_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PathNet/PathNet_1.16.0_src_all.tar.gz']), ('md5', '35fd5d2a6f7365635db7f0f225dcd0a5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PathNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PathNet.html'), ('license', 'GPL-3'), ('summary', 'PathNet uses topological information present in pathways and differential expression levels of genes (obtained from microarray experiment) to identify pathways that are 1) significantly enriched and 2) associated with each other in the context of differential expression. The algorithm is described in: PathNet: A tool for pathway analysis using topological information. Dutta B, Wallqvist A, and Reifman J. Source Code for Biology and Medicine 2012 Sep 24;7(1):10.')]))])), 'bioconductor-pathview': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathview'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pathview_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pathview_<{ version }}.tar.gz'), ('sha256', '21db53ffa15bea36c1c6699eee60eae9537f078b6d68806e9defe062652ea9fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pathview\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pathview.html'), ('license', 'GPL (>=3.0)'), ('summary', 'Pathview is a tool set for pathway based data integration and visualization. It maps and renders a wide variety of biological data on relevant pathway graphs. All users need is to supply their data and specify the target pathway. Pathview automatically downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, Pathview also seamlessly integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.')]))])), 'bioconductor-pcaexplorer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcaexplorer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcaexplorer'), ('version', '2.0.0')])), ('source', CommentedMap([('fn', 'pcaExplorer_2.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pcaExplorer_2.0.0.tar.gz'), ('md5', '456fefb3d662f833b26a86810f26d692')])), ('build', CommentedMap([('skip', True), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pcaExplorer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html'), ('license', 'MIT + file LICENSE'), ('summary', 'This package provides functionality for interactive visualization of RNA-seq datasets based on Principal Components Analysis. The methods provided allow for quick information extraction and effective data exploration. A Shiny application encapsulates the whole analysis.')]))])), 'bioconductor-pcamethods': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcamethods/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcamethods'), ('version', '1.64.0')])), ('source', CommentedMap([('fn', 'pcaMethods_1.64.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pcaMethods_1.64.0.tar.gz'), ('md5', '301af6f5250cdefbb002170ea6f24ab4')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcpp >=0.11.3', 'r-mass', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcpp >=0.11.3', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pcaMethods\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pcaMethods.html'), ('license', 'GPL (>= 3)'), ('summary', 'Provides Bayesian PCA, Probabilistic PCA, Nipals PCA, Inverse Non-Linear PCA and the conventional SVD PCA. A cluster based method for missing value estimation is included for comparison. BPCA, PPCA and NipalsPCA may be used to perform PCA on incomplete data as well as for accurate missing value estimation. A set of methods for printing and plotting the results is also provided. All PCA methods make use of the same data structure (pcaRes) to provide a common interface to the PCA results. Initiated at the Max-Planck Institute for Molecular Plant Physiology, Golm, Germany.')]))])), 'bioconductor-pcan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcan'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'PCAN_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PCAN_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/PCAN_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/PCAN/PCAN_1.4.0_src_all.tar.gz']), ('md5', 'ac243936d5a2e1c52abeb00ce1ea8372')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base']), ('run', ['bioconductor-biocparallel', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PCAN.html'), ('license', 'CC BY-NC-ND 4.0'), ('summary', 'Phenotypes comparison based on a pathway consensus approach. Assess the relationship between candidate genes and a set of phenotypes based on additional genes related to the candidate (e.g. Pathways or network neighbors).')]))])), 'bioconductor-pchicdata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pchicdata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pchicdata'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'PCHiCdata_1.2.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/data/experiment/src/contrib/PCHiCdata_1.2.0.tar.gz'), ('md5', '35d4564d71a1f06afc369b890cfa8047')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-chicago', 'r-base']), ('run', ['bioconductor-chicago', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PCHiCdata\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/PCHiCdata.html'), ('license', 'Artistic-2.0'), ('summary', 'Subsets of Promoter Capture Hi-C data conveniently packaged for Chicago users. Data includes interactions detected for chromosomes 20 and 21 in GM12878 cells and for chromosomes 18 and 19 in mESC.')]))])), 'bioconductor-peca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-peca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-peca'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'PECA_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PECA_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/PECA_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/PECA/PECA_1.12.0_src_all.tar.gz']), ('md5', '854f5e5a4abfea8cc18b6404c516fcc0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-rots', 'r-aroma.affymetrix', 'r-aroma.core', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-rots', 'r-aroma.affymetrix', 'r-aroma.core', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PECA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PECA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Calculates Probe-level Expression Change Averages (PECA) to identify differential expression in Affymetrix gene expression microarray studies or in proteomic studies using peptide-level mesurements respectively.')]))])), 'bioconductor-pepstat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepstat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepstat'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepStat_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepStat_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepStat_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepStat/pepStat_1.10.0_src_all.tar.gz']), ('md5', '37bf864d9516d0a41f16c5c79d1d2591')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepStat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepStat.html'), ('license', 'Artistic-2.0'), ('summary', 'Statistical analysis of peptide microarrays')]))])), 'bioconductor-pepxmltab': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepxmltab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepxmltab'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepXMLTab_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepXMLTab_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepXMLTab_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepXMLTab/pepXMLTab_1.10.0_src_all.tar.gz']), ('md5', '3d0ccaed81b9dd5dba8fb0fb7fab829e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml >=3.98-1.1']), ('run', ['r-base', 'r-xml >=3.98-1.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepXMLTab\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepXMLTab.html'), ('license', 'Artistic-2.0'), ('summary', 'Parsing pepXML files based one XML package. The package tries to handle pepXML files generated from different softwares. The output will be a peptide-spectrum-matching tabular file. The package also provide function to filter the PSMs based on FDR.')]))])), 'bioconductor-phyloseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-phyloseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-phyloseq'), ('version', '1.19.1')])), ('source', CommentedMap([('fn', 'phyloseq_1.19.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/phyloseq_1.19.1.tar.gz'), ('md5', 'b8daecd87cf8528c8abed4ab0b8729d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'phyloseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/phyloseq.html'), ('license', 'AGPL-3'), ('summary', 'phyloseq provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.')]))])), 'bioconductor-piano': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-piano/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-piano'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'piano_1.12.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/piano_1.12.1.tar.gz'), ('md5', '38972b1001c9c336cb0e2f6b5368c832')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations', 'libgcc', 'llvm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'piano\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/piano.html'), ('license', 'GPL (>=2)'), ('summary', 'Piano performs gene set analysis using various statistical methods, from different gene level statistics and a wide range of gene-set collections. Furthermore, the Piano package contains functions for combining the results of multiple runs of gene set analyses.')]))])), 'bioconductor-pickgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pickgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pickgene'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'pickgene_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pickgene_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/pickgene_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/pickgene/pickgene_1.48.0_src_all.tar.gz']), ('md5', 'c1dafca6dc33faac3f80efedd37dd6f8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pickgene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pickgene.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions to Analyze Microarray (Gene Expression) Data.')]))])), 'bioconductor-pics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pics'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'PICS_2.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PICS_2.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/PICS_2.20.0.tar.gz', 'https://depot.galaxyproject.org/software/PICS/PICS_2.20.0_src_all.tar.gz']), ('md5', '9a4ac1081ae3a764caae3774aefd55b4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PICS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PICS.html'), ('license', 'Artistic-2.0'), ('summary', 'Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.')]))])), 'bioconductor-pkgdeptools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pkgdeptools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pkgdeptools'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'pkgDepTools_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pkgDepTools_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/pkgDepTools_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/pkgDepTools/pkgDepTools_1.42.0_src_all.tar.gz']), ('md5', '5ccffeff4f1e1d22577771a338b88585')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pkgDepTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pkgDepTools.html'), ('license', 'GPL-2'), ('summary', 'This package provides tools for computing and analyzing dependency relationships among R packages.  It provides tools for building a graph-based representation of the dependencies among all packages in a list of CRAN-style package repositories.  There are also utilities for computing installation order of a given package.  If the RCurl package is available, an estimate of the download size required to install a given package and its dependencies can be obtained.')]))])), 'bioconductor-plgem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plgem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plgem'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'plgem_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plgem_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/plgem_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/plgem/plgem_1.48.0_src_all.tar.gz']), ('md5', 'c501ba945aba8b65309f38785925cab0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plgem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plgem.html'), ('license', 'GPL-2'), ('summary', 'The Power Law Global Error Model (PLGEM) has been shown to faithfully model the variance-versus-mean dependence that exists in a variety of genome-wide datasets, including microarray and proteomics data. The use of PLGEM has been shown to improve the detection of differentially expressed genes or proteins in these datasets.')]))])), 'bioconductor-plier': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plier/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plier'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'plier_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plier_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/plier_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/plier/plier_1.46.0_src_all.tar.gz']), ('md5', 'b8745e56a92d4ac4c1637e1e84ab76c2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plier\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plier.html'), ('license', 'GPL (>= 2)'), ('summary', 'The PLIER (Probe Logarithmic Error Intensity Estimate) method produces an improved signal by accounting for experimentally observed patterns in probe behavior and handling error at the appropriately at low and high signal values.')]))])), 'bioconductor-plw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plw'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'plw_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plw_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/plw/plw_1.36.0_src_all.tar.gz']), ('md5', 'e8a4447aeb071da118ad6c79b39c9998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass']), ('run', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plw\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plw.html'), ('license', 'GPL-2'), ('summary', 'Probe level Locally moderated Weighted median-t (PLW) and Locally Moderated Weighted-t (LMW).')]))])), 'bioconductor-podkat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-podkat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-podkat'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'podkat_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/podkat_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/podkat_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/podkat/podkat_1.8.0_src_all.tar.gz']), ('md5', '9737525b87ec32df82788005e106a37b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.32.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-matrix', 'r-rcpp >=0.11.1', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.32.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-matrix', 'r-rcpp >=0.11.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'podkat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/podkat.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides an association test that is capable of dealing with very rare and even private variants. This is accomplished by a kernel-based approach that takes the positions of the variants into account. The test can be used for pre-processed matrix data, but also directly for variant data stored in VCF files. Association testing can be performed whole-genome, whole-exome, or restricted to pre-defined regions of interest. The test is complemented by tools for analyzing and visualizing the results.')]))])), 'bioconductor-polyfit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-polyfit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-polyfit'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Polyfit_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Polyfit_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Polyfit_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Polyfit/Polyfit_1.10.0_src_all.tar.gz']), ('md5', 'c07c8cbf0bf82f73e22a2b15892e8187')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-deseq', 'r-base']), ('run', ['bioconductor-deseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Polyfit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Polyfit.html'), ('license', 'GPL (>= 3)'), ('summary', 'Polyfit is an add-on to the packages DESeq which ensures the p-value distribution is uniform over the interval [0, 1] for data satisfying the null hypothesis of no differential expression, and uses an adpated Storey-Tibshiran method to calculate q-values.')]))])), 'bioconductor-prada': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-prada/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-prada'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'prada_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/prada_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/prada_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/prada/prada_1.52.0_src_all.tar.gz']), ('md5', '32323c2251cc7bc1b9e40620ce77facd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-rcolorbrewer', 'r-rrcov']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-rcolorbrewer', 'r-rrcov'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'prada\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/prada.html'), ('license', 'LGPL'), ('summary', 'Tools for analysing and navigating data from high-throughput phenotyping experiments based on cellular assays and fluorescent detection (flow cytometry (FACS), high-content screening microscopy).')]))])), 'bioconductor-preprocesscore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-preprocesscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-preprocesscore'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'preprocessCore_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/preprocessCore_1.34.0.tar.gz'), ('md5', '845eddebdf2c68a93103622ac9cc31a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'preprocessCore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/preprocessCore.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A library of core preprocessing routines')]))])), 'bioconductor-probamr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-probamr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-probamr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'proBAMr_1.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/proBAMr_1.8.0.tar.gz'), ('md5', '438ad7d0502c170a0a7c29a17dc592cb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biostrings', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biostrings', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'proBAMr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/proBAMr.html'), ('license', 'Artistic-2.0'), ('summary', 'Mapping PSMs back to genome. The package builds SAM file from shotgun proteomics data The package also provides function to prepare annotation from GTF file.')]))])), 'bioconductor-profia': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-profia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-profia'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'proFIA_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/proFIA_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/proFIA/proFIA_1.2.0_src_all.tar.gz']), ('md5', '1298970f476ef1086577e96a66062c66')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'proFIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/proFIA.html'), ('license', 'CeCILL'), ('summary', 'Flow Injection Analysis coupled to High-Resolution Mass Spectrometry is a promising approach for high-throughput metabolomics. FIA- HRMS data, however, cannot be pre-processed with current software tools which rely on liquid chromatography separation, or handle low resolution data only. Here we present the proFIA package, which implements a new methodology to pre-process FIA-HRMS raw data (netCDF, mzData, mzXML, and mzML) including noise modelling and injection peak reconstruction, and generate the peak table. The workflow includes noise modelling, band detection and filtering then signal matching and missing value imputation. The peak table can then be exported as a .tsv file for further analysis. Visualisations to assess the quality of the data and of the signal made are easely produced.')]))])), 'bioconductor-promise': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-promise/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-promise'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'PROMISE_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROMISE_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROMISE_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/PROMISE/PROMISE_1.28.0_src_all.tar.gz']), ('md5', 'ebb9cf1b6d9d5b580753c9c24ab49730')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROMISE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROMISE.html'), ('license', 'GPL (>= 2)'), ('summary', 'A general tool to identify genomic features with a specific biologically interesting pattern of associations with multiple endpoint variables as described in Pounds et. al. (2009) Bioinformatics 25: 2013-2019')]))])), 'bioconductor-proper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-proper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-proper'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'PROPER_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROPER_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROPER_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/PROPER/PROPER_1.8.0_src_all.tar.gz']), ('md5', 'ada1ccfbcd1f50a3715bbca5ecddf46e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'r-base']), ('run', ['bioconductor-edger', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROPER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROPER.html'), ('license', 'GPL'), ('summary', 'This package provide simulation based methods for evaluating the statistical power in differential expression analysis from RNA-seq data.')]))])), 'bioconductor-protgenerics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-protgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-protgenerics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ProtGenerics_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ProtGenerics_1.4.0.tar.gz'), ('md5', '129914ba3eba2a8efcc7995d0656a8cf')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ProtGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ProtGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by Bioconductor proteomics packages.')]))])), 'bioconductor-psea': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-psea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-psea'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'PSEA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PSEA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/PSEA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/PSEA/PSEA_1.10.0_src_all.tar.gz']), ('md5', '544a65f736601f4acdb44e409639fc2b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PSEA.html'), ('license', 'Artistic-2.0'), ('summary', 'Deconvolution of gene expression data by Population-Specific Expression Analysis (PSEA).')]))])), 'bioconductor-psicquic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-psicquic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-psicquic'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'PSICQUIC_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PSICQUIC_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/PSICQUIC_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/PSICQUIC/PSICQUIC_1.14.0_src_all.tar.gz']), ('md5', '1db512605bb1da83e7ce5371a99db009')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSICQUIC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PSICQUIC.html'), ('license', 'Apache License 2.0'), ('summary', 'PSICQUIC is a project within the HUPO Proteomics Standard Initiative (HUPO-PSI).  It standardises programmatic access to molecular interaction databases.')]))])), 'bioconductor-purecn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-purecn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-purecn'), ('version', '1.6.3')])), ('source', CommentedMap([('fn', 'PureCN_1.6.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PureCN_1.6.3.tar.gz', 'https://depot.galaxyproject.org/software/PureCN/PureCN_1.6.3_src_all.tar.gz']), ('md5', '9dce6c929693f3d4967e019b83e79fef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PureCN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PureCN.html'), ('license', 'Artistic-2.0'), ('summary', 'This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality. PureCN is designed for targeted short read sequencing data, integrates well with standard somatic variant detection and copy number pipelines, and has support for tumor samples without matching normal samples.')]))])), 'bioconductor-pvac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pvac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pvac'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'pvac_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pvac_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/pvac_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/pvac/pvac_1.24.0_src_all.tar.gz']), ('md5', '2a9ca3fc296ccadb3769422a20869e67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pvac\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pvac.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'The package contains the function for filtering genes by the proportion of variation accounted for by the first principal component (PVAC).')]))])), 'bioconductor-pviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Pviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Pviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Pviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Pviz/Pviz_1.10.0_src_all.tar.gz']), ('md5', '55a484071ac1974537ab7b70163b8896')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Pviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Pviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Pviz adapts the Gviz package for protein sequences and data.')]))])), 'bioconductor-qdnaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qdnaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qdnaseq'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'QDNAseq_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/QDNAseq_1.10.0.tar.gz'), ('md5', 'd46bbb4166bdeeb8be196a3047e28963')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0']), ('run', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'QDNAseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/QDNAseq.html'), ('license', 'GPL'), ('summary', 'Quantitative DNA sequencing for chromosomal aberrations. The genome is divided into non-overlapping fixed-sized bins, number of sequence reads in each counted, adjusted with a simultaneous two-dimensional loess correction for sequence mappability and GC content, and filtered to remove spurious regions in the genome. Downstream steps of segmentation and calling are also implemented via packages DNAcopy and CGHcall, respectively.')]))])), 'bioconductor-qpcrnorm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qpcrnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qpcrnorm'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'qpcrNorm_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qpcrNorm_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/qpcrNorm_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/qpcrNorm/qpcrNorm_1.34.0_src_all.tar.gz']), ('md5', '83dec7f7f085cd88683b982104254259')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qpcrNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qpcrNorm.html'), ('license', 'LGPL (>= 2)'), ('summary', 'The package contains functions to perform normalization of high-throughput qPCR data. Basic functions for processing raw Ct data plus functions to generate diagnostic plots are also available.')]))])), 'bioconductor-qrqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qrqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qrqc'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'qrqc_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qrqc_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/qrqc_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/qrqc/qrqc_1.30.0_src_all.tar.gz']), ('md5', 'cd845c1919613e24d83aa91b4641afc9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qrqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qrqc.html'), ('license', 'GPL (>=2)'), ('summary', 'Quickly scans reads and gathers statistics on base and quality frequencies, read length, k-mers by position, and frequent sequences. Produces graphical output of statistics for use in quality control pipelines, and an optional HTML quality report. S4 SequenceSummary objects allow specific tests and functionality to be written around the data collected.')]))])), 'bioconductor-quantro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantro'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'quantro_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantro_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantro_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/quantro/quantro_1.10.0_src_all.tar.gz']), ('md5', '0928efa544591e3f4e3c50147cda79d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantro.html'), ('license', 'GPL (>=3)'), ('summary', 'A data-driven test for the assumptions of quantile normalization using raw data such as objects that inherit eSets (e.g. ExpressionSet, MethylSet). Group level information about each sample (such as Tumor / Normal status) must also be provided because the test assesses if there are global differences in the distributions between the user-defined groups.')]))])), 'bioconductor-quantsmooth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantsmooth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantsmooth'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'quantsmooth_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantsmooth_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantsmooth_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/quantsmooth/quantsmooth_1.42.0_src_all.tar.gz']), ('md5', '09f500a2a2f6d55d66a40a73f66be4d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-quantreg']), ('run', ['r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantsmooth\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantsmooth.html'), ('license', 'GPL-2'), ('summary', 'Implements quantile smoothing as introduced in: Quantile smoothing of array CGH data; Eilers PH, de Menezes RX; Bioinformatics. 2005 Apr 1;21(7):1146-53.')]))])), 'bioconductor-qusage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qusage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qusage'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'qusage_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qusage_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/qusage_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/qusage/qusage_2.10.0_src_all.tar.gz']), ('md5', '7dbdfa04a1b8db789c84d0952841b45b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=3.14', 'r-base', 'r-lsmeans', 'r-nlme']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=3.14', 'r-base', 'r-lsmeans', 'r-nlme'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qusage\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qusage.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package is an implementation the Quantitative Set Analysis for Gene Expression (QuSAGE) method described in (Yaari G. et al, Nucl Acids Res, 2013). This is a novel Gene Set Enrichment-type test, which is designed to provide a faster, more accurate, and easier to understand test for gene expression studies. qusage accounts for inter-gene correlations using the Variance Inflation Factor technique proposed by Wu et al. (Nucleic Acids Res, 2012). In addition, rather than simply evaluating the deviation from a null hypothesis with a single number (a P value), qusage quantifies gene set activity with a complete probability density function (PDF). From this PDF, P values and confidence intervals can be easily extracted. Preserving the PDF also allows for post-hoc analysis (e.g., pair-wise comparisons of gene set activity) while maintaining statistical traceability. Finally, while qusage is compatible with individual gene statistics from existing methods (e.g., LIMMA), a Welch-based method is implemented that is shown to improve specificity. For questions, contact Chris Bolen (cbolen1@gmail.com) or Steven Kleinstein (steven.kleinstein@yale.edu)')]))])), 'bioconductor-qvalue': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qvalue/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qvalue'), ('version', '2.6.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-qvalue/bioconductor-qvalue_2.6.0_src_all.tar.gz'), ('sha256', 'd8690a917dd548425387ca00a2c2ad1c8fbadbbc09e019ed4b22cfb0438032b6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-reshape2']), ('run', ['r-base', 'r-ggplot2', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qvalue\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/qvalue.html'), ('license', 'LGPL'), ('summary', \"This package takes a list of p-values resulting from the simultaneous testing of many hypotheses and estimates their q-values and local FDR values. The q-value of a test measures the proportion of false positives incurred (called the false discovery rate) when that particular test is called significant. The local FDR measures the posterior probability the null hypothesis is true given the test's p-value. Various plots are automatically generated, allowing one to make sensible significance cut-offs. Several mathematical results have recently been shown on the conservative accuracy of the estimated q-values from this software. The software can be applied to problems in genomics, brain imaging, astrophysics, and data mining.\")]))])), 'bioconductor-r3cseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r3cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r3cseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'r3Cseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/r3Cseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/r3Cseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/r3Cseq/r3Cseq_1.22.0_src_all.tar.gz']), ('md5', '46b95951434f664799a4cf783ace3bd8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'r3Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/r3Cseq.html'), ('license', 'GPL-3'), ('summary', 'This package is an implementation of data analysis for the long-range interactions from 3C-seq assay.')]))])), 'bioconductor-r4rna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r4rna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r4rna'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'R4RNA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/R4RNA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/R4RNA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/R4RNA/R4RNA_1.4.0_src_all.tar.gz']), ('md5', '1569e97b37150559d3ac7c82addf415a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.38.0', 'r-base']), ('run', ['bioconductor-biostrings >=2.38.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'R4RNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/R4RNA.html'), ('license', 'GPL-3'), ('summary', 'A package for RNA basepair analysis, including the visualization of basepairs as arc diagrams for easy comparison and annotation of sequence and structure.  Arc diagrams can additionally be projected onto multiple sequence alignments to assess basepair conservation and covariation, with numerical methods for computing statistics for each.')]))])), 'bioconductor-rain': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rain'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rain_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rain_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rain_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rain/rain_1.10.0_src_all.tar.gz']), ('md5', '7eb25126c0851e7feb3e576d1e910d91')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'r-base', 'r-gmp']), ('run', ['bioconductor-multtest', 'r-base', 'r-gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rain\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rain.html'), ('license', 'GPL-2'), ('summary', 'This package uses non-parametric methods to detect rhythms in time series. It deals with outliers, missing values and is optimized for time series comprising 10-100 measurements. As it does not assume expect any distinct waveform it is optimal or detecting oscillating behavior (e.g. circadian or cell cycle) in e.g. genome- or proteome-wide biological measurements such as: micro arrays, proteome mass spectrometry, or metabolome measurements.')]))])), 'bioconductor-rama': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rama/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rama'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'rama_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rama_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/rama_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/rama/rama_1.50.0_src_all.tar.gz']), ('md5', 'ba1171551c6b72493af478168bd0cbaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rama\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rama.html'), ('license', 'GPL (>= 2)'), ('summary', 'Robust estimation of cDNA microarray intensities with replicates. The package uses a Bayesian hierarchical model for the robust estimation. Outliers are modeled explicitly using a t-distribution, and the model also addresses classical issues such as design effects, normalization, transformation, and nonconstant variance.')]))])), 'bioconductor-ramigo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ramigo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ramigo'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RamiGO_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RamiGO_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RamiGO_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RamiGO/RamiGO_1.22.0_src_all.tar.gz']), ('md5', '746153d0c5edfd483a34ae3bd046d22b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl']), ('run', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RamiGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RamiGO.html'), ('license', 'Artistic-2.0'), ('summary', 'R interface sending requests to AmiGO visualize, retrieving DAG GO trees, parsing GraphViz DOT format files and exporting GML files for Cytoscape. Also uses RCytoscape to interactively display AmiGO trees in Cytoscape.')]))])), 'bioconductor-randpack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-randpack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-randpack'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'randPack_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/randPack_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/randPack_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/randPack/randPack_1.22.0_src_all.tar.gz']), ('md5', '0914bfeeeb98eec4683e397668d10d4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'randPack\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/randPack.html'), ('license', 'Artistic 2.0'), ('summary', 'A suite of classes and functions for randomizing patients in clinical trials.')]))])), 'bioconductor-rbcbook1': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbcbook1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbcbook1'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'RbcBook1_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RbcBook1_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/RbcBook1_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/RbcBook1/RbcBook1_1.44.0_src_all.tar.gz']), ('md5', 'dd996704b9c253ffb42eb80116e16d7f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-rpart']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-rpart'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RbcBook1\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RbcBook1.html'), ('license', 'Artistic-2.0'), ('summary', 'tools for building book')]))])), 'bioconductor-rbgl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbgl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbgl'), ('version', '1.48.1')])), ('source', CommentedMap([('fn', 'RBGL_1.48.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/RBGL_1.48.1.tar.gz'), ('md5', 'e012e1022b4392c9dd90677ac69798c6')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBGL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RBGL.html'), ('license', 'Artistic-2.0'), ('summary', 'A fairly extensive and comprehensive interface to the graph algorithms contained in the BOOST library.')]))])), 'bioconductor-rbioinf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbioinf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbioinf'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'RBioinf_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RBioinf_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/RBioinf_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/RBioinf/RBioinf_1.36.0_src_all.tar.gz']), ('md5', '474f950d9d6f39c07ecc6fce324d4b36')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBioinf\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RBioinf.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions and datasets and examples to accompany the monograph R For Bioinformatics.')]))])), 'bioconductor-rbm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbm'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RBM_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RBM_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RBM_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RBM/RBM_1.8.0_src_all.tar.gz']), ('md5', 'd773904a351f356c9a08a884f3af855e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RBM.html'), ('license', 'GPL (>= 2)'), ('summary', 'Use A Resampling-Based Empirical Bayes Approach to Assess Differential Expression in Two-Color Microarrays and RNA-Seq data sets.')]))])), 'bioconductor-rbowtie': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbowtie/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbowtie'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'Rbowtie_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rbowtie_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rbowtie_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/Rbowtie/Rbowtie_1.16.0_src_all.tar.gz']), ('md5', '36b323e887779324cce1b8d9e0ee332d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rbowtie\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rbowtie.html'), ('license', 'Artistic-1.0 | file LICENSE'), ('summary', 'This package provides an R wrapper around the popular bowtie short read aligner and around SpliceMap, a de novo splice junction discovery and alignment tool. The package is used by the QuasR bioconductor package. We recommend to use the QuasR package instead of using Rbowtie directly.')]))])), 'bioconductor-rbsurv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbsurv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbsurv'), ('version', '2.34.0')])), ('source', CommentedMap([('fn', 'rbsurv_2.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rbsurv_2.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/rbsurv_2.34.0.tar.gz', 'https://depot.galaxyproject.org/software/rbsurv/rbsurv_2.34.0_src_all.tar.gz']), ('md5', '3f7ac25ebdeb23271b55987a8a8dd8b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rbsurv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rbsurv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package selects genes associated with survival.')]))])), 'bioconductor-rcas': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcas'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'v1.1.1.tar.gz'), ('url', 'https://github.com/BIMSBbioinfo/RCAS/archive/v1.1.1.tar.gz'), ('md5', 'c97766ff5ad5fa58da34a3451fdb4354')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix']), ('run', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCAS\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/BIMSBbioinfo/RCAS/tree/rcas_R/rpackage/RCAS'), ('license', 'Artistic-2.0'), ('summary', 'RNA Centric Annotation System (RCAS) is an automated system that provides dynamic annotations for custom input files that contain transcriptomic targe regions.')]))])), 'bioconductor-rcaspar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcaspar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcaspar'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RCASPAR_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RCASPAR_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RCASPAR_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RCASPAR/RCASPAR_1.22.0_src_all.tar.gz']), ('md5', 'd5c6b6dfeedee31203b56b1a9b9e1d9c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCASPAR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RCASPAR.html'), ('license', 'GPL (>=3)'), ('summary', 'The package is the R-version of the C-based software \\\\bold{CASPAR} (Kaderali,2006: \\\\url{http://bioinformatics.oxfordjournals.org/content/22/12/1495}). It is meant to help predict survival times in the presence of high-dimensional explanatory covariates. The model is a piecewise baseline hazard Cox regression model with an Lq-norm based prior that selects for the most important regression coefficients, and in turn the most relevant covariates for survival analysis. It was primarily tried on gene expression and aCGH data, but can be used on any other type of high-dimensional data and in disciplines other than biology and medicine.')]))])), 'bioconductor-rcytoscape': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcytoscape/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcytoscape'), ('version', '1.21.1')])), ('source', CommentedMap([('fn', 'RCytoscape_1.21.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RCytoscape_1.21.1.tar.gz'), ('md5', '06aa503a70ada9bae79b38cc5de1eba2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCytoscape\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RCytoscape.html'), ('license', 'GPL-2'), ('summary', 'Interactvive viewing and exploration of graphs, connecting R to Cytoscape.')]))])), 'bioconductor-rdgidb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rdgidb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rdgidb'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'rDGIdb_1.2.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rDGIdb_1.2.1.tar.gz', 'https://bioarchive.galaxyproject.org/rDGIdb_1.2.1.tar.gz', 'https://depot.galaxyproject.org/software/rDGIdb/rDGIdb_1.2.1_src_all.tar.gz']), ('md5', 'f5b16f45076b3bcacd4fb1fc2659ba04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-httr', 'r-jsonlite']), ('run', ['r-base', 'r-httr', 'r-jsonlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rDGIdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rDGIdb.html'), ('license', 'MIT + file LICENSE'), ('summary', 'The rDGIdb package provides a wrapper for the Drug Gene Interaction Database (DGIdb). For simplicity, the wrapper query function and output resembles the user interface and results format provided on the DGIdb website (http://dgidb.genome.wustl.edu/).')]))])), 'bioconductor-readqpcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-readqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-readqpcr'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'ReadqPCR_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ReadqPCR_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/ReadqPCR_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/ReadqPCR/ReadqPCR_1.22.0_src_all.tar.gz']), ('md5', '356a63511d979779b0ec27dd1c393b10')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ReadqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ReadqPCR.html'), ('license', 'LGPL-3'), ('summary', 'The package provides functions to read raw RT-qPCR data of different platforms.')]))])), 'bioconductor-recoup': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-recoup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-recoup'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'recoup_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/recoup_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/recoup_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/recoup/recoup_1.4.0_src_all.tar.gz']), ('md5', 'b05790f192cee6ed1bec0031f6bae3c5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'recoup\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/recoup.html'), ('license', 'GPL (>= 3)'), ('summary', 'recoup calculates and plots signal profiles created from short sequence reads derived from Next Generation Sequencing technologies. The profiles provided are either sumarized curve profiles or heatmap profiles. Currently, recoup supports genomic profile plots for reads derived from ChIP-Seq and RNA-Seq experiments. The package uses ggplot2 and ComplexHeatmap graphics facilities for curve and heatmap coverage profiles respectively.')]))])), 'bioconductor-regioner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-regioner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-regioner'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'regioneR_1.6.2.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-regioner/bioconductor-regioner_1.6.2_src_all.tar.gz'), ('md5', '93ecdf4d04f690580c0f1681bddc1246')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'regioneR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/regioneR.html'), ('license', 'Artistic-2.0'), ('summary', 'regioneR offers a statistical framework based on customizable permutation tests to assess the association between genomic region sets and other genomic features.')]))])), 'bioconductor-repitools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-repitools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-repitools'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'Repitools_1.20.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-repitools/bioconductor-repitools_1.20.0_src_all.tar.gz'), ('md5', '3f27f4e075425ed0b3a9bb47bcaf8aaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster']), ('run', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Repitools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Repitools.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Tools for the analysis of enrichment-based epigenomic data.  Features include summarization and visualization of epigenomic data across promoters according to gene expression context, finding regions of differential methylation/binding, BayMeth for quantifying methylation etc.')]))])), 'bioconductor-rfpred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rfpred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rfpred'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'rfPred_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rfPred_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/rfPred_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/rfPred/rfPred_1.14.0_src_all.tar.gz']), ('md5', 'a509fb9fe3f7fd2ac7bfc6b19fbc74bf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-data.table']), ('run', ['bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-data.table'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rfPred\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rfPred.html'), ('license', 'GPL (>=2 )'), ('summary', 'Based on external numerous data files where rfPred scores are pre-calculated on all genomic positions of the human exome, the package gives rfPred scores to missense variants identified by the chromosome, the position (hg19 version), the referent and alternative nucleotids and the uniprot identifier of the protein. Note that for using the package, the user has to be connected on the Internet or to download the TabixFile and index (approximately 3.3 Go).')]))])), 'bioconductor-rgadem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgadem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgadem'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'rGADEM_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/rGADEM_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/rGADEM_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/rGADEM/rGADEM_2.22.0_src_all.tar.gz']), ('md5', '68855b9cecf4cb029027b6df0a0b6b2d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rGADEM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rGADEM.html'), ('license', 'Artistic-2.0'), ('summary', 'rGADEM is an efficient de novo motif discovery tool for large-scale genomic sequence data. It is an open-source R package, which is based on the GADEM software.')]))])), 'bioconductor-rgalaxy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgalaxy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgalaxy'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'RGalaxy_1.18.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RGalaxy_1.18.0.tar.gz'), ('md5', '51d1b7553447676c5a48d48133ba6902')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGalaxy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RGalaxy.html'), ('license', 'Artistic-2.0'), ('summary', 'Given an R function and its manual page, make the documented function available in Galaxy.')]))])), 'bioconductor-rgraph2js': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraph2js/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraph2js'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'RGraph2js_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGraph2js_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGraph2js_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/RGraph2js/RGraph2js_1.4.0_src_all.tar.gz']), ('md5', '8c976e7f4782606ff7c209333c47ecf1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker']), ('run', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGraph2js\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGraph2js.html'), ('license', 'GPL-2'), ('summary', 'Generator of web pages which display interactive network/graph visualizations with D3js, jQuery and Raphael.')]))])), 'bioconductor-rgraphviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraphviz/2.13.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraphviz'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Rgraphviz_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rgraphviz_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rgraphviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html'), ('license', 'EPL'), ('summary', 'Interfaces R with the AT and T graphviz library for plotting R graph objects from the graph package.')]))])), 'bioconductor-rgreat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgreat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgreat'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'rGREAT_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rGREAT_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/rGREAT_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/rGREAT/rGREAT_1.8.0_src_all.tar.gz']), ('md5', '27fae08a68adaa6135412d6fa403f8bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-getoptlong >=0.0.9', 'r-rcurl', 'r-rjson']), ('run', ['bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-getoptlong >=0.0.9', 'r-rcurl', 'r-rjson'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rGREAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rGREAT.html'), ('license', 'GPL (>= 2)'), ('summary', \"This package makes GREAT (Genomic Regions Enrichment of Annotations Tool) analysis automatic by constructing a HTTP POST request according to user's input and automatically retrieving results from GREAT web server.\")]))])), 'bioconductor-rgsea': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgsea'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'RGSEA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGSEA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGSEA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/RGSEA/RGSEA_1.10.0_src_all.tar.gz']), ('md5', '95fe347168bedf66f6005b4b39262d9f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGSEA.html'), ('license', 'GPL(>=3)'), ('summary', 'Combining bootstrap aggregating and Gene set enrichment analysis (GSEA), RGSEA is a classfication algorithm with high robustness and no over-fitting problem. It performs well especially for the data generated from different exprements.')]))])), 'bioconductor-rgsepd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgsepd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgsepd'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'rgsepd_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rgsepd_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/rgsepd_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/rgsepd/rgsepd_1.8.0_src_all.tar.gz']), ('md5', 'f46c8807b54858e819528559af19cbf7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-go.db', 'bioconductor-goseq >=1.17', 'bioconductor-org.hs.eg.db', 'bioconductor-summarizedexperiment', 'r-base', 'r-gplots', 'r-hash']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-go.db', 'bioconductor-goseq >=1.17', 'bioconductor-org.hs.eg.db', 'bioconductor-summarizedexperiment', 'r-base', 'r-gplots', 'r-hash'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rgsepd\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rgsepd.html'), ('license', 'GPL-3'), ('summary', 'R/GSEPD is a bioinformatics package for R to help disambiguate transcriptome samples (a matrix of RNA-Seq counts at RefSeq IDs) by automating differential expression (with DESeq2), then gene set enrichment (with GOSeq), and finally a N-dimensional projection to quantify in which ways each sample is like either treatment group.')]))])), 'bioconductor-rhdf5': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rhdf5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rhdf5'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'rhdf5_2.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/rhdf5_2.16.0.tar.gz'), ('md5', 'e583de82b6ac3dbd69d7ea1fd50c3c08'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base', 'gcc', 'llvm']), ('run', ['bioconductor-zlibbioc', 'r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rhdf5\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/rhdf5.html'), ('license', 'Artistic-2.0'), ('summary', \"This R/Bioconductor package provides an interface between HDF5 and R. HDF5's main features are the ability to store and access very large and/or complex datasets and a wide variety of metadata on mass storage (disk) through a completely portable file format. The rhdf5 package is thus suited for the exchange of large and/or complex datasets between R and other software package, and for letting R applications work on datasets that are larger than the available RAM.\")]))])), 'bioconductor-rhtslib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rhtslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rhtslib'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Rhtslib_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rhtslib_1.6.0.tar.gz'), ('md5', 'a4749fbe0ac9bcbc544decde28dac713'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-zlibbioc', 'autoconf 2.69 pl5.20.3.1_4', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-zlibbioc', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rhtslib\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rhtslib.html'), ('license', 'LGPL (>= 2)'), ('summary', 'This package provides version 1.1 of the \"HTSlib\" C library for high-throughput sequence analysis. The package is primarily useful to developers of other R packages who wish to make use of HTSlib. Motivation and instructions for use of this package are in the vignette, vignette(package=\"Rhtslib\", \"Rhtslib\").')]))])), 'bioconductor-riboprofiling': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboprofiling/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboprofiling'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'RiboProfiling_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RiboProfiling_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RiboProfiling_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RiboProfiling/RiboProfiling_1.6.0_src_all.tar.gz']), ('md5', 'cb557c96cdcd6a681da0cffe90ed616a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RiboProfiling\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RiboProfiling.html'), ('license', 'GPL-3'), ('summary', \"Starting with a BAM file, this package provides the necessary functions for quality assessment, read start position recalibration, the counting of reads on CDS, 3'UTR, and 5'UTR, plotting of count data: pairs, log fold-change, codon frequency and coverage assessment, principal component analysis on codon coverage.\")]))])), 'bioconductor-riboseqr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboseqr'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'riboSeqR_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/riboSeqR_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/riboSeqR_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/riboSeqR/riboSeqR_1.10.0_src_all.tar.gz']), ('md5', '422b50a08dd8ba4b1e7bde5935806998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base']), ('run', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'riboSeqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/riboSeqR.html'), ('license', 'GPL-3'), ('summary', 'Plotting functions, frameshift detection and parsing of sequencing data from ribosome profiling experiments.')]))])), 'bioconductor-rimmport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rimmport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rimmport'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'RImmPort_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RImmPort_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/RImmPort/RImmPort_1.4.2_src_all.tar.gz']), ('md5', '26cb626afa74b29ca5219e078b1ce8f2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf']), ('run', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RImmPort\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RImmPort.html'), ('license', 'GPL-3'), ('summary', 'The RImmPort package simplifies access to ImmPort data for analysis in the R environment. It provides a standards-based interface to the ImmPort study data that is in a proprietary format.')]))])), 'bioconductor-ringo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ringo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ringo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Ringo_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/Ringo_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Ringo_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Ringo/Ringo_1.38.0_src_all.tar.gz']), ('md5', '17a1b47d16f09dd4525f26b272c48539')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Ringo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Ringo.html'), ('license', 'Artistic-2.0'), ('summary', 'The package Ringo facilitates the primary analysis of ChIP-chip data. The main functionalities of the package are data read-in, quality assessment, data visualisation and identification of genomic regions showing enrichment in ChIP-chip. The package has functions to deal with two-color oligonucleotide microarrays from NimbleGen used in ChIP-chip projects, but also contains more general functions for ChIP-chip data analysis, given that the data is supplied as RGList (raw)  or ExpressionSet (pre- processed). The package employs functions from various other packages of the Bioconductor project and provides additional ChIP-chip-specific and NimbleGen-specific functionalities.')]))])), 'bioconductor-ripseeker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ripseeker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ripseeker'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RIPSeeker_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RIPSeeker_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RIPSeeker_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RIPSeeker/RIPSeeker_1.16.0_src_all.tar.gz']), ('md5', '41eb01d93e0fda73703df182d861b5c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RIPSeeker\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RIPSeeker.html'), ('license', 'GPL-2'), ('summary', 'Infer and discriminate RIP peaks from RIP-seq alignments using two-state HMM with negative binomial emission probability. While RIPSeeker is specifically tailored for RIP-seq data analysis, it also provides a suite of bioinformatics tools integrated within this self-contained software package comprehensively addressing issues ranging from post-alignments processing to visualization and annotation.')]))])), 'bioconductor-risa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-risa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-risa'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'Risa_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Risa_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/Risa_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/Risa/Risa_1.18.0_src_all.tar.gz']), ('md5', 'cc2cbabbe34dd60ba8a10e1f3a6c6736')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase >=2.4.0', 'bioconductor-biocviews', 'bioconductor-xcms', 'r-base >=2.0.9', 'r-rcpp >=0.9.13', 'gcc', 'llvm', 'readline 6.2']), ('run', ['bioconductor-affy', 'bioconductor-biobase >=2.4.0', 'bioconductor-biocviews', 'bioconductor-xcms', 'r-base >=2.0.9', 'r-rcpp >=0.9.13', 'readline 6.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Risa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Risa.html'), ('license', 'LGPL'), ('summary', 'The Investigation / Study / Assay (ISA) tab-delimited format is a general purpose framework with which to collect and communicate complex metadata (i.e. sample characteristics, technologies used, type of measurements made) from experiments employing a combination of technologies, spanning from traditional approaches to high-throughput techniques. Risa allows to access metadata/data in ISA-Tab format and build Bioconductor data structures. Currently, data generated from microarray, flow cytometry and metabolomics-based (i.e. mass spectrometry) assays are supported. The package is extendable and efforts are undergoing to support metadata associated to proteomics assays.')]))])), 'bioconductor-rlmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rlmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rlmm'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'RLMM_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RLMM_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/RLMM_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/RLMM/RLMM_1.38.0_src_all.tar.gz']), ('md5', 'af5d1ecdc28a9d4d1563a298d6673fa3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RLMM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RLMM.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A classification algorithm, based on a multi-chip, multi-SNP approach for Affymetrix SNP arrays. Using a large training sample where the genotype labels are known, this aglorithm will obtain more accurate classification results on new data. RLMM is based on a robust, linear model and uses the Mahalanobis distance for classification. The chip-to-chip non-biological variation is removed through normalization. This model-based algorithm captures the similarities across genotype groups and probes, as well as thousands other SNPs for accurate classification. NOTE: 100K-Xba only at for now.')]))])), 'bioconductor-rmat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rmat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rmat'), ('version', '3.26.0')])), ('source', CommentedMap([('fn', 'rMAT_3.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rMAT_3.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/rMAT_3.26.0.tar.gz', 'https://depot.galaxyproject.org/software/rMAT/rMAT_3.26.0_src_all.tar.gz']), ('md5', '5a4637f45a1c0f73bf9532534d2b6bcd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base']), ('run', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rMAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rMAT.html'), ('license', 'Artistic-2.0'), ('summary', \"This package is an R version of the package MAT and contains functions to parse and merge Affymetrix BPMAP and CEL tiling array files (using C++ based Fusion SDK and Bioconductor package affxparser), normalize tiling arrays using sequence specific models, detect enriched regions from ChIP-chip experiments. Note: users should have GSL and GenomeGraphs installed. Windows users: 'consult the README file available in the inst directory of the source distribution for necessary configuration instructions'. Snow Leopard users can take advantage of increase speed with Grand Central Dispatch!\")]))])), 'bioconductor-rnaprobr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaprobr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaprobr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RNAprobR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RNAprobR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RNAprobR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RNAprobR/RNAprobR_1.8.0_src_all.tar.gz']), ('md5', '2c13899031d3b95778f1de8ca0963ef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1']), ('run', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNAprobR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RNAprobR.html'), ('license', 'GPL (>=2)'), ('summary', 'This package facilitates analysis of Next Generation Sequencing data for which positional information with a single nucleotide resolution is a key. It allows for applying different types of relevant normalizations, data visualization and export in a table or UCSC compatible bedgraph file.')]))])), 'bioconductor-rnaseqcomp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqcomp'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'rnaseqcomp_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rnaseqcomp_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/rnaseqcomp_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/rnaseqcomp/rnaseqcomp_1.6.0_src_all.tar.gz']), ('md5', '3aabf98f1c100ac627d37fc797e48bb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rnaseqcomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rnaseqcomp.html'), ('license', 'GPL-3'), ('summary', 'Several quantitative and visualized benchmarks for RNA-seq quantification pipelines. Two-condition quantifications for genes, transcripts, junctions or exons by each pipeline with necessary meta information should be organized into numeric matrices in order to proceed the evaluation.')]))])), 'bioconductor-rnaseqmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqmap'), ('version', '2.34.0')])), ('source', CommentedMap([('fn', 'rnaSeqMap_2.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rnaSeqMap_2.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/rnaSeqMap_2.34.0.tar.gz', 'https://depot.galaxyproject.org/software/rnaSeqMap/rnaSeqMap_2.34.0_src_all.tar.gz']), ('md5', '98c22474845881bd4958bfa5de4249fa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-deseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-dbi']), ('run', ['bioconductor-biobase', 'bioconductor-deseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rnaSeqMap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rnaSeqMap.html'), ('license', 'GPL-2'), ('summary', 'The rnaSeqMap library provides classes and functions to analyze the RNA-sequencing data using the coverage profiles in multiple samples at a time')]))])), 'bioconductor-rnaseqpower': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqpower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqpower'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'RNASeqPower_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RNASeqPower_1.14.0.tar.gz'), ('md5', 'b803cf98a37c07405f6843dc75f3aef9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNASeqPower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RNASeqPower.html'), ('license', 'LGPL (>=2)'), ('summary', 'RNA-seq, sample size')]))])), 'bioconductor-rnits': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnits/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnits'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Rnits_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rnits_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rnits_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Rnits/Rnits_1.10.0_src_all.tar.gz']), ('md5', 'b0c1ae5bb71e7210ff21b886de98a4a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rnits\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rnits.html'), ('license', 'GPL-3'), ('summary', 'R/Bioconductor package for normalization, curve registration and inference in time course gene expression data')]))])), 'bioconductor-roar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roar'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'roar_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/roar_1.10.0.tar.gz'), ('md5', 'b1d65f416358c3398f4334d6671d12c1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'roar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/roar.html'), ('license', 'GPL-3'), ('summary', 'Identify preferential usage of APA sites, comparing two biological conditions, starting from known alternative sites and alignments obtained from standard RNA-seq experiments.')]))])), 'bioconductor-roc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roc'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'ROC_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROC_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROC_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/ROC/ROC_1.52.0_src_all.tar.gz']), ('md5', '15d20f25f96473afc11ffe9ee9a73b93')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROC.html'), ('license', 'Artistic-2.0'), ('summary', 'utilities for ROC, with uarray focus')]))])), 'bioconductor-roleswitch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roleswitch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roleswitch'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Roleswitch_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Roleswitch_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Roleswitch_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Roleswitch/Roleswitch_1.14.0_src_all.tar.gz']), ('md5', 'fd8b45d19324e41f4390d8d7803d4bb4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Roleswitch\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Roleswitch.html'), ('license', 'GPL-2'), ('summary', 'Infer Probabilities of MiRNA-mRNA Interaction Signature (ProMISe) using paired expression data from a single sample. Roleswitch operates in two phases by inferring the probability of mRNA (miRNA) being the targets (\"targets\") of miRNA (mRNA), taking into account the expression of all of the mRNAs (miRNAs) due to their potential competition for the same miRNA (mRNA). Due to dynamic miRNA repression in the cell, Roleswitch assumes that the total transcribed mRNA levels are higher than the observed (equilibrium) mRNA levels and iteratively updates the total transcription of each mRNA targets based on the above inference. NB: in the paper, we used ProMISe as both the model name and inferred score name.')]))])), 'bioconductor-rols': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rols/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rols'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'rols_2.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rols_2.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/rols_2.4.0.tar.gz', 'https://depot.galaxyproject.org/software/rols/rols_2.4.0_src_all.tar.gz']), ('md5', 'e9856508a8820274ca553f93cb731557')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-jsonlite', 'r-progress']), ('run', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-jsonlite', 'r-progress'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rols\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rols.html'), ('license', 'GPL-2'), ('summary', 'An interface to the Ontology Lookup Service (OLS) to access and query hundred of ontolgies directly from R.')]))])), 'bioconductor-rontotools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rontotools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rontotools'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'ROntoTools_2.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROntoTools_2.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROntoTools_2.4.0.tar.gz', 'https://depot.galaxyproject.org/software/ROntoTools/ROntoTools_2.4.0_src_all.tar.gz']), ('md5', '00ef46ebced2cb6b09f4bf7f0aa5d178')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROntoTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROntoTools.html'), ('license', 'CC BY-NC-ND 4.0 + file LICENSE'), ('summary', 'Suite of tools for functional analysis.')]))])), 'bioconductor-ropls': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ropls/1.2.14/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ropls'), ('version', '1.2.14')])), ('source', CommentedMap([('fn', 'ropls_1.2.14.tar.gz'), ('url', 'https://bioconductor.riken.jp/packages/3.2/bioc/src/contrib/ropls_1.2.14.tar.gz'), ('md5', '0d0cfa9b7458da283ce343eff9211a81')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ropls\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ropls.html'), ('license', 'CeCILL'), ('summary', 'Latent variable modeling with Principal Component Analysis (PCA) and Partial Least Squares (PLS) are powerful methods for visualization, regression, classification, and feature selection of omics data where the number of variables exceeds the number of samples and with multicollinearity among variables. Orthogonal Partial Least Squares (OPLS) enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation. While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to scores, loadings and weights plots, the package provides metrics and graphics to determine the optimal number of components (e.g. with the R2 and Q2 coefficients), check the validity of the model by permutation testing, detect outliers, and perform feature selection (e.g. with Variable Importance in Projection or regression coefficients). The package can be accessed via a user interface on the Workflow4Metabolomics.org online resource for computational metabolomics (built upon the Galaxy environment).')]))])), 'bioconductor-rots': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rots'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ROTS_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ROTS_1.0.0.tar.gz'), ('md5', '4891e471a606cc72998bd0d87da0066d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'r-base', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROTS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ROTS.html'), ('license', 'GPL (>= 2)'), ('summary', 'Calculates the Reproducibility-Optimized Test Statistic (ROTS) for differential testing in omics data.')]))])), 'bioconductor-rpa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpa'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'RPA_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RPA_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/RPA_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/RPA/RPA_1.32.0_src_all.tar.gz']), ('md5', '06e6dde0f7e06243e740837923498434')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RPA.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'Probabilistic analysis of probe reliability and differential gene expression on short oligonucleotide arrays. Lahti et al. \"Probabilistic Analysis of Probe Reliability in Differential Gene Expression Studies with Short Oligonucleotide Arrays\", TCBB/IEEE, 2011. http://doi.ieeecomputersociety.org/10.1109/TCBB.2009.38')]))])), 'bioconductor-rpx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpx'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'rpx_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rpx_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/rpx/rpx_1.12.1_src_all.tar.gz']), ('md5', '4174cd01da022a04523e8745204f28a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-xml2']), ('run', ['r-base', 'r-rcurl', 'r-xml2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rpx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rpx.html'), ('license', 'GPL-2'), ('summary', 'This package implements an interface to proteomics data submitted to the ProteomeXchange consortium.')]))])), 'bioconductor-rrdp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrdp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrdp'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rRDP_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rRDP_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rRDP_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rRDP/rRDP_1.10.0_src_all.tar.gz']), ('md5', '51ee5791ce729b0b57b62931b901736a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.26.2', 'r-base']), ('run', ['bioconductor-biostrings >=2.26.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rRDP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rRDP.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Seamlessly interfaces RDP classifier (version 2.9).')]))])), 'bioconductor-rrho': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrho/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrho'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RRHO_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RRHO_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RRHO_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RRHO/RRHO_1.16.0_src_all.tar.gz']), ('md5', '5975b6636247b09cc94e3770f215a064')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-venndiagram']), ('run', ['r-base', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RRHO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RRHO.html'), ('license', 'GPL-2'), ('summary', 'The package is aimed at inference on the amount of agreement in two sorted lists using the Rank-Rank Hypergeometric Overlap test.')]))])), 'bioconductor-rsamtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsamtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsamtools'), ('version', '1.26.1')])), ('source', CommentedMap([('fn', 'Rsamtools_1.26.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-rsamtools/bioconductor-rsamtools_1.26.1_src_all.tar.gz'), ('md5', '53b695db2aaea899348d7ccea464197e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-xvector', 'bioconductor-zlibbioc', 'r-base', 'r-bitops']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-xvector', 'bioconductor-zlibbioc', 'r-base', 'r-bitops'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rsamtools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rsamtools.html'), ('license', 'Artistic-2.0 | file LICENSE'), ('summary', \"This package provides an interface to the 'samtools', 'bcftools', and 'tabix' utilities (see 'LICENCE') for manipulating SAM (Sequence Alignment / Map), FASTA, binary variant call (BCF) and compressed indexed tab-delimited (tabix) files.\")]))])), 'bioconductor-rsubread': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsubread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsubread'), ('version', '1.25.2')])), ('source', CommentedMap([('fn', 'Rsubread_1.25.2.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rsubread_1.25.2.tar.gz'), ('md5', 'b883dd93fcf007438c1780a0b2a46fb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rsubread\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rsubread.html'), ('license', 'GPL-3'), ('summary', 'Provides powerful and easy-to-use tools for analyzing next-gen sequencing read data. Includes quality assessment of sequence reads, read alignment, read summarization, exon-exon junction detection, fusion detection, detection of short and long indels, absolute expression calling and SNP calling. Can be used with reads generated from any of the major sequencing platforms including Illumina GA/HiSeq/MiSeq, Roche GS-FLX, ABI SOLiD and LifeTech Ion PGM/Proton sequencers.')]))])), 'bioconductor-rsvsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsvsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsvsim'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RSVSim_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RSVSim_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RSVSim_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RSVSim/RSVSim_1.16.0_src_all.tar.gz']), ('md5', 'a74e5ea3c69cc4d16df35fb61e6cce86')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RSVSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RSVSim.html'), ('license', 'LGPL-3'), ('summary', 'RSVSim is a package for the simulation of deletions, insertions, inversion, tandem-duplications and translocations of various sizes in any genome available as FASTA-file or BSgenome data package. SV breakpoints can be placed uniformly accross the whole genome, with a bias towards repeat regions and regions of high homology (for hg19) or at user-supplied coordinates.')]))])), 'bioconductor-rtca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtca'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'RTCA_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTCA_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTCA_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/RTCA/RTCA_1.28.0_src_all.tar.gz']), ('md5', '7e777771cf80ffe7769d99aa84d1af5f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-gtools', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'r-base', 'r-gtools', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTCA.html'), ('license', 'LGPL-3'), ('summary', 'Import, analyze and visualize data from Roche(R) xCELLigence RTCA systems. The package imports real-time cell electrical impedance data into R. As an alternative to commercial software shipped along the system, the Bioconductor package RTCA provides several unique transformation (normalization) strategies and various visualization tools.')]))])), 'bioconductor-rtcgatoolbox': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtcgatoolbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtcgatoolbox'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'RTCGAToolbox_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTCGAToolbox_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTCGAToolbox_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RTCGAToolbox/RTCGAToolbox_2.6.0_src_all.tar.gz']), ('md5', '7ec10941e9be27dfb553305e96f4007c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml']), ('run', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTCGAToolbox\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTCGAToolbox.html'), ('license', 'GPL (>= 2)'), ('summary', 'Managing data from large scale projects such as The Cancer Genome Atlas (TCGA) for further analysis is an important and time consuming step for research projects. Several efforts, such as Firehose project, make TCGA pre-processed data publicly available via web services and data portals but it requires managing, downloading and preparing the data for following steps. We developed an open source and extensible R based data client for Firehose pre-processed data and demonstrated its use with sample case studies. Results showed that RTCGAToolbox could improve data management for researchers who are interested with TCGA data. In addition, it can be integrated with other analysis pipelines for following data analysis.')]))])), 'bioconductor-rtopper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtopper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtopper'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RTopper_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTopper_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTopper_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RTopper/RTopper_1.22.0_src_all.tar.gz']), ('md5', 'e5900c1e1f7e33a2a42c5abe3f6bce79')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTopper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTopper.html'), ('license', 'GPL (>= 3)'), ('summary', 'the RTopper package is designed to perform and integrate gene set enrichment results across multiple genomic platforms.')]))])), 'bioconductor-rtracklayer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtracklayer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtracklayer'), ('version', '1.34.2')])), ('source', CommentedMap([('fn', 'rtracklayer_1.34.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/rtracklayer_1.34.2.tar.gz'), ('md5', 'f631b7060bd3b397e5e938bb0af94f22')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.3.14', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.20', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.17.8', 'bioconductor-s4vectors >=0.9.33', 'bioconductor-xvector >=0.9.4', 'bioconductor-zlibbioc', 'r-base', 'r-rcurl >=1.4*', 'r-xml >=1.98*']), ('run', ['bioconductor-biocgenerics >=0.13.8', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.3.14', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.20', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.17.8', 'bioconductor-s4vectors >=0.9.33', 'bioconductor-xvector >=0.9.4', 'bioconductor-zlibbioc', 'r-base', 'r-rcurl >=1.4*', 'r-xml >=1.98*'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rtracklayer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/rtracklayer.html'), ('license', 'Artistic-2.0 + file LICENSE'), ('summary', 'Extensible framework for interacting with multiple genome browsers (currently UCSC built-in) and manipulating annotation tracks in various formats (currently GFF, BED, bedGraph, BED15, WIG, BigWig and 2bit built-in). The user may export/import tracks to/from the supported browsers, as well as query and modify the browser state, such as the current viewport.')]))])), 'bioconductor-rtreemix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtreemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtreemix'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Rtreemix_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rtreemix_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rtreemix_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Rtreemix/Rtreemix_1.38.0_src_all.tar.gz']), ('md5', 'c1093bf1a6e197a241eed28f980f4bdf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rtreemix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rtreemix.html'), ('license', 'LGPL'), ('summary', 'Rtreemix is a package that offers an environment for estimating the mutagenetic trees mixture models from cross-sectional data and using them for various predictions. It includes functions for fitting the trees mixture models, likelihood computations, model comparisons, waiting time estimations, stability analysis, etc.')]))])), 'bioconductor-rtrm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtrm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtrm'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'rTRM_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rTRM_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/rTRM_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/rTRM/rTRM_1.14.0_src_all.tar.gz']), ('md5', '3c7c12c8294ca635073deff5caeb8712')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rTRM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rTRM.html'), ('license', 'GPL-3'), ('summary', 'rTRM identifies transcriptional regulatory modules (TRMs) from protein-protein interaction networks.')]))])), 'bioconductor-ruvseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ruvseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ruvseq'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'RUVSeq_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RUVSeq_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/RUVSeq_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/RUVSeq/RUVSeq_1.10.0_src_all.tar.gz']), ('md5', '92ded604e6c32727edce35b0f9483927')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-edaseq >=1.99.1', 'bioconductor-edger', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase', 'bioconductor-edaseq >=1.99.1', 'bioconductor-edger', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RUVSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RUVSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements the remove unwanted variation (RUV) methods of Risso et al. (2014) for the normalization of RNA-Seq read counts between samples.')]))])), 'bioconductor-s4vectors': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-s4vectors/0.8.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-s4vectors'), ('version', '0.8.11')])), ('source', CommentedMap([('fn', 'S4Vectors_0.8.11.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.11.tar.gz'), ('md5', '02b20a992bf46bb2cbdef8c90d76b421')])), ('build', CommentedMap([('skip', False), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.15.10', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.15.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'S4Vectors\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/S4Vectors.html'), ('license', 'Artistic-2.0'), ('summary', 'The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).')]))])), 'bioconductor-safe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-safe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-safe'), ('version', '3.16.0')])), ('source', CommentedMap([('fn', 'safe_3.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/safe_3.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/safe_3.16.0.tar.gz', 'https://depot.galaxyproject.org/software/safe/safe_3.16.0_src_all.tar.gz']), ('md5', '85352d24a00e7203935a166cf87d253f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-sparsem']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-sparsem'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'safe\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/safe.html'), ('license', 'GPL (>= 2)'), ('summary', 'SAFE is a resampling-based method for testing functional categories in gene expression experiments. SAFE can be applied to 2-sample and multi-class comparisons, or simple linear regressions. Other experimental designs can also be accommodated through user-defined functions.')]))])), 'bioconductor-sagenhaft': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagenhaft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagenhaft'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sagenhaft_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sagenhaft_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sagenhaft/sagenhaft_1.46.0_src_all.tar.gz']), ('md5', 'f093d7720c376de3edf1795cc487f2dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-sparsem >=0.73']), ('run', ['r-base', 'r-sparsem >=0.73'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sagenhaft\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sagenhaft.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements several functions useful for analysis of gene expression data by sequencing tags as done in SAGE (Serial Analysis of Gene Expressen) data, i.e. extraction of a SAGE library from sequence files, sequence error correction, library comparison. Sequencing error correction is implementing using an Expectation Maximization Algorithm based on a Mixture Model of tag counts.')]))])), 'bioconductor-sagx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagx'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'SAGx_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SAGx_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/SAGx_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/SAGx/SAGx_1.50.0_src_all.tar.gz']), ('md5', 'abcc404aac2dafb403caefd16764ac41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SAGx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SAGx.html'), ('license', 'GPL-3'), ('summary', 'A package for retrieval, preparation and analysis of data from the Affymetrix GeneChip. In particular the issue of identifying differentially expressed genes is addressed.')]))])), 'bioconductor-sangerseqr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sangerseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sangerseqr'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'sangerseqR_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sangerseqR_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/sangerseqR_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/sangerseqR/sangerseqR_1.12.0_src_all.tar.gz']), ('md5', 'd526e7a5fbc4694c861b34b583efbd35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-shiny']), ('run', ['bioconductor-biostrings', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sangerseqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sangerseqR.html'), ('license', 'GPL-2'), ('summary', 'This package contains several tools for analyzing Sanger Sequencing data files in R, including reading .scf and .ab1 files, making basecalls and plotting chromatograms.')]))])), 'bioconductor-santa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-santa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-santa'), ('version', '2.14.0')])), ('source', CommentedMap([('fn', 'SANTA_2.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SANTA_2.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SANTA_2.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SANTA/SANTA_2.14.0_src_all.tar.gz']), ('md5', '913f31736c9f499d9075b762a6c8f07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-matrix', 'r-snow']), ('run', ['r-base', 'r-igraph', 'r-matrix', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SANTA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SANTA.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides methods for measuring the strength of association between a network and a phenotype. It does this by measuring clustering of the phenotype across the network (Knet). Vertices can also be individually ranked by their strength of association with high-weight vertices (Knode).')]))])), 'bioconductor-savr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-savr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-savr'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'savR_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/savR_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/savR_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/savR/savR_1.14.0_src_all.tar.gz']), ('md5', '8b2eb7908fd014ae3ac9f1d4ac74fbf7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'savR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/savR.html'), ('license', 'AGPL-3'), ('summary', 'Parse Illumina Sequence Analysis Viewer (SAV) files, access data, and generate QC plots.')]))])), 'bioconductor-scan.upc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scan.upc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scan.upc'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'SCAN.UPC_2.18.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/SCAN.UPC_2.18.0.tar.gz'), ('md5', '26993908ea71b6f444aac156ca972b1d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass']), ('run', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SCAN.UPC\\')\"'])])), ('about', CommentedMap([('license', 'MIT'), ('home', 'http://bioconductor.org/packages/release/bioc/html/SCAN.UPC.html'), ('summary', 'SCAN is a microarray normalization method to facilitate personalized-medicine workflows.')]))])), 'bioconductor-scater': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scater/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scater'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'scater_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scater_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/scater_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/scater/scater_1.4.0_src_all.tar.gz']), ('md5', '8033f360d74414444205ec87898675e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scater\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scater.html'), ('license', 'GPL (>= 2)'), ('summary', 'A collection of tools for doing various analyses of single-cell RNA-seq gene expression data, with a focus on quality control.')]))])), 'bioconductor-scran': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scran/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scran'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'scran_1.4.5.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scran_1.4.5.tar.gz', 'https://depot.galaxyproject.org/software/scran/scran_1.4.5_src_all.tar.gz']), ('sha256', '91eac3907d4cf75c532d0fbe9a3f3f88652523d67b9d852fe391be37bfa93977')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scran\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scran.html'), ('license', 'GPL-3'), ('summary', 'Implements a variety of low-level analyses of single-cell RNA-seq data. Methods are provided for normalization of cell-specific biases, assignment of cell cycle phase, and detection of highly variable and significantly correlated genes.')]))])), 'bioconductor-segmentseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-segmentseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-segmentseq'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'segmentSeq_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/segmentSeq_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/segmentSeq_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/segmentSeq/segmentSeq_2.10.0_src_all.tar.gz']), ('md5', '523f393753946e0f560b0f72b1c89d1c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bayseq >=2.9.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-shortread', 'r-abind', 'r-base']), ('run', ['bioconductor-bayseq >=2.9.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-shortread', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'segmentSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/segmentSeq.html'), ('license', 'GPL-3'), ('summary', 'High-throughput sequencing technologies allow the production of large volumes of short sequences, which can be aligned to the genome to create a set of matches to the genome. By looking for regions of the genome which to which there are high densities of matches, we can infer a segmentation of the genome into regions of biological significance. The methods in this package allow the simultaneous segmentation of data from multiple samples, taking into account replicate data, in order to create a consensus segmentation. This has obvious applications in a number of classes of sequencing experiments, particularly in the discovery of small RNA loci and novel mRNA transcriptome discovery.')]))])), 'bioconductor-semdist': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-semdist/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-semdist'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'SemDist_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SemDist_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/SemDist_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/SemDist/SemDist_1.10.0_src_all.tar.gz']), ('md5', 'da15643d4f007c039781767b4e8b3084')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SemDist\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SemDist.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements methods to calculate information accretion for a given version of the gene ontology and uses this data to calculate remaining uncertainty, misinformation, and semantic similarity for given sets of predicted annotations and true annotations from a protein function predictor.')]))])), 'bioconductor-sepa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sepa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sepa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SEPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SEPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SEPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SEPA/SEPA_1.6.0_src_all.tar.gz']), ('md5', '93c9ffd58ae48df9f51a396fc0dbe6d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny']), ('run', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SEPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SEPA.html'), ('license', 'GPL(>=2)'), ('summary', 'Given single-cell RNA-seq data and true experiment time of cells or pseudo-time cell ordering, SEPA provides convenient functions for users to assign genes into different gene expression patterns such as constant, monotone increasing and increasing then decreasing. SEPA then performs GO enrichment analysis to analysis the functional roles of genes with same or similar patterns.')]))])), 'bioconductor-seqgsea': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqgsea'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'SeqGSEA_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SeqGSEA_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SeqGSEA_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SeqGSEA/SeqGSEA_1.16.0_src_all.tar.gz']), ('md5', 'd2f4041cdee40dbd41dbe49d2111f0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SeqGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SeqGSEA.html'), ('license', 'GPL (>= 3)'), ('summary', \"The package generally provides methods for gene set enrichment analysis of high-throughput RNA-Seq data by integrating differential expression and splicing. It uses negative binomial distribution to model read count data, which accounts for sequencing biases and biological variation. Based on permutation tests, statistical significance can also be achieved regarding each gene's differential expression and splicing, respectively.\")]))])), 'bioconductor-seqlogo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqlogo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqlogo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'seqLogo_1.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/seqLogo_1.38.0.tar.gz'), ('md5', '7fe87115d75b80a1b25def5fc254b260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqLogo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqLogo.html'), ('license', 'LGPL (>= 2)'), ('summary', 'seqLogo takes the position weight matrix of a DNA sequence motif and plots the corresponding sequence logo as introduced by Schneider and Stephens (1990).')]))])), 'bioconductor-seqpattern': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqpattern/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqpattern'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'seqPattern_1.6.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-seqpattern/bioconductor-seqpattern_1.6.0_src_all.tar.gz'), ('md5', 'b0f51e76f8e30b7bcd2969f97622d83c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqPattern\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqPattern.html'), ('license', 'GPL-3'), ('summary', 'Visualising oligonucleotide patterns and sequence motifs occurrences across a large set of sequences centred at a common reference point and sorted by a user defined feature.')]))])), 'bioconductor-seqtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqtools'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'seqTools_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/seqTools_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/seqTools_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/seqTools/seqTools_1.10.0_src_all.tar.gz']), ('md5', 'ac3d1b9efc95565f58068ba5825e2ed4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/seqTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Analyze read length, phred scores and alphabet frequency and DNA k-mers on uncompressed and compressed fastq files.')]))])), 'bioconductor-sgseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sgseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sgseq'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'SGSeq_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SGSeq_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/SGSeq/SGSeq_1.10.0_src_all.tar.gz']), ('md5', '238a9ef01aff7fd5dab36d775c63b376')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-summarizedexperiment', 'r-base', 'r-igraph', 'r-runit']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-summarizedexperiment', 'r-base', 'r-igraph', 'r-runit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SGSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SGSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'SGSeq is a software package for analyzing splice events from RNA-seq data. Input data are RNA-seq reads mapped to a reference genome in BAM format. Genes are represented as a splice graph, which can be obtained from existing annotation or predicted from the mapped sequence reads. Splice events are identified from the graph and are quantified locally using structurally compatible reads at the start or end of each splice variant. The software includes functions for splice event prediction, quantification, visualization and interpretation.')]))])), 'bioconductor-shortread': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-shortread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-shortread'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'ShortRead_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ShortRead_1.32.0.tar.gz'), ('md5', '71d56164fd7995c0d257027bbd9e24d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ShortRead\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ShortRead.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements sampling, iteration, and input of FASTQ files. The package includes functions for filtering and trimming reads, and for generating a quality assessment report. Data are represented as DNAStringSet-derived objects, and easily manipulated for a diversity of purposes.  The package also contains legacy support for early single-end, ungapped alignment formats.')]))])), 'bioconductor-siggenes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-siggenes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-siggenes'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'siggenes_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/siggenes_1.46.0.tar.gz'), ('md5', '106de15f38812cccfd5b3e20c3fb13fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'siggenes\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/siggenes.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Identification of differentially expressed genes and estimation of the False Discovery Rate (FDR) using both the Significance Analysis of Microarrays (SAM) and the Empirical Bayes Analyses of Microarrays (EBAM).')]))])), 'bioconductor-sights': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sights/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sights'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'sights_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sights_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/sights_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/sights/sights_1.2.0_src_all.tar.gz']), ('md5', '26e07f1d549bd6b6eb5ad2c1dcb8e3fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4']), ('run', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sights\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sights.html'), ('license', 'GPL-3 | file LICENSE'), ('summary', 'SIGHTS is a suite of normalization methods, statistical tests, and diagnostic graphical tools for high throughput screening (HTS) assays. HTS assays use microtitre plates to screen large libraries of compounds for their biological, chemical, or biochemical activity.')]))])), 'bioconductor-signer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-signer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-signer'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'signeR_1.0.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/signeR_1.0.1.tar.gz', 'https://depot.galaxyproject.org/software/signeR/signeR_1.0.1_src_all.tar.gz']), ('md5', 'edc96646e9c9e8f88a0a147f4662095b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'gcc']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'signeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/signeR.html'), ('license', 'GPL-3'), ('summary', 'The signeR package provides an empirical Bayesian approach to mutational signature discovery. It is designed to analyze single nucleotide variaton (SNV) counts in cancer genomes, but can also be applied to other features as well. Functionalities to characterize signatures or genome samples according to exposure patterns are also provided.')]))])), 'bioconductor-sigpathway': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sigpathway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sigpathway'), ('version', '1.44.1')])), ('source', CommentedMap([('fn', 'sigPathway_1.44.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sigPathway_1.44.1.tar.gz', 'https://depot.galaxyproject.org/software/sigPathway/sigPathway_1.44.1_src_all.tar.gz']), ('md5', '0a34f8a57894f81ef21c17bac663e15b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sigPathway\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sigPathway.html'), ('license', 'GPL-2'), ('summary', 'Conducts pathway analysis by calculating the NT_k and NE_k statistics as described in Tian et al. (2005)')]))])), 'bioconductor-simbindprofiles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-simbindprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-simbindprofiles'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'SimBindProfiles_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SimBindProfiles_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SimBindProfiles_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SimBindProfiles/SimBindProfiles_1.14.0_src_all.tar.gz']), ('md5', '34be39f2a337a2975203476b8aab849f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SimBindProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SimBindProfiles.html'), ('license', 'GPL-3'), ('summary', 'SimBindProfiles identifies common and unique binding regions in genome tiling array data. This package does not rely on peak calling, but directly compares binding profiles processed on the same array platform. It implements a simple threshold approach, thus allowing retrieval of commonly and differentially bound regions between datasets as well as events of compensation and increased binding.')]))])), 'bioconductor-similarpeak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-similarpeak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-similarpeak'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'similaRpeak_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/similaRpeak_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/similaRpeak_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/similaRpeak/similaRpeak_1.8.0_src_all.tar.gz']), ('md5', '05ee87ede75ff28acf79a8e3e1858e2c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-r6 >=2.0']), ('run', ['r-base', 'r-r6 >=2.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'similaRpeak\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/similaRpeak.html'), ('license', 'Artistic-2.0'), ('summary', 'This package calculates metrics which assign a level of similarity between ChIP-Seq profiles.')]))])), 'bioconductor-simpleaffy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-simpleaffy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-simpleaffy'), ('version', '2.52.0')])), ('source', CommentedMap([('fn', 'simpleaffy_2.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/simpleaffy_2.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/simpleaffy_2.52.0.tar.gz', 'https://depot.galaxyproject.org/software/simpleaffy/simpleaffy_2.52.0_src_all.tar.gz']), ('md5', 'aa305099a57b3d868be53dc8c539b74e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.33.6', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-gcrma', 'bioconductor-genefilter', 'r-base']), ('run', ['bioconductor-affy >=1.33.6', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-gcrma', 'bioconductor-genefilter', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'simpleaffy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/simpleaffy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Provides high level functions for reading Affy .CEL files, phenotypic data, and then computing simple things with it, such as t-tests, fold changes and the like. Makes heavy use of the affy library. Also has some basic scatter plot functions and mechanisms for generating high resolution journal figures...')]))])), 'bioconductor-sispa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sispa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sispa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SISPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SISPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SISPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SISPA/SISPA_1.6.0_src_all.tar.gz']), ('md5', '8af8654a5736fa2ad073a1ebab30d9ec')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SISPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SISPA.html'), ('license', 'GPL-2'), ('summary', 'Sample Integrated Set Profile Analysis (SISPA) is a method designed to define sample groups with similar gene set enrichment profiles.')]))])), 'bioconductor-sizepower': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sizepower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sizepower'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sizepower_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sizepower_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/sizepower_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sizepower/sizepower_1.46.0_src_all.tar.gz']), ('md5', '98c92ff897d7b422932cd648018532c7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sizepower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sizepower.html'), ('license', 'LGPL'), ('summary', 'This package has been prepared to assist users in computing either a sample size or power value for a microarray experimental study. The user is referred to the cited references for technical background on the methodology underpinning these calculations. This package provides support for five types of sample size and power calculations. These five types can be adapted in various ways to encompass many of the standard designs encountered in practice.')]))])), 'bioconductor-slqpcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-slqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-slqpcr'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'SLqPCR_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SLqPCR_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/SLqPCR_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/SLqPCR/SLqPCR_1.42.0_src_all.tar.gz']), ('md5', 'db88b99f4c365a7f8381611d89e5fa8c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SLqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SLqPCR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH')]))])), 'bioconductor-smap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-smap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-smap'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'SMAP_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SMAP_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/SMAP_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/SMAP/SMAP_1.40.0_src_all.tar.gz']), ('md5', '8e6d638d02011fe362a6bc7c012a77ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SMAP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SMAP.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for DNA copy number profiling of array-CGH data')]))])), 'bioconductor-snpediar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpediar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpediar'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SNPediaR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SNPediaR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SNPediaR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SNPediaR/SNPediaR_1.2.0_src_all.tar.gz']), ('md5', 'e27dc812f1fe1de1ab81d0ea9b64780e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-jsonlite', 'r-rcurl']), ('run', ['r-base', 'r-jsonlite', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SNPediaR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SNPediaR.html'), ('license', 'GPL-2'), ('summary', 'SNPediaR provides some tools for downloading and parsing data from the SNPedia web site <http://www.snpedia.com>. The implemented functions allow users to import the wiki text available in SNPedia pages and to extract the most relevant information out of them. If some information in the downloaded pages is not automatically processed by the library functions, users can easily implement their own parsers to access it in an efficient way.')]))])), 'bioconductor-snphood': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snphood/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snphood'), ('version', '1.6.1')])), ('source', CommentedMap([('fn', 'SNPhood_1.6.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SNPhood_1.6.1.tar.gz', 'https://depot.galaxyproject.org/software/SNPhood/SNPhood_1.6.1_src_all.tar.gz']), ('md5', '496522c1179c2486ea5911f484926d71')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-deseq2', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base', 'r-checkmate', 'r-cluster', 'r-data.table', 'r-ggplot2', 'r-gridextra', 'r-lattice', 'r-rcolorbrewer', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-deseq2', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base', 'r-checkmate', 'r-cluster', 'r-data.table', 'r-ggplot2', 'r-gridextra', 'r-lattice', 'r-rcolorbrewer', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SNPhood\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SNPhood.html'), ('license', 'LGPL (>= 3)'), ('summary', 'To date, thousands of single nucleotide polymorphisms (SNPs) have been found to be associated with complex traits and diseases. However, the vast majority of these disease-associated SNPs lie in the non-coding part of the genome, and are likely to affect regulatory elements, such as enhancers and promoters, rather than function of a protein. Thus, to understand the molecular mechanisms underlying genetic traits and diseases, it becomes increasingly important to study the effect of a SNP on nearby molecular traits such as chromatin environment or transcription factor (TF) binding. Towards this aim, we developed SNPhood, a user-friendly *Bioconductor* R package to investigate and visualize the local neighborhood of a set of SNPs of interest for NGS data such as chromatin marks or transcription factor binding sites from ChIP-Seq or RNA- Seq experiments. SNPhood comprises a set of easy-to-use functions to extract, normalize and summarize reads for a genomic region, perform various data quality checks, normalize read counts using additional input files, and to cluster and visualize the regions according to the binding pattern. The regions around each SNP can be binned in a user-defined fashion to allow for analysis of very broad patterns as well as a detailed investigation of specific binding shapes. Furthermore, SNPhood supports the integration with genotype information to investigate and visualize genotype-specific binding patterns. Finally, SNPhood can be employed for determining, investigating, and visualizing allele-specific binding patterns around the SNPs of interest.')]))])), 'bioconductor-snpstats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpstats'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'snpStats_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/snpStats_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/snpStats_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/snpStats/snpStats_1.26.0_src_all.tar.gz']), ('md5', '14404c3e01849fa7836f704be8a085cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival']), ('run', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'snpStats\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/snpStats.html'), ('license', 'GPL-3'), ('summary', 'Classes and statistical methods for large SNP association studies. This extends the earlier snpMatrix package, allowing for uncertainty in genotypes.')]))])), 'bioconductor-specond': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-specond/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-specond'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'SpeCond_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SpeCond_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/SpeCond_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/SpeCond/SpeCond_1.30.0_src_all.tar.gz']), ('md5', '05b8043512a1d5c01d51b16435ed49cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SpeCond\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SpeCond.html'), ('license', 'LGPL (>=2)'), ('summary', 'This package performs a gene expression data analysis to detect condition-specific genes. Such genes are significantly up- or down-regulated in a small number of conditions. It does so by fitting a mixture of normal distributions to the expression values. Conditions can be environmental conditions, different tissues, organs or any other sources that you wish to compare in terms of gene expression.')]))])), 'bioconductor-spem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spem'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'SPEM_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPEM_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPEM_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SPEM/SPEM_1.16.0_src_all.tar.gz']), ('md5', 'b5bdc5f1a9a55a8318fd4920003b9b0e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-rsolnp']), ('run', ['bioconductor-biobase', 'r-base', 'r-rsolnp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPEM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPEM.html'), ('license', 'GPL-2'), ('summary', 'This package can optimize the parameter in S-system models given time series data')]))])), 'bioconductor-spia': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spia'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'SPIA_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPIA_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPIA_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/SPIA/SPIA_2.28.0_src_all.tar.gz']), ('md5', '5aff2977b9b84cf4fa21de1d8325c8d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-kegggraph', 'r-base']), ('run', ['bioconductor-kegggraph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPIA.html'), ('license', 'file LICENSE'), ('summary', 'This package implements the Signaling Pathway Impact Analysis (SPIA) which uses the information form a list of differentially expressed genes and their log fold changes together with signaling pathways topology, in order to identify the pathways most relevant to the condition under the study.')]))])), 'bioconductor-spikeli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spikeli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spikeli'), ('version', '2.36.0')])), ('source', CommentedMap([('fn', 'spikeLI_2.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/spikeLI_2.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/spikeLI_2.36.0.tar.gz', 'https://depot.galaxyproject.org/software/spikeLI/spikeLI_2.36.0_src_all.tar.gz']), ('md5', 'fcfab36e7d98f614b8850e8fa5470890')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'spikeLI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/spikeLI.html'), ('license', 'GPL-2'), ('summary', 'SpikeLI  is a package that performs the analysis of the Affymetrix spike-in data using the Langmuir Isotherm. The aim of this package is to show the advantages of a physical-chemistry based analysis of the Affymetrix microarray data compared to the traditional methods. The spike-in (or Latin square) data for the HGU95 and HGU133 chipsets have been downloaded from the Affymetrix web site. The model used in the spikeLI package is described in details in E. Carlon and T. Heim, Physica A 362, 433 (2006).')]))])), 'bioconductor-spktools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spktools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spktools'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'spkTools_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/spkTools_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/spkTools_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/spkTools/spkTools_1.32.0_src_all.tar.gz']), ('md5', '534b7f254944917cb6139e6bfce64f23')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-gtools', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-gtools', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'spkTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/spkTools.html'), ('license', 'GPL (>= 2)'), ('summary', 'The package contains functions that can be used to compare expression measures on different array platforms.')]))])), 'bioconductor-splicegear': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splicegear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splicegear'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'splicegear_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splicegear_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/splicegear_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/splicegear/splicegear_1.48.0_src_all.tar.gz']), ('md5', '8ba5ae791768e9328c6f3b55f806b159')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splicegear\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splicegear.html'), ('license', 'LGPL'), ('summary', 'A set of tools to work with alternative splicing')]))])), 'bioconductor-splicer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splicer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splicer'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'spliceR_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/spliceR_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/spliceR_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/spliceR/spliceR_1.18.0_src_all.tar.gz']), ('md5', '20e45197315f6cdba5d0970493691d1b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cummerbund', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-plyr', 'r-rcolorbrewer', 'r-venndiagram']), ('run', ['bioconductor-cummerbund', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-plyr', 'r-rcolorbrewer', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'spliceR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/spliceR.html'), ('license', 'GPL (>=2)'), ('summary', 'An R package for classification of alternative splicing and prediction of coding potential from RNA-seq data.')]))])), 'bioconductor-splinter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splinter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splinter'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SPLINTER_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPLINTER_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPLINTER_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SPLINTER/SPLINTER_1.2.0_src_all.tar.gz']), ('md5', '3ecdf9ad450532bdb2b6cfbabb80b8e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPLINTER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPLINTER.html'), ('license', 'GPL-2'), ('summary', 'SPLINTER provides tools to analyze alternative splicing sites, interpret outcomes based on sequence information, select and design primers for site validiation and give visual representation of the event to guide downstream experiments.')]))])), 'bioconductor-splots': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splots'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'splots_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splots_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/splots_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/splots/splots_1.42.0_src_all.tar.gz']), ('md5', 'ce2288e62b5fcd7c9ae038f6338e8aad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splots\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splots.html'), ('license', 'LGPL'), ('summary', 'The splots package provides the plotScreen function for visualising data in microtitre plate or slide format.')]))])), 'bioconductor-squadd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-squadd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-squadd'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'SQUADD_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SQUADD_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/SQUADD_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/SQUADD/SQUADD_1.26.0_src_all.tar.gz']), ('md5', 'e031d825157b32d711e31564181dfaba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SQUADD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SQUADD.html'), ('license', 'GPL (>=2)'), ('summary', 'This package SQUADD is a SQUAD add-on. It permits to generate SQUAD simulation matrix, prediction Heat-Map and Correlation Circle from PCA analysis.')]))])), 'bioconductor-sradb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sradb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sradb'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'SRAdb_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/SRAdb_1.32.0.tar.gz'), ('md5', '37c10ff958a239ef1b778ca15d166fe0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SRAdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/SRAdb.html'), ('license', 'Artistic-2.0'), ('summary', 'The Sequence Read Archive (SRA) is the largest public repository of sequencing data from the next generation of sequencing platforms including Roche 454 GS System, Illumina Genome Analyzer, Applied Biosystems SOLiD System, Helicos Heliscope, and others. However, finding data of interest can be challenging using current tools. SRAdb is an attempt to make access to the metadata associated with submission, study, sample, experiment and run much more feasible. This is accomplished by parsing all the NCBI SRA metadata into a SQLite database that can be stored and queried locally. Fulltext search in the package make querying metadata very flexible and powerful.  fastq and sra files can be downloaded for doing alignment locally. Beside ftp protocol, the SRAdb has funcitons supporting fastp protocol (ascp from Aspera Connect) for faster downloading large data files over long distance. The SQLite database is updated regularly as new data is added to SRA and can be downloaded at will for the most up-to-date metadata.')]))])), 'bioconductor-sscore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscore'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'sscore_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscore_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscore_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/sscore/sscore_1.48.0_src_all.tar.gz']), ('md5', 'af130ad415483c52c333ca179d6b1b60')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscore.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains an implementation of the S-Score algorithm as described by Zhang et al (2002).')]))])), 'bioconductor-sscu': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscu/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscu'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'sscu_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscu_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscu_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/sscu/sscu_2.6.0_src_all.tar.gz']), ('md5', '7bdf279b3cfb60538d13df4e4fa25eb2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3']), ('run', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscu\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscu.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package calculates the indexes for selective stength in codon usage in bacteria species. (1) The package can calculate the strength of selected codon usage bias (sscu, also named as s_index) based on Paul Sharp's method. The method take into account of background mutation rate, and focus only on four pairs of codons with universal translational advantages in all bacterial species. Thus the sscu index is comparable among different species. (2) The package can detect the strength of translational accuracy selection by Akashi's test. The test tabulating all codons into four categories with the feature as conserved/variable amino acids and optimal/non-optimal codons. (3) Optimal codon lists (selected codons) can be calculated by either op_highly function (by using the highly expressed genes compared with all genes to identify optimal codons), or op_corre_CodonW/op_corre_NCprime function (by correlative method developed by Hershberg & Petrov). Users will have a list of optimal codons for further analysis, such as input to the Akashi's test. (4) The detailed codon usage information, such as RSCU value, number of optimal codons in the highly/all gene set, as well as the genomic gc3 value, can be calculate by the optimal_codon_statistics and genomic_gc3 function. (5) Furthermore, we added one test function low_frequency_op in the package. The function try to find the low frequency optimal codons, among all the optimal codons identified by the op_highly function.\")]))])), 'bioconductor-sseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sseq'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'sSeq_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sSeq_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/sSeq_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/sSeq/sSeq_1.14.0_src_all.tar.gz']), ('md5', 'acd8bce328fafcfdc09ae23091ce85a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-catools', 'r-rcolorbrewer']), ('run', ['r-base', 'r-catools', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The purpose of this package is to discover the genes that are differentially expressed between two conditions in RNA-seq experiments. Gene expression is measured in counts of transcripts and modeled with the Negative Binomial (NB) distribution using a shrinkage approach for dispersion estimation. The method of moment (MM) estimates for dispersion are shrunk towards an estimated target, which minimizes the average squared difference between the shrinkage estimates and the initial estimates. The exact per-gene probability under the NB model is calculated, and used to test the hypothesis that the expected expression of a gene in two conditions identically follow a NB distribution.')]))])), 'bioconductor-ssize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssize'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'ssize_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssize_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssize_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/ssize/ssize_1.50.0_src_all.tar.gz']), ('md5', 'f7d12775503e22319187141e941f8060')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gdata', 'r-xtable']), ('run', ['r-base', 'r-gdata', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssize\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssize.html'), ('license', 'LGPL'), ('summary', 'Functions for computing and displaying sample size information for gene expression arrays.')]))])), 'bioconductor-sspa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sspa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sspa'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'SSPA_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SSPA_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SSPA_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SSPA/SSPA_2.16.0_src_all.tar.gz']), ('md5', 'c0c2b503e9eb785c66b222dd6dc8d45c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice']), ('run', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SSPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SSPA.html'), ('license', 'GPL (>= 2)'), ('summary', 'General Sample size and power analysis for microarray and next-generation sequencing data.')]))])), 'bioconductor-ssviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ssviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ssviz/ssviz_1.10.0_src_all.tar.gz']), ('md5', 'e9b7d0ef0379968ee84d3228753ebc58')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape']), ('run', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssviz.html'), ('license', 'GPL-2'), ('summary', 'Small RNA sequencing viewer')]))])), 'bioconductor-starank': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-starank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-starank'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'staRank_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/staRank_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/staRank_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/staRank/staRank_1.18.0_src_all.tar.gz']), ('md5', 'eb644e71d012e4dccde98ac02410adc0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cellhts2', 'r-base']), ('run', ['bioconductor-cellhts2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'staRank\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/staRank.html'), ('license', 'GPL'), ('summary', 'Detecting all relevant variables from a data set is challenging, especially when only few samples are available and data is noisy. Stability ranking provides improved variable rankings of increased robustness using resampling or subsampling.')]))])), 'bioconductor-starr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-starr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-starr'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'Starr_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Starr_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/Starr_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/Starr/Starr_1.32.0_src_all.tar.gz']), ('md5', 'f1593771aa0eaa26868fe88e4af5944d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline']), ('run', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Starr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Starr.html'), ('license', 'Artistic-2.0'), ('summary', 'Starr facilitates the analysis of ChIP-chip data, in particular that of Affymetrix tiling arrays. The package provides functions for data import, quality assessment, data visualization and exploration. Furthermore, it includes high-level analysis features like association of ChIP signals with annotated features, correlation analysis of ChIP signals and other genomic data (e.g. gene expression), peak-finding with the CMARRT algorithm and comparative display of multiple clusters of ChIP-profiles. It uses the basic Bioconductor classes ExpressionSet and probeAnno for maximum compatibility with other software on Bioconductor. All functions from Starr can be used to investigate preprocessed data from the Ringo package, and vice versa. An important novel tool is the the automated generation of correct, up-to-date microarray probe annotation (bpmap) files, which relies on an efficient mapping of short sequences (e.g. the probe sequences on a microarray) to an arbitrary genome.')]))])), 'bioconductor-stategra': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stategra/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stategra'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'STATegRa_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/STATegRa_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/STATegRa_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/STATegRa/STATegRa_1.10.0_src_all.tar.gz']), ('md5', 'f34d346fb5af6b548d3b95a771d8fe3a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-calibrate', 'r-foreach', 'r-ggplot2', 'r-gplots', 'r-gridextra', 'r-mass']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-calibrate', 'r-foreach', 'r-ggplot2', 'r-gplots', 'r-gridextra', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'STATegRa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/STATegRa.html'), ('license', 'GPL-2'), ('summary', 'Classes and tools for multi-omics data integration.')]))])), 'bioconductor-stepnorm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stepnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stepnorm'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'stepNorm_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/stepNorm_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/stepNorm_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/stepNorm/stepNorm_1.48.0_src_all.tar.gz']), ('md5', 'e0553374d83c0b631912e27f0f829ff7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'stepNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/stepNorm.html'), ('license', 'LGPL'), ('summary', 'Stepwise normalization functions for cDNA microarray data.')]))])), 'bioconductor-streamer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-streamer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-streamer'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'Streamer_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Streamer_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/Streamer_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/Streamer/Streamer_1.22.0_src_all.tar.gz']), ('md5', 'c1c584dd90ea170f2d19db19df0b4de1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Streamer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Streamer.html'), ('license', 'Artistic-2.0'), ('summary', \"Large data files can be difficult to work with in R, where data generally resides in memory. This package encourages a style of programming where data is 'streamed' from disk into R via a `producer' and through a series of `consumers' that, typically reduce the original data to a manageable size. The package provides useful Producer and Consumer stream components for operations such as data input, sampling, indexing, and transformation; see package?Streamer for details.\")]))])), 'bioconductor-stringdb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stringdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stringdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'STRINGdb_1.16.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/STRINGdb_1.16.0.tar.gz'), ('md5', '2892f288a504491e56686a22650aa4e5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf']), ('run', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'STRINGdb\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/STRINGdb.html'), ('license', 'GPL-2'), ('summary', 'The STRINGdb package provides a R interface to the STRING protein-protein interactions database (http://www.string-db.org)')]))])), 'bioconductor-subseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-subseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-subseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'subSeq_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/subSeq_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/subSeq_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/subSeq/subSeq_1.6.0_src_all.tar.gz']), ('md5', 'c45c6d6082f6f0c36553f4bc3e5aaf0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'subSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/subSeq.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Subsampling of high throughput sequencing count data for use in experiment design and analysis.')]))])), 'bioconductor-summarizedexperiment': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-summarizedexperiment/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-summarizedexperiment'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SummarizedExperiment_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/SummarizedExperiment_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SummarizedExperiment_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SummarizedExperiment/SummarizedExperiment_1.4.0_src_all.tar.gz']), ('md5', '35159a339981efa22e96db639dadc850')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SummarizedExperiment\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SummarizedExperiment.html'), ('license', 'Artistic-2.0'), ('summary', 'The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.')]))])), 'bioconductor-suprahex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-suprahex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-suprahex'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'supraHex_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/supraHex_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/supraHex_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/supraHex/supraHex_1.14.0_src_all.tar.gz']), ('md5', '751bb08f19253c574cc51f9c46ec5eee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-ape', 'r-base', 'r-hexbin', 'r-mass']), ('run', ['r-ape', 'r-base', 'r-hexbin', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'supraHex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/supraHex.html'), ('license', 'GPL-2'), ('summary', 'A supra-hexagonal map is a giant hexagon on a 2-dimensional grid seamlessly consisting of smaller hexagons. It is supposed to train, analyse and visualise a high-dimensional omics input data. The supraHex is able to carry out gene clustering/meta-clustering and sample correlation, plus intuitive visualisations to facilitate exploratory analysis. More importantly, it allows for overlaying additional data onto the trained map to explore relations between input and additional data. So with supraHex, it is also possible to carry out multilayer omics data comparisons. Newly added utilities are advanced heatmap visualisation and tree-based analysis of sample relationships. Uniquely to this package, users can ultrafastly understand any tabular omics data, both scientifically and artistically, especially in a sample-specific fashion but without loss of information on large genes.')]))])), 'bioconductor-sushi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sushi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sushi'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Sushi_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Sushi_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Sushi_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Sushi/Sushi_1.14.0_src_all.tar.gz']), ('md5', '045d9d479da85852673e02940c6ab843')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'r-base', 'r-zoo']), ('run', ['bioconductor-biomart', 'r-base', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Sushi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Sushi.html'), ('license', 'GPL (>= 2)'), ('summary', 'Flexible, quantitative, and integrative genomic visualizations for publication-quality multi-panel figures')]))])), 'bioconductor-sva': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sva/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sva'), ('version', '3.20.0')])), ('source', CommentedMap([('fn', 'sva_3.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/sva_3.20.0.tar.gz'), ('md5', 'cca24ad4d83f6dba6ccddb70e1a834db')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'bioconductor-genefilter', 'r-base', 'r-mgcv']), ('run', ['libgcc', 'bioconductor-genefilter', 'r-base', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sva\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/sva.html'), ('license', 'Artistic-2.0'), ('summary', 'The sva package contains functions for removing batch effects and other\\nunwanted variation in high-throughput experiment. Specifically, the sva package\\ncontains functions for the identifying and building surrogate variables for high-dimensional\\ndata sets. Surrogate variables are covariates constructed directly from high-dimensional\\ndata (like gene expression/RNA sequencing/methylation/brain imaging data) that\\ncan be used in subsequent analyses to adjust for unknown, unmodeled, or latent\\nsources of noise. The sva package can be used to remove artifacts in three ways:\\n(1) identifying and estimating surrogate variables for unknown sources of variation\\nin high-throughput experiments (Leek and Storey 2007 PLoS Genetics,2008 PNAS),\\n(2) directly removing known batch effects using ComBat (Johnson et al. 2007 Biostatistics)\\nand (3) removing batch effects with known control probes (Leek 2014 biorXiv).\\nRemoving batch effects and using surrogate variables in differential expression\\nanalysis have been shown to reduce dependence, stabilize error rate estimates,\\nand improve reproducibility, see (Leek and Storey 2007 PLoS Genetics, 2008 PNAS\\nor Leek et al. 2011 Nat. Reviews Genetics).\\n')]))])), 'bioconductor-swath2stats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-swath2stats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-swath2stats'), ('version', '1.6.1')])), ('source', CommentedMap([('fn', 'SWATH2stats_1.6.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SWATH2stats_1.6.1.tar.gz', 'https://depot.galaxyproject.org/software/SWATH2stats/SWATH2stats_1.6.1_src_all.tar.gz']), ('md5', 'a16451dba71171874af6c8c6ae82d9b5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-ggplot2', 'r-reshape2']), ('run', ['r-base', 'r-data.table', 'r-ggplot2', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SWATH2stats\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SWATH2stats.html'), ('license', 'GPL-3'), ('summary', 'This package is intended to transform SWATH data from the OpenSWATH software into a format readable by other statistics packages while performing filtering, annotation and FDR estimation.')]))])), 'bioconductor-swathxtend': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-swathxtend/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-swathxtend'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SwathXtend_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SwathXtend_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SwathXtend_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SwathXtend/SwathXtend_1.4.0_src_all.tar.gz']), ('md5', 'a3fb504b0b49d5b000d51516d2b36044')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram']), ('run', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SwathXtend\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SwathXtend.html'), ('license', 'GPL-2'), ('summary', 'It contains utility functions for integrating spectral libraries for SWATH and statistical data analysis for SWATH generated data.')]))])), 'bioconductor-switchbox': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-switchbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-switchbox'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'switchBox_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/switchBox_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/switchBox_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/switchBox/switchBox_1.12.0_src_all.tar.gz']), ('md5', '564232ea6cb5ddfbd11515e09ec56bf0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-proc']), ('run', ['r-base', 'r-gplots', 'r-proc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'switchBox\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/switchBox.html'), ('license', 'GPL-2'), ('summary', 'The package offer different classifiers based on comparisons of pair of features (TSP), using various decision rules (e.g., majority wins principle).')]))])), 'bioconductor-switchde': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-switchde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-switchde'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'switchde_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/switchde_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/switchde/switchde_1.2.0_src_all.tar.gz']), ('md5', 'ce5a7cf7527304adcaeee2f1bcb46cca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2']), ('run', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'switchde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/switchde.html'), ('license', 'GPL (>= 2)'), ('summary', 'Inference and detection of switch-like differential expression across single-cell RNA-seq trajectories.')]))])), 'bioconductor-synergyfinder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-synergyfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-synergyfinder'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'synergyfinder_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/synergyfinder_1.0.0.tar.gz'), ('md5', '2b282463638871a0074fe6cb829d112a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr']), ('run', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'synergyfinder\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/synergyfinder.html'), ('license', 'Artistic License 2.0'), ('summary', 'Efficient implementations for all the popular synergy scoring models for drug combinations, including HSA, Loewe, Bliss and ZIP and visualization of the synergy scores as either a two-dimensional or a three-dimensional interaction surface over the dose matrix.')]))])), 'bioconductor-systempiper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-systempiper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['http://bioconductor.org/packages/release/bioc/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz']), ('md5', '9790794a52e4ff0c6abd61d24f31b1eb')])), ('build', CommentedMap([('skip', True), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-shortread', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base', 'r-batchjobs', 'r-ggplot2', 'r-pheatmap', 'r-rjson']), ('run', ['bioconductor-annotate', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-shortread', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-base', 'r-batchjobs', 'r-ggplot2', 'r-pheatmap', 'r-rjson'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'<{ name }}\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/systemPipeR.html'), ('license', 'Artistic-2.0'), ('summary', 'R package for building and running automated end-to-end analysis workflows for a wide range of next generation sequence (NGS) applications such as RNA-Seq, ChIP-Seq, VAR-Seq and Ribo-Seq. Important features include a uniform workflow interface across different NGS applications, automated report generation, and support for running both R and command-line software, such as NGS aligners or peak/variant callers, on local computers or compute clusters. Efficient handling of complex sample sets and experimental designs is facilitated by a consistently implemented sample annotation infrastructure. Instructions for using systemPipeR are given in the Overview Vignette (HTML). The remaining Vignettes, linked below, are workflow templates for common NGS use cases.')]))])), 'bioconductor-targetscore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-targetscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-targetscore'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TargetScore_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TargetScore_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TargetScore_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TargetScore/TargetScore_1.14.0_src_all.tar.gz']), ('md5', '63f01d25aec571ebaaa4cce97b0ff42c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrix', 'r-pracma']), ('run', ['r-base', 'r-matrix', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TargetScore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TargetScore.html'), ('license', 'GPL-2'), ('summary', 'Infer the posterior distributions of microRNA targets by probabilistically modelling the likelihood microRNA-overexpression fold-changes and sequence-based scores. Variaitonal Bayesian Gaussian mixture model (VB-GMM) is applied to log fold-changes and sequence scores to obtain the posteriors of latent variable being the miRNA targets. The final targetScore is computed as the sigmoid-transformed fold-change weighted by the averaged posteriors of target components over all of the features.')]))])), 'bioconductor-tarseqqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tarseqqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tarseqqc'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'TarSeqQC_1.6.2.tar.gz'), ('url', ['https://bioconductor.org/packages/release/bioc/src/contrib/TarSeqQC_1.6.2.tar.gz', 'https://depot.galaxyproject.org/software/bioconductor-tarseqqc/bioconductor-tarseqqc_1.6.2_src_all.tar.gz']), ('sha256', '2a5fe2812cab300afb6c0fbfe9640d506da6e64427052973cc8ad5cdca92c959')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.20.4', 'bioconductor-s4vectors', 'r-base', 'r-cowplot', 'r-ggplot2', 'r-hmisc', 'r-openxlsx', 'r-plyr', 'r-reshape2']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.20.4', 'bioconductor-s4vectors', 'r-base', 'r-cowplot', 'r-ggplot2', 'r-hmisc', 'r-openxlsx', 'r-plyr', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TarSeqQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TarSeqQC.html'), ('license', 'GPL (>=2)'), ('summary', 'The package allows the representation of targeted experiment in R. This is based on current packages and incorporates functions to do a quality control over this kind of experiments and a fast exploration of the sequenced regions. An xlsx file is generated as output.')]))])), 'bioconductor-teqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-teqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-teqc'), ('version', '3.16.0')])), ('source', CommentedMap([('fn', 'TEQC_3.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TEQC_3.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/TEQC_3.16.0.tar.gz', 'https://depot.galaxyproject.org/software/TEQC/TEQC_3.16.0_src_all.tar.gz']), ('md5', '8ebc30b256d07069ab9a012f23469a54')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter']), ('run', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TEQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TEQC.html'), ('license', 'GPL (>= 2)'), ('summary', 'Target capture experiments combine hybridization-based (in solution or on microarrays) capture and enrichment of genomic regions of interest (e.g. the exome) with high throughput sequencing of the captured DNA fragments. This package provides functionalities for assessing and visualizing the quality of the target enrichment process, like specificity and sensitivity of the capture, per-target read coverage and so on.')]))])), 'bioconductor-ternarynet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ternarynet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ternarynet'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'ternarynet_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ternarynet_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/ternarynet_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/ternarynet/ternarynet_1.20.0_src_all.tar.gz']), ('md5', '834763ffbe919907c65bccc73c167358')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph']), ('run', ['r-base', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ternarynet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ternarynet.html'), ('license', 'GPL (>= 2)'), ('summary', 'A computational Bayesian approach to ternary gene regulatory network estimation from gene perturbation experiments.')]))])), 'bioconductor-tfbstools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tfbstools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tfbstools'), ('version', '1.12.2')])), ('source', CommentedMap([('fn', 'TFBSTools_1.12.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/TFBSTools_1.12.2.tar.gz', 'https://depot.galaxyproject.org/software/TFBSTools/TFBSTools_1.12.2_src_all.tar.gz']), ('sha256', '8e495f17804c8d2b0923053251a08814390c035f7e9961d8ae2f18966f7f7964')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.28', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biocparallel >=1.2.21', 'bioconductor-biostrings >=2.36.4', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-cner >=1.4.0', 'bioconductor-dirichletmultinomial >=1.10.0', 'bioconductor-genomeinfodb >=1.6.1', 'bioconductor-genomicranges >=1.20.6', 'bioconductor-iranges >=2.2.7', 'bioconductor-rtracklayer >=1.28.10', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-seqlogo >=1.34.0', 'bioconductor-xvector >=0.8.0', 'r-base', 'r-catools >=1.17.1', 'r-gtools >=3.5.0', 'r-rsqlite >=1.0.0', 'r-tfmpvalue >=0.0.5', 'r-xml >=3.98-1.3']), ('run', ['bioconductor-biobase >=2.28', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biocparallel >=1.2.21', 'bioconductor-biostrings >=2.36.4', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-cner >=1.4.0', 'bioconductor-dirichletmultinomial >=1.10.0', 'bioconductor-genomeinfodb >=1.6.1', 'bioconductor-genomicranges >=1.20.6', 'bioconductor-iranges >=2.2.7', 'bioconductor-rtracklayer >=1.28.10', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-seqlogo >=1.34.0', 'bioconductor-xvector >=0.8.0', 'r-base', 'r-catools >=1.17.1', 'r-gtools >=3.5.0', 'r-rsqlite >=1.0.0', 'r-tfmpvalue >=0.0.5', 'r-xml >=3.98-1.3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TFBSTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/TFBSTools.html'), ('license', 'GPL-2'), ('summary', 'TFBSTools is a package for the analysis and manipulation of transcription factor binding sites. It includes matrices conversion between Position Frequency Matirx (PFM), Position Weight Matirx (PWM) and Information Content Matrix (ICM). It can also scan putative TFBS from sequence/alignment, query JASPAR database and provides a wrapper of de novo motif discovery software.')]))])), 'bioconductor-tigre': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tigre/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tigre'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'tigre_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tigre_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/tigre_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/tigre/tigre_1.30.0_src_all.tar.gz']), ('md5', 'fc1c86cef4cc6174b7f0dea788444603')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-dbi', 'r-gplots', 'r-rsqlite']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-dbi', 'r-gplots', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tigre\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tigre.html'), ('license', 'AGPL-3'), ('summary', 'The tigre package implements our methodology of Gaussian process differential equation models for analysis of gene expression time series from single input motif networks. The package can be used for inferring unobserved transcription factor (TF) protein concentrations from expression measurements of known target genes, or for ranking candidate targets of a TF.')]))])), 'bioconductor-timecourse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-timecourse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-timecourse'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'timecourse_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/timecourse_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/timecourse_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/timecourse/timecourse_1.48.0_src_all.tar.gz']), ('md5', '8c0c49039012d1c74aa81c877a19cd33')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'timecourse\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/timecourse.html'), ('license', 'LGPL'), ('summary', 'Functions for data analysis and graphical displays for developmental microarray time course data.')]))])), 'bioconductor-titancna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-titancna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-titancna'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TitanCNA_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TitanCNA_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TitanCNA_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TitanCNA/TitanCNA_1.14.0_src_all.tar.gz']), ('md5', '989f84cb8d0d687859c4175cea634fdc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2']), ('run', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TitanCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TitanCNA.html'), ('license', 'GPL-3'), ('summary', 'Hidden Markov model to segment and predict regions of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH), and estimate cellular prevalenece of clonal clusters in tumour whole genome sequencing data.')]))])), 'bioconductor-topgo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-topgo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-topgo'), ('version', '2.24.0')])), ('source', CommentedMap([('fn', 'topGO_2.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/topGO_2.24.0.tar.gz'), ('md5', '633251cfa90f4e9f7346ca0b48dcc413')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice']), ('run', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'topGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/topGO.html'), ('license', 'LGPL'), ('summary', 'topGO package provides tools for testing GO terms while accounting for the topology of the GO graph. Different test statistics and different methods for eliminating local similarities and dependencies between GO terms can be implemented and applied.')]))])), 'bioconductor-transview': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-transview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-transview'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'TransView_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TransView_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/TransView_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/TransView/TransView_1.20.0_src_all.tar.gz']), ('md5', '634dc44651702b77eb7c5e7dc9718656')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TransView\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TransView.html'), ('license', 'GPL-3'), ('summary', 'This package provides efficient tools to generate, access and display read densities of sequencing based data sets such as from RNA-Seq and ChIP-Seq.')]))])), 'bioconductor-traser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-traser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-traser'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'traseR_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/traseR_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/traseR_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/traseR/traseR_1.6.0_src_all.tar.gz']), ('md5', '736ed1e0061e31f181a4bc800440241c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base']), ('run', ['bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'traseR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/traseR.html'), ('license', 'GPL'), ('summary', 'traseR performs GWAS trait-associated SNP enrichment analyses in genomic intervals using different hypothesis testing approaches, also provides various functionalities to explore and visualize the results.')]))])), 'bioconductor-triform': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triform/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triform'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'triform_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triform_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/triform_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/triform/triform_1.18.0_src_all.tar.gz']), ('md5', '81b0783c1cb3e8f7b2e2fe2f034b99fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml']), ('run', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triform\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triform.html'), ('license', 'GPL-2'), ('summary', 'The Triform algorithm uses model-free statistics to identify peak-like distributions of TF ChIP sequencing reads, taking advantage of an improved peak definition in combination with known profile characteristics.')]))])), 'bioconductor-trio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-trio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-trio'), ('version', '3.14.0')])), ('source', CommentedMap([('fn', 'trio_3.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/trio_3.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/trio_3.14.0.tar.gz', 'https://depot.galaxyproject.org/software/trio/trio_3.14.0_src_all.tar.gz']), ('md5', 'c9c8042544db35d7a6a2c21f462a01df')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'trio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/trio.html'), ('license', 'LGPL-2'), ('summary', 'Testing SNPs and SNP interactions with a genotypic TDT. This package furthermore contains functions for computing pairwise values of LD measures and for identifying LD blocks, as well as functions for setting up matched case pseudo-control genotype data for case-parent trios in order to run trio logic regression, for imputing missing genotypes in trios, for simulating case-parent trios with disease risk dependent on SNP interaction, and for power and sample size calculation in trio data.')]))])), 'bioconductor-triplex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triplex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triplex'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'triplex_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triplex_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/triplex_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/triplex/triplex_1.16.0_src_all.tar.gz']), ('md5', 'b43ae8cf4b947dfb76e2d7bf91a336d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base']), ('run', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triplex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triplex.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'This package provides functions for identification and visualization of potential intramolecular triplex patterns in DNA sequence. The main functionality is to detect the positions of subsequences capable of folding into an intramolecular triplex (H-DNA) in a much larger sequence. The potential H-DNA (triplexes) should be made of as many cannonical nucleotide triplets as possible. The package includes visualization showing the exact base-pairing in 1D, 2D or 3D.')]))])), 'bioconductor-tscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tscan'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TSCAN_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSCAN_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSCAN_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TSCAN/TSCAN_1.14.0_src_all.tar.gz']), ('md5', '490a22ef7ecccd8e5c5ff45a6c12ee7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny']), ('run', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSCAN.html'), ('license', 'GPL(>=2)'), ('summary', 'TSCAN enables users to easily construct and tune pseudotemporal cell ordering as well as analyzing differentially expressed genes. TSCAN comes with a user-friendly GUI written in shiny. More features will come in the future.')]))])), 'bioconductor-tspair': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tspair/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tspair'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'tspair_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tspair_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/tspair_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/tspair/tspair_1.34.0_src_all.tar.gz']), ('md5', '53493dd6c05e65839d80be76df2c033f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.4.0', 'r-base']), ('run', ['bioconductor-biobase >=2.4.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tspair\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tspair.html'), ('license', 'GPL-2'), ('summary', 'These functions calculate the pair of genes that show the maximum difference in ranking between two user specified groups. This \"top scoring pair\" maximizes the average of sensitivity and specificity over all rank based classifiers using a pair of genes in the data set. The advantage of classifying samples based on only the relative rank of a pair of genes is (a) the classifiers are much simpler and often more interpretable than more complicated classification schemes and (b) if arrays can be classified using only a pair of genes, PCR based tests could be used for classification of samples. See the references for the tspcalc() function for references regarding TSP classifiers.')]))])), 'bioconductor-tssi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tssi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tssi'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'TSSi_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSSi_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSSi_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/TSSi/TSSi_1.22.0_src_all.tar.gz']), ('md5', '2de488ce7fa30a257e2c1b3b3f09a2a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr']), ('run', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSSi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSSi.html'), ('license', 'GPL-3'), ('summary', 'Identify and normalize transcription start sites in high-throughput sequencing data.')]))])), 'bioconductor-tweedeseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tweedeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tweedeseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'tweeDEseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tweeDEseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/tweeDEseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/tweeDEseq/tweeDEseq_1.22.0_src_all.tar.gz']), ('md5', '0e960d8738383e8da9e8127a8d1701e7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass']), ('run', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tweeDEseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tweeDEseq.html'), ('license', 'GPL (>= 2)'), ('summary', 'Differential expression analysis of RNA-seq using the Poisson-Tweedie family of distributions.')]))])), 'bioconductor-twilight': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-twilight/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-twilight'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'twilight_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/twilight_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/twilight_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/twilight/twilight_1.52.0_src_all.tar.gz']), ('md5', '5de401cba564f99a6cb3987c03c6d37a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.12.0', 'r-base']), ('run', ['bioconductor-biobase >=1.12.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'twilight\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/twilight.html'), ('license', 'GPL (>= 2)'), ('summary', \"In a typical microarray setting with gene expression data observed under two conditions, the local false discovery rate describes the probability that a gene is not differentially expressed between the two conditions given its corrresponding observed score or p-value level. The resulting curve of p-values versus local false discovery rate offers an insight into the twilight zone between clear differential and clear non-differential gene expression. Package 'twilight' contains two main functions: Function twilight.pval performs a two-condition test on differences in means for a given input matrix or expression set and computes permutation based p-values. Function twilight performs a stochastic downhill search to estimate local false discovery rates and effect size distributions. The package further provides means to filter for permutations that describe the null distribution correctly. Using filtered permutations, the influence of hidden confounders could be diminished.\")]))])), 'bioconductor-txdb.celegans.ucsc.ce11.refgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.celegans.ucsc.ce11.refgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.celegans.ucsc.ce11.refgene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Celegans.UCSC.ce11.refGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Celegans.UCSC.ce11.refGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Celegans.UCSC.ce11.refGene/TxDb.Celegans.UCSC.ce11.refGene_3.4.0_src_all.tar.gz']), ('sha256', 'cb0478cc70f55c189897e05fdf00c6b70d775e99048281f024c0c0894093e06b')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Celegans.UCSC.ce11.refGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Celegans.UCSC.ce11.refGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.celegans.ucsc.ce6.ensgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.celegans.ucsc.ce6.ensgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Celegans.UCSC.ce6.ensGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Celegans.UCSC.ce6.ensGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Celegans.UCSC.ce6.ensGene/TxDb.Celegans.UCSC.ce6.ensGene_3.2.2_src_all.tar.gz']), ('sha256', '20c51a06664f1135500b3ce1b9eeacf5154995ab4847217524beb131d4bef7e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Celegans.UCSC.ce6.ensGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Celegans.UCSC.ce6.ensGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Dmelanogaster.UCSC.dm3.ensGene_3.2.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/data/annotation/src/contrib/TxDb.Dmelanogaster.UCSC.dm3.ensGene_3.2.2.tar.gz'), ('md5', '46b7ffe0c516edf8a2a3b5d78e0d8b67')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Dmelanogaster.UCSC.dm3.ensGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/TxDb.Dmelanogaster.UCSC.dm3.ensGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.dmelanogaster.ucsc.dm6.ensgene'), ('version', '3.3.0')])), ('source', CommentedMap([('fn', 'TxDb.Dmelanogaster.UCSC.dm6.ensGene_3.3.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Dmelanogaster.UCSC.dm6.ensGene_3.3.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Dmelanogaster.UCSC.dm6.ensGene/TxDb.Dmelanogaster.UCSC.dm6.ensGene_3.3.0_src_all.tar.gz']), ('sha256', 'fca3e415acf022ebab7643296467380bd19dc0cac27d7c752df89ff18759fccd')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.23.28', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.23.28', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Dmelanogaster.UCSC.dm6.ensGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Dmelanogaster.UCSC.dm6.ensGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.drerio.ucsc.danrer10.refgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.drerio.ucsc.danrer10.refgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.drerio.ucsc.danrer10.refgene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Drerio.UCSC.danRer10.refGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Drerio.UCSC.danRer10.refGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Drerio.UCSC.danRer10.refGene/TxDb.Drerio.UCSC.danRer10.refGene_3.4.0_src_all.tar.gz']), ('sha256', '5bca5219e666358143d233349cdd0d3700bf846d387589adad0f7f2ccd92fb6d')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Drerio.UCSC.danRer10.refGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Drerio.UCSC.danRer10.refGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.hsapiens.ucsc.hg18.knowngene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Hsapiens.UCSC.hg18.knownGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg18.knownGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg18.knownGene/TxDb.Hsapiens.UCSC.hg18.knownGene_3.2.2_src_all.tar.gz']), ('sha256', 'bc9ca40b4eab87f5ca64a4b876d42502b9b8e9f5983d745bfe0ee349d97b69fa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Hsapiens.UCSC.hg18.knownGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Hsapiens.UCSC.hg18.knownGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.hsapiens.ucsc.hg19.knowngene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2.tar.gz'), ('md5', '61be88a81433958571b57d0952be48a0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Hsapiens.UCSC.hg19.knownGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/TxDb.Hsapiens.UCSC.hg19.knownGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.hsapiens.ucsc.hg19.lincrnastranscripts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.hsapiens.ucsc.hg19.lincrnastranscripts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.hsapiens.ucsc.hg19.lincrnastranscripts'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts/TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts_3.2.2_src_all.tar.gz']), ('sha256', '4f68ee75c1722b53de4a5382a455730673dab8c0fd7585f86771efe6e7b9ab2e')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.hsapiens.ucsc.hg38.knowngene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg38.knownGene/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0_src_all.tar.gz']), ('md5', '1d5e07631ea58e96b11905d39e76ca6e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('run', ['wget', 'bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.18', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Hsapiens.UCSC.hg38.knownGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Hsapiens.UCSC.hg38.knownGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.mmusculus.ucsc.mm10.ensgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.mmusculus.ucsc.mm10.ensgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.mmusculus.ucsc.mm10.ensgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Mmusculus.UCSC.mm10.ensGene_3.2.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/data/annotation/src/contrib/TxDb.Mmusculus.UCSC.mm10.ensGene_3.2.2.tar.gz'), ('md5', '11da70249e8a8f09f4b4c4eb40ae8b67')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Mmusculus.UCSC.mm10.ensGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/annotation/html/TxDb.Mmusculus.UCSC.mm10.ensGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.mmusculus.ucsc.mm10.knowngene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Mmusculus.UCSC.mm10.knownGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Mmusculus.UCSC.mm10.knownGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Mmusculus.UCSC.mm10.knownGene/TxDb.Mmusculus.UCSC.mm10.knownGene_3.4.0_src_all.tar.gz']), ('sha256', '6b2e9ce3b2050cb770cca989b14b09c50e20ef88568b19cac7ec6ecd93daea21')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Mmusculus.UCSC.mm10.knownGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Mmusculus.UCSC.mm10.knownGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.mmusculus.ucsc.mm9.knowngene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Mmusculus.UCSC.mm9.knownGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Mmusculus.UCSC.mm9.knownGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Mmusculus.UCSC.mm9.knownGene/TxDb.Mmusculus.UCSC.mm9.knownGene_3.2.2_src_all.tar.gz']), ('sha256', 'a2d760206336d0c1ba48108ba3e23b201aa429a6b88a19437270980180ef7299')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Mmusculus.UCSC.mm9.knownGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Mmusculus.UCSC.mm9.knownGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Rnorvegicus.UCSC.rn4.ensGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Rnorvegicus.UCSC.rn4.ensGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Rnorvegicus.UCSC.rn4.ensGene/TxDb.Rnorvegicus.UCSC.rn4.ensGene_3.2.2_src_all.tar.gz']), ('sha256', '15325d09a9da4292678768164e1850a78595dd96351970553844f5eeac5f683f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Rnorvegicus.UCSC.rn4.ensGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/data/annotation/html/TxDb.Rnorvegicus.UCSC.rn4.ensGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.rnorvegicus.ucsc.rn5.refgene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Rnorvegicus.UCSC.rn5.refGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Rnorvegicus.UCSC.rn5.refGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Rnorvegicus.UCSC.rn5.refGene/TxDb.Rnorvegicus.UCSC.rn5.refGene_3.4.0_src_all.tar.gz']), ('sha256', '58fde8628058bfa39919df0eb1558767d3a9b19f25971ef956fb9e4ec43b75dc')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Rnorvegicus.UCSC.rn5.refGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Rnorvegicus.UCSC.rn5.refGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.rnorvegicus.ucsc.rn6.refgene'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'TxDb.Rnorvegicus.UCSC.rn6.refGene_3.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Rnorvegicus.UCSC.rn6.refGene_3.4.0.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Rnorvegicus.UCSC.rn6.refGene/TxDb.Rnorvegicus.UCSC.rn6.refGene_3.4.0_src_all.tar.gz']), ('sha256', 'bd1aa80a44b991dbc55c6e065146e3454df12816461d297ae9a797a1bc092bb8')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.25.17', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Rnorvegicus.UCSC.rn6.refGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Rnorvegicus.UCSC.rn6.refGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.scerevisiae.ucsc.saccer2.sgdgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.scerevisiae.ucsc.saccer2.sgdgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.scerevisiae.ucsc.saccer2.sgdgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Scerevisiae.UCSC.sacCer2.sgdGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Scerevisiae.UCSC.sacCer2.sgdGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Scerevisiae.UCSC.sacCer2.sgdGene/TxDb.Scerevisiae.UCSC.sacCer2.sgdGene_3.2.2_src_all.tar.gz']), ('sha256', 'eaf35b5a3e4706328e7a5b89d7de846055f5ac0aa34109d305a6ffbd2067af50')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Scerevisiae.UCSC.sacCer2.sgdGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Scerevisiae.UCSC.sacCer2.sgdGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-txdb.scerevisiae.ucsc.saccer3.sgdgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-txdb.scerevisiae.ucsc.saccer3.sgdgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-txdb.scerevisiae.ucsc.saccer3.sgdgene'), ('version', '3.2.2')])), ('source', CommentedMap([('fn', 'TxDb.Scerevisiae.UCSC.sacCer3.sgdGene_3.2.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/data/annotation/src/contrib/TxDb.Scerevisiae.UCSC.sacCer3.sgdGene_3.2.2.tar.gz', 'https://depot.galaxyproject.org/software/TxDb.Scerevisiae.UCSC.sacCer3.sgdGene/TxDb.Scerevisiae.UCSC.sacCer3.sgdGene_3.2.2_src_all.tar.gz']), ('sha256', '6dac0b4f88ce54c8543bb0ce236507cd46dd78ff7b3e3457ef7fd0b1dca15cea')])), ('build', CommentedMap([('number', 6), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-genomicfeatures >=1.21.30', 'r-base', 'wget'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TxDb.Scerevisiae.UCSC.sacCer3.sgdGene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/data/annotation/html/TxDb.Scerevisiae.UCSC.sacCer3.sgdGene.html'), ('license', 'Artistic-2.0'), ('summary', 'Exposes an annotation databases generated from UCSC by exposing these as TxDb objects')]))])), 'bioconductor-tximport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tximport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tximport'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'tximport_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/tximport_1.4.0.tar.gz'), ('md5', 'a2a9235064ea1fa89af71d260b50b790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tximport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tximport.html'), ('license', 'GPL (>=2)'), ('summary', 'Imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for different expression of gene-level counts.')]))])), 'bioconductor-uniprot.ws': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-uniprot.ws/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-uniprot.ws'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'UniProt.ws_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/UniProt.ws_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/UniProt.ws_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/UniProt.ws/UniProt.ws_2.16.0_src_all.tar.gz']), ('md5', 'e2e671c9899e415d9049df31523d084f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'UniProt.ws\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/UniProt.ws.html'), ('license', 'Artistic License 2.0'), ('summary', 'A collection of functions for retrieving, processing and repackaging the UniProt web services.')]))])), 'bioconductor-variantannotation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-variantannotation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-variantannotation'), ('version', '1.20.3')])), ('source', CommentedMap([('fn', 'VariantAnnotation_1.20.3.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/VariantAnnotation_1.20.3.tar.gz'), ('md5', 'a16f2e9d24eb4c19607c2e5fcd1d5ea3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantAnnotation\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html'), ('license', 'Artistic-2.0'), ('summary', 'Annotate variants, compute amino acid coding changes, predict coding outcomes.')]))])), 'bioconductor-varianttools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-varianttools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-varianttools'), ('version', '1.18.1')])), ('source', CommentedMap([('fn', 'VariantTools_1.18.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/VariantTools_1.18.1.tar.gz', 'https://depot.galaxyproject.org/software/VariantTools/VariantTools_1.18.1_src_all.tar.gz']), ('md5', 'ef1a9ba2218f49e5dbaf3aba1d0b26b1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/VariantTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Explore, diagnose, and compare variant calls using filters.')]))])), 'bioconductor-vbmp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vbmp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vbmp'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'vbmp_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/vbmp_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/vbmp_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/vbmp/vbmp_1.44.0_src_all.tar.gz']), ('md5', 'fcb2f0ee9fa8829d5a29a038769f65e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'vbmp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/vbmp.html'), ('license', 'GPL (>= 2)'), ('summary', 'Variational Bayesian Multinomial Probit Regression with Gaussian Process Priors. It estimates class membership posterior probability employing variational and sparse approximation to the full posterior. This software also incorporates feature weighting by means of Automatic Relevance Determination.')]))])), 'bioconductor-vega': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vega/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vega'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'Vega_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Vega_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/Vega_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/Vega/Vega_1.24.0_src_all.tar.gz']), ('md5', 'd76349a49dc175ab36939bc89a69c03e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Vega\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Vega.html'), ('license', 'GPL-2'), ('summary', 'Vega (Variational Estimator for Genomic Aberrations) is an algorithm that adapts a very popular variational model (Mumford and Shah) used in image segmentation so that chromosomal aberrant regions can be efficiently detected.')]))])), 'bioconductor-viper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-viper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-viper'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'viper_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/viper_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/viper_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/viper/viper_1.10.0_src_all.tar.gz']), ('md5', '4d7ad13a2d8fe5542a235f0d42f0d6d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'viper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/viper.html'), ('license', 'file LICENSE'), ('summary', 'Inference of protein activity from gene expression data, including the VIPER and msVIPER algorithms')]))])), 'bioconductor-vsn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vsn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vsn'), ('version', '3.38.0')])), ('source', CommentedMap([('fn', 'vsn_3.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/vsn_3.38.0.tar.gz'), ('md5', '1acea08a752560f8c187a2eb010567f7')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin']), ('run', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin'])])), ('test', CommentedMap([('commands', ['$R -e \"require(\\'vsn\\')\"'])])), ('about', CommentedMap([('home', 'http://www.bioconductor.org/packages/release/bioc/html/vsn.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.')]))])), 'bioconductor-weaver': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-weaver/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-weaver'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'weaver_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/weaver_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/weaver_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/weaver/weaver_1.42.0_src_all.tar.gz']), ('md5', 'b7f8eeadaa00abc9e9b56b27af2bd1f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-codetools', 'r-digest']), ('run', ['r-base', 'r-codetools', 'r-digest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'weaver\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/weaver.html'), ('license', 'GPL-2'), ('summary', 'This package provides enhancements on the Sweave() function in the base package.  In particular a facility for caching code chunk results is included.')]))])), 'bioconductor-xbseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xbseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xbseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'XBSeq_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/XBSeq_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/XBSeq_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/XBSeq/XBSeq_1.6.0_src_all.tar.gz']), ('md5', '666261b05630cb254a1834c5dbbde21a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-deseq2', 'bioconductor-roar', 'r-base', 'r-dplyr', 'r-ggplot2', 'r-locfit', 'r-magrittr', 'r-matrixstats', 'r-pracma']), ('run', ['bioconductor-biobase', 'bioconductor-deseq2', 'bioconductor-roar', 'r-base', 'r-dplyr', 'r-ggplot2', 'r-locfit', 'r-magrittr', 'r-matrixstats', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XBSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/XBSeq.html'), ('license', 'GPL (>=3)'), ('summary', 'We developed a novel algorithm, XBSeq, where a statistical model was established based on the assumption that observed signals are the convolution of true expression signals and sequencing noises. The mapped reads in non-exonic regions are considered as sequencing noises, which follows a Poisson distribution. Given measureable observed and noise signals from RNA-seq data, true expression signals, assuming governed by the negative binomial distribution, can be delineated and thus the accurate detection of differential expressed genes.')]))])), 'bioconductor-xcms': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xcms/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'xcms_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.48.0.tar.gz'), ('md5', '83adb6afdb8f6d819bd6d37c7bf583f7')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))])), 'bioconductor-xde': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xde'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'XDE_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/XDE_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/XDE_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/XDE/XDE_2.22.0_src_all.tar.gz']), ('md5', '690f26084d79dfccdc0da9903ff79d2c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XDE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/XDE.html'), ('license', 'LGPL-2'), ('summary', 'Multi-level model for cross-study detection of differential gene expression.')]))])), 'bioconductor-xmapbridge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xmapbridge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xmapbridge'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'xmapbridge_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/xmapbridge_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/xmapbridge_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/xmapbridge/xmapbridge_1.34.0_src_all.tar.gz']), ('md5', '42211b57ad537cdd17c64231b085eac8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xmapbridge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/xmapbridge.html'), ('license', 'LGPL-3'), ('summary', 'xmapBridge can plot graphs in the X:Map genome browser. This package exports plotting files in a suitable format.')]))])), 'bioconductor-xvector': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xvector/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xvector'), ('version', '0.14.1')])), ('source', CommentedMap([('fn', 'XVector_0.14.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/XVector_0.14.1.tar.gz'), ('md5', '27445c0311b9eab3cf1cf032bc55f538')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XVector\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/XVector.html'), ('license', 'Artistic-2.0'), ('summary', 'Memory efficient S4 classes for storing sequences \"externally\" (behind an R external pointer, or on disk).')]))])), 'bioconductor-zlibbioc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-zlibbioc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-zlibbioc'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'zlibbioc_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/zlibbioc_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/zlibbioc_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/zlibbioc/zlibbioc_1.20.0_src_all.tar.gz']), ('md5', 'f68b45fca70f39c7958fb43c0504c546')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'zlibbioc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/zlibbioc.html'), ('license', 'Artistic-2.0 + file LICENSE'), ('summary', 'This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have these available via other means (most Linux and Mac users should have system-level access to zlib, and no direct need for this package). See the vignette for instructions on use.')]))])), 'biofluff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biofluff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biofluff'), ('version', '2.1.3')])), ('source', CommentedMap([('url', 'https://github.com/simonvh/fluff/releases/download/v2.1.3/fluff-2.1.3.tar.gz'), ('fn', 'biofluff-2.1.3.tar.gz'), ('md5', '2a3bbfd07de369c7fba53ca27fb6c8d3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'scipy', 'matplotlib >=2', 'pysam', 'htseq', 'pybedtools', 'colorbrewer', 'pycluster', 'pybigwig']), ('run', ['python', 'scipy', 'matplotlib >=2', 'pysam', 'htseq', 'pybedtools', 'colorbrewer', 'pycluster', 'pybigwig'])])), ('test', CommentedMap([('imports', ['fluff'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/fluff'), ('license', 'MIT'), ('summary', 'Exploratory analysis and visualization of high-throughput sequencing data')]))])), 'biogridpy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biogridpy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biogridpy'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'biogridpy-0.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/97/3f/389d2928ed5d59a4fde16a506e8c6e0e04620f7f61fc84caefca2aa7d84d/biogridpy-0.1.1.tar.gz'), ('md5', '5a39971ec8517658de774dd0b0919a4e')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['biogridpy'])])), ('about', CommentedMap([('home', 'https://github.com/arvkevi/biogridpy'), ('license', 'MIT'), ('summary', 'Python client for the BioGRID REST API webservice'), ('license_family', 'MIT')]))])), 'biokit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biokit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biokit'), ('version', '0.4.1')])), ('source', CommentedMap([('fn', 'biokit-0.4.1.tar.gz'), ('url', 'https://pypi.python.org/packages/e3/fe/9bc827910cbd19a1a03bb5b0a1c4aa583903696526d7f6e51fbc9f849e28/biokit-0.4.1.tar.gz'), ('md5', '755efcfa4982fd23d31d1ebdd4972c22')])), ('build', CommentedMap([('skip', True), ('number', 4)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'easydev >=0.9.34', 'pandas', 'colormap', 'scipy', 'xmltodict', 'numpydoc', 'bioservices', 'mesalib']), ('run', ['python', 'easydev >=0.9.34', 'pandas', 'bioservices >=1.4.16', 'colormap', 'scipy', 'xmltodict', 'numpydoc', 'bioservices', 'matplotlib', 'libgcc', 'mesalib'])])), ('test', CommentedMap([('imports', ['biokit'])])), ('about', CommentedMap([('home', ['http://pypi.python.org/pypi/biokit']), ('license', 'BSD'), ('summary', 'Set of visualisation and analysis tools for biological data sets')]))])), 'biomaj': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biomaj/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biomaj'), ('version', '3.0.19')])), ('source', CommentedMap([('fn', 'biomaj-3.0.19.tar.gz'), ('url', 'https://pypi.python.org/packages/ae/27/1ca70bd9b7c33dc66068d970158a0af1f79ad8823c5bb04cccb21a4eec3b/biomaj-3.0.19.tar.gz'), ('md5', '834631961cab1d87b1a454e08489ef09'), ('patches', ['deps.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'mock', 'drmaa', 'future', 'tabulate', 'elasticsearch']), ('run', ['python', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'drmaa', 'future', 'tabulate', 'elasticsearch'])])), ('test', CommentedMap([('imports', ['biomaj', 'biomaj.download', 'biomaj.process']), ('commands', ['biomaj-cli.py --version'])])), ('about', CommentedMap([('home', 'http://biomaj.genouest.org'), ('license', 'GNU Affero General Public License v3 or later (AGPLv3+)'), ('license_family', 'AGPL'), ('summary', 'Automates the update cycle and the supervision of the locally mirrored databank repository')]))])), 'biopet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biopet/0.8.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/biopet/biopet'), ('summary', 'Biopet (Bio Pipeline Execution Toolkit) is the main pipeline development framework of the LUMC Sequencing Analysis Support Core team.'), ('license', 'https://github.com/biopet/biopet/blob/develop/biopet-core/src/main/resources/nl/lumc/sasc/biopet/License.txt')])), ('package', CommentedMap([('name', 'biopet'), ('version', '0.8.0')])), ('source', CommentedMap([('url', 'https://github.com/biopet/biopet/releases/download/v0.8.0/Biopet-0.8.0-26269612.jar'), ('sha256', '95e93cef4232e1bc4623b30b580c66f4bb76c58278e22815e78aee1095ee2f2b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk', 'python', 'r-base', 'r-ggplot2', 'r-reshape', 'r-argparse'])])), ('test', CommentedMap([('commands', ['biopet version', 'biopet license'])]))])), 'bioservices': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioservices/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioservices'), ('version', '1.4.16')])), ('source', CommentedMap([('fn', 'bioservices.1.4.16.tar.gz'), ('url', 'https://pypi.python.org/packages/40/b0/a105fb56c0210693352dc7136bf3f34ef622768d4c51f796d89bee87f559/bioservices-1.4.16.tar.gz'), ('md5', '1a5509f892ac47db176b183a106a623a')])), ('build', CommentedMap([('skip', False), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'grequests', 'requests', 'requests-cache', 'easydev', 'beautifulsoup4', 'suds-jurko', 'appdirs', 'wrapt', 'numpydoc', 'xmltodict']), ('run', ['python', 'grequests', 'requests', 'requests-cache', 'easydev', 'beautifulsoup4', 'suds-jurko', 'appdirs', 'wrapt', 'numpydoc'])])), ('test', CommentedMap([('imports', ['bioservices', 'bioservices.apps', 'bioservices.mapping'])])), ('about', CommentedMap([('home', ['http://pypi.python.org/pypi/bioservices']), ('license', 'GNU Library or Lesser General Public License (LGPL)'), ('summary', 'Access to Biological Web Services from Python')]))])), 'bismark': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bismark/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bismark'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bismark-v<{ version }}.tar.gz'), ('url', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/bismark_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'samtools', 'bowtie2'])])), ('about', CommentedMap([('home', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('license_family', 'GPL3'), ('summary', 'Bismark is a program to map bisulfite treated sequencing reads to a genome of interest and perform methylation calls in a single step. The output can be easily imported into a genome viewer, such as SeqMonk, and enables a researcher to analyse the methylation levels of their samples straight away.')])), ('test', CommentedMap([('commands', ['bam2nuc --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark --help > /dev/null', 'bismark2bedGraph --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2report --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2summary --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark_genome_preparation --help > /dev/null', 'bismark_methylation_extractor --help > /dev/null', 'coverage2cytosine --help 2>&1 | grep SYNOPSIS > /dev/null'])]))])), 'bitstring': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bitstring/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bitstring'), ('version', '3.1.5')])), ('source', CommentedMap([('fn', 'bitstring-3.1.5.zip'), ('url', 'https://pypi.python.org/packages/f3/e5/dfe4c49c93d174a5fd807ed307d3a3f38c6b3e140972945f81a5f5578ca7/bitstring-3.1.5.zip'), ('md5', '70689a282f66625d0c7c3579a13e66db')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('about', CommentedMap([('home', 'https://github.com/scott-griffiths/bitstring'), ('license', 'MIT License'), ('summary', 'Simple construction, analysis and modification of binary data.'), ('license_family', 'MIT')]))])), 'blasr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blasr/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('git_url', 'https://github.com/PacificBiosciences/blasr.git'), ('git_rev', 'f8bfa9c5565ae01ea669af872edaae5e55471d98')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'hdf5', 'python', 'pbbam', 'blasr_libcpp']), ('run', ['libgcc', 'hdf5', 'pbbam', 'blasr_libcpp'])])), ('test', CommentedMap([('commands', ['blasr --version', 'loadPulses 2>&1 | grep \"loadPulses\"', 'pls2fasta 2>&1 | grep \"pls2fasta\"', 'sawriter --help', 'samtoh5 --help', 'samtom4 --help', 'samFilter --help', 'sawriter --help', 'sdpMatcher 2>&1 | grep \"sdpMatcher\"'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/blasr'), ('license', 'BSD-3-Clause-Clear'), ('license_file', 'LICENSE'), ('summary', 'BLASR - The PacBio long read aligner')]))])), 'blasr_libcpp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blasr_libcpp/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('url', 'https://github.com/PacificBiosciences/blasr_libcpp/archive/fd288b6.tar.gz'), ('md5', '9939867583bdcd9a354997650a617487')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'hdf5', 'python', 'htslib', 'pbbam', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'hdf5', 'boost <{CONDA_BOOST}}*'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/lib/libblasr.so', 'test -f ${PREFIX}/lib/libblasr.dylib', 'test -f ${PREFIX}/lib/libpbihdf.so', 'test -f ${PREFIX}/lib/libpbihdf.dylib', 'test -f ${PREFIX}/lib/libpbdata.so', 'test -f ${PREFIX}/lib/libpbdata.dylib'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/<{ name }}'), ('license', 'BSD-3-Clause-Clear'), ('summary', 'Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for analyzing PacBio sequences')]))])), 'blast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blast/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blast'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('url', 'http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/<{ version }}/ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('md5', '9a2ac418e3031ed266787ea06585d468'), ('patches', ['boost_106400.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'gcc']), ('run', ['boost <{CONDA_BOOST}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['blastn -help', 'blastp -help'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs'), ('license', 'Public Domain'), ('summary', 'BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.')]))])), 'blastalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blastalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blastalign'), ('version', '1.4')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('source', CommentedMap([('fn', 'BlastAlign.tar'), ('url', 'https://sites.google.com/site/belshawrobert/software/BlastAlign.tar'), ('md5', 'c5476addbe673dca1f74ae7a5f62ce9b'), ('patches', ['BlastAlign.patch', 'BlastAlign_fix_Divisionby0.patch', 'BlastAlignP.patch'])])), ('requirements', CommentedMap([('build', ['python', 'perl']), ('run', ['python', 'perl', 'blast-legacy'])])), ('test', CommentedMap([('commands', ['BlastAlign 2>&1'])])), ('about', CommentedMap([('home', 'http://evolve.zoo.ox.ac.uk/Evolve/Blastalign.html'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('authors', 'Aris Katzourakis and Robert Belshaw'), ('summary', 'BlastAlign uses NCBI Blast to align nucleotide sequences that have large indels or are otherwise difficult to align globally.')]))])), 'blat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blat'), ('version', 35)])), ('source', CommentedMap([('fn', 'blatSrc35.zip'), ('url', 'https://users.soe.ucsc.edu/~kent/src/blatSrc35.zip'), ('md5', '16e546b8843b85e0b0f2fa603cd78724')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['libpng', 'zlib']), ('run', ['libpng', 'zlib'])])), ('about', CommentedMap([('home', 'https://genome.ucsc.edu/FAQ/FAQblat.html'), ('license', 'Free for academic, nonprofit and personal use'), ('summary', 'BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.')]))])), 'bleach': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bleach/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bleach'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'bleach-1.4.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/bleach/bleach-1.4.2.tar.gz'), ('md5', '6a8f4d8afed000e2aea14c3aa02a1ea9')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'html5lib >=0.999']), ('run', ['python', 'six', 'html5lib >=0.999'])])), ('test', CommentedMap([('imports', ['bleach', 'bleach.tests']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'http://github.com/jsocol/bleach'), ('license', 'Apache Software License'), ('summary', 'An easy whitelist-based HTML-sanitizing tool.')]))])), 'blockbuster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blockbuster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html'), ('summary', 'Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'blockbuster'), ('version', '0.0.1.1')])), ('source', CommentedMap([('fn', 'blockbuster-0.0.1.1.tar.gz'), ('md5', '23f971cbedfa70caf461e22f96b245c8'), ('url', 'https://github.com/bgruening/download_store/raw/master/blockbuster/blockbuster-0.0.1.1.tar.gz')])), ('test', CommentedMap([('commands', ['blockbuster.x 2>&1 | grep \"^usage\" > /dev/null'])]))])), 'bmfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bmfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bmfilter'), ('version', 3.101)])), ('source', CommentedMap([('fn', 'bmfilter'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/mac-os/bmfilter'), ('md5', '719474a0e3b54449cbe06f6e442994dd')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['bmfilter -h &> /dev/null'])])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/'), ('license', 'Public Domain'), ('summary', 'bmfilter is part of BMTagger aka Best Match Tagger, for removing human reads from metagenomics datasets')]))])), 'bmtagger': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bmtagger/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bmtagger'), ('version', 3.101)])), ('source', CommentedMap([('fn', 'bmtagger.sh'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/mac-os/bmtagger.sh'), ('md5', '719474a0e3b54449cbe06f6e442994dd'), ('patches', ['bmtagger.sh.osxpatch'])])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('build', ['blast', 'srprism', 'bmfilter', 'bmtool', 'extract_fullseq']), ('run', ['gnu-getopt', 'blast', 'srprism', 'bmfilter', 'bmtool', 'extract_fullseq'])])), ('extra', CommentedMap([('notes', 'You may find it necessary to create a bmtagger.conf file to specify paired or single-end searching. The file should contain the line \\'srprismopts=\"-b 100000000 -n 5 -R 0 -r 1 -M 7168\"\\' along with \\'-p true\\' or \\'-p false\\' for paired, and single, respectively. This config file can be passed to bmtagger.sh via the \\'-C\\' option.')])), ('test', CommentedMap([('commands', ['bmtagger.sh -V &> /dev/null'])])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/'), ('license', 'Public Domain'), ('summary', 'BMTagger aka Best Match Tagger is for removing human reads from metagenomics datasets')]))])), 'bmtool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bmtool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bmtool'), ('version', 3.101)])), ('source', CommentedMap([('fn', 'bmtool'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/mac-os/bmtool'), ('md5', '719474a0e3b54449cbe06f6e442994dd')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['bmtool -h &> /dev/null'])])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/'), ('license', 'Public Domain'), ('summary', 'bmtool is part of BMTagger aka Best Match Tagger, for removing human reads from metagenomics datasets')]))])), 'boltons': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/boltons/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'boltons'), ('version', '16.4.1')])), ('source', CommentedMap([('git_url', 'https://github.com/mahmoud/boltons.git'), ('git_rev', '16.4.1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'pytest']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['boltons'])]))])), 'bowtie': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie/1.0.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/index.shtml'), ('license', 'GPLv3'), ('summary', 'An ultrafast memory-efficient short read aligner')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bowtie'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'bowtie-1.0.0-src.zip'), ('sha256', '51e434a78e053301f82ae56f4e94f71f97b19f7175324777a7305c8f88c5bac5'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie/1.0.0/bowtie-1.0.0-src.zip')])), ('test', CommentedMap([('commands', ['(bowtie --version 2>&1) > /dev/null'])]))])), 'bowtie2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie2/2.2.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'), ('license', 'GPLv3'), ('summary', 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.   http://bowtie-bio.sourceforge.net/bowtie2/index.shtml ')])), ('package', CommentedMap([('name', 'bowtie2'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', 'bowtie2-2.2.5-macos-x86_64.zip'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie2/2.2.5/bowtie2-2.2.5-macos-x86_64.zip'), ('sha256', '0f142477738accc73276f3e47877d54a64b6f7ac7911d0609b662adcd15374a0')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'python']), ('run', ['python', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['bowtie2 --version', 'bowtie2-build --help', 'bowtie2-inspect-l --help', 'bowtie2-align-s --help'])]))])), 'bpipe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bpipe/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://docs.bpipe.org/'), ('license', 'BSD-3-clause'), ('summary', 'Bpipe - a tool for running and managing bioinformatics pipelines')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://download.bpipe.org/versions/<{ name }}-<{ version }}.tar.gz'), ('sha256', '29986da8dfb89d9789c25d3bf7e3486acfd99eb1fbac887e7f3aebc8e07ee471')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk 8.0* zulu8*'])])), ('test', CommentedMap([('commands', ['bpipe --help'])]))])), 'bpp-core': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bpp-core/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://biopp.univ-montp2.fr/repos/sources/<{ name }}-<{ version }}.tar.gz'), ('md5', '5789ed2ae8687d13664140cd77203477')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/lib/libbpp-core.a', 'test -f ${PREFIX}/lib/libbpp-core.so'])])), ('about', CommentedMap([('home', 'https://github.com/BioPP/bpp-core'), ('license', 'CeCILL'), ('license_file', 'COPYING.txt'), ('summary', 'Bio++ is a set of C++ libraries for Bioinformatics.')]))])), 'bpp-phyl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bpp-phyl/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://biopp.univ-montp2.fr/repos/sources/<{ name }}-<{ version }}.tar.gz'), ('md5', '5c40667ec0bf37e0ecaba321be932770')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'bpp-seq']), ('run', ['libgcc', 'bpp-seq'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/lib/libbpp-phyl.a', 'test -f ${PREFIX}/lib/libbpp-phyl.so'])])), ('about', CommentedMap([('home', 'https://github.com/BioPP/bpp-phyl'), ('license', 'CeCILL'), ('license_file', 'COPYING.txt'), ('summary', 'Bio++ is a set of C++ libraries for Bioinformatics.')]))])), 'bpp-seq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bpp-seq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://biopp.univ-montp2.fr/repos/sources/<{ name }}-<{ version }}.tar.gz'), ('md5', '44adef0ff4d5ca4e69ccf258c9270633')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'bpp-core']), ('run', ['bpp-core'])])), ('test', None), ('about', CommentedMap([('home', 'https://github.com/BioPP/bpp-seq'), ('license', 'CeCILL'), ('license_file', 'COPYING.txt'), ('summary', 'Bio++ is a set of C++ libraries for Bioinformatics.')]))])), 'bracken': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bracken/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jenniferlu717/Bracken/archive/<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-list-moreutils', 'perl-parallel-forkmanager', 'python', 'kraken'])])), ('test', CommentedMap([('commands', ['count-kmer-abundances.pl --help', 'est_abundance.py -h', 'generate_kmer_distribution.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/jenniferlu717/Bracken'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.')]))])), 'braker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/braker/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bioinf.uni-greifswald.de/augustus/'), ('license', ''), ('summary', 'BRAKER1: Unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'braker'), ('version', '1.9')])), ('requirements', CommentedMap([('build', ['perl-module-build']), ('run', ['perl-app-cpanminus', 'perl-parallel-forkmanager', 'perl-pathtools', 'perl-scalar-util-numeric'])])), ('source', CommentedMap([('fn', 'braker-1.9.tar.gz'), ('url', 'http://exon.gatech.edu/genemark/Braker/BRAKER1.tar.gz')])), ('test', CommentedMap([('commands', ['filterGenemark.pl --help', 'filterIntronsFindStrand.pl --help'])]))])), 'brass': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/brass/meta.yaml', CommentedMap([('build', CommentedMap([('number', 7), ('skip', True)])), ('package', CommentedMap([('name', 'brass'), ('version', '5.1.6')])), ('source', CommentedMap([('fn', '5.1.6.tar.gz'), ('url', 'https://github.com/cancerit/BRASS/archive/v5.1.6.tar.gz'), ('md5', 'bb33dec7048cd1634845f143ed706e3a'), ('patches', ['makefile.patch', 'cansam_headers.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-pcap', 'perl-grass', 'cansam', 'boost', 'perl-extutils-makemaker', 'perl-app-cpanminus', 'perl-math-combinatorics', 'perl-bioperl-core ==1.6.924', 'perl-file-sharedir', 'perl-file-sharedir-install', 'perl-bio-db-hts', 'perl-capture-tiny', 'perl-graph-readwrite', 'perl-number-format', 'perl-statistics-basic', 'perl-text-diff', 'perl-bio-featureio']), ('run', ['libgcc', 'blat', 'velvet', 'exonerate', 'bedtools', 'cansam', 'perl-pcap', 'perl-grass', 'perl-math-combinatorics', 'perl-bioperl-core ==1.6.924', 'perl-file-sharedir', 'perl-file-sharedir-install', 'perl-bio-db-hts', 'perl-capture-tiny', 'perl-graph-readwrite', 'perl-number-format', 'perl-statistics-basic', 'perl-text-diff', 'perl-bio-featureio'])])), ('test', CommentedMap([('commands', ['brass-group --help', 'brass.pl --help'])]))])), 'breakdancer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/breakdancer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/genome/breakdancer'), ('license', 'GPLv3'), ('summary', 'SV detection from paired end reads mapping')])), ('package', CommentedMap([('name', 'breakdancer'), ('version', '1.4.5')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'perl', 'cmake >=2.8', 'gcc', 'zlib', 'ncurses']), ('run', ['libgcc', 'perl-math-cdf', 'perl-statistics-descriptive', 'zlib'])])), ('test', CommentedMap([('commands', ['breakdancer-max 2>&1 | grep \"Usage: breakdancer-max\" > /dev/null'])])), ('source', CommentedMap([('fn', 'v1.4.5.tar.gz'), ('url', 'https://github.com/genome/breakdancer/archive/v1.4.5.tar.gz'), ('md5', '3a3fcb3731b0d9e0573c528bdcaeb18b'), ('patches', ['builddeps.patch'])]))])), 'breakseq2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/breakseq2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'breakseq2'), ('version', '2.2')])), ('source', CommentedMap([('url', 'https://github.com/bioinform/breakseq2/archive/2.2.tar.gz'), ('fn', '2.2.tar.gz'), ('md5', '6fd5a103c2781717b0b1d0efcbdc17e7'), ('patches', ['setup.py.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython', 'pysam ==0.7.7', 'biopython ==1.65']), ('run', ['python', 'cython', 'pysam ==0.7.7', 'biopython ==1.65', 'samtools ==0.1.19', 'bwa'])])), ('test', CommentedMap([('commands', ['run_breakseq2.py -h'])])), ('about', CommentedMap([('home', 'http://bioinform.github.io/breakseq2'), ('license', 'BSD-2-Clause'), ('license_file', 'LICENSE.txt'), ('summary', 'BreakSeq2: Ultrafast and accurate nucleotide-resolution analysis of structural variants.')]))])), 'breseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/breseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/barricklab/breseq/releases/download/v0.29.0/breseq-0.29.0-Source.tar.gz'), ('md5', '7e8d690d0d0f2d0fdd72abb61f91a4f3')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib', 'r-base', 'bowtie2'])])), ('test', CommentedMap([('commands', ['breseq --help 2>&1 | grep Usage', 'gdtools --help 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/barricklab/breseq'), ('license', 'GPLv3'), ('license_file', 'COPYING'), ('summary', 'A computational pipeline for finding mutations relative to a reference sequence in short-read DNA re-sequencing data.')]))])), 'bs-seeker2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bs-seeker2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bs-seeker2'), ('version', '2.1.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'BSseeker2_v2.1.0.tar.bz2'), ('url', 'http://pellegrini.mcdb.ucla.edu/BS_Seeker2/BSseeker2_v2.1.0.tar.bz'), ('md5', '2f3cb74a75453140f20e396ee68402bf')])), ('requirements', CommentedMap([('build', None), ('run', ['python', 'pysam', 'bowtie2'])])), ('about', CommentedMap([('home', 'http://pellegrini.mcdb.ucla.edu/BS_Seeker2/'), ('license', 'MIT'), ('summary', 'BS Seeker 2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated short reads.')])), ('test', CommentedMap([('commands', ['bs_seeker2-align.py --help > /dev/null', 'bs_seeker2-build.py --help > /dev/null', 'bs_seeker2-call_methylation.py --help > /dev/null'])]))])), 'bucketcache': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bucketcache/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bucketcache'), ('version', '0.12.0')])), ('source', CommentedMap([('fn', 'BucketCache-0.12.0.tar.gz'), ('url', 'https://pypi.python.org/packages/68/07/946404561ee0ab94a815a69488f11c7e57060ae8f13fa57b77a0d9f52f05/BucketCache-0.12.0.tar.gz'), ('md5', '40a42a9eed7200a038d2983c151f2113')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'decorator >=4.0.2', 'six >=1.9.0', 'logbook >=0.12.5', 'represent >=1.5.1', 'boltons', 'python-dateutil', 'pathlib']), ('run', ['python', 'decorator >=4.0.2', 'six >=1.9.0', 'logbook >=0.12.5', 'represent >=1.5.1', 'boltons', 'python-dateutil', 'pathlib'])])), ('test', CommentedMap([('imports', ['bucketcache', 'bucketcache.compat'])])), ('about', CommentedMap([('home', 'https://github.com/RazerM/bucketcache'), ('license', 'MIT License'), ('summary', 'Versatile persisent file cache.'), ('license_family', 'MIT')]))])), 'bumbershoot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bumbershoot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bumbershoot'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'http://teamcity.labkey.org/guestAuth/repository/download/BumbershootSubsetSourceTarball/<{ buildid }}:id/bumbershoot-src-without-t-<{ version }}.tar.bz2'), ('sha1', '05cd791cd3f406b32290def486547c44b36a7fb2')])), ('build', CommentedMap([('script', 'bash -c \"./quickbuild.sh -j<{ CPU_COUNT }} address-model=64 runtime-link=shared <{ targets }} && cp build-linux-x86_64/gcc-release-x86_64/* $PREFIX/bin\"'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://proteowizard.sourceforge.net'), ('license', 'Apache 2.0'), ('summary', 'The Bumbershoot tool suite for analyzing shotgun proteomic data')])), ('test', CommentedMap([('commands', ['myrimatch --help', 'directag --help', 'quameter --help', 'pepitome --help'])]))])), 'busco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/busco/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 3)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://gitlab.com/ezlab/busco/repository/archive.tar.bz2?ref=<{ version }}'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'wget', 'blast', 'hmmer', 'augustus >=3.2.3', 'r-ggplot2 >=2.2.1'])])), ('test', CommentedMap([('commands', ['run_BUSCO.py -h', 'run_busco -h', 'generate_plot.py -h', 'generate_plot -h', 'run_busco | grep \"Please add it in the config file\"'])])), ('about', CommentedMap([('home', 'http://busco.ezlab.org/'), ('license', 'GPL'), ('summary', 'BUSCO provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs selected from OrthoDB.')]))])), 'bwa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bwa/0.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.7.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.7.8.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.8.tar.bz2'), ('md5', '51ca7a29907022080398b2b88207e33c')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))])), 'bwakit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bwakit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bwakit'), ('version', '0.7.12')])), ('source', CommentedMap([('fn', 'bwakit-0.7.12_x64-linux.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwakit/bwakit-0.7.12_x64-linux.tar.bz2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', [\"run-bwamem 2>&1 | grep 'Usage:   run-bwamem'\"])])), ('about', CommentedMap([('home', 'https://github.com/lh3/bwa/tree/master/bwakit'), ('license', 'GPLv3'), ('summary', 'A self-consistent installation-free package of scripts and precompiled binaries, providing an end-to-end solution to read mapping')]))])), 'bwameth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bwameth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bwameth'), ('version', '0.2.0')])), ('source', CommentedMap([('fn', 'bwa-meth-0.2.0.tar.gz'), ('sha256', '726b6d0d24b51ff522eaeb72e782ac99cca286bbb8fcfcc55759eafa01132274'), ('url', 'https://github.com/brentp/bwa-meth/archive/v0.2.0.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'toolshed >=0.3.9']), ('run', ['python', 'toolshed >=0.3.9', 'bwa', 'samtools'])])), ('test', CommentedMap([('commands', ['bwameth.py --version'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/bwa-meth'), ('license', 'MIT'), ('summary', 'A fast and accurate aligner of BS-seq reads')]))])), 'bx-python': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bx-python/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bx-python'), ('version', '0.7.3')])), ('source', CommentedMap([('fn', 'bx-python-0.7.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/bx-python/bx-python-0.7.3.tar.gz'), ('md5', 'd8c50c01c9e421bae0bbdbfa00fef6e4')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'nose', 'numpy x.x', 'gcc']), ('run', ['python', 'setuptools', 'numpy x.x', 'libgcc'])])), ('test', CommentedMap([('imports', ['bx.align.sitemask', 'bx.tabular', 'bx.intervals', 'bx.align', 'bx.bbi', 'bx.bbi.bigwig_file', 'bx.cookbook', 'bx.arrays', 'bx.motif.io', 'bx.seq', 'bx.intervals.operations', 'bx.pwm', 'bx.misc', 'bx.motif', 'bx.phylo', 'bx.motif.logo', 'bx_extras', 'bx.align.tools', 'bx', 'bx.intseq'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/james_taylor/bx-python/wiki/Home'), ('license', 'MIT'), ('summary', 'Toolkit to enable rapid implementation of genome scale analyses.')]))])), 'byobu': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/byobu/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'byobu'), ('version', '5.98')])), ('source', CommentedMap([('fn', 'byobu_5.98.orig.tar.gz'), ('url', 'https://launchpad.net/byobu/trunk/5.98/+download/byobu_5.98.orig.tar.gz'), ('md5', '2e09c9d2f9e3b6e2953bd581b027239c')])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'newt', 'ncurses <{CONDA_NCURSES}}*', 'libevent', 'tmux']), ('run', ['libgcc', 'newt', 'ncurses <{CONDA_NCURSES}}*', 'libevent', 'tmux'])])), ('test', CommentedMap([('commands', ['byobu -c \"echo \\'hello world\\'\"'])])), ('about', CommentedMap([('home', 'http://byobu.co/'), ('license', 'GPLv3'), ('summary', 'Byobu is a GPLv3 open source text-based window manager and terminal multiplexer.')]))])), 'c-ares': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/c-ares/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'c-ares'), ('version', '1.11.0')])), ('source', CommentedMap([('fn', 'c-ares-1.11.0.tar.gz'), ('url', 'http://c-ares.haxx.se/download/c-ares-1.11.0.tar.gz'), ('md5', 'd5c6d522cfc54bb6f215a0b7912d46be')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/lib/libcares.a', 'test -f ${PREFIX}/lib/libcares.so.2.1.0', 'test -f ${PREFIX}/lib/libcares.2.dylib', 'test -f ${PREFIX}/include/ares.h'])])), ('about', CommentedMap([('home', 'http://c-ares.haxx.se/'), ('summary', 'c-ares is a C library for asynchronous DNS requests (including name resolves)'), ('license', 'MIT')]))])), 'cage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cage'), ('version', '2016.05.13')])), ('source', CommentedMap([('git_url', 'https://github.com/adambloniarz/CAGe.git'), ('git_rev', 'fcaa9bb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['bamtools', 'sqlite']), ('run', ['python', 'bamtools', 'sqlite'])])), ('test', CommentedMap([('commands', ['cage --version', 'bamdump --help'])])), ('about', CommentedMap([('home', 'https://github.com/adambloniarz/CAGe'), ('license', 'Apache v2'), ('summary', 'Changepoint Analysis for Efficient Variant Calling')]))])), 'cancerit-allelecount': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cancerit-allelecount/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cancerit-allelecount'), ('version', '2.1.2')])), ('source', CommentedMap([('fn', 'alleleCount-v2.1.2.tar.gz'), ('url', 'https://github.com/cancerit/alleleCount/archive/v2.1.2.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['alleleCounter -h'])])), ('about', CommentedMap([('home', 'https://github.com/cancerit/alleleCount'), ('license', 'GPLv3'), ('summary', 'Support code for NGS copy number algorithms')]))])), 'cansam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cansam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cansam'), ('version', '21d64bb')])), ('source', CommentedMap([('fn', '21d64bb.tar.gz'), ('url', 'https://github.com/jmarshall/cansam/archive/21d64bb.tar.gz'), ('md5', '017d2ebc7338be995af3b9e6056f5fe0'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'zlib']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'zlib'])])), ('test', CommentedMap([('commands', ['samcat --help'])])), ('about', CommentedMap([('home', 'https://github.com/jmarshall/cansam/'), ('license', 'BSD-3-Clause'), ('summary', 'C++ binding for SAM/BAM files'), ('license_family', 'BSD')]))])), 'cansnper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cansnper/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/adrlar/CanSNPer/archive/v<{ version }}.tar.gz'), ('md5', '3d46bb7caa3d6bdbb9c5d583636cb5e2')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ete2', 'numpy', 'pyqt 4.*']), ('run', ['python', 'ete2', 'numpy', 'pyqt 4.*'])])), ('test', CommentedMap([('commands', ['CanSNPer --help'])])), ('about', CommentedMap([('home', 'https://github.com/adrlar/CanSNPer/'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'A hierarchical genotype classifier of clonal pathogens.')]))])), 'canu': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/canu/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/marbl/<{ name|lower }}/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'gnuplot >=5.0.5']), ('run', ['libgcc', 'perl', 'perl-filesys-df', 'openjdk', 'gnuplot >=5.0.5'])])), ('test', CommentedMap([('commands', ['canu --version'])])), ('about', CommentedMap([('home', 'http://http://canu.readthedocs.org/'), ('license', 'GPLv2 and others'), ('summary', 'Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing.')]))])), 'canvas': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/canvas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'canvas'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'Canvas-<{ version }}_x64.tar.gz'), ('url', 'https://github.com/Illumina/canvas/releases/download/v<{ version }}/Canvas-<{ dlversion }}_x64.tar.gz'), ('md5', '33516d6034957300a90c7da6c5e43eb6')])), ('requirements', CommentedMap([('build', None), ('run', ['mono'])])), ('test', CommentedMap([('commands', ['Canvas -h', 'EvaluateCNV -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/canvas'), ('license', 'GPLv3'), ('summary', 'Copy number variant (CNV) calling from DNA sequencing data')]))])), 'cap-mirseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cap-mirseq/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bioinformaticstools.mayo.edu/research/cap-mirseq/'), ('license', 'GPL'), ('summary', 'A comprehensive analysis pipeline for deep microRNA sequencing')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'cap-mirseq'), ('version', '2014')])), ('requirements', CommentedMap([('run', ['perl', 'openjdk', 'r-base', 'r-essentials', 'bioconductor-edger >=3.14.0', 'bowtie >=1.1.2', 'samtools >=0.1.19', 'picard >=1.97', 'fastqc >=0.11.5', 'bedtools >=2.17.0', 'mirdeep2 >=2.0.0.7', 'vcftools >=0.1.11'])])), ('source', CommentedMap([('fn', 'cap-mirseq.tar.gz'), ('url', 'https://s3-us-west-2.amazonaws.com/mayo-bic-tools/cap-mirna/CAP-miRSEQ.tgz'), ('md5', '00e158d8585decd854998db58f821da0')])), ('test', CommentedMap([('commands', [\"miRDeep2.pl --help 2>&1 | grep -A20 -B20 'Example of use'\"])]))])), 'cap3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cap3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cap3'), ('version', '10.2011')])), ('source', CommentedMap([('fn', 'cap3.macosx.intel64.tar'), ('url', 'http://seq.cs.iastate.edu/CAP3/cap3.macosx.intel64.tar')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['cap3 2>&1 | grep \"02/10/15\"', 'cap3 2>&1 | grep \"12/21/07\"'])])), ('about', CommentedMap([('home', 'http://seq.cs.iastate.edu/'), ('license', 'Michigan Tech.'), ('authors', 'Xiaoqiu Huang and Anup Madan')]))])), 'carna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/carna/1.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'carna'), ('version', '1.3.2')])), ('about', CommentedMap([('home', 'https://www.bioinf.uni-leipzig.de/~will/Software/CARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Constraint-based Alignment of RNA Ensembles')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'carna-1.3.2.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/Releases/carna-1.3.2.tar.gz'), ('sha256', '279ce4417c8cd637511d4d35efb83cb63194cee7608d89a75542f14d4c5ee48c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gecode 4.4.0', 'viennarna 2.2.8', 'locarna 1.8.11']), ('run', ['libgcc', 'gecode 4.4.0', 'locarna 1.8.11'])])), ('test', CommentedMap([('commands', ['carna --version'])]))])), 'cassiopee': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cassiopee/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cassiopee'), ('version', '1.0.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '1.0.5.tar.gz'), ('url', 'https://github.com/osallou/cassiopee-c/archive/1.0.5.tar.gz'), ('md5', '38f07e5d2ee2a55679281bce73f9993e')])), ('requirements', CommentedMap([('build', ['glog', 'cppunit', 'boost ==<{CONDA_BOOST}}', 'autoconf', 'gcc', 'llvm', 'cmake', 'icu 58.*', '<{CONDA_BZIP2}}', 'zlib ==<{CONDA_ZLIB}}']), ('run', ['glog', 'libgcc', 'boost ==<{CONDA_BOOST}}', 'icu 58.*', '<{CONDA_BZIP2}}', 'zlib ==<{CONDA_ZLIB}}'])])), ('test', CommentedMap([('commands', ['cassiopee -h'])])), ('about', CommentedMap([('home', 'https://github.com/osallou/cassiopee-c'), ('license', 'GPL-3+'), ('license_file', 'LICENSE'), ('summary', 'scan an input genomic sequence (dna/rna/protein) and search for a subsequence with exact match or allowing substitutions (Hamming distance) and/or insertion/deletions')]))])), 'cd-hit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cd-hit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cd-hit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cd-hit-<{ version }}.tar.gz'), ('url', 'https://github.com/weizhongli/cdhit/archive/V<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"cd-hit --help | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/weizhongli/cdhit'), ('license', 'GPLv2'), ('summary', 'Clusters and compares protein or nucleotide sequences')]))])), 'cd-hit-auxtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cd-hit-auxtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cd-hit-auxtools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cd-hit-<{ version }}.tar.gz'), ('url', 'https://github.com/weizhongli/cdhit/archive/V<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"cd-hit-dup --help | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/weizhongli/cdhit'), ('license', 'GPLv2'), ('summary', 'Clusters and compares protein or nucleotide sequences')]))])), 'cdbtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cdbtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cdbtools'), ('version', 0.99)])), ('source', CommentedMap([('fn', 'cdbfasta.tar.gz'), ('url', 'ftp://occams.dfci.harvard.edu/pub/bio/tgi/software/cdbfasta/cdbfasta.tar.gz'), ('md5', 'ed07e474d67e0e79e7e89f9f43ee96df'), ('patches', ['zlib.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib', 'gcc', 'llvm']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['cdbfasta -v', 'cdbyank -v'])])), ('about', CommentedMap([('home', 'http://compbio.dfci.harvard.edu/tgi'), ('license', 'Public Domain'), ('summary', 'CDB (Constant DataBase) indexing and retrieval tools for FASTA files')]))])), 'ceas': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ceas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ceas'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'CEAS-Package-1.0.2.tar.gz'), ('md5', 'faf202ff613794ab9cd07f282c6e2663'), ('url', 'http://liulab.dfci.harvard.edu/CEAS/src/CEAS-Package-1.0.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k or osx]')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'mysql-python']), ('run', ['python', 'mysql-python'])])), ('test', CommentedMap([('imports', ['CEAS']), ('commands', ['ceas --version > /dev/null', 'build_genomeBG  --version > /dev/null', 'sitepro  --version > /dev/null', 'gca --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://liulab.dfci.harvard.edu/CEAS'), ('license', 'Artistic'), ('license_file', 'Artistic_License.pdf'), ('summary', 'CEAS: Cis-regulatory Element Annotation System')]))])), 'centrifuge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/centrifuge/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/infphilo/centrifuge/archive/f39767eb57d8e175029c.tar.gz'), ('md5', '17750d3be1039d2b428fe237a9cd7e45')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'python']), ('run', ['libgcc', 'zlib', 'perl', 'python'])])), ('test', CommentedMap([('commands', ['centrifuge --help'])])), ('about', CommentedMap([('home', 'https://github.com/infphilo/centrifuge'), ('license', 'GPL3'), ('summary', 'Classifier for metagenomic sequences.')]))])), 'cgat-pipelines-nosetests': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgat-pipelines-nosetests/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgat-pipelines-nosetests'), ('version', '0.0.4')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('run', ['nomkl', 'libgcc', 'zlib', 'cgat-scripts-nosetests', 'bedtools', 'samtools', 'python', 'cgat-report', 'biopython', 'cython', 'brewer2mpl', 'beautifulsoup4', 'python-drmaa', 'future', 'httplib2', 'intermine', 'matplotlib', 'mygene', 'mysql-python', 'nose', 'numpy', 'pandas', 'pep8', 'pika', 'pybedtools', 'pybigwig', 'psycopg2', 'pysam', 'rpy2', 'ruffus', 'scipy', 'seaborn', 'setuptools', 'six', 'scikit-learn', 'sqlalchemy', 'toposort', 'pyyaml', 'r-base', 'bioconductor-hpar'])])), ('test', CommentedMap([('imports', ['numpy', 'pandas', 'pysam', 'pybedtools', 'rpy2']), ('commands', ['python --version', 'R --version'])])), ('about', CommentedMap([('home', 'https://www.cgat.org/downloads/public/cgatpipelines/documentation'), ('license', 'MIT'), ('summary', 'Metapackage to test CGAT Pipelines')]))])), 'cgat-report': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgat-report/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgat-report'), ('version', '0.7.6.1')])), ('source', CommentedMap([('fn', 'CGATReport-0.7.6.1.tar.gz'), ('url', 'https://pypi.python.org/packages/df/d5/eaa91e9d1f0582d06016205026f18c383d28331fc5cf20bf804f3c0687c1/CGATReport-0.7.6.1.tar.gz'), ('md5', 'aacacfb5a0221b0c535508216a567fde')])), ('build', CommentedMap([('skip', False), ('number', 1)])), ('requirements', CommentedMap([('build', ['bokeh', 'docutils', 'future', 'matplotlib >=2.0', 'nose', 'numpy', 'pandas', 'scipy', 'seaborn', 'six', 'sphinx', 'sqlalchemy', 'python', 'matplotlib-venn', 'seaborn', 'sqlalchemy', 'openpyxl', 'mpld3', 'pillow', 'pep8']), ('run', ['bokeh', 'docutils', 'future', 'matplotlib >=2.0', 'nose', 'numpy', 'pandas', 'scipy', 'seaborn', 'six', 'sphinx', 'sqlalchemy', 'python', 'matplotlib-venn', 'seaborn', 'sqlalchemy', 'openpyxl', 'mpld3', 'pillow', 'pep8'])])), ('test', CommentedMap([('commands', ['cgatreport-clean -h'])])), ('about', CommentedMap([('home', 'https://github.com/AndreasHeger/CGATReport'), ('license', 'BSD'), ('summary', 'A report generator in python based on sphinx')]))])), 'cgat-scripts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgat-scripts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgat-scripts'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/C/CGAT/CGAT-<{ version }}.tar.gz'), ('md5', '7aca54f911659af4952c48939abc12ea')])), ('build', CommentedMap([('skip', True), ('number', 0), ('entry_points', ['cgat = CGAT.cgat:main'])])), ('requirements', CommentedMap([('build', ['python', 'alignlib-lite 0.3.*', 'biopython 1.69.*', 'cython 0.25.*', 'future 0.16.*', 'matplotlib 2.0.*', 'nose 1.3.*', 'numpy 1.12.*', 'pandas 0.20.*', 'pep8 1.7.*', 'pybedtools 0.7.*', 'pysam 0.11.1', 'python-lzo 1.11.*', 'pyyaml 3.12.*', 'rpy2 2.8.*', 'scipy 0.19.*', 'setuptools 27.2.*', 'six 1.10.*', 'bedtools 2.26.*', 'coreutils 8.25', 'gcc 4.8.*', 'grep 2.14', 'libpng 1.6.*', 'ucsc-bedgraphtobigwig 332', 'ucsc-bedtobigbed 332', 'ucsc-wigtobigwig 332', 'zlib 1.2.*']), ('run', ['python', 'alignlib-lite 0.3.*', 'biopython 1.69.*', 'cython 0.25.*', 'future 0.16.*', 'matplotlib 2.0.*', 'nose 1.3.*', 'numpy 1.12.*', 'pandas 0.20.*', 'pep8 1.7.*', 'pybedtools 0.7.*', 'pysam 0.11.1', 'python-lzo 1.11.*', 'pyyaml 3.12.*', 'rpy2 2.8.*', 'scipy 0.19.*', 'six 1.10.*', 'bedtools 2.26.*', 'coreutils 8.25', 'libgcc 4.8.*', 'grep 2.14', 'libpng 1.6.*', 'ucsc-bedgraphtobigwig 332', 'ucsc-bedtobigbed 332', 'ucsc-wigtobigwig 332', 'zlib 1.2.*'])])), ('test', CommentedMap([('imports', ['numpy', 'pandas', 'pysam', 'pybedtools', 'rpy2']), ('commands', ['cgat --help >& /dev/null', 'cgat --help Genomics >& /dev/null', 'cgat --help Conversion >& /dev/null'])])), ('about', CommentedMap([('home', 'https://www.cgat.org/downloads/public/cgat/documentation'), ('license', 'BSD'), ('summary', 'Computational Genomics Analysis Toolkit')]))])), 'cgat-scripts-devel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgat-scripts-devel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgat-scripts-devel'), ('version', '0.2.6')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('run', ['nomkl', 'libgcc', 'zlib', 'libpng', 'mysqlclient', 'bedtools', 'samtools', 'ucsc-wigtobigwig', 'ucsc-bedtobigbed', 'ucsc-bedgraphtobigwig', 'python', 'biopython', 'cython', 'mysql-python', 'alignlib-lite', 'bx-python', 'future', 'ggplot', 'jinja2', 'matplotlib', 'networkx', 'nose', 'numpy', 'openpyxl', 'pandas', 'pep8', 'psycopg2', 'pybedtools', 'pysam', 'python-weblogo', 'rdflib', 'rpy2', 'scipy', 'setuptools', 'six', 'scikit-learn', 'python-weblogo', 'pyyaml', 'r-base', 'r-coloc', 'r-flashclust', 'r-ggplot2', 'r-gplots', 'r-gridbase', 'r-gridextra', 'r-gtools', 'r-maps', 'r-optparse', 'r-plyr', 'r-qqman', 'r-rcolorbrewer', 'r-reshape', 'r-reshape2', 'r-samr', 'r-scales', 'r-sleuth', 'r-snow', 'r-vegan', 'r-venndiagram', 'r-wgcna', 'bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-hilbertvis', 'bioconductor-ihw', 'bioconductor-limma', 'bioconductor-masigpro', 'bioconductor-medips', 'bioconductor-metagenomeseq', 'bioconductor-qvalue', 'bioconductor-rtracklayer', 'bioconductor-siggenes', 'libglu'])])), ('test', CommentedMap([('imports', ['numpy', 'pandas', 'pysam', 'pybedtools', 'rpy2']), ('commands', ['python --version', 'R --version', 'samtools --version', 'bedtools --version'])])), ('about', CommentedMap([('home', 'https://www.cgat.org/downloads/public/cgat/documentation'), ('license', 'BSD'), ('summary', 'Computational Genomics Analysis Toolkit')]))])), 'cgat-scripts-nosetests': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgat-scripts-nosetests/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgat-scripts-nosetests'), ('version', '0.0.4')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('run', ['nomkl', 'libgcc', 'zlib', 'libpng', 'bedtools', 'samtools', 'ucsc-wigtobigwig', 'ucsc-bedtobigbed', 'ucsc-bedgraphtobigwig', 'python', 'biopython', 'cython', 'alignlib-lite', 'bx-python', 'future', 'ggplot', 'jinja2', 'matplotlib', 'networkx', 'nose', 'numpy', 'openpyxl', 'pandas', 'pep8', 'pybedtools', 'pysam', 'python-weblogo', 'rdflib', 'rpy2', 'scipy', 'setuptools', 'six', 'scikit-learn', 'pyyaml', 'r-base', 'r-coloc', 'r-ggplot2', 'r-qqman'])])), ('test', CommentedMap([('imports', ['numpy', 'pandas', 'pysam', 'pybedtools', 'rpy2']), ('commands', ['python --version', 'R --version'])])), ('about', CommentedMap([('home', 'https://www.cgat.org/downloads/public/cgat/documentation/'), ('license', 'BSD'), ('summary', 'Computational Genomics Analysis Toolkit')]))])), 'cgcloud-lib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cgcloud-lib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cgcloud-lib'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'cgcloud-lib-1.6.0.tar.gz'), ('url', 'https://pypi.python.org/packages/71/e3/0ce3f6b3e3f11f37a51634cd84fa5d95bcee418843ef9b30239de4df0160/cgcloud-lib-1.6.0.tar.gz'), ('md5', '76cb2e9b3b6f716c0790bf7a55594c94')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bd2k-python-lib', 'boto ==2.38.0']), ('run', ['python', 'bd2k-python-lib', 'boto ==2.38.0'])])), ('test', CommentedMap([('imports', ['cgcloud', 'cgcloud.lib', 'cgcloud_Crypto', 'cgcloud_Crypto.Hash', 'cgcloud_Crypto.IO', 'cgcloud_Crypto.PublicKey', 'cgcloud_Crypto.Util'])])), ('about', CommentedMap([('home', 'https://github.com/BD2KGenomics/cgcloud'), ('license', 'Apache 2.0'), ('summary', 'Components shared between cgcloud-core and cgcloud-agent')]))])), 'chainmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chainmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chainmap'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'chainmap-1.0.2.tar.gz'), ('url', 'https://pypi.python.org/packages/56/09/0f2f2e693bb102083b7e77fc2b7f97b9195dc4c90dcfa85038a2d128161b/chainmap-1.0.2.tar.gz'), ('md5', '2ddf536db921c82c9559791047b88caf')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools']), ('run', ['python >=2.7,<3'])])), ('test', CommentedMap([('imports', ['chainmap'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jeunice/chainmap'), ('license', 'Python Software Foundation License'), ('summary', 'Backport/clone of ChainMap for py26, py32, and pypy3.')]))])), 'chanjo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chanjo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chanjo'), ('version', '3.3.0')])), ('source', CommentedMap([('fn', 'chanjo-3.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/c/chanjo/chanjo-3.3.0.tar.gz'), ('md5', '7b7d67863c1cb74a731b1ddb79d5595a')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True), ('entry_points', ['chanjo = chanjo.__main__:root_command']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click', 'toolz', 'path.py', 'pyyaml', 'sqlalchemy >=0.8.2']), ('run', ['python', 'click', 'toolz', 'path.py', 'pyyaml', 'sqlalchemy >=0.8.2', 'sambamba'])])), ('test', CommentedMap([('imports', ['chanjo', 'chanjo.annotate', 'chanjo.annotate.sambamba', 'chanjo.bootstrap', 'chanjo.cli', 'chanjo.config', 'chanjo.demo', 'chanjo.load', 'chanjo.parse', 'chanjo.parse.bed', 'chanjo.parse.sambamba', 'chanjo.sex', 'chanjo.store']), ('commands', ['LANG=C.UTF-8 chanjo --help'])])), ('about', CommentedMap([('home', 'http://www.chanjo.co/'), ('license', 'MIT License'), ('summary', 'Coverage analysis tool for clinical sequencing')]))])), 'check-sort-order': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/check-sort-order/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'check-sort-order'), ('version', '0.0.3')])), ('source', CommentedMap([('fn', 'check-sort-order-linux_amd64'), ('url', 'https://github.com/gogetdata/ggd-utils/releases/download/v0.0.3/check-sort-order-linux_amd64')])), ('build', CommentedMap([('number', 0)])), ('requirements', None), ('test', CommentedMap([('commands', ['check-sort-order --help'])])), ('about', CommentedMap([('home', 'https://github.com/gogetdata/ggd-utils'), ('license', 'MIT'), ('summary', 'check sort-order of genomic files according to a genomefile')]))])), 'checkm-genome': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/checkm-genome/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'checkm-genome'), ('version', '1.0.7')])), ('source', CommentedMap([('fn', 'checkm-genome-1.0.7.tar.gz'), ('url', 'https://pypi.python.org/packages/55/62/35b519866fff020ef0b213e0665dc4d43cb1bb588a6575dd3270d3708095/checkm-genome-1.0.7.tar.gz'), ('md5', 'e62f95b73c11cb4b68ce32d840a5e17e')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.8.0', 'scipy >=0.9.0', 'matplotlib >=1.3.1', 'pysam >=0.8.3', 'dendropy >=4.0.0', 'screamingbackpack >=0.2.333']), ('run', ['python', 'numpy >=1.8.0', 'scipy >=0.9.0', 'matplotlib >=1.3.1', 'pysam >=0.8.3', 'dendropy >=4.0.0', 'screamingbackpack >=0.2.333'])])), ('test', CommentedMap([('imports', ['checkm', 'checkm.plot', 'checkm.test', 'checkm.util'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/checkm/'), ('license', 'GPL3'), ('summary', 'Assess the quality of putative genome bins.')]))])), 'chemfp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chemfp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chemfp'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'chemfp-1.3.tar.gz'), ('url', 'http://dalkescientific.com/releases/chemfp-1.3.tar.gz'), ('sha256', '86f77a0ae85004dc6329d8dd74dc7c81c7a2abf7a7f4023845f01b4b6953d65b')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'openbabel', 'rdkit']), ('run', ['python', 'openbabel', 'rdkit'])])), ('test', CommentedMap([('imports', ['chemfp', 'chemfp.commandline', 'chemfp.futures', 'chemfp.progressbar'])])), ('about', CommentedMap([('home', 'https://chemfp.com'), ('license', 'MIT License'), ('summary', 'chemfp is a set of command-lines tools for generating cheminformatics fingerprints and searching those\\nfingerprints by Tanimoto similarity, as well as a Python library which can be used to build new tools.\\n\\nThese algorithms are designed for the dense, 100-10,000 bit\\nfingerprints which occur in small-molecule/pharmaceutical\\nchemisty. The Tanimoto search algorithms are implemented in C for\\nperformance and support both threshold and k-nearest searches.\\n\\nFingerprint generation can be done either by extracting existing\\nfingerprint data from an SD file or by using an existing chemistry\\ntoolkit. chemfp supports the Python libraries from Open Babel,\\nOpenEye, and RDKit toolkits.\\n')]))])), 'chewbbaca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chewbbaca/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/B-UMMI/chewBBACA/archive/<{ version }}.tar.gz'), ('sha256', 'c4a34c3ecd780cacfdabbb116cba36ced11cba36fefdf4ca9bea9ecd3c918d71')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy 1.13.1', 'scipy >=0.13.3', 'plotly 1.12.*', 'biopython 1.70', 'prodigal >=2.6.0', 'blast >=2.5.0'])])), ('test', CommentedMap([('commands', ['chewBBACA.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/B-UMMI/chewBBACA'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Whole genome and core genome MultiLocus Sequence Typing.')]))])), 'chicagotools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chicagotools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chicagotools'), ('version', '1.2.0')])), ('source', CommentedMap([('url', 'https://bitbucket.org/chicagoTeam/chicago/get/c95eda46cb72.zip'), ('md5', '28498241217219ce6db5752615868690'), ('patches', ['patchFitDistCurve.txt', 'patchMakeDesignFile.txt', 'patchMakeNBaitsPerBinFile.txt', 'patchMakeNPerBinFile.txt', 'patchMakePeakMatrix.txt', 'patchMakeProxOEFile.txt', 'patchRunChicago.txt'])])), ('requirements', CommentedMap([('run', ['python', 'r-base', 'bioconductor-chicago', 'r-argparser'])])), ('test', CommentedMap([('commands', ['bam2chicago.sh -h', 'fitDistCurve.R -h', 'runChicago.R -h', 'makeNBaitsPerBinFile.py -h || [[ \"$?\" == 120 ]]', 'makeNPerBinFile.py -h || [[ \"$?\" == 120 ]]', 'makeProxOEFile.py -h || [[ \"$?\" == 120 ]]', 'makeDesignFiles.py -h || [[ \"$?\" == 120 ]]'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/chicagoTeam/chicago/src/c95eda46cb72e30e25ece06780e517efb3c06cc2/chicagoTools/?at=master'), ('license', 'artistic license 2.0')]))])), 'chromhmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chromhmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chromhmm'), ('version', 1.12)])), ('source', CommentedMap([('fn', 'ChromHMM.zip'), ('url', 'http://compbio.mit.edu/ChromHMM/ChromHMM.zip'), ('md5', '0689d01a9a80a57cd3cb3fc49efa443d')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk', 'unzip'])])), ('test', CommentedMap([('commands', [\"ChromHMM.sh -Xmx512M BinarizeBam | grep -q 'usage BinarizeBam'\"])])), ('about', CommentedMap([('home', 'http://compbio.mit.edu/ChromHMM/'), ('license', 'GPLv3'), ('summary', 'ChromHMM is software for learning and characterizing chromatin states. ChromHMM can integrate multiple chromatin datasets such as ChIP-seq data of various histone modifications to discover de novo the major re-occuring combinatorial and spatial patterns of marks.')])), ('extra', CommentedMap([('notes', 'ChromHMM comes with about 36MB of example data which is not included in the recipe. This recipe installs a script, \"download_chromhmm_data.sh\", which downloads the data in the proper location, and which can be run after ChromHMM has been installed.')]))])), 'circexplorer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circexplorer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'circexplorer'), ('version', '1.1.10')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'CIRCexplorer-1.1.10.tar.gz'), ('url', 'https://pypi.python.org/packages/f0/3c/a93e1572570fe42222c77669232e9ebb686092969490bd4b81d5daafbe3d/CIRCexplorer-1.1.10.tar.gz'), ('md5', '4ed911a2308a41ba021bf04982dcf049')])), ('requirements', CommentedMap([('build', ['python', 'pysam >=0.8.4', 'docopt']), ('run', ['python', 'pysam >=0.8.4', 'docopt'])])), ('test', CommentedMap([('commands', ['CIRCexplorer.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/YangLab/CIRCexplorer'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'A combined strategy to identify circular RNAs (circRNAs and ciRNAs)')]))])), 'circexplorer2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circexplorer2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'circexplorer2'), ('version', '2.3.1')])), ('source', CommentedMap([('fn', 'CIRCexplorer2-2.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/91/97/37698fec559743e51a4bbcd7028eaa0eba982ecd2ad7917ab72a62ecd73e/CIRCexplorer2-2.3.1.tar.gz'), ('md5', '16958567416ba8b718483d1c3194d70f')])), ('build', CommentedMap([('entry_points', ['CIRCexplorer2=circ2.command_parse:main', 'fetch_ucsc.py=circ2.fetch_ucsc:main', 'fast_circ.py=circ2.fast_circ:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'requests', 'pysam >=0.8.4', 'pybedtools', 'docopt', 'scipy']), ('run', ['python', 'requests', 'pysam >=0.8.4', 'pybedtools', 'docopt', 'scipy'])])), ('test', CommentedMap([('imports', ['circ2.align', 'circ2.parse', 'circ2.annotate', 'circ2.assemble', 'circ2.denovo', 'circ2.fast_circ', 'circ2.fetch_ucsc'])])), ('about', CommentedMap([('home', 'https://github.com/YangLab/CIRCexplorer2'), ('license', 'MIT License'), ('summary', 'Circular RNA analysis toolkits')]))])), 'circlator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circlator/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/10/16/0024094199f7613f7037a0b7c47dcabde7191fa171c9f5d4b5388ce693f6/circlator-1.5.1.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'openpyxl', 'pyfastaq >=3.12.1', 'pysam >=0.8.1,<=0.9.1', 'pymummer >=0.9.0', 'bio_assembly_refinement >=0.4.0']), ('run', ['python', 'openpyxl', 'pyfastaq >=3.12.1', 'pysam >=0.8.1,<=0.9.1', 'pymummer >=0.9.0', 'bio_assembly_refinement >=0.4.0', 'bwa', 'prodigal', 'samtools', 'spades'])])), ('test', CommentedMap([('imports', ['circlator', 'circlator.tasks']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/circlator'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'circlator: a tool to circularise genome assemblies'), ('license_family', 'GPL')]))])), 'circos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'circos'), ('version', '<{ version }}.<{ rev }}')])), ('source', CommentedMap([('fn', 'circos-<{ version }}-<{ rev }}.tgz'), ('url', 'http://circos.ca/distribution/circos-<{ version }}-<{ rev }}.tgz'), ('sha256', '3eb868b8f5881f73d714115ce3d235f79f4353e1879e17a479a8d9d5a3627d5f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'perl-clone', 'perl-config-general', 'perl-digest-perl-md5', 'perl-font-ttf', 'perl-gd', 'perl-list-moreutils', 'perl-math-bezier', 'perl-math-round', 'perl-math-vecstat', 'perl-params-validate', 'perl-readonly', 'perl-regexp-common', 'perl-svg', 'perl-set-intspan', 'perl-statistics-basic', 'perl-text-format', 'perl-time-hires'])])), ('test', CommentedMap([('commands', ['circos -modules', 'circos -v'])])), ('about', CommentedMap([('home', 'http://circos.ca'), ('summary', 'Circos is a software package for visualizing data and information. It visualizes data in a circular layout'), ('license', 'GNU General Public License v2 (GPLv2)'), ('license_family', 'GPL2')]))])), 'ciso8601': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ciso8601/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ciso8601'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'ciso8601-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/c/ciso8601/ciso8601-1.0.1.tar.gz'), ('md5', 'feb9bf324a903daeeee40d28d20960cb')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['ciso8601'])])), ('about', CommentedMap([('home', 'https://github.com/elasticsales/ciso8601'), ('summary', 'Fast ISO8601 date time parser for Python written in C'), ('license', 'MIT')]))])), 'cityhash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cityhash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cityhash'), ('version', '0.1.7')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'cityhash-0.1.7.tar.gz'), ('url', 'https://pypi.python.org/packages/c8/7b/9ae60cb970412362f654fe1469b5b8fb230f6089ea0956fff2469d844be5/cityhash-0.1.7.tar.gz'), ('md5', 'ecddde03ae5f62eec8551b3c20632c2a')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['cityhash'])])), ('about', CommentedMap([('home', 'https://github.com/escherba/python-cityhash'), ('license', 'MIT License'), ('summary', 'Python-bindings for CityHash, a fast non-cryptographic hash algorithm')]))])), 'clark': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clark/1.2.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clark'), ('version', '1.2.3')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'CLARKV1.2.3.tar.gz'), ('url', 'http://clark.cs.ucr.edu/Download/CLARKV1.2.3.tar.gz'), ('md5', '32d5643476452c7e70539663fe253ff2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CLARK --help'])])), ('about', CommentedMap([('home', 'http://clark.cs.ucr.edu/'), ('license', 'GPL3'), ('license_file', 'LICENSE_GNU_GPL.txt'), ('summary', 'Fast, accurate and versatile k-mer based classification system')]))])), 'clearcut': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clearcut/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clearcut'), ('version', '1.0.9')])), ('source', CommentedMap([('fn', 'Clearcut.source.zip'), ('url', 'http://www.mothur.org/w/images/9/91/Clearcut.source.zip'), ('md5', 'c7c01922d75d5bb6e218d7a9adc72978')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('test', CommentedMap([('commands', ['clearcut -V'])])), ('about', CommentedMap([('home', 'http://www.mothur.org'), ('license', 'BSD'), ('license_file', 'clearcut/LICENSE'), ('summary', 'The reference implementation for Relaxed Neighbor Joining (RNJ).')]))])), 'clever-toolkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clever-toolkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clever-toolkit'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 3), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('source', CommentedMap([('fn', 'clever-toolkit-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/tobiasmarschall/clever-toolkit/downloads/clever-toolkit-v<{ version }}.tar.gz'), ('sha256', '5e48d82c31d988c9e971802d899fb588c55fcb5418f8f43d9639054ba37db230')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'cmake', 'zlib']), ('run', ['libgcc', 'python', 'boost <{CONDA_BOOST}}*', 'zlib', 'bedtools 2.26.*', 'samtools 1.*', 'bcftools 1.*'])])), ('test', CommentedMap([('commands', ['clever --help > /dev/null', 'laser --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/tobiasmarschall/clever-toolkit'), ('license', 'GPLv3'), ('summary', 'The clever toolkit (CTK) is a suite of tools to analyze next-generation sequencing data and, in particular, to discover and genotype insertions and deletions from paired-end reads.')]))])), 'clinod': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clinod/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clinod'), ('version', 1.3)])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/clinod/clinod_1.3_src_all.jar'), ('fn', 'clinod-1.3.jar'), ('sha256', '45d80662ba109a7af28aeafcfb2ca417a7d575ac3700f69bd40965a69d41e072')])), ('requirements', CommentedMap([('build', ['snns']), ('run', ['python', 'openjdk >=6', 'snns'])])), ('about', CommentedMap([('home', 'http://www.compbio.dundee.ac.uk/nod'), ('license', 'Apache License, Version 2.0 + others used internally'), ('summary', 'Command line NoD (clinod), for  predicting nucleolar localization sequences.')])), ('test', CommentedMap([('commands', ['clinod | grep -i \"^NucleOlar localization sequence Detector\"', 'echo -e \">Test\\\\nMHPAVFLSLPDLRCSLLLLVTWVFTPVTTEITSLDTENIDEILNNADVALVNFYADWCRF\\\\n\" > example.fasta && clinod -in=example.fasta -f=MEDIUM_TAB -nonols -clean_sequence'])])), ('extra', CommentedMap([('notes', 'Command line NoD (clinod) is Java program which is packaged with a custom\\nwrapper shell script. This shell wrapper is called \"clinod\" and is on $PATH\\nby default. By default \"-Xms512m -Xmx1g\" is set in the wrapper. If you want\\nto overwrite it you can specify these values directly after your binaries.\\nIf you have _JAVA_OPTIONS set globally this will take precedence.\\nFor example run it with \"clinod -Xms512m -Xmx1g ...\"\\n')]))])), 'clustalo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clustalo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clustalo'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'clustal-omega-<{ version }}-macosx'), ('url', 'http://www.clustal.org/omega/clustal-omega-<{ version }}-macosx'), ('md5', 'e8ec7292c258b8b1ef4f3780b3952fe8')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['clustalo --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.clustal.org/omega/'), ('license', 'GPLv2'), ('summary', 'Latest version of Clustal: a multiple sequence alignment program for DNA or proteins')]))])), 'clustalw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clustalw/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.clustal.org/clustal2/'), ('license', 'GNU Lesser GPL'), ('summary', 'ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.')])), ('package', CommentedMap([('name', 'clustalw'), ('version', 2.1)])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'clustalw-2.1.tar.gz'), ('url', 'http://www.clustal.org/download/current/clustalw-2.1.tar.gz'), ('md5', '144df8440a0ae083d5167616c8ceeb41')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['clustalw2 H 2>&1 | grep \"CLUSTAL 2.1\"', 'clustalw H 2>&1 | grep \"CLUSTAL 2.1\"'])]))])), 'cluster-picker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cluster-picker/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://hiv.bio.ed.ac.uk/software.html'), ('license', 'GPLv3'), ('summary', 'The Cluster Picker identifies clusters in newick-formatted trees containing thousands of sequences within a few minutes. Cut-offs for within cluster genetic distance and bootstrap support are selected by the user.')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('package', CommentedMap([('name', 'cluster-picker'), ('version', '1.2.3')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ClusterPicker_1.2.3.jar'), ('url', 'http://hiv.bio.ed.ac.uk/clusterPicker/ClusterPicker_1.2.3.jar'), ('md5', '95c19264a268f2522fccc520f52989b6')])), ('test', CommentedMap([('commands', ['which cluster-picker', 'echo -e \">seq1\\\\nATTA\\\\n>seq2\\\\nATTA\\\\n>seq3\\\\nATTA\\\\n>seq4\\\\nATTA\" > dummy.fasta', 'echo \"(((seq1:0.001,seq2:0.002):0.001,seq3:0.005):0.004,seq4:0.003):0.001;\" > dummy.nwk', 'echo -e \"y\\\\ny\\\\ny\\\\ny\\\\ny\\\\n\" | cluster-picker dummy.fasta dummy.nwk'])]))])), 'clustergrammer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clustergrammer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clustergrammer'), ('version', '1.13.5')])), ('source', CommentedMap([('fn', 'clustergrammer-1.13.5.tar.gz'), ('url', 'https://pypi.python.org/packages/f4/ad/c0d0fd6e1c917d51df418d084201ef87c9b17288284870a31ac3ecebd571/clustergrammer-1.13.5.tar.gz'), ('md5', 'd7b600b34c5674710f83efbeddbc50f7')])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scikit-learn', 'pandas']), ('run', ['libgcc', 'python', 'numpy', 'scikit-learn', 'pandas'])])), ('test', CommentedMap([('imports', ['clustergrammer'])])), ('about', CommentedMap([('home', 'https://github.com/MaayanLab/clustergrammer-py'), ('license', 'MIT license'), ('summary', 'A python module for the Clustergrammer visualization project'), ('license_family', 'MIT')]))])), 'cmappy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cmappy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cmappy'), ('version', '2.0.1')])), ('source', CommentedMap([('fn', 'cmapPy-2.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/f1/08/b803a94071566ff86dfd7debd86b96f85f741ebf4b7ffaf601d901005b5e/cmapPy-2.0.1.tar.gz#'), ('md5', '5c03d064678cd1fd46b9372d4db34fa2')])), ('build', CommentedMap([('skip', True), ('entry_points', ['gctx2gct=cmapPy.pandasGEXpress.gctx2gct:main', 'gct2gctx=cmapPy.pandasGEXpress.gct2gctx:main', 'concat_gctoo=cmapPy.pandasGEXpress.concat_gctoo:main', 'slice_gct=cmapPy.pandasGEXpress.slice_gct:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'setuptools', 'numpy >=1.11.2', 'pandas >=0.18', 'h5py >=2.6.0', 'requests >=2.13.0']), ('run', ['python', 'libgcc', 'numpy >=1.11.2', 'pandas >=0.18', 'h5py >=2.6.0', 'requests >=2.13.0'])])), ('test', CommentedMap([('imports', ['cmapPy', 'cmapPy.clue_api_client', 'cmapPy.pandasGEXpress', 'cmapPy.set_io']), ('commands', ['gctx2gct --help', 'gct2gctx --help', 'concat_gctoo --help'])])), ('about', CommentedMap([('home', 'https://github.com/cmap/cmapPy'), ('license', 'BSD 3-clause'), ('summary', 'Assorted tools for interacting with .gct, .gctx, .grp, and .gmt files as well as other Connectivity Map (Broad Institute) data/tools'), ('license_family', 'BSD')]))])), 'cmfinder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cmfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cmfinder'), ('version', '0.2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'CMfinder_0.2.tgz'), ('url', 'http://bio.cs.washington.edu/yzizhen/CMfinder/CMfinder_0.2.tgz'), ('md5', 'f4ff2c87b9cb29dc79e56ac30c9c30a5')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['blast', 'libgcc'])])), ('test', CommentedMap([('commands', ['cmfinder.pl -h'])])), ('about', CommentedMap([('home', 'http://bio.cs.washington.edu/yzizhen/CMfinder/'), ('license', 'GPL2'), ('summary', 'CMfinder - A Covariance Model Based RNA Motif Finding Algorithm')]))])), 'cmph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cmph/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://downloads.sourceforge.net/project/cmph/cmph/cmph-2.0.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['cmph -h'])])), ('about', CommentedMap([('home', 'http://cmph.sourceforge.net/'), ('summary', 'CMPH - C Minimal Perfect Hashing Library')]))])), 'cmv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cmv/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/eggzilla/cmv/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['stack >=1.5.1', 'gmp >=5.0.1,<7', 'cairo', 'pango', 'zlib', 'perl', 'xorg-libxrender', 'xorg-libsm', 'xorg-libxext']), ('run', ['gmp >=5.0.1,<7', 'libgcc', 'cairo', 'pango', 'zlib', 'xorg-libxrender', 'xorg-libsm', 'xorg-libxext'])])), ('test', CommentedMap([('commands', ['CMV --help'])])), ('about', CommentedMap([('home', 'https://github.com/eggzilla/cmv'), ('license', 'GPL-3'), ('summary', 'A collection of tools for the visualisation of Hidden Markov Models (HMMV) and RNA-family models (CMV).')]))])), 'cnvkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cnvkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cnvkit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cnvkit-4c94834.tar.gz'), ('url', 'https://github.com/etal/cnvkit/archive/4c94834.tar.gz'), ('md5', 'ee3e12b06673a3f64a019c05e35c4993')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'atlas', 'numpy >=1.9', 'scipy >=0.9', 'pandas >=0.18.1', 'matplotlib >=1.1', 'biopython >=1.62', 'reportlab >=3.0', 'pysam >=0.9.1.4', 'r-pscbs', 'r-cghflasso', 'pyfaidx >=0.4.7', 'future >=0.15.2', 'futures >=3.0'])])), ('test', CommentedMap([('imports', ['cnvlib', 'skgenome']), ('commands', ['cnvkit.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/etal/cnvkit'), ('license', 'Apache License 2.0'), ('summary', 'Copy number variant detection from targeted DNA sequencing')]))])), 'cobra': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cobra/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'cobra'), ('version', '0.4.0')])), ('source', CommentedMap([('git_url', 'https://github.com/opencobra/cobrapy.git'), ('git_tag', 'de484b5a044da58496b4733f9d7c0302a55f4285')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'cython', 'gcc', 'glpk']), ('run', ['python', 'six', 'numpy', 'scipy', 'lxml', 'python-libsbml', 'glpk', 'libgcc'])])), ('test', CommentedMap([('imports', ['cobra', 'cobra.core', 'cobra.design', 'cobra.flux_analysis', 'cobra.io', 'cobra.manipulation', 'cobra.solvers', 'cobra.test', 'cobra.topology']), ('requires', ['jsonschema'])])), ('about', CommentedMap([('home', 'https://opencobra.github.io/cobrapy'), ('license', 'GNU Lesser General Public License v2 or later (LGPLv2+) or GNU General Public License v2 or later (GPLv2+)'), ('summary', 'COBRApy is a package for constraint-based modeling of biological networks')]))])), 'cofold': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cofold/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cofold'), ('version', '2.0.4')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/jujubix/cofold/archive/11815003a775196f4d1819491d5e0bc90bd5047e.tar.gz'), ('patches', ['gnu.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CoFold --help'])])), ('about', CommentedMap([('home', 'http://www.e-rna.org/cofold/'), ('license', 'MIT-like'), ('summary', 'An RNA secondary structure prediction method that takes co-transcriptional folding into account.')]))])), 'cogent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cogent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cogent'), ('version', '1.5.3')])), ('source', CommentedMap([('fn', 'cogent-1.5.3.tgz'), ('url', 'https://pypi.python.org/packages/source/c/cogent/cogent-1.5.3.tgz'), ('md5', 'ecdfa0bc89e0c04c9d4d39cac9984498')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k or osx]')])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.3', 'zlib', 'matplotlib >=1.1.0', 'mpi4py >=1.0', 'mysql-python >=1.2.2', 'sqlalchemy >=0.5']), ('run', ['python', 'numpy >=1.3', 'zlib', 'matplotlib >=1.1.0', 'mpi4py >=1.0', 'mysql-python >=1.2.2', 'sqlalchemy >=0.5'])])), ('test', CommentedMap([('imports', ['cogent', 'cogent.align', 'cogent.align.weights', 'cogent.app', 'cogent.cluster', 'cogent.core', 'cogent.data', 'cogent.evolve', 'cogent.format', 'cogent.maths', 'cogent.maths.matrix', 'cogent.maths.spatial', 'cogent.maths.stats', 'cogent.maths.stats.cai', 'cogent.maths.unifrac', 'cogent.motif', 'cogent.parse', 'cogent.phylo', 'cogent.recalculation', 'cogent.seqsim', 'cogent.struct', 'cogent.util', 'cogent.db.ensembl'])])), ('about', CommentedMap([('home', 'http://www.pycogent.org'), ('license', 'GNU General Public License (GPL)'), ('summary', 'COmparative GENomics Toolkit')]))])), 'collectl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/collectl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'collectl'), ('version', '4.0.4')])), ('source', CommentedMap([('url', 'http://downloads.sourceforge.net/project/collectl/collectl/collectl-4.0.4/collectl-4.0.4.src.tar.gz'), ('sha256', '03fc26ece3e9db6e410b708ada45fdec8c566ad82e67cb933c6ff8c005058ecf'), ('patches', ['usrenvbin.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'zlib']), ('run', ['perl-threaded', 'zlib'])])), ('test', CommentedMap([('commands', ['collectl --help'])])), ('about', CommentedMap([('home', 'http://collectl.sourceforge.net/'), ('license', 'Artistic License'), ('summary', 'collectl monitoring tool')]))])), 'colorbrewer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/colorbrewer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'colorbrewer'), ('version', '0.1.1')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/source/c/colorbrewer/colorbrewer-0.1.1.tar.gz'), ('fn', 'colorbrewer-0.1.1.tar.gz'), ('md5', 'b271dd4e93a2b2498c89760bca025412')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['colorbrewer'])])), ('about', CommentedMap([('home', 'http://noble.gs.washington.edu/~mmh1/software/colorbrewer/'), ('license', 'MIT'), ('summary', 'An easy way to get access to ColorBrewer schemes from within a Python program')]))])), 'combined-pvalues': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/combined-pvalues/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'combined-pvalues'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/brentp/combined-pvalues/archive/v<{ version }}.tar.gz'), ('md5', '807d3adb4bc8d61a0e7dbe7248b202cd')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ez_setup', 'interlap']), ('run', ['python', 'numpy', 'scipy', 'toolshed', 'interlap'])])), ('test', CommentedMap([('imports', ['cpv']), ('command', ['comb-p'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/combined-pvalues'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A library to combine, analyze, group and correct p-values in BED files.\\nUnique tools involve correction for spatial autocorrelation.\\nThis is useful for ChIP-Seq probes and Tiling arrays, or any data with spatial correlation.\\n')]))])), 'commet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/commet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'commet'), ('version', '24.7.14')])), ('source', CommentedMap([('fn', '6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('url', 'https://github.com/pierrepeterlongo/commet/archive/6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('md5', '20f906448270e2d329f32c1071344a9d'), ('patches', ['commet.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'r-base']), ('run', ['libgcc', 'r-base', 'r-gplots', 'libgfortran', 'python'])])), ('test', CommentedMap([('commands', ['Commet.py -h'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/commet/'), ('license', 'GNU Affero General Public License'), ('summary', 'Comparing and combining multiple metagenomic datasets')]))])), 'concoct': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/concoct/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'concoct'), ('version', '0.4.0')])), ('source', CommentedMap([('fn', 'concoct-0.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/4a/ca/66f48bf7480cc06a1b017a9713b5417ba30a24a5300db1f52fb8f4faeb7e/concoct-0.4.0.tar.gz'), ('md5', '0e783d0e6dabbb7329ff07c11bfd7299'), ('patches', ['concoct.patch'])])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython >=0.19.1', 'numpy >=1.7.1', 'scipy >=0.12.0', 'pandas >=0.11.0', 'biopython >=1.62b', 'scikit-learn >=0.13.1', 'nose >=1.3.0', 'sphinx_rtd_theme >=0.1.6', 'sphinx >=1.2.2', 'gsl <{ CONDA_GSL }}*']), ('run', ['python', 'setuptools', 'cython >=0.19.1', 'numpy >=1.7.1', 'scipy >=0.12.0', 'pandas >=0.11.0', 'biopython >=1.62b', 'scikit-learn >=0.13.1', 'nose >=1.3.0', 'sphinx_rtd_theme >=0.1.6', 'sphinx >=1.2.2', 'gsl <{ CONDA_GSL }}*'])])), ('test', CommentedMap([('imports', ['concoct'])])), ('about', CommentedMap([('home', 'https://github.com/BinPro/CONCOCT'), ('license', 'FreeBSD'), ('summary', 'Clustering cONtigs with COverage and ComposiTion'), ('license_family', 'BSD')]))])), 'connor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/connor/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/umich-brcf-bioinf/Connor/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('script', 'python setup.py install --single-version-externally-managed --record record.txt')])), ('requirements', CommentedMap([('build', ['python', 'cython', 'sortedcontainers', 'pysam >0.8.3,<=0.12', 'setuptools']), ('run', ['python', 'cython', 'sortedcontainers', 'pysam >0.8.3,<=0.12'])])), ('test', CommentedMap([('commands', ['connor --help'])])), ('about', CommentedMap([('home', 'https://github.com/umich-brcf-bioinf/Connor'), ('license', 'Apache-2.0'), ('license_family', 'Apache'), ('license_file', 'LICENSE'), ('summary', 'A command-line tool to deduplicate bam files based on custom, inline barcoding.')])), ('extra', CommentedMap([('recipe-maintainers', ['sebastian-luna-valero', 'cgates'])]))])), 'control-freec': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/control-freec/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'control-freec'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/BoevaLab/FREEC/archive/v<{ version }}.tar.gz'), ('fn', 'control-freec-<{ version }}.tar.gz'), ('md5', 'fb9e4f3c78343689c38fecae14bc13fc'), ('patches', ['makeGraph.R.patch', 'assess_significance.R.patch', 'makeGraph_Chromosome.R.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'r-base', 'bioconductor-rtracklayer'])])), ('test', CommentedMap([('commands', ['freec', 'freec2bed.pl', 'freec2circos.pl', 'which get_fasta_lengths.pl', 'which makeGraph.R', 'which makeGraph_Chromosome.R', 'vcf2snpFreec.pl', 'which assess_significance.R'])])), ('about', CommentedMap([('home', 'https://github.com/BoevaLab/FREEC'), ('license', 'GPL (>=2)'), ('summary', 'Copy number and genotype annotation from whole genome and whole exome \\nsequencing data.\\n')])), ('extra', CommentedMap([('notes', 'The tool will be available as `freec` in the command line.\\nSee the homepage for example config files. Auxiliary scripts\\nlike e.g. freec2bed.pl and freec2circos.pl (see homepage) are available in the \\ncommand line as well.\\n')]))])), 'cooler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cooler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cooler'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cooler-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/d7/b7/d39db1fd12d495ec3293b5e8694a49973f76a2489d1c0274d82de4a27c84/cooler-<{ version }}.tar.gz'), ('md5', '266262d27e2ed371e2554ca2d6a93e4e')])), ('build', CommentedMap([('number', 0), ('entry_points', ['cooler = cooler.cli:cli']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['zlib', 'hdf5', 'python', 'six', 'multiprocess', 'numpy', 'scipy', 'pandas', 'h5py', 'dask', 'click', 'biopython', 'pysam >0.8', 'pyfaidx', 'pairix'])])), ('test', CommentedMap([('imports', ['cooler', 'cooler.cli', 'cooler.contrib']), ('commands', ['LANG=C.UTF-8 cooler --help'])])), ('about', CommentedMap([('home', 'https://github.com/mirnylab/cooler'), ('license', 'BSD'), ('summary', 'Sparse binary format for genomic interaction matrices')]))])), 'coreutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/coreutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'coreutils'), ('version', 8.25)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/coreutils/coreutils_8.25_src_all.tar.bz2'), ('fn', 'coreutils_8.25_src_all.tar.bz2'), ('sha256', '031a9842298864ed0d7ae87e008759465c8b4716ed4bf97d723fe8ced5145e94')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm'])])), ('test', CommentedMap([('commands', ['wc --version', 'cat --version'])])), ('about', CommentedMap([('home', 'http://www.gnu.org/software/coreutils/'), ('license', 'GPL'), ('summary', 'The GNU Core Utilities are the basic file, shell and text manipulation utilities of the GNU operating system. These are the core utilities which are expected to exist on every operating system.')]))])), 'corset': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/corset/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Oshlack/Corset/archive/version-1.06.tar.gz'), ('md5', 'decc3bc6b499d6d7ac448ff3683e07aa')])), ('requirements', CommentedMap([('build', ['gcc', 'samtools 0.1.19']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['corset 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/Oshlack/Corset'), ('license', 'GPLv3'), ('summary', 'Software for clustering de novo assembled transcripts and counting overlapping reads.')]))])), 'cosi2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cosi2/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'cosi2'), ('version', '2.02')])), ('source', CommentedMap([('fn', 'cosi2-2.02.tar.gz'), ('url', 'https://github.com/broadinstitute/cosi2/archive/v2.02.tar.gz'), ('md5', '355300429e4b93afd7a915f5e7ecda18')])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf', 'automake', 'libtool', 'pkg-config', 'python']), ('run', ['libgcc', 'python'])])), ('test', CommentedMap([('commands', ['coalescent -h 2>&1 | grep \"Specifying the model\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.broadinstitute.org/mpg/cosi2/'), ('license', 'GPLv3'), ('summary', 'cosi2 is an efficient coalescent simulator with support for selection, population structure, variable recombination rates, and gene conversion. It supports exact and approximate simulation modes.')]))])), 'cpat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cpat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cpat'), ('version', '1.2.2')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'CPAT-1.2.2.tar.gz'), ('url', 'http://sourceforge.net/projects/rna-cpat/files/CPAT-1.2.2.tar.gz'), ('md5', 'fc296ec5fbd8b8062fa8353458b89c62'), ('patches', ['setup.py.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bx-python', 'cython', 'pysam', 'nose', 'numpy']), ('run', ['python', 'setuptools', 'bx-python', 'pysam'])])), ('test', CommentedMap([('commands', ['cpat.py -h', 'make_hexamer_tab.py -h', 'make_logitModel.py -h'])])), ('about', CommentedMap([('home', 'http://dldcc-web.brc.bcm.edu/lilab/liguow/CGI/cpat/_build/html/index.html'), ('license', 'GNU'), ('summary', 'Coding Potential Assessment Tool')]))])), 'cramtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cramtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cramtools'), ('version', '3.0.b127')])), ('source', CommentedMap([('fn', 'cramtools-3.0.jar'), ('url', 'https://github.com/enasequence/cramtools/raw/9a5977ff59bbc6319bd4b0fb06ccea414492d064/cramtools-3.0.jar'), ('md5', 'be01d4ad46549d48ec6cb77e48f868dd')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['cramtools -h'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/ena/software/cram-toolkit'), ('license', 'Apache v2.0'), ('summary', 'A set of Java tools and APIs for efficient compression of sequence read data')]))])), 'crb-blast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/crb-blast/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'crb-blast'), ('version', '0.6.6')])), ('build', CommentedMap([('number', 0), ('skip', 'True [osx]')])), ('requirements', CommentedMap([('build', ['ruby']), ('run', ['ruby', 'blast'])])), ('test', CommentedMap([('commands', ['crb-blast --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/cboursnell/crb-blast'), ('license', 'MIT'), ('summary', 'Conditional Reciprocal Best BLAST - high confidence ortholog assignment.')]))])), 'crimson': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/crimson/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'crimson'), ('version', '0.3.0')])), ('source', CommentedMap([('fn', 'Crimson-0.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/bf/8e/cdddc397f92fc1199195f01446bd3407dcb6f6ab8983f32a5e8798348809/Crimson-0.3.0.tar.gz'), ('md5', '2087a80794c3d579058bbe4a93a6d39a')])), ('build', CommentedMap([('number', 0), ('entry_points', ['crimson=crimson.main:cli']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click >=6.6', 'pyyaml >=3.11', 'wheel >=0.23.0']), ('run', ['python', 'click >=6.6', 'pyyaml >=3.11'])])), ('test', CommentedMap([('imports', ['crimson']), ('commands', ['LANG=C.UTF-8 crimson --help'])])), ('about', CommentedMap([('home', 'https://github.com/bow/crimson'), ('license', 'BSD License'), ('summary', 'Bioinformatics tool outputs converter to JSON or YAML.'), ('license_family', 'BSD')]))])), 'crisper_recognition_tool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/crisper_recognition_tool/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.room220.com/crt/'), ('license', 'Public domain software'), ('summary', 'A tool for automatic detection of clustered regularly interspaced palindromic repeats (CRISPR).')])), ('package', CommentedMap([('name', 'crisper_recognition_tool'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'CRT1.2-CLI.jar.zip'), ('url', 'http://www.room220.com/crt/CRT1.2-CLI.jar.zip'), ('md5', '1300514e388821b885c2d5fa6d19f137')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['crt -help'])]))])), 'crispresso': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/crispresso/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'crispresso'), ('version', '1.0.8')])), ('source', CommentedMap([('fn', 'CRISPResso-1.0.8.zip'), ('url', 'https://github.com/lucapinello/CRISPResso/archive/d63a974c78d0f34d3980e0ea8980a866c953916f.zip'), ('md5', '718ace1c2cc69262a0ccb67f52a433d0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'mock', 'nose', 'numpy >=1.10.4', 'pandas >=0.16', 'matplotlib >=1.3.1', 'biopython >=1.6.5', 'argparse', 'trimmomatic', 'flash', 'emboss', 'openjdk', 'gcc', 'samtools', 'bowtie2', 'seaborn']), ('run', ['python', 'mock', 'nose', 'numpy >=1.10.4', 'pandas >=0.16', 'matplotlib >=1.3.1', 'biopython >=1.6.5', 'argparse', 'trimmomatic', 'flash', 'emboss', 'openjdk >=8', 'libgcc', 'samtools', 'bowtie2', 'seaborn'])])), ('test', CommentedMap([('commands', ['CRISPResso -h'])])), ('about', CommentedMap([('home', 'https://github.com/lucapinello/CRISPResso'), ('license', 'GPLv3'), ('summary', 'A software pipeline for the analysis of targeted CRISPR-Cas9 sequencing data')]))])), 'cromwell': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cromwell/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/broadinstitute/cromwell'), ('license', 'BSD'), ('summary', 'A Workflow Management System geared towards scientific workflows described in WDL')])), ('package', CommentedMap([('name', 'cromwell'), ('version', 0.26)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'cromwell-0.26.zip'), ('url', 'https://github.com/broadinstitute/cromwell/archive/26.zip'), ('md5', 'c1a249c71641255a90271828b31a6c80')])), ('requirements', CommentedMap([('build', ['openjdk >=8', 'sbt', 'scala']), ('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', ['cromwell --help 2>&1 | grep \"Given a WDL file\"'])]))])), 'crossmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/crossmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'crossmap'), ('version', '0.2.5')])), ('source', CommentedMap([('url', 'http://sourceforge.net/projects/crossmap/files/CrossMap-0.2.5.tar.gz'), ('fn', 'CrossMap-0.2.5.tar.gz'), ('md5', '546710d26e299cf039497af1ec6c80dd')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.17', 'numpy', 'nose', 'pysam >=0.6', 'bx-python']), ('run', ['python', 'cython >=0.17', 'numpy', 'nose', 'pysam >=0.6', 'bx-python', 'ucsc-wigtobigwig'])])), ('test', CommentedMap([('commands', ['CrossMap.py bed', 'CrossMap.py vcf'])])), ('about', CommentedMap([('home', 'http://crossmap.sourceforge.net'), ('license', 'GPL'), ('summary', 'Convert genomic coordiates between assemblies')]))])), 'csb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/csb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'csb'), ('version', '1.2.5')])), ('source', CommentedMap([('fn', 'csb-1.2.5.zip'), ('url', 'https://pypi.python.org/packages/0a/85/730b460d84c816e66ab98727cbba960ca4c1ea9cd86cf2a4504013620020/csb-1.2.5.zip'), ('md5', '06a81ebee65994743ea751819a71620a')])), ('build', CommentedMap([('entry_points', ['csb-test = csb.test.app:main', 'csb-bfit = csb.apps.bfit:main', 'csb-bfite = csb.apps.bfite:main', 'csb-csfrag = csb.apps.csfrag:main', 'csb-hhfrag = csb.apps.hhfrag:main', 'csb-buildhmm = csb.apps.buildhmm:main', 'csb-hhsearch = csb.apps.hhsearch:main', 'csb-precision = csb.apps.precision:main', 'csb-promix = csb.apps.promix:main', 'csb-embd = csb.apps.embd:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'matplotlib']), ('run', ['python', 'numpy', 'scipy', 'matplotlib', 'libgcc'])])), ('test', CommentedMap([('imports', ['csb', 'csb.apps', 'csb.bio', 'csb.bio.fragments', 'csb.bio.hmm', 'csb.bio.io', 'csb.bio.nmr', 'csb.bio.sequence', 'csb.bio.structure', 'csb.bio.utils', 'csb.core', 'csb.io', 'csb.numeric', 'csb.statistics', 'csb.statistics.pdf', 'csb.statistics.samplers', 'csb.statistics.samplers.mc', 'csb.statmech', 'csb.test', 'csb.test.cases', 'csb.test.cases.bio', 'csb.test.cases.bio.fragments', 'csb.test.cases.bio.hmm', 'csb.test.cases.bio.io', 'csb.test.cases.bio.io.clans', 'csb.test.cases.bio.io.cs', 'csb.test.cases.bio.io.fasta', 'csb.test.cases.bio.io.hhpred', 'csb.test.cases.bio.io.mrc', 'csb.test.cases.bio.io.noe', 'csb.test.cases.bio.io.procheck', 'csb.test.cases.bio.io.whatif', 'csb.test.cases.bio.io.wwpdb', 'csb.test.cases.bio.nmr', 'csb.test.cases.bio.sequence', 'csb.test.cases.bio.sequence.alignment', 'csb.test.cases.bio.structure', 'csb.test.cases.bio.utils', 'csb.test.cases.core', 'csb.test.cases.io', 'csb.test.cases.numeric', 'csb.test.cases.statistics', 'csb.test.cases.statistics.pdf', 'csb.test.cases.statistics.samplers', 'csb.test.cases.statmech']), ('commands', ['csb-test --help', 'csb-bfit --help', 'csb-bfite --help', 'csb-csfrag --help', 'csb-hhfrag --help', 'csb-buildhmm --help', 'csb-hhsearch --help', 'csb-precision --help', 'csb-promix --help', 'csb-embd --help'])])), ('about', CommentedMap([('home', 'http://github.com/csb-toolbox'), ('license', 'MIT License'), ('summary', 'Computational Structural Biology Toolbox'), ('license_family', 'MIT')]))])), 'csvtk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/csvtk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'csvtk'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'https://github.com/shenwei356/csvtk'), ('summary', 'A cross-platform, efficient, practical and pretty CSV/TSV toolkit'), ('license', 'MIT')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'csvtk_v<{ version }}_linux64.tar.gz'), ('url', 'https://github.com/shenwei356/csvtk/releases/download/v<{ version }}/csvtk_linux_amd64.tar.gz'), ('md5', 'd51711263d759a5847a7f1960f5b8b38')])), ('test', CommentedMap([('commands', ['csvtk version --help'])]))])), 'cufflinks': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cufflinks/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cole-trapnell-lab.github.io/cufflinks/'), ('license', 'Boost Software License'), ('summary', 'Transcriptome assembly and differential expression analysis for RNA-Seq.')])), ('package', CommentedMap([('name', 'cufflinks'), ('version', '2.2.1')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['(cuffmerge --help 2>&1) | grep Options > /dev/null'])])), ('source', CommentedMap([('fn', 'cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('url', 'http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('patches', ['cuffmerge.python3.patch'])])), ('build', CommentedMap([('number', 1), ('skip', False)]))])), 'curesim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/curesim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'curesim'), ('version', '1.3')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'CuReSim1.3.zip'), ('url', 'http://www.pegase-biosciences.com/wp-content/uploads/2015/08/CuReSim1.3.zip'), ('sha256', '617c2ae560b9d93909a17ba1d88a8bc59c9cbe51e333c77fc02a6bce6eab24df')])), ('requirements', CommentedMap([('run', ['java-jdk >=8', 'coreutils'])])), ('test', CommentedMap([('commands', ['printf \">a\\\\nAAAAA\" > ref.fa && curesim -f ref.fa'])])), ('about', CommentedMap([('home', 'http://www.pegase-biosciences.com/curesim-a-customized-read-simulator/'), ('license', 'unknown'), ('summary', 'CuReSim (Customized Read Simulator) is a customized tool which generates synthetic New-Generation Sequencing reads, supporting read simulation for major letter-base sequencing platforms..')]))])), 'customprodb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/customprodb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-customprodb'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'customProDB_1.14.0.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/customProDB_1.14.0.tar.gz'), ('sha256', '075e88ebb6e5b271178b108e144e97b9f957708c23ddb4a1ddef5b102f1cb0c3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.26.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.10.2', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.13.44', 'r >=3.0.1', 'r-plyr', 'r-rcurl', 'r-rsqlite', 'r-stringr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.26.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.10.2', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.13.44', 'r >=3.0.1', 'r-plyr', 'r-rcurl', 'r-rsqlite', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'customProDB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/customProDB.html'), ('license', 'Artistic-2.0'), ('summary', 'Generate customized protein sequence database from RNA-Seq data for proteomics search')]))])), 'cutadapt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cutadapt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cutadapt'), ('version', '1.14')])), ('about', CommentedMap([('home', 'https://cutadapt.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'trim adapters from high-throughput sequencing reads')])), ('source', CommentedMap([('fn', 'cutadapt-1.14.tar.gz'), ('url', 'https://pypi.python.org/packages/16/e3/06b45eea35359833e7c6fac824b604f1551c2fc7ba0f2bd318d8dd883eb9/cutadapt-1.14.tar.gz'), ('md5', '57403c3794aa940699df3032a5e9b208')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'xopen >=0.1.1']), ('run', ['python', 'xopen >=0.1.1'])])), ('test', CommentedMap([('imports', ['cutadapt', 'cutadapt.scripts']), ('commands', ['cutadapt --version'])]))])), 'cwl2wdl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cwl2wdl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cwl2wdl'), ('version', '0.1dev44')])), ('source', CommentedMap([('fn', 'cwl2wdl-c242b63.tar.gz'), ('url', 'https://github.com/chapmanb/cwl2wdl/archive/c242b63.tar.gz'), ('md5', 'de4c77b6b6c035ab8dc388c405541de0')])), ('build', CommentedMap([('skip', False), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyyaml']), ('run', ['python', 'pyyaml'])])), ('test', CommentedMap([('commands', ['cwl2wdl --help'])])), ('about', CommentedMap([('home', 'https://github.com/adamstruck/cwl2wdl'), ('license', 'MIT'), ('summary', \"Proof of concept converter from Common Workflow Language (CWL) to the Broad Institute's Workflow Definition Language (WDL).\")]))])), 'cwltest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cwltest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cwltest'), ('version', '1.0.20170214185319')])), ('source', CommentedMap([('fn', 'cwltest-1.0.20170214185319.tar.gz'), ('url', 'https://pypi.python.org/packages/07/b0/c7b2c65152471274e496d1b99635955ba3d51ff389cfca06ec11127412ee/cwltest-1.0.20170214185319.tar.gz'), ('md5', '0f4cd403e1cc08a450437c4ab5c5218e')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'schema-salad', 'typing', 'junit-xml', 'futures']), ('run', ['python', 'schema-salad', 'typing', 'junit-xml', 'futures'])])), ('test', CommentedMap([('imports', ['cwltest']), ('commands', ['cwltest --help'])])), ('about', CommentedMap([('home', 'https://github.com/common-workflow-language/cwltest'), ('license', 'Apache License 2.0'), ('summary', 'Framework for testing CWL tools and workflows')]))])), 'cwltool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cwltool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cwltool'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cwltool-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/c/cwltool/cwltool-<{ version }}.tar.gz'), ('md5', 'c6baabec59b98fa00671ee100ce1e592')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'shellescape', 'cwltest', 'schema-salad >=2.6,<3.0', 'rdflib >=4.2.2', 'rdflib-jsonld >=0.3.0', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'typing >=3.5.3,<3.6']), ('run', ['python', 'requests', 'shellescape', 'cwltest', 'schema-salad >=2.6,<3.0', 'rdflib >=4.2.2', 'rdflib-jsonld >=0.3.0', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'typing >=3.5.3,<3.6'])])), ('test', CommentedMap([('imports', ['cwltool']), ('commands', ['cwltool --help'])])), ('about', CommentedMap([('home', 'https://github.com/common-workflow-language/cwltool'), ('license', 'Apache License 2.0'), ('summary', 'Common Workflow Language reference implementation')]))])), 'cyordereddict': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cyordereddict/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cyordereddict'), ('version', '0.2.2')])), ('source', CommentedMap([('fn', 'cyordereddict-0.2.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/c/cyordereddict/cyordereddict-0.2.2.tar.gz'), ('md5', '6279eb0bf9819f0293ad5315b2d484d0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'cython']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['cyordereddict', 'cyordereddict.benchmark'])])), ('about', CommentedMap([('home', 'https://github.com/shoyer/cyordereddict'), ('license', 'BSD License'), ('summary', \"Cython implementation of Python's collections.OrderedDict\")]))])), 'cyvcf2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cyvcf2/0.7.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cyvcf2'), ('version', '0.7.2')])), ('source', CommentedMap([('url', 'https://github.com/brentp/cyvcf2/archive/v0.7.2.tar.gz'), ('git_tag', 'v0.7.2'), ('md5', 'bd42bb44534db93b52ee2e97836abb6c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'gcc', 'zlib', 'cython']), ('run', ['python', 'numpy', 'matplotlib', 'seaborn', 'pandas', 'libgcc', 'zlib'])])), ('test', CommentedMap([('imports', ['cyvcf2'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/cyvcf2'), ('summary', 'A cython wrapper around htslib built for fast parsing of Variant Call Format (VCF) files'), ('license', 'MIT')]))])), 'daisysuite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/daisysuite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'daisysuite'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'DaisySuite_1.2.1.tar.gz'), ('url', 'https://gitlab.com/eseiler/DaisySuite/repository/1.2.1/archive.tar.gz'), ('md5', '488c26bb319ee46f1eb9764791e9c56c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['snakemake >=3.9.0', 'sak >=0.4.6', 'mason >=2.0.7', 'gustaf >=1.0.8', 'yara >=0.9.6', 'bwa >=0.7.8', 'clever-toolkit >=2.0rc3', 'stellar >=1.4.9', 'samtools >=0.1.19', 'bedtools >=0.2.22', 'pandas >=0.18.1', 'pysam >=0.9.1.4', 'scipy', 'biopython'])])), ('test', CommentedMap([('commands', ['DaisySuite --no-hooks --configfile $PREFIX/opt/daisysuite/example/example.yaml --config outputdir=\"$PREFIX/opt/daisysuite/example/output\" -n'])])), ('about', CommentedMap([('home', 'https://gitlab.com/eseiler/DaisySuite'), ('license', 'GNU GPLv3'), ('summary', 'DaisySuite - mapping-based pipeline for horizontal gene transfer (HGT) detection using sequencing data')]))])), 'daligner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/daligner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'daligner'), ('version', '1.0p1')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'daligner_V1.0p1.tar.gz'), ('md5', 'e49ca399c153333b1ec4e69d8d6daea8'), ('url', 'https://github.com/thegenemyers/DALIGNER/archive/a9458dcb34eca61513c994f4217fb7f75915e25c.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['daligner 2>&1 | grep Usage', 'LAsort 2>&1 | grep Usage', 'LAmerge 2>&1 | grep Usage', 'LAshow 2>&1 | grep Usage', 'LAcat 2>&1 | grep Usage', 'LAsplit 2>&1 | grep Usage', 'LAcheck 2>&1 | grep Usage', 'HPC.daligner 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/thegenemyers/DALIGNER'), ('license', 'Custom'), ('license_file', 'README.md'), ('summary', 'DALIGNER: Find all significant local alignments between reads')]))])), 'damidseq_pipeline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/damidseq_pipeline/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/owenjm/damidseq_pipeline'), ('license', 'GPL-2.0'), ('summary', 'An automated pipeline for processing DamID sequencing datasets')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'damidseq_pipeline'), ('version', '1.4')])), ('source', CommentedMap([('fn', 'damidseq_pipeline-1.4.tar.gz'), ('url', 'https://github.com/owenjm/damidseq_pipeline/archive/1.4.tar.gz'), ('sha256', 'cf3a9608a5f5453f2d491bde3759eaf154909387c9b4a1f882be45b542acbe73')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['bowtie2 >=2.3.0', 'igvtools', 'perl', 'r-base', 'samtools <1.3.0'])])), ('test', CommentedMap([('commands', ['damidseq_pipeline --help 2>&1 || true', 'gatc.track.maker.pl --help 2>&1 || true', 'gff2tdf.pl --help 2>&1 || true'])]))])), 'dammit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dammit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dammit'), ('version', '0.3')])), ('source', CommentedMap([('url', 'https://github.com/camillescott/dammit/archive/v0.3.tar.gz'), ('fn', 'dammit-v0.3.tar.gz'), ('md5', 'e7d3c00ab88c81a3a27088b14ab5472f'), ('patches', ['busco.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'pandas', 'numexpr >=2.3.1', 'khmer >=2.0', 'sphinx >1.3.1', 'sphinx_rtd_theme >=0.1.9', 'nose ==1.3.4', 'nose-capturestderr ==1.0', 'ficus >=0.1', 'doit >=0.29.0', 'infernal', 'blast', 'crb-blast', 'hmmer', 'transdecoder', 'last', 'busco ==1.2']), ('run', ['python', 'numpy', 'pandas', 'numexpr >=2.3.1', 'khmer >=2.0', 'sphinx >1.3.1', 'sphinx_rtd_theme >=0.1.9', 'nose ==1.3.4', 'nose-capturestderr ==1.0', 'ficus >=0.1', 'doit >=0.29.0', 'infernal', 'blast', 'crb-blast', 'hmmer', 'transdecoder', 'last', 'busco ==1.2', 'parallel'])])), ('test', CommentedMap([('imports', None), ('commands', ['dammit -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.camillescott.org/dammit/'), ('license', 'BSD'), ('summary', 'simple de novo transcriptome annotation')]))])), 'dascrubber': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dascrubber/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dascrubber'), ('version', '0.0.1a1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'dascrubber_0.0.1a1.tar.gz'), ('md5', '8c3c244315daa3e25c1bf67d605c1f5c'), ('url', 'https://github.com/thegenemyers/DASCRUBBER/archive/00749143f092ccff4d9ffb4b35b3baddc856b063.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['DASqv  2>&1 | grep Usage > /dev/null', 'DAStrim  2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/thegenemyers/DASCRUBBER'), ('license', 'Custom'), ('license_file', 'LICENSE'), ('summary', 'Alignment-based Scrubbing pipeline')]))])), 'data_hacks': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/data_hacks/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'data_hacks'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'data_hacks-v0.3.1.tar.gz'), ('sha256', '188afca43dba6a8267d76f2ffd961a50ded17313a2b273c90e3c9561007a4efa'), ('url', 'https://github.com/bitly/data_hacks/archive/v0.3.1.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['histogram.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/bitly/data_hacks'), ('license', 'Apache 2.0'), ('summary', 'Command line utilities for data analysis.')]))])), 'dazz_db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dazz_db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dazz_db'), ('version', '1.0p1')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'dazz_db_V1.0p1.tar.gz'), ('md5', 'be7950d41f776b542d98ab2376d92a25'), ('url', 'https://github.com/thegenemyers/DAZZ_DB/archive/e31d4517ab9f4fad6571a351ba10f87db35b8bc8.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['fasta2DB 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/thegenemyers/DAZZ_DB'), ('license', 'Custom'), ('license_file', 'README.md'), ('summary', 'DAZZ_DB: The Dazzler Data Base')]))])), 'dbg2olc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dbg2olc/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('script', 'mkdir -p $PREFIX/bin; g++ -O3 -o DBG2OLC *.cpp; mv DBG2OLC $PREFIX/bin')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/yechengxi/DBG2OLC/archive/25baf29bac46ef588d58880fbd3814f3a917771c.tar.gz'), ('md5', '5ad2eb2bed6be30f53d753e555aab814')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib', 'sparseassembler', 'sparc', 'assemblyutility'])])), ('test', CommentedMap([('commands', ['DBG2OLC | grep \"Chengxi Ye cxy@umd.edu\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/yechengxi/DBG2OLC'), ('summary', 'Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies')]))])), 'dcmtk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dcmtk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dcmtk'), ('version', '<{ DCMTK_VERSION }}')])), ('source', CommentedMap([('fn', 'dcmtk-<{ DCMTK_VERSION }}_<{ DCMTK_VERSION_DATE }}.tar.gz'), ('url', 'http://dicom.offis.de/download/dcmtk/snapshot/old/dcmtk-<{ DCMTK_VERSION }}_<{ DCMTK_VERSION_DATE }}.tar.gz'), ('sha1', '9daa6b64767443a60d5cb8946905e4e7003d1658')])), ('build', CommentedMap([('number', 4)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc >=4.7 [linux]', 'openssl', 'libpng', 'libtiff', 'libxml2', 'zlib', 'jpeg']), ('run', ['openssl', 'libpng', 'libtiff', 'jpeg', 'libxml2', 'zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['dcmdata_tests', 'dcmftest', 'dcmiod_tests', 'dcmnet_tests', 'dcmrt_tests', 'dcmseg_tests', 'dcmsr_tests', 'drttest', 'ofstd_tests'])])), ('about', CommentedMap([('home', 'http://dicom.offis.de'), ('summary', 'DCMTK is a collection of libraries and applications implementing large parts the DICOM standard'), ('license', 'BSD')]))])), 'ddocent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ddocent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ddocent'), ('version', '2.2.20')])), ('source', CommentedMap([('fn', 'v2.2.20.zip'), ('sha256', '1bcc44673685e3c0ac9f44990dfd8409efef1ca3497b3478e919375ba322107b'), ('url', 'https://github.com/jpuritz/dDocent/archive/v2.2.20.zip')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'curl', 'cmake', 'freebayes', 'gawk', 'stacks', 'trimmomatic', 'mawk', 'bwa', 'samtools', 'vcftools ==0.1.14', 'rainbow', 'seqtk', 'cd-hit', 'bedtools ==2.26.0', 'vcflib', 'gnuplot', 'parallel', 'bamtools', 'openjdk', 'pear', 'unzip']), ('run', ['libgcc', 'curl', 'cmake', 'freebayes', 'gawk', 'stacks', 'trimmomatic', 'mawk', 'bwa', 'samtools', 'vcftools ==0.1.14', 'rainbow', 'seqtk', 'cd-hit', 'bedtools ==2.26.0', 'vcflib', 'gnuplot', 'parallel', 'bamtools', 'openjdk', 'pear', 'unzip'])])), ('test', CommentedMap([('commands', ['dDocent --help'])])), ('about', CommentedMap([('home', 'http://ddocent.com'), ('license', 'The MIT License (MIT)'), ('summary', ' dDocent is  an interactive bash wrapper to QC, assemble, map, and call SNPs from all types of RAD data')]))])), 'ddrage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ddrage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ddrage'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'ddrage-1.2.2.tar.gz'), ('url', 'https://pypi.python.org/packages/23/4f/b2de08852b6a58169556b34b757b0337dac80f17da32a8be15413c35ab5f/ddrage-1.2.2.tar.gz'), ('md5', '9bca00e2700d874d4a7479e4adc3d425')])), ('build', CommentedMap([('skip', True), ('entry_points', ['rage = ddrage.__main__:main', 'randomize_fastq = ddrage.tools.randomize_fastq:main', 'learn_qmodel = ddrage.tools.learn_qmodel:main', 'visualize_bbd = ddrage.tools.bbd_visualization:main_standalone', 'remove_annotation = ddrage.tools.remove_annotation:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'numba', 'matplotlib', 'scipy', 'pyyaml', 'libgcc']), ('run', ['python', 'numpy', 'numba', 'matplotlib', 'scipy', 'pyyaml', 'libgcc'])])), ('test', CommentedMap([('imports', ['ddrage', 'ddrage.barcode_handler', 'ddrage.tools']), ('commands', ['rage --help', 'randomize_fastq --help', 'learn_qmodel --help', 'remove_annotation --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/genomeinformatics/rage'), ('license', 'MIT License'), ('summary', 'Simulator for ddRADseq (double digest restriction site associdated DNA squencing) datasets. Generates reads (FASTQ format) that can be analyzed and validated using a ground truth file (YAML).'), ('license_family', 'MIT')]))])), 'deap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'deap'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'deap-1.0.2.post2.tar.gz'), ('url', 'https://pypi.python.org/packages/5b/d7/a49d3dd7aa8cbaf2b1ac8f4d6495824c886fea8b3dac4a73dc4df94cad76/deap-1.0.2.post2.tar.gz'), ('md5', 'ccf5ed7562e4d6236c9416e3b5a9d941')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy', 'matplotlib', 'scoop'])])), ('test', CommentedMap([('imports', ['deap', 'deap.benchmarks', 'deap.tests', 'deap.tools'])])), ('about', CommentedMap([('home', 'https://www.github.com/deap'), ('license', 'GNU Library or Lesser General Public License (LGPL)'), ('license_file', 'LICENSE.txt'), ('summary', 'Distributed Evolutionary Algorithms in Python')]))])), 'debwt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/debwt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'debwt'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('git_url', 'https://github.com/DixianZhu/deBWT.git'), ('git_rev', '972b5338b2ad027244bb60d185c5a88586affba6')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['deBWT --help 2>&1 | grep deBWT > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/DixianZhu/deBWT'), ('license', 'Unknown'), ('summary', 'A efficient method to construct BWT index of a given DNA sequence, especially\\nuseful for gigantic and high similar genome.\\nDeBWT has good scalability to construct BWT in parallel computing.\\nIt is well-suited to run on multiple core servers or clusters to\\nconstruct the BWT of large collections of genome sequences.\\n')]))])), 'deepmedic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deepmedic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'deepmedic'), ('version', '<{ DEEPMEDIC_VERSION }}')])), ('source', CommentedMap([('fn', 'v<{ DEEPMEDIC_VERSION}}.zip'), ('url', 'https://codeload.github.com/Kamnitsask/deepmedic/zip/v<{ DEEPMEDIC_VERSION}}'), ('md5', '7f2cb90f90f1da7ea86c7c41c5f76cab'), ('patches', ['setup.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['theano', 'nose', 'nibabel', 'pp', 'six', 'scipy', 'numpy']), ('run', ['theano', 'nibabel', 'pp', 'six', 'scipy', 'numpy'])])), ('test', CommentedMap([('commands', ['deepMedicRun -h'])])), ('about', CommentedMap([('home', 'https://github.com/Kamnitsask/deepmedic'), ('license', 'BSD'), ('summary', 'Efficient Multi-Scale 3D Convolutional Neural Network for Brain Lesion Segmentation.')]))])), 'deeptools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deeptools/2.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/2e/21/a6836deaee7a48025182bf8588acb58716b7b216342479b6bf225f0b147b/deepTools-<{ version }}.tar.gz'), ('md5', '9ab5327fa198ed317fc20d2bea71196b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0', 'gcc']), ('run', ['python', 'pybigwig >=0.2.3', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))])), 'delly': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/delly/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'delly'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/dellytools/delly/archive/v<{ version }}.tar.gz'), ('sha256', '72298ef36be82fa0bd83c77c9c38d5bac48c9219595f1a206c26d6eeeff07c36'), ('patches', ['Makefile.patch'])])), ('build', CommentedMap([('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'bzip2', 'bcftools', 'boost <{CONDA_BOOST}}*', 'htslib']), ('run', ['libgcc', 'zlib', 'bzip2', 'bcftools', 'boost <{CONDA_BOOST}}*', 'htslib'])])), ('test', CommentedMap([('commands', ['delly --help'])])), ('about', CommentedMap([('home', 'https://github.com/dellytools/delly'), ('license', 'GPLv3'), ('summary', 'Structural variant discovery by integrated paired-end and split-read analysis')]))])), 'deltabs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deltabs/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/UCanCompBio/deltaBS/archive/8257207c960e8d694986d72e56af27a2a48d9918.tar.gz'), ('md5', 'ab65383cbdba9c9453d2af46ce1608ed')])), ('requirements', CommentedMap([('build', None), ('run', ['perl-threaded', 'perl-bioperl', 'perl-statistics-distributions', 'hmmer'])])), ('test', CommentedMap([('commands', [\"deltaBS.pl --help 2>&1 | grep 'Usage'\"])])), ('about', CommentedMap([('home', 'https://github.com/UCanCompBio/deltaBS/wiki'), ('license', 'GPL3'), ('summary', 'Quantifying the significance of genetic variation using probabilistic profile-based methods.')]))])), 'dendropy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dendropy/3.12.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dendropy'), ('version', '3.12.3')])), ('source', CommentedMap([('fn', 'DendroPy-3.12.3.tar.gz'), ('url', 'https://pypi.python.org/packages/3d/c0/e0a20d1cf4b268757842a03f36452095fccb8eae5ebd541548e29c6ca1e3/DendroPy-3.12.3.tar.gz'), ('md5', '13de66112a18e4eb283f4d4b18af7a65')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools']), ('run', ['python >=2.7,<3'])])), ('test', CommentedMap([('imports', ['dendropy', 'dendropy.coalescent', 'dendropy.continuous', 'dendropy.dataio', 'dendropy.dataobject', 'dendropy.interop', 'dendropy.mathlib', 'dendropy.popgensim', 'dendropy.popgenstat', 'dendropy.reconcile', 'dendropy.scm', 'dendropy.seqmodel', 'dendropy.seqsim', 'dendropy.test', 'dendropy.treecalc', 'dendropy.treemanip', 'dendropy.treesim', 'dendropy.treesplit', 'dendropy.treesum', 'dendropy.utility'])])), ('about', CommentedMap([('home', 'http://packages.python.org/DendroPy/'), ('license', 'BSD License'), ('summary', 'A Python library for phylogenetics and phylogenetic computing: reading, writing, simulation, processing and manipulation of phylogenetic trees (phylogenies) and characters.')]))])), 'denovogear': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/denovogear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'denovogear'), ('version', '1.1.1')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'denovogear_v1.1.1.tar.gz'), ('url', 'https://github.com/denovogear/denovogear/archive/v1.1.1.tar.gz'), ('md5', 'da30e46851c3a774653e57f98fe62e5f')])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'boost <{CONDA_BOOST}}*', 'eigen', 'zlib', 'htslib']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'eigen', 'zlib', 'htslib'])])), ('test', CommentedMap([('commands', ['dng help call 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/denovogear/denovogear'), ('license', 'GPL3'), ('license_file', 'COPYING.txt'), ('summary', 'A program to detect denovo-variants using next-generation sequencing data.')]))])), 'detonate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/detonate/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://deweylab.biostat.wisc.edu/detonate/'), ('license', 'GPL'), ('summary', 'DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation) consists of two component packages, RSEM-EVAL and REF-EVAL.')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'detonate'), ('version', '1.11')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'ncurses', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'ncurses', 'samtools 0.1.18', 'perl-threaded', 'boost <{CONDA_BOOST}}*'])])), ('source', CommentedMap([('fn', 'detonate-1.11.tar.gz.tar.gz'), ('url', 'http://deweylab.biostat.wisc.edu/detonate/detonate-1.11.tar.gz'), ('patches', ['ref-eval.sam.Makefile.patch', 'ref-eval.Makefile.patch', 'ref-eval.sam.misc.Makefile.patch', 'rsem-eval.sam.Makefile.patch', 'rsem-eval.Makefile.patch'])])), ('test', CommentedMap([('commands', ['ref-eval --help'])]))])), 'dextractor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dextractor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dextractor'), ('version', '1.0p1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'dextractor_V1.0p1.tar.gz'), ('md5', 'f9a79c200783017a1b4c99deeaa5ba2c'), ('url', 'https://github.com/thegenemyers/DEXTRACTOR/archive/e0e2374a50e680b788aadc494bb1d25e4c94f1a9.tar.gz'), ('patches', ['dextractor.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'hdf5']), ('run', ['libgcc', 'hdf5'])])), ('test', CommentedMap([('commands', ['dextract 2>&1 | grep Usage > /dev/null', 'dexta 2>&1 | grep Usage > /dev/null', 'undexta 2>&1 | grep Usage > /dev/null', 'dexqv 2>&1 | grep Usage > /dev/null', 'undexqv 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/thegenemyers/DEXTRACTOR'), ('license', 'Custom'), ('license_file', 'README'), ('summary', 'Bax File Decoder and Data Compressor')]))])), 'dfam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dfam/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'dfam.org'), ('license', 'GPL'), ('summary', 'The Dfam database is a collection of Repetitive DNA element sequence alignments, hidden Markov models (HMMs) and matches lists for complete Eukaryote genomes')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'dfam'), ('version', '2.0')])), ('requirements', CommentedMap([('run', ['perl-threaded', 'wget', 'hmmer', 'repeatmasker'])])), ('source', CommentedMap([('fn', 'dfamscan.pl'), ('url', 'http://dfam.org/web_download/Tools/dfamscan.pl')])), ('test', CommentedMap([('commands', ['dfamscan.pl --help 2>&1 |grep dfam_infile', 'download-dfam.py --help'])]))])), 'dia_umpire': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dia_umpire/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/Nesvilab/DIA-Umpire'), ('license', 'Apache License, Version 2.0'), ('summary', 'DIA-Umpire is an open source Java program for computational analysis of data independent acquisition (DIA) \\nmass spectrometry-based proteomics data. It enables untargeted peptide and protein identification and quantitation \\nusing DIA data, and also incorporates targeted extraction to reduce the number of cases of missing quantitation. \\n')])), ('package', CommentedMap([('name', 'dia_umpire'), ('version', '2.1.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'v2.1.2.zip'), ('url', 'https://github.com/Nesvilab/DIA-Umpire/releases/download/v2.1.2/v2.1.2.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['dia_umpire_se -Xms512m -Xmx1g', 'dia_umpire_quant -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'DIA-Umpire is a Java program.\\n')]))])), 'dinopy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dinopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dinopy'), ('version', '2.0.0')])), ('source', CommentedMap([('fn', 'dinopy-2.0.0.tar.gz'), ('url', 'https://pypi.python.org/packages/4b/f8/85977ab9cf64cbc4ba0b76287f81b9596951ef60b3c3e6aa2b306501b6ec/dinopy-2.0.0.tar.gz'), ('md5', 'c185d50fec4c086e7e787a51cb71fa1d')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9', 'gcc', 'llvm']), ('run', ['python', 'numpy >=1.9', 'libgcc'])])), ('test', CommentedMap([('imports', ['dinopy'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/HenningTimm/dinopy'), ('license', 'MIT License'), ('summary', 'DNA input and output library for Python and Cython. Includes reader and writer for FASTA and FASTQ files, support for samtools faidx files, and generators for solid and gapped q-grams (k-mers).')]))])), 'disco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/disco/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/abiswas-odu/Disco/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'openmpi >=1.8', 'zlib']), ('run', ['libgcc', 'openmpi >=1.8', 'zlib'])])), ('test', CommentedMap([('commands', ['runDisco.sh -h 2>&1 | grep \"runDisco\"', 'runDisco-MPI.sh -h  2>&1 | grep \"runDisco\"', 'runDisco-MPI-ALPS.sh -h 2>&1 | grep \"runDisco\"', 'runDisco-MPI-SLURM.sh -h 2>&1 | grep \"runDisco\"', 'runAssembly.sh -h 2>&1 | grep \"runAssembly\"', 'runAssembly-MPI.sh -h 2>&1 | grep \"runAssembly\"', 'runECC.sh -h 2>&1 | grep \"runECC\"'])])), ('about', CommentedMap([('home', 'http://disco.omicsbio.org/'), ('license', 'GPL-3.0'), ('summary', 'Multi-threaded Distributed Memory Overlap-Layout-Consensus (OLC) Metagenome Assembler')]))])), 'discosnp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discosnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'discosnp'), ('version', '2.2.10')])), ('source', CommentedMap([('fn', 'DiscoSNP.-v2.2.10-Source.tar.gz'), ('url', 'https://github.com/GATB/DiscoSnp/releases/download/v2.2.10/DiscoSNP.-v2.2.10-Source.tar.gz'), ('md5', 'a5590184534e387a21cd35630dc09529')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['run_discoSnp++.sh -h'])])), ('about', CommentedMap([('home', 'https://gatb.inria.fr/software/discosnp/'), ('license', 'GNU Affero General Public License v3.0'), ('summary', 'Designed for discovering all kinds of SNPs')]))])), 'discovar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discovar/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/software/discovar'), ('license', 'MIT'), ('summary', 'Suitable for variant calling with reference and de novo assembly of small genomes.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'discovar'), ('version', '52488')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'discovar-52488.tar.gz'), ('url', 'ftp://ftp.broadinstitute.org/pub/crd/Discovar/latest_source_code/discovar-52488.tar.gz')]))])), 'discovar-denovo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discovar-denovo/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/software/discovar/'), ('license', 'MIT'), ('summary', 'Suitable for de novo assembly of large and small genomes.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'discovardenovo'), ('version', '52488')])), ('requirements', CommentedMap([('build', ['gcc', 'jemalloc', 'zlib']), ('run', ['libgcc', 'jemalloc', 'zlib'])])), ('source', CommentedMap([('fn', 'discovardenovo-52488.tar.gz'), ('url', 'ftp://ftp.broadinstitute.org/pub/crd/DiscovarDeNovo/latest_source_code/discovardenovo-52488.tar.gz')]))])), 'disty': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/disty/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'disty'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'disty-<{ version }}.tar.gz'), ('url', 'https://github.com/c2-d2/disty/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib ==1.2.8', 'libgcc']), ('run', ['zlib ==1.2.8', 'libgcc'])])), ('test', CommentedMap([('commands', ['disty -v'])])), ('about', CommentedMap([('home', 'https://github.com/c2-d2/disty'), ('license', 'MIT'), ('summary', 'Disty McMatrixface - compute a distance matrix from a core genome alignment file.')]))])), 'dkfz-bias-filter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dkfz-bias-filter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dkfz-bias-filter'), ('version', '1.2.3a')])), ('source', CommentedMap([('git_url', 'https://github.com/chapmanb/DKFZBiasFilter.git'), ('git_tag', '4086221e36df7c04400907a47d3c8847bef79b95')])), ('build', CommentedMap([('number', 4), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'matplotlib', 'numpy', 'pysam', 'scipy'])])), ('test', CommentedMap([('commands', ['dkfzbiasfilter.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/eilslabs/DKFZBiasFilter'), ('license', 'GPLv3'), ('summary', 'The DKFZ bias filter flags SNVs that appear to be biased based on the variant read support')]))])), 'dlcpar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dlcpar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dlcpar'), ('version', '1.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'dlcpar-1.0.tar.gz'), ('url', 'https://www.cs.hmc.edu/~yjw/software/dlcpar/pub/sw/dlcpar-1.0.tar.gz'), ('sha256', '774319caba0f10d1230b8f85b8a147eda5871f9a316d7b3381b91c1bde97aa0a')])), ('requirements', CommentedMap([('build', ['python', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('commands', ['dlcpar --help > /dev/null', 'dlcpar_search --help > /dev/null', 'dlcpar_to_dlcoal --help > /dev/null', 'dlcoal_to_dlcpar --help > /dev/null', 'tree-events-dlc --help > /dev/null', 'tree-events-dlcpar --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/davidemms/OrthoFinder'), ('summary', 'Accurate inference of orthogroups, orthologues, gene trees and rooted species tree made easy!'), ('license', 'GPLv3')]))])), 'dnapi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dnapi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dnapi'), ('version', '1.1')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'dnapi_1.1.zip'), ('url', 'https://github.com/jnktsj/DNApi/archive/v1.1.zip'), ('md5', '8da9b9af6e3a9d89b4fa4c9f8fd38df0')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('command', 'qual_offset.py -h qual_trim.py -h to_fasta.py -h')])), ('about', CommentedMap([('home', 'https://github.com/jnktsj/DNApi'), ('license', 'MIT License'), ('summary', 'De novo adapter prediction (iterative) algorithm for small RNA sequencing data.')]))])), 'doit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/doit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'doit'), ('version', '0.29.0')])), ('source', CommentedMap([('fn', 'doit-0.29.0.tar.gz'), ('url', 'https://pypi.python.org/packages/9e/56/7884e2db9350f82c63097fc536781cfe68a02020b92724316d71f9e9491f/doit-0.29.0.tar.gz'), ('md5', '9968e1a669bad0a999212ab9f5c591a3')])), ('build', CommentedMap([('skip', True), ('entry_points', ['doit = doit.__main__:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'cloudpickle', 'pyinotify', 'configparser']), ('run', ['python', 'six', 'cloudpickle', 'pyinotify', 'configparser'])])), ('test', CommentedMap([('imports', ['doit']), ('commands', ['doit --help'])])), ('about', CommentedMap([('home', 'http://pydoit.org'), ('license', 'MIT License'), ('summary', 'doit - Automation Tool')]))])), 'domclust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/domclust/1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'domclust'), ('version', '1.0')])), ('about', CommentedMap([('home', 'http://mbgd.genome.ad.jp/domclust/'), ('license', 'GPL3'), ('summary', 'Effective tool for orthologous grouping in multiple genomes')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'DomClust_1.0_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/DomClust/DomClust_1.0_src_all.tar.gz'), ('sha256', 'a91cd437d2d19f67097a103bd2b49ad8524668b1d946209de65183d50173352a')])), ('requirements', CommentedMap([('build', ['gcc', 'perl']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['domclust -h'])]))])), 'dominate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dominate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dominate'), ('version', '2.1.16')])), ('source', CommentedMap([('fn', 'dominate-2.1.16.zip'), ('url', 'https://pypi.python.org/packages/source/d/dominate/dominate-2.1.16.zip'), ('md5', 'f11d2007f47a343c1323583c5b1360e9')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['dominate'])])), ('about', CommentedMap([('home', 'http://github.com/Knio/dominate/'), ('license', 'GNU Lesser General Public License v3 (LGPLv3)'), ('summary', 'Dominate is a Python library for creating and manipulating HTML documents using an elegant DOM API.')]))])), 'dr-disco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dr-disco/0.9.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/yhoogstrate/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy']), ('run', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['drdisco']), ('commands', ['LANG=C.UTF-8 dr-disco --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/dr-disco/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Dr. Disco: fusion gene and genomic breakpoint detection in random hexamer RNA-seq data')]))])), 'dreamtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dreamtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dreamtools'), ('version', '1.3.0')])), ('source', CommentedMap([('fn', 'dreamtools-1.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/d/dreamtools/dreamtools-1.3.0.tar.gz'), ('md5', 'aab79625f890fba24644627b1886e43c')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'easydev >=0.9.14', 'pandas', 'bioservices >=1.4.5', 'biokit', 'colormap', 'scipy', 'scikit-learn', 'numexpr', 'synapseclient', 'xlrd', 'tabulate', 'fitter']), ('run', ['python', 'easydev >=0.9.14', 'cython', 'pandas', 'bioservices >=1.4.5', 'biokit', 'colormap', 'scipy', 'scikit-learn', 'numexpr', 'synapseclient', 'xlrd', 'tabulate', 'fitter', 'libgcc'])])), ('test', CommentedMap([('imports', ['dreamtools']), ('commands', ['dreamtools --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', ['http://pypi.python.org/pypi/dreamtools']), ('license', 'BSD License'), ('summary', 'Scoring functions for the DREAM / SAGE challenges')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'drive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/drive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'drive'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'drive-v<{ version }}-osx'), ('url', 'https://github.com/odeke-em/drive/releases/download/v<{ version }}/drive_darwin')])), ('build', CommentedMap([('number', 0), ('skip', False), ('binary_relocation', False)])), ('requirements', None), ('test', CommentedMap([('commands', ['drive version', 'drive cp -h'])])), ('about', CommentedMap([('home', 'https://github.com/odeke-em/drive'), ('license', 'Apache v2.0'), ('summary', 'Google Drive client for the commandline')]))])), 'dsk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsk'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'dsk-v2.2.0-Source.tar.gz'), ('url', 'https://github.com/GATB/dsk/releases/download/v2.2.0/dsk-v2.2.0-Source.tar.gz'), ('md5', 'dbdfe583b45e78f965e6f78cf2c9628c'), ('patches', ['install_paths.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'llvm', 'zlib', 'cppunit']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['dsk -h'])])), ('about', CommentedMap([('home', 'https://github.com/GATB/dsk/'), ('license', 'AGPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'DSK is a k-mer counter for reads or genomes.')]))])), 'dsrc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsrc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsrc'), ('version', '2014.12.17')])), ('source', CommentedMap([('fn', 'v2.0.2.tar.gz'), ('url', 'https://github.com/lrog/dsrc/archive/v2.0.2.tar.gz')])), ('build', CommentedMap([('skip', True), ('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'boost <{CONDA_BOOST}}*'])])), ('tests', CommentedMap([('commands', ['dsrc -h'])])), ('about', CommentedMap([('home', 'https://github.com/lrog/dsrc'), ('summary', 'high-performance compression of sequencing reads stored in FASTQ format'), ('license', 'GNU GPL 2')]))])), 'dudes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dudes/0.06/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dudes'), ('version', '0.06')])), ('source', CommentedMap([('fn', 'dudes_v0_06.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/dudes/dudes_v0_06.tar.gz'), ('md5', 'dde2c8dbca04448cf21a0ed6b092b047')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy', 'pandas', 'bsddb'])])), ('test', CommentedMap([('commands', ['DUDes.py -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/dudes/'), ('license', 'The MIT License (MIT)'), ('summary', 'DUDes is a taxonomic profiler for NGS metagenomic samples')]))])), 'dunovo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dunovo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dunovo'), ('version', '0.8.1')])), ('source', CommentedMap([('fn', 'v0.8.1.tar.gz'), ('url', 'https://github.com/galaxyproject/dunovo/archive/v0.8.1.tar.gz'), ('sha256', 'f85fd35ef67c8f76af0d556a4babe9acf7b8abdd8a77232d4f4763cc7de60eed')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['make', 'gcc', 'python', 'wget']), ('run', ['libgcc', 'python', 'mafft 7.221', 'samtools 0.1.18', 'bowtie2 2.2.5', 'networkx', 'paste', 'gawk'])])), ('test', CommentedMap([('commands', ['correct.py -h > /dev/null', 'align_families.py -h > /dev/null', 'dunovo.py --version > /dev/null']), ('imports', ['networkx'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/dunovo'), ('license', 'BSD'), ('license_file', 'LICENSE.txt'), ('summary', 'Du Novo: A pipeline for processing duplex sequencing data.')]))])), 'dwgsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dwgsim/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/nh13/DWGSIM'), ('license', 'GNU General Public License v2 (GPLv2)'), ('license_family', 'GPL'), ('summary', 'Whole Genome Simulator for Next-Generation Sequencing')])), ('build', CommentedMap([('number', 3)])), ('package', CommentedMap([('name', 'dwgsim'), ('version', '1.1.11')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'ncurses']), ('run', ['libgcc', 'zlib', 'ncurses'])])), ('test', CommentedMap([('commands', ['printf \">a\\\\nAAAAA\" > ref.fa; dwgsim ref.fa pref'])])), ('source', CommentedMap([('fn', 'dwgsim-0.1.11.tar.gz'), ('url', 'https://github.com/nh13/DWGSIM/archive/dwgsim.0.1.11.tar.gz'), ('sha256', '49e4b558e313f4cd5755961f7f24ba48ad580c0324772d7080b59cb69ce0148b')]))])), 'dxpy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dxpy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dxpy'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'dxpy-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/d/dxpy/dxpy-<{ version }}.tar.gz'), ('md5', 'd14aabf5b0a644d6dbb5066ec66562fb')])), ('build', CommentedMap([('number', 0), ('skip', True), ('entry_points', ['dx-mappings-to-fastq = dxpy.scripts.dx_mappings_to_fastq:main', 'dx-reads-to-fastq = dxpy.scripts.dx_reads_to_fastq:main', 'dx-app-wizard = dxpy.scripts.dx_app_wizard:main', 'dx-spans-to-bed = dxpy.scripts.dx_spans_to_bed:main', 'dx-mappings-to-sam = dxpy.scripts.dx_mappings_to_sam:main', 'dx-build-applet = dxpy.scripts.dx_build_applet:main', 'dx-build-report-html = dxpy.scripts.dx_build_report_html:main', 'dx-gtable-to-tsv = dxpy.scripts.dx_gtable_to_tsv:main', 'dx = dxpy.scripts.dx:main', 'dx-build-asset = dxpy.scripts.dx_build_asset:main', 'dx-genes-to-gtf = dxpy.scripts.dx_genes_to_gtf:main', 'dx-variants-to-vcf = dxpy.scripts.dx_variants_to_vcf:main', 'dx-gtable-to-csv = dxpy.scripts.dx_gtable_to_csv:main', 'dx-genes-to-gff = dxpy.scripts.dx_genes_to_gff:main', 'dx-build-app = dxpy.scripts.dx_build_app:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'xattr', 'ws4py >=0.3.2', 'python-dateutil >=2.3', 'python-magic >=0.4.6', 'beautifulsoup4 >=4.4.1', 'psutil >=3.3.0', 'requests >=2.7.0', 'futures >=3.0.4', 'fusepy >=2.0.2', 'xattr >=0.6.4']), ('run', ['python', 'xattr', 'ws4py >=0.3.2', 'python-dateutil >=2.3', 'python-magic >=0.4.6', 'beautifulsoup4 >=4.4.1', 'psutil >=3.3.0', 'requests >=2.7.0', 'futures >=3.0.4', 'fusepy >=2.0.2', 'xattr >=0.6.4'])])), ('test', CommentedMap([('imports', ['dxpy', 'dxpy.bindings', 'dxpy.cli', 'dxpy.packages', 'dxpy.packages.argcomplete', 'dxpy.scripts', 'dxpy.templating', 'dxpy.utils']), ('commands', ['dx-mappings-to-fastq --help', 'dx-reads-to-fastq --help', 'dx-app-wizard --help', 'dx-spans-to-bed --help', 'dx-mappings-to-sam --help', 'dx-build-applet --help', 'dx-build-report-html --help', 'dx-gtable-to-tsv --help', 'dx --help', 'dx-build-asset --help', 'dx-genes-to-gtf --help', 'dx-variants-to-vcf --help', 'dx-gtable-to-csv --help', 'dx-genes-to-gff --help', 'dx-build-app --help']), ('requires', ['coverage >=3.7.1', 'pexpect >=4.2', 'pyopenssl >=16.2.0'])])), ('about', CommentedMap([('home', 'https://github.com/dnanexus/dx-toolkit'), ('license', 'Apache v2.0'), ('summary', 'DNAnexus Platform API bindings for Python')]))])), 'dxua': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dxua/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://wiki.dnanexus.com/Downloads#Upload-Agent'), ('license', 'Apache v2.0'), ('summary', 'command-line tool for uploading files to the DNAnexus Platform')])), ('package', CommentedMap([('name', 'dxua'), ('version', '<{ version }}')])), ('requirements', None), ('test', CommentedMap([('commands', ['dxua --help &> /dev/null'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'dnanexus-upload-agent-<{ version }}-osx.zip'), ('url', 'https://wiki.dnanexus.com/images/files/dnanexus-upload-agent-<{ version }}-osx.zip'), ('md5', '83fd9bf4bbee46c887e66195048f8e87')]))])), 'e-pcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/e-pcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'e-pcr'), ('version', '2.3.12')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'e-PCR-2.3.12-1-src.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/schuler/e-PCR/e-PCR-2.3.12-1-src.tar.gz'), ('md5', '516b111f78769af24393e61b0bf7fd80')])), ('requirements', CommentedMap([('build', ['gcc [linux]', 'llvm [osx]']), ('run', ['libgcc [linux]'])])), ('test', CommentedMap([('commands', ['e-PCR -h 2>&1 | grep usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.ncbi.nlm.nih.gov/tools/epcr/'), ('license', 'GPLv2'), ('summary', 'e-PCR identifies sequence tagged sites(STSs)within DNA sequences..')]))])), 'ea-utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ea-utils/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ea-utils'), ('version', '1.1.2.537')])), ('source', CommentedMap([('fn', 'ea-utils.1.1.2-537.tar.gz'), ('url', 'https://ea-utils.googlecode.com/files/ea-utils.1.1.2-537.tar.gz'), ('sha1', '688bddb1891ed186be0070d0d581816a35f7eb4e'), ('patches', ['buildflags.patch'])])), ('requirements', CommentedMap([('build', ['gsl'])])), ('test', CommentedMap([('commands', ['fastq-stats -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://code.google.com/p/ea-utils/'), ('license', 'MIT'), ('summary', 'Command-line tools for processing biological sequencing data.')]))])), 'eagle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eagle'), ('version', '0.9.0')])), ('source', CommentedMap([('fn', 'eagle-0.9.tar.gz'), ('url', 'https://bitbucket.org/christopherschroeder/eagle/get/v0.9.tar.gz'), ('md5', '5f1abfc540f234551760043a53b26268')])), ('build', CommentedMap([('skip', True), ('entry_points', ['eagle=run:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover']), ('run', ['python', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover'])])), ('test', CommentedMap([('commands', ['eagle --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/christopherschroeder/eagle'), ('license', 'MIT License'), ('summary', 'Eagle is a webtool for genome variants and snp analysis'), ('license_family', 'MIT')]))])), 'ebfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ebfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ebfilter'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'EBFilter-<{ version }}.tar.gz'), ('url', 'https://github.com/Genomon-Project/EBFilter/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python <3', 'pysam', 'scipy', 'numpy', 'pyvcf']), ('run', ['python <3', 'pysam', 'scipy', 'numpy', 'pyvcf'])])), ('test', CommentedMap([('commands', ['EBFilter --help'])])), ('about', CommentedMap([('home', 'https://github.com/Genomon-Project/EBFilter'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'EBFilter (Empirical Bayesian Mutation Filtering)'), ('license_family', 'GPL')]))])), 'ebisearch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ebisearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/09/70/41c23c12b7a3610625b97b989fca91ecf66df6e30053128ef97ff9fef75f/ebisearch-0.0.3.tar.gz'), ('md5', '55b6349bdd84b632cde8bb5bd55efbda')])), ('build', CommentedMap([('entry_points', ['ebisearch = ebisearch.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'click', 'flake8', 'pytest', 'pytest-runner']), ('run', ['python', 'requests', 'click', 'flake8'])])), ('test', CommentedMap([('imports', ['ebisearch', 'ebisearch_data']), ('commands', ['ebisearch --help', 'ebi_metagenomics --help']), ('requires', ['pytest'])])), ('about', CommentedMap([('home', 'https://github.com/bebatut/ebisearch'), ('license', 'MIT License'), ('summary', \"A Python library for interacting with EBI Search's API\"), ('license_family', 'MIT')]))])), 'ecmwfapi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ecmwfapi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ecmwfapi'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'ecmwf-api-client-python.tgz'), ('url', 'https://software.ecmwf.int/wiki/download/attachments/56664858/ecmwf-api-client-python.tgz'), ('sha256', '3606f466531b34adca79b25b08a6ff401af80c991a337640490118918a637d1b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['ecmwfapi'])])), ('about', CommentedMap([('home', 'https://software.ecmwf.int/wiki/display/WEBAPI/Access+ECMWF+Public+Datasets#AccessECMWFPublicDatasets-clientlibraries'), ('license', 'Apache Licence Version 2.0'), ('summary', 'ECMWF API is a set of services developed by ECMWF http://www.ecmwf.int to allow users from the outside to access some internal features and data of the centre.')]))])), 'ecopcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ecopcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://git.metabarcoding.org/obitools/ecopcr/uploads/d9d948afac087657225f60f873d912a8/ecoPCR.tar.gz'), ('md5', '60e5a11a93994a3b4089c5c058071602')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['python', 'libgcc [not osx]', 'zlib', 'reportlab'])])), ('test', CommentedMap([('commands', ['ecoPCR -h', 'ecogrep -h', 'ecofind -h', 'ecoPCRFormat.py -h'])])), ('about', CommentedMap([('home', 'https://git.metabarcoding.org/obitools/ecopcr/wikis/home'), ('license', 'CeCill v2'), ('summary', 'ecoPCR is an electronic PCR software that helps you estimate Barcode primers quality.')]))])), 'ecoprimers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ecoprimers/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://git.metabarcoding.org/obitools/ecoprimers/uploads/37c37c8d4dd62f53d66dae7a7f7b2318/ecoPrimers.tar.gz'), ('md5', '4c6bcb67f622fdf7dc8b9cc4860d1520')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc [not osx]', 'zlib'])])), ('test', CommentedMap([('commands', ['ecoPrimers -h'])])), ('about', CommentedMap([('home', 'https://git.metabarcoding.org/obitools/ecoprimers/wikis/home'), ('license', 'CeCill v2'), ('summary', 'ecoPrimers is a software that finds primers from a set of sequence.')]))])), 'ecopy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ecopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ecopy'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'ecopy-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/aa/05/0ecbe8762d3d770e224812e1f2174d57932f00e21f7b37db76c80ddde61e/ecopy-0.1.2.tar.gz'), ('md5', 'd1fe96addd852eb93c4121260ebf9f7f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy >=1.7', 'scipy >=0.14', 'matplotlib >=1.3.1', 'pandas >=0.13', 'patsy >=0.3.0', 'gcc']), ('run', ['python', 'cython', 'numpy >=1.7', 'scipy >=0.14', 'matplotlib >=1.3.1', 'pandas >=0.13', 'patsy >=0.3.0', 'libgcc'])])), ('test', CommentedMap([('imports', ['ecopy', 'ecopy.base_funcs', 'ecopy.diversity', 'ecopy.matrix_comp', 'ecopy.ordination', 'ecopy.regression'])])), ('about', CommentedMap([('home', 'https://github.com/Auerilas/ecopy'), ('license', 'MIT License'), ('summary', 'EcoPy: Ecological Data Analysis in Python'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'eden': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eden/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eden'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'v2.0.tar.gz'), ('url', 'https://github.com/fabriziocosta/EDeN/archive/v2.0.tar.gz'), ('sha256', '1856d79d37d7116c3b33034929be85d5e96dbf584622946b411ecaaec3b3fdac')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'dill', 'joblib', 'matplotlib', 'networkx <=1.10', 'numpy', 'pandas', 'pybedtools', 'reportlab', 'requests', 'scikit-learn >=0.17.0', 'scipy >=0.14.0', 'esmre', 'rdkit']), ('run', ['python', 'dill', 'joblib', 'matplotlib', 'networkx <=1.10', 'numpy', 'pandas', 'pybedtools', 'reportlab', 'requests', 'scikit-learn >=0.17.0', 'scipy >=0.14.0', 'esmre', 'rdkit', 'py-graphviz', 'biopython', 'weblogo', 'cvxopt', 'openbabel', 'muscle', 'rnashapes'])])), ('test', CommentedMap([('imports', ['eden', 'eden.util', 'eden.converter', 'eden.modifier', 'eden.converter.graph', 'eden.converter.molecule', 'eden.converter.rna', 'eden.modifier', 'eden.modifier.rna', 'eden.modifier.graph', 'graphviz'])])), ('about', CommentedMap([('home', 'https://github.com/fabriziocosta/EDeN'), ('license', 'MIT'), ('summary', 'The Explicit Decomposition with Neighborhoods (EDeN) is a decompositional kernel based on the Neighborhood Subgraph Pairwise Distance Kernel (NSPDK) that can be used to induce an explicit feature representation for graphs. This in turn allows the adoption of machine learning algorithm to perform supervised and unsupervised learning task in a scalable way (e.g. using fast stochastic gradient descent methods in classification and approximate neighborhood queries in clustering).')]))])), 'edena': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/edena/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'EdenaV<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://www.genomic.ch/edena/EdenaV<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['edena -h'])])), ('about', CommentedMap([('home', 'http://www.genomic.ch/edena.php'), ('license', 'GPL3'), ('summary', 'de novo short reads assembler')]))])), 'editdistance': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/editdistance/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'editdistance'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'editdistance-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/a8/08/97ed5b60023ab4b67a46ee24bf1d3db93a9abc5b3be7e8f90de18160aaed/editdistance-0.3.1.tar.gz'), ('md5', '372752ca61cf4abe60941357cdb7065f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc', 'llvm']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['editdistance'])])), ('about', CommentedMap([('home', 'https://www.github.com/aflc/editdistance'), ('license', 'MIT License'), ('summary', 'Fast implementation of the edit distance (Levenshtein distance)'), ('license_family', 'MIT')]))])), 'edittag': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/edittag/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'edittag'), ('version', '1.1')])), ('source', CommentedMap([('url', 'https://github.com/faircloth-lab/edittag/archive/v1.1.tar.gz'), ('md5', '3815d876086980d64900e36b0701202b'), ('patches', ['edittag.patch'])])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.3', 'setuptools']), ('run', ['python', 'numpy >=1.3'])])), ('test', CommentedMap([('imports', ['edittag'])])), ('about', CommentedMap([('home', 'http://github.com/faircloth-lab/edittag/'), ('license', 'http://www.opensource.org/licenses/BSD-3-Clause'), ('summary', 'Design and check sets of edit metric sequence tags.')]))])), 'edlib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/edlib/1.1.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'edlib'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/Martinsos/edlib/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libgcc', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['echo \">\" > test.fa && echo \"AC\" >> test.fa && edlib-aligner test.fa test.fa'])])), ('about', CommentedMap([('home', 'https://github.com/Martinsos/edlib/'), ('license', 'MIT'), ('summary', 'C/C++ library and program for sequence alignment using edit (Levenshtein) distance')]))])), 'effectivet3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/effectivet3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'effectivet3'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/TTSS_GUI/TTSS_GUI_1.0.1_src_all.jar'), ('fn', 'TTSS_GUI-1.0.1.jar'), ('sha256', '5a811a86a14dffbc5ae91dc45f40791642b8866266ffd2337b97d4c298129cab')])), ('requirements', CommentedMap([('build', ['openjdk >=6', 'curl', 'coreutils']), ('run', ['openjdk >=6', 'python'])])), ('about', CommentedMap([('home', 'http://www.compbio.dundee.ac.uk/nod'), ('license', 'Apache License, Version 2.0 + others used internally'), ('summary', 'Command line NoD (clinod), for  predicting nucleolar localization sequences.')])), ('test', CommentedMap([('files', ['empty.fasta']), ('commands', ['effectivet3 -f empty.fasta -m TTSS_ANIMAL-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_PLANT-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-2.0.2.jar -t selective -q'])])), ('extra', CommentedMap([('notes', 'EffectiveT3 is Java program which is packaged with a custom wrapper shell\\nscript. This shell wrapper is called \"effectivet3\" and is on $PATH by\\ndefault. By default \"-Xms512m -Xmx1g\" is set in the wrapper. If you want\\nto overwrite it you can specify these values directly after your binaries.\\nIf you have _JAVA_OPTIONS set globally this will take precedence.\\nFor example run it with \"effectivet3 -Xms512m -Xmx1g ...\"\\n')]))])), 'eigensoft': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eigensoft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eigensoft'), ('version', '6.0.1')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'EIG6.0.1.tar.gz'), ('url', 'https://data.broadinstitute.org/alkesgroup/EIGENSOFT/OLD/EIG6.0.1.tar.gz'), ('md5', 'a6ed8f2cb48fe76886dcb97af289596c')])), ('requirements', CommentedMap([('build', ['gcc', 'openblas', 'gsl ==1.16']), ('run', ['libgcc', 'openblas', 'gsl ==1.16', 'perl'])])), ('test', CommentedMap([('commands', ['smartpca 2>&1 | grep \"no parameters\"'])])), ('about', CommentedMap([('home', 'http://www.hsph.harvard.edu/alkes-price/software/'), ('license', 'Custom OSS'), ('summary', 'Population Genetics - Principal Components Analysis (PCA)')]))])), 'elasticluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/elasticluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'elasticluster'), ('version', '0.1.3bcbio')])), ('source', CommentedMap([('fn', 'elasticluster-0.1.3bcbio_11.tar.gz'), ('url', 'https://github.com/chapmanb/elasticluster/archive//8d6d108.tar.gz')])), ('build', CommentedMap([('number', 11), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'ansible', 'azure', 'pycli', 'paramiko', 'voluptuous', 'configobj', 'boto', 'google-api-python-client', 'oauth2client', 'python-gflags'])])), ('test', CommentedMap([('imports', ['elasticluster', 'elasticluster.conf', 'elasticluster.main', 'elasticluster.providers.ansible_provider', 'elasticluster.providers.ec2_boto'])])), ('about', CommentedMap([('home', 'https://github.com/gc3-uzh-ch/elasticluster'), ('license', 'GPL'), ('summary', 'Create, manage and setup computing clusters hosted on a public or private cloud infrastructure.')]))])), 'embassy-phylip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/embassy-phylip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'embassy-phylip'), ('version', '3.69.650')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'PHYLIPNEW-3.69.650.tar.gz'), ('url', 'ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/PHYLIPNEW-3.69.650.tar.gz'), ('md5', '488d117d32a391a9b53c60260595e67b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'zlib', 'emboss']), ('run', ['zlib', 'emboss'])])), ('about', CommentedMap([('home', 'http://emboss.open-bio.org/'), ('license', 'GPL'), ('summary', 'embassy/emboss wrappers for phylip functions')])), ('test', CommentedMap([('commands', ['fneighbor --help', 'ffactor --help', 'fmix --help'])]))])), 'embl-validator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/embl-validator/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.ebi.ac.uk/ena/software/flat-file-validator'), ('license', None), ('summary', 'ENA flat file validator for submission')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk', 'python'])])), ('package', CommentedMap([('name', 'embl-api-validator'), ('version', '1.1.173')])), ('source', CommentedMap([('fn', 'embl-api-validator-1.1.171.jar'), ('url', 'http://central.maven.org/maven2/uk/ac/ebi/ena/sequence/embl-api-validator/1.1.173/embl-api-validator-1.1.173.jar'), ('sha256', '91fc44f005746bf8ce2e876bccf3bc239f68a9aaba211a3c187a5932303a2483')])), ('test', CommentedMap([('commands', ['embl-api-validator -version'])]))])), 'emboss': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emboss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'emboss'), ('version', '6.5.7')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'EMBOSS-6.5.7.tar.gz'), ('url', 'ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz'), ('md5', '6a2cb3f93d5e9415c74ab0f6b1ede5f0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'http://emboss.open-bio.org/'), ('license', 'GPL'), ('summary', 'The European Molecular Biology Open Software Suite')])), ('test', CommentedMap([('commands', ['needle --help', 'transeq --help', 'sixpack --help', 'seqret --help'])]))])), 'emirge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emirge/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install --single-version-externally-managed --record record.txt')])), ('requirements', CommentedMap([('build', ['cython', 'zlib', 'gcc', 'llvm', 'python', 'setuptools', 'numpy']), ('run', ['zlib', 'libgcc', 'python', 'numpy', 'pysam', 'scipy', 'biopython', 'bowtie', 'samtools', 'vsearch'])])), ('test', CommentedMap([('imports', ['pykseq', '_emirge_amplicon', '_emirge']), ('commands', ['emirge_makedb.py --help >/dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/csmiller/EMIRGE'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'EMIRGE reconstructs full length sequences from short sequencing reads'), ('license_family', 'GPL3')]))])), 'emperor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emperor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'emperor'), ('version', '0.9.51')])), ('source', CommentedMap([('fn', 'emperor-0.9.51.tar.gz'), ('url', 'https://pypi.python.org/packages/source/e/emperor/emperor-0.9.51.tar.gz'), ('md5', '9b763cae89fe7b938b8acab4b39b92ba')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k or osx]')])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.7', 'qcli', 'scikit-bio >=0.2.1,<0.3.0']), ('run', ['python', 'numpy >=1.7', 'qcli', 'scikit-bio >=0.2.1,<0.3.0'])])), ('test', CommentedMap([('imports', ['emperor', 'emperor.qiime_backports']), ('commands', ['make_emperor.py --version'])])), ('about', CommentedMap([('home', 'http://github.com/biocore/emperor'), ('license', 'BSD License'), ('license_file', 'LICENSE.md'), ('summary', 'Emperor a tool for the analysis and visualization of large microbial ecology datasets')]))])), 'ensembl-vep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ensembl-vep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ensembl-vep'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'vep-<{ version }}.tar.gz'), ('url', 'https://github.com/Ensembl/ensembl-vep/archive/release/<{ version }}.tar.gz'), ('md5', 'efd312434d7a8b5aa1d50b31dc59015d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['curl', 'perl', 'perl-archive-zip', 'perl-lwp-simple', 'unzip', 'perl-dbi']), ('run', ['perl', 'htslib <{CONDA_HTSLIB}}*', 'perl-bioperl', 'perl-bio-db-hts >=2.7', 'perl-cgi', 'perl-dbi', 'perl-dbd-mysql', 'perl-io-compress', 'perl-json', 'perl-perlio-gzip', 'perl-sereal', 'perl-set-intervaltree', 'unzip'])])), ('test', CommentedMap([('commands', ['vep --help', 'vep_convert_cache --help', 'vep_install --help', 'haplo --help', 'variant_recoder --help'])])), ('about', CommentedMap([('home', 'http://www.ensembl.org/info/docs/tools/vep/index.html'), ('license', 'Apache 2.0'), ('summary', 'The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.')])), ('extra', CommentedMap([('notes', \"This package installs only the variant effect predictor (VEP) library\\ncode. To install data libraries, you can use the 'vep_install' command\\ninstalled along with it. For example, to install the VEP library for human\\nGRCh38 to a directory\\n\\nvep_install -a cf -s homo_sapiens -y GRCh38 -c /output/path/to/GRCh38/vep --CONVERT\\n\\n(note that vep_install is renamed from INSTALL.pl\\n to avoid having generic script names in the PATH)\\n\\nThe --CONVERT flag is not required but improves lookup speeds during\\nruns. See the VEP documentation for more details\\n\\nhttp://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html\\n\")]))])), 'entrez-direct': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/entrez-direct/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'entrez-direct'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'edirect.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/entrez/entrezdirect/versions/<{ version }}.20170714/edirect.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['ftp-cp.patch', 'xtract.patch'])])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl', 'perl-lwp-protocol-https'])])), ('test', CommentedMap([('commands', ['efetch --help', 'epost --help', 'efilter --help', 'eproxy --help', 'einfo --help', 'esearch --help', 'econtact --help', 'elink --help', 'espell --help', 'enotify --help', 'esummary --help', 'entrez-phrase-search --help', 'xtract -version'])])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/entrez/entrezdirect/versions/7.00.20170710/README'), ('license', 'PUBLIC DOMAIN'), ('summary', \"Entrez Direct (EDirect) is an advanced method for accessing the NCBI's set of interconnected databases (publication, sequence, structure, gene, variation, expression, etc.) from a UNIX terminal window. Functions take search terms from command-line arguments. Individual operations are combined to build multi-step queries. Record retrieval and formatting normally complete the process.\")]))])), 'eoulsan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eoulsan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eoulsan'), ('version', '2.0_beta4')])), ('source', CommentedMap([('fn', 'eoulsan-2.0-beta4.tar.gz'), ('url', 'https://github.com/GenomicParisCentre/eoulsan/releases/download/v2.0-beta4/eoulsan-2.0-beta4.tar.gz'), ('sha256', '97782ed5ec9b184bdb5eb79ee41d3e8c910be68db6475cf79d667ef28da35b8b')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['eoulsan.sh --version'])])), ('about', CommentedMap([('home', 'http://www.tools.genomique.biologie.ens.fr/eoulsan/'), ('license', 'LGPL'), ('summary', 'A pipeline and a framework for NGS analysis (RNA-Seq and Chip-Seq)')]))])), 'epic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/epic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'epic'), ('version', '0.2.5')])), ('source', CommentedMap([('fn', 'bioepic-0.2.5.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/69/6b/0d3e7bf8333b2144fb14326d44f7e98a01c5236d75692959063b2509d335/bioepic-0.2.5.tar.gz'), ('sha256', '8ffc81919f4df1d70b8bac5b237400ffbe03998f94bdc5da340197cf39765a85')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy', 'pandas', 'numpy', 'natsort', 'joblib', 'docopt', 'pyfaidx', 'functools32', 'typing', 'pybigwig']), ('run', ['python', 'scipy', 'pandas', 'numpy', 'natsort', 'joblib', 'docopt', 'pyfaidx', 'functools32', 'bedtools', 'typing', 'pybigwig'])])), ('test', CommentedMap([('imports', ['epic', 'epic.config', 'epic.run', 'epic.scripts', 'epic.statistics', 'epic.utils', 'epic.windows', 'epic.windows.cluster', 'epic.windows.count']), ('commands', ['epic -h'])])), ('about', CommentedMap([('home', 'http://github.com/endrebak/epic'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Chip-Seq broad peak/domain finder.')]))])), 'epydoc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/epydoc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'epydoc'), ('version', '3.0.1')])), ('source', CommentedMap([('fn', 'epydoc-3.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/e/epydoc/epydoc-3.0.1.tar.gz'), ('md5', '36407974bd5da2af00bf90ca27feeb44')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['epydoc', 'epydoc.docwriter', 'epydoc.markup', 'epydoc.test'])])), ('about', CommentedMap([('home', 'http://epydoc.sourceforge.net'), ('license', 'MIT License'), ('summary', \"Edward Loper's API Documentation Generation Tool\")]))])), 'eqtlbma': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eqtlbma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eqtlbma'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'v1.3.1.tar.gz'), ('url', 'https://github.com/timflutre/eqtlbma/archive/v1.3.1.tar.gz'), ('md5', '7c9469c61b5d50cd7113c620d30a0142'), ('patches', ['utils_script.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'libtool', 'gsl >=1.15,<2.0', 'zlib', 'texinfo']), ('run', ['libgcc', 'gsl >=1.15,<2.0', 'zlib', 'r-base', 'r-mass', 'bioconductor-genomicranges'])])), ('test', CommentedMap([('commands', ['eqtlbma_bf -h', 'eqtlbma_hm -h', 'eqtlbma_avg_bfs -h'])])), ('about', CommentedMap([('home', 'https://github.com/timflutre/eqtlbma'), ('license', 'GPLv3'), ('license_file', 'COPYING'), ('summary', 'Package to detect eQTLs jointly in multiple subgroups (e.g. tissues) via Bayesian Model Averaging.')]))])), 'ericscript': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ericscript/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ericscript'), ('version', '0.5.5')])), ('source', CommentedMap([('fn', 'ericscript-0.5.5.tar.bz2'), ('url', 'https://sourceforge.net/projects/ericscript/files/ericscript-0.5.5.tar.bz2'), ('md5', 'a871d2c1f92c407ee26a6cabdc870dc6')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('run', ['r-base', 'perl', 'bedtools', 'blat', 'bwa', 'r-ada', 'samtools 0.1.19', 'seqtk'])])), ('test', CommentedMap([('commands', ['ericscript.pl | grep 0.5.5'])])), ('about', CommentedMap([('home', 'https://sites.google.com/site/bioericscript'), ('summary', 'EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data. It is able to generate synthetic gene fusions by using the EricScript simulator and calculate a number of statistical measures for evaluating gene fusion detection methods performance with EricScript CalcStats.'), ('license', 'GPL3'), ('license_family', 'GPL3')]))])), 'erne': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/erne/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/<{ name }}/<{ version }}/<{ name }}-<{ version }}-source.tar.gz'), ('md5', '3c26b80654e1ef6ef61085f0ecf75e65'), ('patches', ['osxfixes.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'llvmdev', 'libcxx', 'boost ==1.63.0', 'boost <{CONDA_BOOST}}*', 'zlib', 'bzip2']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['erne-map --version | grep ERNE', 'erne-bs5 --version | grep ERNE', 'erne-create --version | grep ERNE', 'erne-filter --version | grep ERNE', 'erne-meth --version | grep ERNE'])])), ('about', CommentedMap([('home', 'http://erne.sourceforge.net'), ('license', 'GPLv3'), ('summary', 'ERNE - Extended Randomized Numerical alignEr')]))])), 'esmre': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/esmre/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'esmre'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'esmre-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/e/esmre/esmre-0.3.1.tar.gz'), ('md5', '95ace12bac0c79cf95712336489bc4a4')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['esm'])])), ('about', CommentedMap([('home', 'http://code.google.com/p/esmre/'), ('license', 'GNU Library or Lesser General Public License (LGPL)'), ('summary', 'Regular expression accelerator')]))])), 'estmapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/estmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Getting_Started_with_ESTmapper'), ('license', 'GPL'), ('summary', 'software package for the high-throughput alignment of large cDNA (EST, mRNA) sequence sets to a large eukaryotic genome of the same species')])), ('source', CommentedMap([('fn', 'ESTmapper-r2008.tar.bz2'), ('url', 'http://sourceforge.net/projects/kmer/files/ESTmapper-r2008.tar.bz2'), ('md5', '6d310460dec848fe057d0f51b1e4ccd9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'bzip2', 'perl']), ('run', ['libgcc', 'perl', 'python'])])), ('test', CommentedMap([('commands', ['ESTmapper.pl 2>&1 | grep ESTmapper'])]))])), 'eta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eta'), ('version', '0.9.7')])), ('source', CommentedMap([('fn', 'eta-0.9.7.tar.gz'), ('url', 'https://pypi.python.org/packages/56/f4/3def7582d454d511a0b48e5eabdfd1319ccb6b5fafc810fea7dee4ca6c4d/eta-0.9.7.tar.gz'), ('md5', 'fd6d815b2a3276ca07655c4037472576')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['eta'])])), ('about', CommentedMap([('home', 'http://github.com/mbreese/eta/'), ('license', 'BSD'), ('license_file', '<{ RECIPE_DIR }}/LICENSE.txt'), ('summary', 'ETA Progress bar for command-line utilities')]))])), 'ete2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ete2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ete2'), ('version', '2.3.10')])), ('source', CommentedMap([('fn', 'ete2-2.3.10.tar.gz'), ('url', 'https://pypi.python.org/packages/source/e/ete2/ete2-2.3.10.tar.gz'), ('md5', '87bf020c25c293e8e4e1b1e02a3dc2b1')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyqt 4.*', 'numpy', 'lxml', 'mysql-python']), ('run', ['python', 'pyqt 4.*', 'numpy', 'lxml', 'mysql-python'])])), ('test', CommentedMap([('imports', ['ete2'])])), ('about', CommentedMap([('home', 'http://etetoolkit.org/'), ('license', 'GPLv3'), ('summary', 'Phylogenetic tree analyses and exploration')]))])), 'ete3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ete3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ete3'), ('version', '3.0.0b35')])), ('source', CommentedMap([('fn', 'ete3-3.0.0b35.tar.gz'), ('url', 'https://pypi.python.org/packages/f0/c5/e1bb20c02f0e7cd924eab783901b4e2acf8fda79c0f7c7303b740bf7b98e/ete3-3.0.0b35.tar.gz'), ('md5', 'f4797833ba9afc8547061bfda286c6f5')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['lxml', 'numpy', 'pyqt 4.11.*', 'python', 'six']), ('run', ['lxml', 'numpy', 'pyqt 4.11.*', 'python', 'scipy', 'six', 'xorg-libxext', 'xorg-xextproto', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau', 'libgcc'])])), ('test', CommentedMap([('imports', ['ete3', 'ete3.treeview'])])), ('about', CommentedMap([('home', 'http://etetoolkit.org/'), ('license', 'GPLv3'), ('summary', 'A Python framework for the analysis and visualization of trees')]))])), 'exonerate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exonerate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exonerate'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'exonerate-<{ version }}.tar.gz'), ('url', 'http://ftp.ebi.ac.uk/pub/software/vertebrategenomics/exonerate/exonerate-<{ version }}.tar.gz'), ('md5', '126fbade003b80b663a1d530c56f1904')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'pcre', 'pkg-config', 'automake', 'autoconf']), ('run', ['libgcc', 'glib', 'pcre'])])), ('test', CommentedMap([('commands', ['exonerate 2>&1 | grep \"exonerate: A generic sequence comparison tool\"'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate'), ('license', 'GPL-3.0'), ('summary', 'Exonerate - A generic tool for pairwise sequence comparison / alignment')]))])), 'expansionhunter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/expansionhunter/meta.yaml', CommentedMap([('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Illumina/ExpansionHunter/archive/v<{ version }}.tar.gz'), ('md5', '24d39d1f1c4b013f87a589932bc88976')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'boost <{CONDA_BOOST}}*', 'icu']), ('run', ['boost <{CONDA_BOOST}}*', 'icu', 'libgcc', 'libcxx'])])), ('test', CommentedMap([('commands', ['ExpansionHunter --help 2>&1 | grep \"help\"'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/ExpansionHunter'), ('license', 'GPL3'), ('license_file', 'LICENSE.txt'), ('summary', 'a tool for estimating repeat sizes')]))])), 'exparna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exparna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exparna'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/ExpaRNA/archive/1.0.1.tar.gz'), ('md5', '2120711de20d4f62ded1e0394cd0e601')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'automake', 'autoconf', 'perl-threaded', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['ExpaRNA -h'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/ExpaRNA'), ('license', 'GPL-2.0'), ('license_file', 'LICENSE'), ('summary', 'The program finds the longest common subsequence of exact pattern matches (LCS-EPM)')]))])), 'export2graphlan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/export2graphlan/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/CibioCM/export2graphlan/overview'), ('summary', 'export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn'), ('license', 'MIT')])), ('package', CommentedMap([('name', 'export2graphlan'), ('version', '0.19')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('url', 'https://bitbucket.org/CibioCM/export2graphlan/get/d6c10c1.tar.gz'), ('sha256', 'b8a79cd6de73226e1fcab43ebc941783c72b782f5d44b954a52b39a06652af25'), ('patches', ['export2graphlan.py.patch'])])), ('requirements', CommentedMap([('run', ['python', 'matplotlib', 'pandas', 'scipy', 'numpy', 'hclust2', 'libgcc'])])), ('test', CommentedMap([('commands', ['export2graphlan.py -h'])]))])), 'express': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/express/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'express'), ('version', '1.5.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'express_1.5.1_darwin_x64.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/express/express_1.5.1_darwin_x64.tar.gz')])), ('test', CommentedMap([('commands', ['express --help &> /dev/null || [[ \"$?\" == 1 ]]'])])), ('about', CommentedMap([('home', 'http://bio.math.berkeley.edu/eXpress/'), ('license', 'Artistic License 2.0'), ('summary', 'eXpress is a streaming DNA/RNA sequence quantification tool.')]))])), 'extern': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/extern/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'extern'), ('version', '0.2.1')])), ('source', CommentedMap([('fn', 'extern-0.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/cf/86/c861f334f2af20a577348c3eb9e05b8d0bbf3a9b7a7126609541becd7033/extern-0.2.1.tar.gz'), ('md5', '093c4022301cc57eef0344c577088a09')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['extern'])])), ('about', CommentedMap([('home', 'https://pypi.python.org/pypi/extern/'), ('license', 'MIT'), ('summary', 'Extern is an opinionated version of Pythons subprocess, making it just that little bit more convenient to run shell commands from within Python code.'), ('license_family', 'MIT')]))])), 'extract-sv-reads': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/extract-sv-reads/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'extract-sv-reads'), ('version', '1.1.2')])), ('source', CommentedMap([('git_url', 'https://github.com/hall-lab/extract_sv_reads.git'), ('git_tag', '9bb74fc82d44f1abd73033cbb2e11ba6cf59f885')])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'perl', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['extract-sv-reads -h', 'extract-sv-reads --version'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/extract_sv_reads'), ('license', 'MIT'), ('summary', 'Tool for extracting splitter or discordant reads from a BAM or CRAM file.')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))])), 'extract_fullseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/extract_fullseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'extract_fullseq'), ('version', 3.101)])), ('source', CommentedMap([('fn', 'extract_fullseq'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/mac-os/extract_fullseq'), ('md5', '719474a0e3b54449cbe06f6e442994dd')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['extract_fullseq 2>&1 | grep extract_fullseq > /dev/null'])])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/'), ('license', 'Public Domain'), ('summary', 'extract_fullseq is part of BMTagger aka Best Match Tagger, for removing human reads from metagenomics datasets')]))])), 'extract_genome_region': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/extract_genome_region/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'extract_genome_region'), ('version', '0.0.3')])), ('source', CommentedMap([('fn', 'extract_genome_region_0.0.3.tar.gz'), ('md5', '3d45e601ff036ebf62405ce7fa482bd8'), ('url', 'https://github.com/xguse/extract-genome-region/archive/v0.0.3.tar.gz')])), ('build', CommentedMap([('entry_points', ['extract_genome_region = extract_genome_region.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click', 'pyfaidx']), ('run', ['python', 'click', 'pyfaidx'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 extract_genome_region --help'])])), ('about', CommentedMap([('home', 'https://github.com/xguse/extract-genome-region'), ('license', 'BSD License'), ('summary', 'Given a CSV file of variable information defining the regions of interest, return a file that contains a fasta-formatted representation of these regions.')]))])), 'extractpirs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/extractpirs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'extractpirs'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'extractPIRs.v1.r68.x86_64.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/files/extractPIRs.v1.r68.x86_64.tgz'), ('md5', 'de81232195f6e2eeed17a3ac028a02b6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['libgcc']), ('run', ['zlib', 'libgcc'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'), ('license', 'Free for Academic Use'), ('summary', 'A phase informative read (PIR) is a sequencing read that span at least 2 heterozyguous sites. In the following, we require first that the sequence data is stored in BAM files and second that the genotype data to be phased is in VCF format. We developed a small tool, extractPIRs, to extract the PIRs from BAM files that relies on the samtools API for efficiency')])), ('test', CommentedMap([('commands', ['extractPIRs --vcf null --bam null --out null'])]))])), 'fadapa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fadapa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fadapa'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'fadapa-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/fadapa/fadapa-0.3.1.tar.gz'), ('md5', '4924717f0a02c1fa9fb8985970d59bf0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['fadapa'])])), ('about', CommentedMap([('home', 'https://github.com/fadapa/fadapa'), ('license', 'uncopyrighted'), ('summary', 'FAstqc DAta PArser - A minimal parser to parse FastQC output data')]))])), 'fann': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fann/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fann'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'FANN-2.2.0-Source.zip'), ('url', 'http://downloads.sourceforge.net/project/fann/fann/2.2.0/FANN-2.2.0-Source.zip'), ('md5', '292d914684c9eab1d84eb1f59b0fbf69')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://leenissen.dk/fann/wp/'), ('summary', 'Fast Artificial Neural Network Library'), ('license', 'LGPL-2.1')]))])), 'fast5': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fast5/0.6.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mateidavid/fast5/archive/091849dd7d6fce0812c8d1fc1266b8cb8e071a67.tar.gz'), ('md5', '4d431c91b3e8b7ac6f82636a8eb1a2dc')])), ('requirements', CommentedMap([('build', None), ('run', ['hdf5'])])), ('test', CommentedMap([('commands', ['test -e ${PREFIX}/include/fast5/fast5.hpp'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/fast5'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A C++ header-only library for reading Oxford Nanopore Fast5 files.')]))])), 'fasta3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fasta3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fasta3'), ('version', '36.3.8')])), ('source', CommentedMap([('fn', 'fasta-36.3.8.tar.gz'), ('url', 'http://faculty.virginia.edu/wrpearson/fasta/fasta36/fasta-36.3.8.tar.gz'), ('md5', '071c1a33534e3b47c9a680b87f2a73d7'), ('patches', ['libpath.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['fasta36 > /dev/null', 'fastf36 > /dev/null', 'fastm36 > /dev/null', 'fasts36 > /dev/null', 'fastx36 > /dev/null', 'fasty36 > /dev/null', 'ggsearch36 > /dev/null', 'glsearch36 > /dev/null', 'lalign36 > /dev/null', 'ssearch36 > /dev/null', 'tfastf36 > /dev/null', 'tfastm36 > /dev/null', 'tfasts36 > /dev/null', 'tfastx36 > /dev/null', 'tfasty36 > /dev/null'])])), ('about', CommentedMap([('home', 'http://faculty.virginia.edu/wrpearson/fasta'), ('license', 'Apache 2.0'), ('summary', 'The FASTA package - protein and DNA sequence similarity searching and alignment programs'), ('license_file', 'LICENSE')]))])), 'fastahack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastahack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastahack'), ('version', '2016.07.2')])), ('source', CommentedMap([('fn', 'fastahack.zip'), ('url', 'https://github.com/ekg/fastahack/archive/bbc645f2f7966cb7b44446200c02627c3168b399.zip'), ('md5', 'd32a9dee44a8da7477631930e92f5095')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastahack 2>&1 | grep \"usage\" >/dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/fastahack'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'fastahack --- *fast* FASTA file indexing, subsequence and sequence extraction')]))])), 'fastaindex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastaindex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastaindex'), ('version', '0.11c')])), ('source', CommentedMap([('git_url', 'https://github.com/lpryszcz/FastaIndex'), ('git_rev', '4f369ba763e8a59224d2a9320c557018d4f83fd5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['FastaIndex'])])), ('about', CommentedMap([('home', 'https://github.com/lpryszcz/FastaIndex'), ('license', 'GPLv3'), ('summary', 'FastA indexing and sequence retrival.')]))])), 'fastdtw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastdtw/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'fastdtw'), ('version', '0.2.0')])), ('source', CommentedMap([('fn', 'fastdtw-0.2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/fastdtw/fastdtw-0.2.0.tar.gz'), ('md5', 'bc0d4a886db08a2c03eb403d8015af3f')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('about', CommentedMap([('home', 'https://github.com/slaypni/fastdtw'), ('license', 'MIT License'), ('summary', 'Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity')]))])), 'fastlmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastlmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastlmm'), ('version', '0.2.24')])), ('source', CommentedMap([('fn', 'fastlmm-0.2.24.zip'), ('url', 'https://pypi.python.org/packages/source/f/fastlmm/fastlmm-0.2.24.zip'), ('md5', '009793380099e6bf4cda2215b0fc9c08')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.16.0', 'numpy >=1.9.3', 'pandas >=0.16.2', 'statsmodels >=0.6.1', 'matplotlib >=1.4.3', 'scikit-learn >=0.16.1', 'pysnptools >=0.3.8', 'dill']), ('run', ['python', 'scipy >=0.16.0', 'numpy >=1.9.3', 'pandas >=0.16.2', 'statsmodels >=0.6.1', 'matplotlib >=1.4.3', 'scikit-learn >=0.16.1', 'pysnptools >=0.3.8', 'dill'])])), ('test', CommentedMap([('imports', ['fastlmm', 'fastlmm.association', 'fastlmm.association.tests', 'fastlmm.external', 'fastlmm.external.util', 'fastlmm.feature_selection', 'fastlmm.inference', 'fastlmm.pyplink', 'fastlmm.pyplink.altset_list', 'fastlmm.pyplink.snpreader', 'fastlmm.pyplink.snpset', 'fastlmm.util', 'fastlmm.util.runner', 'fastlmm.util.standardizer', 'fastlmm.util.stats', 'fastlmm.util.stats.quadform', 'fastlmm.util.stats.quadform.qfc_src'])])), ('about', CommentedMap([('home', 'http://research.microsoft.com/en-us/um/redmond/projects/mscompbio/fastlmm/'), ('license', 'Apache 2.0'), ('summary', 'Fast GWAS')]))])), 'fastme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastme'), ('version', '2.1.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'fastme-2.1.5.tar.gz'), ('url', 'http://www.atgc-montpellier.fr/download/sources/fastme/fastme-2.1.5.tar.gz'), ('sha256', 'af636864902f1c038203ad370da6ed2b97edaaa97bfde060ec7aab7b05672277')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastme --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/fastme/binaries.php'), ('summary', 'a comprehensive, accurate and fast distance-based phylogeny inference program.'), ('license', 'GPLv3')]))])), 'fastool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastool'), ('version', '0.1.4')])), ('source', CommentedMap([('fn', '0.1.4.tar.gz'), ('url', 'https://github.com/fstrozzi/Fastool/archive/0.1.4.tar.gz'), ('sha256', 'f86f02fced9479b77d6d27a442b66e66f0c55d40d5ad2ddd91a752de57024540')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['fastool 2>&1 | grep append_to > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/fstrozzi/Fastool'), ('license', 'MIT'), ('summary', 'A simple and quick tool to read huge FastQ and FastA files (both normal and gzipped) and manipulate them.')]))])), 'fastq-and-furious': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastq-and-furious/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastq-and-furious'), ('version', '0.1.0')])), ('source', CommentedMap([('fn', 'fastq-and-furious-0.1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/e0/52/96cc1e9cda6d65ad4179facd18f50aeb21a146f7bd08ef9249ae8f8c6d1e/fastq-and-furious-0.1.0.tar.gz'), ('md5', '45a9403e708fa267140f7a49f1cb106f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['fastqandfurious', 'fastqandfurious.demo'])])), ('about', CommentedMap([('home', 'https://github.com/lgautier/fastq-and-furious'), ('license', 'MIT License'), ('summary', 'Fast handling of FASTQ files'), ('license_family', 'MIT')]))])), 'fastq-join': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastq-join/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastq-join'), ('version', '1.3.1')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'fastq-join-1.3.1.tar.gz'), ('url', 'https://github.com/brwnj/fastq-join/archive/v1.3.1.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['fastq-join -h'])])), ('about', CommentedMap([('home', 'https://github.com/brwnj/fastq-join'), ('license', 'MIT'), ('summary', 'Similar to audy\\'s stitch program, but in C, more efficient and supports some automatic benchmarking and tuning. It uses the same \"squared distance for anchored alignment\" as other tools.')]))])), 'fastq-multx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastq-multx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastq-multx'), ('version', '1.3.1')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'fastq-multx-1.3.1.tar.gz'), ('url', 'https://github.com/brwnj/fastq-multx/archive/v1.3.1.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastq-multx -h'])])), ('about', CommentedMap([('home', 'https://github.com/brwnj/fastq-multx'), ('license', 'MIT'), ('summary', \"Demultiplexes a fastq. Capable of auto-determining barcode id's based on a master set fields. Keeps multiple reads in-sync during demultiplexing. Can verify that the reads are in-sync as well, and fail if they're not.\")]))])), 'fastq-screen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastq-screen/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/'), ('license', 'GPLv3'), ('summary', 'FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['bowtie', 'bowtie2', 'perl', 'perl-gdgraph']), ('run', ['bowtie', 'bowtie2', 'perl', 'perl-gdgraph'])])), ('package', CommentedMap([('name', 'fastq-screen'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'fastq_screen_v<{ version }}.tar.gz'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/fastq_screen_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('test', CommentedMap([('commands', ['fastq_screen -h >/dev/null', 'fastq_screen --version'])]))])), 'fastqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastqc/0.11.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'), ('license', 'GPL'), ('summary', 'A quality control tool for high throughput sequence data.')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk', 'perl-threaded'])])), ('package', CommentedMap([('name', 'fastqc'), ('version', '0.11.3')])), ('source', CommentedMap([('fn', 'fastqc_v0.11.3.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.3.zip'), ('patches', ['hashbang.patch'])])), ('test', CommentedMap([('commands', [\"fastqc --version | grep '^FastQC v0\\\\.11\\\\.3\\\\b'\"])]))])), 'fasttree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fasttree/2.1.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fasttree'), ('version', '2.1.8')])), ('source', CommentedMap([('fn', 'FastTree.c'), ('url', 'http://meta.microbesonline.org/fasttree/FastTree-2.1.8.c'), ('md5', 'c7e85689a26cabba241378d4f633c2fa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('test', CommentedMap([('commands', ['FastTree | cat'])])), ('about', CommentedMap([('home', 'http://www.microbesonline.org/fasttree/'), ('license', 'GPL v2'), ('summary', 'FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences')]))])), 'fastuniq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastuniq/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/fastuniq/'), ('license', 'Creative Commons Attribution License'), ('summary', 'FastUniq, A Fast De Novo Duplicates Removal Tool for Paired Short Reads')])), ('package', CommentedMap([('name', 'fastuniq'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'FastUniq-1.1.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/fastuniq/FastUniq-1.1.tar.gz'), ('sha256', '9ebf251566d097226393fb5aa9db30a827e60c7a4bd9f6e06022b4af4cee0eae')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastuniq  2>&1 >/dev/null | grep \"The input file list of paired\"'])]))])), 'fastx_toolkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastx_toolkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastx_toolkit'), ('version', '0.0.14')])), ('source', CommentedMap([('url', 'https://github.com/agordon/fastx_toolkit/releases/download/0.0.14/fastx_toolkit-0.0.14.tar.bz2'), ('fn', 'fastx_toolkit-0.0.14.tar.bz2'), ('md5', 'bf1993c898626bb147de3d6695c20b40'), ('patches', ['fastx.patch'])])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 5), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc [not osx]', 'llvm [osx]', 'cython', 'nose', 'libgtextutils']), ('run', ['libgcc [not osx]', 'cython', 'nose', 'libgtextutils', 'gnuplot >=5.0.5', 'perl', 'perl-perlio-gzip', 'perl-gd', 'perl-gdgraph-histogram'])])), ('test', CommentedMap([('commands', ['SCRIPT=$(which fasta_clipping_histogram.pl) && perl5.22.0 \"$SCRIPT\"', 'fastx_trimmer -h | grep \"Part of FASTX Toolkit 0.0.14 by A. Gordon (assafgordon@gmail.com)\"'])])), ('about', CommentedMap([('home', 'https://github.com/agordon/fastx_toolkit'), ('license', 'AGPL'), ('summary', 'The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.\\nNext-Generation sequencing machines usually produce FASTA or FASTQ files, containing multiple short-reads sequences (possibly with quality information).\\nThe main processing of such FASTA/FASTQ files is mapping (aka aligning) the sequences to reference genomes or other databases using specialized programs. Example of such mapping programs are: Blat, SHRiMP, LastZ, MAQ and many many others.\\nHowever, it is sometimes more productive to preprocess the FASTA/FASTQ files before mapping the sequences to the genome - manipulating the sequences to produce better mapping results.\\nThe FASTX-Toolkit tools perform some of these preprocessing tasks.')]))])), 'fcp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fcp/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://github.com/dparks1134/FragmentClassificationPackage/archive/v<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python', 'blast'])])), ('test', CommentedMap([('commands', ['FCP_install.py 2>&1 | grep \"FCP_install\"', 'BuildBlastDB.py', 'nb-classify --help', 'BLASTN.py', 'NB-BL.py', 'Epsilon-NB.py', 'LCA.py 2>&1 | grep \"LCA\"', 'LCA+Epsilon-NB.py', 'TaxonomicSummary.py', 'AddModel.py', 'nb-train --help'])])), ('about', CommentedMap([('home', 'https://github.com/dparks1134/FragmentClassificationPackage'), ('license', 'GPL3'), ('summary', 'Homology- and composition-based classifiers for assigning a taxonomic attribution to metagenomic fragments.')]))])), 'fermi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fermi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fermi'), ('version', '1.1_r751_beta')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'fermi_1.1-r751-beta.tar.gz'), ('url', 'https://github.com/lh3/fermi/archive/01cfc0750e654130670846728937076dedfe97b7.tar.gz'), ('md5', 'b8a8345cf266136405f8dcae4eb58494')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'perl-threaded', 'zlib'])])), ('test', CommentedMap([('commands', ['fermi  2>&1 | grep fermi > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/fermi'), ('license', 'Unknown'), ('summary', 'A WGS de novo assembler based on the FMD-index for large genomes')]))])), 'fermi2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fermi2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fermi2'), ('version', 'r193')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'fermi2_r193.tar.gz'), ('url', 'https://github.com/lh3/fermi2/archive/580ec8ffd30b73e33c7dda5b13a5bfb4281466ac.tar.gz'), ('sha256', '278fff4a0aa0203ce288a1b4d9c66245cfe38ac10fece321fed3da869a678536')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'bfc', 'ropebwt2', 'perl', 'zlib'])])), ('test', CommentedMap([('commands', ['fermi2  2>&1 | grep fermi2 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/fermi2'), ('license', 'Unknown'), ('summary', 'Fermi2 focuses on the exploration of FMD-index as a graph.')]))])), 'fermikit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fermikit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fermikit'), ('version', '0.14.dev1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'fermikit-0.14.dev1.tar.bz2'), ('url', 'https://github.com/lh3/fermikit/archive/96f7820c91b01113abf24f4d30cf86a89d8a1ccc.tar.gz'), ('sha256', 'c4f5474fb6f6d23427fbf10564de0606621015ef585f88df0ca0e97353db0fec')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib']), ('run', ['libgcc', 'perl', 'zlib', 'bfc ==r181', 'bwa ==0.7.15', 'fermi2 ==r193', 'htsbox ==r327', 'ropebwt2 ==r187', 'seqtk ==r82', 'trimadap ==r10'])])), ('test', CommentedMap([('source_files', ['test/test.sh', 'test/11_1910000_1940000.fa', 'test/HS1011_11_1910000_1940000.fastq.gz']), ('commands', [\"sed -i.bak 's|\\\\.\\\\.\\\\/fermi.kit\\\\/||g' test/test.sh && bash test/test.sh\"])])), ('about', CommentedMap([('home', 'https://github.com/lh3/fermikit'), ('license', 'Unknown'), ('summary', 'FermiKit is a de novo assembly based variant calling pipeline for deep Illumina resequencing data.')]))])), 'fgbio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fgbio/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/fulcrumgenomics/fgbio'), ('license', 'MIT'), ('summary', 'A set of tools for working with genomic and high throughput sequencing data, including UMIs')])), ('package', CommentedMap([('name', 'fgbio'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'fgbio-<{ tag }}.tar.gz'), ('url', 'https://github.com/fulcrumgenomics/fgbio/archive/<{ tag }}.tar.gz'), ('md5', 'c770b50025284f4baf9dceb45a7fd826')])), ('requirements', CommentedMap([('build', ['openjdk >=8', 'sbt', 'scala']), ('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', ['fgbio 2>&1 | grep \"CallMolecularConsensusReads\"', 'fgbio --version 2>&1 | grep \"Version\"'])]))])), 'ficus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ficus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ficus'), ('version', '0.3')])), ('source', CommentedMap([('fn', 'ficus-0.3.tar.gz'), ('url', 'https://pypi.python.org/packages/1a/ec/b92090badf7093ab2a329f2be6e7ded0803f7dfe0b8196bc3881a53b19b2/ficus-0.3.tar.gz'), ('md5', '979a9b00e2241d3dbc77687bda1a1859')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'matplotlib', 'nose-capturestderr']), ('run', ['python', 'matplotlib', 'nose-capturestderr'])])), ('test', CommentedMap([('imports', ['ficus'])])), ('about', CommentedMap([('home', 'https://github.com/camillescott/ficus'), ('license', 'BSD'), ('summary', 'Ficus provides a context manager for matplotlib figures.')]))])), 'fido': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fido/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fido'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'fido.tgz'), ('url', 'https://noble.gs.washington.edu/proj/fido/fido.tgz'), ('md5', '6ee89316f492105a6fb06729475c37e0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['Fido'])])), ('about', CommentedMap([('home', 'https://noble.gs.washington.edu/proj/fido/'), ('license', 'MIT license'), ('summary', 'A method for protein identification in MS/MS proteomics. Think of it like a protein delivery dog. You bring it the scored matches between peptides and spectra, and it fetches a list of proteins ranked by posterior probability by doing clever tricks.')]))])), 'filechunkio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/filechunkio/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'filechunkio'), ('version', '1.6')])), ('source', CommentedMap([('fn', 'filechunkio-1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/filechunkio/filechunkio-1.6.tar.gz'), ('md5', 'c168a11ad94cd2ec42a219f0f8869a7b')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['filechunkio'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/fabian/filechunkio'), ('license', 'MIT license'), ('summary', 'FileChunkIO represents a chunk of an OS-level file containing bytes data')]))])), 'filtlong': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/filtlong/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/rrwick/Filtlong/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['filtlong --help'])])), ('about', CommentedMap([('home', 'https://github.com/rrwick/Filtlong'), ('license', 'GPL-3.0'), ('summary', 'Filtlong is a tool for filtering long reads. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter.')]))])), 'find_differential_primers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/find_differential_primers/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/widdowquinn/<{ name|lower }}/archive/9e22b40d09de55c65d6c43cabda89608efde55f9.tar.gz'), ('md5', '7b67965653158c35ede1ca1b1e9b6144')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'bx-python', 'emboss', 'primer3 <=1.1.4', 'prodigal', 'blast']), ('run', ['python', 'biopython', 'bx-python', 'emboss', 'primer3  <=1.1.4', 'prodigal', 'blast'])])), ('test', CommentedMap([('commands', ['find_differential_primers.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/widdowquinn/find_differential_primers'), ('license', 'GPLv3'), ('summary', 'Scripts to aid the design of differential primers for diagnostic PCR.')]))])), 'fisher': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fisher/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fisher'), ('version', '0.1.4')])), ('source', CommentedMap([('fn', 'fisher-0.1.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/fisher/fisher-0.1.4.tar.gz'), ('md5', 'bfc763b7333a1f428e4c447dd8a85968')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['fisher'])])), ('about', CommentedMap([('home', 'http://github.com/brentp/fishers_exact_test'), ('license', 'BSD License'), ('summary', \"Fast Fisher's Exact Test\")]))])), 'fishtaco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fishtaco/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fishtaco'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'FishTaco-1.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/35/8c/369067253747db764f45baa3d86054c86d73871c95a2fd11d67c55aa8716/FishTaco-1.1.1.tar.gz'), ('md5', '984f506d327353f7ef3d5fabfdfd44fa')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.6.1', 'scipy >=0.9', 'scikit-learn >=0.15.2', 'statsmodels >=0.5.0', 'pandas >=0.14', 'musicc >=1.0.1']), ('run', ['python', 'numpy >=1.6.1', 'scipy >=0.9', 'scikit-learn >=0.15.2', 'statsmodels >=0.5.0', 'pandas >=0.14', 'musicc >=1.0.1'])])), ('test', CommentedMap([('imports', ['fishtaco']), ('commands', ['run_fishtaco.py -h'])])), ('about', CommentedMap([('home', 'http://omanor.github.io/fishtaco/'), ('license', 'UNKNOWN'), ('summary', 'FishTaco: a metagenomic computational framework, aiming to identify the taxa that are driving functional shifts in microbiomes.'), ('license_family', 'Other')]))])), 'fitter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fitter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fitter'), ('version', '1.0.4')])), ('source', CommentedMap([('fn', 'fitter-1.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/fitter/fitter-1.0.4.tar.gz'), ('md5', 'f9a5b0dea365750168ad08c94cf7ce10')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib', 'numpy', 'scipy', 'pandas']), ('run', ['python', 'matplotlib', 'numpy', 'scipy', 'pandas'])])), ('test', CommentedMap([('imports', ['fitter'])])), ('about', CommentedMap([('home', ['http://github.com/cokelaer/fitter']), ('license', 'GNU Library or Lesser General Public License (LGPL)'), ('summary', 'A tool to fit data to many distributions and best one(s)')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'flaimapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flaimapper/v2.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flaimapper'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'v2.4.0.tar.gz'), ('url', 'https://github.com/yhoogstrate/flaimapper/archive/v2.4.0.tar.gz'), ('sha256', '3ac2f90894bf3687f27e9e35730c0893ce4e6cf689d940c3c1a55cc2f30cb2f4')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'htseq', 'pysam ==0.11.1']), ('run', ['python', 'numpy', 'pysam ==0.11.1'])])), ('test', CommentedMap([('imports', ['flaimapper']), ('commands', ['flaimapper --help', 'flaimapper --version', \"python -c 'import pysam; pysam.AlignmentFile'\"])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/flaimapper/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'FlaiMapper: Detecting small ncRNA derived fragments in small RNA-Seq data')]))])), 'flash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flash/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'flash'), ('version', '1.2.11')])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/FLASH/'), ('license', 'GPLv3+'), ('summary', 'Merge mates from fragments that are shorter than twice the read length')])), ('source', CommentedMap([('fn', 'FLASH-1.2.11.tar.gz'), ('md5', 'e4d355023a766afaaab2d62f912b605c'), ('url', 'http://downloads.sourceforge.net/project/flashpage/FLASH-1.2.11.tar.gz')])), ('test', CommentedMap([('commands', ['flash -v > /dev/null'])]))])), 'flask-admin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-admin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-admin'), ('version', '1.4.0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'Flask-Admin-1.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/F/Flask-Admin/Flask-Admin-1.4.0.tar.gz'), ('md5', '7b24933924f1de60c7dafc371bcbb6f4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask >=0.7', 'wtforms']), ('run', ['python', 'flask >=0.7', 'wtforms'])])), ('test', CommentedMap([('imports', ['flask_admin'])])), ('about', CommentedMap([('home', 'https://github.com/flask-admin/flask-admin/'), ('license', 'BSD License'), ('summary', 'Simple and extensible admin interface framework for Flask')]))])), 'flask-bootstrap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-bootstrap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-bootstrap'), ('version', '3.3.5.7')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'Flask-Bootstrap-3.3.5.7.tar.gz'), ('url', 'https://pypi.python.org/packages/source/F/Flask-Bootstrap/Flask-Bootstrap-3.3.5.7.tar.gz'), ('md5', '4471ad68dbf71a3c68e00bd0e1301a9f')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask >=0.8', 'dominate', 'visitor']), ('run', ['python', 'flask >=0.8', 'dominate', 'visitor'])])), ('test', CommentedMap([('imports', ['flask_bootstrap'])])), ('about', CommentedMap([('home', 'http://github.com/mbr/flask-bootstrap'), ('license', 'BSD License'), ('summary', 'An extension that includes Bootstrap in your project, without any boilerplate code.')]))])), 'flask-bower': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-bower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-bower'), ('version', '1.2.1')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'Flask-Bower-1.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/F/Flask-Bower/Flask-Bower-1.2.1.tar.gz'), ('md5', '91fa024f89af4adb5c67798efcf44428')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask >=0.9']), ('run', ['python', 'flask >=0.9'])])), ('test', CommentedMap([('imports', ['flask_bower'])])), ('about', CommentedMap([('home', 'http://github.com/lobeck/flask-bower'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'An extension to manage and serve your javascript assets with bower')]))])), 'flask-markdown': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-markdown/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-markdown'), ('version', '0.3')])), ('source', CommentedMap([('fn', 'Flask-Markdown-0.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/F/Flask-Markdown/Flask-Markdown-0.3.tar.gz'), ('md5', '37bf5cf3171adb8ba8d2235cb4ef049f')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'markdown', 'flask']), ('run', ['python', 'markdown', 'flask'])])), ('test', CommentedMap([('imports', ['flaskext.markdown']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://github.com/dcolish/flask-markdown'), ('license', 'BSD License'), ('summary', 'Small extension to make using markdown easy')]))])), 'flask-nav': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-nav/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-nav'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'flask-nav-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/flask-nav/flask-nav-0.5.tar.gz'), ('md5', 'a3b49c2d294acd593464b4f01a299492')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask', 'visitor', 'dominate']), ('run', ['python', 'flask', 'visitor', 'dominate'])])), ('test', CommentedMap([('imports', ['flask_nav'])])), ('about', CommentedMap([('home', 'http://github.com/mbr/flask-nav'), ('license', 'MIT'), ('summary', 'Easily create navigation for Flask applications.')]))])), 'flask-potion': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flask-potion/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flask-potion'), ('version', '0.12.1')])), ('source', CommentedMap([('fn', 'Flask-Potion-0.12.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/F/Flask-Potion/Flask-Potion-0.12.1.tar.gz'), ('md5', '73882c3a3d9ca9286ed468c5b17448d6')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask >=0.10', 'jsonschema >=2.4.0', 'aniso8601 >=0.84', 'blinker >=1.3', 'six >=1.8.0']), ('run', ['python', 'flask >=0.10', 'jsonschema >=2.4.0', 'aniso8601 >=0.84', 'blinker >=1.3', 'six >=1.8.0'])])), ('test', CommentedMap([('imports', ['flask_potion'])])), ('about', CommentedMap([('home', 'http://potion.readthedocs.org/en/latest/'), ('license', 'MIT License'), ('summary', 'Powerful REST API framework for Flask and SQLAlchemy')]))])), 'flock': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flock/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flock'), ('version', '1.0')])), ('source', CommentedMap([('fn', '12b153db8d1e0192c449296d9348d2f33d279293.tar.gz'), ('url', 'https://github.com/cristhomas/immport-test/archive/12b153db8d1e0192c449296d9348d2f33d279293.tar.gz'), ('md5', '0d4821d92847d0f9376249bec7ce8ee8')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['flock1 2>&1| grep \"Incorrect number of input parameters\"'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/immportflock/')]))])), 'flux-simulator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flux-simulator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flux-simulator'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'flux-simulator-1.2.1.tgz'), ('url', 'http://artifactory.sammeth.net/artifactory/barna/barna/barna.simulator/1.2.1/flux-simulator-1.2.1.tgz'), ('md5', '1f071df3a577cb5d7c03755ffc0586eb')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', [\"flux-simulator --help 2>&1 | grep 'Tool specific options' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://sammeth.net/confluence/display/SIM/Home'), ('license', 'BSD'), ('summary', 'Tool for modeling RNA-Seq experiments in silico')]))])), 'footprint': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/footprint/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'footprint'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'footprint-pipeline-1.0.0.tar.gz'), ('url', 'http://bimsbstatic.mdc-berlin.de/ohler/asli/footprint-pipeline-1.0.0.tar.gz'), ('md5', '2ab6d789e1a1ed53bb1337c7b06baf28')])), ('build', CommentedMap([('skip', True), ('number', 2)])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-genomicranges', 'r-segmented', 'r-mixtools', 'r-gtools', 'samtools ==1.1', 'bedtools ==2.17.0', 'perl']), ('run', ['r-base', 'bioconductor-genomicranges', 'r-segmented', 'r-mixtools', 'r-gtools', 'samtools ==1.1', 'bedtools ==2.17.0', 'perl'])])), ('test', CommentedMap([('commands', ['find_footprints.sh 2>&1  | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/Reproducible_footprinting_139/'), ('license', 'GPL (>= 2)'), ('summary', 'This is a pipeline to find transcription factor footprints in ATAC-seq or DNase-seq data.')]))])), 'forked-path': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/forked-path/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'forked-path'), ('version', '0.2.3')])), ('source', CommentedMap([('fn', 'forked-path-0.2.3.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/e1/d5/e8b0475ba29140189b21a7d103086a0d29cc9434a753281d6ae8c8976d82/forked-path-0.2.3.tar.gz'), ('md5', '21a618ba91fb045ece0b426cdd21be2b')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['path'])])), ('about', CommentedMap([('home', 'http://github.com/Singletoned/forked-path'), ('license', 'UNKNOWN'), ('summary', 'An object oriented file path module')]))])), 'fqtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fqtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/alastair-droop/<{ name|lower }}/archive/986e451.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'htslib']), ('run', ['libgcc', 'zlib', 'htslib'])])), ('test', CommentedMap([('commands', ['fqtools -h'])])), ('about', CommentedMap([('home', 'https://github.com/alastair-droop/fqtools'), ('license', 'GPLv3'), ('summary', 'An efficient FASTQ manipulation suite.')]))])), 'fraggenescan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fraggenescan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/fraggenescan/<{ name }}<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['hmm_lib.c.patch', 'util_lib.c.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['FragGeneScan &> /dev/null || [[ \"$?\" == 1 ]]', 'run_FragGeneScan.pl &> /dev/null || [[ \"$?\" == 1 ]]'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/fraggenescan/'), ('license', 'BSD'), ('summary', 'FragGeneScan is an application for finding (fragmented) genes in short reads.'), ('description', 'FragGeneScan is an application for finding (fragmented) genes in short\\nreads. It can also be applied to predict prokaryotic genes in incomplete\\nassemblies or complete genomes.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1093/nar/gkq747')]))])), 'frc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/frc/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '5b3f53e.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://github.com/vezzi/FRC_align/archive/5b3f53e.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'bamtools', 'samtools', 'boost <{CONDA_BOOST}}*'])])), ('test', CommentedMap([('commands', ['FRC --help'])])), ('about', CommentedMap([('home', 'https://github.com/vezzi/FRC_align'), ('license', 'GPLv3'), ('summary', 'Computes FRC from SAM/BAM file and not from afg files')]))])), 'freebayes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/freebayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'freebayes'), ('version', '1.1.0.46')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/freebayes.git'), ('git_tag', '8d2b3a060da473e1f4f89be04edfce5cba63f1d3'), ('patches', ['qcfail.diff'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake', 'zlib 1.2.8', 'bzip2 1.0.*', 'python', 'gcc', 'llvm']), ('run', ['zlib 1.2.8', 'bzip2 1.0.*', 'python', 'libgcc', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['freebayes --version', 'vcfstreamsort -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/freebayes'), ('license', 'MIT'), ('summary', 'Bayesian haplotype-based polymorphism discovery and genotyping')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))])), 'fsnviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fsnviz/0.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fsnviz'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'FsnViz-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/a8/e9/535bde7ffdf5c0678d0b75d0667dc846bbf95c27f6f540eb69b7240b901c/FsnViz-<{ version }}.tar.gz'), ('md5', '20a051a88d1631e76536828b8b69cac4')])), ('build', CommentedMap([('skip', True), ('number', 0), ('entry_points', ['fsnviz=fsnviz.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click >=6.6', 'crimson >=0.3.0', 'jinja2 ==2.9.5', 'circos >=0.69.2']), ('run', ['python', 'click >=6.6', 'crimson >=0.3.0', 'jinja2 ==2.9.5', 'circos >=0.69.2'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 fsnviz --help'])])), ('about', CommentedMap([('home', 'https://github.com/bow/fsnviz'), ('license', 'BSD'), ('summary', 'Tool for plotting gene fusion events detected by various tools using Circos.'), ('license_family', 'BSD')]))])), 'ftputil': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ftputil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ftputil'), ('version', '3.2')])), ('source', CommentedMap([('fn', 'ftputil-3.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/ftputil/ftputil-3.2.tar.gz'), ('md5', 'e714790766b753bfdd712955bf3584f8')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['ftputil'])])), ('about', CommentedMap([('home', 'http://ftputil.sschwarzer.net/'), ('license', 'BSD License'), ('summary', 'High-level FTP client library (virtual file system and more)')]))])), 'fuma': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fuma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fuma'), ('version', '3.0.5')])), ('source', CommentedMap([('fn', 'v3.0.5.tar.gz'), ('url', 'https://github.com/yhoogstrate/fuma/archive/v3.0.5.tar.gz'), ('sha256', '89821fcf388a74b9ebd4e61e3d5b2ceb7bd9d3a52738dacd79f4f75aaef09114')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'htseq']), ('run', ['python', 'numpy', 'htseq', 'setuptools'])])), ('test', CommentedMap([('imports', ['fuma']), ('commands', ['fuma --help', 'fuma --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/fuma/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'FuMa: reporting overlap in RNA-seq detected fusion genes')]))])), 'fusepy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fusepy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fusepy'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'fusepy-2.0.4.tar.gz'), ('url', 'https://pypi.io/packages/source/f/fusepy/fusepy-2.0.4.tar.gz'), ('md5', '2b9ac15c1ee854e402ed2f63039d59dd')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', None), ('about', CommentedMap([('home', 'http://github.com/terencehonles/fusepy'), ('license', 'ISC License (ISCL)'), ('summary', 'Simple ctypes bindings for FUSE')]))])), 'fwdpp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fwdpp/0.5.4/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.github.com/molpopgen/fwdpp'), ('license', 'https://raw.githubusercontent.com/molpopgen/fwdpp/master/LICENSE'), ('summary', 'A C++ template library for forward-time population genetic simulation. '), ('doi', '10.1534/genetics.114.165019')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'libsequence', 'boost >=1.60.0'])])), ('build', CommentedMap([('number', 2)])), ('package', CommentedMap([('name', 'fwdpp'), ('version', '0.5.4')])), ('test', CommentedMap([('commands', ['fwdppConfig --version | grep 0.5.4 > /dev/null'])])), ('source', CommentedMap([('fn', '0.5.4.tar.gz'), ('url', 'http://github.com/molpopgen/fwdpp/archive/0.5.4.tar.gz'), ('md5', '36efa5cf0ff3730b154a2c73ec8f390b')]))])), 'fwdpy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fwdpy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fwdpy'), ('version', '0.0.4pre1')])), ('source', CommentedMap([('fn', 'fwdpy-0.0.4pre1.tar.gz'), ('url', 'https://pypi.python.org/packages/ac/8a/01a4a85d20a924abfc981a57aba9b364d6f7e00a0830c5e654242afc9c39/fwdpy-0.0.4pre1.tar.gz'), ('md5', 'f3377dfd2ec9e84beba2d0edcedb8772')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'numpy >=1.10', 'pandas >=0.18', 'libsequence', 'fwdpp >=0.5.4', 'gsl <{CONDA_GSL}}*']), ('run', ['python', 'numpy >=1.10', 'pandas >=0.18', 'libsequence', 'gsl <{CONDA_GSL}}*', 'libgcc'])])), ('test', CommentedMap([('imports', ['fwdpy'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/fwdpy'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Forward-time population genetic simulation in Python.')]))])), 'fwdpy11': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fwdpy11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fwdpy11'), ('version', '0.1.3.post1')])), ('source', CommentedMap([('fn', 'fwdpy11-0.1.3.post1.tar.gz'), ('url', 'https://pypi.python.org/packages/b9/ed/56afc2a56799d1e8c541463bb8c26d6ce1c811c113db584dd1239b925216/fwdpy11-0.1.3.post1.tar.gz'), ('md5', '0d05a5c871f2651b339c88bf0e16f077')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'py<{CONDA_PY}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'numpy >=1.10', 'pybind11 >=2.2.0', 'gsl <{CONDA_GSL}}*']), ('run', ['python', 'numpy >=1.10', 'libgcc', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('imports', ['fwdpy11'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/fwdpy11'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Forward-time population genetic simulation in Python.')]))])), 'gadem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gadem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gadem'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'gadem_v1.3.1.tar.gz'), ('url', 'https://www.niehs.nih.gov/research/resources/assets/docs/gadem_v131targz.gz'), ('sha1', '694a122e0eab3e16277d508a722b8288c20635aa')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gadem'])])), ('about', CommentedMap([('home', 'https://www.niehs.nih.gov/research/resources/software/biostatistics/gadem/index.cfm'), ('license', 'GPLv3'), ('summary', 'A Genetic Algorithm Guided Formation of Spaced Dyads Coupled with an EM Algorithm for Motif Discovery')]))])), 'galaxy-lib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/galaxy-lib/16.7.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'galaxy-lib'), ('version', '16.7.10')])), ('source', CommentedMap([('fn', 'galaxy-lib-16.7.10.tar.gz'), ('url', 'https://pypi.python.org/packages/5d/36/68594f5b76323133c5849834cf2d781a73b7166296b1456cd52aaa26e2ac/galaxy-lib-16.7.10.tar.gz'), ('md5', '048cdb1ba001fba302f9d0e4e835bd20')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'pyyaml', 'six >=1.9.0', 'markupsafe']), ('run', ['python', 'six >=1.9.0', 'pyyaml', 'markupsafe'])])), ('test', CommentedMap([('imports', ['galaxy', 'galaxy.exceptions', 'galaxy.jobs', 'galaxy.jobs.metrics', 'galaxy.jobs.metrics.collectl', 'galaxy.jobs.metrics.instrumenters', 'galaxy.objectstore', 'galaxy.tools', 'galaxy.tools.cwl', 'galaxy.tools.deps', 'galaxy.tools.deps.resolvers', 'galaxy.tools.linters', 'galaxy.tools.parser', 'galaxy.tools.toolbox', 'galaxy.tools.toolbox.filters', 'galaxy.tools.toolbox.lineages', 'galaxy.tools.verify', 'galaxy.tools.verify.asserts', 'galaxy.util'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/galaxy-lib'), ('license', 'Academic Free License (AFL)'), ('summary', 'Subset of Galaxy (http://galaxyproject.org/) core code base designed to be used a library.'), ('license_family', 'Apache')]))])), 'galaxy-ops': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/galaxy-ops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'galaxy-ops'), ('version', '1.1.0')])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/gops'), ('license', 'Academic Free License version 3.0'), ('summary', 'Galaxy interval operations')])), ('source', CommentedMap([('url', 'https://github.com/galaxyproject/gops/archive/v1.1.0.tar.gz'), ('fn', 'v1.1.0.tar.gz'), ('sha256', '93020461079ff8bc804252a01d651b072d621b11270d9ece081617ff843d32fb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['bx-python', 'python']), ('run', ['bx-python', 'python'])])), ('test', CommentedMap([('imports', ['galaxy.tools.util.galaxyops'])]))])), 'galaxy_sequence_utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/galaxy_sequence_utils/1.0.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'galaxy_sequence_utils'), ('version', '1.0.0')])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/sequence_utils'), ('license', 'Academic Free License version 3.0'), ('summary', 'Sequence Utilities from the Galaxy project')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/package/source/galaxy_sequence_utils/galaxy_sequence_utils-1.0.0-st.tgz'), ('fn', 'galaxy_sequence_utils-1.0.0-st.tgz'), ('md5', 'f3f329671c42cfa40e608b54e84e6090')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['galaxy_utils'])]))])), 'gap2seq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gap2seq/2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gap2seq'), ('version', '2.0')])), ('build', CommentedMap([('number', 6), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'Gap2Seq-2.0.tar.gz'), ('url', 'https://www.cs.helsinki.fi/u/lmsalmel/Gap2Seq/Gap2Seq-2.0.tar.gz'), ('md5', '67af9e2ac4171dbbeff569115705405c'), ('patches', ['0001-use-boost-unordered-map-and-set-in-lieu-of-standard.patch', '0002-renamed-lock-to-Gap2Seq_lock-to-avoid-name-conflicts.patch'])])), ('requirements', CommentedMap([('build', ['gcc >=4.7', 'libgcc', 'boost <{CONDA_BOOST}}*', 'llvmdev ==3.9.1', 'libcxx', 'boost ==1.63.0', 'zlib', 'cmake', 'make']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'libcxx', 'boost ==1.63.0', 'zlib'])])), ('test', CommentedMap([('commands', ['GapMerger 2>&1 | grep \"USAGE for\"  > /dev/null', 'GapCutter 2>&1 | grep \"USAGE for\"  > /dev/null', 'Gap2Seq 2>&1 | grep \"USAGE for\"  > /dev/null', 'Gap2Seq.sh 2>&1 | grep \"Required parameters\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.cs.helsinki.fi/u/lmsalmel/Gap2Seq/'), ('license', 'GPLv3'), ('license_file', 'LICENCE'), ('summary', 'Gap2Seq is a tool for filling gaps between contigs in genome assemblies.')])), ('extra', CommentedMap([('notes', 'Gap2Seq is only tested on Linux x86_64 by its author.'), ('recipe-maintainers', ['notestaff'])]))])), 'garnet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/garnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'garnet'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'GarNet-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/g/garnet/GarNet-<{ version }}.tar.gz'), ('md5', 'f5dbae74daf84f1c92ed1a5e3115aec0')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'pandas', 'statsmodels', 'matplotlib', 'intervaltree', 'jinja2']), ('run', ['python', 'numpy', 'pandas', 'statsmodels', 'matplotlib', 'intervaltree', 'jinja2', 'libgcc'])])), ('test', CommentedMap([('imports', ['GarNet'])])), ('about', CommentedMap([('home', 'https://github.com/fraenkel-lab/GarNet'), ('license', 'MIT'), ('summary', 'UNKNOWN'), ('license_family', 'MIT')]))])), 'gat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gat'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/g/gat/gat-<{ version }}.tar.gz'), ('md5', 'c0271d7235eddfdc912afa1ee4b7d278')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'gcc', 'llvm', 'setuptools >=1.1', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0']), ('run', ['python', 'zlib', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0'])])), ('test', CommentedMap([('imports', ['gat']), ('commands', ['gat-run.py -h'])])), ('about', CommentedMap([('summary', 'Genomic Association Tester'), ('license', 'MIT'), ('home', 'https://github.com/AndreasHeger/gat')]))])), 'gatk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gatk/3.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/gatk/'), ('license', 'https://www.broadinstitute.org/gatk/about/#licensing'), ('summary', 'The full Genome Analysis Toolkit (GATK) framework, license restricted.')])), ('package', CommentedMap([('name', 'gatk'), ('version', '3.5')])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8,<9', 'bzip2'])])), ('extra', CommentedMap([('notes', 'Due to license restrictions, this recipe cannot distribute and install GATK directly. To fully install GATK, you must download a licensed copy of GATK from the Broad Institute (https://www.broadinstitute.org/gatk/download/), install this package, and call \"gatk-register /path/to/GenomeAnalysisTK[-$PKG_VERSION.tar.bz2|.jar]\", which will copy GATK into your conda environment.')]))])), 'gatk-framework': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gatk-framework/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/chapmanb/gatk'), ('license', 'MIT'), ('summary', 'The core MIT-licensed Genome Analysis Toolkit (GATK) framework, free for all uses')])), ('package', CommentedMap([('name', 'gatk-framework'), ('version', '3.6.24')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('source', CommentedMap([('fn', 'gatk-framework-3.6-24.tar.gz'), ('url', 'https://github.com/chapmanb/gatk/releases/download/v3.6-24-framework/gatk-framework-3.6-24.tar.gz'), ('md5', '91f783c78569384ba8eccc747220adb2')])), ('requirements', CommentedMap([('run', ['openjdk >=8,<9'])])), ('test', CommentedMap([('commands', ['gatk-framework --version'])]))])), 'gatk4': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gatk4/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/gatk/'), ('license', 'BSD'), ('summary', 'Genome Analysis Toolkit (GATK4)')])), ('package', CommentedMap([('name', 'gatk4'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'gatk-<{ version }}.zip'), ('url', 'https://github.com/broadinstitute/gatk/releases/download/4.beta.5/gatk-4.beta.5.zip'), ('md5', '928fd86820d9e5aef7ca1b30c9342f81')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=8,<9', 'python'])])), ('test', CommentedMap([('commands', ['gatk-launch -h', 'gatk-launch --list', 'gatk-launch HaplotypeCaller --help'])]))])), 'gawk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gawk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gawk'), ('version', '4.1.3')])), ('source', CommentedMap([('url', 'http://ftpmirror.gnu.org/gawk/gawk-4.1.3.tar.xz'), ('fn', 'gawk-4.1.3.tar.xz'), ('sha1', '76b0acbbdeaa0e58466675c5faf68895eedd8306')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'mpfr', 'readline', 'gmp']), ('run', ['mpfr', 'readline', 'gmp'])])), ('test', CommentedMap([('commands', ['gawk --version', 'gawk --help'])])), ('about', CommentedMap([('home', 'https://www.gnu.org/software/gawk/'), ('license', 'GPLv3'), ('summary', 'The awk utility interprets a special-purpose programming language that makes it possible to handle simple data-reformatting jobs with just a few lines of code.')]))])), 'gb_taxonomy_tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gb_taxonomy_tools/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/spond/gb_taxonomy_tools'), ('summary', 'These are four simple utilities which perform certain manipulations and  visualization tasks on GenBank taxonomic information.')])), ('package', CommentedMap([('name', 'gb_taxonomy_tools'), ('version', '1.0.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake'])])), ('test', CommentedMap([('commands', ['which taxonomy-reader'])])), ('source', CommentedMap([('fn', '1.0.0.tar.gz'), ('url', 'https://github.com/spond/gb_taxonomy_tools/archive/1.0.0.tar.gz')]))])), 'gclib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gclib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gclib'), ('version', '0.0.1')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('source', CommentedMap([('fn', 'gclib-0.0.1.tar.gz'), ('url', 'https://github.com/gpertea/gclib/archive/907a9d6484b92ea0c357baf2da5d85bc6e6b82b1.tar.gz'), ('md5', '13891db103fab3457beb621ab43f8843'), ('patches', ['Makefile.patch', 'Makefile.osx.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gtest', 'test -f ${PREFIX}/lib/libgffc.so', 'test -f ${PREFIX}/lib/libgffc.dylib'])])), ('about', CommentedMap([('home', 'https://github.com/gpertea/gclib'), ('license', 'Artistic License 2.0'), ('summary', 'Genomic C libraries by Geo Pertea')]))])), 'gcs_oauth2_boto_plugin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gcs_oauth2_boto_plugin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gcs-oauth2-boto-plugin'), ('version', '1.9')])), ('source', CommentedMap([('fn', 'gcs-oauth2-boto-plugin-1.9.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/gcs-oauth2-boto-plugin/gcs-oauth2-boto-plugin-1.9.tar.gz'), ('md5', '30ce54681c83a23c93f1692c618f3b55')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cryptography', 'python', 'setuptools', 'boto >=2.29.1', 'httplib2 >=0.8', 'google-api-python-client >=1.1', 'pyopenssl >=0.13', 'socksipy-branch ==1.01', 'retry_decorator >=1.0.0']), ('run', ['python', 'boto >=2.29.1', 'httplib2 >=0.8', 'google-api-python-client >=1.1', 'pyopenssl >=0.13', 'socksipy-branch ==1.01', 'retry_decorator >=1.0.0'])])), ('test', CommentedMap([('imports', ['gcs_oauth2_boto_plugin'])])), ('about', CommentedMap([('home', 'https://developers.google.com/storage/docs/gspythonlibrary'), ('license', 'Apache 2.0'), ('summary', 'Auth plugin allowing use the use of OAuth 2.0 credentials for Google Cloud Storage in the Boto library.')]))])), 'gdal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gdal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gdal'), ('version', '1.9.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'gdal-1.9.2.tar.gz'), ('url', 'ftp://ftp.remotesensing.org/gdal/gdal-1.9.2.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://gdal.org'), ('license', 'Several OSS'), ('license_file', 'LICENSE.TXT'), ('summary', 'Geospatial Data Abstraction Library')]))])), 'gdk-pixbuf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gdk-pixbuf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gdk-pixbuf'), ('version', '2.32.2')])), ('source', CommentedMap([('fn', 'gdk-pixbuf-2.32.2.tar.xz'), ('sha256', 'd3ab06fc123b13effed4c27c77cebdfad2173ff20628d82c397b7660ae926145'), ('url', 'http://ftp.gnome.org/pub/gnome/sources/gdk-pixbuf/2.32/gdk-pixbuf-2.32.2.tar.xz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'm4', 'autoconf', 'automake', 'libtool', 'pkg-config', 'glib >=2.48.0', 'gobject-introspection', 'libffi', 'libtiff', 'libpng', 'perl', 'gettext ==0.19.7']), ('run', ['libgcc', 'glib >=2.48.0', 'gobject-introspection', 'libffi', 'libtiff', 'libpng', 'gettext ==0.19.7'])])), ('test', CommentedMap([('commands', ['gdk-pixbuf-csource --version', 'gdk-pixbuf-query-loaders --help', 'gdk-pixbuf-pixdata --help'])])), ('about', CommentedMap([('home', 'https://developer.gnome.org/gdk-pixbuf/'), ('license', 'GNU'), ('summary', 'The GdkPixbuf Structure — Information that describes an image.')]))])), 'gdsctools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gdsctools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('noarch_python', False), ('entry_points', ['gdsctools_anova=gdsctools.scripts.anova:main', 'gdsctools_regression=gdsctools.scripts.regression:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'easydev >=0.9.34', 'reports >=0.3.1', 'pandas ==0.20.1', 'matplotlib >=1.4.3', 'reports >=0.3.1', 'colorlog', 'scipy ==0.19.1', 'statsmodels ==0.8.0', 'colormap ==1.0.1', 'jinja2', 'biokit ==0.4.1', 'scikit-learn ==0.18.2', 'numexpr', 'xlrd']), ('run', ['python', 'numpy', 'easydev >=0.9.34', 'reports >=0.3.1', 'pandas ==0.20.1', 'matplotlib >=1.4.3', 'reports >=0.3.1', 'colorlog', 'scipy ==0.19.1', 'statsmodels ==0.8.0', 'colormap ==1.0.1', 'jinja2', 'biokit ==0.4.1', 'scikit-learn ==0.18.2', 'numexpr', 'xlrd'])])), ('test', None), ('about', CommentedMap([('home', ['http://pypi.python.org/pypi/gdsctools']), ('license', 'BSD 3-clause'), ('summary', 'Set of tools and pipelines to analyse GDSC data (cancerrxgene.org)'), ('license_family', 'BSD')])), ('extra', CommentedMap([('recipe-maintainers', ['cokelaer'])]))])), 'gecode': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gecode/5.0.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gecode'), ('version', '5.0.0')])), ('about', CommentedMap([('home', 'http://www.gecode.org/'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Generic constraint development environment')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gecode-5.0.0.tar.gz'), ('url', 'http://www.gecode.org/download/gecode-5.0.0.tar.gz'), ('sha256', 'f4ff2fa115fed8c09a049b2d8520363b1f9b1a39d80461f597e29dab2ba9e77b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'mpfr', 'gmp <{CONDA_GMP}}*']), ('run', ['libgcc', 'mpfr', 'gmp <{CONDA_GMP}}*'])])), ('test', CommentedMap([('commands', ['fzn-gecode --help >& /dev/null', 'magic-sequence --help >& /dev/null'])]))])), 'gemini': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gemini/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/arq5x/gemini/archive/v<{ version }}.tar.gz'), ('md5', 'd703e6f4b3b29e389903b94cac9b20a3')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'snappy', 'python-snappy']), ('run', ['python', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'sqlalchemy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'grabix', 'snappy', 'python-snappy'])])), ('test', CommentedMap([('commands', ['gemini -h']), ('imports', ['gemini'])])), ('about', CommentedMap([('home', 'https://github.com/arq5x/gemini'), ('license', 'MIT License'), ('summary', 'a lightweight db framework for disease and population genetics.')]))])), 'genblasta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genblasta/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://genome.sfu.ca/genblast/download.html'), ('license', 'GNU'), ('summary', 'genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'genblasta'), ('version', '1.0.4')])), ('requirements', CommentedMap([('run', ['blast'])])), ('source', CommentedMap([('fn', 'genblast_v1.0.4'), ('url', 'http://genome.sfu.ca/genblast/latest/genblast_v138_mac_os_X')])), ('test', CommentedMap([('commands', ['genblastA'])]))])), 'genblastg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genblastg/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://genome.sfu.ca/genblast/download.html'), ('license', 'GNU'), ('summary', 'genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'genblastg'), ('version', '1.38')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib', 'blast'])])), ('source', CommentedMap([('fn', 'genblast_v138.zip'), ('url', 'http://genome.sfu.ca/genblast/latest/genblast_v139.zip')])), ('test', CommentedMap([('commands', ['genblastG'])]))])), 'genehunter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genehunter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ghm'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'ghm-3.1.tgz'), ('url', 'https://www.helmholtz-muenchen.de/fileadmin/GENEPI/downloads/ghm-3.1.tgz'), ('md5', '0cfdc1aa80e7501f4766fa9537f96897')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['echo \"q\" | ghm'])])), ('about', CommentedMap([('home', 'https://www.helmholtz-muenchen.de/en/ige/service/software-download/genehunter-modscore/index.html'), ('license', 'INDIVIDUAL'), ('summary', 'A MOD-score analysis in which parametric LOD scores are maximized over the parameters of the trait model')]))])), 'geneimpacts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/geneimpacts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'geneimpacts'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'geneimpacts-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/g/geneimpacts/geneimpacts-<{ version }}.tar.gz'), ('md5', '1566cce67bfe51413071e4a6a7baf5b5')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['geneimpacts'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/geneimpacts'), ('summary', 'prioritize effects of variant annotations from VEP, SnpEff, et al.'), ('license', 'MIT')]))])), 'genepop': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genepop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genepop'), ('version', '4.6')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'sources.tar.gz'), ('url', 'http://kimura.univ-montp2.fr/~rousset/sources.tar.gz'), ('sha256', '9a87c9f5401d9de5a9a675e8c4db64ae6f63e83e7924d678c7da12a2210c9475')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://kimura.univ-montp2.fr/~rousset/Genepop.htm'), ('license', 'CeCILL'), ('summary', 'The Genepop population genetics package.')])), ('test', CommentedMap([('commands', ['echo does_not_exist.txt | Genepop | grep \"Genepop version\"'])]))])), 'genesplicer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genesplicer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genesplicer'), ('version', '0_2003.04.03')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'GeneSplicer.tar.gz'), ('url', 'http://www.cs.jhu.edu/~genomics/GeneSplicer/GeneSplicer.tar.gz'), ('md5', 'e490f8632f03b5fe3d42dd5b73f1e3c5')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['genesplicer'])])), ('about', CommentedMap([('home', 'http://www.cs.jhu.edu/~genomics/GeneSplicer'), ('license', 'OSI'), ('summary', 'GeneSplicer : A computational method for splice site prediction')]))])), 'genologics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genologics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genologics'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'genologics-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/b4/04/bd023ff57246cd5724aec7fca289d2da519aa3c96cffac8e3a239f073c1e/genologics-<{ version }}.tar.gz'), ('md5', '1812a2f27873c230d5a9bc86419bba8a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'requests']), ('run', ['python', 'requests'])])), ('test', CommentedMap([('imports', ['genologics'])])), ('about', CommentedMap([('home', 'https://github.com/scilifelab/genologics'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Python interface to the GenoLogics LIMS (Laboratory Information Management System) server via its REST API.'), ('license_family', 'GPL3')]))])), 'genomebaser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomebaser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomebaser'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'GenomeBaser-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/G/GenomeBaser/GenomeBaser-0.1.2.tar.gz'), ('md5', '11bbc12de57eda659ea93f9fcfe92029')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'click']), ('run', ['python', 'biopython', 'click'])])), ('test', CommentedMap([('imports', ['GenomeBaser'])])), ('about', CommentedMap([('home', 'http://github.com/mscook/GenomeBaser'), ('license', 'ECL 2.0'), ('summary', 'GenomeBaser manages complete (bacterial) genomes from NCBI')]))])), 'genomedata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomedata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomedata'), ('version', '1.3.6')])), ('source', CommentedMap([('fn', 'genomedata-1.3.6.tar.gz'), ('url', 'https://pypi.python.org/packages/cd/3a/a466d62a79ccc0926f32c9998f712c14140a0593fc9a487504b202dd18bd/genomedata-1.3.6.tar.gz'), ('md5', '55bcd4d4ca8e31617abad7f11ddb70b6')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['genomedata-info=genomedata._info:main', 'genomedata-query=genomedata._query:main', 'genomedata-histogram=genomedata._histogram:main', 'genomedata-load=genomedata.load_genomedata:main', 'genomedata-load-seq=genomedata._load_seq:main', 'genomedata-load-assembly=genomedata._load_seq:main', 'genomedata-open-data=genomedata._open_data:main', 'genomedata-close-data=genomedata._close_data:main', 'genomedata-report=genomedata._report:main', 'genomedata-erase-data=genomedata._erase_data:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'gcc', 'hdf5', 'hgtools']), ('run', ['python', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'libgcc', 'hdf5', 'hgtools'])])), ('test', CommentedMap([('imports', ['genomedata']), ('commands', ['genomedata-info --help', 'genomedata-query --help', 'genomedata-histogram --help', 'genomedata-load --help', 'genomedata-load-seq --help', 'genomedata-load-assembly --help', 'genomedata-open-data --help', 'genomedata-close-data --help', 'genomedata-report --help', 'genomedata-erase-data --help'])])), ('about', CommentedMap([('home', 'http://genomedata.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'tools for accessing large amounts of genomic data')]))])), 'genomepy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomepy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomepy'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'genomepy-0.3.1.tar.gz'), ('url', 'https://github.com/simonvh/genomepy/releases/download/0.3.1/genomepy-0.3.1.tar.gz'), ('md5', '415f1a86fbbd0b819faaacf27e8a2e88')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'click', 'pyfaidx', 'norns >0.0.1', 'xmltodict', 'bucketcache', 'msgpack-python', 'requests']), ('run', ['python', 'nose', 'click', 'pyfaidx', 'norns >0.0.1', 'xmltodict', 'bucketcache', 'msgpack-python', 'requests'])])), ('test', CommentedMap([('imports', ['genomepy'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/genomepy'), ('license', 'MIT License'), ('summary', 'Easy install of genomes.'), ('license_family', 'MIT')]))])), 'genometools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genometools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genometools'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'genometools-1.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/genometools/genometools-1.2.1.tar.gz'), ('md5', '8ff7f54c101656ac1cde5ea8009925e1')])), ('build', CommentedMap([('skip', True), ('entry_points', ['ensembl_filter_fasta.py = genometools.ensembl.filter_fasta:main', 'ensembl_extract_protein_coding_genes.py = genometools.ensembl.extract_protein_coding_genes:main', 'ensembl_extract_protein_coding_gene_ids.py = genometools.ensembl.extract_protein_coding_gene_ids:main', 'ensembl_extract_protein_coding_exon_annotations.py = genometools.ensembl.extract_protein_coding_exon_annotations:main', 'ncbi_extract_entrez2gene.py = genometools.ncbi.extract_entrez2gene:main', 'geo_generate_sample_sheet.py = genometools.geo.generate_sample_sheet:main', 'sra_find_experiment_runs.py = genometools.sra.find_experiment_runs:main', 'seq_trim_fastq.py = genometools.seq.trim_fastq:main', 'rnaseq_stringtie_gene_level_expression.py = genometools.rnaseq.stringtie_gene_level_expression:main', 'exp_convert_entrez2gene.py = genometools.expression.convert_entrez2gene:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'unicodecsv', 'xmltodict', 'requests']), ('run', ['python', 'unicodecsv', 'xmltodict', 'requests', 'numpy', 'ftputil'])])), ('test', CommentedMap([('imports', ['genometools', 'genometools.basic', 'genometools.ebi', 'genometools.ensembl', 'genometools.expression', 'genometools.geo', 'genometools.misc', 'genometools.ncbi', 'genometools.rnaseq', 'genometools.seq', 'genometools.sra']), ('commands', [\"ensembl_extract_protein_coding_genes.py --help | grep 'usage' > /dev/null\", \"ensembl_extract_protein_coding_gene_ids.py --help | grep 'usage' > /dev/null\", \"ensembl_extract_protein_coding_exon_annotations.py --help | grep 'usage' > /dev/null\", \"ncbi_extract_entrez2gene.py --help | grep 'usage' > /dev/null\", \"geo_generate_sample_sheet.py --help | grep 'usage' > /dev/null\", \"sra_find_experiment_runs.py --help | grep 'usage' > /dev/null\", \"seq_trim_fastq.py --help | grep 'usage' > /dev/null\", \"rnaseq_stringtie_gene_level_expression.py --help | grep 'usage' > /dev/null\", \"exp_convert_entrez2gene.py --help | grep 'usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/flo-compbio/genometools'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'GenomeTools: Scripts and Classes For Working With Genomic Data.')]))])), 'genometools-genometools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genometools-genometools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genometools-genometools'), ('version', '1.5.9')])), ('source', CommentedMap([('fn', 'genometools-1.5.9.tar.gz'), ('url', 'https://github.com/genometools/genometools/archive/v1.5.9.tar.gz'), ('sha1', '733530dfe875cd058da0c3e1b6530b9e085517f4')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'pango']), ('run', ['gettext', 'pango', 'xorg-libxrender', 'xorg-libsm', 'xorg-libxext', 'libgcc'])])), ('test', CommentedMap([('commands', ['gt -version > /dev/null']), ('imports', ['gt'])])), ('about', CommentedMap([('home', 'http://genometools.org/'), ('summary', 'GenomeTools genome analysis system.'), ('license', 'BSD'), ('license_file', 'LICENSE')]))])), 'genomicassertions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomicassertions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomicassertions'), ('version', '0.2.5')])), ('source', CommentedMap([('fn', 'genomicassertions-0.2.5.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/8a/84/6c7342e1099338f435ee2ea2d63e3c5b1fb3bc2f3f0cf51b0d2e2e55d001/genomicassertions-0.2.5.tar.gz'), ('md5', 'b6414decd6d0d024a607b91cdc4aad14')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyvcf', 'pysam']), ('run', ['python', 'pyvcf', 'pysam'])])), ('test', CommentedMap([('imports', ['genomicassertions'])])), ('about', CommentedMap([('home', 'https://github.com/dakl/genomicassertions'), ('license', 'MIT'), ('summary', 'A package to test common files in genomics (.vcf.gz, .bam)')]))])), 'genomicconsensus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomicconsensus/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/PacificBiosciences/GenomicConsensus/archive/smrtanalysis-<{ version }}.tar.gz'), ('md5', '47bd0ecc00a8cdaf61c4662dfb88fcc7')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc', 'llvm', 'h5py >=2.0.1', 'numpy >=1.6.0', 'python-consensuscore >=1.0.1', 'pbcommand >=0.3.20', 'pbcore >=1.2.9']), ('run', ['python', 'libgcc', 'h5py >=2.0.1', 'numpy >=1.6.0', 'python-consensuscore >=1.0.1', 'pbcommand >=0.3.20', 'pbcore >=1.2.9'])])), ('test', CommentedMap([('imports', ['GenomicConsensus.main'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/GenomicConsensus'), ('license', 'Redistribution and use in source and binary forms, with or without modification')]))])), 'genonets': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genonets/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genonets'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'genonets-1.1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c5/fa/7c26ef0b264b1217184dcbc20b5eed3abea107484a20c756197f97ffadf6/genonets-1.1.6.tar.gz'), ('md5', 'a3ad65614c18f8ec436325e9a08e3515')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'python-igraph >=0.6', 'numpy >=1.8.2']), ('run', ['python', 'python-igraph >=0.6', 'numpy >=1.8.2'])])), ('test', CommentedMap([('imports', ['genonets'])])), ('about', CommentedMap([('home', 'https://github.com/fkhalid/genonets'), ('license', 'MIT License'), ('summary', 'Framework for creating and analyzing genotype networks from data.'), ('license_family', 'MIT')]))])), 'geodl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/geodl/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/jduc/geoDL'), ('license', 'GPL'), ('summary', 'Dowload FASTQ files from GEO-NCBI with ease')])), ('package', CommentedMap([('name', 'geodl'), ('version', '1.0b1')])), ('source', CommentedMap([('git_url', 'https://github.com/jduc/geoDL.git'), ('git_rev', '3782d3cc9673e431cc90e53d96ee0f5c7bd4e8e1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'colorama', 'beautifulsoup4']), ('run', ['python', 'colorama', 'beautifulsoup4', 'six'])])), ('test', CommentedMap([('commands', ['geoDL -h'])]))])), 'gevent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gevent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gevent'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'gevent-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/gevent/gevent-1.0.1.tar.gz'), ('md5', '7b952591d1a0174d6eb6ac47bd975ab6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'greenlet >=0.4.9', 'greenlet >=0.4']), ('run', ['python', 'greenlet >=0.4.9', 'greenlet >=0.4'])])), ('test', CommentedMap([('imports', ['gevent'])])), ('about', CommentedMap([('home', 'http://www.gevent.org/'), ('license', 'MIT License'), ('summary', 'Coroutine-based network library')]))])), 'gfapy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gfapy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gfapy'), ('version', '1.0.0rc10')])), ('source', CommentedMap([('fn', 'gfapy-1.0.0rc10.tar.gz'), ('url', 'https://pypi.python.org/packages/8f/d6/02932eb5b49e6e2f8690021de0028684cd797ca9c94cd28d146d72c81449/gfapy-1.0.0rc10.tar.gz'), ('md5', '40c1e1339838106d0d26ed1315919c2f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['gfapy', 'gfapy.alignment', 'gfapy.field', 'gfapy.graph_operations', 'gfapy.line', 'gfapy.line.comment', 'gfapy.line.common', 'gfapy.line.custom_record', 'gfapy.line.edge', 'gfapy.line.edge.common', 'gfapy.line.edge.containment', 'gfapy.line.edge.gfa1', 'gfapy.line.edge.gfa2', 'gfapy.line.edge.link', 'gfapy.line.fragment', 'gfapy.line.gap', 'gfapy.line.group', 'gfapy.line.group.gfa2', 'gfapy.line.group.ordered', 'gfapy.line.group.path', 'gfapy.line.group.unordered', 'gfapy.line.header', 'gfapy.line.segment', 'gfapy.line.unknown', 'gfapy.lines', 'tests']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'https://github.com/ggonnella/gfapy'), ('license', 'ISC License (ISCL)'), ('summary', 'Library for handling data in the GFA1 and GFA2 formats'), ('license_family', 'Other')]))])), 'gff3toembl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gff3toembl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gff3toembl'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/sanger-pathogens/gff3toembl/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['urllib.patch'])])), ('requirements', CommentedMap([('build', ['python', 'genometools-genometools', 'six']), ('run', ['python', 'genometools-genometools', 'six'])])), ('test', CommentedMap([('commands', ['gff3_to_embl --version'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/gff3toembl/'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('summary', 'Submitting annotated genomes to EMBL is a very difficult and time consuming process. This software converts GFF3 files from the most commonly use prokaryote annotation tool Prokka into a format that is suitable for submission to EMBL. It has been used to prepare more than 30% of all annotated genomes in EMBL/GenBank.')]))])), 'gffcompare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffcompare/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'gffcompare-<{ version }}.tar.gz'), ('url', 'https://github.com/gpertea/gffcompare/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}'), ('patches', ['Makefile.patch', 'Makefile.osx.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gclib']), ('run', ['gclib', 'libgcc'])])), ('test', CommentedMap([('commands', ['gffcompare -v'])])), ('about', CommentedMap([('home', 'https://github.com/gpertea/gffcompare'), ('license', 'Artistic License 2.0'), ('summary', 'GffCompare by Geo Pertea')]))])), 'gffread': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffread/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gffread-<{ version }}.OSX_x86_64.tar.gz'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/gffread-<{ version }}.OSX_x86_64.tar.gz'), ('sha256', '65c2bda3107dd3fce524bf61636e06e0db29113611850e0e1f23c19254691c50')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['gffread --version'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/gff.shtml'), ('license', 'Artistic License 2.0'), ('summary', 'GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more.')]))])), 'gffutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gffutils'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gffutils-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/b4/ea/54ca403a8d471849606fb432b9afcf73c8c5105ea2dd87b8d38bd5217c5a/gffutils-<{ version }}.tar.gz'), ('sha256', 'e1b0bf2b422ec3491be13d77bf7baf71d274abd844e8f8b6c1217a95b236218b')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson']), ('run', ['python', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson'])])), ('test', CommentedMap([('imports', ['gffutils']), ('commands', ['gffutils-cli -h'])])), ('about', CommentedMap([('home', 'https://github.com/daler/gffutils'), ('license', 'MIT'), ('summary', 'Work with GFF and GTF files in a flexible database framework')]))])), 'gfinisher': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gfinisher/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://downloads.sourceforge.net/project/gfinisher/GenomeFinisher_<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['gfinisher --help'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/gfinisher/'), ('license', 'Unknown'), ('summary', 'GFinisher is an application tools for refinement and finalization of prokaryotic genomes assemblies using the bias of GC Skew to identify assembly errors and organizes the contigs/scaffolds with genomes references.')]))])), 'gfold': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gfold/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}'), ('skip', True), ('detect_binary_files_with_prefix', True)])), ('package', CommentedMap([('name', 'gfold'), ('version', '1.1.4')])), ('source', CommentedMap([('fn', 'gfold.V1.1.4.tar.gz'), ('url', 'https://bitbucket.org/feeldead/gfold/downloads/gfold.V1.1.4.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'gsl <{ CONDA_GSL }}*']), ('run', ['gsl <{ CONDA_GSL }}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['gfold'])])), ('about', CommentedMap([('home', 'http://compbio.tongji.edu.cn/~fengjx/GFOLD/gfold.html'), ('license', 'MIT'), ('summary', 'Find differentially expressed genes from RNA-seq data with few replicates using generalized fold changes.')]))])), 'ggplot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ggplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ggplot'), ('version', '0.6.8')])), ('source', CommentedMap([('fn', 'ggplot-0.6.8.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/ggplot/ggplot-0.6.8.tar.gz'), ('md5', '047167b6b90b0584c37dfc836dd5e364')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'statsmodels', 'patsy', 'brewer2mpl']), ('run', ['python', 'matplotlib', 'pandas', 'numpy', 'scipy', 'statsmodels', 'patsy', 'brewer2mpl'])])), ('test', CommentedMap([('imports', ['ggplot'])])), ('about', CommentedMap([('home', 'https://github.com/yhat/ggplot/'), ('license', 'BSD'), ('summary', 'An extremely un-pythonic package for doing exactly what ggplot2 does')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'ghmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ghmm/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '<{ name }}-<{ version }}-rc3.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://downloads.sourceforge.net/project/ghmm/ghmm/ghmm%200.9-rc3/<{ name }}-<{ version }}-rc3.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libxml2', 'python', 'swig', 'autoconf', 'automake', 'libtool', 'perl']), ('run', ['libgcc', 'python', 'swig', 'libxml2'])])), ('test', CommentedMap([('imports', ['ghmm', 'ghmmhelper', 'modhmmer', 'class_change', 'ghmmwrapper'])])), ('about', CommentedMap([('home', 'http://ghmm.org/'), ('license', 'GPL2'), ('summary', 'General Hidden Markov Model library (GHMM) is a freely available C library implementing efficient data structures and algorithms for basic and extended HMMs with discrete and continous emissions')]))])), 'ghostscript': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ghostscript/9.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ghostscript'), ('version', 9.1)])), ('source', CommentedMap([('fn', 'ghostscript-9.10.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/ghostscript/GPL%20Ghostscript/9.10/ghostscript-9.10.tar.gz'), ('sha256', '913fc974433238ffd4e0549ce11ba2a3360d1d159cf5c3b988d72a77acb74d04')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gs --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://ghostscript.com/'), ('summary', 'An interpreter for the PostScript language and for PDF.'), ('license', 'Affero GPL'), ('license_file', 'LICENSE')]))])), 'gimmemotifs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gimmemotifs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gimmemotifs'), ('version', '0.10.0')])), ('source', CommentedMap([('url', 'https://github.com/simonvh/gimmemotifs/releases/download/0.10.0/gimmemotifs-0.10.0.tar.gz'), ('fn', 'gimmemotifs-0.10.0.tar.gz'), ('md5', '6e0ab0269db321819457eb5845cf6c6b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'statsmodels', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'gcc', 'llvm', 'htslib <1.4', 'ete3']), ('run', ['libgcc', 'python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'bedtools', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'ucsc-genepredtobed', 'statsmodels', 'r-robustrankaggreg', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'xgboost', 'htslib <1.4', 'pillow', 'ete3'])])), ('test', CommentedMap([('imports', ['gimmemotifs']), ('commands', ['gimme --help'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/gimmemotifs'), ('license', 'MIT'), ('summary', 'Motif prediction pipeline and various motif-related tools')]))])), 'glimmer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glimmer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glimmer'), ('version', 3.02)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/glimmer/index.shtml'), ('license', 'Custom'), ('license_file', 'LICENSE'), ('summary', 'Glimmer is a system for finding genes in microbial DNA')])), ('source', CommentedMap([('fn', 'glimmer302b.tar.gz'), ('url', 'https://ccb.jhu.edu/software/glimmer/glimmer302b.tar.gz'), ('md5', '344d012ae12596de905866fe9eb7f16c')])), ('build', CommentedMap([('number', 2)])), ('test', CommentedMap([('commands', ['glimmer3 | cat'])]))])), 'glimmerHMM': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glimmerHMM/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glimmerhmm'), ('version', '3.0.4')])), ('source', CommentedMap([('md5', 'af3626a5eceb2893ba4e19373dfad1b0'), ('fn', 'GlimmerHMM-3.0.4.tar.gz'), ('url', 'ftp://ccb.jhu.edu/pub/software/glimmerhmm/GlimmerHMM-3.0.4.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['glimmerhmm -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/glimmerhmm/'), ('license', 'Artistic License'), ('summary', 'GlimmerHMM is a gene finder based on a Generalized Hidden Markov Model (GHMM)')]))])), 'glob2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glob2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glob2'), ('version', '0.4.1')])), ('source', CommentedMap([('fn', 'glob2-0.4.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/glob2/glob2-0.4.1.tar.gz'), ('md5', '0a4f0030d202dc0a21d64946f6fd5f5d')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['glob2'])])), ('about', CommentedMap([('home', 'http://github.com/miracle2k/python-glob2/'), ('license', 'BSD License'), ('summary', 'Version of the glob module that can capture patterns and supports recursive wildcards')]))])), 'glpk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glpk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glpk'), ('version', 4.57)])), ('source', CommentedMap([('fn', 'glpk-4.57.tar.gz'), ('url', 'http://ftpmirror.gnu.org/glpk/glpk-4.57.tar.gz'), ('md5', '237531a54f73155842f8defe51aedb0f')])), ('build', CommentedMap([('features', ['vc9', 'vc10', 'vc14']), ('number', 0)])), ('requirements', CommentedMap([('build', ['gmp', 'gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://www.gnu.org/software/glpk/'), ('license', 'GNU General Public License v3'), ('summary', 'GNU Linear Programming Kit')]))])), 'gmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gmap/2017.05.08/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2017.05.08')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2017-05-08.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2017-05-08.tar.gz'), ('sha1', '592797008a0251175e95ce0653bdfa764ee99617')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'bzip2']), ('run', ['libgcc', 'perl', 'zlib', 'bzip2'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 0), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.sse42', 'bin/gmap.nosimd', 'bin/gmapl.sse42', 'bin/gmapl.nosimd', 'bin/gsnap.sse42', 'bin/gsnap.nosimd', 'bin/gsnapl.sse42', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))])), 'gmtk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gmtk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gmtk'), ('version', '1.4.4')])), ('source', CommentedMap([('fn', 'gmtk-1.4.4.tar.gz'), ('url', 'http://melodi.ee.washington.edu/downloads/gmtk/gmtk-1.4.4.tar.gz'), ('md5', 'aba3b8d26f5308ec21c1f2364263d132')])), ('requirements', CommentedMap([('build', ['gcc', 'automake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gmtkPrint -version'])])), ('about', CommentedMap([('home', 'http://melodi.ee.washington.edu/gmtk/'), ('license', 'OSL-3.0'), ('summary', 'A publicly available toolkit for rapidly prototyping statistical models using dynamic graphical models (DGMs) and dynamic Bayesian networks (DBNs)')]))])), 'gnu-getopt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gnu-getopt/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://software.frodo.looijaard.name/getopt/'), ('license', 'GPLv2'), ('summary', 'Command-line option parsing library')])), ('build', CommentedMap([('number', 4), ('skip', True)])), ('package', CommentedMap([('name', 'gnu-getopt'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'getopt-1.1.6.tar.gz'), ('md5', '4eaff9e6d8b2504a2e0d29ff0e9fd956'), ('url', 'http://frodo.looijaard.name/system/files/software/getopt/getopt-1.1.6.tar.gz'), ('patches', ['Makefile.osxpatch'])])), ('requirements', CommentedMap([('build', ['gcc', 'gettext']), ('run', ['libgcc', 'gettext'])])), ('extra', CommentedMap([('notes', \"On Linux systems gnu-getopt is simply called 'getopt', however due to potential collision with the native BSD getopt on OSX, this binary is renamed 'gnu-getopt'.\")])), ('test', CommentedMap([('commands', ['$PREFIX/bin/gnu-getopt -V'])]))])), 'gnu-wget': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gnu-wget/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gnu-wget'), ('version', '1.18')])), ('source', CommentedMap([('fn', 'wget-1.18.tar.xz'), ('url', 'http://ftp.gnu.org/gnu/wget/wget-1.18.tar.xz'), ('md5', 'af9ca95a4bb8ac4a9bf10aeae66fa5ec')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python >=3', 'zlib', 'openssl', 'pkg-config', 'pcre']), ('run', ['libgcc', 'zlib', 'openssl', 'pcre'])])), ('test', CommentedMap([('commands', ['wget --help'])])), ('about', CommentedMap([('home', 'http://www.gnu.org/software/wget/'), ('license', 'GPLv3'), ('summary', 'Retrieve files using HTTP, HTTPS and FTP')]))])), 'gnuplot-py': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gnuplot-py/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gnuplot-py'), ('version', '1.8')])), ('source', CommentedMap([('fn', 'gnuplot-py-1.8.tar.gz'), ('url', 'http://prdownloads.sourceforge.net/gnuplot-py/gnuplot-py-1.8.tar.gz'), ('md5', 'abd6f571e7aec68ae7db90a5217cd5b1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'gnuplot']), ('run', ['python', 'numpy', 'gnuplot'])])), ('test', CommentedMap([('imports', ['Gnuplot'])])), ('about', CommentedMap([('home', 'http://gnuplot-py.sourceforge.net'), ('license', 'LGPL'), ('summary', 'A Python interface to the gnuplot plotting program.'), ('license_family', 'LGPL')]))])), 'goatools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/goatools/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/tanghaibao/goatools'), ('summary', 'Python scripts to find enrichment of GO terms'), ('license', 'BSD')])), ('package', CommentedMap([('name', 'goatools'), ('version', '0.6.10')])), ('source', CommentedMap([('fn', 'goatools-0.6.10.tar.gz'), ('url', 'https://pypi.python.org/packages/fe/65/6a88bcd02bb866801aaba653b14b39f02583addc79fd67fcfef485595b61/goatools-0.6.10.tar.gz'), ('md5', '7ade4270227d1fd7906accdec68bdc46')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'fisher', 'wget', 'pandas', 'numpy', 'scipy', 'nose', 'xlsxwriter', 'statsmodels']), ('run', ['python', 'setuptools', 'fisher', 'wget', 'pandas', 'numpy', 'scipy', 'nose', 'xlsxwriter', 'statsmodels', 'graphviz', 'pygraphviz', 'pyparsing', 'pydot'])])), ('test', CommentedMap([('requires', ['nose']), ('commands', ['find_enrichment.py -h', 'map_to_slim.py -h', 'plot_go_term.py -h']), ('imports', ['goatools', 'goatools.obo_parser', 'goatools.go_enrichment', 'goatools.associations'])]))])), 'goleft': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/goleft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'goleft'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'goleft-v<{ version }}-osx'), ('url', 'https://github.com/brentp/goleft/releases/download/v<{ version }}/goleft_osx'), ('md5', '077103f6cbf20e9b2763a4178cf92fba')])), ('build', CommentedMap([('number', 0), ('skip', False), ('binary_relocation', False)])), ('requirements', CommentedMap([('run', ['samtools'])])), ('test', CommentedMap([('commands', ['goleft depth -h', 'goleft indexcov -h'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/goleft'), ('license', 'MIT'), ('summary', 'goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary')]))])), 'google-api-python-client': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/google-api-python-client/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'google-api-python-client'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'google-api-python-client-1.3.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/google-api-python-client/google-api-python-client-1.4.2.tar.gz'), ('md5', '7033985a645e39d3ccf1b2971ab7b6b8')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'httplib2', 'oauth2client', 'uritemplate'])])), ('test', CommentedMap([('imports', ['apiclient', 'apiclient.discovery'])])), ('about', CommentedMap([('home', 'https://github.com/google/google-api-python-client/'), ('license', 'Apache 2.0')]))])), 'google-cloud-sdk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/google-cloud-sdk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'google-cloud-sdk'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'google-cloud-sdk-<{ version }}-darwin-x86_64.tar.gz'), ('url', 'https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-<{ version }}-darwin-x86_64.tar.gz'), ('sha1', 'd38a38ec3c84859884e0939e1376c758b82122e3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['gcloud version', 'gsutil help'])])), ('about', CommentedMap([('home', 'https://cloud.google.com/sdk/'), ('license', 'Apache v2.0'), ('summary', 'Command-line interface for Google Cloud Platform products and services')]))])), 'google-sparsehash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/google-sparsehash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'google-sparsehash'), ('version', '2.0.3')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'sparsehash-2.0.3.tar.gz'), ('url', 'https://github.com/sparsehash/sparsehash/archive/sparsehash-2.0.3.tar.gz'), ('md5', 'd8d5e2538c1c25577b3f066d7a55e99e')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['sparsetable_unittest 2>&1 | grep PASS > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/sparsehash/sparsehash'), ('license', 'BSD'), ('summary', 'Several hash-map implementations.')]))])), 'gottcha': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gottcha/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gottcha'), ('version', '1.0')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '23a7f0df7b5203a56354ed4926d40c19e7e8f6bd.tar.gz'), ('url', 'https://github.com/LANL-Bioinformatics/GOTTCHA/archive/23a7f0df7b5203a56354ed4926d40c19e7e8f6bd.tar.gz'), ('md5', 'ce904c44f9a0f7a45a81268967e143cd')])), ('requirements', CommentedMap([('build', ['unzip', 'bwa', 'dmd', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-tie-ixhash', 'perl-yaml']), ('run', ['perl-threaded', 'bwa', 'perl-yaml'])])), ('test', CommentedMap([('commands', [\"gottcha.pl --help | grep -q '^VERSION: 1.0b'; echo $?\"])])), ('about', CommentedMap([('home', 'https://github.com/LANL-Bioinformatics/GOTTCHA'), ('license', 'GNU GPL v3'), ('summary', 'Genomic Origin Through Taxonomic CHAllenge (GOTTCHA)')]))])), 'gprofiler-official': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gprofiler-official/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gprofiler-official'), ('version', '0.2.3')])), ('source', CommentedMap([('fn', 'gprofiler-official-0.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/fa/67/92a393d6518fb46a5e5f17133598e25482a9a9f3901c05da1f6a1f3d4fbe/gprofiler-official-0.2.3.tar.gz'), ('md5', '71e98ce80b311ad19af1d4de2fea37db')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['gprofiler'])])), ('about', CommentedMap([('home', 'http://biit.cs.ut.ee/gprofiler'), ('license', 'BSD License'), ('summary', 'Functional enrichment analysis and more via the g:Profiler toolkit'), ('license_family', 'BSD')]))])), 'gqt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gqt/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/ryanlayer/gqt/archive/7f64e8b.tar.gz'), ('fn', '7f64e8b.tar.gz'), ('sha256', '211dbac4de074532ee6908e69dea11b10816e4d8b8a013845bdf43ad9b75036c'), ('patches', ['src-makefile.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'flex', 'sqlite', 'htslib']), ('run', ['libgcc', 'sqlite', 'htslib'])])), ('test', CommentedMap([('commands', ['gqt'])])), ('about', CommentedMap([('home', 'https://github.com/ryanlayer/gqt'), ('license', 'MIT'), ('summary', 'GQT is a genotype query interface.')]))])), 'grabix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/grabix/0.1.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/arq5x/grabix'), ('license', 'MIT'), ('summary', \"grabix leverages the fantastic BGZF library in samtools to provide random access into text files that have been compressed with bgzip. grabix creates it's own index (.gbi) of the bgzipped file. Once indexed, one can extract arbitrary lines from the file with the grab command.  https://github.com/arq5x/grabix \")])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'grabix'), ('version', '0.1.3')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('git_rev', '989b021db5b3b68c501dd295fbd62e2457b061c4'), ('git_url', 'https://github.com/arq5x/grabix.git')])), ('test', CommentedMap([('commands', ['grabix'])]))])), 'graftm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graftm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'graftm'), ('version', '0.10.1')])), ('source', CommentedMap([('fn', 'graftm-0.10.1.tar.gz'), ('url', 'https://pypi.python.org/packages/d3/45/fb464bff4ea40aae267c7f3f877b7d7fda506691d158aa7f60ed1ccc6bd2/graftm-0.10.1.tar.gz'), ('md5', 'bcafb5ca4654617856162065ee1ecd27')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython >=1.64', 'seqmagick >=0.5.0', 'subprocess32 >=3.2.6', 'biom-format >=2.1.4', 'extern >=0.0.4', 'taxtastic >=0.5.4', 'tempdir >=0.6', 'dendropy >=4.1.0', 'nose >=1.0']), ('run', ['python', 'biopython >=1.64', 'seqmagick >=0.5.0', 'subprocess32 >=3.2.6', 'biom-format >=2.1.4', 'extern >=0.0.4', 'taxtastic >=0.5.4', 'tempdir >=0.6', 'dendropy >=4.1.0', 'nose >=1.0'])])), ('test', CommentedMap([('imports', ['graftm'])])), ('about', CommentedMap([('home', 'http://geronimp.github.io/graftM'), ('license', 'GPL3+'), ('summary', 'GraftM is a pipeline used for identifying and classifying marker gene reads from metagenomic datasets'), ('license_family', 'GPL3')]))])), 'graph_embed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graph_embed/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://github.com/fabriziocosta/GraphEmbed'), ('license', 'MIT'), ('summary', 'Compute a 2D embedding of a data matrix given supervised class information.')])), ('package', CommentedMap([('name', 'graph_embed'), ('version', 1.0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3']), ('run', ['python >=2.7,<3', 'numpy', 'scikit-learn', 'docopt', 'networkx', 'matplotlib'])])), ('test', CommentedMap([('commands', ['graph_embed --version'])])), ('source', CommentedMap([('fn', '1.0.tar.gz'), ('url', 'https://github.com/fabriziocosta/GraphEmbed/archive/1.0.tar.gz'), ('sha256', '842fd418309a44dc65b8e802ce0365b4061658cfbdd15444214f02ee56ecdf8a')]))])), 'graphclust-wrappers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphclust-wrappers/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'graphclust-wrappers'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', '0.7.6-galaxy0.3.1.tar.gz'), ('url', 'https://github.com/BackofenLab/GraphClust/archive/0.7.6-galaxy0.3.1.tar.gz'), ('sha256', 'bcdabaf0ad36f088ee9f81476a30b30cafa684359fa106ed7d3d9dca09523493')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl >=5.22.0', 'perl-math-round', 'perl-array-utils']), ('run', ['perl >=5.22.0', 'perl-math-round', 'perl-array-utils'])])), ('test', CommentedMap([('commands', ['mloc2stockholm.pl --h 2>&1 | grep \"USAGE\" > /dev/null', 'fasta2shrep_gspan.pl --h 2>&1 | grep \"Usage\" > /dev/null', 'structure_2_gspan.pl --h 2>&1 | grep \"Usage\" > /dev/null', 'rnaclustScores2Dist.pl --help 2>&1 | grep \"Usage\" > /dev/null', 'alifold.pl  2>&1 | grep \"USAGE\" > /dev/null', 'gc_res.pl -h 2>&1 | grep \"Argument\" > /dev/null', 'scoreAln.pl 2>&1 | grep \"usage\" > /dev/null', \"foldFasta.pl --help 2>&1 | grep 'help' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/GraphClust/'), ('license', 'GPLv3'), ('summary', 'The set of individual perl wrappers extracted from GraphClust pipeline')]))])), 'graphlan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphlan/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/nsegata/graphlan/wiki/Home'), ('summary', 'GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.')])), ('package', CommentedMap([('name', 'graphlan'), ('version', '1.0.0')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('hg_url', 'https://hg@bitbucket.org/nsegata/graphlan'), ('hg_tag', 1.0)])), ('requirements', CommentedMap([('build', ['mercurial']), ('run', ['python 2.7*', 'matplotlib 1.5*', 'biopython 1.66*'])])), ('test', CommentedMap([('commands', ['graphlan.py -h', 'graphlan_annotate.py -h'])]))])), 'graphlite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphlite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'graphlite'), ('version', '1.0.5')])), ('source', CommentedMap([('fn', 'graphlite-1.0.5.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/15/56/cee648eeb1537ab86b7363bacb1f254be0f00c38dd8de113da1091fa6a93/graphlite-1.0.5.tar.gz'), ('md5', '34681e73c5e9595970c33c1156b34d5c')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['graphlite'])])), ('about', CommentedMap([('home', 'https://github.com/eugene-eeo/graphlite'), ('license', 'MIT License'), ('summary', 'embedded graph datastore')]))])), 'graphmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/isovic/<{ name|lower }}/archive/eb8c75d68b03be95464318afa69b645a59f8f6b7.tar.gz'), ('sha256', '<{ hash }}'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gnu-wget', 'gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['graphmap align'])])), ('about', CommentedMap([('home', 'https://github.com/isovic/graphmap'), ('license', 'MIT'), ('license_file', 'LICENCE'), ('summary', 'A highly sensitive and accurate mapper for long, error-prone reads.')]))])), 'graphprot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphprot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'graphprot'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/dmaticzka/GraphProt/archive/v<{ version }}.tar.gz'), ('fn', 'GraphProt-v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'openmp', 'zlib <{CONDA_ZLIB}}*', 'make', 'coreutils', 'gawk', 'pytest', 'scripttest', 'rnashapes <3', 'weblogo >=3', 'libsvm', 'perl-getopt-long', 'r-plyr', 'r-prroc']), ('run', ['libgcc', 'openmp', 'zlib <{CONDA_ZLIB}}*', 'make', 'coreutils', 'gawk', 'rnashapes <3', 'weblogo >=3', 'libsvm', 'perl-getopt-long', 'r-plyr', 'r-prroc', 'curl'])])), ('test', CommentedMap([('commands', ['curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/testclip.train.negatives.fa', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/testclip.train.positives.fa', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/test_data_full_A.train.fa', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/test_data_full_A.train.affys', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/CL_ls.params', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/CL_train.model', 'curl -O https://raw.githubusercontent.com/dmaticzka/GraphProt/v<{ version }}/tests/REG_ls.params', 'GraphProt.pl -mode classification -action ls --onlyseq \\\\ -fasta testclip.train.positives.fa \\\\ -negfasta testclip.train.negatives.fa \\\\', 'GraphProt.pl \\\\ -mode classification -action cv --onlyseq -fasta testclip.train.positives.fa -negfasta testclip.train.negatives.fa -params CL_ls.params', 'GraphProt.pl -mode classification -action train -fasta testclip.train.positives.fa -negfasta testclip.train.negatives.fa -abstraction 1 -R 0 -D 0 -epochs 2 -lambda 0.1 -bitsize 10', 'GraphProt.pl -mode classification -action motif -fasta testclip.train.positives.fa -model CL_train.model -abstraction 1 -R 0 -D 0 -epochs 2 -lambda 0.1 -bitsize 10', 'GraphProt.pl -mode regression -action ls --onlyseq -fasta test_data_full_A.train.fa -affinities test_data_full_A.train.affys -abstraction 1 -R 0 -D 0 -epsilon 0.11 -c 11 -bitsize 10', 'GraphProt.pl -mode regression -action cv --onlyseq -fasta test_data_full_A.train.fa -affinities test_data_full_A.train.affys -params REG_ls.params', 'GraphProt.pl -mode regression -action train -fasta test_data_full_A.train.fa -affinities test_data_full_A.train.affys -abstraction 1 -R 0 -D 0 -epsilon 0.11 -c 11 -bitsize 10'])])), ('about', CommentedMap([('home', 'https://github.com/dmaticzka/graphprot'), ('license', 'MIT'), ('summary', 'GraphProt is a tool for modelling binding preferences of RNA-binding proteins from high-throughput experiments such as CLIP-seq and RNAcompete.')]))])), 'grep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/grep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'grep'), ('version', 2.14)])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'grep_2.14_src_all.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/grep/grep_2.14_src_all.tar.bz2'), ('sha256', 'abff7acabf664fc7c25a29c9ae8b3b5b44a84e95a553810f275019e568f7c6e3')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pcre']), ('run', None)])), ('about', CommentedMap([('home', 'https://www.gnu.org/software/grep/'), ('license', 'GPL'), ('summary', 'Grep searches one or more input files for lines containing a match to a specified pattern')])), ('test', CommentedMap([('commands', ['grep --version', 'echo \"foo\" > foo.txt && grep -P \\'foo\\' foo.txt'])]))])), 'gridss': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gridss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gridss'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gridss-<{ version }}.jar'), ('url', 'https://github.com/PapenfussLab/gridss/releases/download/v<{ version }}/gridss-<{version}}-jar-with-dependencies.jar'), ('md5', '6ccca8c848540e40da45407550e5db5a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', [\"gridss gridss.CallVariants --help 2>&1 | grep 'USAGE: CallVariants'\"])])), ('about', CommentedMap([('home', 'https://github.com/PapenfussLab/gridss'), ('license', 'GPLv3'), ('summary', 'GRIDSS: a Genomic Rearrangement IDentification Software Suite')]))])), 'gromacs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gromacs/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.gromacs.org/'), ('license', 'GNU Lesser General Public License (LGPL)'), ('summary', 'GROMACS is a versatile package to perform molecular dynamics.')])), ('package', CommentedMap([('name', 'gromacs'), ('version', '4.6.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gromacs-4.6.5.tar.gz'), ('url', 'ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.5.tar.gz'), ('md5', '184a537ea52429dbc2a681f98b46ed84')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mdrun -version'])]))])), 'group_humann2_uniref_abundances_to_go': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/group_humann2_uniref_abundances_to_go/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/ASaiM/group_humann2_uniref_abundances_to_GO'), ('summary', 'Group abundances of UniRef50 gene families obtained with HUMAnN2 to Gene Ontology (GO) slim terms with relative abundances'), ('license', 'Apache 2')])), ('package', CommentedMap([('name', 'group_humann2_uniref_abundances_to_go'), ('version', '1.2.0')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'v1.2.0.zip'), ('md5', 'd84001d850a004ae9d43dbf5a54c15b2'), ('url', 'https://github.com/ASaiM/group_humann2_uniref_abundances_to_GO/archive/v1.2.0.zip')])), ('requirements', CommentedMap([('run', ['python', 'humann2', 'goatools', 'wget'])])), ('test', CommentedMap([('commands', ['group_humann2_uniref_abundances_to_GO.sh -h'])]))])), 'gseapy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gseapy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gseapy'), ('version', '0.8.6')])), ('source', CommentedMap([('fn', 'gseapy-0.8.6.tar.gz'), ('url', 'https://pypi.python.org/packages/5d/ad/cb7bff2c3df77b77f42c2a89b66b6f73deeabf9d5ff819f7e3d997957ba1/gseapy-0.8.6.tar.gz'), ('sha256', '740a39b64e41474ec00f2181eeb32eadb6a23da759481a9beb386953d0fbf261')])), ('build', CommentedMap([('entry_points', ['gseapy = gseapy.__main__:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.8.0', 'pandas >=0.16', 'matplotlib >=1.4.3', 'beautifulsoup4 >=4.4.1', 'requests', 'lxml', 'html5lib']), ('run', ['python', 'numpy >=1.8.0', 'pandas >=0.16', 'matplotlib >=1.4.3', 'beautifulsoup4 >=4.4.1', 'requests', 'lxml', 'html5lib'])])), ('test', CommentedMap([('imports', ['gseapy']), ('commands', ['gseapy --help'])])), ('about', CommentedMap([('home', 'https://github.com/BioNinja/gseapy'), ('license', 'MIT License'), ('summary', 'Gene Set Enrichment Analysis in Python'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'gsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gsort'), ('version', '0.0.2')])), ('source', CommentedMap([('fn', 'gsort_linux_amd64'), ('url', 'https://github.com/brentp/gsort/releases/download/v0.0.2/gsort_linux_amd64')])), ('build', CommentedMap([('number', 0)])), ('requirements', None), ('test', CommentedMap([('commands', ['gsort --help'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/gsort'), ('license', 'MIT'), ('summary', 'sort genomic files according to a genomefile')]))])), 'gtf_to_genes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gtf_to_genes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gtf_to_genes'), ('version', '1.40')])), ('source', CommentedMap([('fn', 'gtf_to_genes-1.40.tar.gz'), ('url', 'https://pypi.python.org/packages/35/1d/eb9f5d711ee1b9717f8cfc41eadd0086066a08b99e35da95a0a42a1ae266/gtf_to_genes-1.40.tar.gz'), ('md5', '15dea634078a0fc1367c7adb8c13c280')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['gtf_to_genes'])])), ('about', CommentedMap([('home', 'http://code.google.com/p/gtf-to-genes/'), ('license', 'MIT License'), ('summary', 'Fast GTF parser'), ('license_family', 'MIT')]))])), 'gubbins': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gubbins/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/sanger-pathogens/<{ name|lower }}/archive/863d44f56e827531a02b92e94b46786c95c3e747.tar.gz'), ('md5', '<{ md5 }}'), ('patches', ['gubbins.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libtool', 'autoconf', 'automake', 'perl', 'zlib', 'python', 'setuptools', 'nose >=1.3', 'reportlab >=3.0', 'dendropy  >=4.0.2', 'biopython >=1.59']), ('run', ['libgcc', 'zlib', 'python', 'nose >=1.3', 'reportlab >=3.0', 'dendropy  >=4.0.2', 'biopython >=1.59', 'fasttree', 'raxml'])])), ('test', CommentedMap([('commands', ['run_gubbins.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/gubbins'), ('license', 'GPL-2.0'), ('license_file', 'LICENSE'), ('summary', 'Rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences using Gubbins.')]))])), 'guidescan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/guidescan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'guidescan'), ('version', '1.0')])), ('source', CommentedMap([('git_url', 'https://bitbucket.org/arp2012/guidescan_public.git'), ('git_rev', 'ff0745d')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python ==2.7.13', 'samtools ==1.3.1', 'coreutils', 'rename', 'biopython >=1.66', 'pysam ==0.8.3', 'pyfaidx ==0.4.7.1', 'bx-python ==0.7.3', 'scikit-learn ==0.16.1', 'numpy', 'psutil', 'xlwt', 'pandas', 'openblas']), ('run', ['libgcc', 'python ==2.7.13', 'samtools ==1.3.1', 'coreutils', 'rename', 'biopython >=1.66', 'pysam ==0.8.3', 'pyfaidx ==0.4.7.1', 'bx-python ==0.7.3', 'scikit-learn ==0.16.1', 'numpy', 'psutil', 'xlwt', 'pandas', 'openblas'])])), ('test', CommentedMap([('imports', ['guidescan']), ('commands', ['guidescan_processer -h', 'guidescan_bamdata -h', 'guidescan_guidequery -h', 'guidescan_cutting_efficiency_processer -h', 'guidescan_cutting_specificity_processer -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/arp2012/guidescan_public'), ('license', 'Unknown'), ('summary', 'Tools to create and interface genome-wide CRISPR guideRNA databases')]))])), 'gustaf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gustaf/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'gustaf'), ('version', '1.0.8')])), ('source', CommentedMap([('fn', 'gustaf-1.0.8-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/gustaf/gustaf-1.0.8-Linux-x86_64.tar.xz'), ('md5', 'a873ad2048eb2a74e65aff220bba8132')])), ('test', CommentedMap([('commands', ['gustaf --help 2>&1 > /dev/null', 'gustaf_mate_joining --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/gustaf/README.rst'), ('license', 'https://github.com/seqan/seqan/tree/master/apps/gustaf/LICENSE'), ('summary', 'Gustaf is a tool primarily designed for multi-split mapping of sequencing reads.')]))])), 'gvcf-regions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gvcf-regions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gvcf-regions'), ('version', '2016.06.23')])), ('source', CommentedMap([('fn', 'gvcf_regions-4201320.tar.gz'), ('url', 'https://github.com/lijiayong/gvcf_regions/archive/4201320.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'argparse [py26]'])])), ('test', CommentedMap([('commands', ['gvcf_regions.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/lijiayong/gvcf_regions'), ('license', 'GPLv3'), ('summary', 'Convert a gVCF file in multiple formats into a BED file of callable regions')]))])), 'gvcftools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gvcftools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gvcftools'), ('version', '0.17.0')])), ('source', CommentedMap([('fn', 'gvcftools-0.17.0.tar.gz'), ('url', 'https://github.com/sequencing/gvcftools/releases/download/v0.17.0/gvcftools-0.17.0.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'perl', 'zlib'])])), ('test', CommentedMap([('commands', [\"extract_variants -h 2>&1 | grep 'extract_variants extracts variants from a VCF file'\"])])), ('about', CommentedMap([('home', 'https://sites.google.com/site/gvcftools/home'), ('license', 'MIT'), ('summary', 'a set of utilities to help create and analyze Genome VCF (gVCF) files.')]))])), 'hap.py': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hap.py/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hap.py'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'hap.py-<{ version }}.tar.gz'), ('url', 'https://github.com/Illumina/hap.py/archive/v<{ version }}.tar.gz'), ('md5', '3052070d05dc08964e355c9da57f37ea')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc', 'zlib', 'cython', 'numpy', 'pandas', 'pybedtools', 'pysam', 'nose', 'bx-python', 'cmake']), ('run', ['python', 'libgcc', 'cython', 'zlib', 'numpy', 'pandas', 'pybedtools', 'pysam', 'scipy', 'nose', 'bx-python', 'samtools', 'bcftools'])])), ('test', CommentedMap([('commands', ['hap.py -h', 'som.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/sequencing/hap.py'), ('license', 'BSD'), ('summary', 'Haplotype VCF comparison tools')]))])), 'hapbin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hapbin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hapbin'), ('version', '1.0.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'v1.0.0.tar.gz'), ('url', 'https://github.com/evotools/hapbin/archive/v1.0.0.tar.gz'), ('md5', 'e5990a38b1e3cdd3f2229cf1e9310804')])), ('requirements', CommentedMap([('build', ['cmake >=3.3.1', 'gcc']), ('run', ['gcc'])])), ('about', CommentedMap([('home', 'https://github.com/evotools/hapbin'), ('license', 'GPLv3'), ('summary', 'hapbin is a collection of tools for efficiently calculating Extended Haplotype Homozygosity (EHH), the Integrated Haplotype Score (iHS) and the Cross Population Extended Haplotype Homozogysity (XP-EHH) statistic.')])), ('test', CommentedMap([('commands', ['ihsbin --help | grep \"Voight\"', 'ehhbin --help | grep \"Voight\"', 'xpehhbin --help | grep \"Voight\"'])]))])), 'hapflk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hapflk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hapflk'), ('version', '1.3.0')])), ('source', CommentedMap([('fn', 'hapflk-1.3.0.tar.gz'), ('url', 'https://forge-dga.jouy.inra.fr/attachments/download/5770/hapflk-1.3.0.tar.gz'), ('md5', '842a1d30f0ccdc22cc1573719380fc4b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'gcc', 'llvm']), ('run', ['python', 'numpy', 'scipy', 'libgcc'])])), ('test', CommentedMap([('commands', ['hapflk -h'])])), ('about', CommentedMap([('home', 'https://forge-dga.jouy.inra.fr/projects/hapflk'), ('license', 'GPLv3'), ('license_file', 'LICENSE.txt'), ('summary', 'hapflk is a software implementing the hapFLK and FLK tests for the detection of selection signatures based on multiple population genotyping data.')]))])), 'hardklor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hardklor/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/mhoopmann/hardklor/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['hardklor --help | grep \"University of Washington\"'])])), ('about', CommentedMap([('home', 'https://github.com/mhoopmann/hardklor'), ('license', 'Apache License, Version 2.0'), ('summary', 'Analyze mass spectra')]))])), 'haystack_bio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/haystack_bio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'haystack_bio'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'haystack_bio-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/pinellolab/haystack_bio/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('entry_points', ['haystack_pipeline = haystack.run_pipeline:main', 'haystack_hotspots = haystack.find_hotspots:main', 'haystack_motifs = haystack.find_motifs:main', 'haystack_tf_activity_plane = haystack.generate_tf_activity_plane:main', 'haystack_download_genome = haystack.download_genome:main', 'haystack_run_test = haystack.haystack_common:run_testdata']), ('number', 6)])), ('about', CommentedMap([('home', 'https://github.com/pinellolab/haystack_bio'), ('license', 'GPLv3'), ('summary', 'Epigenetic Variability and Transcription Factor Motifs Analysis Pipeline')])), ('requirements', CommentedMap([('build', ['python', 'openjdk', 'meme 4.11.2', 'bedtools', 'sambamba', 'ucsc-bigwigaverageoverbed', 'ucsc-bedgraphtobigwig', 'bx-python', 'numpy', 'scipy', 'matplotlib', 'jinja2', 'pandas', 'tqdm', 'weblogo']), ('run', ['openjdk', 'meme 4.11.2', 'bedtools', 'sambamba', 'ucsc-bigwigaverageoverbed', 'ucsc-bedgraphtobigwig', 'bx-python', 'numpy', 'scipy', 'matplotlib', 'jinja2', 'pandas', 'tqdm', 'weblogo'])])), ('test', CommentedMap([('commands', ['haystack_hotspots -h'])]))])), 'hclust2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hclust2/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://bitbucket.org/nsegata/hclust2/get/3d589ab.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'libgcc']), ('run', ['python', 'libgcc', 'numpy', 'matplotlib', 'scipy', 'pandas'])])), ('test', CommentedMap([('imports', ['hclust2']), ('commands', ['hclust2.py -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/nsegata/hclust2'), ('license', 'License'), ('summary', 'Hclust2 is a handy tool for plotting heat-maps with several useful options to produce high quality figures that can be used in publication.')]))])), 'hcluster_sg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hcluster_sg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hcluster_sg'), ('version', '0.5.1')])), ('source', CommentedMap([('url', 'https://github.com/douglasgscofield/hcluster/archive/8da348eb73d134110c7876466196dd4dbb9ac54a.tar.gz'), ('md5', 'bdcbc25899be101b421d2e01166876e7')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"hcluster_sg -h 2>&1|grep 'Usage'\"])])), ('about', CommentedMap([('home', 'https://github.com/douglasgscofield/hcluster'), ('license', 'GPLv2'), ('summary', 'A tool for hierarchically clustering on a sparse graph')]))])), 'helperlibs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/helperlibs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'helperlibs'), ('version', '0.1.8')])), ('source', CommentedMap([('fn', 'bioinf-helperlibs-release-0-1-8.tar.gz'), ('url', 'https://github.com/kblin/bioinf-helperlibs/archive/release-0-1-8.tar.gz'), ('md5', '80283b45767e08fb16f03c951732cd0d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'biopython >=1.62']), ('run', ['python', 'biopython >=1.62'])])), ('test', CommentedMap([('imports', ['helperlibs', 'helperlibs.bio', 'helperlibs.tests', 'helperlibs.tests.bio', 'helperlibs.tests.wrappers', 'helperlibs.wrappers'])])), ('about', CommentedMap([('home', 'https://github.com/kblin/bioinf-helperlibs/wiki'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'A collection of bioinformatics-related helper functions'), ('license_family', 'GPL3')]))])), 'hera': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hera/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bioturing/hera/archive/hera-v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}'), ('patches', ['hera.patch', 'hera.makefile.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib', 'jemalloc >=4.5.0', 'htslib', 'hdf5', 'libdivsufsort', 'xz', 'bzip2']), ('run', ['python', 'libgcc', 'zlib', 'jemalloc >=4.5.0', 'htslib', 'hdf5', 'libdivsufsort', 'xz', 'bzip2'])])), ('test', CommentedMap([('commands', [\"hera -h 2>&1 | grep 'Hera'\", 'hera_build -h'])])), ('about', CommentedMap([('home', 'https://github.com/bioturing/hera'), ('license', 'MIT'), ('summary', 'hera is a bioinformatics tool that helps analyze RNA-seq data, providing base-to-base alignment BAM files, transcript abundance estimation, and fusion gene detection.')]))])), 'hgtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hgtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hgtools'), ('version', '6.5.1')])), ('source', CommentedMap([('fn', 'hgtools-6.5.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/26/15/a902b096c5c4cb33269d7b69e779101e00cf77a9081250a947f2b2db41e5/hgtools-6.5.1.tar.gz'), ('md5', 'ce8413687e43d5626cdcfee5024a9bc0')])), ('build', CommentedMap([('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'setuptools_scm']), ('run', ['python', 'backports.unittest_mock'])])), ('test', CommentedMap([('imports', ['hgtools', 'hgtools.managers', 'hgtools.tests']), ('requires', ['backports.unittest_mock', 'pytest >=2.8'])])), ('about', CommentedMap([('home', 'https://github.com/jaraco/hgtools'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Classes and setuptools plugin for Mercurial and Git repositories')]))])), 'hicexplorer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hicexplorer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hicexplorer'), ('version', '1.7.2')])), ('source', CommentedMap([('fn', '1.7.2.tar.gz'), ('url', 'https://github.com/maxplanck-ie/HiCExplorer/archive/1.7.2.tar.gz'), ('sha256', '6c30d21b35a8ed87611c5b9e18776deea961c8194483f1a5f18acf3865e3331c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.10.4', 'scipy >=0.17.1', 'matplotlib >=1.5.3', 'pysam', 'intervaltree >=2.1.0', 'biopython >=1.65', 'pytables >=3.2.2', 'pybigwig >=0.2.8', 'pandas', 'htslib >=1.3,<1.4']), ('run', ['python', 'pybigwig >=0.2.8', 'numpy >=1.10.4', 'scipy >=0.17.1', 'matplotlib >=1.5.3', 'pysam', 'intervaltree >=2.1.0', 'biopython >=1.65', 'pytables >=3.2.2', 'pybigwig >=0.2.8', 'pandas', 'htslib >=1.3,<1.4'])])), ('test', CommentedMap([('imports', ['hicexplorer']), ('commands', ['hicBuildMatrix --version', 'hicCorrelate --version'])])), ('about', CommentedMap([('home', 'https://github.com/maxplanck-ie/HiCExplorer'), ('license', 'GPL3'), ('summary', 'Set of programs to process, analyze and visualize Hi-C data')]))])), 'hichipper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hichipper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hichipper'), ('version', '0.7.0')])), ('source', CommentedMap([('fn', 'hichipper-0.7.0.tar.gz'), ('url', 'https://pypi.python.org/packages/92/f8/a3f97448d434704c4ace551b034e6b441b21f25e4e40cdc6e6956ca35704/hichipper-0.7.0.tar.gz'), ('md5', '522cee6121a499cd92438dfc7715ba36')])), ('build', CommentedMap([('entry_points', ['hichipper = hichipper.cli:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click', 'pyyaml', 'numpy', 'macs2']), ('run', ['python', 'click', 'pyyaml', 'numpy', 'macs2'])])), ('test', CommentedMap([('imports', ['hichipper']), ('commands', ['hichipper --help'])])), ('about', CommentedMap([('home', 'https://github.com/aryeelab/hichipper'), ('license', 'BSD License'), ('summary', 'Processing HiChIP data into loops.'), ('license_family', 'BSD')]))])), 'hicup': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hicup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hicup'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl', 'bowtie2', 'bowtie', 'r-base', 'samtools'])])), ('source', CommentedMap([('fn', 'hicup_v<{ version }}.tar.gz'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/hicup_v<{ version }}.tar.gz'), ('md5', '58702fb9c7f2f4887f4113bf284d1b98'), ('patches', ['get_captured_reads.patch'])])), ('test', CommentedMap([('commands', ['hicup_mapper', 'hicup_digester', 'hicup_truncater', 'hicup_deduplicator', 'hicup_truncater', 'hicup', 'hicup2hicpipe', 'hicup2gothic', 'hicup2fithic', 'hicup2homer', 'get_captured_reads || [[ $? == 255 ]]'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/'), ('license', 'GPLv3'), ('summary', 'A tool for mapping and performing quality control on Hi-C data')]))])), 'hifive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hifive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hifive'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'hifive-1.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/f7/ca/1ea86c773adccf966beeb4c3be0be7c9a6ee10ba8b41d120ae103b3bf953/hifive-1.4.0.tar.gz'), ('md5', 'ae547d4768507758421fec822339fbb9')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'h5py', 'cython', 'setuptools_cython']), ('run', ['python', 'numpy', 'scipy', 'h5py', 'cython', 'setuptools_cython', 'pyx ==0.12.1', 'pysam', 'pil', 'mpi4py'])])), ('test', CommentedMap([('imports', ['hifive', 'hifive.commands', 'hifive.libraries']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'https://github.com/bxlab/hifive'), ('license', 'MIT'), ('summary', 'Python library for normalizing and analyzing HiC and 5C data')]))])), 'hisat2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hisat2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hisat2'), ('version', '2.1.0')])), ('source', CommentedMap([('fn', 'hisat2-2.1.0-OSX_x86_64.zip'), ('url', 'ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads/hisat2-2.1.0-OSX_x86_64.zip'), ('sha256', '7416d5381ffa64914e15f6a930cfb9eb3ab1f5b9894ef8189930d1670384cd81')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/hisat2/index.shtml'), ('license', 'GPLv3'), ('summary', 'graph-based alignment of next generation sequencing reads to a population of genomes')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['libgcc', 'python', 'perl'])])), ('test', CommentedMap([('commands', ['hisat2 --version'])]))])), 'hivtrace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hivtrace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hivtrace'), ('version', '0.1.6')])), ('source', CommentedMap([('git_rev', '0.1.6'), ('git_url', 'https://github.com/veg/hivtrace.git'), ('patches', ['dependency_links.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython >=1.58', 'python-bioext >=0.17.4', 'python-hyphy-python >=0.1.1', 'python-hppy >=0.9.6', 'tornado >=4.3', 'python-hivclustering >=1.2.0']), ('run', ['python', 'biopython >=1.58', 'python-bioext >=0.17.4', 'python-hyphy-python >=0.1.1', 'python-hppy >=0.9.6', 'tornado >=4.3', 'python-hivclustering >=1.2.0'])])), ('test', CommentedMap([('commands', [\"hivtrace -h 2>&1 | grep 'usage' > /dev/null\", \"hivtrace_strip_drams -h 2>&1 | grep 'usage' > /dev/null\", \"hivtrace_viz -h 2>&1 | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/veg/hivtrace'), ('license', 'MIT'), ('summary', 'HIV TRACE is an application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.')]))])), 'hlama': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hlama/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hlama'), ('version', '0.3.1')])), ('build', CommentedMap([('number', 1), ('skip', True), ('entry_points', ['hlama = hlama.app:main'])])), ('source', CommentedMap([('fn', 'hlama-0.3.1.tar.gz'), ('url', 'https://github.com/bihealth/hlama/archive/v0.3.1.tar.gz'), ('md5', 'be667006ccc332751ac6841da0db3e5c')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'snakemake ==3.7.1']), ('run', ['python', 'snakemake ==3.7.1', 'optitype >=1.2'])])), ('test', CommentedMap([('commands', ['hlama --help'])])), ('about', CommentedMap([('home', 'https://github.com/bihealth/hlama'), ('license', 'MIT'), ('summary', 'Simple matching of HTS samples based on HLA typing')]))])), 'hlaprofiler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hlaprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hlaprofiler'), ('version', '1.0.5')])), ('source', CommentedMap([('fn', 'v1.0.5-nodb.tar.gz'), ('url', 'https://github.com/ExpressionAnalysis/HLAProfiler/archive/v1.0.5-nodb.tar.gz'), ('md5', '15d847230d41004f09cae271609f79fe')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['perl-base', 'kraken-ea', 'perl-statistics-basic', 'perl-parallel-forkmanager', 'perl-file-slurp', 'perl-file-compare', 'perl-test-trap', 'perl-class-load']), ('run', ['perl-base', 'kraken-ea', 'perl-statistics-basic', 'perl-parallel-forkmanager', 'perl-file-slurp', 'perl-file-compare', 'perl-test-trap', 'perl-class-load'])])), ('test', CommentedMap([('requires', ['perl-base']), ('commands', ['HLAProfiler.pl test_modules'])])), ('about', CommentedMap([('home', 'https://github.com/ExpressionAnalysis/HLAProfiler'), ('license', 'Custom non-commercial license'), ('summary', 'HLAProfiler uses k-mer profiles to predict HLA types from paired-end RNA-seq data.')]))])), 'hmmer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hmmer/2.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hmmer'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'hmmer-2.3.2.tar.gz'), ('url', 'http://eddylab.org/software/hmmer/2.3.2/hmmer-2.3.2.tar.gz'), ('sha256', 'd20e1779fcdff34ab4e986ea74a6c4ac5c5f01da2993b14e92c94d2f076828b4')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', None)])), ('test', CommentedMap([('commands', ['hmmsearch -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmalign -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmbuild -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmcalibrate -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmconvert -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmemit -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmfetch -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmindex -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmpfam -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null', 'hmmsearch -h 2>&1 | grep \"HMMER 2\\\\.3\\\\.2\" > /dev/null'])])), ('about', CommentedMap([('summary', 'Biosequence analysis using profile hidden Markov models'), ('home', 'http://hmmer.org/'), ('license', 'GPLv2'), ('license_file', 'LICENSE'), ('note', 'See also the \"hmmer2\" package which appends \"2\" to the HMMER\\nbinary tool names, allowing it to be installed in parallel with\\nthe later versions of this \"hmmer\" package.\\n')]))])), 'hmmer2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hmmer2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hmmer2'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'hmmer-2.3.2.tar.gz'), ('url', 'http://eddylab.org/software/hmmer/2.3.2/hmmer-2.3.2.tar.gz'), ('sha256', 'd20e1779fcdff34ab4e986ea74a6c4ac5c5f01da2993b14e92c94d2f076828b4'), ('patches', ['Makefile.in.patch'])])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['hmmsearch2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmalign2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmbuild2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmcalibrate2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmconvert2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmemit2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmfetch2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmindex2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmpfam2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null', 'hmmsearch2 -h 2>&1 | grep \"HMMER 2\\\\.\" > /dev/null'])])), ('about', CommentedMap([('summary', 'Biosequence analysis using profile hidden Markov models'), ('home', 'http://hmmer.org/'), ('license', 'GPLv2'), ('license_file', 'LICENSE'), ('notes', 'This package appends \"2\" to the all HMMER tool binary names,\\nallowing use in parallel with HMMER version 3 onwards.\\n')]))])), 'hmmlearn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hmmlearn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hmmlearn'), ('version', '20151031')])), ('source', CommentedMap([('fn', 'hmmlearn-dd67a47.tar.gz'), ('url', 'https://github.com/hmmlearn/hmmlearn/archive/dd67a47.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'scipy', 'scikit-learn']), ('run', ['python', 'setuptools', 'numpy', 'scipy', 'scikit-learn', 'libgcc'])])), ('test', CommentedMap([('imports', ['hmmlearn.hmm'])])), ('about', CommentedMap([('home', 'https://github.com/hmmlearn/hmmlearn'), ('license', 'BSD'), ('summary', 'Hidden Markov Models in Python, with scikit-learn like API')]))])), 'homer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/homer/meta.yaml', CommentedMap([('about', CommentedMap([('home', ['http://homer.ucsd.edu/homer/index.html']), ('license', 'GPL3'), ('summary', 'Software for motif discovery and next generation sequencing analysis'), ('license_family', 'GPL'), ('maintainer', 'BioNinja, cokelaer, simonvh')])), ('package', CommentedMap([('name', 'homer'), ('version', '4.9.1')])), ('source', CommentedMap([('fn', 'configureHomer.pl'), ('md5', 'e71ff4258c8a9cfcdd419c7078848a0d'), ('url', 'http://homer.salk.edu/homer/configureHomer.pl')])), ('build', CommentedMap([('number', 5)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'perl', 'wget', 'gcc', 'llvm', 'unzip', 'zip']), ('run', ['perl', 'libgcc'])])), ('test', CommentedMap([('requires', ['perl']), ('commands', ['findMotifs.pl -h > /dev/null', 'homer2 > /dev/null'])]))])), 'hs-blastn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hs-blastn/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/chenying2016/queries/archive/dd60e628155af28ccaf73a1f160561db0009df47.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'blast'])])), ('test', CommentedMap([('commands', ['hs-blastn align -h'])])), ('about', CommentedMap([('home', 'https://github.com/chenying2016/queries'), ('license', 'GPL-3.0'), ('summary', 'hs-blastn, a fast and accurate nucleotide-nucleotide sequences aligner.')]))])), 'htsbox': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htsbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'htsbox_<{ version }}.tar.gz'), ('url', 'https://github.com/lh3/htsbox/archive/a9ce49542ae6a092fb93ef17abed082f6ff7c63a.tar.gz'), ('sha256', '<{ sha256hash }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['htsbox 2>&1 | grep htsbox > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/htsbox'), ('license', 'Unknown'), ('summary', 'HTSbox is a fork of early HTSlib. It is a collection of small experimental tools manipulating HTS-related files.')]))])), 'htseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htseq/0.7.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htseq'), ('version', '0.7.2')])), ('source', CommentedMap([('fn', 'HTSeq-0.7.2.tar.gz'), ('url', 'https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz'), ('sha256', 'b9da6d6a9004e8e05a40ecf08f74f765ec0e0be0e835e82205a117aba2ce375b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'pysam >=0.9.0']), ('run', ['python', 'numpy', 'matplotlib', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['HTSeq._HTSeq_internal', 'HTSeq.StepVector', 'HTSeq._version', 'HTSeq.scripts.count', 'HTSeq.scripts.qa']), ('commands', ['htseq-count -h', 'htseq-qa -h'])])), ('about', CommentedMap([('home', 'http://www-huber.embl.de/users/anders/HTSeq/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A framework to process and analyze data from high-throughput sequencing (HTS) assays'), ('license_family', 'LGPL')]))])), 'htslib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htslib'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'htslib-<{ version }}.tar.bz2'), ('url', 'https://github.com/samtools/htslib/releases/download/<{ version }}/htslib-<{ version }}.tar.bz2'), ('md5', 'e7c269d519a0c01de680bbc72fc76a57')])), ('requirements', CommentedMap([('build', ['curl', 'bzip2', 'xz']), ('run', ['curl', 'bzip2', 'xz'])])), ('about', CommentedMap([('home', 'https://github.com/samtools/htslib'), ('license', 'MIT'), ('summary', 'C library for high-throughput sequencing data formats.')]))])), 'httpretty': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/httpretty/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'httpretty'), ('version', '0.8.10')])), ('source', CommentedMap([('fn', 'httpretty-0.8.10.tar.gz'), ('url', 'https://pypi.python.org/packages/source/h/httpretty/httpretty-0.8.10.tar.gz'), ('md5', '9c130b16726cbf85159574ae5761bce7')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt'), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['httpretty']), ('requires', ['coverage >=3.7.1', 'httplib2 >=0.9', 'mock >=1.0.1', 'nose >=1.3.0', 'requests >=2.5.1', 'sure >=1.2.3', 'tornado >=4.0.2', 'urllib3 >1.12'])])), ('about', CommentedMap([('home', 'http://github.com/gabrielfalcao/httpretty'), ('license', 'MIT License'), ('summary', 'HTTP client mock for Python')]))])), 'hubward': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hubward/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hubward'), ('version', '0.2.2')])), ('source', CommentedMap([('fn', 'hubward-0.2.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/h/hubward/hubward-0.2.2.tar.gz'), ('md5', '335a36f6eb5e38bfb5f9d0f518aba2f7')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'argh', 'colorama', 'jsonschema', 'pyyaml', 'pybedtools', 'bleach', 'matplotlib', 'numpy', 'fabric', 'trackhub', 'docutils', 'pycurl', 'pyaml', 'functools32']), ('run', ['python', 'argh', 'colorama', 'jsonschema', 'pyyaml', 'pybedtools', 'bleach', 'matplotlib', 'numpy', 'fabric', 'trackhub', 'docutils', 'pycurl', 'pyaml', 'functools32'])])), ('test', CommentedMap([('imports', ['hubward']), ('commands', ['hubward -h'])])), ('about', CommentedMap([('home', 'https://github.com/daler/hubward'), ('license', 'BSD License'), ('summary', \"Manage the visualization of large amounts of other people's [often messy] genomics data\")]))])), 'hubward-all': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hubward-all/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hubward-all'), ('version', '0.2.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['hubward', 'ucsc-bedtobigbed', 'ucsc-bedgraphtobigwig', 'ucsc-wigtobigwig', 'ucsc-bigbedtobed', 'ucsc-fetchchromsizes', 'ucsc-liftover', 'ucsc-bigwigtobedgraph', 'ucsc-bigwigtowig', 'bedtools', 'crossmap'])])), ('about', CommentedMap([('summary', 'Meta-package for hubward including bedtools and UCSC tools'), ('license', 'The license for this meta-package is MIT; individual tools vary')]))])), 'hurry.filesize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hurry.filesize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hurry.filesize'), ('version', '0.9')])), ('source', CommentedMap([('fn', 'hurry.filesize-0.9.tar.gz'), ('url', 'https://pypi.python.org/packages/source/h/hurry.filesize/hurry.filesize-0.9.tar.gz'), ('md5', '8549ccd09bb31b5ff1e8e8c1eacc7794')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['hurry', 'hurry.filesize'])])), ('about', CommentedMap([('home', None), ('license', 'ZPL 2.1'), ('summary', 'A simple Python library for human readable file sizes (or anything sized in bytes).')]))])), 'ibdne': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ibdne/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ibdne'), ('version', '04Sep15.e78')])), ('source', CommentedMap([('fn', 'ibdne.04Sep15.e78.jar'), ('url', 'http://faculty.washington.edu/browning/ibdne/ibdne.04Sep15.e78.jar'), ('md5', 'fb24ca3f3620fe6b94e84a2de2146603')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=6.0.77'])])), ('test', CommentedMap([('commands', ['ibdne | grep \"Browning\"'])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/ibdne.html'), ('license', 'Apache v2.0'), ('summary', 'The IBDNe program estimates the historical effective population size of a homogenous population (from the output of IDBseq).')]))])), 'ibdseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ibdseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ibdseq'), ('version', 'r1206')])), ('source', CommentedMap([('fn', 'ibdseq.r1206.jar'), ('url', 'http://faculty.washington.edu/browning/ibdseq/ibdseq.r1206.jar'), ('md5', '3963b1c810b2fe71082b0a1d44805cd9')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=6.0.77'])])), ('test', CommentedMap([('commands', ['ibdseq | grep \"[parameters]\"'])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/ibdseq.html'), ('license', 'Apache v2.0'), ('summary', 'IBDseq is a software program for detecting segments of identity-by-descent (IBD) and homozygosity-by-descent (HBD) in unphased genetic sequence data.')]))])), 'icqsol': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/icqsol/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'icqsol'), ('version', '0.3.26')])), ('about', CommentedMap([('home', 'https://github.com/pletzer/icqsol'), ('summary', 'A collection of utilities for constructing complex geometries from primitive shapes'), ('license', 'MIT')])), ('source', CommentedMap([('git_url', 'https://github.com/pletzer/icqsol.git'), ('git_rev', 'c2f9aeb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('test', CommentedMap([('imports', ['csg', 'icqsol', 'numpy', 'triangle', 'vtk'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'numpy', 'python', 'pycsg ==0.3.12', 'pytriangle ==1.0.9', 'vtk ==6.3.0']), ('run', ['numpy', 'python', 'pycsg ==0.3.12', 'pytriangle ==1.0.9', 'vtk ==6.3.0'])]))])), 'idba': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/idba/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/'), ('license', ''), ('summary', 'IDBA is a practical iterative De Bruijn Graph De Novo Assembler for sequence assembly in bioinformatics.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['autoconf', 'automake', 'gcc', 'gawk', 'python', 'perl-threaded']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/loneknightpy/idba/archive/<{ version }}.tar.gz'), ('md5', '303d9b4af7a7498b56ac9698028b4e15')])), ('test', CommentedMap([('commands', ['idba --help 2>&1 | grep \"Iterative de Bruijn Graph Assembler\"'])]))])), 'idr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/idr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'idr'), ('version', '2.0.3')])), ('source', CommentedMap([('fn', '2.0.3.tar.gz'), ('url', 'https://github.com/nboley/idr/archive/2.0.3.tar.gz'), ('md5', '1bec61b526bb1dcf92a5ab99c27852d0')])), ('build', CommentedMap([('number', 4), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'matplotlib', 'scipy']), ('run', ['python', 'numpy', 'matplotlib', 'scipy', 'libgcc'])])), ('test', CommentedMap([('commands', ['idr -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/nboley/idr'), ('summary', 'The IDR (Irreproducible Discovery Rate) framework is a uniﬁed approach to measure the reproducibility of ﬁndings identiﬁed from replicate experiments and provide highly stable thresholds based on reproducibility.'), ('license', 'GPLv2')]))])), 'ig-checkfcs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ig-checkfcs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ig-checkfcs'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ig-checkfcs-release-0317.v1.0.tar.gz'), ('md5', 'a0a477e85cf470627b5e0a92d8e648a2'), ('url', 'https://github.com/ImmPortDB/ig-checkfcs/archive/release-0317.v1.0.tar.gz')])), ('requirements', CommentedMap([('build', ['r >=2.10.0', 'bioconductor-flowcore']), ('run', ['r >=2.10.0', 'bioconductor-flowcore'])])), ('test', CommentedMap([('commands', ['checkFCS.R'])])), ('about', CommentedMap([('home', 'https://github.com/ImmPortDB/ig-checkfcs'), ('license', 'BSD License'), ('summary', 'quick FCS datatype check'), ('license_family', 'BSD')]))])), 'ig-checkflowtypes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ig-checkflowtypes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ig-checkflowtypes'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ig-checkflowtypes.v1.0.0.tar.gz'), ('md5', 'a1af73503d5e5c56e2f9eb7037fc1b2b'), ('url', 'https://github.com/ImmPortDB/ig-checkflowtypes/archive/ig-checkflowtypes.v1.0.0.tar.gz')])), ('requirements', CommentedMap([('build', ['r >=2.10.0', 'bioconductor-flowcore']), ('run', ['r >=2.10.0', 'bioconductor-flowcore'])])), ('test', CommentedMap([('commands', ['checkFCS.R'])])), ('about', CommentedMap([('home', 'https://github.com/ImmPortDB/ig-checkflowtypes'), ('license', 'BSD License'), ('summary', 'quick flow-related datatype for galaxy checks'), ('license_family', 'BSD')]))])), 'ig-flowtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ig-flowtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ig-flowtools'), ('version', '2.0.2')])), ('source', CommentedMap([('fn', 'immport-galaxy-tools.v2.0.2.tar.gz'), ('md5', 'b9726cb00c6dc4b67cfe3961bbdea20c'), ('url', 'https://github.com/ImmPortDB/ig-flowtools/archive/immport-galaxy-tools.v2.0.2.tar.gz')])), ('requirements', CommentedMap([('build', ['python', 'pandas', 'numpy', 'scipy', 'jinja2', 'matplotlib', 'r', 'bioconductor-flowcore', 'bioconductor-flowcl', 'bioconductor-flowai', 'bioconductor-flowdensity', 'bioconductor-flowviz', 'bioconductor-ggcyto', 'bioconductor-flowsom', 'clustergrammer', 'flock']), ('run', ['python', 'pandas', 'numpy', 'scipy', 'jinja2', 'matplotlib', 'r', 'bioconductor-flowcore', 'bioconductor-flowcl', 'bioconductor-flowai', 'bioconductor-flowdensity', 'bioconductor-flowviz', 'bioconductor-ggcyto', 'bioconductor-flowsom', 'clustergrammer', 'flock'])])), ('test', CommentedMap([('commands', ['auto_collapse_pops.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/ImmPortDB/ig-flowtools'), ('license', 'BSD License'), ('summary', 'set of tools for flow cytometry analysis'), ('license_family', 'BSD')]))])), 'igblast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igblast/1.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igblast'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ncbi-igblast-1.4.0-universal-macosx.tar.gz'), ('url', 'ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/1.4.0/ncbi-igblast-1.4.0-universal-macosx.tar.gz')])), ('about', CommentedMap([('home', 'http://www.ncbi.nlm.nih.gov/projects/igblast/'), ('license', 'Public Domain'), ('summary', 'A tool for analyzing immunoglobulin (IG) and T cell receptor (TR) sequences.')])), ('build', CommentedMap([('number', 6)])), ('requirements', CommentedMap([('run', ['perl-threaded >=5.8.8'])])), ('test', CommentedMap([('commands', ['igblastn -h'])]))])), 'igdiscover': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igdiscover/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://igdiscover.se/'), ('license', 'MIT'), ('summary', 'Analyze antibody repertoires and discover new V genes')])), ('package', CommentedMap([('name', 'igdiscover'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igdiscover-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/i/igdiscover/igdiscover-<{ version }}.tar.gz'), ('md5', '8322eb84ac78d91fdcd0f9fc4a6ba8e8')])), ('requirements', CommentedMap([('run', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml', 'muscle', 'pear', 'flash', 'igblast ==1.5.0', 'vsearch ==1.9.7', 'fastqc']), ('build', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'python3 setup.py install')])), ('test', CommentedMap([('commands', ['igdiscover --help > /dev/null'])]))])), 'igv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igv'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'IGV_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/IGV_<{ version }}.zip'), ('sha256', '82a261e41f46d2490f2a490f8706575bd3bc07a4ccf9b38dcf6c6989fad64dae')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/software/igv/home'), ('license', 'MIT'), ('summary', 'Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations')]))])), 'igvtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igvtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igvtools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igvtools_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/igvtools_<{ version }}.zip'), ('sha256', '1f15d226c0773d7a39bb839a9f32043214cd45399c08d34aef89cb69b1f777f0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['igvtools help'])])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/igv/'), ('summary', 'command line tools for IGV'), ('license', 'MIT License'), ('license_family', 'MIT')]))])), 'illumina-interop': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/illumina-interop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'illumina-interop'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'Interop_v<{ version }}.tar.gz'), ('url', 'https://github.com/Illumina/interop/archive/v<{ version }}.tar.gz'), ('md5', 'c8cb7bae77faa8f87c4c0b3a32122171')])), ('requirements', CommentedMap([('build', ['cmake', 'gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['interop_dumptext | grep Version > /dev/null', 'interop_imaging_table | grep Version > /dev/null', 'interop_summary | grep Version > /dev/null', 'interop_plot_by_cycle | grep Version > /dev/null', 'interop_plot_by_lane | grep Version > /dev/null', 'interop_plot_flowcell | grep Version > /dev/null', 'interop_plot_qscore_histogram | grep Version > /dev/null', 'interop_plot_qscore_heatmap | grep Version > /dev/null', 'interop_plot_sample_qc | grep Version > /dev/null', 'interop_index-summary | grep Version > /dev/null', 'interop_dumpbin | grep Version > /dev/null', 'interop_aggregate | grep Version > /dev/null'])])), ('about', CommentedMap([('home', 'http://illumina.github.io/interop/index.html'), ('license', 'GNU General Public License v3.0'), ('summary', 'The Illumina InterOp libraries are a set of common routines used for reading and writing InterOp metric files. These metric files are binary files produced during a run providing detailed statistics about a run. In a few cases, the metric files are produced after a run during secondary analysis (index metrics) or for faster display of a subset of the original data (collapsed quality scores).')]))])), 'illuminate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/illuminate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'illuminate'), ('version', '0.6.3')])), ('source', CommentedMap([('fn', 'illuminate-0.6.3.zip'), ('url', 'https://bitbucket.org/ilmn8/illuminate/get/e19b8afefead.zip'), ('md5', 'cfea74ea8b646884df2396eb7e6d6e9e')])), ('build', CommentedMap([('entry_points', ['illuminate = illuminate.__main__:collect_args'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'gcc']), ('run', ['python', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'libgcc'])])), ('test', CommentedMap([('imports', ['illuminate']), ('commands', ['illuminate --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/invitae/illuminate'), ('license', 'MIT'), ('summary', 'Analytics toolkit for Illumina sequencer metrics.'), ('license_family', 'MIT')]))])), 'impute2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/impute2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'impute2'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'impute_v2.3.2.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_MacOSX_Intel.tgz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['impute2 2>&1 | grep IMPUTE'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.html'), ('license', 'Academic use, commerically restricted (http://www.stats.ox.ac.uk/~marchini/software/gwas/gwas.html#licence)'), ('summary', 'Genotype imputation and haplotype phasing')]))])), 'imseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/imseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'imseq'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'seqan-imseq_<{ version }}-source.tgz'), ('url', 'https://github.com/lkuchenb/imseq/releases/download/v<{ version }}/seqan-imseq_<{ version }}-source.tgz'), ('sha256', 'e6c9ffb29b77feecd158f3079ac55a15ed5187a2445455514a78da7c3ee6e73c')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'cmake']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://www.imtools.org/'), ('license', 'GPLv2'), ('summary', 'IMSEQ is a fast, PCR and sequencing error aware tool to analyze high throughput data from recombined T-cell receptor or immunoglobulin gene sequencing experiments\\n')])), ('test', CommentedMap([('commands', [\"imseq 2>&1 | grep 'SeqAn'\"])]))])), 'indelfixer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/indelfixer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/cbg-ethz/InDelFixer'), ('license', 'GNU General Public License v3.0'), ('summary', 'A sensitive aligner for 454, Illumina and PacBio data, employing a full Smith-Waterman alignment against a reference.')])), ('package', CommentedMap([('name', 'indelfixer'), ('version', '1.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'InDelFixer.jar'), ('url', 'https://github.com/cbg-ethz/InDelFixer/releases/download/v1.1/InDelFixer.jar'), ('sha256', '7e4a84d8dde4f7cd02fe1efd13793b466ee1198c69df231b03884c6fc84f88af')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['InDelFixer'])])), ('extra', CommentedMap([('notes', 'InDelFixer is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"InDelFixer\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"InDelFixer -Xms512m -Xmx1G\"\\n')]))])), 'infernal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/infernal/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'infernal'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'infernal-1.0.2.tar.gz'), ('url', 'ftp://selab.janelia.org/pub/software/infernal/infernal-1.0.2.tar.gz'), ('md5', '9f9b4c0d0d741574bd5b1b553a0ba59a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('test', CommentedMap([('commands', ['cmalign -h > /dev/null', 'cmbuild -h > /dev/null', 'cmcalibrate -h > /dev/null', 'cmemit -h > /dev/null', 'cmscore -h > /dev/null', 'cmsearch -h > /dev/null', 'cmstat -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://infernal.janelia.org/'), ('license', 'GPLv3'), ('license_file', 'COPYRIGHT'), ('summary', 'Infernal (\"INFERence of RNA ALignment\") is for searching DNA sequence databases for RNA structure and sequence similarities.')]))])), 'infinity': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/infinity/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'infinity'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'infinity-1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/i/infinity/infinity-1.3.tar.gz'), ('md5', '0aa8feadec140b7d091bac142411e6d9')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['infinity'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/infinity'), ('license', 'BSD License'), ('summary', 'All-in-one infinity value for Python. Can be compared to any object.')]))])), 'inforna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/inforna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'inforna'), ('version', '2.1.2')])), ('build', CommentedMap([('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/INFO-RNA/archive/1e9bb8236c13f5861f07dc19625c394aaeae4c79.tar.gz'), ('md5', '219ddf683b78c1c314d896c1b367847e')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['INFO-RNA -h', 'INFO-RNA \"(...)\"'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/INFO-RNA'), ('license', 'MIT-like'), ('summary', 'A server for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.')]))])), 'inheritance': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/inheritance/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'inheritance'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'inheritance-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/12/28/c51a03e8e186f6cc5004fa066ff1ad873cf41ab60052ca40bf542ed5dd59/inheritance-0.1.2.tar.gz'), ('md5', '887a67cb749c7d082d676cdb5597f1fc')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['inheritance'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/inheritance'), ('summary', 'inheritance models for mendelian diseases'), ('license', 'MIT')]))])), 'intarna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intarna'), ('version', '2.0.5')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software'), ('license', 'MIT'), ('summary', 'Efficient target prediction incorporating seeding and accessibility of interaction sites')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'intarna-2.0.5.tar.gz'), ('url', 'https://github.com/BackofenLab/IntaRNA/releases/download/v2.0.5/intaRNA-2.0.5.tar.gz'), ('sha256', '64e107b94e0cf732556e96a1ae13b380f1009d36cdae1f31e001cf06c9007295')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'llvmdev ==3.9.1', 'libcxx', 'boost ==1.63.0', 'viennarna >=2.3.0,<3', 'doxygen']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'perl'])])), ('test', CommentedMap([('commands', ['IntaRNA -h', 'IntaRNA -t CCCCCCCCGGGGGGGGGGGGGG -q CCCCCCC'])]))])), 'intemap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intemap/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'InteMAP_v<{ version }}.tar.gz'), ('url', 'http://cqb.pku.edu.cn/ZhuLab/InteMAP/InteMAP_v<{ version }}.tar.gz'), ('md5', '49cc274772b87eb1433b5f2a1d65338c')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'idba', 'wgs-assembler', 'abyss', 'quake', 'jellyfish', 'mummer', 'bowtie2', 'python']), ('run', ['libgcc', 'idba', 'wgs-assembler', 'abyss', 'quake', 'bowtie2', 'python', 'jellyfish', 'mummer'])])), ('test', CommentedMap([('commands', ['runInteMAP.py --help'])])), ('about', CommentedMap([('home', 'http://cqb.pku.edu.cn/ZhuLab/InteMAP/index.html'), ('license', ''), ('summary', 'Integrated metagenomic assembly pipeline for short reads')]))])), 'intervals': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervals/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervals'), ('version', '0.6.0')])), ('source', CommentedMap([('fn', 'intervals-0.6.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/i/intervals/intervals-0.6.0.tar.gz'), ('md5', '633ea6469430f8529afe472a08c4e64a')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'infinity >=0.1.3']), ('run', ['python', 'infinity >=0.1.3'])])), ('test', CommentedMap([('imports', ['intervals'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/intervals'), ('license', 'BSD License'), ('summary', 'Python tools for handling intervals (ranges of comparable objects).')]))])), 'intervalstats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervalstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervalstats'), ('version', 1.01)])), ('source', CommentedMap([('fn', 'IntervalStats.tar.gz'), ('url', 'http://sonorus.princeton.edu/IntervalStats/IntervalStats.tar.gz'), ('md5', 'eb056302e48e1b8042dea4d26a43e885')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['IntervalStats -h'])])), ('about', CommentedMap([('home', 'http://sonorus.princeton.edu/IntervalStats/'), ('license', 'unknown'), ('summary', 'Tool for assessing similarity between sets of intervals')]))])), 'intervaltree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervaltree/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervaltree'), ('version', '2.1.0')])), ('source', CommentedMap([('fn', 'intervaltree-2.1.0.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/ca/c1/450d109b70fa58ca9d77972b02f69222412f9175ccf99fdeaf167be9583c/intervaltree-2.1.0.tar.gz'), ('md5', '33bef3448aaf30b78aa093dc7c315c2c')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sortedcontainers']), ('run', ['python', 'sortedcontainers'])])), ('test', CommentedMap([('imports', ['intervaltree']), ('requires', ['pytest'])])), ('about', CommentedMap([('home', 'https://github.com/chaimleib/intervaltree'), ('license', 'Apache Software License'), ('summary', 'Editable interval tree data structure for Python 2 and 3')]))])), 'intervaltree_bio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervaltree_bio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervaltree_bio'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'intervaltree_bio-1.0.1.zip'), ('url', 'https://pypi.python.org/packages/de/e6/c542057e7054cdb4139162041588d3b20f49dace223e7303a99a240bf6fd/intervaltree_bio-1.0.1.zip'), ('md5', 'aed813f69920e8f53fa8ce678256e226')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'intervaltree']), ('run', ['python', 'intervaltree'])])), ('test', CommentedMap([('imports', ['intervaltree_bio']), ('requires', ['pytest'])])), ('about', CommentedMap([('home', 'https://github.com/konstantint/intervaltree-bio'), ('license', 'MIT License'), ('summary', 'Interval tree convenience classes for genomic data')]))])), 'intervene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervene'), ('version', '0.5.8')])), ('source', CommentedMap([('fn', 'intervene-0.5.8.tar.gz'), ('url', 'https://pypi.python.org/packages/a8/68/3d52c21b3cce52d22207973471fed7c113cfc4fbcaa49718620de14957a3/intervene-0.5.8.tar.gz'), ('md5', '31c74e6da434d6df38743072e24c6cac')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn']), ('run', ['python', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn'])])), ('test', CommentedMap([('imports', ['intervene', 'intervene.example_data', 'intervene.example_data.ENCODE_hESC', 'intervene.example_data.Gene_list', 'intervene.example_data.dbSUPER_mm9', 'intervene.modules', 'intervene.modules.pairwise', 'intervene.modules.upset', 'intervene.modules.venn'])])), ('about', CommentedMap([('home', 'https://github.com/asntech/intervene'), ('license', 'MIT License'), ('summary', 'A tool for intersection and visualization of multiple gene or genomic region sets'), ('license_family', 'MIT')]))])), 'involucro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/involucro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'involucro'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'involucro.darwin'), ('url', 'https://github.com/involucro/involucro/releases/download/v1.1.2/involucro.darwin'), ('sha256', '5c50c79232f34ca439c8dbeb2c9d09998a2e90c8ee42af349a6fdb7ede7752da')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['involucro --help'])])), ('about', CommentedMap([('home', 'https://github.com/involucro/involucro'), ('license', 'Apache 2.0'), ('summary', 'Build and Deliver Software with Containers')]))])), 'ipython-cluster-helper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ipython-cluster-helper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ipython-cluster-helper'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ipython-cluster-helper-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/i/ipython-cluster-helper/ipython-cluster-helper-<{ version }}.tar.gz'), ('md5', '8c56019c6d35ba2624b17827142fe067')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'netifaces']), ('run', ['python', 'ipyparallel >=4.0,<5.0', 'zeromq', 'pyzmq', 'netifaces', 'libgcc'])])), ('test', CommentedMap([('imports', None)])), ('about', CommentedMap([('home', 'https://github.com/roryk/ipython-cluster-helper'), ('license', 'MIT'), ('summary', 'Tool to easily start up an IPython cluster on different schedulers')]))])), 'iqtree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/iqtree/1.5.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Cibiv/IQ-TREE/archive/v<{ version }}.tar.gz'), ('md5', 'ca5e7f5e4cd0723da59c67744415b1cc')])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'eigen', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['iqtree-omp', 'iqtree'])])), ('about', CommentedMap([('home', 'http://www.iqtree.org/'), ('license', 'GPL-2.0'), ('summary', 'Efficient phylogenomic software by maximum likelihood')]))])), 'irep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/irep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'irep'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'iRep-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/i/irep/iRep-<{ version }}.tar.gz'), ('md5', '8be4e4939fad1828232c06570c3a6743')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'lmfit', 'numpy', 'scipy', 'pandas', 'seaborn', 'matplotlib']), ('run', ['python', 'lmfit', 'numpy', 'scipy', 'pandas', 'seaborn', 'matplotlib'])])), ('test', CommentedMap([('imports', ['iRep'])])), ('about', CommentedMap([('home', 'https://github.com/christophertbrown/iRep'), ('license', 'MIT'), ('summary', 'calculate iRep replication rates from metagenome sequencing'), ('license_family', 'MIT')]))])), 'irfinder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/irfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('git_rev', 'v<{ version }}'), ('url', 'https://github.com/williamritchie/IRFinder/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl', 'r-base'])])), ('test', CommentedMap([('commands', ['IRFinder -v'])])), ('about', CommentedMap([('home', 'https://github.com/williamritchie/IRFinder'), ('license', 'MIT'), ('summary', 'Intron Retention Finder')]))])), 'irida-linker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/irida-linker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'irida-linker'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', '1.0.1.tar.gz'), ('url', 'https://github.com/phac-nml/irida-linker/archive/1.0.1.tar.gz'), ('md5', '958689540fb5fde2746af9354438d695')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl >=5.22', 'perl-lwp-simple', 'perl-json', 'perl-getopt-long', 'perl-file-path', 'perl-config-simple', 'perl-termreadkey', 'perl-http-message', 'perl-libwww-perl']), ('run', ['perl >=5.22', 'perl-lwp-simple', 'perl-json', 'perl-getopt-long', 'perl-file-path', 'perl-config-simple', 'perl-termreadkey', 'perl-http-message', 'perl-libwww-perl'])])), ('test', CommentedMap([('commands', ['ngsArchiveLinker.pl'])])), ('about', CommentedMap([('home', None), ('license', 'apache_2.0'), ('license_file', 'LICENSE'), ('summary', 'The NGS Archive Linker is a Perl script used to generate a structure of links for files stored in the IRIDA platform.')]))])), 'itk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/itk/4.6.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'itk'), ('version', '<{ ITK_VERSION }}')])), ('source', CommentedMap([('fn', 'InsightToolkit-<{ ITK_VERSION }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/itk/itk/<{ ITK_VERSION_SHORT }}/InsightToolkit-<{ ITK_VERSION }}.tar.gz'), ('md5', '2c84eae50ab2452cdad32aaadced3c37')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'libtiff', 'libpng', 'jpeg', 'zlib']), ('run', ['python', 'libgcc', 'zlib', 'libtiff', 'libpng', 'jpeg'])])), ('test', CommentedMap([('commands', ['test -f $PREFIX/lib/libITKCommon-<{ ITK_VERSION_SHORT }}.so'])])), ('about', CommentedMap([('home', 'http://www.itk.org/'), ('license', 'Apache 2.0'), ('summary', 'ITK is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis.')]))])), 'itsx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/itsx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'itsx'), ('version', '1.1b')])), ('source', CommentedMap([('url', 'http://microbiology.se/sw/ITSx_1.1b.tar.gz'), ('md5', '32dcfd0383a79caa223e18e0e60c9964')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', ['hmmer >=3.1b2', 'perl'])])), ('test', CommentedMap([('commands', ['ITSx -h'])])), ('about', CommentedMap([('home', 'http://microbiology.se/software/itsx/'), ('license', 'GNU General Public License v3.0'), ('summary', 'ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi.')]))])), 'iva': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/iva/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'iva'), ('version', '1.0.7')])), ('source', CommentedMap([('fn', 'iva-1.0.7.tar.gz'), ('url', 'https://github.com/sanger-pathogens/iva/archive/v1.0.7.tar.gz'), ('md5', '45d4f46d47b3a2cd1a0ed799f266149f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfastaq >=3.10.0', 'networkx >=1.7', 'pysam >=0.8.1', 'kmc', 'smalt', 'mummer', 'samtools']), ('run', ['python', 'pyfastaq >=3.10.0', 'networkx >=1.7', 'pysam >=0.8.1', 'kmc', 'smalt', 'mummer', 'samtools'])])), ('test', CommentedMap([('imports', ['iva']), ('commands', ['iva -h 2>&1 | grep \"SMALT mapping options\"'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/iva'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Iterative Virus Assembler'), ('license_family', 'GPL')]))])), 'jaffa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jaffa/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'jaffa-<{ version }}.tar.gz'), ('url', 'https://github.com/Oshlack/JAFFA/releases/download/version-<{ version }}/JAFFA-version-<{ version }}.tar.gz'), ('md5', '964cc8049db5f4afc644446e7ef8b4a8'), ('patches', ['jaffa.patch'])])), ('build', CommentedMap([('skipe', True), ('number', 0)])), ('requirements', CommentedMap([('run', ['bpipe', 'oases', 'velvet', 'trimmomatic', 'samtools ==1.1', 'bowtie2', 'blat', 'fastx_toolkit', 'bbmap', 'bioconductor-iranges'])])), ('about', CommentedMap([('home', 'https://github.com/Oshlack/JAFFA'), ('summary', 'JAFFA is a multi-step pipeline that takes either raw RNA-Seq reads, or pre-assembled transcripts then searches\\nfor gene fusions.  This package contains the wrappers jaffa-direct, jaffa-assembly, and jaffa-hybrid.\\nYou can pass the \"refSeq\" parameter in the environment variables JAFFA_REF_BASE. Otherwise, pass any paramters\\nto the wrappers as you would to the bpipe JAFFA_{method} call in the manual.\\n'), ('license', 'GPLv3')]))])), 'jamm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jamm/1.0.7.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jamm'), ('version', '1.0.7.2')])), ('source', CommentedMap([('fn', 'v1.0.7.2.zip'), ('url', 'https://github.com/mahmoudibrahim/JAMM/archive/v1.0.7.2.zip'), ('md5', '0902ea81e0559a8804c49dd7b1f02efd'), ('patches', ['setpath.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('run', ['r-base', 'perl-threaded', 'r-signal', 'r-mclust >=4'])])), ('test', CommentedMap([('commands', ['JAMM.sh | grep JAMM > /dev/null', 'SignalGenerator.sh | grep JAMM > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mahmoudibrahim/JAMM'), ('license', 'GPL'), ('license_file', 'LICENSE'), ('summary', 'JAMM is a peak finder for NGS datasets (ChIP-Seq, ATAC-Seq, DNase-Seq..etc.) that can integrate replicates and assign peak boundaries accurately.')]))])), 'jannovar-cli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jannovar-cli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jannovar-cli'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'jannovar-cli-<{ version }}-bin.zip'), ('url', 'https://search.maven.org/remotecontent?filepath=de/charite/compbio/jannovar-cli/<{ version }}/jannovar-cli-<{ version }}-bin.zip'), ('md5', '<{ md5sum }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk >=8'])])), ('test', CommentedMap([('commands', ['jannovar --version'])])), ('about', CommentedMap([('home', 'https://github.com/charite/jannovar'), ('summary', 'Java tool for performing annotation of VCF files'), ('license', 'BSD2')]))])), 'java-jdk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/java-jdk/7.0.91/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'java-jdk'), ('version', '7.0.91')])), ('build', CommentedMap([('number', 1), ('binary_relocation', False)])), ('test', CommentedMap([('commands', ['java -version'])])), ('about', CommentedMap([('home', 'http://www.azulsystems.com/products/zulu'), ('license', 'GPL'), ('summary', 'The Zulu OpenJDK build.')]))])), 'java-lsd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/java-lsd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lsd'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'lsd-2.2.3.tgz'), ('url', 'http://downloads.lappsgrid.org/lsd-2.2.3.tgz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['lsd -version'])])), ('about', CommentedMap([('home', 'http://github.com/lappsgrid-incubator/org.anc.lapps.dsl'), ('license', 'Apache v2.0'), ('summary', 'The LAPPS Grid Services DSL (LSD). Used to invoke LAPPS web services from the command line.')]))])), 'jbrowse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jbrowse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jbrowse'), ('version', '1.12.3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'jbrowse-1.12.3.zip'), ('sha256', 'd97aa89e1862efb8f7be12762528f1e32b0f93d252ee197d753eb724c5730465'), ('url', 'http://jbrowse.org/wordpress/wp-content/plugins/download-monitor/download.php?id=109')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-json', 'perl-clone', 'perl-io-string', 'perl-bioperl', 'perl-test-differences', 'perl-test-deep', 'perl-test-most', 'perl-test-exception', 'perl-test-nowarnings', 'perl-test-simple', 'perl-test-warn', 'perl-exporter-tiny', 'perl-list-moreutils', 'perl-class-data-inheritable', 'perl-devel-stacktrace', 'perl-exception-class', 'perl-sub-uplevel', 'perl-algorithm-diff', 'perl-capture-tiny', 'perl-text-diff', 'perl-data-stag', 'perl-uri', 'perl-scalar-list-utils', 'perl-perlio-gzip', 'perl-bio-gff3', 'perl-devel-size', 'perl-digest-crc32', 'perl-file-next', 'perl-hash-merge', 'perl-heap-simple', 'perl-text-tabs-wrap', 'perl-db-file'])])), ('test', CommentedMap([('commands', ['command -v add-json.pl'])])), ('about', CommentedMap([('home', 'http://jbrowse.org/'), ('license', 'LGPL'), ('summary', 'The JBrowse Genome Browser')]))])), 'je-suite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/je-suite/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://gbcs.embl.de/Je'), ('license', 'MIT License'), ('license_family', 'MIT'), ('summary', 'Je is a suite to handle barcoded fastq files with (or without) Unique Molecule Identifiers (UMIs) and filter\\nread duplicates using these UMIs\\n')])), ('package', CommentedMap([('name', 'je-suite'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://git.embl.de/grp-gbcs/Je/raw/release_<{ version }}/dist/<{ name }}_<{ version }}.tar.gz'), ('md5', '83960fb59a4915fff7cbbfc05f2d9f16')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=7'])])), ('test', CommentedMap([('commands', ['je version'])])), ('extra', CommentedMap([('notes', 'Je is Java program that comes with a wrapper shell script.\\nBy default \"-Xmx4G -Xms256m\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"je -Xms512m -Xmx1g\"')]))])), 'jellyfish': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jellyfish/2.2.3/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'jellyfish'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'v2.2.3.tar.gz'), ('url', 'https://github.com/gmarcais/Jellyfish/archive/v2.2.3.tar.gz'), ('md5', 'a63398ce1cbde0eddcb3493f5b84a9b7')])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf', 'automake', 'libtool', 'pkg-config', 'yaggo >=1.5.8']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['jellyfish --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.genome.umd.edu/jellyfish.html'), ('license', 'GPLv3'), ('summary', 'Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence')]))])), 'jemalloc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jemalloc/3.6.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jemalloc'), ('version', '3.6.0')])), ('build', CommentedMap([('number', 1), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('requires', ['gcc']), ('files', ['test.c']), ('commands', ['test -e ${PREFIX}/lib/libjemalloc.dylib', 'test -e ${PREFIX}/lib/libjemalloc.so', 'gcc -I${PREFIX}/include/ test.c -o test.o'])])), ('source', CommentedMap([('fn', 'jemalloc-3.6.0.tar.bz2'), ('url', 'https://github.com/jemalloc/jemalloc/releases/download/3.6.0/jemalloc-3.6.0.tar.bz2'), ('sha256', 'e16c2159dd3c81ca2dc3b5c9ef0d43e1f2f45b04548f42db12e7c12d7bdf84fe')])), ('about', CommentedMap([('home', 'https://github.com/jemalloc/jemalloc'), ('summary', 'Efficient multithreaded memory allocator'), ('license', 'BSD')]))])), 'joblib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/joblib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'joblib'), ('version', '0.9.3')])), ('source', CommentedMap([('fn', 'joblib-0.9.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/j/joblib/joblib-0.9.3.tar.gz'), ('md5', 'b90652daf3f5458e1d65b9b9d0f74756')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['joblib'])])), ('about', CommentedMap([('home', 'http://packages.python.org/joblib/'), ('license', 'BSD License'), ('summary', 'Lightweight pipelining: using Python functions as pipeline jobs.')]))])), 'jq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jq'), ('version', 1.5)])), ('source', CommentedMap([('fn', 'jq-1.5.tar.gz'), ('url', 'https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz'), ('md5', '0933532b086bd8b6a41c1b162b1731f9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['jq --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://stedolan.github.io/jq'), ('license', 'MIT (code), CC-BY-3.0 (docs)'), ('license_file', 'COPYING'), ('summary', 'jq is a lightweight and flexible command-line JSON processor.')]))])), 'json-collect-data-source': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/json-collect-data-source/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'json_collect_data_source'), ('version', '1.0.1')])), ('about', CommentedMap([('home', 'https://github.com/fabio-cumbo/galaxy-json-collect-data-source'), ('summary', 'This tool is able to receive multiple datasets (optionally with their metadata) in a single query. As an extension of the galaxy-json-data-source tool (https://github.com/mdshw5/galaxy-json-data-source), it allows to handle archives (gz, bz2, tar, and zip) organizing their content in a collection.'), ('license', 'BSD')])), ('source', CommentedMap([('fn', 'package_galaxy_json_collect_data_source_1.0.1.tar.gz'), ('url', 'https://github.com/fabio-cumbo/package_galaxy_json_collect_data_source/archive/1.0.1.tar.gz'), ('md5', '0fcea68e4f77eab22419c524ef584a68')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['python'])])), ('test', CommentedMap([('command', ['json_collect_data_source.py --version'])]))])), 'junit-xml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/junit-xml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'junit-xml'), ('version', '1.7')])), ('source', CommentedMap([('fn', 'junit-xml-1.7.tar.gz'), ('url', 'https://pypi.python.org/packages/9f/96/5c2b6968d9d5b53931618b801392759b331e0dbb8ed9bdb705fab8209b24/junit-xml-1.7.tar.gz'), ('md5', '5e6a96edb8a1592f2832241cfd99983e')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['junit_xml'])])), ('about', CommentedMap([('home', 'https://github.com/kyrus/python-junit-xml'), ('license', 'MIT License'), ('summary', 'Creates JUnit XML test result documents that can be read by tools such as Jenkins'), ('license_family', 'MIT')]))])), 'kaiju': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kaiju/1.4.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kaiju'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('url', 'https://github.com/bioinformatics-centre/kaiju/archive/c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('md5', 'dea77fa5856cb9261f523536eb917e68')])), ('build', CommentedMap([('number', 0), ('no_link', ['bin/makeDB.sh'])])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'gnu-wget >=1.16'])])), ('test', CommentedMap([('commands', [\"kaiju 2>&1 | head -n 1 | grep -q '^Error:'; echo $?\"])])), ('about', CommentedMap([('home', 'http://kaiju.binf.ku.dk/'), ('license', 'GNU GPL v3'), ('summary', 'Fast and sensitive taxonomic classification for metagenomics'), ('license_file', 'LICENSE')]))])), 'kallisto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kallisto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kallisto'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'kallisto-<{ version }}.tar.gz'), ('url', 'https://github.com/pachterlab/kallisto/archive/v<{ version }}.tar.gz'), ('sha256', '2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e')])), ('build', CommentedMap([('number', 0), ('string', 'hdf5<{CONDA_HDF5}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['hdf5 <{CONDA_HDF5}}', 'cmake', 'zlib', 'gcc']), ('run', ['hdf5 <{CONDA_HDF5}}', 'libgcc'])])), ('test', CommentedMap([('files', ['t.fa', 't.fq']), ('commands', ['kallisto version'])])), ('about', CommentedMap([('home', 'http://pachterlab.github.io/kallisto'), ('license', 'BSD_2_Clause'), ('summary', 'Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.')]))])), 'kaptive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kaptive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kaptive'), ('version', '0.3')])), ('source', CommentedMap([('url', 'https://github.com/katholt/Kaptive/archive/0.3.tar.gz'), ('md5', 'c68074382cc8eb0c86c914f66405a546')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['biopython ==1.68', 'blast ==2.2.31']), ('run', ['blast ==2.2.31', 'biopython ==1.68', 'python'])])), ('test', CommentedMap([('commands', ['kaptive.py  -h'])])), ('about', CommentedMap([('home', 'https://github.com/katholt/Kaptive'), ('license', 'GPLv3.0'), ('summary', 'Kaptive reports information about capsular (K) loci found in genome assemblies')]))])), 'khmer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/khmer/2.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://khmer.readthedocs.org/'), ('license', 'BSD-3-Clause'), ('summary', 'khmer is a library and suite of command line tools for working with DNA sequence. It is primarily aimed at short-read sequencing data such as that produced by the Illumina platform. khmer takes a k-mer-centric approach to sequence analysis, hence the name.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'khmer'), ('version', '2.0')])), ('requirements', CommentedMap([('build', ['python', 'bz2file ==0.98', 'screed ==0.9', 'gcc', 'llvm']), ('run', ['python', 'bz2file ==0.98', 'screed ==0.9', 'libgcc'])])), ('source', CommentedMap([('fn', 'khmer-2.0.tar.gz'), ('sha256', '6a8c6809b50adaf2d7d7307eefcca173f4b536329562530400fb1c88a12e6071'), ('url', 'https://pypi.python.org/packages/source/k/khmer/khmer-2.0.tar.gz')])), ('test', CommentedMap([('imports', ['khmer', 'khmer.kfile', 'khmer.khmer_args', 'khmer.utils'])]))])), 'kid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kid'), ('version', '0.9.6')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/source/k/kid/kid-0.9.6.tar.gz'), ('fn', 'kid-0.9.6.tar.gz'), ('md5', '4233e592eee906039ad3dcbedbf6e972')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['kid'])])), ('about', CommentedMap([('home', 'https://pypi.python.org/pypi/kid'), ('license', 'MIT'), ('summary', 'A simple and pythonic XML template language')]))])), 'kissplice': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kissplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kissplice'), ('version', '2.4.0p1')])), ('source', CommentedMap([('fn', 'kissplice-2.4.0-p1.tar.gz'), ('url', 'ftp://pbil.univ-lyon1.fr/pub/logiciel/kissplice/download/kissplice-2.4.0-p1.tar.gz'), ('md5', '36ef9e22a0607f0ab4516cc0d4cf9fea')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*', 'python']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'python'])])), ('test', CommentedMap([('commands', ['kissplice -h'])])), ('about', CommentedMap([('home', 'http://kissplice.prabi.fr'), ('license', 'CeCILL license'), ('summary', 'A local transcriptome assembler for SNPs, indels and AS events')]))])), 'kiwidist': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kiwidist/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kiwidist'), ('version', '0.3.6')])), ('source', CommentedMap([('fn', 'KiwiDist-0.3.6.tar.gz'), ('url', 'https://pypi.python.org/packages/53/79/6a1a0511fbfa338eea24b4d5386c522ea285f8fe9aea7c16bc16495edf58/KiwiDist-0.3.6.tar.gz'), ('md5', '28cf18a12ee088a04f18a604f0ceccb4')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'matplotlib >=1.3.1,<=1.4.3', 'mygene >=2.1.0', 'networkx >=1.8.1', 'numpy >=1.8.0', 'pandas >=0.13.1', 'scipy >=0.13.3,<=0.16.0', 'six >=1.5']), ('run', ['python', 'matplotlib >=1.3.1,<=1.4.3', 'mygene >=2.1.0', 'networkx >=1.8.1', 'numpy >=1.8.0', 'pandas >=0.13.1', 'scipy >=0.13.3,<=0.16.0', 'six >=1.5'])])), ('test', CommentedMap([('imports', ['kiwi'])])), ('about', CommentedMap([('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Combining gene-set analysis with network properties')]))])), 'kmc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kmc/meta.yaml', CommentedMap([('build', CommentedMap([('number', 3), ('string', 'noasmlib_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'kmc'), ('version', '2.3.0')])), ('source', CommentedMap([('fn', 'kmc-2.3.0.tar.gz'), ('url', 'https://github.com/refresh-bio/KMC/archive/2.3.tar.gz'), ('md5', '7854235469948b77e54f31a3b7d78f40'), ('patches', ['0001-kmc-added-compilation-via-autotools.patch'])])), ('requirements', CommentedMap([('build', ['gcc >=4.7', 'libgcc', 'libtool', 'pkg-config', 'zlib', 'bzip2', 'gnu-wget']), ('run', ['libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['kmc 2>&1 | grep \"K-Mer Counter\" > /dev/null', 'kmc_dump 2>&1 | grep \"KMC dump ver.\" > /dev/null', 'kmc_tools 2>&1 | grep \"kmc_tools ver.\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://sun.aei.polsl.pl/REFRESH/index.php?page=projects&project=kmc&subpage=about'), ('license', 'GPLv2'), ('summary', 'K-mer Counter is a utility designed for counting k-mers (sequences of consecutive k symbols) in a set of reads from genome sequencing projects.')])), ('extra', CommentedMap([('note', 'this recipe omits the asmlib speedup due to licensing issues')]))])), 'kmergenie': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kmergenie/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://kmergenie.bx.psu.edu/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'zlib', 'r-base']), ('run', ['python', 'libgcc', 'zlib', 'r-base'])])), ('test', CommentedMap([('commands', ['kmergenie --help'])])), ('about', CommentedMap([('home', 'http://kmergenie.bx.psu.edu/'), ('license', 'free software license'), ('summary', 'KmerGenie estimates the best k-mer length for genome de novo assembly')]))])), 'kneaddata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kneaddata/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/f8/6a/3c8f200d3088c5bc0dd2c85b74d5199e3ca1f2fb7d8baae44708f5b1b928/kneaddata-0.6.1.tar.gz'), ('md5', '863be9ccbb374702ebb1c3d03e3dc6f1')])), ('build', CommentedMap([('entry_points', ['kneaddata = kneaddata.knead_data:main', 'kneaddata_database = kneaddata.download_db:main', 'kneaddata_build_database = kneaddata.generate_db:main', 'kneaddata_test = kneaddata.tests.kneaddata_test:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'trimmomatic', 'bowtie2', 'bmtagger', 'fastqc', 'trf', 'samtools'])])), ('test', CommentedMap([('imports', ['kneaddata', 'kneaddata.db_preprocessing', 'kneaddata.tests']), ('commands', ['kneaddata --help', 'kneaddata_database --help', 'kneaddata_build_database --help', 'kneaddata_test --help'])])), ('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/kneaddata'), ('license', 'MIT'), ('summary', 'KneadData is a tool designed to perform quality control on metagenomic and metatranscriptomic sequencing data, especially data from microbiome experiments.'), ('license_family', 'MIT')]))])), 'knot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/knot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'knot'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'KNOT-1.0.0.tgz'), ('url', 'http://mathbio.nimr.mrc.ac.uk/download/KNOT/KNOT-1.0.0.tgz'), ('md5', '2556876109bae506fe171a45902db0b6')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('test', CommentedMap([('commands', ['which knot', 'knot || [[ $? == 139 ]]'])])), ('about', CommentedMap([('home', 'http://mathbio.nimr.mrc.ac.uk/wiki/Software#KNOT'), ('summary', 'Detection of knots in protein folds.'), ('license', 'GPL')]))])), 'kobas': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kobas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kobas'), ('version', '2.1.1')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('url', 'http://kobas.cbi.pku.edu.cn/kobas-2.1.1/kobas-2.1.1.tar.gz'), ('md5', '38cd6c000de06de24f71e6d29599928f'), ('patches', ['package.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools', 'bioconductor-qvalue', 'blast', 'numpy', 'pandas', 'biopython', 'sqlite', 'r-base', 'rpy2 >=2.8.5'])])), ('test', CommentedMap([('imports', ['kobas']), ('commands', ['kobas-annotate -h', 'kobas-identify -h'])])), ('about', CommentedMap([('home', 'http://http://kobas.cbi.pku.edu.cn/index.php')]))])), 'koeken': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/koeken/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/twbattaglia/koeken/archive/b4d71457d4eff0cd28520d8d69643cd0e798dee4.tar.gz'), ('md5', 'cec172d1d978049e7139cde8d5b2c664'), ('patches', ['setup.patch'])])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'qiime', 'rpy2', 'numpy', 'matplotlib', 'pandas', 'biopython', 'r-optparse', 'r-gtools', 'r-klar', 'r-survival', 'r-mvtnorm', 'r-modeltools', 'r-coin', 'r-mass']), ('run', ['python', 'qiime', 'rpy2', 'numpy', 'matplotlib', 'pandas', 'biopython', 'r-optparse', 'r-gtools', 'r-klar', 'r-survival', 'r-mvtnorm', 'r-modeltools', 'r-coin', 'r-mass'])])), ('test', CommentedMap([('commands', ['koeken.py -v'])])), ('about', CommentedMap([('home', 'https://github.com/twbattaglia/koeken'), ('license', 'MIT'), ('summary', 'A Linear Discriminant Analysis (LEfSe) wrapper.'), ('license_family', 'MIT')]))])), 'kpal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kpal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kpal'), ('version', '2.1.1')])), ('about', CommentedMap([('home', 'https://cutadapt.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'Analysis toolkit and programming library for k-mer profiles')])), ('source', CommentedMap([('url', 'https://github.com/LUMC/kPAL/archive/v2.1.1.tar.gz'), ('fn', 'kpal-2.1.1.tar.gz'), ('md5', 'b5fe62fc749c7f975cf7d6cc9f1954d3')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'h5py >=2.1', 'future', 'semantic_version', 'biopython']), ('run', ['python', 'h5py >=2.1', 'future', 'semantic_version', 'biopython'])])), ('test', CommentedMap([('commands', ['kpal -v'])]))])), 'kraken': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kraken/0.10.5beta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kraken'), ('version', '0.10.5beta')])), ('source', CommentedMap([('fn', 'kraken-0.10.5-beta.tar.gz'), ('url', 'https://github.com/DerrickWood/kraken/archive/v0.10.5-beta.tar.gz'), ('md5', '0231a7bfc067f564ad28fa91e9f71606')])), ('build', CommentedMap([('skip', True), ('has_prefix_files', ['libexec/kraken', 'libexec/kraken-build', 'libexec/kraken-filter', 'libexec/kraken-mpa-report', 'libexec/kraken-report', 'libexec/kraken-translate'])])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['kraken --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/kraken/'), ('license', 'GPLv3'), ('summary', 'Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.')]))])), 'kraken-biom': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kraken-biom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kraken-biom'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'kraken-biom-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ce/54/0ce6705a673629787f5cb9ab4f08d338921a7dd4ec9ac68e2b195215f639/kraken-biom-1.0.1.tar.gz'), ('md5', '12de68accb1a24f7c42727f6518acb15')])), ('build', CommentedMap([('entry_points', ['kraken-biom = kraken_biom:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biom-format >=2.1.5']), ('run', ['python', 'biom-format >=2.1.5'])])), ('test', CommentedMap([('commands', ['kraken-biom --version'])])), ('about', CommentedMap([('home', 'https://github.com/smdabdoub/kraken-biom'), ('license', 'MIT License'), ('summary', 'Create BIOM-format tables from Kraken output.'), ('license_family', 'MIT')]))])), 'kraken-ea': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kraken-ea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kraken-ea'), ('version', '0.10.5ea.3')])), ('source', CommentedMap([('fn', 'v0.10.5-beta-ea.3.tar.gz'), ('url', 'https://github.com/ExpressionAnalysis/kraken/archive/v0.10.5-beta-ea.3.tar.gz'), ('md5', '34dce0103874e199a13eeed9bf20bd73')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', ['perl', 'zlib', 'gcc']), ('run', ['zlib', 'jellyfish 1.1.11', 'perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['kraken --version 2>&1 > /dev/null', 'kraken-build --version'])])), ('about', CommentedMap([('home', 'https://github.com/ExpressionAnalysis/kraken/tree/v0.10.5-beta-ea.2'), ('license', 'GPLv3'), ('summary', 'Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. This is a modified version allowing for splitting of fastq files based on read classifications.')]))])), 'krona': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/krona/2.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/marbl/Krona'), ('license', 'BSD'), ('summary', 'Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files.')])), ('build', CommentedMap([('number', 4)])), ('package', CommentedMap([('name', 'krona'), ('version', 2.6)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('source', CommentedMap([('fn', 'KronaTools-2.6.tar'), ('sha256', '4f04ece119f0d32fb4e61e36ffd1a7809ad053d4db5dcb5f2e7dd4fd3fc8b1de'), ('url', 'https://github.com/marbl/Krona/releases/download/v2.6/KronaTools-2.6.tar')])), ('test', CommentedMap([('commands', ['ktImportTaxonomy', 'ktImportXML'])]))])), 'kronik': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kronik/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '4dd6d1a69c61fbe62e4effc773b54b74d34fe689.tar.gz'), ('url', 'https://github.com/mhoopmann/kronik/archive/4dd6d1a69c61fbe62e4effc773b54b74d34fe689.tar.gz')])), ('build', CommentedMap([('number', 0), ('skipt', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['kronik --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mhoopmann/kronik'), ('license', 'Apache License, Version 2.0'), ('summary', 'Utility for processing Hardklor features to find candidate peptides by chromatographic profiling')]))])), 'lace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lace/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Oshlack/Lace/archive/v<{ version }}.tar.gz'), ('md5', 'e5d93919b0b4eb3e3eebd7246b1cad3a'), ('patches', ['lace.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python', 'pandas', 'networkx', 'numpy', 'matplotlib', 'blat'])])), ('test', CommentedMap([('commands', ['Lace.py -h', 'Lace_Checker.py -h', 'Mobius.py -h', 'STViewer.py -h', 'BuildSuperTranscript.py'])])), ('about', CommentedMap([('home', 'https://github.com/Oshlack/Lace'), ('license', 'Unknown'), ('summary', 'Building SuperTranscripts: A linear representation of transcriptome data')])), ('extra', CommentedMap([('notes', \"Prefix with 'Lace_' for the script name that is rather generic: 'Checker.py'.\"), ('container', CommentedMap([('extended-base', True)]))]))])), 'lap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lap/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'assembly-eval-code-186-trunk.zip'), ('url', 'https://sourceforge.net/code-snapshots/svn/a/as/assembly-eval/code/assembly-eval-code-186-trunk.zip'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python', 'bowtie2'])])), ('test', CommentedMap([('commands', ['calc_prob.py --help', 'gen_rand_samp.py --help', 'sum_prob.py --help', 'mean --help', 'probability --help', 'mprobability --help', 'sample --help', 'stitch --help'])])), ('about', CommentedMap([('home', 'http://assembly-eval.sourceforge.net/'), ('license', ''), ('summary', 'De novo genome assembly evaluation')]))])), 'lapack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lapack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lapack'), ('version', '3.6.0')])), ('source', CommentedMap([('fn', 'lapack-3.6.0.tgz'), ('url', 'http://www.netlib.org/lapack/lapack-3.6.0.tgz'), ('sha1', '7e993de16d80d52b22b6093465eeb90c93c7a2e7')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://www.netlib.org/lapack'), ('license', 'BSD'), ('summary', 'Linear Algebra PACKage')]))])), 'last': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/last/874/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '874')])), ('source', CommentedMap([('fn', 'last-874.zip'), ('md5', '499c75e5f7dd66fd8692d87e06843a13'), ('url', 'http://last.cbrc.jp/last-874.zip'), ('patches', ['maf-convert.23patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc', 'future'])])), ('test', CommentedMap([('commands', ['lastal -V', 'lastdb -V', 'lastal8 -h', 'lastdb8 -h', 'last-split -V', 'last-pair-probs -V', 'last-merge-batches -V', 'maf-convert -h'])]))])), 'lastz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lastz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lastz'), ('version', '1.0.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'lastz-1.02.00.tar.gz'), ('url', 'http://www.bx.psu.edu/miller_lab/dist/lastz-1.02.00.tar.gz'), ('md5', 'b5b9a9b4e088aad720ff5ad6eaa7cac4'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['lastz --version  2>&1 | grep 1.02.00 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.bx.psu.edu/~rsharris/lastz/'), ('license', 'MIT'), ('summary', 'LASTZ is a program for aligning DNA sequences, a pairwise aligne')]))])), 'latentstrainanalysis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/latentstrainanalysis/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/brian-cleary/LatentStrainAnalysis/archive/7f18468.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'gensim', 'parallel', 'pyro4']), ('run', ['python', 'numpy', 'scipy', 'gensim', 'parallel', 'pyro4'])])), ('test', CommentedMap([('commands', ['command -v HashCounting.sh', 'command -v KmerSVDClustering.sh', 'command -v ReadPartitioning.sh', 'HashCounting.sh | grep \"Usage:\"', 'KmerSVDClustering.sh | grep \"Usage:\"', 'ReadPartitioning.sh | grep \"Usage:\"'])])), ('about', CommentedMap([('home', 'https://github.com/brian-cleary/LatentStrainAnalysis'), ('license', 'MIT'), ('summary', 'Partitioning and analysis methods for large, complex sequence datasets'), ('description', 'LSA was developed as a pre-assembly tool for partitioning metagenomic reads.\\nIt uses a hyperplane hashing function and streaming SVD in order to find\\ncovariance relations between k-mers. The code, and the process outline in\\nLSFScripts in particular, have been optimized to scale to massive data\\nsets in fixed memory with a highly distributed computing environment.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('documentation', 'http://latentstrainanalysis.readthedocs.io/en/latest/')]))])), 'lefse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lefse/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/nsegata/lefse/get/54694b4b0d9e335ff1ecafff8db4f1e0cf7004da.tar.gz'), ('sha256', 'f261fe98cf28fdb9f06951a21c9efe0058367220d0b3146989bdb47dccd5511b')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'r-base', 'r-survival', 'r-mvtnorm', 'r-modeltools', 'r-coin', 'r-mass', 'rpy2 >=2.8.5', 'numpy', 'matplotlib >=1.0', 'argparse [py26]', 'biom-format >=2.1.5'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('extra', CommentedMap([('notes', \"Prefix with 'lefse-'' some script names that are rather generic: 'format_input.py', 'plot_cladogram.py', 'plot_features.py', 'plot_res.py'.\")])), ('test', CommentedMap([('commands', ['run_lefse.py -h', 'lefse-format_input.py -h', 'lefse2circlader.py -h', 'lefse-plot_cladogram.py -h', 'lefse-plot_features.py -h', 'lefse-plot_res.py -h', 'qiime2lefse.py -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/nsegata/lefse'), ('summary', 'LDA Effect Size (LEfSe) (Segata et. al 2010) is an algorithm for high-dimensional biomarker discovery and explanation that identifies genomic features (genes, pathways, or taxa) characterizing the differences between two or more biological conditions.'), ('license', 'Custom'), ('license_file', 'license.txt')]))])), 'libbigwig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libbigwig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'libbigwig'), ('version', '<{ LIBBIGWIG_VERSION }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'libBigWig-<{ LIBBIGWIG_VERSION }}.tar.gz'), ('url', 'https://github.com/dpryan79/libBigWig/archive/<{ LIBBIGWIG_VERSION }}.tar.gz'), ('sha256', '<{ LIBBIGWIG_SHA256 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'curl', 'zlib']), ('run', ['gcc', 'curl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/dpryan79/libBigWig'), ('license', 'MIT'), ('summary', 'A C library for handling bigWig files')])), ('test', CommentedMap([('commands', None)]))])), 'libdb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libdb/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'libdb'), ('version', '6.1.26')])), ('source', CommentedMap([('fn', 'db-6.1.26.tar.gz'), ('url', 'http://download.oracle.com/berkeley-db/db-6.1.26.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://www.oracle.com/technology/software/products/berkeley-db/index.html'), ('license', 'AGPLv3'), ('summary', 'The Berkeley DB embedded database system.')]))])), 'libdivsufsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libdivsufsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/y-256/libdivsufsort/archive/5f60d6f026c30fb4ac296f696b3c8b0eb71bd428.tar.gz'), ('md5', '209482686af0b9283659637bd6d51297')])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['test -e ${PREFIX}/include/divsufsort.h', 'test -e ${PREFIX}/include/divsufsort64.h'])])), ('about', CommentedMap([('home', 'https://github.com/y-256/libdivsufsort'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A lightweight suffix-sorting library')]))])), 'libgtextutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libgtextutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'libgtextutils'), ('version', 0.7)])), ('source', CommentedMap([('url', 'https://github.com/agordon/libgtextutils/releases/download/0.7/libgtextutils-0.7.tar.gz'), ('fn', 'libgtextutils-0.7.tar.gz'), ('sha256', '792e0ea3c96ffe3ad65617a104b7dc50684932bc96d2adab501c952fd65c3e4a')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc [not osx]', 'llvm [osx]']), ('run', ['libgcc [not osx]', 'cython'])])), ('test', CommentedMap([('commands', ['test -f $PREFIX/include/gtextutils/gtextutils/tuple_parser.h', 'test -f ${PREFIX}/lib/libgtextutils.a', 'test -f ${PREFIX}/lib/libgtextutils.dylib'])])), ('about', CommentedMap([('home', 'https://github.com/agordon/libgtextutils'), ('license', 'AGPL'), ('summary', 'Gordon Text utils Library'), ('license_file', 'COPYING')]))])), 'libidn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libidn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'libidn'), ('version', '7.45.0')])), ('source', CommentedMap([('fn', 'libidn-1.32.tar.gz'), ('url', 'http://ftp.gnu.org/gnu/libidn/libidn-1.32.tar.gz'), ('md5', '4dd8356ba577287ea7076bfa1554b534')])), ('requirements', CommentedMap([('build', ['autoconf'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('commands', ['true'])])), ('about', CommentedMap([('home', 'https://www.gnu.org/software/libidn/'), ('license', 'LGPLv3'), ('license_file', 'COPYING.LESSERv3'), ('summary', 'Library for internationalized domain name support')]))])), 'libmagic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libmagic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'libmagic'), ('version', 5.25)])), ('source', CommentedMap([('fn', 'autoconf-2.69.tar.gz'), ('url', 'https://fossies.org/linux/misc/file-5.25.tar.gz'), ('sha256', '3735381563f69fb4239470b8c51b876a80425348b8285a7cded8b61d6b890eca')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['file --help &> /dev/null'])])), ('about', CommentedMap([('home', 'http://www.darwinsys.com/file/'), ('summary', 'Implementation of the file(1) command'), ('license', 'https://raw.githubusercontent.com/file/file/master/COPYING')]))])), 'libmuscle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libmuscle/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.zip'), ('url', 'https://github.com/druvus/mauve_snapshot/raw/master/mauve-code-4736-muscle-trunk.zip'), ('md5', 'ccd276f428a3b33ee00c9efa22e69d4c')])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf', 'automake', 'perl', 'libtool']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['test -e $PREFIX/lib/libMUSCLE-3.7.la'])])), ('about', CommentedMap([('home', 'http://darlinglab.org/mauve/'), ('license', 'GPLv2'), ('summary', 'libMuscle header files.')]))])), 'libsbml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libsbml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-libsbml'), ('version', '5.12.0')])), ('source', CommentedMap([('fn', 'python-libsbml_5.12.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/python-libsbml/python-libsbml_5.12.0.tar.gz'), ('md5', 'f05ed071b48702295a5ea44e52723247'), ('patches', ['fix_path.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'libxml2', 'gcc', 'bzip2', 'libxml2']), ('run', ['python', 'libgcc', 'bzip2', 'libxml2'])])), ('test', CommentedMap([('imports', ['libsbml'])])), ('about', CommentedMap([('home', 'http://sbml.org'), ('license', 'LGPL'), ('summary', 'LibSBML Python API. LibSBML is a library for reading, writing and manipulating the Systems Biology Markup Language (SBML).')]))])), 'libsequence': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libsequence/1.9.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://http://molpopgen.github.io/libsequence/'), ('license', 'GNU Lesser General Public License v2 or later (LGPLv2+)'), ('summary', 'A C++ class library for evolutionary genetics.')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'tbb', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['libgcc', 'zlib', 'tbb', 'htslib <{CONDA_HTSLIB}}*'])])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'libsequence'), ('version', '1.9.0')])), ('test', CommentedMap([('commands', ['libsequenceConfig --version | grep 1.9.0 > /dev/null'])])), ('source', CommentedMap([('fn', '1.9.0.tar.gz'), ('url', 'http://github.com/molpopgen/libsequence/archive/1.9.0.tar.gz'), ('md5', '6fff32f756c8bac9e36e887a79e8c913')]))])), 'lighter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lighter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lighter'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'v1.1.1.tar.gz'), ('url', 'https://github.com/mourisl/Lighter/archive/v1.1.1.tar.gz')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['lighter -h'])])), ('about', CommentedMap([('home', 'https://github.com/mourisl/Lighter'), ('license', 'GPL 3'), ('summary', 'ighter is a kmer-based error correction method for whole genome sequencing data')]))])), 'lightning': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lightning/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lightning'), ('version', '0.2.dev0')])), ('source', CommentedMap([('url', 'https://github.com/scikit-learn-contrib/lightning/tarball/3eb7155'), ('fn', 'lightning-0.2.dev0.tar.gz'), ('md5', 'ca950c86f9a54e8f681e8cbd36a82f13')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'scikit-learn', 'numpy', 'scipy']), ('run', ['python', 'scikit-learn', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['lightning'])])), ('about', CommentedMap([('home', 'http://contrib.scikit-learn.org/lightning/'), ('license', 'Unknown'), ('summary', 'lightning is a library for large-scale linear classification, regression and ranking in Python')]))])), 'limix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/limix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'limix'), ('version', '0.7.12')])), ('source', CommentedMap([('fn', 'limix-0.7.12.tar.gz'), ('url', 'https://pypi.python.org/packages/source/l/limix/limix-0.7.12.tar.gz'), ('md5', '4437c627f35afade5b4c62fcfb65d995')])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'gcc', 'setuptools', 'cython', 'h5py', 'numpy', 'scipy']), ('run', ['python >=2.7,<3', 'gcc', 'cython', 'numpy', 'scipy', 'h5py'])])), ('build', CommentedMap([('skip', True)])), ('test', CommentedMap([('imports', ['limix', 'limix.core', 'limix.core.association', 'limix.core.covar', 'limix.core.covar.test', 'limix.core.covar.util', 'limix.core.gp', 'limix.core.likelihood', 'limix.core.linalg', 'limix.core.mean', 'limix.core.optimize', 'limix.core.test', 'limix.core.type', 'limix.core.utils', 'limix.deprecated', 'limix.deprecated.io', 'limix.deprecated.modules', 'limix.deprecated.stats', 'limix.deprecated.utils', 'limix.ensemble', 'limix.io', 'limix.io_', 'limix.modules', 'limix.mtSet', 'limix.mtSet.core', 'limix.stats', 'limix.utils'])])), ('about', CommentedMap([('home', 'http://pmbio.github.io/limix/'), ('license', 'BSD'), ('summary', 'A flexible and fast mixed model toolbox written in C++/python')]))])), 'links': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/links/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'links'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'links_<{ version }}.tar.gz'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/links/releases/1.8.4/links_v1-8-4.tar.gz'), ('md5', '2777029b27c7b9359a7ecef936adcab8'), ('patches', ['links.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'swig', 'gcc', 'llvm']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', [\"LINKS  | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('summary', 'Long Interval Nucleotide K-mer Scaffolder'), ('home', 'http://www.bcgsc.ca/platform/bioinfo/software/links'), ('license', 'GPLv3'), ('license_file', 'LINKS-readme.txt')]))])), 'locarna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/locarna/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.9.1')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'locarna-1.9.1.tar.gz'), ('url', 'https://github.com/s-will/LocARNA/releases/download/v1.9.1/locarna-1.9.1.tar.gz'), ('sha256', 'd5c51282ff2f33e9fc90d6b3d85e807d1a2503272c4401d1bdce77f546178b45')])), ('requirements', CommentedMap([('build', ['viennarna ==2.3.3', 'llvm', 'gcc']), ('run', ['viennarna ==2.3.3', 'perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))])), 'lofreq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lofreq/2.1.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lofreq'), ('version', '2.1.2')])), ('source', CommentedMap([('fn', 'lofreq_star-2.1.2.tgz'), ('url', 'http://downloads.sourceforge.net/project/lofreq/lofreq_star-2.1.2_macosx.tgz'), ('md5', 'd688ba34a4161d9690e264648aae2def')])), ('build', CommentedMap([('number', 0), ('script', 'mkdir -p $PREFIX/bin; chmod a+x bin/*; cp bin/* $PREFIX/bin'), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['lofreq version'])])), ('about', CommentedMap([('home', 'http://csb5.github.io/lofreq/'), ('license', 'MIT'), ('summary', 'A fast and sensitive variant-caller for inferring SNVs and indels from next-generation sequencing data')]))])), 'log4cpp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/log4cpp/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://log4cpp.sourceforge.net/'), ('license', 'Lesser General Public License'), ('summary', 'Log4cpp is library of C++ classes for flexible logging to files, syslog, IDSA and other destinations')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.1.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/<{ name }}/<{ name }}-<{ version }}.x%20%28new%29/<{ name }}-<{ version }}/<{ name }}-<{ version }}.1.tar.gz'), ('md5', '1e173df8ee97205f412ff84aa93b8fbe')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['log4cpp-config --help'])]))])), 'logbook': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/logbook/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'logbook'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'Logbook-1.0,0.tar.gz'), ('url', 'https://pypi.python.org/packages/34/e8/6419c217bbf464fe8a902418120cccaf476201bd03b50958db24d6e90f65/Logbook-1.0.0.tar.gz'), ('md5', '22fea705e067c118925898f0c30b0508')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['logbook'])])), ('about', CommentedMap([('home', 'http://logbook.pocoo.org/'), ('license', 'BSD'), ('summary', 'A logging replacement for Python')]))])), 'lordec': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lordec/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lordec'), ('version', '0.6')])), ('source', CommentedMap([('fn', 'LoRDEC-0.6-Source.tar.gz'), ('url', 'http://gatb-tools.gforge.inria.fr/versions/src/LoRDEC-0.6-Source.tar.gz'), ('md5', '6b82f9e232eb886293411e3f0d090ab0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['{ lordec-trim -h 2>&1 | grep Usage; }'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/lordec/'), ('license', 'CeCILL A license'), ('summary', 'A hybrid error correction program for long, PacBio reads')]))])), 'lrzip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lrzip/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'lrzip'), ('version', '0.621')])), ('source', CommentedMap([('fn', 'lrzip-0.621.tar.bz2'), ('url', 'http://ck.kolivas.org/apps/lrzip/lrzip-0.621.tar.bz2'), ('md5', '53a12cc4d19aa030d0ab7f0a21db2cfe')])), ('requirements', CommentedMap([('build', ['gcc', 'bzip2', 'lzo']), ('run', ['libgcc', 'bzip2', 'lzo'])])), ('test', CommentedMap([('commands', ['lrzip --version'])])), ('about', CommentedMap([('home', 'https://github.com/ckolivas/lrzip'), ('license', 'GPLv2'), ('summary', 'Long Range ZIP or Lzma RZIP. This is a compression program optimised for large files. The larger the file and the more memory you have, the better the compression advantage this will provide, especially once the files are larger than 100MB. The advantage can be chosen to be either size (much smaller than bzip2) or speed (much faster than bzip2).')]))])), 'lsc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lsc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lsc'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'LSC-2.0.tar.gz'), ('url', 'https://www.healthcare.uiowa.edu/labs/au/LSC/files/LSC-2.0.tar.gz'), ('md5', '10e0e85f74a7975cd46ef4ea43ab7c5d'), ('patches', ['install_paths.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'python', 'samtools', 'bowtie2'])])), ('test', CommentedMap([('commands', ['runLSC.py -h'])])), ('about', CommentedMap([('home', 'https://www.healthcare.uiowa.edu/labs/au/LSC/'), ('license', 'Apache 2.0'), ('license_file', 'LICENSE'), ('summary', 'LSC is a long read error correction tool that offers fast correction with high sensitivity and good accuracy.')]))])), 'lumpy-sv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lumpy-sv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lumpy-sv'), ('version', '0.2.13')])), ('source', CommentedMap([('fn', 'lumpy-sv-v0.2.13.tar.gz'), ('url', 'https://github.com/arq5x/lumpy-sv/releases/download/0.2.13/lumpy-sv-v0.2.13.tar.gz'), ('sha256', '1a7a5d18875b297acaa68ee1e864ca048654ebc85348920b518dc689d2b7a3cf')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib', 'python', 'samtools', 'gawk', 'libgcc', 'pysam', 'numpy'])])), ('test', CommentedMap([('commands', ['lumpy 2>&1 | grep -q structural', 'lumpyexpress 2>&1 | grep -q lumpyexpress'])])), ('about', CommentedMap([('home', 'https://github.com/arq5x/lumpy-sv'), ('license', 'MIT'), ('summary', 'a general probabilistic framework for structural variant discovery')]))])), 'lz4-bin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lz4-bin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lz4-bin'), ('version', 131)])), ('about', CommentedMap([('home', 'http://cyan4973.github.io/lz4'), ('license', 'BSD'), ('summary', 'Extremely Fast Compression Application')])), ('extra', CommentedMap([('notes', \"This package is for the lz4 C binary, while the package in the default channel is (as of 9/9/16) for the lz4 Python bindings (hence the '-bin' suffix of this package)\")])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'r131.zip'), ('url', 'https://github.com/Cyan4973/lz4/archive/r131.zip'), ('md5', '1ba15809a7eb321f94c8f68d17548dc9')])), ('requirements', CommentedMap([('build', ['gcc ==4.8.2', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['lz4 -h &> /dev/null'])]))])), 'maaslin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maaslin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maaslin'), ('version', '0.04')])), ('source', CommentedMap([('fn', 'maaslin_0.0.4.tar.gz'), ('url', 'https://bitbucket.org/biobakery/maaslin/downloads/Maaslin_0.0.4.tar.gz'), ('md5', '887d1e347e9a2ad2a38ab78a00838162')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-agricolae', 'r-gam', 'r-gamlss', 'r-gbm', 'r-glmnet', 'r-inlinedocs', 'r-logging', 'r-optparse', 'r-outliers', 'r-penalized', 'r-pscl', 'r-robustbase']), ('run', ['r-base', 'r-agricolae', 'r-gam', 'r-gamlss', 'r-gbm', 'r-glmnet', 'r-inlinedocs', 'r-logging', 'r-optparse', 'r-outliers', 'r-penalized', 'r-pscl', 'r-robustbase'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Maaslin\\')\"'])])), ('about', CommentedMap([('license', 'Custom'), ('summary', 'MaAsLin is a multivariate statistical framework that finds associations between clinical metadata and microbial community abundance or function.'), ('license_file', 'LICENSE'), ('home', 'https://huttenhower.sph.harvard.edu/maaslin')]))])), 'macs2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/macs2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'macs2'), ('version', '2.1.1.20160309')])), ('source', CommentedMap([('fn', 'MACS2-2.1.1.20160309.tar.gz'), ('url', 'https://pypi.python.org/packages/9f/99/a8ac96b357f6b0a6f559fe0f5a81bcae12b98579551620ce07c5183aee2c/MACS2-2.1.1.20160309.tar.gz'), ('md5', '1cc2fab44b232992060c7ab592bf43fa')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'gcc']), ('run', ['python', 'numpy', 'r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['macs2 --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/taoliu/MACS/'), ('license', 'BSD New'), ('summary', 'MACS -- Model-based Analysis of ChIP-Seq')]))])), 'mageck': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mageck/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mageck'), ('version', '0.5.6')])), ('source', CommentedMap([('fn', 'liulab-mageck-3d33a2ffb217.tar.gz'), ('url', 'https://bitbucket.org/liulab/mageck/get/v0.5.6.tar.gz'), ('md5', 'd958195dd23779f0e0d3cf764fe25011')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'ipython']), ('run', ['libgcc', 'python', 'numpy', 'scipy', 'ipython'])])), ('test', CommentedMap([('commands', ['mageck --help > /dev/null', 'RRA', 'mageckGSEA'])])), ('about', CommentedMap([('home', 'http://mageck.sourceforge.net'), ('license', 'BSD License'), ('summary', 'Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout.')]))])), 'mageck-vispr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mageck-vispr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mageck-vispr'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'mageck-vispr-<{ version }}.tar.bz2'), ('url', 'https://bitbucket.org/liulab/mageck-vispr/get/v<{ version }}.tar.bz2'), ('md5', '34197d99acc1ca3d303e252ea9daecad')])), ('build', CommentedMap([('skip', True), ('entry_points', ['mageck-vispr = mageck_vispr.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'jinja2 >=2.8']), ('run', ['python', 'setuptools', 'snakemake >=3.6.0', 'mageck >=0.5.3', 'vispr >=0.4.4', 'fastqc >=0.11.4', 'cutadapt >=1.9.1', 'jinja2 >=2.8', 'rpy2 >=0.7.4', 'bioconductor-sva >=3.15.0'])])), ('test', CommentedMap([('commands', ['mageck-vispr --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/liulab/mageck-vispr'), ('license', 'MIT License'), ('summary', 'MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9 screens based on MAGeCK, VISPR, Snakemake, FastQC and cutadapt.')]))])), 'maker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maker'), ('version', '2.31.9')])), ('source', CommentedMap([('fn', 'maker-2.31.9.tgz.tar.gz'), ('url', 'http://yandell.topaz.genetics.utah.edu/maker_downloads/static/maker-2.31.9.tgz'), ('md5', '63d79e37d25c60d236658f847b74ea18')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql']), ('run', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql'])])), ('about', CommentedMap([('home', 'http://www.yandell-lab.org/software/maker.html'), ('license', 'GPL3'), ('summary', 'MAKER is a portable and easily configurable genome annotation pipeline.')])), ('test', CommentedMap([('commands', ['maker --version'])]))])), 'manormfast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/manormfast/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'manormfast'), ('version', '0.1.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'MAnormFast-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/0a/05/ecd88bb8616d5c59f718ad6eaa4eaf0919cddeb46022b65c6bd0025901e3/MAnormFast-0.1.2.tar.gz'), ('md5', '880a47b84db377174250063221c456c1')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy', 'scipy', 'statsmodels', 'matplotlib'])])), ('test', CommentedMap([('imports', ['MAnormFast']), ('command', ['MAnormFast --help'])])), ('about', CommentedMap([('home', 'https://github.com/semal/MAnormFast'), ('license', 'GNU General Public License'), ('summary', 'Fast, memory-intensive version of MAnorm'), ('license_family', 'GPL')]))])), 'manta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'manta'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'manta-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/manta/releases/download/v<{ version }}/manta-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '30d0a28d83ed513e5e785f41291e64d8')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configManta.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/manta'), ('license', 'GPLv3'), ('summary', 'Structural variant and indel caller for mapped sequencing data')]))])), 'mapdamage2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mapdamage2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mapdamage2'), ('version', '2.0.6')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '2.0.6.tar.gz'), ('url', 'https://github.com/ginolhac/mapDamage/archive/2.0.6.tar.gz'), ('md5', 'f14e1ca52d68fd9c7e42e1f4e6f40f8c'), ('patches', ['makefile.patch', 'composition.patch'])])), ('requirements', CommentedMap([('build', ['python >=2.7,<3']), ('run', ['python >=2.7,<3', 'pysam', 'seqtk'])])), ('test', CommentedMap([('commands', ['mapDamage 2>&1 | grep Usage > /dev/null', \"mapDamage -h 2>&1 | grep 'Options for the statistical estimation' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/ginolhac/mapDamage'), ('license', 'MIT'), ('summary', 'mapDamage: tracking and quantifying damage patterns in ancient DNA sequences http://geogenetics.ku.dk/all_literature/mapdamage/')]))])), 'mappy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mappy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mappy'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'mappy-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/m/mappy/mappy-<{ version }}.tar.gz'), ('md5', 'dfc2aefe98376124beb81ce7dcefeccb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'zlib <{CONDA_ZLIB}}*']), ('run', ['python', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['mappy'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/minimap2'), ('license', 'MIT License'), ('summary', 'Minimap2 Python binding'), ('license_family', 'MIT')]))])), 'mapsembler2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mapsembler2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mapsembler2'), ('version', '2.2.4')])), ('source', CommentedMap([('fn', 'mapsembler2_2.2.4.zip'), ('url', 'http://www.irisa.fr/symbiose/people/ppeterlongo/mapsembler2_2.2.4.zip'), ('md5', '2165dfa0372356aac2b0a50f53a838ff')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['run_mapsembler2_pipeline.sh -h'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/mapsembler2/'), ('license', 'GNU Affero General Public License'), ('summary', 'Targeted assembly software')]))])), 'mapsplice': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mapsplice'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'MapSplice-v2.2.0.zip'), ('url', 'http://protocols.netlab.uky.edu/~zeng/MapSplice-v2.2.0.zip'), ('md5', '171975d48d2ddcc111a314d926f3527a'), ('patches', ['mapsplice.patch', 'samtools.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'py<{CONDA_PY}}_ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['mapsplice.py -v 2>&1 |grep MapSplice > /dev/null'])])), ('about', CommentedMap([('license', 'Custom'), ('summary', 'MapSplice is a software for mapping RNA-seq data to reference genome for splice junction discovery that depends only on reference genome, and not on any further annotations.'), ('license_file', 'Copyright.txt')]))])), 'mash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mash'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'mash-<{ version }}.tar.gz'), ('url', 'https://github.com/marbl/Mash/archive/v<{ version }}.tar.gz'), ('sha256', '7bea8cd3c266640bbd97f2e1c9d0168892915c1c14f7d03a9751bf7a3709dd01')])), ('build', CommentedMap([('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'zlib <{CONDA_ZLIB}}*', 'capnproto', 'gsl <{ CONDA_GSL }}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'capnproto', 'gsl <{ CONDA_GSL }}*'])])), ('test', CommentedMap([('commands', ['mash -h'])])), ('about', CommentedMap([('home', 'https://github.com/marbl/Mash'), ('license', 'https://raw.githubusercontent.com/marbl/Mash/master/LICENSE.txt'), ('summary', 'Fast sequence distance estimator that uses MinHash')]))])), 'mashmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mashmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/marbl/MashMap/archive/v1.0.tar.gz'), ('md5', '215cf5117e52ca0d074979ba9e839b5a')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'perl', 'zlib', 'gsl <{ CONDA_GSL }}*']), ('run', ['libgcc', 'zlib', 'gsl <{ CONDA_GSL }}*'])])), ('test', CommentedMap([('commands', [\"mashmap -h 2>&1 | grep 'Approximate'\"])])), ('about', CommentedMap([('home', 'https://github.com/marbl/MashMap'), ('license', 'Custom'), ('license_file', 'LICENSE.txt'), ('summary', 'A fast and approximate long read mapper')]))])), 'mashtree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mashtree/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/lskatz/mashtree/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'perl-module-build', 'perl-bioperl', 'perl-dbd-sqlite']), ('run', ['libgcc', 'perl', 'perl-bioperl', 'perl-dbd-sqlite', 'mash  >=1.1', 'sqlite'])])), ('test', CommentedMap([('commands', ['mashtree.pl --version', \"mashtree_wrapper.pl --help 2>&1  | grep 'Usage'\"])])), ('about', CommentedMap([('home', 'https://github.com/lskatz/mashtree'), ('license', 'GPL-3.0'), ('summary', 'Create a tree using Mash distances')]))])), 'mason': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mason/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'mason'), ('version', '2.0.7')])), ('source', CommentedMap([('fn', 'mason2-2.0.7-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/mason2/mason2-2.0.7-Linux-x86_64.tar.xz'), ('md5', 'af8ae28f177ad95840a9f4ea62901f78')])), ('test', CommentedMap([('commands', ['mason_frag_sequencing --help 2>&1 > /dev/null', 'mason_genome --help 2>&1 > /dev/null', 'mason_materializer --help 2>&1 > /dev/null', 'mason_methylation --help 2>&1 > /dev/null', 'mason_simulator --help 2>&1 > /dev/null', 'mason_splicing --help 2>&1 > /dev/null', 'mason_variator --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/mason2/README'), ('license', 'https://github.com/seqan/seqan/tree/master/apps/mason2/LICENSE'), ('summary', 'Mason is a collection of tools for the simulation of biological sequences.')]))])), 'mathstats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mathstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mathstats'), ('version', '0.2.4')])), ('source', CommentedMap([('fn', 'mathstats-0.2.4.tar.gz'), ('url', 'https://pypi.python.org/packages/84/c4/bfff926871482d970da9990bb655cde6e0d5b7edd3adb61228661413cf76/mathstats-0.2.4.tar.gz'), ('md5', '2abbaf2b319fadc99896369f22438534')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.9']), ('run', ['python', 'scipy >=0.9'])])), ('test', CommentedMap([('imports', ['mathstats', 'mathstats.normaldist', 'mathstats.normaldist.truncatedskewed'])])), ('about', CommentedMap([('home', 'https://github.com/ksahlin/mathstats'), ('license', 'GPLv3'), ('summary', 'Statistical functions, goodness-of-fit tests and special and special distributions not implemented in scipy/numpy .')]))])), 'mawk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mawk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mawk'), ('version', '1.3.4')])), ('source', CommentedMap([('fn', 'mawk.tar.gz'), ('url', 'http://invisible-island.net/datafiles/release/mawk.tar.gz'), ('sha256', '461673c7c4572e1e67e69e7bf7582e02d3c175b814485f2aa52c2c28099b3c6f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mawk -W usage'])])), ('about', CommentedMap([('home', 'http://invisible-island.net/mawk/'), ('license', 'Copyright (c) 2009-2014,2015 by Thomas E. Dickey'), ('summary', 'mawk is an interpreter for the AWK Programming Language.')]))])), 'maxbin2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maxbin2/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'MaxBin-<{ version }}.tar.gz'), ('url', 'http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc >=4.8']), ('run', ['libgcc', 'fraggenescan >=1.30', 'bowtie2', 'hmmer', 'idba', 'perl-lwp-simple', 'r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['run_MaxBin.pl > /dev/null'])])), ('about', CommentedMap([('home', 'http://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html'), ('license', 'BSD 3-clause'), ('summary', 'MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.')])), ('extra', CommentedMap([('recipe-maintainers', ['colinbrislawn', 'keuv-grvl']), ('doi', ['10.1186/2049-2618-2-26', '10.1093/bioinformatics/btv638'])]))])), 'maxentpy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maxentpy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maxentpy'), ('version', '0.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'maxentpy_0.0.1.tar.gz'), ('url', 'https://github.com/kepbod/maxentpy/archive/0.0.1.tar.gz'), ('md5', '66948d2acf796f69c497197a772bbe05')])), ('requirements', CommentedMap([('build', ['python', 'cython', 'msgpack-python']), ('run', ['python', 'cython', 'msgpack-python'])])), ('test', CommentedMap([('imports', ['maxentpy'])])), ('about', CommentedMap([('home', 'https://github.com/kepbod/maxentpy'), ('license', 'MIT'), ('summary', 'maxentpy is a python wrapper for MaxEntScan to calculate splice site strength.')]))])), 'maxentscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maxentscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maxentscan'), ('version', '0_2004.04.21')])), ('source', CommentedMap([('fn', 'MaxEntScan.tar.gz'), ('url', 'http://genes.mit.edu/burgelab/maxent/download/fordownload.tar.gz'), ('md5', '6f1fabda518998bf8c001149c9cb3b5f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['maxentscan_score3.pl || [[ $? == 2 ]]', 'maxentscan_score5.pl || [[ $? == 2 ]]'])])), ('about', CommentedMap([('home', 'http://genes.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html'), ('summary', \"MaxEntScan is based on the approach for modeling the sequences of short sequence motifs such as those involved in RNA splicing which simultaneously accounts for non-adjacent as well as adjacent dependencies between positions. This method is based on the 'Maximum Entropy Principle' and generalizes most previous probabilistic models of sequence motifs such as weight matrix models and inhomogeneous Markov models.\")])), ('extra', CommentedMap([('notes', 'Due to the generic name of the included scripts, \"maxentscan_\" has been prefixed all executables, e.g. score3.pl can be called as maxentscan_score3.pl.')]))])), 'mbuffer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mbuffer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mbuffer'), ('version', '20160228')])), ('source', CommentedMap([('fn', 'mbuffer-20160228.tgz'), ('url', 'http://www.maier-komor.de/software/mbuffer/mbuffer-20160228.tgz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['mbuffer -h'])])), ('about', CommentedMap([('home', 'http://www.maier-komor.de/mbuffer.html'), ('license', 'GPLv3'), ('summary', 'mbuffer is a tool for buffering data streams with a large set of unique features')]))])), 'mcl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mcl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mcl'), ('version', '14.137')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'mcl-14-137.tar.gz'), ('url', 'http://micans.org/mcl/src/mcl-14-137.tar.gz'), ('md5', 'bc8740456cf51019d0a9ac5eba665bb5'), ('patches', ['mcl.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['mcl 2>&1 | grep usage > /dev/null', 'mcxdeblast --help'])])), ('about', CommentedMap([('home', 'http://micans.org/mcl/'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'MCL - a cluster algorithm for graphs')]))])), 'mdasim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mdasim/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://sourceforge.net/projects/mdasim/files/<{name}}<{version}}.tar.gz'), ('md5', '9bfc68508863082b02714bf9d59f6245'), ('patches', ['bugfix-and-openmpi.patch'])])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'openmpi']), ('run', ['libgcc', 'openmpi'])])), ('test', CommentedMap([('source_files', ['examples/Staphylococcus_aureus_USA300_FPR3757.fa', 'examples/primerList.fasta']), ('commands', ['mdasim --help', 'mdasim --version'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/mdasim/'), ('license', 'GPL-3'), ('license_file', 'LICENSE.txt'), ('summary', 'Tagliavi Z, Draghici S. MDAsim: A multiple displacement amplification simulator. 2012 IEEE International Conference on Bioinformatics and Biomedicine (BIBM). 2012. pp. 1–4. doi:10.1109/BIBM.2012.6392622')]))])), 'mdust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mdust/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://compbio.dfci.harvard.edu/tgi/'), ('license', 'The Artistic License'), ('summary', 'mdust from DFCI Gene Indices Software Tools')])), ('package', CommentedMap([('name', 'mdust'), ('version', '2006.10.17')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mdust -h 2>&1|grep \"Usage: mdust\"'])])), ('source', CommentedMap([('fn', 'mdust.tar.gz'), ('url', 'ftp://occams.dfci.harvard.edu/pub/bio/tgi/software/seqclean/mdust.tar.gz'), ('sha256', '9722806920fc6772dc92f99e7a59554c5f7c35b596dca0df9ed03ff2e5c6440d')]))])), 'medusa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/medusa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'medusa-data-fusion'), ('version', '0.1')])), ('source', CommentedMap([('git_url', 'https://github.com/marinkaz/medusa.git'), ('git_rev', 'f3060f6c3e2c56ec8c7cf564f124780818759944')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy']), ('run', ['python', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['medusa'])])), ('about', CommentedMap([('home', 'https://github.com/marinkaz/medusa'), ('license', 'GPLv3'), ('summary', 'Medusa is an approach to detect size-k modules of objects that, taken together,\\nappear most significant to another set of objects. It builds on collective\\nmatrix factorization to derive different semantics, and it formulates the\\ngrowing of the modules as a submodular optimization program.\\n')]))])), 'megagta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/megagta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'megagta'), ('version', '0.1_alpha')])), ('source', CommentedMap([('fn', 'megagta-v0.1alpha-src.tar.gz'), ('url', 'https://github.com/HKU-BAL/megagta/releases/download/v0.1-alpha/megagta-v0.1alpha-src.tar.gz')])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'm4', 'gcc', 'llvm', 'zlib', 'ant']), ('run', ['libgcc', 'zlib', 'python >=2.7,<3', 'hmmer >=3.1b2'])])), ('build', CommentedMap([('skip', True)])), ('test', CommentedMap([('requires', ['python >=2.7,<3']), ('commands', ['megagta.py -v > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/HKU-BAL/megagta'), ('license', 'GNU General Public License v3 (GPLv3'), ('summary', 'HMM-guided metagenomic gene-targeted assembler using iterative de Bruijn graphs')]))])), 'megahit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/megahit/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/voutcn/<{ name }}/archive/v<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib 1.2.8']), ('run', ['libgcc', 'zlib 1.2.8', 'python'])])), ('test', CommentedMap([('commands', ['megahit --12 $PREFIX/share/megahit/example/readsInterleaved1.fa.gz && rm -r megahit_out'])])), ('about', CommentedMap([('home', 'https://github.com/voutcn/megahit'), ('license', 'GPL3'), ('summary', 'MEGAHIT: An ultra-fast single-node solution for large and complex\\nmetagenomics assembly via succinct de Bruijn graph\\n')]))])), 'melt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/melt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'melt'), ('version', '1.0.3')])), ('source', CommentedMap([('fn', 'melt-1.0.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/m/melt/melt-1.0.3.tar.gz'), ('md5', '9b84a279a26098cbcea97631d568e644')])), ('build', CommentedMap([('entry_points', ['Tm = melting:main']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['Tm --help'])])), ('about', CommentedMap([('home', 'https://github.com/eclarke/melt'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A nucleotide melt temp calculator')]))])), 'meme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/meme/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://alternate.meme-suite.org/'), ('license', 'Custom'), ('license_file', 'COPYING'), ('summary', 'Motif based sequence Analysis tools')])), ('package', CommentedMap([('name', 'meme'), ('version', '4.11.2')])), ('build', CommentedMap([('number', 0), ('skip', True), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'yaml', 'python', 'perl', 'ghostscript', 'zlib', 'libxslt', 'libxml2', 'expat', 'perl-xml-parser', 'perl-app-cpanminus', 'perl-yaml', 'perl-xml-simple', 'perl-html-template', 'perl-cgi']), ('run', ['libgcc', 'yaml', 'expat', 'python', 'perl', 'zlib', 'libxslt', 'libxml2', 'ghostscript', 'perl-xml-parser', 'perl-yaml', 'perl-xml-simple', 'perl-html-template', 'perl-cgi'])])), ('source', CommentedMap([('fn', 'meme_4.11.2.tar.gz'), ('url', 'http://alternate.meme-suite.org/meme-software/4.11.2/meme_4.11.2.tar.gz'), ('sha1', '317c230772b62ae3a44d2e7dcc2466eadedf6662'), ('patches', ['mast.patch'])])), ('test', CommentedMap([('commands', ['meme -version', 'mast -version', 'meme-chip 2>&1 | grep FIMO > /dev/null', 'centrimo 2>&1 | grep maxreg > /dev/null'])]))])), 'mendelscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mendelscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mendelscan'), ('version', 'v1.2.2')])), ('source', CommentedMap([('fn', 'MendelScan.v1.2.2.jar'), ('url', 'https://github.com/genome/mendelscan/releases/download/1.2.2/MendelScan.v1.2.2.jar'), ('md5', '78b7f643c1683afb073c17b914c60a86')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['mendelscan -h 2>&1 | grep USAGE > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/genome/mendelscan'), ('license', 'Unknown'), ('summary', 'Analyze exome data for Mendelian disorders.')]))])), 'mentalist': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mentalist/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/WGS-TB/MentaLiST/archive/v<{ version }}.tar.gz'), ('sha256', '0121110f87264423d9ffd18a6262753b1d643a84cbaabf0842c0d2f909d7ccf6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['julia 0.5.2', 'hdf5', 'mpfr']), ('run', ['julia 0.5.2', 'hdf5', 'mpfr'])])), ('test', CommentedMap([('commands', ['mentalist -h'])])), ('about', CommentedMap([('home', 'https://github.com/WGS-TB/MentaLiST'), ('summary', 'The MLST pipeline developed by the PathOGiST research group.'), ('license', 'MIT'), ('license_file', 'LICENSE')]))])), 'mercat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mercat/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/pnnl/mercat/archive/<{ version }}.tar.gz'), ('md5', 'd5bef2073368f8a82a46015bebc8962d')])), ('build', CommentedMap([('number', 0), ('script', 'python setup.py install')])), ('preserve_egg_dir', 'yes'), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'scikit-bio ==0.2.3', 'prodigal', 'trimmomatic', 'dask', 'pandas', 'numpy', 'humanize', 'plotly', 'psutil', 'joblib'])])), ('test', CommentedMap([('commands', ['mercat --help'])])), ('about', CommentedMap([('home', None), ('license', 'BSD'), ('summary', 'Mercat: a versatile counter and diversity estimator for data base independent property analysis obtained from whole community sequencing data.'), ('license_family', 'BSD')])), ('extra', CommentedMap([('recipe-maintainers', ['ajaypanyala', 'raw937']), ('container', CommentedMap([('extended-base', True)]))]))])), 'merfishtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/merfishtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'py<{CONDA_PY}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name }}-<{version}}.tar.gz'), ('url', 'https://github.com/<{ name }}/<{ name }}/archive/v<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['rust >=1.19', 'gcc 4.8.2', 'gsl <{CONDA_GSL}}*', 'python', 'matplotlib', 'numpy', 'pandas']), ('run', ['python', 'matplotlib', 'numpy', 'pandas', 'gsl <{CONDA_GSL}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['merfishtools --help']), ('imports', ['merfishtools'])])), ('about', CommentedMap([('home', 'https://merfishtools.github.io'), ('license', 'MIT'), ('summary', 'MERFISHtools implement a Bayesian framework for accurately predicting gene or transcript expression from MERFISH data. On top, differential expression analysis can be performed for two or multiple conditions, including credible intervals for fold change and coefficient of variation, and controlling the expected false discovery rate.')]))])), 'mergevcf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mergevcf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mergevcf'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'mergevcf-0ac3e63.tar.gz'), ('url', 'https://github.com/ljdursi/mergevcf/archive/0ac3e63.tar.gz'), ('md5', 'f3f1af91d1e2cdc89d51fc14a351be21')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyvcf']), ('run', ['python', 'pyvcf'])])), ('test', CommentedMap([('run', ['mergevcf -h'])])), ('about', CommentedMap([('home', 'https://github.com/ljdursi/mergevcf'), ('license', 'MIT'), ('summary', 'Python package and routines for merging VCF files')]))])), 'merlin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/merlin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'merlin'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'merlin-1.1.2.tar.gz'), ('url', 'http://csg.sph.umich.edu/abecasis/merlin/download/merlin-1.1.2.tar.gz'), ('md5', '01ce17395aff3fba8d49447aeb2128fa'), ('sha1', 'd5227276eae862506c7ed4475255c9be182585df'), ('sha256', 'aa3c087b1c717209389705bf2ace4c24884575014a561dff250c0be93fe1e411')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['merlin | grep MERLIN'])])), ('about', CommentedMap([('home', 'http://csg.sph.umich.edu/abecasis/merlin'), ('license', 'OpenSource'), ('license_file', 'LICENSE.twister'), ('summary', 'MERLIN uses sparse trees to represent gene flow in pedigrees and is a fast pedigree analysis package')]))])), 'meryl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/meryl/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Getting_Started_with_Meryl'), ('license', 'GPL'), ('summary', 'meryl is a multi-threaded, multi-process, out-of-core k-mer counter')])), ('source', CommentedMap([('fn', '<{ name }}-r<{ version }}.tar.xz'), ('url', 'http://downloads.sourceforge.net/project/kmer/meryl-r2013.tar.xz'), ('md5', '445c243c5b9845acda971ec05254a025')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['existDB 2>&1 | grep existDB', 'kmer-mask 2>&1 | grep kmer-mask', 'mapMers 2>&1 | grep mapMers', 'mapMers-depth 2>&1 | grep mapMers-depth', 'meryl 2>&1 | grep meryl', 'mt19937ar-test', 'positionDB 2>&1 | grep positionDB', 'simple 2>&1 | grep \"no input\"', 'test-merStream', 'test-seqCache'])]))])), 'metacluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metacluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'http://i.cs.hku.hk/~alse/MetaCluster/files/MetaCluster5.tar.bz2'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['MetaCluster5_1 > /dev/null 2&>1 || [ \"$?\" == \"255\" ]', 'MetaCluster5_2 > /dev/null 2&>1 || [ \"$?\" == \"255\" ]'])])), ('about', CommentedMap([('home', 'http://i.cs.hku.hk/~alse/MetaCluster/'), ('license', 'GPLv2'), ('summary', 'MetaCluster5.1 is a new software for binning short pair-end reads'), ('description', 'MetaCluster5.1 is an unsupervised binning method that can (1) samples with\\nlow-abundance species, or (2) samples (even with high-abundance) with many\\nextremely-low-abundance species. The input file should be in fasta format.\\nEvery odd-number read and its next read are supposed to be pair-end reads.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1093/bioinformatics/bts397')]))])), 'metameta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metameta/1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metameta'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'v1.1.tar.gz'), ('url', 'https://github.com/pirovc/metameta/archive/v1.1.tar.gz'), ('md5', 'e21addb746996be6faab624a62f097d0')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('run', ['snakemake ==3.9.1'])])), ('test', CommentedMap([('commands', ['metameta --configfile $PREFIX/opt/metameta/sampledata/sample_data_archaea_bacteria.yaml --config workdir=\"$PREFIX/opt/metameta/sampledata/files/\" -n'])])), ('about', CommentedMap([('home', 'https://github.com/pirovc/metameta/'), ('license', 'The MIT License (MIT)'), ('summary', 'MetaMeta - pipeline for integrating metagenome analysis tools to improve taxonomic profiling')]))])), 'metametamerge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metametamerge/1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metametamerge'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'v1.0.tar.gz'), ('url', 'https://github.com/pirovc/metametamerge/archive/v1.0.tar.gz'), ('md5', '64403a1e778aa82032211209541e92a9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('commands', ['MetaMetaMerge.py -h 2>&1 | echo $?'])])), ('about', CommentedMap([('home', 'https://github.com/pirovc/metametamerge/'), ('license', 'The MIT License (MIT)'), ('summary', 'Merging module of the MetaMeta Pipeline')]))])), 'metaphyler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaphyler/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'MetaPhylerV1.25.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://metaphyler.cbcb.umd.edu/MetaPhylerV1.25.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'blast-legacy']), ('run', ['libgcc', 'perl', 'blast-legacy'])])), ('test', CommentedMap([('commands', ['runMetaphyler.pl 2>&1 | grep \"Usage\"', 'buildMetaphyler.pl 2>&1 | grep \"Usage\"', 'runClassifier.pl 2>&1 | grep \"Usage\"', 'simuReads 2>&1 | grep \"Usage\"', 'metaphylerTrain 2>&1 | grep \"Usage\"', 'metaphylerClassify 2>&1 | grep \"Usage\"', 'combine 2>&1 | grep \"Usage\"', 'taxprof 2>&1 | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'http://metaphyler.cbcb.umd.edu/'), ('license', 'Artistic License 2.0'), ('summary', 'Estimating Bacterial Composition from Metagenomic Sequences')]))])), 'metaprob': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaprob/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/samu661/metaprob/downloads/<{ name }}_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'eigen', 'boost <{ CONDA_BOOST }}*']), ('run', ['libgcc', 'eigen', 'boost <{ CONDA_BOOST }}*'])])), ('test', CommentedMap([('commands', ['command -v MetaProb', 'MetaProb > /dev/null'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/samu661/metaprob/'), ('license', 'copyright'), ('summary', 'assembly-assisted tool for un-supervised metagenomic binning'), ('description', 'MetaProb is a novel assembly-assisted tool for un-supervised metagenomic\\nbinning. The novelty of MetaProb derives from solving a few important\\nproblems: how to divide reads into groups of independent reads, so that\\nl-mer frequencies are not overestimated; how to convert l-mer counts into\\nprobabilistic sequence signatures, that will correct for variable\\ndistribution of l-mers, and for unbalanced groups of reads, in order to\\nproduce better estimates of the underlying genome statistic. We show that\\nMetaProb is effective for both simulated and real datasets. It can\\naccurately (with F-measures of 87 for short reads and 97 long reads) and\\nefficiently bin short and long reads with varying abundance ratios.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl'])]))])), 'metaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metaseq'), ('version', '0.5.6')])), ('source', CommentedMap([('fn', 'metaseq-0.5.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/m/metaseq/metaseq-0.5.6.tar.gz'), ('md5', 'f29575db23019a5cd1003bfccb599847')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher']), ('run', ['python', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher'])])), ('test', CommentedMap([('imports', ['metaseq'])])), ('about', CommentedMap([('home', 'http://github.com/daler/metaseq'), ('license', 'MIT License'), ('summary', 'Integrative analysis of high-thoughput sequencing data')]))])), 'metaseq-all': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaseq-all/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metaseq-all'), ('version', '0.5.6')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['metaseq', 'bedtools', 'samtools', 'ucsc-bedgraphtobigwig', 'ucsc-bedtobigbed', 'ucsc-bigbedtobed', 'ucsc-bigwigsummary'])])), ('about', CommentedMap([('summary', 'Meta-package for metaseq including bedtools and UCSC tools'), ('license', 'The license for this meta-package is MIT; individual tools vary')]))])), 'metasv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metasv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metasv'), ('version', '0.5.4')])), ('source', CommentedMap([('fn', '0.5.4.tar.gz'), ('url', 'https://github.com/bioinform/metasv/archive/0.5.4.tar.gz'), ('md5', 'de2e21ac4f86bc4d1830bdfff95d8391')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'pybedtools ==0.6.9', 'pyvcf ==0.6.7', 'pysam ==0.7.7']), ('run', ['python', 'pybedtools ==0.6.9', 'pyvcf ==0.6.7', 'pysam ==0.7.7', 'age-metasv', 'spades'])])), ('test', CommentedMap([('commands', ['run_metasv.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/bioinform/metasv'), ('summary', 'An accurate and integrative structural-variant caller for next generation sequencing'), ('license', 'MIT')]))])), 'metavelvet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metavelvet/1.1.01/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp'), ('license', 'GNU General Public License'), ('summary', 'MetaVelvet : An extension of Velvet assembler to de novo metagenome assembly from short sequence reads')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'metavelvet'), ('version', '1.1.01')])), ('requirements', CommentedMap([('build', ['libgcc', 'gcc', 'zlib']), ('run', ['libgcc', 'zlib', 'velvet', 'metavelvet-annois'])])), ('source', CommentedMap([('fn', 'metavelvet-1.1.01.tgz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/src/MetaVelvet-1.1.01.tgz'), ('patches', ['metavelvet.velvet.makefile.patch'])])), ('test', CommentedMap([('commands', ['meta-velvetg --help 2>&1 | grep \"contiging and scaffolding program for metagenomics NGS data\"'])]))])), 'metavelvet-annois': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metavelvet-annois/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp'), ('license', 'GNU General Public License'), ('summary', 'Metavelvet AnnoIS - an extra package for metavelvet for versions < 1.2.01')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'metavelvet-annois'), ('version', '0.2.01')])), ('requirements', CommentedMap([('build', ['perl-module-build', 'perl-threaded']), ('run', ['perl-threaded', 'perl-module-build'])])), ('source', CommentedMap([('fn', 'metavelvet-annois-0.2.01.tgz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/src/annoIS.0.2.01.tgz'), ('patches', ['run-annoIS.pl.patch'])])), ('test', CommentedMap([('commands', ['run-annoIS.pl --help 2>&1 | grep \"IS fasta file\"'])]))])), 'metavelvet-sl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metavelvet-sl/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'metavelvet-sl-1.0.tar.gz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/srcSL/MetaVelvetSLv1.0.tar.gz'), ('md5', '<{ md5 }}'), ('patches', ['metavelvet.velvet.makefile.patch', 'run-annoIS.pl.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'metavelvet-sl-feature-extraction', 'velvet', 'metaphlan2', 'dwgsim', 'libsvm'])])), ('test', CommentedMap([('commands', ['meta-velvete -h 2>&1 | grep \"meta-\"', 'meta-velvetg -h 2>&1 | grep \"meta-\"', 'run-annoIS.pl -h 2>&1 | grep \"## Options\"'])])), ('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp/MSL.html'), ('license', ''), ('summary', 'MetaVelvet-SL : An extension of Velvet assembler to de novo metagenomic assembler utilizing supervised learning')]))])), 'metavelvet-sl-feature-extraction': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metavelvet-sl-feature-extraction/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'LearningModelFeatures.tar.gz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/srcSL/LearningModelFeatures.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl', 'perl-app-cpanminus', 'perl-module-build'])])), ('test', CommentedMap([('commands', ['eval.pl -h 2>&1 | grep \"## Options\"', 'FeatureExtract.pl 2>&1 | grep \"file\"', 'FeatureExtractPredict.pl 2>&1 | grep \"file\"'])])), ('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp/MSL.html'), ('license', ''), ('summary', 'Perl libraries that do feature extraction for metavelvet-sl')]))])), 'metavelvet-sl-pipeline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metavelvet-sl-pipeline/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'MetaVelvetSL_Pipeline.tar.gz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/srcSL/MetaVelvetSL_Pipeline.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'metavelvet-sl', 'metavelvet-sl-feature-extraction', 'velvet', 'metaphlan2', 'dwgsim', 'libsvm'])])), ('test', CommentedMap([('commands', ['GenerateCommand.pl -h', 'ReadTaxon.pl 2>&1 | grep \"taxonomy\"'])])), ('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp/MSL.html'), ('license', ''), ('summary', 'Perl libraries that run the full pipeline for metavelvet-sl')]))])), 'metawatt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metawatt/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}-<{ version }}.zip'), ('md5', '<{ md5 }}'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', None), ('run', ['wget', 'openjdk', 'prodigal', 'diamond 0.7*', 'hmmer', 'bbmap', 'aragorn', 'samtools', 'mafft', 'blast', 'fasttree'])])), ('test', CommentedMap([('commands', ['metawatt --check-dependencies 2>&1 >/dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/metawatt/'), ('license', 'AFL'), ('summary', 'MetaWatt is a metagenomic binning tool'), ('description', 'The Metawatt binner is a graphical binning tool that makes use of\\nmultivariate statistics of tetranucleotide frequencies and differential\\ncoverage based binning. It also performs taxonomic assessment of binning\\nquality (via diamond BLASTx). Created bins can be edited and exported as\\nfasta. The Metawatt is implemented in Java SWING and minimally depends on\\nDiamond, HMMer3.1, BBMap, Prodigal and the Batik library for the export of\\nSVG graphics.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.3389/fmicb.2012.00410'), ('notes', 'metawatt --help')]))])), 'metfrag': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metfrag/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://c-ruttkies.github.io/MetFrag/'), ('license', 'GNU Lesser General Public License version 2.1 or later.'), ('summary', 'MetFrag is a freely available software for the annotation of high precision tandem mass spectra of metabolites which is a first and critical step for the identification of a molecular structure. Candidate molecules of different databases are fragmented in silico and matched against mass to charge values. A score calculated using the fragment peak matches gives hints to the quality of the candidate spectrum assignment.')])), ('package', CommentedMap([('name', 'metfrag'), ('version', '2.3.1')])), ('source', CommentedMap([('fn', 'MetFrag2.3.1-CL.jar'), ('url', 'http://msbi.ipb-halle.de/~cruttkie/metfrag/MetFrag2.3.1-CL.jar'), ('sha256', '4261ea5abd023016bd99ef5fdd197eb74306586a4a1d637c87d45e36809d21f4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk >=7'])])), ('test', CommentedMap([('commands', ['metfrag --help'])]))])), 'methyldackel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/methyldackel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'methyldackel'), ('version', '0.3.0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'MethylDackel-0.3.0.tar.gz'), ('sha256', 'b5a49f213bba9113d2c9951de9daf676efd9b9c94327097e960b8623765f1b91'), ('url', 'https://github.com/dpryan79/MethylDackel/archive/0.3.0.tar.gz')])), ('requirements', CommentedMap([('build', ['zlib', 'gcc', 'llvm']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['MethylDackel --version'])])), ('about', CommentedMap([('home', 'https://github.com/dpryan79/MethylDackel'), ('license', 'MIT'), ('summary', 'A (mostly) universal methylation extractor for BS-seq experiments. Formerly named PileOMeth.')]))])), 'metilene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metilene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metilene'), ('version', '0.2.6')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-leipzig.de/Software/metilene/'), ('license', 'GPLv2'), ('license_file', 'LICENSE'), ('summary', 'Fast and sensitive detection of differential DNA methylation')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'metilene_v02-6.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/Software/metilene/metilene_v02-6.tar.gz'), ('sha256', '832255cc58d6f3d4a1a157ca6a97186f0ff66a42480d3d0e45eceb4771f3978b'), ('patches', ['patch.0'])])), ('requirements', CommentedMap([('build', ['llvm', 'gcc']), ('run', ['perl-threaded', 'bedtools >=2.24', 'r-base', 'r-ggplot2 >=2.0.0', 'libgcc'])])), ('test', CommentedMap([('commands', ['metilene --help 2>&1 | grep VERSION', 'metilene_input.pl --help 2>&1 | grep usage', 'metilene_output.pl --help 2>&1 | grep usage'])]))])), 'mgf-formatter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mgf-formatter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mgf-formatter'), ('version', '1.0.0')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'd3fdf38.tar.gz'), ('url', 'https://bitbucket.org/galaxyp-applications/mgf-formatter/get/d3fdf38.tar.gz'), ('md5', '04b711a5d04caa02f4713c3fbda8c968')])), ('requirements', CommentedMap([('run', ['openjdk', 'python'])])), ('test', CommentedMap([('commands', ['mgf-formatter --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/galaxyp-applications/mgf-formatter'), ('license', 'Eclipse Public License'), ('summary', 'Tools for convert peak lists into MGF files formatted for particular downstream applications')]))])), 'mgkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mgkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/bd/2b/a227340be64f3ab18ee4e864b87803f1f613bb700f355bcefd6ea20d3680/mgkit-0.3.0.tar.gz'), ('md5', '44632a22c6957d694e26418981e51e84')])), ('build', CommentedMap([('skip', True), ('entry_points', ['download_data = mgkit.workflow.download_data:main', 'download_profiles = mgkit.workflow.download_profiles:main', 'filter-gff = mgkit.workflow.filter_gff:main', 'add-gff-info = mgkit.workflow.add_gff_info:main', 'get-gff-info = mgkit.workflow.extract_gff_info:main', 'hmmer2gff = mgkit.workflow.hmmer2gff:main', 'blast2gff = mgkit.workflow.blast2gff:main', 'snp_parser = mgkit.workflow.snp_parser:main', 'translate_seq = mgkit.workflow.nuc2aa:main', 'fastq_utils = mgkit.workflow.fastq_utils:main', 'taxon_utils = mgkit.workflow.taxon_utils:main', 'taxon-utils = mgkit.workflow.taxon_utils:main', 'json2gff = mgkit.workflow.json2gff:main', 'fasta-utils = mgkit.workflow.fasta_utils:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.2', 'pysam >=0.8.2.1', 'pandas >=0.16.2', 'matplotlib >=1.5', 'msgpack-python >=0.4.6', 'scipy >=0.15.1', 'semidbm >=0.5.1', 'pymongo >=3.1.1', 'r-base', 'htseq >=0.6.0', 'rpy2 >=2.8.5', 'enum34']), ('run', ['python', 'numpy >=1.9.2', 'pysam >=0.8.2.1', 'pandas >=0.16.2', 'matplotlib >=1.5', 'msgpack-python >=0.4.6', 'scipy >=0.15.1', 'semidbm >=0.5.1', 'pymongo >=3.1.1', 'r-base', 'htseq >=0.6.0', 'rpy2 >=2.8.5', 'enum34'])])), ('test', CommentedMap([('imports', ['mgkit', 'mgkit.counts', 'mgkit.db', 'mgkit.filter', 'mgkit.io', 'mgkit.net', 'mgkit.plots', 'mgkit.snps', 'mgkit.utils', 'mgkit.workflow']), ('commands', ['download_data --help', 'download_profiles --version', 'filter-gff --version', 'add-gff-info --version', 'get-gff-info --version', 'hmmer2gff --version', 'blast2gff --version', 'snp_parser --version', 'translate_seq --version', 'fastq_utils --version', 'taxon_utils --version', 'taxon-utils --version', 'json2gff --version', 'fasta-utils --version'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/setsuna80/mgkit/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('summary', 'Metagenomics Framework')]))])), 'mglex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mglex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mglex'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'MGLEX-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/m/mglex/MGLEX-0.2.0.tar.gz'), ('sha256', '2f5dcfd1e381f156558249e56da15df24a34d31f656da0b427c8d4cdb50fc246')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'setuptools_scm', 'gcc', 'llvm', 'numpy >=1.8.2', 'scipy >=0.13.3', 'docopt >=0.6.2']), ('run', ['python', 'libgcc', 'numpy >=1.8.2', 'scipy >=0.13.3', 'docopt >=0.6.2'])])), ('test', CommentedMap([('imports', ['mglex', 'mglex.cli', 'mglex.models'])])), ('about', CommentedMap([('home', 'https://github.com/fungs/mglex'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'MGLEX - MetaGenome Likelihood EXtractor'), ('license_family', 'GPL3')]))])), 'mgltools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mgltools/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mgltools.scripps.edu/'), ('license_file', 'LICENSES'), ('summary', 'MGLTools is an program for visualization and analisys of molecular structures.')])), ('package', CommentedMap([('name', 'mgltools'), ('version', '1.5.6')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mgltools_i86Darwin9_1.5.6.tar.gz'), ('url', 'http://mgltools.scripps.edu/downloads/downloads/tars/releases/REL1.5.6/mgltools_i86Darwin9_1.5.6.tar.gz'), ('md5', 'c515b187cddeec72fe5a81ecb4a92d6b')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['pmv --help', 'prepare_ligand4.py', 'prepare_receptor4.py'])]))])), 'mgs-canopy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mgs-canopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/fplaza/mgs-canopy-algorithm/archive/00a60e0.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['boost <{ CONDA_BOOST }}*', 'gcc', 'llvm']), ('run', ['boost <{ CONDA_BOOST }}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['command -v cc.bin', 'command -v canopy'])])), ('about', CommentedMap([('home', 'https://github.com/fplaza/mgs-canopy-algorithm'), ('license', 'GPL3'), ('summary', 'Canopy clustering algorithm')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1038/nbt.2939')]))])), 'mhap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mhap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mhap'), ('version', '2.1.1')])), ('source', CommentedMap([('fn', 'mhap-2.1.1.jar.gz'), ('url', 'https://github.com/marbl/MHAP/releases/download/v2.1.1/mhap-2.1.1.jar.gz'), ('md5', 'fcc79d0b7483d55d3d4d744c860bbceb')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk', 'python'])])), ('test', CommentedMap([('commands', ['mhap -h'])])), ('about', CommentedMap([('home', 'https://github.com/marbl/MHAP'), ('license', 'Apache License 2.0'), ('summary', 'MHAP: MinHash Alignment Protocol. A tool for finding overlaps of long-read sequences (such as PacBio or Nanopore) in bioinformatics.')]))])), 'midas': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/midas/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/snayfach/MIDAS/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}'), ('patches', ['midas.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pandas', 'pysam', 'numpy', 'biopython']), ('run', ['python', 'pandas', 'pysam', 'numpy', 'biopython', 'hs-blastn', 'bowtie2', 'samtools >=1.4.0'])])), ('test', CommentedMap([('commands', ['build_midas_db.py --help', 'call_consensus.py --help', 'compare_genes.py --help', 'merge_midas.py  --help', 'query_by_compound.py --help', 'run_midas.py --help', 'snp_diversity.py --help', 'strain_tracking.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/snayfach/MIDAS'), ('license', 'GPL-3.0'), ('summary', 'An integrated pipeline for estimating strain-level genomic variation from metagenomic data')]))])), 'migmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/migmap/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'migmap'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'migmap-<{ version }}.zip'), ('url', 'https://github.com/mikessh/migmap/releases/download/<{ version }}/migmap-<{ version }}.zip'), ('sha256', '35936c19bb0ac8b59322ee4e602e92e90fc1afdcf0dd5018521a4b28f09f7842')])), ('about', CommentedMap([('home', 'https://github.com/mikessh/migmap'), ('license', 'https://github.com/mikessh/migmap/blob/master/LICENSE'), ('summary', 'A wrapper for IgBlast V-(D)-J mapping tool designed to facilitate analysis immune receptor libraries profiled using high-throughput sequencing.\\n')])), ('requirements', CommentedMap([('run', ['igblast', 'openjdk'])]))])), 'mimodd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mimodd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mimodd'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'MiModD-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/93/ef/721a2cd06e7880ba00ce13ea5c5986a68155de09032d1eb3733dd4171f73/MiModD-0.1.7.3.tar.gz#md5=c0665a7a4ccc2c69107fcac6551ac43b'), ('md5', 'c0665a7a4ccc2c69107fcac6551ac43b')])), ('requirements', CommentedMap([('build', ['zlib', 'gcc', 'llvm', 'python']), ('run', ['zlib', 'libgcc', 'python', 'rpy2'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/mimodd'), ('license', 'GPL3'), ('summary', 'Tools for Mutation Identification in Model Organism Genomes')])), ('test', CommentedMap([('commands', ['mimodd help'])]))])), 'minced': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minced/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/ctSkennerton/minced/archive/<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ hash }}')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['openjdk']), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['minced --version | grep MinCED'])])), ('about', CommentedMap([('home', 'https://github.com/ctSkennerton/minced'), ('license', 'GPL-3.0'), ('summary', 'MinCED - Mining CRISPRs in Environmental Datasets')]))])), 'minepy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minepy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'minepy'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'minepy-1.2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/e0/d1/b871d4779776aa3be524a892d085a964392a057476765dd07b1fd78687ae/minepy-1.2.0.tar.gz'), ('md5', 'b22db1dfb5e96d909a46bacc072a77e6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.3.0', 'gcc', 'llvm']), ('run', ['python', 'numpy >=1.3.0', 'libgcc'])])), ('test', CommentedMap([('imports', ['minepy'])])), ('about', CommentedMap([('home', 'http://minepy.readthedocs.io'), ('license', 'GNU General Public License (GPL)'), ('summary', 'minepy - Maximal Information-based Nonparametric Exploration'), ('license_family', 'GPL')]))])), 'minialign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minialign/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/ocxtal/minialign/archive/minialign-0.5.3.tar.gz'), ('md5', 'd6fc134ae301662376da62e01a130729')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['minialign -v'])])), ('about', CommentedMap([('home', 'https://github.com/ocxtal/minialign'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Fast and accurate alignment tool for PacBio and Nanopore long reads.'), ('description', 'Minialign is a little bit fast and moderately accurate nucleotide sequence \\nalignment tool designed for PacBio and Nanopore long reads. It is built on \\nthree key algorithms, minimizer-based index of the minimap overlapper, \\narray-based seed chaining, and SIMD-parallel Smith-Waterman-Gotoh extension.\\n')]))])), 'miniasm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/miniasm/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/lh3/miniasm/archive/40ec2809489b72410f9b436e1145f4cf593f259c.tar.gz'), ('md5', 'fea96683cfac16cb431196051d42ae45')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['miniasm -V'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/miniasm'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Ultrafast de novo assembly for long noisy reads (though having no consensus step)')]))])), 'minimap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minimap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'minimap'), ('version', '0.2_r124')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'minimap_v0.2-r124.tar.gz'), ('url', 'https://github.com/lh3/minimap/archive/1cd6ae3bc7c7a6f9e7c03c0b7a93a12647bba244.tar.gz'), ('md5', 'c4ef2c2d5ed19e97d4bb29ad8f4dfea9')])), ('requirements', CommentedMap([('build', ['gcc [linux]', 'llvm [osx]']), ('run', ['libgcc [linux]'])])), ('test', CommentedMap([('commands', ['minimap -V  > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/minimap'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Experimental tool to find approximate mapping positions between long sequences')]))])), 'minimap2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minimap2/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/lh3/minimap2/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', [\"minimap2 2>&1 | grep 'Usage'\"])])), ('about', CommentedMap([('home', 'https://github.com/lh3/minimap2'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'A fast sequence mapping and alignment program')]))])), 'minimock': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minimock/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'minimock'), ('version', '1.2.8')])), ('source', CommentedMap([('fn', 'MiniMock-1.2.8.tar.gz'), ('url', 'https://pypi.python.org/packages/43/c8/0928c5f684d6f86fb34c12e4b6bcde2dd8b951d060f304b5e4cc4d13d106/MiniMock-1.2.8.tar.gz'), ('md5', '76593aaba949b5e010fec95283556449')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['minimock'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/MiniMock'), ('license', 'MIT License'), ('summary', 'The simplest possible mock library'), ('license_family', 'MIT')]))])), 'mintmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mintmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mintmap'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'MINTmap_codesOnlyNotRunnable_v1.0.tar.gz'), ('url', 'https://github.com/TJU-CMC-Org/MINTmap/archive/MINTmap_codesOnlyNotRunnable_v1.0.tar.gz'), ('md5', '7edebe2092c108a989147124c6207b97'), ('patches', ['mintmap.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl-base'])])), ('test', CommentedMap([('commands', ['MINTmap.pl'])])), ('about', CommentedMap([('home', 'https://github.com/TJU-CMC-Org/MINTmap'), ('license', 'open source GNU GPL v3.0 license'), ('summary', 'Generate tRF profiles from short RNA-Seq datasets')]))])), 'minvar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/minvar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'minvar'), ('version', '1.2b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/ozagordi/MinVar/archive/v1.2b.tar.gz'), ('md5', '3cfd62be162fe5cec5549afb4038ac40')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'setuptools_scm', 'setuptools_scm_git_archive']), ('run', ['bedtools', 'biopython', 'blast >=2.3', 'bwa', 'emboss', 'htslib', 'lofreq >=2.1.3.1', 'pandas', 'pandoc', 'python', 'samtools >=1.3', 'seqtk', 'setuptools_scm_git_archive'])])), ('about', CommentedMap([('home', 'https://git.io/minvar'), ('summary', 'A tool to detect minority variants in HIV-1 populations'), ('license', 'Custom')])), ('test', CommentedMap([('commands', ['minvar -h', 'minvar -v'])]))])), 'mir-prefer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mir-prefer/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/hangelwen/miR-PREFeR/archive/v0.24.tar.gz'), ('md5', 'c78ae8bbb7daba266648da1a9d4a2f27'), ('patches', ['patches/bowtie-align-reads.py.patch', 'patches/convert-mirdeep2-fasta.py.patch', 'patches/convert-readcount-file.py.patch', 'patches/miR_PREFeR.py.patch', 'patches/process-reads-fasta.py.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'viennarna', 'samtools ==0.1.19', 'bowtie ==1.2.0'])])), ('test', CommentedMap([('commands', ['bowtie-align-reads.py -h 2>&1 | grep \"python bowtie-align-reads.py\" > /dev/null', 'convert-mirdeep2-fasta.py -h 2>&1 | grep \"python process-reads-fasta.py\" > /dev/null', 'convert-readcount-file.py -h 2>&1 | grep \"python process-reads-fasta.py\" > /dev/null', 'miR_PREFeR.py -h 2>&1 | grep \"python mir_PREFeR.py\" > /dev/null', 'process-reads-fasta.py -h 2>&1 | grep \"python preocess-reads-fasta.py\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/hangelwen/miR-PREFeR'), ('description', 'microRNA PREdiction From small RNA-seq data'), ('license', 'GPL')]))])), 'mira': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mira/4.9.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '4.9.6')])), ('source', CommentedMap([('fn', 'mira_4.9.6.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_4.9.6_darwin_x64.tar.bz2'), ('sha256', '3d0bb15adad0e1e27899d21223cb552968d34d35ad9b1184e85832fc0b3d22ce')])), ('test', CommentedMap([('commands', ['mira --help'])]))])), 'miranda': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/miranda/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'miranda'), ('version', '3.3a')])), ('source', CommentedMap([('fn', 'miRanda-aug2010.tar.gz'), ('url', 'http://cbio.mskcc.org/microrna_data/miRanda-aug2010.tar.gz'), ('sha256', 'a671da562cf4636ef5085b27349df2df2f335774663fd423deb08f31212ec778')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['miranda --version'])])), ('about', CommentedMap([('home', 'http://www.microrna.org/'), ('license', 'GPLv2 + RNAlib license (no commercial redistribution)'), ('summary', 'An algorithm for finding genomic targets for microRNAs')]))])), 'mixcr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mixcr/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'mixcr'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'mixcr-<{ version }}.zip'), ('url', 'https://github.com/milaboratory/mixcr/releases/download/v<{ version }}/mixcr-<{ version }}.zip'), ('sha256', '012ca495754d4904493d17a2f27e3102d531132746d17722e1c6b319fbec76b4')])), ('about', CommentedMap([('home', 'https://github.com/milaboratory/mixcr'), ('license', 'https://github.com/milaboratory/mixcr/blob/develop/LICENSE'), ('summary', 'MiXCR is a universal software for fast and accurate analysis of raw T- or B- cell receptor repertoire sequencing data.\\n')])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['mixcr -h'])]))])), 'mlst': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mlst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mlst'), ('version', '2.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mlst_2.9.tar.gz'), ('url', 'https://github.com/tseemann/mlst/archive/2.9.tar.gz'), ('md5', '7421ad5a423440570f6eede2cfc4a684')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl', 'perl-bioperl', 'perl-moo', 'perl-list-moreutils', 'blast'])])), ('test', CommentedMap([('commands', ['mlst --list 2>&1 | grep leptospira > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/mlst'), ('license', 'GPL2'), ('license_file', 'LICENSE'), ('summary', 'Scan contig files against PubMLST typing schemes')]))])), 'mobster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mobster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/jyhehir/mobster'), ('license', 'GPL3'), ('summary', 'NGS tool for detecting MEI and gene retrotransposition events in WGS and WES data, see Thung et al. Genome Biol. 2014 for more information.')])), ('package', CommentedMap([('name', 'mobster'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'mobster-<{ version }}.tar.gz'), ('url', 'https://github.com/jyhehir/mobster/archive/v<{ version }}.tar.gz'), ('sha256', 'fb6b13d7442444f1b402262df80b2c469cfde7e088a7f04cdaee4657c7854536')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['maven']), ('run', ['openjdk', 'python', 'mosaik'])])), ('test', CommentedMap([('commands', ['mobster --help', 'mobster-to-vcf --help'])])), ('extra', CommentedMap([('notes', 'After installation, mobster is available as command `mobster`. \\nFurther, you can convert mobster output to vcf with the command `mobster-to-vcf`.\\n')]))])), 'moca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/moca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'moca'), ('version', '0.3.4')])), ('source', CommentedMap([('fn', 'moca-0.3.4.tar.gz'), ('url', 'https://pypi.python.org/packages/1c/d1/6f35a10eda0d458c4c921756c0e4cd3dc11bac664023784e82dc8c1d0fb7/moca-0.3.4.tar.gz'), ('md5', '8fc7a56a695bc339cc6b306f52ff6339')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['moca=scripts.mocacli:cli'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython >=1.68', 'certifi >=2016.2.28', 'cffi >=1.9.1', 'click >=6.6', 'click-help-colors >=0.3', 'coverage >=4.2', 'cryptography >=1.7.1', 'cycler >=0.10.0', 'enum34 >=1.1.6', 'functools32 >=3.2.3.post2', 'future >=0.16.0', 'idna >=2.2', 'ipaddress >=1.0.18', 'matplotlib >=2.0.0', 'mmtf-python >=1.0.5', 'msgpack-python >=0.4.8', 'numpy >=1.11.3', 'olefile >=0.44', 'pandas >=0.19.2', 'patsy >=0.4.1', 'pillow >=4.0.0', 'py >=1.4.32', 'pyasn1 >=0.1.9', 'pybedtools >=0.7.9', 'pybigwig >=0.2.8', 'pycairo >=1.10.0', 'pycparser >=2.17', 'pyparsing >=2.1.4', 'pysam >=0.9.1.4', 'pytest >=3.0.5', 'pytest-cov >=2.4.0', 'pytest-mpl >=0.5', 'python-dateutil >=2.3', 'pytz >=2016.10', 'pyyaml >=3.12', 'reportlab >=3.3.0', 'scipy >=0.18.1', 'seaborn >=0.7.1', 'six >=1.10.0', 'statsmodels >=0.6.1', 'subprocess32 >=3.2.7', 'tqdm >=4.7.2']), ('run', ['python', 'setuptools', 'biopython >=1.68', 'certifi >=2016.2.28', 'cffi >=1.9.1', 'click >=6.6', 'click-help-colors >=0.3', 'coverage >=4.2', 'cryptography >=1.7.1', 'cycler >=0.10.0', 'enum34 >=1.1.6', 'functools32 >=3.2.3.post2', 'future >=0.16.0', 'idna >=2.2', 'ipaddress >=1.0.18', 'matplotlib >=2.0.0', 'mmtf-python >=1.0.5', 'msgpack-python >=0.4.8', 'numpy >=1.11.3', 'olefile >=0.44', 'pandas >=0.19.2', 'patsy >=0.4.1', 'pillow >=4.0.0', 'py >=1.4.32', 'pyasn1 >=0.1.9', 'pybedtools >=0.7.9', 'pybigwig >=0.2.8', 'pycairo >=1.10.0', 'pycparser >=2.17', 'pyparsing >=2.1.4', 'pysam >=0.9.1.4', 'pytest >=3.0.5', 'pytest-cov >=2.4.0', 'pytest-mpl >=0.5', 'python-dateutil >=2.3', 'pytz >=2016.10', 'pyyaml >=3.12', 'reportlab >=3.3.0', 'scipy >=0.18.1', 'seaborn >=0.7.1', 'six >=1.10.0', 'statsmodels >=0.6.1', 'subprocess32 >=3.2.7', 'tqdm >=4.7.2'])])), ('test', CommentedMap([('imports', ['moca', 'moca.bedoperations', 'moca.helpers', 'moca.pipeline', 'moca.plotter', 'moca.wigoperations', 'scripts']), ('commands', ['LANG=C.UTF-8 moca --help']), ('requires', ['pytest', 'pytest-cov', 'pytest-mpl'])])), ('about', CommentedMap([('home', 'https://github.com/saketkc/moca'), ('license', 'ISC License (ISCL)'), ('summary', 'Tool for motif conservation analysis'), ('license_family', 'Public-Domain')]))])), 'monovar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/monovar/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://bitbucket.org/dlaehnemann/monovar/get/<{ version }}.tar.gz'), ('md5', 'a5b49a53aa660ce084b1ef0f3d5bbb32')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'pysam']), ('run', ['python', 'numpy', 'scipy', 'pysam'])])), ('test', CommentedMap([('commands', ['monovar 2>&1 | grep \" Reference genome file not provided. \"'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/hamimzafar/monovar'), ('license', 'MIT'), ('summary', 'single cell joint genotyper')]))])), 'moods': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/moods/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'moods'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jhkorhonen/MOODS/releases/download/v<{ version }}/MOODS-python-<{ version }}.tar.gz'), ('fn', 'moods-<{ version }}.tar.gz'), ('md5', '40113c829ec6c699bd59978f313add1f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['MOODS.scan'])])), ('about', CommentedMap([('home', 'https://github.com/jhkorhonen/MOODS'), ('license', 'GPL-3.0'), ('summary', 'MOODS, Motif Occurrence Detection Suite')]))])), 'moreutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/moreutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'moreutils'), ('version', '0.5.7')])), ('source', CommentedMap([('fn', 'moreutils-0.5.7.tar.gz'), ('url', 'http://http.debian.net/debian/pool/main/m/moreutils/moreutils_0.57.orig.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['sponge -h'])])), ('about', CommentedMap([('home', 'http://joeyh.name/code/moreutils/'), ('license', 'GPLv2'), ('summary', 'a growing collection of the unix tools that nobody thought to write long ago when unix was young')]))])), 'morpheus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/morpheus/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/cwenger/Morpheus/'), ('license', 'MIT'), ('summary', 'mass spectrometry–based proteomics database search algorithm')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'cp * $PREFIX/bin && cd $PREFIX/bin && ln -s ./morpheus_mzml_mono_cl.exe ./morpheus')])), ('source', CommentedMap([('fn', 'Morpheus_mzML_Mono.tar.gz'), ('url', 'https://github.com/cwenger/Morpheus/releases/download/r<{ version }}/Morpheus_mzML_Mono.tar.gz'), ('md5', '918207ce1ecf20f6cdcd4aeb03b84e4b')])), ('requirements', CommentedMap([('build', None), ('run', ['mono >=4.0.0'])])), ('test', CommentedMap([('commands', [\"mono `which morpheus` | grep 'mzML Mono command line'\"])]))])), 'mosaik': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mosaik/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/wanpinglee/MOSAIK'), ('license', 'GPL 2.0+'), ('summary', 'MOSAIK is a stable, sensitive and open-source program for mapping second and third-generation sequencing reads to a reference genome.')])), ('package', CommentedMap([('name', 'mosaik'), ('version', '2.2.26')])), ('source', CommentedMap([('fn', 'e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('md5', '2fd32477444978ec7662df77226cc896'), ('url', 'https://github.com/wanpinglee/MOSAIK/archive/e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('patches', ['osx-makefile.patch', 'linux.inc.patch'])])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['MosaikAligner --help | grep Usage > /dev/null'])])), ('build', CommentedMap([('number', 2), ('skip', False)]))])), 'mosdepth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mosdepth/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{revision}}.tar.gz'), ('url', 'https://github.com/brentp/mosdepth/archive/<{ revision }}.tar.gz'), ('sha256', '<{ sha256hash }}'), ('patches', ['install.diff'])])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'pcre', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['libgcc', 'pcre', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['mosdepth -h'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/mosdepth'), ('license', 'MIT'), ('summary', 'Fast BAM/CRAM depth calculation for WGS, exome, or targetted sequencing.')]))])), 'mothur': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mothur/1.36.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mothur'), ('version', '1.36.1')])), ('source', CommentedMap([('fn', 'v1.36.1.tar.gz'), ('url', 'https://github.com/mothur/mothur/archive/v1.36.1.tar.gz'), ('md5', '96e176b223468e4d0cda02859a4ac7aa'), ('patches', ['Build-Flags.patch'])])), ('build', CommentedMap([('number', 3), ('string', 'boost1.60_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'ncurses', 'readline', 'boost >=1.55,1.60*', 'blast-legacy']), ('run', ['libgcc', 'zlib', 'ncurses', 'readline', 'boost 1.60*', 'blast-legacy'])])), ('test', CommentedMap([('commands', ['mothur \"#help()\"', 'uchime --version'])])), ('about', CommentedMap([('home', 'http://www.mothur.org'), ('license', 'GPL'), ('summary', 'An expandable software to fill the bioinformatics needs of the microbial ecology community.')]))])), 'motus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/motus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'motus'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'mOTUs.Linux64bits.tar.gz'), ('url', 'http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz'), ('md5', '4c1fc6dc2ea803c9ee58d96bc6301b8d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mOTUs.pl | head -2 | tail -1 | grep -q \"^For single-sample mode:\"; echo $?'])])), ('about', CommentedMap([('home', 'http://www.bork.embl.de/software/mOTU'), ('license', 'GNU GPL v3'), ('summary', 'mOTUs Metagenomic species profiling using universal phylogenetic marker genes')]))])), 'mpa-portable': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mpa-portable/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/meta-proteome-analyzer'), ('license', 'Apache License, Version 2.0'), ('summary', 'MPA Portable is a light-weight and stand-alone software for the identification of proteins and in-depth analysis of metaproteomics.\\n')])), ('package', CommentedMap([('name', 'mpa-portable'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'mpa-portable-<{ version }}.zip'), ('url', 'https://github.com/compomics/meta-proteome-analyzer/releases/download/v<{ version }}/mpa-portable-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['mpa-portable de.mpa.cli.CmdLineInterface -Djava.awt.headless=true -Xmx2048m 2>&1 | grep MPA'])])), ('extra', CommentedMap([('notes', 'mpa-protable is Java program that comes with a custom wrapper shell script.\\n')]))])), 'mqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mqc'), ('version', '1.7')])), ('source', CommentedMap([('url', 'https://github.com/Biobix/mQC/archive/1.7.tar.gz'), ('md5', '62e4c0cc26ff5a71aed087ca2f21bddd')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('run', ['python', 'numpy', 'pandas', 'matplotlib', 'seaborn', 'r-base', 'perl', 'perl-app-cpanminus', 'perl-dbi', 'perl-dbd-sqlite', 'perl-parallel-forkmanager', 'plastid', 'samtools'])])), ('test', CommentedMap([('commands', ['mQC.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/Biobix/mQC'), ('license', 'GNU General Public License v3.0'), ('license_file', 'LICENSE'), ('summary', 'qualtiy control tool to assess the mapping quality of a ribosome profiling experiment')]))])), 'mrbayes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mrbayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://sourceforge.net/projects/<{ name | lower }}/files/<{ name | lower }}/<{ version }}/<{ name | lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'readline 6.*', 'openmpi', 'beagle-lib']), ('run', ['libgcc', 'readline 6.*', 'openmpi', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['echo Version | mb', 'echo Version | mpirun -mca plm_rsh_agent true --allow-run-as-root mb-mpi'])])), ('about', CommentedMap([('home', 'http://mrbayes.sourceforge.net'), ('license', 'GPLv3'), ('license_family', 'GPL'), ('license_file', 'src/gpl.txt'), ('summary', 'Bayesian Inference of Phylogeny'), ('description', 'MrBayes is a program for Bayesian inference and model choice across a wide\\nrange of phylogenetic and evolutionary models. MrBayes uses Markov chain\\nMonte Carlo (MCMC) methods to estimate the posterior distribution of model\\nparameters.\\n')]))])), 'mreps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mreps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mreps'), ('version', '0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mreps-2.6.tar'), ('url', 'http://mreps.univ-mlv.fr/mreps-2.6.tar'), ('md5', '4cc5444676501adeddf09f0aaf68dde8')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('files', ['DmelChr4.fa']), ('commands', ['mreps -res 3 -exp 3.0 -from 10000 -to 12000 DmelChr4.fa'])])), ('about', CommentedMap([('home', 'http://mreps.univ-mlv.fr/'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('license_file', 'LICENSE.md'), ('summary', 'mreps is a flexible and efficient software for identifying serial repeats (usually called tandem repeats) in DNA sequences.')]))])), 'ms': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ms'), ('version', '2014_03_04')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc'])])), ('source', CommentedMap([('fn', 'ms.tar.gz'), ('url', 'https://webshare.uchicago.edu/users/rhudson1/Public/ms.folder/ms.tar.gz'), ('md5', 'ef693dcc424fd5293b0bea4c68953700')])), ('test', CommentedMap([('commands', ['ms 2 1 -T > /dev/null'])])), ('about', CommentedMap([('home', 'http://home.uchicago.edu/rhudson1/source/mksamples.html'), ('license', 'Unknown'), ('summary', 'Generates random independent samples according to a simple Wright-Fisher neutral model.')]))])), 'msgf_plus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msgf_plus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msgf_plus'), ('version', '2017.07.21')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'v20170721.zip'), ('url', 'https://github.com/sangtaekim/msgfplus/releases/download/v2017.07.21/v20170721.zip'), ('sha256', '71071d16474513459f9ab2255fd872e463357093952a2599aecda70aec3d133f')])), ('requirements', CommentedMap([('run', ['openjdk', 'python'])])), ('test', CommentedMap([('commands', ['msgf_plus'])])), ('about', CommentedMap([('home', 'https://omics.pnl.gov/software/ms-gf'), ('license', 'https://github.com/sangtaekim/msgfplus/blob/master/LICENSE.txt'), ('summary', 'MS-GF+ is a new MS/MS database search tool that is sensitive (it identifies more peptides than other database search tools and as many peptides as spectral library search tools) and universal (works well for diverse types of spectra, different configurations of MS instruments and different experimental protocols).')]))])), 'msms': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msms'), ('version', '2.6.1')])), ('source', CommentedMap([('fn', 'msms_i86_64Linux2_2.6.1.tar.gz'), ('url', 'http://mgltools.scripps.edu/downloads/tars/releases/MSMSRELEASE/REL2.6.1/msms_i86_64Linux2_2.6.1.tar.gz'), ('md5', '650d8c6fd49d9ce291f30ed530d5f313'), ('patches', ['pdb_to_xyzr.patch', 'pdb_to_xyzrn.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('has_prefix_files', ['bin/pdb_to_xyzr', 'bin/pdb_to_xyzrn'])])), ('test', CommentedMap([('commands', ['msms -h'])])), ('about', CommentedMap([('home', 'http://mgltools.scripps.edu/packages/MSMS/'), ('license', 'Free for academic use.'), ('summary', 'MSMS is a program written in the C programming language to compute molecular surfaces.')]))])), 'msprime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msprime/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msprime'), ('version', '0.4.0')])), ('source', CommentedMap([('fn', 'msprime-0.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/db/65/7010c12c8594b6a36b7b6bb7f108cce17003b45d676aafb3f275c7cd598a/msprime-0.4.0.tar.gz'), ('md5', '83f78872174741d2de4db6689ae3d74f')])), ('build', CommentedMap([('entry_points', ['mspms=msprime.cli:mspms_main', 'msp=msprime.cli:msp_main']), ('number', 2), ('string', 'py<{CONDA_PY}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'setuptools_scm', 'svgwrite', 'hdf5', 'gsl <{CONDA_GSL}}*']), ('run', ['python', 'svgwrite', 'hdf5', 'numpy', 'h5py', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('imports', ['msprime']), ('commands', ['msp --help > /dev/null', 'mspms 2 1 -T > /dev/null'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/msprime'), ('license', 'GNU Lesser General Public License v3 or later (LGPLv3+)'), ('summary', 'A fast and accurate coalescent simulator.')]))])), 'msproteomicstools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msproteomicstools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msproteomicstools'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'msproteomicstools-0.5.0.tar.gz'), ('url', 'https://pypi.python.org/packages/39/2a/c91256aaf64b3bd5f89d1dbd3203a0445d6f2675ad6d16d5b437be9e1a9b/msproteomicstools-0.5.0.tar.gz'), ('md5', '9580ef61bd56ef9c12cd1c60dce260c3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'mercurial', 'python-cluster ==1.3.3', 'pyteomics >=2.4.0', 'xlsxwriter >=0.5.3', 'scikits-datasmooth', 'pymzml ==0.7.5', 'lxml', 'configobj', 'biopython', 'xlwt']), ('run', ['python', 'numpy', 'scipy', 'python-cluster ==1.3.3', 'pyteomics >=2.4.0', 'xlsxwriter >=0.5.3', 'scikits-datasmooth', 'pymzml ==0.7.5', 'lxml', 'configobj', 'biopython', 'xlwt'])])), ('test', CommentedMap([('imports', ['msproteomicstoolslib', 'msproteomicstoolslib.algorithms', 'msproteomicstoolslib.algorithms.PADS', 'msproteomicstoolslib.algorithms.alignment', 'msproteomicstoolslib.algorithms.shared', 'msproteomicstoolslib.data_structures', 'msproteomicstoolslib.format', 'msproteomicstoolslib.math', 'msproteomicstoolslib.util', 'openswathgui', 'openswathgui.models', 'openswathgui.views'])])), ('about', CommentedMap([('home', 'https://code.google.com/p/msproteomicstools'), ('license', 'BSD License'), ('summary', 'Tools for MS-based proteomics'), ('license_family', 'BSD')]))])), 'msstitch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msstitch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msstitch'), ('version', '2.5')])), ('source', CommentedMap([('fn', 'msstitch-2.5.tar.gz'), ('url', 'https://pypi.python.org/packages/2e/5e/9d9d0276d107d66147830475def0d49f223c38fc9d5e26e09888d241275f/msstitch-2.5.tar.gz'), ('md5', 'e096baeaf679e14091f9f58beaf25388')])), ('build', CommentedMap([('skip', True), ('number', 0), ('entry_points', ['msspercolator=app.pycolator:main', 'msslookup=app.mslookup:main', 'msspsmtable=app.mzidtsv:main', 'msspeptable=app.peptable:main', 'mssprottable=app.prottable:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'lxml', 'biopython >=1.69', 'pyyaml']), ('run', ['python', 'numpy', 'lxml', 'biopython >=1.69', 'pyyaml', 'libgcc'])])), ('test', CommentedMap([('imports', ['app', 'app.actions', 'app.actions.headers', 'app.actions.mslookup', 'app.actions.mzidtsv', 'app.actions.peptable', 'app.actions.prottable', 'app.actions.pycolator', 'app.actions.shared', 'app.dataformats', 'app.drivers', 'app.drivers.mslookup', 'app.drivers.mzidtsv', 'app.drivers.peptable', 'app.drivers.prottable', 'app.drivers.pycolator', 'app.lookups', 'app.lookups.sqlite', 'app.readers', 'app.writers']), ('commands', ['msspercolator --help', 'msslookup --help', 'msspsmtable --help', 'msspeptable --help', 'mssprottable --help'])])), ('about', CommentedMap([('home', 'https://github.com/glormph/msstitch'), ('license', 'MIT License'), ('summary', 'MS proteomics post processing utilities'), ('license_family', 'MIT')]))])), 'mugsy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mugsy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mugsy'), ('version', '1.2.3')])), ('source', CommentedMap([('fn', 'mugsy_x86-64-v1r2.3.tgz'), ('url', 'http://downloads.sourceforge.net/project/mugsy/mugsy_x86-64-v1r2.3.tgz'), ('md5', 'c4208cce45ff76f3fc7643764453617b'), ('patches', ['perl-env.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://mugsy.sourceforge.net'), ('license', 'Artistic License 2.0'), ('summary', 'Mugsy is a multiple whole genome aligner.'), ('license_file', 'LICENSE')]))])), 'multiqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/multiqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'multiqc'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'v1.2.tar.gz'), ('url', 'https://github.com/ewels/MultiQC/archive/v1.2.tar.gz'), ('md5', '55dac8b0dc055810ae09ec9b72e0603e')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2']), ('run', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2', 'libgcc'])])), ('test', CommentedMap([('imports', ['multiqc', 'multiqc.modules', 'multiqc.modules.adapterRemoval', 'multiqc.modules.afterqc', 'multiqc.modules.bamtools', 'multiqc.modules.bcftools', 'multiqc.modules.bcl2fastq', 'multiqc.modules.biobloomtools', 'multiqc.modules.bismark', 'multiqc.modules.bowtie1', 'multiqc.modules.bowtie2', 'multiqc.modules.busco', 'multiqc.modules.clusterflow', 'multiqc.modules.conpair', 'multiqc.modules.custom_content', 'multiqc.modules.cutadapt', 'multiqc.modules.disambiguate', 'multiqc.modules.fastq_screen', 'multiqc.modules.fastqc', 'multiqc.modules.featureCounts', 'multiqc.modules.flexbar', 'multiqc.modules.gatk', 'multiqc.modules.goleft_indexcov', 'multiqc.modules.hicup', 'multiqc.modules.hisat2', 'multiqc.modules.homer', 'multiqc.modules.htseq', 'multiqc.modules.jellyfish', 'multiqc.modules.kallisto', 'multiqc.modules.leehom', 'multiqc.modules.macs2', 'multiqc.modules.methylQA', 'multiqc.modules.peddy', 'multiqc.modules.picard', 'multiqc.modules.preseq', 'multiqc.modules.prokka', 'multiqc.modules.qorts', 'multiqc.modules.qualimap', 'multiqc.modules.quast', 'multiqc.modules.rna_seqc', 'multiqc.modules.rseqc', 'multiqc.modules.salmon', 'multiqc.modules.samblaster', 'multiqc.modules.samtools', 'multiqc.modules.skewer', 'multiqc.modules.slamdunk', 'multiqc.modules.snpeff', 'multiqc.modules.sortmerna', 'multiqc.modules.star', 'multiqc.modules.theta2', 'multiqc.modules.tophat', 'multiqc.modules.trimmomatic', 'multiqc.modules.vcftools', 'multiqc.plots', 'multiqc.templates', 'multiqc.templates.default', 'multiqc.templates.geo', 'multiqc.templates.sections', 'multiqc.templates.simple', 'multiqc.utils']), ('commands', ['LC_ALL=C.UTF-8 LANG=C.UTF-8 multiqc --version', 'LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 multiqc --version'])])), ('about', CommentedMap([('home', 'http://multiqc.info'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Create aggregate bioinformatics analysis reports across many samples and tools'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'multiqc-bcbio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/multiqc-bcbio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'multiqc-bcbio'), ('version', '0.2.1')])), ('source', CommentedMap([('fn', 'multiqc-bcbio-<{ tag }}.tar.gz'), ('url', 'https://github.com/MultiQC/MultiQC_bcbio/archive/<{ tag }}.tar.gz'), ('md5', '22aa1180b12a2929157e919ccbcb5e19')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'multiqc >=1.2']), ('run', ['python', 'multiqc >=1.2'])])), ('test', CommentedMap([('imports', ['multiqc_bcbio.bcbio'])])), ('about', CommentedMap([('home', 'http://multiqc.info'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'MultiQC plugin for bcbio report visualization.')]))])), 'mummer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mummer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mummer.sourceforge.net/'), ('license', 'The Artistic License'), ('summary', 'MUMmer is a system for rapidly aligning entire genomes')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('package', CommentedMap([('name', 'mummer'), ('version', '3.23')])), ('source', CommentedMap([('fn', 'MUMmer3.23.tar.gz'), ('sha256', '1efad4f7d8cee0d8eaebb320a2d63745bb3a160bb513a15ef7af46f330af662f'), ('url', 'https://downloads.sourceforge.net/project/mummer/mummer/3.23/MUMmer3.23.tar.gz')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mummer -h &> /dev/null', 'mummerplot -h &> /dev/null', 'nucmer -h &> /dev/null', 'promer -h &> /dev/null', 'show-aligns -h &> /dev/null'])]))])), 'munkres': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/munkres/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'munkres'), ('version', '<{ MUNKRES_VERSION }}')])), ('source', CommentedMap([('fn', 'munkres-<{ MUNKRES_VERSION }}.tar.gz'), ('url', 'https://pypi.python.org/packages/source/m/munkres/munkres-<{ MUNKRES_VERSION }}.tar.gz'), ('md5', 'd534612326f7c7cadcfa61d109f96289')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['setuptools', 'python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['munkres'])])), ('about', CommentedMap([('home', 'http://software.clapper.org/munkres/'), ('license', 'Apache License'), ('summary', 'munkres algorithm for the Assignment Problem')]))])), 'muscle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/muscle/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '1b7c9661f275a82d3cf708f923736bf8'), ('url', 'http://www.drive5.com/muscle/muscle_src_3.8.1551.tar.gz'), ('patches', ['osx-makefile.patch'])])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['muscle -version'])])), ('about', CommentedMap([('home', 'http://www.drive5.com/muscle/'), ('license', 'http://www.drive5.com/muscle/manual/license.html'), ('summary', 'MUSCLE: multiple sequence alignment with high accuracy and high throughput')]))])), 'muse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/muse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'muse'), ('version', '1.0.rc')])), ('source', CommentedMap([('md5', 'c63fdb48c041f6f9545879f1a7e4da58'), ('url', 'https://github.com/danielfan/MuSE/archive/v1.0-rc.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['MuSE 2>&1 | grep Version > /dev/null'])])), ('about', CommentedMap([('home', 'http://bioinformatics.mdanderson.org/main/MuSE'), ('license', 'https://github.com/danielfan/MuSE/blob/master/LICENSE'), ('summary', 'Somatic point mutation caller')]))])), 'music': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/music/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'music'), ('version', '1.0.0')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('url', 'https://github.com/gersteinlab/MUSIC/archive/c5c65c3c6a67f22111234e265a1420f93810d13e.tar.gz'), ('md5', '37391461abf38c5999ee54bae0e7eb74')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'samtools'])])), ('test', CommentedMap([('commands', ['MUSIC  -v'])])), ('about', CommentedMap([('home', 'http://music.gersteinlab.org'), ('license', 'academic'), ('summary', 'MUltiScale enrIchment Calling for ChIP-Seq Datasets')]))])), 'musicc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/musicc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'musicc'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'MUSiCC-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/M/MUSiCC/MUSiCC-1.0.1.tar.gz'), ('md5', '270b23e925c6673643af8610d63070d4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.6.1', 'scipy >=0.9', 'scikit-learn >=0.15.2', 'pandas >=0.14']), ('run', ['python', 'numpy >=1.6.1', 'scipy >=0.9', 'scikit-learn >=0.15.2', 'pandas >=0.14'])])), ('test', CommentedMap([('imports', ['musicc'])])), ('about', CommentedMap([('home', 'http://elbo.gs.washington.edu/software_musicc.html'), ('license', 'BSD License'), ('summary', 'MUSICC: A marker genes based framework for metagenomic normalization and accurate profiling of gene abundances in the microbiome.')]))])), 'mvicuna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mvicuna/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/vicuna'), ('license', 'https://www.broadinstitute.org/node/4238'), ('summary', 'M-Vicuna is a modularized version of VICUNA, a de novo assembly program targeting populations with high mutation rates')])), ('package', CommentedMap([('name', 'mvicuna'), ('version', '1.0')])), ('source', CommentedMap([('git_url', 'https://github.com/broadinstitute/mvicuna.git')])), ('build', CommentedMap([('number', 4)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mvicuna -h | grep \"Parameters\" &> /dev/null'])]))])), 'mygene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mygene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mygene'), ('version', '3.0.0')])), ('source', CommentedMap([('fn', 'mygene-3.0.0.tar.gz'), ('url', 'https://pypi.python.org/packages/53/94/6c9975eef99f64cecef912de80615223abfb6dab804bc600b4fe1104eee8/mygene-3.0.0.tar.gz'), ('md5', 'cf1e1182619c7506bf15d0d4f88badf2')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.3.0']), ('run', ['python', 'requests >=2.3.0'])])), ('test', CommentedMap([('imports', ['mygene'])])), ('about', CommentedMap([('home', 'https://github.com/suLab/mygene.py'), ('license', 'BSD License'), ('summary', 'Python Client for MyGene.Info services.'), ('license_family', 'BSD')]))])), 'myriad': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/myriad/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'myriad'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'myriad-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/fc/10/280fdc6b6989d7ee559bc79d7060e911af7d7f7b99cbc6c01d6a10f8dc49/myriad-0.1.3.tar.gz'), ('md5', 'd1114c7bb9fce8eff89364e4def99036')])), ('build', CommentedMap([('skip', True), ('entry_points', ['myriad = myriad.components:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['myriad']), ('commands', ['myriad --help'])])), ('about', CommentedMap([('home', 'https://github.com/cjw85/myriad'), ('license', 'MIT License'), ('summary', 'Simple distributed computing.'), ('license_family', 'MIT')]))])), 'mysql-connector-c': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mysql-connector-c/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mysql-connector-c'), ('version', '6.1.6')])), ('source', CommentedMap([('fn', 'mysql-connector-c-6.1.6-src.tar.gz'), ('url', 'http://ftp.iij.ad.jp/pub/db/mysql/Downloads/Connector-C/mysql-connector-c-6.1.6-src.tar.gz'), ('md5', 'cf0190bace2217d9e6d22e9e4783ae1e')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake >=2.6']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mysql_config --version', 'my_print_defaults --version'])])), ('about', CommentedMap([('home', 'https://dev.mysql.com/downloads/connector/c/'), ('license', 'LGPL'), ('summary', 'MySQL Connector/C, the C interface for communicating with MySQL servers.')]))])), 'mysqlclient': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mysqlclient/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/m/mysqlclient/<{ name }}-<{ version }}.tar.gz'), ('md5', 'e7fb95c4055e2d8a3322db5c85ab6fc8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'mysql-connector-c']), ('run', ['python', 'mysql-connector-c'])])), ('test', CommentedMap([('imports', ['MySQLdb'])])), ('about', CommentedMap([('home', 'https://github.com/PyMySQL/mysqlclient-python'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Python interface to MySQL')]))])), 'mz_to_sqlite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mz_to_sqlite/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/jj-umn/mztosqlite'), ('license', 'GNU-GPL, Version 2.0'), ('summary', 'mzToSQLite extracts data from proteomics mzIdentML and mass spec scan files: mzML, MGF, etc and \\nstores in a SQLite database.The intended purpose is to provide a \\nGalaxy dataset that can support an interactive Galaxy visualization plugin. \\n')])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.jar'), ('url', 'https://github.com/galaxyproteomics/mzToSQLite/releases/download/<{ version }}/<{ name }}-<{ version }}.jar'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['mz_to_sqlite --help'])])), ('extra', CommentedMap([('notes', 'mzToSQLite extracts data from proteomics mzIdentML and mass spec scan files: mzML, MGF, etc and \\nstores in a SQLite database.The intended purpose is to provide a \\nGalaxy dataset that can support an interactive Galaxy visualization plugin.\\n')]))])), 'mzml2isa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mzml2isa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mzml2isa'), ('version', '0.4.24')])), ('source', CommentedMap([('fn', 'mzml2isa-0.4.24.tar.bz2'), ('url', 'https://pypi.python.org/packages/dd/03/bea5554f2cba1a52f26fc497a775e3c078d5efff0f489e5d948a5b6c24b6/mzml2isa-0.4.24.tar.bz2'), ('md5', '01c71fae82689807f0126c2b7bcf986f')])), ('build', CommentedMap([('skip', True), ('number', 1), ('entry_points', ['mzml2isa = mzml2isa.parsing:run'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'lxml', 'six', 'pronto', 'chainmap']), ('run', ['python', 'lxml', 'six', 'pronto', 'chainmap'])])), ('test', CommentedMap([('imports', ['mzml2isa']), ('commands', ['mzml2isa --help'])])), ('about', CommentedMap([('home', 'http://www.biosciences.bham.ac.uk/labs/viant/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'mzml2isa - mzML to ISA-tab parsing tool')]))])), 'nanoblaster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoblaster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanoblaster'), ('version', '0.16')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'nanoblaster_0.16.tar.gz'), ('url', 'https://github.com/ruhulsbu/NanoBLASTer/archive/417c48c8ae8d824ba591922c20b0e4ec09b738cb.tar.gz'), ('md5', '020d10c335691829308c9c8a0ff272bf')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['nanoblaster -h  2>&1 | grep \"Use the following\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ruhulsbu/NanoBLASTer'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Basic Local Alignment and Search Tool for Oxford Nanopore Long Sequences')]))])), 'nanocall': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanocall/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanocall'), ('version', 'v0.7.4')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('git_url', 'https://github.com/mateidavid/nanocall.git'), ('git_rev', 'v0.7.4')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'hdf5']), ('run', ['libgcc', 'hdf5'])])), ('test', CommentedMap([('commands', ['nanocall --version'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/nanocall'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'An Oxford Nanopore Basecaller')]))])), 'nanofilt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanofilt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanofilt'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'NanoFilt-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanofilt/NanoFilt-<{ version }}.tar.gz'), ('md5', 'b77efe9323171feb82b044bd4a5f6db0')])), ('build', CommentedMap([('entry_points', ['NanoFilt=nanofilt.NanoFilt:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'nanomath >=0.12.5', 'nanoget >=0.11.5']), ('run', ['python', 'biopython', 'nanomath >=0.12.5', 'nanoget >=0.11.5'])])), ('test', CommentedMap([('imports', ['nanofilt']), ('commands', ['NanoFilt --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanofilt'), ('license', 'MIT License'), ('summary', 'Filtering and trimming of Oxford Nanopore Sequencing data'), ('license_family', 'MIT')]))])), 'nanoget': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoget/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanoget'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'nanoget-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanoget/nanoget-<{ version }}.tar.gz'), ('md5', '8a820ed4ce1d3b0af93905f50604efa1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pandas', 'numpy', 'biopython', 'pysam >0.10.0.0', 'nanomath']), ('run', ['python', 'pandas', 'numpy', 'biopython', 'pysam >0.10.0.0', 'nanomath'])])), ('test', CommentedMap([('imports', ['nanoget'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanoget'), ('license', 'MIT License'), ('summary', 'Functions to extract information from Oxford Nanopore sequencing data and alignments.'), ('license_family', 'MIT')]))])), 'nanolyse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanolyse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanolyse'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'NanoLyse-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanolyse/NanoLyse-<{ version }}.tar.gz'), ('md5', '277a9c83735f318a31a3c1480eee636c')])), ('build', CommentedMap([('entry_points', ['NanoLyse=nanolyse.NanoLyse:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'mappy']), ('run', ['python', 'biopython', 'mappy'])])), ('test', CommentedMap([('imports', ['nanolyse']), ('commands', ['NanoLyse --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/NanoLyse'), ('license', 'MIT License'), ('summary', 'Removing lambda DNA control reads from fastq dataset'), ('license_family', 'MIT')]))])), 'nanomath': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanomath/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanomath'), ('version', '0.12.5')])), ('source', CommentedMap([('fn', 'nanomath-0.12.5.tar.gz'), ('url', 'https://pypi.python.org/packages/73/b3/065e700ea8c21d5c74a5ec1ce0bcb6ff1d4115317d71e6060c0150735a3b/nanomath-0.12.5.tar.gz'), ('md5', '7fb1d454f2cd0d3a9a74597667242fb3')])), ('build', CommentedMap([('build', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pandas', 'numpy >1.8']), ('run', ['python', 'pandas', 'numpy >1.8', 'libgcc'])])), ('test', CommentedMap([('imports', ['nanomath'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanomath'), ('license', 'MIT License'), ('summary', 'A few simple math function for other Oxford Nanopore processing scripts'), ('license_family', 'MIT')]))])), 'nanonet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanonet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanonet'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'nanonet_<{ version }}.tar.gz'), ('url', 'https://github.com/nanoporetech/nanonet/archive/v2.0.0.tar.gz'), ('md5', 'bc3e34049a2af62edc1d7cea6fb6d1ba'), ('patches', ['nanonet.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'numpy', 'cython', 'h5py', 'watchdog', 'hdf5', 'myriad >=0.1.2', 'six', 'boost <{CONDA_BOOST}}*', 'pyyaml >=3.10', 'argh', 'pathtools >=0.1.1']), ('run', ['libgcc', 'python', 'numpy', 'cython', 'h5py', 'watchdog', 'hdf5', 'myriad >=0.1.2', 'six', 'boost <{CONDA_BOOST}}*', 'pyyaml >=3.10', 'argh', 'pathtools >=0.1.1'])])), ('test', CommentedMap([('commands', ['nanonetcall --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/nanoporetech/nanonet'), ('license', 'MPL-2.0'), ('summary', 'Nanonet provides recurrent neural network basecalling for Oxford Nanopore MinION data.')]))])), 'nanoplot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanoplot'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'NanoPlot-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanoplot/NanoPlot-<{ version }}.tar.gz'), ('md5', 'a04c00f6c6c853c658d57b5b73687869')])), ('build', CommentedMap([('entry_points', ['NanoPlot=nanoplot.NanoPlot:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'pysam >0.10.0.0', 'pandas', 'numpy', 'scipy', 'matplotlib', 'python-dateutil', 'seaborn', 'nanoplotter >=0.13.1', 'nanoget >=0.11.0', 'nanomath >=0.12.0']), ('run', ['python', 'biopython', 'pysam >0.10.0.0', 'pandas', 'numpy', 'scipy', 'matplotlib', 'python-dateutil', 'seaborn', 'nanoplotter >=0.13.1', 'nanoget >=0.11.0', 'nanomath >=0.12.0'])])), ('test', CommentedMap([('imports', ['nanoplot']), ('commands', ['NanoPlot --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/NanoPlot'), ('license', 'MIT License'), ('summary', 'Plotting suite for Oxford Nanopore sequencing data and alignments'), ('license_family', 'MIT')]))])), 'nanoplotter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoplotter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanoplotter'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'nanoplotter-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanoplotter/nanoplotter-<{ version }}.tar.gz'), ('md5', 'cc73355e88930c02708e3a0f8dca0644')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pandas', 'numpy', 'scipy', 'matplotlib >=2.0.0', 'seaborn']), ('run', ['python', 'pandas', 'numpy', 'scipy', 'matplotlib >=2.0.0', 'seaborn'])])), ('test', CommentedMap([('imports', ['nanoplotter'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanoplotter'), ('license', 'MIT License'), ('summary', 'Plotting functions of Oxford Nanopore sequencing data'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'nanopolish': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanopolish/0.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanopolish'), ('version', '0.4.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'nanopolish_v0.4.0.tar.gz'), ('url', 'https://github.com/jts/nanopolish/archive/v0.4.0.tar.gz'), ('md5', 'b5aa0325568d1e3089162e9e3e54c6c2'), ('patches', ['nanopolish.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'fast5', 'htslib', 'hdf5']), ('run', ['libgcc', 'fast5', 'htslib', 'hdf5', 'biopython'])])), ('test', CommentedMap([('commands', ['nanopolish --help  2>&1 | grep usage  > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/jts/nanopolish'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Signal-level algorithms for MinION data')]))])), 'nanoqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanoqc'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'nanoQC-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/5c/d6/6a803fdeb9937a20b6a08b17b25a297f473d56cf6748cb955208bcb7d7e7/nanoQC-0.3.3.tar.gz'), ('md5', 'c0148697f22a2e8d5a0521b7b2505af9')])), ('build', CommentedMap([('entry_points', ['nanoQC=nanoQC.nanoQC:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'seaborn', 'numpy', 'matplotlib >=2.0.0', 'biopython']), ('run', ['python', 'seaborn', 'numpy', 'matplotlib >=2.0.0', 'biopython'])])), ('test', CommentedMap([('imports', ['nanoQC']), ('commands', ['nanoQC --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanoQC'), ('license', 'MIT License'), ('summary', 'Create fastQC-like plots for Oxford Nanopore sequencing data'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'nanoraw': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanoraw/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/marcus1487/nanoraw/archive/<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('entry_points', ['nanoraw = nanoraw.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'h5py', 'rpy2 >=2.4.2', 'numpy', 'scipy', 'r-base', 'r-ggplot2', 'r-cowplot', 'hdf5', 'graphmap', 'bwa']), ('run', ['python', 'h5py', 'rpy2 >=2.4.2', 'numpy', 'scipy', 'r-base', 'r-ggplot2', 'r-cowplot', 'hdf5', 'graphmap', 'bwa'])])), ('test', CommentedMap([('imports', ['nanoraw']), ('commands', ['nanoraw --help'])])), ('about', CommentedMap([('home', 'https://github.com/marcus1487/nanoraw'), ('license', 'Other'), ('license_file', 'LICENSE.txt'), ('summary', 'Analysis of nanopore sequencing data.')]))])), 'nanosim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanosim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanosim'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'nanosim-<{ version }}.tar.gz'), ('url', 'https://github.com/bcgsc/NanoSim/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'last', 'r-base']), ('run', ['python', 'numpy', 'last', 'r-base'])])), ('test', CommentedMap([('commands', ['read_analysis.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/bcgsc/NanoSim'), ('license', 'GPL'), ('summary', 'NanoSim is a fast and scalable read simulator that captures the technology-specific features of ONT data, and allows for adjustments upon improvement of nanopore sequencing technology.')]))])), 'nanosim-h': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanosim-h/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanosim-h'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'NanoSim-H-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/N/NanoSim-H/NanoSim-H-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('script', 'python setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'last', 'r-base', 'progressbar2']), ('run', ['python', 'numpy', 'last', 'r-base', 'progressbar2'])])), ('test', CommentedMap([('commands', ['nanosim-h -h', 'nanosim-h-train -h', 'curl \"https://www.ncbi.nlm.nih.gov/sviewer/viewer.fcgi?db=nuccore&dopt=fasta&val=545778205&sendto=on\" | nanosim-h -n 100 -'])])), ('about', CommentedMap([('home', 'https://github.com/karel-brinda/NanoSim-H'), ('license', 'GPLv3'), ('summary', 'NanoSim-H is a simulator of Oxford Nanopore reads that captures the technology-specific features of ONT data, and allows for adjustments upon improvement of Nanopore sequencing technology.')]))])), 'nanosplit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanosplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanosplit'), ('version', '0.1.4')])), ('source', CommentedMap([('fn', 'NanoSplit-0.1.4.tar.gz'), ('url', 'https://pypi.python.org/packages/38/7f/cba70434da0dced7319a980597e85d808347b654de74169713dc631b022c/NanoSplit-0.1.4.tar.gz'), ('md5', '8491b0adb87b522cad9625b3ac492da8')])), ('build', CommentedMap([('entry_points', ['NanoSplit=nanosplit.NanoSplit:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'nanomath', 'nanoget', 'pypandoc']), ('run', ['python', 'biopython', 'nanomath', 'nanoget', 'pypandoc'])])), ('test', CommentedMap([('imports', ['nanosplit']), ('commands', ['NanoSplit --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanosplit'), ('license', 'MIT License'), ('summary', 'Perform splitting of Oxford Nanopore sequencing data in a fail and pass dataset.'), ('license_family', 'MIT')]))])), 'nanostat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanostat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nanostat'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'NanoStat-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/nanostat/NanoStat-<{ version }}.tar.gz'), ('md5', '635b444c4699c8156599b8e90cacd931')])), ('build', CommentedMap([('entry_points', ['NanoStat=nanostat.NanoStat:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nanoget >=0.10.1', 'nanomath >=0.12.0']), ('run', ['python', 'nanoget >=0.10.1', 'nanomath >=0.12.0'])])), ('test', CommentedMap([('imports', ['nanostat']), ('commands', ['NanoStat --help'])])), ('about', CommentedMap([('home', 'https://github.com/wdecoster/nanostat'), ('license', 'MIT License'), ('summary', 'Calculate statistics for Oxford Nanopore sequencing data and alignments'), ('license_family', 'MIT')]))])), 'nanosv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nanosv/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/mroosmalen/nanosv/archive/57d1f824a64c113e11041ae33f56cd21c61b5fae.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['nanosv.pl -h'])])), ('about', CommentedMap([('home', 'https://github.com/mroosmalen/nanosv'), ('license', 'MIT'), ('summary', 'SV caller for nanopore data')]))])), 'nasm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nasm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nasm'), ('version', '2.11.08')])), ('source', CommentedMap([('fn', 'nasm-2.11.08.tar.gz'), ('url', 'http://www.nasm.us/pub/nasm/releasebuilds/2.11.08/nasm-2.11.08.tar.gz'), ('md5', '6f94c4d8945e9fa6c0790a2e485090d7')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['nasm -v > /dev/null', 'ndisasm -v > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'http://www.nasm.us'), ('license', 'Simplified BSD License'), ('license_file', 'LICENSE'), ('summary', 'NASM, the Netwide Assembler.')]))])), 'nasp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nasp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nasp'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'nasp-<{ version }}.tar.gz'), ('url', 'https://github.com/TGenNorth/NASP/archive/00d22ec67cb513a5df3a0216b0537803e494505e.tar.gz'), ('md5', 'cc220411742d7871875c38d1348e2fa0')])), ('build', CommentedMap([('skip', True), ('entry_points', ['nasp = nasp.nasp:main', 'format_fasta = nasp.format_fasta:main', 'find_duplicates = nasp.find_duplicates:main', 'convert_external_genome = nasp.convert_external_genome:main', 'vcf_to_matrix = nasp.vcf_to_matrix:main']), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'samtools <1.3', 'trimmomatic', 'mummer >=3.23'])])), ('test', CommentedMap([('imports', ['nasp']), ('commands', ['nasp --help', 'format_fasta --help', 'find_duplicates --help', 'convert_external_genome --help', 'vcf_to_matrix --help'])])), ('about', CommentedMap([('home', 'https://github.com/TGenNorth/nasp'), ('license', 'Academic and Research License'), ('summary', 'The Northern Arizona SNP Pipeline (NASP)')]))])), 'ncbi-genome-download': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ncbi-genome-download/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/n/ncbi-genome-download/ncbi-genome-download-<{ version }}.tar.gz'), ('md5', '127909bdbb7146fb232a08ddaf8fd885')])), ('build', CommentedMap([('entry_points', ['ncbi-genome-download=ncbi_genome_download.__main__:main']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.4.3', 'enum34']), ('run', ['python', 'requests >=2.4.3', 'enum34'])])), ('test', CommentedMap([('imports', ['ncbi_genome_download']), ('commands', ['ncbi-genome-download --help']), ('requires', ['coverage', 'pytest', 'pytest-cov', 'pytest-mock', 'requests-mock'])])), ('about', CommentedMap([('home', 'https://github.com/kblin/ncbi-genome-download/'), ('license', 'Apache Software License'), ('summary', 'Download genome files from the NCBI FTP server.'), ('license_family', 'Apache')]))])), 'neptune': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/neptune/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'neptune'), ('version', '1.2.5')])), ('source', CommentedMap([('url', 'https://github.com/phac-nml/neptune/archive/v1.2.5.tar.gz'), ('md5', 'ed48846deffefe25784c9f8b7db2de6e')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'drmaa', 'numpy', 'scipy', 'biopython', 'blast']), ('run', ['python', 'drmaa', 'numpy', 'scipy', 'biopython', 'blast'])])), ('test', CommentedMap([('commands', [\"neptune -h 2>&1 | grep 'usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/phac-nml/neptune'), ('license', 'Apache 2.0'), ('summary', 'Neptune: Genomic Signature Discovery')]))])), 'netifaces': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/netifaces/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'netifaces'), ('version', '0.10.4')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'netifaces-0.10.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/n/netifaces/netifaces-0.10.4.tar.gz'), ('md5', '36da76e2cfadd24cc7510c2c0012eb1e')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['netifaces'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/al45tair/netifaces'), ('license', 'MIT')]))])), 'netreg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/netreg/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/dirmeier/netReg'), ('license', 'GPL3'), ('summary', 'netReg fits linear regression models using network-penalization.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'netreg'), ('version', '1.0.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libcxx', 'cmake >=3.8', 'boost ==1.63.0', 'armadillo >=7.800.1', 'openblas >=0.2.19', 'zlib']), ('run', ['libgcc', 'libcxx', 'boost ==1.63.0', 'armadillo >=7.800.1', 'openblas >=0.2.19', 'zlib'])])), ('source', CommentedMap([('fn', 'netreg-1.0.0.tar.gz'), ('sha256', '72d5cfcfd291ff6bd7f31782b3fab0077e93cd242fc102e6a22f68f7d364c349'), ('url', 'https://github.com/dirmeier/netReg/archive/v1.0.0.tar.gz')])), ('test', CommentedMap([('commands', ['netReg -h'])]))])), 'networkxgmml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/networkxgmml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'networkxgmml'), ('version', '0.1.6')])), ('source', CommentedMap([('fn', 'networkxgmml-0.1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/n/networkxgmml/networkxgmml-0.1.6.tar.gz'), ('md5', '7a272461cf96cf6c2cebd06f2ab2ee4f')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'networkx']), ('run', ['python', 'setuptools', 'networkx'])])), ('about', CommentedMap([('home', 'https://github.com/informationsea/networkxxgmml'), ('license', 'UNKNOWN'), ('summary', 'XGMML parser for networkx')]))])), 'newick_utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/newick_utils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'newick_utils'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('url', 'https://github.com/tjunier/newick_utils/archive/<{ revision }}.tar.gz'), ('fn', 'newick_utils-<{ revision }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libxml2 2.9.*', 'autoconf', 'automake', 'libtool', 'bison', 'flex']), ('run', ['libgcc', 'libxml2 2.9.*'])])), ('test', CommentedMap([('commands', ['nw_clade -h', 'nw_condense -h', 'nw_distance -h', 'nw_duration -h', 'nw_ed -h', 'nw_gen -h', 'nw_indent -h', 'nw_labels -h', 'nw_match -h', 'nw_order -h', 'nw_prune -h', 'nw_rename -h', 'nw_reroot -h', 'nw_stats -h', 'nw_support -h', 'nw_topology -h', 'nw_trim -h', 'nw_display -h'])])), ('about', CommentedMap([('home', 'http://cegg.unige.ch/newick_utils'), ('license', 'BSD License'), ('summary', 'The Newick Utilities are a suite of Unix shell tools for processing phylogenetic trees. We distribute the package under the BSD License. Functions include re-rooting, extracting subtrees, trimming, pruning, condensing, drawing (ASCII graphics or SVG).')]))])), 'newt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/newt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'newt'), ('version', '0.52.18')])), ('source', CommentedMap([('fn', 'newt-0.52.18.tar.gz'), ('url', 'http://fedorahosted.org/releases/n/e/newt/newt-0.52.18.tar.gz'), ('md5', '685721bee1a318570704b19dcf31d268')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['popt', 'slang', 'python']), ('run', ['popt', 'slang', 'python'])])), ('test', CommentedMap([('commands', ['whiptail --help']), ('imports', ['snack'])])), ('about', CommentedMap([('home', 'https://fedorahosted.org/newt/'), ('license', 'GPL')]))])), 'nextflow': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nextflow/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nextflow'), ('version', '0.25.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'nextflow'), ('url', 'https://www.nextflow.io/releases/v0.25.1/nextflow'), ('md5', '8924b4ccdc3117112c6825f6dc307fe9')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0', 'coreutils', 'curl']), ('run', ['openjdk >=7.0', 'coreutils', 'curl'])])), ('test', CommentedMap([('commands', ['nextflow -v'])])), ('about', CommentedMap([('home', 'https://github.com/nextflow-io/nextflow'), ('license', 'GPL3'), ('summary', 'A DSL for data-driven computational pipelines http://nextflow.io')]))])), 'nextgenmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nextgenmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nextgenmap'), ('version', '0.5.3')])), ('source', CommentedMap([('fn', 'ngm-v0.5.3.tar.gz'), ('url', 'https://github.com/Cibiv/NextGenMap/archive/v0.5.3.tar.gz'), ('md5', '6416f0e581338266f0520bbbaeb3a20c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['ngm 2>&1 | grep \"NextGenMap\"', 'ngm-utils 2>&1 | grep \"ngm-utils\"'])])), ('about', CommentedMap([('home', 'https://github.com/philres/NextGenMap'), ('license', 'Artistic License (Perl) 1.0'), ('summary', 'NextGenMap is a flexible highly sensitive short read mapping tool that handles much higher mismatch rates than comparable algorithms while still outperforming them in terms of runtime.')]))])), 'nglview': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nglview/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('script', 'python setup.py install --conda --single-version-externally-managed --record record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'libgcc', 'ipywidgets >=<{ minimum_ipywidgets_version }},<6', 'notebook'])])), ('test', CommentedMap([('imports', ['nglview', 'notebook'])])), ('about', CommentedMap([('home', 'http://github.com/arose/nglview'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'An IPython widget to interactively view molecular structures and trajectories. Utilizes the embeddable NGL Viewer for rendering.'), ('doc_url', 'https://arose.github.io/nglview/latest/'), ('dev_url', 'https://github.com/arose/nglview')])), ('extra', CommentedMap([('recipe-maintainers', ['hainm'])]))])), 'ngmlr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngmlr/0.2.4/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/philres/ngmlr'), ('license', 'MIT'), ('summary', 'ngmlr is a long-read mapper designed to align PacBio or Oxford Nanopore reads to a reference genome and optimized for structural variation detection')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'ngmlr'), ('version', '0.2.4')])), ('requirements', CommentedMap([('build', ['gcc >=4.8.2', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('source', CommentedMap([('fn', 'ngmlr-v0.2.4.tar.gz'), ('md5', '476259d0e8e7b79f3eb911329189a226'), ('url', 'https://github.com/philres/ngmlr/archive/v0.2.4.tar.gz')])), ('test', CommentedMap([('commands', ['ngmlr --help'])]))])), 'ngs-disambiguate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngs-disambiguate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngs-disambiguate'), ('version', '2016.11.10')])), ('source', CommentedMap([('git_url', 'https://github.com/AstraZeneca-NGS/disambiguate'), ('git_tag', '559452477737ea10d20c582aa9a3af880b197bd9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'bamtools']), ('run', ['libgcc', 'bamtools'])])), ('test', CommentedMap([('commands', ['ngs_disambiguate --help'])])), ('about', CommentedMap([('home', 'https://github.com/AstraZeneca-NGS/disambiguate'), ('license', 'MIT'), ('summary', 'Disambiguation algorithm for reads aligned to human and mouse genomes using Tophat or BWA mem')]))])), 'ngseqbasic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngseqbasic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngseqbasic'), ('version', '2.0.1')])), ('source', CommentedMap([('fn', 'NGseqBasic.tar.gz'), ('url', 'https://github.com/telenius/NGseqBasic/raw/master/NGseqBasic.tar.gz'), ('md5', '3992a5c4ed260e548db9b47cad3009be')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('run', ['samtools ==0.1.19', 'bowtie ==1.0.0', 'bowtie2 ==2.3.0', 'bedtools ==2.17.0', 'ucsc-bedgraphtobigwig ==332', 'ucsc-bedclip ==332', 'ucsc-bedgraphpack ==332', 'ucsc-bedtobigbed ==332', 'flash ==1.2.11', 'trim-galore ==0.4.1', 'perl 5.22.0.1'])])), ('test', CommentedMap([('commands', ['find $PREFIX/share -iname \"*testEnvironment.sh*\" -exec {} \\\\; >& /dev/null', 'NGseqBasic --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://userweb.molbiol.ox.ac.uk/public/telenius/NGseqBasicManual/outHouseUsers/instructionsBioconda.html'), ('license', 'GPL-3.0'), ('summary', 'Basic ChIP/DNaseI/ATAC analysis - from FASTQ to visualisation of tracks, in one command.')]))])), 'ngsutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngsutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngsutils'), ('version', '0.5.9')])), ('source', CommentedMap([('url', 'https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz'), ('md5', '6212d6de740c42aa2c6b5f6785465860')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign']), ('run', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign'])])), ('test', CommentedMap([('commands', ['python -m unittest discover ngsutils'])])), ('about', CommentedMap([('home', 'http://ngsutils.org'), ('license', 'BSD'), ('license_file', 'LICENSE'), ('summary', 'Tools for next-generation sequencing analysis http://ngsutils.org')]))])), 'nlstradamus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nlstradamus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nlstradamus'), ('version', 1.8)])), ('about', CommentedMap([('home', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'NLStradamus: a simple Hidden Markov Model for nuclear localization signal prediction.')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'NLStradamus.1.8.tar.gz'), ('url', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/NLStradamus/NLStradamus.1.8.tar.gz'), ('sha256', '73ef3a77c6c1132f213d44bc2a48a8182e652b01b8cd83d06b30832a682ec0d0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['NLStradamus -h'])]))])), 'nmrglue': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nmrglue/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nmrglue'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'nmrglue-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/48/1e/be85936901a3a82a1dc14a8b41017ca86dcb8b966717c786e361e752c898/nmrglue-0.5.tar.gz'), ('md5', '4cf63caceae0b9f0a144f07b6663e670')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'numpy', 'scipy']), ('run', ['libgcc', 'python', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['nmrglue', 'nmrglue.analysis', 'nmrglue.fileio', 'nmrglue.process', 'nmrglue.process.nmrtxt', 'nmrglue.util'])])), ('about', CommentedMap([('home', 'http://www.nmrglue.com'), ('license', 'BSD License'), ('summary', 'A module for working with NMR data in Python'), ('license_family', 'BSD')]))])), 'nmrml2isa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nmrml2isa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nmrml2isa'), ('version', '0.3.0')])), ('source', CommentedMap([('fn', 'nmrml2isa-0.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/61/87/eb7ed3b5dfcdaa5109355daf7ccab722ce984cf49283bead437d6a51cd8a/nmrml2isa-0.3.0.tar.gz'), ('md5', '3f56e532da5ab8d7ee62bd32c1d21253')])), ('build', CommentedMap([('skip', True), ('number', 0), ('entry_points', ['nmrml2isa = nmrml2isa.parsing:run'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'lxml', 'six', 'pronto', 'chainmap']), ('run', ['python', 'lxml', 'six', 'pronto', 'chainmap'])])), ('test', CommentedMap([('imports', ['nmrml2isa']), ('commands', ['nmrml2isa --help'])])), ('about', CommentedMap([('home', 'http://github.com/althonos/nmrml2isa'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'nmrml2isa - nmrML to ISA-Tab parsing tool')]))])), 'nmrpro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nmrpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nmrpro'), ('version', 20161019)])), ('source', CommentedMap([('fn', 'nmrpro-20161019.tar.gz'), ('url', 'https://github.com/ahmohamed/nmrpro/archive/a11111965bd701e1d208673e8435a0c45df3ba36.tar.gz'), ('md5', '2a233d60fd8a175b602b0241e0bc679d'), ('patches', ['nmrpro_setup.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'nmrglue >=0.5']), ('run', ['python', 'numpy', 'scipy', 'nmrglue >=0.5'])])), ('test', CommentedMap([('imports', ['nmrpro'])])), ('about', CommentedMap([('home', 'https://github.com/ahmohamed/nmrpro'), ('license', 'MIT'), ('summary', 'NMRPro reads and processes different types of NMR spectra.')]))])), 'nonpareil': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nonpareil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nonpareil'), ('version', '2.4.01')])), ('source', CommentedMap([('git_url', 'https://github.com/lmrodriguezr/nonpareil.git'), ('git_rev', 'v2.4.01')])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'm4', 'gcc', 'r-base']), ('run', ['r-base'])])), ('build', CommentedMap([('skip', False)])), ('test', CommentedMap([('commands', ['nonpareil -V > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lmrodriguezr/nonpareil'), ('license', 'Artistic License 2.0'), ('summary', 'Estimate average coverage and create Nonpareil curves for metagenomic datasets.')]))])), 'norns': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/norns/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'norns'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'norns-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/c4/74/e151195a1986c6f3c795810de395d91dbe8d80365758d6a09d7dd7d5b35f/norns-0.1.2.tar.gz'), ('md5', 'd39a21f7bc06b3ecda43b33c949e5de4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'appdirs', 'pyyaml']), ('run', ['python', 'nose', 'appdirs', 'pyyaml'])])), ('test', CommentedMap([('imports', ['norns'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/norns'), ('license', 'MIT License'), ('summary', 'Simple yaml-based config module'), ('license_family', 'MIT')]))])), 'nose-capturestderr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nose-capturestderr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nose-capturestderr'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'nose-capturestderr-1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/92/a2/2798b7596040b90217bde9d578085ff652523f69e4de1f2622e8716f9649/nose-capturestderr-1.0.tar.gz'), ('md5', '498707ee990533b77e3ffb0265e6802b')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose >=0.11.1']), ('run', ['python', 'setuptools', 'nose >=0.11.1'])])), ('test', CommentedMap([('imports', ['nose_capturestderr'])])), ('about', CommentedMap([('home', 'http://github.com/sio2project/nose-capturestderr'), ('license', 'GPL'), ('summary', 'Nose plugin for capturing stderr.')]))])), 'novoalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/novoalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'novoalign'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'novocraftV<{ version }}.tar.gz'), ('url', 'http://www.novocraft.com/homebrew/novocraftV<{ version }}.MacOSX.tar.gz'), ('md5', '1a0f98015fe0bf7c8e135132063b312f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['novoalign 2>&1 | grep ^Novoalign'])])), ('about', CommentedMap([('home', 'http://www.novocraft.com/products/novoalign/'), ('license', 'Commercial (requires license for multithreaded use)'), ('summary', 'Powerful tool designed for mapping of short reads onto a reference genome from Illumina, Ion Torrent, and 454 NGS platforms')])), ('extra', CommentedMap([('notes', 'Novoalign runs in single-threaded mode by default unless a \"novoalign.lic\" license file is provided in the same directory as its binaries. The license file can be copied in to the conda environment via the \"novoalign-license-register\" command.')]))])), 'novoplasty': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/novoplasty/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/ndierckx/NOVOPlasty/archive/2.2.2.tar.gz'), ('md5', '980a95dba3d36f3e9d78486d5d002e3f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl', 'perl-app-cpanminus', 'perl-module-build'])])), ('test', CommentedMap([('commands', ['NOVOPlasty2.2.2.pl  2>&1 | grep \"Can\\'t open the configuration file, please check the manual!\" > /dev/null', 'NOVOPlasty.pl 2>&1 | grep \"Can\\'t open the configuration file, please check the manual!\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ndierckx/NOVOPlasty'), ('description', 'NOVOPlasty is a de novo assembler for short circular genomes.'), ('license_file', 'LICENSE')]))])), 'np-likeness-scorer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/np-likeness-scorer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/np-likeness/'), ('license', None), ('summary', 'Calculates Natural Product(NP)-likeness of a molecule, i.e. the similarity of the molecule to the structure space covered by known natural products. NP-likeness is a useful criterion to screen compound libraries and to design new lead compounds')])), ('package', CommentedMap([('name', 'np-likeness-scorer'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'Natural_Product_Likeness_calculator_2.1.zip'), ('url', 'https://sourceforge.net/projects/np-likeness/files/latest/download?source=typ_redirect'), ('md5', '5ed2c3096a10403c6d3d0dd80c20be7a')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['np-likeness -help', 'np-likeness'])]))])), 'npyscreen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/npyscreen/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'npyscreen'), ('version', '4.10.5')])), ('source', CommentedMap([('fn', 'npyscreen-4.10.5.tar.gz'), ('url', 'https://pypi.python.org/packages/93/48/91b8321280f17d135918895b57f891f727be84a88f62fc62485a7039de00/npyscreen-4.10.5.tar.gz'), ('md5', '8f2e2e404d87393fbd277db6430508c0')])), ('build', CommentedMap([('number', 0), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['python', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('imports', ['npyscreen', 'npyscreen.compatibility_code'])])), ('about', CommentedMap([('home', 'http://www.npcole.com/npyscreen/'), ('license', 'BSD License'), ('summary', 'Writing user interfaces without all that ugly mucking about in hyperspace'), ('license_family', 'BSD')]))])), 'nspdk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nspdk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nspdk'), ('version', 9.2)])), ('source', CommentedMap([('fn', 'NSPDK-9.2.tar.gz'), ('url', 'https://raw.githubusercontent.com/BackofenLab/GraphClust/master/NSPDK_src/NSPDK-9.2.tar.gz'), ('sha256', '90955549e7d3e268d194cb89a38018a323f050c1c524d35630733f4cd97664a6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['NSPDK -h  2>&1 | grep \"Usage\" >/dev/null'])])), ('about', CommentedMap([('home', 'http://dtai.cs.kuleuven.be/ml/systems/nspdk'), ('license', 'GNUv3'), ('license_file', None), ('summary', 'Neighborhood Subgraph Pairwise Distance Kernel (NSPDK).')]))])), 'ntcard': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ntcard/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/bcgsc/ntCard/archive/<{ version }}.tar.gz'), ('md5', '41d987f7926466d9afd1e845f369f2b9')])), ('requirements', CommentedMap([('build', ['gcc', 'automake', 'autoconf', 'perl', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['ntcard --help'])])), ('about', CommentedMap([('home', 'https://github.com/bcgsc/ntCard'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Estimating k-mer coverage histogram of genomics data')]))])), 'nucleoatac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nucleoatac/0.3.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://nucleoatac.readthedocs.io/en/latest/'), ('license', 'MIT'), ('summary', 'Python package for calling nucleosomes using ATAC-Seq data. Also includes general scripts for working with paired-end ATAC-Seq data (or potentially other paired-end data).'), ('license_family', 'MIT')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/GreenleafLab/NucleoATAC/archive/v<{ version }}.tar.gz'), ('md5', 'c2273553d768cc41fcdf22cd5045c1af')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.22', 'numpy >=1.9.1', 'scipy', 'matplotlib', 'pysam >=0.8.1']), ('run', ['python', 'cython >=0.22', 'numpy >=1.9.1', 'scipy', 'matplotlib', 'pysam >=0.8.1'])])), ('test', CommentedMap([('commands', ['nucleoatac run --help', 'pyatac --help'])]))])), 'nudup': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nudup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nudup'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'v2.3.2.tar.gz'), ('url', 'https://github.com/nugentechnologies/nudup/archive/v2.3.2.tar.gz'), ('sha256', '519cf12ba68508fa0db5eac74020855333e84251ec3ff06dffc41e9a09b57f71')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'samtools >=1.2', 'coreutils', 'sed', 'grep'])])), ('test', CommentedMap([('commands', ['nudup.py -h'])])), ('about', CommentedMap([('home', 'http://nugentechnologies.github.io/nudup/'), ('license', 'GNU Lesser General Public License 3.0'), ('license_file', 'LICENSE.md')]))])), 'oases': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oases/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'oases'), ('version', '0.2.09')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('git_url', 'https://github.com/dzerbino/oases.git'), ('git_rev', '0.2.09')])), ('requirements', CommentedMap([('build', ['gcc', 'openmpi']), ('run', ['libgcc', 'velvet'])])), ('test', CommentedMap([('commands', ['oases | grep \"oases - De novo transcriptome assembler for the Velvet package\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/~zerbino/oases/'), ('license', 'GPL'), ('summary', 'De novo transcriptome assembler for short reads')]))])), 'oauth2client': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oauth2client/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'oauth2client'), ('version', '1.5.2')])), ('source', CommentedMap([('fn', 'oauth2client-1.5.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/o/oauth2client/oauth2client-1.5.2.tar.gz'), ('md5', '842ab2fba8bce5b36d31f630ed80f17d')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'httplib2', 'pyasn1', 'pyasn1-modules', 'rsa', 'six'])])), ('test', CommentedMap([('imports', ['oauth2client', 'oauth2client.client'])])), ('about', CommentedMap([('home', 'https://github.com/google/oauth2client/'), ('license', 'Apache 2.0'), ('summary', 'client library for OAuth 2.0')]))])), 'ococo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ococo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ococo'), ('version', '0.1.2.4')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '0.1.2.4.tar.gz'), ('url', 'https://github.com/karel-brinda/ococo/archive/0.1.2.4.tar.gz'), ('md5', 'e0e5bd2daf71c89820bad2e745b6def6')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib ==1.2.8', 'libgcc']), ('run', ['zlib ==1.2.8', 'libgcc'])])), ('test', CommentedMap([('commands', ['ococo -v'])])), ('about', CommentedMap([('home', 'http://github.com/karel-brinda/ococo'), ('license', 'MIT'), ('summary', 'Ococo, the first online consensus caller.')]))])), 'odose': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/odose/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'odose'), ('version', 1.0)])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/ODoSE/odose.nl/archive/ab0a66d3807f330727c5ed23d939cd2a917e9a09.tar.gz'), ('md5', '1d59b9e73a786cfd1cb6b4c7f2b6cc8a')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'biopython >=1.64', 'matplotlib >=1.4.2', 'mysql-connector-python', 'numpy >=1.9.1', 'poster >=0.8.1', 'rpy2 >=2.8.5', 'orthomcl', 'translatorx', 'paml', 'phylip', 'phipack'])])), ('test', CommentedMap([('commands', ['versions target'])])), ('about', CommentedMap([('home', 'https://github.com/ODoSE/odose.nl'), ('license', 'MIT'), ('licence_file', 'docs/LICENSE.txt'), ('summary', 'Ortholog Direction of Selection Engine.')]))])), 'ogdf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ogdf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ogdf'), ('version', '201207')])), ('source', CommentedMap([('fn', 'sakura-201207.tar.gz'), ('url', 'https://github.com/ogdf/ogdf/archive/sakura-201207.tar.gz'), ('sha256', '2676d0ce8940079dcf7dd7e94ec35b67beb31f86587b33fd94a019e947c43e6e'), ('patches', ['osx.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python >=2.7,<3']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://http://ogdf.net/doku.php'), ('license', 'GPLv3'), ('summary', 'The Open Graph Drawing Framework is a self-contained C++ class library for the automatic layout of diagrams.')]))])), 'oligotyping': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oligotyping/2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'oligotyping'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'oligotyping-2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/o/oligotyping/oligotyping-2.0.tar.gz'), ('md5', '2159126a355bf3429418224ab17d5a56')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib', 'biopython', 'scipy', 'django']), ('run', ['python', 'matplotlib', 'biopython', 'scipy', 'django', 'blast', 'r-vegan', 'r-ggplot2', 'r-gplots', 'r-gtools', 'r-reshape', 'r-optparse', 'r-pheatmap', 'r-rcolorbrewer', 'r-compute.es'])])), ('test', CommentedMap([('imports', ['Oligotyping', 'Oligotyping.lib', 'Oligotyping.utils', 'Oligotyping.utils.html', 'Oligotyping.visualization'])])), ('about', CommentedMap([('home', 'http://oligotyping.org'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'The oligotyping and minimum entropy decomposition (MED) pipeline for the analysis of marker gene amplicons')]))])), 'oncofuse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oncofuse/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'oncofuse-<{ version }}.zip'), ('url', 'https://github.com/mikessh/oncofuse/releases/download/<{ version }}/oncofuse-<{ version }}.zip'), ('sha256', '<{ sha256hash }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['oncofuse -h 2>&1 | grep \"^usage: Oncofuse\"'])])), ('about', CommentedMap([('home', 'https://github.com/mikessh/oncofuse'), ('license', 'Apache v2.0'), ('summary', 'Predicting oncogenic potential of gene fusions')]))])), 'ont-fast5-api': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ont-fast5-api/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ont-fast5-api'), ('version', '0.4.1')])), ('source', CommentedMap([('fn', 'ont-fast5-api-0.4.1.tar.gz'), ('url', 'https://pypi.python.org/packages/a1/89/ce95c0f39ce47dac75ff21be8a8194f0be70e1d9e5ddda8e3a4fc98359b5/ont-fast5-api-0.4.1.tar.gz'), ('md5', 'b5674b9fabfa3d42e18c53108de8dd47')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'h5py', 'numpy >=1.8.1']), ('run', ['python', 'h5py', 'numpy >=1.8.1'])])), ('test', CommentedMap([('imports', ['ont_fast5_api', 'ont_fast5_api.analysis_tools'])])), ('about', CommentedMap([('home', 'https://github.com/nanoporetech/ont_fast5_api'), ('license', 'Mozilla Public License 2.0 (MPL 2.0)'), ('summary', 'Oxford Nanopore Technologies fast5 API software'), ('license_family', 'OTHER')]))])), 'onto2nx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/onto2nx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'onto2nx'), ('version', '0.1.0')])), ('source', CommentedMap([('fn', 'onto2nx-0.1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/ba/c0/d0ee2400caee5b0245c2034df042c214c63c929721d07193c1f9a5187a00/onto2nx-0.1.0.tar.gz'), ('md5', 'edd3797e4dc00aea48266d3da82eda7c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'networkx', 'rdflib', 'rdflib-jsonld', 'sparqlwrapper', 'colorama', 'requests', 'pyfiglet', 'click', 'pygments']), ('run', ['python', 'networkx', 'rdflib', 'rdflib-jsonld', 'sparqlwrapper', 'colorama', 'requests', 'pyfiglet', 'click', 'pygments'])])), ('test', CommentedMap([('imports', ['onto2nx', 'onto2nx.ontospy', 'onto2nx.ontospy.core'])])), ('about', CommentedMap([('home', 'https://github.com/cthoyt/onto2nx'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'A package for parsing ontologies into NetworkX graphs')]))])), 'openmg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openmg/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/openmg'), ('license', 'GNU General Public License'), ('summary', 'Exhaustive generation of chemical structures')])), ('package', CommentedMap([('name', 'openmg'), ('version', 0.1)])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'OMG.jar'), ('url', 'http://downloads.sourceforge.net/project/openmg/OMG.jar'), ('sha256', '214c9ff862aa069752152cea0309df9a2e1854d31799382d32871a807d04ce5a')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6'])])), ('test', CommentedMap([('commands', [\"openmg -h 2>&1 | grep 'Optional Parameters'\"])]))])), 'openms': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openms/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower  }}'), ('version', '<{ version  }}')])), ('source', CommentedMap([('fn', 'OpenMS-<{ version }}_src_contrib_doc.tar.gz'), ('url', 'https://github.com/OpenMS/OpenMS/releases/download/Release2.2.0/OpenMS-<{ version }}-src.tar.gz'), ('sha256', '61e7d24890abb6f462e2f026efb0dc88b84639b228d96e919c8074d3b11d286e')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_boost<{ boost }}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'glib', 'autoconf', 'automake', 'python', 'zlib', 'libtool', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'seqan-library ==1.4.2', 'coinmp']), ('run', ['python', 'libgcc', 'glib', 'zlib', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'coinmp'])])), ('test', CommentedMap([('commands', ['FileMerger --help'])])), ('about', CommentedMap([('home', 'https://github.com/OpenMS/OpenMS'), ('license', 'BSD'), ('summary', 'OpenMS is an open-source software C++ library for LC-MS data management and analyses')])), ('extra', CommentedMap([('notes', 'This package can exceed the travis build time. If so, this package will be build by @bioconda/core locally via\\nsimulate-travis.py and pushed directly. If you need an update for this package please make your changes,\\ncreate a PR and contact @bioconda/core.\\n')]))])), 'openpyxl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openpyxl/1.8.6/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'openpyxl'), ('version', '1.8.6')])), ('source', CommentedMap([('fn', 'openpyxl-1.8.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c2/ae/4ac6659a37bac4c496cc59378b95e5fbbaf3c1d6249827bb7c7516de979a/openpyxl-1.8.6.tar.gz'), ('md5', 'a610d32ca5460eb78df4eb83c896ba34')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['openpyxl', 'openpyxl.reader', 'openpyxl.shared', 'openpyxl.shared.compat', 'openpyxl.styles', 'openpyxl.writer'])])), ('about', CommentedMap([('home', 'http://openpyxl.readthedocs.org'), ('license', 'MIT License'), ('summary', 'A Python library to read/write Excel 2007 xlsx/xlsm files'), ('license_family', 'MIT')]))])), 'openslide': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openslide/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'openslide'), ('version', '<{OPENSLIDE_VERSION}}')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'v<{OPENSLIDE_VERSION}}.zip'), ('url', 'https://github.com/openslide/openslide/archive/v<{OPENSLIDE_VERSION}}.zip'), ('md5', 'fd69a74481a57d6dc188c7a276a09ee7')])), ('requirements', CommentedMap([('build', ['autoconf', 'pkg-config', 'automake', 'perl', 'gcc >=4.7', 'llvm', 'zlib', 'jpeg', 'libtiff >=4', 'openjpeg >=2.1', 'libpng', 'libxml2', 'sqlite >=3.6.2', 'cairo >=1.2', 'glib >=2.48.0', 'gdk-pixbuf', 'libtool', 'xorg-libxrender', 'xorg-libxext', 'xorg-libxau', 'xorg-libxdmcp']), ('run', ['libgcc', 'zlib', 'jpeg', 'libpng', 'libtiff >=4', 'openjpeg >=2.1', 'libxml2', 'sqlite >=3.6.2', 'cairo >=1.2', 'glib >=2.48.0', 'gdk-pixbuf'])])), ('test', CommentedMap([('commands', ['test -f $PREFIX/lib/libopenslide.so', 'test -f $PREFIX/lib/libopenslide.dylib'])])), ('about', CommentedMap([('home', 'http://openslide.org/'), ('license', 'GNU LGPL 2.1'), ('summary', 'OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).')]))])), 'optbuild': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/optbuild/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'optbuild'), ('version', '0.1.11')])), ('source', CommentedMap([('fn', 'optbuild-0.1.11.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/b0/5e/b88949a1e557724fa61a81dfdd54a3067441050624ace665a5ed605b6fd3/optbuild-0.1.11.tar.gz'), ('md5', '7c3aa06d509bff214e715e670db7b615')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'autolog']), ('run', ['python', 'autolog'])])), ('test', CommentedMap([('imports', ['optbuild'])])), ('about', CommentedMap([('home', 'http://noble.gs.washington.edu/~mmh1/software/optbuild/'), ('license', 'GNU GPLv2'), ('summary', 'build command lines for external programs')]))])), 'optitype': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/optitype/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'optitype'), ('version', '1.2.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'OptiType-1.2.1.tar.gz'), ('url', 'https://github.com/FRED-2/OptiType/archive/v1.2.1.tar.gz'), ('md5', '3e3e3e2675d24ce672f667b5c253161c')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy', 'pyomo', 'hdf5', 'pytables', 'pandas', 'pysam', 'matplotlib', 'samtools', 'six', 'future', 'glpk', 'razers3'])])), ('test', CommentedMap([('commands', ['OptiTypePipeline.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/FRED-2/OptiType'), ('license', 'BSD'), ('summary', 'Precision HLA typing from next-generation sequencing data')]))])), 'optplus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/optplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'optplus'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'optplus-0.1.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/1e/93/0e68324166920d816ff434e1cf5f098570029110eaa636a452c00966759c/optplus-0.1.1.tar.gz'), ('md5', '6dcd292165e21888837705f6369e3b1b')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['optplus'])])), ('about', CommentedMap([('home', 'http://noble.gs.washington.edu/~mmh1/software/optplus/'), ('license', 'UNKNOWN'), ('summary', 'additional options for optparse')]))])), 'orfm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/orfm/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/wwood/OrfM/archive/v<{ version }}.tar.gz'), ('md5', '29319b37ca2c066a073fb802bdca9221')])), ('requirements', CommentedMap([('build', ['gnu-wget', 'gcc', 'llvm', 'autoconf', 'automake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['orfm -h'])])), ('about', CommentedMap([('home', 'https://github.com/wwood/OrfM'), ('license', 'GPL3'), ('license_file', 'LICENSE.txt'), ('summary', 'OrfM is a simple and not slow ORF caller')]))])), 'orthofinder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/orthofinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'orthofinder'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'OrthoFinder-<{ version }}_source.tar.gz'), ('url', 'https://github.com/davidemms/OrthoFinder/releases/download/<{ version }}/OrthoFinder-<{ version }}_source.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('run', ['python', 'scipy', 'blast', 'mcl', 'fastme', 'dlcpar'])])), ('test', CommentedMap([('commands', ['orthofinder > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/davidemms/OrthoFinder'), ('summary', 'Accurate inference of orthogroups, orthologues, gene trees and rooted species tree made easy!'), ('license', 'GPLv3')]))])), 'orthomcl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/orthomcl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'orthomcl'), ('version', '2.0.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'orthomclSoftware-v2.0.9.tar.gz'), ('url', 'http://orthomcl.org/common/downloads/software/v2.0/orthomclSoftware-v2.0.9.tar.gz'), ('md5', '2e0202ed4e36a753752c3567edb9bba9'), ('patches', ['orthomcl.patch'])])), ('requirements', CommentedMap([('run', ['perl', 'mcl', 'blast', 'mysqlclient'])])), ('test', CommentedMap([('commands', ['orthomclBlastParser 2>&1 | grep EXAMPLE', 'orthomclFilterFasta 2>&1 | grep EXAMPLE', 'orthomclReduceFasta 2>&1 | grep EXAMPLE', 'orthomclReduceGroups 2>&1 | grep EXAMPLE', 'orthomclSingletons 2>&1 | grep EXAMPLE', 'orthomclAdjustFasta 2>&1 | grep EXAMPLE'])])), ('about', CommentedMap([('home', 'http://orthomcl.org/orthomcl/'), ('license', 'EuPathDB Bioinformatics Resource Center'), ('license_file', 'doc/OrthoMCLEngine/Main/SoftwareLicense.txt'), ('summary', 'Ortholog groups of protein sequences')]))])), 'outrigger': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/outrigger/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'outrigger'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'outrigger-1.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ca/03/860e2dda28aeba28f9fc552c6af9e9e2ff13e806a4437726c8c024c0efd2/outrigger-1.1.1.tar.gz'), ('md5', 'ac911a16daa78480902ae7cdcde655b0')])), ('build', CommentedMap([('entry_points', ['outrigger = outrigger.commandline:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest >=3.0.0', 'pandas >=0.17.0', 'coverage', 'gffutils >=0.8.7.1', 'pybedtools', 'biopython', 'joblib', 'pysam', 'graphlite']), ('run', ['python', 'pytest >=3.0.0', 'pandas >=0.17.0', 'coverage', 'gffutils >=0.8.7.1', 'pybedtools', 'biopython', 'joblib', 'pysam', 'graphlite'])])), ('test', CommentedMap([('imports', ['outrigger', 'outrigger.index', 'outrigger.io', 'outrigger.psi', 'outrigger.validate']), ('commands', ['outrigger --help']), ('requires', ['flake8', 'pytest'])])), ('about', CommentedMap([('home', 'https://yeolab.github.io/outrigger'), ('license', 'BSD License'), ('summary', 'Outrigger detects *de novo* exons and quantifies their percent spliced-in'), ('license_family', 'BSD')]))])), 'p7zip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/p7zip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'p7zip'), ('version', '15.09')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('source', CommentedMap([('fn', 'p7zip_15.09_src_all.tar.bz2'), ('url', 'http://iweb.dl.sourceforge.net/project/p7zip/p7zip/15.09/p7zip_15.09_src_all.tar.bz2'), ('md5', 'ab69f4f13ba0ec57eca2cf4c9edd9678')])), ('test', CommentedMap([('commands', ['touch xxx', '7z a xxx.7z xxx', '7za a xxx.7z xxx', '7zr a xxx.7z xxx'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/p7zip/'), ('summary', 'p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip, see www.7-zip.org ) for Unix.'), ('license', 'LGPLv2')]))])), 'pacbio_falcon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pacbio_falcon/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/FALCON'), ('license', 'Standard PacBio Open Source License'), ('summary', 'A set of tools for fast aligning long reads for consensus and assembly')])), ('source', CommentedMap([('fn', 'funzip_052016.tar.gz'), ('url', 'https://github.com/PacificBiosciences/FALCON/archive/funzip_052016.tar.gz'), ('md5', 'be4a225262ee4bd9729b6b7e81a83615')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pypeflow', 'networkx']), ('run', ['python', 'pypeflow', 'rdflib', 'networkx'])])), ('test', CommentedMap([('commands', ['fc_run.py 2>&1 | grep fc_run'])]))])), 'pairix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pairix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pairix'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/4dn-dcic/pairix/archive/<{ version }}.tar.gz'), ('md5', 'cf72bfaa9c6d076a2f1b186e2d3ef15d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'python', 'setuptools']), ('run', ['htslib', 'python'])])), ('test', CommentedMap([('imports', ['pypairix']), ('commands', ['pairix --help'])])), ('about', CommentedMap([('home', 'https://github.com/4dn-dcic/pairix'), ('license', 'MIT'), ('summary', '2D indexing on bgzipped text files of paired genomic coordinates'), ('license_family', 'MIT')]))])), 'paladin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paladin/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/twestbrookunh/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'curl']), ('run', ['libgcc', 'zlib', 'curl'])])), ('test', CommentedMap([('commands', ['command -v paladin'])])), ('about', CommentedMap([('home', 'http://genomebio.org/paladin/'), ('license', 'MIT'), ('summary', 'Protein Alignment and Detection Interface'), ('description', 'PALADIN is a protein sequence alignment tool designed for the accurate\\nfunctional characterization of metagenomes.\\n'), ('dev_url', 'https://github.com/twestbrookunh/paladin')])), ('extra', CommentedMap([('recipe-maintainers', ['keuv-grvl'])]))])), 'pancake': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pancake/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pancake'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'pancake-1.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/c6/ac/1cadba4bf3d0cb647da3d04df10ce8c4b085b50f08efd50117531fdb743c/pancake-1.1.2.tar.gz'), ('md5', '2bc2b40b6e46477aae5fc9ae4807dcef')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'biopython']), ('run', ['python', 'numpy', 'biopython'])])), ('test', CommentedMap([('imports', ['pancake'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/CorinnaErnst/pancake'), ('license', 'MIT License'), ('summary', 'A Data Structure for Pangenomes -- Identification of Singletons and Core Regions Dependent on Pairwise Sequence Similarities'), ('license_family', 'MIT')]))])), 'pandaseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pandaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pandaseq'), ('version', '2.11')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'pandaseq_v2.11.tar.gz'), ('url', 'https://github.com/neufeld/pandaseq/archive/v2.11.tar.gz'), ('md5', 'a8ae0e938bac592fc07dfa668147d80b')])), ('requirements', CommentedMap([('build', ['gcc [linux]', 'llvm [osx]', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'zlib', 'pkg-config', 'libtool', 'bzip2']), ('run', ['libtool [linux]', 'libgcc [linux]'])])), ('test', CommentedMap([('commands', ['pandaseq -v 2>&1 | grep \"pandaseq\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/neufeld/pandaseq'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence.')]))])), 'panoct': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/panoct/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://downloads.sourceforge.net/project/panoct/panoct_v3.23.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['panoct.pl -h', 'gene_order.pl -h', 'paralog_matchtable.pl -h'])])), ('about', CommentedMap([('home', 'https://panoct.sourceforge.io/'), ('license', 'GPL-3.0'), ('summary', 'PanOCT, Pan-genome Ortholog Clustering Tool, is a program for pan-genomic analysis of closely related prokaryotic species or strains')]))])), 'pantools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pantools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/Sheikhizadeh/pantools/archive/v<{ version }}.tar.gz'), ('md5', '28c6ce2af0ecd25e50f8152c1c87a4f1')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python', 'kmc'])])), ('test', CommentedMap([('commands', [\"pantools --help | grep 'reconstruct' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/Sheikhizadeh/pantools'), ('license', 'Unknown'), ('summary', 'PanTools is a disk-based java application for computational pan-genomics')]))])), 'parafly': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/parafly/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'parafly'), ('version', 'r2013_01_21')])), ('source', CommentedMap([('fn', 'parafly-r2013-01-21.tgz'), ('url', 'http://downloads.sourceforge.net/project/parafly/parafly-r2013-01-21.tgz'), ('sha256', '64cf7ac2d4af0801b78d58f4057a1489d76b2b8ae59c78997f434d1239fa4abe')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['ParaFly 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://parafly.sourceforge.net/'), ('license', 'The Broad Institute (own license thingy)'), ('summary', 'Given a file containing a list of unix commands, multithreading is used to process the commands in parallel on a single server. Success/failure is captured, and failed commands are retained and reported.')]))])), 'parallel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/parallel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'parallel'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'http://ftpmirror.gnu.org/parallel/parallel-<{ version }}.tar.bz2'), ('fn', 'parallel-<{ version }}.tar.bz2'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['parallel --version'])])), ('about', CommentedMap([('home', 'http://www.gnu.org/software/parallel/'), ('license', 'GPLv3'), ('summary', 'GNU parallel is a shell tool for executing jobs in parallel using one or more computers.')]))])), 'parallel-fastq-dump': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/parallel-fastq-dump/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'parallel-fastq-dump'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/rvalieris/parallel-fastq-dump/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'sra-tools'])])), ('test', CommentedMap([('commands', ['parallel-fastq-dump --version'])])), ('about', CommentedMap([('home', 'https://github.com/rvalieris/parallel-fastq-dump'), ('license', 'MIT License'), ('license_family', 'MIT'), ('summary', 'parallel fastq-dump wrapper')]))])), 'paralyzer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paralyzer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paralyzer'), ('version', 1.5)])), ('source', CommentedMap([('fn', 'PARalyzer_v1_5.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PARalyzer/PARalyzer_v1_5.tar.gz'), ('md5', '09d82ffbda7787a6f16553576b40e6ca')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['java-jdk']), ('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['PARalyzer 2G'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/'), ('license', 'PARalyzer is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PARalyzer algorithm to generate a high resolution map of interaction sites between RNA-binding proteins and their targets. The algorithm utilizes the deep sequencing reads generated by PAR-CLIP (Photoactivatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation) protocol.The use of photoactivatable nucleotides in the PAR-CLIP protocol results in more efficient crosslinking between the RNA-binding protein and its target relative to other CLIP methods; in addition a nucleotide substitution occurs at the site of crosslinking, providing for single-nucleotide resolution binding information. PARalyzer utilizes this nucleotide substition in a kernel density estimate classifier to generate the high resolution set of Protein-RNA interaction sites.')]))])), 'pash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pash'), ('version', '3.0.6.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'pash-3.0.6.2.tar.gz'), ('url', 'https://genboree.org/pash-supp/pash-3.0.6.2.tar.gz'), ('md5', 'f475acdb1ad8dd712cdf16913849b860')])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'ruby']), ('run', ['libgcc', 'glib', 'ruby'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/'), ('license', 'Unknown'), ('summary', 'A versatile software package for read mapping and integrative analysis of genomic and epigenomic variation using massively parallel DNA sequencing')])), ('test', CommentedMap([('commands', ['keyFreq --help 2> /dev/null', 'makeIgnoreList --help 2> /dev/null', 'pash2SAM --help 2> /dev/null', 'pash-3.0lx --help > /dev/null', 'buildFastaIndex.rb --help | grep \"PROGRAM DESCRIPTION\" > /dev/null', 'pashToLff.rb --help 2>&1 | grep \"pash output\" > /dev/null'])]))])), 'pathoscope': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pathoscope/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('about', CommentedMap([('home', 'https://github.com/PathoScope/PathoScope'), ('license', 'GPLv3'), ('summary', 'Species identification and strain attribution with unassembled sequencing data')])), ('package', CommentedMap([('name', 'pathoscope'), ('version', '2.0.6')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'samtools <1.0', 'bowtie2'])])), ('test', CommentedMap([('commands', ['pathoscope --version', 'pathoscope LIB -h &> /dev/null', 'pathoscope MAP -h &> /dev/null', 'pathoscope ID -h &> /dev/null', 'pathoscope REP -h &> /dev/null'])])), ('source', CommentedMap([('url', 'https://github.com/PathoScope/PathoScope/archive/v2.0.6.tar.gz'), ('md5', '86118e61435ff4814794b1af603d8461'), ('sha1', '469ca1036f4c05ef15938b5bee1de28ad1b6519b'), ('sha256', 'ab32ac49ac1e30bf67ed8ca73d9675f60c022983cfc1849e14954da815cb7469')]))])), 'pauda': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pauda/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://ab.inf.uni-tuebingen.de/software/pauda'), ('license', 'GPL'), ('summary', 'PAUDA is a new approach toward the problem of comparing DNA reads against a database of protein reference sequences that is applicable to very large datasets consisting of hundreds of millions or billions of reads.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'pauda'), ('version', '1.0.1')])), ('requirements', CommentedMap([('run', ['java-jdk', 'bowtie2'])])), ('source', CommentedMap([('fn', 'pauda-1.0.1.zip'), ('url', 'http://ab.inf.uni-tuebingen.de/data/software/pauda/download/pauda-1.0.1.zip')])), ('test', CommentedMap([('commands', ['pauda-run | grep Usage > /dev/null', 'pauda-build | grep Usage > /dev/null'])]))])), 'pauvre': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pauvre/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pauvre'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'pauvre-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/85/55/143c757130d51325f12c07b749e9b507de6ddbf95c91d0c1ba9efe1bdebc/pauvre-0.1.3.tar.gz'), ('md5', '807199fb32746f9ccee70c1d5e8c9e89')])), ('build', CommentedMap([('entry_points', ['pauvre=pauvre.pauvre_main:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib', 'biopython', 'pandas']), ('run', ['python', 'matplotlib', 'biopython', 'pandas'])])), ('test', CommentedMap([('imports', ['pauvre']), ('commands', ['pauvre --help'])])), ('about', CommentedMap([('home', 'https://github.com/conchoecia/gloTK'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Tools for plotting Oxford Nanopore and other long-read data.'), ('license_family', 'GPL3')]))])), 'pbbam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbbam/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', False), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('git_url', 'https://github.com/PacificBiosciences/<{ name }}.git'), ('git_rev', '928605a86ac024aced821fe4a4bce0b43d5ee680')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'cmake', 'boost <{CONDA_BOOST}}*', 'zlib']), ('run', ['libgcc', 'htslib', 'zlib'])])), ('test', CommentedMap([('commands', ['bam2sam --help', 'pbindex --help', 'pbindexdump --help', 'pbmerge --help'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/<{ name }}'), ('license', 'BSD-3-Clause-Clear'), ('summary', 'PacBio BAM C++ library, with SWIG bindings')]))])), 'pbcommand': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbcommand/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/ed/10/8cb0bb4586a11ddac7383634ad48b095692313b97f1153f0e77e1b70eacc/pbcommand-0.3.29.tar.gz'), ('md5', 'f1b3c2b6039d6bf7cf166bd5df6f3cd8')])), ('build', CommentedMap([('skip', True), ('entry_points', ['pbservice = pbcommand.services.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'xmlbuilder', 'jsonschema', 'avro', 'requests', 'iso8601', 'numpy']), ('run', ['python', 'xmlbuilder', 'jsonschema', 'avro', 'requests', 'iso8601', 'numpy'])])), ('test', CommentedMap([('imports', ['pbcommand', 'pbcommand.cli', 'pbcommand.cli.examples', 'pbcommand.engine', 'pbcommand.models', 'pbcommand.pb_io', 'pbcommand.schemas', 'pbcommand.services', 'pbcommand.testkit', 'tests']), ('commands', ['pbservice --help']), ('requires', ['avro', 'iso8601', 'jsonschema', 'nose', 'numpy', 'requests', 'sphinx-argparse', 'sphinx-bootstrap-theme', 'xmlbuilder', 'numpy'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/pbcommand'), ('license', 'BSD-3-Clause'), ('summary', 'Library and Tools for interfacing to PacBio pbsmrtpipe workflow engine.')]))])), 'pbcore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbcore/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/63/23/3c2dd8c5b957be17af9d00197fc157cd60952592c3b46e7b418029df9cca/pbcore-1.2.10.tar.gz'), ('md5', 'e33bee180b9d91e90a3f0f022764742c')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy >=1.7.1', 'h5py >=2.0.1', 'pysam >=0.9.0']), ('run', ['python', 'cython', 'numpy >=1.7.1', 'h5py >=2.0.1', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['pbcore', 'pbcore.chemistry', 'pbcore.data', 'pbcore.data.datasets', 'pbcore.io', 'pbcore.io.align', 'pbcore.io.dataset', 'pbcore.model', 'pbcore.util']), ('requires', ['nose', 'pyxb', 'sphinx'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/pbcore'), ('license', 'BSD-3-Clause'), ('summary', 'A Python library for reading and writing PacBio data files')]))])), 'pbdagcon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbdagcon/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('git_url', 'https://github.com/PacificBiosciences/<{ name }}'), ('git_rev', '87f393fe133ea626552037c1a0b9c584808bfdd8')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'boost <{CONDA_BOOST}}*', 'python']), ('run', ['libgcc', 'daligner', 'blasr_libcpp', 'dazz_db'])])), ('test', CommentedMap([('commands', ['pbdagcon --help', 'dazcon --help'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/<{ name }}'), ('license', 'BSD-3-Clause-Clear'), ('summary', 'A sequence consensus algorithm implementation based on using directed acyclic graphs to encode multiple sequence alignment')]))])), 'pbgzip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbgzip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pbgzip'), ('version', '2015.10.28')])), ('source', CommentedMap([('fn', 'bgzip-6abda41a6.tar.gz'), ('url', 'https://github.com/nh13/pbgzip/archive/6abda41a6.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'https://github.com/nh13/pbgzip'), ('summary', 'Parallel Block GZIP'), ('license', 'MIT/Expat')]))])), 'pbh5tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbh5tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'smrtanalysis-2.3.0p4.tar.gz'), ('url', 'https://github.com/PacificBiosciences/pbh5tools/archive/smrtanalysis-2.3.0p4.tar.gz'), ('md5', 'ef2781ff8adb9a613c23c8d945ac7217')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'pbcore >=0.8.0', 'numpy >=1.6.0', 'h5py >=1.3.0', 'setuptools']), ('run', ['python', 'libgcc', 'pbcore >=0.8.0', 'numpy >=1.6.0', 'h5py >=1.3.0'])])), ('test', CommentedMap([('commands', ['bash5tools.py --version > /dev/null', 'cmph5tools.py --version > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/pbh5tools'), ('license', 'BSD-3-Clause'), ('license_file', 'LICENSES.txt'), ('summary', 'A swiss-army knife for interrogating PacBio® HDF5 files (cmp.h5, bas.h5).')]))])), 'pbsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pbsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pbsim'), ('version', '1.0.3')])), ('about', CommentedMap([('home', 'https://code.google.com/archive/p/pbsim/'), ('license', 'GPLv2'), ('summary', 'PBSIM simulates PacBio reads')])), ('source', CommentedMap([('url', 'https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz'), ('md5', 'c070394d42b6e4cdae0bb2aaf4856782')])), ('requirements', CommentedMap([('build', ['gcc'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('commands', [\"pbsim 2>&1 | grep -q 'USAGE: pbsim'\"])]))])), 'pdfkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pdfkit/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'pdfkit'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'pdfkit-0.5.0.zip'), ('url', 'https://pypi.python.org/packages/source/p/pdfkit/pdfkit-0.5.0.zip'), ('md5', '5cbe42c43d463f0794272a01e37a553f')])), ('requirements', CommentedMap([('build', ['wkhtmltopdf', 'python']), ('run', ['wkhtmltopdf', 'python'])])), ('test', CommentedMap([('imports', ['pdfkit'])])), ('about', CommentedMap([('home', 'https://pypi.python.org/pypi/pdfkit'), ('license', 'MIT License'), ('summary', 'Wkhtmltopdf python wrapper to convert html to pdf using the webkit rendering engine and qt')]))])), 'peakachu': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakachu/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakachu'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'peakachu-<{ version }}.tar.gz'), ('url', 'https://github.com/tbischler/PEAKachu/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'biopython >=1.69', 'matplotlib >=2.0.2', 'pandas >=0.20.2', 'pysam >=0.11.2.2', 'numexpr >=2.6.2', 'bcbiogff >=0.6.4', 'statsmodels >=0.8.0', 'rpy2 >=2.8.5']), ('run', ['libgcc', 'python', 'biopython >=1.69', 'matplotlib >=2.0.2', 'pandas >=0.20.2', 'pysam >=0.11.2.2', 'numexpr >=2.6.2', 'bcbiogff >=0.6.4', 'statsmodels >=0.8.0', 'rpy2 >=2.8.5', 'bioconductor-deseq2', 'bioconductor-edger', 'blockbuster'])])), ('test', CommentedMap([('imports', ['peakachulib', 'peakachulib.intersection']), ('commands', ['peakachu 2>&1 | grep \"^usage\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/tbischler/PEAKachu'), ('license', 'ISCL'), ('summary', 'Peak calling tool for CLIP-seq data.')]))])), 'peakranger': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakranger/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakranger'), ('version', 1.18)])), ('source', CommentedMap([('fn', 'PeakRanger-1.18-src.zip'), ('url', 'http://downloads.sourceforge.net/project/ranger/Source%20codes/PeakRanger-1.18-src.zip'), ('md5', '4b47352ee3200096f2ee8745a436b559'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*']), ('run', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*', 'r-base'])])), ('test', CommentedMap([('commands', ['peakranger nr --version'])])), ('about', CommentedMap([('home', 'http://ranger.sourceforge.net'), ('summary', 'PeakRanger is a multi-purporse software suite for analyzing next-generation sequencing (NGS) data.'), ('license', 'Artistic License 2.0'), ('license_file', 'COPYING')]))])), 'peakzilla': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakzilla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakzilla'), ('version', 1.0)])), ('source', CommentedMap([('fn', '0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('url', 'https://github.com/steinmann/peakzilla/archive/0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('sha256', '982d2bbcde736e4ce7837a7758ce37e8040cd1cc148c64a3856b9ea837f2740c'), ('patches', ['indent.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['peakzilla.py -h'])])), ('about', CommentedMap([('home', 'http://stark.imp.ac.at/data/peakzilla'), ('summary', 'Peakzilla identifies sites of enrichment and transcription factor binding sites from transcription factor ChIP-seq and ChIP-exo experiments at hight accuracy and resolution.'), ('license', 'GPLv2')]))])), 'pear': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pear'), ('version', '0.9.6')])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/pear/'), ('license', 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported'), ('summary', 'paired-end read merger')])), ('source', CommentedMap([('fn', 'pear-0.9.6-src.tar.gz'), ('md5', '5ed68d50c1620cd55c58681e38771799'), ('url', 'http://sco.h-its.org/exelixis/web/software/pear/files/pear-0.9.6-src.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkgconfig', 'zlib', 'bzip2']), ('run', ['libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', [\"pear 2>&1 | grep 'PEAR v0.9.6'\", \"pearRM 2>&1 | grep 'PEAR v0.9.6'\"])]))])), 'ped_parser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ped_parser/1.6.5/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ped_parser'), ('version', '1.6.5')])), ('source', CommentedMap([('fn', 'ped_parser-1.6.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/ped_parser/ped_parser-1.6.5.tar.gz'), ('md5', '1f404e131c3d210d32e6253ac500b200')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest', 'click']), ('run', ['python', 'pytest', 'click'])])), ('test', CommentedMap([('imports', ['ped_parser']), ('commands', ['LANG=C.UTF-8 ped_parser --help'])])), ('about', CommentedMap([('home', 'https://github.com/moonso/ped_parser'), ('license', 'BSD License'), ('summary', 'A ped file parser.')]))])), 'pedagree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pedagree/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'pedagree'), ('version', '0.0.11')])), ('source', CommentedMap([('fn', 'pedagree-0.0.11.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pedagree/pedagree-0.0.11.tar.gz'), ('md5', '42d4d309c9b47921870c404a1e3bb406')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pedagree', 'pedagree.tests']), ('requires', ['numpy', 'cython', 'cyvcf2', 'networkx'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/pedagree'), ('license', 'MIT License'), ('license_file', 'LICENSE'), ('summary', 'pleasingly pythonic pedigree manipulation')]))])), 'peddy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peddy/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'peddy-<{ version }}.tar.gz'), ('url', 'https://github.com/brentp/peddy/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}'), ('patches', ['peddy-py.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'networkx', 'cython', 'cyvcf2 >=0.5.3', 'numpy']), ('run', ['python', 'networkx', 'cyvcf2 >=0.5.3', 'numpy', 'scikit-learn', 'toolshed', 'matplotlib >=1.5.0', 'click', 'coloredlogs', 'seaborn', 'pandas'])])), ('test', CommentedMap([('imports', ['peddy', 'peddy.tests', 'toolshed']), ('commands', ['python -m peddy -h'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/peddy'), ('license', 'MIT License'), ('summary', 'genotype :: ped correspondence check, ancestry check, sex check. directly, quickly on VCF')]))])), 'peewee': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peewee/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'peewee'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'peewee-2.8.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/peewee/peewee-2.8.0.tar.gz'), ('md5', '973250df50a081b67d4216b88f694945')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['playhouse'])])), ('about', CommentedMap([('home', 'http://github.com/coleifer/peewee/'), ('license', 'MIT License'), ('summary', 'a little orm')]))])), 'pepnovo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pepnovo/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://proteomics.ucsd.edu/Software/PepNovo/'), ('license_file', 'LICENSE'), ('summary', 'PepNovo serves as a high throughput de novo peptide sequencing tool for tandem mass spectrometry data')])), ('package', CommentedMap([('name', 'pepnovo'), ('version', 20101117)])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'PepNovo.20120423.zip'), ('url', 'http://proteomics.ucsd.edu/Software/PepNovo/PepNovo.20120423.zip'), ('md5', 'af2a863487cb53bf0e92f00d395c4f0d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['pepnovo -h 2>&1 | grep \"PepNovo+\"'])]))])), 'pepr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pepr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pepr'), ('version', '1.1.18')])), ('source', CommentedMap([('fn', 'PePr-1.1.18.tar.gz'), ('url', 'https://pypi.python.org/packages/6b/1a/39f62045d220248e4c5bf7e718eb598ced97dff077b4cce9ea18515b4f45/PePr-1.1.18.tar.gz'), ('md5', 'dcc72e55149c9bdb994d105c69c6e755'), ('patches', ['setup.py.patch'])])), ('build', CommentedMap([('entry_points', ['PePr=PePr.PePr:argless_main', 'PePr-preprocess=PePr.PePr:pre_processing_module', 'PePr-postprocess=PePr.post_processing.post_process_PePr:post_processing_module'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.6.0', 'scipy >=0.14.0', 'pysam', 'sharedmem']), ('run', ['python', 'numpy >=1.6.0', 'scipy >=0.14.0', 'pysam', 'sharedmem'])])), ('test', CommentedMap([('imports', ['PePr', 'PePr.post_processing']), ('commands', ['PePr --help', 'PePr-preprocess --help', 'PePr-postprocess --help'])])), ('about', CommentedMap([('home', 'https://github.com/shawnzhangyx/PePr/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Peak-calling and Prioritization pipeline for replicated ChIP-Seq data'), ('license_family', 'GPL')]))])), 'peptide-shaker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peptide-shaker/1.16.4/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip'), ('md5', '14a48413e28a25614f5fda2b381d7197')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))])), 'percolator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/percolator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'percolator'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'rel-3-01.tar.gz'), ('url', 'https://github.com/percolator/percolator/archive/rel-3-01.tar.gz'), ('md5', 'fcb37bc5fcd0149cc775c3e72ff3dc24')])), ('build', CommentedMap([('number', 2), ('string', 'boost_1.62<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake ==2.8.10.2', 'gcc', 'zlib', 'boost ==1.62', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd']), ('run', ['libgcc', 'zlib', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd'])])), ('test', CommentedMap([('commands', ['percolator --help 2>&1 > /dev/null | grep \"Percolator version 3.01\"', 'msgf2pin --help'])])), ('about', CommentedMap([('home', 'https://github.com/percolator/percolator'), ('license', 'Apache 2.0'), ('summary', 'Semi-supervised learning for peptide identification from shotgun proteomics datasets')]))])), 'perl-aceperl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-aceperl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-aceperl'), ('version', '1.92')])), ('source', CommentedMap([('fn', 'AcePerl-1.92.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LD/LDS/AcePerl-1.92.tar.gz'), ('md5', '485e54655a15d8dc5c471fed07092414'), ('patches', ['makefile-options.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Ace'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/AcePerl'), ('license', 'unknown'), ('summary', 'Object-Oriented Access to ACEDB Databases')]))])), 'perl-algorithm-cluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-algorithm-cluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-algorithm-cluster'), ('version', '1.52')])), ('source', CommentedMap([('fn', 'Algorithm-Cluster-1.52.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MD/MDEHOON/Algorithm-Cluster-1.52.tar.gz'), ('md5', '8a80e644e705c829e02a9339025c4225')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Algorithm::Cluster', 'Algorithm::Cluster::Record'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Algorithm::Cluster'), ('license', 'unknown'), ('summary', 'Perl interface to the C Clustering Library')]))])), 'perl-algorithm-dependency': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-algorithm-dependency/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-algorithm-dependency'), ('version', '1.110')])), ('source', CommentedMap([('fn', 'Algorithm-Dependency-1.110.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Algorithm-Dependency-1.110.tar.gz'), ('md5', 'dc1dab2ac44c315caa092397423b1429')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-test-classapi', 'perl-scalar-list-utils', 'perl-params-util', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-params-util', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Algorithm::Dependency'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Algorithm-Dependency'), ('license', 'perl_5'), ('summary', 'Base class for implementing various dependency trees')]))])), 'perl-algorithm-diff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-algorithm-diff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-algorithm-diff'), ('version', '1.1903')])), ('source', CommentedMap([('fn', 'Algorithm-Diff-1.1903.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TY/TYEMQ/Algorithm-Diff-1.1903.tar.gz'), ('md5', '0e8add21a641b8d66436df0c2024bf3b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Algorithm::Diff'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Algorithm-Diff'), ('license', 'unknown'), ('summary', 'Compute intelligent differences between two files / lists but use the old (<=0.59) interface.')]))])), 'perl-algorithm-munkres': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-algorithm-munkres/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-algorithm-munkres'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Algorithm-Munkres-0.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TP/TPEDERSE/Algorithm-Munkres-0.08.tar.gz'), ('md5', '56349287513f955002720c8e2523582b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Algorithm::Munkres'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Algorithm-Munkres'), ('license', 'unknown'), ('summary', 'Munkres.pm')]))])), 'perl-aliased': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-aliased/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-aliased'), ('version', '0.34')])), ('source', CommentedMap([('fn', 'aliased-0.34.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/aliased-0.34.tar.gz'), ('md5', 'f7f659f689699a87115da1262eb6edc6')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-simple', 'perl-pathtools', 'perl-module-build-tiny', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['aliased'])])), ('about', CommentedMap([('home', 'https://github.com/karenetheridge/aliased'), ('license', 'perl_5'), ('summary', 'Use shorter versions of class names.')]))])), 'perl-apache-test': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-apache-test/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-apache-test'), ('version', '1.40')])), ('source', CommentedMap([('fn', 'Apache-Test-1.40.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHAY/Apache-Test-1.40.tar.gz'), ('md5', '78e059bccfa0c2f0297ff2fca591eca9')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-pathtools'])])), ('test', CommentedMap([('imports', ['Apache::Test'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Apache::Test'), ('license', 'unknown'), ('summary', 'Special Tests Sequence Failure Finder')]))])), 'perl-app-cpanminus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-app-cpanminus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-app-cpanminus'), ('version', '1.7039')])), ('source', CommentedMap([('fn', 'App-cpanminus-1.7039.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIYAGAWA/App-cpanminus-1.7039.tar.gz')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('imports', ['App::cpanminus']), ('commands', ['cpanm --help'])])), ('about', CommentedMap([('home', 'https://github.com/miyagawa/cpanminus'), ('license', 'perl_5'), ('summary', 'get, unpack, build and install modules from CPAN')]))])), 'perl-appconfig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-appconfig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-appconfig'), ('version', '1.71')])), ('source', CommentedMap([('fn', 'AppConfig-1.71.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/AppConfig-1.71.tar.gz'), ('md5', '7747d9241561ed5567d5e134b8648707')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-pod']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['AppConfig', 'AppConfig::Args', 'AppConfig::CGI', 'AppConfig::File', 'AppConfig::Getopt', 'AppConfig::State', 'AppConfig::Sys'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/AppConfig'), ('license', 'perl_5'), ('summary', 'AppConfig is a bundle of Perl5 modules for reading configuration files and parsing command line arguments.')]))])), 'perl-archive-extract': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-archive-extract/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-archive-extract'), ('version', '0.76')])), ('source', CommentedMap([('fn', 'Archive-Extract-0.76.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/Archive-Extract-0.76.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Archive::Extract'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Archive::Extract'), ('license', 'Perl'), ('summary', 'A generic archive extracting mechanism')]))])), 'perl-archive-tar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-archive-tar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-archive-tar'), ('version', 2.18)])), ('source', CommentedMap([('fn', 'Archive-Tar-2.18.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/B/BI/BINGOS/Archive-Tar-2.18.tar.gz'), ('md5', 'ed4d4ca0124adece439f5d97b05d3987')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-io-compress', 'perl-io-zlib', 'perl-test-harness', 'perl-test-simple', 'perl-module-build']), ('run', ['perl', 'perl-pathtools', 'perl-io-compress', 'perl-io-zlib'])])), ('test', CommentedMap([('imports', ['Archive::Tar', 'Archive::Tar::Constant', 'Archive::Tar::File'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Archive::Tar'), ('license', 'perl_5'), ('summary', 'Manipulates TAR archives')]))])), 'perl-archive-zip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-archive-zip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-archive-zip'), ('version', '1.55')])), ('source', CommentedMap([('fn', 'Archive-Zip-1.55.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.55.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-app-cpanminus']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Archive::Zip'), ('license', 'Perl'), ('summary', 'Provide an interface to ZIP archive files')]))])), 'perl-array-compare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-array-compare/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-array-compare'), ('version', '2.11')])), ('source', CommentedMap([('fn', 'Array-Compare-2.11.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DA/DAVECROSS/Array-Compare-2.11.tar.gz'), ('md5', '57ff4aa504d4c2d1ecef342bd19c0039')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-moo', 'perl-type-tiny', 'perl-module-build', 'perl-test-nowarnings']), ('run', ['perl-threaded', 'perl-moo', 'perl-type-tiny'])])), ('test', CommentedMap([('imports', ['Array::Compare'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Array-Compare'), ('license', 'perl_5'), ('summary', 'Perl extension for comparing arrays.')]))])), 'perl-array-utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-array-utils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-array-utils'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'Array-Utils-0.5.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/Z/ZM/ZMIJ/Array/Array-Utils-0.5.tar.gz'), ('md5', 'ac15e6dce2c7c9d1855ecab9eb00aee6')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Array::Utils'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Array::Utils'), ('license', 'unknown'), ('summary', 'small utils for array manipulation')]))])), 'perl-authen-sasl-saslprep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-authen-sasl-saslprep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-authen-sasl-saslprep'), ('version', '1.011')])), ('source', CommentedMap([('fn', 'Authen-SASL-SASLprep-1.011.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CF/CFAERBER/Authen-SASL-SASLprep-1.011.tar.gz'), ('md5', 'eb9cc1f59968047e23ac44c02a0ecc21')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-test-nowarnings', 'perl-unicode-stringprep']), ('run', ['perl-threaded', 'perl-unicode-stringprep'])])), ('test', CommentedMap([('imports', ['Authen::SASL::SASLprep'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Authen-SASL-SASLprep'), ('license', 'perl_5'), ('summary', 'A Stringprep Profile for User Names and Passwords (RFC 4013)')]))])), 'perl-bio-asn1-entrezgene': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-asn1-entrezgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-asn1-entrezgene'), ('version', 1.72)])), ('source', CommentedMap([('fn', 'Bio-ASN1-EntrezGene-1.72.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/C/CJ/CJFIELDS/Bio-ASN1-EntrezGene-1.72.tar.gz'), ('md5', 'ea51d9d1774e0873e702909bf7ccf0cb')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl-core']), ('run', ['perl-threaded', 'perl-bioperl-core'])])), ('test', CommentedMap([('imports', ['Bio::ASN1::EntrezGene', 'Bio::ASN1::EntrezGene::Indexer', 'Bio::ASN1::Sequence', 'Bio::ASN1::Sequence::Indexer'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Bio-ASN1-EntrezGene'), ('license', 'perl_5'), ('summary', 'Regular expression-based Perl Parser for NCBI Entrez Gene')]))])), 'perl-bio-cigar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-cigar/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Bio-Cigar-1.01.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/T/TS/TSIBLEY/Bio-Cigar-1.01.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp', 'perl-termreadkey', 'perl-class-methodmaker', 'perl-bioperl', 'perl-test-more', 'perl-extutils-makemaker', 'perl-namespace-clean', 'perl-moo', 'perl-type-tiny']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp', 'perl-termreadkey', 'perl-class-methodmaker', 'perl-bioperl', 'perl-test-more', 'perl-extutils-makemaker', 'perl-namespace-clean', 'perl-moo', 'perl-type-tiny'])])), ('test', CommentedMap([('commands', ['perl -e \"use Bio::Cigar\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~tsibley/Bio-Cigar-1.01'), ('license', 'perl_5'), ('summary', 'Parse CIGAR strings and translate coordinates to/from reference/query')]))])), 'perl-bio-db-hts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-db-hts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-db-hts'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Bio-DB-HTS-<{ version }}.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RI/RISHIDEV/Bio-DB-HTS-<{ version }}.tar.gz'), ('md5', 'a09468d02a7e58673b177e60915f28b0')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['zlib', 'perl', 'perl-module-build', 'perl-bioperl', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['zlib', 'perl', 'perl-bioperl', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', None), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Bio::DB::HTS'), ('license', 'Apache v2.0'), ('summary', 'Read files using HTSlib including BAM/CRAM, Tabix and BCF database files')]))])), 'perl-bio-db-sam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-db-sam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-db-sam'), ('version', '1.41')])), ('source', CommentedMap([('fn', 'Bio-SamTools-1.41.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LD/LDS/Bio-SamTools-1.41.tar.gz')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-bioperl']), ('run', ['libgcc', 'perl-threaded', 'perl-bioperl'])])), ('test', CommentedMap([('imports', ['Bio::DB::Sam'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Bio::DB::Sam'), ('license', 'Perl'), ('summary', 'Read SAM/BAM database files')]))])), 'perl-bio-eutilities': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-eutilities/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-eutilities'), ('version', '1.75')])), ('source', CommentedMap([('fn', 'Bio-EUtilities-1.75.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/C/CJ/CJFIELDS/Bio-EUtilities-1.75.tar.gz'), ('md5', '8814608d46fc6517814f320766c24739')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-bioperl ==1.6.924', 'perl-test-most', 'perl-test-differences', 'perl-text-diff', 'perl-capture-tiny', 'perl-test-exception', 'perl-sub-uplevel', 'perl-test-deep', 'perl-test-warn', 'perl-exception-class', 'perl-class-data-inheritable', 'perl-devel-stacktrace', 'perl-test-simple', 'perl-data-stag', 'perl-bio-asn1-entrezgene', 'perl-text-csv', 'perl-xml-simple', 'perl-xml-sax', 'perl-xml-sax-base', 'perl-xml-namespacesupport', 'perl-xml-sax-expat']), ('run', ['perl', 'perl-bioperl ==1.6.924', 'perl-test-most', 'perl-test-differences', 'perl-text-diff', 'perl-capture-tiny', 'perl-test-exception', 'perl-sub-uplevel', 'perl-test-deep', 'perl-test-warn', 'perl-exception-class', 'perl-class-data-inheritable', 'perl-devel-stacktrace', 'perl-test-simple', 'perl-data-stag', 'perl-bio-asn1-entrezgene', 'perl-text-csv', 'perl-xml-simple', 'perl-xml-sax', 'perl-xml-sax-base', 'perl-xml-namespacesupport', 'perl-xml-sax-expat'])])), ('test', CommentedMap([('imports', ['Bio::DB::EUtilities', 'Bio::Tools::EUtilities'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Bio-EUtilities'), ('license', 'perl_5'), ('summary', '-')]))])), 'perl-bio-featureio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-featureio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-featureio'), ('version', '1.6.905')])), ('source', CommentedMap([('fn', 'Bio-FeatureIO-1.6.905.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS/Bio-FeatureIO-1.6.905.tar.gz'), ('md5', '7e455995c0a9cff706f43b1412dfdb9c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl-core', 'perl-tree-dag_node', 'perl-uri', 'perl-xml-dom', 'perl-xml-dom-xpath', 'perl-test-most']), ('run', ['perl-threaded', 'perl-bioperl-core', 'perl-tree-dag_node', 'perl-uri', 'perl-xml-dom', 'perl-xml-dom-xpath'])])), ('test', CommentedMap([('imports', ['Bio::FeatureIO', 'Bio::FeatureIO::bed', 'Bio::FeatureIO::gff', 'Bio::FeatureIO::gtf', 'Bio::FeatureIO::interpro', 'Bio::FeatureIO::ptt', 'Bio::FeatureIO::vecscreen_simple', 'Bio::SeqFeature::Annotated'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Bio-FeatureIO'), ('license', 'perl_5'), ('summary', 'Modules for reading, writing, and manipulating sequence features')]))])), 'perl-bio-gff3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-gff3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-gff3'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'Bio-GFF3-2.0.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RB/RBUELS/Bio-GFF3-2.0.tar.gz'), ('md5', '8416fe2b09abddc37aed3df7040b1670')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-string', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-file-readbackwards', 'perl-uri', 'perl-list-moreutils', 'perl-scalar-list-utils', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-io-string', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-file-readbackwards', 'perl-uri', 'perl-list-moreutils', 'perl-scalar-list-utils', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Bio::GFF3::LowLevel', 'Bio::GFF3::LowLevel::Parser', 'Bio::GFF3::LowLevel::Parser::1_0_backcompat', 'Bio::GFF3::Transform::FromFasta', 'Bio::GFF3::Transform::SyncDirectives'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Bio-GFF3'), ('license', 'perl_5'), ('summary', 'fast, low-level GFF3 manipulation')]))])), 'perl-bio-phylo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-phylo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-phylo'), ('version', '0.58')])), ('source', CommentedMap([('fn', 'Bio-Phylo-0.58.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RV/RVOSA/Bio-Phylo-0.58.tar.gz'), ('md5', '7e7a8956222831282cab7444f6632c0c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pdf-api2', 'perl-gd', 'perl-html-treebuilder-xpath', 'perl-json', 'perl-math-cdf', 'perl-math-random', 'perl-svg', 'perl-xml-libxml', 'perl-xml-twig', 'perl-template-toolkit']), ('run', ['perl-threaded', 'perl-pdf-api2', 'perl-gd', 'perl-html-treebuilder-xpath', 'perl-json', 'perl-math-cdf', 'perl-math-random', 'perl-svg', 'perl-xml-libxml', 'perl-xml-twig', 'perl-template-toolkit'])])), ('test', CommentedMap([('imports', ['Bio::Phylo', 'Bio::Phylo::EvolutionaryModels', 'Bio::Phylo::Factory', 'Bio::Phylo::Forest', 'Bio::Phylo::Forest::DrawNodeRole', 'Bio::Phylo::Forest::DrawTreeRole', 'Bio::Phylo::Forest::Node', 'Bio::Phylo::Forest::NodeRole', 'Bio::Phylo::Forest::Tree', 'Bio::Phylo::Forest::TreeRole', 'Bio::Phylo::Generator', 'Bio::Phylo::IO', 'Bio::Phylo::Identifiable', 'Bio::Phylo::Listable', 'Bio::Phylo::ListableRole', 'Bio::Phylo::Matrices', 'Bio::Phylo::Matrices::Character', 'Bio::Phylo::Matrices::Characters', 'Bio::Phylo::Matrices::Datatype', 'Bio::Phylo::Matrices::Datatype::Continuous', 'Bio::Phylo::Matrices::Datatype::Custom', 'Bio::Phylo::Matrices::Datatype::Dna', 'Bio::Phylo::Matrices::Datatype::Illumina', 'Bio::Phylo::Matrices::Datatype::Mixed', 'Bio::Phylo::Matrices::Datatype::Protein', 'Bio::Phylo::Matrices::Datatype::Restriction', 'Bio::Phylo::Matrices::Datatype::Rna', 'Bio::Phylo::Matrices::Datatype::Sanger', 'Bio::Phylo::Matrices::Datatype::Solexa', 'Bio::Phylo::Matrices::Datatype::Standard', 'Bio::Phylo::Matrices::Datum', 'Bio::Phylo::Matrices::DatumRole', 'Bio::Phylo::Matrices::Matrix', 'Bio::Phylo::Matrices::MatrixRole', 'Bio::Phylo::Matrices::TypeSafeData', 'Bio::Phylo::Mediators::TaxaMediator', 'Bio::Phylo::Models::Substitution::Dna', 'Bio::Phylo::Models::Substitution::Dna::F81', 'Bio::Phylo::Models::Substitution::Dna::GTR', 'Bio::Phylo::Models::Substitution::Dna::HKY85', 'Bio::Phylo::Models::Substitution::Dna::JC69', 'Bio::Phylo::Models::Substitution::Dna::K80', 'Bio::Phylo::NeXML::DOM', 'Bio::Phylo::NeXML::DOM::Document', 'Bio::Phylo::NeXML::DOM::Document::Libxml', 'Bio::Phylo::NeXML::DOM::Document::Twig', 'Bio::Phylo::NeXML::DOM::Element', 'Bio::Phylo::NeXML::DOM::Element::Libxml', 'Bio::Phylo::NeXML::DOM::Element::Twig', 'Bio::Phylo::NeXML::Entities', 'Bio::Phylo::NeXML::Meta', 'Bio::Phylo::NeXML::Meta::XMLLiteral', 'Bio::Phylo::NeXML::Writable', 'Bio::Phylo::Parsers::Abstract', 'Bio::Phylo::Parsers::Adjacency', 'Bio::Phylo::Parsers::Fasta', 'Bio::Phylo::Parsers::Fastq', 'Bio::Phylo::Parsers::Figtree', 'Bio::Phylo::Parsers::Newick', 'Bio::Phylo::Parsers::Nexml', 'Bio::Phylo::Parsers::Nexus', 'Bio::Phylo::Parsers::Phylip', 'Bio::Phylo::Parsers::Phyloxml', 'Bio::Phylo::Parsers::Table', 'Bio::Phylo::Parsers::Taxlist', 'Bio::Phylo::Parsers::Tnrs', 'Bio::Phylo::Parsers::Tolweb', 'Bio::Phylo::Parsers::Ubiocbmeta', 'Bio::Phylo::Parsers::Ubiometa', 'Bio::Phylo::Parsers::Ubiosearch', 'Bio::Phylo::PhyloWS', 'Bio::Phylo::PhyloWS::Client', 'Bio::Phylo::PhyloWS::Resource', 'Bio::Phylo::PhyloWS::Resource::Description', 'Bio::Phylo::PhyloWS::Service', 'Bio::Phylo::PhyloWS::Service::Timetree', 'Bio::Phylo::PhyloWS::Service::Tolweb', 'Bio::Phylo::PhyloWS::Service::UbioClassificationBank', 'Bio::Phylo::PhyloWS::Service::UbioNameBank', 'Bio::Phylo::Project', 'Bio::Phylo::Set', 'Bio::Phylo::Taxa', 'Bio::Phylo::Taxa::TaxaLinker', 'Bio::Phylo::Taxa::Taxon', 'Bio::Phylo::Taxa::TaxonLinker', 'Bio::Phylo::Treedrawer', 'Bio::Phylo::Treedrawer::Abstract', 'Bio::Phylo::Treedrawer::Canvas', 'Bio::Phylo::Treedrawer::Gif', 'Bio::Phylo::Treedrawer::Jpeg', 'Bio::Phylo::Treedrawer::Pdf', 'Bio::Phylo::Treedrawer::Png', 'Bio::Phylo::Treedrawer::Processing', 'Bio::Phylo::Treedrawer::Svg', 'Bio::Phylo::Unparsers::Abstract', 'Bio::Phylo::Unparsers::Adjacency', 'Bio::Phylo::Unparsers::Cdao', 'Bio::Phylo::Unparsers::Fasta', 'Bio::Phylo::Unparsers::Hennig86', 'Bio::Phylo::Unparsers::Html', 'Bio::Phylo::Unparsers::Mrp', 'Bio::Phylo::Unparsers::Newick', 'Bio::Phylo::Unparsers::Nexml', 'Bio::Phylo::Unparsers::Nexus', 'Bio::Phylo::Unparsers::Nwmsrdf', 'Bio::Phylo::Unparsers::Pagel', 'Bio::Phylo::Unparsers::Phylip', 'Bio::Phylo::Unparsers::Phyloxml', 'Bio::Phylo::Unparsers::Rss1', 'Bio::Phylo::Unparsers::Taxlist', 'Bio::Phylo::Util::CONSTANT', 'Bio::Phylo::Util::CONSTANT::Int', 'Bio::Phylo::Util::Dependency', 'Bio::Phylo::Util::Exceptions', 'Bio::Phylo::Util::IDPool', 'Bio::Phylo::Util::Logger', 'Bio::Phylo::Util::MOP', 'Bio::Phylo::Util::OptionalInterface', 'Bio::Phylo::Util::StackTrace', 'Bio::PhyloRole'])])), ('about', CommentedMap([('home', 'http://biophylo.blogspot.com/'), ('license', 'perl_5'), ('summary', 'An object-oriented Perl toolkit for analyzing and manipulating phyloinformatic data.')]))])), 'perl-bio-samtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-samtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-samtools'), ('version', '1.43')])), ('source', CommentedMap([('fn', 'Bio-SamTools-1.43.tar.gz'), ('url', 'http://www.cpan.org/authors/id/L/LD/LDS/Bio-SamTools-1.43.tar.gz'), ('md5', 'd36d7bfe91d6eaa8d220a2cb08d04a94'), ('patches', ['build-pl.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl-core', 'perl-module-build', 'samtools <=0.1.17', 'zlib']), ('run', ['perl-threaded', 'perl-bioperl-core', 'zlib'])])), ('test', CommentedMap([('imports', ['Bio::DB::Bam::AlignWrapper', 'Bio::DB::Bam::Alignment', 'Bio::DB::Bam::Pileup', 'Bio::DB::Bam::PileupWrapper', 'Bio::DB::Bam::Query', 'Bio::DB::Bam::Target', 'Bio::DB::Sam', 'Bio::DB::Sam::Constants'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~lds/Bio-SamTools-1.43/'), ('license', 'perl_5'), ('summary', 'Read SAM/BAM files')]))])), 'perl-bio-viennangs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bio-viennangs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bio-viennangs'), ('version', 'v0.16')])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Bio-ViennaNGS/'), ('license', 'Perl5'), ('summary', 'Bio::ViennaNGS is a distribution of Perl modules and utilities for building efficient Next-Generation Sequencing (NGS) analysis pipelines.')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('source', CommentedMap([('fn', 'Bio-ViennaNGS-v0.16.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/MT/MTW/Bio-ViennaNGS-v0.16.tar.gz'), ('sha256', '29797f0720b72a282f2d327cb41bf6c7c90de24a15ec21dfbed280e3c44428d0')])), ('requirements', CommentedMap([('build', ['bedtools >=2.24', 'perl-threaded', 'perl-bio-db-sam', 'perl-statistics-r', 'ucsc-fetchchromsizes', 'ucsc-bedgraphtobigwig', 'perl-app-cpanminus', 'libgcc', 'llvm', 'gcc']), ('run', ['bedtools >=2.24', 'perl-threaded', 'perl-bio-db-sam', 'perl-statistics-r', 'ucsc-fetchchromsizes', 'ucsc-bedgraphtobigwig', 'r-ggplot2 >=2.0.0', 'libgcc', 'llvm'])])), ('test', CommentedMap([('imports', ['Bio::ViennaNGS'])]))])), 'perl-bioperl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bioperl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bioperl'), ('version', '1.6.924')])), ('build', CommentedMap([('number', 7)])), ('requirements', CommentedMap([('build', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run']), ('run', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/BioPerl'), ('license', 'perl_5'), ('summary', 'Bioinformatics Toolkit')]))])), 'perl-bioperl-core': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bioperl-core/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bioperl-core'), ('version', '1.6.924')])), ('source', CommentedMap([('fn', 'BioPerl-1.6.924.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS/BioPerl-1.6.924.tar.gz'), ('md5', '3090437478919f2a1bd1bcb3d6df5839')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl', 'perl-data-stag', 'perl-io-string', 'perl-test-most', 'perl-module-build', 'perl-uri', 'perl-aceperl', 'perl-algorithm-munkres', 'perl-array-compare', 'perl-bio-phylo', 'perl-clone', 'perl-convert-binary-c', 'perl-db-file', 'perl-dbd-sqlite', 'perl-dbi', 'perl-error', 'perl-gd', 'perl-graphviz', 'perl-html-tableextract', 'perl-io-stringy', 'perl-list-moreutils', 'perl-postscript', 'perl-set-scalar', 'perl-soap-lite', 'perl-sort-naturally', 'perl-spreadsheet-parseexcel', 'perl-svg', 'perl-svg-graph', 'perl-xml-dom', 'perl-xml-dom-xpath', 'perl-xml-sax-writer', 'perl-xml-simple', 'perl-xml-twig', 'perl-xml-writer', 'perl-yaml']), ('run', ['perl', 'perl-data-stag', 'perl-io-string', 'perl-aceperl', 'perl-algorithm-munkres', 'perl-array-compare', 'perl-bio-asn1-entrezgene', 'perl-bio-featureio', 'perl-bio-phylo', 'perl-bio-samtools', 'perl-clone', 'perl-convert-binary-c', 'perl-db-file', 'perl-dbd-sqlite', 'perl-dbi', 'perl-error', 'perl-gd', 'perl-graphviz', 'perl-html-tableextract', 'perl-io-stringy', 'perl-list-moreutils', 'perl-postscript', 'perl-set-scalar', 'perl-soap-lite', 'perl-sort-naturally', 'perl-spreadsheet-parseexcel', 'perl-svg', 'perl-svg-graph', 'perl-xml-dom', 'perl-xml-dom-xpath', 'perl-xml-sax-writer', 'perl-xml-simple', 'perl-xml-twig', 'perl-xml-writer', 'perl-yaml'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/BioPerl'), ('license', 'perl_5'), ('summary', '\"Core\" packages for the BioPerl toolkit; you really should install perl-bioperl.')]))])), 'perl-bioperl-run': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bioperl-run/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bioperl-run'), ('version', '1.006900')])), ('source', CommentedMap([('fn', 'BioPerl-Run-1.006900.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS/BioPerl-Run-1.006900.tar.gz'), ('md5', 'c0ce55a64c1d7ab206c1b8f3d3ebd0aa')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl-core', 'perl-bio-samtools', 'perl-file-sort', 'perl-module-build', 'perl-test-most']), ('run', ['perl-threaded', 'perl-bioperl-core', 'perl-bio-samtools', 'perl-file-sort'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/BioPerl-Run'), ('license', 'perl_5'), ('summary', 'BioPerl-Run - wrapper toolkit')]))])), 'perl-biox-workflow-plugin-filedetails': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-biox-workflow-plugin-filedetails/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-biox-workflow-plugin-filedetails'), ('version', '0.11')])), ('source', CommentedMap([('fn', 'BioX-Workflow-Plugin-FileDetails-0.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/BioX-Workflow-Plugin-FileDetails-0.11.tar.gz'), ('md5', '05fbeddfb38e470cb22140511741fba4')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-file-find-rule', 'perl-moose', 'perl-biox-workflow', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-file-find-rule', 'perl-moose', 'perl-biox-workflow'])])), ('test', CommentedMap([('imports', ['BioX::Workflow::Plugin::FileDetails'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/BioX-Workflow-Plugin-FileDetails'), ('license', 'perl_5'), ('summary', 'Get metadata for files in directories processed by BioX::Workflow')]))])), 'perl-biox-workflow-plugin-fileexists': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-biox-workflow-plugin-fileexists/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-biox-workflow-plugin-fileexists'), ('version', '0.13')])), ('source', CommentedMap([('fn', 'BioX-Workflow-Plugin-FileExists-0.13.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/BioX-Workflow-Plugin-FileExists-0.13.tar.gz'), ('md5', '32d4be6dcf5a578e334c0c2d63d07359')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-moose', 'perl-module-build', 'perl-biox-workflow', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-moose', 'perl-biox-workflow'])])), ('test', CommentedMap([('imports', ['BioX::Workflow::Plugin::FileExists'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/BioX-Workflow-Plugin-FileExists'), ('license', 'perl_5'), ('summary', 'a plugin to BioX::Workflow')]))])), 'perl-bit-vector': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bit-vector/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bit-vector'), ('version', '7.4')])), ('source', CommentedMap([('fn', 'Bit-Vector-7.4.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/ST/STBEY/Bit-Vector-7.4.tar.gz'), ('md5', 'bf67f144e5be5327ed79d4c69e6e0086')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Bit::Vector'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/distribution/Bit-Vector/Vector.pod'), ('license', 'Perl'), ('summary', 'Efficient bit vector, set of integers and \"big int\" math library')]))])), 'perl-bloom-faster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bloom-faster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bloom-faster'), ('version', '1.7')])), ('source', CommentedMap([('fn', 'Bloom-Faster-1.7.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PA/PALVARO/Bloom-Faster-1.7.tar.gz'), ('md5', '6d0d26512f4e8e8f9df0523115a60092')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Bloom-Faster'), ('license', 'unknown'), ('summary', 'Perl extension for the c library libbloom.')]))])), 'perl-border-style': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-border-style/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-border-style'), ('version', '0.01')])), ('source', CommentedMap([('fn', 'Border-Style-0.01.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Border-Style-0.01.tar.gz'), ('md5', '341e4e4d3fa5b6c5b01e0622306cdb6b')])), ('requirements', CommentedMap([('build', ['perl', 'perl-term-app-roles', 'perl-moo', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-test-simple', 'perl-app-cpanminus']), ('run', ['perl', 'perl-term-app-roles', 'perl-moo', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Border::Style', 'Border::Style::Role'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Border-Style'), ('license', 'perl_5'), ('summary', 'Border style structure')]))])), 'perl-canary-stability': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-canary-stability/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-canary-stability'), ('version', '2006')])), ('source', CommentedMap([('fn', 'Canary-Stability-2006.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN/Canary-Stability-2006.tar.gz'), ('md5', '570ffd5fd8a399b6ba392c8451c8f5ab')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Canary::Stability'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Canary-Stability'), ('license', 'unknown'), ('summary', \"canary to check perl compatibility for schmorp's modules\")]))])), 'perl-capture-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-capture-tiny/0.34/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-capture-tiny'), ('version', '0.34')])), ('source', CommentedMap([('fn', 'Capture-Tiny-0.34.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Capture-Tiny-0.34.tar.gz'), ('md5', 'bca6ac7cd6cd88fa9cd9c99e52833a72')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Capture::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Capture-Tiny'), ('license', 'Apache 2.0'), ('summary', 'Capture STDOUT and STDERR from Perl, XS or external programs')]))])), 'perl-cgi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-cgi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-cgi'), ('version', '4.22')])), ('source', CommentedMap([('fn', 'CGI-4.22.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEEJO/CGI-4.22.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['CGI'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/distribution/CGI/lib/CGI.pod'), ('license', 'GPL'), ('summary', 'A generic file fetching mechanism')]))])), 'perl-class-accessor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-accessor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-accessor'), ('version', '0.34')])), ('source', CommentedMap([('fn', 'Class-Accessor-0.34.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-0.34.tar.gz'), ('md5', '0d9640d237a13276145f7e44b4855b89')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Class::Accessor'), ('license', 'perl_5'), ('summary', 'Automated accessor generation')]))])), 'perl-class-data-inheritable': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-data-inheritable/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-data-inheritable'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Class-Data-Inheritable-0.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TM/TMTM/Class-Data-Inheritable-0.08.tar.gz'), ('md5', 'fc0fe65926eb8fb932743559feb54eb9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', [\"perl -e 'use base qw(Class::Data::Inheritable);'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Class-Data-Inheritable'), ('license', 'perl_5'), ('summary', 'Inheritable, overridable class data')]))])), 'perl-class-inspector': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-inspector/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-inspector'), ('version', '1.28')])), ('source', CommentedMap([('fn', 'Class-Inspector-1.28.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Class-Inspector-1.28.tar.gz'), ('md5', '7ab3ca22d3147236dddfb58f2366ad22')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Class::Inspector', 'Class::Inspector::Functions'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Class-Inspector'), ('license', 'perl_5'), ('summary', 'Get information about a class and its structure')]))])), 'perl-class-load': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-load/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-load'), ('version', '0.23')])), ('source', CommentedMap([('fn', 'Class-Load-0.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Class-Load-0.23.tar.gz'), ('md5', '080973026b82051117b6c18f5d77aaa9')])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-data-optlist', 'perl-extutils-makemaker', 'perl-test-requires', 'perl-module-runtime', 'perl-apache-test', 'perl-module-implementation', 'perl-test-fatal', 'perl-try-tiny', 'perl-scalar-list-utils', 'perl-test-simple', 'perl-package-stash']), ('run', ['perl', 'perl-data-optlist', 'perl-module-runtime', 'perl-module-implementation', 'perl-apache-test', 'perl-try-tiny', 'perl-scalar-list-utils', 'perl-package-stash'])])), ('test', CommentedMap([('imports', ['Class::Load', 'Class::Load::PP'])])), ('about', CommentedMap([('home', 'https://github.com/moose/Class-Load'), ('license', 'perl_5'), ('summary', 'A working (require \"Class::Name\") and more')]))])), 'perl-class-load-xs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-load-xs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-load-xs'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'Class-Load-XS-0.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Class-Load-XS-0.09.tar.gz'), ('md5', '98eb8daf7f23c872fc7f503a7e34f598')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-test-requires', 'perl-class-load', 'perl-module-implementation', 'perl-apache-test', 'perl-test-fatal', 'perl-xsloader', 'perl-test-simple']), ('run', ['perl', 'perl-xsloader', 'perl-apache-test', 'perl-class-load'])])), ('test', CommentedMap([('imports', ['Class::Load::XS'])])), ('about', CommentedMap([('home', 'https://github.com/moose/Class-Load-XS'), ('license', 'artistic_2'), ('summary', 'XS implementation of parts of Class::Load')]))])), 'perl-class-method-modifiers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-method-modifiers/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-method-modifiers'), ('version', '2.11')])), ('source', CommentedMap([('fn', 'Class-Method-Modifiers-2.11.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/E/ET/ETHER/Class-Method-Modifiers-2.11.tar.gz'), ('md5', 'c6982e9e89325713d5afaff82cfe1744')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-requires', 'perl-test-fatal']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Class::Method::Modifiers'])])), ('about', CommentedMap([('home', 'https://github.com/moose/Class-Method-Modifiers'), ('license', 'perl_5'), ('summary', 'Provides Moose-like method modifiers')]))])), 'perl-class-methodmaker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-methodmaker/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Class-MethodMaker-2.24.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/S/SC/SCHWIGON/class-methodmaker/Class-MethodMaker-2.24.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp', 'perl-getopt-long']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp', 'perl-getopt-long'])])), ('test', CommentedMap([('commands', ['perl -e \"use Class::MethodMaker\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~schwigon/Class-MethodMaker-2.24/'), ('license', 'perl_5'), ('summary', 'Create generic methods for OO Perl')]))])), 'perl-class-std': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-std/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-std'), ('version', '0.013')])), ('source', CommentedMap([('fn', 'Class-Std-0.013.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CH/CHORNY/Class-Std-0.013.tar.gz'), ('md5', '82bcc56d9769edce671b0bd0e183ec3a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Class::Std'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Class-Std'), ('license', 'perl_5'), ('summary', 'Support for creating standard \"inside-out\" classes')]))])), 'perl-class-std-fast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-class-std-fast/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-class-std-fast'), ('version', '0.0.8')])), ('source', CommentedMap([('fn', 'Class-Std-Fast-v0.0.8.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AC/ACID/Class-Std-Fast-v0.0.8.tar.gz'), ('md5', 'd06d084cb84f295b0aabf2eee78ef746')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-scalar-list-utils', 'perl-module-build', 'perl-class-std']), ('run', ['perl-threaded', 'perl-class-std', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Class::Std::Fast'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Class-Std-Fast'), ('license', 'perl_5'), ('summary', 'faster but less secure than Class::Std')]))])), 'perl-clone': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-clone/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-clone'), ('version', '0.38')])), ('source', CommentedMap([('fn', 'Clone-0.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GA/GARU/Clone-0.38.tar.gz'), ('md5', '17d119f9579b4b3ec43c87a4138ed44a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Clone'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Clone'), ('license', 'perl_5'), ('summary', 'recursively copy Perl datatypes')]))])), 'perl-clone-pp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-clone-pp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-clone-pp'), ('version', '1.06')])), ('source', CommentedMap([('fn', 'Clone-PP-1.06.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/N/NE/NEILB/Clone-PP-1.06.tar.gz'), ('md5', '1aa5a17c47f168488d013bd6051658fa')])), ('requirements', CommentedMap([('build', ['perl', 'perl-apache-test', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-apache-test'])])), ('test', CommentedMap([('imports', ['Clone::PP'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Clone::PP'), ('license', 'perl_5'), ('summary', 'Recursively copy Perl datatypes')]))])), 'perl-color-theme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-color-theme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-color-theme'), ('version', '0.01')])), ('source', CommentedMap([('fn', 'Color-Theme-0.01.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Color-Theme-0.01.tar.gz'), ('md5', '732aacdbca2ee6c5dbf59c7043602ec7')])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-simple', 'perl-moo', 'perl-extutils-makemaker', 'perl-term-app-roles', 'perl-pathtools', 'perl-app-cpanminus']), ('run', ['perl', 'perl-term-app-roles', 'perl-moo', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Color::Theme', 'Color::Theme::Role', 'Color::Theme::Role::ANSI', 'Color::Theme::Util'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Color-Theme'), ('license', 'perl_5'), ('summary', 'Color theme structure')]))])), 'perl-common-sense': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-common-sense/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-common-sense'), ('version', '3.74')])), ('source', CommentedMap([('fn', 'common-sense-3.74.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/ML/MLEHMANN/common-sense-3.74.tar.gz'), ('md5', '35b273147200c4c95eef7816f83e572d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['common::sense'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~mlehmann/common-sense-3.74/'), ('license', 'perl_5'), ('summary', 'Sane defaults for Perl programs')]))])), 'perl-compress-raw-bzip2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-compress-raw-bzip2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-compress-raw-bzip2'), ('version', 2.069)])), ('source', CommentedMap([('fn', 'Compress-Raw-Bzip2-2.069.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PM/PMQS/Compress-Raw-Bzip2-2.069.tar.gz'), ('md5', '18c4312b4fb8b644e9d697803687ea4f')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Compress::Raw::Bzip2'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Compress::Raw::Bzip2'), ('license', 'perl_5'), ('summary', 'Low-Level Interface to bzip2 compression library')]))])), 'perl-compress-raw-zlib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-compress-raw-zlib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-compress-raw-zlib'), ('version', '2.069')])), ('source', CommentedMap([('fn', 'Compress-Raw-Zlib-2.069.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PM/PMQS/Compress-Raw-Zlib-2.069.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Compress::Raw::Zlib'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Compress::Raw::Zlib'), ('license', 'Perl'), ('summary', 'Low-Level Interface to zlib compression library')]))])), 'perl-config-any': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-config-any/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-config-any'), ('version', '0.27')])), ('source', CommentedMap([('fn', 'Config-Any-0.27.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BR/BRICAS/Config-Any-0.27.tar.gz'), ('md5', '0c8db36ebfeea250f554ebc50ee5b643')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Config::Any', 'Config::Any::Base', 'Config::Any::General', 'Config::Any::INI', 'Config::Any::JSON', 'Config::Any::Perl', 'Config::Any::XML', 'Config::Any::YAML'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Config-Any'), ('license', 'perl_5'), ('summary', 'Load configuration from different file formats, transparently')]))])), 'perl-config-autoconf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-config-autoconf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-config-autoconf'), ('version', '0.311')])), ('source', CommentedMap([('fn', 'Config-AutoConf-0.311.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RE/REHSACK/Config-AutoConf-0.311.tar.gz'), ('md5', 'fa19f46bbd5bb0bd23ee41e8fd7b533f')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'perl-capture-tiny']), ('run', ['perl-threaded', 'perl-capture-tiny'])])), ('test', CommentedMap([('imports', ['Config::AutoConf'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Config-AutoConf'), ('license', 'perl_5'), ('summary', 'A module to implement some of AutoConf macros in pure perl.')]))])), 'perl-config-general': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-config-general/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-config-general'), ('version', '2.61')])), ('source', CommentedMap([('fn', 'Config-General-2.61.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TL/TLINDEN/Config-General-2.61.tar.gz'), ('md5', '4def4783b285a902543aeb0a3b163ac6')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-pathtools'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Config-General'), ('license', 'perl_5'), ('summary', 'Generic Config Module')]))])), 'perl-config-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-config-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-config-simple'), ('version', '4.58')])), ('source', CommentedMap([('fn', 'Config-Simple-4.58.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SH/SHERZODR/Config-Simple-4.58.tar.gz'), ('md5', 'f014aec54f0a1e2e880d317180fce502')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Config::Simple'), ('license', 'unknown'), ('summary', 'simple configuration file class')]))])), 'perl-config-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-config-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-config-tiny'), ('version', '2.23')])), ('source', CommentedMap([('fn', 'Config-Tiny-2.23.tgz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSAVAGE/Config-Tiny-2.23.tgz'), ('md5', '6c3bc610c4bc587e1441e651e01584ce')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-pathtools', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-pathtools'])])), ('test', CommentedMap([('imports', ['Config::Tiny'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Config::Tiny'), ('license', 'perl_5'), ('summary', 'Read/Write .ini style files with as little code as possible')]))])), 'perl-convert-binary-c': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-convert-binary-c/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-convert-binary-c'), ('version', '0.78')])), ('source', CommentedMap([('fn', 'Convert-Binary-C-0.78.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MH/MHX/Convert-Binary-C-0.78.tar.gz'), ('md5', '89ebf7298890c1a98098006b40c62828')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Convert::Binary::C', 'Convert::Binary::C::Cached'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~mhx/Convert-Binary-C/'), ('license', 'perl_5'), ('summary', 'Binary Data Conversion using C Types')]))])), 'perl-convert-binhex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-convert-binhex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-convert-binhex'), ('version', '1.125')])), ('source', CommentedMap([('fn', 'Convert-BinHex-1.125.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/ST/STEPHEN/Convert-BinHex-1.125.tar.gz'), ('md5', '2f485e1283387ead5b92243e1aacff9d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-file-slurp', 'perl-test-most']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Convert::BinHex'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Convert-BinHex'), ('license', 'perl_5'), ('summary', 'extract data from Macintosh BinHex files')]))])), 'perl-cpan-meta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-cpan-meta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-cpan-meta'), ('version', 2.120921)])), ('source', CommentedMap([('fn', 'CPAN-Meta-2.120921.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/CPAN-Meta-2.120921.tar.gz'), ('md5', '83d00ee341ca2a6d8d5e5f9d4bd9d41e')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/cpan-meta'), ('license', 'perl_5'), ('summary', 'the distribution metadata for a CPAN dist')]))])), 'perl-cpan-meta-check': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-cpan-meta-check/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-cpan-meta-check'), ('version', '0.012')])), ('source', CommentedMap([('fn', 'CPAN-Meta-Check-0.012.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/CPAN-Meta-Check-0.012.tar.gz'), ('md5', 'adb415398390e83f09e4b91f73a6fd7f')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-deep']), ('run', ['perl-threaded'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['CPAN::Meta::Check'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/CPAN-Meta-Check'), ('license', 'perl_5'), ('summary', 'Verify requirements in a CPAN::Meta object')]))])), 'perl-crypt-rc4': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-crypt-rc4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-crypt-rc4'), ('version', '2.02')])), ('source', CommentedMap([('fn', 'Crypt-RC4-2.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SI/SIFUKURT/Crypt-RC4-2.02.tar.gz'), ('md5', '4ca59a7e58ac9597c3b4f3f46ea22629')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Crypt::RC4'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Crypt-RC4'), ('license', 'unknown'), ('summary', 'Perl implementation of the RC4 encryption algorithm')]))])), 'perl-data-dump': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-dump/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-data-dump'), ('version', '1.23')])), ('source', CommentedMap([('fn', 'Data-Dump-1.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GA/GAAS/Data-Dump-1.23.tar.gz'), ('md5', '762c111e525c82ff23d62c90821b26e9')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl']), ('run', ['libgcc', 'perl'])])), ('test', None), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Data::Dump'), ('license', 'perl_5'), ('summary', 'Pretty printing of data structure')]))])), 'perl-data-dumper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-dumper/2.161/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Dumper-2.161.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/Data-Dumper-2.161.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Data::Dumper'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data::Dumper'), ('license', 'unknown'), ('summary', 'seeds germane, yet not germinated')]))])), 'perl-data-optlist': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-optlist/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-data-optlist'), ('version', '0.110')])), ('source', CommentedMap([('fn', 'Data-OptList-0.110.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Data-OptList-0.110.tar.gz'), ('md5', 'f9236c9ea5607134ad8a2b3dc901c4c5')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-pathtools', 'perl-apache-test', 'perl-scalar-list-utils', 'perl-test-simple', 'perl-params-util', 'perl-sub-install']), ('run', ['perl-threaded', 'perl-scalar-list-utils', 'perl-apache-test', 'perl-params-util', 'perl-sub-install'])])), ('test', CommentedMap([('imports', ['Data::OptList'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Data-OptList'), ('license', 'perl_5'), ('summary', 'parse and validate simple name/value option pairs')]))])), 'perl-data-stag': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-stag/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-data-stag'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'Data-Stag-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CM/CMUNGALL/Data-Stag-0.14.tar.gz'), ('md5', 'f803acf74f1bfccc118aeac5483ee871')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-string', 'perl-graph', 'perl-json', 'perl-xml-libxml', 'perl-libxml-perl', 'perl-xml-libxslt', 'perl-mldbm']), ('run', ['perl-threaded', 'perl-io-string', 'perl-graph', 'perl-json', 'perl-xml-libxml', 'perl-libxml-perl', 'perl-xml-libxslt', 'perl-mldbm'])])), ('test', CommentedMap([('imports', ['Data::Stag', 'Data::Stag::Arr2HTML', 'Data::Stag::Base', 'Data::Stag::BaseGenerator', 'Data::Stag::BaseHandler', 'Data::Stag::ChainHandler', 'Data::Stag::DTDWriter', 'Data::Stag::GraphHandler', 'Data::Stag::HashDB', 'Data::Stag::ITextParser', 'Data::Stag::ITextWriter', 'Data::Stag::IndentParser', 'Data::Stag::IndentWriter', 'Data::Stag::JSONWriter', 'Data::Stag::PerlWriter', 'Data::Stag::PodParser', 'Data::Stag::Simple', 'Data::Stag::StagDB', 'Data::Stag::StagI', 'Data::Stag::StagImpl', 'Data::Stag::SxprParser', 'Data::Stag::SxprWriter', 'Data::Stag::Util', 'Data::Stag::Writer', 'Data::Stag::XMLParser', 'Data::Stag::XMLWriter', 'Data::Stag::XSLHandler', 'Data::Stag::XSLTHandler', 'Data::Stag::null'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data-Stag'), ('license', 'unknown'), ('summary', 'Structured Tags')]))])), 'perl-data-uuid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-uuid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-data-uuid'), ('version', '1.221')])), ('source', CommentedMap([('fn', 'Data-UUID-1.221.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Data-UUID-1.221.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Data::UUID'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Data::UUID'), ('license', 'Perl'), ('summary', 'Globally/Universally Unique Identifiers (GUIDs/UUIDs)')]))])), 'perl-date-manip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-date-manip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-date-manip'), ('version', '6.57')])), ('source', CommentedMap([('fn', 'Date-Manip-6.57.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SB/SBECK/Date-Manip-6.57.tar.gz'), ('md5', 'cd87e060f87662fc4cfebe7b3867ddf6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-encode-locale', 'perl-test-inter', 'perl-io-all', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-encode-locale', 'perl-io-all'])])), ('test', CommentedMap([('imports', ['Date::Manip', 'Date::Manip::Base', 'Date::Manip::DM5', 'Date::Manip::DM5abbrevs', 'Date::Manip::DM6', 'Date::Manip::Date', 'Date::Manip::Delta', 'Date::Manip::Lang::catalan', 'Date::Manip::Lang::danish', 'Date::Manip::Lang::dutch', 'Date::Manip::Lang::english', 'Date::Manip::Lang::finnish', 'Date::Manip::Lang::french', 'Date::Manip::Lang::german', 'Date::Manip::Lang::index', 'Date::Manip::Lang::italian', 'Date::Manip::Lang::norwegian', 'Date::Manip::Lang::polish', 'Date::Manip::Lang::portugue', 'Date::Manip::Lang::romanian', 'Date::Manip::Lang::russian', 'Date::Manip::Lang::spanish', 'Date::Manip::Lang::swedish', 'Date::Manip::Lang::turkish', 'Date::Manip::Obj', 'Date::Manip::Recur', 'Date::Manip::TZ', 'Date::Manip::TZ_Base', 'Date::Manip::TZdata', 'Date::Manip::Zones'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Date::Manip'), ('license', 'perl_5'), ('summary', 'Date manipulation routines')]))])), 'perl-db-file': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-db-file/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-db-file'), ('version', 1.835)])), ('source', CommentedMap([('fn', 'DB_File-1.835.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PM/PMQS/DB_File-1.835.tar.gz')])), ('build', CommentedMap([('number', 5)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'libdb']), ('run', ['libgcc', 'perl-threaded', 'libdb'])])), ('test', CommentedMap([('imports', ['DB_File'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Set::IntervalTree'), ('license', 'perl_5'), ('summary', 'Perl5 access to Berkeley DB version 1.x.')]))])), 'perl-dbd-mysql': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbd-mysql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbd-mysql'), ('version', '4.033')])), ('source', CommentedMap([('fn', 'DBD-mysql-4.033.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.033.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi', 'mysqlclient']), ('run', ['libgcc', 'perl-threaded', 'perl-dbi', 'mysqlclient'])])), ('test', CommentedMap([('imports', ['DBD::mysql'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/DBD::mysql'), ('license', 'Perl'), ('summary', 'MySQL driver for the Perl5 Database Interface (DBI)')]))])), 'perl-dbd-pg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbd-pg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbd-pg'), ('version', '3.5.3')])), ('source', CommentedMap([('fn', 'DBD-Pg-3.5.3.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/T/TU/TURNSTEP/DBD-Pg-3.5.3.tar.gz'), ('md5', '21cdf31a8d1f77466920375aa766c164')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-dbi', 'postgresql']), ('run', ['perl-threaded', 'perl-dbi', 'postgresql'])])), ('test', CommentedMap([('imports', ['DBD::Pg'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/DBD-Pg/'), ('license', 'perl_5'), ('summary', 'PostgreSQL database driver for the DBI module')]))])), 'perl-dbd-sqlite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbd-sqlite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbd-sqlite'), ('version', '1.50')])), ('source', CommentedMap([('fn', 'DBD-SQLite-1.50.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/I/IS/ISHIGAKI/DBD-SQLite-1.50.tar.gz')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi']), ('run', ['perl-threaded', 'perl-dbi'])])), ('test', CommentedMap([('imports', ['DBD::SQLite'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/DBD::SQLite'), ('license', 'Perl'), ('summary', 'Self Contained RDBMS in a DBI Driver')]))])), 'perl-dbi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbi'), ('version', '1.636')])), ('source', CommentedMap([('fn', 'DBI-1.636.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TI/TIMB/DBI-1.636.tar.gz'), ('md5', '60f291e5f015550dde71d1858dfe93ba')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/DBI'), ('license', 'Perl'), ('summary', 'Database independent interface for Perl')]))])), 'perl-dbm-deep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbm-deep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbm-deep'), ('version', '2.0013')])), ('source', CommentedMap([('fn', 'DBM-Deep-2.0013.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RK/RKINYON/DBM-Deep-2.0013.tar.gz'), ('md5', 'ea3e7afa296a4c7b87cf06eac3b674bf')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-file-path', 'perl-test-simple', 'perl-test-deep', 'perl-scalar-list-utils', 'perl-test-warn', 'perl-test-exception', 'perl-module-build', 'perl-digest-md5', 'perl-dbi']), ('run', ['perl-threaded', 'perl-digest-md5', 'perl-scalar-list-utils', 'perl-dbi'])])), ('test', CommentedMap([('imports', ['DBM::Deep', 'DBM::Deep::Array', 'DBM::Deep::Engine', 'DBM::Deep::Engine::DBI', 'DBM::Deep::Engine::File', 'DBM::Deep::Hash', 'DBM::Deep::Iterator', 'DBM::Deep::Iterator::DBI', 'DBM::Deep::Iterator::File', 'DBM::Deep::Iterator::File::BucketList', 'DBM::Deep::Iterator::File::Index', 'DBM::Deep::Null', 'DBM::Deep::Sector', 'DBM::Deep::Sector::DBI', 'DBM::Deep::Sector::DBI::Reference', 'DBM::Deep::Sector::DBI::Scalar', 'DBM::Deep::Sector::File', 'DBM::Deep::Sector::File::BucketList', 'DBM::Deep::Sector::File::Data', 'DBM::Deep::Sector::File::Index', 'DBM::Deep::Sector::File::Null', 'DBM::Deep::Sector::File::Reference', 'DBM::Deep::Sector::File::Scalar', 'DBM::Deep::Storage', 'DBM::Deep::Storage::DBI', 'DBM::Deep::Storage::File'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/DBM::Deep'), ('license', 'perl_5'), ('summary', 'A pure perl multi-level hash/array DBM that supports transactions')]))])), 'perl-devel-checkbin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-devel-checkbin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-devel-checkbin'), ('version', '0.04')])), ('source', CommentedMap([('fn', 'Devel::CheckBin-0.04.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TOKUHIROM/Devel-CheckBin-0.04.tar.gz'), ('md5', '042b68e48d9b53de7d3ef4c726d57cb2')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Devel::CheckBin'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Devel::CheckBin'), ('license', 'perl_5'), ('summary', 'check that a command is available')]))])), 'perl-devel-globaldestruction': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-devel-globaldestruction/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-devel-globaldestruction'), ('version', '0.13')])), ('source', CommentedMap([('fn', 'Devel-GlobalDestruction-0.13.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/H/HA/HAARG/Devel-GlobalDestruction-0.13.tar.gz'), ('md5', 'e7be00040827e204b2b6cba2f3166074')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-sub-exporter-progressive']), ('run', ['perl-threaded', 'perl-sub-exporter-progressive'])])), ('test', CommentedMap([('imports', ['Devel::GlobalDestruction'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Devel-GlobalDestruction'), ('license', 'perl_5'), ('summary', \"Provides function returning the equivalent of ${^GLOBAL_PHASE} eq 'DESTRUCT' for older perls.\")]))])), 'perl-devel-size': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-devel-size/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-devel-size'), ('version', '0.80')])), ('source', CommentedMap([('fn', 'Devel-Size-0.80.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NW/NWCLARK/Devel-Size-0.80.tar.gz'), ('md5', '49d5060edaf076bc757a028614c60f41')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Devel::Size'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Devel::Size'), ('license', 'perl_5'), ('summary', 'Perl extension for finding the memory usage of Perl variables')]))])), 'perl-devel-stacktrace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-devel-stacktrace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-devel-stacktrace'), ('version', '2.00')])), ('source', CommentedMap([('fn', 'Devel-StackTrace-2.00.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Devel-StackTrace-2.00.tar.gz'), ('md5', '826ed2bc7cdd8d852d7d2d8b69aa313c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Devel::StackTrace', 'Devel::StackTrace::Frame'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Devel-StackTrace'), ('license', 'artistic_2'), ('summary', 'An object representing a stack trace')]))])), 'perl-digest-crc32': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-crc32/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-digest-crc32'), ('version', '0.01')])), ('source', CommentedMap([('fn', 'Digest-Crc32-0.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/F/FA/FAYS/Digest-Crc32-0.01.tar.gz'), ('md5', '4808a7fd130218cd1193d8a496d8ec92')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Digest::Crc32'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest::Crc32'), ('license', 'perl_5'), ('summary', 'Cyclic Redundency Check digests implementation')]))])), 'perl-digest-hmac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-hmac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-digest-hmac'), ('version', '1.03')])), ('source', CommentedMap([('fn', 'Digest-HMAC-1.03.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/Digest-HMAC-1.03.tar.gz'), ('md5', 'e6a5d6f552da16eacb5157ea4369ff9d')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Digest::HMAC'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest-HMAC'), ('license', 'perl_5'), ('summary', 'Keyed-Hashing for Message Authentication')]))])), 'perl-digest-md5': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-md5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-digest-md5'), ('version', '2.52')])), ('source', CommentedMap([('fn', 'Digest-MD5-2.52.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/Digest-MD5-2.52.tar.gz'), ('sha256', '0a7caed9931af2123e64f110e284e33d88cdd5d9a7e0514ae5a8b7ae1b90e43e')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Digest::MD5'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest-MD5'), ('license', 'perl_5'), ('summary', 'Perl interface to the MD-5 algorithm')]))])), 'perl-digest-md5-file': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-md5-file/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-digest-md5-file'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Digest-MD5-File-0.08.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/D/DM/DMUEY/Digest-MD5-File-0.08.tar.gz'), ('sha256', 'adb43a54e32627b4f7e57c9640e6eb06d0bb79d8ea54cd0bd79ed35688fb1218')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-lwp-simple']), ('run', ['perl', 'perl-lwp-simple'])])), ('test', CommentedMap([('imports', ['Digest::MD5::File'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Digest-MD5-File/'), ('license', 'perl_5'), ('summary', 'Perl extension for getting MD5 sums for files and urls.')]))])), 'perl-digest-perl-md5': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-perl-md5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-digest-perl-md5'), ('version', '1.9')])), ('source', CommentedMap([('fn', 'Digest-Perl-MD5-1.9.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DE/DELTA/Digest-Perl-MD5-1.9.tar.gz'), ('md5', '41f1160e11ce0d99021fc8d6c725fa3d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Digest::Perl::MD5'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest-Perl-MD5'), ('license', 'unknown'), ('summary', \"Perl Implementation of Rivest's MD5 algorithm\")]))])), 'perl-dist-checkconflicts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dist-checkconflicts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dist-checkconflicts'), ('version', '0.11')])), ('source', CommentedMap([('fn', 'Dist-CheckConflicts-0.11.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DO/DOY/Dist-CheckConflicts-0.11.tar.gz'), ('md5', 'c8725a92b9169708b0f63036812070f2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-runtime', 'perl-test-fatal']), ('run', ['perl-threaded', 'perl-module-runtime'])])), ('test', CommentedMap([('imports', ['Dist::CheckConflicts'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Dist-CheckConflicts'), ('license', 'perl_5'), ('summary', 'declare version conflicts for your dist')]))])), 'perl-email-date-format': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-email-date-format/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-email-date-format'), ('version', '1.005')])), ('source', CommentedMap([('fn', 'Email-Date-Format-1.005.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RJ/RJBS/Email-Date-Format-1.005.tar.gz'), ('md5', '030dcee3bc1a44674900b0132925bd03')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Email::Date::Format'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Email-Date-Format'), ('license', 'perl_5'), ('summary', 'produce RFC 2822 date strings')]))])), 'perl-encode-locale': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-encode-locale/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-encode-locale'), ('version', '1.05')])), ('source', CommentedMap([('fn', 'Encode-Locale-1.05.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GA/GAAS/Encode-Locale-1.05.tar.gz')])), ('build', CommentedMap([('number', 4)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('imports', ['Encode::Locale'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Encode::Locale'), ('license', 'Perl'), ('summary', 'Determine the locale encoding')]))])), 'perl-env-path': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-env-path/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-env-path'), ('version', '0.19')])), ('source', CommentedMap([('fn', 'Env-Path-0.19.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DS/DSB/Env-Path-0.19.tar.gz'), ('md5', 'e0990baf757ce4f8145dd18817a589a9')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Env::Path'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Env::Path'), ('license', 'unknown'), ('summary', 'Advanced operations on path variables')]))])), 'perl-error': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-error/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-error'), ('version', '0.17024')])), ('source', CommentedMap([('fn', 'Error-0.17024.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/Error-0.17024.tar.gz'), ('md5', '7e3676a7fb14f65f568f12d4b40a2a72')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Error', 'Error::Simple'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Error'), ('license', 'perl_5'), ('summary', 'Error/exception handling in an OO-ish way')]))])), 'perl-excel-writer-xlsx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-excel-writer-xlsx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-excel-writer-xlsx'), ('version', '0.95')])), ('source', CommentedMap([('fn', 'Excel-Writer-XLSX-0.95.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JM/JMCNAMARA/Excel-Writer-XLSX-0.95.tar.gz'), ('md5', '377094b38e541afdbb21a02a7d433ac3')])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-archive-zip']), ('run', ['perl', 'perl-archive-zip'])])), ('test', CommentedMap([('imports', ['Excel::Writer::XLSX', 'Excel::Writer::XLSX::Chart', 'Excel::Writer::XLSX::Chart::Area', 'Excel::Writer::XLSX::Chart::Bar', 'Excel::Writer::XLSX::Chart::Column', 'Excel::Writer::XLSX::Chart::Doughnut', 'Excel::Writer::XLSX::Chart::Line', 'Excel::Writer::XLSX::Chart::Pie', 'Excel::Writer::XLSX::Chart::Radar', 'Excel::Writer::XLSX::Chart::Scatter', 'Excel::Writer::XLSX::Chart::Stock', 'Excel::Writer::XLSX::Chartsheet', 'Excel::Writer::XLSX::Drawing', 'Excel::Writer::XLSX::Examples', 'Excel::Writer::XLSX::Format', 'Excel::Writer::XLSX::Package::App', 'Excel::Writer::XLSX::Package::Comments', 'Excel::Writer::XLSX::Package::ContentTypes', 'Excel::Writer::XLSX::Package::Core', 'Excel::Writer::XLSX::Package::Custom', 'Excel::Writer::XLSX::Package::Packager', 'Excel::Writer::XLSX::Package::Relationships', 'Excel::Writer::XLSX::Package::SharedStrings', 'Excel::Writer::XLSX::Package::Styles', 'Excel::Writer::XLSX::Package::Table', 'Excel::Writer::XLSX::Package::Theme', 'Excel::Writer::XLSX::Package::VML', 'Excel::Writer::XLSX::Package::XMLwriter', 'Excel::Writer::XLSX::Shape', 'Excel::Writer::XLSX::Utility', 'Excel::Writer::XLSX::Workbook', 'Excel::Writer::XLSX::Worksheet'])])), ('about', CommentedMap([('home', 'http://jmcnamara.github.com/excel-writer-xlsx/'), ('license', 'perl_5'), ('summary', 'Create a new file in the Excel 2007+ XLSX format.')]))])), 'perl-exception-class': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-exception-class/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-exception-class'), ('version', '1.40')])), ('source', CommentedMap([('fn', 'Exception-Class-1.40.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Exception-Class-1.40.tar.gz'), ('md5', '07cd7a38be5f8134955e943cf666d453')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-class-data-inheritable', 'perl-devel-stacktrace']), ('run', ['perl-threaded', 'perl-class-data-inheritable', 'perl-devel-stacktrace'])])), ('test', CommentedMap([('imports', ['Exception::Class', 'Exception::Class::Base'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Exception-Class'), ('license', 'perl_5'), ('summary', 'A module that allows you to declare real exception classes in Perl')]))])), 'perl-exporter-tidy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-exporter-tidy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-exporter-tidy'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Exporter-Tidy-0.08.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JU/JUERD/Exporter-Tidy-0.08.tar.gz'), ('md5', '025866016bd6ca361052acb30bcf07f4')])), ('requirements', CommentedMap([('build', ['perl-threaded >=5.22.0']), ('run', ['perl-threaded >=5.22.0'])])), ('test', CommentedMap([('imports', ['Exporter::Tidy'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Exporter-Tidy'), ('license', 'unknown'), ('summary', 'Another way of exporting symbols')]))])), 'perl-exporter-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-exporter-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-exporter-tiny'), ('version', '0.042')])), ('source', CommentedMap([('fn', 'Exporter-Tiny-0.042.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TO/TOBYINK/Exporter-Tiny-0.042.tar.gz'), ('md5', '77ba432037f35bc9b84b717639075fc6')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-fatal', 'perl-test-warnings']), ('run', ['perl-threaded'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['Exporter::Shiny', 'Exporter::Tiny'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Exporter-Tiny'), ('license', 'perl_5'), ('summary', 'Exporter with the features of Sub::Exporter but only core dependencies')]))])), 'perl-extutils-config': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-config/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-extutils-config'), ('version', '0.008')])), ('source', CommentedMap([('fn', 'ExtUtils-Config-0.008.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/L/LE/LEONT/ExtUtils-Config-0.008.tar.gz'), ('md5', '565a7b09c7cac5907a25bbe2c959a717')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['ExtUtils::Config'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/ExtUtils-Config'), ('license', 'perl_5'), ('summary', \"A wrapper for perl's configuration\")]))])), 'perl-extutils-helpers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-helpers/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-extutils-helpers'), ('version', '0.022')])), ('source', CommentedMap([('fn', 'ExtUtils-Helpers-0.022.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/L/LE/LEONT/ExtUtils-Helpers-0.022.tar.gz'), ('md5', 'cf4fd6f8caa6daac33b1111c9e93162b')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-pathtools'])])), ('test', CommentedMap([('imports', ['ExtUtils::Helpers', 'ExtUtils::Helpers::Unix', 'ExtUtils::Helpers::VMS', 'ExtUtils::Helpers::Windows'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/ExtUtils-Helpers'), ('license', 'perl_5'), ('summary', 'Various portability utilities for module builders')]))])), 'perl-extutils-installpaths': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-installpaths/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-extutils-installpaths'), ('version', '0.011')])), ('source', CommentedMap([('fn', 'ExtUtils-InstallPaths-0.011.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/ExtUtils-InstallPaths-0.011.tar.gz'), ('md5', '9c75894c3c8c899ab6bfafc5eaa97999')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-extutils-config']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-extutils-config'])])), ('test', CommentedMap([('imports', ['ExtUtils::InstallPaths'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/ExtUtils-InstallPaths'), ('license', 'perl_5'), ('summary', 'Build.PL install path logic made easy')]))])), 'perl-extutils-makemaker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-makemaker/7.24/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ExtUtils-MakeMaker-7.24.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.24.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-data-dumper']), ('run', ['perl', 'perl-app-cpanminus', 'perl-data-dumper'])])), ('test', CommentedMap([('commands', ['perl -e \"use ExtUtils::Command\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/ExtUtils-MakeMaker'), ('license', 'perl_5'), ('summary', 'Create a module Makefile')]))])), 'perl-extutils-manifest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-manifest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-extutils-manifest'), ('version', '1.70')])), ('source', CommentedMap([('fn', 'ExtUtils-Manifest-1.70.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/ExtUtils-Manifest-1.70.tar.gz'), ('md5', '8ccb9c1f3bca14c117a74bc7b43be095')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['ExtUtils::Manifest'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/ExtUtils-Manifest'), ('license', 'perl_5'), ('summary', 'Utilities to write and check a MANIFEST file')]))])), 'perl-extutils-parsexs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-extutils-parsexs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-extutils-parsexs'), ('version', '3.28')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['ExtUtils::ParseXS', 'ExtUtils::ParseXS::Constants', 'ExtUtils::ParseXS::CountLines', 'ExtUtils::ParseXS::Eval', 'ExtUtils::ParseXS::Utilities', 'ExtUtils::Typemaps', 'ExtUtils::Typemaps::Cmd', 'ExtUtils::Typemaps::InputMap', 'ExtUtils::Typemaps::OutputMap', 'ExtUtils::Typemaps::Type'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/ExtUtils::ParseXS'), ('license', 'unknown'), ('summary', 'converts Perl XS code into C code')]))])), 'perl-file-chdir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-chdir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-chdir'), ('version', '0.1010')])), ('source', CommentedMap([('fn', 'File-chdir-0.1010.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/File-chdir-0.1010.tar.gz'), ('md5', 'aacfafa4a8f13d45be67451e0f9b483e')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-test-simple', 'perl-extutils-makemaker', 'perl-apache-test']), ('run', ['perl-threaded', 'perl-pathtools'])])), ('test', CommentedMap([('imports', ['File::chdir'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/File-chdir'), ('license', 'perl_5'), ('summary', 'a more sensible way to change directories')]))])), 'perl-file-copy-recursive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-copy-recursive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-copy-recursive'), ('version', '0.38')])), ('source', CommentedMap([('fn', 'File-Copy-Recursive-0.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DM/DMUEY/File-Copy-Recursive-0.38.tar.gz')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Copy::Recursive'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/File::Copy::Recursive'), ('license', 'Perl')]))])), 'perl-file-details': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-details/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-details'), ('version', '0.003')])), ('source', CommentedMap([('fn', 'File-Details-0.003.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RE/RECSKY/File-Details-0.003.tar.gz'), ('md5', '557b420008cf640740932249957c6b89')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-class-accessor']), ('run', ['perl-threaded', 'perl-class-accessor'])])), ('test', CommentedMap([('imports', ['File::Details'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Details'), ('license', 'perl_5'), ('summary', 'File details in an object, stat, hash, etc..')]))])), 'perl-file-fetch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-fetch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-fetch'), ('version', '0.48')])), ('source', CommentedMap([('fn', 'File-Fetch-0.48.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/File-Fetch-0.48.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Fetch'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/File::Fetch'), ('license', 'Perl'), ('summary', 'A generic file fetching mechanism')]))])), 'perl-file-find-rule': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-find-rule/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-find-rule'), ('version', '0.34')])), ('source', CommentedMap([('url', 'https://cpan.metacpan.org/authors/id/R/RC/RCLAMP/File-Find-Rule-0.34.tar.gz'), ('md5', 'a7aa9ad4d8ee87b2a77b8e3722768712')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-app-cpanminus', 'perl-extutils-makemaker', 'perl-module-build', 'perl-number-compare', 'perl-text-glob']), ('run', ['perl-app-cpanminus', 'perl-module-build', 'perl-number-compare', 'perl-text-glob'])])), ('test', CommentedMap([('imports', ['File::Find::Rule'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/File::Find::Rule'), ('license', 'perl_5'), ('summary', 'Alternative interface to File::Find')]))])), 'perl-file-grep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-grep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-grep'), ('version', '0.02')])), ('source', CommentedMap([('fn', 'File-Grep-0.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MN/MNEYLON/File-Grep-0.02.tar.gz'), ('md5', '6ea1d06a385229b1c015db23e1ef93fa')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Grep'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Grep'), ('license', 'unknown'), ('summary', 'Find matches to a pattern in a series of files and related functions')]))])), 'perl-file-homedir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-homedir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-homedir'), ('version', '1.00')])), ('source', CommentedMap([('fn', 'File-HomeDir-1.00.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/File-HomeDir-1.00.tar.gz'), ('md5', 'aa290cab9db52e2f76280fd1febf794e')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-file-which', 'perl-extutils-makemaker', 'perl-file-path']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-file-which', 'perl-file-path'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-HomeDir'), ('license', 'perl_5'), ('summary', 'Find your home and other directories on any platform')]))])), 'perl-file-listing': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-listing/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-listing'), ('version', '6.04')])), ('source', CommentedMap([('fn', 'File-Listing-6.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/File-Listing-6.04.tar.gz'), ('md5', '83f636b477741f3a014585bb9cc079a6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-http-date']), ('run', ['perl-threaded', 'perl-http-date'])])), ('test', CommentedMap([('imports', ['File::Listing'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-Listing'), ('license', 'perl_5'), ('summary', 'parse directory listing')]))])), 'perl-file-next': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-next/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-next'), ('version', '1.16')])), ('source', CommentedMap([('fn', 'File-Next-1.16.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PETDANCE/File-Next-1.16.tar.gz'), ('md5', '103bc329cd2b64ec270ac53a7dacb378')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['File::Next'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Next'), ('license', 'artistic_2'), ('summary', 'File-finding iterator')]))])), 'perl-file-path': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-path/2.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'File-Path-2.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RI/RICHE/File-Path-2.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-carp', 'perl-devel-symdump', 'perl-app-cpanminus']), ('run', ['perl', 'perl-carp', 'perl-devel-symdump', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use File::Path\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Path'), ('license', 'perl_5'), ('summary', 'Create or remove directory trees')]))])), 'perl-file-readbackwards': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-readbackwards/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-readbackwards'), ('version', '1.05')])), ('source', CommentedMap([('fn', 'File-ReadBackwards-1.05.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/U/UR/URI/File-ReadBackwards-1.05.tar.gz'), ('md5', '613d9d02de6c1d86d5fa5b8816a6b214')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::ReadBackwards'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::ReadBackwards'), ('license', 'unknown'), ('summary', 'Read a file backwards by lines.')]))])), 'perl-file-sharedir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-sharedir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-sharedir'), ('version', '1.102')])), ('source', CommentedMap([('fn', 'File-ShareDir-1.102.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RE/REHSACK/File-ShareDir-1.102.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['File::ShareDir'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/File::ShareDir'), ('license', 'Perl'), ('summary', 'Locate per-dist and per-module shared files')]))])), 'perl-file-sharedir-install': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-sharedir-install/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-sharedir-install'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'File-ShareDir-Install-0.10.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GW/GWYN/File-ShareDir-Install-0.10.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['File::ShareDir::Install'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/File::ShareDir::Install'), ('license', 'Perl'), ('summary', 'Install shared files')]))])), 'perl-file-slurp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-slurp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-slurp'), ('version', '9999.19')])), ('source', CommentedMap([('fn', 'File-Slurp-9999.19.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/U/UR/URI/File-Slurp-9999.19.tar.gz'), ('md5', '7d584cd15c4f8b9547765eff8c4ef078')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Slurp'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-Slurp'), ('license', 'perl_5'), ('summary', 'Simple and Efficient Reading/Writing/Modifying of Complete Files')]))])), 'perl-file-slurp-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-slurp-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-slurp-tiny'), ('version', '0.004')])), ('source', CommentedMap([('fn', 'File-Slurp-Tiny-0.004.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/File-Slurp-Tiny-0.004.tar.gz'), ('md5', '7575b81543281ea57cdb7e5eb3f73264')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Slurp::Tiny'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-Slurp-Tiny'), ('license', 'perl_5'), ('summary', 'A simple, sane and efficient file slurper [DISCOURAGED]')]))])), 'perl-file-slurper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-slurper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-slurper'), ('version', '0.008')])), ('source', CommentedMap([('fn', 'File-Slurper-0.008.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/File-Slurper-0.008.tar.gz'), ('md5', '6e4f8ab76e38dc3f3fec6a0f575bf132')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Slurper'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-Slurper'), ('license', 'perl_5'), ('summary', 'A simple, sane and efficient module to slurp a file')]))])), 'perl-file-sort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-sort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-sort'), ('version', '1.01')])), ('source', CommentedMap([('fn', 'File-Sort-1.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CN/CNANDOR/File-Sort-1.01.tar.gz'), ('md5', '1feb4128bba2edde68ada7838e55863a')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['File::Sort'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File-Sort'), ('license', 'unknown'), ('summary', 'Sort a file or merge sort multiple files')]))])), 'perl-file-which': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-which/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-file-which'), ('version', '1.20')])), ('source', CommentedMap([('fn', 'File-Which-1.20.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/File-Which-1.20.tar.gz'), ('md5', '78479c6e602f9aa562900ee790b39d50')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['File::Which'])])), ('about', CommentedMap([('home', 'http://perl.wdlabs.com/File-Which'), ('license', 'perl_5'), ('summary', 'Perl implementation of the which utility as an API')]))])), 'perl-filesys-df': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-filesys-df/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-filesys-df'), ('version', '0.92')])), ('source', CommentedMap([('fn', 'Filesys-Df-0.92.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/I/IG/IGUTHRIE/Filesys-Df-0.92.tar.gz'), ('md5', 'a8b0aa3e5151a8a6c8b3067625980934')])), ('build', CommentedMap([('skip', False), ('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc']), ('run', ['perl-threaded', 'libgcc'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Filesys-Df'), ('license', 'unknown'), ('summary', 'Perl extension for filesystem disk space information.')]))])), 'perl-font-afm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-font-afm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-font-afm'), ('version', '1.20')])), ('source', CommentedMap([('fn', 'Font-AFM-1.20.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/Font-AFM-1.20.tar.gz'), ('md5', 'b14ecf4dc4edbd63940aa354eb13c499')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Font::AFM', 'Font::Metrics::Courier', 'Font::Metrics::CourierBold', 'Font::Metrics::CourierBoldOblique', 'Font::Metrics::CourierOblique', 'Font::Metrics::Helvetica', 'Font::Metrics::HelveticaBold', 'Font::Metrics::HelveticaBoldOblique', 'Font::Metrics::HelveticaOblique', 'Font::Metrics::TimesBold', 'Font::Metrics::TimesBoldItalic', 'Font::Metrics::TimesItalic', 'Font::Metrics::TimesRoman'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Font-AFM'), ('license', 'perl_5'), ('summary', 'Interface to Adobe Font Metrics files')]))])), 'perl-font-ttf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-font-ttf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-font-ttf'), ('version', '1.05')])), ('source', CommentedMap([('fn', 'Font-TTF-1.05.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MH/MHOSKEN/Font-TTF-1.05.tar.gz'), ('md5', 'a9d0acf4cb9ebaee875d71732b83dfe5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-string', 'perl-xml-parser']), ('run', ['perl-threaded', 'perl-io-string', 'perl-xml-parser'])])), ('test', CommentedMap([('imports', ['Font::TTF', 'Font::TTF::AATKern', 'Font::TTF::AATutils', 'Font::TTF::Anchor', 'Font::TTF::Bsln', 'Font::TTF::Cmap', 'Font::TTF::Coverage', 'Font::TTF::DSIG', 'Font::TTF::Delta', 'Font::TTF::Dumper', 'Font::TTF::EBDT', 'Font::TTF::EBLC', 'Font::TTF::Fdsc', 'Font::TTF::Feat', 'Font::TTF::Features::Cvar', 'Font::TTF::Features::Size', 'Font::TTF::Features::Sset', 'Font::TTF::Fmtx', 'Font::TTF::GDEF', 'Font::TTF::GPOS', 'Font::TTF::GSUB', 'Font::TTF::Glat', 'Font::TTF::Gloc', 'Font::TTF::Glyf', 'Font::TTF::Glyph', 'Font::TTF::GrFeat', 'Font::TTF::Hdmx', 'Font::TTF::Head', 'Font::TTF::Hhea', 'Font::TTF::Hmtx', 'Font::TTF::Kern', 'Font::TTF::Kern::ClassArray', 'Font::TTF::Kern::CompactClassArray', 'Font::TTF::Kern::OrderedList', 'Font::TTF::Kern::StateTable', 'Font::TTF::Kern::Subtable', 'Font::TTF::LTSH', 'Font::TTF::Loca', 'Font::TTF::Maxp', 'Font::TTF::Mort', 'Font::TTF::Mort::Chain', 'Font::TTF::Mort::Contextual', 'Font::TTF::Mort::Insertion', 'Font::TTF::Mort::Ligature', 'Font::TTF::Mort::Noncontextual', 'Font::TTF::Mort::Rearrangement', 'Font::TTF::Mort::Subtable', 'Font::TTF::OS_2', 'Font::TTF::OTTags', 'Font::TTF::OldCmap', 'Font::TTF::OldMort', 'Font::TTF::PCLT', 'Font::TTF::PSNames', 'Font::TTF::Prop', 'Font::TTF::Silf', 'Font::TTF::Sill', 'Font::TTF::Ttopen', 'Font::TTF::Vhea', 'Font::TTF::Vmtx', 'Font::TTF::Win32', 'Font::TTF::Woff', 'Font::TTF::Woff::MetaData', 'Font::TTF::Woff::PrivateData', 'Font::TTF::XMLparse'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Font-TTF'), ('license', 'artistic_2'), ('summary', 'TTF font support for Perl')]))])), 'perl-forks': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-forks/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-forks'), ('version', '0.36')])), ('source', CommentedMap([('fn', 'forks-0.36.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/R/RY/RYBSKEJ/forks-0.36.tar.gz'), ('md5', '48a1819dd2ff67d756f268771e1a9827')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['forks'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~rybskej/forks/lib/forks.pm'), ('license', 'Perl'), ('summary', 'drop-in replacement for Perl threads using fork()')]))])), 'perl-gd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-gd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-gd'), ('version', '2.56')])), ('source', CommentedMap([('fn', 'GD-2.56.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LD/LDS/GD-2.56.tar.gz'), ('md5', 'c4b3afd98b2c4ce3c2e1027d101a8f1e')])), ('build', CommentedMap([('number', 6), ('skip', True)])), ('requirements', CommentedMap([('build', ['llvm', 'libgd >=2.2.4', 'perl', 'perl-module-build']), ('run', ['libgcc', 'libgd >=2.2.4', 'perl'])])), ('test', CommentedMap([('imports', ['GD', 'GD::Simple', 'GD::Polygon'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/GD'), ('license', 'perl_5'), ('summary', 'Perl interface to the gd2 graphics library')]))])), 'perl-gdgraph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-gdgraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-gdgraph'), ('version', '1.49')])), ('source', CommentedMap([('fn', 'GDGraph-1.49.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RU/RUZ/GDGraph-1.49.tar.gz'), ('md5', '8bb270a30fea27491302b39b701b3d4e')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'perl-gdtextutil', 'perl-gd']), ('run', ['libgcc', 'perl', 'perl-gd', 'perl-gdtextutil'])])), ('test', CommentedMap([('imports', ['GD::Graph'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/GDGraph'), ('license', 'perl_5'), ('summary', 'Produces charts with GD')]))])), 'perl-gdgraph-histogram': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-gdgraph-histogram/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-gdgraph-histogram'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'GDGraph-histogram-1.1.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/W/WH/WHIZDOG/GDGraph-histogram-1.1.tar.gz'), ('md5', '633c97212412d0d85bb3dc7251f5bad9')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'perl-gdgraph', 'libgd 2.2.*']), ('run', ['perl', 'perl-gdgraph', 'libgd 2.2.*'])])), ('test', CommentedMap([('imports', ['GD::Graph::histogram'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/GDGraph-histogram'), ('license', 'unknown'), ('summary', 'Histogram plotting module for Perl5')]))])), 'perl-gdtextutil': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-gdtextutil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-gdtextutil'), ('version', '0.86')])), ('source', CommentedMap([('fn', 'GDTextUtil-0.86.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MV/MVERB/GDTextUtil-0.86.tar.gz'), ('md5', '941ad06eadc86b47f3a32da405665c41')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'perl-gd']), ('run', ['libgcc', 'perl', 'perl-gd'])])), ('test', CommentedMap([('imports', ['GD::Text'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/GDTextUtil'), ('license', 'unknown'), ('summary', 'Text utilities for use with GD')]))])), 'perl-getopt-long': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-getopt-long/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-getopt-long'), ('version', '2.49')])), ('source', CommentedMap([('fn', 'Getopt-Long-2.49.1.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JV/JV/Getopt-Long-2.49.1.tar.gz'), ('md5', 'bb7c92be20117be5d5043b07b6fd20a1')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Getopt::Long'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Getopt::Long'), ('license', 'unknown'), ('summary', 'Module to handle parsing command line options')]))])), 'perl-git-wrapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-git-wrapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-git-wrapper'), ('version', '0.047')])), ('source', CommentedMap([('fn', 'Git-Wrapper-0.047.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GE/GENEHACK/Git-Wrapper-0.047.tar.gz'), ('md5', '58ab70ecfd306a0fd678d405b15715e9')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-devel-checkbin', 'perl-test-exception', 'perl-extutils-makemaker', 'perl-file-chdir', 'perl-pathtools', 'perl-file-path', 'perl-apache-test', 'perl-test-simple', 'perl-sort-versions', 'perl-test-deep', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-devel-checkbin', 'perl-file-chdir', 'perl-apache-test', 'perl-sort-versions', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Git::Wrapper', 'Git::Wrapper::Exception', 'Git::Wrapper::File::RawModification', 'Git::Wrapper::Log', 'Git::Wrapper::Status', 'Git::Wrapper::Statuses'])])), ('about', CommentedMap([('home', 'http://genehack.github.com/Git-Wrapper/'), ('license', 'perl_5'), ('summary', 'Wrap git(7) command-line interface')]))])), 'perl-git-wrapper-plus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-git-wrapper-plus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-git-wrapper-plus'), ('version', '0.004010')])), ('source', CommentedMap([('fn', 'Git-Wrapper-Plus-0.004010.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/K/KE/KENTNL/Git-Wrapper-Plus-0.004010.tar.gz'), ('md5', 'f8065aedb316a2125b42856feed5f2a5')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-scalar-list-utils', 'perl-moo', 'perl-git-wrapper', 'perl-path-tiny', 'perl-sub-exporter-progressive', 'perl-extutils-makemaker', 'perl-test-simple', 'perl-try-tiny', 'perl-sort-versions', 'perl-pathtools', 'perl-test-fatal', 'perl-apache-test', 'perl-test-requires', 'perl-perl-version']), ('run', ['perl', 'perl-try-tiny', 'perl-scalar-list-utils', 'perl-moo', 'perl-git-wrapper', 'perl-apache-test', 'perl-sort-versions', 'perl-path-tiny', 'perl-sub-exporter-progressive'])])), ('test', CommentedMap([('imports', ['Git::Wrapper::Plus', 'Git::Wrapper::Plus::Branches', 'Git::Wrapper::Plus::Ref', 'Git::Wrapper::Plus::Ref::Branch', 'Git::Wrapper::Plus::Ref::Tag', 'Git::Wrapper::Plus::Refs', 'Git::Wrapper::Plus::Support', 'Git::Wrapper::Plus::Support::Arguments', 'Git::Wrapper::Plus::Support::Behaviors', 'Git::Wrapper::Plus::Support::Commands', 'Git::Wrapper::Plus::Support::Range', 'Git::Wrapper::Plus::Support::RangeDictionary', 'Git::Wrapper::Plus::Support::RangeSet', 'Git::Wrapper::Plus::Tags', 'Git::Wrapper::Plus::Tester', 'Git::Wrapper::Plus::Util', 'Git::Wrapper::Plus::Versions'])])), ('about', CommentedMap([('home', 'https://github.com/kentfredric/Git-Wrapper-Plus'), ('license', 'perl_5'), ('summary', 'A Toolkit for working with Git::Wrapper in an Object Oriented Way.')]))])), 'perl-graph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-graph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-graph'), ('version', '0.9704')])), ('source', CommentedMap([('fn', 'Graph-0.9704.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JH/JHI/Graph-0.9704.tar.gz'), ('md5', '1ab4e49420e56eeb1bc81d842aa8f3af')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Graph', 'Graph::AdjacencyMap', 'Graph::AdjacencyMatrix', 'Graph::BitMatrix', 'Graph::Directed', 'Graph::Matrix', 'Graph::TransitiveClosure', 'Graph::TransitiveClosure::Matrix', 'Graph::Traversal', 'Graph::Traversal::BFS', 'Graph::Traversal::DFS', 'Graph::Undirected', 'Graph::UnionFind'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Graph'), ('license', 'perl_5'), ('summary', 'a Perl extension for keeping data partially sorted')]))])), 'perl-graph-readwrite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-graph-readwrite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-graph-readwrite'), ('version', '2.09')])), ('source', CommentedMap([('fn', 'Graph-ReadWrite-2.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Graph-ReadWrite-2.09.tar.gz'), ('md5', '5cd9191eadd2fe8fe8bb431575434f67')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp']), ('run', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp'])])), ('test', CommentedMap([('imports', ['Graph::Writer::Dot'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Graph::Writer::Dot'), ('license', 'perl_5'), ('summary', 'Write out directed graph in Dot format')]))])), 'perl-graphviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-graphviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-graphviz'), ('version', '2.20')])), ('source', CommentedMap([('fn', 'GraphViz-2.20.tgz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSAVAGE/GraphViz-2.20.tgz'), ('md5', '714dedddd36d86950cd436d87d1d52b6')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-file-which', 'perl-ipc-run', 'perl-libwww-perl', 'perl-parse-recdescent', 'perl-test-pod', 'perl-xml-twig', 'perl-xml-xpath', 'graphviz']), ('run', ['perl-threaded', 'perl-file-which', 'perl-ipc-run', 'perl-libwww-perl', 'perl-parse-recdescent', 'perl-test-pod', 'perl-xml-twig', 'perl-xml-xpath', 'graphviz'])])), ('test', CommentedMap([('imports', ['GraphViz', 'GraphViz::Data::Grapher', 'GraphViz::No', 'GraphViz::Parse::RecDescent', 'GraphViz::Parse::Yacc', 'GraphViz::Parse::Yapp', 'GraphViz::Regex', 'GraphViz::Small', 'GraphViz::XML'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/GraphViz'), ('license', 'artistic_2'), ('summary', \"Interface to AT&T's GraphViz. Deprecated. See GraphViz2\")]))])), 'perl-hash-merge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hash-merge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hash-merge'), ('version', '0.200')])), ('source', CommentedMap([('fn', 'Hash-Merge-0.200.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RE/REHSACK/Hash-Merge-0.200.tar.gz'), ('md5', '54b5743103fb3b3986a384b5424f21f4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-clone', 'perl-extutils-makemaker', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-clone'])])), ('test', CommentedMap([('imports', ['Hash::Merge'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Hash::Merge'), ('license', 'perl_5'), ('summary', 'Merges arbitrarily deep hashes into a single hash')]))])), 'perl-heap-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-heap-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-heap-simple'), ('version', '0.13')])), ('source', CommentedMap([('fn', 'Heap-Simple-0.13.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TH/THOSPEL/Heap-Simple-0.13.tar.gz'), ('md5', '7a70c8e7e52b35c33bbbb9ad8cdbdeb4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-cgi', 'perl-heap-simple-xs', 'perl-heap-simple-perl']), ('run', ['perl-threaded', 'perl-cgi', 'perl-heap-simple-xs', 'perl-heap-simple-perl'])])), ('test', CommentedMap([('imports', ['Heap::Simple'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Heap::Simple'), ('license', 'unknown'), ('summary', 'Fast and easy to use classic heaps')]))])), 'perl-heap-simple-perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-heap-simple-perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-heap-simple-perl'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'Heap-Simple-Perl-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TH/THOSPEL/Heap-Simple-Perl-0.14.tar.gz'), ('md5', 'a2723ab28b7fcb35131037525a5b2f93')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Heap::Simple::Perl'), ('license', 'unknown'), ('summary', 'A pure perl implementation of the Heap::Simple interface')]))])), 'perl-heap-simple-xs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-heap-simple-xs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-heap-simple-xs'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'Heap-Simple-XS-0.10.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TH/THOSPEL/Heap-Simple-XS-0.10.tar.gz'), ('md5', '0611ac7984f6ddc3aaa60c75d2257803')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Heap::Simple::XS'), ('license', 'unknown'), ('summary', 'An XS implementation of the Heap::Simple interface')]))])), 'perl-hpc-runner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner'), ('version', '2.48')])), ('source', CommentedMap([('fn', 'HPC-Runner-2.48.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-2.48.tar.gz'), ('md5', 'd1a1c0c45638d19444909923772e52b6')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-module-build-tiny', 'perl-template-toolkit', 'perl-pathtools', 'perl-log-log4perl', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-path-tiny', 'perl-termreadkey', 'perl-config-any', 'perl-file-slurp', 'perl-capture-tiny', 'perl-extutils-helpers', 'perl-extutils-config', 'perl-extutils-installpaths']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-log-log4perl', 'perl-template-toolkit', 'perl-path-tiny', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-termreadkey', 'perl-config-any', 'perl-file-slurp'])])), ('test', CommentedMap([('imports', ['HPC::Runner'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HPC-Runner'), ('license', 'perl_5'), ('summary', 'HPC Runner::Slurm, Runner::MCE and Runner::Threads base class')]))])), 'perl-hpc-runner-mce': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-mce/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-mce'), ('version', '2.41')])), ('source', CommentedMap([('fn', 'HPC-Runner-MCE-2.41.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-MCE-2.41.tar.gz'), ('md5', '15b20cdd0ca4a8d6785af7f7bee2046f')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-termreadkey', 'perl-app-cpanminus']), ('run', ['perl-threaded', 'perl-termreadkey'])])), ('test', CommentedMap([('imports', ['HPC::Runner::MCE'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jerowe/hpc-runner-mce'), ('license', 'perl_5'), ('summary', 'Job submission using MCE')]))])), 'perl-hpc-runner-pbs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-pbs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-pbs'), ('version', '0.12')])), ('source', CommentedMap([('fn', 'HPC-Runner-PBS-0.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-PBS-0.12.tar.gz'), ('md5', '3504d03c2d155ac73cf8e1dd1e616860')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-termreadkey']), ('run', ['perl-threaded', 'perl-termreadkey'])])), ('test', CommentedMap([('imports', ['HPC::Runner::PBS'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jerowe/hpc-runner-pbs'), ('license', 'perl_5'), ('summary', 'Submit jobs to a PBS job scheduler.')]))])), 'perl-hpc-runner-scheduler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-scheduler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-scheduler'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'HPC-Runner-Scheduler-0.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Scheduler-0.09.tar.gz'), ('md5', 'a95b4c35f448afc63a27261638b9ae42')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-json', 'perl-template-toolkit', 'perl-module-build-tiny', 'perl-test-simple', 'perl-pathtools', 'perl-moose', 'perl-moosex-getopt', 'perl-log-log4perl', 'perl-hpc-runner', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-json', 'perl-template-toolkit', 'perl-module-build-tiny', 'perl-pathtools', 'perl-moose', 'perl-moosex-getopt', 'perl-log-log4perl', 'perl-hpc-runner', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['HPC::Runner::Scheduler'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Scheduler'), ('license', 'perl_5'), ('summary', 'Base Library for HPC::Runner::Slurm and HPC::Runner::PBS')]))])), 'perl-html-element-extended': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-element-extended/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-element-extended'), ('version', '1.18')])), ('source', CommentedMap([('fn', 'HTML-Element-Extended-1.18.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MS/MSISK/HTML-Element-Extended-1.18.tar.gz'), ('md5', '41ec9dcc7cefa03b204d0e8ca8e1c112')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-html-tree']), ('run', ['perl-threaded', 'perl-html-tree'])])), ('test', CommentedMap([('imports', ['HTML::ElementGlob', 'HTML::ElementRaw', 'HTML::ElementSuper', 'HTML::ElementTable'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Element-Extended'), ('license', 'unknown'), ('summary', 'Perl extension for manipulating a table composed of HTML::Element style components.')]))])), 'perl-html-entities-numbered': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-entities-numbered/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-entities-numbered'), ('version', '0.04')])), ('source', CommentedMap([('fn', 'HTML-Entities-Numbered-0.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TA/TANIGUCHI/HTML-Entities-Numbered-0.04.tar.gz'), ('md5', '9bc61132c10aa506d2629b37a2012729')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Entities-Numbered'), ('license', 'unknown'), ('summary', 'Conversion of numbered HTML entities')]))])), 'perl-html-formatter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-formatter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-formatter'), ('version', '2.14')])), ('source', CommentedMap([('fn', 'HTML-Formatter-2.14.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NI/NIGELM/HTML-Formatter-2.14.tar.gz'), ('md5', 'af35f37c2114a355923d924aa8536253')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-file-slurper', 'perl-font-afm', 'perl-html-tree', 'perl-test-cpan-meta', 'perl-test-eol', 'perl-test-notabs']), ('run', ['perl-threaded', 'perl-font-afm', 'perl-html-tree'])])), ('test', CommentedMap([('imports', ['HTML::FormatMarkdown', 'HTML::FormatPS', 'HTML::FormatRTF', 'HTML::FormatText', 'HTML::Formatter'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/HTML-Formatter'), ('license', 'perl_5'), ('summary', 'Base class for HTML formatters')]))])), 'perl-html-parser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-parser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-parser'), ('version', '3.72')])), ('source', CommentedMap([('fn', 'HTML-Parser-3.72.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GA/GAAS/HTML-Parser-3.72.tar.gz'), ('md5', 'eb7505e5f626913350df9dd4a03d54a8')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'perl-html-tagset']), ('run', ['perl', 'perl-html-tagset'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Parser'), ('license', 'perl_5'), ('summary', 'HTML parser class')]))])), 'perl-html-tableextract': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-tableextract/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-tableextract'), ('version', '2.13')])), ('source', CommentedMap([('fn', 'HTML-TableExtract-2.13.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MS/MSISK/HTML-TableExtract-2.13.tar.gz'), ('md5', '0d1d3f46a6a0336c5d20e61128e59cb6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-html-parser', 'perl-html-element-extended']), ('run', ['perl-threaded', 'perl-html-parser', 'perl-html-element-extended'])])), ('test', CommentedMap([('imports', ['HTML::TableExtract'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-TableExtract'), ('license', 'unknown'), ('summary', 'Perl module for extracting the content contained in tables within an HTML document, either as text or encoded element trees.')]))])), 'perl-html-tagset': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-tagset/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-tagset'), ('version', '3.20')])), ('source', CommentedMap([('fn', 'HTML-Tagset-3.20.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PE/PETDANCE/HTML-Tagset-3.20.tar.gz'), ('md5', 'd2bfa18fe1904df7f683e96611e87437')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['HTML::Tagset'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Tagset'), ('license', 'unknown'), ('summary', 'data tables useful in parsing HTML')]))])), 'perl-html-template': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-template/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-template'), ('version', '2.95')])), ('source', CommentedMap([('fn', 'HTML-Template-2.95.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/W/WO/WONKO/HTML-Template-2.95.tar.gz'), ('md5', '6f08e9631af52e6a5f6e4648b89265fe')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-scalar-list-utils', 'perl-cgi']), ('run', ['perl-threaded', 'perl-scalar-list-utils', 'perl-cgi'])])), ('test', CommentedMap([('imports', ['HTML::Template'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Template'), ('license', 'perl_5'), ('summary', 'Perl module to use HTML-like templating language')]))])), 'perl-html-tidy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-tidy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-tidy'), ('version', '1.56')])), ('source', CommentedMap([('fn', 'HTML-Tidy-1.56.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PE/PETDANCE/HTML-Tidy-1.56.tar.gz'), ('md5', 'dd4c0909e93033512606abf2f4e65dae'), ('patches', ['makefile-pl.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'tidyp']), ('run', ['perl-threaded', 'tidyp'])])), ('test', CommentedMap([('imports', ['HTML::Tidy', 'HTML::Tidy::Message'])])), ('about', CommentedMap([('home', 'http://github.com/petdance/html-tidy'), ('license', 'artistic_2'), ('summary', '(X)HTML validation in a Perl object')]))])), 'perl-html-tree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-tree/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-tree'), ('version', '5.03')])), ('source', CommentedMap([('fn', 'HTML-Tree-5.03.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CJ/CJM/HTML-Tree-5.03.tar.gz'), ('md5', 'd9271d60b872ed6fbe68b2d0fe8c450e')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-html-parser', 'perl-module-build', 'perl-html-tagset', 'perl-test-fatal', 'perl-libwww-perl']), ('run', ['perl-threaded', 'perl-html-parser', 'perl-html-tagset', 'perl-libwww-perl'])])), ('test', CommentedMap([('imports', ['HTML::AsSubs', 'HTML::Element', 'HTML::Parse', 'HTML::Tree', 'HTML::TreeBuilder'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-Tree'), ('license', 'perl_5'), ('summary', 'Work with HTML in a DOM-like tree structure')]))])), 'perl-html-treebuilder-xpath': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-html-treebuilder-xpath/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-html-treebuilder-xpath'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'HTML-TreeBuilder-XPath-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MI/MIROD/HTML-TreeBuilder-XPath-0.14.tar.gz'), ('md5', 'a3fa3b73ff51dd6ec63be394dcd2a3b5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-html-tree', 'perl-xml-xpathengine']), ('run', ['perl-threaded', 'perl-html-tree', 'perl-xml-xpathengine'])])), ('test', CommentedMap([('imports', ['HTML::TreeBuilder::XPath'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTML-TreeBuilder-XPath'), ('license', 'perl_5'), ('summary', 'add XPath support to HTML::TreeBuilder')]))])), 'perl-http-cookies': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-http-cookies/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-http-cookies'), ('version', '6.01')])), ('source', CommentedMap([('fn', 'HTTP-Cookies-6.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Cookies-6.01.tar.gz'), ('md5', 'ecfd0eeb88512033352c2f13c9580f03')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-http-date', 'perl-http-message']), ('run', ['perl-threaded', 'perl-http-date', 'perl-http-message'])])), ('test', CommentedMap([('imports', ['HTTP::Cookies'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTTP-Cookies'), ('license', 'perl_5'), ('summary', 'HTTP cookie jars')]))])), 'perl-http-daemon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-http-daemon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-http-daemon'), ('version', '6.01')])), ('source', CommentedMap([('fn', 'HTTP-Daemon-6.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Daemon-6.01.tar.gz'), ('md5', 'ed0ae02d25d7f1e89456d4d69732adc2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-http-date', 'perl-http-message', 'perl-lwp-mediatypes']), ('run', ['perl-threaded', 'perl-http-date', 'perl-http-message', 'perl-lwp-mediatypes'])])), ('test', CommentedMap([('imports', ['HTTP::Daemon'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTTP-Daemon'), ('license', 'perl_5'), ('summary', 'a simple http server class')]))])), 'perl-http-date': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-http-date/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-http-date'), ('version', '6.02')])), ('source', CommentedMap([('fn', 'HTTP-Date-6.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Date-6.02.tar.gz'), ('md5', '52b7a0d5982d61be1edb217751d7daba')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['HTTP::Date'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTTP-Date'), ('license', 'perl_5'), ('summary', 'date conversion routines')]))])), 'perl-http-message': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-http-message/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-http-message'), ('version', '6.11')])), ('source', CommentedMap([('fn', 'HTTP-Message-6.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/HTTP-Message-6.11.tar.gz'), ('md5', '4ed7add10daea3ab30abfeab6d03872f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-encode-locale', 'perl-http-date', 'perl-io-html', 'perl-lwp-mediatypes', 'perl-uri']), ('run', ['perl-threaded', 'perl-encode-locale', 'perl-http-date', 'perl-io-html', 'perl-lwp-mediatypes', 'perl-uri'])])), ('test', CommentedMap([('imports', ['HTTP::Config', 'HTTP::Headers', 'HTTP::Headers::Auth', 'HTTP::Headers::ETag', 'HTTP::Headers::Util', 'HTTP::Message', 'HTTP::Request', 'HTTP::Request::Common', 'HTTP::Response', 'HTTP::Status'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTTP-Message'), ('license', 'perl_5'), ('summary', 'HTTP style messages')]))])), 'perl-http-negotiate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-http-negotiate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-http-negotiate'), ('version', '6.01')])), ('source', CommentedMap([('fn', 'HTTP-Negotiate-6.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Negotiate-6.01.tar.gz'), ('md5', '1236195250e264d7436e7bb02031671b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-http-message']), ('run', ['perl-threaded', 'perl-http-message'])])), ('test', CommentedMap([('imports', ['HTTP::Negotiate'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/HTTP-Negotiate'), ('license', 'perl_5'), ('summary', 'choose a variant to serve')]))])), 'perl-image-exiftool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-image-exiftool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-image-exiftool'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Image-ExifTool-<{ version }}.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/E/EX/EXIFTOOL/Image-ExifTool-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Image::ExifTool']), ('commands', ['exiftool -ver'])])), ('about', CommentedMap([('home', 'http://www.sno.phy.queensu.ca/~phil/exiftool'), ('license', 'perl_5'), ('summary', 'ExifTool is a platform-independent Perl library plus a command-line application for reading, writing and editing meta information in a wide variety of files.')]))])), 'perl-image-info': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-image-info/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-image-info'), ('version', '1.38')])), ('source', CommentedMap([('fn', 'Image-Info-1.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SR/SREZIC/Image-Info-1.38.tar.gz'), ('md5', '0fa4db4bdf14b6f453655bafa62d766a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml']), ('run', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Image-Info'), ('license', 'perl_5'), ('summary', 'Extract meta information from image files')]))])), 'perl-image-size': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-image-size/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-image-size'), ('version', '3.300')])), ('source', CommentedMap([('fn', 'Image-Size-3.300.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJRAY/Image-Size-3.300.tar.gz'), ('md5', 'c8f7f5097cadb984e57e9c0cad1f16b3')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Image::Size'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Image-Size'), ('license', 'perl_5'), ('summary', 'A library to extract height/width from images')]))])), 'perl-importer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-importer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-importer'), ('version', '0.024')])), ('source', CommentedMap([('fn', 'Importer-0.024.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Importer-0.024.tar.gz'), ('md5', '2d29c79e65e221105a23e28c239b688f')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Importer'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Importer'), ('license', 'perl_5'), ('summary', 'Alternative but compatible interface to modules that export symbols.')]))])), 'perl-inline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-inline/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-inline'), ('version', '0.80')])), ('source', CommentedMap([('fn', 'Inline-0.80.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/I/IN/INGY/Inline-0.80.tar.gz'), ('md5', '510bbac46e727bcaf240b7feac2646c9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-warn']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Inline/'), ('license', 'perl_5'), ('summary', 'Write Perl Subroutines in Other Programming Languages')]))])), 'perl-inline-c': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-inline-c/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-inline-c'), ('version', '0.76')])), ('source', CommentedMap([('fn', 'Inline-C-0.76.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/I/IN/INGY/Inline-C-0.76.tar.gz'), ('md5', 'c0fbfdd058075c9271a1384c822c9a87')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-all', 'perl-inline', 'perl-parse-recdescent', 'perl-pegex', 'perl-file-copy-recursive', 'perl-yaml-libyaml']), ('run', ['perl-threaded', 'perl-io-all', 'perl-inline', 'perl-parse-recdescent', 'perl-pegex', 'perl-file-copy-recursive', 'perl-yaml-libyaml'])])), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/inline-c-pm'), ('license', 'perl_5'), ('summary', 'A library to extract height/width from images')]))])), 'perl-io-all': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-all/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-all'), ('version', '0.86')])), ('source', CommentedMap([('fn', 'IO-All-0.86.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/I/IN/INGY/IO-All-0.86.tar.gz'), ('md5', 'b257d3f742867825d018e74f5a5d549b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-scalar-list-utils', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-scalar-list-utils', 'perl-extutils-makemaker'])])), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/io-all-pm'), ('license', 'perl_5'), ('summary', 'Combines all of the best Perl IO modules into a single nifty object oriented interface')]))])), 'perl-io-compress': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-compress/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-compress'), ('version', 2.069)])), ('source', CommentedMap([('fn', 'IO-Compress-2.069.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PM/PMQS/IO-Compress-2.069.tar.gz'), ('md5', 'b26925161e3f01919f60344d1bbb49c4')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-compress-raw-zlib', 'perl-compress-raw-bzip2 >=2.069', 'perl-scalar-list-utils', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-compress-raw-zlib', 'perl-compress-raw-bzip2 >=2.069', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['IO::Compress::Adapter::Bzip2', 'IO::Compress::Adapter::Deflate', 'IO::Compress::Adapter::Identity', 'IO::Compress::Base', 'IO::Compress::Base::Common', 'IO::Compress::Bzip2', 'IO::Compress::Deflate', 'IO::Compress::Gzip', 'IO::Compress::Gzip::Constants', 'IO::Compress::RawDeflate', 'IO::Compress::Zip', 'IO::Compress::Zip::Constants', 'IO::Compress::Zlib::Constants', 'IO::Compress::Zlib::Extra', 'IO::Uncompress::Adapter::Bunzip2', 'IO::Uncompress::Adapter::Identity', 'IO::Uncompress::Adapter::Inflate', 'IO::Uncompress::AnyInflate', 'IO::Uncompress::AnyUncompress', 'IO::Uncompress::Base', 'IO::Uncompress::Bunzip2', 'IO::Uncompress::Gunzip', 'IO::Uncompress::Inflate', 'IO::Uncompress::RawInflate', 'IO::Uncompress::Unzip'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-Compress'), ('license', 'perl_5'), ('summary', 'IO Interface to compressed data files/buffers')]))])), 'perl-io-gzip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-gzip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-gzip'), ('version', '0.20')])), ('source', CommentedMap([('fn', 'PerlIO-gzip-0.20.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NW/NWCLARK/PerlIO-gzip-0.20.tar.gz'), ('md5', '0393eae5d0b23df6cf40ed44af7d711c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'zlib <{CONDA_ZLIB}}*']), ('run', ['perl', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['PerlIO::gzip'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/PerlIO::gzip'), ('license', 'perl_5'), ('summary', 'Perl extension to provide a PerlIO layer to gzip/gunzip')]))])), 'perl-io-html': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-html/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-html'), ('version', '1.001')])), ('source', CommentedMap([('fn', 'IO-HTML-1.001.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CJ/CJM/IO-HTML-1.001.tar.gz'), ('md5', '3f8958718844dc96b9f6946f21d70d22')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['IO::HTML'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-HTML'), ('license', 'perl_5'), ('summary', 'Open an HTML file with automatic charset detection')]))])), 'perl-io-interactive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-interactive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-interactive'), ('version', '1.021')])), ('source', CommentedMap([('fn', 'IO-Interactive-1.021.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BD/BDFOY/IO-Interactive-1.021.tar.gz'), ('md5', '1636503b9e7b34d88450233e2504b73f')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['IO::Interactive'])])), ('about', CommentedMap([('home', 'https://github.com/briandfoy/io-interactive'), ('license', 'perl_5'), ('summary', 'Utilities for interactive I/O')]))])), 'perl-io-prompt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-prompt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/D/DC/DCONWAY/IO-Prompt-0.997004.tar.gz'), ('md5', '24d7c674fd8ec43edf8a05eeb068fc3a'), ('patches', ['patches/clear.pl.patch', 'patches/cmdline.pl.patch', 'patches/default.pl.patch', 'patches/hand.pl.patch', 'patches/interactive.pl.patch', 'patches/lingua.pl.patch', 'patches/nonl.pl.patch', 'patches/passwd.pl.patch', 'patches/to.pl.patch', 'patches/tty.pl.patch', 'patches/while_until.pl.patch', 'patches/yesno.pl.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-extutils-makemaker', 'perl-termreadkey', 'perl-want']), ('run', ['perl', 'perl-termreadkey', 'perl-want'])])), ('test', CommentedMap([('imports', ['IO::Prompt'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/IO-Prompt/lib/IO/Prompt.pm'), ('license', 'perl_5'), ('summary', 'Interactively prompt for user input')]))])), 'perl-io-sessiondata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-sessiondata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-sessiondata'), ('version', '1.03')])), ('source', CommentedMap([('fn', 'IO-SessionData-1.03.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PH/PHRED/IO-SessionData-1.03.tar.gz'), ('md5', '790f9e05465c774cf9a6299500463104')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['IO::SessionData', 'IO::SessionSet'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-SessionData'), ('license', 'unknown'), ('summary', 'supporting module for SOAP::Lite')]))])), 'perl-io-socket-inet6': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-socket-inet6/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-socket-inet6'), ('version', '2.72')])), ('source', CommentedMap([('fn', 'IO-Socket-INET6-2.72.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/IO-Socket-INET6-2.72.tar.gz'), ('md5', '510ddc1bd75a8340ca7226123fb545c1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-socket6']), ('run', ['perl-threaded', 'perl-socket6'])])), ('test', CommentedMap([('imports', ['IO::Socket::INET6'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-Socket-INET6'), ('license', 'perl_5'), ('summary', 'Object interface for AF_INET/AF_INET6 domain sockets')]))])), 'perl-io-socket-ssl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-socket-ssl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-socket-ssl'), ('version', '2.024')])), ('source', CommentedMap([('fn', 'IO-Socket-SSL-2.024.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.024.tar.gz'), ('md5', '9d1be9159475fe67afa5f2e238808086')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-net-ssleay']), ('run', ['perl-threaded', 'perl-net-ssleay'])])), ('test', CommentedMap([('imports', ['IO::Socket::SSL'])])), ('about', CommentedMap([('home', 'https://github.com/noxxi/p5-io-socket-ssl'), ('license', 'perl_5'), ('summary', 'Nearly transparent SSL encapsulation for IO::Socket::INET.')]))])), 'perl-io-string': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-string/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-string'), ('version', '1.08')])), ('source', CommentedMap([('fn', 'IO-String-1.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz'), ('md5', '250e5424f290299fc3d6b5d1e9da3835')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-String'), ('license', 'unknown'), ('summary', 'Emulate file interface for in-core strings')]))])), 'perl-io-stringy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-stringy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-stringy'), ('version', '2.111')])), ('source', CommentedMap([('fn', 'IO-stringy-2.111.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DS/DSKOLL/IO-stringy-2.111.tar.gz'), ('md5', 'e91acf0a800b190d13585a47de775bdd')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['IO::AtomicFile', 'IO::InnerFile', 'IO::Lines', 'IO::Scalar', 'IO::ScalarArray', 'IO::Stringy', 'IO::Wrap', 'IO::WrapTie'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-stringy'), ('license', 'unknown'), ('summary', 'write a file which is updated atomically')]))])), 'perl-io-tty': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-tty/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-io-tty'), ('version', '1.12')])), ('source', CommentedMap([('fn', 'IO-Tty-1.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TODDR/IO-Tty-1.12.tar.gz'), ('md5', '11695a1a516b3bd1b90ce75ff0ce3e6d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['IO::Pty', 'IO::Tty'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO-Tty'), ('license', 'perl_5'), ('summary', 'Pseudo ttys and constants')]))])), 'perl-ipc-run': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ipc-run/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ipc-run'), ('version', '0.94')])), ('source', CommentedMap([('fn', 'IPC-Run-0.94.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TODDR/IPC-Run-0.94.tar.gz'), ('md5', 'c963902894d3796b16624b3b4aea94b4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-tty']), ('run', ['perl-threaded', 'perl-io-tty'])])), ('test', CommentedMap([('imports', ['IPC::Run'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IPC-Run'), ('license', 'perl_5'), ('summary', 'system() and background procs w/ piping, redirs, ptys (Unix, Win32)')]))])), 'perl-ipc-system-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ipc-system-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ipc-system-simple'), ('version', '1.25')])), ('source', CommentedMap([('fn', 'IPC-System-Simple-1.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PJ/PJF/IPC-System-Simple-1.25.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['IPC::System::Simple'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/IPC::System::Simple'), ('license', 'Perl'), ('summary', 'Run commands simply, with detailed diagnostics')]))])), 'perl-jcode': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-jcode/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-jcode'), ('version', '2.07')])), ('source', CommentedMap([('fn', 'Jcode-2.07.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Jcode-2.07.tar.gz'), ('md5', 'f6c52253ff69a44c38a9183c469f6eb0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Jcode'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Jcode'), ('license', 'unknown'), ('summary', 'Japanese Charset Handler')]))])), 'perl-json': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-json/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-json'), ('version', '2.90')])), ('source', CommentedMap([('fn', 'JSON-2.90.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA/JSON-2.90.tar.gz'), ('md5', 'e1512169a623e790a3f69b599cc1d3b9')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-json-xs <3']), ('run', ['perl-threaded', 'perl-json-xs <3'])])), ('test', CommentedMap([('imports', ['JSON'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/JSON'), ('license', 'perl_5'), ('summary', 'JSON (JavaScript Object Notation) encoder/decoder')]))])), 'perl-json-maybexs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-json-maybexs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-json-maybexs'), ('version', '1.003008')])), ('source', CommentedMap([('fn', 'JSON-MaybeXS-1.003008.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/JSON-MaybeXS-1.003008.tar.gz'), ('md5', '5fc1cd29d699dbad7f41e05ccd14118d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-pathtools', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-app-cpanminus']), ('run', ['perl', 'perl-scalar-list-utils', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['JSON::MaybeXS'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/JSON::MaybeXS'), ('license', 'perl_5'), ('summary', 'Use Cpanel::JSON::XS with a fallback to JSON::XS and JSON::PP')]))])), 'perl-json-xs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-json-xs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-json-xs'), ('version', '2.34')])), ('source', CommentedMap([('fn', 'JSON-XS-2.34.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN/JSON-XS-2.34.tar.gz'), ('md5', 'dda3c03335665cdeff91c37a269b833b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-common-sense']), ('run', ['perl-threaded', 'perl-common-sense'])])), ('test', CommentedMap([('imports', ['JSON::XS'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~mlehmann/JSON-XS/'), ('license', 'perl_5'), ('summary', 'JSON serialising/deserialising, done correctly and fast')]))])), 'perl-libwww-perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-libwww-perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-libwww-perl'), ('version', '6.15')])), ('source', CommentedMap([('fn', 'libwww-perl-6.15.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/libwww-perl-6.15.tar.gz'), ('md5', '6888c9d8728cd6f3ea3c9754461c9f94')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-ntlm', 'perl-encode-locale', 'perl-file-listing', 'perl-html-parser', 'perl-http-cookies', 'perl-http-daemon', 'perl-http-date', 'perl-http-message', 'perl-http-negotiate', 'perl-lwp-mediatypes', 'perl-net-http', 'perl-uri', 'perl-www-robotrules']), ('run', ['perl-threaded', 'perl-ntlm', 'perl-encode-locale', 'perl-file-listing', 'perl-html-parser', 'perl-http-cookies', 'perl-http-daemon', 'perl-http-date', 'perl-http-message', 'perl-http-negotiate', 'perl-lwp-mediatypes', 'perl-net-http', 'perl-uri', 'perl-www-robotrules'])])), ('test', CommentedMap([('imports', ['LWP', 'LWP::Authen::Basic', 'LWP::Authen::Digest', 'LWP::Authen::Ntlm', 'LWP::ConnCache', 'LWP::Debug', 'LWP::DebugFile', 'LWP::MemberMixin', 'LWP::Protocol', 'LWP::Protocol::cpan', 'LWP::Protocol::data', 'LWP::Protocol::file', 'LWP::Protocol::ftp', 'LWP::Protocol::gopher', 'LWP::Protocol::http', 'LWP::Protocol::loopback', 'LWP::Protocol::nntp', 'LWP::Protocol::nogo', 'LWP::RobotUA', 'LWP::Simple', 'LWP::UserAgent'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/libwww-perl'), ('license', 'perl_5'), ('summary', 'The World-Wide Web library for Perl')]))])), 'perl-libxml-perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-libxml-perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-libxml-perl'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'libxml-perl-0.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/K/KM/KMACLEOD/libxml-perl-0.08.tar.gz'), ('md5', '0ed5fbdda53d1301ddaed88db10503bb')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-parser']), ('run', ['perl-threaded', 'perl-xml-parser'])])), ('test', CommentedMap([('imports', ['Data::Grove', 'Data::Grove::Parent', 'Data::Grove::Visitor', 'XML::ESISParser', 'XML::Handler::CanonXMLWriter', 'XML::Handler::Subs', 'XML::Handler::XMLWriter', 'XML::Parser::PerlSAX', 'XML::PatAct::Amsterdam', 'XML::PatAct::MatchName', 'XML::PatAct::ToObjects', 'XML::Perl2SAX', 'XML::SAX2Perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/libxml-perl'), ('license', 'unknown'), ('summary', 'Perl SAX parser using nsgmls')]))])), 'perl-list-moreutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-list-moreutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-list-moreutils'), ('version', '0.413')])), ('source', CommentedMap([('fn', 'List-MoreUtils-0.413.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.413.tar.gz'), ('md5', 'bc577c86455e2d1397187bce286a53b9')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-exporter-tiny']), ('run', ['perl-threaded', 'perl-exporter-tiny'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['List::MoreUtils', 'List::MoreUtils::PP', 'List::MoreUtils::XS'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/List-MoreUtils'), ('license', 'perl_5'), ('summary', 'Provide the stuff missing in List::Util')]))])), 'perl-list-uniq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-list-uniq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-list-uniq'), ('version', '0.20')])), ('source', CommentedMap([('fn', 'List-Uniq-0.20.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/J/JF/JFITZ/List-Uniq-0.20.tar.gz'), ('md5', '06abbe2fddf2522b9100d672a48fae59')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/List::Uniq'), ('license', 'perl_5'), ('summary', 'extract the unique elements of a list')]))])), 'perl-log-any': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-log-any/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-log-any'), ('version', '1.045')])), ('source', CommentedMap([('fn', 'Log-Any-1.045.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PR/PREACTION/Log-Any-1.045.tar.gz'), ('md5', '99ee3bfe069838008110ec52cac8c8c3')])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-apache-test', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-apache-test', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Log::Any', 'Log::Any::Adapter', 'Log::Any::Adapter::Base', 'Log::Any::Adapter::File', 'Log::Any::Adapter::Null', 'Log::Any::Adapter::Stderr', 'Log::Any::Adapter::Stdout', 'Log::Any::Adapter::Test', 'Log::Any::Adapter::Util', 'Log::Any::Manager', 'Log::Any::Proxy', 'Log::Any::Proxy::Null', 'Log::Any::Proxy::Test', 'Log::Any::Test'])])), ('about', CommentedMap([('home', 'https://github.com/preaction/Log-Any'), ('license', 'perl_5'), ('summary', 'Bringing loggers and listeners together')]))])), 'perl-log-log4perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-log-log4perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-log-log4perl'), ('version', '1.47')])), ('source', CommentedMap([('fn', 'Log-Log4perl-1.47.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MS/MSCHILLI/Log-Log4perl-1.47.tar.gz'), ('md5', '53a0a2e68da51fa445c3557e4852d3a4')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-dbi']), ('run', ['libgcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-dbi'])])), ('test', CommentedMap([('imports', ['Log::Log4perl', 'Log::Log4perl::Config', 'Log::Log4perl::Level', 'Log::Log4perl::Logger'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Log-Log4perl'), ('license', 'unknown'), ('summary', 'Log4j implementation for Perl')]))])), 'perl-lwp-mediatypes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-lwp-mediatypes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-lwp-mediatypes'), ('version', '6.02')])), ('source', CommentedMap([('fn', 'LWP-MediaTypes-6.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/LWP-MediaTypes-6.02.tar.gz'), ('md5', '8c5f25fb64b974d22aff424476ba13c9')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['LWP::MediaTypes']), ('commands', [\"perl -e 'use LWP::MediaTypes qw(guess_media_type);'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/LWP-MediaTypes'), ('license', 'perl_5'), ('summary', 'Guess media type for a file or a URL')]))])), 'perl-lwp-protocol-https': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-lwp-protocol-https/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-lwp-protocol-https'), ('version', '6.06')])), ('source', CommentedMap([('fn', 'LWP-Protocol-https-6.06.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MS/MSCHILLI/LWP-Protocol-https-6.06.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-libwww-perl', 'perl-io-socket-ssl', 'perl-mozilla-ca', 'perl-net-http']), ('run', ['perl-threaded', 'perl-libwww-perl', 'perl-io-socket-ssl', 'perl-mozilla-ca', 'perl-net-http'])])), ('test', CommentedMap([('imports', ['LWP::Protocol::https'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/LWP::Protocol::https'), ('license', 'Perl'), ('summary', 'Provide https support for LWP::UserAgent')]))])), 'perl-lwp-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-lwp-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-lwp-simple'), ('version', '6.15')])), ('source', CommentedMap([('fn', 'libwww-perl-6.15.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/libwww-perl-6.15.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-encode-locale', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded', 'perl-encode-locale'])])), ('test', CommentedMap([('imports', ['LWP::Simple'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/LWP::Simple'), ('license', 'Perl'), ('summary', 'simple procedural interface to LWP')]))])), 'perl-mailtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mailtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mailtools'), ('version', '2.14')])), ('source', CommentedMap([('fn', 'MailTools-2.14.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MARKOV/MailTools-2.14.tar.gz'), ('md5', '813ae849683367bb75e6be89e4e8cc46')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-timedate']), ('run', ['perl-threaded', 'perl-timedate'])])), ('test', CommentedMap([('imports', ['Mail::Address', 'Mail::Cap', 'Mail::Field', 'Mail::Field::AddrList', 'Mail::Field::Date', 'Mail::Field::Generic', 'Mail::Filter', 'Mail::Header', 'Mail::Internet', 'Mail::Mailer', 'Mail::Send', 'Mail::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MailTools'), ('license', 'perl_5'), ('summary', 'Various e-mail related modules')]))])), 'perl-math-base-convert': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-base-convert/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-base-convert'), ('version', '0.11')])), ('source', CommentedMap([('fn', 'Math-Base-Convert-0.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKER/Math-Base-Convert-0.11.tar.gz'), ('md5', 'aed8356d8e2870ee1d27a7b6c5752714')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Base-Convert'), ('license', 'unknown'), ('summary', 'very fast base to base conversion')]))])), 'perl-math-bezier': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-bezier/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-bezier'), ('version', '0.01')])), ('source', CommentedMap([('fn', 'Math-Bezier-0.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AB/ABW/Math-Bezier-0.01.tar.gz'), ('md5', 'ba6874d8754e2d64ab9c7d15e0eb56c2')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Bezier'), ('license', 'unknown'), ('summary', 'solution of Bezier Curves')]))])), 'perl-math-cdf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-cdf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-cdf'), ('version', '0.1')])), ('source', CommentedMap([('fn', 'Math-CDF-0.1.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CA/CALLAHAN/Math-CDF-0.1.tar.gz'), ('md5', '7866c7b6b9d27f0ce4b7637334478ab7')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Math::CDF'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-CDF'), ('license', 'Public Domain'), ('summary', 'Generate probabilities and quantiles from several statistical probability functions')]))])), 'perl-math-combinatorics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-combinatorics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-combinatorics'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'Math-Combinatorics-0.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AL/ALLENDAY/Math-Combinatorics-0.09.tar.gz'), ('md5', 'de556a046f00789c94cf58cb7c2d9d89')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Math::Combinatorics'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Combinatorics'), ('license', 'unknown'), ('summary', 'Perform combinations and permutations on lists')]))])), 'perl-math-derivative': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-derivative/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-derivative'), ('version', '0.04')])), ('source', CommentedMap([('fn', 'Math-Derivative-0.04.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JG/JGAMBLE/Math-Derivative-0.04.tar.gz'), ('md5', '7e357abf9b3106cc511bfe0e2689df05')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Math::Derivative'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Derivative'), ('license', 'perl_5'), ('summary', 'Numeric 1st and 2nd order differentiation.')]))])), 'perl-math-random': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-random/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-random'), ('version', '0.72')])), ('source', CommentedMap([('fn', 'Math-Random-0.72.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GR/GROMMEL/Math-Random-0.72.tar.gz'), ('md5', 'b0b961e2ae06920801a9c9843c48d0bc')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Math::Random'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Random'), ('license', 'unknown'), ('summary', 'Random Number Generators')]))])), 'perl-math-round': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-round/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-round'), ('version', '0.07')])), ('source', CommentedMap([('fn', 'Math-Round-0.07.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GR/GROMMEL/Math-Round-0.07.tar.gz'), ('md5', 'b74c5d2a3f891c19dafd7ce638732fd1')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Math::Round'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Round'), ('license', 'unknown'), ('summary', 'Perl extension for rounding numbers')]))])), 'perl-math-spline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-spline/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-spline'), ('version', '0.02')])), ('source', CommentedMap([('fn', 'Math-Spline-0.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHORNY/Math-Spline-0.02.tar.gz'), ('md5', '64c7bb3829e1fa7e49c2bf50b358c7c7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-math-derivative']), ('run', ['perl-threaded', 'perl-math-derivative'])])), ('test', CommentedMap([('imports', ['Math::Spline'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-Spline'), ('license', 'perl_5'), ('summary', 'Cubic Spline Interpolation of data')]))])), 'perl-math-vecstat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-math-vecstat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-math-vecstat'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Math-VecStat-0.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AS/ASPINELLI/Math-VecStat-0.08.tar.gz'), ('md5', '3df23fb451f73bb49fd4ea344ba94020')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Math-VecStat'), ('license', 'unknown'), ('summary', 'Some basic numeric stats on vectors')]))])), 'perl-memoize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-memoize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-memoize'), ('version', 1.03)])), ('source', CommentedMap([('fn', 'Memoize-1.03.tgz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MJ/MJD/Memoize-1.03.tgz'), ('md5', '6de4628f734fc898429c5b21367c02ab')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-extutils-makemaker'])])), ('test', CommentedMap([('imports', ['Memoize'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Memoize'), ('license', 'perl_5'), ('summary', 'Make functions faster by trading space for time')]))])), 'perl-mime-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mime-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mime-lite'), ('version', '3.030')])), ('source', CommentedMap([('fn', 'MIME-Lite-3.030.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RJ/RJBS/MIME-Lite-3.030.tar.gz'), ('md5', '5a6d90329e049eee77248d667343acc7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-email-date-format', 'perl-mailtools', 'perl-mime-types']), ('run', ['perl-threaded', 'perl-email-date-format', 'perl-mailtools', 'perl-mime-types'])])), ('test', CommentedMap([('imports', ['MIME::Lite'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MIME-Lite'), ('license', 'perl_5'), ('summary', 'Handy-dandy MIME mailing class')]))])), 'perl-mime-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mime-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mime-tools'), ('version', '5.507')])), ('source', CommentedMap([('fn', 'MIME-tools-5.507.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DS/DSKOLL/MIME-tools-5.507.tar.gz'), ('md5', '98e1997b6ddd973ced9b2f6feb9d6aaf')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-convert-binhex', 'perl-mailtools', 'perl-test-deep']), ('run', ['perl-threaded', 'perl-convert-binhex', 'perl-mailtools'])])), ('test', CommentedMap([('imports', ['MIME::Body', 'MIME::Decoder', 'MIME::Decoder::Base64', 'MIME::Decoder::BinHex', 'MIME::Decoder::Binary', 'MIME::Decoder::Gzip64', 'MIME::Decoder::NBit', 'MIME::Decoder::QuotedPrint', 'MIME::Decoder::UU', 'MIME::Entity', 'MIME::Field::ConTraEnc', 'MIME::Field::ContDisp', 'MIME::Field::ContType', 'MIME::Field::ParamVal', 'MIME::Head', 'MIME::Parser', 'MIME::Parser::Filer', 'MIME::Parser::Reader', 'MIME::Parser::Results', 'MIME::Tools', 'MIME::WordDecoder', 'MIME::Words'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MIME-tools'), ('license', 'perl_5'), ('summary', 'Tools to manipulate MIME messages')]))])), 'perl-mime-types': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mime-types/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mime-types'), ('version', '2.12')])), ('source', CommentedMap([('fn', 'MIME-Types-2.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MARKOV/MIME-Types-2.12.tar.gz'), ('md5', 'bbd17e44183d0e958edd27ebf4673272')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['MIME::Type', 'MIME::Types'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MIME-Types'), ('license', 'perl_5'), ('summary', 'Definition of MIME types')]))])), 'perl-mixin-linewise': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mixin-linewise/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mixin-linewise'), ('version', '0.108')])), ('source', CommentedMap([('fn', 'Mixin-Linewise-0.108.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Mixin-Linewise-0.108.tar.gz'), ('md5', 'c0f62761c6ce2ae6dc3e54153d042009')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-pathtools', 'perl-extutils-makemaker'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Mixin-Linewise'), ('license', 'perl_5'), ('summary', 'write your linewise code for handles; this does the rest')]))])), 'perl-mldbm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mldbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mldbm'), ('version', '2.05')])), ('source', CommentedMap([('fn', 'MLDBM-2.05.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHORNY/MLDBM-2.05.tar.gz'), ('md5', '9b7ec37bfc36b0e83db9323be92beb50')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['MLDBM'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MLDBM'), ('license', 'perl_5'), ('summary', 'store multi-level Perl hash structure in single level tied hash')]))])), 'perl-mldbm-sync': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mldbm-sync/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mldbm-sync'), ('version', '0.30')])), ('source', CommentedMap([('fn', 'MLDBM-Sync-0.30.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHAMAS/MLDBM-Sync-0.30.tar.gz'), ('md5', '1bb4e8d4bd6a30eee3f1126956409321')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-mldbm', 'perl-tie-cache']), ('run', ['perl-threaded', 'perl-mldbm', 'perl-tie-cache'])])), ('test', CommentedMap([('imports', ['MLDBM::Sync'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MLDBM-Sync'), ('license', 'unknown'), ('summary', 'safe concurrent access to MLDBM databases')]))])), 'perl-module-build': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-build/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-module-build'), ('version', '0.4214')])), ('source', CommentedMap([('fn', 'Module-Build-0.4214.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4214.tar.gz'), ('md5', '7b7ca5a47bef48c50c8b5906ca3ac7fb')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('imports', ['Module::Build', 'Module::Build::Base', 'Module::Build::Compat', 'Module::Build::Config', 'Module::Build::Cookbook', 'Module::Build::Dumper', 'Module::Build::Notes', 'Module::Build::PPMMaker', 'Module::Build::Platform::Default', 'Module::Build::Platform::MacOS', 'Module::Build::Platform::Unix', 'Module::Build::Platform::VMS', 'Module::Build::Platform::VOS', 'Module::Build::Platform::Windows', 'Module::Build::Platform::aix', 'Module::Build::Platform::cygwin', 'Module::Build::Platform::darwin', 'Module::Build::Platform::os2', 'Module::Build::PodParser'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module-Build'), ('license', 'perl_5'), ('summary', 'Build and install Perl modules')]))])), 'perl-module-build-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-build-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-module-build-tiny'), ('version', '0.039')])), ('source', CommentedMap([('fn', 'Module-Build-Tiny-0.039.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-Tiny-0.039.tar.gz'), ('md5', '2332c90c17454107fea3f2614e11a3a9')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-extutils-makemaker', 'perl-pathtools']), ('run', ['perl-threaded', 'perl-extutils-makemaker', 'perl-pathtools'])])), ('test', CommentedMap([('imports', ['Module::Build::Tiny'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module-Build-Tiny'), ('license', 'perl_5'), ('summary', 'A tiny replacement for Module::Build')]))])), 'perl-module-implementation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-implementation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-module-implementation'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'Module-Implementation-0.09.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Module-Implementation-0.09.tar.gz'), ('md5', '52e3fe0ca6b1eff0488d59b7aacc0667')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-requires', 'perl-module-runtime', 'perl-test-fatal', 'perl-try-tiny']), ('run', ['perl-threaded', 'perl-module-runtime', 'perl-try-tiny'])])), ('test', CommentedMap([('imports', ['Module::Implementation'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Module-Implementation'), ('license', 'artistic_2'), ('summary', 'Loads one of several alternate underlying implementations for a module')]))])), 'perl-module-runtime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-runtime/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-module-runtime'), ('version', '0.014')])), ('source', CommentedMap([('fn', 'Module-Runtime-0.014.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/Z/ZE/ZEFRAM/Module-Runtime-0.014.tar.gz'), ('md5', 'a6597bc25536a30476f0d75b64d35734')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Module::Runtime'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module-Runtime'), ('license', 'perl_5'), ('summary', 'runtime module handling')]))])), 'perl-moo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moo/2.000002/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moo'), ('version', '2.000002')])), ('source', CommentedMap([('fn', 'Moo-2.000002.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/H/HA/HAARG/Moo-2.000002.tar.gz'), ('md5', '8b84a7289fc6247de5ec5d151105fd6b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-runtime', 'perl-role-tiny', 'perl-class-method-modifiers', 'perl-devel-globaldestruction', 'perl-test-fatal']), ('run', ['perl-threaded', 'perl-module-runtime', 'perl-role-tiny', 'perl-class-method-modifiers', 'perl-devel-globaldestruction', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Moo'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Moo'), ('license', 'perl_5'), ('summary', 'Minimalist Object Orientation (with Moose compatibility)')]))])), 'perl-moose': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moose/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moose'), ('version', '2.1804')])), ('source', CommentedMap([('fn', 'Moose-2.1804.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Moose-2.1804.tar.gz'), ('md5', '9e755f8f08892184c5a3cde086e9a199')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-module-runtime', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-test-warnings', 'perl-scalar-list-utils >=1.45']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-module-runtime', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-test-warnings', 'perl-scalar-list-utils >=1.45'])])), ('test', CommentedMap([('imports', ['Moose', 'Moose::Exception', 'Moose::Exception::AccessorMustReadWrite', 'Moose::Exception::AddParameterizableTypeTakesParameterizableType', 'Moose::Exception::AddRoleTakesAMooseMetaRoleInstance', 'Moose::Exception::AddRoleToARoleTakesAMooseMetaRole', 'Moose::Exception::ApplyTakesABlessedInstance', 'Moose::Exception::AttachToClassNeedsAClassMOPClassInstanceOrASubclass', 'Moose::Exception::AttributeConflictInRoles', 'Moose::Exception::AttributeConflictInSummation', 'Moose::Exception::AttributeExtensionIsNotSupportedInRoles', 'Moose::Exception::AttributeIsRequired', 'Moose::Exception::AttributeMustBeAnClassMOPMixinAttributeCoreOrSubclass', 'Moose::Exception::AttributeNamesDoNotMatch', 'Moose::Exception::AttributeValueIsNotAnObject', 'Moose::Exception::AttributeValueIsNotDefined', 'Moose::Exception::AutoDeRefNeedsArrayRefOrHashRef', 'Moose::Exception::BadOptionFormat', 'Moose::Exception::BothBuilderAndDefaultAreNotAllowed', 'Moose::Exception::BuilderDoesNotExist', 'Moose::Exception::BuilderMethodNotSupportedForAttribute', 'Moose::Exception::BuilderMethodNotSupportedForInlineAttribute', 'Moose::Exception::BuilderMustBeAMethodName', 'Moose::Exception::CallingMethodOnAnImmutableInstance', 'Moose::Exception::CallingReadOnlyMethodOnAnImmutableInstance', 'Moose::Exception::CanExtendOnlyClasses', 'Moose::Exception::CanOnlyConsumeRole', 'Moose::Exception::CanOnlyWrapBlessedCode', 'Moose::Exception::CanReblessOnlyIntoASubclass', 'Moose::Exception::CanReblessOnlyIntoASuperclass', 'Moose::Exception::CannotAddAdditionalTypeCoercionsToUnion', 'Moose::Exception::CannotAddAsAnAttributeToARole', 'Moose::Exception::CannotApplyBaseClassRolesToRole', 'Moose::Exception::CannotAssignValueToReadOnlyAccessor', 'Moose::Exception::CannotAugmentIfLocalMethodPresent', 'Moose::Exception::CannotAugmentNoSuperMethod', 'Moose::Exception::CannotAutoDerefWithoutIsa', 'Moose::Exception::CannotAutoDereferenceTypeConstraint', 'Moose::Exception::CannotCalculateNativeType', 'Moose::Exception::CannotCallAnAbstractBaseMethod', 'Moose::Exception::CannotCallAnAbstractMethod', 'Moose::Exception::CannotCoerceAWeakRef', 'Moose::Exception::CannotCoerceAttributeWhichHasNoCoercion', 'Moose::Exception::CannotCreateHigherOrderTypeWithoutATypeParameter', 'Moose::Exception::CannotCreateMethodAliasLocalMethodIsPresent', 'Moose::Exception::CannotCreateMethodAliasLocalMethodIsPresentInClass', 'Moose::Exception::CannotDelegateLocalMethodIsPresent', 'Moose::Exception::CannotDelegateWithoutIsa', 'Moose::Exception::CannotFindDelegateMetaclass', 'Moose::Exception::CannotFindType', 'Moose::Exception::CannotFindTypeGivenToMatchOnType', 'Moose::Exception::CannotFixMetaclassCompatibility', 'Moose::Exception::CannotGenerateInlineConstraint', 'Moose::Exception::CannotInitializeMooseMetaRoleComposite', 'Moose::Exception::CannotInlineTypeConstraintCheck', 'Moose::Exception::CannotLocatePackageInINC', 'Moose::Exception::CannotMakeMetaclassCompatible', 'Moose::Exception::CannotOverrideALocalMethod', 'Moose::Exception::CannotOverrideBodyOfMetaMethods', 'Moose::Exception::CannotOverrideLocalMethodIsPresent', 'Moose::Exception::CannotOverrideNoSuperMethod', 'Moose::Exception::CannotRegisterUnnamedTypeConstraint', 'Moose::Exception::CannotUseLazyBuildAndDefaultSimultaneously', 'Moose::Exception::CircularReferenceInAlso', 'Moose::Exception::ClassDoesNotHaveInitMeta', 'Moose::Exception::ClassDoesTheExcludedRole', 'Moose::Exception::ClassNamesDoNotMatch', 'Moose::Exception::CloneObjectExpectsAnInstanceOfMetaclass', 'Moose::Exception::CodeBlockMustBeACodeRef', 'Moose::Exception::CoercingWithoutCoercions', 'Moose::Exception::CoercionAlreadyExists', 'Moose::Exception::CoercionNeedsTypeConstraint', 'Moose::Exception::ConflictDetectedInCheckRoleExclusions', 'Moose::Exception::ConflictDetectedInCheckRoleExclusionsInToClass', 'Moose::Exception::ConstructClassInstanceTakesPackageName', 'Moose::Exception::CouldNotCreateMethod', 'Moose::Exception::CouldNotCreateWriter', 'Moose::Exception::CouldNotEvalConstructor', 'Moose::Exception::CouldNotEvalDestructor', 'Moose::Exception::CouldNotFindTypeConstraintToCoerceFrom', 'Moose::Exception::CouldNotGenerateInlineAttributeMethod', 'Moose::Exception::CouldNotLocateTypeConstraintForUnion', 'Moose::Exception::CouldNotParseType', 'Moose::Exception::CreateMOPClassTakesArrayRefOfAttributes', 'Moose::Exception::CreateMOPClassTakesArrayRefOfSuperclasses', 'Moose::Exception::CreateMOPClassTakesHashRefOfMethods', 'Moose::Exception::CreateTakesArrayRefOfRoles', 'Moose::Exception::CreateTakesHashRefOfAttributes', 'Moose::Exception::CreateTakesHashRefOfMethods', 'Moose::Exception::DefaultToMatchOnTypeMustBeCodeRef', 'Moose::Exception::DelegationToAClassWhichIsNotLoaded', 'Moose::Exception::DelegationToARoleWhichIsNotLoaded', 'Moose::Exception::DelegationToATypeWhichIsNotAClass', 'Moose::Exception::DoesRequiresRoleName', 'Moose::Exception::EnumCalledWithAnArrayRefAndAdditionalArgs', 'Moose::Exception::EnumValuesMustBeString', 'Moose::Exception::ExtendsMissingArgs', 'Moose::Exception::HandlesMustBeAHashRef', 'Moose::Exception::IllegalInheritedOptions', 'Moose::Exception::IllegalMethodTypeToAddMethodModifier', 'Moose::Exception::IncompatibleMetaclassOfSuperclass', 'Moose::Exception::InitMetaRequiresClass', 'Moose::Exception::InitializeTakesUnBlessedPackageName', 'Moose::Exception::InstanceBlessedIntoWrongClass', 'Moose::Exception::InstanceMustBeABlessedReference', 'Moose::Exception::InvalidArgPassedToMooseUtilMetaRole', 'Moose::Exception::InvalidArgumentToMethod', 'Moose::Exception::InvalidArgumentsToTraitAliases', 'Moose::Exception::InvalidBaseTypeGivenToCreateParameterizedTypeConstraint', 'Moose::Exception::InvalidHandleValue', 'Moose::Exception::InvalidHasProvidedInARole', 'Moose::Exception::InvalidNameForType', 'Moose::Exception::InvalidOverloadOperator', 'Moose::Exception::InvalidRoleApplication', 'Moose::Exception::InvalidTypeConstraint', 'Moose::Exception::InvalidTypeGivenToCreateParameterizedTypeConstraint', 'Moose::Exception::InvalidValueForIs', 'Moose::Exception::IsaDoesNotDoTheRole', 'Moose::Exception::IsaLacksDoesMethod', 'Moose::Exception::LazyAttributeNeedsADefault', 'Moose::Exception::Legacy', 'Moose::Exception::MOPAttributeNewNeedsAttributeName', 'Moose::Exception::MatchActionMustBeACodeRef', 'Moose::Exception::MessageParameterMustBeCodeRef', 'Moose::Exception::MetaclassIsAClassNotASubclassOfGivenMetaclass', 'Moose::Exception::MetaclassIsARoleNotASubclassOfGivenMetaclass', 'Moose::Exception::MetaclassIsNotASubclassOfGivenMetaclass', 'Moose::Exception::MetaclassMustBeASubclassOfMooseMetaClass', 'Moose::Exception::MetaclassMustBeASubclassOfMooseMetaRole', 'Moose::Exception::MetaclassMustBeDerivedFromClassMOPClass', 'Moose::Exception::MetaclassNotLoaded', 'Moose::Exception::MetaclassTypeIncompatible', 'Moose::Exception::MethodExpectedAMetaclassObject', 'Moose::Exception::MethodExpectsFewerArgs', 'Moose::Exception::MethodExpectsMoreArgs', 'Moose::Exception::MethodModifierNeedsMethodName', 'Moose::Exception::MethodNameConflictInRoles', 'Moose::Exception::MethodNameNotFoundInInheritanceHierarchy', 'Moose::Exception::MethodNameNotGiven', 'Moose::Exception::MustDefineAMethodName', 'Moose::Exception::MustDefineAnAttributeName', 'Moose::Exception::MustDefineAnOverloadOperator', 'Moose::Exception::MustHaveAtLeastOneValueToEnumerate', 'Moose::Exception::MustPassAHashOfOptions', 'Moose::Exception::MustPassAMooseMetaRoleInstanceOrSubclass', 'Moose::Exception::MustPassAPackageNameOrAnExistingClassMOPPackageInstance', 'Moose::Exception::MustPassEvenNumberOfArguments', 'Moose::Exception::MustPassEvenNumberOfAttributeOptions', 'Moose::Exception::MustProvideANameForTheAttribute', 'Moose::Exception::MustSpecifyAtleastOneMethod', 'Moose::Exception::MustSpecifyAtleastOneRole', 'Moose::Exception::MustSpecifyAtleastOneRoleToApplicant', 'Moose::Exception::MustSupplyAClassMOPAttributeInstance', 'Moose::Exception::MustSupplyADelegateToMethod', 'Moose::Exception::MustSupplyAMetaclass', 'Moose::Exception::MustSupplyAMooseMetaAttributeInstance', 'Moose::Exception::MustSupplyAnAccessorTypeToConstructWith', 'Moose::Exception::MustSupplyAnAttributeToConstructWith', 'Moose::Exception::MustSupplyArrayRefAsCurriedArguments', 'Moose::Exception::MustSupplyPackageNameAndName', 'Moose::Exception::NeedsTypeConstraintUnionForTypeCoercionUnion', 'Moose::Exception::NeitherAttributeNorAttributeNameIsGiven', 'Moose::Exception::NeitherClassNorClassNameIsGiven', 'Moose::Exception::NeitherRoleNorRoleNameIsGiven', 'Moose::Exception::NeitherTypeNorTypeNameIsGiven', 'Moose::Exception::NoAttributeFoundInSuperClass', 'Moose::Exception::NoBodyToInitializeInAnAbstractBaseClass', 'Moose::Exception::NoCasesMatched', 'Moose::Exception::NoConstraintCheckForTypeConstraint', 'Moose::Exception::NoDestructorClassSpecified', 'Moose::Exception::NoImmutableTraitSpecifiedForClass', 'Moose::Exception::NoParentGivenToSubtype', 'Moose::Exception::OnlyInstancesCanBeCloned', 'Moose::Exception::OperatorIsRequired', 'Moose::Exception::OverloadConflictInSummation', 'Moose::Exception::OverloadRequiresAMetaClass', 'Moose::Exception::OverloadRequiresAMetaMethod', 'Moose::Exception::OverloadRequiresAMetaOverload', 'Moose::Exception::OverloadRequiresAMethodNameOrCoderef', 'Moose::Exception::OverloadRequiresAnOperator', 'Moose::Exception::OverloadRequiresNamesForCoderef', 'Moose::Exception::OverrideConflictInComposition', 'Moose::Exception::OverrideConflictInSummation', 'Moose::Exception::PackageDoesNotUseMooseExporter', 'Moose::Exception::PackageNameAndNameParamsNotGivenToWrap', 'Moose::Exception::PackagesAndModulesAreNotCachable', 'Moose::Exception::ParameterIsNotSubtypeOfParent', 'Moose::Exception::ReferencesAreNotAllowedAsDefault', 'Moose::Exception::RequiredAttributeLacksInitialization', 'Moose::Exception::RequiredAttributeNeedsADefault', 'Moose::Exception::RequiredMethodsImportedByClass', 'Moose::Exception::RequiredMethodsNotImplementedByClass', 'Moose::Exception::Role::Attribute', 'Moose::Exception::Role::AttributeName', 'Moose::Exception::Role::Class', 'Moose::Exception::Role::EitherAttributeOrAttributeName', 'Moose::Exception::Role::Instance', 'Moose::Exception::Role::InstanceClass', 'Moose::Exception::Role::InvalidAttributeOptions', 'Moose::Exception::Role::Method', 'Moose::Exception::Role::ParamsHash', 'Moose::Exception::Role::Role', 'Moose::Exception::Role::RoleForCreate', 'Moose::Exception::Role::RoleForCreateMOPClass', 'Moose::Exception::Role::TypeConstraint', 'Moose::Exception::RoleDoesTheExcludedRole', 'Moose::Exception::RoleExclusionConflict', 'Moose::Exception::RoleNameRequired', 'Moose::Exception::RoleNameRequiredForMooseMetaRole', 'Moose::Exception::RolesDoNotSupportAugment', 'Moose::Exception::RolesDoNotSupportExtends', 'Moose::Exception::RolesDoNotSupportInner', 'Moose::Exception::RolesDoNotSupportRegexReferencesForMethodModifiers', 'Moose::Exception::RolesInCreateTakesAnArrayRef', 'Moose::Exception::RolesListMustBeInstancesOfMooseMetaRole', 'Moose::Exception::SingleParamsToNewMustBeHashRef', 'Moose::Exception::TriggerMustBeACodeRef', 'Moose::Exception::TypeConstraintCannotBeUsedForAParameterizableType', 'Moose::Exception::TypeConstraintIsAlreadyCreated', 'Moose::Exception::TypeParameterMustBeMooseMetaType', 'Moose::Exception::UnableToCanonicalizeHandles', 'Moose::Exception::UnableToCanonicalizeNonRolePackage', 'Moose::Exception::UnableToRecognizeDelegateMetaclass', 'Moose::Exception::UndefinedHashKeysPassedToMethod', 'Moose::Exception::UnionCalledWithAnArrayRefAndAdditionalArgs', 'Moose::Exception::UnionTakesAtleastTwoTypeNames', 'Moose::Exception::ValidationFailedForInlineTypeConstraint', 'Moose::Exception::ValidationFailedForTypeConstraint', 'Moose::Exception::WrapTakesACodeRefToBless', 'Moose::Exception::WrongTypeConstraintGiven', 'Moose::Exporter', 'Moose::Meta::Attribute', 'Moose::Meta::Attribute::Native', 'Moose::Meta::Attribute::Native::Trait::Array', 'Moose::Meta::Attribute::Native::Trait::Bool', 'Moose::Meta::Attribute::Native::Trait::Code', 'Moose::Meta::Attribute::Native::Trait::Counter', 'Moose::Meta::Attribute::Native::Trait::Hash', 'Moose::Meta::Attribute::Native::Trait::Number', 'Moose::Meta::Attribute::Native::Trait::String', 'Moose::Meta::Class', 'Moose::Meta::Instance', 'Moose::Meta::Method', 'Moose::Meta::Method::Accessor', 'Moose::Meta::Method::Augmented', 'Moose::Meta::Method::Constructor', 'Moose::Meta::Method::Delegation', 'Moose::Meta::Method::Destructor', 'Moose::Meta::Method::Meta', 'Moose::Meta::Method::Overridden', 'Moose::Meta::TypeCoercion', 'Moose::Meta::TypeCoercion::Union', 'Moose::Meta::TypeConstraint', 'Moose::Meta::TypeConstraint::Class', 'Moose::Meta::TypeConstraint::DuckType', 'Moose::Meta::TypeConstraint::Enum', 'Moose::Meta::TypeConstraint::Parameterizable', 'Moose::Meta::TypeConstraint::Parameterized', 'Moose::Meta::TypeConstraint::Registry', 'Moose::Meta::TypeConstraint::Role', 'Moose::Meta::TypeConstraint::Union', 'Moose::Object', 'Moose::Util', 'Moose::Util::MetaRole', 'Moose::Util::TypeConstraints'])])), ('about', CommentedMap([('home', 'http://moose.perl.org/'), ('license', 'perl_5'), ('summary', 'A postmodern object system for Perl 5')]))])), 'perl-moosex-app': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-app/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-app'), ('version', '1.35')])), ('source', CommentedMap([('fn', 'MooseX-App-1.35.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MAROS/MooseX-App-1.35.tar.gz'), ('md5', 'be0ef3961d5a394e2366f7fbc0b62282')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-extutils-makemaker', 'perl-path-class', 'perl-config-any', 'perl-moosex-types-path-class', 'perl-moose', 'perl-pod-elemental', 'perl-scalar-list-utils', 'perl-io-interactive', 'perl-termreadkey', 'perl-file-homedir']), ('run', ['perl-threaded', 'perl-path-class', 'perl-config-any', 'perl-moosex-types-path-class', 'perl-moose', 'perl-pod-elemental', 'perl-scalar-list-utils', 'perl-io-interactive', 'perl-termreadkey', 'perl-file-homedir'])])), ('test', CommentedMap([('imports', ['MooseX::App', 'MooseX::App::Command', 'MooseX::App::Exporter', 'MooseX::App::Message::Block', 'MooseX::App::Message::BlockColor', 'MooseX::App::Message::Envelope', 'MooseX::App::Meta::Role::Attribute::Option', 'MooseX::App::Meta::Role::Class::Base', 'MooseX::App::Meta::Role::Class::Command', 'MooseX::App::Meta::Role::Class::Documentation', 'MooseX::App::Meta::Role::Class::Simple', 'MooseX::App::ParsedArgv', 'MooseX::App::ParsedArgv::Element', 'MooseX::App::Plugin::BashCompletion', 'MooseX::App::Plugin::BashCompletion::Command', 'MooseX::App::Plugin::BashCompletion::Meta::Class', 'MooseX::App::Plugin::Color', 'MooseX::App::Plugin::Color::Meta::Class', 'MooseX::App::Plugin::Config', 'MooseX::App::Plugin::Config::Meta::Class', 'MooseX::App::Plugin::ConfigHome', 'MooseX::App::Plugin::ConfigHome::Meta::Class', 'MooseX::App::Plugin::Depends', 'MooseX::App::Plugin::Depends::Meta::Attribute', 'MooseX::App::Plugin::Depends::Meta::Class', 'MooseX::App::Plugin::Env', 'MooseX::App::Plugin::Fuzzy', 'MooseX::App::Plugin::Man', 'MooseX::App::Plugin::Man::Command', 'MooseX::App::Plugin::Man::Meta::Class', 'MooseX::App::Plugin::MutexGroup', 'MooseX::App::Plugin::MutexGroup::Meta::Attribute', 'MooseX::App::Plugin::MutexGroup::Meta::Class', 'MooseX::App::Plugin::Term', 'MooseX::App::Plugin::Term::Meta::Attribute', 'MooseX::App::Plugin::Term::Meta::Class', 'MooseX::App::Plugin::Typo', 'MooseX::App::Plugin::Typo::Meta::Class', 'MooseX::App::Plugin::Version', 'MooseX::App::Plugin::Version::Command', 'MooseX::App::Plugin::Version::Meta::Class', 'MooseX::App::Simple', 'MooseX::App::Utils'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MooseX::App'), ('license', 'perl_5'), ('summary', 'Write user-friendly command line apps with even less suffering')]))])), 'perl-moosex-app-role-log4perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-app-role-log4perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-app-role-log4perl'), ('version', '0.03')])), ('source', CommentedMap([('fn', 'MooseX-App-Role-Log4perl-0.03.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PS/PSYDEFECT/MooseX-App-Role-Log4perl-0.03.tar.gz'), ('md5', 'cf4138d6de167e554c5f1364461cdae9')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-moosex-app', 'perl-log-log4perl', 'perl-module-build']), ('run', ['perl-threaded', 'perl-moosex-app', 'perl-log-log4perl'])])), ('test', CommentedMap([('imports', ['MooseX::App::Role::Log4perl'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/MooseX::App::Role::Log4perl'), ('license', 'perl_5'), ('summary', 'Add basic Log::Log4perl logging to a MooseX::App application as a role.')]))])), 'perl-moosex-fileattribute': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-fileattribute/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-fileattribute'), ('version', '0.02')])), ('source', CommentedMap([('fn', 'MooseX-FileAttribute-0.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/J/JR/JROCKWAY/MooseX-FileAttribute-0.02.tar.gz'), ('md5', '6024b41ed243cfcb65f97ed0b7750627')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-moosex-types', 'perl-test-exception', 'perl-moosex-types-path-class', 'perl-moose']), ('run', ['perl-threaded', 'perl-moosex-types', 'perl-moosex-types-path-class', 'perl-moose'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MooseX-FileAttribute'), ('license', 'perl_5'), ('summary', 'sugar for classes that have file or directory attributes')]))])), 'perl-moosex-getopt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-getopt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-getopt'), ('version', '0.71')])), ('source', CommentedMap([('fn', 'MooseX-Getopt-0.71.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/MooseX-Getopt-0.71.tar.gz'), ('md5', '7ac31336c49315143b27c856e10862b0')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-test-requires', 'perl-module-runtime', 'perl-test-deep', 'perl-moose', 'perl-path-tiny', 'perl-try-tiny', 'perl-module-build-tiny', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-moose', 'perl-try-tiny', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['MooseX::Getopt', 'MooseX::Getopt::Basic', 'MooseX::Getopt::Dashes', 'MooseX::Getopt::GLD', 'MooseX::Getopt::Meta::Attribute', 'MooseX::Getopt::Meta::Attribute::NoGetopt', 'MooseX::Getopt::Meta::Attribute::Trait', 'MooseX::Getopt::Meta::Attribute::Trait::NoGetopt', 'MooseX::Getopt::OptionTypeMap', 'MooseX::Getopt::ProcessedArgv', 'MooseX::Getopt::Strict'])])), ('about', CommentedMap([('home', 'https://github.com/moose/MooseX-Getopt'), ('license', 'perl_5'), ('summary', 'A Moose role for processing command line options')]))])), 'perl-moosex-types': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-types/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-types'), ('version', '0.46')])), ('source', CommentedMap([('fn', 'MooseX-Types-0.46.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/MooseX-Types-0.46.tar.gz'), ('md5', 'e1debd6007b9cc494636499397c6dabf')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-app-cpanminus', 'perl-threaded', 'perl-scalar-list-utils', 'perl-module-runtime', 'perl-moose', 'perl-module-build-tiny', 'perl-extutils-makemaker', 'perl-pathtools']), ('run', ['perl-threaded', 'perl-scalar-list-utils', 'perl-module-runtime', 'perl-moose'])])), ('test', CommentedMap([('imports', ['MooseX::Types', 'MooseX::Types::Base', 'MooseX::Types::CheckedUtilExports', 'MooseX::Types::Combine', 'MooseX::Types::Moose', 'MooseX::Types::TypeDecorator', 'MooseX::Types::UndefinedType', 'MooseX::Types::Util', 'MooseX::Types::Wrapper'])])), ('about', CommentedMap([('home', 'https://github.com/moose/MooseX-Types'), ('license', 'perl_5'), ('summary', 'Organise your Moose types in libraries')]))])), 'perl-moosex-types-path-class': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-moosex-types-path-class/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-moosex-types-path-class'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'MooseX-Types-Path-Class-0.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/MooseX-Types-Path-Class-0.09.tar.gz'), ('md5', '6ef4c82511793594d37bcabcc29fa124')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-path-class', 'perl-moosex-types', 'perl-moose', 'perl-module-build-tiny']), ('run', ['perl-threaded', 'perl-path-class', 'perl-moosex-types'])])), ('test', CommentedMap([('imports', ['MooseX::Types::Path::Class'])])), ('about', CommentedMap([('home', 'https://github.com/moose/MooseX-Types-Path-Class'), ('license', 'perl_5'), ('summary', 'A Path::Class type library for Moose')]))])), 'perl-mozilla-ca': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mozilla-ca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-mozilla-ca'), ('version', '20160104')])), ('source', CommentedMap([('fn', 'Mozilla-CA-20160104.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AB/ABH/Mozilla-CA-20160104.tar.gz'), ('md5', '1b91edb15953a8188f011ab5ff433300')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Mozilla::CA'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Mozilla-CA'), ('license', 'unknown'), ('summary', \"Mozilla's CA cert bundle in PEM format\")]))])), 'perl-net-http': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-net-http/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-net-http'), ('version', '6.09')])), ('source', CommentedMap([('fn', 'Net-HTTP-6.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Net-HTTP-6.09.tar.gz'), ('md5', '3d84d17f64c7316d69e7eb7b4e292b9a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-socket-ssl', 'perl-uri']), ('run', ['perl-threaded', 'perl-io-socket-ssl', 'perl-uri'])])), ('test', CommentedMap([('imports', ['Net::HTTP', 'Net::HTTP::Methods', 'Net::HTTP::NB', 'Net::HTTPS'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net-HTTP'), ('license', 'perl_5'), ('summary', 'Low-level HTTP connection (client)')]))])), 'perl-net-ssleay': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-net-ssleay/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-net-ssleay'), ('version', '1.74')])), ('source', CommentedMap([('fn', 'Net-SSLeay-1.74.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKEM/Net-SSLeay-1.74.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'openssl', 'zlib']), ('run', ['perl-threaded', 'openssl', 'zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['perl -e \\'use Net::SSLeay qw(get_https); get_https(\"www.google.com\", 443, \"/\");\\''])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net-SSLeay'), ('license', 'perl_5'), ('summary', 'Perl extension for using OpenSSL')]))])), 'perl-ntlm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ntlm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ntlm'), ('version', '1.09')])), ('source', CommentedMap([('fn', 'NTLM-1.09.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/N/NB/NBEBOUT/NTLM-1.09.tar.gz'), ('md5', 'da314ee94b14af2a2f39b6f2c0046e73')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-digest-hmac']), ('run', ['perl-threaded', 'perl-digest-hmac'])])), ('test', CommentedMap([('imports', ['Authen::NTLM'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/NTLM'), ('license', 'perl_5'), ('summary', 'An NTLM authentication module')]))])), 'perl-number-compare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-number-compare/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-number-compare'), ('version', '0.03')])), ('source', CommentedMap([('url', 'https://cpan.metacpan.org/authors/id/R/RC/RCLAMP/Number-Compare-0.03.tar.gz'), ('md5', 'ded4085a8fc96328742785574ca65208')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Number::Compare'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Number::Compare'), ('license', 'perl_5'), ('summary', 'Numeric comparisons')]))])), 'perl-number-format': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-number-format/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-number-format'), ('version', '1.75')])), ('source', CommentedMap([('fn', 'Number-Format-1.75.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/W/WR/WRW/Number-Format-1.75.tar.gz'), ('md5', '2285c537dcf89a5bf556effaf706fd5f')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Number::Format'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Number-Format'), ('license', 'perl_5'), ('summary', 'Perl extension for formatting numbers')]))])), 'perl-ole-storage_lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ole-storage_lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ole-storage_lite'), ('version', '0.19')])), ('source', CommentedMap([('fn', 'OLE-Storage_Lite-0.19.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/J/JM/JMCNAMARA/OLE-Storage_Lite-0.19.tar.gz'), ('md5', '1a713f5342c7d90e54ab0d9659650296')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['OLE::Storage_Lite'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/OLE-Storage_Lite'), ('license', 'unknown'), ('summary', 'Read and write OLE storage files.')]))])), 'perl-onto-perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-onto-perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-onto-perl'), ('version', '1.45')])), ('source', CommentedMap([('fn', 'ONTO-PERL-1.45.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EA/EASR/ONTO-PERL/ONTO-PERL-1.45.tar.gz'), ('md5', '998c417cb74a1140b3e7018aab0af478')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-date-manip', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-date-manip'])])), ('test', CommentedMap([('imports', ['OBO::Core::Term']), ('commands', ['go2csv.pl 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/ONTO-PERL'), ('license', 'perl_5'), ('summary', 'PERL modules for manipulating OBO-formatted ontologies, such as the Gene Ontology (GO)')]))])), 'perl-package-stash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-package-stash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-package-stash'), ('version', '0.37')])), ('source', CommentedMap([('fn', 'Package-Stash-0.37.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DO/DOY/Package-Stash-0.37.tar.gz'), ('md5', '7e47a8261312e1cf3d12bd2007916b66')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-text-parsewords', 'perl-test-requires', 'perl-extutils-makemaker', 'perl-getopt-long', 'perl-apache-test', 'perl-module-implementation', 'perl-test-fatal', 'perl-scalar-list-utils', 'perl-dist-checkconflicts', 'perl-test-simple']), ('run', ['perl', 'perl-getopt-long', 'perl-module-implementation', 'perl-apache-test', 'perl-dist-checkconflicts', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Package::Stash', 'Package::Stash::PP'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Package-Stash'), ('license', 'perl_5'), ('summary', 'routines for manipulating stashes')]))])), 'perl-parallel-forkmanager': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-parallel-forkmanager/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-parallel-forkmanager'), ('version', '1.17')])), ('source', CommentedMap([('fn', 'Parallel-ForkManager-1.17.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/Y/YA/YANICK/Parallel-ForkManager-1.17.tar.gz'), ('md5', 'be82b8b9bd3bff04f0fcec37b1de43d2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-warn']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Parallel::ForkManager'])])), ('about', CommentedMap([('home', 'https://github.com/dluxhu/perl-parallel-forkmanager'), ('license', 'perl_5'), ('summary', 'A simple parallel processing fork manager')]))])), 'perl-params-util': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-params-util/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-params-util'), ('version', '1.07')])), ('source', CommentedMap([('fn', 'Params-Util-1.07.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Params-Util-1.07.tar.gz'), ('md5', '02db120c0eef87aae1830cc62bdec37b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-scalar-list-utils', 'perl-extutils-makemaker', 'perl-test-simple', 'perl-pathtools']), ('run', ['perl-threaded', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Params::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Params-Util'), ('license', 'perl_5'), ('summary', 'Simple, compact and correct param-checking functions')]))])), 'perl-params-validate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-params-validate/1.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Params-Validate-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Params-Validate-1.26.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-implementation', 'perl-test-fatal', 'perl-test-requires', 'perl-xsloader', 'perl-module-build', 'perl-extutils-makemaker']), ('run', ['perl', 'libgcc', 'perl-module-implementation', 'perl-xsloader'])])), ('test', CommentedMap([('imports', ['Params::Validate'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Params-Validate'), ('license', 'artistic_2'), ('summary', 'Validate method/function parameters')]))])), 'perl-parse-recdescent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-parse-recdescent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-parse-recdescent'), ('version', '1.967013')])), ('source', CommentedMap([('fn', 'Parse-RecDescent-1.967013.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967013.tar.gz'), ('md5', '98f582240556a4a92e5d57e480f9d39d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-test-warn']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Parse::RecDescent'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Parse-RecDescent'), ('license', 'unknown'), ('summary', 'Generate Recursive-Descent Parsers')]))])), 'perl-parse-yapp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-parse-yapp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-parse-yapp'), ('version', '1.05')])), ('source', CommentedMap([('fn', 'Parse-Yapp-1.05.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/F/FD/FDESAR/Parse-Yapp-1.05.tar.gz'), ('md5', '7bfca736d6af36c51edf7a97111a8f3b')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Parse::Yapp'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Parse::Yapp'), ('license', 'unknown'), ('summary', 'Perl extension for generating and using LALR parsers.')]))])), 'perl-path-class': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-path-class/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-path-class'), ('version', '0.36')])), ('source', CommentedMap([('fn', 'Path-Class-0.36.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/K/KW/KWILLIAMS/Path-Class-0.36.tar.gz'), ('md5', '3e15307faf0702fb916409f594552b94')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Path::Class', 'Path::Class::Dir', 'Path::Class::Entity', 'Path::Class::File'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Path-Class'), ('license', 'perl_5'), ('summary', 'Cross-platform path specification manipulation')]))])), 'perl-path-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-path-tiny/0.076/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-path-tiny'), ('version', '0.076')])), ('source', CommentedMap([('fn', 'Path-Tiny-0.076.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Path-Tiny-0.076.tar.gz'), ('md5', '495b965187a59af9b99c0f4709013746')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Path::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Path-Tiny'), ('license', 'apache_2_0'), ('summary', 'File path utility')]))])), 'perl-pathtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pathtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pathtools'), ('version', 3.4)])), ('source', CommentedMap([('fn', 'PathTools-3.40.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SM/SMUELLER/PathTools-3.40.tar.gz'), ('md5', '7da19c34dc2d1017cbb675a6ff4fe283')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-scalar-list-utils'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PathTools'), ('license', 'unknown'), ('summary', 'get pathname of current working directory')]))])), 'perl-pbkdf2-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pbkdf2-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pbkdf2-tiny'), ('version', '0.005')])), ('source', CommentedMap([('fn', 'PBKDF2-Tiny-0.005.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/PBKDF2-Tiny-0.005.tar.gz'), ('md5', '6628cad808d13bedb20054eea84b0f72')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['PBKDF2::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/PBKDF2-Tiny'), ('license', 'apache_2_0'), ('summary', 'Minimalist PBKDF2 (RFC 2898) with HMAC-SHA1 or HMAC-SHA2')]))])), 'perl-pcap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pcap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pcap'), ('version', '1.11.1')])), ('source', CommentedMap([('fn', 'PCAP-v1.11.1.tar.gz'), ('url', 'https://github.com/ICGC-TCGA-PanCancer/PCAP-core/archive/v1.11.1.tar.gz')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-file-sharedir-install', 'perl-module-build', 'perl-bio-db-sam', 'perl-bioperl', 'perl-gd', 'perl-ipc-system-simple', 'perl-sanger-cgp-vcf', 'perl-encode-locale', 'perl-lwp-simple', 'perl-xml-parser']), ('run', ['libgcc', 'perl-threaded', 'perl-bio-db-sam', 'perl-bioperl', 'perl-encode-locale', 'perl-gd', 'perl-ipc-system-simple', 'perl-sanger-cgp-vcf', 'perl-encode-locale', 'perl-lwp-simple', 'perl-xml-parser'])])), ('test', CommentedMap([('imports', ['PCAP'])])), ('about', CommentedMap([('home', 'https://github.com/ICGC-TCGA-PanCancer/PCAP-core'), ('license', 'GPLv3'), ('summary', 'NGS reference implementations and helper code for the IGCG/TCGA Pan-Cancer Analysis Project')]))])), 'perl-pdf-api2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pdf-api2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pdf-api2'), ('version', '2.025')])), ('source', CommentedMap([('fn', 'PDF-API2-2.025.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SS/SSIMMS/PDF-API2-2.025.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['PDF::API2'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/PDF::API2'), ('license', 'Perl'), ('summary', 'Facilitates the creation and modification of PDF files')]))])), 'perl-pegex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pegex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pegex'), ('version', '0.61')])), ('source', CommentedMap([('fn', 'Pegex-0.61.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/I/IN/INGY/Pegex-0.61.tar.gz'), ('md5', '04ab96185d73bdad02dfbabb1152af40')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-file-sharedir-install', 'perl-yaml-libyaml']), ('run', ['perl-threaded', 'perl-extutils-makemaker', 'perl-file-sharedir-install', 'perl-yaml-libyaml'])])), ('test', CommentedMap([('imports', ['Pegex'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~ingy/Pegex-0.61/lib/Pegex.pod'), ('license', 'perl_5'), ('summary', 'Pegex Grammar for the Pegex Grammar Language')]))])), 'perl-perl-osnames': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perl-osnames/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-perl-osnames'), ('version', '0.11')])), ('source', CommentedMap([('fn', 'Perl-osnames-0.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Perl-osnames-0.11.tar.gz'), ('md5', '43a842825d93e9da7e83236567c4b194')])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-test-simple', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Perl::osnames'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Perl-osnames'), ('license', 'perl_5'), ('summary', 'List possible $^O ($OSNAME) values, with description')]))])), 'perl-perl-unsafe-signals': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perl-unsafe-signals/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-perl-unsafe-signals'), ('version', '0.03')])), ('source', CommentedMap([('fn', 'Perl-Unsafe-Signals-0.03.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RG/RGARCIA/Perl-Unsafe-Signals-0.03.tar.gz'), ('md5', '3eced0ffa43e5f9978aa9f83a0e13562')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Perl::Unsafe::Signals'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Perl::Unsafe::Signals'), ('license', 'Perl'), ('summary', 'Allow unsafe handling of signals in selected blocks')]))])), 'perl-perl-version': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perl-version/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-perl-version'), ('version', '1.013')])), ('source', CommentedMap([('fn', 'Perl-Version-1.013.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/B/BD/BDFOY/Perl-Version-1.013.tar.gz'), ('md5', '8955a7292d1039f01d76272464812b0e')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-file-slurp-tiny', 'perl-extutils-makemaker', 'perl-getopt-long']), ('run', ['perl-threaded', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-file-slurp-tiny', 'perl-getopt-long'])])), ('test', CommentedMap([('imports', ['Perl::Version'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Perl::Version'), ('license', 'perl_5'), ('summary', 'Parse and manipulate Perl version strings')]))])), 'perl-perlio-gzip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perlio-gzip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-perlio-gzip'), ('version', '0.19')])), ('source', CommentedMap([('fn', 'PerlIO-gzip-0.19.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NW/NWCLARK/PerlIO-gzip-0.19.tar.gz'), ('md5', 'dbcfc1450f6b593b65048b8ced061c98')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'zlib']), ('run', ['perl', 'zlib'])])), ('test', CommentedMap([('imports', ['PerlIO::gzip'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO-gzip'), ('license', 'perl_5'), ('summary', 'PerlIO interface to gzip/gunzip')]))])), 'perl-perlio-utf8_strict': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perlio-utf8_strict/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-perlio-utf8_strict'), ('version', '0.006')])), ('source', CommentedMap([('fn', 'PerlIO-utf8_strict-0.006.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.006.tar.gz'), ('md5', '9bd26d4cf55f41d9e17f03e644cc9a1b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'perl-test-simple', 'perl-test-exception', 'perl-apache-test', 'perl-extutils-makemaker', 'perl-pathtools']), ('run', ['libgcc', 'perl-threaded', 'perl-apache-test'])])), ('test', CommentedMap([('imports', ['PerlIO::utf8_strict'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO::utf8_strict'), ('license', 'perl_5'), ('summary', 'Fast and correct UTF-8 IO')]))])), 'perl-pod-elemental': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-elemental/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-pod-elemental'), ('version', '0.103004')])), ('source', CommentedMap([('fn', 'Pod-Elemental-0.103004.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Pod-Elemental-0.103004.tar.gz'), ('md5', '15458de0d87404ee94285633acadec9f')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-moosex-types', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-moose', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-moosex-types', 'perl-moose', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Pod::Elemental', 'Pod::Elemental::Autoblank', 'Pod::Elemental::Autochomp', 'Pod::Elemental::Command', 'Pod::Elemental::Document', 'Pod::Elemental::Element::Generic::Blank', 'Pod::Elemental::Element::Generic::Command', 'Pod::Elemental::Element::Generic::Nonpod', 'Pod::Elemental::Element::Generic::Text', 'Pod::Elemental::Element::Nested', 'Pod::Elemental::Element::Pod5::Command', 'Pod::Elemental::Element::Pod5::Data', 'Pod::Elemental::Element::Pod5::Nonpod', 'Pod::Elemental::Element::Pod5::Ordinary', 'Pod::Elemental::Element::Pod5::Region', 'Pod::Elemental::Element::Pod5::Verbatim', 'Pod::Elemental::Flat', 'Pod::Elemental::Node', 'Pod::Elemental::Objectifier', 'Pod::Elemental::Paragraph', 'Pod::Elemental::Selectors', 'Pod::Elemental::Transformer', 'Pod::Elemental::Transformer::Gatherer', 'Pod::Elemental::Transformer::Nester', 'Pod::Elemental::Transformer::Pod5', 'Pod::Elemental::Types'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Pod-Elemental'), ('license', 'perl_5'), ('summary', 'work with nestable Pod elements')]))])), 'perl-pod-usage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-usage/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Usage-1.69.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/MA/MAREKR/Pod-Usage-1.69.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp', 'perl-getopt-long']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp', 'perl-getopt-long'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Usage\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~marekr/Pod-Usage-1.69/'), ('license', 'perl_5'), ('summary', 'print a usage message from embedded pod documentation ')]))])), 'perl-postscript': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-postscript/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-postscript'), ('version', '0.06')])), ('source', CommentedMap([('fn', 'PostScript-0.06.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SH/SHAWNPW/PostScript-0.06.tar.gz'), ('md5', '2a91cc23f2d8958e063f7ff09163faa0'), ('patches', ['elements-version.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['PostScript::Document', 'PostScript::Elements', 'PostScript::Metrics', 'PostScript::TextBlock'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PostScript'), ('license', 'unknown'), ('summary', 'helper module for PostScript::TextBlock')]))])), 'perl-proc-fork': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-proc-fork/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-proc-fork'), ('version', '0.804')])), ('source', CommentedMap([('fn', 'Proc-Fork-0.804.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AR/ARISTOTLE/Proc-Fork-0.804.tar.gz'), ('md5', '0fd512eb21f365a11d9ff7de3b77a624')])), ('requirements', CommentedMap([('build', ['perl-threaded >=5.22.0', 'perl-exporter-tidy']), ('run', ['perl-threaded >=5.22.0', 'perl-exporter-tidy'])])), ('test', CommentedMap([('imports', ['Proc::Fork'])])), ('about', CommentedMap([('home', 'http://github.com/ap/Proc-Fork'), ('license', 'perl_5'), ('summary', 'simple, intuitive interface to the fork() system call')]))])), 'perl-readonly': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-readonly/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-readonly'), ('version', 1.04)])), ('source', CommentedMap([('fn', 'Readonly-1.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SA/SANKO/Readonly-1.04.tar.gz'), ('md5', '875c911ed43d0ecf2c5abed60ee6ae18')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-cpan-meta']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'https://github.com/sanko/readonly'), ('license', 'perl_5'), ('summary', 'Facility for creating read-only scalars, arrays, hashes')]))])), 'perl-regexp-common': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-regexp-common/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-regexp-common'), ('version', '2016060801')])), ('source', CommentedMap([('fn', 'Regexp-Common-2016060801.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AB/ABIGAIL/Regexp-Common-2016060801.tar.gz'), ('md5', '566c7c851b6a1cbf1a82a5d3bc7d5d11')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-extutils-makemaker'])])), ('test', CommentedMap([('imports', ['Regexp::Common', 'Regexp::Common::CC', 'Regexp::Common::balanced', 'Regexp::Common::comment', 'Regexp::Common::delimited', 'Regexp::Common::lingua', 'Regexp::Common::list', 'Regexp::Common::net', 'Regexp::Common::number', 'Regexp::Common::profanity', 'Regexp::Common::whitespace', 'Regexp::Common::zip'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Regexp-Common'), ('license', 'mit'), ('summary', 'Provide commonly requested regular expressions')]))])), 'perl-role-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-role-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-role-tiny'), ('version', '2.000001')])), ('source', CommentedMap([('fn', 'Role-Tiny-2.000001.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/H/HA/HAARG/Role-Tiny-2.000001.tar.gz'), ('md5', 'f350f1f8c13652bf85da172380b39ec8')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-fatal']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Role::Tiny', 'Role::Tiny::With'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Role-Tiny'), ('license', 'perl_5'), ('summary', 'Roles. Like a nouvelle cuisine portion size slice of Moose.')]))])), 'perl-sanger-cgp-allelecount': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sanger-cgp-allelecount/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sanger-cgp-allelecount'), ('version', '2.1.2')])), ('source', CommentedMap([('fn', 'alleleCount-v2.1.2.tar.gz'), ('url', 'https://github.com/cancerit/alleleCount/archive/v2.1.2.tar.gz')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-sanger-cgp-vcf', 'perl-bio-db-sam', 'perl-bioperl']), ('run', ['libgcc', 'perl-threaded', 'perl-sanger-cgp-vcf', 'perl-bio-db-sam', 'perl-bioperl'])])), ('test', CommentedMap([('imports', ['Sanger::CGP::AlleleCount'])])), ('about', CommentedMap([('home', 'https://github.com/cancerit/alleleCount'), ('license', 'GPLv3'), ('summary', 'Support code for NGS copy number algorithm')]))])), 'perl-sanger-cgp-battenberg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sanger-cgp-battenberg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sanger-cgp-battenberg'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'cgpBattenberg-1.4.1_3.tar.gz'), ('url', 'https://github.com/chapmanb/cgpBattenberg/archive/a4fec2c.tar.gz')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-file-sharedir', 'perl-file-sharedir-install', 'perl-pcap', 'perl-sanger-cgp-allelecount', 'cancerit-allelecount', 'impute2']), ('run', ['libgcc', 'perl-threaded', 'perl-file-sharedir', 'perl-file-sharedir-install', 'perl-pcap', 'perl-sanger-cgp-allelecount', 'cancerit-allelecount', 'impute2'])])), ('test', CommentedMap([('imports', ['Sanger::CGP::Battenberg']), ('commands', ['battenberg.pl 2>&1 | grep battenberg.pl'])])), ('about', CommentedMap([('home', 'https://github.com/cancerit/cgpBattenberg'), ('license', 'GPLv3'), ('summary', 'detect subclonality and copy number in matched NGS data')]))])), 'perl-sanger-cgp-vagrent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sanger-cgp-vagrent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sanger-cgp-vagrent'), ('version', '3.2.0')])), ('source', CommentedMap([('fn', 'v3.2.0.tar.gz'), ('url', 'https://github.com/cancerit/VAGrENT/archive/v3.2.0.tar.gz'), ('md5', '09996ac5a3fff126f10e163b5bd32ecf')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-file-sharedir-install', 'perl-bio-db-hts']), ('run', ['libgcc', 'perl-threaded', 'perl-bio-db-hts'])])), ('test', CommentedMap([('imports', ['Sanger::CGP::Vagrent'])])), ('about', CommentedMap([('home', 'https://github.com/cancerit/VAGrENT'), ('license', 'GPLv3'), ('summary', 'A toolset for comparing genomic variants to reference genome annotation to identify potential biological consequences')]))])), 'perl-sanger-cgp-vcf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sanger-cgp-vcf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sanger-cgp-vcf'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'cgpVcf-v1.3.1.tar.gz'), ('url', 'https://github.com/cancerit/cgpVcf/archive/v1.3.1.tar.gz')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-data-uuid', 'perl-vcftools-vcf']), ('run', ['libgcc', 'perl-threaded', 'perl-data-uuid', 'perl-vcftools-vcf'])])), ('test', CommentedMap([('imports', ['Sanger::CGP::Vcf'])])), ('about', CommentedMap([('home', 'https://github.com/cancerit/cgpVcf'), ('license', 'GPLv3'), ('summary', 'a set of common perl utilities for generating consistent Vcf headers')]))])), 'perl-scalar-util-numeric': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-scalar-util-numeric/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-scalar-util-numeric'), ('version', '0.40')])), ('source', CommentedMap([('fn', 'Scalar-Util-Numeric-0.40.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CH/CHOCOLATE/Scalar-Util-Numeric-0.40.tar.gz'), ('md5', '6a5451f33c3ec5c17d0672a4d5b0d24c')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Scalar::Util::Numeric'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scalar-Util-Numeric'), ('license', 'perl_5'), ('summary', 'numeric tests for perl scalars')]))])), 'perl-scope-guard': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-scope-guard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-scope-guard'), ('version', '0.21')])), ('source', CommentedMap([('fn', 'Scope-Guard-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CH/CHOCOLATE/Scope-Guard-0.21.tar.gz'), ('md5', 'be57b915d23ddac7677ef2ad9e52b92a')])), ('build', None), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Scope::Guard'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scope::Guard'), ('license', 'perl_5'), ('summary', 'lexically-scoped resource management')]))])), 'perl-sereal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sereal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sereal'), ('version', '3.015')])), ('source', CommentedMap([('fn', 'Sereal-3.015.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/Y/YV/YVES/Sereal-3.015.tar.gz'), ('md5', '732fc663a7ffb9087bf8dbebe29f21bd')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-pathtools', 'perl-test-warn', 'perl-sereal-decoder', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-extutils-makemaker', 'perl-sereal-encoder', 'perl-test-longstring']), ('run', ['perl-threaded', 'perl-sereal-decoder', 'perl-sereal-encoder'])])), ('test', CommentedMap([('imports', ['Sereal'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sereal'), ('license', 'perl_5'), ('summary', 'Fast, compact, powerful binary (de-)serialization')]))])), 'perl-sereal-decoder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sereal-decoder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sereal-decoder'), ('version', '3.015')])), ('source', CommentedMap([('fn', 'Sereal-Decoder-3.015.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/Y/YV/YVES/Sereal-Decoder-3.015.tar.gz'), ('md5', 'de91784e359e25aee07f87a8b9b2f90b')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-file-path', 'perl-test-warn', 'perl-test-simple', 'perl-extutils-makemaker', 'perl-test-longstring', 'perl-scalar-list-utils', 'perl-pathtools', 'perl-extutils-parsexs']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Sereal::Decoder', 'Sereal::Decoder::Constants', 'Sereal::Performance'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sereal::Decoder'), ('license', 'perl_5'), ('summary', 'Fast, compact, powerful binary deserialization')]))])), 'perl-sereal-encoder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sereal-encoder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sereal-encoder'), ('version', '3.015')])), ('source', CommentedMap([('fn', 'Sereal-Encoder-3.015.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/Y/YV/YVES/Sereal-Encoder-3.015.tar.gz'), ('md5', '78428860e055561e371e3292093c1fc5')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-scalar-list-utils', 'perl-extutils-makemaker', 'perl-file-path', 'perl-pathtools', 'perl-test-longstring', 'perl-extutils-parsexs', 'perl-test-warn', 'perl-test-simple', 'perl-sereal-decoder']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Sereal::Encoder', 'Sereal::Encoder::Constants'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sereal::Encoder'), ('license', 'perl_5'), ('summary', 'Fast, compact, powerful binary serialization')]))])), 'perl-set-intervaltree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-set-intervaltree/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-set-intervaltree'), ('version', 0.1)])), ('source', CommentedMap([('fn', 'Set-IntervalTree-0.10.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/B/BE/BENBOOTH/Set-IntervalTree-0.10.tar.gz'), ('sha256', 'e3bd9ccf0d074b5f879eef1ed88254983697bf83d02744fce62150ee46553ebc')])), ('build', CommentedMap([('number', 4)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('imports', ['Set::IntervalTree'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Set::IntervalTree'), ('license', 'perl_5'), ('summary', 'An interval tree implementation in PERL.')]))])), 'perl-set-intspan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-set-intspan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-set-intspan'), ('version', '1.19')])), ('source', CommentedMap([('fn', 'Set-IntSpan-1.19.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SW/SWMCD/Set-IntSpan-1.19.tar.gz'), ('md5', '7c21c2fedce5b7be7d8200e2cf2bdca1')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Set::IntSpan'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Set-IntSpan'), ('license', 'unknown'), ('summary', 'Manages sets of integers, newsrc style')]))])), 'perl-set-scalar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-set-scalar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-set-scalar'), ('version', '1.29')])), ('source', CommentedMap([('fn', 'Set-Scalar-1.29.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DA/DAVIDO/Set-Scalar-1.29.tar.gz'), ('md5', '4af73c0d6f562d1eddd4ebf6014ac67f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Set::Scalar', 'Set::Scalar::Base', 'Set::Scalar::Null', 'Set::Scalar::Real', 'Set::Scalar::Universe', 'Set::Scalar::Valued', 'Set::Scalar::ValuedUniverse', 'Set::Scalar::Virtual'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Set-Scalar'), ('license', 'perl_5'), ('summary', 'basic set operations')]))])), 'perl-soap-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-soap-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-soap-lite'), ('version', '1.19')])), ('source', CommentedMap([('fn', 'SOAP-Lite-1.19.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PH/PHRED/SOAP-Lite-1.19.tar.gz'), ('md5', 'eceb84ca5aabebf855e37c53f53d39ce')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-mime-lite', 'perl-mime-tools', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-test-warn', 'perl-test-xml', 'perl-xml-parser-lite']), ('run', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-mime-tools', 'perl-mime-lite'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/SOAP-Lite'), ('license', 'perl_5'), ('summary', \"Perl's Web Services Toolkit\")]))])), 'perl-socket6': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-socket6/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-socket6'), ('version', '0.25')])), ('source', CommentedMap([('fn', 'Socket6-0.25.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/U/UM/UMEMOTO/Socket6-0.25.tar.gz'), ('md5', 'e6c40d662b1fc5ffd436b7f50daa1f04')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Socket6'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Socket6'), ('license', 'unknown'), ('summary', 'IPv6 related part of the C socket.h defines and structure manipulators')]))])), 'perl-sort-naturally': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sort-naturally/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sort-naturally'), ('version', '1.03')])), ('source', CommentedMap([('fn', 'Sort-Naturally-1.03.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/B/BI/BINGOS/Sort-Naturally-1.03.tar.gz'), ('md5', 'fef2e36d1b1ea23ee79ec68b5872453f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Sort::Naturally'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sort-Naturally'), ('license', 'perl_5'), ('summary', 'sort lexically, but sort numeral parts numerically')]))])), 'perl-sort-versions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sort-versions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sort-versions'), ('version', '1.62')])), ('source', CommentedMap([('fn', 'Sort-Versions-1.62.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Sort-Versions-1.62.tar.gz'), ('md5', 'c975afa9dd114951d902aa4a81ead685')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-test-simple', 'perl-apache-test']), ('run', ['perl-threaded', 'perl-apache-test'])])), ('test', CommentedMap([('imports', ['Sort::Versions'])])), ('about', CommentedMap([('home', 'https://github.com/neilb/Sort-Versions'), ('license', 'perl_5'), ('summary', 'a perl 5 module for sorting of revision-like numbers')]))])), 'perl-spiffy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-spiffy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-spiffy'), ('version', '0.46')])), ('source', CommentedMap([('fn', 'Spiffy-0.46.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/I/IN/INGY/Spiffy-0.46.tar.gz'), ('md5', '2ec7007c111a6bc2775bb7c57673a24b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Spiffy'])])), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/spiffy-pm'), ('license', 'perl_5'), ('summary', 'Spiffy Perl Interface Framework For You')]))])), 'perl-spreadsheet-parseexcel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-spreadsheet-parseexcel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-spreadsheet-parseexcel'), ('version', '0.65')])), ('source', CommentedMap([('fn', 'Spreadsheet-ParseExcel-0.65.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/D/DO/DOUGW/Spreadsheet-ParseExcel-0.65.tar.gz'), ('md5', '4b8857e3a391d86501c1b742b459ac9e')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-crypt-rc4', 'perl-digest-perl-md5', 'perl-io-stringy', 'perl-ole-storage_lite', 'perl-jcode', 'perl-spreadsheet-writeexcel', 'perl-unicode-map']), ('run', ['perl-threaded', 'perl-crypt-rc4', 'perl-digest-perl-md5', 'perl-io-stringy', 'perl-ole-storage_lite', 'perl-jcode', 'perl-spreadsheet-writeexcel', 'perl-unicode-map'])])), ('test', CommentedMap([('imports', ['Spreadsheet::ParseExcel', 'Spreadsheet::ParseExcel::Cell', 'Spreadsheet::ParseExcel::Dump', 'Spreadsheet::ParseExcel::FmtDefault', 'Spreadsheet::ParseExcel::FmtJapan', 'Spreadsheet::ParseExcel::FmtJapan2', 'Spreadsheet::ParseExcel::FmtUnicode', 'Spreadsheet::ParseExcel::Font', 'Spreadsheet::ParseExcel::Format', 'Spreadsheet::ParseExcel::SaveParser', 'Spreadsheet::ParseExcel::SaveParser::Workbook', 'Spreadsheet::ParseExcel::SaveParser::Worksheet', 'Spreadsheet::ParseExcel::Utility', 'Spreadsheet::ParseExcel::Workbook', 'Spreadsheet::ParseExcel::Worksheet'])])), ('about', CommentedMap([('home', 'http://github.com/runrig/spreadsheet-parseexcel/'), ('license', 'perl_5'), ('summary', 'Read information from an Excel file.')]))])), 'perl-spreadsheet-writeexcel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-spreadsheet-writeexcel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-spreadsheet-writeexcel'), ('version', '2.40')])), ('source', CommentedMap([('fn', 'Spreadsheet-WriteExcel-2.40.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/J/JM/JMCNAMARA/Spreadsheet-WriteExcel-2.40.tar.gz'), ('md5', 'b3262290e96985ee12d81c861a389790')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-parse-recdescent', 'perl-ole-storage_lite']), ('run', ['perl-threaded', 'perl-parse-recdescent', 'perl-ole-storage_lite'])])), ('test', CommentedMap([('imports', ['Spreadsheet::WriteExcel', 'Spreadsheet::WriteExcel::BIFFwriter', 'Spreadsheet::WriteExcel::Big', 'Spreadsheet::WriteExcel::Chart', 'Spreadsheet::WriteExcel::Chart::Area', 'Spreadsheet::WriteExcel::Chart::Bar', 'Spreadsheet::WriteExcel::Chart::Column', 'Spreadsheet::WriteExcel::Chart::External', 'Spreadsheet::WriteExcel::Chart::Line', 'Spreadsheet::WriteExcel::Chart::Pie', 'Spreadsheet::WriteExcel::Chart::Scatter', 'Spreadsheet::WriteExcel::Chart::Stock', 'Spreadsheet::WriteExcel::Examples', 'Spreadsheet::WriteExcel::Format', 'Spreadsheet::WriteExcel::Formula', 'Spreadsheet::WriteExcel::OLEwriter', 'Spreadsheet::WriteExcel::Properties', 'Spreadsheet::WriteExcel::Utility', 'Spreadsheet::WriteExcel::Workbook', 'Spreadsheet::WriteExcel::Worksheet'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Spreadsheet-WriteExcel'), ('license', 'perl_5'), ('summary', 'Write to a cross platform Excel binary file')]))])), 'perl-sql-statement': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sql-statement/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sql-statement'), ('version', '1.407')])), ('source', CommentedMap([('fn', 'SQL-Statement-1.407.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RE/REHSACK/SQL-Statement-1.407.tar.gz'), ('md5', 'abeedda4a2c085613cb77fb01631e5de')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-clone', 'perl-math-base-convert', 'perl-module-runtime', 'perl-params-util', 'perl-text-soundex', 'perl-test-deep']), ('run', ['perl-threaded', 'perl-clone', 'perl-math-base-convert', 'perl-module-runtime', 'perl-params-util', 'perl-text-soundex'])])), ('test', CommentedMap([('imports', ['SQL::Dialects::ANSI', 'SQL::Dialects::AnyData', 'SQL::Dialects::CSV', 'SQL::Dialects::Role', 'SQL::Eval', 'SQL::Parser', 'SQL::Statement', 'SQL::Statement::Function', 'SQL::Statement::Functions', 'SQL::Statement::GetInfo', 'SQL::Statement::Operation', 'SQL::Statement::Placeholder', 'SQL::Statement::RAM', 'SQL::Statement::Term', 'SQL::Statement::TermFactory', 'SQL::Statement::Util'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/SQL-Statement'), ('license', 'perl_5'), ('summary', 'SQL parsing and processing engine')]))])), 'perl-statistics-basic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-basic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-basic'), ('version', '1.6611')])), ('source', CommentedMap([('fn', 'Statistics-Basic-1.6611.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JE/JETTERO/Statistics-Basic-1.6611.tar.gz'), ('md5', '1ee961c3a4b9c0a594dc3cfccbcd1ed1')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-number-format', 'perl-scalar-list-utils']), ('run', ['perl-threaded', 'perl-number-format', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Statistics::Basic', 'Statistics::Basic::ComputedVector', 'Statistics::Basic::Correlation', 'Statistics::Basic::Covariance', 'Statistics::Basic::LeastSquareFit', 'Statistics::Basic::Mean', 'Statistics::Basic::Median', 'Statistics::Basic::Mode', 'Statistics::Basic::StdDev', 'Statistics::Basic::Variance', 'Statistics::Basic::Vector'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Statistics-Basic'), ('license', 'open_source'), ('summary', '')]))])), 'perl-statistics-descriptive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-descriptive/3.0609/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-descriptive'), ('version', '3.0609')])), ('source', CommentedMap([('fn', 'Statistics-Descriptive-3.0609.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/Statistics-Descriptive-3.0609.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Statistics::Descriptive'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Statistics::Descriptive'), ('license', 'Perl'), ('summary', 'Module of basic descriptive statistical functions')]))])), 'perl-statistics-distributions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-distributions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-distributions'), ('version', '1.02')])), ('source', CommentedMap([('fn', 'Statistics-Distributions-1.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MI/MIKEK/Statistics-Distributions-1.02.tar.gz'), ('md5', '526cb415b30ffeec52563dbcb436ca60')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Statistics::Distributions'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Statistics-Distributions'), ('license', 'perl_5'), ('summary', 'Perl module for calculating critical values and upper probabilities of common statistical distributions')]))])), 'perl-statistics-frequency': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-frequency/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-frequency'), ('version', 0.04)])), ('source', CommentedMap([('fn', 'Statistics-Frequency-0.04.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JH/JHI/Statistics-Frequency-0.04.tar.gz'), ('md5', 'e1ecb945b642d574d3708d6ef5d0be0c')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Statistics::Frequency'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Statistics::Frequency'), ('license', 'perl_5'), ('summary', 'simple counting of elements')]))])), 'perl-statistics-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-lite'), ('version', '3.62')])), ('source', CommentedMap([('fn', 'Statistics-Lite-3.62.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BR/BRIANL/Statistics-Lite-3.62.tar.gz'), ('md5', '63bb672beb9463e35d1abce850508852')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Statistics::Lite'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Statistics-Lite'), ('license', 'perl_5'), ('summary', 'Small stats stuff.')]))])), 'perl-statistics-r': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-r/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-r'), ('version', '0.34')])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Statistics-R/'), ('license', 'Perl5'), ('summary', 'Statistics::R - Perl interface with the R statistical program')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'Statistics-R-0.34.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/F/FA/FANGLY/Statistics-R-0.34.tar.gz'), ('sha256', '782dd064876ac94680d97899f24fb0e727df42c05ba474ec096a9116438fbed4')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-ipc-run', 'perl-regexp-common', 'perl-text-balanced >=1.97', 'perl-text-wrap >=0.77', 'r-essentials', 'gcc']), ('run', ['perl-threaded', 'perl-ipc-run', 'perl-regexp-common', 'perl-text-balanced >=1.97', 'perl-text-wrap >=0.77', 'r-essentials', 'libgcc', 'gcc'])])), ('test', CommentedMap([('imports', ['Statistics::R'])]))])), 'perl-statistics-ttest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-statistics-ttest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-statistics-ttest'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'Statistics-TTest-1.1.0.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/Y/YU/YUNFANG/Statistics-TTest-1.1.0.tar.gz'), ('md5', '605466db48e2b16063abd16550e7d345')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-statistics-distributions', 'perl-statistics-descriptive']), ('run', ['perl-threaded', 'perl-statistics-distributions', 'perl-statistics-descriptive'])])), ('test', CommentedMap([('imports', ['Statistics::TTest', 'Statistics::PointEstimation'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Statistics-TTest'), ('license', 'perl_5'), ('summary', 'Perl module to perform T-test on 2 independent samples Statistics::TTest::Sufficient - Perl module to perfrom T-Test on 2 indepdent samples using sufficient statistics')]))])), 'perl-sub-exporter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-exporter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sub-exporter'), ('version', '0.987')])), ('source', CommentedMap([('fn', 'Sub-Exporter-0.987.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RJ/RJBS/Sub-Exporter-0.987.tar.gz'), ('md5', '5332d269a7ba387773fcd140b72a0ed2')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-params-util', 'perl-apache-test', 'perl-extutils-makemaker', 'perl-pathtools', 'perl-test-simple', 'perl-data-optlist', 'perl-sub-install']), ('run', ['perl-threaded', 'perl-apache-test', 'perl-params-util', 'perl-data-optlist', 'perl-sub-install'])])), ('test', CommentedMap([('imports', ['Sub::Exporter', 'Sub::Exporter::Util'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Sub-Exporter'), ('license', 'perl_5'), ('summary', 'a sophisticated exporter for custom-built routines')]))])), 'perl-sub-exporter-progressive': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-exporter-progressive/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sub-exporter-progressive'), ('version', '0.001011')])), ('source', CommentedMap([('fn', 'Sub-Exporter-Progressive-0.001011.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/F/FR/FREW/Sub-Exporter-Progressive-0.001011.tar.gz'), ('md5', 'bb50b3ba1538902b197c04818a84230a')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Sub::Exporter::Progressive'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Sub-Exporter-Progressive/'), ('license', 'perl_5'), ('summary', 'Only use Sub::Exporter if you need it')]))])), 'perl-sub-install': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-install/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sub-install'), ('version', '0.928')])), ('source', CommentedMap([('fn', 'Sub-Install-0.928.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RJ/RJBS/Sub-Install-0.928.tar.gz'), ('md5', 'e1ce4f9cb6b2f6b8778b036c31afa5ab')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-apache-test', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-apache-test', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Sub::Install'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Sub-Install'), ('license', 'perl_5'), ('summary', 'install subroutines into packages easily')]))])), 'perl-sub-uplevel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-uplevel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-sub-uplevel'), ('version', '0.25')])), ('source', CommentedMap([('fn', 'Sub-Uplevel-0.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Sub-Uplevel-0.25.tar.gz'), ('md5', '5d0752dbfa94d0c91b25a264f47f5675')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Sub::Uplevel'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Sub-Uplevel'), ('license', 'perl_5'), ('summary', 'apparently run a function in a higher stack frame')]))])), 'perl-svg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-svg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-svg'), ('version', '2.64')])), ('source', CommentedMap([('fn', 'SVG-2.64.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SZ/SZABGAB/SVG-2.64.tar.gz'), ('md5', 'ba4445ee72e2f3c29da332cb0119a2ad')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['SVG', 'SVG::DOM', 'SVG::Element', 'SVG::Extension', 'SVG::XML'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/SVG'), ('license', 'perl_5'), ('summary', 'Perl extension for generating Scalable Vector Graphics (SVG) documents')]))])), 'perl-svg-graph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-svg-graph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-svg-graph'), ('version', '0.02')])), ('source', CommentedMap([('fn', 'SVG-Graph-0.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AL/ALLENDAY/SVG-Graph-0.02.tar.gz'), ('md5', 'e922f0b323056ecf306eb8085db283f5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-tree-dag_node', 'perl-math-derivative', 'perl-math-spline', 'perl-svg', 'perl-statistics-descriptive']), ('run', ['perl-threaded', 'perl-tree-dag_node', 'perl-math-spline', 'perl-math-derivative', 'perl-svg', 'perl-statistics-descriptive'])])), ('test', CommentedMap([('imports', ['SVG::Graph', 'SVG::Graph::Data', 'SVG::Graph::Data::Datum', 'SVG::Graph::Data::Node', 'SVG::Graph::Data::Tree', 'SVG::Graph::File', 'SVG::Graph::Frame', 'SVG::Graph::Glyph', 'SVG::Graph::Glyph::axis', 'SVG::Graph::Glyph::bar', 'SVG::Graph::Glyph::barflex', 'SVG::Graph::Glyph::bezier', 'SVG::Graph::Glyph::bubble', 'SVG::Graph::Glyph::heatmap', 'SVG::Graph::Glyph::line', 'SVG::Graph::Glyph::pictogram', 'SVG::Graph::Glyph::scatter', 'SVG::Graph::Glyph::tree', 'SVG::Graph::Glyph::wedge'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/SVG-Graph'), ('license', 'unknown'), ('summary', 'Visualize your data in Scalable Vector Graphics (SVG) format.')]))])), 'perl-task-weaken': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-task-weaken/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-task-weaken'), ('version', '1.04')])), ('source', CommentedMap([('fn', 'Task-Weaken-1.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Task-Weaken-1.04.tar.gz'), ('md5', 'affd0c395515bb95d29968404d7fe6de')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Task::Weaken'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Task-Weaken'), ('license', 'perl_5'), ('summary', 'Ensure that a platform has weaken support')]))])), 'perl-template-toolkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-template-toolkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-template-toolkit'), ('version', '2.26')])), ('source', CommentedMap([('fn', 'Template-Toolkit-2.26.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AB/ABW/Template-Toolkit-2.26.tar.gz'), ('md5', '4c289856ebee446ebbe2629b24f0734b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-appconfig', 'perl-cgi', 'perl-image-info', 'perl-image-size', 'perl-test-leaktrace']), ('run', ['perl-threaded', 'perl-appconfig', 'perl-cgi', 'perl-image-info', 'perl-image-size', 'perl-test-leaktrace'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Template-Toolkit'), ('license', 'perl_5'), ('summary', 'comprehensive template processing system')]))])), 'perl-term-app-roles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-term-app-roles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-term-app-roles'), ('version', '0.01')])), ('source', CommentedMap([('fn', 'Term-App-Roles-0.01.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Term-App-Roles-0.01.tar.gz'), ('md5', '975376a638b180738cb0bb32d8af3f78')])), ('requirements', CommentedMap([('build', ['perl', 'perl-pathtools', 'perl-extutils-makemaker', 'perl-term-detect-software', 'perl-moo', 'perl-test-simple']), ('run', ['perl', 'perl-moo', 'perl-term-detect-software'])])), ('test', CommentedMap([('imports', ['Term::App::Role::Attrs', 'Term::App::Roles'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Term-App-Roles'), ('license', 'perl_5'), ('summary', 'Collection of roles for terminal-based application')]))])), 'perl-term-detect-software': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-term-detect-software/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-term-detect-software'), ('version', '0.21')])), ('source', CommentedMap([('fn', 'Term-Detect-Software-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Term-Detect-Software-0.21.tar.gz'), ('md5', 'acfe8c62d87ee4ce04f3c8ccc0b1a9a0')])), ('requirements', CommentedMap([('build', ['perl', 'perl-file-which', 'perl-perl-osnames', 'perl-extutils-makemaker', 'perl-pathtools', 'perl-test-simple']), ('run', ['perl', 'perl-file-which'])])), ('test', CommentedMap([('imports', ['Term::Detect::Software'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Term-Detect-Software'), ('license', 'perl_5'), ('summary', 'Detect terminal (emulator) software and its capabilities')]))])), 'perl-term-encoding': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-term-encoding/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-term-encoding'), ('version', '0.02')])), ('source', CommentedMap([('fn', 'Term-Encoding-0.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MI/MIYAGAWA/Term-Encoding-0.02.tar.gz'), ('md5', '67b7fb38ec7779a3fb79a047fd7edbb5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Term::Encoding'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Term::Encoding'), ('license', 'perl_5'), ('summary', 'Detect encoding of the current terminal')]))])), 'perl-term-progressbar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-term-progressbar/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Term-ProgressBar-2.21.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/MA/MANWAR/Term-ProgressBar-2.21.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp', 'perl-termreadkey', 'perl-class-methodmaker', 'perl-capture-tiny', 'perl-test-exception']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp', 'perl-termreadkey', 'perl-class-methodmaker', 'perl-capture-tiny', 'perl-test-exception'])])), ('test', CommentedMap([('imports', ['Term::ProgressBar'])])), ('about', CommentedMap([('home', 'https://github.com/manwar/Term-ProgressBar'), ('license', 'perl_5'), ('summary', 'A progress bar for things that take a while')]))])), 'perl-termreadkey': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-termreadkey/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-termreadkey'), ('version', '2.32')])), ('source', CommentedMap([('fn', 'TermReadKey-2.32.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JS/JSTOWE/TermReadKey-2.32.tar.gz'), ('md5', 'c55420c3456b8e470a01d6557b1045a8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('imports', ['Term::ReadKey'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TermReadKey'), ('license', 'perl_5'), ('summary', 'A perl module for simple terminal control')]))])), 'perl-test-base': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-base/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-base'), ('version', '0.88')])), ('source', CommentedMap([('fn', 'Test-Base-0.88.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/I/IN/INGY/Test-Base-0.88.tar.gz'), ('md5', 'ced0cd86b099f9fd2cd73df1f46ab5e9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-spiffy', 'perl-text-diff', 'perl-test-deep']), ('run', ['perl-threaded', 'perl-spiffy', 'perl-text-diff'])])), ('test', CommentedMap([('imports', ['Test::Base::Filter']), ('commands', [\"perl -e 'use Test::Base tests => 1; pass;'\"])])), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/test-base-pm'), ('license', 'perl_5'), ('summary', 'A Data Driven Testing Framework')]))])), 'perl-test-builder-tester': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-builder-tester/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-builder-tester'), ('version', '1.23_002')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Builder::Tester'), ('license', 'unknown'), ('summary', 'test testsuites that have been built with Test::Builder')]))])), 'perl-test-classapi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-classapi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-classapi'), ('version', '1.06')])), ('source', CommentedMap([('fn', 'Test-ClassAPI-1.06.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Test-ClassAPI-1.06.tar.gz'), ('md5', 'ee31a9e1ecedcf720a9e89461a83442e')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-class-inspector', 'perl-params-util', 'perl-pathtools', 'perl-test-simple', 'perl-config-tiny']), ('run', ['perl-threaded', 'perl-test-simple', 'perl-config-tiny', 'perl-class-inspector', 'perl-params-util', 'perl-pathtools'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::ClassAPI'), ('license', 'perl_5'), ('summary', 'Provides basic first-pass API testing for large class trees')]))])), 'perl-test-cpan-meta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-cpan-meta/0.25/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-CPAN-Meta-0.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BA/BARBIE/Test-CPAN-Meta-0.25.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::CPAN::Meta\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::CPAN::Meta'), ('license', 'artistic_2'), ('summary', 'Validate your CPAN META.json files')]))])), 'perl-test-deep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-deep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-deep'), ('version', '1.120')])), ('source', CommentedMap([('fn', 'Test-Deep-1.120.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Test-Deep-1.120.tar.gz'), ('md5', 'a90859f4f362578d1671e3b9aae0d8c1')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['Test::Deep', 'Test::Deep::All', 'Test::Deep::Any', 'Test::Deep::Array', 'Test::Deep::ArrayEach', 'Test::Deep::ArrayElementsOnly', 'Test::Deep::ArrayLength', 'Test::Deep::ArrayLengthOnly', 'Test::Deep::Blessed', 'Test::Deep::Boolean', 'Test::Deep::Cache', 'Test::Deep::Cache::Simple', 'Test::Deep::Class', 'Test::Deep::Cmp', 'Test::Deep::Code', 'Test::Deep::Hash', 'Test::Deep::HashEach', 'Test::Deep::HashElements', 'Test::Deep::HashKeys', 'Test::Deep::HashKeysOnly', 'Test::Deep::Ignore', 'Test::Deep::Isa', 'Test::Deep::ListMethods', 'Test::Deep::MM', 'Test::Deep::Methods', 'Test::Deep::NoTest', 'Test::Deep::None', 'Test::Deep::Number', 'Test::Deep::Obj', 'Test::Deep::Ref', 'Test::Deep::RefType', 'Test::Deep::Regexp', 'Test::Deep::RegexpMatches', 'Test::Deep::RegexpOnly', 'Test::Deep::RegexpRef', 'Test::Deep::RegexpRefOnly', 'Test::Deep::RegexpVersion', 'Test::Deep::ScalarRef', 'Test::Deep::ScalarRefOnly', 'Test::Deep::Set', 'Test::Deep::Shallow', 'Test::Deep::Stack', 'Test::Deep::String'])])), ('about', CommentedMap([('home', 'http://github.com/rjbs/Test-Deep/'), ('license', 'perl_5'), ('summary', 'Extremely flexible deep comparison')]))])), 'perl-test-differences': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-differences/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-differences'), ('version', '0.64')])), ('source', CommentedMap([('fn', 'Test-Differences-0.64.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DC/DCANTRELL/Test-Differences-0.64.tar.gz'), ('md5', 'ecfda620fe133e36a6e392d94ab8424d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-capture-tiny', 'perl-text-diff', 'perl-module-build']), ('run', ['perl-threaded', 'perl-capture-tiny', 'perl-text-diff'])])), ('test', CommentedMap([('imports', ['Test::Differences'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-Differences'), ('license', 'Perl'), ('summary', 'Test strings and data structures and show differences if not ok')]))])), 'perl-test-eol': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-eol/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-eol'), ('version', '1.6')])), ('source', CommentedMap([('fn', 'Test-EOL-1.6.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/F/FR/FREW/Test-EOL-1.6.tar.gz'), ('md5', '1255e6ef49c417845aa70441c4b40c90')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Test::EOL'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Test-EOL'), ('license', 'perl_5'), ('summary', 'Check the correct line endings in your project')]))])), 'perl-test-exception': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-exception/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-exception'), ('version', '0.43')])), ('source', CommentedMap([('fn', 'Test-Exception-0.43.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Test-Exception-0.43.tar.gz'), ('md5', '572d355026fb0b87fc2b8c64b83cada0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-sub-uplevel']), ('run', ['perl-threaded', 'perl-sub-uplevel'])])), ('test', CommentedMap([('imports', ['Test::Exception'])])), ('about', CommentedMap([('home', 'https://github.com/Test-More/test-exception'), ('license', 'perl_5'), ('summary', 'Test exception-based code')]))])), 'perl-test-fatal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-fatal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-fatal'), ('version', '0.014')])), ('source', CommentedMap([('fn', 'Test-Fatal-0.014.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Test-Fatal-0.014.tar.gz'), ('md5', '7954f6d2e3607be10c0fbd69063a3d1b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-try-tiny']), ('run', ['perl-threaded', 'perl-try-tiny'])])), ('test', CommentedMap([('imports', ['Test::Fatal'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Test-Fatal'), ('license', 'perl_5'), ('summary', 'incredibly simple helpers for testing code with exceptions')]))])), 'perl-test-files': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-files/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-files'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'Test-Files-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PH/PHILCROW/Test-Files-0.14.tar.gz'), ('md5', '52139ac7bfcd77ebc0469d1dae18b80e')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-builder-tester', 'perl-test-simple', 'perl-algorithm-diff', 'perl-text-diff']), ('run', ['perl-threaded', 'perl-test-builder-tester', 'perl-test-simple', 'perl-algorithm-diff', 'perl-text-diff'])])), ('test', CommentedMap([('imports', ['Test::Files'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Files'), ('license', 'unknown'), ('summary', 'A Test::Builder based module to ease testing with files and dirs')]))])), 'perl-test-harness': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-harness/3.36/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Harness-3.36.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/Test-Harness-3.36.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Harness\"'])])), ('about', CommentedMap([('home', 'http://testanything.org/'), ('license', 'perl_5'), ('summary', 'contributing to TAP::Harness')]))])), 'perl-test-inter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-inter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-inter'), ('version', '1.06')])), ('source', CommentedMap([('fn', 'Test-Inter-1.06.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SB/SBECK/Test-Inter-1.06.tar.gz'), ('md5', '61da056da6dae8ee256d2ccbd2e24a73')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-io-all', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-io-all'])])), ('test', CommentedMap([('imports', ['Test::Inter'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Inter'), ('license', 'perl_5'), ('summary', 'framework for more readable interactive test scripts')]))])), 'perl-test-leaktrace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-leaktrace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-leaktrace'), ('version', '0.15')])), ('source', CommentedMap([('fn', 'Test-LeakTrace-0.15.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GF/GFUJI/Test-LeakTrace-0.15.tar.gz'), ('md5', '5eac99e7882970f41886c3e0cec185d8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Test::LeakTrace', 'Test::LeakTrace::Script'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-LeakTrace'), ('license', 'perl_5'), ('summary', 'Traces memory leaks')]))])), 'perl-test-longstring': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-longstring/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-longstring'), ('version', '0.17')])), ('source', CommentedMap([('fn', 'Test-LongString-0.17.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RG/RGARCIA/Test-LongString-0.17.tar.gz'), ('md5', '8baab770f61ebd5647a9dd3679f42fbd')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-test-builder-tester', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-test-builder-tester', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Test::LongString'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::LongString'), ('license', 'perl_5'), ('summary', 'tests strings for equality, with more helpful failures')]))])), 'perl-test-most': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-most/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-most'), ('version', '0.34')])), ('source', CommentedMap([('fn', 'Test-Most-0.34.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/O/OV/OVID/Test-Most-0.34.tar.gz'), ('md5', '6fa1f097bedbc18f46b08ac9eb79bba9')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-exception-class', 'perl-test-deep', 'perl-test-differences', 'perl-test-exception', 'perl-test-warn']), ('run', ['perl-threaded', 'perl-exception-class', 'perl-test-deep', 'perl-test-differences', 'perl-test-exception', 'perl-test-warn'])])), ('test', CommentedMap([('imports', ['Test::Most', 'Test::Most::Exception'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-Most'), ('license', 'unknown'), ('summary', 'Most commonly needed test functions and features')]))])), 'perl-test-notabs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-notabs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-notabs'), ('version', '1.4')])), ('source', CommentedMap([('fn', 'Test-NoTabs-1.4.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BO/BOBTFISH/Test-NoTabs-1.4.tar.gz'), ('md5', '1b135805eebb93ca502df585655f34ca')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Test::NoTabs'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-NoTabs'), ('license', 'perl_5'), ('summary', 'Check the presence of tabs in your project')]))])), 'perl-test-nowarnings': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-nowarnings/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-nowarnings'), ('version', '1.04')])), ('source', CommentedMap([('fn', 'Test-NoWarnings-1.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/A/AD/ADAMK/Test-NoWarnings-1.04.tar.gz'), ('md5', '682ed043f7d3e38f3dfd8745fd21c49a')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', [\"perl -e 'use Test::More tests => x + 1; use Test::NoWarnings;'\"])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Test::NoWarnings'), ('license', 'Perl'), ('summary', \"Make sure you didn't emit any warnings while testing\")]))])), 'perl-test-output': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-output/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-output'), ('version', '1.03')])), ('source', CommentedMap([('fn', 'Test-Output-1.03.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/B/BD/BDFOY/Test-Output-1.03.tar.gz'), ('md5', '903485edb382f5a18a5029978d1febf6')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker', 'perl-sub-exporter', 'perl-capture-tiny', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-capture-tiny', 'perl-sub-exporter', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Test::Output'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-Output'), ('license', 'perl_5'), ('summary', 'Utilities to test STDOUT and STDERR messages.')]))])), 'perl-test-pod': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-pod/1.51/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Pod-1.51.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Test-Pod-1.51.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Pod\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Test-Pod/'), ('license', 'perl_5'), ('summary', 'check for POD errors in files')]))])), 'perl-test-requires': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-requires/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-requires'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'Test-Requires-0.10.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TOKUHIROM/Test-Requires-0.10.tar.gz'), ('md5', '0d5da779609d0c8fa6f796b45ff8c6f3')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Test::Requires'])])), ('about', CommentedMap([('home', 'https://github.com/tokuhirom/Test-Requires'), ('license', 'perl_5'), ('summary', 'Checks to see if the module can be loaded')]))])), 'perl-test-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-simple/1.302075/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test2']), ('run', ['perl', 'perl-test2'])])), ('test', CommentedMap([('imports', ['Test::Simple'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test2'), ('license', 'perl_5'), ('summary', 'Framework for writing test tools that all work together.')]))])), 'perl-test-trap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-trap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-trap'), ('version', '0.3.2')])), ('source', CommentedMap([('fn', 'Test-Trap-v0.3.2.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EB/EBHANSSEN/Test-Trap-v0.3.2.tar.gz'), ('md5', '5ba3fd70ad9776adf972f734872b4760')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl >5.22.0', 'perl-data-dump', 'perl-test-simple', 'perl-module-build']), ('run', ['libgcc', 'perl >5.22.0', 'perl-data-dump', 'perl-test-simple', 'perl-module-build'])])), ('test', CommentedMap([('requires', ['perl >5.22.0']), ('imports', None)])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Test::Trap'), ('license', 'perl_5'), ('summary', 'Trap exit codes, exceptions, output, etc')]))])), 'perl-test-warn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-warn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-warn'), ('version', '0.30')])), ('source', CommentedMap([('fn', 'Test-Warn-0.30.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHORNY/Test-Warn-0.30.tar.gz'), ('md5', '8306b998a96d2cc69266b5248d550472')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-sub-uplevel']), ('run', ['perl-threaded', 'perl-sub-uplevel'])])), ('test', CommentedMap([('imports', ['Test::Warn'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-Warn'), ('license', 'perl_5'), ('summary', 'Perl extension to test methods for warnings')]))])), 'perl-test-warnings': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-warnings/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-warnings'), ('version', '0.021')])), ('source', CommentedMap([('fn', 'Test-Warnings-0.021.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Test-Warnings-0.021.tar.gz'), ('md5', 'fcbc42414de51392348c27420d3c14b0')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-cpan-meta-check']), ('run', ['perl-threaded'])])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['Test::Warnings'])])), ('about', CommentedMap([('home', 'https://github.com/karenetheridge/Test-Warnings'), ('license', 'perl_5'), ('summary', 'Test for warnings and the lack of them')]))])), 'perl-test-xml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-xml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-xml'), ('version', '0.08')])), ('source', CommentedMap([('fn', 'Test-XML-0.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SE/SEMANTICO/Test-XML-0.08.tar.gz'), ('md5', '490b487e07908358eb64a3c3c5b48143')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-parser', 'perl-xml-sax', 'perl-xml-sax-writer', 'perl-xml-semanticdiff', 'perl-xml-twig', 'perl-xml-xpath', 'perl-module-build']), ('run', ['perl-threaded', 'perl-xml-parser', 'perl-xml-sax', 'perl-xml-sax-writer', 'perl-xml-semanticdiff', 'perl-xml-twig', 'perl-xml-xpath'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test-XML'), ('license', 'perl_5'), ('summary', 'Compare XML in perl tests')]))])), 'perl-test-yaml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-yaml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-yaml'), ('version', '1.06')])), ('source', CommentedMap([('fn', 'Test-YAML-1.06.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/I/IN/INGY/Test-YAML-1.06.tar.gz'), ('md5', '92e6ea57576428095fe3c9b8e00e6f29')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-base']), ('run', ['perl-threaded', 'perl-test-base'])])), ('test', CommentedMap([('commands', [\"perl -e 'use Test::YAML tests => 1; pass;'\"])])), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/test-yaml-pm'), ('license', 'perl_5'), ('summary', 'Testing Module for YAML Implementations')]))])), 'perl-test2-suite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test2-suite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test2-suite'), ('version', '0.000061')])), ('source', CommentedMap([('fn', 'Test2-Suite-0.000061.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Test2-Suite-0.000061.tar.gz'), ('md5', 'ee5b2dc4c1e60d4a2db3f601b1c4b71e')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-importer', 'perl-extutils-makemaker']), ('run', ['perl-threaded', 'perl-test-simple', 'perl-scalar-list-utils', 'perl-importer'])])), ('test', CommentedMap([('imports', ['Test2::Bundle', 'Test2::Bundle::Extended', 'Test2::Bundle::More', 'Test2::Bundle::Simple', 'Test2::Compare', 'Test2::Compare::Array', 'Test2::Compare::Bag', 'Test2::Compare::Base', 'Test2::Compare::Bool', 'Test2::Compare::Custom', 'Test2::Compare::DeepRef', 'Test2::Compare::Delta', 'Test2::Compare::Event', 'Test2::Compare::EventMeta', 'Test2::Compare::Hash', 'Test2::Compare::Meta', 'Test2::Compare::Negatable', 'Test2::Compare::Number', 'Test2::Compare::Object', 'Test2::Compare::OrderedSubset', 'Test2::Compare::Pattern', 'Test2::Compare::Ref', 'Test2::Compare::Regex', 'Test2::Compare::Scalar', 'Test2::Compare::Set', 'Test2::Compare::String', 'Test2::Compare::Undef', 'Test2::Compare::Wildcard', 'Test2::Mock', 'Test2::Plugin', 'Test2::Plugin::BailOnFail', 'Test2::Plugin::DieOnFail', 'Test2::Plugin::ExitSummary', 'Test2::Plugin::SRand', 'Test2::Plugin::UTF8', 'Test2::Require', 'Test2::Require::AuthorTesting', 'Test2::Require::EnvVar', 'Test2::Require::Fork', 'Test2::Require::Module', 'Test2::Require::Perl', 'Test2::Require::RealFork', 'Test2::Require::Threads', 'Test2::Suite', 'Test2::Todo', 'Test2::Tools', 'Test2::Tools::Basic', 'Test2::Tools::Class', 'Test2::Tools::ClassicCompare', 'Test2::Tools::Compare', 'Test2::Tools::Defer', 'Test2::Tools::Encoding', 'Test2::Tools::Event', 'Test2::Tools::Exception', 'Test2::Tools::Exports', 'Test2::Tools::Grab', 'Test2::Tools::Mock', 'Test2::Tools::Ref', 'Test2::Tools::Subtest', 'Test2::Tools::Target', 'Test2::Tools::Warnings', 'Test2::Util::Grabber', 'Test2::Util::Ref', 'Test2::Util::Stash', 'Test2::Util::Sub', 'Test2::Util::Table', 'Test2::Util::Table::LineBreak'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test2::Suite'), ('license', 'perl_5'), ('summary', 'Distribution with a rich set of tools built upon the Test2 framework.')]))])), 'perl-text-balanced': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-balanced/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-balanced'), ('version', '2.03')])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/text/balanced/'), ('license', 'Perl5'), ('summary', 'Text::Balanced - Extract delimited text sequences from strings')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'Text-Balanced-2.03.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/S/SH/SHAY/Text-Balanced-2.03.tar.gz'), ('sha256', '057753f8f0568b53921f66a60a89c30092b73329bcc61a2c43339ab70c9792c8')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'libgcc', 'llvm', 'gcc']), ('run', ['perl-threaded', 'libgcc', 'llvm', 'gcc'])])), ('test', CommentedMap([('imports', ['Text::Balanced'])]))])), 'perl-text-csv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-csv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-csv'), ('version', '1.33')])), ('source', CommentedMap([('fn', 'Text-CSV-1.33.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/MA/MAKAMAKA/Text-CSV-1.33.tar.gz'), ('md5', '358e2f2ddb6c7746c91e49a7db97b634')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Text::CSV'), ('license', 'perl_5'), ('summary', 'comma-separated values manipulator (using XS or PurePerl)')]))])), 'perl-text-diff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-diff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-diff'), ('version', '1.44')])), ('source', CommentedMap([('fn', 'Text-Diff-1.44.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Text-Diff-1.44.tar.gz'), ('md5', '1f0bcd24b64b50a29ad1cf997c2247e9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-algorithm-diff']), ('run', ['perl-threaded', 'perl-algorithm-diff'])])), ('test', CommentedMap([('imports', ['Text::Diff'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text-Diff'), ('license', 'perl_5'), ('summary', 'Perform diffs on files and record sets')]))])), 'perl-text-format': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-format/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-format'), ('version', '0.59')])), ('source', CommentedMap([('fn', 'Text-Format-0.59.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/Text-Format-0.59.tar.gz'), ('md5', 'ab4ae26dbbb7eed101dc7a4324ac9c10')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Text::Format'])])), ('about', CommentedMap([('home', 'http://www.shlomifish.org/open-source/projects/Text-Format/'), ('license', 'perl_5'), ('summary', 'Format text')]))])), 'perl-text-glob': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-glob/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-glob'), ('version', '0.11')])), ('source', CommentedMap([('url', 'https://cpan.metacpan.org/authors/id/R/RC/RCLAMP/Text-Glob-0.11.tar.gz'), ('md5', 'd001559c504a2625dd117bd1558f07f7')])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Text::Glob'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Text::Glob'), ('license', 'perl_5'), ('summary', 'match globbing patterns against text')]))])), 'perl-text-nsp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-nsp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-nsp'), ('version', '1.31')])), ('source', CommentedMap([('fn', 'Text-NSP-1.31.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TP/TPEDERSE/Text-NSP-1.31.tar.gz'), ('md5', 'ed15ed2f35947c0a486fa9751f339b09')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text-NSP'), ('license', 'open_source'), ('summary', 'Extract collocations and Ngrams from text')]))])), 'perl-text-parsewords': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-parsewords/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-parsewords'), ('version', 3.29)])), ('source', CommentedMap([('fn', 'Text-ParseWords-3.29.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHORNY/Text-ParseWords-3.29.tar.gz'), ('md5', 'a3d5e6349529a66c56367fc94e99d6f9')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-scalar-list-utils']), ('run', ['perl', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Text::ParseWords'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::ParseWords'), ('license', 'perl_5'), ('summary', 'parse text into an array of tokens or array of arrays')]))])), 'perl-text-soundex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-soundex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-soundex'), ('version', '3.05')])), ('source', CommentedMap([('fn', 'Text-Soundex-3.05.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/R/RJ/RJBS/Text-Soundex-3.05.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Text::Soundex'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~rjbs/Text-Soundex-3.05/Soundex.pm'), ('license', 'perl_5'), ('summary', 'Implementation of the soundex algorithm.')]))])), 'perl-text-tabs-wrap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-tabs-wrap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-text-tabs-wrap'), ('version', '2013.0523')])), ('source', CommentedMap([('fn', 'Text-Tabs+Wrap-2013.0523.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MU/MUIR/modules/Text-Tabs+Wrap-2013.0523.tar.gz'), ('md5', '3e421ed3810803512af8d13bad548993')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Text::Tabs', 'Text::Wrap'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Tabs+Wrap'), ('license', 'unknown'), ('summary', 'Expand tabs and do simple line wrapping')]))])), 'perl-text-wrap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-wrap/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Wrap'), ('license', 'perl_5'), ('summary', 'line wrapping to form simple paragraphs')]))])), 'perl-threaded': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-threaded/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-threaded'), ('version', '5.22.0')])), ('build', CommentedMap([('number', 12)])), ('requirements', CommentedMap([('run', ['perl 5.22.0*'])])), ('test', CommentedMap([('commands', ['perl --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.perl.org/'), ('license', 'Perl'), ('summary', 'The Perl language.')]))])), 'perl-tie-cache': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-cache/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tie-cache'), ('version', '0.21')])), ('source', CommentedMap([('fn', 'Tie-Cache-0.21.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHAMAS/Tie-Cache-0.21.tar.gz'), ('md5', '8210f295e8676c8540a5934f88bf5fed')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Tie::Cache'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie-Cache'), ('license', 'unknown'), ('summary', 'LRU Cache in Memory')]))])), 'perl-tie-cacher': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-cacher/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tie-cacher'), ('version', '0.09')])), ('source', CommentedMap([('fn', 'Tie-Cacher-0.09.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TH/THOSPEL/Tie-Cacher-0.09.tar.gz'), ('md5', 'ef6b5ead22f10731ae007a921342c959')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple']), ('run', ['perl-threaded', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Tie::Cacher'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie::Cacher'), ('license', 'perl_5'), ('summary', 'Cache a (sub)set of key/value pairs. Tie and OO interface.')]))])), 'perl-tie-hash-indexed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-hash-indexed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tie-hash-indexed'), ('version', '0.05')])), ('source', CommentedMap([('fn', 'Tie-Hash-Indexed-0.05.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/M/MH/MHX/Tie-Hash-Indexed-0.05.tar.gz'), ('md5', '41e2d013df8cfafdd239bc65bbada014')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Tie::Hash::Indexed'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Tie::Hash::Indexed'), ('license', 'perl_5'), ('summary', 'Ordered hashes for Perl')]))])), 'perl-tie-ixhash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-ixhash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tie-ixhash'), ('version', '1.23')])), ('source', CommentedMap([('fn', 'Tie-IxHash-1.23.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CH/CHORNY/Tie-IxHash-1.23.tar.gz'), ('md5', '8393f2ced231533c3e714abad582f291')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Tie::IxHash'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie-IxHash'), ('license', 'perl_5'), ('summary', 'ordered associative arrays for Perl')]))])), 'perl-tie-log4perl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-log4perl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tie-log4perl'), ('version', '0.1')])), ('source', CommentedMap([('fn', 'Tie-Log4perl-0.1.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/F/FR/FRODWITH/Tie-Log4perl-0.1.tar.gz'), ('md5', '6f6cef7f335b2326b5c91dbef46d4560')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-log-log4perl']), ('run', ['perl', 'perl-log-log4perl'])])), ('test', CommentedMap([('imports', ['Tie::Log4perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie::Log4perl'), ('license', 'perl_5'), ('summary', 'Tie a filehandle to log via Log4perl')]))])), 'perl-time-hires': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-time-hires/1.9726/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Time::HiRes'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::HiRes'), ('license', 'perl_5'), ('summary', 'High resolution alarm, sleep, gettimeofday, interval timers')]))])), 'perl-timedate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-timedate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-timedate'), ('version', '2.30')])), ('source', CommentedMap([('fn', 'TimeDate-2.30.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GB/GBARR/TimeDate-2.30.tar.gz'), ('md5', 'b1d91153ac971347aee84292ed886c1c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TimeDate'), ('license', 'perl_5'), ('summary', 'Date formating subroutines')]))])), 'perl-tree-dag_node': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tree-dag_node/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-tree-dag_node'), ('version', '1.29')])), ('source', CommentedMap([('fn', 'Tree-DAG_Node-1.29.tgz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSAVAGE/Tree-DAG_Node-1.29.tgz'), ('md5', '0d2c2f7cb7f12afafd28bbab75e15442')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-file-slurp-tiny']), ('run', ['perl-threaded', 'perl-file-slurp-tiny'])])), ('test', CommentedMap([('imports', ['Tree::DAG_Node'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tree-DAG_Node'), ('license', 'artistic_2'), ('summary', 'An N-ary tree')]))])), 'perl-try-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-try-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-try-tiny'), ('version', '0.24')])), ('source', CommentedMap([('fn', 'Try-Tiny-0.24.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Try-Tiny-0.24.tar.gz'), ('md5', '1a12a51cfeb7e2c301e4ae093c7ecdfb')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['perl', 'perl-capture-tiny']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Try::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/karenetheridge/Try-Tiny'), ('license', 'mit'), ('summary', 'minimal try/catch with proper preservation of $@')]))])), 'perl-type-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-type-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-type-tiny'), ('version', '1.000005')])), ('source', CommentedMap([('fn', 'Type-Tiny-1.000005.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TOBYINK/Type-Tiny-1.000005.tar.gz'), ('md5', '2914b0e6b35c35f657d426fba29b4520')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-exporter-tiny', 'perl-moo']), ('run', ['perl-threaded', 'perl-exporter-tiny'])])), ('test', CommentedMap([('imports', ['Type::Coercion', 'Type::Coercion::Union', 'Type::Library', 'Type::Params', 'Type::Parser', 'Type::Registry', 'Type::Tiny', 'Type::Tiny::Class', 'Type::Tiny::Duck', 'Type::Tiny::Enum', 'Type::Tiny::Intersection', 'Type::Tiny::Role', 'Type::Tiny::Union', 'Type::Utils', 'Types::Common::Numeric', 'Types::Common::String', 'Types::Standard', 'Types::Standard::ArrayRef', 'Types::Standard::Dict', 'Types::Standard::HashRef', 'Types::Standard::Map', 'Types::Standard::ScalarRef', 'Types::Standard::Tuple', 'Types::TypeTiny'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Type-Tiny'), ('license', 'perl_5'), ('summary', 'tiny, yet Moo(se)-compatible type constraint')]))])), 'perl-types-serialiser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-types-serialiser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-types-serialiser'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'Types-Serialiser-1.0.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/ML/MLEHMANN/Types-Serialiser-1.0.tar.gz'), ('md5', '76460a2bfbc644672499af89192e03fe')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-common-sense']), ('run', ['perl-threaded', 'perl-common-sense'])])), ('test', CommentedMap([('imports', ['Types::Serialiser'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~mlehmann/Types-Serialiser-1.0/Serialiser.pm'), ('license', 'perl_5'), ('summary', 'Simple data types for common serialisation formats')]))])), 'perl-types-standard': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-types-standard/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Type-Tiny-1.002001.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/T/TO/TOBYINK/Type-Tiny-1.002001.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp', 'perl-exporter-tiny']), ('run', ['perl', 'perl-test-more', 'perl-pod-escapes', 'perl-carp', 'perl-extutils-makemaker', 'perl-exporter-tiny'])])), ('test', CommentedMap([('commands', ['perl -e \"use Types::Standard\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~tobyink/Type-Tiny-1.002001/lib/Types/Standard.pm'), ('license', 'perl_5'), ('summary', 'bundled set of built-in types for Type::Tiny')]))])), 'perl-unicode-map': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-unicode-map/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-unicode-map'), ('version', '0.112')])), ('source', CommentedMap([('fn', 'Unicode-Map-0.112.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MS/MSCHWARTZ/Unicode-Map-0.112.tar.gz'), ('md5', 'edaa8fc5ddf0e5d805e274283dd0625d')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Unicode::Map']), ('commands', ['perl -e \\'use Unicode::Map(); $map = new Unicode::Map(\"ISO-8859-1\");\\''])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Unicode-Map'), ('license', 'unknown'), ('summary', 'Map charsets from and to UTF-16 unicode')]))])), 'perl-unicode-stringprep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-unicode-stringprep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-unicode-stringprep'), ('version', '1.105')])), ('source', CommentedMap([('fn', 'Unicode-Stringprep-1.105.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/C/CF/CFAERBER/Unicode-Stringprep-1.105.tar.gz'), ('md5', 'bf9ffbc387cc12d67a3875be1cd8e105')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-test-nowarnings']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Unicode::Stringprep'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Unicode-Stringprep'), ('license', 'perl_5'), ('summary', 'Preparation of Internationalized Strings (RFC 3454)')]))])), 'perl-uri': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-uri/1.71/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-uri'), ('version', '1.71')])), ('source', CommentedMap([('fn', 'URI-1.71.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/URI-1.71.tar.gz'), ('md5', '247c3da29a794f72730e01aa5a715daf')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/URI'), ('license', 'perl_5'), ('summary', 'Uniform Resource Identifiers (absolute and relative)')]))])), 'perl-vcftools-vcf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-vcftools-vcf/0.840/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-vcftools-vcf'), ('version', '0.840')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Vcf;\"'])])), ('about', CommentedMap([('home', 'https://github.com/vcftools/vcftools'), ('license', 'LGPLv3'), ('summary', 'cpanm ready distribution of VCFtools Perl libraries')]))])), 'perl-velvetoptimiser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-velvetoptimiser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-velvetoptimiser'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', '2.2.5.tar.gz'), ('url', 'https://github.com/tseemann/VelvetOptimiser/archive/2.2.5.tar.gz'), ('md5', '66966ce03f61321a2f81a5cfb45a3be9'), ('patches', ['perl_path.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'perl-bioperl >=1.4', 'velvet >=0.7.51'])])), ('test', CommentedMap([('commands', ['VelvetOptimiser.pl --version > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/VelvetOptimiser'), ('license', 'GPLv2'), ('summary', \"Automatically optimise three of Velvet's assembly parameters.\")]))])), 'perl-version-next': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-version-next/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-version-next'), ('version', '1.000')])), ('source', CommentedMap([('fn', 'Version-Next-1.000.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Version-Next-1.000.tar.gz'), ('md5', '4bafa527ff40edef40822f1f9a80fe51')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-test-exception', 'perl-pathtools', 'perl-apache-test', 'perl-extutils-makemaker', 'perl-sub-exporter', 'perl-perl-version']), ('run', ['perl-threaded', 'perl-apache-test', 'perl-sub-exporter', 'perl-perl-version'])])), ('test', CommentedMap([('imports', ['Version::Next'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Version-Next'), ('license', 'apache_2_0'), ('summary', 'increment module version numbers simply and correctly')]))])), 'perl-want': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-want/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/R/RO/ROBIN/Want-0.29.tar.gz'), ('md5', '33b2dae5db59781b9a0434fa1db04aab')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Want'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~robin/Want-0.29/Want.pm'), ('license', 'perl_5'), ('summary', 'This module generalises the mechanism of the wantarray function, allowing a function to determine in some detail how its return value is going to be immediately used.')]))])), 'perl-www-robotrules': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-www-robotrules/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-www-robotrules'), ('version', '6.02')])), ('source', CommentedMap([('fn', 'WWW-RobotRules-6.02.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GA/GAAS/WWW-RobotRules-6.02.tar.gz'), ('md5', 'b7186e8b8b3701e70c22abf430742403')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-uri']), ('run', ['perl-threaded', 'perl-uri'])])), ('test', CommentedMap([('imports', ['WWW::RobotRules'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/WWW-RobotRules'), ('license', 'perl_5'), ('summary', 'database of robots.txt-derived permissions')]))])), 'perl-xml-dom': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-dom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-dom'), ('version', '1.45')])), ('source', CommentedMap([('fn', 'XML-DOM-1.45.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TJ/TJMATHER/XML-DOM-1.45.tar.gz'), ('md5', 'f95fd6929b43d140db1afc0282513e65')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-libwww-perl', 'perl-libxml-perl', 'perl-xml-parser', 'perl-xml-regexp']), ('run', ['perl-threaded', 'perl-libwww-perl', 'perl-libxml-perl', 'perl-xml-parser', 'perl-xml-regexp'])])), ('test', CommentedMap([('imports', ['XML::DOM'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-DOM'), ('license', 'unknown'), ('summary', 'A perl module for building DOM Level 1 compliant document structures')]))])), 'perl-xml-dom-xpath': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-dom-xpath/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-dom-xpath'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'XML-DOM-XPath-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MI/MIROD/XML-DOM-XPath-0.14.tar.gz'), ('md5', '51a40df96c2f92829e1a4f84782fa75e')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-dom', 'perl-xml-xpathengine']), ('run', ['perl-threaded', 'perl-xml-dom', 'perl-xml-xpathengine'])])), ('test', CommentedMap([('imports', ['XML::DOM::XPath'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-DOM-XPath'), ('license', 'unknown'), ('summary', 'Perl extension to add XPath support to XML::DOM, using XML::XPath engine')]))])), 'perl-xml-filter-buffertext': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-filter-buffertext/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-filter-buffertext'), ('version', '1.01')])), ('source', CommentedMap([('fn', 'XML-Filter-BufferText-1.01.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/R/RB/RBERJON/XML-Filter-BufferText-1.01.tar.gz'), ('md5', '2992c0387632583b966ab9c965b25512')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-sax-base']), ('run', ['perl-threaded', 'perl-xml-sax-base'])])), ('test', CommentedMap([('imports', ['XML::Filter::BufferText'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-Filter-BufferText'), ('license', 'unknown'), ('summary', 'Filter to put all characters() in one event')]))])), 'perl-xml-libxml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-libxml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-libxml'), ('version', '2.0124')])), ('source', CommentedMap([('fn', 'XML-LibXML-2.0124.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0124.tar.gz'), ('md5', '30436b85454fba3ae5f6494df598e65c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-sax', 'perl-xml-namespacesupport', 'libxml2']), ('run', ['perl-threaded', 'perl-xml-sax', 'perl-xml-namespacesupport', 'libxml2'])])), ('test', CommentedMap([('imports', ['XML::LibXML', 'XML::LibXML::AttributeHash', 'XML::LibXML::Boolean', 'XML::LibXML::Common', 'XML::LibXML::Devel', 'XML::LibXML::ErrNo', 'XML::LibXML::Error', 'XML::LibXML::Literal', 'XML::LibXML::NodeList', 'XML::LibXML::Number', 'XML::LibXML::Reader', 'XML::LibXML::SAX', 'XML::LibXML::SAX::Builder', 'XML::LibXML::SAX::Generator', 'XML::LibXML::SAX::Parser', 'XML::LibXML::XPathContext'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/shlomif/perl-xml-libxml'), ('license', 'Perl'), ('summary', 'Interface to Gnome libxml2 xml parsing and DOM library')]))])), 'perl-xml-libxslt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-libxslt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-libxslt'), ('version', '1.94')])), ('source', CommentedMap([('fn', 'XML-LibXSLT-1.94.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXSLT-1.94.tar.gz'), ('md5', 'a32045699c001d960d6f94891f9e853c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-libxml', 'libxslt']), ('run', ['perl-threaded', 'perl-xml-libxml', 'libxslt'])])), ('test', CommentedMap([('imports', ['XML::LibXSLT'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-LibXSLT'), ('license', 'perl_5'), ('summary', 'Interface to GNOME libxslt library')]))])), 'perl-xml-namespacesupport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-namespacesupport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-namespacesupport'), ('version', '1.11')])), ('source', CommentedMap([('fn', 'XML-NamespaceSupport-1.11.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-NamespaceSupport-1.11.tar.gz'), ('md5', '222cca76161cd956d724286d36b607da')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::NamespaceSupport'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-NamespaceSupport'), ('license', 'perl_5'), ('summary', 'a simple generic namespace support class')]))])), 'perl-xml-parser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-parser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-parser'), ('version', '2.44')])), ('source', CommentedMap([('fn', 'XML-Parser-2.44.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-2.44.tar.gz'), ('md5', 'af4813fe3952362451201ced6fbce379')])), ('build', CommentedMap([('number', 4)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'expat', 'perl-libwww-perl']), ('run', ['libgcc', 'perl-threaded', 'expat', 'perl-libwww-perl'])])), ('test', CommentedMap([('imports', ['XML::Parser', 'XML::Parser::Expat', 'XML::Parser::Style::Debug', 'XML::Parser::Style::Objects', 'XML::Parser::Style::Stream', 'XML::Parser::Style::Subs', 'XML::Parser::Style::Tree'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/XML::Parser'), ('license', 'Perl'), ('summary', 'A perl module for parsing XML documents')]))])), 'perl-xml-parser-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-parser-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-parser-lite'), ('version', '0.721')])), ('source', CommentedMap([('fn', 'XML-Parser-Lite-0.721.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PH/PHRED/XML-Parser-Lite-0.721.tar.gz'), ('md5', 'ad8a87b9bf413aa540c7cb724d650808')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-requires']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::Parser::Lite'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-Parser-Lite'), ('license', 'perl_5'), ('summary', 'Lightweight pure-perl XML Parser (based on regexps)')]))])), 'perl-xml-regexp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-regexp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-regexp'), ('version', '0.04')])), ('source', CommentedMap([('fn', 'XML-RegExp-0.04.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/T/TJ/TJMATHER/XML-RegExp-0.04.tar.gz'), ('md5', '2e38ea2340f2d2eb56bd81c4d739fbe0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::RegExp'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-RegExp'), ('license', 'unknown'), ('summary', 'Regular expressions for XML tokens')]))])), 'perl-xml-sax': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-sax/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-sax'), ('version', '0.99')])), ('source', CommentedMap([('fn', 'XML-SAX-0.99.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-0.99.tar.gz'), ('md5', '290f5375ae87fdebfdb5bc3854019f24')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-namespacesupport', 'perl-xml-sax-base']), ('run', ['perl-threaded', 'perl-xml-sax-base', 'perl-xml-namespacesupport'])])), ('test', CommentedMap([('imports', ['XML::SAX'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/~grantm/XML-SAX-0.99'), ('license', 'Perl'), ('summary', 'Simple API for XML')]))])), 'perl-xml-sax-base': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-sax-base/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-sax-base'), ('version', '1.08')])), ('source', CommentedMap([('fn', 'XML-SAX-Base-1.08.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.08.tar.gz'), ('md5', '38c8c3247dfd080712596118d70dbe32')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::SAX::Base'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-SAX-Base'), ('license', 'perl_5'), ('summary', 'Base class for SAX Drivers and Filters')]))])), 'perl-xml-sax-expat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-sax-expat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-sax-expat'), ('version', '0.51')])), ('source', CommentedMap([('fn', 'XML-SAX-Expat-0.51.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/B/BJ/BJOERN/XML-SAX-Expat-0.51.tar.gz'), ('md5', '74c540095d84e3f054d1f8703a73ef76')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-namespacesupport', 'perl-xml-sax', 'perl-xml-parser', 'perl-xml-sax-base']), ('run', ['perl-threaded', 'perl-xml-parser', 'perl-xml-sax-base', 'perl-xml-sax', 'perl-xml-namespacesupport'])])), ('test', CommentedMap([('imports', ['XML::SAX::Expat'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-SAX-Expat'), ('license', 'perl_5'), ('summary', 'SAX Driver for Expat')]))])), 'perl-xml-sax-writer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-sax-writer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-sax-writer'), ('version', '0.56')])), ('source', CommentedMap([('fn', 'XML-SAX-Writer-0.56.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-SAX-Writer-0.56.tar.gz'), ('md5', 'e5fa5e2b2f6867b1aca9b5b43e1ce361')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-filter-buffertext', 'perl-xml-namespacesupport', 'perl-xml-sax-base']), ('run', ['perl-threaded', 'perl-xml-filter-buffertext', 'perl-xml-namespacesupport', 'perl-xml-sax-base'])])), ('test', CommentedMap([('imports', ['XML::SAX::Writer', 'XML::SAX::Writer::XML'])])), ('about', CommentedMap([('home', 'https://github.com/perigrin/xml-sax-writer'), ('license', 'perl_5'), ('summary', 'SAX2 XML Writer')]))])), 'perl-xml-semanticdiff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-semanticdiff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-semanticdiff'), ('version', '1.0004')])), ('source', CommentedMap([('fn', 'XML-SemanticDiff-1.0004.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-SemanticDiff-1.0004.tar.gz'), ('md5', '0d13434cbc3eb2721b45c942ddb05d97')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-xml-parser']), ('run', ['perl-threaded', 'perl-xml-parser'])])), ('test', CommentedMap([('imports', ['XML::SemanticDiff', 'XML::SemanticDiff::BasicHandler'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-SemanticDiff'), ('license', 'perl_5'), ('summary', 'Perl extension for comparing XML documents.')]))])), 'perl-xml-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-simple'), ('version', '2.22')])), ('source', CommentedMap([('fn', 'XML-Simple-2.22.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-Simple-2.22.tar.gz'), ('md5', '0914abddfce749453ed89b54029f2643')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-namespacesupport', 'perl-xml-sax-expat', 'perl-xml-sax']), ('run', ['perl-threaded', 'perl-xml-sax-expat', 'perl-xml-sax', 'perl-xml-namespacesupport'])])), ('test', CommentedMap([('imports', ['XML::Simple'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-Simple'), ('license', 'perl_5'), ('summary', 'An API for simple XML files')]))])), 'perl-xml-twig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-twig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-twig'), ('version', '3.49')])), ('source', CommentedMap([('fn', 'XML-Twig-3.49.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIROD/XML-Twig-3.49.tar.gz'), ('md5', 'afb5786e15cfe7823add6756382e7df7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-parser', 'perl-xml-xpathengine', 'perl-tie-ixhash', 'perl-html-tree', 'perl-html-formatter', 'perl-html-entities-numbered', 'perl-html-tidy', 'perl-html-parser', 'perl-xml-simple', 'perl-xml-sax-writer', 'perl-io-stringy']), ('run', ['perl-threaded', 'perl-xml-parser', 'perl-xml-xpathengine', 'perl-tie-ixhash', 'perl-html-tree', 'perl-html-formatter', 'perl-html-entities-numbered', 'perl-html-tidy', 'perl-html-parser'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-Twig'), ('license', 'perl_5'), ('summary', 'XML, The Perl Way')]))])), 'perl-xml-writer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-writer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-writer'), ('version', '0.625')])), ('source', CommentedMap([('fn', 'XML-Writer-0.625.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/J/JO/JOSEPHW/XML-Writer-0.625.tar.gz'), ('md5', 'b9c2420c243c6a36ce45a008740fcede')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::Writer'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-Writer'), ('license', 'unrestricted'), ('summary', 'Easily generate well-formed, namespace-aware XML.')]))])), 'perl-xml-xpath': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-xpath/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-xpath'), ('version', '1.33')])), ('source', CommentedMap([('fn', 'XML-XPath-1.33.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MANWAR/XML-XPath-1.33.tar.gz'), ('md5', 'dcbd03d687de06a402fed58197eb94f1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-xml-parser', 'perl-path-tiny']), ('run', ['perl-threaded', 'perl-xml-parser'])])), ('test', CommentedMap([('imports', ['XML::XPath', 'XML::XPath::Boolean', 'XML::XPath::Builder', 'XML::XPath::Expr', 'XML::XPath::Function', 'XML::XPath::Literal', 'XML::XPath::LocationPath', 'XML::XPath::Node', 'XML::XPath::Node::Attribute', 'XML::XPath::Node::Comment', 'XML::XPath::Node::Element', 'XML::XPath::Node::Namespace', 'XML::XPath::Node::PI', 'XML::XPath::Node::Text', 'XML::XPath::NodeSet', 'XML::XPath::Number', 'XML::XPath::Parser', 'XML::XPath::PerlSAX', 'XML::XPath::Root', 'XML::XPath::Step', 'XML::XPath::Variable', 'XML::XPath::XMLParser'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-XPath'), ('license', 'artistic_2'), ('summary', 'Parse and evaluate XPath statements.')]))])), 'perl-xml-xpathengine': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xml-xpathengine/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-xml-xpathengine'), ('version', '0.14')])), ('source', CommentedMap([('fn', 'XML-XPathEngine-0.14.tar.gz'), ('url', 'http://cpan.metacpan.org/authors/id/M/MI/MIROD/XML-XPathEngine-0.14.tar.gz'), ('md5', 'd840cbefa57c23041f0bed14940a6b22')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['XML::XPathEngine', 'XML::XPathEngine::Boolean', 'XML::XPathEngine::Expr', 'XML::XPathEngine::Function', 'XML::XPathEngine::Literal', 'XML::XPathEngine::LocationPath', 'XML::XPathEngine::NodeSet', 'XML::XPathEngine::Number', 'XML::XPathEngine::Root', 'XML::XPathEngine::Step', 'XML::XPathEngine::Variable'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/XML-XPathEngine'), ('license', 'unknown'), ('summary', 'a re-usable XPath engine for DOM-like trees')]))])), 'perl-xsloader': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-xsloader/0.22/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'XSLoader-0.22.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SA/SAPER/XSLoader-0.22.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl', 'perl-test-more'])])), ('test', CommentedMap([('commands', ['perl -e \"use XSLoader\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/module/XSLoader'), ('license', 'perl_5'), ('summary', 'Dynamically load C libraries into Perl code')]))])), 'perl-yaml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-yaml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-yaml'), ('version', '1.18')])), ('source', CommentedMap([('fn', 'YAML-1.18.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/T/TI/TINITA/YAML-1.18.tar.gz'), ('md5', '79b6cc821fe157790a6448d1f791f6de')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-yaml']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'https://github.com/ingydotnet/yaml-pm'), ('license', 'perl_5'), ('summary', \"YAML Ain't Markup Language\")]))])), 'perl-yaml-libyaml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-yaml-libyaml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-yaml-libyaml'), ('version', '0.63')])), ('source', CommentedMap([('fn', 'YAML-LibYAML-0.63.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/T/TI/TINITA/YAML-LibYAML-0.63.tar.gz'), ('md5', 'a6a2d7e2124b101236613e831458172f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/YAML-LibYAML'), ('license', 'perl_5'), ('summary', 'Perl YAML Serialization using XS and libyaml ')]))])), 'perl-yaml-tiny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-yaml-tiny/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-yaml-tiny'), ('version', '1.70')])), ('source', CommentedMap([('fn', 'YAML-1.70.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/E/ET/ETHER/YAML-Tiny-1.70.tar.gz'), ('md5', 'a475714e8d512aafc410ca4d7a117aa2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['YAML::Tiny'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/YAML-Tiny/lib/YAML/Tiny.pm'), ('license', 'perl'), ('summary', 'Read/Write YAML files with as little code as possible')]))])), 'pgma-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pgma-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pgma-simple'), ('version', 0.1)])), ('source', CommentedMap([('fn', 'PGMAsimple-0.1.tar.gz'), ('url', 'https://raw.githubusercontent.com/BackofenLab/GraphClust/master/pgma_src/PGMAsimple-0.1.tar.gz'), ('sha256', '785c86b3e57461ffd6c4b8d1002c24d69fc33552b2e81bcd0a9da589330d2b70')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"pgma 2>&1| grep 'USAGE'\"])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/GraphClust'), ('license', 'GNUv3'), ('license_file', None), ('summary', 'Pgma is a simple program for building WPGMA trees.')]))])), 'phipack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phipack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phipack'), ('version', '1.0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'PhiPack.tar.gz'), ('url', 'http://www.maths.otago.ac.nz/~dbryant/software/PhiPack.tar.gz'), ('sha256', 'af43326e021f1f2e590be2cba3caa44a0963f237386e63209ccc26b5bfb02db9')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['Phi 2>&1 | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'http://www.maths.otago.ac.nz/~dbryant/software.html'), ('license', 'GNU General Public License v3.0'), ('summary', 'Simple, rapid, and statistically efficient test for recombination.')]))])), 'phizz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phizz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phizz'), ('version', '0.0.1')])), ('source', CommentedMap([('fn', 'phizz-0.0.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/e9/a9/ffdd3c5fa5418b93e29101dd4d2ac84f8143bdd0d34b897bd61b2b8b6208/phizz-0.0.1.tar.gz'), ('md5', '29387dba01cd9148e47d2998fa932961')])), ('build', CommentedMap([('preserve_egg_dir', True), ('entry_points', ['phizz = phizz.__main__:cli']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click', 'configobj']), ('run', ['python', 'setuptools', 'click', 'configobj'])])), ('test', CommentedMap([('imports', ['phizz', 'phizz.database', 'phizz.utils']), ('commands', ['LANG=C.UTF-8 phizz --help'])])), ('about', CommentedMap([('home', 'https://github.com/moonso/query_hpo'), ('license', 'MIT License'), ('summary', 'Tool to query hpo database and some other sources')]))])), 'phylip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phylip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phylip'), ('version', 3.696)])), ('source', CommentedMap([('fn', 'phylip-3.696.tar.gz'), ('url', 'http://evolution.gs.washington.edu/phylip/download/phylip-3.696.tar.gz'), ('md5', 'dbe5abc26f6089ead3dba41c2db526ee'), ('patches', ['makefile.patch', 'font-paths.patch', 'maxtrees.patch'])])), ('build', CommentedMap([('number', 3), ('detect_binary_files_with_prefix', True), ('script_env', ['BB_ARCH_FLAGS', 'BB_OPT_FLAGS', 'BB_MAKE_JOBS'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['openjdk >=6', 'libgcc', 'python'])])), ('test', CommentedMap([('commands', ['mkdir __tmp4phyliptests__; pushd __tmp4phyliptests__; phylip test; pushd; rm -rf __tmp4phyliptests__'])])), ('about', CommentedMap([('home', 'http://evolution.genetics.washington.edu/phylip/'), ('license', 'BSD'), ('summary', 'Package of programs for inferring phylogenies')]))])), 'phyloflash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phyloflash/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower}}'), ('version', \"<{ version|replace('-','') }}\")])), ('source', CommentedMap([('url', 'https://github.com/HRGV/<{ name }}/archive/v<{ version }}.tar.gz'), ('fn', 'v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['real_bin.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.13.2', 'bbmap', 'emirge', 'pigz', 'samtools', 'vsearch', 'bedtools', 'mafft', 'spades', 'r-base', 'r-ggdendro', 'r-gtable', 'r-reshape2', 'r-ggplot2', 'r-optparse'])])), ('test', CommentedMap([('commands', ['phyloFlash.pl -check_env || true', 'phyloFlash.pl -check_env 2>&1 | grep \"Please provide a path using -dbhome\"', 'phyloFlash_makedb.pl --help', 'phyloFlash_heatmap.R --help |tee | grep \"Generates a heatmap plot\"'])])), ('about', CommentedMap([('home', 'https://github.com/HRGV/phyloFlash'), ('license', 'GPLv3'), ('license_family', 'GPL'), ('summary', 'phyloFlash is a pipeline to rapidly reconstruct the SSU rRNAs and explore\\nphylogenetic composition of an illumina (meta)genomic dataset.\\n')]))])), 'phylotoast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phylotoast/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/smdabdoub/phylotoast/archive/da6a606ed162ab082a9afa0ef0e0f3b5e94a872e.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'matplotlib <=1.5.3', 'biopython >=1.60', 'scikit-bio <=0.4.2', 'scikit-learn', 'pandas', 'statsmodels', 'palettable', 'biom-format >=2.1.5', 'h5py']), ('run', ['python', 'numpy', 'scipy', 'matplotlib <=1.5.3', 'biopython >=1.60', 'scikit-bio <=0.4.2', 'scikit-learn', 'pandas', 'statsmodels', 'palettable', 'biom-format >=2.1.5', 'h5py'])])), ('test', CommentedMap([('imports', ['phylotoast', 'phylotoast.test']), ('commands', ['barcode_filter.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/smdabdoub/phylotoast'), ('license', 'MIT'), ('summary', 'Tools for phylogenetic data analysis including visualization and cluster-computing support. ')]))])), 'phylowgs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phylowgs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phylowgs'), ('version', '20150714')])), ('source', CommentedMap([('fn', 'phylowgs-9a108f4.tar.gz'), ('url', 'https://github.com/morrislab/phylowgs/archive/9a108f4.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'ete2', 'gsl']), ('run', ['python', 'setuptools', 'numpy', 'scipy', 'ete2', 'gsl'])])), ('about', CommentedMap([('home', 'https://github.com/morrislab/phylowgs'), ('license', 'GPLv3'), ('summary', 'Application for inferring subclonal composition and evolution from whole-genome sequencing data')]))])), 'phyml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phyml/3.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/stephaneguindon/<{ name | lower }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['pin_version.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'pkg-config', 'openmpi 2.*', 'beagle-lib', 'libtool']), ('run', ['libgcc', 'openmpi 2.*', 'beagle-lib', 'libtool'])])), ('test', CommentedMap([('requirements', ['conda']), ('commands', ['if command -v conda; then conda inspect linkages -p $PREFIX $PKG_NAME; fi', 'if command -v conda; then conda inspect objects -p $PREFIX $PKG_NAME; fi', 'phyml --help | grep \"PhyML <{ version }}\"', 'mpirun -mca orte_rsh_agent true --allow-run-as-root phyml-mpi --help | grep \"PhyML <{ version }}\"', 'phyml-beagle --help | grep \"PhyML <{ version }}\"'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/phyml/'), ('dev_url', 'https://github.com/stephaneguindon/phyml/'), ('license', 'GPLv2'), ('license_family', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Phylogenetic estimation using (Maximum) Likelihood'), ('description', 'PhyML is a software that estimates maximum likelihood phylogenies from\\nalignments of nucleotide or amino acid sequences. The main strength of\\nPhyML lies in the large number of substitution models coupled to various\\noptions to search the space of phylogenetic tree topologies, going from\\nvery fast and efficient methods to slower but generally more accurate\\napproaches. PhyML was designed to process moderate to large data sets. In\\ntheory, alignments with up to 4,000 sequences 2,000,000 character-long\\ncan be processed. PhyML can process data sets made of multiple genes and\\nfit sophisticated substitution models with heterogeneous components\\nacross partition elements.\\n')]))])), 'picard': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/picard/2.9.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '2.9.2')])), ('source', CommentedMap([('fn', 'picard.jar'), ('url', 'https://github.com/broadinstitute/picard/releases/download/2.9.2/picard.jar'), ('md5', '0449279a6a89830917e8bcef3a976ef7')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', ['picard 2>&1 | grep USAGE:'])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))])), 'picopore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/picopore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'picopore'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'picopore-1.2.0.zip'), ('url', 'https://files.pythonhosted.org/packages/98/e6/2431eebb5370c84e4071475206be805f7059d83ead8c53ba509ecf6e25d6/picopore-1.2.0.zip'), ('md5', '5022bda29808de873c676b47727d1d6d')])), ('build', CommentedMap([('entry_points', ['picopore = picopore.__main__:main', 'picopore-test = picopore.test:main', 'picopore-realtime = picopore.realtime:main', 'picopore-rename = picopore.rename:main']), ('number', 0), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'h5py >2.2.0', 'watchdog', 'hdf5', 'future']), ('run', ['python', 'h5py >2.2.0', 'watchdog', 'hdf5', 'future'])])), ('test', CommentedMap([('imports', ['picopore']), ('commands', ['picopore --help', 'picopore-test --help', 'picopore-realtime --help', 'picopore-rename --help'])])), ('about', CommentedMap([('home', 'https://github.com/scottgigante/picopore'), ('license', 'GNU General Public License (GPL)'), ('summary', \"A tool for reducing the size of Oxford Nanopore Technologies' datasets without losing information.\"), ('license_family', 'LGPL')]))])), 'picrust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/picrust/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/picrust/picrust/archive/v<{ version }}.tar.gz'), ('sha256', '3b2d1bc12b95df4b401f4f3a05913d63c78bbb5f005e60ae4868562ca2c2e4a3')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biom-format >=2.1.4,<2.2.0', 'numpy >=1.5.1', 'cogent >=1.5.3', 'h5py >=1.7.0', 'future >=0.16']), ('run', ['python', 'biom-format >=2.1.4,<2.2.0', 'numpy >=1.5.1', 'cogent >=1.5.3', 'h5py >=1.7.0', 'future >=0.16'])])), ('test', CommentedMap([('commands', ['normalize_by_copy_number.py -h', 'predict_metagenomes.py -h', 'categorize_by_function.py -h', 'metagenome_contributions.py -h', 'compare_biom.py -h', 'download_picrust_files.py -h'])])), ('about', CommentedMap([('home', 'http://picrust.github.com'), ('license', 'GPL-3.0'), ('summary', 'PICRUSt: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States')])), ('extra', CommentedMap([('notes', 'PICRUST requires a set of large pre-calculated data files. PICRUST contains a script, \"download_picrust_files.py\", which downloads the data in the proper location, and which can be run after PICRUST has been installed.')]))])), 'pileometh': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pileometh/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pileometh'), ('version', '0.1.13')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'PileOMeth-0.1.13.tar.gz'), ('sha256', '9f70f847223ebf3bd074553928a4097fa9a6db5cdc5e7632b2062c96edb3c6a8'), ('url', 'https://github.com/dpryan79/PileOMeth/archive/0.1.13.tar.gz')])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['PileOMeth --version'])])), ('about', CommentedMap([('home', 'https://github.com/dpryan79/PileOMeth'), ('license', 'MIT'), ('summary', 'A (mostly) universal methylation extractor for BS-seq experiments.')]))])), 'piler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/piler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'piler'), ('version', '0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'piler_pals_osx_src.tar'), ('url', 'http://drive5.com/piler/piler_pals_osx_src.tar.gz'), ('md5', '8a70e346a45d9f0531642e0ee17e5d86')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['piler -version'])])), ('about', CommentedMap([('home', 'http://www.drive5.com/piler'), ('license', 'public domain'), ('license_file', None), ('summary', 'PILER is public domain software for analyzing repetitive DNA found in genome sequences.')]))])), 'pilon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pilon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pilon'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pilon-<{ version }}.jar'), ('url', 'https://github.com/broadinstitute/pilon/releases/download/v<{ version }}/pilon-<{ version }}.jar'), ('md5', '3c45568dc1b878a9a0316410ec62ab04')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk', 'python'])])), ('test', CommentedMap([('commands', [\"pilon --help | grep  'Usage'\", 'pilon --jar_dir'])])), ('about', CommentedMap([('home', 'https://github.com/broadinstitute/pilon/'), ('license', 'GPLv2'), ('summary', 'Pilon is an automated genome assembly improvement and variant detection tool.')]))])), 'pindel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pindel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pindel'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'pindel-<{ version }}.tar.gz'), ('url', 'https://github.com/genome/pindel/archive/v<{ version }}.tar.gz'), ('sha256', '7f21fda0b751d420831724d96e60873ce332139cfd24396e81c7f1ae2f707a19')])), ('requirements', CommentedMap([('build', ['htslib <{CONDA_HTSLIB}}*', 'gcc']), ('run', ['htslib <{CONDA_HTSLIB}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['pindel -h 2>&1 | grep \"Pindel version\"', 'pindel2vcf  --help | grep Example:'])])), ('about', CommentedMap([('home', 'http://gmt.genome.wustl.edu/packages/pindel/index.html'), ('license', 'GPLv3'), ('summary', 'Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data')]))])), 'pipmir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pipmir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pipmir'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'PIPmiR_v1_1b_executable.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PIPmiR/PIPmiR_v1_1b_executable.tar.gz'), ('md5', '4103f0a148e4cc0217c616c1606f11c5')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['java-jdk', 'samtools', 'viennarna 1.8.5']), ('run', ['java-jdk', 'samtools', 'viennarna 1.8.5'])])), ('test', CommentedMap([('commands', ['PIPmiR'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/'), ('license', 'PIPmiR is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PIPmiR algorithm to identify novel plant miRNA genes from a combination of deep sequencing data and genomic features.')]))])), 'piranha': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/piranha/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/piranha/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Piranha is a peak-caller for CLIP- and RIP-Seq data.')])), ('package', CommentedMap([('name', 'piranha'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'piranha-1.2.1.tar.gz'), ('url', 'http://smithlabresearch.org/downloads/piranha-1.2.1.tar.gz'), ('md5', '4ca70055b54ae0212d2b3641aafcb976'), ('patches', ['0001-compile-with-conda-gsl-patch.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc', 'llvm', 'gawk', 'python', 'bamtools', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bamtools', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['Piranha 2>&1 | grep \"^Usage\"'])]))])), 'pizzly': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pizzly/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pizzly'), ('version', '0.37.3')])), ('source', CommentedMap([('fn', 'pizzly_mac.tar.gz'), ('url', 'https://github.com/pmelsted/pizzly/releases/download/v0.37.3/pizzly_mac.tar.gz'), ('sha256', 'bdd07e0aaaded045fee03e5eb0cae9a9b5c509c28cca501617ded3e0e1307138')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['wget']), ('run', ['python', 'numpy', 'h5py'])])), ('test', CommentedMap([('commands', ['pizzly --help', 'pizzly_flatten_json.py', 'pizzly_get_fragment_length.py'])])), ('about', CommentedMap([('summary', 'Fast fusion detection using kallisto'), ('home', 'https://github.com/pmelsted/pizzly'), ('license', 'BSD-2'), ('license_file', 'LICENSE')]))])), 'planemo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/planemo/0.34.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'planemo'), ('version', '0.34.1')])), ('source', CommentedMap([('fn', 'planemo-0.34.1.tar.gz'), ('url', 'https://pypi.python.org/packages/99/3c/6f1004343bad1e79106eb47030a055b02655e08137f55036ca13f71f55cc/planemo-0.34.1.tar.gz'), ('md5', 'cdb907816e915acaa873f61522166739')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['planemo=planemo.cli:planemo']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'click', 'pyaml', 'bioblend >=0.8.0', 'jinja2', 'virtualenv', 'galaxy-lib >=16.7.10', 'aenum']), ('run', ['python', 'setuptools', 'six', 'glob2', 'click', 'pyaml', 'bioblend >=0.8.0', 'jinja2', 'virtualenv', 'galaxy-lib >=16.7.10', 'aenum'])])), ('test', CommentedMap([('imports', ['planemo', 'planemo.commands', 'planemo.conda_verify', 'planemo.cwl', 'planemo.cwl.cwl2script', 'planemo.database', 'planemo.engine', 'planemo.galaxy', 'planemo.galaxy.test', 'planemo.linters', 'planemo.reports', 'planemo.shed', 'planemo.shed2tap', 'planemo.test', 'planemo.xml']), ('commands', ['LANG=C.UTF-8 planemo --help'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/planemo'), ('license', 'Academic Free License (AFL)'), ('summary', 'Command-line utilities to assist in building tools for the Galaxy project (http://galaxyproject.org/).'), ('license_family', 'Apache')]))])), 'plasmidtron': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plasmidtron/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/sanger-pathogens/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfastaq >=3.12.0', 'matplotlib', 'biopython']), ('run', ['python', 'pyfastaq >=3.12.0', 'matplotlib', 'biopython', 'kmc >=2.3.0', 'spades >=3.9.0'])])), ('test', CommentedMap([('commands', ['plasmidtron -h'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/plasmidtron'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'PlasmidTron: assembling the cause of phenotypes from NGS data')])), ('extra', CommentedMap([('recipe-maintainers', ['andrewjpage'])]))])), 'plastid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plastid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plastid'), ('version', '0.4.7')])), ('source', CommentedMap([('url', 'https://github.com/joshuagryphon/plastid/archive/v0.4.7.tar.gz'), ('md5', '1a1c750b9081d46fa688cf8fcbaa2ad3')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython >=0.22.0', 'numpy >=1.9.0', 'pysam >=0.8.4', 'biopython', 'termcolor', 'scipy >=0.15.1', 'pandas >=0.17.0', 'matplotlib >=1.4.0', 'biopython >=1.64', 'twobitreader >=3.0.0', 'zlib', 'gcc', 'llvm']), ('run', ['python', 'cython >=0.22.0', 'numpy >=1.9.0', 'pysam >=0.8.4', 'biopython', 'termcolor', 'scipy >=0.15.1', 'pandas >=0.17.0', 'matplotlib >=1.4.0', 'biopython >=1.64', 'twobitreader >=3.0.0', 'zlib', 'bowtie', 'ucsc-bedtobigbed', 'fastx_toolkit', 'libgcc'])])), ('test', CommentedMap([('imports', ['plastid']), ('commands', ['counts_in_region -h', 'cs -h', 'get_count_vectors -h', 'make_wiggle -h', 'metagene -h', 'phase_by_size -h', 'psite -h', 'crossmap -h', 'gff_parent_types -h', 'reformat_transcripts -h', 'findjuncs -h', 'slidejuncs -h'])])), ('about', CommentedMap([('home', 'http://plastid.readthedocs.io/en/latest/'), ('license', 'BSD 3-Clause'), ('summary', 'plastid is a Python library for genomic analysis -- in particular, high-throughput sequencing data')]))])), 'platypus-conquistador': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/platypus-conquistador/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'platypus-conquistador'), ('version', '0.9.0')])), ('source', CommentedMap([('fn', 'platypus-conquistador-0.9.0.zip'), ('url', 'https://pypi.python.org/packages/source/p/platypus-conquistador/platypus-conquistador-0.9.0.zip'), ('md5', '609e38711a1ebc7cb2b54a14c3a258f3')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'scikit-bio >=0.2.1,<0.3.0', 'click']), ('run', ['python', 'scikit-bio >=0.2.1,<0.3.0', 'click'])])), ('test', CommentedMap([('imports', ['platypus'])])), ('about', CommentedMap([('home', 'http://github.com/biocore/platypus'), ('license', 'BSD License'), ('summary', 'Platypus Conquistador: Confirming specific taxonomic groups within your metagenomic samples.')]))])), 'platypus-variant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/platypus-variant/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.well.ox.ac.uk/platypus'), ('license', 'GPLv3'), ('summary', 'A Haplotype-Based Variant Caller For Next Generation Sequence Data')])), ('package', CommentedMap([('name', 'platypus-variant'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'Platypus_<{ version }}.tgz'), ('url', 'https://github.com/chapmanb/Platypus/releases/download/<{ version }}/Platypus_<{ version }}.tgz'), ('md5', '84b1ef968bf638e357c447abab39c463')])), ('requirements', CommentedMap([('build', ['python', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['python', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['platypus callVariants -h'])]))])), 'plink': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plink/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plink'), ('version', '1.90b4')])), ('source', CommentedMap([('fn', '8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('url', 'https://github.com/chrchang/plink-ng/archive/8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('md5', '88290f567034153f48e8ca6bb595b3a8'), ('sha1', '9f41c77daa84c365aa659adf0619197fde1337bf'), ('sha256', 'af7bfa27bb303f9bf19b15d4a99e5e8f85de76013b54c05f3479385830598b4d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openblas', 'zlib']), ('run', ['libgcc', 'openblas', 'zlib'])])), ('test', CommentedMap([('commands', ['plink --help | grep PLINK'])])), ('about', CommentedMap([('home', 'https://www.cog-genomics.org/plink'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.')]))])), 'plink2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plink2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plink2'), ('version', '1.90b3.35')])), ('source', CommentedMap([('fn', '8990fc72295b299c1ace05266861884d319cf082.tar.gz'), ('url', 'https://github.com/chrchang/plink-ng/archive/8990fc72295b299c1ace05266861884d319cf082.tar.gz')])), ('requirements', CommentedMap([('build', ['openblas']), ('run', ['openblas'])])), ('test', CommentedMap([('commands', ['plink2 --help | grep \"PLINK v1.90p 64-bit (25 Mar 2016)\"'])])), ('about', CommentedMap([('home', 'https://www.cog-genomics.org/plink2'), ('license', 'GPL'), ('license_file', 'LICENSE'), ('summary', 'Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.')]))])), 'poa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/poa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'poa'), ('version', '2.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'poaV2.tar.gz'), ('md5', '9e2eb270d4867114406f53dab1311b2b'), ('url', 'https://downloads.sourceforge.net/project/poamsa/poamsa/2.0/poaV2.tar.gz'), ('patches', ['pscores.patch'])])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'blast-legacy'])])), ('test', CommentedMap([('commands', ['poa 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/poamsa'), ('license', 'GPLv2'), ('summary', 'POA is Partial Order Alignment, a fast program for multiple sequence alignment in bioinformatics. Its advantages are speed, scalability, sensitivity, and the superior ability to handle branching / indels in the alignment.')]))])), 'pomegranate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pomegranate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pomegranate'), ('version', '0.3.7')])), ('source', CommentedMap([('fn', 'pomegranate-0.3.7.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pomegranate/pomegranate-0.3.7.tar.gz'), ('md5', '28826d27bde9294478bc468509f5ec9d')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.22.1', 'numpy >=1.8.0', 'joblib >=0.9.0b4', 'networkx >=1.8.1', 'scipy']), ('run', ['python', 'cython >=0.22.1', 'numpy >=1.8.0', 'joblib >=0.9.0b4', 'networkx >=1.8.1', 'scipy', 'libgcc'])])), ('test', CommentedMap([('imports', ['pomegranate'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/pomegranate/'), ('license', 'LICENSE.txt'), ('summary', 'Pomegranate is a graphical models library for Python, implemented in Cython for speed.')]))])), 'popt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/popt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'popt'), ('version', 1.16)])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'popt-1.16.tar.gz'), ('url', 'ftp://anduin.linuxfromscratch.org/BLFS/popt/popt-1.16.tar.gz'), ('md5', '3743beefa3dd6247a73f8f7a32c14c33')])), ('test', CommentedMap([('commands', ['test -f $PREFIX/include/popt.h', 'test -f $PREFIX/lib/libpopt.a', 'test -f $PREFIX/lib/libpopt.so', 'test -f $PREFIX/lib/libpopt.dylib'])])), ('about', CommentedMap([('home', 'http://rpm5.org/'), ('license', 'MIT'), ('license_file', 'COPYING'), ('summary', 'Popt is a C library for parsing command line parameters.')]))])), 'poretools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/poretools/0.5.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'poretools'), ('version', '0.5.1')])), ('source', CommentedMap([('fn', 'poretools_v0.5.1.tar.gz'), ('url', 'https://github.com/arq5x/poretools/archive/v0.5.1.tar.gz'), ('md5', 'd246f30b6dd25c1e1580eb81cfd95b85')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'hdf5 >=1.8.7', 'h5py >=2.0', 'matplotlib', 'seaborn', 'pandas']), ('run', ['python', 'hdf5 >=1.8.7', 'h5py >=2.0', 'matplotlib', 'seaborn', 'pandas'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('test', CommentedMap([('commands', ['poretools -v'])])), ('about', CommentedMap([('home', 'http://poretools.readthedocs.org'), ('summary', 'poretools: a toolkit for working with nanopore sequencing data from Oxford Nanopore'), ('license', 'GPL')]))])), 'pout2mzid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pout2mzid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pout2mzid'), ('version', '0.3.03')])), ('source', CommentedMap([('git_url', 'https://github.com/percolator/pout2mzid.git'), ('git_rev', '5e390a7be408a2eee13b24aabb964c2a82b31933')])), ('build', CommentedMap([('number', 2), ('string', 'boost1.62_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'boost ==1.62', 'xerces-c', 'xsd']), ('run', ['libgcc', 'xerces-c', 'boost ==1.62', 'xsd'])])), ('test', CommentedMap([('commands', ['pout2mzid -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/percolator/pout2mzid'), ('summary', 'Adds percolator statistics to mzIdentML files that were used as input to percolator')]))])), 'pp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pp/1.6.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pp'), ('version', '1.6.4')])), ('source', CommentedMap([('fn', 'pp-1.6.4.zip'), ('url', 'http://www.parallelpython.com/downloads/pp/pp-1.6.4.zip'), ('md5', '0162fd2813d7218a4159430ca9e6f26a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pp'])])), ('about', CommentedMap([('home', 'http://www.parallelpython.com'), ('license', 'BSD License'), ('summary', 'Parallel and distributed programming for Python'), ('license_family', 'BSD')]))])), 'ppanini': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ppanini/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/10/56/61250c8e6ba48f76c1270a337f9985808e41aa13c377fd76b818abc6b488/ppanini-0.6.4.tar.gz'), ('md5', '8768da1bd4546fc78bf29c84e26491de')])), ('build', CommentedMap([('skip', True), ('entry_points', ['ppanini = ppanini.ppanini:_main', 'ppanini_abundance_table = ppanini.utils.preppanini:main', 'ppanini_eval_roc = ppanini.utils.plot_roc:main', 'ppanini_test = ppanini.tests.ppanini_test:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.2', 'scikit-learn  >=0.14.1', 'matplotlib >=1.1.1', 'biopython >=1.66']), ('run', ['python', 'numpy >=1.9.2', 'scikit-learn  >=0.14.1', 'matplotlib >=1.1.1', 'biopython >=1.66'])])), ('test', CommentedMap([('imports', ['ppanini', 'ppanini.tests', 'ppanini.tests.data', 'ppanini.utils']), ('commands', ['ppanini --help', 'ppanini_abundance_table --help'])])), ('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/ppanini'), ('license', 'MIT'), ('summary', 'PPANINI: Prioritization and Prediction of functional Annotations for Novel and Important genes via automated data Network Integration.')]))])), 'pplacer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pplacer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pplacer'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pplacer-linux-v<{ version }}.zip'), ('url', 'https://github.com/matsen/pplacer/releases/download/v<{ version }}/pplacer-linux-v<{ version }}.zip'), ('sha256', '3dc8e20fa8642d01daadde5bf9e36df2c180abec8f85c0b2f296f7852b63537c')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('test', CommentedMap([('commands', ['pplacer --help > /dev/null', 'guppy to_csv --help > /dev/null', 'rppr check --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://matsen.fredhutch.org/pplacer/'), ('license', 'GNU General Public License v3.0'), ('summary', 'Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment.')]))])), 'prank': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prank'), ('version', 'v.150803')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'prank.source.150803.tgz'), ('url', 'http://wasabiapp.org/download/prank/prank.source.150803.tgz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://wasabiapp.org/software/prank/'), ('license', 'GPL-3'), ('summary', 'PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences.')])), ('test', CommentedMap([('commands', ['prank -version'])]))])), 'preseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/preseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'preseq'), ('version', '2.0.2')])), ('source', CommentedMap([('url', 'https://github.com/smithlabcode/preseq/releases/download/v2.0.2/preseq_v2.0.2.tar.bz2'), ('fn', 'preseq_v2.0.2.tar.bz2'), ('md5', '9f2a7b597c9f08b821db6ee55e2ea39c'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'zlib', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['preseq', 'bam2mr'])])), ('about', CommentedMap([('home', 'https://github.com/smithlabcode/preseq'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('summary', 'Software for predicting library complexity and genome coverage in high-throughput sequencing')]))])), 'presto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/presto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'presto'), ('version', '0.5.4')])), ('source', CommentedMap([('fn', 'presto-0.5.4.tar.gz'), ('url', 'https://pypi.python.org/packages/ea/89/b2695b05f0e106f700227c742dd7bbebc0e42879c8748741f52919adffd1/presto-0.5.4.tar.gz'), ('md5', 'c58d69852a42fe01058fbbbf142bb4e6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.8', 'scipy >=0.14', 'pandas >=0.15', 'biopython >=1.65']), ('run', ['python', 'numpy >=1.8', 'scipy >=0.14', 'pandas >=0.15', 'biopython >=1.65', 'blast >=2.5', 'muscle >=3.8', 'vsearch >=2.3.2'])])), ('test', CommentedMap([('commands', ['AlignSets.py --version', 'AssemblePairs.py --version', 'BuildConsensus.py --version', 'ClusterSets.py --version', 'CollapseSeq.py --version', 'ConvertHeaders.py --version', 'EstimateError.py --version', 'FilterSeq.py --version', 'MaskPrimers.py --version', 'PairSeq.py --version', 'ParseHeaders.py --version', 'ParseLog.py --version', 'SplitSeq.py --version'])])), ('about', CommentedMap([('home', 'http://presto.readthedocs.io'), ('license', 'Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)'), ('summary', 'A bioinformatics toolkit for processing high-throughput lymphocyte receptor sequencing data.'), ('license_family', 'OTHER')]))])), 'primer3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/primer3/1.1.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/primer3/primer3/1.1.4/primer3-1.1.4.tar.gz'), ('md5', 'd9bbdb749a7eea25dd8c653e272d5f39')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'llvm']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', [\"primer3_core --help 2>&1 | grep 'This is primer3'\"])])), ('about', CommentedMap([('home', 'http://primer3.sourceforge.net/'), ('license', 'GPLv2'), ('license_file', 'COPYING.txt'), ('summary', 'Design PCR primers from DNA sequence. From mispriming libraries to sequence quality data to the generation of internal oligos, primer3 does it.')]))])), 'primer3-py': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/primer3-py/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/93/06/5c3a894ce9694d571c41fed35551d03880a32abee325aecbbd23fec1b858/<{name|lower}}-<{version}}.tar.gz'), ('md5', '6b231a06f8f7f2d2998ccb780adfae9d')])), ('build', CommentedMap([('skip', False), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['primer3'])])), ('about', CommentedMap([('home', 'https://libnano.github.io/primer3-py/'), ('license', 'GPLv2'), ('summary', 'Primer3-py is a Python-abstracted API for the popular Primer3 library.')]))])), 'primerprospector': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/primerprospector/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/pprospector/pprospector-1.0.1.tar.gz'), ('md5', 'a3bffcaa722f07cb6206226781254aca')])), ('requirements', CommentedMap([('build', ['python', 'cogent >=1.5', 'numpy >=1.3.0', 'matplotlib >=0.98.3']), ('run', ['python', 'cogent >=1.5', 'numpy >=1.3.0', 'matplotlib >=0.98.3'])])), ('test', CommentedMap([('commands', ['amplicons_histograms.py --help', 'analyze_primers.py --help', 'check_primer_barcode_dimers.py --help', 'clean_fasta.py --help', 'generate_linkers.py --help', 'generate_primers_denovo.py --help', 'get_amplicons_and_reads.py --help', 'make_pp_rst_file.py --help', 'optimize_primers.py --help', 'sort_denovo_primers.py --help', 'taxa_assignment_report.py --help', 'taxa_coverage.py --help'])])), ('about', CommentedMap([('home', 'http://pprospector.sourceforge.net/'), ('license', 'GPL'), ('summary', 'Primer Prospector is a pipeline of programs to design and analyze PCR primers.')]))])), 'prinseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prinseq/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://prinseq.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data')])), ('package', CommentedMap([('name', 'prinseq'), ('version', '0.20.4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('source', CommentedMap([('fn', 'prinseq-lite-0.20.4.tar.gz'), ('md5', '3be1a572073ebbbecfeba42a42853ff5'), ('url', 'http://iweb.dl.sourceforge.net/project/prinseq/standalone/prinseq-lite-0.20.4.tar.gz')])), ('requirements', CommentedMap([('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['prinseq-lite.pl -version'])]))])), 'probamconvert': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/probamconvert/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'probamconvert'), ('version', '1.0.2')])), ('about', CommentedMap([('home', 'https://github.com/Biobix/proBAMconvert'), ('summary', 'proBAMconvert is a conversion tool to convert common peptide identification files (mzIdentML, pepXML, mzTAB) to the proBAM or proBED format'), ('license', 'Apache License')])), ('source', CommentedMap([('url', 'https://github.com/Biobix/proBAMconvert/archive/a94913a16339ca58ae43a8e40d26acfd6725da40.tar.gz'), ('md5', '17d68ecec97405e8c2198dfd2e7d2383')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('run', ['python', 'cogent', 'pysam >=0.9', 'mysql-python', 'lxml', 'numpy', 'matplotlib', 'pyteomics', 'xz', 'bioservices', 'icu 58.*'])])), ('test', CommentedMap([('commands', ['proBAM.py --help'])]))])), 'prodigal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prodigal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prodigal'), ('version', '2.6.3')])), ('source', CommentedMap([('fn', 'Prodigal-2.6.3.zip'), ('url', 'https://github.com/hyattpd/Prodigal/archive/v2.6.3.zip'), ('mda5sum', '6eb6af4a91f811818261111890aca19c')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['prodigal -h'])])), ('about', CommentedMap([('home', 'http://prodigal.ornl.gov/'), ('license', 'GPL v3'), ('summary', 'Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program')]))])), 'prokka': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prokka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prokka'), ('version', '1.12')])), ('source', CommentedMap([('url', 'https://github.com/tseemann/prokka/archive/v1.12.tar.gz'), ('md5', '658c4c203ddded3623e68a36f94cabec')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['perl-bioperl ==1.6.924', 'blast >=2.6']), ('run', ['aragorn >=1.2', 'barrnap >=0.7', 'blast >=2.6', 'hmmer >=3.1b2', 'infernal >=1.1.2', 'minced >=0.2', 'parallel >=20130422', 'perl-bioperl ==1.6.924', 'perl-xml-simple', 'prodigal >=2.6', 'tbl2asn >=25.3', 'libidn'])])), ('test', CommentedMap([('commands', ['prokka --listdb', 'prokka --version', 'blastp -help', 'barrnap --help 2>&1 | grep Options', 'prodigal -h'])])), ('about', CommentedMap([('home', 'http://www.vicbioinformatics.com/software.prokka.shtml'), ('license', 'GPLv2'), ('summary', 'A tool for the rapid annotation of prokaryotic genomes')]))])), 'pronto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pronto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pronto'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'pronto-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/43/0b/abfdb921e0c869d4ee2f31b9e68cbc55ea07c57a45eb7054ad3b27b05965/pronto-0.3.3.tar.gz'), ('md5', '512eee78637add914a1b62195007aa2c')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools', 'lxml', 'six']), ('run', ['python >=2.7,<3', 'lxml', 'six'])])), ('test', CommentedMap([('imports', ['pronto', 'pronto.parser'])])), ('about', CommentedMap([('home', 'http://github.com/althonos/pronto'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Python frontend to ontologies - a library to parse, create, browse and export ontologies.')]))])), 'proot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/proot/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/proot-me/PRoot/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['proot_event.patch'])])), ('about', CommentedMap([('home', 'https://github.com/proot-me/PRoot'), ('license', 'GPL-2.0'), ('license_file', 'COPYING'), ('summary', 'chroot, mount --bind, and binfmt_misc without privilege/setup')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'talloc']), ('run', ['talloc'])])), ('test', CommentedMap([('commands', ['proot -V |grep proot.me >/dev/null'])]))])), 'prophyle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prophyle/0.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prophyle'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'prophyle-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/p/prophyle/prophyle-<{version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'samtools', 'ete3', 'bitarray', 'scipy', 'wheel', 'zlib', 'psutil', 'pysam']), ('run', ['python', 'samtools', 'ete3', 'bitarray', 'scipy', 'wheel', 'zlib', 'psutil', 'pysam'])])), ('test', CommentedMap([('commands', ['prophyle -h'])])), ('about', CommentedMap([('home', 'https://github.com/karel-brinda/prophyle'), ('license', 'MIT'), ('summary', 'ProPhyle is a metagenomic classifier based on BWT-index and phylogenetic trees. Its indexing strategy uses bottom-up propagation of k-mers in the tree, assembling contigs at each node and matching using a standard full-text search.')]))])), 'prosic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prosic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prosic'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('url', 'https://github.com/PROSIC/prosic2/archive/v<{ version }}.tar.gz'), ('fn', 'prosic-<{ version }}.tar.gz'), ('sha256', '575994be1c6f26d4966eedd44a18a5459e90cad7023dd2d329fbe463cb209b96')])), ('requirements', CommentedMap([('build', ['gcc 4.8.2*', 'rust >=1.12', 'gsl <{CONDA_GSL}}*', 'zlib']), ('run', ['gsl <{CONDA_GSL}}*', 'libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['prosic call-tumor-normal --help'])])), ('about', CommentedMap([('home', 'https://prosic.github.io'), ('license', 'GPLv3'), ('summary', 'A highly sensitive and accurate Bayesian caller for somatic insertions and deletions.')]))])), 'prosolo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prosolo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prosolo'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('url', 'https://github.com/PROSIC/prosolo/archive/v<{ version }}.tar.gz'), ('fn', 'prosolo-<{ version }}.tar.gz'), ('md5', 'f70464c825e1b8e20da7d4382dd53f65')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'rust >=1.18', 'gsl <{CONDA_GSL}}*', 'zlib']), ('run', ['gsl <{CONDA_GSL}}*', 'libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['prosolo single-cell-bulk --help'])])), ('about', CommentedMap([('home', 'https://github.com/PROSIC/prosolo/tree/v<{ version }}'), ('license', 'GPLv3'), ('summary', 'A highly sensitive and accurate Bayesian caller for variants in single cell sequencing data.')]))])), 'proteowizard': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/proteowizard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'proteowizard'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'http://teamcity.labkey.org/guestAuth/repository/download/bt81/365914:id/pwiz-src-<{ version }}.tar.bz2'), ('sha1', '4bf02eb83755b3df822a2d35b39d577be84f8282')])), ('build', CommentedMap([('script', 'bash -c \"./quickbuild.sh -j<{ CPU_COUNT }} address-model=64 runtime-link=shared executables && cp build-linux-x86_64/gcc-release-x86_64/* $PREFIX/bin\"'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://proteowizard.sourceforge.net'), ('license', 'Apache 2.0'), ('summary', 'Tools for dealing with mass spectrometry files (e.g., mzML, mzXML, mzIdentML, MGF)')])), ('test', CommentedMap([('commands', ['msconvert --help'])]))])), 'protrac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/protrac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'protrac'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'proTRAC_<{ version }}.pl'), ('url', 'http://www.smallrnagroup.uni-mainz.de/software/proTRAC_<{ version }}.pl'), ('sha256', '44f5ab6e05d6cf1ec42969bce654ce81b4ced1ec8bd70deae1338f2498fdc15f')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl', 'perl-gd'])])), ('test', CommentedMap([('commands', ['proTRAC_<{ version }}.pl | grep VERSION'])])), ('about', CommentedMap([('home', 'http://www.smallrnagroup.uni-mainz.de/software.html'), ('license', 'CC BY-NC 2.0'), ('summary', 'piRNA detection')]))])), 'pullseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pullseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pullseq'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', '1.0.2.tar.gz'), ('url', 'https://github.com/bcthomas/pullseq/archive/1.0.2.tar.gz'), ('md5', 'a4ef6e5b4b5c77be2ef5d444c44fb6c0')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pcre', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'libtool', 'zlib']), ('run', ['libgcc', 'pcre', 'zlib'])])), ('test', CommentedMap([('commands', ['pullseq --version 2>&1 | grep pullseq', 'seqdiff --help 2>&1 | grep seqdiff'])])), ('about', CommentedMap([('home', 'https://github.com/bcthomas/pullseq'), ('license', 'MIT'), ('license_file', 'COPYING'), ('summary', 'Utility program for extracting sequences from a fasta/fastq file.')]))])), 'pvacseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pvacseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pvacseq'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/griffithlab/pVAC-Seq/archive/v<{ version }}.tar.gz'), ('sha256', 'b218a8b5eaf2966321cc9be58c14060f908226f07105e3fcddc82893e31c5357')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install --single-version-externally-managed --record=record.txt'), ('entry_points', ['pvacseq = pvacseq.pvacseq:main', 'pvacseq-ui = pvacseq.server.app:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'pyvcf', 'requests', 'pyyaml', 'pandas', 'connexion'])])), ('test', CommentedMap([('imports', ['pvacseq', 'pvacseq.lib', 'pvacseq.server']), ('commands', ['pvacseq --version'])])), ('about', CommentedMap([('home', 'https://github.com/griffithlab/pVAC-Seq'), ('license', 'NPOSL-3.0'), ('summary', 'Personalized Variant Antigens by Cancer Sequencing (pVAC-Seq)'), ('license_family', 'Other'), ('doc_url', 'http://pvac-seq.readthedocs.io/')]))])), 'py-graphviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/py-graphviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'py-graphviz'), ('version', '0.4.10')])), ('source', CommentedMap([('fn', 'graphviz-0.4.8.zip'), ('url', 'https://pypi.python.org/packages/source/g/graphviz/graphviz-0.4.10.zip'), ('md5', 'dd1e505934f9a2632084a5e2d29b08a3')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['graphviz'])])), ('about', CommentedMap([('home', 'http://github.com/xflr6/graphviz'), ('license', 'MIT License'), ('summary', 'Simple Python interface for Graphviz')]))])), 'py2bit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/py2bit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'py2bit'), ('version', '0.2.1')])), ('source', CommentedMap([('fn', 'py2bit-0.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/b2/ad/72d0d1cf2a588d9d2b16f5e531063aa33d1c80bf424e810fc1dfe5c5dc72/py2bit-0.2.1.tar.gz'), ('md5', 'eaf5b1c80a0bbf0b35af1f002f83a556')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['py2bit', 'py2bitTest'])])), ('about', CommentedMap([('home', 'https://github.com/dpryan79/py2bit'), ('license', 'MIT'), ('summary', 'A package for accessing 2bit files using lib2bit')]))])), 'pyalveo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyalveo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyalveo'), ('version', '0.7')])), ('source', CommentedMap([('fn', 'pyalveo-0.7.tar.gz'), ('url', 'https://pypi.python.org/packages/90/1c/6c92056e5937d77071df55cc74b7a92cfe0b43a77f34f1bc1c2a5d95056e/pyalveo-0.7.tar.gz'), ('md5', '462a440ae11dde729255596a501c2a5b')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'python-dateutil', 'requests']), ('run', ['python', 'python-dateutil', 'requests'])])), ('test', CommentedMap([('imports', ['pyalveo'])])), ('about', CommentedMap([('home', 'https://github.com/Alveo/pyalveo'), ('license', 'BSD'), ('summary', 'A Python library for interfacing with the Alveo API')]))])), 'pyani': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyani/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/widdowquinn/<{ name|lower }}/archive/v<{ version }}.tar.gz'), ('md5', 'f4da6989c63b6b712ddddcd5bac902de')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'matplotlib', 'pandas', 'seaborn', 'scipy', 'numpy', 'mummer', 'blast', 'blast-legacy']), ('run', ['python', 'biopython', 'matplotlib', 'pandas', 'seaborn', 'scipy', 'numpy', 'mummer', 'blast', 'blast-legacy'])])), ('test', CommentedMap([('imports', ['pyani']), ('commands', ['average_nucleotide_identity.py -h', 'genbank_get_genomes_by_taxon.py -h'])])), ('about', CommentedMap([('home', 'http://widdowquinn.github.io/pyani/'), ('license', 'MIT License'), ('summary', 'pyani provides a package and script for calculation of genome-scale average nucleotide identity.'), ('license_family', 'MIT')]))])), 'pybedtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pybedtools/0.6.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pybedtools'), ('version', '0.6.9')])), ('source', CommentedMap([('url', 'https://github.com/daler/pybedtools/archive/v0.6.9.tar.gz'), ('fn', 'v0.6.9.tar.gz'), ('md5', 'a24e4bcd0c89beb9535295db964f6a4a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'six', 'samtools', 'setuptools', 'pandas', 'cython', 'gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['python', 'bedtools', 'six', 'samtools', 'pandas', 'libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['pybedtools']), ('commands', ['python -c \"import pybedtools; help(pybedtools.BedTool.intersect)\" | grep \"Original BEDTools help\"'])])), ('about', CommentedMap([('home', 'https://github.com/daler/pybedtools'), ('license', 'MIT'), ('summary', 'Wraps BEDTools for use in Python and adds many additional features.')]))])), 'pybel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pybel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pybel'), ('version', '0.5.4')])), ('source', CommentedMap([('fn', 'PyBEL-0.5.4.tar.gz'), ('url', 'https://pypi.python.org/packages/a0/78/11e54154928e59ac4ddd9a0ad1aa3a298e2bf2c1c2d75716b7dd228f8fe2/PyBEL-0.5.4.tar.gz'), ('md5', '3cc9adac52db1b8605f6313475e06b11')])), ('build', CommentedMap([('entry_points', ['pybel = pybel.cli:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'requests-file', 'networkx', 'pyparsing', 'configparser', 'py2neo', 'click', 'sqlalchemy', 'onto2nx']), ('run', ['python', 'requests', 'requests-file', 'networkx', 'pyparsing', 'configparser', 'py2neo', 'click', 'sqlalchemy', 'onto2nx'])])), ('test', CommentedMap([('imports', ['pybel', 'pybel.ext', 'pybel.manager', 'pybel.parser', 'pybel.parser.modifiers']), ('commands', ['LANG=C.UTF-8 pybel --help'])])), ('about', CommentedMap([('home', 'https://github.com/pybel/pybel'), ('license', 'Apache Software License'), ('summary', 'Parsing, validation, and analysis of BEL graphs')]))])), 'pybigwig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pybigwig/0.1.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pybigwig'), ('version', '0.1.11')])), ('source', CommentedMap([('fn', 'pyBigWig-0.1.11.tar.gz'), ('url', 'https://github.com/dpryan79/pyBigWig/archive/0.1.11.tar.gz'), ('sha256', '7b2d5897da5761b125e6a6efbaebe60ee81996546689ecd4b1493cc5ee14ae07')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'curl']), ('run', ['python', 'curl']), ('test', None), ('imports', ['pyBigWig'])])), ('about', CommentedMap([('home', 'https://github.com/dpryan79/pyBigWig'), ('license', 'MIT'), ('summary', 'A python extension written in C for quick access to bigWig files.')]))])), 'pycli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pycli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pycli'), ('version', '2.0.3')])), ('source', CommentedMap([('fn', 'pyCLI-2.0.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyCLI/pyCLI-2.0.3.tar.gz'), ('md5', '89c487e7c8068ce05d63394d2ae7ebad')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['cli', 'cli._ext', 'cli.tests', 'cli.tests.functional'])])), ('about', CommentedMap([('home', 'http://packages.python.org/pyCLI/'), ('license', 'MIT'), ('summary', 'Simple, object-oriented approach to Python CLI apps')]))])), 'pycluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pycluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pycluster'), ('version', '1.52')])), ('source', CommentedMap([('url', 'http://bonsai.hgc.jp/~mdehoon/software/cluster/Pycluster-1.52.tar.gz'), ('fn', 'Pycluster-1.52.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['Pycluster'])])), ('about', CommentedMap([('home', 'http://bonsai.hgc.jp/~mdehoon/software/cluster/software.htm#pycluster'), ('license', 'MIT'), ('summary', 'Clustering module for Python')]))])), 'pycoqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pycoqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/EnrightLab/pycoQC/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'pandas', 'matplotlib', 'seaborn', 'notebook']), ('run', ['python', 'numpy', 'pandas', 'matplotlib', 'seaborn', 'notebook'])])), ('test', CommentedMap([('imports', ['pycoQC'])])), ('about', CommentedMap([('home', 'https://github.com/EnrightLab/pycoQC'), ('license', 'GPL-3.0'), ('summary', 'Python 3 package for Jupyter Notebook, computing metrics and generating plots from Oxford Nanopore Albacore report')]))])), 'pycsg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pycsg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pycsg'), ('version', '0.3.12')])), ('about', CommentedMap([('home', 'https://github.com/pletzer/pycsg'), ('summary', 'Python port of csg.js from Evan Wallace'), ('license', 'MIT')])), ('source', CommentedMap([('git_url', 'https://github.com/pletzer/pycsg'), ('git_rev', 'cee697f')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('imports', ['csg'])])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])]))])), 'pydna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pydna/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/BjornFJohansson/<{ name|lower }}/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython >=1.65', 'networkx >=1.8.1', 'appdirs >=1.3.0', 'prettytable >=0.7.2', 'ordered-set >=2.0.1', 'pyparsing >=2.1.4', 'requests >=2.12', 'pytest-runner']), ('run', ['python', 'biopython >=1.65', 'networkx >=1.8.1', 'appdirs >=1.3.0', 'prettytable >=0.7.2', 'ordered-set >=2.0.1', 'pyparsing >=2.1.4', 'requests >=2.12', 'pytest-runner'])])), ('test', CommentedMap([('imports', ['pydna'])])), ('about', CommentedMap([('home', 'https://github.com/BjornFJohansson/pydna'), ('license', 'BSD-3-Clause'), ('license_family', 'BSD'), ('license_file', 'LICENSE.txt'), ('summary', 'Representing double stranded DNA and simulating cloning, homologous recombination, Gibson assembly, Gel electrophoresis etc.')]))])), 'pyexcelerator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyexcelerator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyexcelerator'), ('version', '0.6.4a')])), ('source', CommentedMap([('fn', 'pyexcelerator-0.6.4.1.tar.bz2'), ('url', 'https://pypi.python.org/packages/09/21/556398d15af938ac28e8f804f840949f833340880883a31ee8c9b533ae0b/pyexcelerator-0.6.4.1.tar.bz2'), ('md5', '478c79a74be39d27eee9bd1b3032dffe')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pyExcelerator'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/pyexcelerator/'), ('license', 'BSD License'), ('summary', 'generating Excel 97+ files; importing Excel 95+ files; Excel files dumper; OLE2 files dumper; xls2txt, xls2csv, xls2html'), ('license_family', 'BSD')]))])), 'pyfaidx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfaidx/0.4.8.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfaidx'), ('version', '0.4.8.1')])), ('source', CommentedMap([('fn', 'pyfaidx-0.4.8.1.tar.gz'), ('url', 'https://pypi.python.org/packages/5f/0e/b22911d548b5fe043629675fccd7c0b8f96b76b0ec0c72a484f4f0615c42/pyfaidx-0.4.8.1.tar.gz'), ('md5', 'f33beb242463eed92381571da4a3aa56')])), ('build', CommentedMap([('skip', False), ('entry_points', ['faidx = pyfaidx.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pyfaidx']), ('commands', ['faidx --help'])])), ('about', CommentedMap([('home', 'http://mattshirley.com'), ('license', 'MIT License'), ('summary', 'pyfaidx: efficient pythonic random access to fasta subsequences')]))])), 'pyfasta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfasta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfasta'), ('version', '0.5.2')])), ('source', CommentedMap([('fn', 'pyfasta-0.5.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyfasta/pyfasta-0.5.2.tar.gz'), ('md5', 'bf61ab997dca329675c3eb2ee7cdfcf2')])), ('build', CommentedMap([('skip', False), ('entry_points', ['pyfasta = pyfasta:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['pyfasta']), ('commands', ['pyfasta extract --help | grep Usage > /dev/null']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://github.com/brentp/pyfasta/'), ('license', 'MIT'), ('summary', 'fast, memory-efficient, pythonic (and command-line) access to fasta sequence files')]))])), 'pyfastaq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfastaq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfastaq'), ('version', '3.14.0')])), ('source', CommentedMap([('fn', 'pyfastaq-3.14.0.tar.gz'), ('url', 'https://pypi.python.org/packages/0e/5d/8b39442b62c43da835c89f4c244d037bc7fcd8b47b0c0fff6e8d9097a035/pyfastaq-3.14.0.tar.gz'), ('md5', '9f7e0e76400dad31679b22934dc2e7bf')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pyfastaq', 'pyfastaq.runners']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/Fastaq'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Script to manipulate FASTA and FASTQ files, plus API for developers'), ('license_family', 'GPL')]))])), 'pygraphviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pygraphviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pygraphviz'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'pygraphviz-1.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/98/bb/a32e33f7665b921c926209305dde66fe41003a4ad934b10efb7c1211a419/pygraphviz-1.3.1.tar.gz'), ('md5', '7f690295dfe77edaa9e552d09d98d279')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'graphviz']), ('run', ['python', 'graphviz'])])), ('test', CommentedMap([('imports', ['pygraphviz', 'pygraphviz.tests']), ('requires', ['nose >=0.10.1'])])), ('about', CommentedMap([('home', 'http://pygraphviz.github.io'), ('license', 'BSD License'), ('summary', 'Python interface to Graphviz'), ('license_family', 'BSD')]))])), 'pygresql': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pygresql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pygresql'), ('version', '5.0.1')])), ('source', CommentedMap([('fn', 'PyGreSQL-5.0.1.zip'), ('url', 'https://pypi.python.org/packages/25/f7/dd8a191de9a7f90b8b262ecf54bff31ca2422b4d564f0ab57e39924aea21/PyGreSQL-5.0.1.zip'), ('md5', 'ccc6b5f37e585d69cab21a6199b9d680')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'postgresql']), ('run', ['python', 'postgresql'])])), ('test', CommentedMap([('imports', ['pg'])])), ('about', CommentedMap([('home', 'http://www.pygresql.org'), ('license', 'Python Software Foundation License'), ('summary', 'Python PostgreSQL Interfaces')]))])), 'pyhashxx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyhashxx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyhashxx'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'pyhashxx-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/c4/a9/630e7c00c4766b0fa03ca9c4c1c3774ab2fa52646711918632daa403b52c/pyhashxx-0.1.3.tar.gz'), ('md5', 'b4dbe8f1cd964628744ac5704fc05f5e')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pyhashxx'])])), ('about', CommentedMap([('home', 'http://github.com/ewencp/pyhashxx'), ('license', 'BSD'), ('summary', 'Python wrapper for xxHash algorithm'), ('license_family', 'BSD')]))])), 'pylibseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pylibseq/0.1.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pylibseq'), ('version', '0.1.8')])), ('source', CommentedMap([('fn', 'pylibseq-0.1.8.tar.gz'), ('url', 'https://pypi.python.org/packages/61/a3/2777266d22ace0f461bb323d6ad2a47d67bcaff68b203ff1b0637adc77da/pylibseq-0.1.8.tar.gz'), ('md5', '53ceb50b9c454a2a23890319d06fd57f')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'libsequence', 'gcc']), ('run', ['python', 'libsequence', 'libgcc'])])), ('test', CommentedMap([('imports', ['libsequence'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/pylibseq'), ('license', 'GNU Lesser General Public License v2 or later (LGPLv2+)'), ('summary', 'A Python interface to libsequence')]))])), 'pyliftover': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyliftover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyliftover'), ('version', '0.3')])), ('source', CommentedMap([('fn', 'pyliftover-0.3.zip'), ('url', 'https://pypi.python.org/packages/95/1a/85ca6e7cbfcc6a3d0e480d446791ff56388235ffc9c922a8067994b5f44f/pyliftover-0.3.zip'), ('md5', 'db3a4cf5fbfce534d8978dac66f2f039')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pyliftover'])])), ('about', CommentedMap([('home', 'https://github.com/konstantint/pyliftover'), ('license', 'MIT License'), ('summary', 'Pure-python implementation of UCSC ``liftOver`` genome coordinate conversion.'), ('license_family', 'MIT')]))])), 'pyloh': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyloh/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyloh'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'pyloh-1.4.1.tar.gz'), ('url', 'https://github.com/uci-cbcl/PyLOH/archive/1.4.1.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'scipy', 'matplotlib', 'pysam'])])), ('test', CommentedMap([('imports', ['pyloh'])])), ('about', CommentedMap([('home', 'https://github.com/uci-cbcl/PyLOH'), ('license', 'GPL'), ('summary', 'Deconvolving tumor purity and ploidy by integrating copy number alterations and loss of heterozygosity')]))])), 'pymix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymix/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '<{ name }}-<{ version }}a.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://www.pymix.org/<{ name }}-<{ version }}a.tar.gz'), ('patches', ['setup.py.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'numpy', 'gsl']), ('run', ['libgcc', 'python', 'numpy', 'gsl', 'ghmm', 'matplotlib >=1.1.0,!=1.4.2,<1.5.0', 'glib', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau'])])), ('test', CommentedMap([('imports', ['mixture', 'mixtureHMM', 'mixtureunittests', 'alphabet', 'plotMixture', 'setPartitions'])])), ('about', CommentedMap([('home', 'http://www.pymix.org/pymix'), ('license', 'GPL2'), ('summary', 'Python mixture package')]))])), 'pymot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pymot'), ('version', '<{ PYMOT_VERSION }}')])), ('source', CommentedMap([('fn', 'pymot-master.zip'), ('url', 'https://github.com/Videmo/pymot/archive/master.zip'), ('md5', '3641a81623d0bd37689abe69fd9ed924')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['setuptools', 'python']), ('run', ['python', 'munkres'])])), ('test', CommentedMap([('imports', ['pymot'])])), ('about', CommentedMap([('home', 'https://github.com/Videmo/pymot'), ('license', 'All Rights Reserved'), ('summary', 'This is a python implementation which determines the MOTP and MOTA metrics from a set of ground truth tracks and a set of hypothesis tracks given by the tracker to be evaluated.')]))])), 'pymummer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymummer/0.10.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pymummer'), ('version', '0.10.1')])), ('source', CommentedMap([('fn', 'pymummer-0.10.1.tar.gz'), ('url', 'https://pypi.python.org/packages/96/04/a67728a727a8214de494b06178bfaca025550156889953d581a141976ec0/pymummer-0.10.1.tar.gz'), ('md5', '1a16e2d27b58f6fdc47f648f977fafd0')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfastaq >=3.10.0', 'mummer']), ('run', ['python', 'pyfastaq >=3.10.0', 'mummer'])])), ('test', CommentedMap([('imports', ['pymummer']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/pymummer'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Wrapper for MUMmer')]))])), 'pymzml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymzml/0.7.7/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pymzml'), ('version', '0.7.7')])), ('source', CommentedMap([('fn', 'pymzML-0.7.7.zip'), ('url', 'https://pypi.python.org/packages/da/35/cbcc508d55f68927c1fb6574b45f7103fa0817d269c770e4bd7a6c52509c/pymzML-0.7.7.zip'), ('md5', '3fb7e7799983e7616257a5e96cfb5911')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pymzml'])])), ('about', CommentedMap([('home', 'http://pymzml.github.com'), ('license', 'GNU General Public License (GPL)'), ('summary', 'high-throughput mzML parsing'), ('license_family', 'LGPL')]))])), 'pynacl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pynacl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pynacl'), ('version', '0.3.0')])), ('source', CommentedMap([('fn', 'PyNaCl-0.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/1a/a1/752a313b753e694c1efa2657143e734fe3fd8368a55bff1cc3afce47a4c7/PyNaCl-0.3.0.tar.gz'), ('md5', '04dc891f6b95e3defa8a35ade57559c9')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cffi >=0.8', 'six']), ('run', ['python', 'cffi >=0.8', 'six'])])), ('test', CommentedMap([('imports', ['nacl', 'nacl._lib', 'nacl.bindings']), ('requires', ['pytest'])])), ('about', CommentedMap([('home', 'https://github.com/pyca/pynacl/'), ('license', 'Apache License 2.0'), ('summary', 'Python binding to the Networking and Cryptography (NaCl) library')]))])), 'pyomo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyomo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyomo'), ('version', '4.1.10527')])), ('source', CommentedMap([('fn', 'Pyomo-4.1.10527.tar.gz'), ('url', 'https://pypi.python.org/packages/source/P/Pyomo/Pyomo-4.1.10527.tar.gz'), ('md5', 'c7c6b933edd5fd667cafcffa04da5b82')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['pyutilib', 'python', 'setuptools', 'ply', 'nose', 'six >=1.6.1']), ('run', ['pyutilib', 'python', 'ply', 'nose', 'six >=1.6.1'])])), ('test', CommentedMap([('imports', ['pyomo', 'pyomo.bilevel', 'pyomo.bilevel.plugins', 'pyomo.bilevel.tests', 'pyomo.checker', 'pyomo.checker.plugins', 'pyomo.checker.plugins.checkers', 'pyomo.checker.plugins.checkers.model', 'pyomo.checker.plugins.checkers.py3k', 'pyomo.checker.plugins.checkers.sample', 'pyomo.checker.tests', 'pyomo.core', 'pyomo.core.base', 'pyomo.core.data', 'pyomo.core.plugins', 'pyomo.core.plugins.data', 'pyomo.core.plugins.transform', 'pyomo.core.preprocess', 'pyomo.core.tests', 'pyomo.core.tests.data', 'pyomo.core.tests.diet', 'pyomo.core.tests.examples', 'pyomo.core.tests.transform', 'pyomo.core.tests.unit', 'pyomo.dae', 'pyomo.dae.plugins', 'pyomo.dae.tests', 'pyomo.data', 'pyomo.environ', 'pyomo.environ.tests', 'pyomo.gdp', 'pyomo.gdp.plugins', 'pyomo.gdp.tests', 'pyomo.mpec', 'pyomo.mpec.plugins', 'pyomo.mpec.tests', 'pyomo.neos', 'pyomo.neos.plugins', 'pyomo.openopt', 'pyomo.openopt.plugins', 'pyomo.openopt.tests', 'pyomo.opt', 'pyomo.opt.base', 'pyomo.opt.blackbox', 'pyomo.opt.parallel', 'pyomo.opt.plugins', 'pyomo.opt.problem', 'pyomo.opt.results', 'pyomo.opt.solver', 'pyomo.opt.testing', 'pyomo.opt.tests', 'pyomo.opt.tests.base', 'pyomo.opt.tests.blackbox', 'pyomo.os', 'pyomo.os.plugins', 'pyomo.os.tests', 'pyomo.pysp', 'pyomo.pysp.plugins', 'pyomo.pysp.scenariotree', 'pyomo.pysp.tests', 'pyomo.pysp.tests.examples', 'pyomo.pysp.tests.unit', 'pyomo.pysp.util', 'pyomo.repn', 'pyomo.repn.plugins', 'pyomo.repn.plugins.ampl', 'pyomo.repn.tests', 'pyomo.repn.tests.ampl', 'pyomo.scripting', 'pyomo.scripting.plugins', 'pyomo.solvers', 'pyomo.solvers.plugins', 'pyomo.solvers.plugins.converter', 'pyomo.solvers.plugins.smanager', 'pyomo.solvers.plugins.solvers', 'pyomo.solvers.plugins.testdriver', 'pyomo.solvers.tests', 'pyomo.solvers.tests.io', 'pyomo.solvers.tests.io.fd', 'pyomo.solvers.tests.mip', 'pyomo.solvers.tests.pickling', 'pyomo.util', 'pyomo.util.tests', 'pyomo.version', 'pyomo.version.tests'])])), ('about', CommentedMap([('home', 'http://pyomo.org'), ('license', 'BSD License'), ('summary', 'Pyomo: Python Optimization Modeling Objects')]))])), 'pyopt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyopt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyopt'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'pyOpt-1.2.0.tar.gz'), ('url', 'http://www.pyopt.org/_downloads/pyOpt-1.2.0.tar.gz'), ('md5', '84f7574cb40919009312ad52d30109db')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.0', 'swig >=1.3', 'mpi4py']), ('run', ['python', 'numpy >=1.0', 'mpi4py'])])), ('about', CommentedMap([('home', 'http://www.pyopt.org/index.html'), ('license', 'LGPLv3'), ('license_file', 'LICENSE')]))])), 'pypeflow': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pypeflow/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/pypeFLOW'), ('license', 'BSD'), ('summary', 'Light weight and reusable make / flow data process library written in Python')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/PacificBiosciences/pypeFLOW/archive/v<{ version }}.tar.gz'), ('md5', '2856278a331ec041fa8cb690412889fa')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'rdfextras']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pypeflow'])]))])), 'pypore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pypore/0.0.5.dev20160304220337/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'pypore'), ('version', '0.0.5.dev20160304220337')])), ('source', CommentedMap([('fn', 'pypore-0.0.5.dev20160304220337.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pypore/pypore-0.0.5.dev20160304220337.tar.gz'), ('md5', '173ebb6a8ca721630468ffd5940029fd')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['pypore', 'pypore.filetypes', 'pypore.filetypes.tests', 'pypore.i_o', 'pypore.i_o.tests', 'pypore.sampledata', 'pypore.sampledata.tests', 'pypore.strategies', 'pypore.strategies.tests', 'pypore.tests', 'pyporegui', 'pyporegui.graphicsItems', 'pyporegui.graphicsItems.tests', 'pyporegui.tests', 'pyporegui.widgets', 'pyporegui.widgets.tests']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://parkin.github.io/pypore/'), ('license', 'Apache License, Version 2.0'), ('summary', 'Pythonic/Cythonic Nanopore Translocation Analysis')]))])), 'pyprophet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyprophet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyprophet'), ('version', '0.22.0')])), ('source', CommentedMap([('fn', 'pyprophet-0.22.0.tar.gz'), ('url', 'https://pypi.python.org/packages/0b/b5/4afe0b5b6ebac3755c2d27a58aeac39d57168d5246a0d52f32752cc46c45/pyprophet-0.22.0.tar.gz'), ('md5', '8e2e0445f161b69514cc9284e89305e4')])), ('build', CommentedMap([('entry_points', ['pyprophet=pyprophet.main:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.0', 'pandas >=0.17', 'scipy >=0.9.0', 'numexpr >=2.1', 'scikit-learn >=0.17', 'matplotlib', 'seaborn']), ('run', ['python', 'numpy >=1.9.0', 'pandas >=0.17', 'scipy >=0.9.0', 'numexpr >=2.1', 'scikit-learn >=0.17', 'matplotlib', 'seaborn'])])), ('test', CommentedMap([('imports', ['pyprophet']), ('commands', ['pyprophet --help'])])), ('about', CommentedMap([('home', 'http://github.com/uweschmitt/pyprophet'), ('license', 'BSD License'), ('summary', 'Python reimplementation of mProphet peak scoring'), ('license_family', 'BSD')]))])), 'pyquant-ms': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyquant-ms/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('<{ hash_type }}', '<{ hash_value }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('entry_points', ['pyQuant = pyquant.command_line:run_pyquant']), ('script', 'python setup.py install  --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'scipy >=0.18.*', 'patsy', 'pythomics >=0.3.41', 'pandas', 'lxml', 'scikit-learn', 'simplejson']), ('run', ['python', 'cython', 'numpy', 'scipy >=0.18.*', 'patsy', 'pythomics >=0.3.41', 'pandas', 'lxml', 'scikit-learn', 'simplejson'])])), ('test', CommentedMap([('imports', ['pyquant', 'pyquant.tests']), ('commands', ['pyQuant --help'])])), ('about', CommentedMap([('home', 'http://www.github.com/pandeylab/pyquant'), ('license', 'MIT License'), ('license_family', 'MIT'), ('summary', 'A framework for the analysis of quantitative mass spectrometry data')]))])), 'pyquery': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyquery'), ('version', '1.2.9')])), ('source', CommentedMap([('fn', '1.2.9.tar.gz'), ('url', 'https://github.com/gawel/pyquery/archive/1.2.9.tar.gz'), ('md5', '902e2ded38899c7c3f66cba6d4a464fb')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'setuptools', 'lxml >=2.1', 'libxslt', 'cssselect >0.7.9']), ('run', ['libgcc', 'python', 'libxslt', 'lxml >=2.1', 'cssselect >0.7.9'])])), ('test', CommentedMap([('imports', ['pyquery'])])), ('extra', CommentedMap([('skip-lints', ['uses_setuptools', 'setup_py_install_args'])])), ('about', CommentedMap([('home', 'https://github.com/gawel/pyquery'), ('license', 'BSD'), ('summary', 'A jquery-like library for python'), ('license_family', 'BSD')]))])), 'pyrad': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyrad/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyrad'), ('version', '3.0.64')])), ('source', CommentedMap([('fn', '3.0.64.tar.gz'), ('url', 'https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz'), ('sha256', '45fd47819489df150427d5e28b8296b7aebae2281b8560412977668475c65796')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'six', 'muscle', 'vsearch']), ('run', ['python', 'numpy', 'scipy', 'six', 'muscle', 'vsearch'])])), ('test', CommentedMap([('imports', ['pyrad']), ('commands', ['pyrad -h'])])), ('about', CommentedMap([('home', 'https://github.com/wichert/pyrad'), ('license', 'GPLv3'), ('summary', 'Assembly and analysis of RADseq data sets')]))])), 'pysam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysam/0.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'pysam-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/40/15/20b22dc3d017ec123e533d062b982b111b0214168905de3221b5caf5f766/pysam-0.9.1.tar.gz'), ('md5', 'd87765ad1e0afe534255c20cce5478e4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'cython', 'python', 'setuptools', 'zlib', 'curl']), ('run', ['libgcc', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'python', 'zlib', 'curl'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', 'Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.')]))])), 'pysamstats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysamstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysamstats'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'v1.0.1.tar.gz'), ('url', 'https://github.com/alimanfoo/pysamstats/archive/v1.0.1.tar.gz'), ('md5', 'db8a5ad0db0027f55fd954a5510a797a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'cython', 'pysam >=0.11', 'zlib', 'gcc', 'numpy', 'pytables', 'nose']), ('run', ['python', 'pysam >=0.11', 'zlib', 'libgcc', 'numpy', 'pytables'])])), ('test', CommentedMap([('imports', ['pysamstats']), ('commands', ['pysamstats --help'])])), ('about', CommentedMap([('home', 'https://github.com/alimanfoo/pysamstats'), ('license', 'MIT'), ('summary', 'A Python utility for calculating statistics against genome position based on sequence alignments from a SAM, BAM or CRAM file.')]))])), 'pyscaf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyscaf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyscaf'), ('version', '0.12a4')])), ('source', CommentedMap([('fn', 'pyScaf-0.12a4.tar.gz'), ('url', 'https://pypi.python.org/packages/ee/52/a947347d00c323a87588d6b6d5ad54b3656a5df2f3bcaad477833a43d1f6/pyScaf-0.12a4.tar.gz'), ('md5', 'c67526747eb04d1e28279ac310916d40')])), ('build', CommentedMap([('entry_points', ['pyScaf = pyScaf:main']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'fastaindex']), ('run', ['python', 'fastaindex'])])), ('test', CommentedMap([('commands', ['pyScaf --help'])])), ('about', CommentedMap([('home', 'https://github.com/lpryszcz/pyScaf'), ('license', 'GPLv3'), ('summary', 'Genome assembly scaffolding using information from paired-end/mate-pair libraries, long reads, and synteny to closely related species.'), ('license_family', 'GPL3')]))])), 'pysftp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysftp/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'pysftp'), ('version', '0.2.9')])), ('source', CommentedMap([('fn', 'pysftp-0.2.9.tar.gz'), ('url', 'https://pypi.python.org/packages/36/60/45f30390a38b1f92e0a8cf4de178cd7c2bc3f874c85430e40ccf99df8fe7/pysftp-0.2.9.tar.gz'), ('md5', 'abc55b7122de3e86f0d547301f4ddf0c')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'paramiko >=1.7.7']), ('run', ['python', 'paramiko >=1.7.7'])])), ('test', CommentedMap([('imports', ['pysftp'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/dundeemt/pysftp'), ('license', 'BSD License'), ('summary', 'A friendly face on SFTP')]))])), 'pysnptools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysnptools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysnptools'), ('version', '0.3.9')])), ('source', CommentedMap([('fn', 'pysnptools-0.3.9.zip'), ('url', 'https://pypi.python.org/packages/source/p/pysnptools/pysnptools-0.3.9.zip'), ('md5', 'ddcc7742930170fb1a240faca39bb9c5')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.15.1', 'numpy >=1.9.2', 'pandas >=0.16.2']), ('run', ['python', 'scipy >=0.15.1', 'numpy >=1.9.2', 'pandas >=0.16.2'])])), ('test', CommentedMap([('imports', ['pysnptools', 'pysnptools.kernelreader', 'pysnptools.kernelstandardizer', 'pysnptools.pstreader', 'pysnptools.snpreader', 'pysnptools.standardizer', 'pysnptools.util'])])), ('about', CommentedMap([('home', 'http://research.microsoft.com/en-us/um/redmond/projects/mscompbio/'), ('license', 'Apache 2.0'), ('summary', 'Python library for reading and manipulating genetic data')]))])), 'pysvg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysvg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysvg'), ('version', '0.2.2')])), ('source', CommentedMap([('fn', 'pysvg-0.2.2.zip'), ('url', 'https://pypi.python.org/packages/49/2e/4ea6adcffa317856f9d07cbf31cc0cd8befbc3e138e19fb4c33b037300b3/pysvg-0.2.2.zip'), ('md5', '9f9b637f5d7205a200d00856df1b2ae2')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'lxml >=2.1', 'cssselect'])])), ('test', CommentedMap([('imports', ['pysvg'])])), ('about', CommentedMap([('home', 'http://codeboje.de/pysvg/'), ('license', 'BSD License'), ('summary', 'Python SVG Library'), ('license_family', 'BSD')]))])), 'pytabix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pytabix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pytabix'), ('version', '0.0.2')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'pytabix-e4a6f81.tar.gz'), ('url', 'https://github.com/slowkow/pytabix/archive/e4a6f81.tar.gz'), ('sha256', '0087bcde70dcccbc67e0a985741c887154319123707c9cdb3d515e39bcb7b41f')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'zlib']), ('run', ['python', 'zlib'])])), ('test', CommentedMap([('imports', ['tabix'])])), ('about', CommentedMap([('home', 'https://github.com/slowkow/pytabix'), ('license', 'MIT'), ('summary', 'Fast random access to sorted files compressed with bgzip and indexed by tabix.')]))])), 'pyteomics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyteomics/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/levitsky/pyteomics'), ('license', 'Apache License, Version 2.0'), ('summary', 'A framework for proteomics data analysis.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'pyteomics'), ('version', '3.4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'lxml', 'matplotlib', 'pandas', 'sqlalchemy'])])), ('source', CommentedMap([('fn', 'pyteomics-3.4.tar.gz'), ('md5', '3c8d1bcc16c53100e531a10a2853f679'), ('url', 'https://pypi.python.org/packages/02/2d/0dfb9bd5b02db6228e3277020209ddb7217e12a76e39a1cb0d1c70756e30/pyteomics-3.4.tar.gz')])), ('test', CommentedMap([('imports', ['pyteomics', 'pyteomics.mass', 'pyteomics.openms'])]))])), 'pytest-marks': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pytest-marks/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pytest-marks'), ('version', '0.4')])), ('source', CommentedMap([('fn', 'pytest-marks-0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/72/b6/9c9dce170c70998a89a95d7ab9706ba7da9b82d0ca5736843d2f60c6b977/pytest-marks-0.4.tar.gz'), ('md5', 'edaf600ddae40685df4a6fe05601e540')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest >2.0.2']), ('run', ['python', 'setuptools', 'pytest >2.0.2'])])), ('test', CommentedMap([('imports', ['marks'])])), ('about', CommentedMap([('home', 'https://github.com/adamgoucher/pytest-marks'), ('license', 'GNU General Public License (GPL)'), ('summary', 'set marks on py.test test methods'), ('license_family', 'LGPL')]))])), 'pytest-timeout': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pytest-timeout/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pytest-timeout'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'pytest-timeout-1.0.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pytest-timeout/pytest-timeout-1.0.0.tar.gz'), ('md5', 'f9f162bd079689980b5614673ddfdae4')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest >=2.8.0']), ('run', ['python', 'setuptools', 'pytest >=2.8.0'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/pytest-dev/pytest-timeout/'), ('license', 'MIT License'), ('summary', 'py.test plugin to abort hanging tests')]))])), 'pythomics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pythomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pythomics'), ('version', '0.3.42')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'pythomics-0.3.42.tar.gz'), ('url', 'https://pypi.python.org/packages/93/73/0ee763885f5fe62dccc266e0f34132ddebcecce03ccdeeaa23601c2615de/pythomics-0.3.42.tar.gz'), ('md5', '3619af158a8ca3568d7620fa047308be')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pythomics', 'pythomics.genomics', 'pythomics.parsers', 'pythomics.proteomics', 'pythomics.templates', 'pythomics.tests', 'pythomics.utils'])])), ('about', CommentedMap([('home', 'https://github.com/pandeylab/pythomics'), ('license', 'GPL3'), ('summary', 'A multi-omic python package'), ('license_family', 'GPL3')]))])), 'python-bioext': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-bioext/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-bioext'), ('version', '0.17.4')])), ('source', CommentedMap([('git_rev', '0.17.4'), ('git_url', 'https://github.com/veg/BioExt.git'), ('patches', ['osx-freetype.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'freetype', 'python', 'setuptools', 'biopython', 'pysam <0.9', 'numpy', 'scipy', 'six', 'python-consensuscore']), ('run', ['libgcc', 'freetype', 'python', 'biopython', 'pysam <0.9', 'numpy', 'scipy', 'six', 'python-consensuscore'])])), ('test', CommentedMap([('imports', ['BioExt', 'BioExt.align', 'BioExt.args', 'BioExt.collections', 'BioExt.errorize', 'BioExt.freetype', 'BioExt.freetype.ft_enums', 'BioExt.io', 'BioExt.io.BamIO', 'BioExt.io.LazyAlignIO', 'BioExt.io.SamIO', 'BioExt.joblib', 'BioExt.joblib.test', 'BioExt.merge', 'BioExt.misc', 'BioExt.ndarray', 'BioExt.optimize', 'BioExt.orflist', 'BioExt.phylo', 'BioExt.quiver', 'BioExt.rateclass', 'BioExt.references', 'BioExt.scorematrices', 'BioExt.stats', 'BioExt.uds', 'BioExt.untranslate']), ('commands', [\"bam2fna 2>&1 | grep 'usage' > /dev/null\", \"bam2msa 2>&1 | grep 'usage' > /dev/null\", \"bealign 2>&1 | grep 'usage' > /dev/null\", \"clipedge 2>&1 | grep 'usage' > /dev/null\", \"consensus 2>&1 | grep 'usage' > /dev/null\", \"msa2bam 2>&1 | grep 'usage' > /dev/null\", \"seqmerge 2>&1 | grep 'usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/veg/BioExt.git'), ('license', 'GPL-3'), ('summary', 'A few handy bioinformatics tools not already in BioPython')]))])), 'python-chado': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-chado/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-chado'), ('version', '2.0.1')])), ('source', CommentedMap([('fn', 'chado-2.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ff/21/de2f7a20a277c72c704910f1c34f53336d83a986114afbd87e6eca51a713/chado-2.0.1.tar.gz'), ('md5', '503b74610eb7f10b35de3e22f72817b1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sqlalchemy', 'psycopg2', 'biopython ==1.67', 'bcbiogff', 'wrapt', 'click', 'pyyaml']), ('run', ['python', 'sqlalchemy', 'psycopg2', 'biopython ==1.67', 'bcbiogff', 'wrapt', 'click', 'pyyaml', 'future'])])), ('test', CommentedMap([('imports', ['chado', 'chado.analysis', 'chado.export', 'chado.organism', 'chado.util']), ('commands', ['LANG=C.UTF-8 chakin --help > /dev/null', 'LANG=C.UTF-8 chakin analysis --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/galaxy-genome-annotation/python-chado'), ('license', 'MIT License'), ('summary', 'Chado library'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'python-cluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-cluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-cluster'), ('version', '1.3.3')])), ('source', CommentedMap([('fn', 'v1.3.3.tar.gz'), ('url', 'https://github.com/exhuma/python-cluster/archive/v1.3.3.tar.gz'), ('sha256', '4de2600408b77749abf6186df8571c3193ae7b729ec256fd8d0c25152c57ddf4')])), ('build', CommentedMap([('number', 0), ('script', ['sed -i.bak \"s/name=\\'cluster\\'/name=\\'python-cluster\\'/g\" setup.py', 'cat setup.py', '$PYTHON setup.py install'])])), ('requirements', CommentedMap([('build', ['setuptools', 'python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['cluster', 'cluster.method'])])), ('about', CommentedMap([('home', 'https://github.com/exhuma/python-cluster'), ('license', 'LGPL'), ('summary', 'python-cluster is a \"simple\" package that allows to create several groups (clusters) of objects from a list'), ('license_family', 'LGPL')]))])), 'python-consensuscore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-consensuscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/PacificBiosciences/ConsensusCore/archive/0ee22b27847f000d08923fdbaf84bf80ea228370.tar.gz'), ('md5', '521fb20a45008b61ec9f1b5e08ed050a')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}py<{CONDA_PY}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'setuptools', 'numpy >=1.6.0', 'boost >=1.47.0,<{CONDA_BOOST}}*', 'swig >=2.0.7']), ('run', ['libgcc', 'python', 'numpy >=1.6.0', 'boost <{CONDA_BOOST}}*', 'swig >=2.0.7'])])), ('test', CommentedMap([('imports', ['ConsensusCore'])])), ('about', CommentedMap([('home', 'https://github.com/PacificBiosciences/ConsensusCore'), ('license', 'BSD-3-Clause-Clear'), ('summary', 'C++ library of multiple-sequence consensus routines with SWIG bindings')]))])), 'python-fakemp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-fakemp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-fakemp'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'fakemp-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/f/fakemp/fakemp-0.9.1.tar.gz'), ('md5', '5d89d0190d0a5eba1f659f2857e37441')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['fakemp'])])), ('about', CommentedMap([('home', 'https://github.com/nlhepler/fakemp'), ('license', 'GPL-3'), ('summary', 'Fake multiprocessing pool objects')]))])), 'python-hivclustering': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-hivclustering/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-hivclustering'), ('version', '1.2.3')])), ('source', CommentedMap([('git_rev', '1.2.3'), ('git_url', 'https://github.com/veg/hivclustering.git'), ('patches', ['dependency_links.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'python-bioext', 'python-hyphy-python', 'python-hppy']), ('run', ['python', 'python-bioext', 'python-hyphy-python', 'python-hppy'])])), ('test', CommentedMap([('imports', ['hivclustering'])])), ('about', CommentedMap([('home', 'https://github.com/veg/hivclustering'), ('license', 'MIT'), ('summary', 'A Python 3 library that makes inferences on HIV-1 transmission networks.')]))])), 'python-hppy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-hppy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-hppy'), ('version', '0.9.6')])), ('source', CommentedMap([('git_rev', '0.9.6'), ('git_url', 'https://github.com/veg/hppy.git'), ('patches', ['dependency_links.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'python-hyphy-python', 'python-fakemp']), ('run', ['python', 'cython', 'python-hyphy-python', 'python-fakemp'])])), ('test', CommentedMap([('imports', ['hppy'])])), ('about', CommentedMap([('home', 'https://github.com/veg/hppy'), ('license', 'GPL-3'), ('summary', 'An intuitive HyPhy interface for Python')]))])), 'python-hyphy-python': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-hyphy-python/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-hyphy-python'), ('version', '0.1.3')])), ('source', CommentedMap([('git_rev', '0.1.3'), ('git_url', 'https://github.com/veg/hyphy-python.git')])), ('requirements', CommentedMap([('build', ['python', 'curl', 'gcc', 'llvm']), ('run', ['python', 'curl', 'libgcc'])])), ('test', CommentedMap([('imports', ['HyPhy'])])), ('about', CommentedMap([('home', 'https://github.com/veg/hyphy-python'), ('license', 'UNKNOWN'), ('summary', 'HyPhy package interface library')]))])), 'python-intermine': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-intermine/meta.yaml', CommentedMap([('package', CommentedMap([('version', '<{ version }}'), ('name', '<{ name|lower }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('<{ hash_type }}', '<{ hash_value }}'), ('fn', '<{ name }}-<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('script', 'python setup.py install')])), ('requirements', CommentedMap([('run', ['python']), ('build', ['python'])])), ('test', CommentedMap([('imports', ['intermine', 'intermine.lists'])])), ('about', CommentedMap([('dev_url', ''), ('description', 'InterMine Webservice Client\\n----------------------------\\n\\nA Python API to access bioinformatics data warehouses powered by the InterMine platform.'), ('license', 'GNU Library or Lesser General Public License (LGPL) or BSD License'), ('license_family', 'LGPL'), ('summary', 'InterMine WebService client'), ('home', 'http://www.intermine.org'), ('license_file', ''), ('doc_url', '')])), ('extra', CommentedMap([('recipe-maintainers', '')]))])), 'python-levenshtein': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-levenshtein/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-levenshtein'), ('version', '0.12.0')])), ('about', CommentedMap([('home', 'https://pypi.python.org/pypi/python-Levenshtein/'), ('license', 'GPL'), ('summary', 'Python extension for computing string edit distances and similarities.')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/source/p/python-Levenshtein/python-Levenshtein-0.12.0.tar.gz'), ('fn', 'python-Levenshtein-0.12.0.tar.gz')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', ['cython', 'nose', 'python']), ('run', ['cython', 'nose', 'python'])])), ('test', CommentedMap([('requires', ['nose']), ('imports', ['Levenshtein'])]))])), 'python-omero': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-omero/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-omero'), ('version', '<{ OMERO_VERSION }}')])), ('source', CommentedMap([('fn', 'OMERO.py-<{ OMERO_VERSION }}-ice36-b674.zip'), ('url', 'http://downloads.openmicroscopy.org/omero/<{ OMERO_VERSION }}/artifacts/OMERO.py-<{ OMERO_VERSION }}-ice36-b74.zip'), ('sha1', 'a17c26a6b027bfdadb487d069b14b4b38bd531c9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['setuptools', 'python']), ('run', ['python', 'zeroc-ice >3.5,<3.7'])])), ('test', CommentedMap([('imports', ['omero', 'omero.gateway'])])), ('about', CommentedMap([('home', 'https://www.openmicroscopy.org/'), ('license', 'GPL 2'), ('summary', 'OME (Open Microscopy Environment) develops open-source software and data format standards for the storage and manipulation of biological light microscopy data.')]))])), 'python-tripal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-tripal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-tripal'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'tripal-2.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/35/69/30bf090fb1eb0f0418506412f308c14a83490084b79f72d1c5a5990313c6/tripal-2.0.4.tar.gz'), ('md5', '8aa799a7be8ab2788b376a7bdfd7c728')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml']), ('run', ['python', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml', 'future'])])), ('test', CommentedMap([('imports', ['tripal', 'tripal.analysis', 'tripal.db', 'tripal.feature', 'tripal.job', 'tripal.organism']), ('commands', ['LANG=C.UTF-8 tripaille --help > /dev/null', 'LANG=C.UTF-8 tripaille analysis --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/galaxy-genome-annotation/python-tripal'), ('license', 'MIT License'), ('summary', 'Tripal API library'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'python-wget': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-wget/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-wget'), ('version', '3.2')])), ('source', CommentedMap([('fn', 'wget-3.2.zip'), ('url', 'https://pypi.python.org/packages/source/w/wget/wget-3.2.zip'), ('md5', 'fb7a04ee59a56bd8db974146df1d7a8b')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/techtonik/python-wget/'), ('license', 'Public Domain'), ('summary', 'pure python download utility')]))])), 'pythonpy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pythonpy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pythonpy'), ('version', '0.4.2')])), ('source', CommentedMap([('fn', 'pythonpy-0.4.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pythonpy/pythonpy-0.4.2.tar.gz'), ('md5', '9e4de105c594ecc04127a0fee088edc0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('commands', ['py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Russell91/pythonpy'), ('summary', 'the swiss army knife of the command line'), ('license', 'Unknown')]))])), 'pytriangle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pytriangle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pytriangle'), ('version', '1.0.9')])), ('about', CommentedMap([('home', 'https://github.com/pletzer/pytriangle'), ('summary', 'A python interface to the 2D triangulation program TRIANGLE'), ('license', 'MIT')])), ('source', CommentedMap([('git_url', 'https://github.com/pletzer/pytriangle'), ('git_rev', 'f87b411')])), ('build', CommentedMap([('number', 1)])), ('test', CommentedMap([('imports', ['triangle'])])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])]))])), 'pyutilib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyutilib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyutilib'), ('version', '5.1.3556')])), ('source', CommentedMap([('fn', 'PyUtilib-5.1.3556.tar.gz'), ('url', 'https://pypi.python.org/packages/source/P/PyUtilib/PyUtilib-5.1.3556.tar.gz'), ('md5', 'fa6b4fba1e8c817d431e9c2aac8a27b5')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'six']), ('run', ['python', 'nose', 'six'])])), ('test', CommentedMap([('imports', ['pyutilib', 'pyutilib.autotest', 'pyutilib.autotest.tests', 'pyutilib.common', 'pyutilib.component', 'pyutilib.component.app', 'pyutilib.component.app.tests', 'pyutilib.component.config', 'pyutilib.component.config.tests', 'pyutilib.component.core', 'pyutilib.component.core.tests', 'pyutilib.component.executables', 'pyutilib.component.loader', 'pyutilib.component.loader.tests', 'pyutilib.dev', 'pyutilib.enum', 'pyutilib.enum.tests', 'pyutilib.excel', 'pyutilib.excel.tests', 'pyutilib.math', 'pyutilib.math.tests', 'pyutilib.misc', 'pyutilib.misc.tests', 'pyutilib.misc.tests.import_data', 'pyutilib.misc.tests.import_data.a', 'pyutilib.misc.tests.import_data.b', 'pyutilib.ply', 'pyutilib.pyro', 'pyutilib.services', 'pyutilib.subprocess', 'pyutilib.subprocess.tests', 'pyutilib.th', 'pyutilib.th.tests', 'pyutilib.workflow', 'pyutilib.workflow.tests'])])), ('about', CommentedMap([('home', 'https://software.sandia.gov/trac/pyutilib'), ('license', 'BSD License'), ('summary', 'PyUtilib: A collection of Python utilities')]))])), 'pyvcf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyvcf/0.6.7/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyvcf'), ('version', '0.6.7')])), ('source', CommentedMap([('fn', 'PyVCF-0.6.7.tar.gz'), ('url', 'https://pypi.python.org/packages/source/P/PyVCF/PyVCF-0.6.7.tar.gz'), ('md5', '51b57ce99e0c2f7be2a18d08d8f87734')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['vcf.test', 'vcf'])])), ('about', CommentedMap([('home', 'https://github.com/jamescasbon/PyVCF'), ('license', 'BSD'), ('summary', 'A Variant Call Format reader for Python')]))])), 'pywdl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pywdl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pywdl'), ('version', '1.0.22')])), ('source', CommentedMap([('fn', 'pywdl-1.0.22.tar.gz'), ('url', 'https://github.com/broadinstitute/pywdl/archive/7a09cab.tar.gz')])), ('build', CommentedMap([('skip', False), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest', 'xtermcolor']), ('run', ['python', 'xtermcolor', 'pytest'])])), ('test', CommentedMap([('commands', ['wdl --help'])])), ('about', CommentedMap([('home', 'https://github.com/broadinstitute/pywdl'), ('license', 'Apache v2.0'), ('summary', 'A Python implementation of a WDL parser and language bindings.')]))])), 'pyx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyx'), ('version', '0.12.1')])), ('source', CommentedMap([('fn', 'PyX-0.12.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/P/PyX/pyx-0.12.1.tar.gz'), ('md5', '04263eb002553dae0e8a1d0eeb805ec1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pyx', 'pyx.dvi', 'pyx.font', 'pyx.graph', 'pyx.graph.axis', 'pyx.metapost'])])), ('about', CommentedMap([('home', 'http://pyx.sourceforge.net/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Python package for the generation of PostScript, PDF, and SVG files')]))])), 'qcli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qcli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qcli'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'qcli-0.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qcli/qcli-0.1.1.tar.gz'), ('md5', '25b7bc209c1e52be16efad295bd37ed4'), ('patches', ['python3.patch [py3k]'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['qcli']), ('commands', ['qcli_make_rst -h', 'qcli_make_script -h'])])), ('about', CommentedMap([('license', 'GPL'), ('summary', 'qcli')]))])), 'qcumber': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qcumber/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qcumber'), ('version', '2.0.4')])), ('source', CommentedMap([('url', 'https://gitlab.com/RKIBioinformaticsPipelines/QCumber/repository/2.0.4/archive.tar.gz'), ('md5', '8547d291c53a3d665427b05729f1db4c')])), ('build', CommentedMap([('skip', True), ('number', '0'), ('string', '0')])), ('requirements', CommentedMap([('build', ['python 3.6.*', 'trimmomatic 0.36.*']), ('run', ['python 3.6.*', 'snakemake 3.12.*', 'r 3.3.*', 'trimmomatic 0.36.*', 'setuptools', 'bowtie2 2.3.*', 'fastqc 0.11.*', 'kraken 0.10.*', 'samtools 1.3.*', 'numpy', 'pandas', 'pyyaml 3.12.*', 'matplotlib 2.0.*', 'jinja2', 'bitstring', 'docopt', 'krona', 'xmltodict', 'seaborn', 'r-quantreg', 'r-ggplot2 2.2.*', 'r-reshape2', 'bioconductor-savr', 'r-stringi'])])), ('about', CommentedMap([('home', 'https://gitlab.com/RKIBioinformaticsPipelines/QCumber'), ('license', 'LGPL3'), ('license_file', 'LICENSE'), ('summary', 'Quality control, quality trimming, adapter removal and sequence content check of NGS data.')])), ('test', CommentedMap([('source_files', ['QCumber-2', 'Snakefile', 'batch_report.html', 'report.tex', 'modules/init.snakefile', 'modules/classification.snakefile', 'modules/fastqc.snakefile', 'modules/mapping.snakefile', 'modules/sav.snakefile', 'modules/trimming.snakefile', 'config/parameter.txt', 'Rscripts/sav.R', 'Rscripts/barplot.R', 'Rscripts/boxplot.R']), ('commands', ['snakemake -h', 'QCumber-2 -h'])]))])), 'qfilt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qfilt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qfilt'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/veg/qfilt/archive/<{ version }}.tar.gz'), ('sha256', 'b0ada01c44c98b1a8bebf88c72a845b24f91a7cc63687e228b1ad872681adc09')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"qfilt -h 2>&1 | grep 'filter sequencing data using some simple heuristics' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/veg/qfilt'), ('license', 'MIT'), ('summary', 'Filter sequencing data using some simple heuristics')]))])), 'qiimetomaaslin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qiimetomaaslin/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/biobakery/<{ name|lower }}/get/<{ version }}.tar.gz'), ('md5', 'fbd3168034d0cd76756935374b1c6cf6')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'blist'])])), ('test', CommentedMap([('commands', ['qiimeToMaaslin.py -h'])])), ('about', CommentedMap([('home', 'https://huttenhower.sph.harvard.edu/maaslin'), ('license', 'Unknown'), ('summary', 'Data munging script to change text Qiime OTU tables to pcl-formatted, maaslin-compatible text files')]))])), 'qorts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qorts/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://hartleys.github.io/QoRTs/'), ('license', 'Public Domain'), ('summary', 'The QoRTs software package is a fast, efficient, and portable multifunction toolkit designed to assist in the analysis, quality control, and data management of RNA-Seq datasets.')])), ('package', CommentedMap([('name', 'qorts'), ('version', '1.1.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'QoRTs.jar'), ('url', 'https://github.com/hartleys/QoRTs/releases/download/v1.1.8/QoRTs.jar'), ('md5', '4ff96e7126c971cf38b55863828096f3')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['qorts QC --man &> /dev/null'])])), ('extra', CommentedMap([('notes', 'QoRTs is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"qorts\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"qorts -Xms512m -Xmx1g\"\\n')]))])), 'qsignature': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qsignature/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qsignature'), ('version', '0.1pre')])), ('source', CommentedMap([('fn', 'qsignature-0.1pre.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/adamajava/qsignature.tar.bz2'), ('md5', '32c168b5ac414af099bcf8cf98a0d073')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['qsignature org.qcmg.sig.SignatureGenerator -h'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/p/adamajava/wiki/Home/'), ('license', 'GPLv3'), ('summary', 'qsignature is a simple and highly effective method for detecting potential sample mix-ups using distance measurements between SNP')]))])), 'qtip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qtip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qtip'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'qtip-<{ version }}.tar.gz'), ('url', 'https://github.com/BenLangmead/qtip/archive/v<{ version }}.tar.gz'), ('sha256', 'fbffb2eccef3248dd6ffd10c9272c0c14091b25dbf3e680d75e7f3ffcf7de1e3')])), ('requirements', CommentedMap([('build', ['make', 'gcc', 'python', 'numpy', 'scikit-learn', 'pandas']), ('run', ['python', 'numpy', 'scikit-learn', 'pandas', 'libgcc'])])), ('test', CommentedMap([('commands', ['qtip --help'])])), ('about', CommentedMap([('home', 'https://github.com/BenLangmead/qtip'), ('license', 'MIT'), ('summary', 'A tandem simulation approach for accurately predicting read alignment\\nmapping qualities.\\n')])), ('extra', CommentedMap([('notes', 'For running qtip, you will need to install any of the supported read mappers,\\n(e.g., bowtie2), in the minimum required version. See homepage for details.\\n')]))])), 'quake': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quake/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://www.cbcb.umd.edu/software/quake/'), ('summary', 'Quake is a package to correct substitution sequencing errors in experiments with deep coverage (e.g. >15X), specifically intended for Illumina sequencing reads')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://www.cbcb.umd.edu/software/quake/downloads/<{ name }}-<{ version }}.tar.gz'), ('md5', 'f8e26c0c29d22fc0d49fcfb5dc8945d3')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'jellyfish', 'r-base', 'r-vgam', 'python'])])), ('test', CommentedMap([('commands', ['quake.py --help'])]))])), 'qualimap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qualimap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qualimap'), ('version', '2.2.2a')])), ('source', CommentedMap([('fn', 'qualimap-build-11-12-16.tar.gz'), ('url', 'https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-11-12-16.tar.gz'), ('md5', '2bacc61c0ed0412f09c6f02cae25bb7c')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['qualimap -h'])])), ('about', CommentedMap([('home', 'http://qualimap.bioinfo.cipf.es/'), ('license', 'GPLv2'), ('summary', 'Quality control of alignment sequencing data and its derivatives like feature counts')]))])), 'quasitools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quasitools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'quasitools'), ('version', '0.1.0')])), ('source', CommentedMap([('fn', 'quasitools-0.1.0.tar.gz'), ('url', 'https://github.com/phac-nml/quasitools/archive/v0.1.0.tar.gz'), ('md5', '8d84ebe5c20e4b6a21b6ef902842617d')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'scipy', 'pysam', 'click', 'biopython']), ('run', ['python', 'scipy', 'pysam', 'click', 'biopython'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 quasitools --help'])])), ('about', CommentedMap([('home', 'https://github.com/phac-nml/quasitools/'), ('license', 'Apache License, Version 2.0'), ('summary', 'Quasitools is a collection of tools for analysing Viral Quasispecies')]))])), 'quast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quast/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '<{ hash }}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'perl', 'python', 'matplotlib', 'openjdk', 'joblib', 'simplejson']), ('run', ['boost <{CONDA_BOOST}}*', 'perl', 'python', 'libgcc', 'matplotlib', 'openjdk', 'joblib', 'simplejson'])])), ('test', CommentedMap([('commands', ['quast --version', 'quast --test', 'metaquast --version', 'metaquast --test'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/<{ name|lower }}/'), ('license', 'GPL2'), ('license_file', 'LICENSE.txt'), ('summary', 'Quality Assessment Tool for Genome Assemblies')]))])), 'query_phenomizer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/query_phenomizer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'query_phenomizer'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'query_phenomizer-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/e3/b2/d8f106d7f59aba6f198a6b2dc14237076517977c4208ca299ea8995fd43e/query_phenomizer-0.5.tar.gz'), ('md5', '6141bd9f5743d9f828ee200eb62f8fbc')])), ('build', CommentedMap([('entry_points', ['query_phenomizer = query_phenomizer.__main__:cli']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'click', 'requests', 'pytest']), ('run', ['python', 'click', 'requests', 'pytest'])])), ('test', CommentedMap([('imports', ['query_phenomizer']), ('commands', ['LANG=C.UTF-8 query_phenomizer --help'])])), ('about', CommentedMap([('home', 'https://www.github.com/moonso/query_phenomizer'), ('license', 'MIT License'), ('summary', 'Tool for query and parsing the phenomizer tool')]))])), 'quorum': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quorum/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Quorum-<{ version }}.tar.gz'), ('url', 'https://github.com/gmarcais/Quorum/releases/download/v<{ version }}/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'jellyfish >=2.0']), ('run', ['libgcc', 'perl-threaded', 'jellyfish >=2.0'])])), ('test', CommentedMap([('commands', ['quorum --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.genome.umd.edu/quorum.html'), ('license', 'GPLv3'), ('summary', 'QuorUM (Quality Optimized Reads from the University of Maryland) is an error corrector for Illumina reads. It is distributed and used with MaSuRCA, or it can be used independently.')]))])), 'r-alakazam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-alakazam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-alakazam'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'alakazam_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/alakazam_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/alakazam/alakazam_<{ version }}.tar.gz']), ('sha256', '83d781817163110e71d3f1ba2205284a33f5e8d6d654a7074e7e27a18e47fb85')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base >=3.4.1', 'r-rcpp >=0.12.11', 'r-dplyr >=0.5.0', 'r-ggplot2 >=2.0.0', 'r-igraph >=1.0.0', 'r-lazyeval', 'r-progress', 'r-readr', 'r-scales', 'r-seqinr', 'r-stringi', 'gcc', 'llvm']), ('run', ['r-base >=3.4.1', 'r-rcpp >=0.12.11', 'r-dplyr >=0.5.0', 'r-ggplot2 >=2.0.0', 'r-igraph >=1.0.0', 'r-lazyeval', 'r-progress', 'r-readr', 'r-scales', 'r-seqinr', 'r-stringi', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'alakazam\\')\"'])])), ('about', CommentedMap([('home', 'http://alakazam.readthedocs.org'), ('license', 'CC BY-SA 4.0'), ('summary', 'Provides immunoglobulin (Ig) sequence lineage reconstruction, diversity profiling, and amino acid property analysis.'), ('license_family', 'OTHER')]))])), 'r-ampvis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-ampvis/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/MadsAlbertsen/ampvis/archive/8d425cead73155e0591ffed64069c557ced8b46d.tar.gz'), ('md5', 'e898bf83d56eb7bfc17535b563932690')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-ggrepel', 'r-reshape2', 'bioconductor-phyloseq', 'r-vegan', 'r-knitr', 'r-igraph', 'r-rcolorbrewer', 'bioconductor-biostrings', 'r-data.table', 'bioconductor-deseq2', 'r-ggdendro', 'r-stringr', 'r-dplyr', 'r-magrittr', 'r-gridextra', 'r-scales']), ('run', ['r-base', 'r-ggplot2', 'r-ggrepel', 'r-reshape2', 'bioconductor-phyloseq', 'r-vegan', 'r-knitr', 'r-igraph', 'r-rcolorbrewer', 'bioconductor-biostrings', 'r-data.table', 'bioconductor-deseq2', 'r-ggdendro', 'r-stringr', 'r-dplyr', 'r-magrittr', 'r-gridextra', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ampvis\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/MadsAlbertsen/ampvis'), ('license', 'AGPL-3'), ('summary', 'A package to visualise amplicon data')]))])), 'r-aroma.affymetrix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-aroma.affymetrix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-aroma.affymetrix'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'aroma.affymetrix_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/aroma.affymetrix_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/aroma.affymetrix/aroma.affymetrix_<{ version }}.tar.gz']), ('sha256', '03c0d30c2e636409d527c0d4253d792218bba133681d5d334d19350c32ae8bea')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-r.cache >=0.12.0', 'r-r.devices >=2.15.1', 'r-r.filesets >=2.11.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-aroma.apd >=0.6.0', 'r-aroma.core >=3.1.0', 'r-future', 'r-listenv', 'r-matrixstats >=0.51.0']), ('run', ['r-base', 'r-mass', 'r-r.cache >=0.12.0', 'r-r.devices >=2.15.1', 'r-r.filesets >=2.11.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-aroma.apd >=0.6.0', 'r-aroma.core >=3.1.0', 'r-future', 'r-listenv', 'r-matrixstats >=0.51.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aroma.affymetrix\\')\"', '\"%R%\" -e \"library(\\'aroma.affymetrix\\')\"'])])), ('about', CommentedMap([('home', 'http://www.aroma-project.org/, https://github.com/HenrikBengtsson/aroma.affymetrix'), ('license', 'LGPL (>= 2.1)'), ('summary', 'A cross-platform R framework that facilitates processing of any number of Affymetrix microarray samples regardless of computer system.  The only parameter that limits the number of chips that can be processed is the amount of available disk space.  The Aroma Framework has successfully been used in studies to process tens of thousands of arrays.  This package has actively been used since 2006.'), ('license_family', 'LGPL')])), ('extra', CommentedMap([('recipe-maintainers', ['johanneskoester', 'bgruening', 'daler', 'jdblischak'])]))])), 'r-aroma.core': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-aroma.core/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-aroma.core'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'aroma.core_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/aroma.core_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/aroma.core/aroma.core_<{ version }}.tar.gz']), ('sha256', '615af96ace9ce276e0277d76637417508ea0d6f15e907732332ea28a029a7055')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-pscbs >=0.62.0', 'r-r.cache >=0.12.0', 'r-r.devices >=2.15.1', 'r-r.filesets >=2.11.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.rsp >=0.40.0', 'r-r.utils >=2.5.0', 'r-rcolorbrewer >=1.1_2', 'r-future', 'r-listenv', 'r-matrixstats >=0.52.2']), ('run', ['r-base', 'r-pscbs >=0.62.0', 'r-r.cache >=0.12.0', 'r-r.devices >=2.15.1', 'r-r.filesets >=2.11.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.rsp >=0.40.0', 'r-r.utils >=2.5.0', 'r-rcolorbrewer >=1.1_2', 'r-future', 'r-listenv', 'r-matrixstats >=0.52.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aroma.core\\')\"', '\"%R%\" -e \"library(\\'aroma.core\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/HenrikBengtsson/aroma.core, http://www.aroma-project.org/'), ('license', 'LGPL (>= 2.1)'), ('summary', 'Core methods and classes used by higher-level aroma.* packages part of the Aroma Project, e.g. aroma.affymetrix and aroma.cn.'), ('license_family', 'LGPL')])), ('extra', CommentedMap([('recipe-maintainers', ['johanneskoester', 'bgruening', 'daler', 'jdblischak'])]))])), 'r-bio3d': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-bio3d/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-bio3d'), ('version', '2.2_3')])), ('source', CommentedMap([('fn', 'bio3d_2.2-3.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/bio3d/bio3d_2.2-3.tar.gz']), ('sha256', '58afd7f9682c2355b62831793bef93159f91ab804a01bc05a16b208994815181')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bio3d\\')\"', '\"%R%\" -e \"library(\\'bio3d\\')\"'])])), ('about', CommentedMap([('home', 'http://thegrantlab.org/bio3d/, http://bitbucket.org/Grantlab/bio3d'), ('license', 'GPL (>= 2)'), ('summary', 'Utilities to process, organize and explore protein structure, sequence and dynamics data.  Features include the ability to read and write structure, sequence and dynamic trajectory data, perform sequence and structure database searches, data summaries, atom selection, alignment, superposition, rigid core identification, clustering, torsion analysis, distance matrix analysis, structure and sequence conservation analysis, normal mode analysis, principal component analysis of heterogeneous structure data, and correlation network analysis from normal mode and molecular dynamics data.  In addition, various utility functions are provided to enable the statistical and graphical power of the R environment to work with biological sequence and structural data.  Please refer to the URLs below for more information.')]))])), 'r-biom': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-biom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-biom'), ('version', '0.3.12')])), ('source', CommentedMap([('fn', 'biom_0.3.12.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/Archive/biom/biom_0.3.12.tar.gz']), ('sha256', '4ad17f7811c7346dc4923bd6596a007c177eebb1944a9f46e5674afcc5fdd5a1')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-plyr', 'r-rjsonio', 'r-matrix']), ('run', ['r-base', 'r-plyr', 'r-rjsonio', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biom\\')\"', '\"%R%\" -e \"library(\\'biom\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/joey711/biom/, http://biom-format.org/'), ('license', 'GPL-2'), ('summary', 'This is an R package for interfacing with the BIOM format.')]))])), 'r-boutroslabplottinggeneral': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-boutroslabplottinggeneral/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'boutroslabplottinggeneral'), ('version', '5.3.4')])), ('source', CommentedMap([('fn', 'BoutrosLab.plotting.general_5.3.4.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/boutroslabplottinggeneral/boutroslabplottinggeneral_5.3.4_src_all.tar.gz'), ('sha256', 'aa72e402e6cd7206ce34dd74a174fe135e7d5ab00e36164effec50279ad6ae52')])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer', 'r-lattice', 'r-latticeextra', 'r-r.methodss3', 'r-hexbin', 'r-proto', 'r-findpython', 'r-getopt', 'r-rjson', 'r-argparse', 'r-assertthat', 'r-r6', 'r-rcpp', 'r-lazyeval', 'r-tibble', 'r-magrittr', 'r-dbi', 'r-bh', 'r-dplyr', 'r-mass', 'r-cluster']), ('run', ['r-base', 'r-rcolorbrewer', 'r-lattice', 'r-latticeextra', 'r-r.methodss3', 'r-hexbin', 'r-proto', 'r-findpython', 'r-getopt', 'r-rjson', 'r-argparse', 'r-assertthat', 'r-r6', 'r-rcpp', 'r-lazyeval', 'r-tibble', 'r-magrittr', 'r-dbi', 'r-bh', 'r-dplyr', 'r-mass', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BoutrosLab.plotting.general\\')\"'])])), ('about', CommentedMap([('home', 'https://labs.oicr.on.ca/boutros-lab/software/bpg')]))])), 'r-chbutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-chbutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-chbutils'), ('version', '0.1_2015_12_21')])), ('source', CommentedMap([('fn', 'CHBUtils-0.1_2015_12_21.tar.gz'), ('url', 'https://github.com/hbc/CHBUtils/archive/d862e27f.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mvtnorm', 'r-rcolorbrewer']), ('run', ['r-base', 'r-mvtnorm', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CHBUtils\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/hbc/CHBUtils'), ('license', 'MIT'), ('summary', 'Useful utility functions used at the Harvard Chan School Bioinformatics core')]))])), 'r-cimpl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-cimpl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-cimpl'), ('version', 1.1)])), ('source', CommentedMap([('url', 'https://github.com/NKI-CCB/cimpl/archive/v1.1.tar.gz'), ('sha256', '4895f221e8d12efc65d892684efbd151e909b32d5c5041d4bcefea9e832995f5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-kernsmooth', 'r-mass', 'r-xtable', 'bioconductor-biomart', 'bioconductor-biostrings']), ('run', ['r-base', 'r-kernsmooth', 'r-mass', 'r-xtable', 'bioconductor-biomart', 'bioconductor-biostrings'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cimpl\\')\"'])])), ('about', CommentedMap([('home', 'http://ccb.nki.nl/software/'), ('license', 'GPL-3'), ('summary', 'An analysis package for multi sample insertional mutagenesis data (including viral- and transposon-based systems) using Gaussian kernel convolution to identify common insertion sites.')]))])), 'r-cnogpro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-cnogpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-cnogpro'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'CNOGpro_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/CNOGpro_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/CNOGpro/CNOGpro_<{ version }}.tar.gz']), ('sha256', '889aba7c5201bda7edc369798e5fb9043ffb73c54b310394dee1d18c1dac3abb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNOGpro\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=CNOGpro'), ('license', 'GPL-2'), ('summary', 'Methods for assigning copy number states and breakpoints in resequencing experiments of prokaryotic organisms.'), ('license_family', 'GPL2')]))])), 'r-coloc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-coloc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-coloc'), ('version', '2.3_1')])), ('source', CommentedMap([('fn', 'coloc_2.3-1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/coloc_2.3-1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/coloc/coloc_2.3-1.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-bma', 'r-mass', 'r-colorspace']), ('run', ['r-base', 'r-bma', 'r-mass', 'r-colorspace'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'coloc\\')\"', '\"%R%\" -e \"library(\\'coloc\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/chr1swallace/coloc'), ('license', 'GPL'), ('summary', 'Performs the colocalisation tests described in Plagnol et al (2009), Wallace et al (2013) and Giambartolomei et al (2013).'), ('license_family', 'GPL3')]))])), 'r-dbchip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-dbchip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-dbchip'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'DBChIP_1.1.6.tar.gz'), ('url', 'http://pages.cs.wisc.edu/~kliang/DBChIP/DBChIP_1.1.6.tar.gz'), ('md5', 'f4b22bb2051ad6b2d33d4687754e8cee')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-deseq', 'bioconductor-edger']), ('run', ['r-base', 'bioconductor-deseq', 'bioconductor-edger'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DBChIP\\')\"'])])), ('about', CommentedMap([('home', 'http://pages.cs.wisc.edu/~kliang/DBChIP'), ('license', 'GPL (>= 2)'), ('summary', 'ChIP-seq differential binding')]))])), 'r-deoptimr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-deoptimr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-deoptimr'), ('version', '1.0_6')])), ('source', CommentedMap([('fn', 'DEoptimR_1.0-6.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/DEoptimR/DEoptimR_1.0-6.tar.gz']), ('md5', '882cac87c736fc4c66c51065e81287ff')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEoptimR\\')\"', '\"%R%\" -e \"library(\\'DEoptimR\\')\"'])])), ('about', CommentedMap([('license', 'GPL (>= 2)'), ('summary', 'An implementation of a bespoke jDE variant of the Differential Evolution stochastic algorithm for global optimization of nonlinear programming problems.'), ('license_family', 'GPL3')]))])), 'r-dpeak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-dpeak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-dpeak'), ('version', '2.0.1')])), ('source', CommentedMap([('git_url', 'https://github.com/dongjunchung/dpeak.git'), ('git_rev', 'b4627c94ef87b3dbd0821230821df4ab7e85be8f')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'bioconductor-iranges', 'bioconductor-bsgenome', 'r-rcpp', 'gcc']), ('run', ['r-base', 'r-mass', 'bioconductor-iranges', 'bioconductor-bsgenome', 'r-rcpp', 'libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dpeak\\')\"'])])), ('about', CommentedMap([('home', 'http://dongjunchung.github.io/dpeak/'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for fitting dPeak, a statistical framework to deconvolve ChIP-seq peaks.')]))])), 'r-drc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-drc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/<{ name }}/<{ name }}_<{ version }}.tar.gz']), ('md5', '50429c71d876eabdb399a81f1b5913a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-car', 'r-gtools', 'r-multcomp', 'r-plotrix', 'r-scales', 'r-nloptr']), ('run', ['r-base', 'r-mass', 'r-car', 'r-gtools', 'r-multcomp', 'r-plotrix', 'r-scales', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'drc\\')\"'])])), ('about', CommentedMap([('home', 'http://www.r-project.org, http://www.bioassay.dk'), ('license', 'GPL-2 | file LICENCE'), ('summary', 'Analysis of dose-response data is made available through a suite of flexible and versatile model fitting and after-fitting functions.'), ('license_family', 'GPL2')]))])), 'r-dynamictreecut': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-dynamictreecut/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-dynamictreecut'), ('version', '1.63_1')])), ('source', CommentedMap([('fn', 'dynamicTreeCut_1.63-1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/dynamicTreeCut_1.63-1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/dynamicTreeCut/dynamicTreeCut_1.63-1.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dynamicTreeCut\\')\"', '\"%R%\" -e \"library(\\'dynamicTreeCut\\')\"'])])), ('about', CommentedMap([('home', 'http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/BranchCutting/'), ('license', 'GPL (>= 2)'), ('summary', 'Contains methods for detection of clusters in hierarchical clustering dendrograms.'), ('license_family', 'GPL3')]))])), 'r-empiricalfdr.deseq2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-empiricalfdr.deseq2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-empiricalfdr.deseq2'), ('version', '1.0.3')])), ('source', CommentedMap([('fn', 'empiricalFDR.DESeq2_1.0.3.tar.gz'), ('url', 'https://cran.r-project.org/src/contrib/empiricalFDR.DESeq2_1.0.3.tar.gz'), ('md5', '676001ea8edfc2c573fd3a8e9a7b55a1')])), ('build', CommentedMap([('skip', True), ('rpaths', ['lib/R/lib/', 'lib'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-deseq2', 'bioconductor-genomicranges']), ('run', ['r-base', 'bioconductor-deseq2', 'bioconductor-genomicranges'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'empiricalFDR.DESeq2\\')\"', '\"%R%\" -e \"library(\\'empiricalFDR.DESeq2\\')\"'])])), ('about', CommentedMap([('license', 'GPLv3'), ('summary', 'Auxiliary functions for the DESeq2 package to simulate read counts according to the null hypothesis (i.e., with empirical sample size factors, per-gene total counts and dispersions, but without effects of predictor variables) and to compute the empirical false discovery rate.')]))])), 'r-exomedepth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-exomedepth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-exomedepth'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'ExomeDepth_1.1.10.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/ExomeDepth_1.1.10.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/ExomeDepth/ExomeDepth_1.1.10.tar.gz']), ('md5', '2354132d3a08d227474cd54efc5f7ae4')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ExomeDepth\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/ExomeDepth/index.html'), ('license', 'GPL-3'), ('summary', 'Calls copy number variants (CNVs) from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.'), ('license_family', 'GPL3')]))])), 'r-factominer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-factominer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/<{ name }}/<{ name }}_<{ version }}.tar.gz']), ('md5', 'bef076181ce942016114dd7a6f5c2348')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base ==3.3.1', 'r-mass', 'r-car', 'r-cluster', 'r-ellipse', 'r-flashclust', 'r-knitr', 'r-lattice', 'r-leaps', 'r-scatterplot3d']), ('run', ['r-base 3.3.1', 'r-mass', 'r-car', 'r-cluster', 'r-ellipse', 'r-flashclust', 'r-knitr', 'r-lattice', 'r-leaps', 'r-scatterplot3d'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'FactoMineR\\')\"'])])), ('about', CommentedMap([('home', 'http://factominer.free.fr'), ('license', 'GPL (>= 2)'), ('summary', 'Exploratory data analysis methods to summarize, visualize and describe datasets. The main principal component methods are available, those with the largest potential in terms of applications: principal component analysis (PCA) when variables are quantitative, correspondence analysis (CA) and multiple correspondence analysis (MCA) when variables are categorical, Multiple Factor Analysis when variables are structured in groups, etc. and hierarchical cluster analysis.'), ('license_family', 'GPL3')]))])), 'r-fnn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-fnn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-fnn'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'FNN_<{ version }}.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/FNN_<{ version }}.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/FNN/FNN_<{ version }}.tar.gz']), ('md5', '8ba8f5b8be271785593e13eae7b8c393')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'FNN\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=FNN'), ('license', 'GPL (>= 2.1)'), ('summary', 'Cover-tree and kd-tree fast k-nearest neighbor search algorithms and related applications including KNN classification, regression and information measures are implemented.'), ('license_family', 'GPL2')]))])), 'r-gam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gam'), ('version', '1.14')])), ('source', CommentedMap([('fn', 'gam_1.14.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/gam/gam_1.14.tar.gz']), ('md5', '6f34c6b3b6133d28c8715102b52b4e86')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-foreach', 'gcc']), ('run', ['r-base', 'r-foreach', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gam\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/gam/index.html'), ('license', 'GPL-2'), ('summary', 'Functions for fitting and working with generalized additive models, as described in chapter 7 of \"Statistical Models in S\" (Chambers and Hastie (eds), 1991), and \"Generalized Additive Models\" (Hastie and Tibshirani, 1990).'), ('license_family', 'GPL2')]))])), 'r-gbm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gbm'), ('version', '2.1.1')])), ('source', CommentedMap([('fn', 'gbm_2.1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/gbm/gbm_2.1.1.tar.gz']), ('md5', '8126037a1a9a216735db074d04d362ed')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice', 'r-survival', 'gcc', 'llvm']), ('run', ['r-base', 'r-lattice', 'r-survival', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gbm\\')\"'])])), ('about', CommentedMap([('home', 'http://code.google.com/p/gradientboostedmodels/'), ('license', 'GPL (>= 2) | file LICENSE'), ('summary', \"An implementation of extensions to Freund and Schapire's AdaBoost algorithm and Friedman's gradient boosting machine. Includes regression methods for least squares, absolute loss, t-distribution loss, quantile regression, logistic, multinomial logistic, Poisson, Cox proportional hazards partial likelihood, AdaBoost exponential loss, Huberized hinge loss, and Learning to Rank measures (LambdaMart).\"), ('license_family', 'GPL3')]))])), 'r-genabel.data': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-genabel.data/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-genabel.data'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'GenABEL.data_1.0.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/GenABEL.data_1.0.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/GenABEL.data/GenABEL.data_1.0.0.tar.gz']), ('md5', '649dde059aed16bb411059efb20f4930')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenABEL.data\\')\"', '\"%R%\" -e \"library(\\'GenABEL.data\\')\"'])])), ('about', CommentedMap([('home', 'http://www.genabel.org, http://forum.genabel.org, http://genabel.r-forge.r-project.org/'), ('license', 'GPL (>= 2)'), ('summary', 'GenABEL.data package consists of a data set used by GenABEL functions'), ('license_family', 'GPL3')]))])), 'r-genenet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-genenet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-genenet'), ('version', '1.2.13')])), ('source', CommentedMap([('fn', 'GeneNet_1.2.13.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/GeneNet_1.2.13.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/GeneNet/GeneNet_1.2.13.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-corpcor >=1.6.8', 'r-fdrtool >=1.2.15', 'r-longitudinal >=1.1.12']), ('run', ['r-base', 'r-corpcor >=1.6.8', 'r-fdrtool >=1.2.15', 'r-longitudinal >=1.1.12'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GeneNet\\')\"', '\"%R%\" -e \"library(\\'GeneNet\\')\"'])])), ('about', CommentedMap([('home', 'http://strimmerlab.org/software/genenet/'), ('license', 'GPL (>= 3)'), ('summary', 'Analyzes gene expression (time series) data with focus on the inference of gene networks. In particular, GeneNet implements the methods of Schaefer and  Strimmer (2005a,b,c) and Opgen-Rhein and Strimmer (2006, 2007) for learning large-scale gene association networks (including assignment of putative directions).  ')]))])), 'r-george': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-george/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-george'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'v1.0.1.zip'), ('url', 'https://github.com/jcapelladesto/geoRge/archive/v1.0.1.zip'), ('md5', '0c30595306dcea334d054b7733bba375')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biobase', 'r-rcpp', 'bioconductor-mzr', 'bioconductor-xcms', 'r-optparse']), ('run', ['r-base', 'bioconductor-mzr', 'bioconductor-xcms', 'r-optparse'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'geoRge\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/jcapelladesto/geoRge/README.md'), ('license', 'GPL (>= 2)'), ('summary', 'geoRge, a computational tool for stable isotope labelling detection in LC/MS-based untargeted metabolomics')]))])), 'r-ggthemes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-ggthemes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-ggthemes'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'ggthemes_3.4.0.tar.gz'), ('url', 'https://cran.rstudio.com/src/contrib/ggthemes_3.4.0.tar.gz'), ('md5', '075dde101a5415becc790dfbdcf0b862')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-assertthat', 'r-colorspace', 'r-ggplot2 >=2.2.0', 'r-scales']), ('run', ['r-base', 'r-assertthat', 'r-colorspace', 'r-ggplot2 >=2.2.0', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggthemes\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.rstudio.com/web/packages/ggthemes/index.html'), ('license', 'GPL-2'), ('summary', 'Some extra themes, geoms, and scales for ggplot2. Provides ggplot2 themes and scales that replicate the look of plots by Edward Tufte, Stephen Few, Fivethirtyeight, The Economist, Stata, Excel, and The Wall Street Journal, among others. Provides geoms for Tufte s box plot and range frame.')]))])), 'r-giant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-giant/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-giant'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'GiANT_1.2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/GiANT_1.2.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/GiANT/GiANT_1.2.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GiANT\\')\"', '\"%R%\" -e \"library(\\'GiANT\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/GiANT/index.html'), ('license', 'Artistic-2.0'), ('summary', 'Toolbox for various enrichment analysis methods and quantification of uncertainty of gene sets.'), ('license_family', 'Proprietary')]))])), 'r-gprofiler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/<{ name }}/<{ name }}_<{ version }}.tar.gz']), ('md5', 'fb040ab9f716b4d8c38ae6e81c529869')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-plyr']), ('run', ['r-base', 'r-rcurl', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gProfileR\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=gProfileR'), ('license', 'GPL (>= 2)'), ('summary', \"Functional enrichment analysis, gene identifier conversion and mapping homologous genes across related organisms via the 'g:Profiler' toolkit (http://biit.cs.ut.ee/gprofiler/).\"), ('license_family', 'GPL3')]))])), 'r-gpseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gpseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gpseq'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'GPseq_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/GPseq_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/GPseq/GPseq_<{ version }}.tar.gz']), ('sha256', '5121b6a4723efbe91a6a86074a100c383257a1ee2628e9d9be574c4c888bbd4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GPseq\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=GPseq'), ('license', 'GPL-2'), ('summary', 'Some functions for modeling sequence read counts as a generalized poisson model and to use this model for detecting differentially expressed genes in different conditions and differentially spliced exons.'), ('license_family', 'GPL2')]))])), 'r-grimport': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-grimport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-grimport'), ('version', '0.9_0')])), ('source', CommentedMap([('fn', 'grImport_0.9-0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/grImport_0.9-0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/grImport/grImport_0.9-0.tar.gz']), ('md5', '35c4035aa3f6b2953698ddc9513f2b96')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml']), ('run', ['r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'grImport\\')\"'])])), ('about', CommentedMap([('license', 'GPL (>= 2)'), ('summary', 'Functions for converting, importing, and drawing PostScript  pictures in R plots.'), ('license_family', 'GPL3')]))])), 'r-gsalib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gsalib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gsalib'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'gsalib_2.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/gsalib_2.1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/gsalib/gsalib_2.1.tar.gz'])])), ('build', CommentedMap([('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gsalib\\')\"', '\"%R%\" -e \"library(\\'gsalib\\')\"'])])), ('about', CommentedMap([('license', 'MIT + file LICENSE'), ('summary', <ruamel.yaml.comments.TaggedScalar object at 0x110f2ddd8>)]))])), 'r-gsmoothr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gsmoothr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gsmoothr'), ('version', '0.1.7')])), ('source', CommentedMap([('fn', 'gsmoothr_0.1.7.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/gsmoothr_0.1.7.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/gsmoothr/gsmoothr_0.1.7.tar.gz']), ('md5', 'd90da237ee71eb665b8d59f210c5b6ee')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gsmoothr\\')\"', '\"%R%\" -e \"library(\\'gsmoothr\\')\"'])])), ('about', CommentedMap([('license', 'LGPL (>= 2.0)'), ('summary', 'Tools rewritten in C for various smoothing tasks'), ('license_family', 'LGPL')]))])), 'r-histonehmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-histonehmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://histonehmm.molgen.mpg.de/histoneHMM_<{ version }}.tar.gz'), ('md5', '<{ hash }}'), ('patches', 'loghmm.patch')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcpp', 'bioconductor-genomicranges', 'bioconductor-rsamtools', 'r-mvtnorm', 'r-optparse', 'bioconductor-biocstyle', 'gcc']), ('run', ['r-base', 'r-rcpp', 'bioconductor-genomicranges', 'bioconductor-rsamtools', 'r-mvtnorm', 'r-optparse', 'bioconductor-biocstyle', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'histoneHMM\\')\"'])])), ('about', CommentedMap([('home', 'http://histonehmm.molgen.mpg.de/'), ('license', 'GPL'), ('summary', 'histoneHMM is a software to analyse ChIP-seq data of histone modifications with broad genomic footprints like H3K27me3. It allows for calling modified regions in single samples as well as for calling differentially modified regions in a comparison of two samples')]))])), 'r-idpmisc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-idpmisc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-idpmisc'), ('version', '1.1.17')])), ('source', CommentedMap([('fn', 'IDPmisc_1.1.17.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/IDPmisc_1.1.17.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/IDPmisc/IDPmisc_1.1.17.tar.gz']), ('md5', '217e100a5a61c9843a0ef56bf11e2fb9')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice >=0.19_26', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'r-lattice >=0.19_26'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IDPmisc\\')\"', '\"%R%\" -e \"library(\\'IDPmisc\\')\"'])])), ('about', CommentedMap([('license', 'GPL (>= 3)'), ('summary', 'The IDPmisc package contains different high-level graphics functions for displaying large datasets, displaying circular data in a very flexible way, finding local maxima, brewing color ramps, drawing nice arrows, zooming 2D-plots, creating figures with differently colored margin and plot region.  In addition, the package contains auxiliary functions for data manipulation like omitting observations with irregular values or selecting data by logical vectors, which include NAs. Other functions are especially useful in spectroscopy and analyses of environmental data: robust baseline fitting, finding peaks in spectra.'), ('license_family', 'GPL3')]))])), 'r-idr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-idr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-idr'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'idr_1.2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/idr_1.2.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/idr/idr_1.2.tar.gz']), ('md5', 'e35d7c0013d35fe0dfa3b26b38feb137')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'idr\\')\"'])])), ('about', CommentedMap([('home', 'https://sites.google.com/site/anshulkundaje/projects/idr'), ('license', 'GPL (>= 2.0)'), ('summary', 'This is a package for estimating the copula mixture model and plotting correspondence curves in \"Measuring reproducibility of high-throughput experiments\" (2011), Annals of Applied Statistics, Vol. 5, No. 3, 1752-1779, by Li, Brown, Huang, and Bickel ')]))])), 'r-inlinedocs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-inlinedocs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-inlinedocs'), ('version', '2013.9.3')])), ('source', CommentedMap([('fn', 'inlinedocs_2013.9.3.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/inlinedocs_2013.9.3.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/inlinedocs/inlinedocs_2013.9.3.tar.gz']), ('md5', '16de1fef5c76f458c06d438fba67791f')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'inlinedocs\\')\"'])])), ('about', CommentedMap([('home', 'http://inlinedocs.r-forge.r-project.org'), ('license', 'GPL-2 | GPL-3'), ('summary', 'Generates Rd files from R source code with comments. The main features of the default syntax are that (1) docs are defined in comments near the relevant code, (2) function argument names are not repeated in comments, and (3) examples are defined in R code, not comments. It is also easy to define a new syntax.'), ('license_family', 'GPL3')]))])), 'r-km.ci': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-km.ci/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-km.ci'), ('version', '0.5_2')])), ('source', CommentedMap([('fn', 'km.ci_0.5-2.tar.gz'), ('url', 'https://cran.rstudio.com/src/contrib/km.ci_0.5-2.tar.gz'), ('md5', '9079410df73a8a06ea0405e7faf85ea7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-survival']), ('run', ['r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'km.ci\\')\"', '\"%R%\" -e \"library(\\'km.ci\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.rstudio.com/web/packages/km.ci/index.html'), ('license', 'GPL-2|GPL-3'), ('summary', \"Computes various confidence intervals for the Kaplan-Meier estimator, namely Petos CI, Rothman CI, CI's based on Greenwoods variance, Thomas and Grunkemeier CI and the simultaneous confidence bands by Nair and Hall and Wellner.\")]))])), 'r-kmsurv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-kmsurv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-kmsurv'), ('version', '0.1_5')])), ('source', CommentedMap([('fn', 'KMsurv_0.1-5.tar.gz'), ('url', 'https://cran.rstudio.com/src/contrib/KMsurv_0.1-5.tar.gz'), ('md5', '4979464e3ee73b03891b66e53231b2be')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KMsurv\\')\"', '\"%R%\" -e \"library(\\'KMsurv\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.rstudio.com/web/packages/KMsurv/index.html'), ('license', 'GPL (>= 3)'), ('summary', 'Data sets and functions for Klein and Moeschberger (1997), \"Survival Analysis, Techniques for Censored and Truncated Data\", Springer.')]))])), 'r-kriging': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-kriging/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/<{ name }}_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/<{ name }}/<{ name }}_<{ version }}.tar.gz']), ('md5', 'e0d34b36f6a31933b88bd012f7df7b0e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'kriging\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=kriging'), ('license', 'GPL-2'), ('summary', 'Simple and highly optimized ordinary kriging algorithm to plot geographical data'), ('license_family', 'GPL2')]))])), 'r-lsd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-lsd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-lsd'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'LSD_3.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/LSD_3.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/LSD/LSD_3.0.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'LSD\\')\"', '\"%R%\" -e \"library(\\'LSD\\')\"'])])), ('about', CommentedMap([('license', 'Unlimited'), ('summary', <ruamel.yaml.comments.TaggedScalar object at 0x1119aae80>)]))])), 'r-maldiquant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-maldiquant/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-maldiquant'), ('version', '1.14')])), ('source', CommentedMap([('fn', 'MALDIquant_1.14.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/MALDIquant/MALDIquant_1.14.tar.gz']), ('sha256', 'ea4ea0c2b4d2099813fe4df4cd32a9a5b7f695017035ff706f306e28540ecfb8')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MALDIquant\\')\"', '\"%R%\" -e \"library(\\'MALDIquant\\')\"'])])), ('about', CommentedMap([('home', 'http://strimmerlab.org/software/maldiquant/ https://github.com/sgibb/MALDIquant/'), ('license', 'GPL (>= 3)'), ('summary', 'A complete analysis pipeline for matrix-assisted laser desorption/ionization-time-of-flight (MALDI-TOF) and other two-dimensional mass spectrometry data. In addition to commonly used plotting and processing methods it includes distinctive features, namely baseline subtraction methods such as morphological filters (TopHat) or the statistics-sensitive non-linear iterative peak-clipping algorithm (SNIP), peak alignment using warping functions, handling of replicated measurements as well as allowing spectra with different resolutions.')]))])), 'r-maldiquantforeign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-maldiquantforeign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-maldiquantforeign'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'MALDIquantForeign_0.10.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/MALDIquantForeign_0.10.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/MALDIquantForeign/MALDIquantForeign_0.10.tar.gz']), ('sha256', '2356bdc2437c147614c1398c391e3ce21b5970b146ddcd837f7c0c3178dd34c0')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-maldiquant >=1.11.7', 'r-xml', 'r-base64enc', 'r-digest', 'r-readbrukerflexdata >=1.7', 'r-readmzxmldata >=2.7']), ('run', ['r-base', 'r-maldiquant >=1.11.7', 'r-xml', 'r-base64enc', 'r-digest', 'r-readbrukerflexdata >=1.7', 'r-readmzxmldata >=2.7'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MALDIquantForeign\\')\"', '\"%R%\" -e \"library(\\'MALDIquantForeign\\')\"'])])), ('about', CommentedMap([('home', 'http://strimmerlab.org/software/maldiquant/ https://github.com/sgibb/MALDIquantForeign/'), ('license', 'GPL (>= 3)'), ('summary', 'Functions for reading (tab, csv, Bruker fid, Ciphergen XML, mzXML, mzML, imzML, Analyze 7.5, CDF, mMass MSD) and writing (tab, csv, mMass MSD, mzML, imzML) different file formats of mass spectrometry data into/from MALDIquant objects.')]))])), 'r-matrixeqtl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-matrixeqtl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-matrixeqtl'), ('version', '2.1.1')])), ('source', CommentedMap([('fn', 'MatrixEQTL_2.1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/MatrixEQTL_2.1.1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/MatrixEQTL/MatrixEQTL_2.1.1.tar.gz']), ('md5', 'd029efe74270a9a45b635ed0589919c9')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MatrixEQTL\\')\"', '\"%R%\" -e \"library(\\'MatrixEQTL\\')\"'])])), ('about', CommentedMap([('home', 'http://www.bios.unc.edu/research/genomic_software/Matrix_eQTL/'), ('license', 'LGPL-3'), ('summary', 'Matrix eQTL is designed for fast eQTL analysis on large datasets. Matrix eQTL can test for association between genotype and gene expression using linear regression  with either additive or ANOVA genotype effects. The models can include covariates to account for factors  as population stratification, gender, and clinical variables.  It also supports models with heteroscedastic and/or correlated errors, false discovery rate estimation and separate treatment of local (cis) and distant (trans) eQTLs.'), ('license_family', 'LGPL')]))])), 'r-micropan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-micropan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-micropan'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'micropan_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/micropan_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/micropan/micropan_<{ version }}.tar.gz']), ('sha256', 'efb4491b04f781a99fb9180689b7788618800618c3433a8af897ee926f556a67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-microseq']), ('run', ['r-base', 'r-igraph', 'r-microseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'micropan\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=micropan'), ('license', 'GPL-2'), ('summary', 'A collection of functions for computations and visualizations of microbial pan-genomes.'), ('license_family', 'GPL2')]))])), 'r-microseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-microseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-microseq'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'microseq_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/microseq_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/microseq/microseq_<{ version }}.tar.gz']), ('sha256', '9ac33165011c856bf149c0e15340a4677888844c7a3e0f5e61f949c099778193')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcpp >=0.11.1', 'gcc']), ('run', ['r-base', 'r-rcpp >=0.11.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'microseq\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=microseq'), ('license', 'GPL-2'), ('summary', 'Basic functions for microbial sequence data analysis.'), ('license_family', 'GPL2')]))])), 'r-mixomics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mixomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mixomics'), ('version', '6.1.1')])), ('source', CommentedMap([('fn', 'mixOmics_6.1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/mixOmics/mixOmics_6.1.1.tar.gz']), ('sha256', '0a0b5cc2cb7ffcadcf2a5eab35c0f2d8cc9d1a60e7c83086a3909742c02bb471')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu']), ('run', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mixOmics\\')\"', '\"%R%\" -e \"library(\\'mixOmics\\')\"'])])), ('about', CommentedMap([('home', 'http://www.mixOmics.org'), ('license', 'GPL (>= 2)'), ('summary', 'Multivariate methods are well suited to large omics data sets where the number of\\nvariables (e.g. genes, proteins, metabolites) is much larger than the number of\\nsamples (patients, cells, mice). They have the appealing properties of reducing\\nthe dimension of the data by using instrumental variables (components), which are\\ndefined as combinations of all variables. Those components are then used to produce\\nuseful graphical outputs that enable better understanding of the relationships and\\ncorrelation structures between the different data sets that are integrated. mixOmics\\noffers a wide range of multivariate methods for the exploration and integration\\nof biological datasets with a particular focus on variable selection. The package\\nproposes several sparse multivariate models we have developed to identify the key\\nvariables that are highly correlated, and/or explain the biological outcome of interest.\\nThe data that can be analysed with mixOmics may come from high throughput sequencing\\ntechnologies, such as omics data (transcriptomics, metabolomics, proteomics, metagenomics\\netc) but also beyond the realm of omics (e.g. spectral imaging). The methods implemented\\nin mixOmics can also handle missing values without having to delete entire rows\\nwith missing data. A non exhaustive list of methods include generalised Canonical\\nCorrelation Analysis, sparse Partial Least Squares and sparse Discriminant Analysis.\\nRecently we implemented integrative methods to combine multiple data sets: horizontal\\nintegration with regularised Generalised Canonical Correlation Analysis and vertical\\nintegration with multi-group Partial Least Squares.\\n'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'r-mlgt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mlgt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mlgt'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'mlgt_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/mlgt_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/mlgt/mlgt_<{ version }}.tar.gz']), ('sha256', 'a93d6a2aafec41946773484a1bc5087cd56b967e4f8de3e68c69e4b7abc16ddb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mlgt\\')\"'])])), ('about', CommentedMap([('home', 'http://personalpages.manchester.ac.uk/staff/David.Gerrard/'), ('license', 'GPL (>= 2)'), ('summary', 'Processing and analysis of high throughput (Roche 454) sequences generated from multiple loci and multiple biological samples. Sequences are assigned to their locus and sample of origin, aligned and trimmed. Where possible, genotypes are called and variants mapped to known alleles.'), ('license_family', 'GPL3')]))])), 'r-mmgenome': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mmgenome/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/MadsAlbertsen/mmgenome/archive/a7e3e7bc5f7c0b36a1403c4cca2628e9a46a0297.tar.gz'), ('md5', '9475010aa10503678d7548d5355fbbc8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-sp >=1.0.15', 'r-gridextra >=0.9.1', 'r-ggplot2 >=1.0.0', 'r-knitr >=1.6', 'r-vegan >=2.0.10', 'bioconductor-biostrings >=2.32.0', 'r-igraph >=1.0.0', 'r-reshape2 >=1.4', 'r-dplyr >=0.4.0']), ('run', ['r-base', 'r-sp >=1.0.15', 'r-gridextra >=0.9.1', 'r-ggplot2 >=1.0.0', 'r-knitr >=1.6', 'r-vegan >=2.0.10', 'bioconductor-biostrings >=2.32.0', 'r-igraph >=1.0.0', 'r-reshape2 >=1.4', 'r-dplyr >=0.4.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mmgenome\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/MadsAlbertsen/mmgenome'), ('license', 'AGPL-3'), ('summary', 'Tools for extracting individual genomes from metagneomes')]))])), 'r-mutationalpatterns': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mutationalpatterns/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mutationalpatterns'), ('version', '0.2b')])), ('source', CommentedMap([('fn', 'MutationalPatterns-v0.2-beta.tar.gz'), ('url', 'https://github.com/CuppenResearch/MutationalPatterns/archive/v0.2-beta.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MutationalPatterns\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/CuppenResearch/MutationalPatterns'), ('license', 'MIT'), ('summary', 'R package for extracting and visualizing mutational patterns in base substitution catalogues')]))])), 'r-mutoss': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mutoss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mutoss'), ('version', '0.1_10')])), ('source', CommentedMap([('fn', 'mutoss_0.1-10.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/mutoss_0.1-10.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/mutoss/mutoss_0.1-10.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-multcomp >=1.1_0', 'bioconductor-multtest >=2.2.0', 'r-mvtnorm', 'r-plotrix']), ('run', ['r-base', 'r-multcomp >=1.1_0', 'bioconductor-multtest >=2.2.0', 'r-mvtnorm', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mutoss\\')\"', '\"%R%\" -e \"library(\\'mutoss\\')\"'])])), ('about', CommentedMap([('home', 'http://mutoss.r-forge.r-project.org/'), ('license', 'GPL'), ('summary', 'The Mutoss package and accompanying mutossGUI package are designed to ease the application and comparison of multiple hypothesis testing procedures.')]))])), 'r-nanostringnorm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-nanostringnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-nanostringnorm'), ('version', '1.1.21')])), ('source', CommentedMap([('fn', 'NanoStringNorm_1.1.21.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/NanoStringNorm_1.1.21.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/NanoStringNorm/NanoStringNorm_1.1.21.tar.gz'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gdata >=2.8.2', 'bioconductor-vsn >=3.22.0', 'r-googlevis', 'r-lme4', 'r-runit']), ('run', ['r-base', 'r-gdata >=2.8.2', 'bioconductor-vsn >=3.22.0', 'r-googlevis', 'r-lme4', 'r-runit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NanoStringNorm\\')\"', '\"%R%\" -e \"library(\\'NanoStringNorm\\')\"'])])), ('about', CommentedMap([('license', 'GPL-2'), ('summary', 'A set of tools for normalizing, diagnostics and visualization of NanoString nCounter data.')]))])), 'r-nastiseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-nastiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-nastiseq'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'NASTIseq_1.0.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/NASTIseq/NASTIseq_1.0.tar.gz'), ('md5', 'e5dd75117e910dec6f6c22b48f570730')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rocr', 'bioconductor-deseq']), ('run', ['r-base', 'r-rocr', 'bioconductor-deseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NASTIseq\\')\"'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/'), ('license', 'GPL-2.0'), ('summary', 'Pairs of RNA molecules transcribed from partially or entirely complementary loci are called cis-natural antisense transcripts (cis-NATs), and they play key roles in the regulation of gene expression in many organisms. A promising experimental tool for profiling sense and antisense transcription is strand-specific RNA sequencing (ssRNA-seq). To identify cis-NATs using ssRNA-seq, we developed a new computational method based on model comparison that incorporates the inherent variable efficiency of generating perfectly strand-specific libraries. Applying the method to new ssRNA-seq data from whole root and cell-type specific Arabidopsis libraries confirmed most of the known cis-NAT pairs and identified hundreds of additional cis-NAT pairs.')]))])), 'r-patpro': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-patpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-patpro'), ('version', '1.1.0')])), ('source', CommentedMap([('fn', 'patPRO_1.1.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/patPRO_1.1.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/patPRO/patPRO_1.1.0.tar.gz']), ('md5', '5eeb50baecd4c01bf51b9fe2b2801d7e')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-reshape2']), ('run', ['r-base', 'r-rcolorbrewer', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'patPRO\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/Microbiology/patPRO'), ('license', 'GPL-3'), ('summary', 'Quickly and easily visualize longitudinal microbiome profiles using standard output from the QIIME microbiome analysis toolkit (see <http://qiime.org> for more information).'), ('license_family', 'GPL3')]))])), 'r-peer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-peer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-peer'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'R_peer_source_1.3.tgz'), ('url', 'https://github.com/downloads/PMBio/peer/R_peer_source_1.3.tgz'), ('md5', 'b5570546d219c53f983bff8a725492f4')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'peer\\')\"'])])), ('about', CommentedMap([('license', 'GPL 2.0'), ('summary', 'PEER is a collection of Bayesian approaches to infer hidden determinants and their effects from gene expression profiles using factor analysis methods. Applications of PEER have (i) detected batch effects and experimental confounders (ii) increased the number of expression QTL findings by threefold and (iii) allowed inference of intermediate cellular traits, such as transcription factor or pathway activations. This project offers an efficient and versatile C++ implementation of the underlying algorithms with user-friendly interfaces to R and python. ')]))])), 'r-plasmidprofiler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-plasmidprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-plasmidprofiler'), ('version', '0.1.6')])), ('source', CommentedMap([('fn', 'Plasmidprofiler_0.1.6.tar.gz'), ('url', 'https://cran.r-project.org/src/contrib/Plasmidprofiler_0.1.6.tar.gz'), ('sha1', 'fb707f9418926600789e4bc3faf951d583eb62b4')])), ('build', CommentedMap([('number', 3), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ape', 'r-dplyr', 'r-gdata', 'r-ggdendro', 'r-ggplot2', 'r-gridextra', 'r-gtable', 'r-htmlwidgets', 'r-magrittr', 'r-plotly', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-stringr', 'pandoc']), ('run', ['r-base', 'r-ape', 'r-dplyr', 'r-gdata', 'r-ggdendro', 'r-ggplot2', 'r-gridextra', 'r-gtable', 'r-htmlwidgets', 'r-magrittr', 'r-plotly', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-stringr', 'pandoc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Plasmidprofiler\\')\"'])])), ('about', CommentedMap([('license', 'GPL'), ('summary', 'Plasmid profiler')]))])), 'r-pore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-pore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-pore'), ('version', '0.24')])), ('source', CommentedMap([('fn', 'poRe_0.24.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/rpore/0.24/poRe_0.24.tar.gz'), ('sha256', '5bcf5a2dfe3e714baef545972cd05901198cd58949cc9951270fed25a94d305e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-rhdf5', 'r-bit64', 'r-shiny', 'r-data.table', 'r-svdialogs', 'parallel']), ('run', ['r-base', 'bioconductor-rhdf5', 'r-bit64', 'r-shiny', 'r-data.table', 'r-svdialogs', 'parallel'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'poRe\\')\"'])])), ('about', CommentedMap([('license', 'BSD'), ('summary', 'An R package to enable organisation and visualisation of nanopore sequencing data'), ('home', 'http://sourceforge.net/projects/rpore/')]))])), 'r-prestor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-prestor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prestor'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.zip'), ('url', ['https://bitbucket.org/javh/prototype-prestor/get/<{ version }}.zip']), ('md5', '085f7c2e289f73ad9dbf779fac187211')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['r-base >=3.4.1', 'r-devtools', 'r-dplyr', 'r-ggplot2', 'r-hexbin', 'r-knitr', 'r-optparse', 'r-rcpp >=0.12.11', 'r-rmarkdown', 'r-roxygen2', 'r-testthat', 'r-tidyr', 'r-bibtex', 'r-captioner', 'r-alakazam']), ('run', ['r-base >=3.4.1', 'r-devtools', 'r-dplyr', 'r-ggplot2', 'r-hexbin', 'r-knitr', 'r-optparse', 'r-rcpp >=0.12.11', 'r-rmarkdown', 'r-roxygen2', 'r-testthat', 'r-tidyr', 'r-bibtex', 'r-captioner', 'r-alakazam', 'pandoc', 'texlive-core'])])), ('test', CommentedMap([('commands', ['Rscript -e \"library(prestor)\"'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/javh/prototype-prestor'), ('summary', 'A prototype package for generating quality control plots from pRESTO output.'), ('license', 'Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)'), ('license_url', 'https://creativecommons.org/licenses/by-sa/4.0/legalcode'), ('license_family', 'OTHER')]))])), 'r-pscbs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-pscbs/0.61.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-pscbs'), ('version', '0.61.0')])), ('source', CommentedMap([('fn', 'PSCBS_0.61.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/PSCBS_0.61.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/PSCBS/PSCBS_0.61.0.tar.gz']), ('sha256', '0497e728d8c12896dbd2cf89a7262718ad1cb5a87f6155ea92acad6c3391ba12')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-dnacopy >=1.40.0', 'bioconductor-aroma.light >=2.2.1', 'r-r.cache >=0.12.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-future >=1.1.1', 'r-listenv >=0.6.0', 'r-matrixstats >=0.51.0']), ('run', ['r-base', 'bioconductor-dnacopy >=1.40.0', 'bioconductor-aroma.light >=2.2.1', 'r-r.cache >=0.12.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-future >=1.1.1', 'r-listenv >=0.6.0', 'r-matrixstats >=0.51.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSCBS\\')\"', '\"%R%\" -e \"library(\\'PSCBS\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/HenrikBengtsson/PSCBS'), ('license', 'GPL (>= 2)'), ('summary', 'Segmentation of allele-specific DNA copy number data and detection of regions with abnormal copy number within each parental chromosome.  Both tumor-normal paired and tumor-only analyses are supported.')]))])), 'r-psych': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-psych/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-psych'), ('version', '1.5.8')])), ('source', CommentedMap([('fn', 'psych_1.5.8.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/psych/psych_1.5.8.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mnormt']), ('run', ['r-base', 'r-mnormt'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'psych\\')\"', '\"%R%\" -e \"library(\\'psych\\')\"'])])), ('about', CommentedMap([('home', 'http://personality-project.org/r/psych http://personality-project.org/r/psych-manual.pdf'), ('license', 'GPL (>= 2)'), ('summary', 'A general purpose toolbox for personality, psychometrics and experimental psychology.   Functions are primarily for multivariate analysis and scale construction using factor analysis, principal component analysis, cluster analysis and reliability analysis, although others provide basic descriptive statistics. Item Response Theory is done using  factor analysis of tetrachoric and polychoric correlations. Functions for analyzing data at multi-levels include within and between group statistics, including correlations and factor analysis.   Functions for simulating particular item and test structures are included. Several functions serve as a useful front end for structural equation modeling.  Graphical displays of path diagrams, factor analysis and structural equation models are created using basic graphics. Some of the functions are written to support a book on psychometrics as well as publications in personality research. For more information, see the personality-project.org/r webpage.')]))])), 'r-qiimer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-qiimer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-qiimer'), ('version', '0.9.4')])), ('source', CommentedMap([('fn', 'qiimer_0.9.4.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/qiimer_0.9.4.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/qiimer/qiimer_0.9.4.tar.gz'])])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-pheatmap']), ('run', ['r-base', 'r-pheatmap'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qiimer\\')\"', '\"%R%\" -e \"library(\\'qiimer\\')\"'])])), ('about', CommentedMap([('home', 'http://www.r-project.org'), ('license', 'GPL (>= 2)'), ('summary', 'Open QIIME output files in R, compute statistics, and create plots from the data.'), ('license_family', 'GPL3')]))])), 'r-qqman': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-qqman/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-qqman'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'qqman_0.1.2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/qqman/qqman_0.1.2.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qqman\\')\"', '\"%R%\" -e \"library(\\'qqman\\')\"'])])), ('about', CommentedMap([('license', 'GPL-3'), ('summary', 'Q-Q and manhattan plots for GWAS data'), ('license_family', 'GPL3')]))])), 'r-rBLAST': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rBLAST/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-rblast'), ('version', '0.99.1')])), ('source', CommentedMap([('fn', '5cbc5039705e45c83d88c324c4f2a64dfc63e4e1.tar.gz'), ('url', ['https://github.com/mhahsler/rBLAST/archive/5cbc5039705e45c83d88c324c4f2a64dfc63e4e1.tar.gz']), ('md5', '5773ffb34b208af1d7999518c645728a')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings']), ('run', ['r-base', 'bioconductor-biostrings', 'blast'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rBLAST\\')\"', '\"%R%\" -e \"library(\\'rBLAST\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/mhahsler/rBLAST'), ('license', 'GPL3'), ('summary', 'Seamlessly interfaces the Basic Local Alignment Search Tool (BLAST) to search genetic sequence data bases. This work was partially supported by grant no. R21HG005912 from the National Human Genome Research Institute.')]))])), 'r-rcircos': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rcircos/1.1.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-rcircos'), ('version', '1.1.3')])), ('source', CommentedMap([('fn', 'RCircos_1.1.3.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/RCircos/RCircos_1.1.3.tar.gz'])])), ('build', CommentedMap([('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCircos\\')\"', '\"%R%\" -e \"library(\\'RCircos\\')\"'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/henryhzhang/rcircos'), ('license', 'GPL (>= 2)'), ('summary', 'A simple and flexible way to generate Circos 2D track plot images for genomic data visualization is implemented in this package. The types of plots include: heatmap, histogram, lines, scatterplot, tiles and plot items for further decorations include connector, link (lines and ribbons), and text (gene) label. All functions require only R graphics package that comes with R base installation.  ')]))])), 'r-readbrukerflexdata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-readbrukerflexdata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-readbrukerflexdata'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'readBrukerFlexData_1.8.2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/readBrukerFlexData/readBrukerFlexData_1.8.2.tar.gz']), ('sha256', '6c745e43e8a21a02660bef933785ce9d8b3d845a6f0976e04177c024a8d94fb1')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'readBrukerFlexData\\')\"', '\"%R%\" -e \"library(\\'readBrukerFlexData\\')\"'])])), ('about', CommentedMap([('home', 'http://strimmerlab.org/software/maldiquant/ https://github.com/sgibb/readBrukerFlexData/'), ('license', 'GPL (>= 3)'), ('summary', 'Reads data files acquired by MALDI-TOF MS on Bruker Daltonics machines of the \\\\*flex series.')]))])), 'r-readmzxmldata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-readmzxmldata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-readmzxmldata'), ('version', '2.8.1')])), ('source', CommentedMap([('fn', 'readMzXmlData_2.8.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/readMzXmlData_2.8.1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/readMzXmlData/readMzXmlData_2.8.1.tar.gz']), ('sha256', '1c3d842d2a647020e19d42a448578e8bd48b4bbe56d8927c8d239623a582960e')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml', 'r-base64enc', 'r-digest']), ('run', ['r-base', 'r-xml', 'r-base64enc', 'r-digest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'readMzXmlData\\')\"', '\"%R%\" -e \"library(\\'readMzXmlData\\')\"'])])), ('about', CommentedMap([('home', 'http://strimmerlab.org/software/maldiquant/ https://github.com/sgibb/readMzXmlData/'), ('license', 'GPL (>= 3)'), ('summary', 'Functions for reading mass spectrometry data in mzXML format.')]))])), 'r-rematch2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rematch2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-rematch2'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'rematch2_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/rematch2_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/rematch2/rematch2_<{ version }}.tar.gz']), ('sha256', '0612bb904334bd022ba6d1e69925b1e85f8e86b15ec65476777828776e89609a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-tibble']), ('run', ['r-base', 'r-tibble'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rematch2\\')\"', '\"%R%\" -e \"library(\\'rematch2\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/r-lib/rematch2#readme'), ('license', 'MIT + file LICENSE'), ('summary', \"Wrappers on 'regexpr' and 'gregexpr' to return the match results in tidy data frames.\"), ('license_family', 'MIT')]))])), 'r-rentrez': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rentrez/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-rentrez'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'rentrez_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/rentrez_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/rentrez/rentrez_<{ version }}.tar.gz']), ('sha256', '8dd46c03cbb05d4a3e787b7fc9a1858fcbdf1bb0faf5ee438a4f2ea4dbca9d56')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml', 'r-httr >=0.5', 'r-jsonlite >=0.9']), ('run', ['r-base', 'r-xml', 'r-httr >=0.5', 'r-jsonlite >=0.9'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rentrez\\')\"', '\"%R%\" -e \"library(\\'rentrez\\')\"'])])), ('about', CommentedMap([('home', 'http://github.com/ropensci/rentrez'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', \"Provides an R interface to the NCBI's EUtils API allowing users to search databases\\nlike GenBank and PubMed, process the results of those searches and pull data into\\ntheir R sessions.\\n\"), ('license_family', 'MIT')]))])), 'r-robustrankaggreg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-robustrankaggreg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-robustrankaggreg'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'RobustRankAggreg_1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/RobustRankAggreg_1.1.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/RobustRankAggreg/RobustRankAggreg_1.1.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc', 'llvm']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RobustRankAggreg\\')\"'])])), ('about', CommentedMap([('license', 'GPL-2'), ('summary', 'Methods for aggregating ranked lists, especially lists of genes. It implements the Robust Rank Aggregation (Kolde et. al in preparation) and some other simple algorithms for the task. RRA method uses a probabilistic model for aggregation that is robust to noise and also facilitates the calculation of significance probabilities for all the elements in the final ranking.')]))])), 'r-rphylip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rphylip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://cran.r-project.org/src/contrib/<{ name }}_<{ version }}.tar.gz'), ('md5', '99597a778fc5bf69c7175e4dde787970'), ('patches', ['rconsense.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ape >=3.0_10']), ('run', ['r-base', 'r-ape >=3.0_10', 'phylip >=3.696'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rphylip\\')\"', '\"%R%\" -e \"library(\\'Rphylip\\')\"'])])), ('about', CommentedMap([('home', 'http://www.phytools.org/Rphylip'), ('license', 'GPL (>= 2)'), ('summary', 'Rphylip provides an R interface for the PHYLIP package. All users of Rphylip will thus first have to install the PHYLIP phylogeny methods program package (Felsenstein 2013). See http://www.phylip.com for more  information about installing PHYLIP.'), ('license_family', 'GPL3')]))])), 'r-samr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-samr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-samr'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'samr_2.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/samr_2.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/samr/samr_2.0.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-impute', 'r-matrixstats', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'bioconductor-impute', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'samr\\')\"', '\"%R%\" -e \"library(\\'samr\\')\"'])])), ('about', CommentedMap([('home', 'http://www-stat.stanford.edu/~tibs/SAM'), ('license', 'LGPL'), ('summary', 'Significance Analysis of Microarrays'), ('license_family', 'LGPL')]))])), 'r-sartools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-sartools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/PF2-pasteur-fr/SARTools/archive/v<{ version }}.tar.gz'), ('sha256', 'e35df9c200e3fbdde43c340a5c5426715e891cd6424e4d4ebe8201d4afb23035')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown']), ('run', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SARTools\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/PF2-pasteur-fr/SARTools'), ('license', 'GPL-2'), ('summary', 'SARTools provides R tools and an environment for the statistical analysis of RNA-Seq projects load and clean data, produce figures, perform statistical analysis/testing with DESeq2 or edgeR, export results and create final report.')]))])), 'r-scrm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-scrm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-scrm'), ('version', '1.6.0_2')])), ('source', CommentedMap([('fn', 'scrm_1.6.0-2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/scrm/scrm_1.6.0-2.tar.gz']), ('sha256', '36b0090d8bbed01482b27705decc184aae328f49001e066535a22846a42975a8')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/']), ('skip', True)])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcpp >=0.11.2', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'r-rcpp >=0.11.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scrm\\')\"', '\"%R%\" -e \"library(\\'scrm\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/scrm/scrm-r'), ('license', 'GPL (>= 3)'), ('summary', 'A coalescent simulator that allows the rapid simulation of biological sequences under neutral models of evolution. Different to other coalescent based simulations, it has an optional approximation parameter that allows for high accuracy while maintaining a linear run time cost for long sequences. It is optimized for simulating massive data sets as produced by Next-Generation Sequencing technologies for up to several thousand sequences.'), ('license_family', 'GPL3')]))])), 'r-seqinr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-seqinr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-seqinr'), ('version', '3.1_3')])), ('source', CommentedMap([('fn', 'seqinr_3.1-3.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/seqinr/seqinr_3.1-3.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ade4']), ('run', ['r-base', 'r-ade4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqinr\\')\"', '\"%R%\" -e \"library(\\'seqinr\\')\"'])])), ('about', CommentedMap([('home', 'http://seqinr.r-forge.r-project.org/, Mailing list: http://r-forge.r-project.org/mail/?group_id=220'), ('license', 'GPL (>= 2)'), ('summary', 'Exploratory data analysis and data visualization for biological sequence (DNA and protein) data.  Includes also utilities for sequence data management under the ACNUC system.')]))])), 'r-sleuth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-sleuth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-sleuth'), ('version', '0.29.0')])), ('source', CommentedMap([('url', 'https://github.com/pachterlab/sleuth/archive/v0.29.0.tar.gz'), ('md5', 'd625174a31bb033d8ddf14c29c703773')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-rhdf5', 'r-ggplot2', 'r-dplyr', 'r-tidyr', 'r-shiny', 'r-data.table', 'r-matrixstats']), ('run', ['r-base', 'bioconductor-rhdf5', 'r-ggplot2', 'r-dplyr', 'r-tidyr', 'r-shiny', 'r-data.table', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sleuth\\')\"'])])), ('about', CommentedMap([('home', 'http://pachterlab.github.io/sleuth'), ('license', 'GPLv3'), ('summary', 'Sleuth is an R library for analysis of RNA-Seq experiments for which transcript abundances have been quantified with kallisto.')]))])), 'r-soap-nmr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-soap-nmr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-soap-nmr'), ('version', '0.1.0.20170207')])), ('source', CommentedMap([('fn', 'soap-nmr-0.1.0.20170207.tar.gz'), ('url', 'https://github.com/ManonMartin/SOAP-NMR/archive/83986fad26becd625c89f0cd7522e40587e83fe3.tar.gz'), ('md5', '79c22ee665352c0bd2fa2f429db48d30')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrix', 'r-ptw', 'r-ggplot2', 'r-gridextra', 'r-matrixstats', 'r-reshape2']), ('run', ['r-base', 'r-matrix', 'r-ptw', 'r-ggplot2', 'r-gridextra', 'r-matrixstats', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"require(\\'SOAP\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/ManonMartin/SOAP-NMR'), ('license', 'GPL2'), ('summary', 'R package for 1H-NMR data pre-treatment ')]))])), 'r-solarius': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-solarius/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-solarius'), ('version', '0.3.0.2')])), ('source', CommentedMap([('fn', 'solarius_0.3.0.2.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/solarius_0.3.0.2.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/solarius/solarius_0.3.0.2.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-ggplot2', 'r-plyr >=1.8.1']), ('run', ['r-base', 'r-data.table', 'r-ggplot2', 'r-plyr >=1.8.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'solarius\\')\"', '\"%R%\" -e \"library(\\'solarius\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/ugcd/solarius'), ('license', 'GPL (>= 3)'), ('summary', 'SOLAR is the standard software program to perform linkage and association mappings of the quantitative trait loci (QTLs) in pedigrees of arbitrary size and complexity. This package allows the user to exploit the variance component methods implemented in SOLAR. It automates such routine operations as formatting pedigree and phenotype data. It also parses the model output and contains summary and plotting functions for exploration of the results. In addition, solarius enables parallel computing of the linkage and association analyses, that makes the calculation of genome-wide scans more efficient. See <http://solar.txbiomedgenetics.org/> for more information about SOLAR.'), ('license_family', 'GPL3')]))])), 'r-speaq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-speaq/1.2.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-speaq'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'speaq_1.2.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/speaq/speaq_1.2.1.tar.gz']), ('md5', '52b3c390a34dca0ec439c424375569ff')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-massspecwavelet']), ('run', ['r-base', 'bioconductor-massspecwavelet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'speaq\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/nghiavtr/speaq'), ('license', 'Apache License 2.0'), ('summary', 'We introduce a novel suite of informatics tools for the quantitative analysis of NMR metabolomic profile data. The core of the processing cascade is a novel peak alignment algorithm, called hierarchical Cluster-based Peak Alignment (CluPA). The algorithm aligns a target spectrum to the reference spectrum in a top-down fashion by building a hierarchical cluster tree from peak lists of reference and target spectra and then dividing the spectra into smaller segments based on the most distant clusters of the tree. To reduce the computational time to estimate the spectral misalignment, the method makes use of Fast Fourier Transformation (FFT) cross-correlation. Since the method returns a high-quality alignment, we can propose a simple methodology to study the variability of the NMR spectra. For each aligned NMR data point the ratio of the between-group and within-group sum of squares (BW-ratio) is calculated to quantify the difference in variability between and within predefined groups of NMR spectra. This differential analysis is related to the calculation of the F-statistic or a one-way ANOVA, but without distributional assumptions. Statistical inference based on the BW-ratio is achieved by bootstrapping the null distribution from the experimental data. See Vu et al. (2011)<DOI:10.1186/1471-2105-12-405> for further information.'), ('license_family', 'Apache')]))])), 'r-spp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-spp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-spp'), ('version', '1.14post')])), ('source', CommentedMap([('fn', '1.14post.zip'), ('url', 'https://github.com/hms-dbmi/spp/archive/c6526d2bc8eeaaa889970cd2f8d30c998f3ea320.zip'), ('sha256', '61f0d21f2c5127e570bf619445c89968009cfbd32b995984cf20788f986e666b'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'r_<{CONDA_R}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'gcc', 'llvm', 'zlib', 'bzip2']), ('run', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(spp)\"', '$R -e \"library(spp); read.bam.tags(system.file(\\\\\"extdata\\\\\", \\\\\"ex1.bam\\\\\", package=\\\\\"Rsamtools\\\\\"))\"'])])), ('about', CommentedMap([('license', 'GPL'), ('summary', 'ChIP-seq peak caller'), ('home', 'http://compbio.med.harvard.edu/Supplements/ChIP-seq')]))])), 'r-tfmpvalue': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-tfmpvalue/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-tfmpvalue'), ('version', '0.0.6')])), ('source', CommentedMap([('fn', 'TFMPvalue_0.0.6.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/TFMPvalue_0.0.6.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/TFMPvalue/TFMPvalue_0.0.6.tar.gz']), ('md5', '69fdf4f9b9a0f408a5cee9ce34bea261')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['r-base', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TFMPvalue\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/ge11232002/TFMPvalue'), ('license', 'GPL-2'), ('summary', 'In putative Transcription Factor Binding Sites (TFBSs)  identification from sequence/alignments, we are interested in the significance of certain match score. TFMPvalue provides the accurate calculation of P-value with  score threshold for Position Weight Matrices,  or the score with given P-value.  This package is an interface to code originally made available by  Helene Touzet and Jean-Stephane Varre, 2007,  Algorithms Mol Biol:2, 15.'), ('license_family', 'GPL2')]))])), 'r-wasabi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-wasabi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-wasabi'), ('version', 0.2)])), ('source', CommentedMap([('url', 'https://github.com/COMBINE-lab/wasabi/archive/master.zip'), ('md5', '32f8e787edb3c19307d9681e8588cca9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-rhdf5', 'r-data.table', 'r-rjson']), ('run', ['r-base', 'bioconductor-rhdf5', 'r-data.table', 'r-rjson'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'wasabi\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/COMBINE-lab/wasabi'), ('license', 'BSD-3-clause'), ('summary', 'Prepare Sailfish and Salmon output for downstream analysis')]))])), 'r-wgcna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-wgcna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-wgcna'), ('version', '1.51')])), ('source', CommentedMap([('fn', 'WGCNA_1.51.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/WGCNA_1.51.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/WGCNA/WGCNA_1.51.tar.gz']), ('sha256', 'bd6be9ea2a71a41cd758b9ae21065afc7e5d1870c88c8ee8428f6f7338b4fb43')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'WGCNA\\')\"', '\"%R%\" -e \"library(\\'WGCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/Rpackages/WGCNA/'), ('license', 'GPL (>= 2)'), ('summary', 'Functions necessary to perform Weighted Correlation Network Analysis on high-dimensional data. Includes functions for rudimentary data cleaning, construction of correlation networks, module identification, summarization, and relating of variables and modules to sample traits. Also includes a number of utility functions for data manipulation and visualization.'), ('license_family', 'GPL3')]))])), 'r-xmlrpc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-xmlrpc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-xmlrpc'), ('version', '0.2_4')])), ('source', CommentedMap([('fn', 'XMLRPC_0.2-4.tar.gz'), ('url', ['http://download.r-forge.r-project.org/src/contrib/XMLRPC_0.2-4.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-xml']), ('run', ['r-base', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XMLRPC\\')\"', '\"%R%\" -e \"library(\\'XMLRPC\\')\"'])])), ('about', CommentedMap([('home', 'https://r-forge.r-project.org'), ('license', 'BSD'), ('summary', 'A simple implementation of XML-RPC for R.')]))])), 'r-xnomial': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-xnomial/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-xnomial'), ('version', '1.0.4')])), ('source', CommentedMap([('fn', 'XNomial_1.0.4.tar.gz'), ('url', 'https://cran.r-project.org/src/contrib/XNomial_1.0.4.tar.gz'), ('md5', '3c3bf5877533c9fcbfc9bf257f5cbc29')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XNomial\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/XNomial/index.html'), ('license', 'GPL-2|GPL-3'), ('summary', 'Tests whether a set of counts fit a given expected ratio. For example, a genetic cross might be expected to produce four types in the relative frequencies of 9:3:3:1. To see whether a set of observed counts fits this expectation, one can examine all possible outcomes with xmulti() or a random sample of them with xmonte() and find the probability of an observation deviating from the expectation by at least as much as the observed. As a measure of deviation from the expected, one can use the log-likelihood ratio, the multinomial probability, or the classic chi-square statistic. A histogram of the test statistic can also be plotted and compared with the asymptotic curve.')]))])), 'r2r': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r2r/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r2r'), ('version', '1.0.4')])), ('about', CommentedMap([('home', 'http://breaker.research.yale.edu/R2R/'), ('license', 'GPL-2'), ('license_file', 'LICENSE'), ('summary', 'software to speed depiction of aesthetic consensus RNA secondary structures')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'R2R-1.0.4.tgz'), ('url', 'http://breaker.research.yale.edu/R2R/R2R-1.0.4.tgz'), ('sha256', '3578c8ad5dfc2a4e6c4f0613ca3e98a1f352af661f60886a1616c5e6d8e7440d')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'perl-threaded', 'perl-clone']), ('run', ['libgcc', 'perl-threaded', 'perl-clone'])])), ('test', CommentedMap([('commands', ['r2r --GSC-weighted-consensus $PREFIX/share/demo1.sto $PREFIX/share/demo1.cons.sto 3 0.97 0.9 0.75 4 0.97 0.9 0.75 0.5 0.1'])]))])), 'ra': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ra/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ra'), ('version', '0.9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'ra_v0.9.tar.gz'), ('url', 'https://github.com/mariokostelac/ra/archive/v0.9.tar.gz'), ('md5', '387006cf89e9cbf331e2a09b62caa002')])), ('requirements', CommentedMap([('build', ['gcc [linux]', 'llvm [osx]']), ('run', ['libgcc [linux]'])])), ('test', CommentedMap([('commands', ['zoom 2>&1 | grep usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mariokostelac/ra'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Ra is short for RNA Assembler and it is a C++ implementation of an overlap-layout-consensus transcriptome assembler.')]))])), 'ra-integrate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ra-integrate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ra-integrate'), ('version', '0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'ra-integrate_v0.1.tar.gz'), ('url', 'https://github.com/mariokostelac/ra-integrate/archive/v0.1.tar.gz'), ('md5', '10a5cb06ccb001ac0959d519df986e30'), ('patches', ['ra-integrate.patch'])])), ('requirements', CommentedMap([('build', ['gcc [linux]', 'llvm [osx]']), ('run', ['libgcc [linux]', 'ruby', 'graphviz', 'ra'])])), ('test', CommentedMap([('commands', ['ra-integrate -h 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mariokostelac/ra-integrate'), ('license', 'GPL3'), ('summary', 'Integration of the Ra assembler - a de novo DNA assembler for third generation sequencing data.')]))])), 'rabix-bunny': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rabix-bunny/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/rabix/bunny'), ('license', 'Apache v2'), ('summary', 'Open-source development kit for the Common Workflow Language from Seven Bridges. The Rabix executor Bunny, which can be used to execute apps locally from the command line.')])), ('package', CommentedMap([('name', 'rabix-bunny'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('source', CommentedMap([('fn', 'rabix-<{ version }}.tar.gz'), ('url', 'https://github.com/rabix/bunny/releases/download/v<{ version }}/rabix-<{ version }}.tar.gz'), ('md5', '3aa876849ac7fbc81de8246ce4ce1069')])), ('requirements', CommentedMap([('run', ['openjdk >=8,<9'])])), ('test', CommentedMap([('commands', ['rabix --version'])]))])), 'racon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/racon/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/isovic/racon/files/741519/racon-v0.5.0.tar.gz'), ('md5', '4de02dc66849b067aa90b26c5610e524')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib', 'python', 'numpy', 'matplotlib', 'mummer'])])), ('test', CommentedMap([('commands', [\"racon 2>&1 | grep 'options'\"])])), ('about', CommentedMap([('home', 'https://github.com/isovic/racon'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.')]))])), 'rad_haplotyper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rad_haplotyper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rad_haplotyper'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'v1.1.6.tar.gz'), ('url', 'https://github.com/chollenbeck/rad_haplotyper/archive/v1.1.6.tar.gz'), ('sha256', 'da7f663922ca66f4b736775df16e70c423dad74aeb03bc4d732572ff01d8466d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'ddocent', 'perl-bioperl', 'perl-vcftools-vcf', 'perl-bio-db-sam', 'perl-data-dumper', 'perl-getopt-long', 'perl-pod-usage', 'perl-list-moreutils', 'perl-parallel-forkmanager', 'perl-term-progressbar', 'perl-bio-cigar']), ('run', ['libgcc', 'ddocent', 'perl', 'perl-bioperl', 'perl-vcftools-vcf', 'perl-app-cpanminus', 'perl-module-build', 'perl-bio-db-sam', 'perl-data-dumper', 'perl-getopt-long', 'perl-pod-usage', 'perl-list-moreutils', 'perl-parallel-forkmanager', 'perl-term-progressbar', 'perl-bio-cigar'])])), ('test', CommentedMap([('commands', ['rad_haplotyper.pl --version 2>&1 | grep Version'])])), ('about', CommentedMap([('home', 'https://github.com/chollenbeck/rad_haplotyper'), ('license', 'Perl'), ('summary', 'A program for building SNP haplotypes from RAD sequencing data')]))])), 'rainbow': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rainbow/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rainbow'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'rainbow_2.0.4.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/bio-rainbow/rainbow_2.0.4.tar.gz'), ('md5', '1212af13c8a4ca241d9131789cea6a1f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['rainbow -h &> /dev/null || [[ \"$?\" == 1 ]]', 'select_all_rbcontig.pl -h &> /dev/null || [[ \"$?\" == 255 ]]', 'select_all_rbcontig.pl -h &> /dev/null || [[ \"$?\" == 255 ]]', 'select_sec_rbcontig.pl -h &> /dev/null || [[ \"$?\" == 255 ]]', 'select_best_rbcontig_plus_read1.pl -h &> /dev/null || [[ \"$?\" == 255 ]]'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/bio-rainbow/'), ('license', 'GNU General Public License version 2.0 (GPLv2)'), ('summary', 'Efficient tool for clustering and assembling short reads, especially for RAD')]))])), 'randfold': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/randfold/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'randfold'), ('version', '2.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'http://bioinformatics.psb.ugent.be/supplementary_data/erbon/nov2003/downloads/randfold-2.0.1.tar.gz'), ('md5', 'c9ebf7dc9d62fa4554a738a15fe1ded8'), ('patches', ['randfold.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['afetch -h', 'alistat -h', 'compalign -h', 'compstruct -h', 'revcomp -h', 'seqsplit -h', 'seqstat -h', 'sfetch -h', 'shuffle -h', 'sindex -h', 'sreformat -h', 'translate -h', 'weight -h', 'randfold -h'])])), ('about', CommentedMap([('home', 'http://bioinformatics.psb.ugent.be/software/details/Randfold'), ('license', 'GNU GPLv2'), ('license_file', 'LICENSE'), ('summary', 'Minimum free energy of folding randomization test software')]))])), 'rapclust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rapclust/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rapclust'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'rapclust-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/3b/ad/7dcaf6c6507d7b8703fde76aba730f83d7dc6d04c5ef88f22e10f19e9d53/rapclust-0.1.2.tar.gz'), ('md5', '7e13cecfe0ec6f12211d61888d7e6cf6')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyyaml', 'click', 'networkx', 'numpy', 'pandas', 'tqdm', 'coloredlogs']), ('run', ['python', 'pyyaml', 'click', 'networkx', 'numpy', 'pandas', 'tqdm', 'coloredlogs'])])), ('test', CommentedMap([('imports', ['rapclust']), ('commands', ['LANG=C.UTF-8 RapClust --help'])])), ('about', CommentedMap([('home', 'https://github.com/COMBINE-lab/RapClust'), ('license', 'BSD with attribution'), ('summary', 'Accurate, Fast and Lightweight Clustering of de novo Transcriptomes using Fragment Equivalence Classes'), ('license_family', 'BSD')]))])), 'rapmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rapmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rapmap'), ('version', 'v0.2.1')])), ('source', CommentedMap([('fn', 'v0.2.1.tar.gz'), ('url', 'https://github.com/COMBINE-lab/RapMap/archive/v0.2.1.tar.gz'), ('sha256', 'bc2d62e5f278fb1b0f6a8591e5ddb5d505dee7152e659d04ee5630d2a5546830')])), ('requirements', CommentedMap([('build', ['cmake']), ('run', None)])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('about', CommentedMap([('home', 'https://github.com/COMBINE-lab/RapMap'), ('license', 'GPL'), ('summary', 'Rapid sensitive and accurate read mapping via quasi-mapping')])), ('test', CommentedMap([('commands', ['rapmap --help'])]))])), 'rapsearch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rapsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rapsearch'), ('version', 2.24)])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'RAPSearch2.24_64bits.tar.gz.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/rapsearch2/RAPSearch2.24_64bits.tar.gz'), ('md5', '573ff48c2815c7e07f0268780c2590a8')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rapsearch 2>&1 | grep rapsearch', 'prerapsearch 2>&1 | grep \"Usage\\\\:\"'])])), ('about', CommentedMap([('home', 'http://omics.informatics.indiana.edu/mg/RAPSearch2/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'RAPSearch2 is a tool for fast protein similarity searches.')]))])), 'rascaf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rascaf/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('script', 'mkdir -p $PREFIX/bin; make; mv *.pl  $PREFIX/bin; mv rascaf $PREFIX/bin; mv rascaf-join $PREFIX/bin')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mourisl/Rascaf/archive/ed779e100853977307bbdc01c1644390d8556a7b.tar.gz'), ('md5', 'b05116262162712098b357871f1a79e4')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['rascaf  2>&1 | grep \"usage:\" > /dev/null', 'rascaf-join 2>&1 | grep \"usage:\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mourisl/Rascaf/commits/master'), ('summary', 'Scaffolding with RNA-seq read alignment'), ('licence', 'GPL2')]))])), 'raxml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/raxml/8.2.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'raxml'), ('version', '8.2.9')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'raxml_v8.2.9.tar.gz'), ('url', 'https://github.com/stamatak/standard-RAxML/archive/v8.2.9.tar.gz'), ('md5', 'ba77f7df7ce5e2a7967decd9e4278a72')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['raxmlHPC -h > /dev/null', 'raxmlHPC-PTHREADS -h > /dev/null', 'raxmlHPC-SSE3 -h > /dev/null', 'raxmlHPC-PTHREADS-SSE3 -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/raxml/index.html'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))])), 'ray': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ray/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Ray-<{ version }}.tar.bz2'), ('url', 'https://downloads.sourceforge.net/project/denovoassembler/Ray-<{ version }}.tar.bz2'), ('md5', '82f693c4db60af4328263c9279701009')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openmpi >=2.0.2', 'cmake']), ('run', ['libgcc', 'openmpi >=2.0.2'])])), ('test', CommentedMap([('commands', ['which Ray'])])), ('about', CommentedMap([('home', 'http://denovoassembler.sourceforge.net/index.html'), ('license', 'GPL3'), ('summary', 'Parallel genome assemblies for parallel DNA sequencing')]))])), 'razers3': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/razers3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'razers3'), ('version', '3.5.0')])), ('source', CommentedMap([('fn', 'seqan-src-2.0.1.tar.gz'), ('url', 'http://packages.seqan.de/seqan-src/seqan-src-2.0.1.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'cmake']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['razers3 --help'])])), ('about', CommentedMap([('home', 'http://www.seqan.de/projects/razers/'), ('license', 'GPLv3'), ('summary', 'a tool for mapping millions of short genomic reads onto a reference genome')]))])), 'rdfextras': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdfextras/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rdfextras'), ('version', '0.4')])), ('source', CommentedMap([('fn', 'rdfextras-0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/9f/1a/55fd2e8f7f08ddc6b86fa891f0fad0ba23ebbc82ae0b8a8f1f0fe2326137/rdfextras-0.4.tar.gz'), ('md5', 'c9905b062568ab5607dcf60573b55c2b')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['rdfpipe = rdfextras.tools.rdfpipe:main', 'csv2rdf = rdfextras.tools.csv2rdf:main', 'rdf2dot = rdfextras.tools.rdf2dot:main', 'rdfs2dot = rdfextras.tools.rdfs2dot:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'rdflib >=3.2.1', 'pyparsing <=1.5.7']), ('run', ['python', 'rdflib >=3.2.1', 'pyparsing <=1.5.7'])])), ('test', CommentedMap([('imports', ['rdfextras', 'rdfextras.sparql', 'rdfextras.sparql.results', 'rdfextras.store', 'rdfextras.store.FOPLRelationalModel', 'rdfextras.tools', 'rdfextras.utils']), ('commands', ['rdfpipe --help', 'rdf2dot -h 2>&1 | grep rdf2dot.py &> /dev/null', 'rdfs2dot -h 2>&1 | grep rdfs2dot.py &> /dev/null', 'csv2rdf --help 2>&1 | grep csv2rdf.py &> /dev/null', 'rdfgraphisomorphism --help', 'rdfpipe -h']), ('requires', ['pyparsing', 'rdflib >=3.2.1'])])), ('about', CommentedMap([('home', 'http://github.com/RDFLib/rdfextras'), ('license', 'BSD License'), ('summary', 'RDFExtras provide tools, extra stores and such for RDFLib.'), ('license_family', 'BSD')]))])), 'rdflib-jsonld': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdflib-jsonld/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rdflib-jsonld'), ('version', '0.4')])), ('source', CommentedMap([('fn', 'rdflib-jsonld-0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/ba/48/edaad22fc9de34500699f0c7fc9124385dd425fd18857244f126a6f7df66/rdflib-jsonld-0.4.0.tar.gz'), ('md5', '69097f57d302791a2959c07e110c47cf')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'rdflib >=4.2.2']), ('run', ['python', 'setuptools', 'rdflib >=4.2.2'])])), ('test', CommentedMap([('imports', ['rdflib_jsonld'])])), ('about', CommentedMap([('home', 'https://github.com/RDFLib/rdflib-jsonld'), ('license', 'BSD License'), ('summary', 'rdflib extension adding JSON-LD parser and serializer')]))])), 'rdock': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdock/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rdock'), ('version', 2013.1)])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'rDock_2013.1_src.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/rDock/rDock_2013.1_src_all.tar.gz'), ('sha256', 'e716998c3f8a2a70205a8d30ba22675bfdb1764d13c858645138c5eadf2a37e9'), ('patches', ['rDock_python_fixes.patch', 'rDock_fix_paths.patch'])])), ('requirements', CommentedMap([('build', ['cppunit', 'gcc', 'perl', 'popt', 'python']), ('run', ['libgcc', 'numpy', 'perl', 'popt', 'openbabel'])])), ('test', CommentedMap([('commands', ['rbcavity --help', 'rbdock --help', 'sdrmsd --help', 'sdsort -h'])])), ('about', CommentedMap([('home', 'http://rdock.sourceforge.net/'), ('license', 'LGPLv3'), ('license_file', 'license.txt'), ('summary', 'A Fast, Versatile and Open Source Program for Docking Ligands to Proteins and Nucleic Acids')]))])), 'rdp-readseq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdp-readseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/rdpstaff/ReadSeq/archive/<{ version }}.tar.gz'), ('md5', 'fb969e436c9d7a2d5e87bc7363c30bbf')])), ('about', CommentedMap([('home', 'https://github.com/rdpstaff/ReadSeq'), ('license', 'GPL'), ('summary', 'Java based common sequence file format reader and sequence file manipulation.')])), ('requirements', CommentedMap([('build', ['openjdk', 'ant']), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"ReadSeq 2>&1 | grep 'ReadSeqMain'\"])]))])), 'rdptools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdptools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('git_url', 'https://github.com/rdpstaff/RDPTools.git'), ('git_rev', '96770fa51a9c15a2686c1c6f2bfb293425c698e8')])), ('requirements', CommentedMap([('build', ['openjdk', 'ant']), ('run', ['openjdk', 'python'])])), ('test', CommentedMap([('commands', [\"AbundanceStats 2>&1 | grep 'Main'\", \"AlignmentTools 2>&1 | grep 'AlignmentToolsMain'\", 'Clustering', 'FrameBot', 'KmerFilter', \"ProbeMatch 2>&1 | grep 'PrimerMatch'\", \"ReadSeq 2>&1 | grep 'ReadSeqMain'\", \"SeqFilters 2>&1 | grep 'InitialProcessorMain'\", \"SequenceMatch 2>&1 | grep 'seqmatch'\", \"hmmgs 2>&1 | grep 'HMMgs'\", 'classifier'])])), ('about', CommentedMap([('home', 'http://rdp.cme.msu.edu/misc/resources.jsp'), ('license', 'GPL'), ('summary', 'Metaproject for RDP Tools')]))])), 'reago': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/reago/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'reago'), ('version', 1.1)])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/chengyuan/reago-1.1/archive/release-2015.12.18.tar.gz'), ('md5', 'ac75a8914c209052bc2e2688c19f63c8'), ('patches', ['reago.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'networkx', 'infernal ==1.1.1', 'genometools-genometools'])])), ('test', CommentedMap([('source_files', ['sample_1.fasta', 'sample_2.fasta']), ('commands', ['reago.py', 'filter_input.py sample_1.fasta sample_2.fasta filter_out $PREFIX/bin/cm ba 10', 'reago.py filter_out/filtered.fasta sample_out -l 101'])])), ('about', CommentedMap([('home', 'https://github.com/chengyuan/reago-1.1'), ('license', None), ('summary', 'An assembly tool for 16S ribosomal RNA recovery from metagenomic data')]))])), 'recon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/recon/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://eddylab.org/software/recon/'), ('license', 'GNU General Public License'), ('summary', 'The RECON package performs de novo identification and classification of repeat sequence families from genomic sequences.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'recon'), ('version', '1.08')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RECON-1.08.tar.gz'), ('url', 'http://www.repeatmasker.org/RECON-1.08.tar.gz')])), ('test', CommentedMap([('commands', ['imagespread 2>&1 | grep usage'])]))])), 'recycler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/recycler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'recycler'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'recycler_<{ version }}.tar.gz'), ('url', 'https://github.com/Shamir-Lab/Recycler/archive/db982e876b99cd8b49e9f6266ded84071a5931b9.tar.gz'), ('md5', '4caffdf6f80b746f7160068b2fe5ae75')])), ('requirements', CommentedMap([('build', ['python', 'networkx', 'pysam', 'nose', 'numpy']), ('run', ['python', 'networkx', 'pysam', 'nose', 'numpy'])])), ('test', CommentedMap([('commands', ['recycle.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/Shamir-Lab/Recycler'), ('license', 'BSD-3-Clause'), ('license_file', 'LICENSE'), ('summary', 'Recycler is a tool designed for extracting circular sequences from de novo assembly graphs')]))])), 'regex': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/regex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'regex'), ('version', '2016.06.24')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'regex-2016.06.24.tar.gz'), ('url', 'https://pypi.python.org/packages/96/4a/9943cf2bc0c60e6c502c1df407393696f97ac0209794ee827cf263cbcb60/regex-2016.06.24.tar.gz'), ('md5', 'e75fcd602b73483543444e443faa4119')])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['regex'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/mrabarnett/mrab-regex'), ('license', 'Python software foundation license'), ('summary', 'Alternative regular expression module, to replace re.')]))])), 'relocate2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/relocate2/2.0.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'relocate2'), ('version', '2.0.1')])), ('source', CommentedMap([('fn', 'v2.0.1.tar.gz'), ('url', 'https://github.com/stajichlab/RelocaTE2/archive/v2.0.1.tar.gz'), ('md5', '395bd7833163bd72fe7a6ec3d85bf9d5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python ==2.7.5', 'perl >=5.10.0', 'samtools ==1.3-0', 'bowtie2 ==2.2.8', 'bedtools ==2.26.0-0', 'bwa ==0.6.2-0', 'seqtk ==1.2-0', 'pysam ==0.9.0', 'blat ==35']), ('run', ['python ==2.7.5', 'perl >=5.10.0', 'samtools ==1.3-0', 'bowtie2 ==2.2.8', 'bedtools ==2.26.0-0', 'bwa ==0.6.2-0', 'seqtk ==1.2-0', 'pysam ==0.9.0', 'blat ==35'])])), ('test', CommentedMap([('import', ['pysam'])]))])), 'remurna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/remurna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'remurna'), ('version', 1.0)])), ('source', CommentedMap([('fn', 'remuRNA.zip'), ('url', 'https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/software/remurna/remuRNA.zip'), ('sha256', '0a46cab65186d93415aa781463e3ae4b230cf6236b961d50472acf4cd0456105')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['printf \">gi|56682960|ref|NM_000146.3| Homo sapiens ferritin, light polypeptide (FTL), mRNA\\\\nGCAGTTCGGCGGTCCCGCGGGTCTGTCTCTTGCTTCAACAGTGTTTGGACGGAACAGATCCGGGGACTCTCTTCCAGCCTCCGACCGCCCTCCGATTTCCTCTCCGCTTGCAACCTCCGGGACCATCTTCTCGGCCATCTCCTGCTTCTGGGACCTGCCAGCACCGTTTTTGTGGTTAGCTCCTTCTTGCCAACCAACCA\\\\n*A3G\" > input.fa && $(which remuRNA) input.fa > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#remurna'), ('summary', 'Measurement of Single-Nucleotide Polymorphism-induced Changes of RNA Conformation')]))])), 'rename': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rename/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rename'), ('version', '1.600')])), ('source', CommentedMap([('fn', 'rename-1.600.tar.gz'), ('url', 'https://github.com/ap/rename/archive/v1.600.tar.gz'), ('sha256', '538fa908c9c2c4e7a08899edb6ddb47f7cbeb9b1a1d04e003d3c19b56fcc7f88')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl'])])), ('test', CommentedMap([('commands', ['touch foo.txt && rename -s .txt .rtf *.txt && test -e foo.rtf'])])), ('about', CommentedMap([('home', 'http://plasmasturm.org/code/rename'), ('license', 'GNU GPLv3'), ('summary', 'Perl-powered file rename script with many helpful built-ins')]))])), 'repeatmasker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmasker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'repeatmasker'), ('version', '4.0.6')])), ('source', CommentedMap([('fn', 'RepeatMasker-open-4-0-6.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatMasker-open-4-0-6.tar.gz'), ('md5', '324bcb707b23b06adfde34704a8ec0c4'), ('patches', ['patch1'])])), ('build', CommentedMap([('number', 9), ('detect_binary_files_with_prefix', False), ('has_prefix_files', ['bin/RepeatMasker']), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'rmblast']), ('run', ['rmblast', 'hmmer', 'trf', 'perl', 'perl-text-soundex'])])), ('test', CommentedMap([('commands', [\"RepeatMasker | grep 'RepeatMasker - Mask repetitive DNA' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://www.repeatmasker.org'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.')]))])), 'repeatmodeler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmodeler/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.repeatmasker.org/RepeatModeler.html'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatModeler is a de-novo repeat family identification and modeling package.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'repeatmodeler'), ('version', '1.0.8')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl', 'repeatmasker', 'rmblast', 'trf', 'repeatscout', 'recon'])])), ('source', CommentedMap([('fn', 'RepeatModeler-open-1-0-8.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatModeler-open-1-0-8.tar.gz'), ('patches', ['abs_path.patch'])])), ('test', CommentedMap([('commands', ['RepeatModeler > /dev/null'])]))])), 'repeatscout': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatscout/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://repeatscout.bioprojects.org'), ('license', 'GNU General Public License'), ('summary', 'De novo identification of repeat families in large genomes.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'repeatscout'), ('version', '1.0.5')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RepeatScout-1.0.5.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatScout-1.0.5.tar.gz')])), ('test', CommentedMap([('commands', ['RepeatScout 2>&1 | grep Version'])]))])), 'represent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/represent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'represent'), ('version', '1.5.1')])), ('source', CommentedMap([('fn', 'Represent-1.5.1.tar.gz'), ('url', 'https://pypi.python.org/packages/e2/45/010aedabc9b78625573865ccc1dec6d55b44dac1d25d85bb97952fb0caa5/Represent-1.5.1.tar.gz'), ('md5', '96a8517b8ccb82dcb23b8e8210e4c330')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six >=1.8.0']), ('run', ['python', 'six >=1.8.0'])])), ('test', CommentedMap([('imports', ['represent', 'represent.compat'])])), ('about', CommentedMap([('home', 'https://github.com/RazerM/represent'), ('license', 'MIT License'), ('summary', 'Create __repr__ automatically or declaratively.'), ('license_family', 'MIT')]))])), 'requests-cache': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/requests-cache/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'requests-cache'), ('version', '0.4.10')])), ('source', CommentedMap([('fn', 'requests-cache-0.4.10.tar.gz'), ('url', 'https://pypi.python.org/packages/source/r/requests-cache/requests-cache-0.4.10.tar.gz'), ('md5', '4231368e27d63591280aafb8095a54cb')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.1.0']), ('run', ['python', 'requests >=1.1.0'])])), ('test', CommentedMap([('imports', ['requests_cache', 'requests_cache.backends', 'requests_cache.backends.storage'])])), ('about', CommentedMap([('home', 'https://github.com/reclosedev/requests-cache'), ('license', 'BSD License'), ('summary', 'Persistent cache for requests library')]))])), 'requests-mock': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/requests-mock/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'requests-mock'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'requests-mock-1.0.0.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/60/d1/dd2ffc8661573be1bc0bed112d392eeba2064ab0e4c5547dc43ba4b31ca8/requests-mock-1.0.0.tar.gz'), ('md5', 'af311a9d7e6b3a10680d1e62da27eb23')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.1', 'six', 'pbr']), ('run', ['python', 'requests >=1.1', 'six', 'pbr'])])), ('about', CommentedMap([('license', 'Apache Software License'), ('summary', 'Mock out responses from the requests package')]))])), 'requests-toolbelt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/requests-toolbelt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'requests-toolbelt'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'requests-toolbelt-0.5.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/r/requests-toolbelt/requests-toolbelt-0.5.0.tar.gz'), ('md5', '317a788caa4eec4e3b8f2433c646eaa8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests']), ('run', ['python', 'requests'])])), ('test', CommentedMap([('imports', ['requests_toolbelt.multipart', 'requests_toolbelt.downloadutils'])])), ('about', CommentedMap([('home', 'https://github.com/sigmavirus24/requests-toolbelt'), ('license', 'Apache License 2.0'), ('summary', 'A toolbelt of useful classes and functions to be used with python-requests')]))])), 'retry_decorator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/retry_decorator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'retry_decorator'), ('version', '1.1.0')])), ('source', CommentedMap([('fn', 'retry_decorator-1.1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/95/7a/807ac21749ecd26ae3337c3069ed6ac8658b9fbc85f109e419a812b18ab7/retry_decorator-1.1.0.tar.gz'), ('md5', 'ee933c38020839e30699519a7611b3f2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['retry_decorator'])])), ('about', CommentedMap([('home', 'https://github.com/pnpnpn/retry-decorator'), ('license', 'MIT'), ('summary', 'Retry Decorator')]))])), 'reveal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/reveal/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/jasperlinthorst/reveal/archive/c9d67109859e8a4d25b8ff0c3e6382d1d404ca8d.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'intervaltree', 'networkx', 'gcc', 'llvm', 'libdivsufsort']), ('run', ['python', 'intervaltree', 'networkx', 'libgcc', 'libdivsufsort', 'matplotlib'])])), ('test', CommentedMap([('commands', ['reveal -h', 'dformat -h'])])), ('about', CommentedMap([('home', 'https://github.com/jasperlinthorst/reveal'), ('license', 'Unknown'), ('summary', 'Graph based multi genome aligner.')]))])), 'ribocode': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ribocode/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ribocode'), ('version', '1.2.6')])), ('source', CommentedMap([('fn', 'RiboCode-1.2.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz'), ('md5', 'e05ef843da351c4588e02bad3d5c3d65')])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'pysam >0.8.4', 'matplotlib', 'scipy', 'pyfasta', 'biopython', 'h5py', 'htseq']), ('run', ['python', 'numpy', 'pysam >0.8.4', 'matplotlib', 'scipy', 'pyfasta', 'biopython', 'h5py', 'htseq'])])), ('test', CommentedMap([('imports', ['RiboCode']), ('commands', ['RiboCode --help', 'prepare_transcripts --help'])])), ('build', CommentedMap([('skip', True), ('entry_points', ['RiboCode=RiboCode.RiboCode:main', 'prepare_transcripts=RiboCode.prepare_transcripts:main', 'metaplots=RiboCode.metaplots:main', 'plot_orf_density=RiboCode.plot_orf_density:main', 'ORFcount=RiboCode.RPF_count_ORF:main']), ('script', 'python setup.py install')])), ('about', CommentedMap([('home', 'https://github.com/xzt41/RiboCode'), ('license', 'MIT'), ('summary', 'A package for identifying the translated ORFs using ribosome-profiling data'), ('license_family', 'MIT')]))])), 'ribodiff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ribodiff/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ribodiff'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ribodiff-<{ version }}.tar.gz'), ('url', 'https://github.com/ratschlab/RiboDiff/archive/v0.2.2.tar.gz'), ('md5', 'b02833b4412959746032f0cf23a944d8')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install')])), ('requirements', CommentedMap([('build', ['python >=2.6.6,<3', 'numpy >=1.8.0', 'scipy >=0.13.3', 'matplotlib >=1.3.0', 'statsmodels >=0.5.0']), ('run', ['python >=2.6.6,<3', 'numpy >=1.8.0', 'scipy >=0.13.3', 'matplotlib >=1.3.0', 'statsmodels >=0.5.0'])])), ('test', CommentedMap([('imports', ['ribodiff']), ('commands', ['TE.py --help'])])), ('about', CommentedMap([('home', 'http://public.bmi.inf.ethz.ch/user/zhongy/RiboDiff/index.html'), ('license', 'GPL 3'), ('summary', 'RiboDiff is a statistical tool that detects the protein translational efficiency change from Ribo-Seq (ribosome footprinting) and RNA-Seq data.')]))])), 'riboplot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/riboplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'riboplot'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'riboplot-0.3.1.tar.gz [py2k]'), ('url', 'https://pypi.python.org/packages/87/d7/e67c64d7ea6312437013be23e91c9478543ca0b29a8dc69aa92676f12bd0/riboplot-0.3.1.tar.gz [py2k]'), ('md5', '6903d5fa1e5e5390f6d8dff8b62307ff [py2k]')])), ('build', CommentedMap([('number', 1), ('entry_points', ['riboplot = riboplot.riboplot:run', 'ribocount = riboplot.ribocount:run']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib <=1.4.3', 'pysam <=0.8.3', 'mock ==1.0.1', 'nose']), ('run', ['python', 'matplotlib <=1.4.3', 'pysam <=0.8.3', 'mock ==1.0.1', 'libgcc', 'bedtools <=2.24.0'])])), ('test', CommentedMap([('imports', ['riboplot']), ('commands', ['riboplot --help', 'ribocount --help'])])), ('about', CommentedMap([('home', 'https://github.com/vimalkvn/riboplot'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Plot read counts of RiboSeq data from BAM format alignment files'), ('license_family', 'LGPL')]))])), 'riboseed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/riboseed/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://test-files.pythonhosted.org/packages/21/59/ef9dbb631f132c312bdd1cc145395050fdd0bfbe46765280c9a157367dcd/riboSeed-0.4.16.tar.gz'), ('<{ hash_type }}', '<{ hash_value }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython ==1.68', 'pysam ==0.11.2.2', 'jenkspy ==0.1.4', 'pyutilsnrw ==0.1.0', 'matplotlib ==1.5.3', 'pandas ==0.20.1', 'coverage ==4.4.1', 'numpy ==1.11.3', 'nose ==1.3.7']), ('run', ['python', 'biopython ==1.68', 'jenkspy ==0.1.4', 'pysam ==0.11.2.2', 'pyutilsnrw ==0.1.0', 'matplotlib ==1.5.3', 'pandas ==0.20.1', 'coverage ==4.4.1', 'numpy ==1.11.3', 'nose ==1.3.7', 'emboss', 'barrnap ==0.7', 'mafft', 'prank', 'spades ==3.9.0', 'bwa ==0.7.8', 'samtools ==1.4.1'])])), ('test', CommentedMap([('imports', ['riboSeed'])])), ('about', CommentedMap([('home', 'https://github.com/nickp60/riboSeed'), ('license', 'MIT License'), ('license_family', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'riboSeed: assemble across rDNA regions'), ('description', 'Genome assembly polisher to bridge rDNA gaps'), ('dev_url', 'https://github.com/nickp60/riboSeed')])), ('extra', CommentedMap([('recipe-maintainers', ['nickp60'])]))])), 'ribotaper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ribotaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ribotaper'), ('version', '1.3.1a')])), ('source', CommentedMap([('fn', 'RiboTaper-1.3.1a.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/RiboTaper/RiboTaper-1.3.1a.tar.gz'), ('md5', '5f753a25646e7f5bb3e3f1865129e1f5')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial']), ('run', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial'])])), ('test', CommentedMap([('commands', ['Ribotaper.sh 2>&1  | grep Usage > /dev/null', 'create_annotations_files.bash 2>&1  | grep Usage > /dev/null', 'create_metaplots.bash 2>&1  | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/'), ('license', 'GPL'), ('summary', 'RiboTaper is a new analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.')]))])), 'rmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmap/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/rmap/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'rmap is a short reads mapper for next-generation sequencing data')])), ('package', CommentedMap([('name', 'rmap'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'rmap-2.1.tar.bz2'), ('url', 'http://smithlabresearch.org/downloads/rmap-2.1.tar.bz2'), ('sha256', '0196139025e1dfaa203b28176607fadcad50b4b0926eb53ef9fc7e95ec5dece7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rmap 2>&1 | grep \"^Usage\"'])]))])), 'rmats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rmats'), ('version', '3.2.5')])), ('source', CommentedMap([('fn', 'rMATS.3.2.5.tgz'), ('sha256', '13e4fb9d97bad1fa0b53b9e8e358e42cfc9f7619c52e1999afb3b8249de67a28'), ('url', 'https://sourceforge.net/projects/rnaseq-mats/files/MATS/rMATS.3.2.5.tgz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'pysam ==0.9.0']), ('run', ['python', 'numpy', 'scipy', 'pysam ==0.9.0', 'samtools ==1.3.1', 'star'])])), ('test', CommentedMap([('commands', ['RNASeq-MATS.py'])])), ('about', CommentedMap([('home', 'http://rnaseq-mats.sourceforge.net'), ('license', 'MIT'), ('summary', 'MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data.')]))])), 'rmats2sashimiplot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmats2sashimiplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rmats2sashimiplot'), ('version', '2.0.0')])), ('source', CommentedMap([('url', 'https://github.com/Xinglab/rmats2sashimiplot/archive/v0.0.1.tar.gz'), ('md5', 'fe92f2dcaceacc18ed864e269f866766')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python ==2.6*|2.7*', 'matplotlib', 'scipy', 'pysam ==0.9.0', 'samtools ==1.3.1'])])), ('build', CommentedMap([('number', 0), ('skip', True), ('preserve_egg_dir', True)])), ('test', CommentedMap([('commands', ['rmats2sashimiplot -h'])])), ('about', CommentedMap([('home', 'https://github.com/Xinglab/rmats2sashimiplot'), ('license', 'GNU GPLv2'), ('license_file', 'LICENSE'), ('summary', 'rmats2sashimiplot is a tool that generates sahimi plots from rMATS outputs.')]))])), 'rmblast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmblast/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rmblast'), ('version', '2.2.28')])), ('source', CommentedMap([('fn', 'ncbi-rmblastn-2.2.28-universal-macosx.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/blast/executables/rmblast/2.2.28/ncbi-rmblastn-2.2.28-universal-macosx.tar.gz'), ('md5', '421c5ddf2c425800d934562c965e8706')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['boost <=1.58', 'gcc', 'bzip2 1.0.6 1', 'zlib']), ('run', ['boost <=1.58', 'libgcc', 'bzip2 1.0.6 1', 'zlib', 'blast'])])), ('test', CommentedMap([('commands', ['rmblastn -help > /dev/null', 'makeblastdb -help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.repeatmasker.org/RMBlast.html'), ('license', 'OSL-2.1'), ('summary', 'RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite.')]))])), 'rna-seqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rna-seqc/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://archive.broadinstitute.org/cancer/cga/rna-seqc'), ('license', 'None'), ('summary', 'RNA-SeQC is a java program which computes a series of quality control metrics for RNA-seq data. The input can be one or more BAM files')])), ('package', CommentedMap([('name', 'rna-seqc'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['{ rna-seqc || true; } | grep RNA-SeQC', '{ rna-seqc || true; } | grep <{ version }}'])])), ('source', CommentedMap([('fn', 'RNA-SeQC_v1.1.8.jar'), ('url', 'http://www.broadinstitute.org/cancer/cga/tools/rnaseqc/RNA-SeQC_v1.1.8.jar'), ('md5', '71d7b5d3b3dcc1893cdc7f6819185d41')]))])), 'rnabob': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnabob/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnabob'), ('version', '2.2.1')])), ('source', CommentedMap([('url', 'http://eddylab.org/software/rnabob/rnabob-2.2.1.tar.gz'), ('md5', '23588aa649169d25d71c124b1013a9ac'), ('patches', ['man_location.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rnabob -h'])])), ('about', CommentedMap([('license_file', 'COPYING'), ('summary', 'fast RNA motif searching')]))])), 'rnalien': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnalien/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/eggzilla/RNAlien/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['stack >=1.3.2', 'gmp', 'zlib', 'perl']), ('run', ['viennarna ==2.3.3', 'rnaz ==2.1', 'locarna ==1.9.1', 'infernal ==1.1.2', 'rnacode ==0.3', 'gmp', 'perl', 'libgcc', 'zlib', 'openssl'])])), ('test', CommentedMap([('commands', ['RNAlien --help'])])), ('about', CommentedMap([('home', 'http://rna.tbi.univie.ac.at/rnalien/tool'), ('license', 'GPL-3'), ('summary', 'A tool for unsupervised construction of RNA family models')]))])), 'rnasnp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnasnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnasnp'), ('version', 1.2)])), ('source', CommentedMap([('fn', 'RNAsnp-1.2.tar.gz'), ('url', 'http://rth.dk/resources/rnasnp/software_src/RNAsnp-1.2.tar.gz'), ('sha256', '4fb63b7ff149eca463e84e2f9e3c430e34d5e381dbda827b89e452408cead094')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['set +e && printf \"A3G\" > snp.txt && printf \">gi|56682960|ref|NM_000146.3| Homo sapiens ferritin, light polypeptide (FTL), mRNA\\\\nGCAGTTCGGCGGTCCCGCGGGTCTGTCTCTTGCTTCAACAGTGTTTGGACGGAACAGATCCGGGGACTCTCTTCCAGCCTCCGACCGCCCTCCGATTTCCTCTCCGCTTGCAACCTCCGGGACCATCTTCTCGGCCATCTCCTGCTTCTGGGACCTGCCAGCACCGTTTTTGTGGTTAGCTCCTTCTTGCCAACCAACCA\" > seq.fa && RNAsnp -f seq.fa -s snp.txt; exit_code=$? && if [ $exit_code != 160 ]; then exit $exit_code ;fi'])])), ('about', CommentedMap([('home', 'http://rth.dk/resources/rnasnp/software'), ('license', 'RTH'), ('license_file', 'COPYING'), ('summary', 'Efficient detection of local RNA secondary structure changes induced by SNPs.')]))])), 'rnaz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnaz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnaz'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'RNAz-2.1.tar.gz'), ('url', 'https://www.tbi.univie.ac.at/~wash/RNAz/RNAz-2.1.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['RNAz -h'])])), ('about', CommentedMap([('home', 'https://www.tbi.univie.ac.at/~wash/RNAz/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'predicting structural noncoding RNAs')]))])), 'rnftools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnftools/0.3.1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnftools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'rnftools-<{ version }}.tar.gz'), ('url', 'https://github.com/karel-brinda/rnftools/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'pip install .')])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'snakemake', 'argparse']), ('run', ['python', 'samtools', 'pysam', 'zlib', 'wgsim', 'dwgsim', 'art', 'mason', 'curesim', 'snakemake', 'argparse', 'pyfaidx', 'bwa', 'gnuplot >=5.0', 'beautifulsoup4', 'reportlab ==3.3.0'])])), ('test', CommentedMap([('commands', ['rnftools -h']), ('imports', ['rnftools'])])), ('about', CommentedMap([('home', 'http://karel-brinda.github.io/rnftools'), ('license', 'MIT'), ('summary', 'RNF framework for NGS: simulation of reads, evaluation of mappers, conversion of RNF-compliant data.')]))])), 'roary': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/roary/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://search.cpan.org/CPAN/authors/id/A/AJ/AJPAGE/Bio-Roary-<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build', 'gcc', 'llvm', 'perl-bioperl <1.7', 'perl-bioperl-core <1.7', 'perl-array-utils', 'perl-text-csv', 'perl-moose', 'perl-exception-class', 'perl-file-grep', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-file-slurper', 'perl-graph-readwrite', 'perl-env-path', 'perl-perlio-utf8_strict', 'mcl', 'bedtools', 'prank', 'mafft', 'cd-hit', 'blast', 'parallel', 'fasttree', 'perl-test-output', 'perl-test-most', 'perl-test-files']), ('run', ['perl', 'libgcc', 'perl-bioperl <1.7', 'perl-bioperl-core <1.7', 'perl-array-utils', 'perl-text-csv', 'perl-moose', 'perl-exception-class', 'perl-file-grep', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-file-slurper', 'perl-graph-readwrite', 'perl-env-path', 'perl-perlio-utf8_strict', 'mcl', 'bedtools', 'prank', 'mafft', 'cd-hit', 'blast', 'parallel', 'fasttree', 'perl-test-output', 'perl-test-most', 'perl-test-files'])])), ('test', CommentedMap([('commands', ['roary -h 2>&1 | grep Usage', 'query_pan_genome -h 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/Roary'), ('license', 'GPL-3.0'), ('license_file', 'GPL-LICENSE'), ('summary', 'Rapid large-scale prokaryote pan genome analysis')]))])), 'roary2fripan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/roary2fripan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/kwongj/roary2fripan/archive/298c32a56f8696898fd3a94ef252e57ebe464119.tar.gz'), ('md5', '1d99d4a5af37a3ea617cdff7e05f039f')])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['roary2fripan.py --version'])])), ('about', CommentedMap([('home', 'https://github.com/kwongj/roary2fripan'), ('license', 'GPLv2'), ('summary', 'Formats Roary output for viewing in FriPan')]))])), 'ropebwt2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ropebwt2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ropebwt2'), ('version', 'r187')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'ropebwt2_r187.tar.gz'), ('url', 'https://github.com/lh3/ropebwt2/archive/e23a7df263571c02aa0c0434e623108482097e3d.tar.gz'), ('md5', 'fc23f90727e28eef748304a12583a585')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'llvm', 'zlib'])])), ('test', CommentedMap([('commands', ['ropebwt2  2>&1 | grep ropebwt2 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/ropebwt2'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'Incremental construction of FM-index for DNA sequences')]))])), 'roprofile': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/roprofile/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/cimendes/roProfile/archive/2a8c52fafc5ae47fe0e7f2b29999579352fdfed1.tar.gz'), ('md5', '52a4b174d5d862f0f4b0f203bfe3dc16')])), ('requirements', CommentedMap([('build', None), ('run', ['python', 'biopython >=1.66', 'matplotlib >=1.5.2', 'mpld3 >=0.2', 'pandas >=0.15.0'])])), ('test', CommentedMap([('commands', ['roProfile.py --version'])])), ('about', CommentedMap([('home', 'https://github.com/cimendes/roProfile'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'Generation of pan-genome profile files using Roary output.')]))])), 'rpkmforgenes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rpkmforgenes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rpkmforgenes'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/danielramskold/S3_species-specific_sequencing/archive/1.0.1.tar.gz'), ('md5', '114afd083b2f11bd57c57e68ec4b607d')])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'numpy']), ('run', ['python', 'pysam', 'numpy'])])), ('test', CommentedMap([('commands', ['rpkmforgenes.py 2>&1 | grep \"Non-optional arguments:\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/danielramskold/S3_species-specific_sequencing'), ('license', 'Creative Commons Attribution License'), ('summary', 'Calculates gene expression from a read mapping file')]))])), 'rsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rsa'), ('version', '3.1.4')])), ('source', CommentedMap([('fn', 'rsa-3.1.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/r/rsa/rsa-3.1.4.tar.gz'), ('md5', 'b6b1c80e1931d4eba8538fd5d4de1355')])), ('build', CommentedMap([('skip', False), ('entry_points', ['pyrsa-priv2pub = rsa.util:private_to_public', 'pyrsa-keygen = rsa.cli:keygen', 'pyrsa-encrypt = rsa.cli:encrypt', 'pyrsa-decrypt = rsa.cli:decrypt', 'pyrsa-sign = rsa.cli:sign', 'pyrsa-verify = rsa.cli:verify', 'pyrsa-encrypt-bigfile = rsa.cli:encrypt_bigfile', 'pyrsa-decrypt-bigfile = rsa.cli:decrypt_bigfile'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyasn1 >=0.1.3']), ('run', ['python', 'pyasn1 >=0.1.3'])])), ('test', CommentedMap([('imports', ['rsa']), ('commands', ['pyrsa-priv2pub --help', 'pyrsa-keygen --help', 'pyrsa-encrypt --help', 'pyrsa-decrypt --help', 'pyrsa-sign --help', 'pyrsa-verify --help', 'pyrsa-encrypt-bigfile --help', 'pyrsa-decrypt-bigfile --help'])])), ('about', CommentedMap([('home', 'http://stuvel.eu/rsa'), ('license', 'Apache Software License'), ('summary', 'Pure-Python RSA implementation')]))])), 'rscape': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rscape/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rscape'), ('version', '0.6.1')])), ('source', CommentedMap([('fn', 'rscape_v0.6.1.tar.gz'), ('url', 'http://eddylab.org/software/rscape/rscape_v0.6.1.tar.gz'), ('md5', 'c4fd21e433c571e483424dd071cc6226')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gnuplot']), ('run', ['libgcc', 'gnuplot'])])), ('test', CommentedMap([('commands', ['printf \"# STOCKHOLM 1.0\\\\n#=GF SQ 3\\\\nmyseq1 ACGCUUUUGCU\\\\nmyseq2 ACUCUUUUGGU\\\\nmyseq3 UCGCUUUUGCA\\\\n//\\\\n\" > tmp.sto; R-scape --cyk --nofigures tmp.sto'])])), ('about', CommentedMap([('home', 'http://eddylab.org/R-scape/'), ('license', 'GPLv3'), ('license_file', 'documentation/R-scape_userguide.pdf'), ('summary', 'R-scape (RNA Structural Covariation Above Phylogenetic Expectation) looks for evidence of a conserved RNA secondary structure by measuring pairwise covariations observed in an input multiple sequence alignment.')]))])), 'rseg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rseg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rseg'), ('version', '0.4.9')])), ('source', CommentedMap([('fn', 'rseg-0.4.9.tar.gz'), ('url', 'https://github.com/smithlabcode/rseg/archive/rseg-0.4.9.tar.gz'), ('md5', '890d9d3b364270ab62da012cd87a79ee'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['rseg -about > /dev/null 2>&1', 'rseg-diff -about > /dev/null 2>&1', 'deadzones -about > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'http://smithlabresearch.org/software/rseg'), ('license', 'GPLv3'), ('summary', 'The RSEG software package is used to analyze ChIP-Seq data, especially for identifying genomic regions and their boundaries marked by diffusive histone modification markers, such as H3K36me3 and H3K27me3.')]))])), 'rsem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rsem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://deweylab.github.io/RSEM/'), ('license', 'GPLv3'), ('summary', 'RSEM (RNA-Seq by Expectation-Maximization)')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/']), ('number', 1), ('skip', False), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'boost <{CONDA_BOOST}}*', 'gcc', 'perl', 'perl-module-build', 'r-base', 'zlib']), ('run', ['bioconductor-biobase', 'bioconductor-ebseq', 'boost <{CONDA_BOOST}}*', 'libgcc', 'perl', 'perl-module-build', 'r-base', 'samtools 1.3', 'zlib'])])), ('test', CommentedMap([('commands', ['rsem-prepare-reference 2>&1 | grep reference_name > /dev/null', 'rsem-for-ebseq-find-DE 2>&1 | grep Usage > /dev/null', \"rsem-bam2wig foo bar foobar 2>&1 | grep 'fail to open file' > /dev/null\", 'which rsem-for-ebseq-calculate-clustering-info'])])), ('source', CommentedMap([('url', 'https://github.com/deweylab/RSEM/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')]))])), 'rseqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rseqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rseqc'), ('version', '2.6.4')])), ('source', CommentedMap([('fn', 'RSeQC-2.6.4.tar.gz'), ('url', 'https://pypi.python.org/packages/61/58/1d9c280d8d852f7c5e873c0635952fc19305b0805e9f0ace50e8ef2f93c3/RSeQC-2.6.4.tar.gz'), ('md5', '935779c452ffc84f3b8b9fb3d485c782')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'nose']), ('run', ['python', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'r-base'])])), ('test', CommentedMap([('imports', ['qcmodule', 'qcmodule.SAM']), ('commands', ['read_distribution.py 2>&1 | grep Usage > /dev/null', 'geneBody_coverage.py 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://rseqc.sourceforge.net/'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'RNA-seq QC Package'), ('license_family', 'GPL2')]))])), 'rtg-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rtg-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rtg-tools'), ('version', '3.8.4')])), ('source', CommentedMap([('fn', 'rtg-tools-3.8.4-nojre.zip'), ('url', 'https://github.com/RealTimeGenomics/rtg-tools/releases/download/3.8.4/rtg-tools-3.8.4-nojre.zip'), ('md5', 'b01349ddff9306da4e0a5b4a15156093')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['rtg vcfeval --help 2>&1 | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'https://github.com/RealTimeGenomics/rtg-tools'), ('license', 'BSD'), ('summary', 'RealTimeGenomics Tools -- Utilities for accurate VCF comparison and manipulation')]))])), 'ruby': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ruby/2.1.7/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ruby'), ('version', '2.1.7')])), ('source', CommentedMap([('fn', 'ruby-2.1.7.tar.gz'), ('url', 'https://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.7.tar.gz'), ('sha256', 'f59c1596ac39cc7e60126e7d3698c19f482f04060674fdfe0124e1752ba6dd81')])), ('test', CommentedMap([('commands', ['ruby --version', 'ruby -e \\'puts \"Hi, from Ruby!\"\\'', 'ruby -e \\'require \"rubygems\"\\'', 'gem --version'])])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'libxml2 ==2.9.2', 'readline ==6.2', 'yaml', 'openssl 1.0.1g', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://www.ruby-lang.org/'), ('license', 'BSD')]))])), 'ruffus': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ruffus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ruffus'), ('version', '2.6.3')])), ('source', CommentedMap([('fn', 'ruffus-2.6.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/r/ruffus/ruffus-2.6.3.tar.gz'), ('md5', '37a8a3f6962d7eabf9513e1a5a8c6890')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['ruffus'])])), ('about', CommentedMap([('home', 'http://www.ruffus.org.uk/'), ('license', 'MIT'), ('summary', 'Light-weight Python Computational Pipeline Management')]))])), 'rust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rust/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rust'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'rust-<{ version }}-x86_64-apple-darwin.tar.gz'), ('url', 'http://static.rust-lang.org/dist/rust-<{ version }}-x86_64-apple-darwin.tar.gz')])), ('test', CommentedMap([('commands', ['rustc --help', 'cargo --help'])])), ('about', CommentedMap([('home', 'https://www.rust-lang.org'), ('license', 'MIT'), ('summary', 'Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.')]))])), 'rust-bio-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rust-bio-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rust-bio-tools'), ('version', '<{version}}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'rust-bio-tools-<{version}}.tar.gz'), ('url', 'https://github.com/rust-bio/rust-bio-tools/archive/v<{version}}.tar.gz'), ('md5', 'a85bceaa2aaed28b05d6faef78b8ca2b')])), ('requirements', CommentedMap([('build', ['rust', 'gcc', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['rbt --help &> /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/rust-bio/rust-bio-tools'), ('license', 'MIT'), ('summary', 'A growing collection of fast and secure command line utililities for dealing with NGS data\\nimplemented on top of Rust-Bio.\\n')]))])), 'rust-overlaps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rust-overlaps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rust-overlaps'), ('version', '<{version}}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'rust-overlaps-<{version}}.tar.gz'), ('url', 'https://github.com/jbaaijens/rust-overlaps/archive/v<{version}}.tar.gz'), ('md5', '5bc2935652b0cc9f4b64f4cb0f5baa79')])), ('requirements', CommentedMap([('build', ['rust >=1.18.0', 'gcc 4.8.2', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['rust-overlaps --help &> /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/jbaaijens/rust-overlaps'), ('license', 'MIT'), ('summary', 'A fast and secure command line utilility for enumerating all suffix-prefix\\noverlaps within a set of sequences, satisfying a user-specified minimal\\noverlap length and maximal error rate.\\n')]))])), 's3gof3r': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/s3gof3r/meta.yaml', CommentedMap([('package', CommentedMap([('name', 's3gof3r'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'gof3r_0.5.0_darwin_amd64.zip'), ('url', 'https://github.com/rlmcpherson/s3gof3r/releases/download/v0.5.0/gof3r_0.5.0_darwin_amd64.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', [\"gof3r cp -h 2>&1 | grep 'gof3r'\"])])), ('about', CommentedMap([('home', 'https://github.com/rlmcpherson/s3gof3r'), ('license', 'MIT'), ('summary', 'Fast, concurrent, streaming access to Amazon S3, including gof3r, a CLI')]))])), 'saffrontree': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/saffrontree/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/sanger-pathogens/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'dendropy >=4.1.0', 'pyfastaq >=3.12.0', 'biopython', 'gcc', 'llvm']), ('run', ['python', 'biopython', 'dendropy >=4.1.0', 'pyfastaq >=3.12.0', 'kmc >=2.3.0', 'libgcc'])])), ('test', CommentedMap([('commands', ['saffrontree -h'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/saffrontree'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'SaffronTree: Reference free rapid phylogenetic tree construction from raw read data')])), ('extra', CommentedMap([('recipe-maintainers', ['andrewjpage']), ('doi', '10.21105/joss.00243')]))])), 'sailfish': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sailfish/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1), ('skip', True)])), ('about', CommentedMap([('home', 'http://www.cs.cmu.edu/~ckingsf/software/sailfish/'), ('license', 'GPLv3'), ('summary', 'Rapid Mapping-based Isoform Quantification from RNA-Seq Reads')])), ('package', CommentedMap([('name', 'sailfish'), ('version', '0.10.1')])), ('source', CommentedMap([('fn', 'sailfish-0.10.1.tar.gz'), ('url', 'https://github.com/kingsfordgroup/sailfish/archive/v0.10.1.tar.gz'), ('md5', 'e6dab4cf3a39f346df7c28f40eb58cad')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost ==1.63.0', 'cmake', 'tbb', 'autoconf', 'unzip', 'perl', 'icu 58.*']), ('run', ['libgcc', 'boost ==1.63.0', 'tbb', 'icu 58.*'])])), ('test', CommentedMap([('commands', ['sailfish --help'])]))])), 'sak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sak/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'sak'), ('version', '0.4.6')])), ('source', CommentedMap([('fn', 'sak-0.4.6-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/sak/sak-0.4.6-Linux-x86_64.tar.xz'), ('md5', 'cfd68084994e666116971dd681b7fe20')])), ('test', CommentedMap([('commands', ['sak --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/sak/README'), ('license', 'https://github.com/seqan/seqan/tree/master/apps/sak/LICENSE'), ('summary', 'This tool allows one to cut sequences and parts of sequences out of sequence files.')]))])), 'salmon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/salmon/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('about', CommentedMap([('home', 'https://github.com/COMBINE-lab/salmon'), ('license', 'GPLv3'), ('summary', 'Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments')])), ('package', CommentedMap([('name', 'salmon'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Salmon-<{ version }}_src.tar.gz'), ('url', 'https://github.com/COMBINE-lab/salmon/archive/v<{ version }}.tar.gz'), ('md5', 'ee512697bc44b13661a16d4e14cf0a00')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc ==4.8.5', 'autoconf 2.69 pl5.*', 'boost ==1.63.0', 'icu 58.*', 'cmake', 'zlib', 'bzip2', 'unzip', 'jemalloc ==4.5.0', 'tbb']), ('run', ['libgcc', 'boost ==1.63.0', 'icu 58.*', 'jemalloc ==4.5.0', 'tbb', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['salmon --help'])]))])), 'sam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sam'), ('version', 3.5)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'sam3.5.i386-darwin9.6.0.tar.gz'), ('url', 'https://compbio.soe.ucsc.edu/sam2src/osx/sam3.5.i386-darwin9.6.0.tar.gz'), ('md5', '214ea1929b3a1c778d9c12f55c27a897')])), ('about', CommentedMap([('home', 'https://compbio.soe.ucsc.edu/sam2src/'), ('license', 'academic'), ('license_file', 'COPYRIGHT'), ('summary', 'SAM - Sequence Alignment and Modeling System')])), ('test', CommentedMap([('commands', ['buildmodel -v', 'get_fisher_scores -v'])]))])), 'sambamba': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sambamba/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sambamba'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'sambamba_v<{ version }}.tar.bz2'), ('url', 'https://github.com/lomereiter/sambamba/releases/download/v<{ version }}/sambamba_v<{ version }}_osx.tar.bz2'), ('md5', '07d82a2102126565e6796658face7c71')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['samtools', 'bcftools'])])), ('test', CommentedMap([('commands', ['sambamba view'])])), ('about', CommentedMap([('home', 'https://github.com/lomereiter/sambamba'), ('license', 'GPLv2'), ('summary', 'Tools for working with SAM/BAM data')]))])), 'samblaster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samblaster/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'samblaster'), ('version', '0.1.24')])), ('source', CommentedMap([('fn', 'samblaster-v.0.1.24.tar.gz'), ('url', 'https://github.com/GregoryFaust/samblaster/releases/download/v.0.1.24/samblaster-v.0.1.24.tar.gz'), ('md5', '885d5782cc277865dfb086fc0a20243e')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['samblaster -h'])])), ('about', CommentedMap([('home', 'https://github.com/GregoryFaust/samblaster'), ('license', 'MIT'), ('summary', 'A tool to mark duplicates and extract discordant and split reads from sam files.')]))])), 'samsift': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samsift/0.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'samsift'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'samsift-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/samsift/samsift-<{version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'pysam']), ('run', ['python', 'pysam'])])), ('test', CommentedMap([('commands', ['samsift -v', 'samsift -h', 'BAM=https://github.com/karel-brinda/samsift/raw/master/tests/test.bam; curl -L ${BAM} | samsift'])])), ('about', CommentedMap([('home', 'https://github.com/karel-brinda/samsift'), ('license', 'MIT'), ('summary', 'Advanced filtering and tagging of SAM/BAM alignments using Python expressions.')]))])), 'samsifter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samsifter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'samsifter'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'SamSifter-1.0.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/SamSifter/SamSifter-1.0.0.tar.gz'), ('md5', '883df68d2f989c4c431ebb253ca104f7')])), ('build', CommentedMap([('skip', True), ('number', 2), ('entry_points', ['compile_stats=samsifter.stats.compile_stats:main', 'count_taxon_reads=samsifter.tools.count_taxon_reads:main', 'filter_read_conservation=samsifter.tools.filter_read_conservation:main', 'filter_read_list=samsifter.tools.filter_read_list:main', 'filter_ref_coverage=samsifter.tools.filter_ref_coverage:main', 'filter_ref_identity=samsifter.tools.filter_ref_identity:main', 'filter_ref_list=samsifter.tools.filter_ref_list:main', 'filter_ref_pmds=samsifter.tools.filter_ref_pmds:main', 'filter_taxon_list=samsifter.tools.filter_taxon_list:main', 'filter_taxon_pmds=samsifter.tools.filter_taxon_pmds:main', 'pmdtools_mod=pmdtools.pmdtools_mod:main', 'summarize_stats=samsifter.stats.summarize_stats:main', 'samsifter=samsifter.samsifter:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'matplotlib >=1.3.1', 'pandas >=0.14.1', 'numpy >=1.6.1', 'pyqt 4.11.*', 'glib', 'xorg-libsm', 'xorg-libxrender']), ('run', ['python', 'matplotlib >=1.3.1', 'pandas >=0.14.1', 'numpy >=1.6.1', 'pyqt 4.11.*', 'libgcc', 'glib', 'xorg-libsm', 'xorg-libxrender'])])), ('test', CommentedMap([('imports', ['pmdtools', 'samsifter', 'samsifter.gui', 'samsifter.models', 'samsifter.resources', 'samsifter.stats', 'samsifter.tools', 'samsifter.util', 'samsifter.views']), ('commands', ['compile_stats --help', 'count_taxon_reads --help', 'filter_read_conservation --help', 'filter_read_list --help', 'filter_ref_coverage --help', 'filter_ref_identity --help', 'filter_ref_list --help', 'filter_ref_pmds --help', 'filter_taxon_list --help', 'filter_taxon_pmds --help', 'pmdtools_mod --help', 'summarize_stats --help', 'samsifter --help'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/SamSifter/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Workflow editor for metagenomic analysis')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'samtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samtools/1.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM, BAM and CRAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '1.1')])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'zlib ==1.2.8']), ('run', ['ncurses ==5.9', 'zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'samtools-1.1.tar.bz2'), ('sha256', 'c24d26c303153d72b5bf3cc11f72c6c375a4ca1140cc485648c8c5473414b7f8'), ('url', 'http://downloads.sourceforge.net/project/samtools/samtools/1.1/samtools-1.1.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))])), 'savage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/savage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'savage'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('source', CommentedMap([('fn', 'savage-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/jbaaijens/savage/get/v<{ version }}.tar.gz'), ('md5', 'eb1622cdd8a22c4d6c7fd34dc064b3cd')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'boost <{CONDA_BOOST}}*']), ('run', ['boost <{CONDA_BOOST}}*', 'python', 'bwa', 'libgcc', 'kallisto >=0.43.0', 'rust-overlaps'])])), ('test', CommentedMap([('commands', ['savage --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jbaaijens/savage'), ('license', 'GPL v3'), ('summary', 'SAVAGE (Strain Aware VirAl GEnome assembly) reconstructs individual (viral) haplotypes from a mixed sample.')]))])), 'sbt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sbt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sbt'), ('version', '0.13.12')])), ('about', CommentedMap([('home', 'http://www.scala-sbt.org/'), ('license', 'BSD')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['java-jdk  >=6']), ('run', ['java-jdk  >=6'])])), ('source', CommentedMap([('fn', 'sbt-launch.tgz'), ('url', 'https://dl.bintray.com/sbt/native-packages/sbt/0.13.12/sbt-0.13.12.tgz'), ('md5', 'cec3071d46ef13334c8097cc3467ff28')])), ('test', CommentedMap([('commands', ['sbt -version'])]))])), 'scala': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scala/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scala'), ('version', '2.11.8')])), ('about', CommentedMap([('home', 'http://www.scala-lang.org/'), ('license', 'BSD')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['java-jdk >=8']), ('run', ['java-jdk >=8'])])), ('source', CommentedMap([('fn', 'scala-2.11.8.tgz'), ('url', 'http://downloads.lightbend.com/scala/2.11.8/scala-2.11.8.tgz'), ('md5', 'e7e1f7ddc8873d228d611e801b8d2795')])), ('test', CommentedMap([('commands', ['scalac -help'])]))])), 'scalpel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scalpel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scalpel'), ('version', '0.5.3')])), ('source', CommentedMap([('fn', 'scalpel-0.5.3.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/scalpel/scalpel-0.5.3.tar.gz'), ('md5', '682c9f1cd6ab2cb11c6866f303c673f0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'bamtools']), ('run', ['bamtools', 'samtools', 'bcftools', 'perl'])])), ('test', CommentedMap([('commands', ['scalpel-discovery --help'])])), ('about', CommentedMap([('home', 'http://scalpel.sourceforge.net/'), ('license', 'MIT'), ('summary', 'Sensitive detection of INDELs (INsertions and DELetions)')]))])), 'scg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scg/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://bitbucket.org/aroth85/scg/downloads/<{ name }}-<{ version }}.tar.gz'), ('md5', '6f0fb91266de66d6a9d73fb741f49e13')])), ('build', CommentedMap([('script', '$PYTHON setup.py install --record=record.txt'), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pandas >=0.16', 'numpy >=1.9.2', 'scipy >=0.15', 'pyyaml']), ('run', ['python', 'pandas >=0.16', 'numpy >=1.9.2', 'scipy >=0.15', 'pyyaml', 'libgcc'])])), ('test', CommentedMap([('source_files', ['examples/config.yaml', 'examples/state_map.yaml', 'examples/snv.tsv.gz', 'examples/breakpoint.tsv.gz']), ('commands', ['scg --version', 'scg --help', 'scg run_doublet_model --config_file examples/config.yaml --state_map_file examples/state_map.yaml'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/aroth85/scg/wiki/Home'), ('license', 'GPL-3'), ('license_file', 'LICENSE.txt'), ('summary', 'The single cell genotyper (SCG): \"Roth A, McPherson A, Laks E, Biele J, Yap D, Wan A, et al. Clonal genotype and population structure inference from single-cell tumor sequencing. Nat Meth. 2016;13: 573–576. doi:10.1038/nmeth.3867\"')]))])), 'schavott': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/schavott/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/emilhaegglund/schavott/archive/<{ version }}.tar.gz'), ('md5', 'c4ed83657af9997719edec86ce651114')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfasta', 'h5py >=2.2.0', 'bokeh', 'watchdog >=0.8.3', 'numpy']), ('run', ['python', 'pyfasta', 'h5py >=2.2.0', 'bokeh', 'watchdog >=0.8.3', 'numpy'])])), ('test', CommentedMap([('imports', ['schavott'])])), ('about', CommentedMap([('home', 'http://github.com/emilhaegglund/schavott'), ('license', 'MIT'), ('summary', 'Assembly and scaffolding of bacterial genomes in real time using MinION-sequencing.'), ('license_file', 'LICENSE')]))])), 'schema': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/schema/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'schema'), ('version', '0.6.6')])), ('source', CommentedMap([('fn', 'schema-0.6.6.tar.gz'), ('url', 'https://pypi.python.org/packages/05/5c/25b7fccf722d579d1368415db031dbefedbbddac125363f7827af6aa5ece/schema-0.6.6.tar.gz'), ('md5', '0da19403e5e2abd49e323b0c076dde18')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('imports', ['schema'])])), ('about', CommentedMap([('home', 'https://github.com/keleshev/schema'), ('license', 'MIT License'), ('summary', 'Simple data validation library')]))])), 'schema-salad': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/schema-salad/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'schema-salad'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'schema-salad-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/schema-salad/schema-salad-<{ version }}.tar.gz'), ('md5', 'f45cd87c34a90ccedae2d00278a9c0af')])), ('build', CommentedMap([('entry_points', ['schema-salad-tool=schema_salad.main:main']), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3']), ('run', ['python', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3'])])), ('test', CommentedMap([('imports', ['schema_salad']), ('commands', ['schema-salad-tool --help'])])), ('about', CommentedMap([('home', 'https://github.com/common-workflow-language/schema_salad'), ('license', 'Apache 2.0'), ('summary', 'Schema Annotations for Linked Avro Data (SALAD)')]))])), 'scikit-bio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scikit-bio/0.2.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scikit-bio'), ('version', '0.2.3')])), ('source', CommentedMap([('fn', 'scikit-bio-0.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/scikit-bio/scikit-bio-0.2.3.tar.gz'), ('md5', '2bb1148468d3e5129fa8ff5b29cbeaa3')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'matplotlib >=1.1.0', 'scipy >=0.13.0', 'pandas', 'future', 'six', 'natsort', 'ipython']), ('run', ['python', 'numpy x.x', 'matplotlib >=1.1.0', 'scipy >=0.13.0', 'pandas', 'future', 'six', 'natsort', 'ipython'])])), ('test', CommentedMap([('imports', ['skbio', 'skbio.alignment', 'skbio.alignment._lib', 'skbio.alignment.tests', 'skbio.diversity', 'skbio.diversity.alpha', 'skbio.diversity.alpha.tests', 'skbio.diversity.beta', 'skbio.diversity.beta.tests', 'skbio.draw', 'skbio.draw.tests', 'skbio.format', 'skbio.format.sequences', 'skbio.format.sequences.tests', 'skbio.io', 'skbio.io.tests', 'skbio.parse', 'skbio.parse.sequences', 'skbio.parse.sequences.tests', 'skbio.parse.tests', 'skbio.sequence', 'skbio.sequence.tests', 'skbio.stats', 'skbio.stats.distance', 'skbio.stats.distance.tests', 'skbio.stats.ordination', 'skbio.stats.ordination.tests', 'skbio.stats.tests', 'skbio.tests', 'skbio.tree', 'skbio.tree.tests', 'skbio.util', 'skbio.util.tests'])])), ('about', CommentedMap([('home', 'http://scikit-bio.org'), ('license', 'BSD License'), ('summary', 'Data structures, algorithms and educational resources for bioinformatics.'), ('license_file', 'COPYING.txt')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'scikits-datasmooth': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scikits-datasmooth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scikits-datasmooth'), ('version', '0.7.0')])), ('source', CommentedMap([('fn', 'scikit-datasmooth-0.7.0.tar.gz'), ('url', 'https://github.com/jjstickel/scikit-datasmooth/releases/download/v0.7.0/scikit-datasmooth-0.7.0.tar.gz'), ('sha256', 'd25f60f5618f9ec3533da197fd6bdd30ea90aebc8400b30b0127d739ad241060')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'cvxopt']), ('run', ['python', 'numpy', 'scipy', 'cvxopt'])])), ('test', CommentedMap([('imports', ['scikits.datasmooth'])])), ('about', CommentedMap([('home', 'https://github.com/jjstickel/scikit-datasmooth/'), ('license', 'BSD'), ('summary', 'Scikits data smoothing package'), ('license_family', 'BSD')]))])), 'scoary': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scoary/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scoary'), ('version', '1.6.9')])), ('source', CommentedMap([('fn', 'v1.6.9.tar.gz'), ('url', 'https://github.com/AdmiralenOla/Scoary/archive/v1.6.9.tar.gz')])), ('build', CommentedMap([('entry_points', ['scoary=scoary.methods:main', 'scoary_GUI=scoary.GUI:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.16', 'argparse', 'ete3', 'six']), ('run', ['python', 'scipy >=0.16', 'argparse', 'ete3', 'six'])])), ('test', CommentedMap([('imports', ['scoary']), ('commands', ['scoary --help'])])), ('about', CommentedMap([('home', 'https://github.com/AdmiralenOla/Scoary'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Microbial pan-GWAS using the output from Roary'), ('license_family', 'GPL3')]))])), 'scoop': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scoop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scoop'), ('version', '0.7.1.1')])), ('source', CommentedMap([('fn', 'scoop-0.7.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/88/78/d175920fd8a6d4d607c45477731835ba7aa5f03924e556f9e8f1b50fb9a1/scoop-0.7.1.1.tar.gz'), ('md5', 'a9af3e9884c4e8f1f512c89490f790ed'), ('patches', ['setup.py.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'greenlet >=0.3.4', 'pyzmq >=13.1.0']), ('run', ['python', 'greenlet >=0.3.4', 'pyzmq >=13.1.0', 'libgcc'])])), ('test', CommentedMap([('imports', ['scoop', 'scoop._comm', 'scoop.bootstrap', 'scoop.broker', 'scoop.discovery', 'scoop.launch'])])), ('about', CommentedMap([('home', 'http://github.com/soravux/scoop'), ('license', 'GNU Library or Lesser General Public License (LGPL)'), ('summary', 'Scalable COncurrent Operations in Python')]))])), 'screamingbackpack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/screamingbackpack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'screamingbackpack'), ('version', '0.2.333')])), ('source', CommentedMap([('fn', 'ScreamingBackpack-0.2.333.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/ScreamingBackpack/ScreamingBackpack-0.2.333.tar.gz'), ('md5', '22d776c6d49eb6e6b2f387af99cc471f')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['screamingbackpack'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/ScreamingBackpack/'), ('license', 'GPLv3'), ('summary', 'ScreamingBackpack')]))])), 'screed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/screed/0.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'screed'), ('version', '0.9')])), ('source', CommentedMap([('fn', 'screed-0.9.tar.gz'), ('md5', '611dd11105489e4153265e055f13debd'), ('url', 'https://pypi.python.org/packages/source/s/screed/screed-0.9.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'bz2file ==0.98']), ('run', ['python', 'bz2file ==0.98'])])), ('test', CommentedMap([('imports', ['screed'])])), ('about', CommentedMap([('home', 'https://github.com/dib-lab/screed'), ('license', 'BSD'), ('summary', 'a simple read-only sequence database, designed for short reads')]))])), 'scrm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scrm/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'scrm'), ('version', '1.7.2')])), ('source', CommentedMap([('fn', 'scrm-src.tar.gz'), ('url', 'https://github.com/scrm/scrm/releases/download/v1.7.2/scrm-src.tar.gz'), ('md5', 'eb55770072d2bd6f42cddc8e1d9bd30f')])), ('test', CommentedMap([('commands', ['scrm 2 1 -T > /dev/null'])])), ('about', CommentedMap([('home', 'https://scrm.github.io/'), ('summary', 'A coalescent simulator for genome-scale sequences'), ('license', 'GNU General Public License v3 or later (GPLv3+)')]))])), 'searchgui': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/searchgui/3.2.8/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', '3c1c82325e19d30dbe8bb7233e9d8c0e')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))])), 'secapr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/secapr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'secapr'), ('version', '1.0.2')])), ('about', CommentedMap([('home', 'https://github.com/AntonelliLab/seqcap_processor'), ('license', 'MIT'), ('summary', 'Process sequence-capture FASTQ files into alignments for phylogenetic analyses. Integrates allele phasing, producing haplotype alignments.')])), ('source', CommentedMap([('url', 'https://github.com/AntonelliLab/seqcap_processor/archive/v1.0.2.tar.gz'), ('sha256', 'ada58295f1d02ca2da0b57a49d74770afcb3ac3137c25c32ecf19bb44937dbfa')])), ('build', CommentedMap([('skip', True), ('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'pandas']), ('run', ['python', 'biopython', 'emboss', 'bcftools', 'samtools', 'picard', 'seqtk', 'bwa', 'lastz', 'mafft', 'muscle', 'cogent', 'trimmomatic', 'trinity', 'abyss', 'fastqc', 'pandas'])])), ('test', CommentedMap([('imports', ['secapr']), ('commands', ['secapr --version'])]))])), 'secimtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/secimtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}<{ rc_version }}.tar.gz'), ('<{ hash_type }}', '<{ hash_value }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install  --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'lxml >=3.3.3', 'matplotlib >=1.5.1,<2.0.0', 'matplotlib-venn >=0.11.1', 'mpld3 ==0.2', 'numpy ==1.11.0', 'palettable', 'pandas ==0.18.1', 'patsy >=0.4.0', 'pymc >=2.3.6', 'rpy2 >=2.3.10', 'scikit-learn >=0.18', 'scipy >=0.15.1', 'seaborn ==0.7.0', 'statsmodels ==0.6.1', 'sympy ==0.7.4.1']), ('run', ['python', 'lxml >=3.3.3', 'matplotlib >=1.5.1,<2.0.0', 'matplotlib-venn >=0.11.1', 'mpld3 ==0.2', 'numpy ==1.11.0', 'palettable', 'pandas ==0.18.1', 'patsy >=0.4.0', 'pymc >=2.3.6', 'rpy2 >=2.3.10', 'scikit-learn >=0.18', 'scipy >=0.15.1', 'seaborn ==0.7.0', 'statsmodels ==0.6.1', 'sympy ==0.7.4.1'])])), ('test', CommentedMap([('imports', ['secimtools', 'secimtools.anovaModules', 'secimtools.dataManager', 'secimtools.visualManager'])])), ('about', CommentedMap([('home', 'https://github.com/secimTools/SECIMTools'), ('license', 'MIT License'), ('license_family', 'MIT'), ('summary', 'Metabolomics tools from the SECIM project'), ('description', 'SECIMTools project aims to develop a suite of tools for processing of metabolomics data, which can be run in a standalone mode or via Galaxy Genomics Framework.')])), ('extra', CommentedMap([('recipe-maintainers', 'om@rc.ufl.edu')]))])), 'seedme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seedme/1.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seedme'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'seedme-1.2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/seedme/seedme-1.2.0.tar.gz'), ('md5', 'd8c342c6c5f345fd5a8afabdaf14a307'), ('patches', ['license.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests ==2.7.0']), ('run', ['python', 'requests ==2.7.0', 'curl'])])), ('test', CommentedMap([('commands', ['seedme.py --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.seedme.org/downloads'), ('license', 'GPL'), ('license_file', 'LICENSE.txt'), ('summary', 'Python REST like client for SeedMe.org')]))])), 'segemehl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segemehl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segemehl'), ('version', '0.2.0')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/'), ('summary', 'Short read mapping with gaps')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'segemehl_0_2_0.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz'), ('sha256', '2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['segemehl.x --help 2>&1 | grep -A1 VERSION', 'lack.x --help 2>&1 | grep -A1 VERSION', 'testrealign.x --help 2>&1 | grep -A1 VERSION'])]))])), 'segmentation-fold': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segmentation-fold/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segmentation-fold'), ('version', '1.7.0')])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/segmentation-fold'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'RNA-Folding with predefined segments including K-turns and loop-E-motifs')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'd1b24fec1da405ba70e07ef27046b385b13e8d1f.tar.gz'), ('url', 'https://github.com/yhoogstrate/segmentation-fold/archive/d1b24fec1da405ba70e07ef27046b385b13e8d1f.tar.gz'), ('sha256', '0995272183e31b0b80be57dee22f0e15c62df10d5034f106a3e4454bf9b82e7a')])), ('requirements', CommentedMap([('build', ['llvmdev ==3.9.1', 'libcxx', 'gcc ==4.8.5', 'boost ==1.63.0', 'cmake', 'python ==2.7.12', 'setuptools', 'pysam >=0.8.1,<=0.8.3', 'htseq >=0.6.1', 'click >=4.0']), ('run', ['libgcc', 'libcxx', 'boost ==1.63.0', 'python ==2.7.12', 'pysam >=0.8.1,<=0.8.3', 'htseq >=0.6.1', 'click >=4.0'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 segmentation-fold --version', 'LANG=C.UTF-8 segmentation-fold-utils --help'])]))])), 'segway': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segway'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'segway-2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/09/13/96398f873521032ee80b8665c59785445f4870159f8cb47840e22e8fd156/segway-2.0.tar.gz'), ('md5', 'b4bd3d4ebfcfe1e5540f8da523956764')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['segway = segway.run:main', 'segway-task = segway.task:main', 'segway-layer = segway.layer:main', 'segway-winner = segway.winner:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'hgtools', 'colorbrewer', 'drmaa >=0.4a3']), ('run', ['python', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'colorbrewer', 'drmaa >=0.4a3'])])), ('test', CommentedMap([('imports', ['segway']), ('commands', ['segway --help'])])), ('about', CommentedMap([('home', 'http://segway.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'a tool for easy pattern discovery and identification in functional genomics data.')]))])), 'selectsequencesfrommsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/selectsequencesfrommsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/eggzilla/SelectSequencesFromMSA/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['stack >=1.3.2', 'gmp', 'zlib', 'perl']), ('run', ['gmp', 'perl', 'libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['SelectSequencesFromMSA --help'])])), ('about', CommentedMap([('home', 'https://github.com/eggzilla/SelectSequencesFromMSA'), ('license', 'GPL-3'), ('summary', 'Tool to select representative sequences from a multiple sequence alignment')]))])), 'selscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/selscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'selscan'), ('version', '1.1.0b')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '1.1.0b.zip'), ('url', 'https://github.com/szpiech/selscan/archive/1.1.0b.zip'), ('md5', '02a39fff61fadcfb5df248d37111141f'), ('patches', ['Makefile.linux.patch', 'Makefile.mac.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', [\"selscan 2>&1 | grep 'selscan' &> /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/szpiech/selscan'), ('license', 'GPLv3'), ('summary', 'a program to calculate EHH-based scans for positive selection in genomes')]))])), 'semeta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/semeta/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://it.hcmute.edu.vn/bioinfo/metapro/Source/<{ name }}<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'blast'])])), ('test', CommentedMap([('commands', ['which SeMeta_Assign', 'which SeMeta_Cluster'])])), ('about', CommentedMap([('home', 'http://it.hcmute.edu.vn/bioinfo/metapro/SeMeta.html'), ('license', 'GPL-3'), ('summary', 'SeMeta is a new software for taxonomic assignment of metagenomic reads. It\\nsupports both single-end and paired-end reads. The software is implemented\\nin C++\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1186/s12859-015-0872-x'), ('notes', 'Databases are required. Please see the project homepage.')]))])), 'semidbm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/semidbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'semidbm'), ('version', '0.5.1')])), ('source', CommentedMap([('fn', 'semidbm-0.5.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/semidbm/semidbm-0.5.1.tar.gz'), ('md5', 'e0e45cf55fb0cfca1da04368619ff116')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['semidbm', 'semidbm.loaders'])])), ('about', CommentedMap([('home', 'https://github.com/jamesls/semidbm'), ('license', 'BSD License'), ('summary', 'Cross platform (fast) DBM interface in python')]))])), 'sentieon': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sentieon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sentieon'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'sentieon-genomics-<{ version }}-linux.tar.gz'), ('url', 'https://sentieon.sharepoint.com/Release/_layouts/15/guestaccess.aspx?docid=0dc87e53e46854d22ad84f9a5082abc8b&authkey=AaHvyGjBt9eddKu9YOFUX-M'), ('md5', '1862e8a129155ee789bec9da3a147da0'), ('patches', ['sentieon_symlinks.patch', 'bwa_symlinks.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True), ('binary_relocation', False)])), ('requirements', CommentedMap([('build', None), ('run', ['libgfortran', 'libgcc', 'zlib'])])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://sentieon.com'), ('license', 'Commercial (requires license for use)'), ('summary', 'Accelerated performance bioinformatics tools for mapping and variant calling')]))])), 'seq-gen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seq-gen/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/rambaut/Seq-Gen/archive/<{ version }}.tar.gz'), ('md5', '14633986c015977ab56cd38fab5f0a08')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['seq-gen -h'])])), ('about', CommentedMap([('home', 'http://tree.bio.ed.ac.uk/software/Seq-Gen/'), ('license', 'Unknown'), ('summary', 'Seq-Gen is a program that will simulate the evolution of nucleotide or amino acid sequences along a phylogeny, using common models of the substitution process.')]))])), 'seq2c': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seq2c/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seq2c'), ('version', '2016.03.23')])), ('source', CommentedMap([('fn', 'seq2c-8e62a3f.tar.gz'), ('url', 'https://github.com/AstraZeneca-NGS/Seq2C/archive/8e62a3f.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['seq2cov.pl -h', 'cov2lr.pl -h', 'lr2gene.pl -h'])])), ('about', CommentedMap([('home', 'https://github.com/AstraZeneca-NGS/Seq2C'), ('license', 'MIT'), ('summary', 'Cohort based copy number calling in gene regions')]))])), 'seq2hla': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seq2hla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seq2hla'), ('version', '2.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '95e554b1d191.zip'), ('url', 'https://bitbucket.org/sebastian_boegel/seq2hla/get/95e554b1d191.zip'), ('patches', ['seq2HLA.patch'])])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'biopython >=1.58', 'bowtie ==1.1.2', 'r-base'])])), ('test', CommentedMap([('commands', ['seq2HLA -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/sebastian_boegel/seq2hla'), ('license', 'MIT'), ('summary', 'Precision HLA typing and expression from next-generation RNA sequencing data')]))])), 'seqbuster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqbuster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqbuster'), ('license', 'MIT'), ('summary', 'miRNA and isomiR annotation')])), ('package', CommentedMap([('name', 'seqbuster'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'seqbuster-3.1.tar.gz'), ('url', 'https://github.com/lpantano/seqbuster/archive/miraligner-3.1.tar.gz'), ('md5', '3b4d23fa9839910ac980c3fb2822fbfa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8'])])), ('test', CommentedMap([('commands', ['miraligner -h'])]))])), 'seqcluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqcluster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqclsuter'), ('license', 'MIT'), ('summary', 'small RNA analysis from NGS data')])), ('package', CommentedMap([('name', 'seqcluster'), ('version', '1.2.4a')])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/seqcluster'), ('git_rev', '9f9f886e6c45b31121a75d1f47a4443dfaa0e96a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc']), ('run', ['python', 'pyyaml', 'pysam >=0.8.1', 'pandas', 'pybedtools', 'progressbar', 'matplotlib'])])), ('test', CommentedMap([('commands', ['seqcluster cluster -h']), ('imports', ['seqcluster'])]))])), 'seqkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seqkit'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'https://github.com/shenwei356/seqkit'), ('summary', 'Cross-platform and ultrafast toolkit for FASTA/Q file manipulation'), ('license', 'MIT')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'seqkit_<{ version }}_linux64.tar.gz'), ('url', 'https://github.com/shenwei356/seqkit/releases/download/v<{ version }}/seqkit_linux_amd64.tar.gz'), ('md5', '6d56f7026fdc8e1240e9ec3b8703f1fc')])), ('test', CommentedMap([('commands', ['seqkit version --help'])]))])), 'seqmagick': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqmagick/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seqmagick'), ('version', '0.6.1')])), ('source', CommentedMap([('fn', 'seqmagick-0.6.1.tar.gz'), ('url', 'https://pypi.python.org/packages/62/f4/5bf0b358551ba5f1a18de039a7001f2e0fad4b08ff07ff0cc0cce492ad47/seqmagick-0.6.1.tar.gz'), ('md5', 'b20043ad6e558fdf8369e86aa2731ad9')])), ('build', CommentedMap([('entry_points', ['seqmagick = seqmagick.scripts.cli:main']), ('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython >=1.58']), ('run', ['python', 'biopython >=1.58'])])), ('test', CommentedMap([('imports', ['seqmagick', 'seqmagick.scripts', 'seqmagick.subcommands', 'seqmagick.test', 'seqmagick.test.integration', 'seqmagick.test.integration.data']), ('commands', ['seqmagick --help'])])), ('about', CommentedMap([('home', 'http://github.com/fhcrc/seqmagick'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Tools for converting and modifying sequence files from the command-line'), ('license_family', 'GPL')]))])), 'seqprep': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqprep/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'seqprep'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'v1.1.tar.gz'), ('url', 'https://github.com/jstjohn/SeqPrep/archive/v1.1.tar.gz'), ('md5', '42bd8e62e8bf99befb37ed29ce047115'), ('patches', ['cflags.patch'])])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://github.com/jstjohn/SeqPrep'), ('license', 'MIT')]))])), 'seqtk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqtk/r82/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seqtk'), ('version', 'r82')])), ('source', CommentedMap([('fn', 'seqtk-r82.tar.gz'), ('url', 'https://github.com/lh3/seqtk/archive/5e1e8dbd506ea1ff8c77d468a1f27b8e8f73eac0.tar.gz'), ('sha256', 'dd4b54a3ba57279659153204aa59aa1d5efe4d307549ac7b8e93b67eb9d3248f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/seqtk'), ('license', 'MIT License'), ('summary', 'Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format')])), ('test', CommentedMap([('commands', ['seqtk seq'])]))])), 'sequana': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sequana/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('number', 0), ('entry_points', ['sequana_gui=sequana.gui.sequana_gui:main', 'sequanix=sequana.gui.sequana_gui:main', 'fastq_head=sequana.scripts.fastq_head:main', 'fastq_count=sequana.scripts.fastq_count:main', 'sequana_fastq_head=sequana.scripts.fastq_head:main', 'sequana_fastq_count=sequana.scripts.fastq_count:main', 'sequana=sequana.scripts.main:main', 'sequana_taxonomy=sequana.scripts.taxonomy:main', 'sequana_coverage=sequana.scripts.coverage:main', 'sequana_summary=sequana.scripts.summary:main', 'sequana_mapping=sequana.scripts.mapping:main', 'sequana_compressor=sequana.scripts.compressor:main', 'sequana_report=sequana.scripts.reports:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'atropos <=1.1.10', 'bioservices >=1.4.14', 'biokit >=0.4.1', 'bwa', 'colorlog', 'cutadapt', 'easydev >=0.9.34', 'graphviz', 'kraken', 'krona', 'mock', 'multiqc', 'numexpr', 'pandas', 'pigz', 'psutil', 'pyyaml', 'pyvcf', 'pysam', 'ruamel.yaml >=0.13.2', 'qtconsole', 'reports >=0.3.1', 'sambamba', 'samtools', 'snakemake', 'shustring', 'xlrd'])])), ('test', CommentedMap([('imports', None), ('commands', None)])), ('about', CommentedMap([('home', ['http://github.com/sequana/']), ('license', 'BSD 3-clause'), ('summary', 'A set of standalone application and pipelines dedicated to NGS (new generation sequencing) analysis'), ('license_family', 'BSD')]))])), 'setuptools_cython': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/setuptools_cython/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'setuptools_cython'), ('version', '0.2.1')])), ('source', CommentedMap([('fn', 'setuptools_cython-0.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/setuptools_cython/setuptools_cython-0.2.1.tar.gz'), ('md5', 'c29fc5bf2894e69cbd98e3800cef4707')])), ('build', CommentedMap([('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython']), ('run', ['python', 'setuptools', 'cython'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/setuptools_cython/'), ('license', 'http://www.gnu.org/licenses/gpl-2.0.html'), ('summary', 'Cython setuptools integration')]))])), 'sevenbridges-python': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sevenbridges-python/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sevenbridges-python'), ('version', '0.7.2')])), ('source', CommentedMap([('fn', 'sevenbridges-python-0.7.2.tar.gz'), ('url', 'https://pypi.python.org/packages/07/cc/f959f1d79c67cb46457ca58c6cf2210acfd1c018cbc4e0431815e6e80b7e/sevenbridges-python-0.7.2.tar.gz'), ('md5', 'f74052f71c9a325f07ed5e5f806619a3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'six', 'futures']), ('run', ['python', 'requests', 'six', 'futures'])])), ('test', CommentedMap([('imports', ['sevenbridges', 'sevenbridges.http', 'sevenbridges.meta', 'sevenbridges.models', 'sevenbridges.models.compound', 'sevenbridges.models.compound.billing', 'sevenbridges.models.compound.files', 'sevenbridges.models.compound.jobs', 'sevenbridges.models.compound.projects', 'sevenbridges.models.compound.tasks', 'sevenbridges.models.compound.volumes', 'sevenbridges.transfer'])])), ('about', CommentedMap([('home', 'https://github.com/sbg/sevenbridges-python'), ('license', 'Apache Software License'), ('summary', 'SBG API python client bindings')]))])), 'sff2fastq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sff2fastq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/indraniel/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['sff2fastq -v'])])), ('about', CommentedMap([('home', 'https://github.com/indraniel/sff2fastq'), ('license', 'GPL-3'), ('license_family', 'GPL'), ('summary', 'Extract 454 Genome Sequencer reads from a SFF file and convert them into a FASTQ formatted output'), ('description', 'The program sff2fastq extracts read information from a SFF file, produced by\\nthe 454 genome sequencer, and outputs the sequences and quality scores in a\\nFASTQ format.\\n')])), ('extra', CommentedMap([('recipe-maintainers', ['keuv-grvl'])]))])), 'sga': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sga/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sga'), ('version', '0.10.13')])), ('source', CommentedMap([('fn', 'v0.10.13.tar.gz'), ('url', 'https://github.com/jts/sga/archive/v0.10.13.tar.gz'), ('md5', 'd4f6aefc48c940dba96cc6513649ecdd')])), ('requirements', CommentedMap([('build', ['bamtools']), ('run', ['bamtools'])])), ('about', CommentedMap([('home', 'https://github.com/jts/sga'), ('license', 'GPLv3'), ('summary', 'SGA - String Graph Assembler. SGA is a de novo assembler for DNA sequence reads. It is based on Gene Myers string graph formulation of assembly and uses the FM-index/Burrows-Wheeler transform to efficiently find overlaps between sequence reads.')]))])), 'shapeit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shapeit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shapeit'), ('version', '2.r837')])), ('source', CommentedMap([('fn', 'shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('md5', 'd778d2e6963a5fdbf4edb1f2dafa9f63')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'boost']), ('run', ['libgcc', 'boost'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'), ('license', 'Free for Academic Use'), ('summary', 'SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.')])), ('test', CommentedMap([('commands', ['shapeit --help'])]))])), 'shapemapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shapemapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shapemapper'), ('version', '1.2')])), ('source', CommentedMap([('url', 'http://www.chem.unc.edu/rna/software-files/ShapeMapper_v1.2.tar.gz'), ('md5', '640bf3f898a37ff8c5ca06f5e68c40d3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'bowtie2', 'matplotlib', 'httplib2', 'rnastructure']), ('run', ['python', 'libgcc', 'bowtie2', 'matplotlib', 'httplib2', 'rnastructure'])])), ('test', CommentedMap([('commands', ['type ShapeMapper.py', 'type trimPhred', 'type parseAlignment', 'type countMutations'])])), ('about', CommentedMap([('home', 'http://www.chem.unc.edu/rna/software.html'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'ShapeMapper converts raw sequencing files into mutational profiles, creates SHAPE reactivity plots, and provides extensive diagnostic information useful for experiment analysis and troubleshooting.')]))])), 'sharedmem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sharedmem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sharedmem'), ('version', '0.3.5')])), ('source', CommentedMap([('fn', 'sharedmem-0.3.5.tar.gz'), ('url', 'https://pypi.python.org/packages/a4/cd/9e0ff25ba14e17682e4a474dfd480c3c454ef5edc885c7a9c16d2e9a3c3e/sharedmem-0.3.5.tar.gz'), ('md5', '1c6341087ec9897daa5ac4e8418b86ed')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['sharedmem'])])), ('about', CommentedMap([('home', 'http://github.com/rainwoodman/sharedmem'), ('license', None), ('summary', 'Dispatch your trivially parallizable jobs with sharedmem. '), ('license_family', 'Public-Domain')]))])), 'shellescape': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shellescape/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shellescape'), ('version', '3.4.1')])), ('source', CommentedMap([('fn', 'shellescape-3.4.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/shellescape/shellescape-3.4.1.tar.gz'), ('md5', '8dc7545a9a27d19dfcc4d7b33a72c296')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['shellescape'])])), ('about', CommentedMap([('home', 'https://github.com/chrissimpkins/shellescape'), ('license', 'MIT License'), ('summary', 'Shell escape a string to safely use it as a token in a shell command (backport of Python shlex.quote for Python versions 2.x & < 3.3)')]))])), 'shiver': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shiver/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shiver'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'shiver-<{ version }}.tar.gz'), ('url', 'https://github.com/ChrisHIV/shiver/archive/v<{ version }}.tar.gz'), ('sha256', '4670bb0c3ee68d8bd492a2b024a255fd31d974f47fd9b691c6158c3f67ed2658'), ('patches', ['0001-changed-tools-to-shiver_tools.patch', '0002-changed-python2-to-python.patch', '0003-added-fastaq-installation-on-demand.patch', '0004-fixed-pyfastaq-installation.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'smalt', 'trimmomatic', 'samtools', 'biopython', 'mafft', 'picard', 'blast >=2.2.28', 'pyfastaq'])])), ('test', CommentedMap([('commands', ['shiver_init.sh 2>&1 | grep \"Arguments\"'])])), ('about', CommentedMap([('home', 'https://github.com/ChrisHIV/shiver'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'SHIVER - Sequences from HIV Easily Reconstructed'), ('license_family', 'Public-Domain')])), ('extra', CommentedMap([('recipe-maintainers', ['notestaff'])]))])), 'shovill': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shovill/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shovill'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('string', 'pl<{CONDA_PERL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'shovill_v<{ version }}.tar.gz'), ('url', 'https://github.com/tseemann/shovill/archive/v<{ version }}.tar.gz'), ('sha256', 'b27cfd00adf541fa75645b9874b9243c9928c194c0c126fd0829cf00771bffa4')])), ('requirements', CommentedMap([('run', ['perl <{CONDA_PERL}}*', 'perl-file-slurp', 'perl-time-piece', 'perl-file-spec', 'perl-findbin', 'unzip', 'lighter', 'bwa', 'kmc >=2', 'flash', 'pilon >=1.22', 'seqtk', 'trimmomatic', 'spades', 'pigz', 'samtools >=1.3'])])), ('test', CommentedMap([('commands', ['shovill --help', 'shovill --version'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/shovill'), ('license', 'GPL2'), ('license_file', 'LICENSE'), ('summary', 'Faster SPAdes assembly of Illumina reads')]))])), 'shustring': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shustring/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://raw.githubusercontent.com/sequana/resources/master/software/shustring-2.6.tar.gz'), ('md5', 'd57ed4d24f2f3e032a059994f6e9ec8d')])), ('build', CommentedMap([('skip', False), ('number', 0)])), ('requrements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['./shustring --help 2>&1 | grep shustring > /dev/null'])])), ('about', CommentedMap([('home', 'http://guanine.evolbio.mpg.de/cgi-bin/shustring/shustring.cgi.pl'), ('summary', 'Program for Computing SHortest Unique SubSTRINGs'), ('license', 'GPL-2'), ('license_family', 'GPL2'), ('license_file', 'COPYING')])), ('extra', CommentedMap([('recipe-maintainers', ['cokelaer'])]))])), 'sickle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sickle/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sickle'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'Sickle-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/Sickle/Sickle-0.5.tar.gz'), ('md5', '9c93e7f3e0db98ab1a18107dfa096082')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.1.0', 'lxml >=3.2.3']), ('run', ['python', 'requests >=1.1.0', 'lxml >=3.2.3'])])), ('test', CommentedMap([('imports', ['sickle']), ('requires', ['mock >=1.0.1', 'nose >=1.0'])])), ('about', CommentedMap([('home', 'http://github.com/mloesch/sickle'), ('license', 'BSD License'), ('summary', 'A lightweight OAI client library for Python')]))])), 'sickle-trim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sickle-trim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sickle-trim'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'sickle-<{ version }}.tar.gz'), ('url', 'https://github.com/najoshi/sickle/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['zlib 1.2.8']), ('run', ['zlib 1.2.8'])])), ('test', CommentedMap([('commands', ['sickle --version'])])), ('about', CommentedMap([('home', 'https://github.com/najoshi/sickle'), ('license', 'MIT'), ('summary', 'Windowed Adaptive Trimming for fastq files using quality')]))])), 'sift4g': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sift4g/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sift4g'), ('version', '2.0.0')])), ('source', CommentedMap([('fn', 'SIFT4G_v2.0.0.tar.gz'), ('url', 'http://sift.bii.a-star.edu.sg/sift4g/SIFT4G_v2.0.0.tar.gz'), ('md5', '99a4de9c84902c95d90e95ed993731b8')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['sift4g -h 2>&1 | grep \"sift4g\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://sift.bii.a-star.edu.sg/sift4g/'), ('license', 'GPL-3.0'), ('summary', 'SIFT 4G is a faster version of SIFT that enables us to scale up and provide SIFT predictions for more organisms.')]))])), 'sim4db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sim4db/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Getting_Started_with_Sim4db'), ('license', 'GPL'), ('summary', 'Sim4db and leaff: Utilities for fast batch spliced alignment and sequence indexing')])), ('source', CommentedMap([('fn', '<{ name }}-r<{ version }}.tar.bz2'), ('url', 'http://sourceforge.net/projects/kmer/files/sim4db-r2008.tar.bz2'), ('md5', 'fa04c8fc71323bbe1aec58c9d6e765be')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'bzip2']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['comparePolishes 2>&1 | grep comparePolishes', 'convertPolishes --help 2>&1 | grep convertPolishes', 'convertToAtac 2>&1 | grep convertToAtac', 'leaff 2>&1 | grep leaff', 'mergePolishes 2>&1 | grep mergePolishes', 'mt19937ar-test', 'pickBestPair 2>&1 | grep pickBestPair', 'removeRedundant 2>&1 | grep removeRedundant', 'reportAlignmentDifferences 2>&1 | grep reportAlignmentDifferences', 'sim4db 2>&1 | grep sim4db'])]))])), 'simlord': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/simlord/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'simlord'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'simlord-1.0.2.zip'), ('url', 'https://pypi.python.org/packages/23/c5/bf4b08125ed36150a5a7e6bf48a07a7a2d25cb9d6855df63d5bc4d6e632f/simlord-1.0.2.zip'), ('md5', 'da15b92001429628c65833e12962da58')])), ('build', CommentedMap([('entry_points', ['simlord = simlord.simlord:main']), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'pysam >=0.8.4', 'cython', 'dinopy']), ('run', ['python', 'numpy', 'scipy', 'pysam >=0.8.4', 'dinopy'])])), ('test', CommentedMap([('imports', ['simlord']), ('commands', ['simlord --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/genomeinformatics/simlord/'), ('license', 'MIT License'), ('summary', 'SimLoRD is a read simulator for long reads from third generation sequencing. Currently, it supports the Pacific Biosciences SMRT error model.')]))])), 'simple_sv_annotation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/simple_sv_annotation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'simple_sv_annotation'), ('version', '2017.02.17')])), ('source', CommentedMap([('git_url', 'https://github.com/AstraZeneca-NGS/simple_sv_annotation'), ('git_tag', '0d1bfeda337d7b6ab4deb80ae967b7684f0e0777')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'pyvcf'])])), ('test', CommentedMap([('commands', ['simple_sv_annotation.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/AstraZeneca-NGS/simple_sv_annotation'), ('license', 'MIT'), ('summary', 'Simplify snpEff annotations for interesting cases')]))])), 'simplejson': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/simplejson/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'simplejson'), ('version', '3.8.1')])), ('source', CommentedMap([('fn', 'simplejson-3.8.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/simplejson/simplejson-3.8.1.tar.gz'), ('md5', 'b8441f1053edd9dc335ded8c7f98a974')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['simplejson', 'simplejson.tests'])])), ('about', CommentedMap([('home', 'http://github.com/simplejson/simplejson'), ('license', 'MIT License or Academic Free License (AFL)'), ('summary', 'Simple, fast, extensible JSON encoder/decoder for Python')]))])), 'sina': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sina/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.bz2'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/epruesse/SINA/releases/download/v<{version}}/sina-<{version}}.tar.bz2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkg-config', 'arb-bio', 'boost 1.64.*']), ('run', ['libgcc', 'arb-bio', 'boost 1.64.*'])])), ('test', CommentedMap([('commands', ['sina --help'])])), ('about', CommentedMap([('home', 'https://github.com/epruesse/SINA'), ('license', 'GPLv3'), ('license_family', 'GPL'), ('summary', 'SINA is a tool for aligning sequences with an existing multiple sequence\\nalignment (MSA) at high accuracy. It can also execute a homology search based\\non the computed alignment and generate a per sequence classifications from the\\nsearch results.\\n')]))])), 'singlecell-barcodes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/singlecell-barcodes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'singlecell-barcodes'), ('version', '0.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '6a669fb.tar.gz'), ('url', 'https://github.com/roryk/singlecell-barcodes/archive/6a669fb.tar.gz'), ('md5', '40f120245806d9de42d9e4845d0b2919')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('test', CommentedMap([('commands', [])])), ('about', CommentedMap([('home', 'https://github.com/roryk/singlecell-barcodes'), ('license', 'MIT'), ('summary', 'whitelisted singlecell barcodes and information regarding where molecular/sample/cellular barcodes are in each read, for various singlecell protocols')]))])), 'singularity': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/singularity/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'singularity'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'singularity-<{ version }}.tar.gz'), ('url', 'https://github.com/singularityware/singularity/releases/download/<{ version }}/singularity-<{ version }}.tar.gz'), ('sha256', '0c2e142d4bfef05ae54448aa09f17d4e5e33a7878d450ca1e51239f68df577b5')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('test', CommentedMap([('commands', ['singularity --help'])])), ('about', CommentedMap([('home', 'http://singularity.lbl.gov'), ('license', 'BSD'), ('summary', 'Singularity - Enabling users to have full control of their environment')]))])), 'sis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sis/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'sis.zip'), ('url', 'http://marte.ic.unicamp.br:8747/static/sis.zip'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['mummer', 'perl', 'python']), ('run', ['mummer', 'perl', 'python'])])), ('test', CommentedMap([('commands', ['python $PREFIX/bin/sis.py'])])), ('about', CommentedMap([('home', 'http://marte.ic.unicamp.br:8747/'), ('license', 'This software is experimental in nature and is supplied \"AS IS\", without obligation by the authors to provide  accompanying services or support.  The entire risk as to the quality  and performance of the Software is with you. The authors EXPRESSLY DISCLAIM ANY AND ALL WARRANTIES REGARDING THE SOFTWARE,  WHETHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO WARRANTIES  PERTAINING TO MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.'), ('summary', 'A tool that uses mummer to scaffold small genomes.'), ('maintainer', 'nickp60')]))])), 'sistr_cmd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sistr_cmd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sistr_cmd'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'sistr_cmd-1.0.2.tar.gz'), ('url', 'https://pypi.python.org/packages/48/d5/bf992cf61aef67c9d89879a99000eb02f34ef612608b6f6e0c9394f915f7/sistr_cmd-1.0.2.tar.gz'), ('md5', '9a531cf6139a56613943627de267572c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.11.1', 'pandas >=0.18.1', 'pytables >=3.3.0']), ('run', ['python', 'numpy >=1.11.1', 'pandas >=0.18.1', 'pytables >=3.3.0', 'blast', 'mafft', 'mash'])])), ('test', CommentedMap([('commands', ['sistr -h', 'sistr -V'])])), ('about', CommentedMap([('home', 'https://github.com/peterk87/sistr_cmd/'), ('license', 'Apache'), ('summary', 'Salmonella In Silico Typing Resource (SISTR) commandline tool for serovar prediction')]))])), 'sixgill': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sixgill/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sixgill'), ('version', '0.2.4')])), ('source', CommentedMap([('fn', 'sixgill-0.2.4.tar.gz'), ('url', 'https://pypi.python.org/packages/84/95/7ce89b21cce62349507433b50faeae068aae6da53e8b6a6078725c4a225d/sixgill-0.2.4.tar.gz'), ('md5', '4578548e681c5e2a3e127db84cb4bb83')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'pysam >=0.9.0']), ('run', ['python', 'biopython', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['sixgill']), ('commands', ['sixgill_build --version'])])), ('about', CommentedMap([('license', 'Apache Software License'), ('summary', 'six-frame genome-inferred libraries for LC-MS/MS'), ('license_family', 'Apache')]))])), 'skewer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/skewer/0.1.126/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'skewer'), ('version', '0.1.126')])), ('source', CommentedMap([('fn', '0.1.126.zip'), ('url', 'https://github.com/relipmoc/skewer/archive/0.1.126.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['skewer 2>&1 | grep \"skewer (0.1.126)\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/relipmoc/skewer.git'), ('license', 'MIT'), ('summary', 'A fast and accurate adapter trimmer for next-generation sequencing paired-end reads.')]))])), 'slang': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/slang/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'slang'), ('version', '2.3.0')])), ('source', CommentedMap([('fn', 'slang-2.3.0.tar.bz2'), ('url', 'http://www.jedsoft.org/releases/slang/slang-2.3.0.tar.bz2'), ('sha1', '6e26e90307d4569e5feef195648c0858ba27f7ac')])), ('requirements', CommentedMap([('build', ['readline', 'ncurses <{CONDA_NCURSES}}*', 'libpng', 'zlib']), ('run', ['readline', 'ncurses <{CONDA_NCURSES}}*', 'libpng', 'zlib'])])), ('test', CommentedMap([('commands', [\"slsh --help 2>&1 | grep 'Usage: '\"])])), ('about', CommentedMap([('home', 'http://www.jedsoft.org/slang/index.html'), ('license', 'GPL')]))])), 'slclust': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/slclust/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'slclust'), ('version', 2022010)])), ('source', CommentedMap([('fn', 'slclust_02022010.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/slclust/slclust/slclust_02022010.tar.gz'), ('sha256', '9e21b65288e7fc35647721380cfedffd68608e164b4240c692c511c18c1620f8')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['slclust --help 2>&1 | grep jaccard > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/slclust/'), ('license', 'Artistic License'), ('summary', 'A utility that performs single-linkage clustering with the option of applying a Jaccard similarity coefficient to break weakly bound clusters into distinct clusters.')]))])), 'smalt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smalt/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'smalt'), ('version', '0.7.6')])), ('source', CommentedMap([('fn', 'smalt-0.7.6-static.tar.gz'), ('url', 'https://sourceforge.net/projects/smalt/files/smalt-0.7.6-static.tar.gz'), ('md5', '57ed011836fda8fb5e34604dd1858b5e')])), ('requirements', CommentedMap([('build', ['gcc', 'libgcc', 'libtool', 'pkg-config', 'bambamc', 'zlib', 'python >=2.6,<3', 'samtools']), ('run', ['bambamc', 'zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['smalt version 2>&1 | grep \"SMALT - Sequence Mapping and Alignment Tool\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.sanger.ac.uk/science/tools/smalt-0'), ('license', 'GPLv3'), ('summary', 'SMALT aligns DNA sequencing reads with a reference genome.')]))])), 'smartdenovo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smartdenovo/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/ruanjue/smartdenovo/archive/3d9c22e25bdf4caf6c08ea1acb41ee58e52f61a8.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['smartdenovo.pl test.fa'])])), ('about', CommentedMap([('home', 'https://github.com/ruanjue/smartdenovo'), ('license', 'GPLv3'), ('summary', 'Ultra-fast de novo assembler using long noisy reads')]))])), 'smashbenchmarking': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smashbenchmarking/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'smashbenchmarking'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'smashbenchmarking-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/smashbenchmarking/smashbenchmarking-1.0.1.tar.gz'), ('md5', '77f0c979549a645f849ec28ef8afcda4')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyvcf', 'pyfasta', 'numpy']), ('run', ['python', 'pyvcf', 'pyfasta', 'numpy'])])), ('test', CommentedMap([('imports', ['smashbenchmarking', 'smashbenchmarking.corrupt', 'smashbenchmarking.parsers', 'smashbenchmarking.vcf_eval'])])), ('about', CommentedMap([('home', 'http://github.com/amplab/smash/'), ('license', 'BSD License'), ('summary', 'Check the accuracy of one VCF callset against another')]))])), 'smhasher': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smhasher/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'smhasher'), ('version', '0.150.1')])), ('source', CommentedMap([('fn', 'smhasher-0.150.1.tar.gz'), ('url', 'https://pypi.python.org/packages/5f/b2/421c060300f1449646787273ecd7df558f6f8e4caee3d6fbe8b193cd60d4/smhasher-0.150.1.tar.gz'), ('md5', '4fc61f36b141e137dd2d71ecd0167866')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['smhasher'])])), ('about', CommentedMap([('home', 'http://github.com/phensley/python-smhasher'), ('license', 'MIT License'), ('summary', 'Python extension for smhasher hash functions'), ('license_family', 'MIT')]))])), 'snakemake': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snakemake/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snakemake'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'snakemake-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/snakemake/snakemake-<{ version }}.tar.gz'), ('sha256', '0876797aeb633be7129ac943d9ff5643d491f9115a19484cbe8594c3e87912e5')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', 'python setup.py install --single-version-externally-managed --record record.txt'), ('entry_points', ['snakemake = snakemake:main', 'snakemake-bash-completion = snakemake:bash_completion'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'wrapt', 'requests', 'ratelimiter', 'configargparse', 'appdirs']), ('run', ['python', 'wrapt', 'requests', 'docutils', 'pyyaml', 'requests >=2.8.1', 'dropbox >=7.2.1', 'ftputil >=3.2', 'filechunkio >=1.6', 'pysftp >=0.2.8', 'psutil', 'aioeasywebdav', 'pandas', 'ratelimiter', 'configargparse', 'appdirs'])])), ('test', CommentedMap([('imports', ['snakemake']), ('commands', ['snakemake --version', 'snakemake --version | grep \"<{ version }}\"'])])), ('about', CommentedMap([('home', 'https://snakemake.readthedocs.io'), ('license', 'MIT'), ('summary', 'Snakemake is a workflow management system that aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern specification language in python style. Snakemake workflows are essentially Python scripts extended by declarative code to define rules. Rules describe how to create output files from input files.')]))])), 'snap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snap'), ('version', '2013_11_29')])), ('source', CommentedMap([('fn', 'snap-2013-11-29.tar.gz'), ('url', 'http://korflab.ucdavis.edu/Software/snap-2013-11-29.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', None)])), ('test', CommentedMap([('commands', ['snap 2>&1 | grep \"Semi-HMM-based Nucleic Acid Parser\"'])])), ('about', CommentedMap([('home', 'http://korflab.ucdavis.edu/software.html'), ('license', 'GNU General Public License version 2.0'), ('summary', 'Semi-HMM-based Nucleic Acid Parser -- gene prediction tool')]))])), 'snap-aligner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snap-aligner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snap-aligner'), ('version', '1.0dev.97')])), ('source', CommentedMap([('fn', 'snap-v1.0dev.97.tar.gz'), ('url', 'https://github.com/amplab/snap/archive/bee5657.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['snap-aligner 2>&1 | grep \"SNAP\"'])])), ('about', CommentedMap([('home', 'http://snap.cs.berkeley.edu/'), ('license', 'Apache v2'), ('summary', 'Scalable Nucleotide Alignment Program -- a fast and accurate read aligner for high-throughput sequencing data')]))])), 'sniffles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sniffles/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/fritzsedlazeck/Sniffles/archive/v<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'zlib', 'bamtools >=2.3.0', 'tclap >=1.2.1']), ('run', ['zlib', 'libgcc', 'bamtools >=2.3.0', 'tclap >=1.2.1'])])), ('test', CommentedMap([('commands', ['sniffles --help'])])), ('about', CommentedMap([('home', 'https://github.com/fritzsedlazeck/Sniffles'), ('license', 'MIT'), ('summary', 'Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore)')]))])), 'snippy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snippy/3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snippy'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'snippy_v3.0.tar.gz'), ('md5', '081ddf6ffad192136de2fc9d6635199d'), ('url', 'https://github.com/tseemann/snippy/archive/v3.0.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl', 'perl-app-cpanminus', 'perl-module-build', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff']), ('run', ['perl-threaded', 'perl-bioperl', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff', 'htslib'])])), ('test', CommentedMap([('commands', ['snippy --version'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/snippy'), ('license', 'GPL2'), ('summary', 'Rapid bacterial SNP calling and core genome alignments')]))])), 'snns': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snns/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snns'), ('version', 4.3)])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/SNNS/SNNS_4.3_linux_x64.tar.gz'), ('fn', 'SNNSv4.3.tar.gz'), ('sha256', '54bf92d23e9198f9030a3c3d2b741472e9b8660b27d3b419ade6393b1ebf6f62')])), ('requirements', CommentedMap([('run', ['xorg-libxaw3d', 'bison', 'flex'])])), ('about', CommentedMap([('home', 'http://www.ra.cs.uni-tuebingen.de/SNNS/'), ('license', 'LGPL v2.1'), ('license_file', 'Readme.license'), ('summary', 'Stuttgart Neural Network Simulator (SNNS)')])), ('test', CommentedMap([('commands', ['analyze -i 2>&1 | grep usage', 'batchman -h 2>&1 | grep SNNS', 'convert2snns -i 2>&1 | grep Network', 'ff_bignet 2>&1 | grep usage', 'linknets 2>&1 | grep usage', 'mkhead 2>&1 | grep usage', 'mkout 2>&1 | grep usage', 'mkpat 2>&1 | grep usage', 'snns2c 2>&1 | grep usage', 'td_bignet 2>&1 | grep usage'])]))])), 'snoscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snoscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://lowelab.ucsc.edu/software/snoscan-0.9.tar.gz'), ('md5', '30719dc179f00c892d50ce070336d0a1'), ('patches', ['patches/sort-snos.patch', 'patches/sqio.c.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'zlib', 'perl'])])), ('test', CommentedMap([('commands', ['snoscan -h > /dev/null', 'sort-snos 2>&1 | grep \"Usage\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://lowelab.ucsc.edu/snoscan'), ('summary', 'Search for C/D box methylation guide snoRNA genes in a genomic sequence'), ('licence', 'GPL')]))])), 'snp-pipeline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snp-pipeline/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snp-pipeline'), ('version', '0.7.0')])), ('source', CommentedMap([('fn', 'snp-pipeline-0.7.0.tar.gz'), ('url', 'https://pypi.python.org/packages/62/6b/5c7c7d1ad8239f2dd051a91e8bc55cd98418df5719d0fd4605f863f71d9a/snp-pipeline-0.7.0.tar.gz'), ('md5', '8f5971ca4729364fd22e26e712be862d')])), ('build', CommentedMap([('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'pyvcf >=0.6.7', 'setuptools', 'psutil', 'biopython']), ('run', ['python', 'pyvcf >=0.6.7', 'setuptools', 'psutil', 'biopython', 'testfixtures'])])), ('test', CommentedMap([('imports', ['snppipeline']), ('requires', ['testfixtures'])])), ('about', CommentedMap([('home', 'https://github.com/CFSAN-Biostatistics/snp-pipeline'), ('license', 'BSD License'), ('summary', 'Script and functions for SNP matrix construction')]))])), 'snp-sites': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snp-sites/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/sanger-pathogens/snp-sites/archive/<{ version }}.tar.gz'), ('sha256', '73210a0c1aedee54546dadd0d7e495131c6d01451fc917d98ea122d9786b838f'), ('patches', ['configure.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'automake', 'autoconf', 'libtool', 'zlib']), ('run', ['libgcc [not osx]', 'zlib'])])), ('test', CommentedMap([('commands', ['snp-sites -V'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/snp-sites'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Finds SNP sites from a multi-FASTA alignment file.')]))])), 'snpeff': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpeff/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/'), ('license', 'LGPLv3'), ('summary', 'Genetic variant annotation and effect prediction toolbox')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['snpEff -version'])]))])), 'snpsift': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpsift/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/SnpSift.html'), ('license', 'LGPLv3'), ('summary', 'Toolbox that allows you to filter and manipulate annotated files')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['SnpSift 2>&1 > /dev/null | grep \"SnpSift version 4.2\"'])]))])), 'snvphyl-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snvphyl-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snvphyl-tools'), ('version', '1.8.1')])), ('source', CommentedMap([('fn', '1.8.1.tar.gz'), ('url', 'https://github.com/phac-nml/snvphyl-tools/archive/1.8.1.tar.gz'), ('md5', '30f7c958a295e40df878658020e4cceb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-text-diff', 'perl-test-exception', 'perl-template-toolkit', 'perl-parallel-forkmanager', 'perl-hash-merge', 'perl-text-csv', 'perl-math-round', 'perl-list-moreutils', 'perl-string-util', 'perl-bioperl', 'vcftools', 'samtools', 'bcftools-snvphyl-plugin']), ('run', ['perl', 'perl-parallel-forkmanager', 'perl-hash-merge', 'perl-text-csv', 'perl-math-round', 'perl-list-moreutils', 'perl-string-util', 'perl-bioperl', 'vcftools', 'samtools', 'bcftools-snvphyl-plugin', 'mummer'])])), ('test', CommentedMap([('commands', ['vcf2snv_alignment.pl -h', 'consolidate_vcfs.pl -h'])])), ('about', CommentedMap([('home', 'https://github.com/phac-nml/snvphyl-tools'), ('license', 'apache_2.0'), ('license_file', 'LICENSE'), ('summary', 'The SNVPhyl (Single Nucleotide Variant PHYLogenomics) pipeline is a pipeline for identifying Single Nucleotide Variants (SNV) within a collection of microbial genomes and constructing a phylogenetic tree')]))])), 'soapdenovo-trans': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapdenovo-trans/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'soapdenovo-trans'), ('version', 1.04)])), ('source', CommentedMap([('fn', 'SOAPdenovo-Trans-src-v1.04.zip'), ('url', 'https://downloads.sourceforge.net/project/soapdenovotrans/SOAPdenovo-Trans/src/v1.04/SOAPdenovo-Trans-src-v1.04.zip'), ('md5', 'be5fc908051bffd95a9499a497ccd3fa'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['SOAPdenovo-Trans-31mer', 'SOAPdenovo-Trans-127mer'])])), ('about', CommentedMap([('home', 'http://soap.genomics.org.cn/SOAPdenovo-Trans.html'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts.')]))])), 'soapdenovo2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapdenovo2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapdenovo.html'), ('license', 'GPL'), ('summary', 'SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for the human-sized genomes.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'soapdenovo2'), ('version', '2.40')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'samtools ==0.1.19']), ('run', ['libgcc', 'zlib', 'samtools ==0.1.19'])])), ('source', CommentedMap([('fn', 'soapdenovo2-t240.tgz'), ('url', 'https://sourceforge.net/projects/soapdenovo2/files/SOAPdenovo2/src/r240/SOAPdenovo2-src-r240-mac.tgz/download'), ('patches', ['makefile.patch', 'sparsePregraph.main.cpp.patch', 'soapdenovo2-sources.patch'])])), ('test', CommentedMap([('commands', ['SOAPdenovo-127mer --help', 'SOAPdenovo-63mer --help'])]))])), 'soapdenovo2-errorcorrection': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapdenovo2-errorcorrection/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapdenovo.html'), ('license', 'GNU'), ('summary', 'Error correction for soapdenovo2.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'soapdenovo2-errorcorrection'), ('version', '2.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'errorcorrection.tar.gz'), ('url', 'http://sourceforge.net/projects/soapdenovo2/files/ErrorCorrection/ErrorCorrection.tgz/download')]))])), 'soapdenovo2-gapcloser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapdenovo2-gapcloser/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapdenovo.html'), ('license', 'GNU'), ('summary', 'a tool named GapCloser for SOAPdenovo.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'soapdenovo2-gapcloser'), ('version', '1.12')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'gapcloser.tar.gz'), ('url', 'https://sourceforge.net/projects/soapdenovo2/files/GapCloser/src/r6/GapCloser-src-v1.12-r6.tgz/download')])), ('test', CommentedMap([('commands', [\"GapCloser 2>&1 | grep 'input scaffold file name'\"])]))])), 'soapdenovo2-prepare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapdenovo2-prepare/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapdenovo.html'), ('license', 'GPL'), ('summary', 'SoapDenovo2 Data prepare module using assembled contig to do scaffold assembly')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'soapdenovo2-prepare'), ('version', '2.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'soapdenovo2-prepare.tar.gz'), ('url', 'https://sourceforge.net/projects/soapdenovo2/files/Prepare/Prepare-src-v2.0.tgz/download'), ('patches', ['Makefile.osx.patch'])])), ('test', CommentedMap([('commands', ['finalFusion'])]))])), 'soapec': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapec/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapdenovo.html'), ('license', 'GNU'), ('summary', 'a correction tool for SOAPdenovo')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'soapec'), ('version', '2.03')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'soapec_v2.03.tar.gz'), ('url', 'https://sourceforge.net/projects/soapdenovo2/files/ErrorCorrection/SOAPec_src_v2.03.tgz/download')])), ('test', CommentedMap([('commands', ['Corrector_AR -h'])]))])), 'soapsplice': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'soapsplice'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'SOAPsplice-v<{ version }}.tar.gz'), ('url', 'http://soap.genomics.org.cn/down/SOAPsplice-v<{ version }}.tar.gz'), ('md5', 'df038fd6657885b77932111652e9203d')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['{ soapsplice || true; } | grep SOAPsplice'])])), ('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapsplice.html'), ('summary', 'We have developed a tool SOAPsplice for genome-wide ab initio detection of splice junction sites from RNA-Seq, a method using new generation sequencing technologies to sequence the messenger RNA.'), ('license', 'freely available')]))])), 'socksipy-branch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/socksipy-branch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'socksipy-branch'), ('version', '1.01')])), ('source', CommentedMap([('fn', 'SocksiPy-branch-1.01.tar.gz'), ('url', 'https://pypi.python.org/packages/69/ed/3659a7e2cff38c3156cf919f8f6ee63360147ba97460c0a8c130f8b781e0/SocksiPy-branch-1.01.tar.gz'), ('md5', 'a59e29647320211202ce79b0cf873106')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('about', CommentedMap([('home', 'http://socksipy.sourceforge.net/'), ('license', 'BSD'), ('summary', 'A Python SOCKS module')]))])), 'somatic-sniper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/somatic-sniper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'somatic-sniper'), ('version', '1.0.5.0')])), ('source', CommentedMap([('fn', 'v1.0.5.0.tar.gz'), ('url', 'https://github.com/genome/somatic-sniper/archive/v1.0.5.0.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['bam-somaticsniper -v'])])), ('about', CommentedMap([('home', 'https://github.com/genome/somatic-sniper/'), ('license', 'MIT'), ('summary', 'A tool to call somatic single nucleotide variants.')]))])), 'sonlib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sonlib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sonlib'), ('version', '1.1.0')])), ('source', CommentedMap([('fn', 'sonLib-1.1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/c5/63/5c597eb4895ad59e6b57050de0dbc579b400e9e2bfc7a698d6b19d91caaf/sonLib-1.1.0.tar.gz'), ('md5', 'b6051a1c664ad51f8620d80a3652aa05')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['sonLib'])])), ('about', CommentedMap([('home', 'https://github.com/benedictpaten/sonLib'), ('license', 'MIT'), ('summary', 'Small general purpose library for C and Python with focus on bioinformatics.')]))])), 'sourmash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sourmash/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/dib-lab/sourmash/archive/ea1310da4695ba29ff6a74e9855c0c35a9c6b6c1.tar.gz'), ('sha256', '<{ hash }}'), ('patches', ['sourmash.patch'])])), ('build', CommentedMap([('entry_points', ['sourmash = sourmash_lib.__main__:main']), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc', 'llvm', 'cython >=0.25.2', 'screed >=0.9', 'pyyaml >=3.11', 'ijson', 'numpy', 'matplotlib', 'scipy', 'khmer >=2.1']), ('run', ['python', 'libgcc', 'cython >=0.25.2', 'screed >=0.9', 'pyyaml >=3.11', 'ijson', 'numpy', 'matplotlib', 'scipy', 'khmer >=2.1'])])), ('test', CommentedMap([('imports', ['sourmash_lib']), ('commands', ['sourmash --help'])])), ('about', CommentedMap([('home', 'https://github.com/dib-lab/sourmash'), ('license', 'BSD License'), ('summary', 'Compute and compare MinHash signatures for DNA data sets.'), ('license_family', 'BSD')]))])), 'spades': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spades/3.9.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spades'), ('version', '3.9.0')])), ('source', CommentedMap([('fn', 'SPAdes-3.9.0-Darwin.tar.gz'), ('url', 'http://spades.bioinf.spbau.ru/release3.9.0/SPAdes-3.9.0-Darwin.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['spades.py --test'])])), ('about', CommentedMap([('home', 'http://bioinf.spbau.ru/en/spades'), ('license', 'GPLv2'), ('summary', 'SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.')]))])), 'spanki': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spanki/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spanki'), ('version', '0.5.1')])), ('source', CommentedMap([('git_url', 'https://github.com/dsturg/Spanki.git'), ('git_rev', 'v0.5.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pyfasta', 'numpy', 'biopython', 'statsmodels', 'fisher', 'pysam']), ('run', ['python', 'pyfasta', 'numpy', 'biopython', 'statsmodels', 'fisher', 'pysam', 'samtools', 'cufflinks'])])), ('test', CommentedMap([('commands', ['spankijunc -h > /dev/null 2>&1', 'merge_jtabs > /dev/null 2>&1 || [[ \"$?\" == 1 ]]', 'make_curated_jtab -h > /dev/null', 'junccomp -h > /dev/null 2>&1', 'spankisplice -h > /dev/null 2>&1', 'splicecomp -h > /dev/null 2>&1', 'spankisim_models -h > /dev/null 2>&1', 'spankisim_transcripts -h > /dev/null 2>&1', 'quickjunc -h > /dev/null 2>&1', 'annotate_junctions -h > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'http://www.cbcb.umd.edu/software/spanki/'), ('license', 'GPLv3'), ('summary', 'Spanki is a set of tools to facilitate analysis of alternative splicing from RNA-Seq data. Spanki compiles quantitative and qualitative information about junction alignments from input BAM files, and analyzes junction-level splicing along with pairwise-defined splicing events. A simulator is also included to evaluate junction detection performance.')]))])), 'sparc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sparc/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('script', 'mkdir -p $PREFIX/bin; g++ -O3 -o Sparc *.cpp; mv Sparc $PREFIX/bin')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/yechengxi/Sparc/archive/ae89503a1613e26432e42febc7f35f6f71152b38.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['Sparc -h | grep \"Chengxi Ye cxy@umd.edu\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/yechengxi/Sparc'), ('summary', 'A sparsity-based consensus algorithm for long erroneous sequencing reads.')]))])), 'sparse-neighbors-search': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sparse-neighbors-search/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sparse-neighbors-search'), ('version', 0.3)])), ('source', CommentedMap([('url', 'https://github.com/joachimwolff/minHashNearestNeighbors/archive/0.3.tar.gz'), ('md5', 'bed04589d2c05c916a7b7ebdf31e71f2')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc', 'scipy', 'numpy', 'scikit-learn', 'cython']), ('run', ['python', 'scipy', 'numpy', 'scikit-learn', 'cython', 'libgcc'])])), ('test', CommentedMap([('imports', ['sparse_neighbors_search'])])), ('about', CommentedMap([('maintainers', ['Joachim Wolff']), ('home', 'https://github.com/joachimwolff/minHashNearestNeighbors'), ('license', 'MIT'), ('license_file', 'LICENSE')]))])), 'sparseassembler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sparseassembler/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('script', 'mkdir -p $PREFIX/bin; g++ -O3 -o SparseAssembler *.cpp; mv SparseAssembler $PREFIX/bin')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/yechengxi/SparseAssembler/archive/7ce06016160de6dc0e245530689cefed8d5db19f.tar.gz'), ('md5', 'e9c6b25a8f9983badff01094615596f0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['SparseAssembler | grep \"Error! Genome size not given.\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/yechengxi/SparseAssembler'), ('summary', 'A sparse k-mer graph based, memory-efficient genome assembler')]))])), 'spectacle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spectacle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spectacle'), ('version', 1.4)])), ('source', CommentedMap([('git_url', 'https://github.com/jiminsong/Spectacle.git'), ('git_rev', 'v1.4')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['java-jdk', 'scipy', 'numpy'])])), ('about', CommentedMap([('home', 'https://github.com/jiminsong/Spectacle'), ('license', 'GPLv3'), ('summary', 'This software implements a spectral learning algorithm for hidden Markov models for epigenomic data. Please see our paper for further details: Song, J and Chen, K. C. Spectacle: fast chromatin state annotation using spectral learning. Genome Biology, 16:33, 2015. http://genomebiology.com/2015/16/1/33')])), ('extra', CommentedMap([('notes', 'The Spectacle github repo weighs in at around 500MB, a large portion of which is data files. These have been removed from the conda recipe, but a script (download_spectacle_data.sh) has been included here which will download those files from github.  In addition, a wrapper script `Spectacle.sh` has been included in this recipe and should be used when calling the program.')]))])), 'spectra-cluster-cli': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spectra-cluster-cli/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/spectra-cluster/spectra-cluster-cli'), ('license', 'Apache License, Version 2.0'), ('summary', 'This is a stand-alone implementation of the new updated PRIDE Cluster algorithm. It is based on the spectra-cluster API and uses a highly similar logic as the Hadoop implementation spectra-cluster-hadoop used to build the PRIDE Cluster resource.')])), ('package', CommentedMap([('name', 'spectra-cluster-cli'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'https://github.com/spectra-cluster/spectra-cluster-cli/releases/download/spectra-cluster-cli-1.0.1/spectra-cluster-cli-1.0.1-bin.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', [\"spectra-cluster-cli 2>&1 | grep 'Spectra Cluster'\"])])), ('extra', CommentedMap([('notes', 'spectra-cluster-cli is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"spectra-cluster-cli -Xms512m -Xmx1g\"\\n')]))])), 'spectral_hk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spectral_hk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spectral_hk'), ('version', 0.1)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'spectral_hk.tar.bz2'), ('url', 'https://bitbucket.org/ncgc/spectral_hk/get/cb39dca4c40d.tar.bz2'), ('sha256', 'c4de3914668aa7db97ab18b89727b858590f5e04eec69f1d9b00d16d82100879')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['spectral_hk --help 2>&1 | grep spectral_hk'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/ncgc/spectral_hk'), ('license', 'PUBLIC DOMAIN'), ('summary', 'NCGC Spectral HashKey')]))])), 'spectrassembler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spectrassembler/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/antrec/spectrassembler/archive/3ef76977896c65af4059d164853295ffecacc687.tar.gz'), ('md5', '95ae6018417128d99b622f37f7e6b50a')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'scipy', 'biopython']), ('run', ['python', 'numpy', 'scipy', 'biopython', 'minimap', 'bwa', 'poa'])])), ('test', CommentedMap([('commands', ['spectral_layout_from_minimap.py -h', 'gen_cons_from_poa.py  -h'])])), ('about', CommentedMap([('home', 'https://github.com/antrec/spectrassembler'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Tool (experimental) to compute layout from overlaps with spectral algorithm')]))])), 'sphinxcontrib-programoutput': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sphinxcontrib-programoutput/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'sphinxcontrib-programoutput'), ('version', '0.8')])), ('source', CommentedMap([('fn', 'sphinxcontrib-programoutput-0.8.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/sphinxcontrib-programoutput/sphinxcontrib-programoutput-0.8.tar.gz'), ('md5', 'bb0be17ef13f268378b2af51ff413c58')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sphinx >=1.1']), ('run', ['python', 'sphinx >=1.1'])])), ('test', CommentedMap([('imports', ['sphinxcontrib'])])), ('about', CommentedMap([('home', 'http://sphinxcontrib-programoutput.readthedocs.org/'), ('license', 'BSD License'), ('summary', 'Sphinx extension to include program output')]))])), 'spine': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spine/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/egonozer/Spine/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}'), ('patches', ['spine.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-file-which']), ('run', ['perl', 'perl-file-which', 'mummer >=3.22'])])), ('test', CommentedMap([('commands', [\"spine.pl  2>&1 | grep 'wrapper'\", 'nucmer_backbone.pl -V', 'nucmer_multi.pl -V'])])), ('about', CommentedMap([('home', 'https://github.com/egonozer/Spine'), ('license', 'GPL-3.0'), ('summary', 'Identification of conserved nucleotide core genome of bacteria and other small genome organisms')]))])), 'spingo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spingo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spingo'), ('version', '1.3')])), ('source', CommentedMap([('url', 'https://github.com/GuyAllard/SPINGO/archive/v1.3.tar.gz'), ('md5', 'e837f57fcb554486a83ab9cc016a5de9')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*'])])), ('test', CommentedMap([('commands', ['spingo --version'])])), ('about', CommentedMap([('home', 'https://github.com/GuyAllard/SPINGO'), ('license', 'https://github.com/GuyAllard/SPINGO/blob/master/LICENSE'), ('summary', 'Species level IdentificatioN of metaGenOmic amplicons')]))])), 'splitmem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/splitmem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'splitmem'), ('version', '1.0')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'splitmem.1.0.tgz'), ('url', 'http://downloads.sourceforge.net/project/splitmem/splitmem.1.0.tgz'), ('md5', '564de17993d4a9e21cf88ac9b340a809')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['splitMEM -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/splitmem/'), ('license', 'Apache License V2.0'), ('summary', 'Graphical pan-genome analysis with suffix skips')]))])), 'sprinkles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sprinkles/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'sprinkles'), ('version', '0.4.6')])), ('source', CommentedMap([('fn', 'sprinkles-0.4.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/sprinkles/sprinkles-0.4.6.tar.gz'), ('md5', 'a653505d58e316b871cafe7df4c79048')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('about', CommentedMap([('home', 'http://an9.org/w/SprinklesPy'), ('license', 'MIT License'), ('summary', 'Plugins! Easy!')]))])), 'sqlalchemy-datatables': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sqlalchemy-datatables/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sqlalchemy-datatables'), ('version', '0.3.0')])), ('source', CommentedMap([('fn', 'sqlalchemy-datatables-0.3.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/sqlalchemy-datatables/sqlalchemy-datatables-0.3.0.tar.gz'), ('md5', 'c79ba42e7cf6298ca168e91bdc2bfadc')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sqlalchemy']), ('run', ['python', 'sqlalchemy'])])), ('test', CommentedMap([('imports', ['datatables'])])), ('about', CommentedMap([('home', 'https://github.com/pegase745/sqlalchemy-datatables'), ('license', 'MIT License'), ('summary', 'SQLAlchemy integration of jQuery DataTables')]))])), 'sqlalchemy-utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sqlalchemy-utils/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sqlalchemy-utils'), ('version', '0.31.6')])), ('source', CommentedMap([('fn', 'SQLAlchemy-Utils-0.31.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/SQLAlchemy-Utils/SQLAlchemy-Utils-0.31.6.tar.gz'), ('md5', '8b939e5eec1baae46e88e6b145c1c1f5')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'sqlalchemy >=1.0']), ('run', ['python', 'six', 'sqlalchemy >=1.0'])])), ('test', CommentedMap([('imports', ['sqlalchemy_utils', 'sqlalchemy_utils.functions', 'sqlalchemy_utils.primitives', 'sqlalchemy_utils.relationships', 'sqlalchemy_utils.types'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/sqlalchemy-utils'), ('license', 'BSD License'), ('summary', 'Various utility functions for SQLAlchemy.')]))])), 'sqlitebrowser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sqlitebrowser/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sqlitebrowser'), ('version', '3.8.0')])), ('source', CommentedMap([('fn', 'sqlitebrowser-3.8.0.tar.gz'), ('url', 'https://codeload.github.com/sqlitebrowser/sqlitebrowser/tar.gz/v3.8.0')])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'sqlite', 'qt']), ('run', ['libgcc', 'sqlite', 'qt'])])), ('about', CommentedMap([('home', 'http://sqlitebrowser.org/'), ('license', 'GPLv3'), ('summary', 'DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.')]))])), 'sqt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sqt/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/marcelm/sqt'), ('license', 'MIT'), ('summary', 'Command-line tools for the analysis of high-throughput sequencing data')])), ('package', CommentedMap([('name', 'sqt'), ('version', '0.8.0')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/09/85/ec30aba5d8bcc54fadcb65ab3387a5ba7921e1d22c2f71a498187ba096d7/sqt-0.8.0.tar.gz'), ('md5', '5a6400b6cb5a331589f831ba0b6020f0')])), ('requirements', CommentedMap([('run', ['python', 'cutadapt', 'xopen', 'pysam', 'matplotlib', 'seaborn', 'libgcc']), ('build', ['python', 'cutadapt', 'xopen', 'pysam', 'matplotlib', 'seaborn', 'gcc', 'llvm'])])), ('test', CommentedMap([('imports', ['sqt']), ('commands', ['sqt --version'])])), ('build', CommentedMap([('skip', True), ('script', 'python3 setup.py install'), ('number', 0)]))])), 'squeakr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/squeakr/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/splatlab/squeakr/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'bzip2', 'zlib <{CONDA_ZLIB}}*', 'boost  <{CONDA_BOOST}}*', 'openssl']), ('run', ['libgcc', 'bzip2', 'zlib <{CONDA_ZLIB}}*', 'boost  <{CONDA_BOOST}}*', 'openssl'])])), ('test', CommentedMap([('commands', ['squeakr-count -h'])])), ('about', CommentedMap([('home', 'https://github.com/splatlab/squeakr'), ('license', 'BSD'), ('summary', 'An Exact and Approximate k-mmer Counting System')]))])), 'sra-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sra-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sra-tools'), ('version', '<{ SRA_TOOLS_VERSION }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'sra-tools-<{ SRA_TOOLS_VERSION }}.tar.gz'), ('url', 'https://github.com/ncbi/sra-tools/archive/<{ SRA_TOOLS_VERSION }}-3.tar.gz'), ('sha256', '<{ SRA_TOOLS_SHA256 }}')])), ('extra', CommentedMap([('notes', 'This package includes ncbi-vdb and ngs-sdk. After installation, you should run the configuration tool: ./vdb-config -i. This tool will setup your download/cache area for downloaded files and references.')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'libiconv', 'libxml2']), ('run', None)])), ('about', CommentedMap([('home', 'https://github.com/ncbi/sra-tools'), ('license', 'Public Domain'), ('summary', 'The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.')])), ('test', CommentedMap([('commands', ['fastq-dump --help > /dev/null', 'prefetch --help > /dev/null', 'sam-dump --help > /dev/null', 'sra-pileup --help > /dev/null', 'vdb-config --help > /dev/null', 'vdb-decrypt --help > /dev/null', \"fastq-dump -N 1 -X 1 SRR000001 && grep '@SRR000001.1' SRR000001.fastq\"])]))])), 'srprism': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/srprism/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'srprism'), ('version', '2.4.24')])), ('about', CommentedMap([('home', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/srprism/'), ('summary', 'SRPRISM - Short Read Alignment Tool')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('source', CommentedMap([('fn', 'srprism.tar.gz'), ('md5', '09b43fcf8f2db2376bcb2f27ac3a1f00'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/src/srprism.tar.gz')])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['srprism help'])]))])), 'srst2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/srst2/0.1.4.6/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'srst2'), ('version', '0.1.4.6')])), ('source', CommentedMap([('fn', 'v0.1.6.tar.gz'), ('url', 'https://github.com/katholt/srst2/archive/v0.1.6.tar.gz'), ('sha256', '24cbd03933d4cc14bbfa7f8a073c9b0caafb13d55afb3bfa25f8b093fc5bc615'), ('patches', ['use_setuptools.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.7.1', 'scipy >=0.12.0']), ('run', ['python', 'numpy >=1.7.1', 'scipy >=0.12.0', 'bowtie2 <=2.2.4', 'samtools ==0.1.18'])])), ('test', CommentedMap([('commands', [\"srst2 -h 2>&1 | grep 'SRST2 - Short Read Sequence Typer' > /dev/null\", \"getmlst.py -h 2>&1 | grep 'usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/katholt/srst2'), ('license', 'BSD'), ('summary', 'Short Read Sequence Typing for Bacterial Pathogens')]))])), 'stacks': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stacks/1.44/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.44')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.44.tar.gz'), ('md5', '10e1c2cb3ed10bebcb035619d94ae7d3'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.44.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))])), 'stacks_summary': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stacks_summary/1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stacks_summary'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'stacks_summary-1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/fd/4d/b66c0171669e42a92b6961cf769fcd7186b60e126868fb7c216d9d9a07ec/stacks_summary-1.0.tar.gz'), ('md5', '47393995a2ce15fe2d48eedebfde011c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['stacks']), ('commands', ['stacks_summary.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/mariabernard/galaxy_wrappers'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Stacks reports generator'), ('license_family', 'Public-Domain')]))])), 'stamp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stamp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stamp'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAMP-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/stamp/STAMP-<{ version }}.tar.gz'), ('md5', '81a5e0df4094d65007c3c27bf1796ec8'), ('patches', ['stamp.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2']), ('run', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2', 'pyqt 4.*'])])), ('test', CommentedMap([('imports', ['stamp', 'stamp.GUI', 'stamp.metagenomics', 'stamp.metagenomics.fileIO', 'stamp.metagenomics.simulations', 'stamp.metagenomics.stats', 'stamp.metagenomics.stats.CI', 'stamp.metagenomics.stats.distributions', 'stamp.metagenomics.stats.empiricalTests', 'stamp.metagenomics.stats.tests', 'stamp.plugins', 'stamp.plugins.common', 'stamp.plugins.common.multipleComparisonCorrections', 'stamp.plugins.groups', 'stamp.plugins.groups.effectSizeFilters', 'stamp.plugins.groups.plots', 'stamp.plugins.groups.plots.configGUI', 'stamp.plugins.groups.statisticalTests', 'stamp.plugins.multiGroups', 'stamp.plugins.multiGroups.effectSizeFilters', 'stamp.plugins.multiGroups.plots', 'stamp.plugins.multiGroups.plots.configGUI', 'stamp.plugins.multiGroups.postHoc', 'stamp.plugins.multiGroups.statisticalTests', 'stamp.plugins.samples', 'stamp.plugins.samples.confidenceIntervalMethods', 'stamp.plugins.samples.effectSizeFilters', 'stamp.plugins.samples.plots', 'stamp.plugins.samples.plots.configGUI', 'stamp.plugins.samples.statisticalTests']), ('commands', ['checkHierarchy.py -h'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/stamp/'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE.txt'), ('summary', 'A graphical software package for analyzing taxonomic and functional profiles.')]))])), 'star': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/alexdobin/STAR/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://github.com/alexdobin/STAR'), ('license', 'GPLv3'), ('summary', 'An RNA-seq read aligner.')])), ('test', CommentedMap([('commands', ['STAR --version'])]))])), 'star-fusion': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star-fusion/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star-fusion'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAR-Fusion_v<{ version }}.FULL.tar.gz'), ('url', 'https://github.com/STAR-Fusion/STAR-Fusion/releases/download/v<{ version }}/STAR-Fusion_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['blast', 'perl', 'perl-set-intervaltree', 'perl-db-file', 'perl-uri', 'star', 'gmap', 'bowtie'])])), ('test', CommentedMap([('commands', ['STAR-Fusion --help 2>&1 | grep Optional > /dev/null', 'prep_genome_lib.pl -h 2>&1 | grep \"STAR-Fusion\" > /dev/null', 'blast_and_promiscuity_filter.pl -h 2>&1 | grep Optional > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/STAR-Fusion/STAR-Fusion'), ('license', 'BSD-3-Clause'), ('summary', 'A fusion gene caller for STAR. The FusionFilter tools to create needed index data structures are not included.')]))])), 'starseqr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/starseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'starseqr'), ('version', '0.5.0')])), ('source', CommentedMap([('fn', 'starseqr-0.5.0.tar.gz'), ('url', 'https://pypi.python.org/packages/b1/88/af89ea23d199ec6a614226c1d53dd1003110c67efa3f4f35952e5dc13ec4/starseqr-0.5.0.tar.gz'), ('md5', '20dfef835d705703476510c9b1be9c7e')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'six', 'networkx', 'gcc', 'llvm', 'gffread', 'numpy', 'pandas >=0.18.1', 'pysam >=0.9.1.4', 'primer3-py', 'intervaltree_bio']), ('run', ['python', 'cython', 'six', 'networkx', 'gffread', 'pandas >=0.18.1', 'numpy', 'pysam >=0.9.1.4', 'primer3-py', 'intervaltree_bio'])])), ('test', CommentedMap([('imports', ['ssw', 'starseqr_utils', 'starseqr_utils.tests']), ('requires', ['nose']), ('commands', ['starseqr.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/ExpressionAnalysis/STAR-SEQR'), ('license', '../../LICENSE'), ('summary', 'RNA Fusion Detection and Quantification')]))])), 'stellar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stellar/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'stellar'), ('version', '1.4.9')])), ('source', CommentedMap([('fn', 'stellar-1.4.9-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/stellar/stellar-1.4.9-Linux-x86_64.tar.xz'), ('md5', '44eb71b174b516bac0bc884467d8bd8f')])), ('test', CommentedMap([('commands', ['stellar --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/stellar/README'), ('license', 'LGPLv3+'), ('license_file', 'LICENSE'), ('summary', 'STELLAR is a tool for finding pairwise local alignments between long genomic or very many short sequences.')]))])), 'strelka': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/strelka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'strelka'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'strelka-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/strelka/releases/download/v<{ version }}/strelka-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '72c57c3c90dd33a0d91c32298bb6320d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configureStrelkaGermlineWorkflow.py -h', 'configureStrelkaSomaticWorkflow.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/strelka'), ('license', 'GPLv3'), ('summary', 'Strelka calls somatic and germline small variants from mapped sequencing reads')]))])), 'stride': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stride/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('url', 'https://github.com/ythuang0522/StriDe/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'google-sparsehash', 'autoconf', 'automake', 'm4', 'perl', 'zlib']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['stride 2>&1 | grep \"stride\"'])])), ('about', CommentedMap([('home', 'https://github.com/ythuang0522/StriDe'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'The StriDe Assembler integrates string and de Bruijn graph by decomposing reads within error-prone regions, while extending paire-end read into long reads for assembly through repetitive regions.')]))])), 'stringmlst': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stringmlst/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jordanlab/stringMLST/archive/<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfaidx', 'lxml']), ('run', ['python', 'bwa', 'bedtools', 'samtools >=1.0', 'pyfaidx', 'lxml'])])), ('test', CommentedMap([('commands', ['stringMLST.py  -v'])])), ('about', CommentedMap([('home', 'https://github.com/jordanlab/stringMLST'), ('license', 'CC BY-NC-SA 4.0'), ('license_file', 'LICENSE'), ('summary', 'Fast k-mer based tool for multi locus sequence typing (MLST) directly from genome sequencing reads')]))])), 'stringtie': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stringtie/1.3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.3.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'stringtie-1.3.0.tar.gz'), ('sha256', '3a21c662904cdab0c749d5c3f5ef988e84fa036857ce2ca35782b3c19469c652'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.0.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))])), 'structure': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/structure/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'structure_kernel_source.tar.gz'), ('md5', '4e0591678cdbfe79347d272b5dceeda1'), ('url', 'https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v<{ version }}/structure_kernel_source.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['structure | grep \"Version <{ version }}\"'])])), ('about', CommentedMap([('home', 'https://web.stanford.edu/group/pritchardlab/structure.html'), ('license', 'MIT'), ('summary', 'The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including SNPS, microsatellites, RFLPs and AFLPs.')]))])), 'subread': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/subread/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://subread.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'High-performance read alignment, quantification, and mutation discovery')])), ('package', CommentedMap([('name', 'subread'), ('version', '1.5.3')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['featureCounts -v 2>&1 > /dev/null'])])), ('source', CommentedMap([('fn', 'subread-1.5.3-MacOSX-x86_64.tar.gz'), ('url', 'https://sourceforge.net/projects/subread/files/subread-1.5.3/subread-1.5.3-MacOSX-x86_64.tar.gz'), ('md5', 'd1d9d05226484a1df184f7ef6d541ffa')]))])), 'suma_package': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/suma_package/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'suma_package'), ('version', '1.0.00')])), ('source', CommentedMap([('fn', 'suma_package_V_1.0.00.tar.gz'), ('url', 'ftp://ftp.microbio.me/pub/QIIME-v1.9.0-dependencies/suma_package_V_1.0.00.tar.gz'), ('md5', 'e528493d05225a605adacd0961b8f50c'), ('patches', ['ldflags.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['sumaclust -h > /dev/null', 'sumatra -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://metabarcoding.org/sumatra'), ('license', 'CeCILL FSLA'), ('license_file', 'Licence_CeCILL_V2-en.txt'), ('summary', 'Fast and exact comparison of sequences')]))])), 'sure': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sure/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sure'), ('version', '1.2.24')])), ('source', CommentedMap([('fn', 'sure-1.2.24.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/sure/sure-1.2.24.tar.gz'), ('md5', 'a396cc3c71d551bfdb9bc45363ca05da')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'mock', 'six']), ('run', ['python', 'mock', 'six'])])), ('test', CommentedMap([('imports', ['sure']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://github.com/gabrielfalcao/sure'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'utility belt for automated testing in python for python')]))])), 'survivor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/survivor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'survivor'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'survivor-<{ version }}.tar.gz'), ('url', 'https://github.com/fritzsedlazeck/SURVIVOR/archive/v<{ version }}.tar.gz'), ('md5', '1387f5f13e0c3ad83b53c52d3b380d4a')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://github.com/fritzsedlazeck/SURVIVOR'), ('license', 'MIT'), ('summary', 'Toolset for SV simulation, comparison and filtering')])), ('test', CommentedMap([('commands', ['SURVIVOR'])]))])), 'svgutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/svgutils/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'svgutils'), ('version', '0.1.0')])), ('source', CommentedMap([('git_url', 'https://github.com/btel/svg_utils.git'), ('git_rev', '145502d680744274dea66ae0d19da4e4c62d77e4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'lxml']), ('run', ['python', 'lxml'])])), ('test', CommentedMap([('imports', ['svgutils'])])), ('about', CommentedMap([('home', 'http://neuroscience.telenczuk.pl'), ('license', 'MIT License'), ('summary', 'Python SVG editor that allows to automatically create publication ready composite SVG figures.')]))])), 'svgwrite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/svgwrite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'svgwrite'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'svgwrite-1.1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/svgwrite/svgwrite-1.1.6.tar.gz'), ('md5', '0d54ccf5584dd1f98fe22b7ac172bef1')])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyparsing >=2.0.1']), ('run', ['python', 'pyparsing >=2.0.1'])])), ('test', CommentedMap([('imports', ['svgwrite'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/mozman/svgwrite'), ('license', 'MIT License'), ('summary', 'A Python library to create SVG drawings.')]))])), 'svtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/svtools/0.3.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'svtools'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'svtools-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/59/5c/d3ec7b8345e4bf53628b9ad047529d134ab0d3c08e18eed4d741c7340a92/svtools-0.3.1.tar.gz'), ('md5', 'bf313bfdd4f46b3136e449cad7f83770')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['svtools=svtools.cli:main']), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools']), ('run', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas'])])), ('test', CommentedMap([('imports', ['svtools', 'svtools.vcf']), ('commands', ['svtools --help', 'create_coordinates --help'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtools'), ('license', 'MIT License'), ('summary', 'Tools for processing and analyzing structural variants')]))])), 'svtyper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/svtyper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'svtyper'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'svtyper-<{ version }}.tar.gz'), ('url', 'https://github.com/hall-lab/svtyper/archive/v<{ version }}.tar.gz'), ('md5', 'cd1b7a73a6cbf002c6398caef5bfa79a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pysam']), ('run', ['python', 'pysam'])])), ('test', CommentedMap([('commands', ['svtyper -h'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtyper'), ('license', 'MIT'), ('summary', 'Bayesian genotyper for structural variants')]))])), 'svviz': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/svviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'svviz'), ('version', '1.5.1')])), ('source', CommentedMap([('fn', 'svviz-1.5.1.tar.gz'), ('url', 'https://pypi.python.org/packages/3b/b1/3571a9cbff54a60cf3a9222c5a7c7c05e3bcca8bb1fc350fed221b8337e9/svviz-1.5.1.tar.gz'), ('md5', '27497e1db3ddd27b5cf94896b7e4596c')])), ('build', CommentedMap([('skip', True), ('entry_points', ['svviz = svviz.app:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'numpy', 'pyfaidx', 'pysam >=0.7.8', 'flask', 'joblib']), ('run', ['python', 'requests', 'numpy', 'pyfaidx', 'pysam >=0.7.8', 'flask', 'joblib'])])), ('test', CommentedMap([('imports', ['ssw', 'svviz']), ('commands', ['svviz --help'])])), ('about', CommentedMap([('home', 'https://github.com/svviz/svviz'), ('license', 'MIT License'), ('summary', 'A read visualizer for structural variants')]))])), 'swalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swalign'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'swalign-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/72/97/cb26a99ee2e1796b0ab28a955e3568607826a44999654f6880d4e00174ac/swalign-0.3.3.tar.gz'), ('md5', '64c55d09775d4875bc419a08a43dc6fa')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['swalign'])])), ('about', CommentedMap([('home', 'http://github.com/mbreese/swalign/'), ('license', 'BSD'), ('license_file', '<{ RECIPE_DIR }}/LICENSE'), ('summary', 'Smith-Waterman local aligner')]))])), 'swarm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swarm/1.2.19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swarm'), ('version', '1.2.19')])), ('source', CommentedMap([('fn', '1.2.19.tar.gz'), ('url', 'https://github.com/torognes/swarm/archive/1.2.19.tar.gz'), ('md5', '4d147b9c94b8966760d0b737499a53d2')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['swarm_breaker.py --version', 'swarm -v 2>&1  | grep Swarm'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/swarm'), ('license', 'Affero GPL'), ('summary', 'A robust and fast clustering method for amplicon-based studies.')]))])), 'sweepfinder2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sweepfinder2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sweepfinder2'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'SF2.tar.gz'), ('url', 'http://www.personal.psu.edu/mxd60/SF2.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['SweepFinder2 --help | grep usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.personal.psu.edu/mxd60/sf2.html'), ('license', 'Unknown'), ('summary', 'The BWA read mapper.')]))])), 'swga': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swga/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/eclarke/swga/archive/c91d2ff81349811369110df81219ec1036ba8129.tar.gz'), ('md5', 'c71b533fed7037be59f852af90944c1d')])), ('build', CommentedMap([('skip', True), ('entry_points', ['swga = swga.main:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfaidx', 'click', 'peewee', 'melt', 'pytest', 'argutils', 'semantic_version', 'workspace', 'gcc']), ('run', ['python', 'setuptools', 'pyfaidx', 'click', 'peewee', 'melt', 'pytest', 'argutils', 'semantic_version', 'workspace'])])), ('test', CommentedMap([('imports', ['swga']), ('commands', ['LANG=C.UTF-8 swga  2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/eclarke/swga'), ('license', 'GPLv3'), ('summary', 'Select primer sets for selective whole genome amplification (SWGA)')]))])), 'synapseclient': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/synapseclient/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'synapseclient'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'synapseclient-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/synapseclient/synapseclient-<{ version }}.tar.gz'), ('md5', '94cafb568a0c3adfe35cd3a4e826f1bb')])), ('build', CommentedMap([('skip', False), ('entry_points', ['synapse = synapseclient.__main__:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.2', 'six', 'future', 'backports.csv']), ('run', ['python', 'requests >=1.2', 'six', 'future', 'backports.csv'])])), ('test', CommentedMap([('imports', ['synapseclient']), ('commands', ['synapse --help']), ('requires', ['mock', 'nose'])])), ('about', CommentedMap([('home', 'http://synapse.sagebase.org/'), ('license', 'Apache Software License'), ('summary', 'A client for Synapse, a collaborative compute space  that allows scientists to share and analyze data together.')]))])), 't_coffee': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/t_coffee/meta.yaml', CommentedMap([('package', CommentedMap([('name', 't_coffee'), ('version', '11.0.8')])), ('source', CommentedMap([('fn', 'T-COFFEE_installer_Version_11.00.8cbe486_linux_x64.bin'), ('url', 'http://www.tcoffee.org/Packages/Stable/Latest/linux/T-COFFEE_installer_Version_11.00.8cbe486_linux_x64.bin'), ('md5', '2e375405c9ebb977adf87c145ca698bb')])), ('build', CommentedMap([('number', 2), ('skip', True), ('entry_points', ['t_coffee = t_coffee.__main__:main'])])), ('requirements', CommentedMap([('build', ['curl', 'openssl', 'python', 'zlib']), ('run', ['curl', 'openssl', 'perl-threaded', 'python', 'zlib'])])), ('test', CommentedMap([('imports', ['t_coffee']), ('commands', ['t_coffee', 't_coffee --help'])])), ('about', CommentedMap([('home', 'http://www.tcoffee.org/Projects/tcoffee/'), ('license', 'GNU'), ('summary', 'A collection of tools for Computing, Evaluating and Manipulating Multiple Alignments of DNA, RNA, Protein Sequences and Structures.')]))])), 'tabview': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tabview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tabview'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'tabview-1.4.2.tar.gz'), ('url', 'https://pypi.python.org/packages/f5/6c/75a807e740dcddbade3194cfaf05db72e378a714d2c16341b60d7044fe0f/tabview-1.4.2.tar.gz'), ('md5', 'bcc8bc045051f4529d3cad43b56b61d2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'ncurses']), ('run', ['python', 'ncurses'])])), ('test', CommentedMap([('imports', ['tabview']), ('commands', ['tabview -h > /dev/null || [[ \"$?\" == 2 ]]'])])), ('about', CommentedMap([('home', 'https://github.com/firecat53/tabview'), ('license', 'MIT License'), ('summary', 'A curses command-line CSV and list (tabular data) viewer')]))])), 'taco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/taco/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'taco'), ('version', 'v0.7.0')])), ('source', CommentedMap([('fn', 'taco-v0.7.0.Linux_x86_64.tar.gz'), ('url', 'https://github.com/tacorna/taco/archive/v0.7.0.tar.gz'), ('sha256', '3b7aab519bf72bfd74fe91d6cb15b92abd1e67370c0da7ebd49646a553909442')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['setuptools ==19.02', 'pyinstaller', 'python', 'numpy', 'cython', 'gcc', 'llvm']), ('run', ['libgcc', 'python', 'pyinstaller'])])), ('test', CommentedMap([('commands', ['taco_run -h'])])), ('about', CommentedMap([('home', 'https://github.com/tacorna/taco'), ('license', 'BSD'), ('summary', 'A tool for multi-sample transcriptome assembly from RNA-Seq')]))])), 'tadarida-c': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tadarida-c/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/ThimotheeV/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-randomforest']), ('run', ['r-base', 'r-data.table', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['tadaridaC.r --help'])])), ('about', CommentedMap([('home', 'https://github.com/YvesBas/Tadarida-C'), ('license', 'GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007'), ('license_file', 'licence.txt'), ('summary', 'Tadarida-C (Toolbox for Animal Detection on Acoustic Recordings - Classification part) for Galaxy use.')]))])), 'tadarida-d': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tadarida-d/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/ThimotheeV/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['qt 4.8.7', 'fftw 3.3.4', 'libsndfile 1.0.27', 'gcc', 'zlib']), ('run', ['qt 4.8.7', 'fftw 3.3.4', 'libsndfile 1.0.27', 'libgcc', 'zlib', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau', 'xorg-libx11'])])), ('test', CommentedMap([('commands', ['tadaridaD --help'])])), ('about', CommentedMap([('home', 'https://github.com/YvesBas/Tadarida-D'), ('license', 'GNU LESSER GENERAL PUBLIC LICENSE Version 3, 29 June 2007'), ('license_file', 'licence.txt'), ('summary', 'Tadarida-D (Toolbox for Animal Detection on Acoustic Recordings - Detection and Feature extraction part) for Galaxy use.')]))])), 'takeabreak': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/takeabreak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'takeabreak'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'TakeABreak-v1.1.2-Source.tar.gz'), ('url', 'https://github.com/GATB/TakeABreak/releases/download/v1.1.2/TakeABreak-v1.1.2-Source.tar.gz'), ('md5', '9170a6d61f51e53189822df830648fd0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['TakeABreak -help'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/takeabreak/'), ('license', 'GNU Affero General Public License'), ('summary', 'tool that can detect inversion breakpoints directly from raw NGS reads, without the need of any reference genome and without de novo assembling the genomes')]))])), 'talloc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/talloc/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://www.samba.org/ftp/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://talloc.samba.org/talloc/doc/html/index.html`'), ('license', 'LGPL-3.0'), ('summary', 'talloc is a hierarchical, reference counted memory pool system with destructors.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python <3', 'gcc'])]))])), 'tantan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tantan/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cbrc3.cbrc.jp/~martin/tantan/'), ('license', 'GPLv3'), ('summary', 'tantan is a tool for masking simple regions (low complexity and short-period tandem repeats) in biological sequences.')])), ('package', CommentedMap([('name', 'tantan'), ('version', '13')])), ('source', CommentedMap([('fn', 'tantan-13.zip'), ('md5', '90a30284a7d0cd04d797527d47bc8bd0'), ('url', 'http://cbrc3.cbrc.jp/~martin/tantan/tantan-13.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['tantan -h'])]))])), 'targetfinder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/targetfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'targetfinder'), ('version', 1.7)])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/carringtonlab/TargetFinder/archive/v1.7.tar.gz'), ('md5', 'fe4713a008442f0f499c53ef7860b3b0'), ('patches', ['targetfinder.patch'])])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'fasta3'])])), ('test', CommentedMap([('commands', ['targetfinder.pl -h', 'targetfinder_threads.pl -h', 'printf \">gi|51107699|gb\\\\nTCTCTAATTCAGCACTGCTCTCTCTCTTCTGTCAAATAGTCCTGGGGCTCTAGAAACTCG\" > sample.fasta', 'targetfinder.pl -s CGACAGAAGAGAGUGAGCAC -d sample.fasta'])])), ('about', CommentedMap([('home', 'https://github.com/carringtonlab/TargetFinder'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Plant small RNA target prediction tool')]))])), 'targqc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/targqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'targqc'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'targqc-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/e0/98/f95a514128e3ee1bfc60a4a9544df9a0497b691383bfebf991973e35c414/TargQC-1.4.4.tar.gz'), ('md5', '75b056abdf2e2cf9860eaa8758ab236a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pip', 'setuptools', 'numpy']), ('run', ['python', 'qualimap', 'sambamba', 'bwa', 'bedtools', 'cython', 'setuptools >=18.5', 'pip', 'numpy', 'tempita', 'pandas', 'pybedtools', 'joblib', 'ipython >=4.0.0,<5.0.0', 'ipyparallel', 'ipython-cluster-helper', 'gffutils', 'nose', 'coverage', 'pysam', 'beautifulsoup4', 'lxml'])])), ('test', CommentedMap([('imports', ['targqc', 'ensembl', 'bed_annotation']), ('commands', ['targqc --version'])])), ('about', CommentedMap([('home', 'https://github.com/vladsaveliev/TargQC'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Target capture coverage QC'), ('license_family', 'GPL3')]))])), 'taxmapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/taxmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'taxmapper'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'taxmapper-1.0.2.tar.gz'), ('url', 'https://bitbucket.org/dbeisser/taxmapper/get/taxmapper-1.0.2.tar.gz'), ('md5', 'debeb04b01fb17f0db47ee46fd94e760')])), ('build', CommentedMap([('skip', True), ('entry_points', ['taxmapper=taxmapper:main'])])), ('requirements', CommentedMap([('build', ['libgcc', 'python', 'setuptools', 'numpy', 'pandas', 'matplotlib', 'deepdish']), ('run', ['libgcc', 'python', 'numpy', 'pandas', 'matplotlib', 'deepdish'])])), ('test', CommentedMap([('commands', ['taxmapper --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/dbeisser/taxmapper'), ('license', 'MIT License'), ('summary', 'Analysis pipeline for metagenomic, microeukaryotic sequencing data.'), ('license_family', 'MIT')]))])), 'taxonkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/taxonkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'taxonkit'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'https://github.com/shenwei356/taxonkit'), ('summary', 'A Cross-platform and Efficient NCBI Taxonomy Toolkit'), ('license', 'MIT')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'taxonkit_<{ version }}_linux64.tar.gz'), ('url', 'https://github.com/shenwei356/taxonkit/releases/download/v<{ version }}/taxonkit_linux_amd64.tar.gz'), ('md5', '50bc1471368fab215a98149621a0aea7')])), ('test', CommentedMap([('commands', ['taxonkit version --help'])]))])), 'taxtastic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/taxtastic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'taxtastic'), ('version', '0.5.4')])), ('source', CommentedMap([('fn', 'taxtastic-0.5.4.tar.gz'), ('url', 'https://pypi.python.org/packages/source/t/taxtastic/taxtastic-0.5.4.tar.gz'), ('md5', '42e82d6370fad3da154c94c9f05000c8')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sqlalchemy >=0.7', 'decorator', 'biopython', 'xlrd']), ('run', ['python', 'sqlalchemy >=0.7', 'decorator', 'biopython', 'xlrd'])])), ('test', CommentedMap([('imports', ['taxtastic', 'taxtastic.scripts', 'taxtastic.subcommands'])])), ('about', CommentedMap([('home', 'https://github.com/fhcrc/taxtastic'), ('license', 'GNU General Public License (GPL)'), ('summary', 'Tools for taxonomic naming and annotation')]))])), 'tbb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tbb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tbb'), ('version', '4.4_20150728')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'tbb44_20150728oss_src.tgz'), ('url', 'https://www.threadingbuildingblocks.org/sites/default/files/software_releases/source/tbb44_20150728oss_src.tgz')])), ('about', CommentedMap([('home', 'https://www.threadingbuildingblocks.org/'), ('license', 'GPLv2')]))])), 'tbl2asn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tbl2asn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tbl2asn'), ('version', '25.3')])), ('source', CommentedMap([('fn', 'linux64.tbl2asn.gz'), ('url', 'ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux64.tbl2asn.gz'), ('sha256', '8d335fa0688e06052752509d99c94f64db126cb47a7f87d256348c121ab3ddd6')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['tbl2asn --help | grep arguments'])])), ('about', CommentedMap([('home', 'https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/'), ('license', 'Public Domain'), ('summary', 'tbl2asn is a program that automates the creation of sequence records for submission to GenBank')]))])), 'tdrmapper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tdrmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tdrmapper'), ('version', '1.1')])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/tDRmapper'), ('git_rev', '6710e07c409961c590ede75a5550d5e569d9fceb')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('test', CommentedMap([('commands', ['TdrMappingScripts.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/sararselitsky/tDRmapper'), ('license', 'academic'), ('summary', 'tRNA detection and quantification')]))])), 'tedna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tedna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tedna'), ('version', '1.2.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'tedna_1.2.2.tar.gz'), ('url', 'https://urgi.versailles.inra.fr/content/download/3481/29402/file/tedna_1.2.2.tar.gz'), ('md5', 'b60b9b974e349db692eea75aad15b5e9')])), ('requirements', CommentedMap([('build', ['google-sparsehash', 'gcc', 'bc']), ('run', ['libgcc', 'google-sparsehash'])])), ('test', CommentedMap([('commands', ['tedna -v'])])), ('about', CommentedMap([('home', 'https://urgi.versailles.inra.fr/Tools/Tedna'), ('license', 'GNU Affero General Public License'), ('summary', 'Tedna is a lightweight de novo transposable element assembler')]))])), 'tepid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tepid/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/ListerLab/TEPID'), ('license', 'GPL'), ('summary', 'TEPID uses paired-end illumina sequencing reads to identify novel TE variants.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'tepid'), ('version', '0.8')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'pysam <0.9,>0.8', 'pybedtools', 'numpy ==1.9.2', 'pandas', 'nose', 'samtools ==1.2', 'bedtools ==2.25.0']), ('run', ['libgcc', 'zlib', 'python', 'pybedtools', 'numpy ==1.9.2', 'pandas', 'pysam <0.9,>0.8', 'nose', 'bowtie2', 'samtools ==1.2', 'samblaster', 'bedtools ==2.25.0', 'yaha'])])), ('source', CommentedMap([('fn', '0.8.tar.gz'), ('url', 'https://github.com/ListerLab/TEPID/archive/0.8.tar.gz'), ('sha256', '2f9ff777705f74abe54f916bd33783085696a4687c47efe61146dd9827c4e241')])), ('test', CommentedMap([('commands', ['tepid-map | grep data', 'tepid-discover --version', 'tepid-refine --version'])]))])), 'testfixtures': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/testfixtures/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'testfixtures'), ('version', '4.8.0')])), ('source', CommentedMap([('fn', 'testfixtures-4.8.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/t/testfixtures/testfixtures-4.8.0.tar.gz'), ('md5', 'f3f558d1be89128e1c4bb4f940a56f7e')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['testfixtures', 'testfixtures.tests'])])), ('about', CommentedMap([('home', 'https://github.com/Simplistix/testfixtures'), ('license', 'MIT License'), ('summary', 'A collection of helpers and mock objects for unit tests and doc tests.')]))])), 'tetoolkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tetoolkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tetoolkit'), ('version', '1.5.1')])), ('source', CommentedMap([('fn', 'TEToolkit-1.5.1.tar.gz'), ('url', 'https://pypi.python.org/packages/3c/a2/b8fc869e59439d3f7523adb6d0caf4ba0f58b002ec72148518dc927042d0/TEToolkit-1.5.1.tar.gz'), ('md5', '05745b2d5109911e95593e423446a831')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'argparse', 'pysam >=0.8']), ('run', ['python', 'argparse', 'pysam >=0.8', 'bioconductor-deseq'])])), ('test', CommentedMap([('imports', ['TEToolkit', 'TEToolkit.IO', 'TEToolkit.ShortRead']), ('commmands', ['TEtranscripts --version'])])), ('about', CommentedMap([('home', 'http://hammelllab.labsites.cshl.edu/software#TEToolkit'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Tools for estimating differential enrichment of Transposable Elements and other highly repetitive regions'), ('license_family', 'GPL3')]))])), 'textinput': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/textinput/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'textinput'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'textinput-0.1.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/d8/e0/9dfec97d21033192dfafcba0813b200b27ebcbd542fca9a5e9ccac3b05e9/textinput-0.1.1.tar.gz'), ('md5', '7eec8a6dff6d489e1183cdfccc26a869')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['textinput', 'tabdelim'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/~hoffman/software/textinput/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'streamlined version of stdlib fileinput')]))])), 'tgt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tgt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tgt'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'tgt-1.4.2.tar.gz'), ('url', 'https://pypi.python.org/packages/bc/90/f1a4c6a1d12ea21db92dc5c2a7b600b276541ba77c5e6cc4f3d7e05fc01f/tgt-1.4.2.tar.gz'), ('md5', 'b396d66ef78edb9e2d51c098b97cd14d')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['tgt'])])), ('about', CommentedMap([('home', 'https://github.com/hbuschme/TextGridTools/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'TextGridTools -- Read, write, and manipulate Praat TextGrid files')]))])), 'theta2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/theta2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'theta2'), ('version', '0.7')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'theta-0.7.tar.gz'), ('url', 'https://github.com/raphael-group/THetA/archive/v0.7.tar.gz')])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'matplotlib', 'joblib', 'numexpr']), ('run', ['python', 'numpy', 'scipy', 'matplotlib', 'joblib', 'numexpr'])])), ('test', CommentedMap([('commands', ['RunTHetA.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/raphael-group/THetA'), ('license', 'Modified MIT (no inclusion in commercial tools)'), ('summary', 'Estimate tumor purity and clonal/subclonal copy number aberrations directly from high-throughput DNA sequencing data')]))])), 'tidyp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tidyp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tidyp'), ('version', '1.04')])), ('requirements', CommentedMap([('build', None)])), ('source', CommentedMap([('fn', 'tidyp-1.04.tar.gz'), ('url', 'http://github.com/downloads/petdance/tidyp/tidyp-1.04.tar.gz'), ('md5', '00a6b804f6625221391d010ca37178e1')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'http://www.tidyp.com/'), ('license', 'BSD-like'), ('summary', 'Program for cleaning up and validating HTML')]))])), 'tn93': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tn93/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tn93'), ('version', '1.0.4')])), ('source', CommentedMap([('fn', 'v1.0.4.tar.gz'), ('url', 'https://github.com/veg/tn93/archive/v1.0.4.tar.gz'), ('sha256', 'c92db46f253b86a05d3c294d78c42565f9c424b76428c23fe32209261d205f75')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"tn93 -h 2>&1 | grep 'compute Tamura Nei 93 distances between aligned sequences' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/veg/tn93'), ('license', 'MIT'), ('summary', 'This is a simple program meant to compute pairwise distances between aligned nucleotide sequences in sequential FASTA format using the Tamura Nei 93 distance.')]))])), 'toil': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/toil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'toil'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'toil-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/t/toil/toil-<{ version }}.tar.gz'), ('md5', 'c1b4ac47cac2d7d298a4aeac82b5d91e')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'future', 'bd2k-python-lib ==1.14a1.dev37', 'dill', 'psutil', 'cgcloud-lib', 'boto ==2.38.0', 'futures', 'azure', 'cwltool ==1.0.20170828135420', 'gcs-oauth2-boto-plugin ==1.9', 'pynacl ==1.1.2', 'rdflib >=4.2.2', 'rdflib-jsonld >=0.3.0', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'typing >=3.5.3,<3.6']), ('run', ['python', 'six', 'future', 'bd2k-python-lib ==1.14a1.dev37', 'dill', 'psutil', 'cgcloud-lib', 'boto ==2.38.0', 'futures', 'azure', 'cwltool ==1.0.20170828135420', 'gcs-oauth2-boto-plugin ==1.9', 'pynacl ==1.1.2', 'rdflib >=4.2.2', 'rdflib-jsonld >=0.3.0', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'typing >=3.5.3,<3.6'])])), ('test', CommentedMap([('commands', ['toil --help', 'cwltoil --help'])])), ('about', CommentedMap([('home', 'https://github.com/BD2KGenomics/toil'), ('license', 'Apache 2.0'), ('summary', 'A scalable, efficient, cross-platform and easy-to-use workflow engine in pure Python')]))])), 'toolshed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/toolshed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'toolshed'), ('version', '0.4.6')])), ('source', CommentedMap([('fn', 'toolshed-0.4.6.tar.gz'), ('url', 'https://pypi.python.org/packages/08/bd/d6259c8d882f6783fdfe69bfaed628afb1ddd291f8a1ac6176d27c62860c/toolshed-0.4.6.tar.gz'), ('md5', 'b0f678e499d5032b9d810271ad26b411')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('requires', ['nose']), ('imports', ['toolshed'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/toolshed'), ('license', 'MIT License'), ('summary', 'flexible and easy file manipulation')]))])), 'tophat-recondition': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tophat-recondition/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/cbrueffer/tophat-recondition'), ('license', 'BSD_2_clause'), ('summary', 'Post-processor for TopHat unmapped reads')])), ('package', CommentedMap([('name', 'tophat-recondition'), ('version', 1.4)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'pysam'])])), ('test', CommentedMap([('commands', ['(tophat-recondition -v 2>&1) > /dev/null'])])), ('source', CommentedMap([('fn', 'v1.4.tar.gz'), ('url', 'https://github.com/cbrueffer/tophat-recondition/archive/v1.4.tar.gz'), ('md5', '4062d5fa0a6f22eb7abed759d278bb34')]))])), 'tophat2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tophat2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/tophat/index.shtml'), ('license', 'Boost Software License'), ('summary', 'A spliced read mapper for RNA-Seq')])), ('package', CommentedMap([('name', 'tophat'), ('version', '2.1.1')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'bowtie2 <=2.2.5'])])), ('test', CommentedMap([('commands', ['(tophat --version 2>&1) > /dev/null'])])), ('source', CommentedMap([('fn', 'tophat-2.1.1.Linux_x86_64.tar.gz'), ('url', 'https://ccb.jhu.edu/software/tophat/downloads/tophat-2.1.1.OSX_x86_64.tar.gz'), ('md5', '1e4e7f8d08f182d2db3202975f284294')]))])), 'trackhub': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trackhub/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trackhub'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'trackhub-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/t/trackhub/trackhub-0.1.3.tar.gz'), ('md5', 'bb141a415676e86f8c8f4fa09e64608c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'fabric']), ('run', ['python', 'fabric'])])), ('test', CommentedMap([('imports', ['trackhub', 'trackhub.scripts', 'trackhub.test', 'trackhub.test.data'])])), ('about', CommentedMap([('home', 'http://github.com/daler/trackhub'), ('license', 'BSD 3-clause'), ('summary', '\\\\nCreate and manage UCSC track hubs from Python\\\\n')]))])), 'transabyss': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/transabyss/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss'), ('license', 'GPL'), ('summary', 'de novo assembly of RNA-Seq data using ABySS')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'transabyss'), ('version', '1.5.4')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['zlib', 'abyss 1.5.2', 'python-igraph', 'samtools 1.3.1', 'bowtie2', 'blat', 'python'])])), ('source', CommentedMap([('fn', 'transabyss-1.5.4.zip'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/releases/1.5.4/transabyss-1.5.4.zip'), ('md5', '0e61575b9341429b18f231fd29eeff60'), ('patches', ['transabyss.patch', 'transabyss-merge.patch'])])), ('test', CommentedMap([('commands', ['transabyss --help'])]))])), 'transcomb': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/transcomb/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/transcriptomeassembly/TransComb_v.1.0.tar.gz'), ('md5', '2bae37e0e59646f900dd0683f572af9b'), ('patches', ['CMakeLists.txt.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'bamtools', 'samtools ==0.1.19', 'zlib', 'cmake']), ('run', ['libgcc', 'zlib', 'boost <{CONDA_BOOST}}*', 'bamtools', 'samtools'])])), ('test', CommentedMap([('commands', ['TransComb -h | grep \"TransComb -b file.bam -s unstranded\"', 'CorrectName -h | grep \"CorrectName fq reads_1.fq reads_2.fq\"'])])), ('about', CommentedMap([('home', 'https://github.com/yechengxi/SparseAssembler'), ('summary', 'A sparse k-mer graph based, memory-efficient genome assembler')]))])), 'transdecoder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/transdecoder/2.1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 3)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/TransDecoder/TransDecoder/archive/v<{ version }}.tar.gz'), ('md5', 'f86c688e13ced3cc51fed3d508a2ea05')])), ('requirements', CommentedMap([('build', None), ('run', ['perl-threaded', 'perl-db-file', 'perl-uri', 'cd-hit'])])), ('test', CommentedMap([('commands', ['TransDecoder.Predict -h 2>&1 | grep USAGE', 'TransDecoder.LongOrfs -h 2>&1 | grep USAGE', 'perl -e \"use DB_File\"'])])), ('about', CommentedMap([('home', 'https://transdecoder.github.io/'), ('license', 'Broad Institute'), ('summary', 'TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.')]))])), 'translatorx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/translatorx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'translatorx'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'translatorx_vLocal.pl'), ('url', 'http://pc16141.mncn.csic.es/cgi-bin/translatorx_vLocal.pl'), ('sha256', 'd0b88c3cedd253ea4972567191ba1dddd4c79841bfb411d6e7bdaae0fbb266c3'), ('patches', ['translatorx.patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'muscle'])])), ('test', CommentedMap([('commands', ['translatorx'])])), ('about', CommentedMap([('home', 'http://pc16141.mncn.csic.es/'), ('license', None), ('summary', 'Multiple alignment of nucleotide sequences guided by amino acid information')]))])), 'transrate-tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/transrate-tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'transrate-tools'), ('version', '1.0.0')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', '<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'transrate-tools-1.0.0.tar.gz'), ('url', 'https://github.com/blahah/transrate-tools/archive/v1.0.0.tar.gz'), ('md5', '93323fbc46fc64a6f0cb1a1fc96c8dc8')])), ('requirements', CommentedMap([('build', ['gcc >=4.8', 'libgcc', 'bamtools', 'zlib', 'cmake', 'make', 'gnu-wget']), ('run', ['libgcc', 'bamtools', 'zlib'])])), ('test', CommentedMap([('commands', ['bam-read 2>&1 | grep \"bam-read version\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/blahah/transrate-tools'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Command-line tools used by transrate for processing bam files.')])), ('extra', CommentedMap([('recipe-maintainers', ['notestaff'])]))])), 'transtermhp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/transtermhp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'transtermhp'), ('version', 2.09)])), ('source', CommentedMap([('fn', 'transtermhp_2.09_src_all.zip'), ('url', 'https://depot.galaxyproject.org/software/transtermhp/transtermhp_2.09_src_all.zip'), ('sha256', 'b636db1c0fe9731cc760d8f3639209d07f7f35309d83013cc9f9e40798ce8be0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['transterm --help | grep TransTermHP'])])), ('about', CommentedMap([('home', 'http://transterm.cbcb.umd.edu/index.php'), ('license', 'GPL'), ('license_file', 'LICENSE.txt'), ('summary', 'TransTermHP finds rho-independent transcription terminators in bacterial genomes')]))])), 'trawler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trawler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trawler'), ('version', 2.0)])), ('source', CommentedMap([('url', 'https://api.github.com/repos/Ramialison-Lab-ARMI/Trawler-2.0/tarball/5f391e0'), ('fn', 'trawler-2.0.tar.gz'), ('sha1', 'c02ce9aaf6bf743b2dcf056bf697a79d10b37be7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['ghostscript', 'openjdk', 'perl >=5.22', 'perl-cgi', 'perl-algorithm-cluster']), ('run', ['ghostscript', 'openjdk', 'perl >=5.22', 'perl-cgi', 'perl-algorithm-cluster'])])), ('test', CommentedMap([('requires', ['perl']), ('commands', ['trawler'])])), ('about', CommentedMap([('home', 'https://trawler.erc.monash.edu.au/help.html'), ('license', 'GPLv2'), ('summary', 'Trawler is a motif discovery tool used to identify enriched motifs in a set of sequenced regions of DNA.')]))])), 'treebest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/treebest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'treebest'), ('version', '1.9.2_ep78')])), ('source', CommentedMap([('url', 'https://github.com/Ensembl/treebest/archive/ensembl_production_78.tar.gz'), ('sha256', '1bdb036e88afb366555d0edf15ae935dcb7fe33acdd3b1c3ee01a06093ac0bca')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'bison', 'flex', 'm4']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"treebest 2>&1 | grep 'Usage:'\"])])), ('about', CommentedMap([('license', 'GPLv2'), ('summary', 'A tool for hierarchically clustering on a sparse graph')]))])), 'treemix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/treemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'treemix'), ('version', 1.12)])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}__gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'treemix-1.12.tar.gz'), ('url', 'https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.12.tar.gz'), ('md5', '183f605751a0fa445e249db840907104')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'gsl <{CONDA_GSL}}*'])])), ('about', CommentedMap([('home', 'http://pritchardlab.stanford.edu/software.html'), ('license', 'GPLv3'), ('summary', 'TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations')])), ('test', CommentedMap([('commands', ['treemix --help | grep 1.12'])]))])), 'treetime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/treetime/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'treetime'), ('version', '0.1')])), ('about', CommentedMap([('home', 'https://github.com/neherlab/treetime'), ('summary', 'Maximum-Likelihood dating and ancestral inference for phylogenetic trees'), ('license', 'MIT')])), ('source', CommentedMap([('fn', 'treetime-de93089.zip'), ('url', 'https://github.com/neherlab/treetime/archive/de93089.zip'), ('md5', '4e5daa8f28bcf579ddb39da3d055ed83')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['ancestral_reconstruction.py -h', 'timetree_inference.py -h'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'numpy >=1.10.4', 'scipy >=0.16.1', 'biopython >=1.66', 'pandas >=0.17.1']), ('run', ['libgcc', 'python <3', 'numpy >=1.10.4', 'scipy >=0.16.1', 'biopython >=1.66', 'pandas >=0.17.1'])]))])), 'trf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trf/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}'), ('url', 'https://tandem.bu.edu/trf/downloads/trf409.macosx'), ('md5', '14784824b6c2fae52f4b5c289ef67255')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['trf -v'])])), ('about', CommentedMap([('home', 'https://tandem.bu.edu/trf/trf.html'), ('license', 'Custom'), ('summary', 'Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences.')]))])), 'triform2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/triform2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'triform2'), ('version', '0.0.5')])), ('source', CommentedMap([('fn', 'triform2-0.0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/e1/c8/c52c93822cbfe5138dbbb41b39ce4b4b55018777e931ecbb0a6a172a6534/triform2-0.0.5.tar.gz'), ('md5', '2f5da13472d663cd9d5974410561a321')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy', 'pandas', 'numpy', 'natsort', 'joblib', 'rpy2 >=2.4.2', 'bioconductor-iranges', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'r-base', 'bx-python']), ('run', ['python', 'setuptools', 'scipy', 'pandas', 'numpy', 'natsort', 'joblib', 'rpy2 >=2.4.2', 'bioconductor-iranges', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'r-base', 'bx-python'])])), ('test', CommentedMap([('imports', ['triform']), ('commands', ['triform2 -h'])])), ('about', CommentedMap([('home', 'http://github.com/endrebak/epic'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Improved sensitivity, specificity and control of false discovery rates in ChIP-Seq peak finding.')]))])), 'trim_galore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trim_galore/0.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trim-galore'), ('version', '0.4.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'trim_galore_v0.4.1.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/trim_galore_v0.4.1.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['perl-threaded', 'fastqc', 'cutadapt'])])), ('test', CommentedMap([('commands', ['trim_galore --help | grep trim_galore > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'), ('license', 'GPL'), ('summary', 'Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control')]))])), 'trimadap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimadap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trimadap'), ('version', 'r10')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'trimadap_r10.tar.gz'), ('url', 'https://github.com/lh3/trimadap/archive/a577c413337df1fbc9c7ac11f925eb7dcc0e8209.tar.gz'), ('sha256', '0d8386429247c08e512c2afd2fd18cfbc68d62d02f92571da956f7dbbad5322b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['trimadap  2>&1 | grep trimadap > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/trimadap'), ('license', 'Unknown'), ('summary', 'Fast but inaccurate adapter trimmer for Illumina reads.')]))])), 'trimmomatic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimmomatic/0.35/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.usadellab.org/cms/?page=trimmomatic'), ('license', 'GPLv3'), ('summary', 'A flexible read trimming tool for Illumina NGS data')])), ('package', CommentedMap([('name', 'trimmomatic'), ('version', '0.35')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('source', CommentedMap([('fn', 'Trimmomatic-0.35.zip'), ('md5', '564e62f4f0f9f56d22762b237d4c69c4'), ('url', 'http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.35.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['unzip -t $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/trimmomatic.jar &> /dev/null'])]))])), 'trinculo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trinculo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trinculo'), ('version', '0.96')])), ('build', CommentedMap([('skip', True)])), ('source', CommentedMap([('fn', 'trinculo_v0.96.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/trinculo/trinculo_v0.96.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'lapack']), ('run', ['libgcc', 'lapack'])])), ('test', CommentedMap([('commands', ['trinculo multinom --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/trinculo/'), ('license', 'MIT'), ('summary', 'A toolkit for carrying out genetic association for\\nmulti-category phenotypes. Implements multinomial and ordinal\\nassociation incorporating covariates, conditional analysis,\\nempirical and non-emperical priors and fine-mapping.\\n')]))])), 'trinity': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trinity/2.1.1/date.2011_11_26/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq'), ('license', 'https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt'), ('summary', 'Trinity assembles transcript sequences from Illumina RNA-Seq data')])), ('build', CommentedMap([('number', 4), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'trinity'), ('version', 'date.2011_11_26')])), ('source', CommentedMap([('fn', 'trinityrnaseq_r2011-11-26.tgz'), ('md5', '3a4c4691908d37042ca63ca469c75fd3'), ('url', 'http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz'), ('patches', ['RunButterfly.cc.patch', 'Trinity.pl.patch', 'samtools_makefile.patch'])])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['perl-threaded', 'libgcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('extra', CommentedMap([('notes', 'Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)')])), ('test', CommentedMap([('commands', ['$PREFIX/bin/Trinity-test &> /dev/null'])]))])), 'trinotate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trinotate/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://trinotate.github.io/'), ('license', 'BSD'), ('summary', 'Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'trinotate'), ('version', '3.0.2')])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi', 'perl-dbd-sqlite', 'perl-file-find-rule']), ('run', ['perl', 'sqlite', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi', 'perl-dbd-sqlite', 'blast', 'hmmer', 'perl-file-find-rule'])])), ('source', CommentedMap([('fn', 'trinotate-3.0.2.tar.gz'), ('url', 'https://github.com/Trinotate/Trinotate/archive/v3.0.2.tar.gz'), ('patches', ['runMe.sh.patch', 'trinotate.patch']), ('md5', 'bf52a1393d029acc43dbb493dd6f31f4')])), ('test', CommentedMap([('commands', ['Trinotate --help 2>&1 | grep \"Trinotate <sqlite.db> <command>\"'])]))])), 'trnascan-se': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trnascan-se/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://eddylab.org/software/tRNAscan-SE/tRNAscan-SE.tar.gz'), ('md5', '714689121a1cf11c5d9c59561769c4d7'), ('patches', ['patches/checkversion.pl.patch', 'patches/fasta2gsi.pl.patch', 'patches/instman.pl.patch', 'patches/sstofa.pl.patch', 'patches/tRNAscan-SE.src.patch', 'patches/Makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl']), ('run', ['libgcc', 'zlib', 'perl'])])), ('test', CommentedMap([('commands', ['tRNAscan-SE -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://eddylab.org/software.html'), ('license', 'GPLv2'), ('summary', 'tRNA detection in large-scale genome sequence')]))])), 'tssar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tssar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tssar'), ('version', '1.0.1')])), ('source', CommentedMap([('url', 'http://nibiru.tbi.univie.ac.at/TSSAR/download/TSSAR_1.0.1.zip'), ('md5', '22f3c1fe591a566b0c31877fff6c31b9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'r-vgam', 'r-skellam', 'perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['TSSAR --help | grep \"help\"'])])), ('about', CommentedMap([('home', 'http://rna.tbi.univie.ac.at/TSSAR'), ('license', 'GNU General Public License v2.0'), ('summary', 'TSSAR is a Web Service for predicting bacterial Transcription Start Sites from dRNA-seq data.')]))])), 'twobitreader': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/twobitreader/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'twobitreader'), ('version', '3.1.4')])), ('source', CommentedMap([('fn', 'twobitreader-3.1.4.tar.gz'), ('url', 'https://pypi.python.org/packages/4c/95/133326b21354b8592d1b6be6d594a17cebf1070325222075df2630d77da4/twobitreader-3.1.4.tar.gz'), ('md5', '1672e7e515d057a93cc2dd6c8d487458')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['twobitreader'])])), ('about', CommentedMap([('home', 'https://github.com/benjschiller/twobitreader'), ('license', 'Artistic License'), ('summary', 'A fast python package for reading .2bit files (used by the UCSC genome browser)')]))])), 'twopaco': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/twopaco/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/medvedevgroup/TwoPaCo/archive/v<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'tbb']), ('run', ['libgcc', 'tbb'])])), ('test', CommentedMap([('commands', ['twopaco --help', 'graphdump --help'])])), ('about', CommentedMap([('home', 'https://github.com/medvedevgroup/TwoPaCo'), ('license', 'BSD-3-Clause'), ('summary', 'A fast constructor of the compressed de Bruijn graph from many genomes')]))])), 'ucsc-addcols': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-addcols/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-addcols'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/addCols'), ('fn', 'addCols'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp addCols $PREFIX/bin; chmod +x $PREFIX/bin/addCols')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which addCols'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Sum columns in a text file.')]))])), 'ucsc-ameme': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-ameme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-ameme'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/ameme'), ('fn', 'ameme'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp ameme $PREFIX/bin; chmod +x $PREFIX/bin/ameme')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which ameme'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'find common patterns in DNA')]))])), 'ucsc-autodtd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-autodtd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-autodtd'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/autoDtd'), ('fn', 'autoDtd'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp autoDtd $PREFIX/bin; chmod +x $PREFIX/bin/autoDtd')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which autoDtd'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Give this a XML document to look at and it will come up with a DTD')]))])), 'ucsc-autosql': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-autosql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-autosql'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/autoSql'), ('fn', 'autoSql'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp autoSql $PREFIX/bin; chmod +x $PREFIX/bin/autoSql')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which autoSql'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'create SQL and C code for permanently storing')]))])), 'ucsc-autoxml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-autoxml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-autoxml'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/autoXml'), ('fn', 'autoXml'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp autoXml $PREFIX/bin; chmod +x $PREFIX/bin/autoXml')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which autoXml'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Generate structures code and parser for XML file from DTD-like spec')]))])), 'ucsc-avecols': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-avecols/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-avecols'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/aveCols'), ('fn', 'aveCols'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp aveCols $PREFIX/bin; chmod +x $PREFIX/bin/aveCols')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which aveCols'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'average together columns')]))])), 'ucsc-axtchain': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-axtchain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-axtchain'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/axtChain'), ('fn', 'axtChain'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp axtChain $PREFIX/bin; chmod +x $PREFIX/bin/axtChain')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which axtChain'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Chain together axt alignments.')]))])), 'ucsc-axtsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-axtsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-axtsort'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/axtSort'), ('fn', 'axtSort'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp axtSort $PREFIX/bin; chmod +x $PREFIX/bin/axtSort')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which axtSort'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Sort axt files')]))])), 'ucsc-axtswap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-axtswap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-axtswap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/axtSwap'), ('fn', 'axtSwap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp axtSwap $PREFIX/bin; chmod +x $PREFIX/bin/axtSwap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which axtSwap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Swap source and query in an axt file')]))])), 'ucsc-axttomaf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-axttomaf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-axttomaf'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/axtToMaf'), ('fn', 'axtToMaf'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp axtToMaf $PREFIX/bin; chmod +x $PREFIX/bin/axtToMaf')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which axtToMaf'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from axt to maf format')]))])), 'ucsc-axttopsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-axttopsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-axttopsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/axtToPsl'), ('fn', 'axtToPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp axtToPsl $PREFIX/bin; chmod +x $PREFIX/bin/axtToPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which axtToPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert axt to psl format')]))])), 'ucsc-bedclip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedclip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedclip'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedClip'), ('fn', 'bedClip'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedClip $PREFIX/bin; chmod +x $PREFIX/bin/bedClip')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedClip'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Remove lines from bed file that refer to off-chromosome places.')]))])), 'ucsc-bedcommonregions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedcommonregions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedcommonregions'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedCommonRegions'), ('fn', 'bedCommonRegions'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedCommonRegions $PREFIX/bin; chmod +x $PREFIX/bin/bedCommonRegions')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedCommonRegions'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Create a bed file (just bed3) that contains the regions common to all inputs.')]))])), 'ucsc-bedcoverage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedcoverage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedcoverage'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedCoverage'), ('fn', 'bedCoverage'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedCoverage $PREFIX/bin; chmod +x $PREFIX/bin/bedCoverage')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedCoverage'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Analyse coverage by bed files - chromosome by ')]))])), 'ucsc-bedextendranges': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedextendranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedextendranges'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedExtendRanges'), ('fn', 'bedExtendRanges'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedExtendRanges $PREFIX/bin; chmod +x $PREFIX/bin/bedExtendRanges')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedExtendRanges'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'extend length of entries in bed 6+ data to be at least the given length,')]))])), 'ucsc-bedgeneparts': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedgeneparts/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedgeneparts'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedGeneParts'), ('fn', 'bedGeneParts'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedGeneParts $PREFIX/bin; chmod +x $PREFIX/bin/bedGeneParts')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedGeneParts'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Given a bed, spit out promoter, first exon, or all introns.')]))])), 'ucsc-bedgraphpack': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedgraphpack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedgraphpack'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedGraphPack'), ('fn', 'bedGraphPack'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedGraphPack $PREFIX/bin; chmod +x $PREFIX/bin/bedGraphPack')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedGraphPack'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Pack together adjacent records representing same value.')]))])), 'ucsc-bedgraphtobigwig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedgraphtobigwig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedgraphtobigwig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedGraphToBigWig'), ('fn', 'bedGraphToBigWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedGraphToBigWig $PREFIX/bin; chmod +x $PREFIX/bin/bedGraphToBigWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedGraphToBigWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert a bedGraph file to bigWig format.')]))])), 'ucsc-bedintersect': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedintersect/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedintersect'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedIntersect'), ('fn', 'bedIntersect'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedIntersect $PREFIX/bin; chmod +x $PREFIX/bin/bedIntersect')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedIntersect'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Intersect two bed files')]))])), 'ucsc-beditemoverlapcount': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-beditemoverlapcount/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-beditemoverlapcount'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedItemOverlapCount'), ('fn', 'bedItemOverlapCount'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedItemOverlapCount $PREFIX/bin; chmod +x $PREFIX/bin/bedItemOverlapCount')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedItemOverlapCount'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'count number of times a base is overlapped by the')]))])), 'ucsc-bedpileups': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedpileups/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedpileups'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedPileUps'), ('fn', 'bedPileUps'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedPileUps $PREFIX/bin; chmod +x $PREFIX/bin/bedPileUps')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedPileUps'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Find (exact) overlaps if any in bed input')]))])), 'ucsc-bedremoveoverlap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedremoveoverlap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedremoveoverlap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedRemoveOverlap'), ('fn', 'bedRemoveOverlap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedRemoveOverlap $PREFIX/bin; chmod +x $PREFIX/bin/bedRemoveOverlap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedRemoveOverlap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Remove overlapping records from a (sorted) bed file.  Gets rid of')]))])), 'ucsc-bedrestricttopositions': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedrestricttopositions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedrestricttopositions'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedRestrictToPositions'), ('fn', 'bedRestrictToPositions'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedRestrictToPositions $PREFIX/bin; chmod +x $PREFIX/bin/bedRestrictToPositions')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedRestrictToPositions'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Filter bed file, restricting to only ones that match chrom/start/ends specified in restrict.bed file.')]))])), 'ucsc-bedsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedsort'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedSort'), ('fn', 'bedSort'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedSort $PREFIX/bin; chmod +x $PREFIX/bin/bedSort')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedSort'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Sort a .bed file by chrom,chromStart')]))])), 'ucsc-bedtobigbed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedtobigbed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedtobigbed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedToBigBed'), ('fn', 'bedToBigBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedToBigBed $PREFIX/bin; chmod +x $PREFIX/bin/bedToBigBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedToBigBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert bed file to bigBed. (BigBed version: 4)')]))])), 'ucsc-bedtogenepred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedtogenepred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedtogenepred'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedToGenePred'), ('fn', 'bedToGenePred'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedToGenePred $PREFIX/bin; chmod +x $PREFIX/bin/bedToGenePred')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedToGenePred'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert bed format files to genePred format')]))])), 'ucsc-bedtopsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedtopsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedtopsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedToPsl'), ('fn', 'bedToPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedToPsl $PREFIX/bin; chmod +x $PREFIX/bin/bedToPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedToPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert bed format files to psl format')]))])), 'ucsc-bedweedoverlapping': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedweedoverlapping/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedweedoverlapping'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedWeedOverlapping'), ('fn', 'bedWeedOverlapping'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedWeedOverlapping $PREFIX/bin; chmod +x $PREFIX/bin/bedWeedOverlapping')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedWeedOverlapping'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', \"Filter out beds that overlap a 'weed.bed' file.\")]))])), 'ucsc-bigbedinfo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigbedinfo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigbedinfo'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigBedInfo'), ('fn', 'bigBedInfo'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigBedInfo $PREFIX/bin; chmod +x $PREFIX/bin/bigBedInfo')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigBedInfo'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Show information about a bigBed file.')]))])), 'ucsc-bigbednameditems': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigbednameditems/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigbednameditems'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigBedNamedItems'), ('fn', 'bigBedNamedItems'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigBedNamedItems $PREFIX/bin; chmod +x $PREFIX/bin/bigBedNamedItems')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigBedNamedItems'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract item of given name from bigBed')]))])), 'ucsc-bigbedsummary': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigbedsummary/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigbedsummary'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigBedSummary'), ('fn', 'bigBedSummary'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigBedSummary $PREFIX/bin; chmod +x $PREFIX/bin/bigBedSummary')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigBedSummary'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract summary information from a bigBed file.')]))])), 'ucsc-bigbedtobed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigbedtobed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigbedtobed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigBedToBed'), ('fn', 'bigBedToBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigBedToBed $PREFIX/bin; chmod +x $PREFIX/bin/bigBedToBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigBedToBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from bigBed to ascii bed format.')]))])), 'ucsc-bigpsltopsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigpsltopsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigpsltopsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigPslToPsl'), ('fn', 'bigPslToPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigPslToPsl $PREFIX/bin; chmod +x $PREFIX/bin/bigPslToPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigPslToPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert bigPsl file to psle')]))])), 'ucsc-bigwigaverageoverbed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigaverageoverbed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigaverageoverbed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigAverageOverBed'), ('fn', 'bigWigAverageOverBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigAverageOverBed $PREFIX/bin; chmod +x $PREFIX/bin/bigWigAverageOverBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigAverageOverBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Compute average score of big wig over each bed, which may have introns.')]))])), 'ucsc-bigwigcat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigcat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigcat'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigCat'), ('fn', 'bigWigCat'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigCat $PREFIX/bin; chmod +x $PREFIX/bin/bigWigCat')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigCat'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'merge non-overlapping bigWig files')]))])), 'ucsc-bigwigcluster': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigcluster/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigcluster'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigCluster'), ('fn', 'bigWigCluster'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigCluster $PREFIX/bin; chmod +x $PREFIX/bin/bigWigCluster')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigCluster'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Cluster bigWigs using a hacTree')]))])), 'ucsc-bigwigcorrelate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigcorrelate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigcorrelate'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigCorrelate'), ('fn', 'bigWigCorrelate'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigCorrelate $PREFIX/bin; chmod +x $PREFIX/bin/bigWigCorrelate')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigCorrelate'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Correlate bigWig files, optionally only on target regions.')]))])), 'ucsc-bigwiginfo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwiginfo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwiginfo'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigInfo'), ('fn', 'bigWigInfo'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigInfo $PREFIX/bin; chmod +x $PREFIX/bin/bigWigInfo')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigInfo'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Print out information about bigWig file.')]))])), 'ucsc-bigwigmerge': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigmerge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigmerge'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigMerge'), ('fn', 'bigWigMerge'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigMerge $PREFIX/bin; chmod +x $PREFIX/bin/bigWigMerge')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigMerge'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Merge together multiple bigWigs into a single output bedGraph.')]))])), 'ucsc-bigwigsummary': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigsummary/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigsummary'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigSummary'), ('fn', 'bigWigSummary'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigSummary $PREFIX/bin; chmod +x $PREFIX/bin/bigWigSummary')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigSummary'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract summary information from a bigWig file.')]))])), 'ucsc-bigwigtobedgraph': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigtobedgraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigtobedgraph'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigToBedGraph'), ('fn', 'bigWigToBedGraph'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigToBedGraph $PREFIX/bin; chmod +x $PREFIX/bin/bigWigToBedGraph')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigToBedGraph'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from bigWig to bedGraph format.')]))])), 'ucsc-bigwigtowig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bigwigtowig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bigwigtowig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bigWigToWig'), ('fn', 'bigWigToWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bigWigToWig $PREFIX/bin; chmod +x $PREFIX/bin/bigWigToWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bigWigToWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert bigWig to wig.  This will keep more of the same structure of the')]))])), 'ucsc-blasttopsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-blasttopsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-blasttopsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/blastToPsl'), ('fn', 'blastToPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp blastToPsl $PREFIX/bin; chmod +x $PREFIX/bin/blastToPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which blastToPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert blast alignments to PSLs.')]))])), 'ucsc-catdir': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-catdir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-catdir'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/catDir'), ('fn', 'catDir'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp catDir $PREFIX/bin; chmod +x $PREFIX/bin/catDir')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which catDir'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'concatenate files in directory to stdout.')]))])), 'ucsc-catuncomment': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-catuncomment/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-catuncomment'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/catUncomment'), ('fn', 'catUncomment'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp catUncomment $PREFIX/bin; chmod +x $PREFIX/bin/catUncomment')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which catUncomment'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', \"Concatenate input removing lines that start with '#'\")]))])), 'ucsc-chainantirepeat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainantirepeat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainantirepeat'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainAntiRepeat'), ('fn', 'chainAntiRepeat'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainAntiRepeat $PREFIX/bin; chmod +x $PREFIX/bin/chainAntiRepeat')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainAntiRepeat'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Get rid of chains that are primarily the results of repeats and degenerate DNA')]))])), 'ucsc-chainfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainfilter'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainFilter'), ('fn', 'chainFilter'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainFilter $PREFIX/bin; chmod +x $PREFIX/bin/chainFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Filter chain files.  Output goes to standard out.')]))])), 'ucsc-chainmergesort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainmergesort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainmergesort'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainMergeSort'), ('fn', 'chainMergeSort'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainMergeSort $PREFIX/bin; chmod +x $PREFIX/bin/chainMergeSort')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainMergeSort'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Combine sorted files into larger sorted file')]))])), 'ucsc-chainnet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainnet'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainNet'), ('fn', 'chainNet'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainNet $PREFIX/bin; chmod +x $PREFIX/bin/chainNet')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainNet'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Make alignment nets out of chains')]))])), 'ucsc-chainprenet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainprenet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainprenet'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainPreNet'), ('fn', 'chainPreNet'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainPreNet $PREFIX/bin; chmod +x $PREFIX/bin/chainPreNet')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainPreNet'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', \"Remove chains that don't have a chance of being netted\")]))])), 'ucsc-chainsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainsort'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainSort'), ('fn', 'chainSort'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainSort $PREFIX/bin; chmod +x $PREFIX/bin/chainSort')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainSort'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Sort chains.  By default sorts by score.')]))])), 'ucsc-chainsplit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainsplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainsplit'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainSplit'), ('fn', 'chainSplit'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainSplit $PREFIX/bin; chmod +x $PREFIX/bin/chainSplit')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainSplit'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Split chains up by target or query sequence')]))])), 'ucsc-chainstitchid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainstitchid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainstitchid'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainStitchId'), ('fn', 'chainStitchId'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainStitchId $PREFIX/bin; chmod +x $PREFIX/bin/chainStitchId')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainStitchId'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Join chain fragments with the same chain ID into a single')]))])), 'ucsc-chainswap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chainswap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chainswap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainSwap'), ('fn', 'chainSwap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainSwap $PREFIX/bin; chmod +x $PREFIX/bin/chainSwap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainSwap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Swap target and query in chain')]))])), 'ucsc-chaintoaxt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chaintoaxt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chaintoaxt'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainToAxt'), ('fn', 'chainToAxt'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainToAxt $PREFIX/bin; chmod +x $PREFIX/bin/chainToAxt')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainToAxt'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from chain to axt file')]))])), 'ucsc-chaintopsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chaintopsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chaintopsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainToPsl'), ('fn', 'chainToPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainToPsl $PREFIX/bin; chmod +x $PREFIX/bin/chainToPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainToPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert chain file to psl format')]))])), 'ucsc-chaintopslbasic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chaintopslbasic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chaintopslbasic'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chainToPslBasic'), ('fn', 'chainToPslBasic'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chainToPslBasic $PREFIX/bin; chmod +x $PREFIX/bin/chainToPslBasic')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chainToPslBasic'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Basic conversion chain file to psl format')]))])), 'ucsc-checkagpandfa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-checkagpandfa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-checkagpandfa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/checkAgpAndFa'), ('fn', 'checkAgpAndFa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp checkAgpAndFa $PREFIX/bin; chmod +x $PREFIX/bin/checkAgpAndFa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which checkAgpAndFa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'takes a .agp file and .fa file and ensures that they are in synch')]))])), 'ucsc-checkcoveragegaps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-checkcoveragegaps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-checkcoveragegaps'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/checkCoverageGaps'), ('fn', 'checkCoverageGaps'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp checkCoverageGaps $PREFIX/bin; chmod +x $PREFIX/bin/checkCoverageGaps')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which checkCoverageGaps'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Check for biggest gap in coverage for a list of tracks.')]))])), 'ucsc-checkhgfindspec': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-checkhgfindspec/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-checkhgfindspec'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/checkHgFindSpec'), ('fn', 'checkHgFindSpec'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp checkHgFindSpec $PREFIX/bin; chmod +x $PREFIX/bin/checkHgFindSpec')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which checkHgFindSpec'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'test and describe search specs in hgFindSpec tables.')]))])), 'ucsc-checktablecoords': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-checktablecoords/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-checktablecoords'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/checkTableCoords'), ('fn', 'checkTableCoords'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp checkTableCoords $PREFIX/bin; chmod +x $PREFIX/bin/checkTableCoords')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which checkTableCoords'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'check invariants on genomic coords in table(s).')]))])), 'ucsc-chopfalines': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chopfalines/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chopfalines'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chopFaLines'), ('fn', 'chopFaLines'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chopFaLines $PREFIX/bin; chmod +x $PREFIX/bin/chopFaLines')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chopFaLines'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Read in FA file with long lines and rewrite it with shorter lines')]))])), 'ucsc-chromgraphfrombin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chromgraphfrombin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chromgraphfrombin'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chromGraphFromBin'), ('fn', 'chromGraphFromBin'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chromGraphFromBin $PREFIX/bin; chmod +x $PREFIX/bin/chromGraphFromBin')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chromGraphFromBin'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert chromGraph binary to ascii format.')]))])), 'ucsc-chromgraphtobin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-chromgraphtobin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-chromgraphtobin'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/chromGraphToBin'), ('fn', 'chromGraphToBin'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp chromGraphToBin $PREFIX/bin; chmod +x $PREFIX/bin/chromGraphToBin')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which chromGraphToBin'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Make binary version of chromGraph.')]))])), 'ucsc-coltransform': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-coltransform/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-coltransform'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/colTransform'), ('fn', 'colTransform'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp colTransform $PREFIX/bin; chmod +x $PREFIX/bin/colTransform')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which colTransform'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Add and/or multiply column by constant.')]))])), 'ucsc-countchars': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-countchars/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-countchars'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/countChars'), ('fn', 'countChars'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp countChars $PREFIX/bin; chmod +x $PREFIX/bin/countChars')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which countChars'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Count the number of occurrences of a particular char')]))])), 'ucsc-crtreeindexbed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-crtreeindexbed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-crtreeindexbed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/crTreeIndexBed'), ('fn', 'crTreeIndexBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp crTreeIndexBed $PREFIX/bin; chmod +x $PREFIX/bin/crTreeIndexBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which crTreeIndexBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Create an index for a bed file.')]))])), 'ucsc-crtreesearchbed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-crtreesearchbed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-crtreesearchbed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/crTreeSearchBed'), ('fn', 'crTreeSearchBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp crTreeSearchBed $PREFIX/bin; chmod +x $PREFIX/bin/crTreeSearchBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which crTreeSearchBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Search a crTree indexed bed file and print all items that overlap query.')]))])), 'ucsc-dbsnoop': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-dbsnoop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-dbsnoop'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/dbSnoop'), ('fn', 'dbSnoop'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp dbSnoop $PREFIX/bin; chmod +x $PREFIX/bin/dbSnoop')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which dbSnoop'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Produce an overview of a database.')]))])), 'ucsc-dbtrash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-dbtrash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-dbtrash'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/dbTrash'), ('fn', 'dbTrash'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp dbTrash $PREFIX/bin; chmod +x $PREFIX/bin/dbTrash')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which dbTrash'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'drop tables from a database older than specified N hours')]))])), 'ucsc-estorient': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-estorient/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-estorient'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/estOrient'), ('fn', 'estOrient'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp estOrient $PREFIX/bin; chmod +x $PREFIX/bin/estOrient')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which estOrient'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', \" Read ESTs from a database and determine orientation based on estOrientInfo table or direction in gbCdnaInfo table.  Update PSLs so that the strand reflects the direction of transcription. By default, PSLs where the direction can't be determined are dropped. \")]))])), 'ucsc-faalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-faalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-faalign'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faAlign'), ('fn', 'faAlign'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faAlign $PREFIX/bin; chmod +x $PREFIX/bin/faAlign')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faAlign'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Align two fasta files')]))])), 'ucsc-facmp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-facmp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-facmp'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faCmp'), ('fn', 'faCmp'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faCmp $PREFIX/bin; chmod +x $PREFIX/bin/faCmp')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faCmp'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Compare two .fa files')]))])), 'ucsc-facount': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-facount/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-facount'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faCount'), ('fn', 'faCount'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faCount $PREFIX/bin; chmod +x $PREFIX/bin/faCount')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faCount'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'count base statistics and CpGs in FA files.')]))])), 'ucsc-fafilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fafilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fafilter'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faFilter'), ('fn', 'faFilter'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faFilter $PREFIX/bin; chmod +x $PREFIX/bin/faFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Filter fa records, selecting ones that match the specified conditions')]))])), 'ucsc-fafiltern': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fafiltern/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fafiltern'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faFilterN'), ('fn', 'faFilterN'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faFilterN $PREFIX/bin; chmod +x $PREFIX/bin/faFilterN')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faFilterN'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', \"Get rid of sequences with too many N's\")]))])), 'ucsc-fafrag': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fafrag/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fafrag'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faFrag'), ('fn', 'faFrag'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faFrag $PREFIX/bin; chmod +x $PREFIX/bin/faFrag')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faFrag'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract a piece of DNA from a .fa file.')]))])), 'ucsc-fanoise': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fanoise/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fanoise'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faNoise'), ('fn', 'faNoise'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faNoise $PREFIX/bin; chmod +x $PREFIX/bin/faNoise')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faNoise'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Add noise to .fa file')]))])), 'ucsc-faonerecord': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-faonerecord/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-faonerecord'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faOneRecord'), ('fn', 'faOneRecord'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faOneRecord $PREFIX/bin; chmod +x $PREFIX/bin/faOneRecord')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faOneRecord'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract a single record from a .FA file')]))])), 'ucsc-fapolyasizes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fapolyasizes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fapolyasizes'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faPolyASizes'), ('fn', 'faPolyASizes'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faPolyASizes $PREFIX/bin; chmod +x $PREFIX/bin/faPolyASizes')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faPolyASizes'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'get poly A sizes')]))])), 'ucsc-farandomize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-farandomize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-farandomize'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faRandomize'), ('fn', 'faRandomize'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faRandomize $PREFIX/bin; chmod +x $PREFIX/bin/faRandomize')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faRandomize'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Program to create random fasta records')]))])), 'ucsc-farc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-farc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-farc'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faRc'), ('fn', 'faRc'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faRc $PREFIX/bin; chmod +x $PREFIX/bin/faRc')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faRc'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Reverse complement a FA file')]))])), 'ucsc-fasize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fasize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fasize'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faSize'), ('fn', 'faSize'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faSize $PREFIX/bin; chmod +x $PREFIX/bin/faSize')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faSize'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'print total base count in fa files.')]))])), 'ucsc-fasomerecords': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fasomerecords/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fasomerecords'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faSomeRecords'), ('fn', 'faSomeRecords'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faSomeRecords $PREFIX/bin; chmod +x $PREFIX/bin/faSomeRecords')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faSomeRecords'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract multiple fa records')]))])), 'ucsc-fasplit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fasplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fasplit'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faSplit'), ('fn', 'faSplit'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faSplit $PREFIX/bin; chmod +x $PREFIX/bin/faSplit')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faSplit'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Split an fa file into several files.')]))])), 'ucsc-fastqtofa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fastqtofa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fastqtofa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/fastqToFa'), ('fn', 'fastqToFa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp fastqToFa $PREFIX/bin; chmod +x $PREFIX/bin/fastqToFa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which fastqToFa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from fastq to fasta format.')]))])), 'ucsc-fatofastq': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fatofastq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fatofastq'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faToFastq'), ('fn', 'faToFastq'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faToFastq $PREFIX/bin; chmod +x $PREFIX/bin/faToFastq')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faToFastq'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert fa to fastq format, just faking quality values.')]))])), 'ucsc-fatotab': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fatotab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fatotab'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faToTab'), ('fn', 'faToTab'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faToTab $PREFIX/bin; chmod +x $PREFIX/bin/faToTab')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faToTab'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert fa file to tab separated file')]))])), 'ucsc-fatotwobit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fatotwobit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fatotwobit'), ('version', '353')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faToTwoBit'), ('fn', 'faToTwoBit'), ('md5', '8335fcbaa7815f368c56e04564fc58e9'), ('patches', ['include.patch'])])), ('requirements', CommentedMap([('build', ['libpng', 'mysql-connector-c', 'zlib', 'openssl', 'libuuid', 'gcc']), ('run', ['libpng', 'mysql-connector-c', 'zlib', 'openssl', 'libuuid', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faToTwoBit', 'faToTwoBit 2>&1 | grep faToTwoBit'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert DNA from fasta to 2bit format')]))])), 'ucsc-fatrans': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fatrans/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fatrans'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/faTrans'), ('fn', 'faTrans'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp faTrans $PREFIX/bin; chmod +x $PREFIX/bin/faTrans')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which faTrans'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Translate DNA .fa file to peptide')]))])), 'ucsc-featurebits': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-featurebits/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-featurebits'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/featureBits'), ('fn', 'featureBits'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp featureBits $PREFIX/bin; chmod +x $PREFIX/bin/featureBits')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which featureBits'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Correlate tables via bitmap projections. ')]))])), 'ucsc-fetchchromsizes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-fetchchromsizes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-fetchchromsizes'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/fetchChromSizes'), ('fn', 'fetchChromSizes'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp fetchChromSizes $PREFIX/bin; chmod +x $PREFIX/bin/fetchChromSizes')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which fetchChromSizes'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', ' used to fetch chrom.sizes information from UCSC for the given <db> ')]))])), 'ucsc-findmotif': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-findmotif/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-findmotif'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/findMotif'), ('fn', 'findMotif'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp findMotif $PREFIX/bin; chmod +x $PREFIX/bin/findMotif')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which findMotif'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'find specified motif in sequence')]))])), 'ucsc-gaptolift': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-gaptolift/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-gaptolift'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/gapToLift'), ('fn', 'gapToLift'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp gapToLift $PREFIX/bin; chmod +x $PREFIX/bin/gapToLift')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which gapToLift'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'create lift file from gap table(s)')]))])), 'ucsc-genepredcheck': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredcheck/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredcheck'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredCheck'), ('fn', 'genePredCheck'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredCheck $PREFIX/bin; chmod +x $PREFIX/bin/genePredCheck')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredCheck'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'validate genePred files or tables')]))])), 'ucsc-genepredfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredfilter'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredFilter'), ('fn', 'genePredFilter'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredFilter $PREFIX/bin; chmod +x $PREFIX/bin/genePredFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'filter a genePred file')]))])), 'ucsc-genepredhisto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredhisto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredhisto'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredHisto'), ('fn', 'genePredHisto'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredHisto $PREFIX/bin; chmod +x $PREFIX/bin/genePredHisto')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredHisto'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'get data for generating histograms from a genePred file.')]))])), 'ucsc-genepredsinglecover': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredsinglecover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredsinglecover'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredSingleCover'), ('fn', 'genePredSingleCover'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredSingleCover $PREFIX/bin; chmod +x $PREFIX/bin/genePredSingleCover')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredSingleCover'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'create single-coverage genePred files')]))])), 'ucsc-genepredtobed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredtobed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredtobed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredToBed'), ('fn', 'genePredToBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredToBed $PREFIX/bin; chmod +x $PREFIX/bin/genePredToBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredToBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from genePred to bed format. Does not yet handle genePredExt')]))])), 'ucsc-genepredtobiggenepred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredtobiggenepred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredtobiggenepred'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredToBigGenePred'), ('fn', 'genePredToBigGenePred'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredToBigGenePred $PREFIX/bin; chmod +x $PREFIX/bin/genePredToBigGenePred')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredToBigGenePred'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'converts genePred or genePredExt to bigGenePred input (bed format with extra fields)')]))])), 'ucsc-genepredtofakepsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredtofakepsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredtofakepsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredToFakePsl'), ('fn', 'genePredToFakePsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredToFakePsl $PREFIX/bin; chmod +x $PREFIX/bin/genePredToFakePsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredToFakePsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Create a psl of fake-mRNA aligned to gene-preds from a file or table.')]))])), 'ucsc-genepredtogtf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredtogtf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredtogtf'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredToGtf'), ('fn', 'genePredToGtf'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredToGtf $PREFIX/bin; chmod +x $PREFIX/bin/genePredToGtf')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredToGtf'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert genePred table or file to gtf.')]))])), 'ucsc-genepredtomafframes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-genepredtomafframes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-genepredtomafframes'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/genePredToMafFrames'), ('fn', 'genePredToMafFrames'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp genePredToMafFrames $PREFIX/bin; chmod +x $PREFIX/bin/genePredToMafFrames')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which genePredToMafFrames'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'create mafFrames tables from a genePreds')]))])), 'ucsc-getrna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-getrna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-getrna'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/getRna'), ('fn', 'getRna'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp getRna $PREFIX/bin; chmod +x $PREFIX/bin/getRna')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which getRna'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Get mrna for GenBank or RefSeq sequences found in a database')]))])), 'ucsc-getrnapred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-getrnapred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-getrnapred'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/getRnaPred'), ('fn', 'getRnaPred'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp getRnaPred $PREFIX/bin; chmod +x $PREFIX/bin/getRnaPred')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which getRnaPred'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Get virtual RNA for gene predictions')]))])), 'ucsc-gff3togenepred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-gff3togenepred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-gff3togenepred'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/gff3ToGenePred'), ('fn', 'gff3ToGenePred'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp gff3ToGenePred $PREFIX/bin; chmod +x $PREFIX/bin/gff3ToGenePred')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which gff3ToGenePred'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert a GFF3 file to a genePred file')]))])), 'ucsc-gtftogenepred': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-gtftogenepred/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-gtftogenepred'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/gtfToGenePred'), ('fn', 'gtfToGenePred'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp gtfToGenePred $PREFIX/bin; chmod +x $PREFIX/bin/gtfToGenePred')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which gtfToGenePred'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert a GTF file to a genePred')]))])), 'ucsc-hggcpercent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-hggcpercent/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-hggcpercent'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/hgGcPercent'), ('fn', 'hgGcPercent'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp hgGcPercent $PREFIX/bin; chmod +x $PREFIX/bin/hgGcPercent')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which hgGcPercent'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Calculate GC Percentage in 20kb windows')]))])), 'ucsc-liftover': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-liftover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-liftover'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/liftOver'), ('fn', 'liftOver'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'libpng', 'mysql', 'openssl', 'zlib']), ('run', ['libgcc', 'libpng', 'mysql', 'openssl', 'zlib'])])), ('test', CommentedMap([('commands', ['which liftOver'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Move annotations from one assembly to another')]))])), 'ucsc-maskoutfa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-maskoutfa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-maskoutfa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/maskOutFa'), ('fn', 'maskOutFa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp maskOutFa $PREFIX/bin; chmod +x $PREFIX/bin/maskOutFa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which maskOutFa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Produce a masked .fa file given an unmasked .fa and')]))])), 'ucsc-netclass': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-netclass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-netclass'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/netClass'), ('fn', 'netClass'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp netClass $PREFIX/bin; chmod +x $PREFIX/bin/netClass')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which netClass'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Add classification info to net')]))])), 'ucsc-netfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-netfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-netfilter'), ('version', '324')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/netFilter'), ('fn', 'netFilter'), ('md5', 'c49c168584c5917081d71df854e3cd9d'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp netFilter $PREFIX/bin; chmod +x $PREFIX/bin/netFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which netFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Filter out parts of net.  What passes')]))])), 'ucsc-netsyntenic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-netsyntenic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-netsyntenic'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/netSyntenic'), ('fn', 'netSyntenic'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp netSyntenic $PREFIX/bin; chmod +x $PREFIX/bin/netSyntenic')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which netSyntenic'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Add synteny info to net.')]))])), 'ucsc-nettoaxt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-nettoaxt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-nettoaxt'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/netToAxt'), ('fn', 'netToAxt'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp netToAxt $PREFIX/bin; chmod +x $PREFIX/bin/netToAxt')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which netToAxt'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert net (and chain) to axt.')]))])), 'ucsc-nettobed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-nettobed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-nettobed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/netToBed'), ('fn', 'netToBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp netToBed $PREFIX/bin; chmod +x $PREFIX/bin/netToBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which netToBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert target coverage of net to a bed file.')]))])), 'ucsc-newpythonprog': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-newpythonprog/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-newpythonprog'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/newPythonProg'), ('fn', 'newPythonProg'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp newPythonProg $PREFIX/bin; chmod +x $PREFIX/bin/newPythonProg')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which newPythonProg'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Make a skeleton for a new python program')]))])), 'ucsc-nibfrag': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-nibfrag/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-nibfrag'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/nibFrag'), ('fn', 'nibFrag'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp nibFrag $PREFIX/bin; chmod +x $PREFIX/bin/nibFrag')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which nibFrag'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Extract part of a nib file as .fa (all bases/gaps lower case by default)')]))])), 'ucsc-nibsize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-nibsize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-nibsize'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/nibSize'), ('fn', 'nibSize'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp nibSize $PREFIX/bin; chmod +x $PREFIX/bin/nibSize')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which nibSize'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'print size of nibs')]))])), 'ucsc-oligomatch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-oligomatch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-oligomatch'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/oligoMatch'), ('fn', 'oligoMatch'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp oligoMatch $PREFIX/bin; chmod +x $PREFIX/bin/oligoMatch')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which oligoMatch'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'find perfect matches in sequence.')]))])), 'ucsc-parafetch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-parafetch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-parafetch'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/paraFetch'), ('fn', 'paraFetch'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp paraFetch $PREFIX/bin; chmod +x $PREFIX/bin/paraFetch')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which paraFetch'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'try to fetch url with multiple connections')]))])), 'ucsc-parasync': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-parasync/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-parasync'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/paraSync'), ('fn', 'paraSync'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp paraSync $PREFIX/bin; chmod +x $PREFIX/bin/paraSync')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which paraSync'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'uses paraFetch to recursively mirror url to given path')]))])), 'ucsc-positionaltblcheck': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-positionaltblcheck/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-positionaltblcheck'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/positionalTblCheck'), ('fn', 'positionalTblCheck'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp positionalTblCheck $PREFIX/bin; chmod +x $PREFIX/bin/positionalTblCheck')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which positionalTblCheck'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'check that positional tables are sorted')]))])), 'ucsc-pslcat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslcat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslcat'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslCat'), ('fn', 'pslCat'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslCat $PREFIX/bin; chmod +x $PREFIX/bin/pslCat')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslCat'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'concatenate psl files')]))])), 'ucsc-pslcdnafilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslcdnafilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslcdnafilter'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslCDnaFilter'), ('fn', 'pslCDnaFilter'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslCDnaFilter $PREFIX/bin; chmod +x $PREFIX/bin/pslCDnaFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslCDnaFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', ' Filter cDNA alignments in psl format.  Filtering criteria are comparative, selecting near best in genome alignments for each given cDNA and non-comparative, based only on the quality of an individual alignment. ')]))])), 'ucsc-pslcheck': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslcheck/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslcheck'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslCheck'), ('fn', 'pslCheck'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslCheck $PREFIX/bin; chmod +x $PREFIX/bin/pslCheck')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslCheck'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'validate PSL files')]))])), 'ucsc-psldropoverlap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-psldropoverlap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-psldropoverlap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslDropOverlap'), ('fn', 'pslDropOverlap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslDropOverlap $PREFIX/bin; chmod +x $PREFIX/bin/pslDropOverlap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslDropOverlap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'deletes all overlapping self alignments. ')]))])), 'ucsc-pslfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslfilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslfilter'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslFilter'), ('fn', 'pslFilter'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslFilter $PREFIX/bin; chmod +x $PREFIX/bin/pslFilter')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslFilter'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'filter out psl file')]))])), 'ucsc-pslhisto': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslhisto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslhisto'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslHisto'), ('fn', 'pslHisto'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslHisto $PREFIX/bin; chmod +x $PREFIX/bin/pslHisto')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslHisto'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', ' Collect counts on PSL alignments for making histograms. These then be analyzed with R, textHistogram, etc. ')]))])), 'ucsc-pslliftsubrangeblat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslliftsubrangeblat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslliftsubrangeblat'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslLiftSubrangeBlat'), ('fn', 'pslLiftSubrangeBlat'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslLiftSubrangeBlat $PREFIX/bin; chmod +x $PREFIX/bin/pslLiftSubrangeBlat')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslLiftSubrangeBlat'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'lift PSLs from blat subrange alignments')]))])), 'ucsc-pslmap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslmap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslMap'), ('fn', 'pslMap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslMap $PREFIX/bin; chmod +x $PREFIX/bin/pslMap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslMap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'map PSLs alignments to new targets using alignments of')]))])), 'ucsc-pslmrnacover': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslmrnacover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslmrnacover'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslMrnaCover'), ('fn', 'pslMrnaCover'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslMrnaCover $PREFIX/bin; chmod +x $PREFIX/bin/pslMrnaCover')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslMrnaCover'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Make histogram of coverage percentage of mRNA in psl.')]))])), 'ucsc-pslpairs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslpairs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslpairs'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslPairs'), ('fn', 'pslPairs'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslPairs $PREFIX/bin; chmod +x $PREFIX/bin/pslPairs')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslPairs'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'join paired ends in psl alignments')]))])), 'ucsc-pslpartition': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslpartition/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslpartition'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslPartition'), ('fn', 'pslPartition'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslPartition $PREFIX/bin; chmod +x $PREFIX/bin/pslPartition')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslPartition'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'split PSL files into non-overlapping sets')]))])), 'ucsc-pslpostarget': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslpostarget/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslpostarget'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslPosTarget'), ('fn', 'pslPosTarget'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslPosTarget $PREFIX/bin; chmod +x $PREFIX/bin/pslPosTarget')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslPosTarget'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'flip psl strands so target is positive and implicit')]))])), 'ucsc-pslpretty': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslpretty/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslpretty'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslPretty'), ('fn', 'pslPretty'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslPretty $PREFIX/bin; chmod +x $PREFIX/bin/pslPretty')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslPretty'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert PSL to human-readable output')]))])), 'ucsc-pslrecalcmatch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslrecalcmatch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslrecalcmatch'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslRecalcMatch'), ('fn', 'pslRecalcMatch'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslRecalcMatch $PREFIX/bin; chmod +x $PREFIX/bin/pslRecalcMatch')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslRecalcMatch'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Recalculate match,mismatch,repMatch columns in psl file.')]))])), 'ucsc-pslreps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslreps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslreps'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslReps'), ('fn', 'pslReps'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslReps $PREFIX/bin; chmod +x $PREFIX/bin/pslReps')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslReps'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Analyze repeats and generate genome-wide best alignments from a')]))])), 'ucsc-pslscore': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslscore'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslScore'), ('fn', 'pslScore'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslScore $PREFIX/bin; chmod +x $PREFIX/bin/pslScore')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslScore'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'calculate web blat score from psl files')]))])), 'ucsc-pslselect': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslselect/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslselect'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslSelect'), ('fn', 'pslSelect'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslSelect $PREFIX/bin; chmod +x $PREFIX/bin/pslSelect')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslSelect'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'select records from a PSL file.')]))])), 'ucsc-pslsort': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslsort/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslsort'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslSort'), ('fn', 'pslSort'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslSort $PREFIX/bin; chmod +x $PREFIX/bin/pslSort')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslSort'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Merge and sort psCluster .psl output files')]))])), 'ucsc-pslstats': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslstats'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslStats'), ('fn', 'pslStats'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslStats $PREFIX/bin; chmod +x $PREFIX/bin/pslStats')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslStats'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'collect statistics from a psl file.')]))])), 'ucsc-pslswap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslswap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslswap'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslSwap'), ('fn', 'pslSwap'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslSwap $PREFIX/bin; chmod +x $PREFIX/bin/pslSwap')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslSwap'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', ' Swap target and query in psls ')]))])), 'ucsc-psltobed': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-psltobed/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-psltobed'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslToBed'), ('fn', 'pslToBed'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslToBed $PREFIX/bin; chmod +x $PREFIX/bin/pslToBed')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslToBed'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', ' transform a psl format file to a bed format file. ')]))])), 'ucsc-psltobigpsl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-psltobigpsl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-psltobigpsl'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslToBigPsl'), ('fn', 'pslToBigPsl'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslToBigPsl $PREFIX/bin; chmod +x $PREFIX/bin/pslToBigPsl')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslToBigPsl'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'converts psl to bigPsl input (bed format with extra fields)')]))])), 'ucsc-psltochain': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-psltochain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-psltochain'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslToChain'), ('fn', 'pslToChain'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslToChain $PREFIX/bin; chmod +x $PREFIX/bin/pslToChain')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslToChain'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert psl records to chain records ')]))])), 'ucsc-psltopslx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-psltopslx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-psltopslx'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslToPslx'), ('fn', 'pslToPslx'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslToPslx $PREFIX/bin; chmod +x $PREFIX/bin/pslToPslx')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslToPslx'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert from psl to pslx format, which includes sequences')]))])), 'ucsc-pslxtofa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-pslxtofa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-pslxtofa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/pslxToFa'), ('fn', 'pslxToFa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp pslxToFa $PREFIX/bin; chmod +x $PREFIX/bin/pslxToFa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which pslxToFa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert pslx (with sequence) to fasta file')]))])), 'ucsc-qacagplift': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-qacagplift/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-qacagplift'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/qacAgpLift'), ('fn', 'qacAgpLift'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp qacAgpLift $PREFIX/bin; chmod +x $PREFIX/bin/qacAgpLift')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which qacAgpLift'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Use AGP to combine per-scaffold qac into per-chrom qac.')]))])), 'ucsc-qactoqa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-qactoqa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-qactoqa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/qacToQa'), ('fn', 'qacToQa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp qacToQa $PREFIX/bin; chmod +x $PREFIX/bin/qacToQa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which qacToQa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert from compressed to uncompressed')]))])), 'ucsc-qactowig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-qactowig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-qactowig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/qacToWig'), ('fn', 'qacToWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp qacToWig $PREFIX/bin; chmod +x $PREFIX/bin/qacToWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which qacToWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert from compressed quality score format to wiggle format.')]))])), 'ucsc-qatoqac': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-qatoqac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-qatoqac'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/qaToQac'), ('fn', 'qaToQac'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp qaToQac $PREFIX/bin; chmod +x $PREFIX/bin/qaToQac')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which qaToQac'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'convert from uncompressed to compressed')]))])), 'ucsc-randomlines': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-randomlines/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-randomlines'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/randomLines'), ('fn', 'randomLines'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp randomLines $PREFIX/bin; chmod +x $PREFIX/bin/randomLines')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which randomLines'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Pick out random lines from file')]))])), 'ucsc-rasqlquery': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-rasqlquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-rasqlquery'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/raSqlQuery'), ('fn', 'raSqlQuery'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp raSqlQuery $PREFIX/bin; chmod +x $PREFIX/bin/raSqlQuery')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which raSqlQuery'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Do a SQL-like query on a RA file.')]))])), 'ucsc-ratolines': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-ratolines/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-ratolines'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/raToLines'), ('fn', 'raToLines'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp raToLines $PREFIX/bin; chmod +x $PREFIX/bin/raToLines')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which raToLines'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Output .ra file stanzas as single lines, with pipe-separated fields.')]))])), 'ucsc-ratotab': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-ratotab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-ratotab'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/raToTab'), ('fn', 'raToTab'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp raToTab $PREFIX/bin; chmod +x $PREFIX/bin/raToTab')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which raToTab'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert ra file to table.')]))])), 'ucsc-rmfadups': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-rmfadups/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-rmfadups'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/rmFaDups'), ('fn', 'rmFaDups'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp rmFaDups $PREFIX/bin; chmod +x $PREFIX/bin/rmFaDups')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which rmFaDups'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'remove duplicate records in FA file')]))])), 'ucsc-rowstocols': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-rowstocols/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-rowstocols'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/rowsToCols'), ('fn', 'rowsToCols'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp rowsToCols $PREFIX/bin; chmod +x $PREFIX/bin/rowsToCols')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which rowsToCols'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert rows to columns and vice versa in a text file.')]))])), 'ucsc-spacedtotab': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-spacedtotab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-spacedtotab'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/spacedToTab'), ('fn', 'spacedToTab'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp spacedToTab $PREFIX/bin; chmod +x $PREFIX/bin/spacedToTab')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which spacedToTab'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert fixed width space separated fields to tab separated')]))])), 'ucsc-splitfile': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-splitfile/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-splitfile'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/splitFile'), ('fn', 'splitFile'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp splitFile $PREFIX/bin; chmod +x $PREFIX/bin/splitFile')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which splitFile'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Split up a file')]))])), 'ucsc-splitfilebycolumn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-splitfilebycolumn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-splitfilebycolumn'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/splitFileByColumn'), ('fn', 'splitFileByColumn'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp splitFileByColumn $PREFIX/bin; chmod +x $PREFIX/bin/splitFileByColumn')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which splitFileByColumn'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Split text input into files named by column value')]))])), 'ucsc-sqltoxml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-sqltoxml/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-sqltoxml'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/sqlToXml'), ('fn', 'sqlToXml'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp sqlToXml $PREFIX/bin; chmod +x $PREFIX/bin/sqlToXml')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which sqlToXml'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'dump out all or part of a relational database to XML, guided')]))])), 'ucsc-stringify': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-stringify/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-stringify'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/stringify'), ('fn', 'stringify'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp stringify $PREFIX/bin; chmod +x $PREFIX/bin/stringify')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which stringify'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert file to C strings')]))])), 'ucsc-subchar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-subchar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-subchar'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/subChar'), ('fn', 'subChar'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp subChar $PREFIX/bin; chmod +x $PREFIX/bin/subChar')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which subChar'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Substitute one character for another throughout a file.')]))])), 'ucsc-subcolumn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-subcolumn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-subcolumn'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/subColumn'), ('fn', 'subColumn'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp subColumn $PREFIX/bin; chmod +x $PREFIX/bin/subColumn')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which subColumn'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Substitute one column in a tab-separated file.')]))])), 'ucsc-taillines': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-taillines/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-taillines'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/tailLines'), ('fn', 'tailLines'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp tailLines $PREFIX/bin; chmod +x $PREFIX/bin/tailLines')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which tailLines'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'add tail to each line of file')]))])), 'ucsc-tdbquery': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-tdbquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-tdbquery'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/tdbQuery'), ('fn', 'tdbQuery'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp tdbQuery $PREFIX/bin; chmod +x $PREFIX/bin/tdbQuery')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which tdbQuery'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Query the trackDb system using SQL syntax.')]))])), 'ucsc-texthistogram': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-texthistogram/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-texthistogram'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/textHistogram'), ('fn', 'textHistogram'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp textHistogram $PREFIX/bin; chmod +x $PREFIX/bin/textHistogram')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which textHistogram'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Make a histogram in ascii')]))])), 'ucsc-ticktodate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-ticktodate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-ticktodate'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/tickToDate'), ('fn', 'tickToDate'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp tickToDate $PREFIX/bin; chmod +x $PREFIX/bin/tickToDate')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which tickToDate'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert seconds since 1970 to time and date')]))])), 'ucsc-tolower': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-tolower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-tolower'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/toLower'), ('fn', 'toLower'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp toLower $PREFIX/bin; chmod +x $PREFIX/bin/toLower')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which toLower'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert upper case to lower case in file. Leave other chars alone')]))])), 'ucsc-toupper': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-toupper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-toupper'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/toUpper'), ('fn', 'toUpper'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp toUpper $PREFIX/bin; chmod +x $PREFIX/bin/toUpper')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which toUpper'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert lower case to upper case in file. Leave other chars alone')]))])), 'ucsc-trfbig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-trfbig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-trfbig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/trfBig'), ('fn', 'trfBig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp trfBig $PREFIX/bin; chmod +x $PREFIX/bin/trfBig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which trfBig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Mask tandem repeats on a big sequence file.')]))])), 'ucsc-twobitdup': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-twobitdup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-twobitdup'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/twoBitDup'), ('fn', 'twoBitDup'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp twoBitDup $PREFIX/bin; chmod +x $PREFIX/bin/twoBitDup')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which twoBitDup'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'check to see if a twobit file has any identical sequences in it')]))])), 'ucsc-twobitinfo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-twobitinfo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-twobitinfo'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/twoBitInfo'), ('fn', 'twoBitInfo'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp twoBitInfo $PREFIX/bin; chmod +x $PREFIX/bin/twoBitInfo')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which twoBitInfo'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'get information about sequences in a .2bit file')]))])), 'ucsc-twobitmask': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-twobitmask/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-twobitmask'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/twoBitMask'), ('fn', 'twoBitMask'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp twoBitMask $PREFIX/bin; chmod +x $PREFIX/bin/twoBitMask')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which twoBitMask'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'apply masking to a .2bit file, creating a new .2bit file')]))])), 'ucsc-twobittofa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-twobittofa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-twobittofa'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/twoBitToFa'), ('fn', 'twoBitToFa'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp twoBitToFa $PREFIX/bin; chmod +x $PREFIX/bin/twoBitToFa')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which twoBitToFa'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert all or part of .2bit file to fasta')]))])), 'ucsc-validatefiles': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-validatefiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-validatefiles'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/validateFiles'), ('fn', 'validateFiles'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp validateFiles $PREFIX/bin; chmod +x $PREFIX/bin/validateFiles')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which validateFiles'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Validates the format of different genomic files.')]))])), 'ucsc-validatemanifest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-validatemanifest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-validatemanifest'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/validateManifest'), ('fn', 'validateManifest'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp validateManifest $PREFIX/bin; chmod +x $PREFIX/bin/validateManifest')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which validateManifest'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Validates the ENCODE3 manifest.txt file.')]))])), 'ucsc-wigcorrelate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-wigcorrelate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-wigcorrelate'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/wigCorrelate'), ('fn', 'wigCorrelate'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp wigCorrelate $PREFIX/bin; chmod +x $PREFIX/bin/wigCorrelate')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which wigCorrelate'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Produce a table that correlates all pairs of wigs.')]))])), 'ucsc-wigtobigwig': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-wigtobigwig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-wigtobigwig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/wigToBigWig'), ('fn', 'wigToBigWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp wigToBigWig $PREFIX/bin; chmod +x $PREFIX/bin/wigToBigWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which wigToBigWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert ascii format wig file (in fixedStep, variableStep')]))])), 'ucsc-wordline': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-wordline/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-wordline'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/wordLine'), ('fn', 'wordLine'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp wordLine $PREFIX/bin; chmod +x $PREFIX/bin/wordLine')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which wordLine'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'chop up words by white space and output them with one')]))])), 'ucsc-xmlcat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-xmlcat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-xmlcat'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/xmlCat'), ('fn', 'xmlCat'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp xmlCat $PREFIX/bin; chmod +x $PREFIX/bin/xmlCat')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which xmlCat'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Concatenate xml files together, stuffing all records inside a single outer tag. ')]))])), 'ucsc-xmltosql': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-xmltosql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-xmltosql'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/xmlToSql'), ('fn', 'xmlToSql'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp xmlToSql $PREFIX/bin; chmod +x $PREFIX/bin/xmlToSql')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which xmlToSql'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert XML dump into a fairly normalized relational database')]))])), 'udocker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/udocker/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/indigo-dc/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://github.com/indigo-dc/udocker'), ('license', 'Apache-2.0'), ('license_file', 'LICENSE'), ('summary', 'Freely available tools for computational molecular biology.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pycurl']), ('run', ['python', 'proot', 'simplejson', 'pycurl'])]))])), 'umi_tools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/umi_tools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'umi_tools'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/u/umi_tools/umi_tools-<{ version }}.tar.gz'), ('sha256', 'a535a0a67d30c98fa2d29f8aa5f3b40cd94c2058073a7541be08383964672912'), ('patches', ['umi_tools.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'matplotlib', 'scipy', 'regex', 'freetype', 'certifi ==2016.9.26', 'setuptools >=1.1', 'cython >=0.19', 'numpy >=1.7', 'pandas >=0.12.0', 'pysam >=0.8.4', 'future']), ('run', ['python', 'matplotlib', 'scipy', 'regex', 'freetype', 'certifi ==2016.9.26', 'cython >=0.19', 'numpy >=1.7', 'pandas >=0.12.0', 'pysam >=0.8.4', 'future'])])), ('test', CommentedMap([('commands', ['umi_tools --help'])])), ('about', CommentedMap([('home', 'https://github.com/CGATOxford/UMI-tools'), ('license', 'MIT License'), ('summary', 'Tools for dealing with Unique Molecular Identifiers (UMIs) / Random Molecular Tags (RMTs)')]))])), 'umis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/umis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'umis'), ('version', '0.7.0a')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'umis-7fa923b.tar.gz'), ('url', 'https://github.com/vals/umis/archive/7fa923b.tar.gz'), ('md5', '816a3f0f6df9de1e80733ee25bfadcde')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'setuptools', 'pandas', 'click', 'pysam', 'regex', 'cython', 'toolz', 'scipy']), ('run', ['python', 'libgcc', 'pandas', 'click', 'pysam', 'regex', 'cython', 'toolz', 'scipy'])])), ('test', CommentedMap([('commands', ['LANG=C.UTF-8 umis --help'])])), ('about', CommentedMap([('home', 'https://github.com/vals/umis'), ('license', 'MIT'), ('summary', 'Tools for processing UMI RNA-tag data')]))])), 'unicycler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/unicycler/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/rrwick/Unicycler/archive/v<{ version }}.tar.gz'), ('sha256', '00200ea13c2ce90ba7b55917060ffb2e425f59ba515423fa516ff3d6519af336'), ('patches', ['Makefile.patch [linux]', 'misc.py.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc >=4.9.4']), ('run', ['python', 'libgcc', 'spades >=3.6.2', 'pilon', 'openjdk', 'bowtie2', 'samtools >=1.0', 'blast', 'freebayes'])])), ('test', CommentedMap([('commands', ['unicycler -h', 'unicycler_align -h', 'unicycler_check -h', 'unicycler_polish -h'])])), ('about', CommentedMap([('home', 'https://github.com/rrwick/Unicycler'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Hybrid assembly pipeline for bacterial genomes')]))])), 'uniprot': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/uniprot/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/41/82/7d57f68ba49394c86f58792f57c47a79d7890589a911cbcaee84fa6081f4/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests']), ('run', ['python', 'requests'])])), ('test', CommentedMap([('imports', ['uniprot'])])), ('about', CommentedMap([('home', 'http://github.com/boscoh/uniprot'), ('license', 'BSD'), ('summary', 'Retrieve protein sequence identifiers and metadata from http://uniprot.org'), ('license_family', 'BSD')]))])), 'uropa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/uropa/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/<{ name[0] }}/<{ name }}/<{ name }}-<{ version }}.tar.gz'), ('fn', '<{ name }}-<{ version }}.tar.gz'), ('md5', '10e53bad46946c6c5f29a577de1220a7')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'pysam', 'r-base', 'r-ggplot2', 'r-gplots', 'r-gridextra', 'r-jsonlite', 'r-venndiagram', 'r-getopt', 'r-tidyr', 'r-upsetr', 'bioconductor-graph', 'bioconductor-rbgl'])])), ('test', CommentedMap([('commands', ['uropa -h', 'uropa_summary.R -h', 'uropa_reformat_output.R -h', 'uropa2gtf.R -h'])])), ('about', CommentedMap([('home', 'https://github.molgen.mpg.de/loosolab/UROPA'), ('doc_url', 'http://uropa-manual.readthedocs.io'), ('license', 'MIT'), ('summary', 'UROPA (Universal RObust Peak Annotator) is a command line based tool, intended for genomic region annotation from e.g. peak calling.\\nIt detects the most appropriate annotation by taking parameters such as feature type, anchor, direction and strand into account.\\nFurthermore, it allows filtering for GTF attribute values, e.g. protein_coding.\\n')])), ('extra', CommentedMap([('recipe-maintainers', ['jenzopr'])]))])), 'ushuffle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ushuffle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ushuffle'), ('version', '1.2.2')])), ('about', CommentedMap([('home', 'http://digital.cs.usu.edu/~mjiang/ushuffle/'), ('license', 'custom'), ('summary', 'uShuffle---a useful tool for shuffling biological sequences while preserving the k-let counts')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ushuffle-v1.2.2.tar.gz'), ('url', 'https://github.com/s-will/ushuffle/archive/v1.2.2.tar.gz'), ('sha256', 'a9cd1c7126265a87bc150b8a5924f5b5ab5f8604c6678d3d470fd93998db5ab9')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['ushuffle --help', 'python -c \"import ushuffle; s=(chr(65)*2+chr(66))*3; print(s,ushuffle.shuffle(s,len(s),2))\"'])]))])), 'validate-fasta-database': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/validate-fasta-database/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'validate-fasta-database'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/caleb-easterly/validate_fasta_database/releases/download/v<{ version }}/validate_fasta_database-<{ version }}.jar'), ('md5', '04fa47141882ee32f5cdaec93e5a890b')])), ('requirements', CommentedMap([('build', ['openjdk >=7']), ('run', ['openjdk >=7', 'python'])])), ('test', CommentedMap([('commands', ['echo -e \">abc|001\\\\nMMCTARRK\" > test.fasta && validate-fasta-database test.fasta good.fasta bad.fasta false true \"\" false 0'])])), ('about', CommentedMap([('home', 'https://github.com/caleb-easterly/validate_fasta_database'), ('license', 'GPL-3.0'), ('summary', 'Code for Galaxy tool for quality control on FASTA database')]))])), 'validators': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/validators/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'validators'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'validators-0.10.tar.gz'), ('url', 'https://pypi.python.org/packages/source/v/validators/validators-0.10.tar.gz'), ('md5', '9ddd9d77aadc047723080da500e0bbf2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six >=1.4.0', 'decorator >=3.4.0']), ('run', ['python', 'six >=1.4.0', 'decorator >=3.4.0'])])), ('test', CommentedMap([('imports', ['validators', 'validators.i18n'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/validators'), ('license', 'BSD License'), ('summary', 'Python Data Validation for Humans.')]))])), 'validictory': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/validictory/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'validictory'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'validictory-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/v/validictory/validictory-1.0.1.tar.gz'), ('md5', '59b619e446e7f8b510c2b76f0973b2e8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['validictory', 'validictory.tests'])])), ('about', CommentedMap([('home', 'http://github.com/jamesturk/validictory'), ('license', 'BSD License'), ('summary', 'general purpose python data validator')]))])), 'vardict': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vardict/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vardict'), ('version', '2017.04.18')])), ('source', CommentedMap([('fn', 'vardict-328e00a.tar.gz'), ('url', 'https://github.com/AstraZeneca-NGS/VarDict/archive/328e00a.tar.gz'), ('md5', '0a33c031b65bacfe62c405f4c8b24619')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['vardict -h'])])), ('about', CommentedMap([('home', 'https://github.com/AstraZeneca-NGS/VarDict'), ('license', 'MIT'), ('summary', 'A sensitive variant caller for both single and paired sample variant calling')]))])), 'vardict-java': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vardict-java/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vardict-java'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'VarDict-<{ version }}.zip'), ('url', 'https://github.com/AstraZeneca-NGS/VarDictJava/releases/download/v<{ version }}/VarDict-<{ version }}.zip'), ('sha256', 'fb4cedfbb7f03514030e07fc0cfc77f7286a14c02bcdc0493bb7e165656db6ac')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['vardict-java -h'])])), ('about', CommentedMap([('home', 'https://github.com/AstraZeneca-NGS/VarDictJava'), ('license', 'MIT'), ('summary', 'Java port of the VarDict variant discovery program')]))])), 'variant-effect-predictor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/variant-effect-predictor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'variant-effect-predictor'), ('version', '87')])), ('source', CommentedMap([('fn', 'vep-87.zip'), ('url', 'https://github.com/Ensembl/ensembl-tools/archive/release/87.zip')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gnu-wget', 'gcc', 'perl-threaded', 'perl-file-copy-recursive', 'perl-archive-extract', 'perl-archive-zip', 'perl-lwp-simple', 'perl-lwp-protocol-https', 'zlib', 'htslib', 'unzip']), ('run', ['perl-threaded', 'perl-bioperl ==1.6.924 4', 'perl-bio-db-hts', 'perl-encode-locale', 'perl-archive-extract', 'perl-archive-zip', 'perl-cgi', 'perl-compress-raw-zlib', 'perl-dbi', 'perl-dbd-mysql', 'perl-file-fetch', 'perl-file-sharedir', 'perl-file-sharedir-install', 'perl-file-copy-recursive', 'perl-ipc-system-simple', 'perl-lwp-simple', 'perl-lwp-protocol-https', 'perl-statistics-descriptive', 'perl-time-hires', 'zlib', 'htslib'])])), ('test', CommentedMap([('commands', ['variant_effect_predictor.pl --help', 'vep_convert_cache.pl --help', 'vep_install.pl --help'])])), ('about', CommentedMap([('home', 'http://www.ensembl.org/info/docs/tools/vep/index.html'), ('license', 'Apache 2.0'), ('summary', 'The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.')])), ('extra', CommentedMap([('notes', \"This package installs only the variant effect predictor (VEP) library\\ncode. To install data libraries, you can use the 'vep_install.pl' command\\ninstalled along with it. For example, to install the VEP library for human\\nhg19/GRCh37 to a directory\\n\\nvep_install.pl -a cf -s homo_sapiens -y GRCh37 -c /output/path/to/hg19/vep\\nvep_convert_cache.pl -species homo_sapiens -version 86_GRCh37 -d /output/path/to/hg19/vep\\n\\n(note that vep_install.pl is renamed from INSTALL.pl\\n and vep_convert_cache.pl from covert_cache.pl\\n to avoid having generic script names in the PATH)\\n\\nThe convert cache step is not required but improves lookup speeds during\\nruns. See the VEP documentation for more details\\n\\nhttp://useast.ensembl.org/info/docs/tools/vep/script/vep_cache.html\\n\")]))])), 'variantbam': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/variantbam/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'variantbam'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('git_url', 'https://github.com/walaj/VariantBam.git'), ('git_rev', '<{ revision }}'), ('md5', 'unused')])), ('requirements', CommentedMap([('build', ['gcc', 'xz', 'bzip2', 'zlib']), ('run', ['libgcc', 'xz', 'bzip2', 'zlib'])])), ('test', CommentedMap([('commands', ['variant --help 2>&1 | grep \"Usage: variant\"'])])), ('about', CommentedMap([('home', 'https://github.com/jwalabroad/VariantBam'), ('license', 'GPLv3'), ('summary', 'Filtering and profiling of next-generational sequencing data using region-specific rules')]))])), 'varscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/varscan/2.4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://dkoboldt.github.io/varscan/'), ('license', 'The Non-Profit Open Software License version 3.0 (NPOSL-3.0)'), ('summary', 'variant detection in massively parallel sequencing data')])), ('package', CommentedMap([('name', 'varscan'), ('version', '2.4.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'VarScan-2.4.2.jar'), ('url', 'https://github.com/dkoboldt/varscan/raw/master/VarScan.v2.4.2.jar'), ('md5', '4b810741505a8145a7f8f9f6791bbacf')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['varscan mpileup2cns --help'])]))])), 'vawk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vawk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vawk'), ('version', '0.0.2')])), ('source', CommentedMap([('fn', 'vawk-bab8237.tar.gz'), ('url', 'https://github.com/cc2qe/vawk/archive/bab8237.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'gawk'])])), ('test', CommentedMap([('commands', ['vawk --help'])])), ('about', CommentedMap([('home', 'https://github.com/cc2qe/vawk'), ('license', 'MIT'), ('summary', 'An awk-like VCF parser')]))])), 'vcf2db': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcf2db/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcf2db'), ('version', '2017.09.14')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'vcf2db-b5251d9.tar.gz'), ('url', 'https://github.com/quinlan-lab/vcf2db/archive/b5251d9.tar.gz'), ('sha256', '6377b1fcc55b1cbf438e69c7e18092ee1c4cfd0a516ffed582924bf11fa38663')])), ('requirements', CommentedMap([('build', ['python', 'perl', 'nomkl', 'snappy', 'python-snappy', 'sqlalchemy', 'cyvcf2', 'numpy', 'peddy >=0.2.9', 'geneimpacts >0.2.0']), ('run', ['python', 'nomkl', 'snappy', 'python-snappy', 'sqlalchemy', 'cyvcf2', 'numpy', 'peddy >=0.2.9', 'geneimpacts >0.2.0'])])), ('test', CommentedMap([('commands', ['vcf2db.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/quinlan-lab/vcf2db'), ('license', 'MIT'), ('summary', 'Create a gemini-compatible database from a VCF')]))])), 'vcf2maf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcf2maf/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/mskcc/vcf2maf/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}')])), ('requirements', CommentedMap([('build', ['perl', 'variant-effect-predictor', 'samtools', 'htslib']), ('run', ['perl', 'variant-effect-predictor', 'samtools', 'htslib'])])), ('test', CommentedMap([('commands', ['vcf2maf.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/mskcc/vcf2maf'), ('license', 'Apache-2.0'), ('summary', 'Convert a VCF into a MAF where each variant is annotated to only one of all possible gene isoforms')])), ('extra', CommentedMap([('notes', 'This package installs only vcf2maf and does not integrate with the variant-effect-predictor (VEP). To\\ndo so, please follow the instructions at https://github.com/mskcc/vcf2maf/blob/master/README.md.\\n')]))])), 'vcfanno': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcfanno/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcfanno'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'vcfanno_linux'), ('url', 'https://github.com/brentp/vcfanno/releases/download/v<{ version }}/vcfanno_linux64'), ('md5', 'e5c665b0ffee8cb09a732768cdaae733')])), ('build', CommentedMap([('number', 0)])), ('requirements', None), ('test', CommentedMap([('commands', ['vcfanno'])])), ('about', CommentedMap([('home', 'https://github.com/brentp/vcfanno'), ('license', 'MIT'), ('summary', 'annotate a VCF with other VCFs/BEDs/tabixed files')]))])), 'vcfkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcfkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcfkit'), ('version', '0.0.4')])), ('source', CommentedMap([('fn', 'vcfkit-0.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/52/49/d0ff24eb7ff1cb9907c76b56da19dc94f5161bb750a13cd3f010300e95fc/vcfkit-0.0.4.tar.gz'), ('md5', '25af9501bb7691f286645cb80ad591d8')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['vk=vcfkit.vk:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'awesome-slugify', 'matplotlib', 'scipy', 'numpy', 'cython', 'cyvcf2 >=0.1.9', 'docopt', 'biopython', 'yahmm', 'clint', 'requests', 'intervaltree', 'tabulate', 'yahmm', 'jinja2', 'pytest-runner']), ('run', ['python', 'setuptools', 'awesome-slugify', 'matplotlib', 'scipy', 'numpy', 'cython', 'cyvcf2 >=0.1.9', 'docopt', 'biopython', 'yahmm', 'clint', 'requests', 'intervaltree', 'tabulate', 'yahmm', 'jinja2', 'pytest-runner'])])), ('test', CommentedMap([('imports', ['vcfkit', 'vcfkit.utils']), ('commands', ['vk --help']), ('requires', ['coveralls', 'pytest'])])), ('about', CommentedMap([('home', 'https://github.com/AndersenLab/VCF-kit'), ('license', 'MIT License'), ('summary', 'Assorted utilities for the variant call format'), ('license_family', 'MIT')]))])), 'vcflatten': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcflatten/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'vcflatten'), ('version', '0.5.2')])), ('source', CommentedMap([('fn', 'vcflatten-0.5.2.zip'), ('url', 'https://github.com/downloads/innovativemedicine/vcfimp/vcflatten-0.5.2.zip'), ('md5', '69864df8a5a228a8fea73e0df5ddf394')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['vcflatten --help | grep vcflatten &> /dev/null'])])), ('about', CommentedMap([('home', 'http://innovativemedicine.ca/tools/vcflatten'), ('license', 'BSD'), ('summary', 'A command line tool for flattening VCF files down to simpler TSV files.')]))])), 'vcflib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcflib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcflib'), ('version', '1.0.0_rc1')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/vcflib.git'), ('git_rev', '3e03c7a10098b3888aaa258d4e339ddcdbf3b951')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['vcfallelicprimitives -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/vcflib'), ('license', 'MIT'), ('summary', 'Command-line tools for manipulating VCF files')]))])), 'vcftoolbox': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcftoolbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcftoolbox'), ('version', '0.1')])), ('source', CommentedMap([('fn', 'vcftoolbox-0.1.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/37/9b/7886bcd25c337ba7de136a43b7e879d249abd3cba10e7da1e028fad12c73/vcftoolbox-0.1.tar.gz'), ('md5', '1b0c44d931b7f1e39befa94e37571106')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest']), ('run', ['python', 'pytest'])])), ('test', CommentedMap([('imports', ['vcftoolbox'])])), ('about', CommentedMap([('home', 'http://github.com/moonso/vcftoolbox'), ('license', 'MIT License'), ('summary', 'Tools for manipulating and parsing vcf files')]))])), 'vcftools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcftools/0.1.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcftools'), ('version', '0.1.11')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['zlib', 'perl', 'perl-vcftools-vcf ==0.840']), ('run', ['zlib', 'perl-vcftools-vcf ==0.840'])])), ('test', CommentedMap([('commands', ['vcftools -h'])])), ('about', CommentedMap([('home', 'https://vcftools.github.io/'), ('license', 'LGPL'), ('summary', 'A set of tools written in Perl and C++ for working with VCF files.')]))])), 'vdjer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vdjer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vdjer'), ('version', '0.12')])), ('source', CommentedMap([('url', 'https://github.com/mozack/vdjer/archive/v0.12.tar.gz'), ('sha256', '09d407c00e29efdc05d11b3b24955d3a15e436cb00974db7cb7390db3fc3bdba')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', [\"vdjer 2>&1 | grep 'ELAPSED_SEC'\"])])), ('about', CommentedMap([('home', 'https://github.com/mozack/vdjer'), ('license', 'https://github.com/mozack/vdjer/blob/master/LICENSE.txt'), ('summary', 'B Cell Receptor Repertoire Reconstruction from short read mRNA-Seq data')]))])), 'velvet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/velvet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'velvet'), ('version', '1.2.10')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'velvet_1.2.10.tgz'), ('url', 'https://www.ebi.ac.uk/~zerbino/velvet/velvet_1.2.10.tgz'), ('sha256', '884dd488c2d12f1f89cdc530a266af5d3106965f21ab9149e8cb5c633c977640')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['velvetg | grep velvetg > /dev/null', 'velveth | grep velveth > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/~zerbino/velvet/'), ('license', 'GPL'), ('summary', 'Sequence Assembler for short reads')]))])), 'velvet-sc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/velvet-sc/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'velvet-sc.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://bix.ucsd.edu/projects/singlecell/velvet-sc.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['velvetg --help', 'velveth --help', 'shuffleSequences_fasta.pl', 'shuffleSequences_fastq.pl'])])), ('about', CommentedMap([('home', 'http://bix.ucsd.edu/projects/singlecell/'), ('license', 'GPL2'), ('summary', 'Efficient de novo assembly of single-cell bacterial genomes from short-read data sets')]))])), 'verifybamid': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/verifybamid/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}LibStatGen.<{ version }}.tgz'), ('url', 'https://github.com/statgen/<{ name }}/releases/download/v<{ version }}/<{ name }}LibStatGen.<{ version }}.tgz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['verifyBamID -h 2>&1 | grep -q verifyBamID'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/VerifyBamID'), ('license', 'GPL3'), ('summary', 'verifyBamID verifies identity and purity of sequence data')]))])), 'verse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/verse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'verse'), ('version', '0.1.5')])), ('source', CommentedMap([('git_url', 'https://github.com/qinzhu/VERSE.git'), ('git_rev', 'e390069b3d1c44340c086c52f5a9c977c8ab0216')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['verse -v > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/qinzhu/VERSE'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('summary', 'VERSE: a versatile and efficient RNA-Seq read counting tool')]))])), 'viennarna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/viennarna/2.3.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.5.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.5.tar.gz'), ('sha256', '26b62a00da21bc5597b580ab8fef4e624234ec446d7d3cb0ce22803a5d7074ca')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'python >3']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'custom'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))])), 'virema': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/virema/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'virema'), ('version', '0.6')])), ('source', CommentedMap([('fn', 'ViReMa_0.6.zip'), ('url', 'https://sourceforge.net/projects/virema/files/ViReMa_0.6/ViReMa_0.6.zip'), ('md5', '2e15fcea133a64b4b043dba39d87f289'), ('patches', ['path.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python', 'bowtie <=1.0.0'])])), ('test', CommentedMap([('commands', ['ViReMa.py -h', 'Compiler_Module.py -h'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/virema/'), ('license', 'Custom OSS'), ('summary', 'ViReMa (Viral Recombination Mapper) detects and reports recombination or fusion events in virus genomes using deep sequencing datasets.')]))])), 'virmet': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/virmet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'virmet'), ('version', 'v1.1.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/ozagordi/VirMet/archive/v1.1.1.tar.gz'), ('md5', 'be64fa64b5c9b9033ac9282fe7e37fac')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['biopython', 'blast >=2.3', 'bwa', 'entrez-direct', 'htslib', 'pandas', 'prinseq', 'python', 'r-ggplot2', 'samtools >=1.3', 'seqtk'])])), ('about', CommentedMap([('home', 'http://virmet.readthedocs.io'), ('summary', 'A pipeline for viral metagenomics'), ('license', 'MIT')])), ('test', CommentedMap([('commands', ['virmet fetch -h', 'virmet wolfpack -h', 'virmet --help'])]))])), 'visceral-evaluatesegmentation': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/visceral-evaluatesegmentation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'visceral-evaluatesegmentation'), ('version', '2015.07.02')])), ('source', CommentedMap([('fn', 'master.zip'), ('url', 'https://github.com/Visceral-Project/EvaluateSegmentation/archive/master.zip'), ('md5', '62128cfcee35060627658eef93da037a')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'itk', 'cmake']), ('run', ['libgcc', 'itk', 'jpeg'])])), ('test', CommentedMap([('commands', ['EvaluateSegmentation --help'])])), ('about', CommentedMap([('home', 'https://github.com/Visceral-Project/EvaluateSegmentation'), ('license', 'Apache License, Version 2.0'), ('summary', 'EvaluateSegmentation is a tool that compares two volumes (a test segmentation and a ground truth segmentation) using 22 different metrics that were selected as a result of a comprehensive research into the metrics used in the medical volume segmentations.')]))])), 'visitor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/visitor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'visitor'), ('version', '0.1.2')])), ('source', CommentedMap([('fn', 'visitor-0.1.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/v/visitor/visitor-0.1.2.tar.gz'), ('md5', 'e9471b44e5d426b0adf5d27694598be0')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['visitor'])])), ('about', CommentedMap([('home', 'http://github.com/mbr/visitor'), ('license', 'MIT'), ('summary', 'A tiny pythonic visitor implementation.')]))])), 'vispr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vispr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vispr'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'vispr-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/v/vispr/vispr-<{ version }}.tar.gz'), ('sha256', 'dc2cf6ac9c8930b0f1f1b3a2b7f57bfae180b2ac3674060123fd2d7fe10ccd82')])), ('build', CommentedMap([('skip', True), ('number', 1), ('entry_points', ['vispr = vispr.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'flask', 'pandas >=0.17.1,<0.20', 'pyyaml', 'scipy', 'scikit-learn', 'appdirs']), ('run', ['python', 'numpy', 'flask', 'pandas >=0.17.1,<0.20', 'pyyaml', 'scipy', 'scikit-learn', 'appdirs', 'libgcc'])])), ('test', CommentedMap([('commands', ['vispr --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/liulab/vispr'), ('license', 'MIT License'), ('summary', 'VISPR is a visualization framework and analysis workflow for CRISPR/Cas9 knockout screens. VISPR is designed to display results calculated by MAGeCK.')]))])), 'vnl': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vnl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vnl'), ('version', '<{ VXL_VERSION }}')])), ('source', CommentedMap([('fn', 'v<{VXL_VERSION}}.zip'), ('url', 'https://github.com/vxl/vxl/archive/v<{VXL_VERSION}}.zip'), ('md5', 'c0065d212bf58b022b94b65026e75506')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'cmake']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://sf.net/projects/vxl/'), ('license', 'BSD'), ('summary', 'A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.')]))])), 'voluptuous': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/voluptuous/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'voluptuous'), ('version', '0.8.8')])), ('source', CommentedMap([('fn', 'voluptuous-0.8.8.tar.gz'), ('url', 'https://pypi.python.org/packages/source/v/voluptuous/voluptuous-0.8.8.tar.gz'), ('md5', 'd39128d69dce5395c8a3c5c6474bf740')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['voluptuous'])])), ('about', CommentedMap([('home', 'https://github.com/alecthomas/voluptuous'), ('license', 'BSD'), ('summary', 'Python data validation library')]))])), 'vphaser2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vphaser2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/v-phaser-2'), ('license', 'single user license for academic non-commercial research purposes only'), ('license_file', 'V-Phaser-2.0/license/V_Phaser_2_License_v20130317.pdf'), ('summary', 'V-Phaser 2 is a tool to call variants in genetically heterogeneous populations from ultra-deep sequence data')])), ('package', CommentedMap([('name', 'vphaser2'), ('version', '2.0')])), ('source', CommentedMap([('url', 'https://github.com/broadinstitute/v-phaser2/archive/6b5483a8e1a9bf24061a0aa211fee18e6b845668.tar.gz'), ('md5', 'fd22b1c4403903576e62defb51b6ae32')])), ('build', CommentedMap([('number', 4), ('rpaths', ['$PREFIX/opt/$PKG_NAME-$PKG_VERSION', 'lib/'])])), ('requirements', CommentedMap([('build', ['bamtools', 'zlib']), ('run', ['bamtools', 'zlib'])])), ('test', CommentedMap([('commands', ['vphaser2 | grep \"BAM file\"'])]))])), 'vsearch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/torognes/vsearch/archive/v<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'm4', 'gcc', 'llvm', 'perl', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['vsearch --version'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/vsearch'), ('license', 'dual-licensed under GPLv3+ or BSD 2-clause'), ('summary', 'a versatile open source tool for metagenomics (USEARCH alternative)')]))])), 'vt': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vt'), ('version', '2015.11.10')])), ('source', CommentedMap([('fn', 'vt-15f90de5.tar.gz'), ('url', 'https://github.com/atks/vt/archive/15f90de5.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['vt'])])), ('about', CommentedMap([('home', 'https://github.com/atks/vt'), ('license', 'MIT'), ('summary', 'A tool set for short variant discovery in genetic sequence data')]))])), 'w4mclassfilter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/w4mclassfilter/0.98.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'w4mclassfilter'), ('version', '0.98.2')])), ('source', CommentedMap([('fn', 'w4mclassfilter_0.98.2.tar.gz'), ('url', 'https://github.com/HegemanLab/w4mclassfilter/releases/download/v0.98.2/w4mclassfilter_0.98.2.tar.gz'), ('md5', '8520732b9903cd2cee3be0cedd31db38')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(w4mclassfilter)\"'])])), ('about', CommentedMap([('home', 'https://github.com/HegemanLab/w4mclassfilter'), ('license', 'MIT'), ('summary', 'Filter Workflow4Metabolomics dataMatrix, sampleMetadata, and variableMetadata files by sample-class, imputing zero for NA values and eliminating zero-variance rows and columns from the data-matrix. MIT Licence allows redistribution.')]))])), 'w4mclstrpeakpics': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/w4mclstrpeakpics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'w4mclstrpeakpics'), ('version', '0.98.1')])), ('source', CommentedMap([('fn', 'w4mclstrpeakpics_0.98.1.tar.gz'), ('url', 'https://github.com/HegemanLab/w4mclstrpeakpics/releases/download/v0.98.1/w4mclstrpeakpics_0.98.1.tar.gz'), ('md5', 'de7ffdeb74972b8b8d8ebc17ae47f532')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-batch', 'r-reshape2', 'r-sqldf']), ('run', ['r-base', 'r-batch', 'r-reshape2', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(w4mclstrpeakpics); library(batch); library(grDevices); library(sqldf); library(gsubfn); options(gsubfn.engine = \\\\\"R\\\\\"); print(\\\\\"library(sqldf)\\\\\"); library(sqldf); print(\\\\\"test completed\\\\\")\"'])])), ('about', CommentedMap([('home', 'https://github.com/HegemanLab/w4mclstrpeakpics'), ('license', 'MIT'), ('summary', 'Visualize W4M sample-cluster peaks - Produce a figure to assess the similarities and differences among peaks in a cluster of samples using XCMS-preprocessed data files as input. MIT Licence allows redistribution.')]))])), 'wdltool': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wdltool/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/broadinstitute/wdltool'), ('license', 'BSD'), ('summary', 'Command line utilities for interacting with WDL')])), ('package', CommentedMap([('name', 'wdltool'), ('version', 0.9)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'wdltool-0.9.zip'), ('url', 'https://github.com/broadinstitute/wdltool/archive/0.9.zip'), ('md5', '29fa21e53445f6729c4897efc575635a')])), ('requirements', CommentedMap([('build', ['openjdk >=8', 'sbt', 'scala']), ('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', ['wdltool --help 2>&1 | grep \"Performs full validation\"'])]))])), 'web.py': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/web.py/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'web.py'), ('version', 0.37)])), ('source', CommentedMap([('fn', 'web.py-0.37.tar.gz'), ('url', 'https://pypi.python.org/packages/source/w/web.py/web.py-0.37.tar.gz'), ('md5', '93375e3f03e74d6bf5c5096a4962a8db')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['web'])])), ('about', CommentedMap([('home', 'http://webpy.org/'), ('license', 'Public domain'), ('summary', 'web.py makes web apps')]))])), 'weblogo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/weblogo/2.8.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://weblogo.berkeley.edu/'), ('license', 'MIT'), ('summary', 'Web based application designed to generate sequence logos')])), ('package', CommentedMap([('name', 'weblogo'), ('version', '2.8.2')])), ('build', CommentedMap([('number', 5)])), ('source', CommentedMap([('url', 'http://weblogo.berkeley.edu/release/weblogo.2.8.2.tar.gz'), ('fn', 'weblogo.2.8.2.tar.gz'), ('md5', '12886e0f41757e32ac2823d0f990ce0c')])), ('requirements', CommentedMap([('run', ['perl'])])), ('test', CommentedMap([('commands', ['seqlogo 2>&1 | grep usage > /dev/null', 'seqlogo -f foo.bar 2>&1 | grep \"input filename\" > /dev/null'])]))])), 'weeder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/weeder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'weeder'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'weeder2.0.tar.gz'), ('url', 'http://depot.galaxyproject.org/software/weeder/weeder_2.0_src_all.tar.gz'), ('md5', '3401675b24ca928b7cc54ce9ebf18b6a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python'])])), ('test', CommentedMap([('commands', ['weeder2'])])), ('about', CommentedMap([('home', 'http://159.149.160.51/modtools/'), ('license', 'GPL3'), ('summary', 'Motif (transcription factor binding sites) discovery in sequences from coregulated genes of a single species. This is a new Weeder release rewritten to be faster and optimized for large ChIP-Seq data.')])), ('extra', CommentedMap([('notes', 'Includes a simple wrapper script to be able to run weeder from any directory. It passes arguments transparently to the weeder2 executable.')]))])), 'wgs-assembler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wgs-assembler/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page'), ('license', 'MIT'), ('summary', 'Celera Assembler (wgs-assembler) is a de novo whole-genome shotgun (WGS) DNA sequence assembler')])), ('source', CommentedMap([('fn', 'wgs-<{ version }}rc2.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/wgs-assembler/wgs-assembler/wgs-<{ version }}/wgs-<{ version }}rc2.tar.bz2'), ('md5', '70e8efea279d7061718d442a8c1217b3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'gcc', 'llvm', 'python', 'bzip2', 'perl', 'estmapper', 'atac', 'sim4db', 'meryl', 'falcon']), ('run', ['libgcc', 'blasr', 'jellyfish', 'samtools', 'pbdagcon', 'perl', 'estmapper', 'atac', 'sim4db', 'meryl', 'falcon'])])), ('test', CommentedMap([('commands', [\"gatekeeper -h 2>&1 | grep 'gatekeeper'\", 'runCA -version'])]))])), 'wgsim': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wgsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wgsim'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libgcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['printf \">a\\\\nAAAAA\" > ref.fa; wgsim -1 1 -2 1 ref.fa /dev/null /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/wgsim'), ('license', 'MIT'), ('summary', 'Wgsim is a small tool for simulating sequence reads from a reference genome.')]))])), 'wham': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wham/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wham'), ('version', '1.7.0.311')])), ('source', CommentedMap([('git_url', 'https://github.com/zeeev/wham.git'), ('git_rev', '4e8c9bee568211596cdc634d15600114e75d2619'), ('md5', 'unneeded')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'git']), ('run', ['libgcc'])])), ('test', None), ('about', CommentedMap([('home', 'https://github.com/jewmanchue/wham'), ('license', 'MIT'), ('summary', 'Structural variant detection and association testing')]))])), 'whatshap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/whatshap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'whatshap'), ('version', '0.14.1')])), ('about', CommentedMap([('home', 'https://whatshap.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'phase genomic variants using DNA sequencing reads (haplotype assembly)')])), ('source', CommentedMap([('fn', 'whatshap-0.14.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ec/d4/d6c19b716926895ee3def919ab23194212172ed5cf89ccb91c2949fb258d/whatshap-0.14.1.tar.gz'), ('md5', 'edaaeada862844285de1fee9d76da93d')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen']), ('run', ['libgcc', 'python', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen'])])), ('build', CommentedMap([('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('test', CommentedMap([('imports', ['whatshap']), ('commands', ['whatshap --help > /dev/null'])]))])), 'wiggletools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wiggletools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wiggletools'), ('version', '<{ WIGGLETOOLS_VERSION }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'wiggletools-<{ WIGGLETOOLS_VERSION }}.tar.gz'), ('url', 'https://github.com/Ensembl/WiggleTools/archive/v<{ WIGGLETOOLS_VERSION }}.tar.gz'), ('sha256', '<{ WIGGLETOOLS_SHA256 }}')])), ('requirements', CommentedMap([('build', ['libbigwig', 'htslib', 'gsl']), ('run', ['libbigwig', 'htslib', 'gsl'])])), ('about', CommentedMap([('home', 'https://github.com/Ensembl/WiggleTools'), ('license', 'Apache'), ('summary', \"The WiggleTools package allows genomewide data files to be manipulated as numerical functions, equipped with all the standard functional analysis operators (sum, product, product by a scalar, comparators), and derived statistics (mean, median, variance, stddev, t-test, Wilcoxon's rank sum test, etc).\")])), ('test', CommentedMap([('commands', ['wiggletools --help'])]))])), 'wkhtmltopdf': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wkhtmltopdf/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'wkhtmltopdf'), ('version', '0.12.3')])), ('source', CommentedMap([('fn', 'wkhtmltox-0.12.3_linux-generic-amd64.tar.xz'), ('url', 'http://download.gna.org/wkhtmltopdf/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz')])), ('requirements', None), ('about', CommentedMap([('home', 'http://wkhtmltopdf.org/'), ('license', 'LGPLv3'), ('summary', 'wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render HTML into PDF and various image formats using the Qt WebKit rendering engine')]))])), 'workspace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/workspace/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'workspace'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'workspace-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/w/workspace/workspace-0.3.1.tar.gz'), ('md5', '643250e65bd836e3ed4810fe4cb86182')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sprinkles >=0.4.4']), ('run', ['python', 'sprinkles >=0.4.4'])])), ('test', CommentedMap([('imports', ['workspace', 'workspace.plugins'])])), ('about', CommentedMap([('home', 'http://an9.org/w/WorkspacePy'), ('license', 'MIT License'), ('summary', 'Workspace Overmind')]))])), 'wtforms-alchemy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wtforms-alchemy/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'wtforms-alchemy'), ('version', '0.15.0')])), ('source', CommentedMap([('fn', 'WTForms-Alchemy-0.15.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/W/WTForms-Alchemy/WTForms-Alchemy-0.15.0.tar.gz'), ('md5', '2b38bd46d005ef833a8e164ef7e9d45d')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'sqlalchemy >=0.8.0', 'wtforms >=1.0.4', 'wtforms-components >=0.9.2', 'sqlalchemy-utils >=0.30.0', 'six >=1.4.1']), ('run', ['python', 'sqlalchemy >=0.8.0', 'wtforms >=1.0.4', 'wtforms-components >=0.9.2', 'sqlalchemy-utils >=0.30.0', 'six >=1.4.1'])])), ('test', CommentedMap([('imports', ['wtforms_alchemy'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/wtforms-alchemy'), ('license', 'BSD License'), ('summary', 'Generates WTForms forms from SQLAlchemy models.')]))])), 'wtforms-components': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wtforms-components/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wtforms-components'), ('version', '0.10.0')])), ('source', CommentedMap([('fn', 'WTForms-Components-0.10.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/W/WTForms-Components/WTForms-Components-0.10.0.tar.gz'), ('md5', '2b5a5201e679bceb32aecc2099a276e1'), ('patches', ['setup.py.patch'])])), ('build', CommentedMap([('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'wtforms >=1.0.4', 'six >=1.4.1', 'validators >=0.5.0', 'intervals >=0.6.0', 'phonenumbers']), ('run', ['python', 'wtforms >=1.0.4', 'six >=1.4.1', 'validators >=0.5.0', 'intervals >=0.6.0', 'phonenumbers'])])), ('test', CommentedMap([('imports', ['wtforms_components', 'wtforms_components.fields'])])), ('about', CommentedMap([('home', 'https://github.com/kvesteri/wtforms-components'), ('license', 'BSD License'), ('summary', 'Additional fields, validators and widgets for WTForms.')]))])), 'wub': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wub/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/nanoporetech/wub/archive/491f9ab3ba22ad3265710190ad9e9fd0c4e14e2e.tar.gz'), ('md5', '9cc4c34618d27c0acf6bc0c9385e7a75'), ('patches', ['wub_setup.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest', 'biopython', 'numpy', 'matplotlib', 'seaborn', 'editdistance', 'pandas', 'pysam', 'tqdm', 'h5py', 'statsmodels', 'htslib <1.4']), ('run', ['python', 'pytest', 'biopython', 'numpy', 'matplotlib', 'seaborn', 'editdistance', 'pandas', 'pysam', 'tqdm', 'h5py', 'statsmodels', 'htslib <1.4'])])), ('test', CommentedMap([('imports', ['wub']), ('commands', ['add_errors.py -h', 'compare_genomes_lastal.py -h', 'reads_stats.py -h', 'sequence_filter.py -h', 'simulate_genome.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/nanoporetech/wub'), ('license', 'MPL-2.0'), ('license_file', 'LICENSE.md'), ('summary', 'Tools and software library developed by the ONT Applications group')]))])), 'xcftools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xcftools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xcftools'), ('version', '1.0.7')])), ('source', CommentedMap([('fn', '196f51790bbaff594525935adeab1247bb798946.zip'), ('url', 'https://github.com/j-jorge/xcftools/archive/196f51790bbaff594525935adeab1247bb798946.zip'), ('md5', 'e62779d7aa0518977e653b077d5ae9ee')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'automake', 'perl', 'libpng >=1.2.13,<1.7', 'gettext']), ('run', ['libgcc', 'libpng >=1.2.13,<1.7', 'gettext'])])), ('test', CommentedMap([('commands', ['xcf2png --version'])])), ('about', CommentedMap([('home', 'https://github.com/j-jorge/xcftools'), ('license', 'GPL v2'), ('summary', 'Provides xcf2pnm, xcf2png, and xcfinfo binaries')]))])), 'xdg': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xdg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xdg'), ('version', '1.0.5')])), ('source', CommentedMap([('fn', 'xdg-1.0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/ac/72/c8d77048d7470a8ff02d8aa2caff65ca605e04bb2dfc81fd4fa320888578/xdg-1.0.5.tar.gz'), ('md5', 'db3caa0921233323f64677796853abd5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['xdg'])])), ('about', CommentedMap([('home', 'https://github.com/srstevenson/xdg'), ('license', 'ISC License (ISCL)'), ('summary', 'Variables defined by the XDG Base Directory Specification'), ('license_family', 'Public-Domain')]))])), 'xmlbuilder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xmlbuilder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xmlbuilder'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'xmlbuilder-1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/ab/50/7d5482347973e449432c7a09385a5d6769db63c95ef98590caa82d0792ae/xmlbuilder-1.0.tar.gz'), ('md5', '17596b2b8596a17abc8ab56085921b67')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['xmlbuilder'])])), ('about', CommentedMap([('home', 'https://pypi.python.org/pypi/xmlbuilder/1.0'), ('license', 'LGPL v3'), ('summary', 'pythonic way to crate xml/(x)html files'), ('license_family', 'LGPL')]))])), 'xmltramp2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xmltramp2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xmltramp2'), ('version', '3.1.1')])), ('source', CommentedMap([('fn', 'xmltramp2-3.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/28/56/a8628b716378b67c8536989ad9347a003328e83a52f0a5c65407331e59d3/xmltramp2-3.1.1.tar.gz'), ('md5', '1e803ef8c64ba619159572bd03f7b24b')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['xmltramp2'])])), ('about', CommentedMap([('home', 'https://github.com/tBaxter/xmltramp2'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'A modern refactoring of the venerable xmltramp application'), ('license_family', 'GPL')]))])), 'xopen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xopen/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/marcelm/xopen'), ('license', 'MIT'), ('summary', 'Open compressed files transparently in Python')])), ('package', CommentedMap([('name', 'xopen'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'xopen-0.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/9a/b7/8906d6b2c4f874c0aacc676709a3d533362ac93bebc4a656c9df19421c9b/xopen-0.1.1.tar.gz'), ('md5', '4e0e955546ee6bee4ea736b54623a671')])), ('requirements', CommentedMap([('run', ['python']), ('build', ['python'])])), ('test', CommentedMap([('imports', ['xopen'])])), ('build', CommentedMap([('script', 'python setup.py install')]))])), 'xsd': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xsd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xsd'), ('version', '4.0.0_dep')])), ('source', CommentedMap([('url', 'http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2'), ('fn', 'xsd-4.0.0+dep.tar.bz2'), ('md5', 'ae64d7fcd258addc9b045fe3f96208bb')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'xerces-c']), ('run', ['libgcc', 'xerces-c'])])), ('test', CommentedMap([('commands', ['xsd version'])])), ('about', CommentedMap([('home', 'http://www.codesynthesis.com/products/xsd/'), ('license', 'GPL'), ('summary', 'CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler. Provided with an XML instance specification (XML Schema), it generates C++ classes that represent the given vocabulary as well as XML parsing and serialization code.')]))])), 'xsv': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xsv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xsv'), ('version', '0.10.3')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'rust']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'xsv-0.10.3-x86_64-unknown-linux-musl.tar.gz'), ('url', 'https://github.com/BurntSushi/xsv/releases/download/0.10.3/xsv-0.10.3-x86_64-unknown-linux-musl.tar.gz')])), ('test', CommentedMap([('commands', ['xsv --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/BurntSushi/xsv'), ('license', 'MIT'), ('summary', 'A fast CSV toolkit written in Rust.')]))])), 'xtandem': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xtandem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.thegpm.org/TANDEM/index.html'), ('license', 'Artistic License'), ('summary', 'X! Tandem open source is software that can match tandem mass spectra with peptide sequences, in a process that has come to be known as protein identification')])), ('package', CommentedMap([('name', 'xtandem'), ('version', '15.12.15.2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'tandem-linux-15-12-15-2.zip'), ('url', 'ftp://ftp.thegpm.org/projects/tandem/source/tandem-linux-15-12-15-2.zip'), ('md5', '40058cd84fc3f56d671d32ff393f7aa3'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'expat']), ('run', ['libgcc', 'zlib', 'expat'])])), ('test', CommentedMap([('commands', [\"xtandem help 2>&1 | grep 'TANDEM'\"])]))])), 'xtermcolor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xtermcolor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xtermcolor'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'xtermcolor-1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/65/46/c17b53f040396fb6bc0ee6afd0e809c12580791a61b801728708b48b6711/xtermcolor-1.3.tar.gz'), ('md5', '9f674649d431536a35b1cf911c44ce2c')])), ('build', CommentedMap([('skip', False), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['xtermcolor --help'])])), ('about', CommentedMap([('home', 'https://github.com/broadinstitute/xtermcolor'), ('license', 'MIT'), ('summary', 'Python library for terminal color support (including 256-color support')]))])), 'xxhash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xxhash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xxhash'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'xxhash-1.0.1.zip'), ('url', 'https://pypi.python.org/packages/06/2d/59697bd6d2e8b277a39a916fcdd17246bd25eeceb107534fe50e128f6e59/xxhash-1.0.1.zip'), ('md5', '39753d608c8f6eedf40b4fe9a008a082')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['xxhash']), ('requires', ['nose >1.3.0'])])), ('about', CommentedMap([('home', 'https://github.com/ifduyue/python-xxhash'), ('license', 'BSD License'), ('summary', 'Python binding for xxHash'), ('license_family', 'BSD')]))])), 'xxmotif': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xxmotif/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xxmotif'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'xxmotif-<{ version }}.tar.gz'), ('url', 'https://github.com/soedinglab/xxmotif/archive/<{ version }}.tar.gz'), ('sha256', '235eb8ad562c4757a549751bb290f443ff70c29a142edbff0f03440f097b61cb')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://github.com/soedinglab/xxmotif'), ('summary', 'eXhaustive, weight matriX-based motif discovery in nucleotide sequences'), ('license', 'GPLv3'), ('license_file', 'LICENSE')])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/bin/XXmotif'])]))])), 'xyalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xyalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xyalign'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'xyalign-1.0.0.tar.gz'), ('url', 'https://pypi.python.org/packages/ca/5f/3508044d4c18caa8773049162baa78dc994b69fc25a2b953a3224289cc71/XYalign-1.0.0.tar.gz'), ('md5', 'b9315052f9a68ae73608aeeddac45c6d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('entry_points', ['xyalign = xyalign.xyalign:main']), ('script', 'python setup.py install  --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest-runner', 'matplotlib', 'numpy', 'pandas', 'pybedtools', 'pysam', 'scipy']), ('run', ['python', 'matplotlib', 'numpy', 'pandas', 'pybedtools', 'pysam', 'scipy', 'bbmap', 'bedtools', 'bwa', 'sambamba', 'samtools', 'platypus-variant'])])), ('test', CommentedMap([('imports', ['xyalign', 'xyalign.test']), ('commands', ['xyalign --help']), ('requires', ['pytest', 'pytest-runner'])])), ('about', CommentedMap([('home', 'https://github.com/WilsonSayresLab/XYalign'), ('license', 'GNU General Public License version 3'), ('summary', 'Command line tools and python library to infer ploidy, correct for sex chromosome complement, and work with NGS data')]))])), 'yaggo': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yaggo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yaggo'), ('version', '1.5.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'yaggo'), ('url', 'https://github.com/gmarcais/yaggo/releases/download/v1.5.9/yaggo')])), ('requirements', CommentedMap([('build', None), ('run', ['ruby'])])), ('test', CommentedMap([('commands', ['yaggo -h 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/gmarcais/yaggo'), ('license', 'GPLv3'), ('summary', 'Yaggo is a tool to generate command line parsers for C++. Yaggo stands for \"Yet Another GenGetOpt\" and is inspired by GNU Gengetopt.')]))])), 'yaha': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yaha/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/GregoryFaust/yaha'), ('license', 'GPL'), ('summary', 'yaha is an open source, flexible, sensitive and accurate DNA aligner designed for single-end reads')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'yaha'), ('version', '0.1.83')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'yaha-0.1.83.tar.gz'), ('url', 'https://github.com/GregoryFaust/yaha/releases/download/v0.1.83/yaha-0.1.83.tar.gz'), ('sha256', '76e052cd92630c6e9871412e37e46f18bfbbf7fc5dd2f345b3b9a73eb74939ef')])), ('test', CommentedMap([('commands', ['yaha 2>&1 | grep version > /dev/null'])]))])), 'yahmm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yahmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yahmm'), ('version', '1.1.3')])), ('source', CommentedMap([('fn', 'yahmm-1.1.3.zip'), ('url', 'https://pypi.python.org/packages/source/y/yahmm/yahmm-1.1.3.zip'), ('md5', '022158915e79a41ff36327a02a3c2265')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.20.1', 'numpy >=1.8.0', 'scipy >=0.13.3', 'networkx >=1.8.1', 'matplotlib >=1.3.1']), ('run', ['python', 'cython >=0.20.1', 'numpy >=1.8.0', 'scipy >=0.13.3', 'networkx >=1.8.1', 'matplotlib >=1.3.1'])])), ('test', CommentedMap([('imports', ['yahmm'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/yahmm/'), ('license', 'MIT'), ('license_file', 'LICENSE.txt'), ('summary', 'YAHMM is a HMM package for Python, implemented in Cython for speed.')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'yamllint': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yamllint/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yamllint'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'yamllint-1.2.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/y/yamllint/yamllint-1.2.1.tar.gz'), ('md5', '51631d9133fbdd5b72f5f9a5c076bcd5')])), ('build', CommentedMap([('entry_points', ['yamllint=yamllint.cli:run'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyyaml']), ('run', ['python', 'pyyaml'])])), ('test', CommentedMap([('imports', ['tests', 'tests.rules', 'yamllint', 'yamllint.rules']), ('commands', ['yamllint --help']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'https://github.com/adrienverge/yamllint'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'A linter for YAML files.')]))])), 'yanagiba': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yanagiba/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yanagiba'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'yanagiba-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/y/yanagiba/yanagiba-<{ version }}.tar.gz'), ('sha256', '7246f157d60568c4a35b734ec904fdc2b3a443ff5433acdca43dcc86e9a5ec06')])), ('build', CommentedMap([('skip', True), ('entry_points', ['yanagiba=yanagiba.cmd_line:main']), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nanomath', 'pandas >=0.20.3', 'biopython >=1.70']), ('run', ['python', 'nanomath', 'pandas >=0.20.3', 'biopython >=1.70'])])), ('test', CommentedMap([('imports', ['yanagiba']), ('commands', ['yanagiba --help'])])), ('about', CommentedMap([('home', 'https://github.com/Adamtaranto/Yanagiba'), ('license', 'MIT License'), ('summary', 'Filterilter short or low quality Oxford Nanopore reads which have been basecalled with Albacore.'), ('license_family', 'MIT')]))])), 'yanc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yanc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yanc'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'yanc-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/y/yanc/yanc-0.3.3.tar.gz'), ('md5', 'e1c6630d9c7e1210975c8d23ca50700f')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['yanc']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'https://github.com/0compute/yanc'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Yet another nose colorer')]))])), 'yara': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yara/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'yara'), ('version', '0.9.6')])), ('source', CommentedMap([('fn', 'yara-0.9.6-Linux-x86_64.zip'), ('url', 'http://packages.seqan.de/yara/yara-0.9.6-Linux-x86_64.zip'), ('md5', 'fca52e83f786b71ce622f108a9e18a24')])), ('test', CommentedMap([('commands', ['yara_indexer --help 2>&1 > /dev/null', 'yara_mapper --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/blob/develop/apps/yara/README.rst'), ('license', 'https://raw.githubusercontent.com/seqan/seqan/develop/apps/yara/LICENSE'), ('summary', 'Yara is an exact tool for aligning DNA sequencing reads to reference genomes.')]))])), 'yass': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yass'), ('version', '1.14')])), ('source', CommentedMap([('fn', 'yass-1.14.tar.gz'), ('url', 'http://bioinfo.lifl.fr/yass/files/yass-current/yass-1.14.tar.gz'), ('sha256', '2ea4d2a32bb17fb6de590b0e8bce5231e2506b490b3456700b4bc029544a1982')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['yass --help'])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/yass/'), ('license', 'CeCILL Free Software License'), ('summary', 'YASS is a genomic similarity search tool, for nucleic (DNA/RNA) sequences in fasta or plain text format.')]))])), 'zagros': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/zagros/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/zagros/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'zagros is a motif-discovery tool for CLIP-Seq data.')])), ('package', CommentedMap([('name', 'zagros'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'zagros-1.0.0.tar.gz'), ('url', 'http://smithlabresearch.org/downloads/zagros-1.0.0.tar.gz'), ('sha256', '3d8495d4217e75059a49673c8eaf524f933e2f35be325502fdbaf107e933c028')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc', 'llvm', 'gawk', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['zagros 2>&1 | grep \"^Usage\"'])]))])), 'zeroc-ice': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/zeroc-ice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'zeroc-ice'), ('version', '3.6.3')])), ('source', CommentedMap([('fn', 'zeroc-ice-3.6.3.tar.gz'), ('url', 'https://pypi.python.org/packages/d4/13/090854b3a35ce66aeba13f6722868dc1f3b113175b4d7cd1a73e29718051/zeroc-ice-3.6.3.tar.gz'), ('md5', '518402167fbb7788619fac3ca641b745')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bzip2', 'gcc', 'llvm']), ('run', ['python', 'bzip2', 'libgcc'])])), ('test', CommentedMap([('imports', ['Ice'])])), ('about', CommentedMap([('home', 'https://github.com/zeroc-ice'), ('license', 'GPL v2 with exceptions'), ('summary', 'Ice is a comprehensive RPC framework that helps you network your software with minimal effort.')]))])), 'zifa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/zifa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'zifa'), ('version', '0.1.0')])), ('source', CommentedMap([('fn', 'zifa-0.1.0.zip'), ('url', 'https://github.com/epierson9/ZIFA/archive/e40fdbf6c41cd7e564ab03d242fc071f5ac543b7.zip')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['ZIFA'])])), ('about', CommentedMap([('home', 'https://github.com/epierson9/ZIFA'), ('license', 'MIT'), ('summary', 'Dimensionality reduction for zero-inflated single-cell gene expression analysis')]))])), 'align-it': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/align-it/1.0.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/software/align-it/1.0.4/align-it.html'), ('license', 'LGPL'), ('summary', 'Align-it is a tool to align molecules according to their pharmacophores. A pharmacophore is an abstract concept based on the specific interactions observed in drug-receptor interactions: hydrogen bonding, charge transfer, electrostatic and hydrophobic interactions.')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('package', CommentedMap([('name', 'align_it'), ('version', '1.0.3')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openbabel ==2.4.1', 'zlib', 'cmake']), ('run', ['openbabel ==2.4.1', 'libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'align-it-1.0.3.tar.gz'), ('url', 'https://github.com/bgruening/silicos-it_store/raw/master/align-it/align-it-1.0.3.tar.gz'), ('sha256', '30e97121ee9d717c140daf9397f31744319a6e840ca4dbc8d748a5aa9039b1f2')])), ('test', CommentedMap([('commands', ['align-it --version'])]))])), 'aragorn': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/aragorn/1.2.36/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/'), ('license', 'GPLv3'), ('summary', 'ARAGORN, tRNA (and tmRNA) detection')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'aragorn'), ('version', '1.2.36')])), ('source', CommentedMap([('fn', 'aragorn1.2.36.tgz'), ('sha256', 'ab06032589e45aa002f8616333568e9ab11034b3a675f922421e5f1c3e95e7b5'), ('url', 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/Downloads/aragorn1.2.36.tgz')])), ('test', CommentedMap([('commands', ['aragorn'])]))])), 'bamhash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamhash/1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamhash'), ('version', 1.1)])), ('source', CommentedMap([('url', 'https://github.com/DecodeGenetics/BamHash/archive/1c987784929f5c6cf28b9d36d00017380b922c71.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'openssl', 'zlib']), ('run', ['openssl', 'libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bamhash_checksum_bam --version', 'bamhash_checksum_fastq --version', 'bamhash_checksum_fasta --version'])])), ('about', CommentedMap([('home', 'https://github.com/joerick/pyinstrument'), ('license', 'BSD'), ('license_file', 'LICENSE')]))])), 'barrnap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/barrnap/0.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/tseemann/<{ name }}/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.22.0', 'hmmer >=3.1b'])])), ('test', CommentedMap([('source_files', ['examples/small.fna']), ('commands', ['(barrnap --help 2>&1; true) | grep -q Options', '(barrnap $PREFIX/lib/barrnap/examples/small.fna 2>&1; true) | grep -q \"Found 3 ribosomal RNA features\"'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/barrnap'), ('license', 'GPLv3'), ('summary', 'Find 5S, 16S and 23S ribosomal RNA genes in bacterial contigs')]))])), 'bedtools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedtools/2.26.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.26.0gx')])), ('source', CommentedMap([('fn', 'bedtools-2.26.0.tar.gz'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.26.0/bedtools-2.26.0.tar.gz'), ('sha256', '65f32f32cbf1b91ba42854b40c604aa6a16c7d3b3ec110d6acf438eb22df0a4a'), ('patches', ['2.26.0_postrelease.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bedtools'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))])), 'bioconductor-diffbind': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-diffbind/2.2.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-diffbind'), ('version', '2.2.12')])), ('source', CommentedMap([('fn', 'DiffBind_2.2.12.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/DiffBind_2.2.12.tar.gz', 'https://depot.galaxyproject.org/software/DiffBind/DiffBind_2.2.12_src_all.tar.gz']), ('sha256', '9ee1454c851559ac713b5ef83e22f577c801aae76f843ada3a4c033725e46d70')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DiffBind\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/DiffBind.html'), ('license', 'Artistic-2.0'), ('summary', 'Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.')]))])), 'bioconductor-qdnaseq.mm10': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qdnaseq.mm10/1.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qdnaseq.mm10'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'QDNAseq.mm10_1.4.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.4/data/experiment/src/contrib/QDNAseq.mm10_1.4.0.tar.gz'), ('md5', 'd5e08fe75728f32ffb64c907e8ac0a68')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qdnaseq', 'r-base']), ('run', ['bioconductor-qdnaseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'QDNAseq.mm10\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/data/experiment/html/QDNAseq.mm10.html'), ('license', 'GPL'), ('summary', 'This package provides QDNAseq bin annotations for the mouse genome build mm10.')]))])), 'biom-format': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biom-format/2.1.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biom-format'), ('version', '2.1.5')])), ('source', CommentedMap([('fn', 'biom-format-2.1.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/biom-format/biom-format-2.1.5.tar.gz'), ('md5', '30964772b5eb9a84d407d1ed33a064c0')])), ('build', CommentedMap([('preserve_egg_dir', True), ('entry_points', ['biom=biom.cli:cli']), ('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyqi ==0.3.2', 'click', 'h5py', 'numpy >=1.3.0', 'future >=0.15.0', 'scipy >=0.13.0']), ('run', ['python', 'setuptools', 'click', 'h5py', 'pyqi ==0.3.2', 'numpy >=1.3.0', 'future >=0.15.0', 'scipy >=0.13.0', 'libgcc'])])), ('test', CommentedMap([('imports', ['biom', 'biom.cli']), ('commands', ['LANG=C.UTF-8 biom --help'])])), ('about', CommentedMap([('home', 'http://www.biom-format.org'), ('license', 'BSD License'), ('summary', 'Biological Observation Matrix (BIOM) format')]))])), 'blast-legacy': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blast-legacy/2.2.22/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blast-legacy'), ('version', '2.2.22')])), ('source', CommentedMap([('fn', 'blast-2.2.22-x64-linux.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/blast/executables/legacy/2.2.22/blast-2.2.22-x64-linux.tar.gz'), ('md5', '84e900b276a6e956dbbff9e6321155cf')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('test', CommentedMap([('commands', ['bl2seq --help | grep bl2seq > /dev/null', 'blastall --help | grep blastall > /dev/null', 'blastclust --help | grep blastclust > /dev/null', 'blastpgp --help | grep blastpgp > /dev/null', 'copymat --help | grep copymat > /dev/null', 'fastacmd --help | grep fastacmd > /dev/null', 'formatdb --help | grep formatdb > /dev/null', 'formatrpsdb --help | grep formatrpsdb > /dev/null', 'impala --help | grep impala > /dev/null', 'makemat --help | grep makemat > /dev/null', 'megablast --help | grep megablast > /dev/null', 'rpsblast --help | grep rpsblast > /dev/null', 'seedtop --help | grep seedtop > /dev/null'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov'), ('license', 'Public Domain'), ('summary', 'The Basic Local Alignment Search Tool (BLAST) finds regions of local similarity between sequences.')]))])), 'burrito': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/burrito/0.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'burrito'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'burrito-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/burrito/burrito-0.9.1.tar.gz'), ('md5', 'ef2b0bee1e8c037a535627a33e304d61')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'future']), ('run', ['python', 'future', 'r-base'])])), ('test', CommentedMap([('imports', ['burrito'])])), ('about', CommentedMap([('home', 'https://github.com/biocore/burrito'), ('license', 'BSD License'), ('summary', 'Framework for wrapping and controlling command-line applications.')]))])), 'burrito-fillings': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/burrito-fillings/0.1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'burrito-fillings'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'burrito-fillings-0.1.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/b/burrito-fillings/burrito-fillings-0.1.1.tar.gz'), ('md5', 'ea65051e674aedfad8bbfdfa63f2c4d8')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scikit-bio >=0.2.1,<0.3.0', 'burrito <1.0.0']), ('run', ['python', 'scikit-bio >=0.2.1,<0.3.0', 'burrito <1.0.0'])])), ('test', CommentedMap([('imports', ['bfillings'])])), ('about', CommentedMap([('home', 'https://github.com/biocore/burrito-fillings'), ('license', 'BSD License'), ('summary', 'burrito-fillings: burrito application controllers for bioinformatics')]))])), 'ccat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ccat/3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/paperCCAT.htm'), ('license', None), ('summary', 'CCAT is a software package for the analysis of ChIP-seq data with negative control.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'ccat'), ('version', 3.0)])), ('source', CommentedMap([('fn', 'CCAT3.0.tar.gz'), ('sha256', 'd8944465c8327bec22eb41059ba73f780fcdcedf4470507bfdae5a54c59a9b07'), ('url', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/tools/CCAT3.0.tar.gz')])), ('test', CommentedMap([('commands', ['CCAT --help'])]))])), 'cuneiform': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cuneiform/1.1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cuneiform'), ('version', 0.21)])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'cuneiform-linux-1.1.0.tar.bz2'), ('url', 'https://launchpad.net/cuneiform-linux/1.1/1.1/+download/cuneiform-linux-1.1.0.tar.bz2')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]', 'graphicsmagick', 'cmake']), ('run', None)])), ('about', CommentedMap([('home', 'https://launchpad.net/cuneiform-linux/'), ('license', 'Simplified BSD Licence'), ('summary', 'Cuneiform is an OCR system.')]))])), 'datamash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/datamash/1.1.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.gnu.org/software/datamash'), ('license', 'GPL3'), ('summary', 'GNU Datamash is a command-line program which performs basic numeric,textual and statistical operations on input textual data files.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'datamash'), ('version', '1.1.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', [])])), ('source', CommentedMap([('fn', 'datamash-1.1.0.tar.gz'), ('sha256', 'a9e5acc86af4dd64c7ac7f6554718b40271aa67f7ff6e9819bdd919a25904bb0'), ('url', 'http://ftpmirror.gnu.org/datamash/datamash-1.1.0.tar.gz')])), ('test', CommentedMap([('commands', ['datamash --help &> /dev/null'])]))])), 'diamond': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/diamond/0.9.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))])), 'eigen': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eigen/3.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://eigen.tuxfamily.org'), ('license', 'MPL2'), ('summary', 'Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('package', CommentedMap([('name', 'eigen'), ('version', '3.2')])), ('requirements', CommentedMap([('build', ['cmake']), ('run', ['cmake'])])), ('source', CommentedMap([('fn', '3.2.6.tar.bz2'), ('sha256', '8a3352f9a5361fe90e451a7305fb1896fc7f771dc16cc0edd8e6b157f52c343e'), ('url', 'https://bitbucket.org/eigen/eigen/get/3.2.6.tar.bz2')])), ('test', CommentedMap([('commands', ['ls -l $PREFIX/include/ &> /dev/null', 'ls -l $PREFIX/share/ &> /dev/null'])]))])), 'emmix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emmix/1.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'emmix'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'EMMIX.f'), ('url', 'https://people.smp.uq.edu.au/GeoffMcLachlan/emmix/EMMIX.f'), ('md5', '66523b6873e820bde7561182dd522898')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['certifi', 'gnu-wget', 'gcc']), ('run', ['libgcc'])])), ('test', None), ('about', CommentedMap([('home', 'https://people.smp.uq.edu.au/GeoffMcLachlan/emmix/emmix.html'), ('summary', 'A tool that fits a mixture model of multivariate normal or t-distributed components to a given data set.'), ('license', 'Available freely for non-commercial use only')]))])), 'enasearch': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/enasearch/0.1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/18/b1/6d22f24d8afd01ee159933f5e54675e866e4a706c9f9a98b791bba40196f/enasearch-0.1.1.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('entry_points', ['enasearch = enasearch.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'click', 'flake8', 'xmltodict', 'biopython', 'dicttoxml', 'pytest', 'pytest-runner']), ('run', ['python', 'requests', 'click', 'flake8', 'xmltodict', 'biopython', 'dicttoxml'])])), ('test', CommentedMap([('imports', ['enasearch', 'enasearch_data']), ('commands', ['enasearch --help'])])), ('about', CommentedMap([('home', 'https://github.com/bebatut/enasearch'), ('license', 'MIT License'), ('summary', \"A Python library for interacting with ENA's API\"), ('license_family', 'MIT')]))])), 'estscan': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/estscan/3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'estscan'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'estscan-3.0.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/estscan/3.0/estscan-3.0.tar.gz'), ('md5', '1252c862b41b03b59687f5ba148b77cd')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc'])])), ('test', CommentedMap([('commands', [\"estscan -v 2>&1 | grep '3.0'\", \"estscan -h 2>&1 | grep 'Available'\"])])), ('about', CommentedMap([('home', 'http://estscan.sourceforge.net'), ('license', 'open source'), ('summary', 'Detects coding regions in DNA sequences even if they are of low quality')]))])), 'fiji': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fiji/20151222/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://fiji.sc'), ('license', 'GNU General Public License'), ('summary', 'Fiji is an image processing package—a \"batteries-included\" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.')])), ('package', CommentedMap([('name', 'fiji'), ('version', 20151222)])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'fiji-macosx-20151222.dmg'), ('url', 'http://downloads.imagej.net/fiji/Life-Line/fiji-macosx-20151222.dmg'), ('md5', '8a0f2acb02b1106255d18b17e6f25c7c')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6'])])), ('test', CommentedMap([('requires', ['python']), ('files', ['hello.py']), ('commands', [\"bunwarpj 2>&1 | grep 'Usage'\", 'ImageJ --ij2 --headless --debug --jython hello.py 2>&1 | grep Bjoern', 'ImageJ --ij2 --headless --debug -DXms=512m -DXmx=1g --jython hello.py 2>&1 | grep Bjoern'])]))])), 'flexbar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flexbar/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flexbar'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/seqan/flexbar/archive/v<{ version }}.tar.gz'), ('fn', 'flexbar_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['flexbar -h'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/flexbar'), ('license', 'BSD-3-Clause'), ('summary', 'Flexible barcode and adapter removal')]))])), 'ghc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ghc/6.8.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ghc'), ('version', '6.8.3')])), ('source', CommentedMap([('fn', 'ghc-6.8.3-x86_64-unknown-linux.tar.bz2'), ('url', 'https://www.haskell.org/ghc/dist/6.8.3/ghc-6.8.3-x86_64-unknown-linux.tar.bz2'), ('sha1', 'e8869f7df7465f1db913ce070e95fb22cc9309e9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gmp']), ('run', ['gmp'])])), ('test', CommentedMap([('commands', ['ghc --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.haskell.org/ghc/'), ('license', 'BSD'), ('summary', 'GHC is a state-of-the-art, open source, compiler and interactive environment for the functional language Haskell.')]))])), 'gocr': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gocr/0.50/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gocr'), ('version', 0.5)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gocr-0.50pre-patched.tgz'), ('url', 'http://downloads.sourceforge.net/project/osra/gocr-patched/gocr-0.50pre-patched.tgz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', ['gocr --help'])])), ('about', CommentedMap([('home', 'http://jocr.sourceforge.net/'), ('license', 'GPL'), ('summary', 'GOCR is an OCR (Optical Character Recognition) program')]))])), 'graphicsmagick': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/graphicsmagick/1.3.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'graphicsmagick'), ('version', '1.3.26')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'GraphicsMagick-1.3.26.tar.gz'), ('url', 'https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.26/GraphicsMagick-1.3.26.tar.gz'), ('sha256', 'ab06d07cda5181f7829714c9f5641c0a93a0c3af2dacd747ed48d534e1ed0b3a')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'ghostscript', 'libpng', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau']), ('run', ['ghostscript', 'libpng', 'libgcc', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau', 'xorg-libx11'])])), ('test', CommentedMap([('commands', ['gm -help | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'http://www.graphicsmagick.org/'), ('license', 'MIT'), ('summary', 'GraphicsMagick is the swiss army knife of image processing')]))])), 'humann2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/humann2/0.9.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://bitbucket.org/biobakery/humann2/get/<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'bowtie2 >=2.2.5', 'metaphlan2 >=2.2.0', 'diamond >=0.7.10,<0.9.0', 'rapsearch >=2.21', 'samtools', 'biom-format'])])), ('test', CommentedMap([('commands', ['humann2 --version', 'humann2_join_tables -h', 'humann2_reduce_table -h', 'humann2_regroup_table -h', 'humann2_rename_table -h', 'humann2_renorm_table -h', 'humann2_rna_dna_norm -h', 'humann2_split_table -h'])])), ('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/humann2'), ('license', 'MIT'), ('summary', 'HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data')]))])), 'imfusion': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/imfusion/0.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'imfusion'), ('version', '0.3.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'https://github.com/NKI-CCB/imfusion/archive/0.3.2.tar.gz'), ('sha256', '957747b0c2cb91c18860991e809be84595d9fc3ed88d575b474344fdbe95bd56')])), ('requirements', CommentedMap([('build', ['setuptools', 'python', 'future', 'pandas', 'numexpr', 'pysam >=0.9.1', 'toolz', 'scipy', 'seaborn', 'pyfaidx >=0.4.8.1', 'intervaltree', 'pathlib2', 'typing', 'htslib <1.4', 'htseq >=0.7.2', 'matplotlib', 'seaborn']), ('run', ['python', 'intervaltree', 'future', 'pandas', 'numexpr', 'pysam >=0.9.1', 'toolz', 'scipy', 'seaborn', 'pyfaidx >=0.4.8.1', 'intervaltree', 'pathlib2', 'typing', 'htslib <1.4', 'htseq >=0.7.2', 'matplotlib', 'seaborn', 'star', 'subread', 'stringtie', 'tophat >=2.1.0', 'bowtie'])])), ('test', CommentedMap([('imports', ['imfusion']), ('commands', ['imfusion-build -h', 'imfusion-insertions -h', 'imfusion-expression -h', 'imfusion-merge -h', 'imfusion-ctg -h', 'STAR --version', 'featureCounts -v'])])), ('about', CommentedMap([('home', 'https://github.com/jrderuiter/imfusion'), ('license', 'MIT'), ('summary', 'IM-Fusion - Tool for identifying transposon insertions and their effects from RNA-sequencing data')]))])), 'kat': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kat/2.3.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Release-<{version}}.tar.gz'), ('url', 'https://github.com/TGAC/KAT/archive/Release-<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://github.com/TGAC/KAT'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['llvm', 'libgcc', 'python', 'zlib', 'autoconf', 'automake', 'libtool', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'perl', 'matplotlib', 'scipy', 'numpy']), ('run', ['python', 'libgcc', 'zlib', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'matplotlib', 'scipy', 'numpy'])])), ('test', CommentedMap([('commands', ['kat --version'])]))])), 'leptonica': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/leptonica/1.73/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'leptonica'), ('version', 1.73)])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'leptonica-1.73.tar.gz'), ('url', 'https://github.com/DanBloomberg/leptonica/archive/v1.73.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'jpeg', 'libpng', 'libtiff', 'zlib', 'cmake']), ('run', ['zlib', 'jpeg', 'libpng', 'libtiff'])])), ('test', CommentedMap([('commands', [\"convertfilestopdf --help 2>&1 | grep 'resolution' &> /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/DanBloomberg/leptonica'), ('license', 'FreeBSD 2-clause license'), ('summary', 'Leptonica is a pedagogically-oriented open source site containing software that is broadly useful for image processing and image analysis applications.')]))])), 'libbambamc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libbambamc/0.5.00/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/gt1/bambamc'), ('license', 'GPLv3')])), ('package', CommentedMap([('name', 'libbambamc'), ('version', '0.0.50')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', '0.0.50-release-20140430085950.tar.gz'), ('sha256', '06d2f17b9e4290ef76275f5011dab3b9967baa4d960c2c42f315c7c2f8a89d04'), ('url', 'https://github.com/gt1/bambamc/archive/0.0.50-release-20140430085950.tar.gz')])), ('test', CommentedMap([('files', ['test.c']), ('commands', ['gcc -I ${PREFIX}/include -L ${PREFIX}/lib -lbambamc test.c'])]))])), 'libsvm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/libsvm/3.16/meta.yaml', CommentedMap([('about', CommentedMap([('home', ''), ('license', ''), ('summary', 'LIBSVM is an integrated software for support vector classification, (C-SVC,nu-SVC), regression (epsilon-SVR, nu-SVR) and distribution estimation(one-class SVM). It supports multi-class classification.http://www.csie.ntu.edu.tw/~cjlin/libsvm/')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'libsvm'), ('version', '3.16')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('git_rev', '041496b61214aff1beba02c8f2957e80bd90c234'), ('git_url', 'https://github.com/cjlin1/libsvm.git')])), ('test', CommentedMap([('commands', [\"svm-train | grep 'Usage' > /dev/null\", \"svm-predict | grep 'Usage' > /dev/null\"])]))])), 'mafft': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mafft/7.310/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mafft.cbrc.jp/alignment/software/'), ('license', 'BSD'), ('summary', 'Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform')])), ('package', CommentedMap([('name', 'mafft'), ('version', '7.310')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'mafft-7.310-without-extensions-src.tgz'), ('md5', 'fcb68a47d4506b93bb8463ff29b3cbc4'), ('url', 'http://mafft.cbrc.jp/alignment/software/mafft-7.310-without-extensions-src.tgz'), ('patches', ['osx-makefile.patch', 'linux-makefile.patch'])])), ('test', CommentedMap([('commands', ['mafft --version'])]))])), 'metaphlan2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaphlan2/2.6.0/meta.yaml', CommentedMap([('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://depot.galaxyproject.org/software/<{ name }}/<{ name }}_<{ version }}_src_all.tar.gz')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('run', ['python', 'biom-format', 'numpy', 'bowtie2 >=2.2.5', 'msgpack-python', 'biopython', 'pandas', 'pysam', 'matplotlib'])])), ('test', CommentedMap([('commands', ['metaphlan2.py -v', 'metaphlan2krona.py -h', 'strainphlan.py -h', 'extract_markers.py -h', 'merge_metaphlan_tables.py -h', 'metaphlan2krona.py -h', 'metaphlan_hclust_heatmap.py -h', 'download_metaphlan2_db.py -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/biobakery/metaphlan2'), ('license', 'MIT'), ('summary', 'MetaPhlAn is a computational tool for profiling the composition of microbial  communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic  shotgun sequencing data with species level resolution. From version 2.0  MetaPhlAn is also able to identify specific strains (in the not-so-frequent  cases in which the sample contains a previously sequenced strains) and to  track strains across samples for all species.\\n')]))])), 'mirdeep2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mirdeep2/2.0.0.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mirdeep2'), ('version', '2.0.0.8')])), ('source', CommentedMap([('url', 'https://www.mdc-berlin.de/45995549/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/mirdeep2_0_0_8.zip'), ('md5', 'a707f7d7ad4a2975fb8b2e78c5bcf483'), ('patches', ['mirdeep2.patch'])])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['bowtie', 'viennarna', 'randfold', 'perl', 'perl-pdf-api2', 'perl-font-ttf', 'perl-compress-raw-zlib', 'perl-lwp-simple'])])), ('test', CommentedMap([('commands', ['make_html.pl', 'miRDeep2.pl -h', 'quantifier.pl || [[ $? == 255 ]]'])])), ('about', CommentedMap([('home', 'https://www.mdc-berlin.de/8551903/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep'), ('license', 'academic'), ('summary', 'A completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs')]))])), 'multigps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/multigps/0.74/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'multigps'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'multigps_v<{ version }}.jar'), ('url', 'http://lugh.bmb.psu.edu/software/multigps/multigps_v<{ version }}.jar'), ('sha256', '34c7d9cd5f718f126a9d2a502dd9b04159dd8426c5c8f66929adfdcdc85a946d')])), ('requirements', CommentedMap([('run', ['r-base', 'bioconductor-edger', 'openjdk >=8', 'meme >=4.11.2'])])), ('test', CommentedMap([('commands', ['multigps --version 2>&1 | grep <{ version }}'])])), ('about', CommentedMap([('home', 'http://mahonylab.org/software/multigps/'), ('license', 'MIT'), ('summary', 'MultiGPS is a framework for analyzing collections of multi-condition ChIP-seq datasets and characterizing differential binding events between conditions.')]))])), 'ngshmmalign': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngshmmalign/0.1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngshmmalign'), ('version', '0.1.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ngshmmalign-0.1.1.tar.bz2'), ('url', 'https://github.com/cbg-ethz/ngshmmalign/releases/download/0.1.1/ngshmmalign-0.1.1.tar.bz2'), ('sha256', 'fb5319a71f52ade75eeed68100f0c39ec065a64fa3dbde428c228ba835dd3e4b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libcxx', 'boost ==1.63.0']), ('run', ['libgcc', 'libcxx', 'boost ==1.63.0'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/ngshmmalign'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('summary', 'ngshmmalign is a profile HMM aligner for NGS reads designed particularly for small genomes')])), ('test', CommentedMap([('commands', ['ngshmmalign -h'])]))])), 'obitools': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/obitools/1.2.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'obitools'), ('version', '1.2.11')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/dc/53/bef677cfebef9aba707abb36f476d474c1a3d67f02fefdb38fa9fcf92b4f/OBITools-1.2.11.tar.gz'), ('md5', 'a96c303a083df545778f3f8663bdfa29'), ('sha1', '07c645a8ea3134ff635f93e4592ab9dd4b2acbea'), ('sha256', '40af7e5187dc145cc9f9bda25b13026719cec213248a5c71e515263feecfeb6b'), ('patches', ['extractreads.patch', 'extractreads2.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.24', 'setuptools', 'sphinx >=1.2.0', 'virtualenv >=1.11.0', 'wheel >=0.24.0']), ('run', ['python', 'ipython >=3.0.0,<6.0', 'xorg-libx11', 'xorg-libxau'])])), ('test', CommentedMap([('commands', ['ali2consensus -h', 'ecodbtaxstat -h', 'ecotag -h', 'ecotaxspecificity -h', 'ecotaxstat -h', 'illuminapairedend -h', 'ngsfilter -h', 'obiaddtaxids -h', 'obiannotate -h', 'obiclean -h', 'obicomplement -h', 'obiconvert -h', 'obicount -h', 'obicut -h', 'obidistribute -h', 'obiextract -h', 'obigrep -h', 'obihead -h', 'obijoinpairedend -h', 'obipr2 -h', 'obisample -h', 'obiselect -h', 'obisilva -h', 'obisort -h', 'obisplit -h', 'obistat -h', 'obisubset -h', 'obitab -h', 'obitail -h', 'obitaxonomy -h', 'obiuniq -h', 'oligotag -h'])])), ('about', CommentedMap([('home', 'http://metabarcoding.org/obitools'), ('license', 'CeCILL-V2'), ('summary', 'The OBITools package is a set of programs specifically designed for analyzing NGS data in a DNA metabarcoding context, taking into account taxonomic information')]))])), 'ocrad': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ocrad/0.21/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ocrad'), ('version', 0.21)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ocrad-0.21.tar.gz'), ('url', 'http://ftpmirror.gnu.org/ocrad/ocrad-0.21.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'https://www.gnu.org/software/ocrad/'), ('license', 'GPL'), ('summary', 'Ocrad is an optical character recognition program.')]))])), 'openbabel': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openbabel/2.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'openbabel'), ('version', '2.4.1')])), ('source', CommentedMap([('fn', 'openbabel-2-4-1.zip'), ('url', 'https://github.com/openbabel/openbabel/archive/openbabel-2-4-1.zip'), ('sha1', '6fefb2355601b665bea5840fe8e6f95bd48023c9'), ('patches', ['fix_library_path_search.diff', 'fix_data_path.diff', 'fix_babel_test_paths.diff', 'fix_libinchi_cmakelists.diff', '372.patch'])])), ('build', CommentedMap([('skip', True), ('number', 1), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib', 'swig', 'eigen', 'bzip2', 'python', 'libxml2', 'cairo']), ('run', ['python', 'libgcc', 'zlib', 'libxml2', 'cairo', 'xorg-libxau', 'xorg-libxdmcp', 'xorg-libxext', 'xorg-libxrender'])])), ('test', CommentedMap([('imports', ['openbabel', 'pybel']), ('commands', ['obabel --help', \"obabel -:'c1ccccc1' --gen3d -omol2\", \"obabel -:'c1ccccc1' --gen3d -oinchi\"])])), ('about', CommentedMap([('home', 'http://www.openbabel.org/'), ('license', 'GPLv2'), ('summary', 'A chemical toolbox designed to speak the many languages of chemical data')]))])), 'opsin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/opsin/2.1.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/dan2097/opsin/'), ('license', 'Artistic License 2.0'), ('summary', 'OPSIN is a Java(1.6+) library for IUPAC name-to-structure conversion offering high recall and precision on organic chemical nomenclature.')])), ('package', CommentedMap([('name', 'opsin'), ('version', '2.1.0')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'opsin-2.1.0-jar-with-dependencies.jar'), ('url', 'https://bitbucket.org/dan2097/opsin/downloads/opsin-2.1.0-jar-with-dependencies.jar')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6'])])), ('test', CommentedMap([('commands', ['opsin --help', 'opsin -Xms512m -Xmx1g --help'])])), ('extra', CommentedMap([('notes', 'Opsin is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run opsin with \"opsin -Xms512m -Xmx1g --help\"\\n')]))])), 'osra': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/osra/2.0.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'osra'), ('version', '2.0.1')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('fn', 'osra-2.0.1.tgz'), ('url', 'https://downloads.sourceforge.net/project/osra/osra/2.0.1/osra-2.0.1.tgz'), ('sha256', '82757ccceb39d8af61cf51abfa8d03fd3d9910ed80c63182fa01774cd83a7dec')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openbabel ==2.4.1', 'graphicsmagick ==1.3.23', 'tesseract', 'potrace', 'tclap', 'gocr', 'ocrad']), ('run', ['graphicsmagick ==1.3.23', 'openbabel ==2.4.1', 'tesseract', 'potrace', 'tclap', 'gocr', 'ocrad'])])), ('test', CommentedMap([('commands', ['osra --help'])])), ('about', CommentedMap([('home', 'http://cactus.nci.nih.gov/osra/'), ('license', 'Simplified BSD Licence'), ('summary', 'OSRA is a utility designed to convert graphical representations of chemical structures, as they appear in journal articles, patent documents, textbooks, trade magazines etc., into SMILES or SDF.')]))])), 'paml': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paml/4.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paml'), ('version', '4.9')])), ('source', CommentedMap([('fn', 'paml_4.9_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/paml/paml_4.9_src_all.tar.gz'), ('md5', '7c3638fdf19ec44b7b2636477d893de6')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['chi2 1 .1', 'evolver 1 | grep EVOLVER', 'codeml /dev/null 2>&1 | grep paml'])])), ('about', CommentedMap([('home', 'http://abacus.gene.ucl.ac.uk/software/paml.html'), ('summary', 'A package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'pasta': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pasta/1.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pasta'), ('version', '1.7.8')])), ('source', CommentedMap([('fn', '43076b8.tar.gz'), ('url', 'https://github.com/smirarab/pasta/archive/43076b8.tar.gz'), ('md5', 'a085bb4f6b4116566a61b785be580ae1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['certifi 2016.2.28 py27_0', 'dendropy 4.1.0 py27_0', 'gcc', 'gnu-wget', 'java-jdk >=6', 'llvm', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0']), ('run', ['dendropy 4.1.0 py27_0', 'java-jdk >=6', 'libgcc', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0'])])), ('test', CommentedMap([('imports', ['pasta']), ('commands', ['clustalw2 -help | grep OPTIONS', 'hmmalign -h | grep HMMER', 'hmmbuild -h | grep HMMER', 'muscle -version | grep MUSCLE', 'prank -help | grep prank', 'raxml -h | grep raxmlHPC', 'raxmlp -h | grep raxmlHPC'])])), ('about', CommentedMap([('home', 'https://github.com/smirarab/pasta'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An implementation of the PASTA (Practical Alignment using Sate and TrAnsitivity) algorithm')]))])), 'perl-attribute-handlers': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-attribute-handlers/0.96/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Attribute::Handlers\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Attribute::Handlers'), ('license', 'perl_5'), ('summary', 'Simpler definition of attribute handlers')]))])), 'perl-autoloader': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-autoloader/5.74/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['AutoLoader'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/AutoLoader'), ('license', 'perl_5'), ('summary', 'load subroutines only on demand')]))])), 'perl-b': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-b/1.48/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/B'), ('license', 'perl_5'), ('summary', 'The Perl Compiler Backend')]))])), 'perl-b-hooks-endofscope': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-b-hooks-endofscope/0.21/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'B-Hooks-EndOfScope-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/B-Hooks-EndOfScope-0.21.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-module-implementation', 'perl-sub-exporter-progressive', 'perl-variable-magic', 'perl-test-more', 'perl-text-parsewords']), ('run', ['perl', 'perl-module-implementation', 'perl-sub-exporter-progressive', 'perl-variable-magic'])])), ('test', CommentedMap([('commands', ['perl -e \"use B::Hooks::EndOfScope\"', 'perl -e \"use B::Hooks::EndOfScope::XS\"'])])), ('about', CommentedMap([('home', 'https://github.com/karenetheridge/B-Hooks-EndOfScope'), ('license', 'perl_5'), ('summary', 'Execute code after a scope finished compilation')]))])), 'perl-base': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-base/2.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'base-2.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/base-2.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use base\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/base'), ('license', 'unknown'), ('summary', 'compile-time class fields')]))])), 'perl-biox-workflow': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-biox-workflow/1.10/1.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-biox-workflow'), ('version', '0.10')])), ('source', CommentedMap([('fn', 'BioX-Workflow-1.10.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/BioX-Workflow-1.10.tar.gz'), ('md5', '2670aed5fec2e545a8ee97f2c04a5a95')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-module-build', 'perl-file-find-rule', 'perl-moose']), ('run', ['libgcc', 'perl-threaded', 'perl-pathtools', 'perl-file-find-rule', 'perl-moose'])])), ('test', CommentedMap([('imports', ['BioX::Workflow'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/BioX-Workflow'), ('license', 'perl_5'), ('summary', 'Template away your bioinformatics workflows.')]))])), 'perl-biox-workflow-command': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-biox-workflow-command/2.2.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'BioX-Workflow-Command-<{ version }}.tar.gz'), ('url', 'https://github.com/biosails/BioX-Workflow-Command/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-test-class-moose', 'perl-file-details', 'perl-capture-tiny', 'perl-text-diff', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-datetime-format-strptime', 'perl-config-any', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-module-build-tiny', 'perl-params-validate', 'perl-file-slurp', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-test2', 'perl-test-class-moose', 'perl-clone', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-list-someutils']), ('run', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-test2', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-file-details', 'perl-moose', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-moosex-getopt', 'perl-datetime-format-strptime', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-params-validate', 'perl-config-any', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-clone', 'perl-json', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-list-someutils'])])), ('test', CommentedMap([('commands', ['biox -h', 'biox run -h', 'biox stats -h', 'biox new -h', 'biox add -h'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/BioX-Workflow-Command'), ('license', 'perl_5'), ('summary', 'Opinionated Bioinformatics Genomics Workflow Creator')]))])), 'perl-blib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-blib/1.06/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/blib'), ('license', 'perl_5'), ('summary', \"Use MakeMaker's uninstalled version of a package\")]))])), 'perl-btlib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-btlib/0.19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-btlib'), ('version', '0.19')])), ('source', CommentedMap([('fn', 'BTLib-0.19.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/BTLib/0.19/BTLib-0.19.tar.gz'), ('md5', '58def6382c59a03b8fb7c3343ec3502a')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', [\"perl -e 'use BTLib; $main::seq = BTLib->new;'\"])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/estscan/files/BTLib'), ('license', 'open source'), ('summary', 'Binary Search Tree library')]))])), 'perl-carp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-carp/1.38/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Carp-1.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Carp-1.38.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-exporter', 'perl-extutils-makemaker'])])), ('test', CommentedMap([('commands', ['perl -e \"use Carp\"', 'perl -e \"use Carp::Heavy\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Carp'), ('license', 'perl_5'), ('summary', 'alternative warn and die for modules')]))])), 'perl-constant': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-constant/1.33/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'constant-1.33.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/constant-1.33.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use constant\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/module/constant'), ('license', 'perl_5'), ('summary', 'Perl pragma to declare constants')]))])), 'perl-cpan-meta-validator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-cpan-meta-validator/2.140640/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/CPAN::Meta::Validator'), ('license', 'perl_5'), ('summary', 'validate CPAN distribution metadata structures')]))])), 'perl-cpan-shell': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-cpan-shell/5.5004/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/CPAN::Shell'), ('license', 'perl_5'), ('summary', '')]))])), 'perl-crypt-openssl-random': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-crypt-openssl-random/0.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Crypt-OpenSSL-Random-0.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RU/RURBAN/Crypt-OpenSSL-Random-0.11.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'openssl', 'perl-extutils-makemaker']), ('run', ['openssl', 'perl'])])), ('test', CommentedMap([('imports', ['Crypt::OpenSSL::Random'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/perl-openssl/'), ('license', 'perl_5'), ('summary', 'OpenSSL/LibreSSL pseudo-random number generator access')]))])), 'perl-crypt-openssl-rsa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-crypt-openssl-rsa/0.28/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Crypt-OpenSSL-RSA-0.28.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLER/Crypt-OpenSSL-RSA-0.28.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-crypt-openssl-random']), ('run', ['perl', 'perl-crypt-openssl-random'])])), ('test', CommentedMap([('imports', ['Crypt::OpenSSL::RSA'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Crypt-OpenSSL-RSA'), ('license', 'perl_5'), ('summary', 'RSA encoding and decoding, using the openSSL libraries')]))])), 'perl-data-utilities': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-utilities/0.04/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Utilities-0.04.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CO/CORNELIS/Data-Utilities-0.04.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-clone', 'perl-test-simple']), ('run', ['perl', 'perl-clone'])])), ('test', CommentedMap([('imports', ['Data::Comparator', 'Data::Differences', 'Data::Merger', 'Data::Transformator', 'Data::Utilities'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data-Utilities'), ('license', 'unknown'), ('summary', 'recursively compare Perl datatypes')]))])), 'perl-data-walk': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-data-walk/2.01/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Walk-2.01.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GU/GUIDO/Data-Walk-2.01.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Data::Walk'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data::Walk'), ('license', 'open_source'), ('summary', 'Traverse Perl data structures.')]))])), 'perl-date-format': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-date-format/2.30/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'TimeDate-2.30.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GB/GBARR/TimeDate-2.30.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Date::Format'), ('license', 'perl_5'), ('summary', 'Date formating subroutines')]))])), 'perl-datetime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-datetime/1.42/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-1.42.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-1.42.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-namespace-autoclean', 'perl-test-more', 'perl-test-warnings', 'perl-xsloader', 'perl-try-tiny', 'perl-warnings-register', 'perl-cpan-meta-check', 'perl-test-fatal', 'perl-specio-exporter', 'perl-variable-magic', 'perl-params-validationcompiler', 'perl-datetime-locale', 'perl-datetime-timezone']), ('run', ['perl', 'perl-namespace-autoclean', 'perl-datetime-timezone', 'perl-try-tiny', 'perl-xsloader', 'perl-warnings-register', 'perl-cpan-meta-check', 'perl-test-fatal', 'perl-specio-exporter', 'perl-params-validationcompiler', 'perl-datetime-locale'])])), ('test', CommentedMap([('imports', ['DateTime', 'DateTime::Duration'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime'), ('license', 'artistic_2'), ('summary', 'A date and time object for Perl')]))])), 'perl-datetime-format-strptime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-datetime-format-strptime/1.73/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-Format-Strptime-1.73.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-Format-Strptime-1.73.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-datetime', 'perl-datetime-timezone', 'perl-test-more', 'perl-specio-exporter', 'perl-datetime-locale', 'perl-params-validationcompiler', 'perl-test-fatal', 'perl-try-tiny', 'perl-sub-name', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-app-cpanminus', 'perl-datetime-locale', 'perl-specio-exporter', 'perl-datetime-locale', 'perl-params-validationcompiler', 'perl-datetime', 'perl-datetime-timezone', 'perl-sub-name', 'perl-try-tiny'])])), ('test', CommentedMap([('imports', ['DateTime::Format::Strptime', 'DateTime::Format::Strptime::Types'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-Format-Strptime'), ('license', 'artistic_2'), ('summary', 'Parse and format strp and strf time patterns')]))])), 'perl-datetime-locale': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-datetime-locale/1.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-Locale-1.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-Locale-1.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-cpan-meta-check', 'perl-test-requires', 'perl-test-warnings', 'perl-extutils-makemaker', 'perl-specio-exporter', 'perl-params-validationcompiler', 'perl-test-more', 'perl-namespace-autoclean', 'perl-dist-checkconflicts']), ('run', ['perl', 'perl-namespace-autoclean', 'perl-dist-checkconflicts', 'perl-params-validationcompiler', 'perl-specio-exporter'])])), ('test', CommentedMap([('imports', ['DateTime::Locale', 'DateTime::Locale::Base', 'DateTime::Locale::Catalog', 'DateTime::Locale::Data', 'DateTime::Locale::FromData', 'DateTime::Locale::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-Locale'), ('license', 'perl_5'), ('summary', 'Localization support for DateTime.pm')]))])), 'perl-datetime-timezone': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-datetime-timezone/2.09/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-TimeZone-2.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-TimeZone-2.09.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-runtime', 'perl-test-requires', 'perl-test-more', 'perl-test-fatal', 'perl-namespace-autoclean', 'perl-specio-exporter', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-params-validationcompiler']), ('run', ['perl', 'perl-app-cpanminus', 'perl-module-runtime', 'perl-namespace-autoclean', 'perl-specio-exporter', 'perl-try-tiny', 'perl-params-validationcompiler'])])), ('test', CommentedMap([('commands', ['perl -e \"use DateTime::TimeZone\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-TimeZone'), ('license', 'perl_5'), ('summary', 'Time zone object base class and factory')]))])), 'perl-devel-symdump': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-devel-symdump/2.18/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Devel-Symdump-2.18.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AN/ANDK/Devel-Symdump-2.18.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Devel::Symdump\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Devel::Symdump'), ('license', 'perl_5'), ('summary', 'dump symbol names or the symbol table')]))])), 'perl-digest-sha': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-digest-sha/5.88/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Digest::SHA\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest::SHA'), ('license', 'perl_5'), ('summary', 'Perl extension for SHA-1/224/256/384/512')]))])), 'perl-dynaloader': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dynaloader/1.25/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/DynaLoader'), ('license', 'perl_5'), ('summary', 'Dynamically load C libraries into Perl code')]))])), 'perl-encode': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-encode/2.88/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Encode-2.88.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Encode-2.88.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-extutils-makemaker', 'perl-parent']), ('run', ['perl', 'perl-exporter', 'perl-parent'])])), ('test', CommentedMap([('commands', ['perl -e \"use Encode\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Encode'), ('license', 'perl_5'), ('summary', 'allows you to write your script in non-ASCII and non-UTF-8')]))])), 'perl-estscan1': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-estscan1/1.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-estscan1'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'ESTScan1-1.3.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/ESTScan1/1.3/ESTScan1-1.3.tar.gz'), ('md5', '457e695657544d56e38b340ef3ebc46c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'perl-btlib'])])), ('test', CommentedMap([('imports', ['ESTScan1']), ('commands', ['perl -e \\'use ESTScan1; my @mat = ESTScan1::LoadMatrix(\"HumanIso.smat\", \"CODING\", 4.0);\\''])])), ('about', CommentedMap([('home', 'http://estscan.sourceforge.net'), ('license', 'open source'), ('summary', 'Detects coding regions in DNA sequences even if they are of low quality. ESTScan.pm contains the Perl part of the code that reads in the matrices file. The C code that does the actual computation is located in estscan.c.')]))])), 'perl-estscan2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-estscan2/2.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-estscan2'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'ESTScan2-2.1.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/ESTScan2/2.1/ESTScan2-2.1.tar.gz'), ('md5', 'e6bd51cc264252f27f6222883e63ae36')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'perl-btlib']), ('run', ['perl', 'perl-btlib'])])), ('test', CommentedMap([('commands', ['perl -e \\'use ESTScan; my @mat = ESTScan::LoadMatrix(\"Hs.smat\", \"CODING\", 4.0);\\''])])), ('about', CommentedMap([('home', 'http://estscan.sourceforge.net'), ('license', 'open source'), ('summary', 'Detects coding regions in DNA sequences even if they are of low quality. ESTScan.pm contains the Perl part of the code that reads in the matrices file. The C code that does the actual computation is located in estscan.c.')]))])), 'perl-eval-closure': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-eval-closure/0.14/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Eval-Closure-0.14.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DO/DOY/Eval-Closure-0.14.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-test-requires', 'perl-test-fatal', 'perl-test-more', 'perl-extutils-makemaker', 'perl-io-handle', 'perl-b', 'perl-test2-suite']), ('run', ['perl', 'perl-app-cpanminus', 'perl-test2-suite'])])), ('test', CommentedMap([('imports', ['Eval::Closure'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Eval-Closure'), ('license', 'perl_5'), ('summary', 'safely and cleanly create closures via string eval')]))])), 'perl-exporter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-exporter/5.72/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Exporter-5.72.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TODDR/Exporter-5.72.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Exporter\"', 'perl -e \"use Exporter::Heavy\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Exporter'), ('license', 'perl_5'), ('summary', 'Implements default import method for modules')]))])), 'perl-file-compare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-compare/1.1006/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Compare'), ('license', 'perl_5'), ('summary', 'Compare files or filehandles')]))])), 'perl-file-find': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-find/1.27/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Find'), ('license', 'perl_5'), ('summary', 'Traverse a directory tree.')]))])), 'perl-file-spec': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-spec/3.48_01/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Spec'), ('license', 'perl_5'), ('summary', 'portably perform operations on file names')]))])), 'perl-file-temp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-file-temp/0.2304/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'File-Temp-0.2304.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/File-Temp-0.2304.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-file-path', 'perl-constant', 'perl-parent', 'perl-extutils-makemaker', 'perl-carp', 'perl-app-cpanminus']), ('run', ['perl', 'perl-exporter', 'perl-file-path', 'perl-constant', 'perl-parent', 'perl-carp', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use File::Temp\"'])])), ('about', CommentedMap([('home', 'https://github.com/Perl-Toolchain-Gang/File-Temp'), ('license', 'perl_5'), ('summary', 'return name and handle of a temporary file safely')]))])), 'perl-filter-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-filter-simple/0.91/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Filter::Simple\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Filter::Simple'), ('license', 'perl_5'), ('summary', 'Simplified source filtering')]))])), 'perl-findbin': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-findbin/1.51/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/FindBin'), ('license', 'perl_5'), ('summary', 'Locate directory of original perl script')]))])), 'perl-getopt-argvfile': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-getopt-argvfile/1.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-getopt-argvfile'), ('version', '1.11')])), ('source', CommentedMap([('fn', 'Getopt-ArgvFile-1.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JS/JSTENZEL/Getopt-ArgvFile-1.11.tar.gz'), ('md5', '42117db2af4a24c062b2da25d4fdf756')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-harness', 'perl-text-parsewords', 'perl-test-simple']), ('run', ['perl', 'perl-test-harness', 'perl-text-parsewords', 'perl-test-simple'])])), ('test', CommentedMap([('imports', ['Getopt::ArgvFile'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Getopt::ArgvFile'), ('license', 'artistic_1'), ('summary', 'interpolates script options from files into @ARGV or another array')]))])), 'perl-hpc-runner-command': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-command/3.2.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-<{version}}.tar.gz'), ('url', 'https://github.com/biosails/HPC-Runner-Command/archive/<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-mce', 'perl-archive-tar', 'perl-test-class-moose', 'perl-file-homedir', 'perl-list-moreutils', 'perl-module-build-tiny', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-moose', 'perl-moosex-app', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-datetime', 'perl-string-approx', 'perl-capture-tiny', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-text-diff', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-datetime', 'perl-test-class-moose', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate']), ('run', ['perl', 'perl-mce', 'perl-app-cpanminus', 'perl-archive-tar', 'perl-moosex-app', 'perl-file-homedir', 'perl-list-moreutils', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-datetime', 'perl-string-approx', 'perl-moose', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-datetime', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))])), 'perl-hpc-runner-command-plugin-logger-sqlite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-command-plugin-logger-sqlite/0.0.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-Plugin-Logger-Sqlite-0.0.3.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Command-Plugin-Logger-Sqlite-0.0.3.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'sqlite', 'perl-app-cpanminus', 'perl-hpc-runner-command', 'perl-json-xs', 'perl-file-path', 'perl-datetime', 'perl-capture-tiny', 'perl-log-log4perl', 'perl-json', 'perl-list-uniq', 'perl-text-asciitable', 'perl-file-slurp', 'perl-base', 'perl-module-build-tiny', 'perl-dbi', 'perl-dbd-sqlite', 'perl-clone', 'perl-test2', 'perl-test-class-moose', 'perl-params-validate']), ('run', ['perl', 'sqlite', 'perl-app-cpanminus', 'perl-hpc-runner-command', 'perl-json-xs', 'perl-datetime', 'perl-hpc-runner-command', 'perl-log-log4perl', 'perl-list-uniq', 'perl-text-asciitable', 'perl-json', 'perl-base', 'perl-dbi', 'perl-dbd-sqlite', 'perl-clone', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl watch_db -h'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/HPC-Runner-Command-Plugin-Logger-Sqlite'), ('license', 'perl_5'), ('summary', 'Log HPC::Runner workflows to a sqlite DB.')]))])), 'perl-hpc-runner-slurm': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-hpc-runner-slurm/2.58/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-slurm'), ('version', '2.58')])), ('source', CommentedMap([('fn', 'HPC-Runner-Slurm-2.58.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Slurm-2.58.tar.gz'), ('md5', '1d5d5329159e21163393f01b1acd7523')])), ('build', CommentedMap([('skip', True), ('number', 2)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-log-log4perl', 'perl-moosex-getopt', 'perl-scalar-list-utils', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-file-slurp', 'perl-template-toolkit', 'perl-path-tiny', 'perl-config-any', 'perl-exporter-tiny', 'perl-json', 'perl-extutils-makemaker', 'perl-capture-tiny', 'perl-termreadkey', 'perl-hpc-runner', 'perl-hpc-runner-scheduler', 'perl-hpc-runner-mce']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-log-log4perl', 'perl-moosex-getopt', 'perl-scalar-list-utils', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-file-slurp', 'perl-template-toolkit', 'perl-path-tiny', 'perl-config-any', 'perl-exporter-tiny', 'perl-json', 'perl-termreadkey', 'perl-hpc-runner', 'perl-hpc-runner-scheduler', 'perl-hpc-runner-mce'])])), ('test', CommentedMap([('imports', ['HPC::Runner::Slurm'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jerowe/hpc-runner-slurm'), ('license', 'perl_5'), ('summary', 'Submit Bioinformatics HPC Workflows with the Slurm scheduler ')]))])), 'perl-integer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-integer/1.01/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/integer'), ('license', 'perl_5'), ('summary', 'Perl pragma to use integer arithmetic instead of floating point')]))])), 'perl-io-compress-deflate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-compress-deflate/2.064/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Compress::Deflate'), ('license', 'perl_5'), ('summary', 'Write RFC 1950 files/buffers')]))])), 'perl-io-handle': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-handle/1.35/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Handle'), ('license', 'perl_5'), ('summary', 'supply object methods for I/O handles')]))])), 'perl-io-scalar': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-scalar/2.111/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'IO-stringy-2.111.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DS/DSKOLL/IO-stringy-2.111.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Scalar'), ('license', 'perl_5'), ('summary', 'IO:: interface for reading/writing a scalar')]))])), 'perl-io-uncompress-bunzip2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-uncompress-bunzip2/2.064/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Uncompress::Bunzip2'), ('license', 'perl_5'), ('summary', 'Read bzip2 files/buffers')]))])), 'perl-io-uncompress-gunzip': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-uncompress-gunzip/2.064/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Uncompress::Gunzip'), ('license', 'perl_5'), ('summary', 'Read RFC 1952 files/buffers')]))])), 'perl-io-uncompress-rawinflate': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-io-uncompress-rawinflate/2.064/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Uncompress::RawInflate'), ('license', 'perl_5'), ('summary', 'Read RFC 1951 files/buffers')]))])), 'perl-json-parse': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-json-parse/0.49/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-json-parse'), ('version', '0.49')])), ('source', CommentedMap([('fn', 'JSON-Parse-0.49.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BK/BKB/JSON-Parse-0.49.tar.gz'), ('md5', '41ffc71a708a2a768580fb1611d5f8e8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['JSON::Parse', 'JSON::Tokenize'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/JSON::Parse'), ('license', 'perl_5'), ('summary', 'Read JSON into a Perl variable')]))])), 'perl-lib': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-lib/0.63/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'lib-0.63.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/lib-0.63.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use lib\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/lib'), ('license', 'perl_5'), ('summary', 'manipulate @INC at compile time')]))])), 'perl-list-compare': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-list-compare/0.53/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'List-Compare-0.53.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JK/JKEENAN/List-Compare-0.53.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['List::Compare'])])), ('about', CommentedMap([('home', 'http://thenceforward.net/perl/modules/List-Compare/'), ('license', 'perl_5'), ('summary', 'Compare elements of two or more lists')]))])), 'perl-list-someutils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-list-someutils/0.53/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'List-SomeUtils-0.53.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/List-SomeUtils-0.53.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker', 'perl-module-implementation', 'perl-text-parsewords']), ('run', ['perl', 'perl-module-implementation', 'perl-text-parsewords'])])), ('test', CommentedMap([('imports', ['List::SomeUtils', 'List::SomeUtils::PP'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/List-SomeUtils'), ('license', 'perl_5'), ('summary', 'Provide the stuff missing in List::Util')]))])), 'perl-list-util': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-list-util/1.38/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/List::Util'), ('license', 'perl_5'), ('summary', 'A selection of general-utility list subroutines')]))])), 'perl-locale': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-locale/1.03/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/locale'), ('license', 'perl_5'), ('summary', 'Perl pragma to use or avoid POSIX locales for built-in operations')]))])), 'perl-locale-maketext-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-locale-maketext-simple/0.21/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Locale::Maketext::Simple'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Locale::Maketext::Simple'), ('license', 'perl_5'), ('summary', 'Simple interface to Locale::Maketext::Lexicon')]))])), 'perl-mce': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mce/1.814/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'MCE-1.814.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MARIOROY/MCE-1.814.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['MCE'])])), ('about', CommentedMap([('home', 'https://github.com/marioroy/mce-perl'), ('license', 'perl_5'), ('summary', 'Many-Core Engine for Perl providing parallel processing capabilities')]))])), 'perl-mime-quotedprint': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-mime-quotedprint/3.13/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MIME::QuotedPrint'), ('license', 'perl_5'), ('summary', 'Encoding and decoding of quoted-printable strings')]))])), 'perl-module-load': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-load/0.32/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Module::Load'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Load'), ('license', 'perl_5'), ('summary', 'Load modules in a DWIM style')]))])), 'perl-module-load-conditional': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-load-conditional/0.62/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Module::Load::Conditional\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Load::Conditional'), ('license', 'perl_5'), ('summary', 'Looking up module information / loading at runtime')]))])), 'perl-module-metadata': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-metadata/1.000019/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Module-Metadata-1.000019.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Module-Metadata-1.000019.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', [\"perl -e 'use Module::Metadata'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Metadata'), ('license', 'perl_5'), ('summary', 'Gather package and POD information from perl module files')]))])), 'perl-module-scandeps': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-module-scandeps/1.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Module-ScanDeps-1.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/Module-ScanDeps-1.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-text-parsewords', 'perl-module-metadata', 'perl-test-more', 'perl-extutils-makemaker', 'perl-test-requires']), ('run', ['perl', 'perl-text-parsewords', 'perl-module-metadata'])])), ('test', CommentedMap([('imports', ['Module::ScanDeps'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::ScanDeps'), ('license', 'perl_5'), ('summary', 'Recursively scan Perl code for dependencies')]))])), 'perl-namespace-autoclean': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-namespace-autoclean/0.28/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'namespace-autoclean-0.28.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/namespace-autoclean-0.28.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-requires', 'perl-namespace-clean', 'perl-test-more', 'perl-b-hooks-endofscope', 'perl-list-util', 'perl-sub-identify', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-namespace-clean', 'perl-b-hooks-endofscope', 'perl-list-util', 'perl-sub-identify'])])), ('test', CommentedMap([('imports', ['namespace::autoclean'])])), ('about', CommentedMap([('home', 'https://github.com/moose/namespace-autoclean'), ('license', 'perl_5'), ('summary', 'Keep imports out of your namespace')]))])), 'perl-namespace-clean': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-namespace-clean/0.27/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'namespace-clean-0.27.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RI/RIBASUSHI/namespace-clean-0.27.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-package-stash', 'perl-b-hooks-endofscope', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl', 'perl-package-stash', 'perl-b-hooks-endofscope'])])), ('test', CommentedMap([('imports', ['namespace::clean'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/namespace-clean'), ('license', 'perl_5'), ('summary', 'Keep imports and functions out of your namespace')]))])), 'perl-net-ftp': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-net-ftp/2.79/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net::FTP'), ('license', 'perl_5'), ('summary', 'FTP Client class')]))])), 'perl-net-netrc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-net-netrc/2.14/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net::Netrc'), ('license', 'perl_5'), ('summary', 'OO interface to users netrc file')]))])), 'perl-number-misc': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-number-misc/1.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-number-misc'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'Number-Misc-1.2.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/Number-Misc-1.2.tar.gz'), ('md5', '7a0009b5be60378f204ed1fc9b2d5a19')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Number::Misc'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Number::Misc'), ('license', 'perl_5'), ('summary', 'Number::Misc - handy utilities for numbers')]))])), 'perl-par': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-par/1.014/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-1.014.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-1.014.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-archive-zip', 'perl-par-dist', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-par-dist', 'perl-archive-zip'])])), ('test', CommentedMap([('commands', ['perl -e \"use PAR\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR'), ('license', 'perl_5'), ('summary', 'Perl Archive Tookit')]))])), 'perl-par-dist': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-par-dist/0.49/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-Dist-0.49.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-Dist-0.49.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-file-find', 'perl-extutils-makemaker', 'perl-file-path']), ('run', ['perl', 'perl-file-find', 'perl-file-path'])])), ('test', CommentedMap([('imports', ['PAR::Dist'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR::Dist'), ('license', 'unknown'), ('summary', 'Create and manipulate PAR distributions')]))])), 'perl-par-packer': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-par-packer/1.036/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-Packer-1.036.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-Packer-1.036.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-text-parsewords', 'perl-extutils-makemaker', 'perl-module-build', 'perl-par', 'perl-par-dist', 'perl-module-scandeps', 'perl-getopt-long', 'perl-getopt-argvfile']), ('run', ['perl', 'perl-text-parsewords', 'perl-par', 'perl-par-dist', 'perl-module-scandeps', 'perl-getopt-long', 'perl-getopt-argvfile'])])), ('test', CommentedMap([('commands', ['perl -e \"use PAR::Filter\"', 'perl -e \"use PAR::Filter::Bleach\"', 'which pp', 'which par.pl']), ('imports', ['PAR::Packer'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR::Packer'), ('license', 'perl_5'), ('summary', 'PAR Packager')]))])), 'perl-params-check': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-params-check/0.38/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Params::Check'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Params::Check'), ('license', 'perl_5'), ('summary', 'Templated based param validation')]))])), 'perl-params-validationcompiler': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-params-validationcompiler/0.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Params-ValidationCompiler-0.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Params-ValidationCompiler-0.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-b', 'perl-extutils-makemaker', 'perl-specio-exporter', 'perl-test-simple', 'perl-eval-closure', 'perl-variable-magic', 'perl-exception-class', 'perl-test2-suite', 'perl-devel-stacktrace']), ('run', ['perl', 'perl-app-cpanminus', 'perl-b', 'perl-eval-closure', 'perl-variable-magic', 'perl-exception-class', 'perl-devel-stacktrace'])])), ('test', CommentedMap([('commands', ['perl -e \"use Params::ValidationCompiler\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Params-ValidationCompiler'), ('license', 'artistic_2'), ('summary', 'Build an optimized subroutine parameter validator once, use it forever')]))])), 'perl-parent': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-parent/0.236/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'parent-0.236.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CO/CORION/parent-0.236.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use parent\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/parent'), ('license', 'perl_5'), ('summary', 'Establish an ISA relationship with base classes at compile time')]))])), 'perl-perlio': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perlio/1.09/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO'), ('license', 'perl_5'), ('summary', 'On demand loader for PerlIO layers and root of PerlIO::* name space')]))])), 'perl-perlio-encoding': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-perlio-encoding/0.18/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO::encoding'), ('license', 'perl_5'), ('summary', 'encoding layer')]))])), 'perl-pod-checker': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-checker/1.60/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Pod::Checker'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Checker'), ('license', 'perl_5'), ('summary', 'Pod::Checker verifies POD documentation contents for compliance with the POD format specifications')]))])), 'perl-pod-coverage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-coverage/0.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Coverage-0.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RC/RCLAMP/Pod-Coverage-0.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-devel-symdump', 'perl-pod-parser', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-devel-symdump', 'perl-pod-parser', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"Pod::Coverage\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Coverage'), ('license', 'unknown'), ('summary', 'Checks if the documentation of a module is comprehensive')]))])), 'perl-pod-coverage-trustpod': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-coverage-trustpod/0.100003/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Coverage-TrustPod-0.100003.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Pod-Coverage-TrustPod-0.100003.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-base', 'perl-lib', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-base', 'perl-lib', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Coverage::TrustPod\"'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Pod-Coverage-TrustPod'), ('license', 'perl_5'), ('summary', \"allow a module's pod to contain Pod::Coverage hints\")]))])), 'perl-pod-escapes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-escapes/1.07/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Escapes-1.07.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Pod-Escapes-1.07.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-exporter'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Escapes\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Escapes'), ('license', 'perl_5'), ('summary', 'for resolving Pod Elt...gt sequences')]))])), 'perl-pod-parser': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-parser/1.63/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Parser-1.63.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MAREKR/Pod-Parser-1.63.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Parser\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Parser'), ('license', 'unknown'), ('summary', 'Modules for parsing/translating POD format documents')]))])), 'perl-pod-plaintext': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-plaintext/2.07/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::PlainText'), ('license', 'perl_5'), ('summary', 'Convert POD data to formatted ASCII text')]))])), 'perl-pod-simple': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-simple/3.35/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Simple-3.35.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/K/KH/KHW/Pod-Simple-3.35.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Simple\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Pod-Simple/'), ('license', 'perl_5'), ('summary', 'framework for parsing Pod')]))])), 'perl-pod-simple-text': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-pod-simple-text/3.28/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Simple::Text'), ('license', 'perl_5'), ('summary', 'format Pod as plaintext')]))])), 'perl-posix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-posix/1.38_03/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/POSIX'), ('license', 'perl_5'), ('summary', '')]))])), 'perl-role-tiny-with': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-role-tiny-with/2.000005/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Role-Tiny-2.000005.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/H/HA/HAARG/Role-Tiny-2.000005.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Role::Tiny::With'), ('license', 'perl_5'), ('summary', 'Neat interface for consumers of Role::Tiny roles')]))])), 'perl-safe': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-safe/2.37/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Safe\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Safe'), ('license', 'unknown'), ('summary', 'Compile and execute code in restricted compartments')]))])), 'perl-scalar-list-utils': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-scalar-list-utils/1.45/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-scalar-list-utils'), ('version', '1.45')])), ('source', CommentedMap([('fn', 'Scalar-List-Utils-1.45.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PEVANS/Scalar-List-Utils-1.45.tar.gz'), ('md5', 'b469e990ec92a4c9386aaf9e92857ae9')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Scalar::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scalar-List-Utils'), ('license', 'perl_5'), ('summary', 'Common Scalar and List utility subroutines')]))])), 'perl-specio-exporter': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-specio-exporter/0.36/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Specio-0.36.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Specio-0.36.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Specio::Exporter'), ('license', 'perl_5'), ('summary', 'Base class for type libraries')]))])), 'perl-string-approx': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-string-approx/3.27/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'String-Approx-3.27.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JH/JHI/String-Approx-3.27.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-test-more'])])), ('test', CommentedMap([('imports', ['String::Approx'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/String::Approx'), ('license', 'unknown'), ('summary', 'Perl extension for approximate matching (fuzzy matching)')]))])), 'perl-string-util': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-string-util/1.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-string-util'), ('version', '1.26')])), ('source', CommentedMap([('fn', 'String-Util-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/String-Util-1.26.tar.gz'), ('md5', 'd0f13f8688bedef7174634bb57f7fbe4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-number-misc', 'perl-test-toolbox']), ('run', ['perl', 'perl-number-misc', 'perl-test-toolbox'])])), ('test', CommentedMap([('imports', ['String::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/String::Util'), ('license', 'perl_5'), ('summary', 'String::Util -- String processing utilities')]))])), 'perl-sub-attribute': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-attribute/0.05/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Attribute-0.05.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GF/GFUJI/Sub-Attribute-0.05.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-parsexs', 'perl-test-more', 'perl-extutils-makemaker', 'perl-xsloader']), ('run', ['perl', 'perl-xsloader'])])), ('test', CommentedMap([('imports', ['Sub::Attribute'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sub::Attribute'), ('license', 'perl_5'), ('summary', 'Reliable subroutine attribute handlers')]))])), 'perl-sub-identify': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-identify/0.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Identify-0.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RG/RGARCIA/Sub-Identify-0.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Sub::Identify'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sub::Identify'), ('license', 'perl_5'), ('summary', 'Retrieve names of code references')]))])), 'perl-sub-name': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-sub-name/0.21/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Name-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Sub-Name-0.21.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Sub::Name'])])), ('about', CommentedMap([('home', 'https://github.com/p5sagit/Sub-Name'), ('license', 'perl_5'), ('summary', '(Re)name a sub')]))])), 'perl-symbol': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-symbol/1.07/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Symbol'), ('license', 'perl_5'), ('summary', 'manipulate Perl symbols and their names')]))])), 'perl-symbol-util': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-symbol-util/0.0203/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Symbol-Util-0.0203.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DE/DEXTER/Symbol-Util-0.0203.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Symbol::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Symbol::Util'), ('license', 'perl_5'), ('summary', 'Additional utils for Perl symbols manipulation')]))])), 'perl-tap-harness-env': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tap-harness-env/3.30/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TAP::Harness::Env'), ('license', 'perl_5'), ('summary', 'Parsing harness related environmental variables where appropriate')]))])), 'perl-test': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test/1.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JE/JESSE/Test-1.26.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-harness']), ('run', ['perl', 'perl-test-harness'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test'), ('license', 'unknown'), ('summary', 'provides a simple framework for writing test scripts')]))])), 'perl-test-class-moose': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-class-moose/0.80/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Class-Moose-0.80.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Test-Class-Moose-0.80.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test2', 'perl-app-cpanminus', 'perl-test-more', 'perl-test-warn', 'perl-test-most', 'perl-test-exception', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-sub-attribute', 'perl-test-requires', 'perl-moose', 'perl-namespace-autoclean', 'perl-parallel-forkmanager', 'perl-exception-class', 'perl-capture-tiny', 'perl-scope-guard', 'perl-list-someutils', 'perl-text-diff']), ('run', ['perl', 'perl-test2', 'perl-app-cpanminus', 'perl-test-more', 'perl-test-warn', 'perl-test-requires', 'perl-test-most', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-sub-attribute', 'perl-moose', 'perl-namespace-autoclean', 'perl-parallel-forkmanager', 'perl-exception-class', 'perl-exception-class', 'perl-capture-tiny', 'perl-scope-guard', 'perl-list-someutils', 'perl-text-diff'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Class::Moose\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Test-Class-Moose/'), ('license', 'perl_5'), ('summary', 'Serious testing for serious Perl')]))])), 'perl-test-more': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-more/1.001002/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::More'), ('license', 'perl_5'), ('summary', 'yet another framework for writing test scripts')]))])), 'perl-test-pod-coverage': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-pod-coverage/1.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Pod-Coverage-1.10.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Test-Pod-Coverage-1.10.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-lib', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', [\"perl -e 'Test::Pod::Coverage'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Pod::Coverage'), ('license', 'artistic_2'), ('summary', 'Check for pod coverage in your distribution')]))])), 'perl-test-toolbox': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-toolbox/0.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-test-toolbox'), ('version', '0.4')])), ('source', CommentedMap([('fn', 'Test-Toolbox-0.4.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/Test-Toolbox-0.4.tar.gz'), ('md5', 'b8a83d332dcd0691b68ed9e0a0a53e7b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Test::Toolbox'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Toolbox'), ('license', 'perl_5'), ('summary', 'Test::Toolbox - tools for testing')]))])), 'perl-test-unit-lite': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test-unit-lite/0.1202/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Unit-Lite-0.1202.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DE/DEXTER/Test-Unit-Lite-0.1202.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Unit::Lite\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Unit::Lite'), ('license', 'perl_5'), ('summary', 'Unit testing without external dependencies')]))])), 'perl-test2': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-test2/1.302075/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Simple-1.302075.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Test-Simple-1.302075.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test2'), ('license', 'perl_5'), ('summary', 'Framework for writing test tools that all work together.')]))])), 'perl-text-ansitable': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-ansitable/0.48/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Text-ANSITable-0.48.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Text-ANSITable-0.48.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-namespace-clean', 'perl-json-maybexs', 'perl-extutils-makemaker', 'perl-moo', 'perl-module-load', 'perl-perl-osnames', 'perl-test-more', 'perl-term-app-roles', 'perl-border-style', 'perl-color-theme', 'perl-perlio', 'perl-test-exception']), ('run', ['perl', 'perl-app-cpanminus', 'perl-namespace-clean', 'perl-json-maybexs', 'perl-extutils-makemaker', 'perl-moo', 'perl-module-load', 'perl-perl-osnames', 'perl-test-more', 'perl-term-app-roles', 'perl-border-style', 'perl-color-theme', 'perl-perlio', 'perl-test-exception', 'perl-json-maybexs'])])), ('test', CommentedMap([('imports', ['Text::ANSITable', 'Text::ANSITable::BorderStyle::Default', 'Text::ANSITable::ColorTheme::Default', 'Text::ANSITable::StyleSet::AltRow'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Text-ANSITable'), ('license', 'perl_5'), ('summary', 'Create nice formatted tables using extended ASCII and ANSI colors')]))])), 'perl-text-asciitable': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-asciitable/0.22/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Text-ASCIITable-0.22.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LU/LUNATIC/Text-ASCIITable-0.22.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Text::ASCIITable'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::ASCIITable'), ('license', 'perl_5'), ('summary', 'Create a nice formatted table using ASCII characters.')]))])), 'perl-text-tabs': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-tabs/2013.0523/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Tabs'), ('license', 'perl_5'), ('summary', 'expand and unexpand tabs like unix expand(1) and unexpand(1)')]))])), 'perl-text-template': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-text-template/1.46/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Text-Template-1.46.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MJ/MJD/Text-Template-1.46.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Text::Template', 'Text::Template::Preprocess'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Template'), ('license', 'unknown'), ('summary', 'Expand template text with embedded Perl')]))])), 'perl-tie-hash': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tie-hash/1.05/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie::Hash'), ('license', 'perl_5'), ('summary', '')]))])), 'perl-time-local': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-time-local/1.2300/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Time::Local'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::Local'), ('license', 'perl_5'), ('summary', 'efficiently compute time from local and GMT time')]))])), 'perl-time-piece': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-time-piece/1.27/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Time::Piece\"', 'perl -e \"use Time::Seconds\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::Piece'), ('license', 'perl_5'), ('summary', 'Object Oriented time objects')]))])), 'perl-unicode-normalize': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-unicode-normalize/1.17/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Unicode::Normalize'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Unicode::Normalize'), ('license', 'perl_5'), ('summary', 'Unicode Normalization Forms')]))])), 'perl-variable-magic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-variable-magic/0.61/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Variable-Magic-0.61.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/V/VP/VPIT/Variable-Magic-0.61.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-xsloader', 'perl-extutils-makemaker', 'perl-test2-suite']), ('run', ['perl', 'perl-xsloader'])])), ('test', CommentedMap([('imports', ['Variable::Magic'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Variable-Magic/'), ('license', 'perl_5'), ('summary', 'Associate user-defined magic to variables from Perl.')]))])), 'perl-vars': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-vars/1.03/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/vars'), ('license', 'perl_5'), ('summary', 'Perl pragma to predeclare global variable names')]))])), 'perl-warnings-register': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-warnings-register/1.03/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/warnings::register'), ('license', 'perl_5'), ('summary', 'warnings import function')]))])), 'plant_tribes_assembly_post_processor': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_assembly_post_processor/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_assembly_post_processor'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['cap3', 'genometools-genometools >=1,<2', 'hmmer >=3', 'mafft >=7,<8', 'perl >=5.22', 'perl-estscan2 2.1 pl5.22.0_1', 'transdecoder >=3,<4', 'trimal >=1.4,<2'])])), ('test', CommentedMap([('commands', ['AssemblyPostProcessor 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'Transcriptome assembly post processing pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'plant_tribes_gene_family_aligner': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_gene_family_aligner/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_gene_family_aligner'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['mafft >=7,<8', 'pasta >=1.7.8', 'perl', 'trimal >=1.4,<2'])])), ('test', CommentedMap([('commands', ['GeneFamilyAligner 2>&1 | grep ALIGN'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'Gene family aligner pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'plant_tribes_gene_family_classifier': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_gene_family_classifier/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_gene_family_classifier'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['blast >=2.2.29', 'hmmer >=3', 'perl'])])), ('test', CommentedMap([('commands', ['GeneFamilyClassifier 2>&1 | grep GENE'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'Gene family classifier pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'plant_tribes_gene_family_integrator': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_gene_family_integrator/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_gene_family_integrator'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl'])])), ('test', CommentedMap([('commands', ['GeneFamilyIntegrator 2>&1 | grep INTEGRATE'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'Gene family integrator pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'plant_tribes_gene_family_phylogeny_builder': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_gene_family_phylogeny_builder/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_gene_family_phylogeny_builder'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['fasttree >=2,<3', 'perl', 'raxml >=8,<9'])])), ('test', CommentedMap([('commands', ['GeneFamilyPhylogenyBuilder 2>&1 | grep PHYLOGENETIC'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'Gene family phylogeny builder pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'plant_tribes_kaks_analysis': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plant_tribes_kaks_analysis/1.0.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plant_tribes_kaks_analysis'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'v1.0.2.tar.gz'), ('url', 'https://github.com/dePamphilis/PlantTribes/archive/v1.0.2.tar.gz'), ('md5', 'c0a4217048fe8a18e0c13d67667da638')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['blast >=2.2.29', 'emmix', 'mafft >=7,<8', 'paml', 'perl'])])), ('test', CommentedMap([('commands', ['KaKsAnalysis 2>&1 | grep KaKsAnalysis'])])), ('about', CommentedMap([('home', 'https://github.com/dePamphilis/PlantTribes'), ('summary', 'KaKs Analysis pipeline'), ('license', 'GNU General Public License v3 (GPLv3)')]))])), 'potrace': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/potrace/1.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'potrace'), ('version', 1.11)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'potrace_1.11_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/potrace/potrace_1.11_src_all.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', ['potrace --help'])])), ('about', CommentedMap([('home', 'http://potrace.sourceforge.net'), ('license', 'GPL'), ('summary', 'A tool for tracing a bitmap, which means, transforming a bitmap into a smooth, scalable image')]))])), 'pymvpa': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymvpa/2.6.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pymvpa-2.6.0.tar.gz'), ('url', 'https://github.com/PyMVPA/PyMVPA/archive/upstream/2.6.0.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'libsvm ==3.21', 'matplotlib', 'scipy', 'numpy', 'swig', 'zlib']), ('run', ['python', 'zlib', 'libsvm ==3.21', 'matplotlib', 'scipy', 'numpy', 'swig', 'libgcc'])])), ('test', CommentedMap([('imports', ['mvpa2'])])), ('about', CommentedMap([('home', 'http://www.pymvpa.org/'), ('license', 'perl_5'), ('summary', 'PyMVPA -- Multivariate Pattern Analysis in Python')]))])), 'pynast': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pynast/1.2.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pynast'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'pynast-1.2.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pynast/pynast-1.2.2.tar.gz'), ('md5', '6f1744d8d3f333eb2dbe81026d4dcb1b')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k]')])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.5.1', 'cogent >=1.5.3']), ('run', ['python', 'numpy >=1.5.1', 'cogent >=1.5.3'])])), ('test', CommentedMap([('imports', ['pynast'])])), ('about', CommentedMap([('home', 'http://qiime.org/pynast'), ('license', 'BSD License'), ('summary', 'The Python Nearest Alignment Space Termination tool')]))])), 'pyqi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyqi/0.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyqi'), ('version', '0.3.2')])), ('source', CommentedMap([('fn', 'pyqi-0.3.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyqi/pyqi-0.3.2.tar.gz'), ('md5', '9507c06eeb22a816d963c860ad8e92ae')])), ('build', CommentedMap([('number', 0), ('skip', 'True    [not py27]')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['pyqi'])])), ('about', CommentedMap([('home', 'http://bipy.github.io/pyqi'), ('license', 'BSD License'), ('summary', 'pyqi: expose your interface')]))])), 'qiime': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qiime/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qiime'), ('version', '1.9.1')])), ('source', CommentedMap([('fn', 'qiime-1.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime/qiime-1.9.1.tar.gz'), ('md5', '32d4db2b2c888dbbfc6aa33f02725526'), ('sha1', '2ab4ac9beb745ed3eca3243312cc9ce14e8a20d8'), ('sha256', 'c12dcdae1fe44762a49668276d7e93f487d87a0dc2ff8ca696ac0eff9d92a8a8')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'xz']), ('run', ['python', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'libgcc', 'glib', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau', 'xz'])])), ('test', CommentedMap([('imports', ['qiime', 'qiime.util', 'qiime.check_id_map', 'qiime.sort']), ('commands', ['add_alpha_to_mapping_file.py -h', 'add_qiime_labels.py -h', 'adjust_seq_orientation.py -h', 'align_seqs.py -h', 'alpha_diversity.py -h', 'alpha_rarefaction.py -h', 'amplicon_contingency_table.py', 'ampliconnoise.py -h', 'assign_taxonomy.py -h', 'beta_diversity.py -h', 'beta_diversity_through_plots.py -h', 'beta_significance.py -h', 'blast_wrapper.py -h', 'categorized_dist_scatterplot.py -h', 'check_id_map.py -h', 'clean_raxml_parsimony_tree.py -h', 'cluster_quality.py -h', 'collapse_samples.py -h', 'collate_alpha.py -h', 'compare_alpha_diversity.py -h', 'compare_categories.py -h', 'compare_distance_matrices.py -h', 'compare_taxa_summaries.py -h', 'compare_trajectories.py -h', 'compute_core_microbiome.py -h', 'compute_taxonomy_ratios.py -h', 'conditional_uncovered_probability.py -h', 'consensus_tree.py -h', 'convert_fastaqual_fastq.py -h', 'convert_unifrac_sample_mapping_to_otu_table.py -h', 'core_diversity_analyses.py -h', 'count_seqs.py -h', 'demultiplex_fasta.py -h', 'denoise_wrapper.py -h', 'denoiser.py -h', 'denoiser_preprocess.py -h', 'denoiser_worker.py -h', 'detrend.py -h', 'differential_abundance.py -h', 'dissimilarity_mtx_stats.py -h', 'distance_matrix_from_mapping.py -h', 'estimate_observation_richness.py -h', 'exclude_seqs_by_blast.py -h', 'extract_barcodes.py -h', 'extract_reads_from_interleaved_file.py -h', 'extract_seqs_by_sample_id.py -h', 'filter_alignment.py -h', 'filter_distance_matrix.py -h', 'filter_fasta.py -h', 'filter_otus_by_sample.py -h', 'filter_otus_from_otu_table.py -h', 'filter_samples_from_otu_table.py -h', 'filter_taxa_from_otu_table.py -h', 'filter_tree.py -h', 'fix_arb_fasta.py -h', 'group_significance.py -h', 'identify_chimeric_seqs.py -h', 'identify_missing_files.py -h', 'identify_paired_differences.py -h', 'inflate_denoiser_output.py -h', 'jackknifed_beta_diversity.py -h', 'join_paired_ends.py -h', 'load_remote_mapping_file.py -h', 'make_2d_plots.py -h', 'make_bipartite_network.py -h', 'make_bootstrapped_tree.py -h', 'make_distance_boxplots.py -h', 'make_distance_comparison_plots.py -h', 'make_fastq.py -h', 'make_library_id_lists.py -h', 'make_otu_heatmap.py -h', 'make_otu_heatmap_html.py -h', 'make_otu_network.py -h', 'make_otu_table.py -h', 'make_per_library_sff.py -h', 'make_phylogeny.py -h', 'make_prefs_file.py -h', 'make_qiime_py_file.py -h', 'make_rarefaction_plots.py -h', 'make_tep.py -h', 'map_reads_to_reference.py -h', 'merge_mapping_files.py -h', 'merge_otu_maps.py -h', 'merge_otu_tables.py -h', 'multiple_extract_barcodes.py -h', 'multiple_join_paired_ends.py -h', 'multiple_rarefactions.py -h', 'multiple_rarefactions_even_depth.py -h', 'multiple_split_libraries_fastq.py -h', 'neighbor_joining.py -h', 'nmds.py -h', 'normalize_table.py -h', 'observation_metadata_correlation.py -h', 'otu_category_significance.py -h', 'parallel_align_seqs_pynast.py -h', 'parallel_alpha_diversity.py -h', 'parallel_assign_taxonomy_blast.py -h', 'parallel_assign_taxonomy_rdp.py -h', 'parallel_assign_taxonomy_uclust.py -h', 'parallel_beta_diversity.py -h', 'parallel_blast.py -h', 'parallel_identify_chimeric_seqs.py -h', 'parallel_map_reads_to_reference.py -h', 'parallel_merge_otu_tables.py -h', 'parallel_multiple_rarefactions.py -h', 'parallel_pick_otus_blast.py -h', 'parallel_pick_otus_sortmerna.py -h', 'parallel_pick_otus_trie.py -h', 'parallel_pick_otus_uclust_ref.py -h', 'parallel_pick_otus_usearch61_ref.py -h', 'pick_closed_reference_otus.py -h', 'pick_de_novo_otus.py -h', 'pick_open_reference_otus.py -h', 'pick_otus.py -h', 'pick_rep_set.py -h', 'plot_rank_abundance_graph.py -h', 'plot_semivariogram.py -h', 'plot_taxa_summary.py -h', 'poller.py -h', 'principal_coordinates.py -h', 'print_metadata_stats.py -h', 'print_qiime_config.py -h', 'process_iseq.py -h', 'process_qseq.py -h', 'process_sff.py -h', 'quality_scores_plot.py -h', 'relatedness.py -h', 'shared_phylotypes.py -h', 'simsam.py -h', 'single_rarefaction.py -h', 'sort_otu_table.py -h', 'split_libraries.py -h', 'split_libraries_fastq.py -h', 'split_libraries_lea_seq.py -h', 'split_otu_table.py -h', 'split_otu_table_by_taxonomy.py -h', 'split_sequence_file_on_sample_ids.py -h', 'start_parallel_jobs.py -h', 'start_parallel_jobs_sc.py -h', 'start_parallel_jobs_slurm.py -h', 'start_parallel_jobs_torque.py -h', 'subsample_fasta.py -h', 'summarize_otu_by_cat.py -h', 'summarize_taxa.py -h', 'summarize_taxa_through_plots.py -h', 'supervised_learning.py -h', 'swarm_breaker.py -h', 'transform_coordinate_matrices.py -h', 'tree_compare.py -h', 'trflp_file_to_otu_table.py -h', 'trim_sff_primers.py -h', 'truncate_fasta_qual_files.py -h', 'truncate_reverse_primer.py -h', 'unweight_fasta.py -h', 'upgma_cluster.py -h', 'validate_demultiplexed_fasta.py -h', 'validate_mapping_file.py -h'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'Quantitative Insights Into Microbial Ecology')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))])), 'qiime-default-reference': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qiime-default-reference/0.1.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qiime-default-reference'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'qiime-default-reference-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime-default-reference/qiime-default-reference-0.1.3.tar.gz'), ('md5', '4a5ccf5930690478de1eb881e8030bff')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['qiime_default_reference'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'CC BY-SA 3.0'), ('summary', 'Default reference data files for use with QIIME.')]))])), 'r-data.table': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-data.table/1.10.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-data.table'), ('version', '1.10.4')])), ('source', CommentedMap([('fn', 'data.table_1.10.4.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/data.table_1.10.4.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/data.table/data.table_1.10.4.tar.gz']), ('sha256', '865fdf6aad389071ad063ec1c75a78ffc86eeb88bba011f3ea5281d243966b7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'data.table\\')\"', '\"%R%\" -e \"library(\\'data.table\\')\"'])])), ('about', CommentedMap([('home', 'http://r-datatable.com'), ('license', 'GPL-3 | file LICENSE'), ('summary', 'Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, a fast friendly file reader and parallel file writer. Offers a natural and flexible syntax, for faster development.'), ('license_family', 'GPL3')]))])), 'r-diptest': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-diptest/0.75_7/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-<{ package }}'), ('version', \"<{ version | replace('-', '_') }}\")])), ('source', CommentedMap([('fn', \"<{ package }}_<{ version | replace('-', '_') }}.tar.gz\"), ('url', ['https://cran.r-project.org/src/contrib/<{ package }}_<{ version }}.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/<{ package }}/<{ package }}_<{ version }}.tar.gz']), ('sha256', '462900100ca598ef21dbe566bf1ab2ce7c49cdeab6b7a600a50489b05f61b61b')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'<{ package }}\\')\"', '\"%R%\" -e \"library(\\'<{ package }}\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=<{ package }}'), ('license', 'GPL (>=2)'), ('summary', \"Compute Hartigan's dip test statistic for unimodality / multimodality and provide a test with simulation based p-values, where the original public code has been corrected.\")]))])), 'r-probmetab': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-probmetab/1.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-probmetab'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'eaa6583ac40a7a253996baabc77a20909bcbc188.tar.gz'), ('url', 'https://github.com/rsilvabioinfo/ProbMetab/archive/eaa6583ac40a7a253996baabc77a20909bcbc188.tar.gz')])), ('build', CommentedMap([('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-camera', 'bioconductor-multtest', 'r-rcpparmadillo', 'r-genenet', 'r-hwriter', 'r-rcurl', 'r-rjson', 'r-xml', 'bioconductor-rcytoscape']), ('run', ['r-base', 'bioconductor-camera', 'bioconductor-multtest', 'r-rcpparmadillo', 'r-genenet', 'r-hwriter', 'r-rcurl', 'r-rjson', 'r-xml', 'bioconductor-rcytoscape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ProbMetab\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/rsilvabioinfo/ProbMetab'), ('license', 'GPL (>= 3)'), ('summary', 'Provides probability ranking to candidate compounds assigned to masses, with the prior assumption of connected sample and additional previous and spectral information modeled by the user.')]))])), 'r-rubic': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-rubic/1.0.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-rubic'), ('version', '1.0.3')])), ('source', CommentedMap([('url', 'https://github.com/NKI-CCB/RUBIC/archive/v1.0.3.tar.gz'), ('sha256', '36dda0bd2f388e610f551a19b177be08e5a1c2cf68b23a3626dcbd5be1705837')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table >1.9.6', 'r-digest', 'r-ggplot2', 'r-pracma', 'r-gtable']), ('run', ['r-base', 'r-data.table >1.9.6', 'r-digest', 'r-ggplot2', 'r-pracma', 'r-gtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RUBIC\\')\"'])])), ('about', CommentedMap([('home', 'http://ccb.nki.nl/software/'), ('license', 'Apache-2.0'), ('summary', 'RUBIC detects recurrent copy number aberrations using copy number breaks, rather than recurrently amplified or deleted regions. This allows for a vastly simplified approach as recursive peak splitting procedures and repeated re-estimation of the background model are avoided. Furthermore, the false discovery rate is controlled on the level of called regions, rather than at the probe level.')]))])), 'rdkit': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdkit/2016.03.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rdkit'), ('version', '2016.03.3')])), ('source', CommentedMap([('url', 'https://github.com/rdkit/rdkit/archive/Release_2016_03_3.tar.gz'), ('fn', 'Release_2016_03_3.tar.gz'), ('sha256', '229b33e63be90b202b771007bd30e17dddb079a96415471ed7c66420ac28d93a')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake [unix]', 'requests', 'boost >=1.48.0,<{CONDA_BOOST}}*', 'python', 'numpy >=1.7', 'pillow', 'freetype', 'py2cairo']), ('run', ['python', 'numpy >=1.7', 'boost <{CONDA_BOOST}}*', 'pillow'])])), ('test', CommentedMap([('imports', ['rdkit', 'rdkit.Avalon', 'rdkit.Dbase', 'rdkit.ML', 'rdkit.RDPaths', 'rdkit.VLib', 'rdkit.Chem', 'rdkit.DistanceGeometry', 'rdkit.Numerics', 'rdkit.RDRandom', 'rdkit.DataManip', 'rdkit.ForceField', 'rdkit.RDConfig', 'rdkit.DataStructs', 'rdkit.Geometry'])])), ('about', CommentedMap([('home', 'http://rdkit.org'), ('license', 'BSD'), ('summary', 'Open-Source Cheminformatics Software')]))])), 'rgi': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rgi/3.2.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('git_url', '<{ source_url }}'), ('git_rev', '<{ commit }}')])), ('build', CommentedMap([('skip', True), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'six >=1.7.0', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'blast', 'biopython', 'six >=1.7.0', 'libgcc', 'zlib', 'prodigal', 'diamond >=0.8.24.86'])])), ('test', CommentedMap([('commands', ['rgi -h', 'rgi_jsontab -h', 'rgi_clean -h', 'rgi_load -h', 'rgi_jsonformat -h', 'rgi --software_version', 'rgi --data_version'])])), ('about', CommentedMap([('home', 'https://card.mcmaster.ca'), ('license', 'https://card.mcmaster.ca/about'), ('summary', 'This tool provides a preliminary annotation of your DNA sequence(s) based upon the data available in The Comprehensive Antibiotic Resistance Database (CARD). Hits to genes tagged with Antibiotic Resistance ontology terms will be highlighted. As CARD expands to include more pathogens, genomes, plasmids, and ontology terms this tool will grow increasingly powerful in providing first-pass detection of antibiotic resistance associated genes. See license at CARD website')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))])), 'rnacode': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnacode/0.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://wash.github.io/rnacode/'), ('license', ''), ('summary', 'RNAcode - Analyze the protein coding potential in multiple sequence alignments RNAcode relies on evolutionary signatures including synonymous/conservative mutations and conservation of the reading frame. It does not use any species specific sequence characteristics whatsoever and does not use any machine learning techniques.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'rnacode'), ('version', '0.3')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'RNAcode-0.3.tar.gz'), ('sha256', '86c600a07ecd04d243fd6eb3c6cc5b20fd6159f34f69423e7391c0ce8f311048'), ('url', 'http://github.com/downloads/wash/rnacode/RNAcode-0.3.tar.gz')])), ('test', CommentedMap([('commands', ['RNAcode --help &> /dev/null'])]))])), 'rnashapes': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnashapes/3.3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bibiserv.techfak.uni-bielefeld.de/rnashapes'), ('license', 'Uknown - Please refer to the tool homepage'), ('summary', \"RNAshape abstraction maps structures to a tree-like domain of shapes, retaining adjacency and nesting of structural features, but disregarding helix lengths. Shape abstraction integrates well with dynamic programming algorithms, and hence it can be applied during structure prediction rather than afterwards. This avoids exponential explosion and can still give us a non-heuristic and complete account of properties of the molecule's folding space.\")])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'rnashapes'), ('version', '3.3.0')])), ('requirements', CommentedMap([('build', ['perl', 'bellmans-gapc', 'gcc', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*']), ('run', ['perl', 'libgcc', 'bellmans-gapc', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*'])])), ('source', CommentedMap([('fn', 'fold-grammars.zip'), ('sha256', 'd5d21ce1035e6cb9952cf803f8ad43cd986c172b6b11c2c43a5ad6722d98f2db'), ('url', 'https://depot.galaxyproject.org/software/Fold-Grammars/Fold-Grammars_1.0_src_all.zip')])), ('test', CommentedMap([('commands', ['RNAshapes  -h', 'RNAshapes  CGCAUGCUAGUCGCG'])]))])), 'rnastructure': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnastructure/5.7/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://rna.urmc.rochester.edu/RNAstructure.html'), ('license', ''), ('summary', 'RNAstructure is a complete package for RNA and DNA secondary structure  prediction and analysis. It includes algorithms for   secondary structure  prediction, including facility to predict base pairing probabilities. It  also can be used to predict bimolecular structures and can predict the  equilibrium binding affinity of an oligonucleotide to a structured RNA  target. This is useful for siRNA design. It can also predict secondary  structures common to two, unaligned sequences, which is much more accurate  than single sequence secondary structure prediction. Finally, RNAstructure  can take a number of different types of experiment mapping data to constrain  or restrain structure prediction. These include chemical mapping, enzymatic  mapping, NMR, and SHAPE data.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'rnastructure'), ('version', '5.7')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RNAstructure-5.7.tgz'), ('sha256', '7d7060c2180b95e0d0e0a840499df48de3571e37ca6bef51325377fe7ca28ae6'), ('url', 'http://depot.galaxyproject.org/package/source/RNAstructure-5.7.tgz')])), ('test', CommentedMap([('commands', ['refold -h', 'ProbKnot -h', 'dot2ct -h'])]))])), 'shape-it': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shape-it/1.0.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shape_it'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'shape-it-1.0.1.tar.gz'), ('url', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/_downloads/shape-it-1.0.1.tar.gz'), ('md5', '65f8cff881d92e1ab41df52b181eda06')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openbabel ==2.4.1', 'cmake']), ('run', ['openbabel ==2.4.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['shape-it --version'])])), ('about', CommentedMap([('home', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/software/shape-it/1.0.1/shape-it.html'), ('license', 'LGPL'), ('summary', 'Shape alignment against a database of molecules')]))])), 'shorah': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shorah/1.1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shorah'), ('version', '1.1.0')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'shorah-1.1.0.tar.bz2'), ('url', 'https://github.com/cbg-ethz/shorah/releases/download/v1.1.0/shorah-1.1.0.tar.bz2'), ('sha256', 'e4fe6cfb854163cb0963a977b9f08faee68027298c247b7e305337f3ebc3234c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'gsl', 'zlib']), ('run', ['python', 'samtools', 'biopython', 'numpy', 'gsl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/shorah'), ('license', 'GPLv3'), ('summary', 'The Short Reads Assembly into Haplotypes (ShoRAH) program for inferring viral haplotypes from NGS data')])), ('test', CommentedMap([('commands', ['dec.py -h', 'shorah.py -h'])]))])), 'smallgenomeutilities': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smallgenomeutilities/0.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'smallgenomeutilities'), ('version', '0.1')])), ('source', CommentedMap([('fn', 'smallgenomeutilities-0.1.tar.gz'), ('url', 'https://pypi.io/packages/source/s/smallgenomeutilities/smallgenomeutilities-0.1.tar.gz'), ('md5', '339439b1f97ecb1fc91378f78a06bebe')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'numpy', 'progress', 'pysam']), ('run', ['python', 'biopython', 'numpy', 'progress', 'pysam'])])), ('test', CommentedMap([('imports', ['smallgenomeutilities']), ('commands', ['convert_qr -h', 'convert_reference -h'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/smallgenomeutilities'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('summary', 'A collection of scripts that are useful for dealing with viral RNA NGS data')]))])), 'sortmerna': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sortmerna/2.1b/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sortmerna'), ('version', '2.1b')])), ('source', CommentedMap([('fn', '2.1b.tar.gz'), ('url', 'https://github.com/biocore/sortmerna/archive/2.1b.tar.gz'), ('sha256', 'b3d122776c323813971b35991cda21a2c2f3ce817daba68a4c4e09d4367c0abe')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['sortmerna --version', \"merge-paired-reads.sh  --help | grep 'merged-reads.fastq'\", \"unmerge-paired-reads.sh  --help | grep 'merged-reads-file.fastq'\"])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/RNA/sortmerna'), ('license', 'LGPL'), ('summary', 'SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.')]))])), 'strip-it': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/strip-it/1.0.2/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'strip-it-1.0.2.tar.gz'), ('url', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/_downloads/strip-it-1.0.2.tar.gz'), ('md5', '963da0b005774a7a2eba4b4c384dca2a')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'strip_it'), ('version', '1.0.2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openbabel ==2.4.1', 'cmake']), ('run', ['openbabel ==2.4.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['strip-it --version'])])), ('about', CommentedMap([('home', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/software/strip-it/1.0.2/strip-it.html'), ('license', 'LGPL'), ('summary', 'Strip-it is a program that extracts predefined scaffolds from organic small molecules.')]))])), 'tabix': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tabix/0.2.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tabix'), ('version', '0.2.5')])), ('source', CommentedMap([('url', 'https://downloads.sourceforge.net/project/samtools/tabix/tabix-0.2.5.tar.bz2'), ('sha256', '6ce0da41a7ef82aa34db413602cc7cd11c08d606f36ebba793da41ec402e2f86')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['zlib', 'gcc', 'llvm']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['test -x \"$PREFIX/bin/tabix\"', 'test -x \"$PREFIX/bin/bgzip\"'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/samtools'), ('license', 'BSD'), ('summary', 'A set of tools written in Perl and C++ for working with VCF files.')]))])), 'tclap': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tclap/1.2.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tclap'), ('version', '1.2.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'tclap-1.2.1.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/tclap/tclap-1.2.1.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://tclap.sourceforge.net/'), ('license', 'MIT'), ('summary', 'TCLAP is a small, flexible library that provides a simple interface for defining and accessing command line arguments.')]))])), 'tesseract': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/tesseract/3.04.01/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'tesseract'), ('version', '3.04.01')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '3.04.01.tar.gz'), ('url', 'https://github.com/tesseract-ocr/tesseract/archive/3.04.01.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libpng', 'jpeg', 'libtiff', 'zlib', 'leptonica']), ('run', ['libgcc', 'libpng', 'jpeg', 'libtiff', 'zlib', 'leptonica'])])), ('test', CommentedMap([('commands', ['tesseract --help'])])), ('about', CommentedMap([('home', 'https://github.com/tesseract-ocr/tesseract'), ('license', 'Apache License, Version 2.0'), ('summary', 'OCR engine')]))])), 'trimal': ('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimal/1.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trimal'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'v1.4.1.tar.gz'), ('url', 'https://github.com/scapella/trimal/archive/v1.4.1.tar.gz'), ('md5', 'cb193397635da45be350e9ac96dafe45')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['readal --version | grep readAl > /dev/null', 'statal --version | grep statAl > /dev/null', 'trimal --version | grep trimAl > /dev/null'])])), ('about', CommentedMap([('home', 'http://trimal.cgenomics.org'), ('summary', 'A tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment'), ('license', 'GNU General Public License v3 or later (GPLv3+)')]))]))}\n",
      "EXTRA has been found: weeder\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EXTRA has been found: bbmap\n",
      "EXTRA has been found: freec\n",
      "EXTRA has been found: searchgui\n",
      "EXTRA has been found: trinity\n",
      "EXTRA has been found: chromhmm\n",
      "EXTRA has been found: fraggenescan\n",
      "EXTRA has been found: gatk\n",
      "EXTRA has been found: tripal\n",
      "EXTRA has been found: openms\n",
      "EXTRA has been found: freebayes\n",
      "EXTRA has been found: vep\n",
      "EXTRA has been found: sff2fastq\n",
      "EXTRA has been found: qiime\n",
      "EXTRA has been found: effectivet3\n",
      "EXTRA has been found: mixomics\n",
      "EXTRA has been found: Novoalign\n",
      "EXTRA has been found: multiqc\n"
     ]
    }
   ],
   "source": [
    "bioconda_recipes_path = \"/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes\"\n",
    "mapping_file          = \"/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-biotools/mapping_matchonly.csv\"\n",
    "mapping = get_annotation_mapping(mapping_file)\n",
    "yaml_files = get_bioconda_yalm(bioconda_recipes_path)\n",
    "print(yaml_files)\n",
    "annotate_yamls(yaml_files, mapping)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'meta.yaml': CommentedMap([('package', CommentedMap([('name', 'zifa'), ('version', '0.1.0')])), ('source', CommentedMap([('fn', 'zifa-0.1.0.zip'), ('url', 'https://github.com/epierson9/ZIFA/archive/e40fdbf6c41cd7e564ab03d242fc071f5ac543b7.zip')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['ZIFA'])])), ('about', CommentedMap([('home', 'https://github.com/epierson9/ZIFA'), ('license', 'MIT'), ('summary', 'Dimensionality reduction for zero-inflated single-cell gene expression analysis')]))]), 'biotools.yaml': CommentedMap([('function', [CommentedMap([('input', [CommentedMap([('dataType', CommentedMap([('term', 'RNA sequence'), ('uri', 'http://edamontology.org/data_3495')])), ('dataFormat', [CommentedMap([('term', 'FASTA'), ('uri', 'http://edamontology.org/format_1929')]), CommentedMap([('term', 'FASTA-like (text)'), ('uri', 'http://edamontology.org/format_2200')])])])]), ('output', [CommentedMap([('dataType', CommentedMap([('term', 'Alignment'), ('uri', 'http://edamontology.org/data_1916')])), ('dataFormat', [CommentedMap([('term', 'ClustalW format'), ('uri', 'http://edamontology.org/format_1982')]), CommentedMap([('term', 'FASTA'), ('uri', 'http://edamontology.org/format_1929')]), CommentedMap([('term', 'PDF'), ('uri', 'http://edamontology.org/format_3508')])])]), CommentedMap([('dataType', CommentedMap([('term', 'Phylogenetic tree'), ('uri', 'http://edamontology.org/data_0872')])), ('dataFormat', [CommentedMap([('term', 'newick'), ('uri', 'http://edamontology.org/format_1910')])])]), CommentedMap([('dataType', CommentedMap([('term', 'RNA structure'), ('uri', 'http://edamontology.org/data_1465')])), ('dataFormat', [CommentedMap([('term', 'EPS'), ('uri', 'http://edamontology.org/format_3466')])])])]), ('functionName', [CommentedMap([('term', 'Structure-based sequence alignment'), ('uri', 'http://edamontology.org/operation_0294')])]), ('functionDescription', 'LocARNA simultaneously folds and aligns input sequences. Specifications of additional constraints or fixed input structures are possible.')])]), ('maturity', 'Stable'), ('additionDate', '2016-01-20T14:57:08Z'), ('cost', 'Free'), ('description', 'Structure-based alignment of RNAs'), ('credits', CommentedMap([('creditsInstitution', ['University Freiburg']), ('creditsDeveloper', ['Sebastian Will'])])), ('resourceType', ['Tool']), ('accessibility', 'Public'), ('contact', [CommentedMap([('contactEmail', 'rna@rna.informatik.uni-freiburg.de'), ('contactName', 'Rolf Backofen'), ('contactRole', ['Developer'])]), CommentedMap([('contactEmail', 'will@bioinf.uni-leipzig.de'), ('contactName', 'Sebastian Will'), ('contactRole', ['Developer'])])]), ('topic', [CommentedMap([('term', 'Sequence comparison'), ('uri', 'http://edamontology.org/topic_0159')])]), ('affiliation', 'rnateam'), ('version', '1.8.7'), ('publications', CommentedMap([('publicationsPrimaryID', '17432929')])), ('interface', [CommentedMap([('interfaceType', 'Command line')])]), ('homepage', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('id', 42667), ('name', 'LocARNA')]), '0.4': CommentedMap([('package', CommentedMap([('name', 'perl-test-toolbox'), ('version', '0.4')])), ('source', CommentedMap([('fn', 'Test-Toolbox-0.4.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/Test-Toolbox-0.4.tar.gz'), ('md5', 'b8a83d332dcd0691b68ed9e0a0a53e7b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Test::Toolbox'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Toolbox'), ('license', 'perl_5'), ('summary', 'Test::Toolbox - tools for testing')]))]), '0.5': CommentedMap([('package', CommentedMap([('name', 'multigps'), ('version', 0.5)])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'multigps_v0.5.jar'), ('url', 'http://lugh.bmb.psu.edu/software/multigps/multigps_v0.5.jar'), ('md5', '102edef55aaffd3b30d446c88a7bf7d6')])), ('requirements', CommentedMap([('run', ['bioconductor-edger', 'openjdk >=7', 'meme'])])), ('test', CommentedMap([('commands', [\"multigps --version 2>&1 | grep 'MultiGPS' &>/dev/null\"])])), ('about', CommentedMap([('home', 'http://mahonylab.org/software/multigps/'), ('license', 'MIT'), ('summary', 'MultiGPS is a framework for analyzing collections of multi-condition ChIP-seq datasets and characterizing differential binding events between conditions.')]))]), '1.5.2': CommentedMap([('package', CommentedMap([('name', 'abyss'), ('version', '1.5.2')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'abyss-1.5.2.tar.gz'), ('url', 'https://github.com/bcgsc/abyss/releases/download/1.5.2/abyss-1.5.2.tar.gz'), ('md5', '10d6d72d1a915e618d41a5cbbcf2364c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'google-sparsehash', 'sqlite']), ('run', ['bwa', 'libgcc', 'sqlite'])])), ('test', CommentedMap([('commands', ['ABYSS --version'])])), ('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler')]))]), '1.9.0': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.9.0')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'locarna-1.9.0.tar.gz'), ('url', 'https://github.com/s-will/LocARNA/releases/download/v1.9.0/locarna-1.9.0.tar.gz'), ('sha256', '83e47940a0ab851b5247cdd1d1814d7568d971b6883d33d911557efe7a4980cd')])), ('requirements', CommentedMap([('build', ['viennarna ==2.3.1', 'llvm', 'gcc']), ('run', ['viennarna ==2.3.1', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))]), '2.0.1-k128': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}-k128'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/<{ name|lower }}/releases/<{ version }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '596c56a476b26a294b46e6e36890c136')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'google-sparsehash', 'sqlite', 'openmpi']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'bwa', 'pandoc', 'sqlite', 'openmpi'])])), ('test', CommentedMap([('commands', ['ABYSS --version'])])), ('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler')]))]), '1.0.3': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.0.3')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'stringtie-1.0.3.tar.gz'), ('sha256', '1567d9d87d9375a3db03fa0b682eaef4e89899df64fd001c14d475cc9e737e08'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.0.3.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '2.4.0': CommentedMap([('package', CommentedMap([('name', 'anvio'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'anvio-2.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/1a/f4/e75b895b3348ec7d0d21e220735d0f9dd0ea2e67ef9a6ebb9c92bd1d7593/anvio-2.4.0.tar.gz'), ('md5', '581caa914116a24938938f99e3227b43')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'setuptools', 'cython', 'numpy', 'bottle >=0.12.9', 'pysam ==0.11.1', 'ete3 ==3.0.0b35', 'scipy >=0.13.3', 'scikit-learn >=0.17.1', 'django >=1.9.7', 'h5py >=2.6.0', 'cherrypy >=8,<9', 'requests >=2.10.0', 'psutil >=5.0.1', 'mistune ==0.7.4', 'openblas', 'gsl ==2.2.1', 'six ==1.10.0']), ('run', ['python', 'libgcc', 'cython', 'numpy', 'bottle >=0.12.9', 'pysam ==0.11.1', 'ete3 ==3.0.0b35', 'scipy >=0.13.3', 'scikit-learn >=0.17.1', 'django >=1.9.7', 'h5py >=2.6.0', 'cherrypy >=8,<9', 'requests >=2.10.0', 'psutil >=5.0.1', 'mistune ==0.7.4', 'six ==1.10.0', 'openblas', 'gsl ==2.2.1', 'sqlite', 'diamond', 'blast', 'prodigal', 'mcl', 'muscle', 'hmmer', 'bwa', 'samtools'])])), ('test', CommentedMap([('commands', ['anvi-pan-genome --help', 'anvi-script-reformat-fasta --help', 'anvi-profile --version'])])), ('about', CommentedMap([('home', 'http://merenlab.org/software/anvio/index.html'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An interactive analysis and visualization platform for omics data'), ('license_family', 'GPL3')]))]), '1.2.36': CommentedMap([('about', CommentedMap([('home', 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/'), ('license', 'GPLv3'), ('summary', 'ARAGORN, tRNA (and tmRNA) detection')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'aragorn'), ('version', '1.2.36')])), ('source', CommentedMap([('fn', 'aragorn1.2.36.tgz'), ('sha256', 'ab06032589e45aa002f8616333568e9ab11034b3a675f922421e5f1c3e95e7b5'), ('url', 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/Downloads/aragorn1.2.36.tgz')])), ('test', CommentedMap([('commands', ['aragorn'])]))]), '3.11.14': CommentedMap([('about', CommentedMap([('home', 'http://www.niehs.nih.gov/research/resources/software/biostatistics/art/'), ('license', 'GPLv2'), ('summary', 'Illumina, 454 and Solid read simulator')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'art'), ('version', '3.11.14')])), ('requirements', CommentedMap([('build', ['gsl ==1.16']), ('run', ['gsl ==1.16'])])), ('source', CommentedMap([('fn', 'artsrcvanillaicecream031114linuxtgz.tgz'), ('sha256', '31f172ac5953ec32e4ae7787b2aa25e62d55b086647ac1e3e6298bbe39e89645'), ('url', 'http://www.niehs.nih.gov/research/resources/assets/docs/artsrcvanillaicecream031114linuxtgz.tgz')])), ('test', CommentedMap([('commands', ['art_454'])]))]), '1.0': CommentedMap([('package', CommentedMap([('name', 'stacks_summary'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'stacks_summary-1.0.tar.gz'), ('url', 'https://pypi.python.org/packages/fd/4d/b66c0171669e42a92b6961cf769fcd7186b60e126868fb7c216d9d9a07ec/stacks_summary-1.0.tar.gz'), ('md5', '47393995a2ce15fe2d48eedebfde011c')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['stacks']), ('commands', ['stacks_summary.py --help'])])), ('about', CommentedMap([('home', 'https://github.com/mariabernard/galaxy_wrappers'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Stacks reports generator'), ('license_family', 'Public-Domain')]))]), '1.1': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM, BAM and CRAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '1.1')])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'zlib ==1.2.8']), ('run', ['ncurses ==5.9', 'zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'samtools-1.1.tar.bz2'), ('sha256', 'c24d26c303153d72b5bf3cc11f72c6c375a4ca1140cc485648c8c5473414b7f8'), ('url', 'http://downloads.sourceforge.net/project/samtools/samtools/1.1/samtools-1.1.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))]), '2.3.0': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.0')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.0.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.0.tar.gz'), ('sha256', 'dd5b8c9c42dabc2521be11de99f1f102e912cf921b3789345e90e51a64be33ff')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '0.2': CommentedMap([('package', CommentedMap([('name', 'pasta'), ('version', '0.2')])), ('source', CommentedMap([('fn', '6d52da1.tar.gz'), ('url', 'https://github.com/smirarab/pasta/archive/6d52da1.tar.gz'), ('md5', '69430d77b13f53505c3c9dc14bc1c4e2')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['certifi', 'dendropy <=3.13', 'gcc', 'gnu-wget', 'java-jdk >=6', 'llvm', 'pcre', 'python >=2.7,<3', 'pymongo', 'setuptools']), ('run', ['dendropy <=3.13', 'java-jdk >=6', 'libgcc', 'pcre', 'python >=2.7,<3', 'pymongo'])])), ('test', CommentedMap([('imports', ['pasta.alignment', 'pasta.configure', 'pasta.errors', 'pasta.filemgr', 'pasta.mainpasta', 'pasta.pastaalignerjob', 'pasta.pastajob', 'pasta.scheduler', 'pasta.settings', 'pasta.tools', 'pasta.treeholder', 'pasta.tree', 'pasta.usersettingclasses', 'pasta.utility']), ('commands', ['clustalw2 -help | grep OPTIONS', 'hmmalign -h | grep HMMER', 'hmmbuild -h | grep HMMER', 'muscle -version | grep MUSCLE', 'prank -help | grep prank', 'raxml -h | grep raxmlHPC', 'raxmlp -h | grep raxmlHPC'])])), ('about', CommentedMap([('home', 'https://github.com/smirarab/pasta'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An implementation of the PASTA (Practical Alignment using Sate and TrAnsitivity) algorithm')]))]), '0.3': CommentedMap([('about', CommentedMap([('home', 'http://wash.github.io/rnacode/'), ('license', ''), ('summary', 'RNAcode - Analyze the protein coding potential in multiple sequence alignments RNAcode relies on evolutionary signatures including synonymous/conservative mutations and conservation of the reading frame. It does not use any species specific sequence characteristics whatsoever and does not use any machine learning techniques.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'rnacode'), ('version', '0.3')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'RNAcode-0.3.tar.gz'), ('sha256', '86c600a07ecd04d243fd6eb3c6cc5b20fd6159f34f69423e7391c0ce8f311048'), ('url', 'http://github.com/downloads/wash/rnacode/RNAcode-0.3.tar.gz')])), ('test', CommentedMap([('commands', ['RNAcode --help &> /dev/null'])]))]), '0.7': CommentedMap([('about', CommentedMap([('home', 'https://github.com/tseemann/barrnap'), ('license', 'GPLv3'), ('summary', 'Find 5S, 16S and 23S ribosomal RNA genes in bacterial contigs')])), ('build', CommentedMap([('number', 3)])), ('package', CommentedMap([('name', 'barrnap'), ('version', '0.7')])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.22.0', 'hmmer >=3.0', 'libgcc'])])), ('source', CommentedMap([('fn', 'barrnap_v0.7.tar.gz'), ('sha256', 'ef2173e250f06cca7569c03404c9d4ab6a908ef7643e28901fbe9a732d20c09b'), ('url', 'https://github.com/tseemann/barrnap/archive/0.7.tar.gz')])), ('test', CommentedMap([('commands', ['barrnap --help 2>&1 | grep Options > /dev/null'])]))]), '0.8': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/tseemann/<{ name }}/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.22.0', 'hmmer >=3.1b'])])), ('test', CommentedMap([('source_files', ['examples/small.fna']), ('commands', ['(barrnap --help 2>&1; true) | grep -q Options', '(barrnap $PREFIX/lib/barrnap/examples/small.fna 2>&1; true) | grep -q \"Found 3 ribosomal RNA features\"'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/barrnap'), ('license', 'GPLv3'), ('summary', 'Find 5S, 16S and 23S ribosomal RNA genes in bacterial contigs')]))]), '37.10': CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/bbmap'), ('license', 'UC-LBL license (see package)'), ('summary', 'BBMap is a short read aligner, as well as various other bioinformatic tools')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}_<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0']), ('run', ['openjdk >=7.0'])])), ('test', CommentedMap([('commands', ['bbmap.sh -V 2>&1 | grep \"Executing\" > /dev/null'])])), ('extra', CommentedMap([('notes', 'BBMap is a series of Java programs, but they come with a number of custom\\nwrapper shell scripts. Each of these is symlinked to the conda bin directory\\nduring install.\\n')]))]), '37.17': CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/bbmap'), ('license', 'UC-LBL license (see package)'), ('summary', 'BBMap is a short read aligner, as well as various other bioinformatic tools')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}_<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0']), ('run', ['openjdk >=7.0'])])), ('test', CommentedMap([('commands', ['bbmap.sh -V 2>&1 | grep \"Executing\" > /dev/null'])])), ('extra', CommentedMap([('notes', 'BBMap is a series of Java programs, but they come with a number of custom\\nwrapper shell scripts. Each of these is symlinked to the conda bin directory\\nduring install.\\n')]))]), '0.6.2': CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.6.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.6.2.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.6.2.tar.bz2'), ('md5', '5e4043769ce3436f6098ec67f6e02332')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))]), '1.2': CommentedMap([('package', CommentedMap([('name', 'perl-number-misc'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'Number-Misc-1.2.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/Number-Misc-1.2.tar.gz'), ('md5', '7a0009b5be60378f204ed1fc9b2d5a19')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Number::Misc'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Number::Misc'), ('license', 'perl_5'), ('summary', 'Number::Misc - handy utilities for numbers')]))]), '1.3': CommentedMap([('package', CommentedMap([('name', 'perl-estscan1'), ('version', '1.3')])), ('source', CommentedMap([('fn', 'ESTScan1-1.3.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/ESTScan1/1.3/ESTScan1-1.3.tar.gz'), ('md5', '457e695657544d56e38b340ef3ebc46c')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'perl-btlib'])])), ('test', CommentedMap([('imports', ['ESTScan1']), ('commands', ['perl -e \\'use ESTScan1; my @mat = ESTScan1::LoadMatrix(\"HumanIso.smat\", \"CODING\", 4.0);\\''])])), ('about', CommentedMap([('home', 'http://estscan.sourceforge.net'), ('license', 'open source'), ('summary', 'Detects coding regions in DNA sequences even if they are of low quality. ESTScan.pm contains the Perl part of the code that reads in the matrices file. The C code that does the actual computation is located in estscan.c.')]))]), '1.3.1': CommentedMap([('package', CommentedMap([('name', 'carna'), ('version', '1.3.1')])), ('about', CommentedMap([('home', 'https://www.bioinf.uni-leipzig.de/~will/Software/CARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Constraint-based Alignment of RNA Ensembles')])), ('build', CommentedMap([('number', 4)])), ('source', CommentedMap([('fn', 'carna-1.3.1.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/carna-1.3.1.tar.gz'), ('sha256', '7c78463542c48322e2770eff544b46cffe5e4ce67f5fc58c05b4ead6063b541a')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gecode 4.4.0', 'viennarna 2.2.5', 'locarna 1.8.9']), ('run', ['libgcc', 'gecode 4.4.0', 'locarna 1.8.9'])])), ('test', CommentedMap([('commands', ['carna --version'])]))]), '2.16.2': CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.16.2')])), ('source', CommentedMap([('fn', 'BEDTools.v2.16.2.tar.gz'), ('url', 'https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/bedtools/BEDTools.v2.16.2.tar.gz'), ('sha256', 'f5f5c864eb3f465ac7fd5fa651e2e4dbc0cd8d9198367148c52f3be3f46c2772'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bedtools > /dev/null', \"bedtools --version | egrep '\\\\bv2\\\\.16\\\\.2\\\\b'\"])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]), '2.17.0': CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.17.0')])), ('source', CommentedMap([('fn', 'BEDTools.v2.17.0.tar.gz'), ('url', 'https://bedtools.googlecode.com/files/BEDTools.v2.17.0.tar.gz'), ('sha256', '6dcfacdcbccc968e872b81ba1f37a086b1bcc486571582ed595db717e924d0fe'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bedtools > /dev/null'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]), '2.19.1': CommentedMap([('about', CommentedMap([('home', ''), ('license', ''), ('summary', 'A swiss army knife for genome arithmetic.  https://github.com/arq5x/bedtools2 ')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'bedtools'), ('version', '2.19.1')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'bedtools-2.19.1.tar.gz'), ('sha256', '17253439e8191cb51a8bffba640c580e09ba158d3a0c66649e53d23ee3e50f77'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.19.1/bedtools-2.19.1.tar.gz')])), ('test', CommentedMap([('commands', [])]))]), '2.20.1': CommentedMap([('about', CommentedMap([('home', ''), ('license', ''), ('summary', 'A swiss army knife for genome arithmetic.  https://github.com/arq5x/bedtools2 ')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'bedtools'), ('version', '2.20.1')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'bedtools-2.20.1.tar.gz'), ('sha256', 'b5401810f8b12b683575f0119521dda64ff2f0a59faa308357405c4ae4e328d3'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.20.1/bedtools-2.20.1.tar.gz')])), ('test', CommentedMap([('commands', [])]))]), '2.22': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['base'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/base'), ('license', 'unknown'), ('summary', 'compile-time class fields')]))]), '2.23.0': CommentedMap([('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A swiss army knife for genome arithmetic.  https://github.com/arq5x/bedtools2 ')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'bedtools'), ('version', '2.23.0')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'bedtools-2.23.0.tar.gz'), ('sha256', '9dacaa561d11ce9835d1d51e5aeb092bcbe117b7119663ec9a671abac6a36056'), ('url', 'https://github.com/arq5x/bedtools2/archive/v2.23.0.tar.gz')])), ('test', CommentedMap([('commands', ['bedtools'])]))]), '2.25': CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.25.0')])), ('source', CommentedMap([('fn', 'bedtools-2.25.0.tar.gz'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.25.0/bedtools-2.25.0.tar.gz'), ('sha256', 'd737ca42e7df76c5455d3e6e0562cdcb62336830eaad290fd4133a328a1ddacc')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bedtools'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]), '2.26.0': CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.26.0gx')])), ('source', CommentedMap([('fn', 'bedtools-2.26.0.tar.gz'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.26.0/bedtools-2.26.0.tar.gz'), ('sha256', '65f32f32cbf1b91ba42854b40c604aa6a16c7d3b3ec110d6acf438eb22df0a4a'), ('patches', ['2.26.0_postrelease.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bedtools'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]), '0.7.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/biopet/biopet'), ('summary', 'Biopet (Bio Pipeline Execution Toolkit) is the main pipeline development framework of the LUMC Sequencing Analysis Support Core team.'), ('license', 'https://github.com/biopet/biopet/blob/develop/biopet-core/src/main/resources/nl/lumc/sasc/biopet/License.txt')])), ('package', CommentedMap([('name', 'biopet'), ('version', '0.7.0')])), ('source', CommentedMap([('url', 'https://github.com/biopet/biopet/releases/download/v0.7.0/Biopet-0.7.0-cb7a051c.jar'), ('sha256', 'e7d866c4705bb06729f3331e8341400f4f2eba9530ba8ab548fd78423fea5490')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk', 'python', 'r-base', 'r-ggplot2', 'r-reshape', 'r-argparse'])])), ('test', CommentedMap([('commands', ['biopet version', 'biopet license'])]))]), '2.12.1': CommentedMap([('package', CommentedMap([('name', 'bioconductor-cummerbund'), ('version', '2.12.1')])), ('source', CommentedMap([('fn', 'cummeRbund_2.12.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/cummeRbund_2.12.1.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cummeRbund\\')\"', '\"%R%\" -e \"library(\\'cummeRbund\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cummeRbund.html'), ('license', 'Artistic-2.0'), ('summary', 'Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.')]))]), '2.8.2': CommentedMap([('about', CommentedMap([('home', 'http://weblogo.berkeley.edu/'), ('license', 'MIT'), ('summary', 'Web based application designed to generate sequence logos')])), ('package', CommentedMap([('name', 'weblogo'), ('version', '2.8.2')])), ('build', CommentedMap([('number', 5)])), ('source', CommentedMap([('url', 'http://weblogo.berkeley.edu/release/weblogo.2.8.2.tar.gz'), ('fn', 'weblogo.2.8.2.tar.gz'), ('md5', '12886e0f41757e32ac2823d0f990ce0c')])), ('requirements', CommentedMap([('run', ['perl'])])), ('test', CommentedMap([('commands', ['seqlogo 2>&1 | grep usage > /dev/null', 'seqlogo -f foo.bar 2>&1 | grep \"input filename\" > /dev/null'])]))]), '2.0.9': CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'http://bioconductor.statistik.tu-dortmund.de/packages/3.3/bioc/src/contrib/DiffBind_2.0.9.tar.gz'), ('md5', '9d00cf8d809f138ce2a3b1e5620bbf51')])), ('build', CommentedMap([('skip', True), ('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-base', 'r-amap', 'r-dplyr', 'r-gplots', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-base', 'r-amap', 'r-dplyr', 'r-gplots', 'r-locfit', 'r-optparse', 'r-rcolorbrewer', 'r-getopt', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'<{ name }}\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DiffBind.html'), ('license', 'Artistic-2.0'), ('summary', 'Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.')]))]), '2.2.12': CommentedMap([('package', CommentedMap([('name', 'bioconductor-diffbind'), ('version', '2.2.12')])), ('source', CommentedMap([('fn', 'DiffBind_2.2.12.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/DiffBind_2.2.12.tar.gz', 'https://depot.galaxyproject.org/software/DiffBind/DiffBind_2.2.12_src_all.tar.gz']), ('sha256', '9ee1454c851559ac713b5ef83e22f577c801aae76f843ada3a4c033725e46d70')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DiffBind\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/DiffBind.html'), ('license', 'Artistic-2.0'), ('summary', 'Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.')]))]), '1.48.0': CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'xcms_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.48.0.tar.gz'), ('md5', '83adb6afdb8f6d819bd6d37c7bf583f7')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))]), '3.12.1': CommentedMap([('package', CommentedMap([('name', 'bioconductor-edger'), ('version', '3.12.1')])), ('source', CommentedMap([('fn', 'c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('url', 'https://github.com/Bioconductor-mirror/edgeR/archive/c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('md5', 'a76554192ad964763f2c1040e04b7347')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'edgeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/edgeR.html'), ('license', 'GPL (>=2)'), ('summary', 'Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.')]))]), '1.22.0': CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggbio'), ('version', '1.22.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/ggbio_1.22.0.tar.gz'), ('sha256', '5ba61f3a2065de08a4ca888d6da3fee6aec5ca42bc5a47df64bd4574f4d81033')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggbio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggbio.html'), ('license', 'Artistic-2.0'), ('summary', 'The ggbio package extends and specializes the grammar of graphics for biological data. The graphics are designed to answer common scientific questions, in particular those often asked of high throughput genomics data. All core Bioconductor data structures are supported, where appropriate. The package supports detailed views of particular genomic regions, as well as genome-wide overviews. Supported overviews include ideograms and grand linear views. High-level plots include sequence fragment length, edge-linked interval to data view, mismatch pileup, and several splicing summaries.')]))]), '3.28.10': CommentedMap([('package', CommentedMap([('name', 'bioconductor-limma'), ('version', '3.28.10')])), ('source', CommentedMap([('fn', 'limma_3.28.10_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/limma/limma_3.28.10_src_all.tar.gz'), ('sha256', '62c9337a640d4ee31abbec77df37ea3b0d75826cbace3de9914561106e04dfdc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'limma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/limma.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.\\n')]))]), '3.28.2': CommentedMap([('package', CommentedMap([('name', 'bioconductor-limma'), ('version', '3.28.2')])), ('source', CommentedMap([('fn', 'limma_3.28.2.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/limma/limma_3.28.2_src_all.tar.gz'), ('sha256', '70acdd49e77b8514afd8ca364c3c47b6fbeef46b7a40881d5312832e90cf9c75')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'limma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/limma.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.\\n')]))]), '3.3.0': CommentedMap([('about', CommentedMap([('home', 'http://bibiserv.techfak.uni-bielefeld.de/rnashapes'), ('license', 'Uknown - Please refer to the tool homepage'), ('summary', \"RNAshape abstraction maps structures to a tree-like domain of shapes, retaining adjacency and nesting of structural features, but disregarding helix lengths. Shape abstraction integrates well with dynamic programming algorithms, and hence it can be applied during structure prediction rather than afterwards. This avoids exponential explosion and can still give us a non-heuristic and complete account of properties of the molecule's folding space.\")])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'rnashapes'), ('version', '3.3.0')])), ('requirements', CommentedMap([('build', ['perl', 'bellmans-gapc', 'gcc', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*']), ('run', ['perl', 'libgcc', 'bellmans-gapc', 'boost <{CONDA_BOOST}}*', 'gsl <{CONDA_GSL}}*'])])), ('source', CommentedMap([('fn', 'fold-grammars.zip'), ('sha256', 'd5d21ce1035e6cb9952cf803f8ad43cd986c172b6b11c2c43a5ad6722d98f2db'), ('url', 'https://depot.galaxyproject.org/software/Fold-Grammars/Fold-Grammars_1.0_src_all.zip')])), ('test', CommentedMap([('commands', ['RNAshapes  -h', 'RNAshapes  CGCAUGCUAGUCGCG'])]))]), '1.4.0': CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/dan2097/opsin/'), ('license', 'Artistic License 2.0'), ('summary', 'OPSIN is a Java(1.6+) library for IUPAC name-to-structure conversion offering high recall and precision on organic chemical nomenclature.')])), ('package', CommentedMap([('name', 'opsin'), ('version', '1.4.0')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'opsin-1.4.0-jar-with-dependencies.jar'), ('url', 'https://bitbucket.org/dan2097/opsin/downloads/opsin-1.4.0-jar-with-dependencies.jar')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6'])])), ('test', CommentedMap([('commands', ['opsin --help', 'opsin -Xms512m -Xmx1g --help'])])), ('extra', CommentedMap([('notes', 'Opsin is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run opsin with \"opsin -Xms512m -Xmx1g --help\"\\n')]))]), '2.13.0': CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraphviz'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Rgraphviz_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rgraphviz_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rgraphviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html'), ('license', 'EPL'), ('summary', 'Interfaces R with the AT and T graphviz library for plotting R graph objects from the graph package.')]))]), '1.2.14': CommentedMap([('package', CommentedMap([('name', 'bioconductor-ropls'), ('version', '1.2.14')])), ('source', CommentedMap([('fn', 'ropls_1.2.14.tar.gz'), ('url', 'https://bioconductor.riken.jp/packages/3.2/bioc/src/contrib/ropls_1.2.14.tar.gz'), ('md5', '0d0cfa9b7458da283ce343eff9211a81')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ropls\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ropls.html'), ('license', 'CeCILL'), ('summary', 'Latent variable modeling with Principal Component Analysis (PCA) and Partial Least Squares (PLS) are powerful methods for visualization, regression, classification, and feature selection of omics data where the number of variables exceeds the number of samples and with multicollinearity among variables. Orthogonal Partial Least Squares (OPLS) enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation. While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to scores, loadings and weights plots, the package provides metrics and graphics to determine the optimal number of components (e.g. with the R2 and Q2 coefficients), check the validity of the model by permutation testing, detect outliers, and perform feature selection (e.g. with Variable Importance in Projection or regression coefficients). The package can be accessed via a user interface on the Workflow4Metabolomics.org online resource for computational metabolomics (built upon the Galaxy environment).')]))]), '0.8.11': CommentedMap([('package', CommentedMap([('name', 'bioconductor-s4vectors'), ('version', '0.8.11')])), ('source', CommentedMap([('fn', 'S4Vectors_0.8.11.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.11.tar.gz'), ('md5', '02b20a992bf46bb2cbdef8c90d76b421')])), ('build', CommentedMap([('skip', False), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.15.10', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.15.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'S4Vectors\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/S4Vectors.html'), ('license', 'Artistic-2.0'), ('summary', 'The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).')]))]), '1.44.0': CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'xcms_1.44.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.44.0.tar.gz'), ('md5', '1197c847d85414bd374f3f8563516c8b')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))]), '1.46.0': CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'xcms_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.46.0.tar.gz'), ('md5', 'ddde0d6cb128f66d5f922f3fd12c5590')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))]), '2.1.4': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '2.1.4')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'SearchGUI-2.1.4-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/SearchGUI/2.1.4/SearchGUI-2.1.4-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '2.1.5': CommentedMap([('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Tools for RNA secondary structure prediction and comparison')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'viennarna'), ('version', '2.1.5')])), ('requirements', CommentedMap([('build', ['perl ==5.18.1']), ('run', ['perl ==5.18.1'])])), ('source', CommentedMap([('fn', 'ViennaRNA-2.1.5.tar.gz'), ('sha256', '2d02103b6a5d1536df26607ee00b8be364758f2cb796aeb010517c311e861d58'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.1.5.tar.gz')])), ('test', CommentedMap([('commands', ['RNAcofold --version'])]))]), '0.8.0': CommentedMap([('package', CommentedMap([('name', 'dr-disco'), ('version', '0.8.0')])), ('source', CommentedMap([('fn', 'v0.8.0.tar.gz'), ('url', 'https://github.com/yhoogstrate/dr-disco/archive/v0.8.0.tar.gz'), ('sha256', '64f9c553389869d563de9423fffd6da4e4eb9df734580a313e7015dfc0559f82')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy']), ('run', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['drdisco']), ('commands', ['LANG=C.UTF-8 dr-disco --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/dr-disco/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Dr. Disco: fusion gene and genomic breakpoint detection in random hexamer RNA-seq data')]))]), '2.2.21': CommentedMap([('package', CommentedMap([('name', 'blast'), ('version', '2.2.21')])), ('source', CommentedMap([('fn', 'ncbi-blast-2.2.21+-universal-macosx.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.2.21/ncbi-blast-2.2.21+-universal-macosx.tar.gz'), ('md5', '03cbf1c151691ce22f22c337ba1f00a5'), ('patches', ['legacy_blast.pl.patch', 'update_blastdb.pl.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['blastdb_aliastool -version > /dev/null', 'blastdbcheck -version > /dev/null', 'blastdbcmd -version > /dev/null', 'blastn -version > /dev/null', 'blastp -version > /dev/null', 'blastx -version > /dev/null', 'convert2blastmask -version > /dev/null', 'dustmasker -version > /dev/null', 'legacy_blast.pl --version > /dev/null', 'makeblastdb -version > /dev/null', 'makembindex -version > /dev/null', 'psiblast -version > /dev/null', 'rpsblast -version > /dev/null', 'rpstblastn -version > /dev/null', 'segmasker -version > /dev/null', 'tblastn -version > /dev/null', 'tblastx -version > /dev/null', 'windowmasker -version > /dev/null', 'update_blastdb.pl --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs'), ('license', 'Public Domain'), ('summary', 'BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.')]))]), '2.5.0': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '2.5.0')])), ('source', CommentedMap([('fn', 'picard-tools-2.5.0.zip'), ('url', 'https://github.com/broadinstitute/picard/releases/download/2.5.0/picard-tools-2.5.0.zip'), ('md5', '0b9e2c9dc0043b5f8ef7b8f9b39780ae')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', [\"picard 2>&1 | grep 'USAGE: PicardCommandLine'\"])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))]), '2.2.22': CommentedMap([('package', CommentedMap([('name', 'blast-legacy'), ('version', '2.2.22')])), ('source', CommentedMap([('fn', 'blast-2.2.22-x64-linux.tar.gz'), ('url', 'ftp://ftp.ncbi.nlm.nih.gov/blast/executables/legacy/2.2.22/blast-2.2.22-x64-linux.tar.gz'), ('md5', '84e900b276a6e956dbbff9e6321155cf')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('test', CommentedMap([('commands', ['bl2seq --help | grep bl2seq > /dev/null', 'blastall --help | grep blastall > /dev/null', 'blastclust --help | grep blastclust > /dev/null', 'blastpgp --help | grep blastpgp > /dev/null', 'copymat --help | grep copymat > /dev/null', 'fastacmd --help | grep fastacmd > /dev/null', 'formatdb --help | grep formatdb > /dev/null', 'formatrpsdb --help | grep formatrpsdb > /dev/null', 'impala --help | grep impala > /dev/null', 'makemat --help | grep makemat > /dev/null', 'megablast --help | grep megablast > /dev/null', 'rpsblast --help | grep rpsblast > /dev/null', 'seedtop --help | grep seedtop > /dev/null'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov'), ('license', 'Public Domain'), ('summary', 'The Basic Local Alignment Search Tool (BLAST) finds regions of local similarity between sequences.')]))]), '1.0.0': CommentedMap([('package', CommentedMap([('name', 'shorah'), ('version', '1.0.0')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', 'shorah-1.0.0.tar.bz2'), ('url', 'https://github.com/cbg-ethz/shorah/releases/download/v1.0.0/shorah-1.0.0.tar.bz2'), ('md5', 'f0d057d3947b533ef4308458de9bfa8d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'gsl', 'zlib']), ('run', ['python', 'samtools', 'biopython', 'gsl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/shorah'), ('license', 'GPLv3'), ('summary', 'The Short Reads Assembly into Haplotypes (ShoRAH) program for inferring viral haplotypes from NGS data')])), ('test', CommentedMap([('commands', ['dec.py -h', 'shorah.py -h'])]))]), '2.2.4': CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'), ('license', 'GPLv3'), ('summary', 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.   http://bowtie-bio.sourceforge.net/bowtie2/index.shtml ')])), ('package', CommentedMap([('name', 'bowtie2'), ('version', '2.2.4')])), ('source', CommentedMap([('fn', 'bowtie2-2.2.4-source.zip'), ('url', 'https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.2.4/bowtie2-2.2.4-source.zip'), ('md5sum', 'd41d8cd98f00b204e9800998ecf8427e')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'python']), ('run', ['python', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['bowtie2 --version', 'bowtie2-build --help', 'bowtie2-inspect-l --help', 'bowtie2-align-s --help', 'bowtie2_wrapper.sh -h'])]))]), '2.2.5': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.2.5')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.2.5.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.2.5.tar.gz'), ('sha256', '12654f0ba5e516d5eeb018dc76485b1c5a3ba010c61f40f1221aad5c3815cc29')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAfold --version', 'RNAplfold --version'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '0.9.1': CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'pysam-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/40/15/20b22dc3d017ec123e533d062b982b111b0214168905de3221b5caf5f766/pysam-0.9.1.tar.gz'), ('md5', 'd87765ad1e0afe534255c20cce5478e4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'cython', 'python', 'setuptools', 'zlib', 'curl']), ('run', ['libgcc', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'python', 'zlib', 'curl'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', 'Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.')]))]), '0.1.1': CommentedMap([('package', CommentedMap([('name', 'svtools'), ('version', '0.1.1')])), ('source', CommentedMap([('fn', 'svtools-3731a7c6e19.tar.gz'), ('url', 'https://github.com/hall-lab/svtools/archive/3731a7c6e19.tar.gz'), ('md5', 'c6ba71cd3a7f02b0c58d1d13a48c02db')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['vcftobedpe -h'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtools'), ('summary', 'Tools for processing and analyzing structural variants.')]))]), '0.5.9': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mateidavid/fast5/archive/v<{ version }}.tar.gz'), ('md5', '5082495779140c74aa3cbfc6119ea338')])), ('requirements', CommentedMap([('build', None), ('run', ['hdf5'])])), ('test', CommentedMap([('requires', ['gcc']), ('files', ['test.c']), ('commands', ['g++ -std=c++11 -I ${PREFIX}/include -L ${PREFIX}/lib  test.c'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/fast5'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A C++ header-only library for reading Oxford Nanopore Fast5 files.')]))]), '0.7.3a': CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.7.3a')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.7.3a.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.3a.tar.bz2'), ('md5', '7ed9f5d750d26f5bf91cee1fa6f65df2')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['bwa 2>&1 | grep \"index sequences\"'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))]), '0.7.8': CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.7.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.7.8.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.8.tar.bz2'), ('md5', '51ca7a29907022080398b2b88207e33c')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))]), '1.3.2': CommentedMap([('package', CommentedMap([('name', 'carna'), ('version', '1.3.2')])), ('about', CommentedMap([('home', 'https://www.bioinf.uni-leipzig.de/~will/Software/CARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Constraint-based Alignment of RNA Ensembles')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'carna-1.3.2.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/Releases/carna-1.3.2.tar.gz'), ('sha256', '279ce4417c8cd637511d4d35efb83cb63194cee7608d89a75542f14d4c5ee48c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gecode 4.4.0', 'viennarna 2.2.8', 'locarna 1.8.11']), ('run', ['libgcc', 'gecode 4.4.0', 'locarna 1.8.11'])])), ('test', CommentedMap([('commands', ['carna --version'])]))]), '3.0': CommentedMap([('package', CommentedMap([('name', 'snippy'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'snippy_v3.0.tar.gz'), ('md5', '081ddf6ffad192136de2fc9d6635199d'), ('url', 'https://github.com/tseemann/snippy/archive/v3.0.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl', 'perl-app-cpanminus', 'perl-module-build', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff']), ('run', ['perl-threaded', 'perl-bioperl', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff', 'htslib'])])), ('test', CommentedMap([('commands', ['snippy --version'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/snippy'), ('license', 'GPL2'), ('summary', 'Rapid bacterial SNP calling and core genome alignments')]))]), '1.2.2-b': CommentedMap([('package', CommentedMap([('name', 'clark'), ('version', '1.2.2_b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'CLARKV1.2.2-b.tar.gz'), ('url', 'http://clark.cs.ucr.edu/Download/CLARKV1.2.2-b.tar.gz'), ('md5', '32d5643476452c7e70539663fe253ff2')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CLARK --help 2>&1 | grep k-spectrum > /dev/null'])])), ('about', CommentedMap([('home', 'http://clark.cs.ucr.edu/'), ('license', 'GPL3'), ('license_file', 'LICENSE_GNU_GPL.txt'), ('summary', 'Fast, accurate and versatile k-mer based classification system')]))]), '1.2.3': CommentedMap([('package', CommentedMap([('name', 'clark'), ('version', '1.2.3')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'CLARKV1.2.3.tar.gz'), ('url', 'http://clark.cs.ucr.edu/Download/CLARKV1.2.3.tar.gz'), ('md5', '32d5643476452c7e70539663fe253ff2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CLARK --help'])])), ('about', CommentedMap([('home', 'http://clark.cs.ucr.edu/'), ('license', 'GPL3'), ('license_file', 'LICENSE_GNU_GPL.txt'), ('summary', 'Fast, accurate and versatile k-mer based classification system')]))]), '1.1.0': CommentedMap([('package', CommentedMap([('name', 'shorah'), ('version', '1.1.0')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'shorah-1.1.0.tar.bz2'), ('url', 'https://github.com/cbg-ethz/shorah/releases/download/v1.1.0/shorah-1.1.0.tar.bz2'), ('sha256', 'e4fe6cfb854163cb0963a977b9f08faee68027298c247b7e305337f3ebc3234c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'gsl', 'zlib']), ('run', ['python', 'samtools', 'biopython', 'numpy', 'gsl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/shorah'), ('license', 'GPLv3'), ('summary', 'The Short Reads Assembly into Haplotypes (ShoRAH) program for inferring viral haplotypes from NGS data')])), ('test', CommentedMap([('commands', ['dec.py -h', 'shorah.py -h'])]))]), '0.7.2': CommentedMap([('package', CommentedMap([('name', 'htseq'), ('version', '0.7.2')])), ('source', CommentedMap([('fn', 'HTSeq-0.7.2.tar.gz'), ('url', 'https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz'), ('sha256', 'b9da6d6a9004e8e05a40ecf08f74f765ec0e0be0e835e82205a117aba2ce375b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'pysam >=0.9.0']), ('run', ['python', 'numpy', 'matplotlib', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['HTSeq._HTSeq_internal', 'HTSeq.StepVector', 'HTSeq._version', 'HTSeq.scripts.count', 'HTSeq.scripts.qa']), ('commands', ['htseq-count -h', 'htseq-qa -h'])])), ('about', CommentedMap([('home', 'http://www-huber.embl.de/users/anders/HTSeq/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A framework to process and analyze data from high-throughput sequencing (HTS) assays'), ('license_family', 'LGPL')]))]), '1.0.6': CommentedMap([('about', CommentedMap([('home', 'http://www.gnu.org/software/datamash'), ('license', 'GPL3'), ('summary', 'GNU Datamash is a command-line program which performs basic numeric,textual and statistical operations on input textual data files.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'datamash'), ('version', '1.0.6')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', [])])), ('source', CommentedMap([('fn', 'datamash-1.0.6.tar.gz'), ('sha256', '0154c25c45b5506b6d618ca8e18d0ef093dac47946ac0df464fb21e77b504118'), ('url', 'http://ftpmirror.gnu.org/datamash/datamash-1.0.6.tar.gz')])), ('test', CommentedMap([('commands', ['datamash --help &> /dev/null'])]))]), '1.5.8.2': CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/73/f6/9bf7814bb88b4fa7116d30dd7fb4304153d738b8eacf2551a2018e63321f/deepTools-<{ version }}.tar.gz'), ('md5', 'd07c09b9e643f3f32f6e83d150b793f8')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools', 'numpy >=1.6.2', 'scipy >=0.10.0', 'pysam >=0.7.7', 'bx-python >=0.5.0', 'gcc']), ('run', ['python >=2.7,<3', 'numpy >=1.6.2', 'scipy >=0.10.0', 'pysam >=0.7.7', 'bx-python >=0.5.0', 'libpng', 'ucsc-bedgraphtobigwig'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))]), '1.5.9.1': CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://files.pythonhosted.org/packages/0a/04/73a64b8a8748b21de16acc882448e475d3d25cc4872aa495373234ad88fb/deepTools-<{ version }}.tar.gz'), ('sha256', '18037085e579e62c07b8334427c889b709a7c903239da00e153bd6a8b0e165de')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools', 'numpy >=1.7.1', 'scipy >=0.12.0', 'pysam >=0.7.7', 'bx-python >=0.5.0', 'gcc']), ('run', ['python >=2.7,<3', 'numpy >=1.7.1', 'scipy >=0.12.0', 'pysam >=0.7.7', 'bx-python >=0.5.0', 'libpng', 'ucsc-bedgraphtobigwig'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))]), '2.3.5': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.5.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.5.tar.gz'), ('sha256', '26b62a00da21bc5597b580ab8fef4e624234ec446d7d3cb0ce22803a5d7074ca')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'python >3']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'custom'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.3.6': CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/75/cc/52774ca1df27e4b0dc503972f782fd01d8ee611d8ad89abfe7b36d7c633f/deepTools-<{ version }}.tar.gz'), ('md5', '25d21c21fdd54fea7fb008ed44afa40f')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0', 'gcc']), ('run', ['python', 'pybigwig >=0.2.3', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))]), '2.4.1': CommentedMap([('package', CommentedMap([('name', 'openbabel'), ('version', '2.4.1')])), ('source', CommentedMap([('fn', 'openbabel-2-4-1.zip'), ('url', 'https://github.com/openbabel/openbabel/archive/openbabel-2-4-1.zip'), ('sha1', '6fefb2355601b665bea5840fe8e6f95bd48023c9'), ('patches', ['fix_library_path_search.diff', 'fix_data_path.diff', 'fix_babel_test_paths.diff', 'fix_libinchi_cmakelists.diff', '372.patch'])])), ('build', CommentedMap([('skip', True), ('number', 1), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib', 'swig', 'eigen', 'bzip2', 'python', 'libxml2', 'cairo']), ('run', ['python', 'libgcc', 'zlib', 'libxml2', 'cairo', 'xorg-libxau', 'xorg-libxdmcp', 'xorg-libxext', 'xorg-libxrender'])])), ('test', CommentedMap([('imports', ['openbabel', 'pybel']), ('commands', ['obabel --help', \"obabel -:'c1ccccc1' --gen3d -omol2\", \"obabel -:'c1ccccc1' --gen3d -oinchi\"])])), ('about', CommentedMap([('home', 'http://www.openbabel.org/'), ('license', 'GPLv2'), ('summary', 'A chemical toolbox designed to speak the many languages of chemical data')]))]), '3.12.3': CommentedMap([('package', CommentedMap([('name', 'dendropy'), ('version', '3.12.3')])), ('source', CommentedMap([('fn', 'DendroPy-3.12.3.tar.gz'), ('url', 'https://pypi.python.org/packages/3d/c0/e0a20d1cf4b268757842a03f36452095fccb8eae5ebd541548e29c6ca1e3/DendroPy-3.12.3.tar.gz'), ('md5', '13de66112a18e4eb283f4d4b18af7a65')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'setuptools']), ('run', ['python >=2.7,<3'])])), ('test', CommentedMap([('imports', ['dendropy', 'dendropy.coalescent', 'dendropy.continuous', 'dendropy.dataio', 'dendropy.dataobject', 'dendropy.interop', 'dendropy.mathlib', 'dendropy.popgensim', 'dendropy.popgenstat', 'dendropy.reconcile', 'dendropy.scm', 'dendropy.seqmodel', 'dendropy.seqsim', 'dendropy.test', 'dendropy.treecalc', 'dendropy.treemanip', 'dendropy.treesim', 'dendropy.treesplit', 'dendropy.treesum', 'dendropy.utility'])])), ('about', CommentedMap([('home', 'http://packages.python.org/DendroPy/'), ('license', 'BSD License'), ('summary', 'A Python library for phylogenetics and phylogenetic computing: reading, writing, simulation, processing and manipulation of phylogenetic trees (phylogenies) and characters.')]))]), '0.7.12': CommentedMap([('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_0')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost >=1.53.0,<{CONDA_BOOST}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*'])])), ('test', CommentedMap([('commands', ['diamond -h 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.22': CommentedMap([('build', CommentedMap([('number', 2)])), ('package', CommentedMap([('name', 'diamond'), ('version', '0.8.22')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/diamond/archive/v0.8.22.tar.gz'), ('md5', '1c31a0d84b5e5a2656b745725e6b0d80')])), ('requirements', CommentedMap([('build', ['gcc', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.24': CommentedMap([('package', CommentedMap([('name', 'diamond'), ('version', '0.8.24')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/diamond/archive/v0.8.24.tar.gz'), ('md5', '7f294908edd97ed6f4adf9439f6825e5')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.26': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.27': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.28': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.29': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.30': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.31': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.8.36': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.9.10': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]), '0.10.0': CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.10.0')])), ('source', CommentedMap([('fn', 'pysam-0.10.0.tar.gz'), ('url', 'https://pypi.python.org/packages/87/8e/d2d8238558970df37c7aa01ddec63057a98042334e939b4c1c69cb9a2504/pysam-0.10.0.tar.gz'), ('md5', 'a1f3333ce60f8de542624cb07c792fc0'), ('patches', ['osx_rpath.patch [osx]'])])), ('build', CommentedMap([('number', 3), ('skip', False), ('binary_relocation', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib >=1.3,<1.4', 'samtools >=1.3,<1.4', 'bcftools >=1.3,<1.4', 'cython', 'python', 'setuptools', 'zlib', 'curl']), ('run', ['libgcc', 'htslib >=1.3,<1.4', 'samtools >=1.3,<1.4', 'bcftools >=1.3,<1.4', 'python', 'zlib', 'curl'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', 'Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.')]))]), '0.8.2': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/yhoogstrate/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy']), ('run', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['drdisco']), ('commands', ['LANG=C.UTF-8 dr-disco --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/dr-disco/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Dr. Disco: fusion gene and genomic breakpoint detection in random hexamer RNA-seq data')]))]), '0.9.0': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/yhoogstrate/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy']), ('run', ['python', 'click', 'htseq ==0.6.1.post1', 'htslib ==1.3', 'pysam ==0.10.0', 'fuma ==3.0.5', 'numpy', 'scipy'])])), ('test', CommentedMap([('imports', ['drdisco']), ('commands', ['LANG=C.UTF-8 dr-disco --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/dr-disco/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'Dr. Disco: fusion gene and genomic breakpoint detection in random hexamer RNA-seq data')]))]), '0.06': CommentedMap([('package', CommentedMap([('name', 'dudes'), ('version', '0.06')])), ('source', CommentedMap([('fn', 'dudes_v0_06.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/dudes/dudes_v0_06.tar.gz'), ('md5', 'dde2c8dbca04448cf21a0ed6b092b047')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'numpy', 'pandas', 'bsddb'])])), ('test', CommentedMap([('commands', ['DUDes.py -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/dudes/'), ('license', 'The MIT License (MIT)'), ('summary', 'DUDes is a taxonomic profiler for NGS metagenomic samples')]))]), '1.1.2': CommentedMap([('package', CommentedMap([('name', 'edlib'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/Martinsos/edlib/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libgcc', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['echo \">\" > test.fa && echo \"AC\" >> test.fa && edlib-aligner test.fa test.fa'])])), ('about', CommentedMap([('home', 'https://github.com/Martinsos/edlib/'), ('license', 'MIT'), ('summary', 'C/C++ library and program for sequence alignment using edit (Levenshtein) distance')]))]), '2.0.17': CommentedMap([('about', CommentedMap([('home', 'http://eigen.tuxfamily.org'), ('license', 'LGPL2'), ('summary', 'Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'eigen'), ('version', '2.0.17')])), ('requirements', CommentedMap([('build', ['cmake ==3.2.3']), ('run', ['cmake ==3.2.3'])])), ('source', CommentedMap([('fn', '2.0.17.tar.gz'), ('sha256', '705028db65db7c1e393cdf55018754bf0696e376603a48a33b781bc85406ff1e'), ('url', 'http://bitbucket.org/eigen/eigen/get/2.0.17.tar.gz')])), ('test', CommentedMap([('commands', ['ls -l $PREFIX/include/ &> /dev/null', 'ls -l $PREFIX/share/ &> /dev/null'])]))]), '3.1.3': CommentedMap([('about', CommentedMap([('home', 'http://eigen.tuxfamily.org'), ('license', 'MPL2'), ('summary', 'Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'eigen'), ('version', '3.1.3')])), ('requirements', CommentedMap([('build', ['cmake ==3.2.3']), ('run', ['cmake ==3.2.3'])])), ('source', CommentedMap([('fn', '3.1.3.tar.gz'), ('sha256', '2ea0ba3eb9ac7ec623d8be92be4ee651191532cb311e0c979d65da9bc6c5143a'), ('url', 'http://bitbucket.org/eigen/eigen/get/3.1.3.tar.gz')])), ('test', CommentedMap([('commands', ['ls -l $PREFIX/include/ &> /dev/null', 'ls -l $PREFIX/share/ &> /dev/null'])]))]), '3.2': CommentedMap([('about', CommentedMap([('home', 'http://eigen.tuxfamily.org'), ('license', 'MPL2'), ('summary', 'Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('package', CommentedMap([('name', 'eigen'), ('version', '3.2')])), ('requirements', CommentedMap([('build', ['cmake']), ('run', ['cmake'])])), ('source', CommentedMap([('fn', '3.2.6.tar.bz2'), ('sha256', '8a3352f9a5361fe90e451a7305fb1896fc7f771dc16cc0edd8e6b157f52c343e'), ('url', 'https://bitbucket.org/eigen/eigen/get/3.2.6.tar.bz2')])), ('test', CommentedMap([('commands', ['ls -l $PREFIX/include/ &> /dev/null', 'ls -l $PREFIX/share/ &> /dev/null'])]))]), '0.0.4': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/96/bc/fd02655f20f6e4771ef8b1e59bffe6873f57feac69f0f5cbbce4fffe490a/enasearch-0.0.4.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('entry_points', ['enasearch = enasearch.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'click', 'flake8', 'xmltodict', 'biopython', 'pytest', 'pytest-runner']), ('run', ['python', 'requests', 'click', 'flake8', 'xmltodict', 'biopython'])])), ('test', CommentedMap([('imports', ['enasearch', 'enasearch_data']), ('commands', ['enasearch --help'])])), ('about', CommentedMap([('home', 'https://github.com/bebatut/enasearch'), ('license', 'MIT License'), ('summary', \"A Python library for interacting with ENA's API\"), ('license_family', 'MIT')]))]), '0.0.5': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/ae/50/057369fdac779e6b5ec2426c7984213f766967cd728ed261cdf66cc8f10a/enasearch-0.0.5.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('entry_points', ['enasearch = enasearch.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'click', 'flake8', 'xmltodict', 'biopython', 'pytest', 'pytest-runner']), ('run', ['python', 'requests', 'click', 'flake8', 'xmltodict', 'biopython'])])), ('test', CommentedMap([('imports', ['enasearch', 'enasearch_data']), ('commands', ['enasearch --help'])])), ('about', CommentedMap([('home', 'https://github.com/bebatut/enasearch'), ('license', 'MIT License'), ('summary', \"A Python library for interacting with ENA's API\"), ('license_family', 'MIT')]))]), '0.0.6': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/a1/fd/dd5183aa4ed33cd051fdfbfb1d8c70b39c537de4174bc63253bf7229f3eb/enasearch-0.0.6.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('entry_points', ['enasearch = enasearch.__main__:main']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'click', 'flake8', 'xmltodict', 'biopython', 'pytest', 'pytest-runner']), ('run', ['python', 'requests', 'click', 'flake8', 'xmltodict', 'biopython'])])), ('test', CommentedMap([('imports', ['enasearch', 'enasearch_data']), ('commands', ['enasearch --help'])])), ('about', CommentedMap([('home', 'https://github.com/bebatut/enasearch'), ('license', 'MIT License'), ('summary', \"A Python library for interacting with ENA's API\"), ('license_family', 'MIT')]))]), '0.5.10a': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mateidavid/fast5/archive/0e1bfb1079151e14837373dd2f8df8e2850f7a43.tar.gz'), ('md5', '7727eaea6a2aff85e46838aeb4b6b91f')])), ('requirements', CommentedMap([('build', None), ('run', ['hdf5'])])), ('test', CommentedMap([('commands', ['test -e ${PREFIX}/include/fast5/fast5.hpp'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/fast5'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A C++ header-only library for reading Oxford Nanopore Fast5 files.')]))]), '0.5.6': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mateidavid/fast5/archive/v<{ version }}.tar.gz'), ('md5', '649a37d83b4199a7f406754bf3dd6d83')])), ('requirements', CommentedMap([('build', None), ('run', ['hdf5'])])), ('test', CommentedMap([('requires', ['gcc']), ('files', ['test.c']), ('commands', ['g++ -std=c++11 -I ${PREFIX}/include -L ${PREFIX}/lib  test.c'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/fast5'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A C++ header-only library for reading Oxford Nanopore Fast5 files.')]))]), '0.5.8': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/mateidavid/fast5/archive/v<{ version }}.tar.gz'), ('md5', '686fd27e76f0f569f2fb4d05a19e4b79')])), ('requirements', CommentedMap([('build', None), ('run', ['hdf5'])])), ('test', CommentedMap([('requires', ['gcc']), ('files', ['test.c']), ('commands', ['g++ -std=c++11 -I ${PREFIX}/include -L ${PREFIX}/lib  test.c'])])), ('about', CommentedMap([('home', 'https://github.com/mateidavid/fast5'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'A C++ header-only library for reading Oxford Nanopore Fast5 files.')]))]), '0.6.1': CommentedMap([('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/humann2'), ('license', 'MIT'), ('summary', 'HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data')])), ('package', CommentedMap([('name', 'humann2'), ('version', '0.6.1')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('source', CommentedMap([('fn', 'humann2_v0.6.1.tar.gz'), ('md5', 'fae7549ac333b973badf7e99805d50d1'), ('url', 'https://bitbucket.org/biobakery/humann2/downloads/humann2_v0.6.1.tar.gz')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'bowtie2 >=2.2.5', 'metaphlan2 >=2.2.0', 'diamond >=0.7.10,<0.9.0'])])), ('test', CommentedMap([('commands', ['humann2 --version', 'humann2_join_tables -h', 'humann2_merge_abundance_tables -h', 'humann2_reduce_table -h', 'humann2_regroup_table -h', 'humann2_rename_table -h', 'humann2_renorm_table -h', 'humann2_rna_dna_norm -h', 'humann2_split_table -h'])]))]), '0.10.1': CommentedMap([('package', CommentedMap([('name', 'pymummer'), ('version', '0.10.1')])), ('source', CommentedMap([('fn', 'pymummer-0.10.1.tar.gz'), ('url', 'https://pypi.python.org/packages/96/04/a67728a727a8214de494b06178bfaca025550156889953d581a141976ec0/pymummer-0.10.1.tar.gz'), ('md5', '1a16e2d27b58f6fdc47f648f977fafd0')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfastaq >=3.10.0', 'mummer']), ('run', ['python', 'pyfastaq >=3.10.0', 'mummer'])])), ('test', CommentedMap([('imports', ['pymummer']), ('requires', ['nose >=1.3'])])), ('about', CommentedMap([('home', 'https://github.com/sanger-pathogens/pymummer'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Wrapper for MUMmer')]))]), '0.11.3': CommentedMap([('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'), ('license', 'GPL'), ('summary', 'A quality control tool for high throughput sequence data.')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk', 'perl-threaded'])])), ('package', CommentedMap([('name', 'fastqc'), ('version', '0.11.3')])), ('source', CommentedMap([('fn', 'fastqc_v0.11.3.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.3.zip'), ('patches', ['hashbang.patch'])])), ('test', CommentedMap([('commands', [\"fastqc --version | grep '^FastQC v0\\\\.11\\\\.3\\\\b'\"])]))]), '2.1.8': CommentedMap([('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Tools for RNA secondary structure prediction and comparison')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'viennarna'), ('version', '2.1.8')])), ('requirements', CommentedMap([('build', ['perl ==5.18.1']), ('run', ['perl ==5.18.1'])])), ('source', CommentedMap([('fn', 'ViennaRNA-2.1.8.tar.gz'), ('sha256', '37a0a0b7c8a167bcdab1a85d88bcb1025ad8e00a4b00a147bf36d45dc8ca7560'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.1.8.tar.gz')])), ('test', CommentedMap([('commands', ['RNAcofold --version'])]))]), '20141125': CommentedMap([('about', CommentedMap([('home', 'http://fiji.sc/'), ('license', 'GNU General Public License'), ('summary', 'Fiji is an image processing package—a \"batteries-included\" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.')])), ('package', CommentedMap([('name', 'fiji'), ('version', 20141125)])), ('build', CommentedMap([('number', 5)])), ('source', CommentedMap([('fn', 'fiji-macosx-20141125.dmg'), ('url', 'http://downloads.imagej.net/fiji/Life-Line/fiji-macosx-20141125.dmg'), ('md5', 'fdb20fc6bfe838d03d065cbe568f908d')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6'])])), ('test', CommentedMap([('commands', [\"ImageJ --ij2 --headless --debug 2>&1 | grep 'Available RAM:'\", \"ImageJ --ij2 --headless --debug -DXms=512m -DXmx=1g 2>&1 | grep 'Available RAM:'\", \"bunwarpj 2>&1 | grep 'Usage: bUnwarpj_'\"])]))]), '20151222': CommentedMap([('about', CommentedMap([('home', 'http://fiji.sc'), ('license', 'GNU General Public License'), ('summary', 'Fiji is an image processing package—a \"batteries-included\" distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.')])), ('package', CommentedMap([('name', 'fiji'), ('version', 20151222)])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'fiji-macosx-20151222.dmg'), ('url', 'http://downloads.imagej.net/fiji/Life-Line/fiji-macosx-20151222.dmg'), ('md5', '8a0f2acb02b1106255d18b17e6f25c7c')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6'])])), ('test', CommentedMap([('requires', ['python']), ('files', ['hello.py']), ('commands', [\"bunwarpj 2>&1 | grep 'Usage'\", 'ImageJ --ij2 --headless --debug --jython hello.py 2>&1 | grep Bjoern', 'ImageJ --ij2 --headless --debug -DXms=512m -DXmx=1g --jython hello.py 2>&1 | grep Bjoern'])]))]), 'v2.3.4': CommentedMap([('package', CommentedMap([('name', 'flaimapper'), ('version', '2.3.4')])), ('source', CommentedMap([('fn', 'v2.3.4.tar.gz'), ('url', 'https://github.com/yhoogstrate/flaimapper/archive/v2.3.4.tar.gz'), ('sha256', '10eabc52839d1c6172a15b9c0f71cdbd6bb13987778519eb73b893a043cfb4f2')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'htseq', 'pysam']), ('run', ['python', 'numpy', 'pysam'])])), ('test', CommentedMap([('imports', ['flaimapper']), ('commands', ['flaimapper --help', 'flaimapper --version'])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/flaimapper/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'FlaiMapper: Detecting small ncRNA derived fragments in small RNA-Seq data')]))]), 'v2.4.0': CommentedMap([('package', CommentedMap([('name', 'flaimapper'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'v2.4.0.tar.gz'), ('url', 'https://github.com/yhoogstrate/flaimapper/archive/v2.4.0.tar.gz'), ('sha256', '3ac2f90894bf3687f27e9e35730c0893ce4e6cf689d940c3c1a55cc2f30cb2f4')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'htseq', 'pysam ==0.11.1']), ('run', ['python', 'numpy', 'pysam ==0.11.1'])])), ('test', CommentedMap([('imports', ['flaimapper']), ('commands', ['flaimapper --help', 'flaimapper --version', \"python -c 'import pysam; pysam.AlignmentFile'\"])])), ('about', CommentedMap([('home', 'https://github.com/yhoogstrate/flaimapper/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'FlaiMapper: Detecting small ncRNA derived fragments in small RNA-Seq data')]))]), '0.1.0': CommentedMap([('package', CommentedMap([('name', 'samsift'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'samsift-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/samsift/samsift-<{version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('requirements', CommentedMap([('build', ['python', 'pysam']), ('run', ['python', 'pysam'])])), ('test', CommentedMap([('commands', ['samsift -h'])])), ('about', CommentedMap([('home', 'https://github.com/karel-brinda/samsift'), ('license', 'MIT'), ('summary', 'Advanced filtering and tagging of SAM/BAM alignments using Python expressions.')]))]), '0.2.0': CommentedMap([('package', CommentedMap([('name', 'svtools'), ('version', '0.2.0')])), ('source', CommentedMap([('fn', 'svtools-0.2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/83/52/798f0e95586c15ceb66ddc4c09d6fca40666cca6e5f761c96868e30b92d7/svtools-0.2.0.tar.gz'), ('md5', '93dd90f37ee696f0c73ccd580657b792')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['svtools=svtools.cli:main']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools']), ('run', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools'])])), ('test', CommentedMap([('imports', ['svtools', 'svtools.vcf']), ('commands', ['svtools --help', 'create_coordinates --help'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtools'), ('license', 'MIT License'), ('summary', 'Tools for processing and analyzing structural variants')]))]), '0.5.3': CommentedMap([('about', CommentedMap([('home', 'https://www.github.com/molpopgen/fwdpp'), ('license', 'https://raw.githubusercontent.com/molpopgen/fwdpp/master/LICENSE'), ('summary', 'A C++ template library for forward-time population genetic simulation. '), ('doi', '10.1534/genetics.114.165019')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'libsequence', 'boost >=1.60.0'])])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'fwdpp'), ('version', '0.5.3')])), ('test', CommentedMap([('commands', ['fwdppConfig --version | grep 0.5.3 > /dev/null'])])), ('source', CommentedMap([('fn', '0.5.3.tar.gz'), ('url', 'http://github.com/molpopgen/fwdpp/archive/0.5.3.tar.gz'), ('md5', '09a00eb8fa1d54c953b5cb4e52a85a1c')]))]), '0.5.4': CommentedMap([('about', CommentedMap([('home', 'https://www.github.com/molpopgen/fwdpp'), ('license', 'https://raw.githubusercontent.com/molpopgen/fwdpp/master/LICENSE'), ('summary', 'A C++ template library for forward-time population genetic simulation. '), ('doi', '10.1534/genetics.114.165019')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'libsequence', 'boost >=1.60.0'])])), ('build', CommentedMap([('number', 2)])), ('package', CommentedMap([('name', 'fwdpp'), ('version', '0.5.4')])), ('test', CommentedMap([('commands', ['fwdppConfig --version | grep 0.5.4 > /dev/null'])])), ('source', CommentedMap([('fn', '0.5.4.tar.gz'), ('url', 'http://github.com/molpopgen/fwdpp/archive/0.5.4.tar.gz'), ('md5', '36efa5cf0ff3730b154a2c73ec8f390b')]))]), '16.10.4': CommentedMap([('package', CommentedMap([('name', 'galaxy-lib'), ('version', '16.10.4')])), ('source', CommentedMap([('fn', 'galaxy-lib-16.10.4.tar.gz'), ('url', 'https://pypi.python.org/packages/47/47/bcf1ad924f721fbd4ee855dfbdaaa5d3eaf888acaede46ce811881379808/galaxy-lib-16.10.4.tar.gz'), ('md5', 'ac51a8668668a8e95d7598dbde8eed5d')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('number', 0), ('entry_points', ['mulled-build=galaxy.tools.deps.mulled.mulled_build:main', 'mulled-build-channel=galaxy.tools.deps.mulled.mulled_build_channel:main', 'mulled-build-tool=galaxy.tools.deps.mulled.mulled_build_tool:main', 'mulled-build-files=galaxy.tools.deps.mulled.mulled_build_files:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyyaml', 'six >=1.9.0', 'markupsafe']), ('run', ['python', 'six >=1.9.0', 'pyyaml', 'markupsafe'])])), ('test', CommentedMap([('imports', ['galaxy', 'galaxy.exceptions', 'galaxy.jobs', 'galaxy.jobs.metrics', 'galaxy.jobs.metrics.collectl', 'galaxy.jobs.metrics.instrumenters', 'galaxy.objectstore', 'galaxy.tools', 'galaxy.tools.cwl', 'galaxy.tools.deps', 'galaxy.tools.deps.container_resolvers', 'galaxy.tools.deps.mulled', 'galaxy.tools.deps.resolvers', 'galaxy.tools.linters', 'galaxy.tools.parser', 'galaxy.tools.toolbox', 'galaxy.tools.toolbox.filters', 'galaxy.tools.toolbox.lineages', 'galaxy.tools.verify', 'galaxy.tools.verify.asserts', 'galaxy.util']), ('commands', ['mulled-build --help', 'mulled-build-channel --help', 'mulled-build-tool --help', 'mulled-build-files --help'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/galaxy-lib'), ('license', 'Academic Free License (AFL)'), ('summary', 'Subset of Galaxy (http://galaxyproject.org/) core code base designed to be used a library.'), ('license_family', 'Apache')]))]), '16.10.8': CommentedMap([('package', CommentedMap([('name', 'galaxy-lib'), ('version', '16.10.8')])), ('source', CommentedMap([('fn', 'galaxy-lib-16.10.8.tar.gz'), ('url', 'https://pypi.python.org/packages/dc/b2/a9d84f67e03362196d7c590dfa4afff9e4a5d8bf32b479c7eecb615c55d1/galaxy-lib-16.10.8.tar.gz'), ('md5', '8f7117c67126f0d3bbc9190f8a69f6d9')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('number', 0), ('entry_points', ['mulled-build=galaxy.tools.deps.mulled.mulled_build:main', 'mulled-build-channel=galaxy.tools.deps.mulled.mulled_build_channel:main', 'mulled-build-tool=galaxy.tools.deps.mulled.mulled_build_tool:main', 'mulled-build-files=galaxy.tools.deps.mulled.mulled_build_files:main']), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyyaml', 'six >=1.9.0', 'markupsafe']), ('run', ['python', 'six >=1.9.0', 'pyyaml', 'markupsafe'])])), ('test', CommentedMap([('imports', ['galaxy', 'galaxy.exceptions', 'galaxy.jobs', 'galaxy.jobs.metrics', 'galaxy.jobs.metrics.collectl', 'galaxy.jobs.metrics.instrumenters', 'galaxy.objectstore', 'galaxy.tools', 'galaxy.tools.cwl', 'galaxy.tools.deps', 'galaxy.tools.deps.container_resolvers', 'galaxy.tools.deps.mulled', 'galaxy.tools.deps.resolvers', 'galaxy.tools.linters', 'galaxy.tools.parser', 'galaxy.tools.toolbox', 'galaxy.tools.toolbox.filters', 'galaxy.tools.toolbox.lineages', 'galaxy.tools.verify', 'galaxy.tools.verify.asserts', 'galaxy.util']), ('commands', ['mulled-build --help', 'mulled-build-channel --help', 'mulled-build-tool --help', 'mulled-build-files --help'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/galaxy-lib'), ('license', 'Academic Free License (AFL)'), ('summary', 'Subset of Galaxy (http://galaxyproject.org/) core code base designed to be used a library.'), ('license_family', 'Apache')]))]), '16.4.0': CommentedMap([('package', CommentedMap([('name', 'galaxy-lib'), ('version', '16.4.0')])), ('source', CommentedMap([('fn', 'galaxy-lib-16.4.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/g/galaxy-lib/galaxy-lib-16.4.0.tar.gz'), ('md5', '3576ebbb7918f24296dd1e85a23b6ae7')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pyyaml', 'six >=1.9.0', 'markupsafe']), ('run', ['python', 'six >=1.9.0', 'pyyaml', 'markupsafe'])])), ('test', CommentedMap([('imports', ['galaxy', 'galaxy.exceptions', 'galaxy.jobs', 'galaxy.jobs.metrics', 'galaxy.jobs.metrics.collectl', 'galaxy.jobs.metrics.instrumenters', 'galaxy.objectstore', 'galaxy.tools', 'galaxy.tools.cwl', 'galaxy.tools.deps', 'galaxy.tools.deps.resolvers', 'galaxy.tools.linters', 'galaxy.tools.parser', 'galaxy.tools.toolbox', 'galaxy.tools.toolbox.filters', 'galaxy.tools.toolbox.lineages', 'galaxy.util'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/galaxy-lib'), ('license', 'Academic Free License (AFL)'), ('summary', 'Subset of Galaxy (http://galaxyproject.org/) core code base designed to be used a library.')]))]), '16.7.10': CommentedMap([('package', CommentedMap([('name', 'galaxy-lib'), ('version', '16.7.10')])), ('source', CommentedMap([('fn', 'galaxy-lib-16.7.10.tar.gz'), ('url', 'https://pypi.python.org/packages/5d/36/68594f5b76323133c5849834cf2d781a73b7166296b1456cd52aaa26e2ac/galaxy-lib-16.7.10.tar.gz'), ('md5', '048cdb1ba001fba302f9d0e4e835bd20')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'pyyaml', 'six >=1.9.0', 'markupsafe']), ('run', ['python', 'six >=1.9.0', 'pyyaml', 'markupsafe'])])), ('test', CommentedMap([('imports', ['galaxy', 'galaxy.exceptions', 'galaxy.jobs', 'galaxy.jobs.metrics', 'galaxy.jobs.metrics.collectl', 'galaxy.jobs.metrics.instrumenters', 'galaxy.objectstore', 'galaxy.tools', 'galaxy.tools.cwl', 'galaxy.tools.deps', 'galaxy.tools.deps.resolvers', 'galaxy.tools.linters', 'galaxy.tools.parser', 'galaxy.tools.toolbox', 'galaxy.tools.toolbox.filters', 'galaxy.tools.toolbox.lineages', 'galaxy.tools.verify', 'galaxy.tools.verify.asserts', 'galaxy.util'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/galaxy-lib'), ('license', 'Academic Free License (AFL)'), ('summary', 'Subset of Galaxy (http://galaxyproject.org/) core code base designed to be used a library.'), ('license_family', 'Apache')]))]), '2.0': CommentedMap([('package', CommentedMap([('name', 'sortmerna'), ('version', 2.0)])), ('source', CommentedMap([('fn', '2.0.tar.gz'), ('url', 'https://github.com/biocore/sortmerna/archive/2.0.tar.gz'), ('md5', 'f233f7734105c75c14bd4c2fa4e2aefe')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['sortmerna --version'])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/RNA/sortmerna'), ('license', 'LGPL'), ('summary', 'SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.')]))]), '3.5': CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/gatk/'), ('license', 'https://www.broadinstitute.org/gatk/about/#licensing'), ('summary', 'The full Genome Analysis Toolkit (GATK) framework, license restricted.')])), ('package', CommentedMap([('name', 'gatk'), ('version', '3.5')])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8,<9', 'bzip2'])])), ('extra', CommentedMap([('notes', 'Due to license restrictions, this recipe cannot distribute and install GATK directly. To fully install GATK, you must download a licensed copy of GATK from the Broad Institute (https://www.broadinstitute.org/gatk/download/), install this package, and call \"gatk-register /path/to/GenomeAnalysisTK[-$PKG_VERSION.tar.bz2|.jar]\", which will copy GATK into your conda environment.')]))]), '4.4.0': CommentedMap([('package', CommentedMap([('name', 'gecode'), ('version', '4.4.0')])), ('about', CommentedMap([('home', 'http://www.gecode.org/'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Generic constraint development environment')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'gecode-4.4.0.tar.gz'), ('url', 'http://www.gecode.org/download/gecode-4.4.0.tar.gz'), ('sha256', 'b45783cc8d0d5dbbd3385a263a2199e6ad7f9a286e92607de81aa0c1105769cb')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fzn-gecode --help >& /dev/null', 'magic-sequence --help >& /dev/null'])]))]), '5.0.0': CommentedMap([('package', CommentedMap([('name', 'gecode'), ('version', '5.0.0')])), ('about', CommentedMap([('home', 'http://www.gecode.org/'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Generic constraint development environment')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gecode-5.0.0.tar.gz'), ('url', 'http://www.gecode.org/download/gecode-5.0.0.tar.gz'), ('sha256', 'f4ff2fa115fed8c09a049b2d8520363b1f9b1a39d80461f597e29dab2ba9e77b')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'mpfr', 'gmp <{CONDA_GMP}}*']), ('run', ['libgcc', 'mpfr', 'gmp <{CONDA_GMP}}*'])])), ('test', CommentedMap([('commands', ['fzn-gecode --help >& /dev/null', 'magic-sequence --help >& /dev/null'])]))]), '6.8.3': CommentedMap([('package', CommentedMap([('name', 'ghc'), ('version', '6.8.3')])), ('source', CommentedMap([('fn', 'ghc-6.8.3-x86_64-unknown-linux.tar.bz2'), ('url', 'https://www.haskell.org/ghc/dist/6.8.3/ghc-6.8.3-x86_64-unknown-linux.tar.bz2'), ('sha1', 'e8869f7df7465f1db913ce070e95fb22cc9309e9')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gmp']), ('run', ['gmp'])])), ('test', CommentedMap([('commands', ['ghc --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.haskell.org/ghc/'), ('license', 'BSD'), ('summary', 'GHC is a state-of-the-art, open source, compiler and interactive environment for the functional language Haskell.')]))]), '9.07': CommentedMap([('package', CommentedMap([('name', 'ghostscript'), ('version', 9.07)])), ('source', CommentedMap([('fn', 'ghostscript-9.07.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/ghostscript/GPL%20Ghostscript/9.07/ghostscript-9.07.tar.gz'), ('sha256', '44800d004c53f13192d1b5db413119198ddfc8a11c4d2a030aac2f2fda822ebf')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gs --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://ghostscript.com/'), ('summary', 'An interpreter for the PostScript language and for PDF.'), ('license', 'Affero GPL'), ('license_file', 'LICENSE')]))]), '9.10': CommentedMap([('package', CommentedMap([('name', 'ghostscript'), ('version', 9.1)])), ('source', CommentedMap([('fn', 'ghostscript-9.10.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/ghostscript/GPL%20Ghostscript/9.10/ghostscript-9.10.tar.gz'), ('sha256', '913fc974433238ffd4e0549ce11ba2a3360d1d159cf5c3b988d72a77acb74d04')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['gs --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://ghostscript.com/'), ('summary', 'An interpreter for the PostScript language and for PDF.'), ('license', 'Affero GPL'), ('license_file', 'LICENSE')]))]), '2014.12.23': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2014.12.23')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2014-12-23.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2014-12-23.tar.gz'), ('sha1', '70d94b67283df3a689c328ef31e5aa5e3bb58c1e')])), ('requirements', CommentedMap([('build', ['perl', 'zlib']), ('run', ['perl', 'zlib'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 3), ('skip', True), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap', 'bin/gmapl', 'bin/gsnap', 'bin/gsnapl', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl', 'bin/sam_sort']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version && gmap --check', 'gsnap --version && gsnap --check'])]))]), '2015.09.10': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2015.09.10')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2015-09-10.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2015-09-10.tar.gz'), ('sha1', 'c175c0d45ee7d79036fc08e357a66ab327dcb14f')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 1), ('skip', True), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap', 'bin/gmapl', 'bin/gsnap', 'bin/gsnapl', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl', 'bin/sam_sort']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]), '2015.12.31': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2015.12.31')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2015-12-31.v10.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2015-12-31.v10.tar.gz'), ('sha1', '815b155601cba941f54a99682e2629867a3fe757')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'bzip2']), ('run', ['libgcc', 'perl', 'zlib', 'bzip2'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 0), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.sse42', 'bin/gmap.nosimd', 'bin/gmapl.sse42', 'bin/gmapl.nosimd', 'bin/gsnap.sse42', 'bin/gsnap.nosimd', 'bin/gsnapl.sse42', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]), '2016.09.23': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2016.09.23')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2016-09-23.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2016-09-23.tar.gz'), ('sha1', '9d56ebcf47da5b4c1326bbeed37e37971657bbf7')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 1), ('skip', True), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.avx2', 'bin/gmap.nosimd', 'bin/gmapl.avx2', 'bin/gmapl.nosimd', 'bin/gsnap.avx2', 'bin/gsnap.nosimd', 'bin/gsnapl.avx2', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]), '2017.02.15': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2017.02.15')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2017-02-15.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2017-02-15.tar.gz'), ('sha1', '89a4814b0aa4bec9c5db4e1161432f8553d65c00')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 2), ('skip', True), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.avx2', 'bin/gmap.nosimd', 'bin/gmapl.avx2', 'bin/gmapl.nosimd', 'bin/gsnap.avx2', 'bin/gsnap.nosimd', 'bin/gsnapl.avx2', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]), '2017.05.08': CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2017.05.08')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2017-05-08.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2017-05-08.tar.gz'), ('sha1', '592797008a0251175e95ce0653bdfa764ee99617')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'bzip2']), ('run', ['libgcc', 'perl', 'zlib', 'bzip2'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 0), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.sse42', 'bin/gmap.nosimd', 'bin/gmapl.sse42', 'bin/gmapl.nosimd', 'bin/gsnap.sse42', 'bin/gsnap.nosimd', 'bin/gsnapl.sse42', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]), '0.50': CommentedMap([('package', CommentedMap([('name', 'gocr'), ('version', 0.5)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gocr-0.50pre-patched.tgz'), ('url', 'http://downloads.sourceforge.net/project/osra/gocr-patched/gocr-0.50pre-patched.tgz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', ['gocr --help'])])), ('about', CommentedMap([('home', 'http://jocr.sourceforge.net/'), ('license', 'GPL'), ('summary', 'GOCR is an OCR (Optical Character Recognition) program')]))]), '0.1.3': CommentedMap([('package', CommentedMap([('name', 'qiime-default-reference'), ('version', '0.1.3')])), ('source', CommentedMap([('fn', 'qiime-default-reference-0.1.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime-default-reference/qiime-default-reference-0.1.3.tar.gz'), ('md5', '4a5ccf5930690478de1eb881e8030bff')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['qiime_default_reference'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'CC BY-SA 3.0'), ('summary', 'Default reference data files for use with QIIME.')]))]), '1.3.20': CommentedMap([('package', CommentedMap([('name', 'graphicsmagick'), ('version', '1.3.23')])), ('build', CommentedMap([('number', 4)])), ('source', CommentedMap([('fn', 'GraphicsMagick-1.3.23.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/graphicsmagick/graphicsmagick/1.3.23/GraphicsMagick-1.3.23.tar.bz2'), ('sha256', '6e14a9e9e42ec074239b2de4db37ebebb8268b0361332d5bc86d7c3fbfe5aabf')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'ghostscript', 'libpng', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau']), ('run', ['ghostscript', 'libpng', 'libgcc', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau', 'xorg-libx11'])])), ('test', CommentedMap([('commands', ['gm -help | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'http://www.graphicsmagick.org/'), ('license', 'MIT'), ('summary', 'GraphicsMagick is the swiss army knife of image processing')]))]), '1.3.26': CommentedMap([('package', CommentedMap([('name', 'graphicsmagick'), ('version', '1.3.26')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'GraphicsMagick-1.3.26.tar.gz'), ('url', 'https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.26/GraphicsMagick-1.3.26.tar.gz'), ('sha256', 'ab06d07cda5181f7829714c9f5641c0a93a0c3af2dacd747ed48d534e1ed0b3a')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'ghostscript', 'libpng', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau']), ('run', ['ghostscript', 'libpng', 'libgcc', 'gnuplot', 'xorg-libsm', 'libwebp', 'freetype', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libxau', 'xorg-libx11'])])), ('test', CommentedMap([('commands', ['gm -help | grep \"Usage\"'])])), ('about', CommentedMap([('home', 'http://www.graphicsmagick.org/'), ('license', 'MIT'), ('summary', 'GraphicsMagick is the swiss army knife of image processing')]))]), '2.3.2': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.2')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.2.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.2.tar.gz'), ('sha256', '6d4f8b4375d0bc68655715303026a7c16c26967d7713ac83b3aca9478374ec6e')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'python >3']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'custom'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '0.6.1.post1': CommentedMap([('package', CommentedMap([('name', 'htseq'), ('version', '0.6.1.post1')])), ('source', CommentedMap([('fn', 'HTSeq-0.6.1p1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/H/HTSeq/HTSeq-0.6.1p1.tar.gz'), ('md5', 'c44d7b256281a8a53b6fe5beaeddd31c')])), ('build', CommentedMap([('preserve_egg_dir', True), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy', 'matplotlib', 'pysam'])])), ('test', CommentedMap([('imports', ['HTSeq._HTSeq_internal', 'HTSeq.StepVector', 'HTSeq._version', 'HTSeq.scripts.qa', 'HTSeq.scripts.count']), ('commands', ['htseq-count -h', 'htseq-qa -h'])])), ('about', CommentedMap([('home', 'http://www-huber.embl.de/users/anders/HTSeq/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A framework to process and analyze data from high-throughput sequencing (HTS) assays')]))]), '0.11.1': CommentedMap([('package', CommentedMap([('name', 'humann2'), ('version', '0.11.1')])), ('build', CommentedMap([('number', 1), ('skip', True), ('entry_points', ['humann2 = humann2.humann2:main', 'humann2_databases = humann2.tools.humann2_databases:main', 'humann2_config = humann2.tools.humann2_config:main', 'humann2_join_tables = humann2.tools.join_tables:main', 'humann2_split_table = humann2.tools.split_table:main', 'humann2_rename_table = humann2.tools.rename_table:main', 'humann2_renorm_table = humann2.tools.renorm_table:main', 'humann2_regroup_table = humann2.tools.regroup_table:main', 'humann2_infer_taxonomy = humann2.tools.infer_taxonomy:main', 'humann2_humann1_kegg = humann2.tools.humann2_humann1_kegg:main', 'humann2_rna_dna_norm = humann2.tools.rna_dna_norm:main', 'humann2_strain_profiler = humann2.tools.strain_profiler:main', 'humann2_reduce_table = humann2.tools.reduce_table:main', 'humann2_unpack_pathways = humann2.tools.merge_abundance:main', 'humann2_test = humann2.tests.humann2_test:main', 'humann2_build_custom_database = humann2.tools.build_custom_database:main', 'humann2_genefamilies_genus_level = humann2.tools.genefamilies_genus_level:main', 'humann2_split_stratified_table = humann2.tools.split_stratified_table:main', 'humann2_associate = humann2.tools.humann2_associate:main', 'humann2_barplot = humann2.tools.humann2_barplot:main', 'humann2_benchmark = humann2.tools.humann2_benchmark:main'])])), ('source', CommentedMap([('fn', 'humann2-0.11.1.tar.gz'), ('url', 'https://pypi.python.org/packages/43/07/ec41577c3c1f9b578875ade8ed549d14fc2944c13cb7504579d542b62a69/humann2-0.11.1.tar.gz'), ('md5', 'b5c5f2823ea535cab27f54ea72e2ca82')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'bowtie2 >=2.2.5', 'metaphlan2 >=2.2.0', 'diamond >=0.7.10,<0.9.0', 'samtools', 'biom-format', 'matplotlib', 'scipy', 'numpy'])])), ('test', CommentedMap([('imports', ['humann2', 'humann2.maintenance', 'humann2.quantify', 'humann2.search', 'humann2.tests', 'humann2.tools']), ('commands', ['humann2 --help', 'humann2_databases --help', 'humann2_config --help', 'humann2_join_tables --help', 'humann2_split_table --help', 'humann2_rename_table --help', 'humann2_renorm_table --help', 'humann2_regroup_table --help', 'humann2_infer_taxonomy --help 2>&1 | grep \"HUMAnN2\"', 'humann2_humann1_kegg --help', 'humann2_rna_dna_norm --help', 'humann2_strain_profiler --help', 'humann2_reduce_table --help', 'humann2_unpack_pathways --help', 'humann2_test --help', 'humann2_build_custom_database --help', 'humann2_genefamilies_genus_level --help', 'humann2_split_stratified_table --help', 'humann2_associate --help', 'humann2_barplot --help', 'humann2_benchmark --help'])])), ('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/humann2'), ('license', 'MIT'), ('summary', 'HUMAnN2: The HMP Unified Metabolic Analysis Network 2'), ('license_family', 'MIT')]))]), '0.9.9': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'https://bitbucket.org/biobakery/humann2/get/<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'bowtie2 >=2.2.5', 'metaphlan2 >=2.2.0', 'diamond >=0.7.10,<0.9.0', 'rapsearch >=2.21', 'samtools', 'biom-format'])])), ('test', CommentedMap([('commands', ['humann2 --version', 'humann2_join_tables -h', 'humann2_reduce_table -h', 'humann2_regroup_table -h', 'humann2_rename_table -h', 'humann2_renorm_table -h', 'humann2_rna_dna_norm -h', 'humann2_split_table -h'])])), ('about', CommentedMap([('home', 'http://huttenhower.sph.harvard.edu/humann2'), ('license', 'MIT'), ('summary', 'HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data')]))]), '0.3.0': CommentedMap([('package', CommentedMap([('name', 'svtools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'svtools-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/svtools/svtools-<{ version }}.tar.gz'), ('md5', '0524f0bac826a51e46a832444460a19e')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['svtools=svtools.cli:main']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools']), ('run', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools'])])), ('test', CommentedMap([('imports', ['svtools', 'svtools.vcf']), ('commands', ['svtools --help', 'create_coordinates --help'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtools'), ('license', 'MIT License'), ('summary', 'Tools for processing and analyzing structural variants')]))]), '0.3.2': CommentedMap([('package', CommentedMap([('name', 'pyqi'), ('version', '0.3.2')])), ('source', CommentedMap([('fn', 'pyqi-0.3.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyqi/pyqi-0.3.2.tar.gz'), ('md5', '9507c06eeb22a816d963c860ad8e92ae')])), ('build', CommentedMap([('number', 0), ('skip', 'True    [not py27]')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools'])])), ('test', CommentedMap([('imports', ['pyqi'])])), ('about', CommentedMap([('home', 'http://bipy.github.io/pyqi'), ('license', 'BSD License'), ('summary', 'pyqi: expose your interface')]))]), '1.0.2': CommentedMap([('source', CommentedMap([('fn', 'strip-it-1.0.2.tar.gz'), ('url', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/_downloads/strip-it-1.0.2.tar.gz'), ('md5', '963da0b005774a7a2eba4b4c384dca2a')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'strip_it'), ('version', '1.0.2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openbabel ==2.4.1', 'cmake']), ('run', ['openbabel ==2.4.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['strip-it --version'])])), ('about', CommentedMap([('home', 'http://silicos-it.be.s3-website-eu-west-1.amazonaws.com/software/strip-it/1.0.2/strip-it.html'), ('license', 'LGPL'), ('summary', 'Strip-it is a program that extracts predefined scaffolds from organic small molecules.')]))]), '1.5.3': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Cibiv/IQ-TREE/archive/v<{ version }}.tar.gz'), ('md5', 'ca5e7f5e4cd0723da59c67744415b1cc')])), ('requirements', CommentedMap([('build', ['gcc', 'cmake', 'eigen', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['iqtree-omp', 'iqtree'])])), ('about', CommentedMap([('home', 'http://www.iqtree.org/'), ('license', 'GPL-2.0'), ('summary', 'Efficient phylogenomic software by maximum likelihood')]))]), '4.6.1': CommentedMap([('package', CommentedMap([('name', 'itk'), ('version', '<{ ITK_VERSION }}')])), ('source', CommentedMap([('fn', 'InsightToolkit-<{ ITK_VERSION }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/itk/itk/<{ ITK_VERSION_SHORT }}/InsightToolkit-<{ ITK_VERSION }}.tar.gz'), ('md5', '2c84eae50ab2452cdad32aaadced3c37')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'libtiff', 'libpng', 'jpeg', 'zlib']), ('run', ['python', 'libgcc', 'zlib', 'libtiff', 'libpng', 'jpeg'])])), ('test', CommentedMap([('commands', ['test -f $PREFIX/lib/libITKCommon-<{ ITK_VERSION_SHORT }}.so'])])), ('about', CommentedMap([('home', 'http://www.itk.org/'), ('license', 'Apache 2.0'), ('summary', 'ITK is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis.')]))]), '1.0.7.2': CommentedMap([('package', CommentedMap([('name', 'jamm'), ('version', '1.0.7.2')])), ('source', CommentedMap([('fn', 'v1.0.7.2.zip'), ('url', 'https://github.com/mahmoudibrahim/JAMM/archive/v1.0.7.2.zip'), ('md5', '0902ea81e0559a8804c49dd7b1f02efd'), ('patches', ['setpath.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('run', ['r-base', 'perl-threaded', 'r-signal', 'r-mclust >=4'])])), ('test', CommentedMap([('commands', ['JAMM.sh | grep JAMM > /dev/null', 'SignalGenerator.sh | grep JAMM > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/mahmoudibrahim/JAMM'), ('license', 'GPL'), ('license_file', 'LICENSE'), ('summary', 'JAMM is a peak finder for NGS datasets (ChIP-Seq, ATAC-Seq, DNase-Seq..etc.) that can integrate replicates and assign peak boundaries accurately.')]))]), '6.0.77': CommentedMap([('package', CommentedMap([('name', 'java-jdk'), ('version', '6.0.77')])), ('source', CommentedMap([('fn', 'zulu6.11.0.2-jdk6.0.77-linux_x64.tar.gz'), ('url', 'http://cdn.azul.com/zulu/bin/zulu6.11.0.2-jdk6.0.77-linux_x64.tar.gz'), ('md5', 'a2d7f9fcbfbdd6c5a9e73a1b3350f041')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('test', CommentedMap([('commands', ['java -version > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'http://www.azulsystems.com/products/zulu'), ('license', 'GPL'), ('summary', 'The Zulu OpenJDK build.')]))]), '7.0.91': CommentedMap([('package', CommentedMap([('name', 'java-jdk'), ('version', '7.0.91')])), ('build', CommentedMap([('number', 1), ('binary_relocation', False)])), ('test', CommentedMap([('commands', ['java -version'])])), ('about', CommentedMap([('home', 'http://www.azulsystems.com/products/zulu'), ('license', 'GPL'), ('summary', 'The Zulu OpenJDK build.')]))]), '1.1.11': CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'jellyfish'), ('version', '1.1.11')])), ('source', CommentedMap([('fn', 'v1.1.11.zip'), ('url', 'https://github.com/gmarcais/Jellyfish/archive/43fc99e4d44d11f115dc6741ff705cf7e113f251.zip'), ('md5', '618afc01dd54305689f4deab656e482e')])), ('requirements', CommentedMap([('build', ['yaggo >=1.5.8', 'gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['jellyfish --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.genome.umd.edu/jellyfish.html'), ('license', 'GPLv3'), ('summary', 'Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence')]))]), '2.2.3': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'BioX-Workflow-Command-<{ version }}.tar.gz'), ('url', 'https://github.com/biosails/BioX-Workflow-Command/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-test-class-moose', 'perl-file-details', 'perl-capture-tiny', 'perl-text-diff', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-datetime-format-strptime', 'perl-config-any', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-module-build-tiny', 'perl-params-validate', 'perl-file-slurp', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-test2', 'perl-test-class-moose', 'perl-clone', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-list-someutils']), ('run', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-test2', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-file-details', 'perl-moose', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-moosex-getopt', 'perl-datetime-format-strptime', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-params-validate', 'perl-config-any', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-clone', 'perl-json', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-list-someutils'])])), ('test', CommentedMap([('commands', ['biox -h', 'biox run -h', 'biox stats -h', 'biox new -h', 'biox add -h'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/BioX-Workflow-Command'), ('license', 'perl_5'), ('summary', 'Opinionated Bioinformatics Genomics Workflow Creator')]))]), '3.6.0': CommentedMap([('package', CommentedMap([('name', 'jemalloc'), ('version', '3.6.0')])), ('build', CommentedMap([('number', 1), ('detect_binary_files_with_prefix', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('requires', ['gcc']), ('files', ['test.c']), ('commands', ['test -e ${PREFIX}/lib/libjemalloc.dylib', 'test -e ${PREFIX}/lib/libjemalloc.so', 'gcc -I${PREFIX}/include/ test.c -o test.o'])])), ('source', CommentedMap([('fn', 'jemalloc-3.6.0.tar.bz2'), ('url', 'https://github.com/jemalloc/jemalloc/releases/download/3.6.0/jemalloc-3.6.0.tar.bz2'), ('sha256', 'e16c2159dd3c81ca2dc3b5c9ef0d43e1f2f45b04548f42db12e7c12d7bdf84fe')])), ('about', CommentedMap([('home', 'https://github.com/jemalloc/jemalloc'), ('summary', 'Efficient multithreaded memory allocator'), ('license', 'BSD')]))]), '1.4.5': CommentedMap([('package', CommentedMap([('name', 'kaiju'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('url', 'https://github.com/bioinformatics-centre/kaiju/archive/c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('md5', 'dea77fa5856cb9261f523536eb917e68')])), ('build', CommentedMap([('number', 0), ('no_link', ['bin/makeDB.sh'])])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'gnu-wget >=1.16'])])), ('test', CommentedMap([('commands', [\"kaiju 2>&1 | head -n 1 | grep -q '^Error:'; echo $?\"])])), ('about', CommentedMap([('home', 'http://kaiju.binf.ku.dk/'), ('license', 'GNU GPL v3'), ('summary', 'Fast and sensitive taxonomic classification for metagenomics'), ('license_file', 'LICENSE')]))]), '2.0.8': CommentedMap([('about', CommentedMap([('home', 'https://github.com/TGAC/KAT/'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts')])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'kat'), ('version', '2.0.8')])), ('requirements', CommentedMap([('build', ['gcc', 'libgcc', 'autoconf', 'automake', 'libtool', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'perl']), ('run', ['boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish'])])), ('source', CommentedMap([('fn', 'Release-2.0.8.tar.gz'), ('url', 'https://github.com/TGAC/KAT/archive/Release-2.0.8.tar.gz'), ('sha1', '76f8a5fba81837dd20d00eaefb3709da476c1ca6')])), ('test', CommentedMap([('commands', ['kat --version'])]))]), '2.3.1': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.1')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.1.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.1.tar.gz'), ('sha256', 'fa7ae1571b6464682be2eff90198e41eb4f54c2cbf135950ea2e246d7cb4bf9d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded', 'python >3']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'custom'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.3.4': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Release-<{version}}.tar.gz'), ('url', 'https://github.com/TGAC/KAT/archive/Release-<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://github.com/TGAC/KAT'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['llvm', 'libgcc', 'python', 'zlib', 'autoconf', 'automake', 'libtool', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'perl', 'matplotlib', 'scipy', 'numpy']), ('run', ['python', 'libgcc', 'zlib', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'matplotlib', 'scipy', 'numpy'])])), ('test', CommentedMap([('commands', ['kat --version'])]))]), '0.10.5beta': CommentedMap([('package', CommentedMap([('name', 'kraken'), ('version', '0.10.5beta')])), ('source', CommentedMap([('fn', 'kraken-0.10.5-beta.tar.gz'), ('url', 'https://github.com/DerrickWood/kraken/archive/v0.10.5-beta.tar.gz'), ('md5', '0231a7bfc067f564ad28fa91e9f71606')])), ('build', CommentedMap([('skip', True), ('has_prefix_files', ['libexec/kraken', 'libexec/kraken-build', 'libexec/kraken-filter', 'libexec/kraken-mpa-report', 'libexec/kraken-report', 'libexec/kraken-translate'])])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['kraken --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/kraken/'), ('license', 'GPLv3'), ('summary', 'Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.')]))]), '2.5': CommentedMap([('about', CommentedMap([('home', 'https://github.com/marbl/Krona'), ('license', 'BSD'), ('summary', 'Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files.')])), ('build', CommentedMap([('number', 4)])), ('package', CommentedMap([('name', 'krona'), ('version', 2.5)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('source', CommentedMap([('fn', 'KronaTools-2.5.tar'), ('sha256', '0a0f04949b153df7ec0a06192278c138cd3215ede3dec36063a3a5708f07fa8a'), ('url', 'https://depot.galaxyproject.org/package/source/KronaTools-2.5.tar')])), ('test', CommentedMap([('commands', ['ktImportTaxonomy', 'ktImportXML'])]))]), '2.6': CommentedMap([('about', CommentedMap([('home', 'https://github.com/marbl/Krona'), ('license', 'BSD'), ('summary', 'Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files.')])), ('build', CommentedMap([('number', 4)])), ('package', CommentedMap([('name', 'krona'), ('version', 2.6)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('source', CommentedMap([('fn', 'KronaTools-2.6.tar'), ('sha256', '4f04ece119f0d32fb4e61e36ffd1a7809ad053d4db5dcb5f2e7dd4fd3fc8b1de'), ('url', 'https://github.com/marbl/Krona/releases/download/v2.6/KronaTools-2.6.tar')])), ('test', CommentedMap([('commands', ['ktImportTaxonomy', 'ktImportXML'])]))]), '490': CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '490')])), ('source', CommentedMap([('fn', 'last-490.zip'), ('md5', 'c4d600c373e5f15dc10fb87b1f9b2d37'), ('url', 'http://last.cbrc.jp/last-490.zip'), ('patches', ['maf-convert.23patch', 'last-pair-probs.23patch'])])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['lastal -h', 'lastdb -h', 'last-split -V', 'last-pair-probs -h', 'last-merge-batches -V', 'maf-convert -h'])]))]), '638': CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '638')])), ('source', CommentedMap([('fn', 'last-638.zip'), ('md5', '6c5edd2706934bfe7b3224073b79463b'), ('url', 'http://last.cbrc.jp/last-638.zip'), ('patches', ['maf-convert.23patch'])])), ('build', CommentedMap([('number', 5)])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['lastal -V', 'lastdb -V', 'last-split -V', 'last-pair-probs -V', 'last-merge-batches -V', 'maf-convert -h'])]))]), '874': CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '874')])), ('source', CommentedMap([('fn', 'last-874.zip'), ('md5', '499c75e5f7dd66fd8692d87e06843a13'), ('url', 'http://last.cbrc.jp/last-874.zip'), ('patches', ['maf-convert.23patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc', 'future'])])), ('test', CommentedMap([('commands', ['lastal -V', 'lastdb -V', 'lastal8 -h', 'lastdb8 -h', 'last-split -V', 'last-pair-probs -V', 'last-merge-batches -V', 'maf-convert -h'])]))]), '1.73': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-Format-Strptime-1.73.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-Format-Strptime-1.73.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-datetime', 'perl-datetime-timezone', 'perl-test-more', 'perl-specio-exporter', 'perl-datetime-locale', 'perl-params-validationcompiler', 'perl-test-fatal', 'perl-try-tiny', 'perl-sub-name', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-app-cpanminus', 'perl-datetime-locale', 'perl-specio-exporter', 'perl-datetime-locale', 'perl-params-validationcompiler', 'perl-datetime', 'perl-datetime-timezone', 'perl-sub-name', 'perl-try-tiny'])])), ('test', CommentedMap([('imports', ['DateTime::Format::Strptime', 'DateTime::Format::Strptime::Types'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-Format-Strptime'), ('license', 'artistic_2'), ('summary', 'Parse and format strp and strf time patterns')]))]), '0.5.00': CommentedMap([('about', CommentedMap([('home', 'https://github.com/gt1/bambamc'), ('license', 'GPLv3')])), ('package', CommentedMap([('name', 'libbambamc'), ('version', '0.0.50')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', '0.0.50-release-20140430085950.tar.gz'), ('sha256', '06d2f17b9e4290ef76275f5011dab3b9967baa4d960c2c42f315c7c2f8a89d04'), ('url', 'https://github.com/gt1/bambamc/archive/0.0.50-release-20140430085950.tar.gz')])), ('test', CommentedMap([('files', ['test.c']), ('commands', ['gcc -I ${PREFIX}/include -L ${PREFIX}/lib -lbambamc test.c'])]))]), '3.16': CommentedMap([('about', CommentedMap([('home', ''), ('license', ''), ('summary', 'LIBSVM is an integrated software for support vector classification, (C-SVC,nu-SVC), regression (epsilon-SVR, nu-SVR) and distribution estimation(one-class SVM). It supports multi-class classification.http://www.csie.ntu.edu.tw/~cjlin/libsvm/')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('package', CommentedMap([('name', 'libsvm'), ('version', '3.16')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('git_rev', '041496b61214aff1beba02c8f2957e80bd90c234'), ('git_url', 'https://github.com/cjlin1/libsvm.git')])), ('test', CommentedMap([('commands', [\"svm-train | grep 'Usage' > /dev/null\", \"svm-predict | grep 'Usage' > /dev/null\"])]))]), '1.8.10': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.8.10')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'locarna-1.8.10.tar.gz'), ('url', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/Releases/locarna-1.8.10.tar.gz'), ('sha256', '311be46bd48bfe5ee422dd44caae09d3822f72c4543c24e16298878d5a1f0567')])), ('requirements', CommentedMap([('build', ['viennarna >=2.2.7', 'llvm', 'gcc']), ('run', ['viennarna >=2.2.7', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version'])]))]), '1.8.11': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.8.11')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'locarna-1.8.11.tar.gz'), ('url', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/Releases/locarna-1.8.11.tar.gz'), ('sha256', '29ea5c21689454340a04a03990ad59c1a9e7b53838b5c2ba905a0fd5476f7762')])), ('requirements', CommentedMap([('build', ['viennarna >=2.2.8', 'llvm', 'gcc']), ('run', ['viennarna >=2.2.8', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))]), '1.8.12': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.8.12')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'locarna-1.8.12.tar.gz'), ('url', 'https://github.com/s-will/LocARNA/releases/download/v1.8.12/locarna-1.8.12.tar.gz'), ('sha256', '5c800c74d6de34b59e5c035410cd8231c843f65ac641d402ed6b1ebf5815a59c')])), ('requirements', CommentedMap([('build', ['viennarna ==2.3.1', 'llvm', 'gcc']), ('run', ['viennarna ==2.3.1', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))]), '1.8.7': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.8.7')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'locarna-1.8.7.tar.gz'), ('url', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/Releases/locarna-1.8.7.tar.gz'), ('sha1', '602a0d5a5652f3e5577099859c6ccfaaed5c6fa1')])), ('requirements', CommentedMap([('build', ['viennarna <=2.1.9']), ('run', ['viennarna <=2.1.9', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'exparna_p --version'])]))]), '1.8.9': CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.8.9')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 3)])), ('source', CommentedMap([('fn', 'locarna-1.8.9.tar.gz'), ('url', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/Releases/locarna-1.8.9.tar.gz'), ('sha256', 'ad78fbbcd946932d368ae3d75dea7b94c8c2974a161036e37f351a70972e90a8')])), ('requirements', CommentedMap([('build', ['viennarna >=2.2.5', 'llvm', 'gcc']), ('run', ['viennarna >=2.2.5', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'exparna_p --version'])]))]), '1.9.1': CommentedMap([('package', CommentedMap([('name', 'qiime'), ('version', '1.9.1')])), ('source', CommentedMap([('fn', 'qiime-1.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime/qiime-1.9.1.tar.gz'), ('md5', '32d4db2b2c888dbbfc6aa33f02725526'), ('sha1', '2ab4ac9beb745ed3eca3243312cc9ce14e8a20d8'), ('sha256', 'c12dcdae1fe44762a49668276d7e93f487d87a0dc2ff8ca696ac0eff9d92a8a8')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'xz']), ('run', ['python', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'libgcc', 'glib', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau', 'xz'])])), ('test', CommentedMap([('imports', ['qiime', 'qiime.util', 'qiime.check_id_map', 'qiime.sort']), ('commands', ['add_alpha_to_mapping_file.py -h', 'add_qiime_labels.py -h', 'adjust_seq_orientation.py -h', 'align_seqs.py -h', 'alpha_diversity.py -h', 'alpha_rarefaction.py -h', 'amplicon_contingency_table.py', 'ampliconnoise.py -h', 'assign_taxonomy.py -h', 'beta_diversity.py -h', 'beta_diversity_through_plots.py -h', 'beta_significance.py -h', 'blast_wrapper.py -h', 'categorized_dist_scatterplot.py -h', 'check_id_map.py -h', 'clean_raxml_parsimony_tree.py -h', 'cluster_quality.py -h', 'collapse_samples.py -h', 'collate_alpha.py -h', 'compare_alpha_diversity.py -h', 'compare_categories.py -h', 'compare_distance_matrices.py -h', 'compare_taxa_summaries.py -h', 'compare_trajectories.py -h', 'compute_core_microbiome.py -h', 'compute_taxonomy_ratios.py -h', 'conditional_uncovered_probability.py -h', 'consensus_tree.py -h', 'convert_fastaqual_fastq.py -h', 'convert_unifrac_sample_mapping_to_otu_table.py -h', 'core_diversity_analyses.py -h', 'count_seqs.py -h', 'demultiplex_fasta.py -h', 'denoise_wrapper.py -h', 'denoiser.py -h', 'denoiser_preprocess.py -h', 'denoiser_worker.py -h', 'detrend.py -h', 'differential_abundance.py -h', 'dissimilarity_mtx_stats.py -h', 'distance_matrix_from_mapping.py -h', 'estimate_observation_richness.py -h', 'exclude_seqs_by_blast.py -h', 'extract_barcodes.py -h', 'extract_reads_from_interleaved_file.py -h', 'extract_seqs_by_sample_id.py -h', 'filter_alignment.py -h', 'filter_distance_matrix.py -h', 'filter_fasta.py -h', 'filter_otus_by_sample.py -h', 'filter_otus_from_otu_table.py -h', 'filter_samples_from_otu_table.py -h', 'filter_taxa_from_otu_table.py -h', 'filter_tree.py -h', 'fix_arb_fasta.py -h', 'group_significance.py -h', 'identify_chimeric_seqs.py -h', 'identify_missing_files.py -h', 'identify_paired_differences.py -h', 'inflate_denoiser_output.py -h', 'jackknifed_beta_diversity.py -h', 'join_paired_ends.py -h', 'load_remote_mapping_file.py -h', 'make_2d_plots.py -h', 'make_bipartite_network.py -h', 'make_bootstrapped_tree.py -h', 'make_distance_boxplots.py -h', 'make_distance_comparison_plots.py -h', 'make_fastq.py -h', 'make_library_id_lists.py -h', 'make_otu_heatmap.py -h', 'make_otu_heatmap_html.py -h', 'make_otu_network.py -h', 'make_otu_table.py -h', 'make_per_library_sff.py -h', 'make_phylogeny.py -h', 'make_prefs_file.py -h', 'make_qiime_py_file.py -h', 'make_rarefaction_plots.py -h', 'make_tep.py -h', 'map_reads_to_reference.py -h', 'merge_mapping_files.py -h', 'merge_otu_maps.py -h', 'merge_otu_tables.py -h', 'multiple_extract_barcodes.py -h', 'multiple_join_paired_ends.py -h', 'multiple_rarefactions.py -h', 'multiple_rarefactions_even_depth.py -h', 'multiple_split_libraries_fastq.py -h', 'neighbor_joining.py -h', 'nmds.py -h', 'normalize_table.py -h', 'observation_metadata_correlation.py -h', 'otu_category_significance.py -h', 'parallel_align_seqs_pynast.py -h', 'parallel_alpha_diversity.py -h', 'parallel_assign_taxonomy_blast.py -h', 'parallel_assign_taxonomy_rdp.py -h', 'parallel_assign_taxonomy_uclust.py -h', 'parallel_beta_diversity.py -h', 'parallel_blast.py -h', 'parallel_identify_chimeric_seqs.py -h', 'parallel_map_reads_to_reference.py -h', 'parallel_merge_otu_tables.py -h', 'parallel_multiple_rarefactions.py -h', 'parallel_pick_otus_blast.py -h', 'parallel_pick_otus_sortmerna.py -h', 'parallel_pick_otus_trie.py -h', 'parallel_pick_otus_uclust_ref.py -h', 'parallel_pick_otus_usearch61_ref.py -h', 'pick_closed_reference_otus.py -h', 'pick_de_novo_otus.py -h', 'pick_open_reference_otus.py -h', 'pick_otus.py -h', 'pick_rep_set.py -h', 'plot_rank_abundance_graph.py -h', 'plot_semivariogram.py -h', 'plot_taxa_summary.py -h', 'poller.py -h', 'principal_coordinates.py -h', 'print_metadata_stats.py -h', 'print_qiime_config.py -h', 'process_iseq.py -h', 'process_qseq.py -h', 'process_sff.py -h', 'quality_scores_plot.py -h', 'relatedness.py -h', 'shared_phylotypes.py -h', 'simsam.py -h', 'single_rarefaction.py -h', 'sort_otu_table.py -h', 'split_libraries.py -h', 'split_libraries_fastq.py -h', 'split_libraries_lea_seq.py -h', 'split_otu_table.py -h', 'split_otu_table_by_taxonomy.py -h', 'split_sequence_file_on_sample_ids.py -h', 'start_parallel_jobs.py -h', 'start_parallel_jobs_sc.py -h', 'start_parallel_jobs_slurm.py -h', 'start_parallel_jobs_torque.py -h', 'subsample_fasta.py -h', 'summarize_otu_by_cat.py -h', 'summarize_taxa.py -h', 'summarize_taxa_through_plots.py -h', 'supervised_learning.py -h', 'swarm_breaker.py -h', 'transform_coordinate_matrices.py -h', 'tree_compare.py -h', 'trflp_file_to_otu_table.py -h', 'trim_sff_primers.py -h', 'truncate_fasta_qual_files.py -h', 'truncate_reverse_primer.py -h', 'unweight_fasta.py -h', 'upgma_cluster.py -h', 'validate_demultiplexed_fasta.py -h', 'validate_mapping_file.py -h'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'Quantitative Insights Into Microbial Ecology')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]), '2.1.2': CommentedMap([('package', CommentedMap([('name', 'lofreq'), ('version', '2.1.2')])), ('source', CommentedMap([('fn', 'lofreq_star-2.1.2.tgz'), ('url', 'http://downloads.sourceforge.net/project/lofreq/lofreq_star-2.1.2_macosx.tgz'), ('md5', 'd688ba34a4161d9690e264648aae2def')])), ('build', CommentedMap([('number', 0), ('script', 'mkdir -p $PREFIX/bin; chmod a+x bin/*; cp bin/* $PREFIX/bin'), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['lofreq version'])])), ('about', CommentedMap([('home', 'http://csb5.github.io/lofreq/'), ('license', 'MIT'), ('summary', 'A fast and sensitive variant-caller for inferring SNVs and indels from next-generation sequencing data')]))]), '7.305': CommentedMap([('about', CommentedMap([('home', 'http://mafft.cbrc.jp/alignment/software/'), ('license', 'BSD'), ('summary', 'Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform')])), ('package', CommentedMap([('name', 'mafft'), ('version', '7.305')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'mafft-7.305-without-extensions-src.tgz'), ('md5', '4f645f32df0d0b2edfa3ead7c4486815'), ('url', 'http://mafft.cbrc.jp/alignment/software/mafft-7.305-without-extensions-src.tgz'), ('patches', ['osx-makefile.patch', 'linux-makefile.patch'])])), ('test', CommentedMap([('commands', ['mafft --version'])]))]), '7.310': CommentedMap([('about', CommentedMap([('home', 'http://mafft.cbrc.jp/alignment/software/'), ('license', 'BSD'), ('summary', 'Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform')])), ('package', CommentedMap([('name', 'mafft'), ('version', '7.310')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'mafft-7.310-without-extensions-src.tgz'), ('md5', 'fcb68a47d4506b93bb8463ff29b3cbc4'), ('url', 'http://mafft.cbrc.jp/alignment/software/mafft-7.310-without-extensions-src.tgz'), ('patches', ['osx-makefile.patch', 'linux-makefile.patch'])])), ('test', CommentedMap([('commands', ['mafft --version'])]))]), '2.6.0': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pymvpa-2.6.0.tar.gz'), ('url', 'https://github.com/PyMVPA/PyMVPA/archive/upstream/2.6.0.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'libsvm ==3.21', 'matplotlib', 'scipy', 'numpy', 'swig', 'zlib']), ('run', ['python', 'zlib', 'libsvm ==3.21', 'matplotlib', 'scipy', 'numpy', 'swig', 'libgcc'])])), ('test', CommentedMap([('imports', ['mvpa2'])])), ('about', CommentedMap([('home', 'http://www.pymvpa.org/'), ('license', 'perl_5'), ('summary', 'PyMVPA -- Multivariate Pattern Analysis in Python')]))]), '1.1.01': CommentedMap([('about', CommentedMap([('home', 'http://metavelvet.dna.bio.keio.ac.jp'), ('license', 'GNU General Public License'), ('summary', 'MetaVelvet : An extension of Velvet assembler to de novo metagenome assembly from short sequence reads')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'metavelvet'), ('version', '1.1.01')])), ('requirements', CommentedMap([('build', ['libgcc', 'gcc', 'zlib']), ('run', ['libgcc', 'zlib', 'velvet', 'metavelvet-annois'])])), ('source', CommentedMap([('fn', 'metavelvet-1.1.01.tgz'), ('url', 'http://metavelvet.dna.bio.keio.ac.jp/src/MetaVelvet-1.1.01.tgz'), ('patches', ['metavelvet.velvet.makefile.patch'])])), ('test', CommentedMap([('commands', ['meta-velvetg --help 2>&1 | grep \"contiging and scaffolding program for metagenomics NGS data\"'])]))]), '3.4.1.1': CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA v3 is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data. Version 3 has since been superceded by version 4.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '3.4.1.1')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('source', CommentedMap([('fn', 'mira_3.4.1.1.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_3.4.1.1_linux_x64.tar.bz2'), ('sha256', '8843c3efc72f0ca8c7e681f9d81748604127b1d9c01d0234a5e8b3a55b798d5c')])), ('test', CommentedMap([('commands', ['mira --version | grep \"[0-9]\\\\.[0-9]\"'])]))]), '4.9.5': CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '4.9.5')])), ('source', CommentedMap([('fn', 'mira_4.9.5.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_4.9.5_linux_x64.tar.bz2'), ('sha256', '3848885cb041cd9bf7aa8f220dd3f084443f5060fa433b669f7d28880ba4c61f')])), ('test', CommentedMap([('commands', ['mira --help'])]))]), '4.9.6': CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '4.9.6')])), ('source', CommentedMap([('fn', 'mira_4.9.6.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_4.9.6_darwin_x64.tar.bz2'), ('sha256', '3d0bb15adad0e1e27899d21223cb552968d34d35ad9b1184e85832fc0b3d22ce')])), ('test', CommentedMap([('commands', ['mira --help'])]))]), '2.0.0.7': CommentedMap([('package', CommentedMap([('name', 'mirdeep2'), ('version', '2.0.0.7')])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/mirdeep2_core.git'), ('git_rev', '0e96444370683a1f5eeb4a5c0a85dc118ec0c01d')])), ('build', CommentedMap([('number', 7), ('skip', True)])), ('requirements', CommentedMap([('run', ['perl-threaded', 'perl-pdf-api2'])])), ('about', CommentedMap([('home', 'https://github.com/lpantano/mirdeep2_core.git'), ('license', 'academic'), ('summary', 'miRNA prediction')]))]), '2.0.0.8': CommentedMap([('package', CommentedMap([('name', 'mirdeep2'), ('version', '2.0.0.8')])), ('source', CommentedMap([('url', 'https://www.mdc-berlin.de/45995549/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/mirdeep2_0_0_8.zip'), ('md5', 'a707f7d7ad4a2975fb8b2e78c5bcf483'), ('patches', ['mirdeep2.patch'])])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['bowtie', 'viennarna', 'randfold', 'perl', 'perl-pdf-api2', 'perl-font-ttf', 'perl-compress-raw-zlib', 'perl-lwp-simple'])])), ('test', CommentedMap([('commands', ['make_html.pl', 'miRDeep2.pl -h', 'quantifier.pl || [[ $? == 255 ]]'])])), ('about', CommentedMap([('home', 'https://www.mdc-berlin.de/8551903/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep'), ('license', 'academic'), ('summary', 'A completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs')]))]), '1.25.0': CommentedMap([('package', CommentedMap([('name', 'mothur'), ('version', '1.25.0')])), ('source', CommentedMap([('fn', 'Mothur.1.25.0.zip'), ('url', 'http://www.mothur.org/w/images/6/6d/Mothur.1.25.0.zip'), ('md5', '846f3aa4b6aa1f210d36df65fb5ae198'), ('patches', ['Build-Flags.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['ncurses <{CONDA_NCURSES}}*', 'readline']), ('run', ['ncurses <{CONDA_NCURSES}}*', 'readline'])])), ('test', CommentedMap([('commands', ['mothur \"#help()\"', 'uchime --version'])])), ('about', CommentedMap([('home', 'http://www.mothur.org'), ('license', 'GPL')]))]), '1.36.1': CommentedMap([('package', CommentedMap([('name', 'mothur'), ('version', '1.36.1')])), ('source', CommentedMap([('fn', 'v1.36.1.tar.gz'), ('url', 'https://github.com/mothur/mothur/archive/v1.36.1.tar.gz'), ('md5', '96e176b223468e4d0cda02859a4ac7aa'), ('patches', ['Build-Flags.patch'])])), ('build', CommentedMap([('number', 3), ('string', 'boost1.60_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'ncurses', 'readline', 'boost >=1.55,1.60*', 'blast-legacy']), ('run', ['libgcc', 'zlib', 'ncurses', 'readline', 'boost 1.60*', 'blast-legacy'])])), ('test', CommentedMap([('commands', ['mothur \"#help()\"', 'uchime --version'])])), ('about', CommentedMap([('home', 'http://www.mothur.org'), ('license', 'GPL'), ('summary', 'An expandable software to fill the bioinformatics needs of the microbial ecology community.')]))]), '0.72': CommentedMap([('package', CommentedMap([('name', 'multigps'), ('version', 0.72)])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', 'multigps_v0.72.jar'), ('url', 'http://lugh.bmb.psu.edu/software/multigps/multigps_v0.72.jar'), ('md5', '158bf14a03ec6a2049ab2328591cfd39')])), ('requirements', CommentedMap([('run', ['r-base', 'bioconductor-edger', 'openjdk >=8', 'meme'])])), ('test', CommentedMap([('commands', [\"multigps --version 2>&1 | grep 'MultiGPS'\"])])), ('about', CommentedMap([('home', 'http://mahonylab.org/software/multigps/'), ('license', 'MIT'), ('summary', 'MultiGPS is a framework for analyzing collections of multi-condition ChIP-seq datasets and characterizing differential binding events between conditions.')]))]), '0.73': CommentedMap([('package', CommentedMap([('name', 'multigps'), ('version', 0.73)])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('source', CommentedMap([('fn', 'multigps_v0.73.jar'), ('url', 'http://lugh.bmb.psu.edu/software/multigps/multigps_v0.73.jar'), ('md5', 'c641425ab18d0a8db8b7c1205aa1b6b5')])), ('requirements', CommentedMap([('run', ['r-base', 'bioconductor-edger', 'openjdk >=8', 'meme >=4.11.2'])])), ('test', CommentedMap([('commands', ['multigps --version 2>&1 | grep 0.73'])])), ('about', CommentedMap([('home', 'http://mahonylab.org/software/multigps/'), ('license', 'MIT'), ('summary', 'MultiGPS is a framework for analyzing collections of multi-condition ChIP-seq datasets and characterizing differential binding events between conditions.')]))]), '0.74': CommentedMap([('package', CommentedMap([('name', 'multigps'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'multigps_v<{ version }}.jar'), ('url', 'http://lugh.bmb.psu.edu/software/multigps/multigps_v<{ version }}.jar'), ('sha256', '34c7d9cd5f718f126a9d2a502dd9b04159dd8426c5c8f66929adfdcdc85a946d')])), ('requirements', CommentedMap([('run', ['r-base', 'bioconductor-edger', 'openjdk >=8', 'meme >=4.11.2'])])), ('test', CommentedMap([('commands', ['multigps --version 2>&1 | grep <{ version }}'])])), ('about', CommentedMap([('home', 'http://mahonylab.org/software/multigps/'), ('license', 'MIT'), ('summary', 'MultiGPS is a framework for analyzing collections of multi-condition ChIP-seq datasets and characterizing differential binding events between conditions.')]))]), '0.4.0': CommentedMap([('package', CommentedMap([('name', 'nanopolish'), ('version', '0.4.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'nanopolish_v0.4.0.tar.gz'), ('url', 'https://github.com/jts/nanopolish/archive/v0.4.0.tar.gz'), ('md5', 'b5aa0325568d1e3089162e9e3e54c6c2'), ('patches', ['nanopolish.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'fast5', 'htslib', 'hdf5']), ('run', ['libgcc', 'fast5', 'htslib', 'hdf5', 'biopython'])])), ('test', CommentedMap([('commands', ['nanopolish --help  2>&1 | grep usage  > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/jts/nanopolish'), ('license', 'MIT'), ('license_file', 'LICENSE'), ('summary', 'Signal-level algorithms for MinION data')]))]), '0.2.2': CommentedMap([('package', CommentedMap([('name', 'ngmlr'), ('version', '0.2.2')])), ('source', CommentedMap([('fn', 'ngmlr-v0.2.2.tar.gz'), ('url', 'https://github.com/philres/ngmlr/archive/v0.2.2.tar.gz'), ('md5', '449f0d972c907c64c6ddee08c2308826')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc >=4.8.2', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['ngmlr --help'])])), ('about', CommentedMap([('home', 'https://github.com/philres/ngmlr'), ('license', 'GPL-3.0'), ('summary', 'ngmlr is a long-read mapper desigend to align PacBilo or Oxford Nanopore to a reference genome optimized for structural variation detection')]))]), '0.2.3': CommentedMap([('package', CommentedMap([('name', 'scikit-bio'), ('version', '0.2.3')])), ('source', CommentedMap([('fn', 'scikit-bio-0.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/scikit-bio/scikit-bio-0.2.3.tar.gz'), ('md5', '2bb1148468d3e5129fa8ff5b29cbeaa3')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'matplotlib >=1.1.0', 'scipy >=0.13.0', 'pandas', 'future', 'six', 'natsort', 'ipython']), ('run', ['python', 'numpy x.x', 'matplotlib >=1.1.0', 'scipy >=0.13.0', 'pandas', 'future', 'six', 'natsort', 'ipython'])])), ('test', CommentedMap([('imports', ['skbio', 'skbio.alignment', 'skbio.alignment._lib', 'skbio.alignment.tests', 'skbio.diversity', 'skbio.diversity.alpha', 'skbio.diversity.alpha.tests', 'skbio.diversity.beta', 'skbio.diversity.beta.tests', 'skbio.draw', 'skbio.draw.tests', 'skbio.format', 'skbio.format.sequences', 'skbio.format.sequences.tests', 'skbio.io', 'skbio.io.tests', 'skbio.parse', 'skbio.parse.sequences', 'skbio.parse.sequences.tests', 'skbio.parse.tests', 'skbio.sequence', 'skbio.sequence.tests', 'skbio.stats', 'skbio.stats.distance', 'skbio.stats.distance.tests', 'skbio.stats.ordination', 'skbio.stats.ordination.tests', 'skbio.stats.tests', 'skbio.tests', 'skbio.tree', 'skbio.tree.tests', 'skbio.util', 'skbio.util.tests'])])), ('about', CommentedMap([('home', 'http://scikit-bio.org'), ('license', 'BSD License'), ('summary', 'Data structures, algorithms and educational resources for bioinformatics.'), ('license_file', 'COPYING.txt')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]), '0.2.4': CommentedMap([('about', CommentedMap([('home', 'https://github.com/philres/ngmlr'), ('license', 'MIT'), ('summary', 'ngmlr is a long-read mapper designed to align PacBio or Oxford Nanopore reads to a reference genome and optimized for structural variation detection')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'ngmlr'), ('version', '0.2.4')])), ('requirements', CommentedMap([('build', ['gcc >=4.8.2', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('source', CommentedMap([('fn', 'ngmlr-v0.2.4.tar.gz'), ('md5', '476259d0e8e7b79f3eb911329189a226'), ('url', 'https://github.com/philres/ngmlr/archive/v0.2.4.tar.gz')])), ('test', CommentedMap([('commands', ['ngmlr --help'])]))]), '0.3.1': CommentedMap([('package', CommentedMap([('name', 'svtools'), ('version', '0.3.1')])), ('source', CommentedMap([('fn', 'svtools-0.3.1.tar.gz'), ('url', 'https://pypi.python.org/packages/59/5c/d3ec7b8345e4bf53628b9ad047529d134ab0d3c08e18eed4d741c7340a92/svtools-0.3.1.tar.gz'), ('md5', 'bf313bfdd4f46b3136e449cad7f83770')])), ('build', CommentedMap([('skip', True), ('preserve_egg_dir', True), ('entry_points', ['svtools=svtools.cli:main']), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas', 'setuptools']), ('run', ['python', 'pysam >=0.8.1', 'numpy', 'scipy', 'statsmodels', 'pandas'])])), ('test', CommentedMap([('imports', ['svtools', 'svtools.vcf']), ('commands', ['svtools --help', 'create_coordinates --help'])])), ('about', CommentedMap([('home', 'https://github.com/hall-lab/svtools'), ('license', 'MIT License'), ('summary', 'Tools for processing and analyzing structural variants')]))]), '1.0.010': CommentedMap([('package', CommentedMap([('name', 'obitools'), ('version', '1.0.010')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/00/33/f2774566fc1c661dcea4015f5423817d39b0dff2a3e3151108cf9100275b/OBITools-1.0.010.tar.gz'), ('md5', '28695dd1334d0b7a4717df64817ef3dd'), ('sha1', 'ae56828d2b08de8264056ae533ed41f6945c7be8'), ('sha256', '63766b4543f1e5871cf306811fc06dfaec2928445693e8f1c2768d578daed0e2'), ('patches', ['extractreads.patch', 'extractreads2.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython 0.21.*', 'setuptools', 'sphinx >=1.2.0', 'virtualenv >=1.11.0', 'wheel >=0.24.0']), ('run', ['python', 'esmre', 'xorg-libx11', 'xorg-libxau'])])), ('test', CommentedMap([('commands', ['ali2consensus -h', 'ecodbtaxstat -h', 'ecotag -h', 'ecotaxspecificity -h', 'ecotaxstat -h', 'ecodbtaxstat -h', 'extractreads -h', 'extractreads2 -h', 'illuminapairedend -h', 'ngsfilter -h', 'obiaddtaxids -h', 'obiannotate -h', 'obiclean -h', 'obicomplement -h', 'obiconvert -h', 'obicount -h', 'obicut -h', 'obidistribute -h', 'obigrep -h', 'obihead -h', 'obijoinpairedend -h', 'obisample -h', 'obiselect -h', 'obisort -h', 'obisplit -h', 'obistat -h', 'obitab -h', 'obitail -h', 'obitaxonomy -h', 'obiuniq -h', 'oligotag -h'])])), ('about', CommentedMap([('home', 'http://metabarcoding.org/obitools'), ('license', 'CeCILL-V2'), ('summary', 'The OBITools package is a set of programs specifically designed for analyzing NGS data in a DNA metabarcoding context, taking into account taxonomic information')]))]), '1.2.10': CommentedMap([('package', CommentedMap([('name', 'obitools'), ('version', '1.2.10')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/c6/24/8145846aed626414b19073c95bcf323304206d302e7e8e3e0831eed993dc/OBITools-1.2.10.tar.gz'), ('md5', '4e1f8a39a304224f5c877f7afb845bcd'), ('sha1', 'cb49f1cc73eef5d97110aa092b78f9143f113249'), ('sha256', 'ffa158a31d1bc01ca0ee9b62a27654fc88a01e7fe0a18984f4bbb8a1a72b1f5e'), ('patches', ['extractreads.patch', 'extractreads2.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.24', 'setuptools', 'sphinx >=1.2.0', 'virtualenv >=1.11.0', 'wheel >=0.24.0']), ('run', ['python', 'esmre', 'ipython >=3.0.0', 'xorg-libx11', 'xorg-libxau'])])), ('test', CommentedMap([('commands', ['ali2consensus -h', 'ecodbtaxstat -h', 'ecotag -h', 'ecotaxspecificity -h', 'ecotaxstat -h', 'extractreads -h', 'extractreads2 -h', 'illuminapairedend -h', 'ngsfilter -h', 'obiaddtaxids -h', 'obiannotate -h', 'obiclean -h', 'obicomplement -h', 'obiconvert -h', 'obicount -h', 'obicut -h', 'obidistribute -h', 'obiextract -h', 'obigrep -h', 'obihead -h', 'obijoinpairedend -h', 'obipr2 -h', 'obisample -h', 'obiselect -h', 'obisilva -h', 'obisort -h', 'obisplit -h', 'obistat -h', 'obisubset -h', 'obitab -h', 'obitail -h', 'obitaxonomy -h', 'obiuniq -h', 'oligotag -h'])])), ('about', CommentedMap([('home', 'http://metabarcoding.org/obitools'), ('license', 'CeCILL-V2'), ('summary', 'The OBITools package is a set of programs specifically designed for analyzing NGS data in a DNA metabarcoding context, taking into account taxonomic information')]))]), '1.2.11': CommentedMap([('package', CommentedMap([('name', 'obitools'), ('version', '1.2.11')])), ('source', CommentedMap([('url', 'https://pypi.python.org/packages/dc/53/bef677cfebef9aba707abb36f476d474c1a3d67f02fefdb38fa9fcf92b4f/OBITools-1.2.11.tar.gz'), ('md5', 'a96c303a083df545778f3f8663bdfa29'), ('sha1', '07c645a8ea3134ff635f93e4592ab9dd4b2acbea'), ('sha256', '40af7e5187dc145cc9f9bda25b13026719cec213248a5c71e515263feecfeb6b'), ('patches', ['extractreads.patch', 'extractreads2.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'cython >=0.24', 'setuptools', 'sphinx >=1.2.0', 'virtualenv >=1.11.0', 'wheel >=0.24.0']), ('run', ['python', 'ipython >=3.0.0,<6.0', 'xorg-libx11', 'xorg-libxau'])])), ('test', CommentedMap([('commands', ['ali2consensus -h', 'ecodbtaxstat -h', 'ecotag -h', 'ecotaxspecificity -h', 'ecotaxstat -h', 'illuminapairedend -h', 'ngsfilter -h', 'obiaddtaxids -h', 'obiannotate -h', 'obiclean -h', 'obicomplement -h', 'obiconvert -h', 'obicount -h', 'obicut -h', 'obidistribute -h', 'obiextract -h', 'obigrep -h', 'obihead -h', 'obijoinpairedend -h', 'obipr2 -h', 'obisample -h', 'obiselect -h', 'obisilva -h', 'obisort -h', 'obisplit -h', 'obistat -h', 'obisubset -h', 'obitab -h', 'obitail -h', 'obitaxonomy -h', 'obiuniq -h', 'oligotag -h'])])), ('about', CommentedMap([('home', 'http://metabarcoding.org/obitools'), ('license', 'CeCILL-V2'), ('summary', 'The OBITools package is a set of programs specifically designed for analyzing NGS data in a DNA metabarcoding context, taking into account taxonomic information')]))]), '0.21': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Name-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Sub-Name-0.21.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Sub::Name'])])), ('about', CommentedMap([('home', 'https://github.com/p5sagit/Sub-Name'), ('license', 'perl_5'), ('summary', '(Re)name a sub')]))]), '1.8.6': CommentedMap([('package', CommentedMap([('name', 'openpyxl'), ('version', '1.8.6')])), ('source', CommentedMap([('fn', 'openpyxl-1.8.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c2/ae/4ac6659a37bac4c496cc59378b95e5fbbaf3c1d6249827bb7c7516de979a/openpyxl-1.8.6.tar.gz'), ('md5', 'a610d32ca5460eb78df4eb83c896ba34')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['openpyxl', 'openpyxl.reader', 'openpyxl.shared', 'openpyxl.shared.compat', 'openpyxl.styles', 'openpyxl.writer'])])), ('about', CommentedMap([('home', 'http://openpyxl.readthedocs.org'), ('license', 'MIT License'), ('summary', 'A Python library to read/write Excel 2007 xlsx/xlsm files'), ('license_family', 'MIT')]))]), '2.1.0': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 3)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/TransDecoder/TransDecoder/archive/v<{ version }}.tar.gz'), ('md5', 'f86c688e13ced3cc51fed3d508a2ea05')])), ('requirements', CommentedMap([('build', None), ('run', ['perl-threaded', 'perl-db-file', 'perl-uri', 'cd-hit'])])), ('test', CommentedMap([('commands', ['TransDecoder.Predict -h 2>&1 | grep USAGE', 'TransDecoder.LongOrfs -h 2>&1 | grep USAGE', 'perl -e \"use DB_File\"'])])), ('about', CommentedMap([('home', 'https://transdecoder.github.io/'), ('license', 'Broad Institute'), ('summary', 'TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.')]))]), '2.0.1': CommentedMap([('package', CommentedMap([('name', 'relocate2'), ('version', '2.0.1')])), ('source', CommentedMap([('fn', 'v2.0.1.tar.gz'), ('url', 'https://github.com/stajichlab/RelocaTE2/archive/v2.0.1.tar.gz'), ('md5', '395bd7833163bd72fe7a6ec3d85bf9d5')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python ==2.7.5', 'perl >=5.10.0', 'samtools ==1.3-0', 'bowtie2 ==2.2.8', 'bedtools ==2.26.0-0', 'bwa ==0.6.2-0', 'seqtk ==1.2-0', 'pysam ==0.9.0', 'blat ==35']), ('run', ['python ==2.7.5', 'perl >=5.10.0', 'samtools ==1.3-0', 'bowtie2 ==2.2.8', 'bedtools ==2.26.0-0', 'bwa ==0.6.2-0', 'seqtk ==1.2-0', 'pysam ==0.9.0', 'blat ==35'])])), ('test', CommentedMap([('import', ['pysam'])]))]), '4.9': CommentedMap([('package', CommentedMap([('name', 'paml'), ('version', '4.9')])), ('source', CommentedMap([('fn', 'paml_4.9_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/paml/paml_4.9_src_all.tar.gz'), ('md5', '7c3638fdf19ec44b7b2636477d893de6')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['chi2 1 .1', 'evolver 1 | grep EVOLVER', 'codeml /dev/null 2>&1 | grep paml'])])), ('about', CommentedMap([('home', 'http://abacus.gene.ucl.ac.uk/software/paml.html'), ('summary', 'A package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.'), ('license', 'GNU General Public License v3 (GPLv3)')]))]), '1.7.8': CommentedMap([('package', CommentedMap([('name', 'pasta'), ('version', '1.7.8')])), ('source', CommentedMap([('fn', '43076b8.tar.gz'), ('url', 'https://github.com/smirarab/pasta/archive/43076b8.tar.gz'), ('md5', 'a085bb4f6b4116566a61b785be580ae1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['certifi 2016.2.28 py27_0', 'dendropy 4.1.0 py27_0', 'gcc', 'gnu-wget', 'java-jdk >=6', 'llvm', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0']), ('run', ['dendropy 4.1.0 py27_0', 'java-jdk >=6', 'libgcc', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0'])])), ('test', CommentedMap([('imports', ['pasta']), ('commands', ['clustalw2 -help | grep OPTIONS', 'hmmalign -h | grep HMMER', 'hmmbuild -h | grep HMMER', 'muscle -version | grep MUSCLE', 'prank -help | grep prank', 'raxml -h | grep raxmlHPC', 'raxmlp -h | grep raxmlHPC'])])), ('about', CommentedMap([('home', 'https://github.com/smirarab/pasta'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An implementation of the PASTA (Practical Alignment using Sate and TrAnsitivity) algorithm')]))]), '1.6.5': CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ped_parser'), ('version', '1.6.5')])), ('source', CommentedMap([('fn', 'ped_parser-1.6.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/ped_parser/ped_parser-1.6.5.tar.gz'), ('md5', '1f404e131c3d210d32e6253ac500b200')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pytest', 'click']), ('run', ['python', 'pytest', 'click'])])), ('test', CommentedMap([('imports', ['ped_parser']), ('commands', ['LANG=C.UTF-8 ped_parser --help'])])), ('about', CommentedMap([('home', 'https://github.com/moonso/ped_parser'), ('license', 'BSD License'), ('summary', 'A ped file parser.')]))]), '1.11.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.13.3': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.13.6': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.14.4': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.14.6': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.15.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.15.1': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.16.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip'), ('md5', '4e19c402e2e97365e5a5cbcda8432cfd')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.16.13': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip'), ('sha256', '2fad59130c97f69182dcfb6db95877ec75957242d1c8703512467e1110ceb634')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.16.3': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip'), ('md5', '57d89093241444aa910f84f15f71440d')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '1.16.4': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/peptide-shaker'), ('license', 'Apache License, Version 2.0'), ('summary', 'PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines,\\ncurrently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML.\\nBy combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference,\\nPeptideShaker attempts to give you the best possible understanding of your proteomics data!\\n')])), ('package', CommentedMap([('name', 'peptide-shaker'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.zip'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/peptideshaker/<{ name }}/<{ version }}/<{ name }}-<{ version }}.zip'), ('md5', '14a48413e28a25614f5fda2b381d7197')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'python'])])), ('test', CommentedMap([('commands', ['peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI', 'peptide-shaker eu.isas.peptideshaker.cmd.PeptideShakerCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'PeptideShaker is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"peptide-shaker -Xms512m -Xmx1g\"\\n')]))]), '0.96': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Attribute::Handlers\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Attribute::Handlers'), ('license', 'perl_5'), ('summary', 'Simpler definition of attribute handlers')]))]), '5.74': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['AutoLoader'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/AutoLoader'), ('license', 'perl_5'), ('summary', 'load subroutines only on demand')]))]), '1.48': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/B'), ('license', 'perl_5'), ('summary', 'The Perl Compiler Backend')]))]), '2.23': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'base-2.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/base-2.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use base\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/base'), ('license', 'unknown'), ('summary', 'compile-time class fields')]))]), '0.27': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'namespace-clean-0.27.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RI/RIBASUSHI/namespace-clean-0.27.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-package-stash', 'perl-b-hooks-endofscope', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl', 'perl-package-stash', 'perl-b-hooks-endofscope'])])), ('test', CommentedMap([('imports', ['namespace::clean'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/namespace-clean'), ('license', 'perl_5'), ('summary', 'Keep imports and functions out of your namespace')]))]), '1.10': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Pod-Coverage-1.10.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Test-Pod-Coverage-1.10.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-lib', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', [\"perl -e 'Test::Pod::Coverage'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Pod::Coverage'), ('license', 'artistic_2'), ('summary', 'Check for pod coverage in your distribution')]))]), '2.2.1': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'BioX-Workflow-Command-<{ version }}.tar.gz'), ('url', 'https://github.com/biosails/BioX-Workflow-Command/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-test-class-moose', 'perl-file-details', 'perl-capture-tiny', 'perl-text-diff', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-datetime-format-strptime', 'perl-config-any', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-module-build-tiny', 'perl-params-validate', 'perl-file-slurp', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-test2', 'perl-test-class-moose', 'perl-clone', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus']), ('run', ['perl', 'perl-app-cpanminus', 'perl-list-compare', 'perl-file-path', 'perl-safe', 'perl-class-load', 'perl-test2', 'perl-try-tiny', 'perl-file-find-rule', 'perl-log-log4perl', 'perl-data-walk', 'perl-file-details', 'perl-moose', 'perl-string-approx', 'perl-text-asciitable', 'perl-yaml', 'perl-namespace-autoclean', 'perl-dbm-deep', 'perl-list-uniq', 'perl-moosex-types', 'perl-moosex-getopt', 'perl-datetime-format-strptime', 'perl-text-template', 'perl-datetime', 'perl-path-tiny', 'perl-params-validate', 'perl-config-any', 'perl-moose', 'perl-moosex-app', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-algorithm-dependency', 'perl-clone', 'perl-json', 'perl-file-slurp', 'perl-git-wrapper', 'perl-git-wrapper-plus'])])), ('test', CommentedMap([('commands', ['biox -h', 'biox run -h', 'biox stats -h', 'biox new -h', 'biox add -h'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/BioX-Workflow-Command'), ('license', 'perl_5'), ('summary', 'Opinionated Bioinformatics Genomics Workflow Creator')]))]), '1.06': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/blib'), ('license', 'perl_5'), ('summary', \"Use MakeMaker's uninstalled version of a package\")]))]), '0.19': CommentedMap([('package', CommentedMap([('name', 'perl-btlib'), ('version', '0.19')])), ('source', CommentedMap([('fn', 'BTLib-0.19.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/BTLib/0.19/BTLib-0.19.tar.gz'), ('md5', '58def6382c59a03b8fb7c3343ec3502a')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', [\"perl -e 'use BTLib; $main::seq = BTLib->new;'\"])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/estscan/files/BTLib'), ('license', 'open source'), ('summary', 'Binary Search Tree library')]))]), '0.34': CommentedMap([('package', CommentedMap([('name', 'perl-capture-tiny'), ('version', '0.34')])), ('source', CommentedMap([('fn', 'Capture-Tiny-0.34.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Capture-Tiny-0.34.tar.gz'), ('md5', 'bca6ac7cd6cd88fa9cd9c99e52833a72')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Capture::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Capture-Tiny'), ('license', 'Apache 2.0'), ('summary', 'Capture STDOUT and STDERR from Perl, XS or external programs')]))]), '1.38': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/List::Util'), ('license', 'perl_5'), ('summary', 'A selection of general-utility list subroutines')]))]), '1.33': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'constant-1.33.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/constant-1.33.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use constant\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/module/constant'), ('license', 'perl_5'), ('summary', 'Perl pragma to declare constants')]))]), '2.140640': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/CPAN::Meta::Validator'), ('license', 'perl_5'), ('summary', 'validate CPAN distribution metadata structures')]))]), '5.5004': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/CPAN::Shell'), ('license', 'perl_5'), ('summary', '')]))]), '0.11': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Crypt-OpenSSL-Random-0.11.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RU/RURBAN/Crypt-OpenSSL-Random-0.11.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'openssl', 'perl-extutils-makemaker']), ('run', ['openssl', 'perl'])])), ('test', CommentedMap([('imports', ['Crypt::OpenSSL::Random'])])), ('about', CommentedMap([('home', 'http://sourceforge.net/projects/perl-openssl/'), ('license', 'perl_5'), ('summary', 'OpenSSL/LibreSSL pseudo-random number generator access')]))]), '0.28': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'namespace-autoclean-0.28.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/namespace-autoclean-0.28.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-requires', 'perl-namespace-clean', 'perl-test-more', 'perl-b-hooks-endofscope', 'perl-list-util', 'perl-sub-identify', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-namespace-clean', 'perl-b-hooks-endofscope', 'perl-list-util', 'perl-sub-identify'])])), ('test', CommentedMap([('imports', ['namespace::autoclean'])])), ('about', CommentedMap([('home', 'https://github.com/moose/namespace-autoclean'), ('license', 'perl_5'), ('summary', 'Keep imports out of your namespace')]))]), '2.161': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Dumper-2.161.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/Data-Dumper-2.161.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Data::Dumper'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data::Dumper'), ('license', 'unknown'), ('summary', 'seeds germane, yet not germinated')]))]), '0.04': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Utilities-0.04.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CO/CORNELIS/Data-Utilities-0.04.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-clone', 'perl-test-simple']), ('run', ['perl', 'perl-clone'])])), ('test', CommentedMap([('imports', ['Data::Comparator', 'Data::Differences', 'Data::Merger', 'Data::Transformator', 'Data::Utilities'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data-Utilities'), ('license', 'unknown'), ('summary', 'recursively compare Perl datatypes')]))]), '2.01': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Data-Walk-2.01.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GU/GUIDO/Data-Walk-2.01.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Data::Walk'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Data::Walk'), ('license', 'open_source'), ('summary', 'Traverse Perl data structures.')]))]), '2.30': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'TimeDate-2.30.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GB/GBARR/TimeDate-2.30.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Date::Format'), ('license', 'perl_5'), ('summary', 'Date formating subroutines')]))]), '1.42': CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.42')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.42.tar.gz'), ('md5', '9fdf24ede5528505d8b63bc2169f74de'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.42.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl-threaded', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]), '1.12': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-Locale-1.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-Locale-1.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-cpan-meta-check', 'perl-test-requires', 'perl-test-warnings', 'perl-extutils-makemaker', 'perl-specio-exporter', 'perl-params-validationcompiler', 'perl-test-more', 'perl-namespace-autoclean', 'perl-dist-checkconflicts']), ('run', ['perl', 'perl-namespace-autoclean', 'perl-dist-checkconflicts', 'perl-params-validationcompiler', 'perl-specio-exporter'])])), ('test', CommentedMap([('imports', ['DateTime::Locale', 'DateTime::Locale::Base', 'DateTime::Locale::Catalog', 'DateTime::Locale::Data', 'DateTime::Locale::FromData', 'DateTime::Locale::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-Locale'), ('license', 'perl_5'), ('summary', 'Localization support for DateTime.pm')]))]), '2.09': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DateTime-TimeZone-2.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/DateTime-TimeZone-2.09.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-runtime', 'perl-test-requires', 'perl-test-more', 'perl-test-fatal', 'perl-namespace-autoclean', 'perl-specio-exporter', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-params-validationcompiler']), ('run', ['perl', 'perl-app-cpanminus', 'perl-module-runtime', 'perl-namespace-autoclean', 'perl-specio-exporter', 'perl-try-tiny', 'perl-params-validationcompiler'])])), ('test', CommentedMap([('commands', ['perl -e \"use DateTime::TimeZone\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/DateTime-TimeZone'), ('license', 'perl_5'), ('summary', 'Time zone object base class and factory')]))]), '2.18': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Devel-Symdump-2.18.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/A/AN/ANDK/Devel-Symdump-2.18.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Devel::Symdump\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Devel::Symdump'), ('license', 'perl_5'), ('summary', 'dump symbol names or the symbol table')]))]), '5.88': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Digest::SHA\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Digest::SHA'), ('license', 'perl_5'), ('summary', 'Perl extension for SHA-1/224/256/384/512')]))]), '1.25': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/DynaLoader'), ('license', 'perl_5'), ('summary', 'Dynamically load C libraries into Perl code')]))]), '2.88': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Encode-2.88.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Encode-2.88.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-extutils-makemaker', 'perl-parent']), ('run', ['perl', 'perl-exporter', 'perl-parent'])])), ('test', CommentedMap([('commands', ['perl -e \"use Encode\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Encode'), ('license', 'perl_5'), ('summary', 'allows you to write your script in non-ASCII and non-UTF-8')]))]), '2.1': CommentedMap([('package', CommentedMap([('name', 'perl-estscan2'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'ESTScan2-2.1.tar.gz'), ('url', 'https://sourceforge.net/projects/estscan/files/ESTScan2/2.1/ESTScan2-2.1.tar.gz'), ('md5', 'e6bd51cc264252f27f6222883e63ae36')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl', 'perl-btlib']), ('run', ['perl', 'perl-btlib'])])), ('test', CommentedMap([('commands', ['perl -e \\'use ESTScan; my @mat = ESTScan::LoadMatrix(\"Hs.smat\", \"CODING\", 4.0);\\''])])), ('about', CommentedMap([('home', 'http://estscan.sourceforge.net'), ('license', 'open source'), ('summary', 'Detects coding regions in DNA sequences even if they are of low quality. ESTScan.pm contains the Perl part of the code that reads in the matrices file. The C code that does the actual computation is located in estscan.c.')]))]), '0.14': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Eval-Closure-0.14.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DO/DOY/Eval-Closure-0.14.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-test-requires', 'perl-test-fatal', 'perl-test-more', 'perl-extutils-makemaker', 'perl-io-handle', 'perl-b', 'perl-test2-suite']), ('run', ['perl', 'perl-app-cpanminus', 'perl-test2-suite'])])), ('test', CommentedMap([('imports', ['Eval::Closure'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Eval-Closure'), ('license', 'perl_5'), ('summary', 'safely and cleanly create closures via string eval')]))]), '5.72': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Exporter-5.72.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/T/TO/TODDR/Exporter-5.72.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Exporter\"', 'perl -e \"use Exporter::Heavy\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Exporter'), ('license', 'perl_5'), ('summary', 'Implements default import method for modules')]))]), '7.24': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ExtUtils-MakeMaker-7.24.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.24.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-data-dumper']), ('run', ['perl', 'perl-app-cpanminus', 'perl-data-dumper'])])), ('test', CommentedMap([('commands', ['perl -e \"use ExtUtils::Command\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/ExtUtils-MakeMaker'), ('license', 'perl_5'), ('summary', 'Create a module Makefile')]))]), '1.1006': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Compare'), ('license', 'perl_5'), ('summary', 'Compare files or filehandles')]))]), '1.27': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Time::Piece\"', 'perl -e \"use Time::Seconds\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::Piece'), ('license', 'perl_5'), ('summary', 'Object Oriented time objects')]))]), '2.12': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'File-Path-2.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RI/RICHE/File-Path-2.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-carp', 'perl-devel-symdump', 'perl-app-cpanminus']), ('run', ['perl', 'perl-carp', 'perl-devel-symdump', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use File::Path\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Path'), ('license', 'perl_5'), ('summary', 'Create or remove directory trees')]))]), '3.48_01': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/File::Spec'), ('license', 'perl_5'), ('summary', 'portably perform operations on file names')]))]), '0.2304': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'File-Temp-0.2304.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/File-Temp-0.2304.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-exporter', 'perl-file-path', 'perl-constant', 'perl-parent', 'perl-extutils-makemaker', 'perl-carp', 'perl-app-cpanminus']), ('run', ['perl', 'perl-exporter', 'perl-file-path', 'perl-constant', 'perl-parent', 'perl-carp', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use File::Temp\"'])])), ('about', CommentedMap([('home', 'https://github.com/Perl-Toolchain-Gang/File-Temp'), ('license', 'perl_5'), ('summary', 'return name and handle of a temporary file safely')]))]), '0.91': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Filter::Simple\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Filter::Simple'), ('license', 'perl_5'), ('summary', 'Simplified source filtering')]))]), '1.51': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Pod-1.51.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Test-Pod-1.51.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Pod\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Test-Pod/'), ('license', 'perl_5'), ('summary', 'check for POD errors in files')]))]), '1.11': CommentedMap([('package', CommentedMap([('name', 'potrace'), ('version', 1.11)])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'potrace_1.11_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/potrace/potrace_1.11_src_all.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', ['potrace --help'])])), ('about', CommentedMap([('home', 'http://potrace.sourceforge.net'), ('license', 'GPL'), ('summary', 'A tool for tracing a bitmap, which means, transforming a bitmap into a smooth, scalable image')]))]), '3.0.1': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-3.0.1.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Command-3.0.1.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-app-cpanminus', 'perl-io-interactive', 'perl-test-requires', 'perl-test2', 'perl-try-tiny', 'perl-test-class-moose', 'perl-module-build', 'perl-role-tiny', 'perl-parallel-forkmanager', 'perl-datetime', 'perl-git-wrapper', 'perl-b', 'perl-b-hooks-endofscope', 'perl-list-moreutils', 'perl-json-xs', 'perl-namespace-autoclean', 'perl-posix', 'perl-list-uniq', 'perl-git-wrapper-plus', 'perl-moosex-app', 'perl-moosex-app-role-log4perl', 'perl-test-more', 'perl-file-slurp', 'perl-algorithm-dependency', 'perl-try-tiny', 'perl-archive-tar', 'perl-file-homedir', 'perl-sort-versions', 'perl-list-util', 'perl-symbol', 'perl-template-toolkit', 'perl-module-build-tiny', 'perl-capture-tiny', 'perl-dbm-deep', 'perl-file-path', 'perl-version-next', 'perl-json', 'perl-moose', 'perl-moosex-types', 'perl-log-log4perl', 'perl-variable-magic', 'perl-mce', 'perl-specio-exporter', 'perl-datetime', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate', 'perl-scope-guard', 'perl-sub-attribute', 'perl-string-approx', 'perl-text-asciitable', 'perl-text-ansitable']), ('run', ['perl', 'perl-app-cpanminus', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-git-wrapper', 'perl-list-moreutils', 'perl-json-xs', 'perl-posix', 'perl-list-uniq', 'perl-git-wrapper-plus', 'perl-moosex-app-role-log4perl', 'perl-test-more', 'perl-file-slurp', 'perl-algorithm-dependency', 'perl-try-tiny', 'perl-moosex-app', 'perl-archive-tar', 'perl-file-homedir', 'perl-sort-versions', 'perl-list-util', 'perl-symbol', 'perl-template-toolkit', 'perl-module-build-tiny', 'perl-capture-tiny', 'perl-dbm-deep', 'perl-file-path', 'perl-version-next', 'perl-json', 'perl-moose', 'perl-moosex-types', 'perl-log-log4perl', 'perl-variable-magic', 'perl-mce', 'perl-specio-exporter', 'perl-datetime', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate', 'perl-scope-guard', 'perl-sub-attribute', 'perl-string-approx', 'perl-text-asciitable', 'perl-text-ansitable'])])), ('test', CommentedMap([('imports', ['HPC::Runner::Command', 'HPC::Runner::Command::Plugin::ConfigHome', 'HPC::Runner::Command::Plugin::ConfigHome::Meta::Class', 'HPC::Runner::Command::Utils::Base', 'HPC::Runner::Command::Utils::Git', 'HPC::Runner::Command::Utils::Log', 'HPC::Runner::Command::Utils::Plugin', 'HPC::Runner::Command::Utils::Traits', 'HPC::Runner::Command::execute_array', 'HPC::Runner::Command::execute_job', 'HPC::Runner::Command::execute_job::Utils::MCE', 'HPC::Runner::Command::new', 'HPC::Runner::Command::submit_jobs', 'HPC::Runner::Command::submit_jobs::Plugin::Dummy', 'HPC::Runner::Command::submit_jobs::Plugin::Slurm', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::Batch', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::Directives', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::Files', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::JobDeps', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::JobStats', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::ParseInput', 'HPC::Runner::Command::submit_jobs::Utils::Scheduler::ResolveDeps'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))]), '3.1.1': CommentedMap([('package', CommentedMap([('name', 'rgi'), ('version', '3.1.1')])), ('source', CommentedMap([('fn', 'RGI-mac.tar.gz'), ('url', 'https://card.mcmaster.ca/download/3/RGI-mac.tar.gz/3.1.1'), ('md5', '07c1f2df1565891ddcad47c0c73c3baa')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'six >=1.7.0', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'blast', 'biopython', 'six >=1.7.0', 'libgcc', 'zlib', 'prodigal'])])), ('test', CommentedMap([('commands', ['rgi -V', 'rgi -h', 'rgi_jsontab -h', 'rgi_clean -h', 'rgi_load -h', 'rgi_jsonformat -h'])])), ('about', CommentedMap([('home', 'https://card.mcmaster.ca'), ('license', 'https://card.mcmaster.ca/about'), ('summary', 'This tool provides a preliminary annotation of your DNA sequence(s) based upon the data available in The Comprehensive Antibiotic Resistance Database (CARD). Hits to genes tagged with Antibiotic Resistance ontology terms will be highlighted. As CARD expands to include more pathogens, genomes, plasmids, and ontology terms this tool will grow increasingly powerful in providing first-pass detection of antibiotic resistance associated genes. See license at CARD website')]))]), '3.1.4': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.0': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/arpcard/rgi_conda_dev/archive/<{ commit }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'six >=1.7.0', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'blast', 'biopython', 'six >=1.7.0', 'libgcc', 'zlib', 'prodigal', 'diamond >=0.8.24.86'])])), ('test', CommentedMap([('commands', ['rgi -h', 'rgi_jsontab -h', 'rgi_clean -h', 'rgi_load -h', 'rgi_jsonformat -h', 'rgi --software_version', 'rgi --data_version'])])), ('about', CommentedMap([('home', 'https://card.mcmaster.ca'), ('license', 'https://card.mcmaster.ca/about'), ('summary', 'This tool provides a preliminary annotation of your DNA sequence(s) based upon the data available in The Comprehensive Antibiotic Resistance Database (CARD). Hits to genes tagged with Antibiotic Resistance ontology terms will be highlighted. As CARD expands to include more pathogens, genomes, plasmids, and ontology terms this tool will grow increasingly powerful in providing first-pass detection of antibiotic resistance associated genes. See license at CARD website')]))]), '3.2.10': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-<{version}}.tar.gz'), ('url', 'https://github.com/biosails/HPC-Runner-Command/archive/<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-mce', 'perl-archive-tar', 'perl-test-class-moose', 'perl-file-homedir', 'perl-list-moreutils', 'perl-module-build-tiny', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-moose', 'perl-moosex-app', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-datetime', 'perl-string-approx', 'perl-capture-tiny', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-text-diff', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-datetime', 'perl-test-class-moose', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate']), ('run', ['perl', 'perl-mce', 'perl-app-cpanminus', 'perl-archive-tar', 'perl-moosex-app', 'perl-file-homedir', 'perl-list-moreutils', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-datetime', 'perl-string-approx', 'perl-moose', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-datetime', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl --help', 'hpcrunner.pl submit_jobs --help', 'hpcrunner.pl execute_job --help'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))]), '3.2.11': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', '79daafa1be7c43ad515d38dab5b182c4')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.2': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-<{version}}.tar.gz'), ('url', 'https://github.com/biosails/HPC-Runner-Command/archive/<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-mce', 'perl-archive-tar', 'perl-test-class-moose', 'perl-file-homedir', 'perl-list-moreutils', 'perl-module-build-tiny', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-moose', 'perl-moosex-app', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-datetime', 'perl-string-approx', 'perl-capture-tiny', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-text-diff', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-datetime', 'perl-test-class-moose', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate']), ('run', ['perl', 'perl-mce', 'perl-app-cpanminus', 'perl-archive-tar', 'perl-moosex-app', 'perl-file-homedir', 'perl-list-moreutils', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-datetime', 'perl-string-approx', 'perl-moose', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-datetime', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))]), '3.2.4': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-<{version}}.tar.gz'), ('url', 'https://github.com/biosails/HPC-Runner-Command/archive/<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-mce', 'perl-archive-tar', 'perl-test-class-moose', 'perl-file-homedir', 'perl-list-moreutils', 'perl-module-build-tiny', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-moose', 'perl-moosex-app', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-datetime', 'perl-string-approx', 'perl-capture-tiny', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-text-diff', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-datetime', 'perl-test-class-moose', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate']), ('run', ['perl', 'perl-mce', 'perl-app-cpanminus', 'perl-archive-tar', 'perl-moosex-app', 'perl-file-homedir', 'perl-list-moreutils', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-datetime', 'perl-string-approx', 'perl-moose', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-datetime', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))]), '3.2.5': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.6': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.7': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.8': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', '3c1c82325e19d30dbe8bb7233e9d8c0e')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.9': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-<{version}}.tar.gz'), ('url', 'https://github.com/biosails/HPC-Runner-Command/archive/<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-mce', 'perl-archive-tar', 'perl-test-class-moose', 'perl-file-homedir', 'perl-list-moreutils', 'perl-module-build-tiny', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-moose', 'perl-moosex-app', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-datetime', 'perl-string-approx', 'perl-capture-tiny', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-text-diff', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-datetime', 'perl-test-class-moose', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate']), ('run', ['perl', 'perl-mce', 'perl-app-cpanminus', 'perl-archive-tar', 'perl-moosex-app', 'perl-file-homedir', 'perl-list-moreutils', 'perl-algorithm-dependency', 'perl-git-wrapper', 'perl-git-wrapper-plus', 'perl-dbm-deep', 'perl-file-path', 'perl-mce', 'perl-json-xs', 'perl-file-slurp', 'perl-log-log4perl', 'perl-sort-versions', 'perl-datetime', 'perl-string-approx', 'perl-moose', 'perl-moosex-types', 'perl-moosex-app-role-log4perl', 'perl-moosex-getopt', 'perl-version-next', 'perl-text-ansitable', 'perl-namespace-autoclean', 'perl-io-interactive', 'perl-list-uniq', 'perl-datetime', 'perl-text-asciitable', 'perl-try-tiny', 'perl-json', 'perl-path-tiny', 'perl-template-toolkit', 'perl-params-validate', 'perl-yaml', 'perl-yaml-libyaml', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl --help'])])), ('about', CommentedMap([('home', 'https://github.com/jerowe/HPC-Runner-Command'), ('license', 'perl_5'), ('summary', 'Create composable bioinformatics hpc analyses.')]))]), '0.0.3': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'HPC-Runner-Command-Plugin-Logger-Sqlite-0.0.3.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Command-Plugin-Logger-Sqlite-0.0.3.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'sqlite', 'perl-app-cpanminus', 'perl-hpc-runner-command', 'perl-json-xs', 'perl-file-path', 'perl-datetime', 'perl-capture-tiny', 'perl-log-log4perl', 'perl-json', 'perl-list-uniq', 'perl-text-asciitable', 'perl-file-slurp', 'perl-base', 'perl-module-build-tiny', 'perl-dbi', 'perl-dbd-sqlite', 'perl-clone', 'perl-test2', 'perl-test-class-moose', 'perl-params-validate']), ('run', ['perl', 'sqlite', 'perl-app-cpanminus', 'perl-hpc-runner-command', 'perl-json-xs', 'perl-datetime', 'perl-hpc-runner-command', 'perl-log-log4perl', 'perl-list-uniq', 'perl-text-asciitable', 'perl-json', 'perl-base', 'perl-dbi', 'perl-dbd-sqlite', 'perl-clone', 'perl-params-validate'])])), ('test', CommentedMap([('commands', ['hpcrunner.pl watch_db -h'])])), ('about', CommentedMap([('home', 'https://github.com/biosails/HPC-Runner-Command-Plugin-Logger-Sqlite'), ('license', 'perl_5'), ('summary', 'Log HPC::Runner workflows to a sqlite DB.')]))]), '2.57': CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-slurm'), ('version', '2.57')])), ('source', CommentedMap([('fn', 'HPC-Runner-Slurm-2.57.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Slurm-2.57.tar.gz'), ('md5', '85023fd438456ded737784f31562ce9f')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-build', 'perl-termreadkey', 'perl-app-cpanminus']), ('run', ['perl-threaded', 'perl-termreadkey'])])), ('test', CommentedMap([('imports', ['HPC::Runner::Slurm'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jerowe/hpc-runner-slurm'), ('license', 'perl_5'), ('summary', 'Job Submission to Slurm')]))]), '2.58': CommentedMap([('package', CommentedMap([('name', 'perl-hpc-runner-slurm'), ('version', '2.58')])), ('source', CommentedMap([('fn', 'HPC-Runner-Slurm-2.58.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/HPC-Runner-Slurm-2.58.tar.gz'), ('md5', '1d5d5329159e21163393f01b1acd7523')])), ('build', CommentedMap([('skip', True), ('number', 2)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-log-log4perl', 'perl-moosex-getopt', 'perl-scalar-list-utils', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-file-slurp', 'perl-template-toolkit', 'perl-path-tiny', 'perl-config-any', 'perl-exporter-tiny', 'perl-json', 'perl-extutils-makemaker', 'perl-capture-tiny', 'perl-termreadkey', 'perl-hpc-runner', 'perl-hpc-runner-scheduler', 'perl-hpc-runner-mce']), ('run', ['perl-threaded', 'perl-app-cpanminus', 'perl-pathtools', 'perl-log-log4perl', 'perl-moosex-getopt', 'perl-scalar-list-utils', 'perl-moose', 'perl-moosex-getopt', 'perl-moosex-types', 'perl-file-slurp', 'perl-template-toolkit', 'perl-path-tiny', 'perl-config-any', 'perl-exporter-tiny', 'perl-json', 'perl-termreadkey', 'perl-hpc-runner', 'perl-hpc-runner-scheduler', 'perl-hpc-runner-mce'])])), ('test', CommentedMap([('imports', ['HPC::Runner::Slurm'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/jerowe/hpc-runner-slurm'), ('license', 'perl_5'), ('summary', 'Submit Bioinformatics HPC Workflows with the Slurm scheduler ')]))]), '1.01': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/integer'), ('license', 'perl_5'), ('summary', 'Perl pragma to use integer arithmetic instead of floating point')]))]), '2.064': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Uncompress::RawInflate'), ('license', 'perl_5'), ('summary', 'Read RFC 1951 files/buffers')]))]), '1.35': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Handle'), ('license', 'perl_5'), ('summary', 'supply object methods for I/O handles')]))]), '2.111': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'IO-stringy-2.111.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DS/DSKOLL/IO-stringy-2.111.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/IO::Scalar'), ('license', 'perl_5'), ('summary', 'IO:: interface for reading/writing a scalar')]))]), '0.49': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-Dist-0.49.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-Dist-0.49.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-file-find', 'perl-extutils-makemaker', 'perl-file-path']), ('run', ['perl', 'perl-file-find', 'perl-file-path'])])), ('test', CommentedMap([('imports', ['PAR::Dist'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR::Dist'), ('license', 'unknown'), ('summary', 'Create and manipulate PAR distributions')]))]), '0.63': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'lib-0.63.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/lib-0.63.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use lib\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/lib'), ('license', 'perl_5'), ('summary', 'manipulate @INC at compile time')]))]), '0.53': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'List-SomeUtils-0.53.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/List-SomeUtils-0.53.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker', 'perl-module-implementation', 'perl-text-parsewords']), ('run', ['perl', 'perl-module-implementation', 'perl-text-parsewords'])])), ('test', CommentedMap([('imports', ['List::SomeUtils', 'List::SomeUtils::PP'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/List-SomeUtils'), ('license', 'perl_5'), ('summary', 'Provide the stuff missing in List::Util')]))]), '1.03': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/warnings::register'), ('license', 'perl_5'), ('summary', 'warnings import function')]))]), '1.814': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'MCE-1.814.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MARIOROY/MCE-1.814.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['MCE'])])), ('about', CommentedMap([('home', 'https://github.com/marioroy/mce-perl'), ('license', 'perl_5'), ('summary', 'Many-Core Engine for Perl providing parallel processing capabilities')]))]), '3.13': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/MIME::QuotedPrint'), ('license', 'perl_5'), ('summary', 'Encoding and decoding of quoted-printable strings')]))]), '0.32': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Module::Load'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Load'), ('license', 'perl_5'), ('summary', 'Load modules in a DWIM style')]))]), '0.62': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Module::Load::Conditional\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Load::Conditional'), ('license', 'perl_5'), ('summary', 'Looking up module information / loading at runtime')]))]), '1.000019': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Module-Metadata-1.000019.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/Module-Metadata-1.000019.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', [\"perl -e 'use Module::Metadata'\"])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::Metadata'), ('license', 'perl_5'), ('summary', 'Gather package and POD information from perl module files')]))]), '1.23': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Module-ScanDeps-1.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/Module-ScanDeps-1.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-text-parsewords', 'perl-module-metadata', 'perl-test-more', 'perl-extutils-makemaker', 'perl-test-requires']), ('run', ['perl', 'perl-text-parsewords', 'perl-module-metadata'])])), ('test', CommentedMap([('imports', ['Module::ScanDeps'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Module::ScanDeps'), ('license', 'perl_5'), ('summary', 'Recursively scan Perl code for dependencies')]))]), '2.000002': CommentedMap([('package', CommentedMap([('name', 'perl-moo'), ('version', '2.000002')])), ('source', CommentedMap([('fn', 'Moo-2.000002.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/H/HA/HAARG/Moo-2.000002.tar.gz'), ('md5', '8b84a7289fc6247de5ec5d151105fd6b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-module-runtime', 'perl-role-tiny', 'perl-class-method-modifiers', 'perl-devel-globaldestruction', 'perl-test-fatal']), ('run', ['perl-threaded', 'perl-module-runtime', 'perl-role-tiny', 'perl-class-method-modifiers', 'perl-devel-globaldestruction', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('imports', ['Moo'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Moo'), ('license', 'perl_5'), ('summary', 'Minimalist Object Orientation (with Moose compatibility)')]))]), '2.79': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net::FTP'), ('license', 'perl_5'), ('summary', 'FTP Client class')]))]), '2.14': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Net::Netrc'), ('license', 'perl_5'), ('summary', 'OO interface to users netrc file')]))]), '1.014': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-1.014.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-1.014.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-archive-zip', 'perl-par-dist', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-par-dist', 'perl-archive-zip'])])), ('test', CommentedMap([('commands', ['perl -e \"use PAR\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR'), ('license', 'perl_5'), ('summary', 'Perl Archive Tookit')]))]), '1.036': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'PAR-Packer-1.036.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RS/RSCHUPP/PAR-Packer-1.036.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'perl-text-parsewords', 'perl-extutils-makemaker', 'perl-module-build', 'perl-par', 'perl-par-dist', 'perl-module-scandeps', 'perl-getopt-long', 'perl-getopt-argvfile']), ('run', ['perl', 'perl-text-parsewords', 'perl-par', 'perl-par-dist', 'perl-module-scandeps', 'perl-getopt-long', 'perl-getopt-argvfile'])])), ('test', CommentedMap([('commands', ['perl -e \"use PAR::Filter\"', 'perl -e \"use PAR::Filter::Bleach\"', 'which pp', 'which par.pl']), ('imports', ['PAR::Packer'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PAR::Packer'), ('license', 'perl_5'), ('summary', 'PAR Packager')]))]), '0.38': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Params::Check'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Params::Check'), ('license', 'perl_5'), ('summary', 'Templated based param validation')]))]), '1.26': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JE/JESSE/Test-1.26.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-harness']), ('run', ['perl', 'perl-test-harness'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test'), ('license', 'unknown'), ('summary', 'provides a simple framework for writing test scripts')]))]), '0.23': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Coverage-0.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RC/RCLAMP/Pod-Coverage-0.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-devel-symdump', 'perl-pod-parser', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-devel-symdump', 'perl-pod-parser', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"Pod::Coverage\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Coverage'), ('license', 'unknown'), ('summary', 'Checks if the documentation of a module is comprehensive')]))]), '0.236': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'parent-0.236.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CO/CORION/parent-0.236.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use parent\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/parent'), ('license', 'perl_5'), ('summary', 'Establish an ISA relationship with base classes at compile time')]))]), '0.076': CommentedMap([('package', CommentedMap([('name', 'perl-path-tiny'), ('version', '0.076')])), ('source', CommentedMap([('fn', 'Path-Tiny-0.076.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Path-Tiny-0.076.tar.gz'), ('md5', '495b965187a59af9b99c0f4709013746')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Path::Tiny'])])), ('about', CommentedMap([('home', 'https://github.com/dagolden/Path-Tiny'), ('license', 'apache_2_0'), ('summary', 'File path utility')]))]), '1.09': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO'), ('license', 'perl_5'), ('summary', 'On demand loader for PerlIO layers and root of PerlIO::* name space')]))]), '0.18': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/PerlIO::encoding'), ('license', 'perl_5'), ('summary', 'encoding layer')]))]), '1.60': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Pod::Checker'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Checker'), ('license', 'perl_5'), ('summary', 'Pod::Checker verifies POD documentation contents for compliance with the POD format specifications')]))]), '0.100003': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Coverage-TrustPod-0.100003.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Pod-Coverage-TrustPod-0.100003.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-base', 'perl-lib', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-base', 'perl-lib', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Coverage::TrustPod\"'])])), ('about', CommentedMap([('home', 'https://github.com/rjbs/Pod-Coverage-TrustPod'), ('license', 'perl_5'), ('summary', \"allow a module's pod to contain Pod::Coverage hints\")]))]), '1.07': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Symbol'), ('license', 'perl_5'), ('summary', 'manipulate Perl symbols and their names')]))]), '1.63': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Parser-1.63.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MA/MAREKR/Pod-Parser-1.63.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Parser\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Parser'), ('license', 'unknown'), ('summary', 'Modules for parsing/translating POD format documents')]))]), '2.07': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::PlainText'), ('license', 'perl_5'), ('summary', 'Convert POD data to formatted ASCII text')]))]), '3.35': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Pod-Simple-3.35.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/K/KH/KHW/Pod-Simple-3.35.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test', 'perl-extutils-makemaker', 'perl-pod-escapes', 'perl-carp']), ('run', ['perl', 'perl-test', 'perl-pod-escapes', 'perl-carp'])])), ('test', CommentedMap([('commands', ['perl -e \"use Pod::Simple\"'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Pod-Simple/'), ('license', 'perl_5'), ('summary', 'framework for parsing Pod')]))]), '3.28': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Pod::Simple::Text'), ('license', 'perl_5'), ('summary', 'format Pod as plaintext')]))]), '1.38_03': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/POSIX'), ('license', 'perl_5'), ('summary', '')]))]), '2.000005': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Role-Tiny-2.000005.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/H/HA/HAARG/Role-Tiny-2.000005.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Role::Tiny::With'), ('license', 'perl_5'), ('summary', 'Neat interface for consumers of Role::Tiny roles')]))]), '2.37': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Safe\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Safe'), ('license', 'unknown'), ('summary', 'Compile and execute code in restricted compartments')]))]), '1.45': CommentedMap([('package', CommentedMap([('name', 'perl-scalar-list-utils'), ('version', '1.45')])), ('source', CommentedMap([('fn', 'Scalar-List-Utils-1.45.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PEVANS/Scalar-List-Utils-1.45.tar.gz'), ('md5', 'b469e990ec92a4c9386aaf9e92857ae9')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Scalar::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scalar-List-Utils'), ('license', 'perl_5'), ('summary', 'Common Scalar and List utility subroutines')]))]), '0.36': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Specio-0.36.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Specio-0.36.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus']), ('run', ['perl', 'perl-app-cpanminus'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Specio::Exporter'), ('license', 'perl_5'), ('summary', 'Base class for type libraries')]))]), '3.0609': CommentedMap([('package', CommentedMap([('name', 'perl-statistics-descriptive'), ('version', '3.0609')])), ('source', CommentedMap([('fn', 'Statistics-Descriptive-3.0609.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/Statistics-Descriptive-3.0609.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['Statistics::Descriptive'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/Statistics::Descriptive'), ('license', 'Perl'), ('summary', 'Module of basic descriptive statistical functions')]))]), '3.27': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'String-Approx-3.27.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/J/JH/JHI/String-Approx-3.27.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-extutils-makemaker']), ('run', ['perl', 'perl-test-more'])])), ('test', CommentedMap([('imports', ['String::Approx'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/String::Approx'), ('license', 'unknown'), ('summary', 'Perl extension for approximate matching (fuzzy matching)')]))]), '0.05': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Attribute-0.05.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/G/GF/GFUJI/Sub-Attribute-0.05.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-parsexs', 'perl-test-more', 'perl-extutils-makemaker', 'perl-xsloader']), ('run', ['perl', 'perl-xsloader'])])), ('test', CommentedMap([('imports', ['Sub::Attribute'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sub::Attribute'), ('license', 'perl_5'), ('summary', 'Reliable subroutine attribute handlers')]))]), '0.12': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Sub-Identify-0.12.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RG/RGARCIA/Sub-Identify-0.12.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Sub::Identify'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Sub::Identify'), ('license', 'perl_5'), ('summary', 'Retrieve names of code references')]))]), '0.0203': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Symbol-Util-0.0203.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DE/DEXTER/Symbol-Util-0.0203.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['perl', 'perl-app-cpanminus'])])), ('test', CommentedMap([('imports', ['Symbol::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Symbol::Util'), ('license', 'perl_5'), ('summary', 'Additional utils for Perl symbols manipulation')]))]), '3.30': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TAP::Harness::Env'), ('license', 'perl_5'), ('summary', 'Parsing harness related environmental variables where appropriate')]))]), '0.80': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Class-Moose-0.80.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Test-Class-Moose-0.80.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test2', 'perl-app-cpanminus', 'perl-test-more', 'perl-test-warn', 'perl-test-most', 'perl-test-exception', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-sub-attribute', 'perl-test-requires', 'perl-moose', 'perl-namespace-autoclean', 'perl-parallel-forkmanager', 'perl-exception-class', 'perl-capture-tiny', 'perl-scope-guard', 'perl-list-someutils', 'perl-text-diff']), ('run', ['perl', 'perl-test2', 'perl-app-cpanminus', 'perl-test-more', 'perl-test-warn', 'perl-test-requires', 'perl-test-most', 'perl-extutils-makemaker', 'perl-try-tiny', 'perl-sub-attribute', 'perl-moose', 'perl-namespace-autoclean', 'perl-parallel-forkmanager', 'perl-exception-class', 'perl-exception-class', 'perl-capture-tiny', 'perl-scope-guard', 'perl-list-someutils', 'perl-text-diff'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Class::Moose\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/release/Test-Class-Moose/'), ('license', 'perl_5'), ('summary', 'Serious testing for serious Perl')]))]), '0.25': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-CPAN-Meta-0.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/B/BA/BARBIE/Test-CPAN-Meta-0.25.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::CPAN::Meta\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::CPAN::Meta'), ('license', 'artistic_2'), ('summary', 'Validate your CPAN META.json files')]))]), '3.36': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Harness-3.36.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/L/LE/LEONT/Test-Harness-3.36.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Harness\"'])])), ('about', CommentedMap([('home', 'http://testanything.org/'), ('license', 'perl_5'), ('summary', 'contributing to TAP::Harness')]))]), '1.001002': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::More'), ('license', 'perl_5'), ('summary', 'yet another framework for writing test scripts')]))]), '1.302075': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Simple-1.302075.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Test-Simple-1.302075.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test2'), ('license', 'perl_5'), ('summary', 'Framework for writing test tools that all work together.')]))]), '0.1202': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Test-Unit-Lite-0.1202.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/D/DE/DEXTER/Test-Unit-Lite-0.1202.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Test::Unit::Lite\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Test::Unit::Lite'), ('license', 'perl_5'), ('summary', 'Unit testing without external dependencies')]))]), '0.48': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Text-ANSITable-0.48.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Text-ANSITable-0.48.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-app-cpanminus', 'perl-namespace-clean', 'perl-json-maybexs', 'perl-extutils-makemaker', 'perl-moo', 'perl-module-load', 'perl-perl-osnames', 'perl-test-more', 'perl-term-app-roles', 'perl-border-style', 'perl-color-theme', 'perl-perlio', 'perl-test-exception']), ('run', ['perl', 'perl-app-cpanminus', 'perl-namespace-clean', 'perl-json-maybexs', 'perl-extutils-makemaker', 'perl-moo', 'perl-module-load', 'perl-perl-osnames', 'perl-test-more', 'perl-term-app-roles', 'perl-border-style', 'perl-color-theme', 'perl-perlio', 'perl-test-exception', 'perl-json-maybexs'])])), ('test', CommentedMap([('imports', ['Text::ANSITable', 'Text::ANSITable::BorderStyle::Default', 'Text::ANSITable::ColorTheme::Default', 'Text::ANSITable::StyleSet::AltRow'])])), ('about', CommentedMap([('home', 'https://metacpan.org/release/Text-ANSITable'), ('license', 'perl_5'), ('summary', 'Create nice formatted tables using extended ASCII and ANSI colors')]))]), '0.22': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'XSLoader-0.22.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SA/SAPER/XSLoader-0.22.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker', 'perl-test-more']), ('run', ['perl', 'perl-test-more'])])), ('test', CommentedMap([('commands', ['perl -e \"use XSLoader\"'])])), ('about', CommentedMap([('home', 'https://metacpan.org/module/XSLoader'), ('license', 'perl_5'), ('summary', 'Dynamically load C libraries into Perl code')]))]), '2013.0523': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Tabs'), ('license', 'perl_5'), ('summary', 'expand and unexpand tabs like unix expand(1) and unexpand(1)')]))]), '1.46': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Text-Template-1.46.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MJ/MJD/Text-Template-1.46.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Text::Template', 'Text::Template::Preprocess'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Text::Template'), ('license', 'unknown'), ('summary', 'Expand template text with embedded Perl')]))]), '1.05': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Tie::Hash'), ('license', 'perl_5'), ('summary', '')]))]), '1.9726': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Time::HiRes'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::HiRes'), ('license', 'perl_5'), ('summary', 'High resolution alarm, sleep, gettimeofday, interval timers')]))]), '1.2300': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Time::Local'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Time::Local'), ('license', 'perl_5'), ('summary', 'efficiently compute time from local and GMT time')]))]), '1.17': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('imports', ['Unicode::Normalize'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Unicode::Normalize'), ('license', 'perl_5'), ('summary', 'Unicode Normalization Forms')]))]), '1.71': CommentedMap([('package', CommentedMap([('name', 'perl-uri'), ('version', '1.71')])), ('source', CommentedMap([('fn', 'URI-1.71.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/URI-1.71.tar.gz'), ('md5', '247c3da29a794f72730e01aa5a715daf')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/URI'), ('license', 'perl_5'), ('summary', 'Uniform Resource Identifiers (absolute and relative)')]))]), '0.61': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Variable-Magic-0.61.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/V/VP/VPIT/Variable-Magic-0.61.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-test-more', 'perl-xsloader', 'perl-extutils-makemaker', 'perl-test2-suite']), ('run', ['perl', 'perl-xsloader'])])), ('test', CommentedMap([('imports', ['Variable::Magic'])])), ('about', CommentedMap([('home', 'http://search.cpan.org/dist/Variable-Magic/'), ('license', 'perl_5'), ('summary', 'Associate user-defined magic to variables from Perl.')]))]), '0.797': CommentedMap([('package', CommentedMap([('name', 'perl-vcftools-vcf'), ('version', '0.797')])), ('source', CommentedMap([('fn', 'vcftools-0.1.10.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/vcftools/vcftools_0.1.10.tar.gz'), ('sha256', '0680dd38dc41a9fdbb5ddb22d79e0314009e5e464a5bb7b6da4845cce0181a23')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Vcf;\"'])])), ('about', CommentedMap([('home', 'https://github.com/vcftools/vcftools'), ('license', 'LGPLv3'), ('summary', 'cpanm ready distribution of VCFtools Perl libraries')]))]), '0.840': CommentedMap([('package', CommentedMap([('name', 'perl-vcftools-vcf'), ('version', '0.840')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Vcf;\"'])])), ('about', CommentedMap([('home', 'https://github.com/vcftools/vcftools'), ('license', 'LGPLv3'), ('summary', 'cpanm ready distribution of VCFtools Perl libraries')]))]), '1.126': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '1.126')])), ('source', CommentedMap([('fn', 'picard-tools-1.126.zip'), ('url', 'https://github.com/broadinstitute/picard/releases/download/1.126/picard-tools-1.126.zip'), ('md5', 'e455213f6b9f46433b7d59da78c5a0fb')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', [\"picard 2>&1 | grep 'USAGE: PicardCommandLine'\"])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))]), '1.141': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '1.141')])), ('source', CommentedMap([('fn', 'picard-tools-1.141.zip'), ('url', 'https://github.com/broadinstitute/picard/releases/download/1.141/picard-tools-1.141.zip'), ('md5', 'c6e6eb8c89ff28b7846ba320ce89ebd0')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', [\"picard 2>&1 | grep 'USAGE: PicardCommandLine'\"])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))]), '1.97': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '1.97')])), ('source', CommentedMap([('fn', '1.97.tar.gz'), ('url', 'https://github.com/broadinstitute/picard/archive/1.97.tar.gz'), ('md5', 'aa5e802b7f1abcfe11e632fe52c25c08')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['ant', 'java-jdk 6.*', 'svn']), ('run', ['java-jdk 6.*'])])), ('test', CommentedMap([('commands', [\"picard AddOrReplaceReadGroups.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard apache-ant-1.8.2-bzip2.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard BamIndexStats.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard BamToBfq.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard BuildBamIndex.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CalculateHsMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CheckIlluminaDirectory.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CleanSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectAlignmentSummaryMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectGcBiasMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectInsertSizeMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectMultipleMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectRnaSeqMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CollectTargetedPcrMetrics.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CompareSAMs.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard CreateSequenceDictionary.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard DownsampleSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard EstimateLibraryComplexity.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ExtractIlluminaBarcodes.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ExtractSequences.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard FastqToSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard FilterSamReads.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard FixMateInformation.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard IlluminaBasecallsToFastq.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard IlluminaBasecallsToSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard IntervalListTools.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MakeSitesOnlyVcf.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MarkDuplicates.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MarkIlluminaAdapters.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MeanQualityByCycle.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MergeBamAlignment.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MergeSamFiles.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard MergeVcfs.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard NormalizeFasta.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard QualityScoreDistribution.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ReorderSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ReplaceSamHeader.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard RevertSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard SamFormatConverter.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard SamToFastq.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard snappy-java-1.0.3-rc3.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard SortSam.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard SplitVcfs.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard tribble-1.97.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ValidateSamFile.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard VcfFormatConverter.jar --version 2>&1 | grep '1.97' > /dev/null\", \"picard ViewSam.jar --version 2>&1 | grep '1.97' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')])), ('extra', CommentedMap([('notes', \"Picard version 1.97 uses a wrapper script symlinked to bin/: picard.  The wrapper script takes as arguments the particular jar file you want to run followed by any arguments for that jar.  For example, 'picard ViewSam.jar --help'.\")]))]), '2.7.1': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '2.7.1')])), ('source', CommentedMap([('fn', 'picard.jar'), ('url', 'https://github.com/broadinstitute/picard/releases/download/2.7.1/picard.jar'), ('md5', '323c048b5c33ce4cccfd8b481539968b')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8', 'python'])])), ('test', CommentedMap([('commands', [\"picard 2>&1 | grep 'USAGE: PicardCommandLine'\"])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))]), '2.9.2': CommentedMap([('package', CommentedMap([('name', 'picard'), ('version', '2.9.2')])), ('source', CommentedMap([('fn', 'picard.jar'), ('url', 'https://github.com/broadinstitute/picard/releases/download/2.9.2/picard.jar'), ('md5', '0449279a6a89830917e8bcef3a976ef7')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', ['picard 2>&1 | grep USAGE:'])])), ('about', CommentedMap([('home', 'http://broadinstitute.github.io/picard/'), ('license', 'MIT'), ('summary', 'Java tools for working with NGS data in the BAM format')]))]), '0.23.0': CommentedMap([('package', CommentedMap([('name', 'planemo'), ('version', '0.23.0')])), ('source', CommentedMap([('fn', 'planemo-0.23.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/planemo/planemo-0.23.0.tar.gz'), ('md5', 'f715cae718013fb1e32a5e0d5a4816c8')])), ('build', CommentedMap([('preserve_egg_dir', True), ('entry_points', ['planemo=planemo.cli:planemo']), ('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'six', 'click', 'pyaml', 'bioblend', 'jinja2', 'glob2', 'virtualenv', 'galaxy-lib >=16.4.0']), ('run', ['python', 'six', 'click', 'pyaml', 'bioblend', 'jinja2', 'glob2', 'virtualenv', 'galaxy-lib >=16.4.0'])])), ('test', CommentedMap([('imports', ['planemo', 'planemo.commands', 'planemo.cwl', 'planemo.cwl.cwl2script', 'planemo.galaxy_test', 'planemo.linters', 'planemo.reports', 'planemo.shed', 'planemo.shed2tap', 'planemo.xml']), ('commands', ['LANG=C.UTF-8 planemo --help'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/planemo'), ('license', 'Academic Free License (AFL)'), ('summary', 'Command-line utilities to assist in building tools for the Galaxy project (http://galaxyproject.org/).')]))]), '0.34.1': CommentedMap([('package', CommentedMap([('name', 'planemo'), ('version', '0.34.1')])), ('source', CommentedMap([('fn', 'planemo-0.34.1.tar.gz'), ('url', 'https://pypi.python.org/packages/99/3c/6f1004343bad1e79106eb47030a055b02655e08137f55036ca13f71f55cc/planemo-0.34.1.tar.gz'), ('md5', 'cdb907816e915acaa873f61522166739')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['planemo=planemo.cli:planemo']), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six', 'click', 'pyaml', 'bioblend >=0.8.0', 'jinja2', 'virtualenv', 'galaxy-lib >=16.7.10', 'aenum']), ('run', ['python', 'setuptools', 'six', 'glob2', 'click', 'pyaml', 'bioblend >=0.8.0', 'jinja2', 'virtualenv', 'galaxy-lib >=16.7.10', 'aenum'])])), ('test', CommentedMap([('imports', ['planemo', 'planemo.commands', 'planemo.conda_verify', 'planemo.cwl', 'planemo.cwl.cwl2script', 'planemo.database', 'planemo.engine', 'planemo.galaxy', 'planemo.galaxy.test', 'planemo.linters', 'planemo.reports', 'planemo.shed', 'planemo.shed2tap', 'planemo.test', 'planemo.xml']), ('commands', ['LANG=C.UTF-8 planemo --help'])])), ('about', CommentedMap([('home', 'https://github.com/galaxyproject/planemo'), ('license', 'Academic Free License (AFL)'), ('summary', 'Command-line utilities to assist in building tools for the Galaxy project (http://galaxyproject.org/).'), ('license_family', 'Apache')]))]), '1.0.1': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.0.1')])), ('requirements', CommentedMap([('build', ['zlib']), ('run', [])])), ('source', CommentedMap([('fn', 'stringtie-1.0.1.tar.gz'), ('sha256', 'b0750b40214814270f7380d04efa350d57ef781eb67cf6f4cf168040e93fd98f'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.0.1.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '0.5.1': CommentedMap([('package', CommentedMap([('name', 'poretools'), ('version', '0.5.1')])), ('source', CommentedMap([('fn', 'poretools_v0.5.1.tar.gz'), ('url', 'https://github.com/arq5x/poretools/archive/v0.5.1.tar.gz'), ('md5', 'd246f30b6dd25c1e1580eb81cfd95b85')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'hdf5 >=1.8.7', 'h5py >=2.0', 'matplotlib', 'seaborn', 'pandas']), ('run', ['python', 'hdf5 >=1.8.7', 'h5py >=2.0', 'matplotlib', 'seaborn', 'pandas'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('test', CommentedMap([('commands', ['poretools -v'])])), ('about', CommentedMap([('home', 'http://poretools.readthedocs.org'), ('summary', 'poretools: a toolkit for working with nanopore sequencing data from Oxford Nanopore'), ('license', 'GPL')]))]), '1.6.4': CommentedMap([('package', CommentedMap([('name', 'pp'), ('version', '1.6.4')])), ('source', CommentedMap([('fn', 'pp-1.6.4.zip'), ('url', 'http://www.parallelpython.com/downloads/pp/pp-1.6.4.zip'), ('md5', '0162fd2813d7218a4159430ca9e6f26a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pp'])])), ('about', CommentedMap([('home', 'http://www.parallelpython.com'), ('license', 'BSD License'), ('summary', 'Parallel and distributed programming for Python'), ('license_family', 'BSD')]))]), '1.1.4': CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/primer3/primer3/1.1.4/primer3-1.1.4.tar.gz'), ('md5', 'd9bbdb749a7eea25dd8c653e272d5f39')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'llvm']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', [\"primer3_core --help 2>&1 | grep 'This is primer3'\"])])), ('about', CommentedMap([('home', 'http://primer3.sourceforge.net/'), ('license', 'GPLv2'), ('license_file', 'COPYING.txt'), ('summary', 'Design PCR primers from DNA sequence. From mispriming libraries to sequence quality data to the generation of internal oligos, primer3 does it.')]))]), '0.6.9': CommentedMap([('package', CommentedMap([('name', 'pybedtools'), ('version', '0.6.9')])), ('source', CommentedMap([('url', 'https://github.com/daler/pybedtools/archive/v0.6.9.tar.gz'), ('fn', 'v0.6.9.tar.gz'), ('md5', 'a24e4bcd0c89beb9535295db964f6a4a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'six', 'samtools', 'setuptools', 'pandas', 'cython', 'gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['python', 'bedtools', 'six', 'samtools', 'pandas', 'libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['pybedtools']), ('commands', ['python -c \"import pybedtools; help(pybedtools.BedTool.intersect)\" | grep \"Original BEDTools help\"'])])), ('about', CommentedMap([('home', 'https://github.com/daler/pybedtools'), ('license', 'MIT'), ('summary', 'Wraps BEDTools for use in Python and adds many additional features.')]))]), '0.1.11': CommentedMap([('package', CommentedMap([('name', 'vcftools'), ('version', '0.1.11')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['zlib', 'perl', 'perl-vcftools-vcf ==0.840']), ('run', ['zlib', 'perl-vcftools-vcf ==0.840'])])), ('test', CommentedMap([('commands', ['vcftools -h'])])), ('about', CommentedMap([('home', 'https://vcftools.github.io/'), ('license', 'LGPL'), ('summary', 'A set of tools written in Perl and C++ for working with VCF files.')]))]), '0.4.7.1': CommentedMap([('package', CommentedMap([('name', 'pyfaidx'), ('version', '0.4.7.1')])), ('source', CommentedMap([('fn', 'pyfaidx-0.4.7.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyfaidx/pyfaidx-0.4.7.1.tar.gz'), ('md5', 'f33604a3550c2fa115ac7d33b952127d')])), ('build', CommentedMap([('skip', False), ('entry_points', ['faidx = pyfaidx.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pyfaidx']), ('commands', ['faidx --help'])])), ('about', CommentedMap([('home', 'http://mattshirley.com'), ('license', 'MIT License'), ('summary', 'pyfaidx: efficient pythonic random access to fasta subsequences')]))]), '0.4.8.1': CommentedMap([('package', CommentedMap([('name', 'pyfaidx'), ('version', '0.4.8.1')])), ('source', CommentedMap([('fn', 'pyfaidx-0.4.8.1.tar.gz'), ('url', 'https://pypi.python.org/packages/5f/0e/b22911d548b5fe043629675fccd7c0b8f96b76b0ec0c72a484f4f0615c42/pyfaidx-0.4.8.1.tar.gz'), ('md5', 'f33beb242463eed92381571da4a3aa56')])), ('build', CommentedMap([('skip', False), ('entry_points', ['faidx = pyfaidx.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pyfaidx']), ('commands', ['faidx --help'])])), ('about', CommentedMap([('home', 'http://mattshirley.com'), ('license', 'MIT License'), ('summary', 'pyfaidx: efficient pythonic random access to fasta subsequences')]))]), '0.1.8': CommentedMap([('package', CommentedMap([('name', 'pylibseq'), ('version', '0.1.8')])), ('source', CommentedMap([('fn', 'pylibseq-0.1.8.tar.gz'), ('url', 'https://pypi.python.org/packages/61/a3/2777266d22ace0f461bb323d6ad2a47d67bcaff68b203ff1b0637adc77da/pylibseq-0.1.8.tar.gz'), ('md5', '53ceb50b9c454a2a23890319d06fd57f')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'libsequence', 'gcc']), ('run', ['python', 'libsequence', 'libgcc'])])), ('test', CommentedMap([('imports', ['libsequence'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/pylibseq'), ('license', 'GNU Lesser General Public License v2 or later (LGPLv2+)'), ('summary', 'A Python interface to libsequence')]))]), '0.7.5': CommentedMap([('package', CommentedMap([('name', 'pymzml'), ('version', '0.7.5')])), ('source', CommentedMap([('fn', '9f40b407a3ee604eb588f3850d3981095f4dc7aa.zip'), ('url', 'https://github.com/pymzml/pymzML/archive/9f40b407a3ee604eb588f3850d3981095f4dc7aa.zip'), ('sha256', '93344fae89592792a04531de0563451ae0e078b93e200a49efdc76ceed6ba4ca')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pymzml'])])), ('about', CommentedMap([('home', 'http://pymzml.github.com'), ('license', 'GNU General Public License (GPL)'), ('summary', 'high-throughput mzML parsing'), ('license_family', 'LGPL')]))]), '0.7.7': CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.7.7')])), ('source', CommentedMap([('fn', 'pysam-0.7.7.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pysam/pysam-0.7.7.tar.gz'), ('sha256', 'c9f3018482eec99ee199dda3fdef2aa7424dde6574672a4c0d209a10985755cc')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'gcc', 'llvm', 'zlib']), ('run', ['python <3.0.0', 'libgcc', 'zlib'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', \"Pysam is a python module for reading and manipulating Samfiles. It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.\")]))]), '1.2.2': CommentedMap([('package', CommentedMap([('name', 'stringtie'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'v1.2.2.tar.gz'), ('url', 'https://github.com/gpertea/stringtie/archive/v1.2.2.tar.gz'), ('sha256', 'e6c1eb68f51a28fd2b0573b334a0d6f4a7626f36248ee3bd49592d83aea638d6')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('test', CommentedMap([('commands', ['stringtie --version'])]))]), '0.0.5.dev20160304220337': CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'pypore'), ('version', '0.0.5.dev20160304220337')])), ('source', CommentedMap([('fn', 'pypore-0.0.5.dev20160304220337.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pypore/pypore-0.0.5.dev20160304220337.tar.gz'), ('md5', '173ebb6a8ca721630468ffd5940029fd')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['pypore', 'pypore.filetypes', 'pypore.filetypes.tests', 'pypore.i_o', 'pypore.i_o.tests', 'pypore.sampledata', 'pypore.sampledata.tests', 'pypore.strategies', 'pypore.strategies.tests', 'pypore.tests', 'pyporegui', 'pyporegui.graphicsItems', 'pyporegui.graphicsItems.tests', 'pyporegui.tests', 'pyporegui.widgets', 'pyporegui.widgets.tests']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://parkin.github.io/pypore/'), ('license', 'Apache License, Version 2.0'), ('summary', 'Pythonic/Cythonic Nanopore Translocation Analysis')]))]), '0.8.3': CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.8.3')])), ('source', CommentedMap([('fn', 'pysam-0.8.3.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pysam/pysam-0.8.3.tar.gz'), ('md5', 'b1ae2a8ec3c6d20be30b2bc1aa995bbf')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'libgcc', 'zlib'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', \"Pysam is a python module for reading and manipulating Samfiles. It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.\")]))]), '0.6.7': CommentedMap([('package', CommentedMap([('name', 'pyvcf'), ('version', '0.6.7')])), ('source', CommentedMap([('fn', 'PyVCF-0.6.7.tar.gz'), ('url', 'https://pypi.python.org/packages/source/P/PyVCF/PyVCF-0.6.7.tar.gz'), ('md5', '51b57ce99e0c2f7be2a18d08d8f87734')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['vcf.test', 'vcf'])])), ('about', CommentedMap([('home', 'https://github.com/jamescasbon/PyVCF'), ('license', 'BSD'), ('summary', 'A Variant Call Format reader for Python')]))]), '1.10.4': CommentedMap([('package', CommentedMap([('name', 'r-data.table'), ('version', '1.10.4')])), ('source', CommentedMap([('fn', 'data.table_1.10.4.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/data.table_1.10.4.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/data.table/data.table_1.10.4.tar.gz']), ('sha256', '865fdf6aad389071ad063ec1c75a78ffc86eeb88bba011f3ea5281d243966b7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'data.table\\')\"', '\"%R%\" -e \"library(\\'data.table\\')\"'])])), ('about', CommentedMap([('home', 'http://r-datatable.com'), ('license', 'GPL-3 | file LICENSE'), ('summary', 'Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, a fast friendly file reader and parallel file writer. Offers a natural and flexible syntax, for faster development.'), ('license_family', 'GPL3')]))]), '0.75_7': CommentedMap([('package', CommentedMap([('name', 'r-<{ package }}'), ('version', \"<{ version | replace('-', '_') }}\")])), ('source', CommentedMap([('fn', \"<{ package }}_<{ version | replace('-', '_') }}.tar.gz\"), ('url', ['https://cran.r-project.org/src/contrib/<{ package }}_<{ version }}.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/<{ package }}/<{ package }}_<{ version }}.tar.gz']), ('sha256', '462900100ca598ef21dbe566bf1ab2ce7c49cdeab6b7a600a50489b05f61b61b')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'<{ package }}\\')\"', '\"%R%\" -e \"library(\\'<{ package }}\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=<{ package }}'), ('license', 'GPL (>=2)'), ('summary', \"Compute Hartigan's dip test statistic for unimodality / multimodality and provide a test with simulation based p-values, where the original public code has been corrected.\")]))]), '0.61.0': CommentedMap([('package', CommentedMap([('name', 'r-pscbs'), ('version', '0.61.0')])), ('source', CommentedMap([('fn', 'PSCBS_0.61.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/PSCBS_0.61.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/PSCBS/PSCBS_0.61.0.tar.gz']), ('sha256', '0497e728d8c12896dbd2cf89a7262718ad1cb5a87f6155ea92acad6c3391ba12')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-dnacopy >=1.40.0', 'bioconductor-aroma.light >=2.2.1', 'r-r.cache >=0.12.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-future >=1.1.1', 'r-listenv >=0.6.0', 'r-matrixstats >=0.51.0']), ('run', ['r-base', 'bioconductor-dnacopy >=1.40.0', 'bioconductor-aroma.light >=2.2.1', 'r-r.cache >=0.12.0', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.21.0', 'r-r.utils >=2.5.0', 'r-future >=1.1.1', 'r-listenv >=0.6.0', 'r-matrixstats >=0.51.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSCBS\\')\"', '\"%R%\" -e \"library(\\'PSCBS\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/HenrikBengtsson/PSCBS'), ('license', 'GPL (>= 2)'), ('summary', 'Segmentation of allele-specific DNA copy number data and detection of regions with abnormal copy number within each parental chromosome.  Both tumor-normal paired and tumor-only analyses are supported.')]))]), '1.1.3': CommentedMap([('package', CommentedMap([('name', 'r-rcircos'), ('version', '1.1.3')])), ('source', CommentedMap([('fn', 'RCircos_1.1.3.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/RCircos/RCircos_1.1.3.tar.gz'])])), ('build', CommentedMap([('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCircos\\')\"', '\"%R%\" -e \"library(\\'RCircos\\')\"'])])), ('about', CommentedMap([('home', 'http://bitbucket.org/henryhzhang/rcircos'), ('license', 'GPL (>= 2)'), ('summary', 'A simple and flexible way to generate Circos 2D track plot images for genomic data visualization is implemented in this package. The types of plots include: heatmap, histogram, lines, scatterplot, tiles and plot items for further decorations include connector, link (lines and ribbons), and text (gene) label. All functions require only R graphics package that comes with R base installation.  ')]))]), '1.2.1': CommentedMap([('package', CommentedMap([('name', 'tclap'), ('version', '1.2.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'tclap-1.2.1.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/tclap/tclap-1.2.1.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc   [linux]', 'llvm  [osx]']), ('run', None)])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://tclap.sourceforge.net/'), ('license', 'MIT'), ('summary', 'TCLAP is a small, flexible library that provides a simple interface for defining and accessing command line arguments.')]))]), '7.3.0': CommentedMap([('package', CommentedMap([('name', 'raxml'), ('version', '7.3.0')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'RAxML-7.3.0.tar.bz2'), ('url', 'http://sco.h-its.org/exelixis/resource/download/software/RAxML-7.3.0.tar.bz2')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['raxmlHPC-PTHREADS -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/raxml/index.html'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))]), '8.2.9': CommentedMap([('package', CommentedMap([('name', 'raxml'), ('version', '8.2.9')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'raxml_v8.2.9.tar.gz'), ('url', 'https://github.com/stamatak/standard-RAxML/archive/v8.2.9.tar.gz'), ('md5', 'ba77f7df7ce5e2a7967decd9e4278a72')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['raxmlHPC -h > /dev/null', 'raxmlHPC-PTHREADS -h > /dev/null', 'raxmlHPC-SSE3 -h > /dev/null', 'raxmlHPC-PTHREADS-SSE3 -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/raxml/index.html'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))]), '2015.09.2': CommentedMap([('package', CommentedMap([('name', 'rdkit'), ('version', '2015.09.2')])), ('source', CommentedMap([('url', 'https://github.com/rdkit/rdkit/archive/Release_2015_09_2.tar.gz'), ('fn', 'Release_2015_09_2.tar.gz'), ('patches', ['rdpaths.patch', 'rdconfig.patch [win]'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake [unix]', 'requests', 'boost >=1.48.0,<{CONDA_BOOST}}*', 'python', 'numpy', 'pillow', 'freetype', 'py2cairo']), ('run', ['python', 'numpy', 'boost <{CONDA_BOOST}}*'])])), ('test', CommentedMap([('imports', ['rdkit', 'rdkit.Avalon', 'rdkit.Dbase', 'rdkit.ML', 'rdkit.RDPaths', 'rdkit.VLib', 'rdkit.Chem', 'rdkit.DistanceGeometry', 'rdkit.Numerics', 'rdkit.RDRandom', 'rdkit.DataManip', 'rdkit.ForceField', 'rdkit.RDConfig', 'rdkit.DataStructs', 'rdkit.Geometry'])])), ('about', CommentedMap([('home', 'http://rdkit.org'), ('license', 'BSD'), ('summary', 'Open-Source Cheminformatics Software')]))]), '2016.03.3': CommentedMap([('package', CommentedMap([('name', 'rdkit'), ('version', '2016.03.3')])), ('source', CommentedMap([('url', 'https://github.com/rdkit/rdkit/archive/Release_2016_03_3.tar.gz'), ('fn', 'Release_2016_03_3.tar.gz'), ('sha256', '229b33e63be90b202b771007bd30e17dddb079a96415471ed7c66420ac28d93a')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'py<{CONDA_PY}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake [unix]', 'requests', 'boost >=1.48.0,<{CONDA_BOOST}}*', 'python', 'numpy >=1.7', 'pillow', 'freetype', 'py2cairo']), ('run', ['python', 'numpy >=1.7', 'boost <{CONDA_BOOST}}*', 'pillow'])])), ('test', CommentedMap([('imports', ['rdkit', 'rdkit.Avalon', 'rdkit.Dbase', 'rdkit.ML', 'rdkit.RDPaths', 'rdkit.VLib', 'rdkit.Chem', 'rdkit.DistanceGeometry', 'rdkit.Numerics', 'rdkit.RDRandom', 'rdkit.DataManip', 'rdkit.ForceField', 'rdkit.RDConfig', 'rdkit.DataStructs', 'rdkit.Geometry'])])), ('about', CommentedMap([('home', 'http://rdkit.org'), ('license', 'BSD'), ('summary', 'Open-Source Cheminformatics Software')]))]), '3.1.2': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/arpcard/rgi_conda_dev/archive/<{ commit }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'six >=1.7.0', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'blast', 'biopython', 'six >=1.7.0', 'libgcc', 'zlib', 'prodigal', 'diamond >=0.8.24.86'])])), ('test', CommentedMap([('commands', ['rgi -h', 'rgi_jsontab -h', 'rgi_clean -h', 'rgi_load -h', 'rgi_jsonformat -h', 'rgi --software_version', 'rgi --data_version'])])), ('about', CommentedMap([('home', 'https://card.mcmaster.ca'), ('license', 'https://card.mcmaster.ca/about'), ('summary', 'This tool provides a preliminary annotation of your DNA sequence(s) based upon the data available in The Comprehensive Antibiotic Resistance Database (CARD). Hits to genes tagged with Antibiotic Resistance ontology terms will be highlighted. As CARD expands to include more pathogens, genomes, plasmids, and ontology terms this tool will grow increasingly powerful in providing first-pass detection of antibiotic resistance associated genes. See license at CARD website')]))]), '3.2.1': CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('git_url', '<{ source_url }}'), ('git_rev', '<{ commit }}')])), ('build', CommentedMap([('skip', True), ('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'biopython', 'six >=1.7.0', 'gcc', 'llvm', 'zlib']), ('run', ['python', 'blast', 'biopython', 'six >=1.7.0', 'libgcc', 'zlib', 'prodigal', 'diamond >=0.8.24.86'])])), ('test', CommentedMap([('commands', ['rgi -h', 'rgi_jsontab -h', 'rgi_clean -h', 'rgi_load -h', 'rgi_jsonformat -h', 'rgi --software_version', 'rgi --data_version'])])), ('about', CommentedMap([('home', 'https://card.mcmaster.ca'), ('license', 'https://card.mcmaster.ca/about'), ('summary', 'This tool provides a preliminary annotation of your DNA sequence(s) based upon the data available in The Comprehensive Antibiotic Resistance Database (CARD). Hits to genes tagged with Antibiotic Resistance ontology terms will be highlighted. As CARD expands to include more pathogens, genomes, plasmids, and ontology terms this tool will grow increasingly powerful in providing first-pass detection of antibiotic resistance associated genes. See license at CARD website')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))]), '2.1.6': CommentedMap([('about', CommentedMap([('home', 'http://bibiserv.techfak.uni-bielefeld.de/rnashapes'), ('license', ''), ('summary', \"RNAshape abstraction maps structures to a tree-like domain of shapes, retaining adjacency and nesting of structural features, but disregarding helix lengths. Shape abstraction integrates well with dynamic programming algorithms, and hence it can be applied during structure prediction rather than afterwards. This avoids exponential explosion and can still give us a non-heuristic and complete account of properties of the molecule's folding space.\")])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'rnashapes'), ('version', '2.1.6')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'RNAshapes-2.1.6.tar.gz'), ('sha256', '6923af49f931efa4f3ad2a05f87a8e5222b6f865d0347e5598de3661e73bb81b'), ('url', 'https://github.com/bgruening/download_store/raw/master/RNAshapes/RNAshapes-2.1.6.tar.gz')])), ('test', CommentedMap([('commands', ['RNAshapes  -h &> /dev/null'])]))]), '5.7': CommentedMap([('about', CommentedMap([('home', 'http://rna.urmc.rochester.edu/RNAstructure.html'), ('license', ''), ('summary', 'RNAstructure is a complete package for RNA and DNA secondary structure  prediction and analysis. It includes algorithms for   secondary structure  prediction, including facility to predict base pairing probabilities. It  also can be used to predict bimolecular structures and can predict the  equilibrium binding affinity of an oligonucleotide to a structured RNA  target. This is useful for siRNA design. It can also predict secondary  structures common to two, unaligned sequences, which is much more accurate  than single sequence secondary structure prediction. Finally, RNAstructure  can take a number of different types of experiment mapping data to constrain  or restrain structure prediction. These include chemical mapping, enzymatic  mapping, NMR, and SHAPE data.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'rnastructure'), ('version', '5.7')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RNAstructure-5.7.tgz'), ('sha256', '7d7060c2180b95e0d0e0a840499df48de3571e37ca6bef51325377fe7ca28ae6'), ('url', 'http://depot.galaxyproject.org/package/source/RNAstructure-5.7.tgz')])), ('test', CommentedMap([('commands', ['refold -h', 'ProbKnot -h', 'dot2ct -h'])]))]), '0.3.0.2': CommentedMap([('package', CommentedMap([('name', 'rnftools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'rnftools-<{ version }}.tar.gz'), ('url', 'https://github.com/karel-brinda/rnftools/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'pip install .')])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'snakemake', 'argparse']), ('run', ['python', 'samtools', 'pysam', 'zlib', 'wgsim', 'dwgsim', 'art', 'mason', 'curesim', 'snakemake', 'argparse'])])), ('test', CommentedMap([('commands', ['rnftools -h']), ('imports', ['rnftools'])])), ('about', CommentedMap([('home', 'http://karel-brinda.github.io/rnftools'), ('license', 'MIT'), ('summary', 'RNF framework for NGS: simulation of reads, evaluation of mappers, conversion of RNF-compliant data.')]))]), '0.3.1.0': CommentedMap([('package', CommentedMap([('name', 'rnftools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'rnftools-<{ version }}.tar.gz'), ('url', 'https://github.com/karel-brinda/rnftools/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'pip install .')])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'snakemake', 'argparse', 'reportlab']), ('run', ['python', 'samtools', 'pysam', 'zlib', 'wgsim', 'dwgsim', 'art', 'mason', 'curesim', 'snakemake', 'argparse', 'pyfaidx', 'bwa', 'gnuplot >=5.0', 'beautifulsoup4', 'reportlab'])])), ('test', CommentedMap([('commands', ['rnftools -h']), ('imports', ['rnftools'])])), ('about', CommentedMap([('home', 'http://karel-brinda.github.io/rnftools'), ('license', 'MIT'), ('summary', 'RNF framework for NGS: simulation of reads, evaluation of mappers, conversion of RNF-compliant data.')]))]), '0.3.1.1': CommentedMap([('package', CommentedMap([('name', 'rnftools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'rnftools-<{ version }}.tar.gz'), ('url', 'https://github.com/karel-brinda/rnftools/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'pip install .')])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'snakemake', 'argparse']), ('run', ['python', 'samtools', 'pysam', 'zlib', 'wgsim', 'dwgsim', 'art', 'mason', 'curesim', 'snakemake', 'argparse', 'pyfaidx', 'bwa', 'gnuplot >=5.0', 'beautifulsoup4', 'reportlab ==3.3.0'])])), ('test', CommentedMap([('commands', ['rnftools -h']), ('imports', ['rnftools'])])), ('about', CommentedMap([('home', 'http://karel-brinda.github.io/rnftools'), ('license', 'MIT'), ('summary', 'RNF framework for NGS: simulation of reads, evaluation of mappers, conversion of RNF-compliant data.')]))]), '1.9': CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ruby'), ('version', '1.9.3')])), ('source', CommentedMap([('fn', 'ruby-1.9.3-p484.tar.gz'), ('url', 'http://cache.ruby-lang.org/pub/ruby/1.9/ruby-1.9.3-p484.tar.gz'), ('sha256', 'd684bc3a5ba72cda9ef30039f783c0f8cdc325bae5c8738c7bf05577cbe8f31d')])), ('test', CommentedMap([('commands', ['ruby --version', 'ruby -e \\'puts \"Hi, from Ruby!\"\\'', 'ruby -e \\'require \"rubygems\"\\'', 'gem --version'])])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'libxml2 ==2.9.2', 'readline ==6.2', 'yaml', 'openssl 1.0.1g', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://www.ruby-lang.org/'), ('license', 'BSD')]))]), '2.1.7': CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'ruby'), ('version', '2.1.7')])), ('source', CommentedMap([('fn', 'ruby-2.1.7.tar.gz'), ('url', 'https://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.7.tar.gz'), ('sha256', 'f59c1596ac39cc7e60126e7d3698c19f482f04060674fdfe0124e1752ba6dd81')])), ('test', CommentedMap([('commands', ['ruby --version', 'ruby -e \\'puts \"Hi, from Ruby!\"\\'', 'ruby -e \\'require \"rubygems\"\\'', 'gem --version'])])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'libxml2 ==2.9.2', 'readline ==6.2', 'yaml', 'openssl 1.0.1g', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://www.ruby-lang.org/'), ('license', 'BSD')]))]), '0.1.12': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.12')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*'])])), ('source', CommentedMap([('fn', 'samtools-0.1.12.tar.bz2'), ('md5', '500ae32fb431de3940e0e955744b9b36'), ('url', 'https://downloads.sourceforge.net/project/samtools/samtools/0.1.12/samtools-0.1.12a.tar.bz2')])), ('test', CommentedMap([('files', ['ex1.fa']), ('commands', ['samtools faidx ex1.fa'])]))]), '0.1.13': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.13')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*'])])), ('source', CommentedMap([('fn', 'samtools-0.1.13.tar.bz2'), ('md5', '2e66b94639f90d10cc935bb145ef71f3'), ('url', 'https://downloads.sourceforge.net/project/samtools/samtools/0.1.13/samtools-0.1.13.tar.bz2')])), ('test', CommentedMap([('files', ['ex1.fa']), ('commands', ['samtools faidx ex1.fa'])]))]), '0.1.14': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.14')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*'])])), ('source', CommentedMap([('fn', 'samtools-0.1.14.tar.bz2'), ('md5', 'aa2bed7b3d96e64cbeb3726f1fb6d880'), ('url', 'https://downloads.sourceforge.net/project/samtools/samtools/0.1.14/samtools-0.1.14.tar.bz2')])), ('test', CommentedMap([('files', ['ex1.fa']), ('commands', ['samtools faidx ex1.fa'])]))]), '0.1.15': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.15')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'ncurses <{CONDA_NCURSES}}*'])])), ('source', CommentedMap([('fn', 'samtools-0.1.15.tar.bz2'), ('md5', 'caf1cac191fdabd57a309e36f359c047'), ('url', 'https://downloads.sourceforge.net/project/samtools/samtools/0.1.15/samtools-0.1.15.tar.bz2')])), ('test', CommentedMap([('files', ['ex1.fa']), ('commands', ['samtools faidx ex1.fa'])]))]), '0.1.16': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.16')])), ('source', CommentedMap([('fn', 'samtools-0.1.16.tar.bz2'), ('sha256', 'ae37c49f2414d1fabe597f68f2351f6cce4b6c0a1a512a5d739bbf759f80a726'), ('url', 'http://depot.galaxyproject.org/package/source/samtools/samtools-0.1.16.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))]), '0.1.17': CommentedMap([('package', CommentedMap([('name', 'samtools'), ('version', '0.1.17')])), ('requirements', CommentedMap([('build', ['gcc', 'pkg-config', 'zlib']), ('run', ['pkg-config', 'zlib'])])), ('source', CommentedMap([('fn', 'samtools-0.1.17.tar.bz2'), ('url', ['http://downloads.sourceforge.net/project/samtools/samtools/0.1.17/samtools-0.1.17.tar.bz2', 'http://download.lab7.io/sources/samtools-0.1.17.tar.bz2']), ('md5', '9549f7a6bba894766df9c3dd18d61f0b'), ('patches', ['makefiles.patch', 'sources.patch'])])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'http://samtools.sourceforge.net/'), ('license', 'MIT'), ('license_file', 'COPYING'), ('summary', 'Library for manipulating alignments in the SAM/BAM format')])), ('test', CommentedMap([('commands', ['samtools --help'])]))]), '0.1.18': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.18')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'samtools-0.1.18.tar.bz2'), ('sha256', 'f3faaf34430d4782956562eb72906289e8e34d44d0c4d73837bdbeead7746b16'), ('url', 'http://depot.galaxyproject.org/package/source/samtools/samtools-0.1.18.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))]), '0.1.19': CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM and BAM files')])), ('build', CommentedMap([('number', 2)])), ('package', CommentedMap([('name', 'samtools'), ('version', '0.1.19')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'ncurses', 'zlib']), ('run', ['libgcc', 'ncurses', 'zlib'])])), ('source', CommentedMap([('fn', 'samtools-0.1.19.tar.bz2'), ('sha256', 'd080c9d356e5f0ad334007e4461cbcee3c4ca97b8a7a5a48c44883cf9dee63d4'), ('url', 'http://depot.galaxyproject.org/package/source/samtools/samtools-0.1.19.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes'])]))]), '0.9': CommentedMap([('package', CommentedMap([('name', 'screed'), ('version', '0.9')])), ('source', CommentedMap([('fn', 'screed-0.9.tar.gz'), ('md5', '611dd11105489e4153265e055f13debd'), ('url', 'https://pypi.python.org/packages/source/s/screed/screed-0.9.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'bz2file ==0.98']), ('run', ['python', 'bz2file ==0.98'])])), ('test', CommentedMap([('imports', ['screed'])])), ('about', CommentedMap([('home', 'https://github.com/dib-lab/screed'), ('license', 'BSD'), ('summary', 'a simple read-only sequence database, designed for short reads')]))]), '2.9.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.13': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', 'd7f9e271e2b86bba1f9d641cadf380ce')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '3.2.3': CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk  >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]), '1.2.0': CommentedMap([('package', CommentedMap([('name', 'seedme'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'seedme-1.2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/source/s/seedme/seedme-1.2.0.tar.gz'), ('md5', 'd8c342c6c5f345fd5a8afabdaf14a307'), ('patches', ['license.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests ==2.7.0']), ('run', ['python', 'requests ==2.7.0', 'curl'])])), ('test', CommentedMap([('commands', ['seedme.py --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.seedme.org/downloads'), ('license', 'GPL'), ('license_file', 'LICENSE.txt'), ('summary', 'Python REST like client for SeedMe.org')]))]), 'r75': CommentedMap([('package', CommentedMap([('name', 'seqtk'), ('version', 'r75')])), ('source', CommentedMap([('git_rev', '08b3625c2a7aae3eca9ab056e1adea52ec22cbef'), ('git_url', 'https://github.com/lh3/seqtk.git')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('about', CommentedMap([('home', 'https://github.com/lh3/seqtk'), ('license', 'MIT License'), ('summary', 'Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format')])), ('test', CommentedMap([('commands', ['seqtk seq'])]))]), 'r82': CommentedMap([('package', CommentedMap([('name', 'seqtk'), ('version', 'r82')])), ('source', CommentedMap([('fn', 'seqtk-r82.tar.gz'), ('url', 'https://github.com/lh3/seqtk/archive/5e1e8dbd506ea1ff8c77d468a1f27b8e8f73eac0.tar.gz'), ('sha256', 'dd4b54a3ba57279659153204aa59aa1d5efe4d307549ac7b8e93b67eb9d3248f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/seqtk'), ('license', 'MIT License'), ('summary', 'Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format')])), ('test', CommentedMap([('commands', ['seqtk seq'])]))]), '0.1.126': CommentedMap([('package', CommentedMap([('name', 'skewer'), ('version', '0.1.126')])), ('source', CommentedMap([('fn', '0.1.126.zip'), ('url', 'https://github.com/relipmoc/skewer/archive/0.1.126.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['skewer 2>&1 | grep \"skewer (0.1.126)\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/relipmoc/skewer.git'), ('license', 'MIT'), ('summary', 'A fast and accurate adapter trimmer for next-generation sequencing paired-end reads.')]))]), '0.1': CommentedMap([('package', CommentedMap([('name', 'smallgenomeutilities'), ('version', '0.1')])), ('source', CommentedMap([('fn', 'smallgenomeutilities-0.1.tar.gz'), ('url', 'https://pypi.io/packages/source/s/smallgenomeutilities/smallgenomeutilities-0.1.tar.gz'), ('md5', '339439b1f97ecb1fc91378f78a06bebe')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'biopython', 'numpy', 'progress', 'pysam']), ('run', ['python', 'biopython', 'numpy', 'progress', 'pysam'])])), ('test', CommentedMap([('imports', ['smallgenomeutilities']), ('commands', ['convert_qr -h', 'convert_reference -h'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/smallgenomeutilities'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('summary', 'A collection of scripts that are useful for dealing with viral RNA NGS data')]))]), '4.1l': CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/SnpSift.html'), ('license', 'LGPLv3'), ('summary', 'Toolbox that allows you to filter and manipulate annotated files')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.1l')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_1l_core.zip'), ('md5', '43dd4b41f3223bbb4cc094ec5c5e2aac'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_1l_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['SnpSift 2>&1 > /dev/null | grep \"SnpSift version 4.1l\"'])]))]), '4.2': CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/SnpSift.html'), ('license', 'LGPLv3'), ('summary', 'Toolbox that allows you to filter and manipulate annotated files')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['SnpSift 2>&1 > /dev/null | grep \"SnpSift version 4.2\"'])]))]), '2.1b': CommentedMap([('package', CommentedMap([('name', 'sortmerna'), ('version', '2.1b')])), ('source', CommentedMap([('fn', '2.1b.tar.gz'), ('url', 'https://github.com/biocore/sortmerna/archive/2.1b.tar.gz'), ('sha256', 'b3d122776c323813971b35991cda21a2c2f3ce817daba68a4c4e09d4367c0abe')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['sortmerna --version', \"merge-paired-reads.sh  --help | grep 'merged-reads.fastq'\", \"unmerge-paired-reads.sh  --help | grep 'merged-reads-file.fastq'\"])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/RNA/sortmerna'), ('license', 'LGPL'), ('summary', 'SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.')]))]), '3.10.1': CommentedMap([('package', CommentedMap([('name', 'spades'), ('version', '3.10.1')])), ('source', CommentedMap([('fn', 'SPAdes-3.10.1-Darwin.tar.gz'), ('url', 'http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Darwin.tar.gz'), ('md5', 'f23bbad7921cff2c283e6589ccee83a2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['spades.py --test'])])), ('about', CommentedMap([('home', 'http://bioinf.spbau.ru/en/spades'), ('license', 'GPLv2'), ('summary', 'SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.')]))]), '3.9.0': CommentedMap([('package', CommentedMap([('name', 'spades'), ('version', '3.9.0')])), ('source', CommentedMap([('fn', 'SPAdes-3.9.0-Darwin.tar.gz'), ('url', 'http://spades.bioinf.spbau.ru/release3.9.0/SPAdes-3.9.0-Darwin.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['spades.py --test'])])), ('about', CommentedMap([('home', 'http://bioinf.spbau.ru/en/spades'), ('license', 'GPLv2'), ('summary', 'SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.')]))]), '0.1.4.6': CommentedMap([('package', CommentedMap([('name', 'srst2'), ('version', '0.1.4.6')])), ('source', CommentedMap([('fn', 'v0.1.6.tar.gz'), ('url', 'https://github.com/katholt/srst2/archive/v0.1.6.tar.gz'), ('sha256', '24cbd03933d4cc14bbfa7f8a073c9b0caafb13d55afb3bfa25f8b093fc5bc615'), ('patches', ['use_setuptools.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.7.1', 'scipy >=0.12.0']), ('run', ['python', 'numpy >=1.7.1', 'scipy >=0.12.0', 'bowtie2 <=2.2.4', 'samtools ==0.1.18'])])), ('test', CommentedMap([('commands', [\"srst2 -h 2>&1 | grep 'SRST2 - Short Read Sequence Typer' > /dev/null\", \"getmlst.py -h 2>&1 | grep 'usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/katholt/srst2'), ('license', 'BSD'), ('summary', 'Short Read Sequence Typing for Bacterial Pathogens')]))]), '1.37': CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.37')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.37.tar.gz'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.37.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['perl-threaded', 'velvet', 'libgcc'])])), ('test', CommentedMap([('commands', ['cstacks  2>&1 | grep cstacks', 'pstacks 2>&1 | grep pstacks', 'process_radtags 2>&1 | grep process_radtags', 'clone_filter 2>&1 | grep clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]), '1.40': CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.40')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.40.tar.gz'), ('md5', '6e37053d3a5fcbbe2b421be9624b19fa'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.40.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'samtools 0.1.19', 'sparsehash']), ('run', ['perl-threaded', 'velvet', 'libgcc', 'zlib', 'samtools 0.1.19', 'sparsehash'])])), ('test', CommentedMap([('commands', ['cstacks  2>&1 | grep cstacks', 'pstacks 2>&1 | grep pstacks', 'process_radtags 2>&1 | grep process_radtags', 'clone_filter 2>&1 | grep clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]), '1.43': CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.43')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.43.tar.gz'), ('md5', '46e00cb1248dd440d5004b6a2f39dc4d'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.43.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl-threaded', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]), '1.44': CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.44')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.44.tar.gz'), ('md5', '10e1c2cb3ed10bebcb035619d94ae7d3'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.44.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]), '0.97': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '0.97')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'stringtie-0.97.tar.gz'), ('sha256', '8a8371827a123a94333036d482cc45ccd39d6dd54f6d394fb536853265241d1d'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-0.97.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '1.0.4': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.0.4')])), ('requirements', CommentedMap([('build', []), ('run', [])])), ('source', CommentedMap([('fn', 'stringtie-1.0.4.tar.gz'), ('sha256', '635099d543bfaf0ec1c84020eb4aa3375714c12e2d0d435dae44901d49fe3ef2'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.0.4.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '1.2.4': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.2.4')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'stringtie-1.2.4.tar.gz'), ('sha256', '7fcd64c3ad73816cdb446c5d703a1c46cf828db88871221267532355168b0dc9'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.2.4.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '1.3.0': CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.3.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'stringtie-1.3.0.tar.gz'), ('sha256', '3a21c662904cdab0c749d5c3f5ef988e84fa036857ce2ca35782b3c19469c652'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.0.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]), '1.2.19': CommentedMap([('package', CommentedMap([('name', 'swarm'), ('version', '1.2.19')])), ('source', CommentedMap([('fn', '1.2.19.tar.gz'), ('url', 'https://github.com/torognes/swarm/archive/1.2.19.tar.gz'), ('md5', '4d147b9c94b8966760d0b737499a53d2')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['swarm_breaker.py --version', 'swarm -v 2>&1  | grep Swarm'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/swarm'), ('license', 'Affero GPL'), ('summary', 'A robust and fast clustering method for amplicon-based studies.')]))]), '0.2.5': CommentedMap([('package', CommentedMap([('name', 'tabix'), ('version', '0.2.5')])), ('source', CommentedMap([('url', 'https://downloads.sourceforge.net/project/samtools/tabix/tabix-0.2.5.tar.bz2'), ('sha256', '6ce0da41a7ef82aa34db413602cc7cd11c08d606f36ebba793da41ec402e2f86')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['zlib', 'gcc', 'llvm']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['test -x \"$PREFIX/bin/tabix\"', 'test -x \"$PREFIX/bin/bgzip\"'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/samtools'), ('license', 'BSD'), ('summary', 'A set of tools written in Perl and C++ for working with VCF files.')]))]), '3.04.01': CommentedMap([('package', CommentedMap([('name', 'tesseract'), ('version', '3.04.01')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '3.04.01.tar.gz'), ('url', 'https://github.com/tesseract-ocr/tesseract/archive/3.04.01.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'libpng', 'jpeg', 'libtiff', 'zlib', 'leptonica']), ('run', ['libgcc', 'libpng', 'jpeg', 'libtiff', 'zlib', 'leptonica'])])), ('test', CommentedMap([('commands', ['tesseract --help'])])), ('about', CommentedMap([('home', 'https://github.com/tesseract-ocr/tesseract'), ('license', 'Apache License, Version 2.0'), ('summary', 'OCR engine')]))]), '0.4.1': CommentedMap([('package', CommentedMap([('name', 'trim-galore'), ('version', '0.4.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'trim_galore_v0.4.1.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/trim_galore_v0.4.1.zip')])), ('requirements', CommentedMap([('build', None), ('run', ['perl-threaded', 'fastqc', 'cutadapt'])])), ('test', CommentedMap([('commands', ['trim_galore --help | grep trim_galore > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'), ('license', 'GPL'), ('summary', 'Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control')]))]), '1.4.1': CommentedMap([('package', CommentedMap([('name', 'trimal'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'v1.4.1.tar.gz'), ('url', 'https://github.com/scapella/trimal/archive/v1.4.1.tar.gz'), ('md5', 'cb193397635da45be350e9ac96dafe45')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['readal --version | grep readAl > /dev/null', 'statal --version | grep statAl > /dev/null', 'trimal --version | grep trimAl > /dev/null'])])), ('about', CommentedMap([('home', 'http://trimal.cgenomics.org'), ('summary', 'A tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment'), ('license', 'GNU General Public License v3 or later (GPLv3+)')]))]), '0.35': CommentedMap([('about', CommentedMap([('home', 'http://www.usadellab.org/cms/?page=trimmomatic'), ('license', 'GPLv3'), ('summary', 'A flexible read trimming tool for Illumina NGS data')])), ('package', CommentedMap([('name', 'trimmomatic'), ('version', '0.35')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('source', CommentedMap([('fn', 'Trimmomatic-0.35.zip'), ('md5', '564e62f4f0f9f56d22762b237d4c69c4'), ('url', 'http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.35.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['unzip -t $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/trimmomatic.jar &> /dev/null'])]))]), '2.1.1': CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq'), ('license', 'https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt'), ('summary', 'Trinity assembles transcript sequences from Illumina RNA-Seq data')])), ('build', CommentedMap([('number', 4), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'trinity'), ('version', 'date.2011_11_26')])), ('source', CommentedMap([('fn', 'trinityrnaseq_r2011-11-26.tgz'), ('md5', '3a4c4691908d37042ca63ca469c75fd3'), ('url', 'http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz'), ('patches', ['RunButterfly.cc.patch', 'Trinity.pl.patch', 'samtools_makefile.patch'])])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['perl-threaded', 'libgcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('extra', CommentedMap([('notes', 'Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)')])), ('test', CommentedMap([('commands', ['$PREFIX/bin/Trinity-test &> /dev/null'])]))]), '2.2.0': CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq/'), ('license', ''), ('summary', 'Trinity RNA-Seq de novo transcriptome assembly')])), ('build', CommentedMap([('number', 6), ('skip', True)])), ('package', CommentedMap([('name', 'trinity'), ('version', '2.2.0')])), ('requirements', CommentedMap([('build', ['libgcc', 'gcc', 'perl-app-cpanminus', 'perl-module-build', 'perl-file-find-rule', 'java-jdk', 'zlib', 'jellyfish 2.2.3', 'samtools 0.1.19', 'trimmomatic', 'collectl', 'fastool', 'parafly', 'slclust']), ('run', ['perl-app-cpanminus', 'perl-module-build', 'perl-file-find-rule', 'java-jdk', 'zlib', 'jellyfish 2.2.3', 'samtools 0.1.19', 'trimmomatic', 'bowtie', 'collectl', 'fastool', 'parafly', 'slclust'])])), ('source', CommentedMap([('fn', 'trinity-2.2.0.tar.gz'), ('url', 'https://github.com/trinityrnaseq/trinityrnaseq/archive/v2.2.0.tar.gz'), ('patches', ['makefile.clean.patch', 'trinity-plugins.makefile.clean.patch'])])), ('test', CommentedMap([('commands', ['Trinity --cite &> /dev/null', 'align_and_estimate_abundance.pl --help &> /dev/null || [[ \"$?\" == 255 ]]', 'abundance_estimates_to_matrix.pl --help &> /dev/null || [[ \"$?\" == 255 ]]', 'run_DE_analysis.pl -h &> /dev/null || [[ \"$?\" == 255 ]]', 'analyze_diff_expr.pl -h &> /dev/null || [[ \"$?\" == 255 ]]', 'PtR --help &> /dev/null || [[ \"$?\" == 255 ]]', 'TrinityStats.pl 2>&1 | grep \"usage:\" &> /dev/null', 'define_clusters_by_cutting_tree.pl 2>&1 | grep \"define K clusters via k-means algorithm\" &> /dev/null', 'contig_ExN50_statistic.pl 2>&1 | grep \"usage:\" &> /dev/null'])]))]), 'date.2011_11_26': CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq'), ('license', 'https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt'), ('summary', 'Trinity assembles transcript sequences from Illumina RNA-Seq data')])), ('build', CommentedMap([('number', 7), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'trinity'), ('version', 'date.2011_11_26')])), ('source', CommentedMap([('fn', 'trinityrnaseq_r2011-11-26.tgz'), ('md5', '3a4c4691908d37042ca63ca469c75fd3'), ('url', 'http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz'), ('patches', ['RunButterfly.cc.patch', 'Trinity.pl.patch', 'samtools_makefile.patch', 'Chrysalis_stacksize.patch', 'headerpad.patch'])])), ('requirements', CommentedMap([('build', ['perl', 'gcc', 'openjdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['perl', 'libgcc', 'openjdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('extra', CommentedMap([('notes', 'Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)')])), ('test', CommentedMap([('commands', ['Trinity-test &> /dev/null'])]))]), '2.3.7': CommentedMap([('about', CommentedMap([('home', 'http://dkoboldt.github.io/varscan/'), ('license', 'The Non-Profit Open Software License version 3.0 (NPOSL-3.0)'), ('summary', 'variant detection in massively parallel sequencing data')])), ('package', CommentedMap([('name', 'varscan'), ('version', '2.3.7')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'VarScan-2.3.7.jar'), ('url', 'http://downloads.sourceforge.net/project/varscan/VarScan.v2.3.7.jar'), ('md5', '9796ac2564ebf60103e874bc68a03524')])), ('requirements', CommentedMap([('build', None), ('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['varscan mpileup2cns --help'])]))]), '2.4.2': CommentedMap([('about', CommentedMap([('home', 'http://dkoboldt.github.io/varscan/'), ('license', 'The Non-Profit Open Software License version 3.0 (NPOSL-3.0)'), ('summary', 'variant detection in massively parallel sequencing data')])), ('package', CommentedMap([('name', 'varscan'), ('version', '2.4.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'VarScan-2.4.2.jar'), ('url', 'https://github.com/dkoboldt/varscan/raw/master/VarScan.v2.4.2.jar'), ('md5', '4b810741505a8145a7f8f9f6791bbacf')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['varscan mpileup2cns --help'])]))]), '0.1.10': CommentedMap([('package', CommentedMap([('name', 'vcftools'), ('version', '0.1.10')])), ('source', CommentedMap([('fn', 'vcftools-0.1.10.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.10.tar.gz'), ('sha256', '0680dd38dc41a9fdbb5ddb22d79e0314009e5e464a5bb7b6da4845cce0181a23')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'perl-vcftools-vcf ==0.797']), ('run', ['libgcc', 'zlib', 'perl-vcftools-vcf ==0.797'])])), ('test', CommentedMap([('commands', ['vcftools -h'])])), ('about', CommentedMap([('home', 'https://vcftools.github.io/'), ('license', 'LGPL'), ('summary', 'A set of tools written in Perl and C++ for working with VCF files.')]))]), '1.8.5': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '1.8.5')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'ViennaRNA-1.8.5.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Archive/ViennaRNA-1.8.5.tar.gz'), ('sha256', 'f4e2d94beaf77165e8321758e4ab0ad1c5d49879cefa12e48b07d09ed2d0ecf9'), ('patches', ['noinline.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['echo GGGUACATACGAUCCC | RNAfold'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Tools for RNA secondary structure prediction and comparison')]))]), '2.1.9': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.1.9')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.1.9.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/download.php?id=viennarna-2.1.9'), ('sha1', '4916031938404aabe3b7fe725e43b1b75fe9527e')])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Tools for RNA secondary structure prediction and comparison')]))]), '2.2.10': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.2.10')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.2.10.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.2.10.tar.gz'), ('sha256', '94f3c8e92a767d08b31f4eb896590fd702330e611036567f68c290c8b0eccc57'), ('patches', ['patch.makedocs'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl-threaded']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.2.7': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.2.7')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.2.7.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.2.7.tar.gz'), ('sha256', 'dae281ca6143b4379cccc31aa68ddeecd79e577ecedb881e3d45d28d4ae71d60')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAfold --version', 'RNAplfold --version'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.2.8': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.2.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.2.8.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.2.8.tar.gz'), ('sha256', '30c40b8da4635388da626a741edfd5173e5712703d7a2ef2905bf666731e302d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAfold --version', 'RNAplfold --version'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.2.9': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.2.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.2.9.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.2.9.tar.gz'), ('sha256', 'cf3806fdad51405ac537e3636f4fa6f56d6b3297841ab4736a84ddd8841d5108')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAfold --version', 'RNAplfold --version'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '2.3.3': CommentedMap([('package', CommentedMap([('name', 'viennarna'), ('version', '2.3.3')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'ViennaRNA-2.3.3.tar.gz'), ('url', 'http://www.tbi.univie.ac.at/RNA/packages/source/ViennaRNA-2.3.3.tar.gz'), ('sha256', 'cf92c05e54dff32c2135433b6ebaa5330c05de02a1ae8b7c3b7a865d42eea514')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'python >3']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['RNAalifold --version', 'RNAeval    --version', 'RNAfold    --version', 'RNAplfold  --version', 'RNAsubopt  --version', 'kinwalker  --help >& /dev/null'])])), ('about', CommentedMap([('home', 'http://www.tbi.univie.ac.at/RNA/'), ('license', 'custom'), ('license_file', 'COPYING'), ('summary', 'Vienna RNA package -- RNA secondary structure prediction and comparison')]))]), '0.98.0': CommentedMap([('package', CommentedMap([('name', 'w4mclassfilter'), ('version', '0.98.0')])), ('source', CommentedMap([('fn', 'w4mclassfilter_0.98.0.tar.gz'), ('url', 'https://github.com/HegemanLab/w4mclassfilter/releases/download/v0.98.0/w4mclassfilter_0.98.0.tar.gz'), ('md5', '2f3ef13570c3d033bd3e24ada5d52e71')])), ('build', CommentedMap([('number', 0), ('string', '0.98.0'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(w4mclassfilter)\"'])])), ('about', CommentedMap([('home', 'https://github.com/HegemanLab/w4mclassfilter'), ('license', 'MIT'), ('summary', 'Filter Workflow4Metabolomics dataMatrix, sampleMetadata, and variableMetadata files by sample-class, imputing zero for NA values and eliminating zero-variance rows and columns from the data-matrix. MIT Licence allows redistribution.')]))]), '0.98.1': CommentedMap([('package', CommentedMap([('name', 'w4mclassfilter'), ('version', '0.98.1')])), ('source', CommentedMap([('fn', 'w4mclassfilter_0.98.1.tar.gz'), ('url', 'https://github.com/HegemanLab/w4mclassfilter/releases/download/v0.98.1/w4mclassfilter_0.98.1.tar.gz'), ('md5', 'e25e7842d96bf25ff50ecea551321dea')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(w4mclassfilter)\"'])])), ('about', CommentedMap([('home', 'https://github.com/HegemanLab/w4mclassfilter'), ('license', 'MIT'), ('summary', 'Filter Workflow4Metabolomics dataMatrix, sampleMetadata, and variableMetadata files by sample-class, imputing zero for NA values and eliminating zero-variance rows and columns from the data-matrix. MIT Licence allows redistribution.')]))]), '0.98.2': CommentedMap([('package', CommentedMap([('name', 'w4mclassfilter'), ('version', '0.98.2')])), ('source', CommentedMap([('fn', 'w4mclassfilter_0.98.2.tar.gz'), ('url', 'https://github.com/HegemanLab/w4mclassfilter/releases/download/v0.98.2/w4mclassfilter_0.98.2.tar.gz'), ('md5', '8520732b9903cd2cee3be0cedd31db38')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(w4mclassfilter)\"'])])), ('about', CommentedMap([('home', 'https://github.com/HegemanLab/w4mclassfilter'), ('license', 'MIT'), ('summary', 'Filter Workflow4Metabolomics dataMatrix, sampleMetadata, and variableMetadata files by sample-class, imputing zero for NA values and eliminating zero-variance rows and columns from the data-matrix. MIT Licence allows redistribution.')]))])}\n",
      "meta.yaml\n",
      "biotools.yaml\n",
      "0.4\n",
      "0.5\n",
      "1.5.2\n",
      "1.9.0\n",
      "2.0.1-k128\n",
      "1.0.3\n",
      "2.4.0\n",
      "1.2.36\n",
      "3.11.14\n",
      "1.0\n",
      "1.1\n",
      "2.3.0\n",
      "0.2\n",
      "0.3\n",
      "0.7\n",
      "0.8\n",
      "37.10\n",
      "37.17\n",
      "0.6.2\n",
      "1.2\n",
      "1.3\n",
      "1.3.1\n",
      "2.16.2\n",
      "2.17.0\n",
      "2.19.1\n",
      "2.20.1\n",
      "2.22\n",
      "2.23.0\n",
      "2.25\n",
      "2.26.0\n",
      "0.7.0\n",
      "2.12.1\n",
      "2.8.2\n",
      "2.0.9\n",
      "2.2.12\n",
      "1.48.0\n",
      "3.12.1\n",
      "1.22.0\n",
      "3.28.10\n",
      "3.28.2\n",
      "3.3.0\n",
      "1.4.0\n",
      "2.13.0\n",
      "1.2.14\n",
      "0.8.11\n",
      "1.44.0\n",
      "1.46.0\n",
      "2.1.4\n",
      "2.1.5\n",
      "0.8.0\n",
      "2.2.21\n",
      "2.5.0\n",
      "2.2.22\n",
      "1.0.0\n",
      "2.2.4\n",
      "2.2.5\n",
      "0.9.1\n",
      "0.1.1\n",
      "0.5.9\n",
      "0.7.3a\n",
      "0.7.8\n",
      "1.3.2\n",
      "3.0\n",
      "1.2.2-b\n",
      "1.2.3\n",
      "1.1.0\n",
      "0.7.2\n",
      "1.0.6\n",
      "1.5.8.2\n",
      "1.5.9.1\n",
      "2.3.5\n",
      "2.3.6\n",
      "2.4.1\n",
      "3.12.3\n",
      "0.7.12\n",
      "0.8.22\n",
      "0.8.24\n",
      "0.8.26\n",
      "0.8.27\n",
      "0.8.28\n",
      "0.8.29\n",
      "0.8.30\n",
      "0.8.31\n",
      "0.8.36\n",
      "0.9.10\n",
      "0.10.0\n",
      "0.8.2\n",
      "0.9.0\n",
      "0.06\n",
      "1.1.2\n",
      "2.0.17\n",
      "3.1.3\n",
      "3.2\n",
      "0.0.4\n",
      "0.0.5\n",
      "0.0.6\n",
      "0.5.10a\n",
      "0.5.6\n",
      "0.5.8\n",
      "0.6.1\n",
      "0.10.1\n",
      "0.11.3\n",
      "2.1.8\n",
      "20141125\n",
      "20151222\n",
      "v2.3.4\n",
      "v2.4.0\n",
      "0.1.0\n",
      "0.2.0\n",
      "0.5.3\n",
      "0.5.4\n",
      "16.10.4\n",
      "16.10.8\n",
      "16.4.0\n",
      "16.7.10\n",
      "2.0\n",
      "3.5\n",
      "4.4.0\n",
      "5.0.0\n",
      "6.8.3\n",
      "9.07\n",
      "9.10\n",
      "2014.12.23\n",
      "2015.09.10\n",
      "2015.12.31\n",
      "2016.09.23\n",
      "2017.02.15\n",
      "2017.05.08\n",
      "0.50\n",
      "0.1.3\n",
      "1.3.20\n",
      "1.3.26\n",
      "2.3.2\n",
      "0.6.1.post1\n",
      "0.11.1\n",
      "0.9.9\n",
      "0.3.0\n",
      "0.3.2\n",
      "1.0.2\n",
      "1.5.3\n",
      "4.6.1\n",
      "1.0.7.2\n",
      "6.0.77\n",
      "7.0.91\n",
      "1.1.11\n",
      "2.2.3\n",
      "3.6.0\n",
      "1.4.5\n",
      "2.0.8\n",
      "2.3.1\n",
      "2.3.4\n",
      "0.10.5beta\n",
      "2.5\n",
      "2.6\n",
      "490\n",
      "638\n",
      "874\n",
      "1.73\n",
      "0.5.00\n",
      "3.16\n",
      "1.8.10\n",
      "1.8.11\n",
      "1.8.12\n",
      "1.8.7\n",
      "1.8.9\n",
      "1.9.1\n",
      "2.1.2\n",
      "7.305\n",
      "7.310\n",
      "2.6.0\n",
      "1.1.01\n",
      "3.4.1.1\n",
      "4.9.5\n",
      "4.9.6\n",
      "2.0.0.7\n",
      "2.0.0.8\n",
      "1.25.0\n",
      "1.36.1\n",
      "0.72\n",
      "0.73\n",
      "0.74\n",
      "0.4.0\n",
      "0.2.2\n",
      "0.2.3\n",
      "0.2.4\n",
      "0.3.1\n",
      "1.0.010\n",
      "1.2.10\n",
      "1.2.11\n",
      "0.21\n",
      "1.8.6\n",
      "2.1.0\n",
      "2.0.1\n",
      "4.9\n",
      "1.7.8\n",
      "1.6.5\n",
      "1.11.0\n",
      "1.13.3\n",
      "1.13.6\n",
      "1.14.4\n",
      "1.14.6\n",
      "1.15.0\n",
      "1.15.1\n",
      "1.16.0\n",
      "1.16.13\n",
      "1.16.3\n",
      "1.16.4\n",
      "0.96\n",
      "5.74\n",
      "1.48\n",
      "2.23\n",
      "0.27\n",
      "1.10\n",
      "2.2.1\n",
      "1.06\n",
      "0.19\n",
      "0.34\n",
      "1.38\n",
      "1.33\n",
      "2.140640\n",
      "5.5004\n",
      "0.11\n",
      "0.28\n",
      "2.161\n",
      "0.04\n",
      "2.01\n",
      "2.30\n",
      "1.42\n",
      "1.12\n",
      "2.09\n",
      "2.18\n",
      "5.88\n",
      "1.25\n",
      "2.88\n",
      "2.1\n",
      "0.14\n",
      "5.72\n",
      "7.24\n",
      "1.1006\n",
      "1.27\n",
      "2.12\n",
      "3.48_01\n",
      "0.2304\n",
      "0.91\n",
      "1.51\n",
      "1.11\n",
      "3.0.1\n",
      "3.1.1\n",
      "3.1.4\n",
      "3.2.0\n",
      "3.2.10\n",
      "3.2.11\n",
      "3.2.2\n",
      "3.2.4\n",
      "3.2.5\n",
      "3.2.6\n",
      "3.2.7\n",
      "3.2.8\n",
      "3.2.9\n",
      "0.0.3\n",
      "2.57\n",
      "2.58\n",
      "1.01\n",
      "2.064\n",
      "1.35\n",
      "2.111\n",
      "0.49\n",
      "0.63\n",
      "0.53\n",
      "1.03\n",
      "1.814\n",
      "3.13\n",
      "0.32\n",
      "0.62\n",
      "1.000019\n",
      "1.23\n",
      "2.000002\n",
      "2.79\n",
      "2.14\n",
      "1.014\n",
      "1.036\n",
      "0.38\n",
      "1.26\n",
      "0.23\n",
      "0.236\n",
      "0.076\n",
      "1.09\n",
      "0.18\n",
      "1.60\n",
      "0.100003\n",
      "1.07\n",
      "1.63\n",
      "2.07\n",
      "3.35\n",
      "3.28\n",
      "1.38_03\n",
      "2.000005\n",
      "2.37\n",
      "1.45\n",
      "0.36\n",
      "3.0609\n",
      "3.27\n",
      "0.05\n",
      "0.12\n",
      "0.0203\n",
      "3.30\n",
      "0.80\n",
      "0.25\n",
      "3.36\n",
      "1.001002\n",
      "1.302075\n",
      "0.1202\n",
      "0.48\n",
      "0.22\n",
      "2013.0523\n",
      "1.46\n",
      "1.05\n",
      "1.9726\n",
      "1.2300\n",
      "1.17\n",
      "1.71\n",
      "0.61\n",
      "0.797\n",
      "0.840\n",
      "1.126\n",
      "1.141\n",
      "1.97\n",
      "2.7.1\n",
      "2.9.2\n",
      "0.23.0\n",
      "0.34.1\n",
      "1.0.1\n",
      "0.5.1\n",
      "1.6.4\n",
      "1.1.4\n",
      "0.6.9\n",
      "0.1.11\n",
      "0.4.7.1\n",
      "0.4.8.1\n",
      "0.1.8\n",
      "0.7.5\n",
      "0.7.7\n",
      "1.2.2\n",
      "0.0.5.dev20160304220337\n",
      "0.8.3\n",
      "0.6.7\n",
      "1.10.4\n",
      "0.75_7\n",
      "0.61.0\n",
      "1.1.3\n",
      "1.2.1\n",
      "7.3.0\n",
      "8.2.9\n",
      "2015.09.2\n",
      "2016.03.3\n",
      "3.1.2\n",
      "3.2.1\n",
      "2.1.6\n",
      "5.7\n",
      "0.3.0.2\n",
      "0.3.1.0\n",
      "0.3.1.1\n",
      "1.9\n",
      "2.1.7\n",
      "0.1.12\n",
      "0.1.13\n",
      "0.1.14\n",
      "0.1.15\n",
      "0.1.16\n",
      "0.1.17\n",
      "0.1.18\n",
      "0.1.19\n",
      "0.9\n",
      "2.9.0\n",
      "3.2.13\n",
      "3.2.3\n",
      "1.2.0\n",
      "r75\n",
      "r82\n",
      "0.1.126\n",
      "0.1\n",
      "4.1l\n",
      "4.2\n",
      "2.1b\n",
      "3.10.1\n",
      "3.9.0\n",
      "0.1.4.6\n",
      "1.37\n",
      "1.40\n",
      "1.43\n",
      "1.44\n",
      "0.97\n",
      "1.0.4\n",
      "1.2.4\n",
      "1.3.0\n",
      "1.2.19\n",
      "0.2.5\n",
      "3.04.01\n",
      "0.4.1\n",
      "1.4.1\n",
      "0.35\n",
      "2.1.1\n",
      "2.2.0\n",
      "date.2011_11_26\n",
      "2.3.7\n",
      "2.4.2\n",
      "0.1.10\n",
      "1.8.5\n",
      "2.1.9\n",
      "2.2.10\n",
      "2.2.7\n",
      "2.2.8\n",
      "2.2.9\n",
      "2.3.3\n",
      "0.98.0\n",
      "0.98.1\n",
      "0.98.2\n"
     ]
    }
   ],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
