{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# overlap study for bioconda and biotools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## get list of biotools entries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "getting page ?page=1\n",
      "getting page ?page=2\n",
      "getting page ?page=3\n",
      "getting page ?page=4\n",
      "getting page ?page=5\n",
      "getting page ?page=6\n",
      "getting page ?page=7\n",
      "getting page ?page=8\n",
      "getting page ?page=9\n",
      "getting page ?page=10\n",
      "getting page ?page=11\n",
      "getting page ?page=12\n",
      "getting page ?page=13\n",
      "getting page ?page=14\n",
      "getting page ?page=15\n",
      "getting page ?page=16\n",
      "getting page ?page=17\n",
      "getting page ?page=18\n",
      "getting page ?page=19\n",
      "getting page ?page=20\n",
      "getting page ?page=21\n",
      "getting page ?page=22\n",
      "getting page ?page=23\n",
      "getting page ?page=24\n",
      "getting page ?page=25\n",
      "getting page ?page=26\n",
      "getting page ?page=27\n",
      "getting page ?page=28\n",
      "getting page ?page=29\n",
      "getting page ?page=30\n",
      "getting page ?page=31\n",
      "getting page ?page=32\n",
      "getting page ?page=33\n",
      "getting page ?page=34\n",
      "getting page ?page=35\n",
      "getting page ?page=36\n",
      "getting page ?page=37\n",
      "getting page ?page=38\n",
      "getting page ?page=39\n",
      "getting page ?page=40\n",
      "getting page ?page=41\n",
      "getting page ?page=42\n",
      "getting page ?page=43\n",
      "getting page ?page=44\n",
      "getting page ?page=45\n",
      "getting page ?page=46\n",
      "getting page ?page=47\n",
      "getting page ?page=48\n",
      "getting page ?page=49\n",
      "getting page ?page=50\n",
      "getting page ?page=51\n",
      "getting page ?page=52\n",
      "getting page ?page=53\n",
      "getting page ?page=54\n",
      "getting page ?page=55\n",
      "getting page ?page=56\n",
      "getting page ?page=57\n",
      "getting page ?page=58\n",
      "getting page ?page=59\n",
      "getting page ?page=60\n",
      "getting page ?page=61\n",
      "getting page ?page=62\n",
      "getting page ?page=63\n",
      "getting page ?page=64\n",
      "getting page ?page=65\n",
      "getting page ?page=66\n",
      "getting page ?page=67\n",
      "getting page ?page=68\n",
      "getting page ?page=69\n",
      "getting page ?page=70\n",
      "getting page ?page=71\n",
      "getting page ?page=72\n",
      "getting page ?page=73\n",
      "getting page ?page=74\n",
      "getting page ?page=75\n",
      "getting page ?page=76\n",
      "getting page ?page=77\n",
      "getting page ?page=78\n",
      "getting page ?page=79\n",
      "getting page ?page=80\n",
      "getting page ?page=81\n",
      "getting page ?page=82\n",
      "getting page ?page=83\n",
      "getting page ?page=84\n",
      "getting page ?page=85\n",
      "getting page ?page=86\n",
      "getting page ?page=87\n",
      "getting page ?page=88\n",
      "getting page ?page=89\n",
      "getting page ?page=90\n",
      "getting page ?page=91\n",
      "getting page ?page=92\n",
      "getting page ?page=93\n",
      "getting page ?page=94\n",
      "getting page ?page=95\n",
      "getting page ?page=96\n",
      "getting page ?page=97\n",
      "getting page ?page=98\n",
      "getting page ?page=99\n",
      "getting page ?page=100\n",
      "getting page ?page=101\n",
      "getting page ?page=102\n",
      "getting page ?page=103\n",
      "getting page ?page=104\n",
      "getting page ?page=105\n",
      "getting page ?page=106\n",
      "getting page ?page=107\n",
      "getting page ?page=108\n",
      "getting page ?page=109\n",
      "getting page ?page=110\n",
      "getting page ?page=111\n",
      "getting page ?page=112\n",
      "getting page ?page=113\n",
      "getting page ?page=114\n",
      "getting page ?page=115\n",
      "getting page ?page=116\n",
      "getting page ?page=117\n",
      "getting page ?page=118\n",
      "getting page ?page=119\n",
      "getting page ?page=120\n",
      "getting page ?page=121\n",
      "getting page ?page=122\n",
      "getting page ?page=123\n",
      "getting page ?page=124\n",
      "getting page ?page=125\n",
      "getting page ?page=126\n",
      "getting page ?page=127\n",
      "getting page ?page=128\n",
      "getting page ?page=129\n",
      "getting page ?page=130\n",
      "getting page ?page=131\n",
      "getting page ?page=132\n",
      "getting page ?page=133\n",
      "getting page ?page=134\n",
      "getting page ?page=135\n",
      "getting page ?page=136\n",
      "getting page ?page=137\n",
      "getting page ?page=138\n",
      "getting page ?page=139\n",
      "getting page ?page=140\n",
      "getting page ?page=141\n",
      "getting page ?page=142\n",
      "getting page ?page=143\n",
      "getting page ?page=144\n",
      "getting page ?page=145\n",
      "getting page ?page=146\n",
      "getting page ?page=147\n",
      "getting page ?page=148\n",
      "getting page ?page=149\n",
      "getting page ?page=150\n",
      "getting page ?page=151\n",
      "getting page ?page=152\n",
      "getting page ?page=153\n",
      "getting page ?page=154\n",
      "getting page ?page=155\n",
      "getting page ?page=156\n",
      "getting page ?page=157\n",
      "getting page ?page=158\n",
      "getting page ?page=159\n",
      "getting page ?page=160\n",
      "getting page ?page=161\n",
      "getting page ?page=162\n",
      "getting page ?page=163\n",
      "getting page ?page=164\n",
      "getting page ?page=165\n",
      "getting page ?page=166\n",
      "getting page ?page=167\n",
      "getting page ?page=168\n",
      "getting page ?page=169\n",
      "getting page ?page=170\n",
      "getting page ?page=171\n",
      "getting page ?page=172\n",
      "getting page ?page=173\n",
      "getting page ?page=174\n",
      "getting page ?page=175\n",
      "getting page ?page=176\n",
      "getting page ?page=177\n",
      "getting page ?page=178\n",
      "getting page ?page=179\n",
      "getting page ?page=180\n",
      "getting page ?page=181\n",
      "getting page ?page=182\n",
      "getting page ?page=183\n",
      "getting page ?page=184\n",
      "getting page ?page=185\n",
      "getting page ?page=186\n",
      "getting page ?page=187\n",
      "getting page ?page=188\n",
      "getting page ?page=189\n",
      "getting page ?page=190\n",
      "getting page ?page=191\n",
      "getting page ?page=192\n",
      "getting page ?page=193\n",
      "getting page ?page=194\n",
      "getting page ?page=195\n",
      "getting page ?page=196\n",
      "getting page ?page=197\n",
      "getting page ?page=198\n",
      "getting page ?page=199\n",
      "getting page ?page=200\n",
      "getting page ?page=201\n",
      "getting page ?page=202\n",
      "getting page ?page=203\n",
      "getting page ?page=204\n",
      "getting page ?page=205\n",
      "getting page ?page=206\n",
      "getting page ?page=207\n",
      "getting page ?page=208\n",
      "getting page ?page=209\n",
      "getting page ?page=210\n",
      "getting page ?page=211\n",
      "getting page ?page=212\n",
      "getting page ?page=213\n",
      "getting page ?page=214\n",
      "getting page ?page=215\n",
      "getting page ?page=216\n",
      "getting page ?page=217\n",
      "getting page ?page=218\n",
      "getting page ?page=219\n",
      "getting page ?page=220\n",
      "getting page ?page=221\n",
      "getting page ?page=222\n",
      "getting page ?page=223\n",
      "getting page ?page=224\n",
      "getting page ?page=225\n",
      "getting page ?page=226\n",
      "getting page ?page=227\n",
      "getting page ?page=228\n",
      "getting page ?page=229\n",
      "getting page ?page=230\n",
      "getting page ?page=231\n",
      "getting page ?page=232\n",
      "getting page ?page=233\n",
      "getting page ?page=234\n",
      "getting page ?page=235\n",
      "getting page ?page=236\n",
      "getting page ?page=237\n",
      "getting page ?page=238\n",
      "getting page ?page=239\n",
      "getting page ?page=240\n",
      "getting page ?page=241\n",
      "getting page ?page=242\n",
      "getting page ?page=243\n",
      "getting page ?page=244\n",
      "getting page ?page=245\n",
      "getting page ?page=246\n",
      "getting page ?page=247\n",
      "getting page ?page=248\n",
      "getting page ?page=249\n",
      "getting page ?page=250\n",
      "getting page ?page=251\n",
      "getting page ?page=252\n",
      "getting page ?page=253\n",
      "getting page ?page=254\n",
      "getting page ?page=255\n",
      "getting page ?page=256\n",
      "getting page ?page=257\n",
      "getting page ?page=258\n",
      "getting page ?page=259\n",
      "getting page ?page=260\n",
      "getting page ?page=261\n",
      "getting page ?page=262\n",
      "getting page ?page=263\n",
      "getting page ?page=264\n",
      "getting page ?page=265\n",
      "getting page ?page=266\n",
      "getting page ?page=267\n",
      "getting page ?page=268\n",
      "getting page ?page=269\n",
      "getting page ?page=270\n",
      "getting page ?page=271\n",
      "getting page ?page=272\n",
      "getting page ?page=273\n",
      "getting page ?page=274\n",
      "getting page ?page=275\n",
      "getting page ?page=276\n",
      "getting page ?page=277\n",
      "getting page ?page=278\n",
      "getting page ?page=279\n",
      "getting page ?page=280\n",
      "getting page ?page=281\n",
      "getting page ?page=282\n",
      "getting page ?page=283\n",
      "getting page ?page=284\n",
      "getting page ?page=285\n",
      "getting page ?page=286\n",
      "getting page ?page=287\n",
      "getting page ?page=288\n",
      "getting page ?page=289\n",
      "getting page ?page=290\n",
      "getting page ?page=291\n",
      "getting page ?page=292\n",
      "getting page ?page=293\n",
      "getting page ?page=294\n",
      "getting page ?page=295\n",
      "getting page ?page=296\n",
      "getting page ?page=297\n",
      "getting page ?page=298\n",
      "getting page ?page=299\n",
      "getting page ?page=300\n",
      "getting page ?page=301\n",
      "getting page ?page=302\n",
      "getting page ?page=303\n",
      "getting page ?page=304\n",
      "getting page ?page=305\n",
      "getting page ?page=306\n",
      "getting page ?page=307\n",
      "getting page ?page=308\n",
      "getting page ?page=309\n",
      "getting page ?page=310\n",
      "getting page ?page=311\n",
      "getting page ?page=312\n",
      "getting page ?page=313\n",
      "getting page ?page=314\n",
      "getting page ?page=315\n",
      "getting page ?page=316\n",
      "getting page ?page=317\n",
      "getting page ?page=318\n",
      "getting page ?page=319\n",
      "getting page ?page=320\n",
      "getting page ?page=321\n",
      "getting page ?page=322\n",
      "getting page ?page=323\n",
      "getting page ?page=324\n",
      "getting page ?page=325\n",
      "getting page ?page=326\n",
      "getting page ?page=327\n",
      "getting page ?page=328\n",
      "getting page ?page=329\n",
      "getting page ?page=330\n",
      "getting page ?page=331\n",
      "getting page ?page=332\n",
      "getting page ?page=333\n",
      "getting page ?page=334\n",
      "getting page ?page=335\n",
      "getting page ?page=336\n",
      "getting page ?page=337\n",
      "getting page ?page=338\n",
      "getting page ?page=339\n",
      "getting page ?page=340\n",
      "getting page ?page=341\n",
      "getting page ?page=342\n",
      "getting page ?page=343\n",
      "getting page ?page=344\n",
      "getting page ?page=345\n",
      "getting page ?page=346\n",
      "getting page ?page=347\n",
      "getting page ?page=348\n",
      "getting page ?page=349\n",
      "getting page ?page=350\n",
      "getting page ?page=351\n",
      "getting page ?page=352\n",
      "getting page ?page=353\n",
      "getting page ?page=354\n",
      "getting page ?page=355\n",
      "getting page ?page=356\n",
      "getting page ?page=357\n",
      "getting page ?page=358\n",
      "getting page ?page=359\n",
      "getting page ?page=360\n",
      "getting page ?page=361\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "getting page ?page=362\n",
      "getting page ?page=363\n",
      "getting page ?page=364\n",
      "getting page ?page=365\n",
      "getting page ?page=366\n",
      "getting page ?page=367\n",
      "getting page ?page=368\n",
      "getting page ?page=369\n",
      "getting page ?page=370\n",
      "getting page ?page=371\n",
      "getting page ?page=372\n",
      "getting page ?page=373\n",
      "getting page ?page=374\n",
      "getting page ?page=375\n",
      "getting page ?page=376\n",
      "getting page ?page=377\n",
      "getting page ?page=378\n",
      "getting page ?page=379\n",
      "getting page ?page=380\n",
      "getting page ?page=381\n",
      "getting page ?page=382\n",
      "getting page ?page=383\n",
      "getting page ?page=384\n",
      "getting page ?page=385\n",
      "getting page ?page=386\n",
      "getting page ?page=387\n",
      "getting page ?page=388\n",
      "getting page ?page=389\n",
      "getting page ?page=390\n",
      "getting page ?page=391\n",
      "getting page ?page=392\n",
      "getting page ?page=393\n",
      "getting page ?page=394\n",
      "getting page ?page=395\n",
      "getting page ?page=396\n",
      "getting page ?page=397\n",
      "getting page ?page=398\n",
      "getting page ?page=399\n",
      "getting page ?page=400\n",
      "getting page ?page=401\n",
      "getting page ?page=402\n",
      "getting page ?page=403\n",
      "getting page ?page=404\n",
      "getting page ?page=405\n",
      "getting page ?page=406\n",
      "getting page ?page=407\n",
      "getting page ?page=408\n",
      "getting page ?page=409\n",
      "getting page ?page=410\n",
      "getting page ?page=411\n",
      "getting page ?page=412\n",
      "getting page ?page=413\n",
      "getting page ?page=414\n",
      "getting page ?page=415\n",
      "getting page ?page=416\n",
      "getting page ?page=417\n",
      "getting page ?page=418\n",
      "getting page ?page=419\n",
      "getting page ?page=420\n",
      "getting page ?page=421\n",
      "getting page ?page=422\n",
      "getting page ?page=423\n",
      "getting page ?page=424\n",
      "getting page ?page=425\n",
      "getting page ?page=426\n",
      "getting page ?page=427\n",
      "getting page ?page=428\n",
      "getting page ?page=429\n",
      "getting page ?page=430\n",
      "getting page ?page=431\n",
      "getting page ?page=432\n",
      "getting page ?page=433\n",
      "getting page ?page=434\n",
      "getting page ?page=435\n",
      "getting page ?page=436\n",
      "getting page ?page=437\n",
      "getting page ?page=438\n",
      "getting page ?page=439\n",
      "getting page ?page=440\n",
      "getting page ?page=441\n",
      "getting page ?page=442\n",
      "getting page ?page=443\n",
      "getting page ?page=444\n",
      "getting page ?page=445\n",
      "getting page ?page=446\n",
      "getting page ?page=447\n",
      "getting page ?page=448\n",
      "getting page ?page=449\n",
      "getting page ?page=450\n",
      "getting page ?page=451\n",
      "getting page ?page=452\n",
      "getting page ?page=453\n",
      "getting page ?page=454\n",
      "getting page ?page=455\n",
      "getting page ?page=456\n",
      "getting page ?page=457\n",
      "getting page ?page=458\n",
      "getting page ?page=459\n",
      "getting page ?page=460\n",
      "getting page ?page=461\n",
      "getting page ?page=462\n",
      "getting page ?page=463\n",
      "getting page ?page=464\n",
      "getting page ?page=465\n",
      "getting page ?page=466\n",
      "getting page ?page=467\n",
      "getting page ?page=468\n",
      "getting page ?page=469\n",
      "getting page ?page=470\n",
      "getting page ?page=471\n",
      "getting page ?page=472\n",
      "getting page ?page=473\n",
      "getting page ?page=474\n",
      "getting page ?page=475\n",
      "getting page ?page=476\n",
      "getting page ?page=477\n",
      "getting page ?page=478\n",
      "getting page ?page=479\n",
      "getting page ?page=480\n",
      "getting page ?page=481\n",
      "getting page ?page=482\n",
      "getting page ?page=483\n",
      "getting page ?page=484\n",
      "getting page ?page=485\n",
      "getting page ?page=486\n",
      "getting page ?page=487\n",
      "getting page ?page=488\n",
      "getting page ?page=489\n",
      "getting page ?page=490\n",
      "getting page ?page=491\n",
      "getting page ?page=492\n",
      "getting page ?page=493\n",
      "getting page ?page=494\n",
      "getting page ?page=495\n",
      "getting page ?page=496\n",
      "getting page ?page=497\n",
      "getting page ?page=498\n",
      "getting page ?page=499\n",
      "getting page ?page=500\n",
      "getting page ?page=501\n",
      "getting page ?page=502\n",
      "getting page ?page=503\n",
      "getting page ?page=504\n",
      "getting page ?page=505\n",
      "getting page ?page=506\n",
      "getting page ?page=507\n",
      "getting page ?page=508\n",
      "getting page ?page=509\n",
      "getting page ?page=510\n",
      "getting page ?page=511\n",
      "getting page ?page=512\n",
      "getting page ?page=513\n",
      "getting page ?page=514\n",
      "getting page ?page=515\n",
      "getting page ?page=516\n",
      "getting page ?page=517\n",
      "getting page ?page=518\n",
      "getting page ?page=519\n",
      "getting page ?page=520\n",
      "getting page ?page=521\n",
      "getting page ?page=522\n",
      "getting page ?page=523\n",
      "getting page ?page=524\n",
      "getting page ?page=525\n",
      "getting page ?page=526\n",
      "getting page ?page=527\n",
      "getting page ?page=528\n",
      "getting page ?page=529\n",
      "getting page ?page=530\n",
      "getting page ?page=531\n",
      "getting page ?page=532\n",
      "getting page ?page=533\n",
      "getting page ?page=534\n",
      "getting page ?page=535\n",
      "getting page ?page=536\n",
      "getting page ?page=537\n",
      "getting page ?page=538\n",
      "getting page ?page=539\n",
      "getting page ?page=540\n",
      "getting page ?page=541\n",
      "getting page ?page=542\n",
      "getting page ?page=543\n",
      "getting page ?page=544\n",
      "getting page ?page=545\n",
      "getting page ?page=546\n",
      "getting page ?page=547\n",
      "getting page ?page=548\n",
      "getting page ?page=549\n",
      "getting page ?page=550\n",
      "getting page ?page=551\n",
      "getting page ?page=552\n",
      "getting page ?page=553\n",
      "getting page ?page=554\n",
      "getting page ?page=555\n",
      "getting page ?page=556\n",
      "getting page ?page=557\n",
      "getting page ?page=558\n",
      "getting page ?page=559\n",
      "getting page ?page=560\n",
      "getting page ?page=561\n",
      "getting page ?page=562\n",
      "getting page ?page=563\n",
      "getting page ?page=564\n",
      "getting page ?page=565\n",
      "getting page ?page=566\n",
      "getting page ?page=567\n",
      "getting page ?page=568\n",
      "getting page ?page=569\n",
      "getting page ?page=570\n",
      "getting page ?page=571\n",
      "getting page ?page=572\n",
      "getting page ?page=573\n",
      "getting page ?page=574\n",
      "getting page ?page=575\n",
      "getting page ?page=576\n",
      "getting page ?page=577\n",
      "getting page ?page=578\n",
      "getting page ?page=579\n",
      "getting page ?page=580\n",
      "getting page ?page=581\n",
      "getting page ?page=582\n",
      "getting page ?page=583\n",
      "getting page ?page=584\n",
      "getting page ?page=585\n",
      "getting page ?page=586\n",
      "getting page ?page=587\n",
      "getting page ?page=588\n",
      "getting page ?page=589\n",
      "getting page ?page=590\n",
      "getting page ?page=591\n",
      "getting page ?page=592\n",
      "getting page ?page=593\n",
      "getting page ?page=594\n",
      "getting page ?page=595\n",
      "getting page ?page=596\n",
      "getting page ?page=597\n",
      "getting page ?page=598\n",
      "getting page ?page=599\n",
      "getting page ?page=600\n",
      "getting page ?page=601\n",
      "getting page ?page=602\n",
      "getting page ?page=603\n",
      "getting page ?page=604\n",
      "getting page ?page=605\n",
      "getting page ?page=606\n",
      "getting page ?page=607\n",
      "getting page ?page=608\n",
      "getting page ?page=609\n",
      "getting page ?page=610\n",
      "getting page ?page=611\n",
      "getting page ?page=612\n",
      "getting page ?page=613\n",
      "getting page ?page=614\n",
      "getting page ?page=615\n",
      "getting page ?page=616\n",
      "getting page ?page=617\n",
      "getting page ?page=618\n",
      "getting page ?page=619\n",
      "getting page ?page=620\n",
      "getting page ?page=621\n",
      "getting page ?page=622\n",
      "getting page ?page=623\n",
      "getting page ?page=624\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "biotools_list = []\n",
    "next_biotools_page = '?page=1'\n",
    "while next_biotools_page is not None:\n",
    "    print('getting page %s' % next_biotools_page)\n",
    "    page = requests.get('https://bio.tools/api/tool/' + next_biotools_page).json()\n",
    "    biotools_list += page['list']\n",
    "    next_biotools_page = page['next']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "_and get the list of biotools ids for these entries_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['TPPRED_2.0', 'PROP', 'SNP2TFBS', 'BIOMET_TOOLBOX', 'CCP4MG', 'CCPN', 'CONSENSUS', 'DATABASE_FOR_BACTERIAL_GROUP_2_INTRONS', 'DIANA-MICROT', 'FREIBURG_RNA_TOOLS', 'VEP', 'BIOSEMANTICS_SUITE', 'SABIO-RK_WS', 'ROSETTADOCK_SERVER', 'INSTITUTE_OF_ENZYMOLOGY_SERVERS', 'GODMD', 'PCIT_R_PACKAGE', 'REPEAT_EXPLORER', 'PYMUT', 'POTAMOS', 'JD2STAT', 'AFNETWORK', 'COCIN', 'NETSTRAT', 'BOOLSPACE', 'MCL-CAW', 'MUTEXSL2832', 'CONTOURV2', 'CONTOURV1', 'PCP', 'MUTEXSL3761', 'MUTEXSL', 'EDGAR', 'DISEASE_ONTOLOGY', 'VISTA', 'ESEFINDER', 'YABI', 'PEREGRINE', 'EXOMEDEPTH', 'UMD-PREDICTOR', 'DVD', 'DBGAP', 'SEQR', 'GENOMATIX_GENEGRID', 'RDRF', 'OSSE', 'ICD-10', 'NCBI_RESOURCES', 'GATK', 'MUTALYZER', 'DELLY2', 'GA4GH_BEACON', 'FIREPROT', 'HAPLOPAINTER', 'INVFEST', 'KNOWLEDGE.BIO_LWAS', 'PATIENT_ARCHIVE', 'DBMAE', 'LINKDATAGEN', 'ELECTRONIC_PHARMACOGENOMICS_ASSISTANT_EPGA', 'CENTOMD', 'MUTATION_TASTER8065', 'UNIVERSAL_MUTATION_DATABASE_UMD', 'CONDEL', 'VAT', 'SCALEUS', 'GNOMAD', 'EGL_GENETICS_EMVCLASS', 'CLINGEN_PATHOGENICITY_CALCULATOR', 'GENEIMPRINT', 'MIT_SPLICE_PREDICTOR', 'MYPROTEINNET', 'EXOME_VARIANT_SERVER_ESP', 'REDIPORTAL', 'VALTRENDSDB', 'HOTSPOT_WIZARD', 'CHANNELSDB', 'PATTERNEXPLORER', 'ATOMICCHARGECALCULATOR', 'MOLE_PYMOL_PLUGIN', 'MOLE', 'BEGDB', 'MSVIZ', 'HUDSEN_ATLAS_AND_SPATIAL_GENE_EXPRESSION_DATABASE', 'MSVIZ1435', 'MSVIZ1188', 'COORDINATESERVER', 'DENSITYSERVER', 'LITEMOL', 'PATTERNQUERY', 'PROTEOSIGN', 'BIOTEXTQUEST', 'MICROTSS', 'DIANA-LNCBASE', 'DIANA-MICROT7101', 'DIANA-MIRPATH', 'DIANA-MIRGEN', 'DIANA-MIREXTRA1333', 'DIANA-MIREXTRA', 'EASYFRAP9342', 'PICKLE', 'LIFEWATCHGREECE_MICROCT_VLAB', 'TRES', 'MIGA', '3DMEM-ENZYME', 'PCASUITE', 'FLEXSERV', 'MODEL', 'MDDMD', 'BUFET', 'BIP_FINDER', 'MR-MICROT', 'EDAL_-_ELECTRONIC_DATA_ARCHIVE_LIBRARY', 'ARTICLE-BASED_PUBMED_SEARCH_ENGINE_APSE', 'MOLEONLINE', 'TRITON', 'DOLCE', 'NEEMP', 'INTAA', 'DECRYPTOR', 'HANA', 'HERVD', 'MADORA', 'VALIDATORDB', 'DNATCO', 'SITEBINDER', 'HCVIVDB', 'IRESITE', 'RPREDICTOR', 'PROTEINCUTTER', 'ABEEM_SOLVER', 'EEM_SOLVER', 'MOTIVEVALIDATOR', 'IPK_BLAST', 'MESBL_GC-MS_METABOLITE_PEAK_DATABASE', 'IOM', 'SWICZ', 'SETTER', 'BIOSPEAN', 'METADYN_VIEW', 'DNAS', 'M-IOLITE', 'SCIPION', 'GPROTPRED', 'AMYL-PRED_2', 'MUTATIONS-META-ANALYSER', 'HAPLOTYPEANALYSISPIPELINE', 'BPM', 'ALIGN-PATHS', 'G-CLASS', 'TRIFORC_DATABASE', 'SCILUIGI', 'AGOTOOL7657', 'AGOTOOL', 'NORMALYZER', 'PSEUDOMONAS_GENOME_PROJECT', 'PSEUDOMONAS_GENOME_DATABASE', 'PRANK_API', 'MEDLINE_RANKER', 'EFETCH_SNP', 'EFETCH_SEQUENCE', 'EFETCH_TAXON', 'EFETCH_PMC', 'EFETCH_GENE', 'EBI_TOOLS_PORTFOLIO', 'DICTYBASE_UPDATE_2011', 'CPHMODELS-3.0', 'CHEBI_WS', 'CCP4', 'BAR-PLUS', 'MAFFT', 'I-MUTANT_SUITE', 'GXA', 'DEEPLOC', 'CHEMBL_API', 'BIOMODELS', 'BIOMART', 'HMMER_CLOUDIFB', 'KNIME', 'BTOD_BLAST', 'STRINGAPP', 'METARANKER', 'MSA', 'bioconductor-MUSCLE', 'MUSCLE', 'NCBI_TAXONOMY_DATABASE', 'ORANGE', 'PDB', 'PFAM', 'PHENOMECENTRAL', 'WISE2DBA', 'SEQCLUSTER', 'GPCRDB', 'MATCHER-EBI', 'PDBEXPLORE', 'SIMAP', 'NETPLANTGENE', 'NETGENE2', 'NETASPGENE', 'PINTRON', 'NETUTR', 'ASPIC', 'CLICK', 'DALI', 'META-BASIC', 'PATSEARCH', 'GENATLASDB', 'FINDZEBRA', 'PILELINE', 'POMBASE', 'UNDERIICRMS', 'APPRIS', 'QMSAT', 'MTDFREML', 'YRGATE', 'X-MATE', 'W2RAP-CONTIGGER', 'VIZSTRUCT', 'VCF2ARLEQUIN', 'STACKS', 'SRA_TOOLKIT', 'SPADES', 'SOAPSPLICE', 'SOAPALIGNER', 'SOAP3', 'SMALT', 'SICKLE', 'SEQTK', 'SEQASSIST', 'SABRE', 'REPEATMASKER', 'PYSAM', 'PYFAIDX', 'PROVEAN', 'POLYMARKER', 'PLATYPUS', 'PENNCNV', 'PENNCNV2', 'PEDSYS', 'PAVIAN', 'OMIA', 'OCSESTDB', 'NGS-SNP', 'NANOOK', 'MUTATIONTASTER', 'MSCENTIPEDE', 'MIKADO', 'METHYLHMM', 'MAQVIEW', 'MAPP', 'MAKER-P', 'MACH2DAT', 'LME4', 'LIBMAUS', 'KALLISTO', 'ICARUS', 'HTSLIB', 'HISAT2', 'GVISTA', 'ZPLAN', 'XP-EHH', 'WOMBAT', 'WIGWAMS', 'WHAMM', 'WIGGLER', 'WAVESEQR', 'WAVEQTL', 'VCE', 'TRF', 'UNCCOMBO', 'TREEMIX', 'TASSEL', 'TREELD', 'SWEMBL', 'SWEEPFINDER', 'SWEED', 'SWEEP', 'SURVIVAL_KIT', 'STRINGTIE', 'STRUCTURE', 'SOLAR-ECLIPSE', 'SOAPSV', 'SOAPINDEL', 'SNPTEST', 'SNPDB', 'SNPSIFT', 'SNMF', 'SINGLE_CELL_GENOTYPER_SCG', 'SKAT', 'SHOTGUN', 'SHAPEIT', 'SELSIM', 'SELSCAN', 'SAMSA', 'SELACTION', 'REPEATMODELER', 'RAMPART', 'R-QTL', 'QFAM', 'QTDT', 'QCTOOL', 'Q', 'PYSAMSTATS', 'PYFASTA', 'PSMC', 'PLINK', 'PLINK-SEQ', 'PHYLOSIFT', 'PHASEBOOK', 'PHASE', 'PEST', 'PEDIG', 'PEAR', 'PBAT', 'ORYZABASE', 'OPERA-LG', 'OMICSDI', 'NOVOALIGNCS', 'NOVOALIGN', 'NGS-QC_GENERATOR', 'NEAT-GENREADS', 'MVBIMBAM', 'MSMS', 'MSMC', 'MSHOT', 'MS', 'MOODS', 'MONOVAR', 'MIXBLUP', 'MIX99', 'MINIMAC3', 'METASKAT', 'METAQUAST', 'METAPHLAN', 'METAL', 'MENDEL', 'MERLIN', 'MEGAN6', 'MARGE', 'MALDSOFT', 'MACH2QTL', 'MACH', 'MACH-ADMIX', 'LINKAGE', 'LDSC', 'KAT', 'JIGSAW', 'IMPUTE2', 'IHS', 'IMETER', 'IGVTOOLS', 'IBDSEQ', 'HOTSPOTTER', 'HAPLOREC', 'HAPLOVIEW', 'HADOOPCNV', 'GWAPP', 'GTOOL', 'GPLINK', 'GLOBETROTTER', 'GLOGS', 'GLIMMER-HMM', 'GFFUTILS', 'GFFREAD', 'GFFCOMPARE', 'GAEVAL', 'GINKGO', 'GENOMETHREADER', 'GENOMESTUDIO', 'GENOME_STRIP', 'GEMINI', 'GENABEL', 'GCTA', 'GAPIT', 'FIXSEQ', 'FIMPUTE', 'FINESTRUCTURE', 'FBAT', 'FASTSTRUCTURE', 'FASTPHASE', 'FASTSKAT', 'FASTOOL', 'FAMFLM', 'F-SEQ', 'EQTLBMA', 'EPIDB', 'EMMAX', 'EMLRT', 'EIGENSOFT', 'EDGE-PRO', 'EAGLE', 'DUPHMM', 'DMU', 'DISSCO', 'DICAL', 'DIAMUND', 'DIAMOND', 'DENDROSCOPE3', 'DBVAR', 'DAD-IS', 'CUTADAPT_1.12', 'CRI-MAP', 'CRAMTOOLS', 'CR-EST', 'COME', 'CLUMPAK', 'CHROMOPAINTER', 'CGQTL', 'CENTRIFUGE', 'CCAT', 'BRIDGER', 'BRACKEN', 'BLUPF90', 'BLIMP', 'BIOBAMBAM2', 'BIMBAM', 'BETASEQ', 'BEDOPS', 'BEAGLE', 'BC_SNPMAX', 'BAYZ', 'BAMUTIL', 'ASREML', 'ARLEQUIN', 'ARAPORT', 'APPLES', 'AMPHORA', 'ALPHAIMPUTE', 'ALLPATHS-LG', 'ALDER', 'AEQUATUS', 'ADMIXTURE', 'ADMIXTOOLS', 'ADAM', 'ABCD', 'RING', 'METASEQR', 'ZT', 'bioconductor-ZLIBBIOC', 'ZLIBBIOC', 'ZOOMA', 'ZORRO', 'ZOOM', 'XNAT.BMIA.NL', 'YINOYANG', 'bioconductor-XMLGET', 'bioconductor-YANK', 'bioconductor-YARN', 'YARN', 'bioconductor-YAQCAFFY', 'YAQCAFFY', 'YMF_3.0', 'YPA', 'YOGY', 'ZINCPHARMER', 'YEASTGFP', 'XMIPP', 'XPRESS', 'bioconductor-XPS', 'XPS', 'XPLORMED', 'YEAST_GENE_ORDER_BROWSER', 'XPLORE3D', 'Y.A.D.A.', 'ZINBA', 'XTANDEM', 'XTANDEM_PARSER', 'bioconductor-YAPSA', 'YAPSA', 'XMATCHVIEW', 'bioconductor-XVECTOR', 'XVECTOR', 'ZINC_FINGER_TOOLS', 'XREI', 'YEASTRACT', 'XPLATCOM', 'ZIPHMM', 'bioconductor-XMLTEXT', 'bioconductor-YAMSS', 'YAMSS', 'XTANDEMPIPELINE', 'YASS', 'YLOC', 'ZIFIT', 'XXMOTIF', 'XDS', 'XIA2', 'WORDMATCH_WS_HUSAR', 'XHEINZ', 'XILMASS', 'WRITERESTOHTML-IP', 'WOLF_PSORT', 'WORDSPY', 'WORFDB', 'WORLD-2DPAGE', 'bioconductor-WOSSDATA', 'bioconductor-WORDCOUNT', 'bioconductor-WOSSINPUT', 'XLINK', 'WWLIGCSRRE', 'WSDBFETCH', 'WSSAS', 'bioconductor-WORDMATCH', 'bioconductor-WOSSOPERATION', 'WORMS', 'WSPREDICTION', 'X_HUNTER', 'bioconductor-XCMS', 'XCMS', 'WURST', 'WS_SNPS_AND_GO', 'XAPERONES', 'bioconductor-WOSSNAME', 'bioconductor-WORDFINDER', 'WSGOBIOSPACE', 'bioconductor-WOSSPARAM', 'XCOMB', 'bioconductor-XDE', 'XDE', 'WORMBASE', 'XEML-LAB', 'bioconductor-WOSSTOPIC', 'bioconductor-WOSSOUTPUT', 'WSLIBRARYSEARCH', 'WHAT_CHECK', 'bioconductor-WHICHDB', 'bioconductor-WOBBLE', 'WHETS', 'WEBMAM', 'WEBPRC', 'WEBMOTIFS', 'WHEAT_GENOME', 'WHAM', 'WENMR_GROMACS', 'WEBLOGO', 'WHATSHAP', 'WEBINTERVIEWER', 'WEBGESTER_DB', 'WHAT_IF', 'WIKIPATHWAYS', 'WHATIZIT', 'WEGO', 'WEBPIPSA', 'WEBLAB', 'WEBMICROSCOPE', 'WINCOOT', 'WENMR', 'WEBNMA', 'WEBPRINSES', 'bioconductor-WIDGETTOOLS', 'WIDGETTOOLS', 'WEBTRACEMINER', 'WIKIGENE', 'WHICHGENES', 'WEBGMAP', 'WEEDERH', 'WEEDER', 'WEP', 'WEBPRANK', 'WHATIF', 'WIFF2DTA', 'WGCNA', 'WEBPROANALYST', 'WATER_WS_HUSAR', 'bioconductor-WATER', 'VMATCH', 'bioconductor-VTPNET', 'VTPNET', 'bioconductor-WEAVER', 'WEAVER', 'bioconductor-WATERMELON', 'WATERMELON', 'WEBFR3D', 'WDAC', 'VISHIC', 'VLDP', 'WAR', 'VIRUSMENTHA', 'VOID_EDITOR', 'bioconductor-WAVETILING', 'WAVETILING', 'VIRUS_GENOTYPING_TOOLS', 'VOGDB', 'VIRUSHUNTER', 'VIRUSSEQ', 'VISSR', 'bioconductor-WEBBIOC', 'WEBBIOC', 'WAVIS', 'WEB_WEEDER', 'WAVICGH', 'VISIBIOWEB', 'VKCDB', 'WEBGESTALT', 'WEBAPOLLO', 'WEBFEATURE', 'VOMBAT', 'VND', 'WEB_3DNA', 'VISANT', 'WEBAUGUSTUS', 'VIRUSMINT', 'WATER-EBI', 'VIRUS_DATABASE_VIDA', 'VITRAM', 'VSEARCH', 'WATER_API-EBI', 'VIRVARSEQ', 'bioconductor-VSN', 'VSN', 'WEB-THERMODYN', 'VELVET_CLOUDIFB', 'VIPR', 'VIRALMBD', 'VIRGO', 'VIENNA_RNA_WEBSUITE', 'VHLDB', 'VECTORBASE', 'VICTOR', 'VIRALZONE', 'VESPUCCI', 'bioconductor-VECTORSTRIP', 'VIRTUAL_FOLDER_FOR_STRUCTURAL_BIOLOGY_PROJECTS', 'VCFUTILS-IP', 'VICUNA', 'bioconductor-VIPER', 'VIPER', 'VIGOR', 'VDJSOLVER', 'VENN_DIAGRAMS', 'VCFHEAD', 'VELVETOPTIMISER', 'VIPERDB', 'VEMS', 'VCFLIB', 'VCFTOOLS', 'VELVET', 'VIRALORFEOME', 'VERIFYBAMID', 'VIP', 'VIRTUAL_RIBOSOME', 'VIENNA-PTM', 'VCFQFILTER-IP', 'VECTORFRIENDS', 'VIRALFUSIONSEQ', 'VIENNA_RNA_PACKAGE', 'VIRHOSTNET', 'UTRDB', 'VARAN', 'VCF_CONSENSUS-IP', 'VCF_INTERSECT-IP', 'VCF_SUBSTRACT-IP', 'VCFFORMATANDMERGE-IP', 'UNOSEQ', 'USEQ', 'VARIANTCLASSIFIER', 'bioconductor-VANILLAICE', 'VANILLAICE', 'bioconductor-VARIANTTOOLS', 'VARIANTTOOLS', 'VARAFT', 'UTRSITE', 'bioconductor-VARIATIONGET', 'VARIANT', 'bioconductor-VARIANCEPARTITION', 'VARIANCEPARTITION', 'bioconductor-USORT', 'USORT', 'VARIATION_TOOLKIT', 'VARIATIONHUNTER', 'VCAKE', 'VARIOWATCH', 'VAMPIRE', 'UNITRAP', 'VCFFILTERJS', 'bioconductor-URLGET', 'VARSCAN', 'VALIDICHRO', 'bioconductor-VARIANTANNOTATION', 'VARIANTANNOTATION', 'VAAL', 'VAAST', 'VARDOM', 'UNITE_RDNA', 'VCF_REMOVE_FILTERED-IP', 'bioconductor-VARIANTFILTERING', 'VARIANTFILTERING', 'V-XTRACTOR', 'bioconductor-VBMP', 'VBMP', 'VADAR', 'UPF', 'UNIREF', 'UNIGENE', 'UCSC_EBOLA_GENOME_PORTAL', 'UNICARB-DB', 'UM-PPS', 'UNDERLYINGAPPROACH', 'UNICARBKB', 'bioconductor-UNION', 'UNIQUEPROT', 'UNIPROT_ALIGN', 'UNIPATHWAY', 'ULTRANET', 'UCL_WEB_SERVICES', 'bioconductor-UNIQUORN', 'UNIQUORN', 'UNIPROT_ID_MAPPING', 'UKPMC', 'UCSC_INTERACTION_BROWSER', 'UNIPROBE', 'UCSC_CANCER_GENOMICS_BROWSER', 'bioconductor-UNIFIEDWMWQPCR', 'UNIFIEDWMWQPCR', 'UNIPROT_BLAST', 'UNIPARC', 'UNIPRIME2', 'UM-BBD', 'UNDERLYINGFILTER', 'UNIPEPT', 'bioconductor-UNIPROT.WS', 'UNIPROT.WS', 'UNIMOD', 'UNFINNIGAN', 'UGENE', 'UNIPROTKB', 'bioconductor-TRIMSPACE', 'bioconductor-TRIMSEQ', 'bioconductor-TRIMEST', 'TRIMMOMATIC', 'TRINITY', 'bioconductor-TWILIGHT', 'TWILIGHT', 'TREEGRAPH_2', 'bioconductor-TWOFEAT', 'bioconductor-TRIFORM', 'TRIFORM', 'bioconductor-TYPEINFO', 'TYPEINFO', 'bioconductor-TSCAN', 'TSCAN', 'bioconductor-TRIPLEX', 'TRIPLEX', 'bioconductor-TSPAIR', 'TSPAIR', 'TRNADB-CE', 'UCL_SERVERS', 'TRICHDB', 'TUBERCULIST', 'TUIUIU', 'TRIMMOMATIC-IP', 'TRIMADAPTOR-IP', 'TRIPAL', 'bioconductor-TWEEDESEQ', 'TWEEDESEQ', 'TREEDYN', 'TRNASCAN-SE', 'TRIM_GALORE', 'TSEMA', 'TTD', 'TRIPLEXRNA', 'TYNA', 'bioconductor-TVTB', 'TVTB', 'bioconductor-TXIMPORT', 'TXIMPORT', 'bioconductor-TURBONORM', 'TURBONORM', 'bioconductor-TRIO', 'TRIO', 'TRIP', 'TRONCO', 'TREEFAM', 'TRIMSEQUENCES-IP', 'bioconductor-TRIGGER', 'TRIGGER', 'bioconductor-TSSI', 'TSSI', 'TROD', 'TOXODB', 'bioconductor-TRACKTABLES', 'TRACKTABLES', 'TRACTS', 'TOPSAN', 'bioconductor-TRANALIGN', 'bioconductor-TRANSLATOME', 'TRANSLATOME', 'TRACEMBLER', 'TRANS-ABYSS', 'bioconductor-TRACKVIEWER', 'TRACKVIEWER', 'TRAMPLE', 'TRANSLATORX', 'TRANSFOLD', 'TOPPGENE_SUITE', 'TRANSFIND', 'TRAPID', 'TRANSALIGN', 'bioconductor-TRANSCRIPTR', 'TRANSCRIPTR', 'TRANSEQ-EBI', 'TRACER', 'TREEDET', 'TRANSMIR', 'TRANSFAC', 'TRANSCLUST', 'TRANALIGN_WS_HUSAR', 'TRANSMART', 'bioconductor-TRANSEQ', 'TOPPR', 'TREECON', 'TOPPIC', 'bioconductor-TRANSVIEW', 'TRANSVIEW', 'TOUCAN_2', 'TREEDOMVIEWER', 'TORQUE', 'TRAPLINE', 'TQDIST', 'TMHMMFIX', 'TOPCONS', 'TOPHAT_PARALLEL-IP', 'bioconductor-TITANCNA', 'TITANCNA', 'bioconductor-TOPGO', 'TOPGO', 'bioconductor-TOFSIMS', 'TOFSIMS', 'TMB-HUNT', 'TMPRIME', 'TOPPCLUSTER', 'TMHMM', 'TOPHAT-FUSION', 'TOPDB', 'TIPS', 'TOM', 'TMSEG', 'TOPHAT-IP', 'TMAP', 'TMH_BENCHMARK', 'TMA_NAVIGATOR', 'TOPFIND', 'TOPOLOGYGSA', 'TOGOWS', 'TNRS', 'TOPHAT2-IP', 'bioconductor-TKWIDGETS', 'TKWIDGETS', 'TMBETA-NET', 'TMPAD', 'TMSOC', 'TOPHAT_CLOUDIFB', 'TOPHAT', 'TIMECLIP', 'THE_INTERACTORIUM', 'THE_FLUX_SIMULATOR', 'bioconductor-TEXTGET', 'THE_MENDEL_SITE', 'TFBSCLUSTER', 'THYME', 'THREAD_MAPPER_STUDIO', 'TIGR_SOFTWARE_TOOLS', 'TIGRFAMS', 'THE_ITHANET_PORTAL', 'bioconductor-TFEXTRACT', 'TERMINE', 'TILEQC', 'TFGD', 'THE_GENOMIC_HYPERBROWSER', 'THE_CHICKPEA_PORTAL', 'THE_FLUX_CAPACITOR', 'bioconductor-TILINGARRAY', 'TILINGARRAY', 'TIARA', 'TICO', 'TFMODELLER', 'THESEED', 'bioconductor-TFSCAN', 'bioconductor-TFM', 'bioconductor-TEXTSEARCH', 'THERMO_MSF_PARSER', 'THEOSPEC', 'TIMAT2', 'bioconductor-TIMECOURSE', 'TIMECOURSE', 'TFM-EXPLORER', 'TFAM', 'THE_AUTOPHAGY_DATABASE', 'TEBACTEN', 'bioconductor-TCODE', 'TAXONOMY_KRONA_CHART-IP', 'TASE', 'TAVERNA4PROTEOMICS', 'bioconductor-TDARACNE', 'TDARACNE', 'bioconductor-TARGETSEARCH', 'TARGETSEARCH', 'TARGETRNA', 'bioconductor-TAXGETDOWN', 'TCOFFEE', 'TAPYR', 'bioconductor-TEQC', 'TEQC', 'TARFISDOCK', 'TARO', 'TARGETFINDER.ORG', 'TARGET_EXPLORER', 'TCDB', 'bioconductor-TAXGETSPECIES', 'bioconductor-TAXGET', 'TAXOPTIMIZER-IP', 'bioconductor-TARGETSCORE', 'TARGETSCORE', 'TCOF-DB', 'TDR_TARGETS', 'TAXONOMIC_METAGENOME_TOOL', 'TARGETP', 'TATP', 'TARGETHUB', 'bioconductor-TAXGETRANK', 'TAXONOMY_SUMMARY-IP', 'bioconductor-TCC', 'TCC', 'bioconductor-TCGABIOLINKS', 'TCGABIOLINKS', 'TARGETIDENTIFIER', 'bioconductor-TAXGETUP', 'TCGA_API', 'TAVERNA', 'TASR', 'TAXMAN', 'TARBASE', 'TASSDB', 'TAKEABREAK', 'TAGTOG', 'bioconductor-SYNERGYFINDER', 'SYNERGYFINDER', 'T2D_CONVERTER', 'SYZYGY', 'TA-SI_PREDICTION', 'TABLET', 'SYNOR', 'TAP', 'TAPIR_CBS', 'SYNTREN', 'TAGDUST', 'TALLYMER', 'bioconductor-SYNAPTER', 'SYNAPTER', 'TACT', 'TADB', 'T1DBASE', 'TABHU', 'T-COFFEE', 'T-COFFEE_API_EBI', 'TABLEANNOTATOR', 'T-LEX', 'T-REG_COMPARATOR', 'T3DB', 'T-REX', 'SYMMDOCK', 'SYSTEM_BIOLOGY_PLATFORM', 'SYMCURV', 'bioconductor-SYNLET', 'SYNLET', 'TAIPAN', 'TALLY', 'SYNTVIEW', 'T-PROFILER', 'TAPIR', 'SYSBIO_MODELICA_LIBRARY', 'TAGCLEANER', 'T-RMSD', 'SWISS-PROT', 'SWISSADME', 'bioconductor-SYCO', 'SVDETECT_BAM_PREPROCESSING_PAIRS-IP', 'SWDMR', 'SYAPSE', 'SWISSSIMILARITY', 'SWISS-MODEL_REPOSITORY', 'SWISSREGULON', 'SYLARRAY', 'SWARM', 'SVDETECT_RUN_PARALLEL_STEP-IP', 'bioconductor-SWATHXTEND', 'SWATHXTEND', 'SVDETECT', 'SWALIGN', 'SWIFT-ILLUMINA', 'SWISS-2DPAGE', 'SWISSDOCK', 'SWIFT', 'SWIPE', 'SVM2', 'SYFPEITHI', 'SWAKK', 'SWISS-MODEL_WORKSPACE', 'SWISSVAR', 'SVMHC', 'SWEPEP', 'SVDETECT_COMPARE_STEP-IP', 'bioconductor-SWITCHDE', 'SWITCHDE', 'bioconductor-SVAPLSSEQ', 'SVAPLSSEQ', 'SVMERGE', 'SXOLIGOSEARCH', 'bioconductor-SWIMR', 'SWIMR', 'SWISS_EMBNET', 'SWISS_MODEL', 'SVM-PROT', 'STRETCHER-EBI', 'SUGARBIND', 'STRETCHER_WS_HUSAR', 'bioconductor-SUPERMATCHER', 'SUBJUNC', 'bioconductor-STRINGDB', 'STRINGDB', 'bioconductor-SURVCOMP', 'SURVCOMP', 'bioconductor-SUSHI', 'SUSHI', 'SUPERPOSE', 'SUPERLOOPER', 'SUPERQUANTNODE', 'STRIDE', 'STRING', 'SUBA3', 'bioconductor-STSSEARCH', 'STRETCHER-API-EBI', 'SURVNET', 'SURVCURVE', 'SUPPA', 'bioconductor-STRETCHER', 'SUBREAD', 'SUFFIXERATOR', 'bioconductor-SUBSEQ', 'SUBSEQ', 'SUMOSP', 'STRUCT2NET', 'SUPERCYP', 'STRICTLYINCLUDEGFF-IP', 'bioconductor-SUPRAHEX', 'SUPRAHEX', 'SUPERSPLAT', 'SUPERIMPOSE', 'SUPERSWEET', 'SUPERPRED', 'STSNP', 'SUPER', 'SUPERPOSE3D', 'SUPERMATCHER_WS_HUSAR', 'bioconductor-SUMMARIZEDEXPERIMENT', 'SUMMARIZEDEXPERIMENT', 'SUPERTARGET', 'bioconductor-STAN', 'STAN', 'bioconductor-SSIZE', 'SSIZE', 'bioconductor-SSPA', 'SSPA', 'STARBASE', 'STITCH', 'STAMP', 'STEMCELLNET', 'bioconductor-STARR', 'STARR', 'SSRCALC', 'SSPACE', 'STADEN', 'STAMPY', 'STANDARDIZED_VELVET_ASSEMBLY_REPORT', 'bioconductor-SSEQ', 'SSEQ', 'bioconductor-SSCU', 'SSCU', 'STAP', 'STRAININFO', 'STAMPY_MAPPER-IP', 'STAVROX', 'STRAND_NGS', 'bioconductor-SSVIZ', 'SSVIZ', 'bioconductor-STATTARGET', 'STATTARGET', 'bioconductor-STEPNORM', 'STEPNORM', 'bioconductor-STREAMER', 'STREAMER', 'STITCHPROFILES.UIO.NO', 'STEINERNET', 'STITCHPROFILES', 'STAMPY_INDEXER-IP', 'STAR', 'STRELKA', 'bioconductor-SSCORE', 'SSCORE', 'bioconductor-STEPWISECM', 'STEPWISECM', 'STING_MILLENIUM', 'STARTSCAN', 'bioconductor-STARBIOTREK', 'STARBIOTREK', 'STEMCHECKER', 'SSEP', 'STRAW', 'SR-TESSELER', 'SPLICETRAP', 'SPP', 'SSA', 'bioconductor-SPOTSEGMENTATION', 'SPOTSEGMENTATION', 'bioconductor-SPLICESITES', 'SPLICESITES', 'bioconductor-SRAP', 'SRAP', 'SRD', 'bioconductor-SPLICINGGRAPHS', 'SPLICINGGRAPHS', 'SPLITSEEK', 'SR-ASM', 'SSAHA', 'bioconductor-SPLOTS', 'SPLOTS', 'SROOGLE', 'SPRING', 'bioconductor-SRGNET', 'SRGNET', 'bioconductor-SQUADD', 'SQUADD', 'SRIDE', 'SPOT', 'SRNA_WORKFLOW', 'bioconductor-SPLITSOURCE', 'SSAHA_PILEUP-IP', 'bioconductor-SPLINTER', 'SPLINTER', 'SSAKE', 'bioconductor-SRADB', 'SRADB', 'bioconductor-SPLINETIMER', 'SPLINETIMER', 'SPRITE', 'SQTLSEEKER', 'SPPG', 'bioconductor-SPLITTER', 'SPLICINGVIEWER', 'SRMA', 'SSAHASNP', 'SPLITPOCKET', 'SPRITZ', 'SPLITDIST', 'SPECARRAY', 'SPLICEPLOT', 'bioconductor-SPIA', 'SPIA', 'SPIKE', 'SPEED', 'SPATT', 'SPARCS', 'SPINDLEP', 'SPLICEAID', 'SPIDER', 'SPIDERMASS', 'SPLICEMAP', 'SPLICEDISEASE_DB', 'SPLICEAID-F', 'SPLICEMACHINE', 'SPECTRAST', 'bioconductor-SPIKELI', 'SPIKELI', 'SPARSE', 'SPIRPRO', 'SPIN', 'SPECLUST', 'SPECTRUM_SIMILARITY', 'SPLICEGRAPHER', 'bioconductor-SPECL', 'SPECL', 'bioconductor-SPIDERMIR', 'SPIDERMIR', 'bioconductor-SPECOND', 'SPECOND', 'SPEER-SERVER', 'SPEDRE', 'SPADES-IP', 'SPIRAL_GENETICS', 'SPECTRUM_LOOK', 'bioconductor-SPLICEGEAR', 'SPLICEGEAR', 'SPLICEPORT', 'SNPSYN', 'SOYBASE', 'SNPTREE', 'SNPMASKER', 'SOLID_QUAL_BOXPLOT-IP', 'SOAP', 'SOCS', 'SOLAS', 'bioconductor-SPACEPAC', 'SPACEPAC', 'SNPEFFECT', 'SOMENA', 'SOLID2FASTQ-IP', 'SORTGFF-IP', 'SNVER', 'bioconductor-SNPSTATS', 'SNPSTATS', 'SNPINFO', 'SPACE', 'SOBA', 'SPACER', 'SNPNEXUS', 'SOMINACLUST', 'bioconductor-SOGGI', 'SOGGI', 'SOAPDENOVO', 'SOAPSNP', 'SOLEXATOOLS', 'SOL_GENOMICS_NETWORK', 'SNPITTY', 'SNPS_AND_GO', 'SOAPALIGNER-IP', 'SOLID_QUAL_STATS-IP', 'SNPSEA', 'SNVMIX', 'SOAPFUSION', 'SOLEXAQA', 'SOLID_SOFTWARE_TOOLS', 'bioconductor-SOMATICSIGNATURES', 'SOMATICSIGNATURES', 'bioconductor-SNPRELATE', 'SNPRELATE', 'SORTMERNA', 'SOAPDB', 'SORT2', 'SOFTWARE.NCRNA.ORG', 'SMIRNADB', 'bioconductor-SNPCHIP', 'SNPCHIP', 'bioconductor-SMITE', 'SMITE', 'bioconductor-SNPEDIAR', 'SNPEDIAR', 'bioconductor-SNAGEE', 'SNAGEE', 'SLEDGEHMMER', 'SLIMSEARCH', 'SLIMDISC', 'SNPACK', 'SNAP2', 'bioconductor-SLGI', 'SLGI', 'SNPBOX', 'SLITHER', 'SNPEDIA', 'SMOG', 'SMART', 'SNPEFF2-IP', 'SLIDESORT', 'SNIP-SEQ', 'bioconductor-SNM', 'SNM', 'SNOSCAN', 'SLUSH-BIOJS', 'SNIPPY', 'SLIDER', 'bioconductor-SMAP', 'SMAP', 'bioconductor-SNAPCGH', 'SNAPCGH', 'SMOLDYN', 'SLIMFINDER', 'SMAP_WS', 'SNP_CUTTER', 'SNPEFF', 'SNPATOR', 'SLIMSUITE', 'SNPEFF4-IP', 'SNPEFF3-IP', 'SLOPE', 'SMASHCOMMUNITY', 'SNIPER', 'SNP-O-MATIC', 'SKYLINE', 'bioconductor-SLQPCR', 'SLQPCR', 'SNOW', 'SNOGPS', 'SISYPHUS', 'bioconductor-SKIPREDUNDANT', 'bioconductor-SISPA', 'SISPA', 'SIRW', 'SITESEER', 'SIMMAP', 'SIMEXPR2SAMPLEDATA', 'bioconductor-SIZESEQ', 'bioconductor-SIRNA', 'SISSRS', 'SKEWER', 'bioconductor-SIMLR', 'SIMLR', 'SIXPACK-EBI', 'SIRNA_SELECTION', 'bioconductor-SIXPACK', 'SIMSEQ', 'bioconductor-SIMULATORZ', 'SIMULATORZ', 'SIVIRUS', 'SITECON', 'bioconductor-SKIPSEQ', 'bioconductor-SIZEPOWER', 'SIZEPOWER', 'bioconductor-SKEWR', 'SKEWR', 'SITEHOUND-WEB', 'SITEENGINE', 'SIRES', 'SIMBIONET', 'SJCOUNT', 'SIRIUS', 'SIMCOMP-SUBCOMP', 'SITEX', 'bioconductor-SIGCLEAVE', 'SIB-BLAST', 'SIGNISITE', 'bioconductor-SHUFFLESEQ', 'SIMAP_WS', 'SIENA', 'SIBELIA', 'SILOCO', 'SIM4CC', 'bioconductor-SIGPATHWAY', 'SIGPATHWAY', 'bioconductor-SIGCHECK', 'SIGCHECK', 'SIDIRECT', 'SIFT', 'SILVANGS', 'SIGNALP', 'bioconductor-SIGHTS', 'SIGHTS', 'SHRIMP', 'SICER', 'bioconductor-SIGFUGE', 'SIGFUGE', 'SIGNATURE', 'bioconductor-SHOWSERVER', 'bioconductor-SILENT', 'bioconductor-SIMBINDPROFILES', 'SIMBINDPROFILES', 'SIM-XL', 'bioconductor-SIGGENES', 'SIGGENES', 'SILVA', 'SIGNOR', 'SHREC', 'bioconductor-SIGNER', 'SIGNER', 'SILKDB', 'SEWAL', 'SHOW_EMFIT', 'SH3-HUNTER', 'SGTARGET', 'bioconductor-SHOWDB', 'SHOWALIGN_WS_HUSAR', 'SHINYMB', 'SHE-RA', 'SHORTFUSE', 'bioconductor-SHINYTANDEM', 'SHINYTANDEM', 'bioconductor-SHORTREAD', 'SHORTREAD', 'SHOW_SIMUL', 'SFMAP', 'SFCHECK', 'SHELX', 'bioconductor-SHOWFEAT', 'bioconductor-SHOWPEP', 'SGA', 'SHERMAN', 'SHORE', 'SHORTY', 'bioconductor-SHINYMETHYL', 'SHINYMETHYL', 'SFOLD', 'SFINX', 'SHAMAN', 'SETUPPROMOTER-IP', 'SFF2FASTQ', 'SHORAH', 'SHOW_VITERBI', 'SGATOOLS', 'SGP2', 'bioconductor-SHOWSEQ', 'bioconductor-SHOWALIGN', 'bioconductor-SHOWORF', 'bioconductor-SEVENBRIDGES', 'SEVENBRIDGES', 'SHARCGS', 'SHOREMAP', 'SEQWARE', 'bioconductor-SEQXREFGET', 'bioconductor-SERVERTELL', 'bioconductor-SEQXREF', 'SEQMONK', 'SEQPREP', 'SEQUENCHER', 'bioconductor-SEQTOOLS', 'SEQTOOLS', 'SEQVISTA', 'SEQUANA_COVERAGE', 'SEQMAP', 'SEQSAW', 'SEQMATCHALL_WS_HUSAR', 'bioconductor-SEQRETSETALL', 'bioconductor-SEQRET', 'SEQMAN_NGEN', 'SEQSITE', 'SEQTRIM', 'SEQUEDEX', 'bioconductor-SEQPATTERN', 'SEQPATTERN', 'SEQUENCE_HARMONY', 'SERBGO', 'SERVICES_FROM_ZHOU_LABORATORY', 'bioconductor-SEQMATCHALL', 'SEQTYPEDETECT-IP', 'SEQMINER', 'SEQSEG', 'SEQWORDS', 'bioconductor-SEQVARTOOLS', 'SEQVARTOOLS', 'SEQRET-EBI', 'bioconductor-SEQRETSPLIT', 'SEQSOLVE', 'SEQSPHERE', 'SEQUENCEVARIANTANALYZER', 'SEL.TS.AREA', 'SEQCKSUM', 'SEQAN', 'SELECTBYTAG-IP', 'SELECTBESTHIT-IP', 'SEQCONS', 'bioconductor-SEPA', 'SEPA', 'SEPPA', 'SELECTOME', 'SELECTHITSCORE-IP', 'SEQ2HLA', 'SEQATOMS', 'SEQLM', 'SEQBUSTER', 'SEQEM', 'bioconductor-SEQLOGO', 'SEQLOGO', 'SEPACS', 'SEM', 'SELOX', 'SELTARBASE', 'SEQ2LOGO', 'SEQDEPOT', 'bioconductor-SEQARRAY', 'SEQARRAY', 'SEPRESA', 'SELENOPROFILES3', 'bioconductor-SEQCOUNT', 'bioconductor-SEQGSEA', 'SEQGSEA', 'bioconductor-SEQBIAS', 'SEQBIAS', 'bioconductor-SEQCNA', 'SEQCNA', 'SELECTON', 'SEQFIRE', 'SEATTLESNPS', 'SECISDESIGN', 'SEAR', 'bioconductor-SEEALSO', 'SEECER', 'SDPPRED', 'SEEMOTIF', 'SDA', 'SDRDB', 'SCRY', 'SEGWAY', 'SEEDGFF-IP', 'SEED', 'SEABED', 'SEGTOOLS', 'SEAVIEW', 'SECRETOMEP', 'SEAGRASS_ZMU_GENOME', 'SECISALN', 'SEEK_FOR_SCIENCE', 'bioconductor-SCRAN', 'SCRAN', 'SEAL', 'SEGTOR', 'SEARCHPKS', 'SDPMOD', 'SEARCHGTR', 'SCOPE', 'SCR_FIND', 'SCORECONS', 'SCOWL', 'SECISEARCH3_SEBLASTIAN', 'SCOP3D', 'SEARCHGUI', 'SCRIPTURE', 'SEGEMEHL', 'bioconductor-SCSR', 'SCSR', 'SDM', 'SCRATCH', 'SEA', 'SARTOOLS_DESEQ2-IP', 'bioconductor-SCATER', 'SCATER', 'SBML', 'SCOOBY_DOMAIN_PREDICTION', 'SATELLOG', 'SATCHMO-JS', 'SAVANT_GENOME_BROWSER', 'SCIPION_WEB_TOOLS', 'SBO_WS', 'SCARF', 'SCANMOT', 'SBC_TOOLS', 'SATRAP', 'SASTBX', 'SCHLORO', 'SCANPROSITE', 'SCMD', 'SCIT', 'SCAMPI', 'bioconductor-SC3', 'SC3', 'bioconductor-SCAN.UPC', 'SCAN.UPC', 'bioconductor-SCDE', 'SCDE', 'SCAFFOLD_VIEWER', 'SCALCE', 'bioconductor-SAVR', 'SAVR', 'bioconductor-SBMLR', 'SBMLR', 'SCLD', 'SCERTF', 'SAS', 'SBGR', 'SCITRACK', 'SARTOOLS_EDGER-IP', 'SCAFFOLDER', 'bioconductor-SCISI', 'SCISI', 'SBT_INTERFACE', 'SCANSITE', 'SCANPS', 'SAVOR', 'RVD', 'SAM2TSV', 'SAPS', 'RXNORM_API', 'SAMTOOLS_SLICE_BAM-IP', 'SAMTOOLS_SORT-IP', 'bioconductor-SAMSPECTRAL', 'SAMSPECTRAL', 'S-MART', 'SABIO-RK', 'SALAMI', 'SABBAC', 'bioconductor-SAGENHAFT', 'SAGENHAFT', 'bioconductor-S4VECTORS', 'S4VECTORS', 'bioconductor-SAGX', 'SAGX', 'bioconductor-SANTA', 'SANTA', 'SAM-T08', 'SARA', 'SA-MOT', 'RVISTA', 'SALAD', 'SARTOOLS', 'SABUND-IP', 'SAMTOOLS_IDXSTATS-IP', 'bioconductor-SANGERSEQR', 'SANGERSEQR', 'SALMOBASE', 'S2P', 'SAMTOOLS_FLAGSTAT-IP', 'SAMTOOLS_FILTER-IP', 'SAMTOOLS', 'bioconductor-SAPFINDER', 'SAPFINDER', 'SAGEMAP', 'SAHG', 'SAMJAVASCRIPT', 'SAM_TO_BAM-IP', 'SAMMATE', 'SAMSCOPE', 'SAMSTAT', 'RUBI', 'bioconductor-RSVSIM', 'RSVSIM', 'bioconductor-RSBML', 'RSBML', 'bioconductor-RTRMUI', 'RTRMUI', 'RTIPS', 'RSNP_GUIDE', 'RT', 'bioconductor-RTANDEM', 'RTANDEM', 'bioconductor-RTN', 'RTN', 'bioconductor-RUVCORR', 'RUVCORR', 'bioconductor-RQUBIC', 'RQUBIC', 'RRNASELECTOR', 'RSTUDIO', 'RSCRIPTS_FOR_DATA_ANALYSIS', 'RSPRED', 'RSAT_PEAK-MOTIFS', 'bioconductor-RSFFREADER', 'RSFFREADER', 'bioconductor-RTOPPER', 'RTOPPER', 'bioconductor-RUVNORMALIZE', 'RUVNORMALIZE', 'bioconductor-RTREEMIX', 'RTREEMIX', 'bioconductor-RTRM', 'RTRM', 'bioconductor-RRDP', 'RRDP', 'bioconductor-RRHO', 'RRHO', 'RSAT', 'RSEQC', 'RTANALYZER', 'bioconductor-RSUBREAD', 'RSUBREAD', 'RSEM', 'RSEQTOOLS', 'RUBIOSEQ', 'bioconductor-RTCGATOOLBOX', 'RTCGATOOLBOX', 'RULEGO', 'RSSSITE', 'ROBUSTRANKAGGREGATION', 'ROUGE', 'ROI_REPORT-IP', 'ROI_PCA-IP', 'ROI_TABLE-IP', 'ROI_PLOT_THUMBNAILS-IP', 'bioconductor-ROTS', 'ROTS', 'bioconductor-RPA', 'RPA', 'RPBS', 'RQUANT.WEB', 'ROCI', 'bioconductor-ROAR', 'ROAR', 'bioconductor-RPSIXML', 'RPSIXML', 'bioconductor-RNITS', 'RNITS', 'ROSETTA_FLEXPEPDOCK', 'ROSETTABACKRUB', 'RONN', 'RODDY', 'ROI_FEATURE_PROJECTION-IP', 'ROI_METRICS-IP', 'RNNOTATOR', 'ROBINA', 'ROLEXA', 'bioconductor-ROLESWITCH', 'ROLESWITCH', 'ROSETTADESIGN', 'ROBETTA', 'RNAZ', 'ROI_SORT-IP', 'RPROTEOMICS', 'bioconductor-RONTOTOOLS', 'RONTOTOOLS', 'ROSETTAANTIBODY', 'RNMR', 'bioconductor-ROPLS', 'ROPLS', 'bioconductor-RPX', 'RPX', 'bioconductor-RQC', 'RQC', 'bioconductor-ROC', 'ROC', 'RPF', 'ROI_FILTER-IP', 'RNAVIRUSDB', 'RNASEQR', 'bioconductor-RNASEQPOWER', 'RNASEQPOWER', 'RNA_MOVIES', 'RNALOGO', 'RNA_HELICASE_DATABASE', 'RNAHYBRID', 'bioconductor-RNASEQCOMP', 'RNASEQCOMP', 'RNAMUTANTS', 'RNA_ANALYZER', 'RNATIPS', 'RNAPROFILE', 'RNAALIFOLDWS', 'RNASPADES-IP', 'RNAMUTE', 'RNABOR', 'RNASNP', 'RNAEDITINGATLAS', 'RNAMMER', 'RNA', 'RNA-MATE', 'RNABINDR', 'RNA_STUDIO', 'RNASTRUCTURE', 'bioconductor-RNASEQSAMPLESIZE', 'RNASEQSAMPLESIZE', 'bioconductor-RNAPROBR', 'RNAPROBR', 'bioconductor-RMAT', 'RMAT', 'RNASOFT', 'RMOTIF', 'RNAPREDATOR', 'RNAMDB', 'RNAIFOLD', 'RNA_COSSMOS', 'RNALOSS', 'RNAMST', 'bioconductor-RMAGPIE', 'RMAGPIE', 'bioconductor-RIBOSEQR', 'RIBOSEQR', 'RIBOGALAXY', 'RIBOTAPER', 'RHYTHM', 'RIBOSWITCH_FINDER', 'RILOGO', 'RGESTLIBCOMP-IP', 'bioconductor-RIBOPROFILING', 'RIBOPROFILING', 'bioconductor-RGRAPHVIZ', 'RGRAPHVIZ', 'RIKEN', 'RIGHTFIELD', 'RGPICFIXMATE-IP', 'RIBOPICKER', 'bioconductor-RHVDM', 'RHVDM', 'RIBEX', 'RISEARCH', 'RMAP', 'bioconductor-RINGO', 'RINGO', 'bioconductor-RLMM', 'RLMM', 'RJPRIMERS', 'RICEXPRO', 'bioconductor-RIMMPORT', 'RIMMPORT', 'bioconductor-RGSEA', 'RGSEA', 'bioconductor-RHTSLIB', 'RHTSLIB', 'bioconductor-RIPSEEKER', 'RIPSEEKER', 'bioconductor-RGRAPH2JS', 'RGRAPH2JS', 'RHEA', 'REPUTER', 'bioconductor-REVSEQ', 'RESTRICTTRANSCRIPTLIST-IP', 'RESTRICTFROMSIZE-IP', 'RESCUE-ESE', 'RF-CLONING.ORG', 'RGD', 'RESFINDER', 'RETROTECTOR', 'RESTRICTSEQUENCELIST-IP', 'REVTRANS', 'RF_DYMHC', 'RFAM_SEARCH', 'bioconductor-RESTOVER', 'bioconductor-RGALAXY', 'RGALAXY', 'bioconductor-REQON', 'REQON', 'bioconductor-RGADEM', 'RGADEM', 'RETENTIONPREDICTION', 'REPTILE', 'RESID', 'bioconductor-RESTRICT', 'RESEQSIM', 'RESPONSENET', 'REPETITA', 'REPROGENOMICS_VIEWER', 'REMOVEEXONLINES-IP', 'REMUS', 'REPEATSDB', 'REPET', 'RELNA', 'REGPHOS', 'RELATE', 'REMODISCOVERY', 'bioconductor-REFSEQGET', 'REFORMAT_FASTQ-IP', 'bioconductor-REGIONER', 'REGIONER', 'RELEX', 'bioconductor-REFPLUS', 'REFPLUS', 'REGPREDICT', 'REPPER', 'REGULONDB', 'REFSEQ', 'REFMAC', 'REPORTER', 'REFORMAT_SNPEFF_FOR_SYNTVIEW-IP', 'bioconductor-REPORTINGTOOLS', 'REPORTINGTOOLS', 'REFCOV', 'bioconductor-REPITOOLS', 'REPITOOLS', 'bioconductor-REFNET', 'REFNET', 'REGRNA', 'REPK', 'REMS', 'REPROF', 'REPLACE_REGEX-IP', 'bioconductor-REGSPLICE', 'REGSPLICE', 'REDUCE', 'REGANALYST', 'REGPRECISE', 'REPTAR', 'RENATO', 'REDIDB', 'REDHOM', 'READFASTAFILE-IP', 'bioconductor-READQPCR', 'READQPCR', 'READEMPTION', 'READOUT', 'RE-MUSIC', 'REACTOME', 'RECODE', 'RECPARS', 'READXPLORER', 'bioconductor-RECODER', 'bioconductor-REDATA', 'RECAL2', 'RECONCILIATOR', 'bioconductor-REDSEQ', 'REDSEQ', 'REDFLY', 'REACTOME_SMALL_MOLECULE_SEARCH', 'READSEQ', 'REDITOOLS', 'bioconductor-RECOUP', 'RECOUP', 'READALIGNER', 'REAPER', 'bioconductor-REB', 'REB', 'bioconductor-RDRTOOLBOX', 'RDRTOOLBOX', 'RECON', 'REAL', 'bioconductor-READAT', 'READAT', 'REDS', 'RECOUNTDB', 'bioconductor-REBASEEXTRACT', 'RECOUNT', 'READW', 'READDEPTH', 'RDIFF', 'bioconductor-RDAVIDWEBSERVICE', 'RDAVIDWEBSERVICE', 'RD-CONNECT', 'bioconductor-RCAS', 'RCAS', 'bioconductor-RDGIDB', 'RDGIDB', 'bioconductor-RCHYOPTIMYX', 'RCHYOPTIMYX', 'RBPDB', 'RDP', 'RAST', 'bioconductor-RBM', 'RBM', 'bioconductor-RCYTOSCAPE', 'RCYTOSCAPE', 'bioconductor-RBSURV', 'RBSURV', 'RBPMOTIF', 'RAW2MZDB', 'RAY', 'RBC_MAFFT-IP', 'RAZERS', 'bioconductor-RDISOP', 'RDISOP', 'RAXML', 'RBDE', 'RCI', 'bioconductor-RCADE', 'RCADE', 'RDP_PYROSEQUENCING_PIPELINE', 'bioconductor-RBIOPAXPARSER', 'RBIOPAXPARSER', 'bioconductor-RBGL', 'RBGL', 'bioconductor-RCYJS', 'RCYJS', 'R_CLOUDIFB', 'bioconductor-RANKPROD', 'RANKPROD', 'R2R', 'bioconductor-RAMIGO', 'RAMIGO', 'bioconductor-RARIANT', 'RARIANT', 'R-COFFEE', 'QUICKSNP', 'RADAR', 'R_CLOUD_WORKBENCH', 'QUERYOR', 'RANKOPTIMIZER-IP', 'bioconductor-R4RNA', 'R4RNA', 'RAAMS', 'QUEST', 'RACA', 'bioconductor-R453PLUS1TOOLBOX', 'R453PLUS1TOOLBOX', 'R3D_ALIGN', 'R_SPIDER', 'RAP', 'RASQUAL', 'RANDFOLD', 'bioconductor-RAIN', 'RAIN', 'bioconductor-RAMA', 'RAMA', 'bioconductor-R3CSEQ', 'R3CSEQ', 'bioconductor-RANDPACK', 'RANDPACK', 'QUICKGO', 'RASMOT-3D_PRO', 'QUICKJOIN', 'RAPHAEL', 'QUIP', 'RAPSEARCH', 'bioconductor-RAREVARIANTVIS', 'RAREVARIANTVIS', 'RACE', 'RAPIDO', 'QUPE', 'QUMA', 'RAPIDNJ', 'QIIME', 'PYTEOMICS', 'bioconductor-QDNASEQ', 'QDNASEQ', 'bioconductor-QUANTRO', 'QUANTRO', 'QUANTUM.LIGAND.DOCK', 'QPALMA', 'QSRA', 'QUADGT', 'bioconductor-QUANTSMOOTH', 'QUANTSMOOTH', 'QGRS-H_PREDICTOR', 'QCLUSTER', 'bioconductor-QUATERNARYPROD', 'QUATERNARYPROD', 'bioconductor-QRQC', 'QRQC', 'QUANTGENIUS', 'QDB', 'QCALL', 'QUALIMAP', 'QGRS_MAPPER', 'QSCOP_BLAST', 'QUASIMOTIFINDER', 'QA-RECOMBINEIT', 'QUALITYSNPNG', 'QMEAN', 'QUALTOFASTQ-IP', 'QUANT', 'bioconductor-QSEA', 'QSEA', 'QRIDGE', 'QSEQ', 'QUAKE', 'QUAST', 'bioconductor-QUARTPAC', 'QUARTPAC', 'bioconductor-QUBIC', 'QUBIC', 'bioconductor-QPCRNORM', 'QPCRNORM', 'QGRID', 'QUADFINDER', 'QARIP', 'PYCHEMINF', 'PUG', 'PYCOGENT', 'bioconductor-PSYGENET2R', 'PSYGENET2R', 'PYMZML', 'PYROMAP', 'PSSRNAMINER', 'PSORTDB', 'PSSRDB', 'PUPASUITE', 'PUDGE', 'PSROBOT', 'PYICOTEO', 'bioconductor-PVAC', 'PVAC', 'PSORT.ORG', 'PTGL', 'PYDOCKWEB', 'PYDOCKSAXS', 'PSYCHOPROT', 'PYMS', 'PYRONOISE', 'PTARGET', 'PVS', 'PXSIM', 'PYBEDTOOLS', 'bioconductor-PVIZ', 'PVIZ', 'PUBTATOR', 'PUBMED_CENTRAL', 'PUBMETH', 'PXIMAGE', 'PUBMED', 'bioconductor-PURECN', 'PURECN', 'PVIEW', 'bioconductor-PWOMICS', 'PWOMICS', 'PSRNATARGET', 'PUBCHEM_BIOASSAY_SERVICE', 'PUBCHEM', 'PUPASVIEW', 'PSCANCHIP', 'PSI-SEARCH', 'PROTEOSAFE', 'bioconductor-PROTEOMICSANNOTATIONHUBDATA', 'PROTEOMICSANNOTATIONHUBDATA', 'PROTEOWIZARD', 'PROTEUS2', 'PSC', 'PSEUDOCAP', 'PSI-MI_VALIDATOR', 'bioconductor-PSIPHI', 'PSI-SEARCH_REST', 'PROTK', 'PROTEUS', 'PROTEOMICS_TOOLBOX', 'PSI-BLAST_REST', 'PROTTER', 'PSEUDOVIEWER', 'PROTINFO_PPC', 'PROTOMATA', 'PROTEOPEDIA', 'PROTFUN', 'PROTOSS', 'bioconductor-PROTEOQC', 'PROTEOQC', 'PROTINFO', 'PROTONET', 'PSIQUIC_DIP', 'PROTEOMICSDB', 'PROTICDB', 'bioconductor-PSICQUIC', 'PSICQUIC', 'PSICQUIC_VIEW', 'bioconductor-PROTGENERICS', 'PROTGENERICS', 'PSICHOMICS', 'PSIPRED', 'PS2', 'bioconductor-PROSEXTRACT', 'PROTEIOS_SOFTWARE_ENVIRONMENT', 'PROTEINATLAS', 'PROSPLICER', 'PROTEOMATIC', 'PROTEINDBS', 'PROTEDNA', 'PROTCID', 'PROTCHEMSI', 'PROSESS', 'PROTEIN_NAMING_UTILITY', 'PROTEOFORMER', 'PROTEINSPECTOR', 'PROTEINGURU', 'bioconductor-PROT2D', 'PROT2D', 'bioconductor-PROSTAR', 'PROSTAR', 'PROTEOMEXCHANGE', 'PROTEOME-PI', 'PROTEMOT', 'PROTEOME_ANALYST', 'PROTEINCCD', 'PROTEIN_FRUSTRATOMETER', 'PROTEO', 'PROTEIN_MODEL_PORTAL', 'PROSITE_SCAN_EBI', 'PROTAG', 'PROT_PI', 'PROSIGHT_PTM_ION_PREDICTOR', 'PROTEINPROSPECTOR', 'PROTEINPROPHET', 'PROTEOMEDIGESTSIM', 'PROSPECT-PSPP', 'PROSMOS', 'PROSA-WEB', 'PROSITE', 'PROSOM', 'PROSIGHT_PTM', 'PROTEIN_DIGESTION_SIMULATOR_BASIC', 'PROTEIN_PEELING_3_D', 'PROSAT2', 'PROTEGEN', 'PROMOTERWISE', 'PROGENETIX', 'PROQ', 'PROLIFIC', 'PRORATA', 'PROMISEC_ENDPOINT_MANAGER', 'bioconductor-PROPER', 'PROPER', 'bioconductor-PROLOCGUI', 'PROLOCGUI', 'PROMATEUS', 'PROFUNC', 'PROFTMB', 'PROFOUND', 'PROMH', 'PROJECTOR_2', 'PROMALS3D', 'PROMOST', 'PROLIC', 'PROFSEC', 'bioconductor-PROFIT', 'PROPHECY', 'PROGMAP', 'PROPORTAL', 'PROMEX', 'PROMAYA', 'PROMOTER', 'PROFISIS', 'PROKKA', 'PROLINE', 'bioconductor-PROFILESCOREDIST', 'PROFILESCOREDIST', 'PROMOTERPLOT', 'PROKARYOTE-RNASEQ', 'PROJECT', 'bioconductor-PROPHET', 'bioconductor-PROLOC', 'PROLOC', 'bioconductor-PROMISE', 'PROMISE', 'PROMOSER', 'PROMOTERCAD', 'PROGLYCPROT', 'PROREPEAT', 'PROMISCUOUS', 'PROKWARE', 'PRIMETV', 'PROFACC', 'PROBEMATCH', 'bioconductor-PROCOIL', 'PROCOIL', 'PROBE', 'PRIMERS4CLADES', 'PRIMER3PLUS', 'PRISM', 'PROFHTM', 'PRODOMCG_TREE', 'PRODOMCG_EVOSCEN', 'PROBCONS_WS_JABAWS', 'PROFCOM', 'PRIMERBANK', 'PRODONET', 'bioconductor-PRINTSEXTRACT', 'bioconductor-PROCESS', 'PROCESS', 'PROBCONS', 'PROCARS', 'PROCON', 'PRINTS', 'PRODOM', 'PRIMER3', 'PROABC', 'PROFDISIS', 'bioconductor-PRIMERSEARCH', 'PRINSEQ', 'bioconductor-PROFIA', 'PROFIA', 'PRODOC', 'PROFEAT', 'PROBE_AND_PRIMER_SEARCH', 'PROBELYNX', 'PRINCE', 'PRO', 'PRIMERSTATION', 'PRESARTOOLS-IP', 'bioconductor-PREPROCESSCORE', 'PREPROCESSCORE', 'PRIDE', 'PRED-GPCR', 'PREX', 'PRIMER_Z', 'PREDICTSNP', 'PRETTYPLOT_WS_HUSAR', 'PREDICTNLS', 'PRIAM_SEARCH', 'bioconductor-PRETTYPLOT', 'PRECOG', 'PRCBENCH', 'PRIDE_TOOLSUITE', 'bioconductor-PREBS', 'PREBS', 'PRIFI', 'PRED-TMBB', 'PREDITOR', 'PREOPDB', 'PRGDB', 'PREDGPI', 'PRECREC', 'PREPAREANNOTATION-IP', 'PRICE', 'bioconductor-PREDICTIONET', 'PREDICTIONET', 'PREBI', 'PRI-CAT', 'PREDICTREGULON', 'PREDYFLEXY', 'PREDTM', 'PREDICTPROTEIN_OPEN', 'PREDICTSNP2', 'bioconductor-PRETTYSEQ', 'bioconductor-PREG', 'PREDIKIN', 'PREDUS', 'PREP_SUITE', 'PREDICTPROTEIN', 'PRDOS', 'PRIDE-ASA-PIPELINE', 'PREDISI', 'PON-DISO', 'PPI', 'POSEVIEW', 'POSTMAN', 'POPOOLATION', 'PRANK', 'bioconductor-PQSFINDER', 'PQSFINDER', 'POPSCOMP', 'POPS', 'POMBASE_MOTIF_SEARCH', 'PRATT', 'POMBASE_VEP', 'POREWALKER', 'PQPQ', 'POWERFIT', 'bioconductor-PPISTATS', 'PPISTATS', 'POXO', 'POSSUM', 'POSMED', 'PPISEARCH', 'POSSUMSEARCH', 'PPSC', 'POTAGE', 'PONGO', 'POWER', 'PPFOLD', 'POMBASE_DATA_SLICER', 'PORELOGO', 'POSSUMSEARCH2', 'PON-P2', 'PPPBENCHMARK', 'POOLHAP', 'PPG', 'POMELO_II', 'PPTASEDB', 'POPITAM', 'POPOOLATION2', 'PRALINE', 'PLOTGENOMECOVERAGE-IP', 'bioconductor-PLRS', 'PLRS', 'POBO', 'PLOTREP', 'POCO', 'PMG', 'PMUT', 'PMAP-CUTDB', 'PMDB', 'bioconductor-PLGEM', 'PLGEM', 'POCKETOME', 'PNU', 'bioconductor-PLOTORF', 'PLOT-IP', 'POISSONSEQ', 'bioconductor-PLW', 'PLW', 'bioconductor-POLYFIT', 'POLYFIT', 'bioconductor-PMM', 'PMM', 'POLYGENICPATHWAYS', 'PMIRKB', 'POLYQ', 'POLYVIEW-MM', 'POL3SCAN', 'POLYDOT_WS_HUSAR', 'bioconductor-PLOTCON', 'PLOTCOVERAGE-IP', 'POCKETQUERY', 'POCKETANNOTATE', 'POLBASE', 'bioconductor-POLYDOT', 'bioconductor-PLPE', 'PLPE', 'bioconductor-POLYESTER', 'POLYESTER', 'PMRD', 'PLNTFDB', 'POLYMIRTS', 'POLYSEARCH', 'PLANTCARE', 'PINTA', 'PIPMAKER', 'PIPMIR', 'PIRANHA', 'PLANDBAFFY', 'PLABIPD', 'PLAZA_DICOTS', 'PIR', 'PLASMODB', 'PLAST', 'PLATO_PLUGIN_FOR_AUTOCOMPLETE_ON_ONTOLOGIES', 'PKLFILEMERGER', 'PLANTMETABOLOMICS', 'PLANTNATSDB', 'PLAZA_MONOCOTS', 'PLADIPUS', 'bioconductor-PKGDEPTOOLS', 'PKGDEPTOOLS', 'PISCES', 'PLECDOM', 'PLANTTFDB_2.0', 'PLEXDB', 'PLANTSDB', 'PIRSF', 'PLANTGENIE.ORG', 'PIPELINE_PILOT', 'bioconductor-PLETHY', 'PLETHY', 'PINTS', 'PLAST-NCRNA', 'PLASMAPPER', 'PKD', 'PLAN2L', 'PITA', 'PISA', 'PLASMIDSPADES-IP', 'PIPEALIGN', 'PKNOTSRG', 'PLANTLOC', 'PLANTGSEA', 'PINGO', 'PIGS', 'PIANA', 'bioconductor-PIGENGENE', 'PIGENGENE', 'PINDEL', 'PINA', 'PID', 'PIMS_LIMS', 'PICARD_ARRG-IP', 'PILEUP2COV-IP', 'PICARD_REPLACESAMHEADER-IP', 'PICARDASMETRICS-IP', 'bioconductor-PICS', 'PICS', 'PICTAR', 'PIE', 'PIC', 'PICMI', 'PICARDINSERTSIZE-IP', 'PINDEL2VCF-IP', 'PICARD_REORDERSAM-IP', 'PICARDHSMETRICS-IP', 'PICALL', 'PINDEL-IP', 'PICARDGCBIASMETRICS-IP', 'PICARD_FASTQTOSAM-IP', 'bioconductor-PIANO', 'PIANO', 'PICARD_TOOLS', 'PICXAA-WEB', 'PICA', 'PILEUP_TO_VCF-IP', 'PICARD_BAMINDEXSTATS-IP', 'PICARD_SAMTOFASTQ-IP', 'bioconductor-PING', 'PING', 'bioconductor-PICKGENE', 'PICKGENE', 'PILGRM', 'PICR', 'PIDNA', 'PHOSTRYP', 'PHYLOGENETICS_AT_JST', 'PHYML-IP', 'PHYSBINDER', 'PHYML_ONECLICK-IP', 'PHYMM', 'PHYDBAC2', 'PHYLOPARS', 'PHOSPHAT', 'PHYLEASPROG', 'PHYML', 'PHYMYCO-DB', 'PHSOL', 'bioconductor-PHYLOSEQ', 'PHYLOSEQ', 'PI2PE', 'PHYLOGENY.FR', 'bioconductor-PI', 'PI', 'PHRED', 'PHREDPHRAPCONSED', 'PHOG', 'PHYLOFACTS_FAT-CAT', 'PHOSSNP', 'PHOSPHO3D', 'PHYD3', 'PHYML-SMS_ONECLICK-IP', 'PHUSER', 'PHYLOSCAN', 'PHYLOMEDB_V3.0', 'PHOSIDA', 'PHOSFINDER', 'PHOSPHOELM', 'PHYLOGENY.LIRMM.FR', 'PHYLEMON', 'PHYSIOLIBRARY', 'PHOSPHO.ELM', 'PHYLODOME', 'PHYTOZOME', 'PIA', 'PHD-SNP', 'bioconductor-PHILR', 'PHILR', 'PFIND_STUDIO', 'PHARMAGIST', 'PHENOHM', 'PFOLD', 'PFAMSCAN_API', 'PHOBIUS_API', 'PHOBIUS', 'PHDHTM', 'PHENETIC_EQTL', 'bioconductor-PHARMACOGX', 'PHARMACOGX', 'PEVOSOAR', 'PHARMMAPPER', 'PHENOM', 'PFP', 'PHISITE', 'PGMRA', 'PHENIX', 'PHEMTO', 'PGD', 'PGP', 'PHAISTOS', 'PHASER', 'PHENOTATOR', 'PHEATMAP', 'PHENETIC', 'PHEPS', 'PHMMER', 'PHAST', 'PHENOTIPS', 'PHARMGKB', 'bioconductor-PGSEA', 'PGSEA', 'PHEVER', 'PFLEXANA', 'PEPTIDEATLAS', 'bioconductor-PEPWINDOW', 'bioconductor-PEPINFO', 'PESI_WS', 'PEPSPLICE', 'PEPVAC', 'PEPSERVE', 'PEPTIDESHAKER', 'PEPWINDOW-EBI', 'bioconductor-PEPSTATS', 'PEPNOVO', 'PEPTIDEPROPHET', 'PEPXML2EXCEL', 'PERM', 'bioconductor-PEPXMLTAB', 'PEPXMLTAB', 'bioconductor-PEPSTAT', 'PEPSTAT', 'PERGOLA', 'PEPINFO-EBI', 'bioconductor-PEPWHEEL', 'bioconductor-PEPCOIL', 'PEPTIDE_MATCH', 'PEPTIDECUTTER', 'PERALIGN', 'PEPX', 'PEROXIBASE', 'PEPMMSMIMIC', 'PESPOTDB', 'PEPSTATS-EBI', 'PEPTIDE-SHAKER', 'PEPSHELL', 'PEPPY', 'bioconductor-PEPNET', 'bioconductor-PEPDIGEST', 'bioconductor-PEPWINDOWALL', 'PEPTIZER', 'PERCOLATOR', 'PEROXISOMEDB', 'PEPSITE', 'PDBEXTRACT', 'PEAK2BED-IP', 'PE_FILTER-IP', 'PEAKS_DE_NOVO', 'PEMER', 'PEAKSEQ', 'PEPBUILD', 'PEFFECT', 'PEAKS_Q', 'PE-ASSEMBLER', 'PEAKZILLA', 'PED', 'PELE', 'PDBEFOLD', 'PDBEMOTIF', 'PDBEPISA', 'PEAKCALLING_FINDPEAKS-IP', 'PEP3D', 'PEAKANALYZER', 'PEAKRANGER', 'PEP-FOLD', 'PDBFINDER', 'PDBSUM', 'PDBSUM_GENERATE', 'PEAKCALLING_MACS14-IP', 'PEAKCALLING_MACS-IP', 'PEAKS_DB', 'bioconductor-PDMCLASS', 'PDMCLASS', 'PEACE', 'PDBFUN', 'PDESTRIAN', 'bioconductor-PDINFOBUILDER', 'PDINFOBUILDER', 'PEAKTRACE', 'PECAN', 'PDBSITESCAN', 'PDBSELECT', 'PCRCLIPREADS', 'bioconductor-PCAGOPROMOTER', 'PCAGOPROMOTER', 'bioconductor-PBASE', 'PBASE', 'PDB2PQR', 'PCRPI-DB', 'PCDB', 'PCHOPPER', 'bioconductor-PBCMC', 'PBCMC', 'PDB_HYDRO', 'PCDDB', 'PBEQ-SOLVER', 'PDB_REDO', 'PATTERNLAB', 'PCONS', 'PBE', 'PCLEAVAGE', 'PCFAMILY', 'PDB-REDO', 'bioconductor-PCAN', 'PCAN', 'bioconductor-PCOT2', 'PCOT2', 'bioconductor-PCAEXPLORER', 'PCAEXPLORER', 'bioconductor-PCPHENO', 'PCPHENO', 'PDA', 'PBIL', 'PCE', 'PBSWORD', 'PCRTILER', 'bioconductor-PATMATDB', 'PATHJAM', 'PARTEK_GENOMICS_SUITE', 'bioconductor-PARODY', 'PARODY', 'bioconductor-PATHIFIER', 'PATHIFIER', 'PATMATCH', 'PARTIGENEDB', 'PASS2', 'PATHWAY_COMMONS', 'PATMAN', 'PAST', 'PATHWAY_EXPLORER', 'PATHIWAYS', 'PATHEMA', 'PASTA', 'PATHBASE_WS', 'bioconductor-PATHOSTAT', 'PATHOSTAT', 'PARSESNP', 'PATROCLES', 'bioconductor-PASTESEQ', 'bioconductor-PATH2PPI', 'PATH2PPI', 'bioconductor-PATHNET', 'PATHNET', 'bioconductor-PATHVIEW', 'PATHVIEW', 'PATHEXPRESS', 'PASS', 'PATHOGENFINDER', 'PATCHFINDER', 'bioconductor-PATMATMOTIFS', 'PATHICULAR', 'PASH', 'bioconductor-PATHVAR', 'PATHVAR', 'bioconductor-PATHRENDER', 'PATHRENDER', 'PATHWAY_EXTRACTION', 'PATHBLAST', 'PARTICLESTATS', 'PATHWAY_INSPECTOR', 'PATCHWORK', 'PATHPRED', 'PATCHDOCK', 'PACMAN', 'PAML', 'bioconductor-PANP', 'PANP', 'bioconductor-PAIRCOMPVIZ', 'PAIRCOMPVIZ', 'PANDORA', 'PARATOME', 'OVERLAP', 'bioconductor-PALINDROME', 'PAGIT', 'bioconductor-PADOG', 'PADOG', 'P-MATCH', 'P2CS', 'PARAMECIUMDB', 'P53', 'PARALYZER', 'bioconductor-PAPI', 'PAPI', 'bioconductor-PANDAR', 'PANDAR', 'PACOMPLEX', 'PANTHER', 'bioconductor-PAA', 'PAA', 'PARMIGENE', 'PANADA', 'PALMA', 'PANGEA', 'bioconductor-PANVIZGENERATOR', 'PANVIZGENERATOR', 'PAL2NAL', 'PANGAEA', 'PAIR', 'PARIGA', 'PANALYZER', 'PAPPSO', 'PALMAPPER', 'PAP', 'PALS', 'ORCAE', 'ORIONE', 'ORTHODISEASE', 'ORFPREDICTOR', 'OREST', 'OPTITOPE', 'OSCAR', 'ORIDB', 'OREGANNO', 'ORDO', 'ORPHADATA', 'OPENMZXY', 'bioconductor-OPPAR', 'OPPAR', 'OPOSSUM', 'ORTHOSELECT', 'ORPHELIA', 'ORPHANET', 'OTP', 'ORANGE_NETWORK', 'OUT2SUMMARY', 'bioconductor-OPOSSOM', 'OPOSSOM', 'bioconductor-OSAT', 'OSAT', 'bioconductor-OUTLIERD', 'OUTLIERD', 'bioconductor-OTUBASE', 'OTUBASE', 'bioconductor-OSCOPE', 'OSCOPE', 'ORTHODB', 'OPENMS', 'OPTIMUS_PRIMER', 'bioconductor-ORDEREDLIST', 'ORDEREDLIST', 'OPTIMIZER', 'OPM', 'OPEN_PHACTS_EXPLORER', 'bioconductor-ONTOCOUNT', 'bioconductor-ONTOGETROOT', 'ONTOBROWSER', 'OMIXON_VARIANT_TOOLKIT', 'OPEN_BABEL', 'ONE-BLOCK_CYRCA', 'OPAL_WS', 'ONTO-TOOLS', 'OPEN_PHACTS', 'OPAAS', 'OPEN_PHACTS_CHEMISTRY_REGISTRY_SYSTEM', 'bioconductor-ONTOGETOBSOLETE', 'bioconductor-ONTOGET', 'bioconductor-ONTOGETSIBS', 'ONTOQUESTMAIN', 'bioconductor-OMICSMARKER', 'OMICSMARKER', 'OPENHUSAR', 'bioconductor-ONTOGETDOWN', 'bioconductor-ONTOGETUP', 'ONLINE_SBML_VALIDATOR', 'ONTOQUEST_WS', 'bioconductor-ONTOISOBSOLETE', 'bioconductor-ONTOTEXT', 'OMSSA_PARSER', 'OPENCHROM', 'bioconductor-ONECHANNELGU', 'ONECHANNELGU', 'bioconductor-OPENCYTO', 'OPENCYTO', 'bioconductor-ONTOCAT', 'ONTOCAT', 'bioconductor-ONTOGETCOMMON', 'OND-CRF', 'bioconductor-ONCOSCORE', 'ONCOSCORE', 'ONTOLOGIZER', 'OMICBROWSE', 'OLS-DIALOG', 'ODIN', 'bioconductor-OLIN', 'OLIN', 'bioconductor-OCPLUS', 'OCPLUS', 'OGNM', 'OLIGOHEATMAP', 'OGTREE_2', 'O-MINER', 'OLIGOWIZ', 'OCTOPUS', 'OLIGO-ANALYSIS', 'OASIS', 'OLEGO', 'bioconductor-OLIGOCLASSES', 'OLIGOCLASSES', 'NXSENSOR', 'ODB', 'OGDRAW', 'OASES_CLOUDIFB', 'OI_GENE', 'NUTRICHEM', 'bioconductor-ODDCOMP', 'NWDISPLAY-IP', 'OBI-WARP', 'OASES', 'bioconductor-OGSA', 'OGSA', 'OLIGOWALK', 'OLS', 'bioconductor-OCTANOL', 'bioconductor-ODSEQ', 'ODSEQ', 'bioconductor-NUPOP', 'NUPOP', 'bioconductor-OLINGUI', 'OLINGUI', 'OLIGOCALC', 'NVT', 'OGEE', 'NUCLEOS', 'NULLARBOR', 'NORSNET', 'bioconductor-NORETURN', 'bioconductor-NTHSEQSET', 'NOMENCLATURE_WS', 'bioconductor-NUCLER', 'NUCLER', 'NOVOR', 'NSSNPANALYZER', 'NOMAD-REF', 'NUCPRED', 'bioconductor-NOSPACE', 'NOVELSEQ', 'NPS', 'bioconductor-NPGSEA', 'NPGSEA', 'NORSP', 'NQ-FLIPPER', 'NTMG', 'NRG-CING', 'NON-B_DB', 'NONCODE', 'bioconductor-NOTAB', 'bioconductor-NOTSEQ', 'NOISY_ONECLICK-IP', 'bioconductor-NORMR', 'NORMR', 'bioconductor-NUDGE', 'NUDGE', 'bioconductor-NORMQPCR', 'NORMQPCR', 'bioconductor-NTW', 'NTW', 'NPD_WS', 'bioconductor-NTHSEQ', 'bioconductor-NUCLEOSIM', 'NUCLEOSIM', 'NONTARGET', 'NOKOI', 'NOISY-IP', 'NOVOCRAFT', 'bioconductor-NORMALIZE450K', 'NORMALIZE450K', 'NRSAS', 'NRPS-PKS', 'NUCLEARDB', 'NEWCPGREPORT-EBI', 'NMPDR', 'NETWORKIN', 'NGS_SIMULATION-IP', 'NEXTGEN_UTILITY_SCRIPTS', 'NEXTGENE', 'bioconductor-NNNORM', 'NNNORM', 'bioconductor-NOISEQ', 'NOISEQ', 'NOBAI', 'NMSIM', 'NNALIGN', 'bioconductor-NOHTML', 'NGSUTILS', 'NIBR_2D-PAGE', 'NIF_REGISTRY_SEARCH', 'NGS.PLOT', 'NEWT', 'NNDB', 'NETWORK_OF_CANCER_GENES', 'NEXTBIO', 'bioconductor-NEWCPGSEEK', 'NEUROELECTRO', 'bioconductor-NEWSEQ', 'NIMEFI', 'NGL_VIEWER', 'NEXALIGN', 'NGS_BACKBONE', 'NGSVIEW', 'bioconductor-NGSCOPY', 'NGSCOPY', 'NEUROPRED', 'NGSMETHDB', 'NIASGBDB', 'NNCON', 'NEXTPROT', 'NEXTPROT-CLI', 'bioconductor-NEWCPGREPORT', 'NEWBLER', 'NGS-PIPELINE', 'NLPROT', 'NMR_CONSTRAINTS_ANALYSER', 'NEWT-OMICS', 'NETSIM', 'NETCORONA', 'NETCTLPAN', 'NETPHOSYEAST', 'NETCHOP', 'bioconductor-NETSAM', 'NETSAM', 'NETSURFP', 'NETNES', 'NETGLYCATE', 'NETMHCII', 'NETCGLYC', 'NETCTL', 'NETWORK_MOTIF_CLUSTERING_TOOLBOX', 'bioconductor-NETPRIOR', 'NETPRIOR', 'NETCSSP', 'NETMHC', 'NETMHCPAN', 'NETNGLYC', 'NETDISEASESNP', 'NETOGLYC', 'NETPHOSBAC', 'bioconductor-NETPATHMINER', 'NETPATHMINER', 'NETPHOS', 'NETSTART', 'NETPHOSK', 'NETPHOREST', 'NETTURNP', 'NETMHCCONS', 'NETMHCIIPAN', 'NETMHCSTAB', 'NETPICORNA', 'NETTER', 'bioconductor-NETRESPONSE', 'NETRESPONSE', 'bioconductor-NETHET', 'NETHET', 'bioconductor-NETBENCHMARK', 'NETBENCHMARK', 'NETMHC-3.0', 'NETALIGNER', 'NEEDLE-EBI', 'NCBI_GENE', 'NDTREE', 'NETACET', 'bioconductor-NEEDLEALL', 'NEEDLES', 'bioconductor-NEM', 'NEM', 'NASTISEQ', 'NAST', 'NAPS', 'NEGATOME', 'NBRP_RESOURCES', 'NET-GE', 'NCOILS', 'bioconductor-NEEDLE', 'NESONI', 'NEPHELE', 'NCBI_GENOTYPING', 'NCPRED', 'NEEDLE-API-EBI', 'NEBCUTTER', 'NEAT', 'NASSAM', 'NCRNAWSDL', 'NEMATODE.NET', 'NCBI_DBPROBE', 'NEEMO', 'NDB_TOOLS', 'NCBI_GEO', 'NDB', 'NCBI_INFLUENZA_VIRUS_SEQUENCE_ANNOTATION_TOOL', 'NAPP', 'NEEDLE_WS_HUSAR', 'bioconductor-NARROWPEAKS', 'NARROWPEAKS', 'NCBI_GENOME_WORKBENCH', 'bioconductor-NCDFFLOW', 'NCDFFLOW', 'bioconductor-NCIGRAPH', 'NCIGRAPH', 'NCBI_EPIGENOMICS', 'MY_FORENSIC_LOCI_QUERIES', 'MYRNA', 'MZJAVA', 'MZEXTRACT', 'bioconductor-MWCONTAM', 'MYGENE.INFO', 'MUXQUANT', 'MZ2MGF', 'bioconductor-MVGST', 'MVGST', 'bioconductor-NANOSTRINGDIFF', 'NANOSTRINGDIFF', 'MUTSCREENER', 'MVBIODATASIM', 'NAFLEX', 'MZXMLPLOT', 'MYRIALIGN', 'bioconductor-MVCCLASS', 'MVCCLASS', 'bioconductor-NANOSTRINGQCPRO', 'NANOSTRINGQCPRO', 'bioconductor-MZR', 'MZR', 'MUTS-NEEDLE-PLOT', 'bioconductor-MWFILTER', 'MYRIMATCH', 'MZXML_VIEWER', 'MZSTAR', 'bioconductor-MZID', 'MZID', 'bioconductor-MUTATIONALPATTERNS', 'MUTATIONALPATTERNS', 'MUTDB', 'MYHITS', 'MYBS', 'NANNOCHOLOPSIS_GADITANA_DATABASE', 'MVIEW_API_EBI', 'MVIEW_EBI', 'MVDA', 'MUTATIONMOTIF', 'MZMINE', 'MUTECT', 'N-GLYCO', 'MYEXPERIMENT', 'MUTALYZER_WS', 'MSV3D', 'MUSCLE-IP', 'MUMRESCUELITE', 'MUSICA_2', 'MUTASCOPE', 'bioconductor-MULTICLUST', 'MULTICLUST', 'MULTIPLX', 'MULTIDISP', 'MSQUANT', 'MTR', 'MUMMER', 'bioconductor-MULTIASSAYEXPERIMENT', 'MULTIASSAYEXPERIMENT', 'MUSCLE_WS_JABAWS', 'MULTIPHYL', 'MULTI_EXPERIMENT_MATRIX', 'MSQROB', 'MULTIFASTA_SEQUENCE_EXTRACTOR-IP', 'MULTIFASTAMIX-IP', 'MULTIPLIERZ', 'MU2A', 'MUMMERGPU', 'bioconductor-MULTTEST', 'MULTTEST', 'MULTIFIT', 'MUPLEX', 'MUD', 'MUSCLE_EBI', 'bioconductor-MULTIDATASET', 'MULTIDATASET', 'MULTI-Q', 'MULTIPSQ', 'MULTIQC', 'bioconductor-MULTISCAN', 'MULTISCAN', 'MULTIBIND', 'MSSTATS', 'bioconductor-MULCOM', 'MULCOM', 'MULTI-HARMONY', 'MSCONVERT', 'MSIREADER', 'MS-ISOTOPE', 'bioconductor-MSPURITY', 'MSPURITY', 'bioconductor-MSNBASE', 'MSNBASE', 'bioconductor-MSGFGUI', 'MSGFGUI', 'MS-LIMS', 'MSPIRE-SIMULATOR', 'MS-GFDB', 'MSPOLYGRAPH', 'MS-SPECTRE', 'MS-DIGEST', 'MSPIRE_MSPIRE-SEQUEST', 'MSALIGN2', 'bioconductor-MSGFPLUS', 'MSGFPLUS', 'MSAVIEWER', 'bioconductor-MSBAR', 'MS2PREPROC', 'MSACCESS', 'bioconductor-MSNID', 'MSNID', 'MSEA', 'MSAPROBS', 'MS-FIT', 'MS-GF', 'MSIMAGEVIEW', 'MSGRAPH', 'MSGFDB2PEPXML', 'MSALIGN', 'MRSFAST', 'MRS', 'MSFILEREADER_PYTHON_BINDINGS', 'MSD_FILE_READER', 'MPAGENOMICS', 'MOTIFSUITE', 'bioconductor-MOTIFDB', 'MOTIFDB', 'bioconductor-MOTIFRG', 'MOTIFRG', 'MPI_TOOLKIT', 'MOTIFVIZ', 'MOUSEINDELDB', 'MOUSE_PROTEOME_PROJECT', 'MRC_IMAGE_PROCESSING_PACKAGE', 'MPILEUP-IP', 'bioconductor-MQTL.NMR', 'MQTL.NMR', 'bioconductor-MOTIFBREAKR', 'MOTIFBREAKR', 'MRCANAVAR', 'MOTIF3D', 'MOVIEMAKER', 'MPD', 'MOTUS', 'MRBUMP', 'MOTIFLAB', 'bioconductor-MOTIV', 'MOTIV', 'MRES2X', 'MOVIES', 'MPROMDB', 'MPA', 'MOSFLM', 'MRBWS', 'MRBAYES', 'MPILEUP_VCF-IP', 'MOSAIK', 'bioconductor-MPFE', 'MPFE', 'MOWSERV', 'MPIUTILS', 'MPSCAN', 'MRFAST', 'MPLOT', 'MOUSEBOOK', 'MOFF', 'MODENCODE_PEAKCALLING_MACS2-IP', 'MONSTER', 'MOLSURFER', 'MOLGENIS_COMPUTE', 'MONOSACCHARIDEDB', 'MODULEORGANIZER', 'MODIFYGENOMICCOORDINATES-IP', 'bioconductor-MONOCLE', 'MONOCLE', 'MODELLER', 'MORPH', 'MODULEDIGGER', 'MOLECULAR_WEIGHT_CALCULATOR', 'bioconductor-MOONLIGHTR', 'MOONLIGHTR', 'MOLMOVDB', 'MONETFAMILY', 'MOLEONLINE_2.0', 'MORCA', 'MODORAMA', 'MODLINK', 'MOLLIGEN', 'MODIFYSEQUENCELIST-IP', 'MODPROPEP', 'MOLLOC', 'MOLREP', 'MODERNA', 'MOLMETH', 'bioconductor-MOSAICS', 'MOSAICS', 'MOM', 'bioconductor-MOPS', 'MOPS', 'MOLGENIS', 'MOLPROBITY', 'MOLAXIS', 'MODELTEST', 'MODI', 'MODFOLD4', 'MOMA-LIGPATH', 'MMSINC', 'MKESA', 'MLGT', 'MOCHIVIEW', 'MMASS', 'bioconductor-MMNET', 'MMNET', 'bioconductor-MLINTERFACES', 'MLINTERFACES', 'bioconductor-MLSEQ', 'MLSEQ', 'MITOPRED', 'MITOGENESISDB', 'MMSSIMILARITY', 'MITOZOA', 'MIXOMICS', 'MKDOM2', 'bioconductor-MLP', 'MLP', 'MOBILOMEFINDER', 'MOD_TOOLS', 'MOBIDB', 'MKT', 'MITOMINER', 'MOCAPY_PLUS_PLUS', 'MODBASE', 'bioconductor-MODA', 'MODA', 'MMIA', 'MISTIC', 'MMDB', 'MMSDUSTY', 'MLST-DB', 'bioconductor-MMDIFF2', 'MMDIFF2', 'MOBIDB_LITE', 'MMMDB', 'MOBI', 'MOAN', 'MLST', 'MMETA', 'MIX', 'MLIBRARY', 'MMSEQ', 'bioconductor-MMPALATEMIRNA', 'MMPALATEMIRNA', 'MOCAT', 'MIRNAKEY', 'MIRSPRING', 'MISO', 'bioconductor-MISSMETHYL', 'MISSMETHYL', 'MIRCONNX', 'MIRBASE', 'MIRIAM', 'MIRMAP', 'bioconductor-MIRLAB', 'MIRLAB', 'MIRA', 'MIRCAT', 'MIRPROF', 'bioconductor-MIRINTEGRATOR', 'MIRINTEGRATOR', 'MIRZ', 'MIRU-VNTRPLUS', 'MIRU', 'MIRVESTIGATOR', 'MIRORTHO', 'MIRANALYZER', 'MIRTOOLS', 'MIRTRIOS', 'bioconductor-MIRNAMECONVERTER', 'MIRNAMECONVERTER', 'bioconductor-MIRAGE', 'MIRAGE', 'MIRMAID', 'MIREDUCE', 'MIRGATOR_V2.0', 'MIST', 'MIRNEST', 'MISMAX', 'MIRGEN', 'MIRIAM_WS', 'MIRDEEP', 'bioconductor-MIRSYNERGY', 'MIRSYNERGY', 'MIRACT', 'MIRTARBASE', 'MICROGRAPHITE', 'MIGENAS_TOOLKIT', 'MINER', 'MINT', 'MICROPAN', 'bioconductor-MICHIP', 'MICHIP', 'MHCPRED', 'MIDAW', 'MIMODB', 'MICROBE_BROWSER', 'MINDTHEGAP', 'MHCCLUSTER', 'MICROPITA-IP', 'MIP_SCAFFOLDER', 'bioconductor-MICRORNA', 'MICRORNA', 'bioconductor-MINIMUMDISTANCE', 'MINIMUMDISTANCE', 'bioconductor-MINFI', 'MINFI', 'MICROMUMMIE', 'MICROKIT', 'MICROARRAY_RETRIEVER', 'MINIA', 'MICROBIAL_ADVANCED_DATABASE_ORGANISATION_MICADO', 'MICRORAZERS', 'MICSA', 'bioconductor-MINEICA', 'MINEICA', 'bioconductor-MINET', 'MINET', 'MIPRED', 'MICROSCOPEMAGE', 'bioconductor-MIPP', 'MIPP', 'MICCA', 'MINACTIONPATH', 'MICROINSPECTOR', 'MICROFOOTPRINTER', 'MICHECK', 'MICROBESONLINE', 'MIPMODDB', 'MINAS', 'MINEPATH', 'MFUNGD', 'MGI', 'MGX', 'METHPIPE', 'bioconductor-METHYLPIPE', 'METHYLPIPE', 'bioconductor-METHPED', 'METHPED', 'bioconductor-METHYLKIT', 'METHYLKIT', 'bioconductor-METHYLSEEKR', 'METHYLSEEKR', 'bioconductor-MGFR', 'MGFR', 'MGIP', 'METSTAT', 'METITREE', 'METHMARKER', 'bioconductor-METHVISUAL', 'METHVISUAL', 'MFOLD', 'MGENE.WEB', 'MGD', 'METFRAG', 'bioconductor-METHYLMIX', 'METHYLMIX', 'bioconductor-MFUZZ', 'MFUZZ', 'MGTOOLS', 'bioconductor-METHTARGETEDNGS', 'METHTARGETEDNGS', 'MFPAQ', 'MEV', 'bioconductor-METHYLUMI', 'METHYLUMI', 'METFUSION', 'MFEPRIMER-2.0', 'METHYLOMEDB', 'MFSEARCHER', 'MEXPRESS', 'METHYLCODER', 'MG-RAST', 'bioconductor-MGFM', 'MGFM', 'bioconductor-METHYANALYSIS', 'METHYANALYSIS', 'bioconductor-MGSA', 'MGSA', 'MGENOMESUBTRACTOR', 'METABOLIGHTS', 'METABOLOMEXCHANGE', 'bioconductor-METAGENOMEFEATURES', 'METAGENOMEFEATURES', 'METABOANALYST_2.0', 'METABIOME', 'METACYC', 'METASTUDENT', 'bioconductor-METACCA', 'METACCA', 'METAGENOME-KIN', 'METAXA', 'bioconductor-METABOMXTR', 'METABOMXTR', 'METAPATH_ONLINE', 'METALPDB', 'METADISORDER', 'METASPADES-IP', 'bioconductor-METAGENOMESEQ', 'METAGENOMESEQ', 'METAPROTEOMEANALYZER', 'METAGENASSIST', 'METASIM', 'bioconductor-METAHDEP', 'METAHDEP', 'bioconductor-METASEQ', 'METASEQ', 'bioconductor-METAMS', 'METAMS', 'METEXPLORE', 'METASHARK', 'bioconductor-METCIRC', 'METCIRC', 'METAREP', 'METALS2', 'bioconductor-METABOSIGNAL', 'METABOSIGNAL', 'METALDETECTOR', 'METAMHC', 'METACROP', 'MENTHA', 'MENDELIAN', 'MESALINA', 'MENU-BUILDER', 'META-PIPE', 'META', 'MERGETRANSCRIPTLISTS-IP', 'MEMOTIF', 'MEROPS_BLAST', 'bioconductor-MERGER', 'bioconductor-METAARRAY', 'METAARRAY', 'bioconductor-METAB', 'METAB', 'MESADB', 'MESORD', 'bioconductor-MERGEOMICS', 'MERGEOMICS', 'bioconductor-MESHES', 'MESHES', 'MERACULOUS', 'META-SNP', 'META-GECKO', 'METABASE', 'MERGESLIDINGWINDOWSCLUSTERS-IP', 'MERGINGDATACLUSTERIZE-IP', 'bioconductor-MERGEMAID', 'MERGEMAID', 'MEMPYPE', 'bioconductor-MEGAMERGER', 'MEMHDX', 'bioconductor-MEDME', 'MEDME', 'bioconductor-MEASUREMENTERROR.COR', 'MEASUREMENTERROR.COR', 'MEDLINERANKER', 'MEM', 'MEGX.NET', 'MEME_SUITE_CLOUDIFB', 'MELANIE', 'MEME_FIMO-IP', 'MEGRAFT', 'bioconductor-MEIGOR', 'MEIGOR', 'MEMO', 'MEMOIR', 'MEMLOCI', 'bioconductor-MEDIPS', 'MEDIPS', 'MEGAN', 'MDMOBY', 'MDWEB', 'MELINAII', 'MELTDB', 'MCPEP', 'MEMBPLUGIN', 'MELANOMAMINE', 'MEERKAT', 'bioconductor-MCRESTIMATE', 'MCRESTIMATE', 'MEME_SUITE', 'MDSCAN', 'MEDUSA', 'MEGAMERGER_WS_HUSAR', 'MDR', 'MEME_MEME-IP', 'bioconductor-MDQC', 'MDQC', 'MEDOCK', 'MAXCOUNTS', 'MAXSPROUT', 'MAUVE', 'MASSYPUP', 'bioconductor-MCAGUI', 'MCAGUI', 'MASTR', 'MATCHER-API-EBI', 'bioconductor-MATTER', 'MATTER', 'MAYU', 'bioconductor-MBASED', 'MBASED', 'MAXIC-Q', 'MAXOCC', 'MATCHER_WS_HUSAR', 'MASTER2PGSNP-IP', 'bioconductor-MBTTEST', 'MBTTEST', 'MASSWOLF', 'MAYDAY', 'bioconductor-MATRIXRIDER', 'MATRIXRIDER', 'MBGD', 'MATRIXDB', 'MBINFO', 'MBLASTALL-IP', 'MASSXPERT', 'MAXIMUS', 'bioconductor-MBAMETHYL', 'MBAMETHYL', 'MATRIX-CLUSTERING', 'MCPATH', 'bioconductor-MATCHER', 'MATRIXCLUST', 'MBWA_WRAPPER-IP', 'MAXQUANT', 'bioconductor-MBCB', 'MBCB', 'MAVID', 'MATRAS', 'MAPSEMBLER', 'MARNA', 'MASS-UP', 'MASCOTDATFILE', 'MASCOT_PEPTIDE_MASS_FINGERPRINT', 'bioconductor-MARRAY', 'MARRAY', 'bioconductor-MARSCAN', 'bioconductor-MASKSEQ', 'MASCOT_IONS_SEARCH', 'MASSMATRIX', 'MASSHUNTER_FILE_READER', 'MASSIFQUANT', 'MAQ', 'MAPPIS', 'bioconductor-MASKAMBIGPROT', 'bioconductor-MASKFEAT', 'MASSWIZ', 'bioconductor-MASIGPRO', 'MASIGPRO', 'bioconductor-MASKBAD', 'MASKBAD', 'bioconductor-MASSARRAY', 'MASSARRAY', 'MARKUS', 'MASSSPECAPI', 'bioconductor-MASKAMBIGNUC', 'MAPPERANALYZER-IP', 'MASSAI', 'MAQGENE', 'MASON', 'bioconductor-MASSSPECWAVELET', 'MASSSPECWAVELET', 'bioconductor-MAPREDICTDSC', 'MAPREDICTDSC', 'MASSNET', 'MAPPINGHSP4TAXO-IP', 'MARKDUPLICATES_IP', 'MASSCHROQ', 'MAPQUANT', 'MAPSPLICE', 'MASSTRIX', 'MAPPERDB', 'MARQ', 'MAGECOMET', 'MALARIA_DATA_SMALL_MOLECULE_SEARCH', 'bioconductor-MAKEPROTSEQ', 'bioconductor-MADSEQ', 'MADSEQ', 'bioconductor-MANOR', 'MANOR', 'bioconductor-MAFTOOLS', 'MAFTOOLS', 'MAKER', 'MAGNOLIA', 'MAGIA2', 'MAPI', 'MAGALLANES', 'MALARIA_DATA_BLAST_SEARCH', 'MAPMI', 'MAGIC', 'MAKETSSDIST-IP', 'bioconductor-MAPKL', 'MAPKL', 'MAGMA', 'MAGIIC-PRO', 'MAMMALIAN_GENE_COLLECTION', 'MAFFT_API', 'bioconductor-MAKENUCSEQ', 'MAM-PIPELINE', 'MAKE_RANDOM', 'bioconductor-MAIGESPACK', 'MAIGESPACK', 'bioconductor-MANTELCORR', 'MANTELCORR', 'MADNET', 'MAIZEGDB', 'MAPDAMAGE', 'bioconductor-MAKECDFENV', 'MAKECDFENV', 'bioconductor-MADE4', 'MADE4', 'MAISTAS', 'MAKESTATSCHIPSEQ-IP', 'MAGICVIEWER', 'bioconductor-MANTA', 'MANTA', 'MAGNET', 'MACSIMS', 'LOCTREE3', 'bioconductor-LYMPHOSEQ', 'LYMPHOSEQ', 'bioconductor-LOLA', 'LOLA', 'bioconductor-LOGICFS', 'LOGICFS', 'LUCY', 'LUX_SCORE', 'LOCTREE2', 'MACSYFINDER', 'bioconductor-M3DROP', 'M3DROP', 'bioconductor-LOGITT', 'LOGITT', 'bioconductor-LPE', 'LPE', 'bioconductor-LVSMIRNA', 'LVSMIRNA', 'bioconductor-MACORRPLOT', 'MACORRPLOT', 'LOWCOMPLEXITY', 'LOCATE', 'LOGOL', 'MACS', 'bioconductor-MACAT', 'MACAT', 'bioconductor-LPSYMPHONY', 'LPSYMPHONY', 'bioconductor-LPNET', 'LPNET', 'bioconductor-M3D', 'M3D', 'LTR_FINDER', 'LYNX_MULTI-GENE_ANNOTATIONS', 'LOOKSEQ', 'LSKB', 'bioconductor-LUMI', 'LUMI', 'M4T', 'MADAP', 'LOCDB', 'LOVD', 'MA', 'LUTEFISK', 'bioconductor-MAANOVA', 'MAANOVA', 'bioconductor-LOL', 'LOL', 'bioconductor-LPEADJ', 'LPEADJ', 'LOCTARGET', 'LSD', 'LOCARNA-P', 'LIST_ONTOLOGIES', 'LOBSTR', 'bioconductor-LOBSTAHS', 'LOBSTAHS', 'LITINSPECTOR', 'LOCALMOVE', 'LNCRNADB', 'LOBO', 'LIVERMORE_METAGENOMICS_ANALYSIS_TOOLKIT', 'bioconductor-LINC', 'LINC', 'LOC3D', 'LIPID_MAPS', 'LIPIDXPLORER', 'LIMMARP', 'LIPOP', 'LISTANNOTATION-IP', 'bioconductor-LIQUIDASSOCIATION', 'LIQUIDASSOCIATION', 'bioconductor-LMDME', 'LMDME', 'LNATOOLS', 'LITMINER', 'LOCALIZOME', 'LISE', 'LIS', 'LOCARNA', 'bioconductor-LISTOR', 'LIPIDCREATOR', 'LIMTOX', 'LNCIPEDIA', 'bioconductor-LINDNA', 'LOAD_MULTIFASTQ-IP', 'LISTINPUTS-IP', 'bioconductor-LINNORM', 'LINNORM', 'LOCAS', 'bioconductor-LMGENE', 'LMGENE', 'LEBIBIPQP', 'LEMONE', 'LGICDB', 'KRONAEXTRACT-IP', 'LA-IMAGES', 'LASERGENE', 'LAST', 'LIBMGF', 'LGA', 'LAILAPS', 'LALIGN', 'LIMANI', 'LEPROMA', 'LIFTOVER', 'LALIGN_API', 'bioconductor-LFA', 'LFA', 'LAHEDES', 'LEGUMEIP', 'LAILAPSIPK', 'LIGSEARCH', 'LCMS_TNO_DECO', 'LCA-IP', 'LAB7', 'LAMININ_DATABASE', 'LEON', 'LEPIDODB', 'LIBFID', 'bioconductor-LES', 'LES', 'bioconductor-LBE', 'LBE', 'bioconductor-LAPMIX', 'LAPMIX', 'LIMIX', 'KUPS', 'LIGHTSTRINGGRAPH', 'LEGIOLIST', 'LAJ_1-IP', 'LIBIPACA', 'LIBRA', 'LIBMPRC', 'LASTZ', 'KEY_PATHWAY_MINER', 'KISMETH', 'KMERGENIE', 'bioconductor-KEGGGRAPH', 'KEGGGRAPH', 'KINASE_SARFARI_BLAST_SEARCH', 'KNIME4BIO', 'bioconductor-KEGGPROFILE', 'KEGGPROFILE', 'KINDOCK', 'bioconductor-KEGGORTHOLOGY', 'KEGGORTHOLOGY', 'bioconductor-KIMOD', 'KIMOD', 'KOBAS', 'KINEFOLD', 'KMER-SVM', 'KINASEPHOS', 'KINASE_SARFARI_SMALL_MOLECULE_SEARCH', 'KEGGANIM', 'KMX', 'KRAKEN', 'KFC', 'KINARI-WEB', 'KOBAMIN', 'KEESEEK', 'KMERFINDER', 'KHMER', 'KLIFS', 'bioconductor-KEGGLINCS', 'KEGGLINCS', 'KRONA', 'KOSMOS', 'KINASEPHOS_2.0', 'KEGG', 'KOMODO', 'JOINTSLM', 'KBASE', 'KAPPA-VIEW4', 'KCAM', 'JPHMM', 'JIFFYNET', 'K-FOLD', 'KAAS', 'JPRED_3', 'JPHYLOIO', 'JVARKIT', 'JSQUID', 'bioconductor-JODA', 'JODA', 'JPROGO', 'KARMA', 'KD4V', 'JPREDICTOR', 'JORCA', 'KALIGN_API_EBI', 'JMZML', 'JSPARKLINES', 'bioconductor-JUNCTIONSEQ', 'JUNCTIONSEQ', 'JRONN_WS_JABAWS', 'JPRED_WS_JABAWS', 'JVIRGEL', 'JGI_GENOME_PORTAL', 'KALIGN', 'KBWS', 'KB-RANK', 'KAIJU', 'JTRAML', 'bioconductor-KEBABS', 'KEBABS', 'JELLYFISH', 'ISOTOP', 'ISOTOPE_CALCULATOR', 'bioconductor-ITALICS', 'ITALICS', 'ISPIDER_CENTRAL', 'JASPAR', 'IUPRED_WS_JABAWS', 'ISOFINDER', 'ISPRED', 'ISOSCALE', 'ISOCHORE-EBI', 'ITSONEDB', 'bioconductor-JASPEXTRACT', 'bioconductor-ISOCHORE', 'ISOEM', 'JBROWSE', 'bioconductor-ISOGENEGUI', 'ISOGENEGUI', 'bioconductor-ITERATIVEBMASURV', 'ITERATIVEBMASURV', 'ISOTOPICA', 'JCAT', 'ISTECH_SNPANALYZER', 'ISOTOP_FS', 'bioconductor-ISOMIRS', 'ISOMIRS', 'JACKHMMER', 'ITS2', 'IUPHAR-DB', 'ITOL', 'JASPARDB', 'bioconductor-ISOBAR', 'ISOBAR', 'bioconductor-ISOLDE', 'ISOLDE', 'ISOBASE', 'JALI', 'bioconductor-JASPSCAN', 'ISOTOPE.PY', 'bioconductor-ITERATIVEBMA', 'ITERATIVEBMA', 'ISOBLAT', 'ISSAKE', 'ISPOT', 'IREFSCAPE', 'IRBIS', 'IPSA', 'ISO2L', 'bioconductor-IPPD', 'IPPD', 'bioconductor-INVERSION', 'INVERSION', 'IPC', 'IPI', 'ISARST', 'IPD', 'IPAVS', 'ISMARA', 'IREFR', 'IONISER', 'bioconductor-ISEQ', 'ISEQ', 'IOBIO', 'ISAAC', 'IPDA', 'INTSCORE', 'IQSEQ', 'bioconductor-IRANGES', 'IRANGES', 'IPBA', 'INTERPROSCAN_WS', 'INTOGEN', 'IREFINDEX', 'bioconductor-IPO', 'IPO', 'ISA_TOOLS', 'bioconductor-IPAC', 'IPAC', 'INTFOLD', 'ISACGH', 'ISFINDER', 'IOMICS', 'ISAS', 'bioconductor-IONTREE', 'IONTREE', 'INTREPID', 'bioconductor-INFOSEQ', 'bioconductor-INFOBASE', 'INNATEDB', 'INGAP', 'bioconductor-INTERACTIVEDISPLAY', 'INTERACTIVEDISPLAY', 'INSIGNIA', 'INPS-MD', 'INTEROLOG', 'INTEREVOL', 'INTENZ', 'INTARNA', 'INFORNA', 'INTEGRON_FINDER', 'INFO-RNA', 'INSERM-REMAP', 'INTERPRO', 'INTERPROSCAN_4', 'INFOALIGN_WS_HUSAR', 'bioconductor-INFORESIDUE', 'INTERFEROME', 'INMEX', 'INSDC', 'bioconductor-INFOASSEMBLY', 'INTERELEMENTGFF-IP', 'bioconductor-INTERACTIVEDISPLAYBASE', 'INTERACTIVEDISPLAYBASE', 'bioconductor-INPOWER', 'INPOWER', 'bioconductor-INSPECT', 'INSPECT', 'INGA', 'INTACT', 'INTERMAP3D', 'INTERMINE', 'bioconductor-INFOALIGN', 'INPARANOID', 'INGENUITY_VARIANT_ANALYSIS', 'INGAP-SV', 'IMAGEJ_CLOUDIFB', 'INFERNAL_CMSCAN', 'IMPC', 'IMGT_HLA-SEQUENCE_ALIGNMENT', 'IMIR', 'IHOP_WS', 'IMAGE', 'INDIGESTION', 'bioconductor-IMMUNESPACER', 'IMMUNESPACER', 'IMEX', 'IKMC', 'IMGT', 'IMOD', 'IMDEV', 'ILM', 'bioconductor-IMMUNOCLUST', 'IMMUNOCLUST', 'bioconductor-IMPUTE', 'IMPUTE', 'INCLUSIVE', 'INDELSCAN', 'IMG', 'IMAAAGINE', 'IMSDB', 'INB-BSC_GENOMICS_CLOUD', 'IGV', 'ILLUMINATE', 'INDICES_DIVERSITY-IP', 'bioconductor-ILLUMINAIO', 'ILLUMINAIO', 'IMGT_HLA', 'IMAGIC', 'ILOOPS', 'INCHI', 'ILLUMINATOR', 'bioconductor-IMPULSEDE', 'IMPULSEDE', 'IMOT', 'IMP', 'INDELFR', 'IDBP', 'IDBASES', 'ICTV', 'IDCONVERT', 'IDCALC', 'bioconductor-ICOBRA', 'ICOBRA', 'bioconductor-IDMAPPINGRETRIEVAL', 'IDMAPPINGRETRIEVAL', 'ICGEBNET_PROTEIN_TOOLS', 'ICM_WEB', 'ICSNPATHWAY', 'ICR', 'IEDB', 'IDEAL', 'bioconductor-IEP', 'ICORN', 'ICPLQUANT', 'ICPL_ESIQUANT', 'bioconductor-ICHECK', 'ICHECK', 'IDT_SCITOOLS', 'IEDB-AR', 'IGBLAST', 'IDTARGET', 'IDBA', 'ID-ALIGN', 'IGDD', 'IELM', 'ICGC_DATA_PORTAL', 'IDENTIFIERS.ORG_RESOLVER', 'bioconductor-IDMAPPINGANALYSIS', 'IDMAPPINGANALYSIS', 'IGDISCOVER', 'ICLC', 'ID_HISTORY_CONVERTER', 'bioconductor-ICHIP', 'ICHIP', 'bioconductor-ICLUSTERPLUS', 'ICLUSTERPLUS', 'bioconductor-IDIOGRAM', 'IDIOGRAM', 'bioconductor-ICENS', 'ICENS', 'bioconductor-IDEOVIZ', 'IDEOVIZ', 'IGB', 'I-MUTANT', 'bioconductor-IASEQ', 'IASEQ', 'I2I-SITEENGINE', 'HUBBA', 'I-ADHORE', 'ICELOGO', 'HUMAN_SPLICING_FINDER', 'IBM_PATTERN_DISCOVERY', 'I-GSEA4GWAS', 'IBWS_WS', 'ICCARE', 'HYPERGENY', 'IBIS', 'bioconductor-ICARE', 'ICARE', 'bioconductor-HYPERDRAW', 'HYPERDRAW', 'bioconductor-HYPERGRAPH', 'HYPERGRAPH', 'I-MUTANT2.0', 'HYPERCLDB', 'HUGE', 'HYBRID-SHREC', 'IC50-TO-KI', 'HUPHO', 'HTSEQCOUNT-IP', 'HTSEQ', 'HYDROGEN_BONDIFIER', 'HX-EXPRESS', 'bioconductor-HYBRIDMTEST', 'HYBRIDMTEST', 'bioconductor-IBMQ', 'IBMQ', 'bioconductor-HTSFILTER', 'HTSFILTER', 'bioconductor-HTSEQTOOLS', 'HTSEQTOOLS', 'bioconductor-IBH', 'IBH', 'I2D', 'HUBMED', 'ICEBERG', 'bioconductor-HOPACH', 'HOPACH', 'HMMSIM_GENOUEST', 'HOGENOM', 'HOTPOINT', 'HPRD', 'HSSP', 'HOTNET', 'HMMGENE', 'HTML4BLAST-IP', 'bioconductor-HPAR', 'HPAR', 'bioconductor-HMMCOPY', 'HMMCOPY', 'HMMEMIT', 'HRTBLDB', 'HORA', 'HMS-ICS', 'HPLUSPLUS', 'HOMOLOSAMPLER', 'bioconductor-HTQPCR', 'HTQPCR', 'HMMSCAN_GENOUEST', 'HPO', 'HOMER-M', 'HMMSPLICER', 'HPEAK', 'HMMFETCH_GENOUEST', 'HMMSEARCH_GENOUEST', 'HOTREGION', 'HOMER-A', 'HMMSCAN_EBI', 'HOVERGEN', 'bioconductor-HMOMENT', 'HOGENOM_TREE', 'HOGENOM_EVOSCEN', 'HPLCSIMULATOR', 'HMMSTAT_GENOUEST', 'HOLOVIR', 'HOMOZYGOSITYMAPPER', 'HITEC', 'bioconductor-HIREADSPROCESSOR', 'HIREADSPROCESSOR', 'bioconductor-HILBERTVISGUI', 'HILBERTVISGUI', 'HMMALIGN_GENOUEST', 'bioconductor-HILBERTVIS', 'HILBERTVIS', 'bioconductor-HITC', 'HITC', 'bioconductor-HIERGWAS', 'HIERGWAS', 'HMMCONVERT_GENOUEST', 'HHPRED', 'HIPATHDB', 'HGVBASE', 'HHSENSER', 'HGV_DAVID-IP', 'HGV_LINKTOGPROFILE-IP', 'HI', 'HICUP', 'HINT', 'HIPIPE', 'HGV_GPASS-IP', 'bioconductor-HIBAG', 'HIBAG', 'HMMBUILD_GENOUEST', 'HITRACE-WEB', 'HMDB', 'HIC-PRO', 'HGV_LPS-IP', 'HGV_LDTOOLS-IP', 'HGV_PASS-IP', 'HGV_SIFT-IP', 'HITPREDICT', 'HHOMP', 'HHREP', 'HLUNGDB', 'HHMD', 'HIERARCHY_WEB_SERVICES', 'HISTONE_CODER', 'HLARESTRICTOR', 'HIT', 'HLA_BIND', 'HISTOME', 'HFAIM', 'HEMAEXPLORER', 'HAPCOL', 'HETEROMIRPRED', 'HALVADE', 'HGV_ADD_SCORES-IP', 'HAPLOWSER', 'HELISPHERE', 'HEXSERVER', 'HARMONY', 'HEXPOCHEM', 'HAWKEYE', 'HGV_BEAM-IP', 'bioconductor-HELP', 'HELP', 'HFV', 'bioconductor-HELIXTURNHELIX', 'bioconductor-HARMAN', 'HARMAN', 'bioconductor-HEM', 'HEM', 'bioconductor-HARSHLIGHT', 'HARSHLIGHT', 'HGMD', 'HAPCLUSTER', 'HAMAP', 'HAIRPIN_ANNOTATION', 'HGV_CODINGSNPS-IP', 'bioconductor-HELLORANGES', 'HELLORANGES', 'HDX_WORKBENCH', 'bioconductor-HEATPLUS', 'HEATPLUS', 'HGD', 'HGCA', 'HGNC', 'bioconductor-HDF5ARRAY', 'HDF5ARRAY', 'HGPD', 'GWASPI', 'GSNAP-IP', 'bioconductor-GSEALM', 'GSEALM', 'bioconductor-GSRI', 'GSRI', 'bioconductor-GWASCAT', 'GWASCAT', 'GUIDANCE', 'GWIDD', 'GRS', 'bioconductor-GWASTOOLS', 'GWASTOOLS', 'bioconductor-GSREG', 'GSREG', 'HADDOCK', 'GSEA', 'GSRC', 'GUIDE_TO_PHARMACOLOGY', 'bioconductor-GSALIGHTNING', 'GSALIGHTNING', 'bioconductor-GVIZ', 'GVIZ', 'bioconductor-GSEABASE', 'GSEABASE', 'GWAS3D', 'H-INVDB', 'GXA_EXPT', 'GWIPS-VIZ', 'H-DBAS', 'GUUGLE', 'GUILDIFY', 'GUILD', 'GROMACS', 'GSNAP', 'bioconductor-GSCA', 'GSCA', 'bioconductor-GSVA', 'GSVA', 'bioconductor-GUIDESEQ', 'GUIDESEQ', 'GYDB', 'GSA-SNP', 'GWASDB', 'GRIDSS', 'bioconductor-GRAPH', 'GRAPH', 'bioconductor-GRAPHALIGNMENT', 'GRAPHALIGNMENT', 'GPS', 'GREP_COUNT-IP', 'bioconductor-GRAPHPAC', 'GRAPHPAC', 'GRABCAS', 'GREENPHYLDB', 'GPSDB', 'GPSEQ', 'bioconductor-GPLS', 'GPLS', 'GREAT', 'GRAPE', 'GRAPE_2.0', 'GRAPHWEB', 'GPCRS', 'GREATSCANPATTERNS', 'GREATSCANMULTIPATTERNS', 'GPMDB_API', 'bioconductor-GRMETRICS', 'GRMETRICS', 'GREYLAG', 'bioconductor-GPREGE', 'GPREGE', 'bioconductor-GRAPHAT', 'GRAPHAT', 'GPCRSCLASS', 'GPU.PROTON.DOCK', 'GRAMM-X', 'GRAPE_DATABASE', 'GRAPH_EXTRACT', 'GRIFFIN', 'GRAPHITEWEB', 'GPSY', 'GRAMENE', 'GOSUMMARIES', 'GPCRHMM', 'bioconductor-GOPROFILES', 'GOPROFILES', 'GOLD', 'bioconductor-GOTHIC', 'GOTHIC', 'GO_TERM_MAPPER', 'GORILLA', 'GPCRPRED', 'GORTH', 'bioconductor-GONAME', 'GOBY_FRAMEWORK', 'GOLDEN_HELIX', 'bioconductor-GOPRO', 'GOPRO', 'bioconductor-GOFUNCTION', 'GOFUNCTION', 'bioconductor-GOTOOLS', 'GOTOOLS', 'GOMINER_WS', 'bioconductor-GOOGLEGENOMICS', 'GOOGLEGENOMICS', 'GOGENE', 'GO_TERM_FINDER', 'GOPUBMED', 'GOMAPMAN', 'GOWINDA', 'GOA_GUBBAR', 'GOBLET', 'GOEAST', 'GOA', 'GPCR_SARFARI_BLAST_SEARCH', 'GPCR_SARFARI_SMALL_MOLECULE_SEARCH', 'bioconductor-GODEF', 'bioconductor-GOSEQ', 'GOSEQ', 'GONUTS', 'GMAP-IP', 'GK_ARRAYS', 'bioconductor-GLOBALANCOVA', 'GLOBALANCOVA', 'GLIMMER', 'GLYCOMEDB', 'GLYCOSCIENCES', 'GMAP_IIT_STORE-IP', 'bioconductor-GLIMMA', 'GLIMMA', 'GLPROBS_WS_JABAWS', 'GLYCOFRAGMENT', 'GLOBPLOTTER', 'GITOOLS', 'GMAP', 'GNUMAP', 'GLYCOMOD', 'bioconductor-GLAD', 'GLAD', 'bioconductor-GIRAFE', 'GIRAFE', 'bioconductor-GLOBALTEST', 'GLOBALTEST', 'GLOBPLOT_WS_JABAWS', 'GLYPROT', 'GLYCOSEARCHMS', 'GMS', 'GMAP_BUILD-IP', 'GINSIM', 'GIS', 'GLYCOVIEWER', 'GNPIS', 'GMD-R', 'GMD', 'GMAP_SNPINDEX-IP', 'GIMMEMOTIFS', 'bioconductor-GLOBALSEQ', 'GLOBALSEQ', 'bioconductor-GMRP', 'GMRP', 'GO_ENRICHMENT_TOOL', 'GLOBPLOT', 'GLAMM', 'GLYNEST_AND_CASPER', 'GO', 'GIARDIADB', 'bioconductor-GETORF', 'GETLETTERDISTRIBUTION1-IP', 'GETDISTRIBUTION-IP', 'GETDIFFERENCE-IP', 'GETNUMBER-IP', 'GETWIGDISTANCE-IP', 'GETWIGPROFILE-IP', 'GETINTRONS-IP', 'GEOSEQ', 'GERP', 'bioconductor-GGBIO', 'GGBIO', 'GESBAP', 'GEPAS', 'GERMONLINE', 'GERM-LINE_V_GENES', 'GFF2PS', 'GETSEQUENCE-IP', 'GETREADDISTRIBUTION-IP', 'GETDIFFERENTIALEXPRESSION-IP', 'bioconductor-GGCYTO', 'GGCYTO', 'GGPLOT2', 'bioconductor-GEOSUBMISSION', 'GEOSUBMISSION', 'GFINDER', 'GGRNA', 'GIBBS_MOTIF_SAMPLER', 'GGOST', 'GEPARD', 'GETSIZES-IP', 'bioconductor-GEWIST', 'GEWIST', 'GEOSEARCH', 'GFF2APLOT', 'GETDISTANCE-IP', 'bioconductor-GGBASE', 'GGBASE', 'GETEXONS-IP', 'GETFLANKING-IP', 'GETWIGDATA-IP', 'bioconductor-GGTOOLS', 'GGTOOLS', 'GENOMEVIEW', 'bioconductor-GENVISR', 'GENVISR', 'bioconductor-GENOMICRANGES', 'GENOMICRANGES', 'bioconductor-GENOSET', 'GENOSET', 'GENOMIC_HYPERBROWSER', 'GENOMICS_VIRTUAL_LAB', 'GENOMEJACK', 'GENOMETOOLS', 'bioconductor-GENOMICINTERACTIONS', 'GENOMICINTERACTIONS', 'bioconductor-GENOMICALIGNMENTS', 'GENOMICALIGNMENTS', 'bioconductor-GENOMICTUPLES', 'GENOMICTUPLES', 'bioconductor-GENOMICFILES', 'GENOMICFILES', 'bioconductor-GEOQUERY', 'GEOQUERY', 'GENOMEMAPPER', 'GENOMICTOOLS', 'bioconductor-GENOMICFEATURES', 'GENOMICFEATURES', 'bioconductor-GENPHEN', 'GENPHEN', 'GENSKEW', 'GENOMETA', 'GENOREAD', 'GENOVIEWER', 'GENSEARCHNGS', 'GENVISION', 'bioconductor-GENOMINATOR', 'GENOMINATOR', 'GENOMEREVIEWS', 'GENOMETESTER', 'GENOMERNAI', 'bioconductor-GENRANK', 'GENRANK', 'bioconductor-GENOMEINTERVALS', 'GENOMEINTERVALS', 'bioconductor-GEOMETADB', 'GEOMETADB', 'GENONETS', 'GENEWISE', 'GENESMASH', 'GENIE', 'GENOMEDATA', 'bioconductor-GENETCLASSIFIER', 'GENETCLASSIFIER', 'bioconductor-GENESIS', 'GENESIS', 'bioconductor-GENESELECTOR', 'GENESELECTOR', 'GENESEEKER', 'GENETRAIL', 'GENES_BY_TRANSMEMBRANE_DOMAIN_COUNT', 'GENETALK', 'bioconductor-GENOGAM', 'GENOGAM', 'GENOMECRISPR', 'GENMR', 'GENO2PHENO', 'GENOLIST', 'GENETRAIL2', 'GENOMEBROWSE', 'GENOME_SURVEYOR', 'GENESIGDB', 'GENOCAD', 'GENESNPS', 'GENNON-H', 'GENIATAGGER', 'bioconductor-GENEXTENDER', 'GENEXTENDER', 'bioconductor-GENESELECTMMD', 'GENESELECTMMD', 'bioconductor-GENOCN', 'GENOCN', 'GENIES', 'GENOME_MAPS', 'GENOME_TRAX', 'bioconductor-GENETICSDESIGN', 'GENETICSDESIGN', 'bioconductor-GENETICSPED', 'GENETICSPED', 'bioconductor-GENOMEGRAPHS', 'GENOMEGRAPHS', 'GENEVESTIGATOR', 'GENESET2MIRNA', 'GENESEQER', 'GENEDB', 'GENEPUBLISHER', 'GENES_BY_ORGANISM', 'GENES_BY_PREDICTED_SIGNAL_PEPTIDE', 'GENES_BY_MOLECULAR_WEIGHT', 'GENEPROF', 'bioconductor-GENEOVERLAP', 'GENEOVERLAP', 'bioconductor-GENEGENEINTER', 'GENEGENEINTER', 'bioconductor-GENEMETA', 'GENEMETA', 'bioconductor-GENEATTRIBUTION', 'GENEATTRIBUTION', 'bioconductor-GENEPLAST', 'GENEPLAST', 'GENEHUB-GEPIS', 'GENEFARM', 'GENEFISHER2', 'GENEBYLOCUSTAG', 'GENES_BY_EPITOPES_PRESENCE', 'bioconductor-GENEFU', 'GENEFU', 'bioconductor-GENENETWORKBUILDER', 'GENENETWORKBUILDER', 'bioconductor-GENEREGIONSCAN', 'GENEREGIONSCAN', 'GENECODIS3', 'GENEIOUS', 'bioconductor-GENEEXPRESSIONSIGNATURE', 'GENEEXPRESSIONSIGNATURE', 'bioconductor-GENERECOMMENDER', 'GENERECOMMENDER', 'GENEFIZZ', 'bioconductor-GENEPLOTTER', 'GENEPLOTTER', 'GENECAT', 'GENEMANIA', 'GENERECON', 'GENEID', 'GENES_BY_ISOELECTRIC_POINT', 'GENEDATA_EXPRESSIONIST', 'GENEPATTERN', 'bioconductor-GENERXCLUSTER', 'GENERXCLUSTER', 'bioconductor-GENEGA', 'GENEGA', 'bioconductor-GENEFILTER', 'GENEFILTER', 'GENEMARK', 'GENEPATH', 'GENENAMES.ORG', 'GENECARDS', 'GENATLAS', 'GENE_SET_TO_DISEASES', 'GEMREADS-IP', 'GEMERR-IP', 'GENE2OLIGO', 'GECKO', 'GENBANK', 'GEDEVO-M', 'GEDEVO', 'GENDBE', 'GENBANK2GFF3-IP', 'bioconductor-GENE.E', 'GENE.E', 'bioconductor-GENBANKR', 'GENBANKR', 'GENE3D', 'GEMS', 'GEMINA', 'GENE_EXPRESSION_BARCODE', 'GENBANK_TO_GENE_PROT_CONVERTOR-IP', 'GENE_STACKER', 'GENDECODER', 'GENDOO', 'GENE3D_STRUCTURAL', 'GENDB', 'bioconductor-GEECEE', 'GEEFU', 'GEM_LIBRARY', 'GENBANK_TO_GENE-IP', 'bioconductor-GENEANSWERS', 'GENEANSWERS', 'bioconductor-GENARISE', 'GENARISE', 'bioconductor-GEM', 'GEM', 'GEMPROSPECTOR', 'GECONT_2', 'GEMTOOLS', 'GEMMAPPER', 'GENALICE_MAP', 'GENE-COUNTER', 'GENE_EXPRESSION_ATLAS', 'GECKO-CSB', 'GECKO-MGV', 'GDTOOLS_VALIDATE-IP', 'GATK2_VARIANT_RECALIBRATOR-IP', 'GBLOCKS_ONECLICK-IP', 'GBLOCKS-IP', 'GDR', 'GATK2_VARIANT_FILTRATION-IP', 'GDTOOLS_SIMULATE-MUTATIONS-IP', 'bioconductor-GCRISPRTOOLS', 'GCRISPRTOOLS', 'GCOCOA', 'GCONVERT', 'GBIF_TAXON_WS', 'GDTOOLS_MASK-IP', 'GDTOOLS_VCF2GD-IP', 'GATK2_VARIANT_SELECT-IP', 'GDTOOLS_SUBTRACT-IP', 'GDTOOLS_GD2VCF-IP', 'GDTOOLS_CHECK-IP', 'GDTOOLS_NORMALIZE-IP', 'GDTOOLS_INTERSECT-IP', 'bioconductor-GCATEST', 'GCATEST', 'bioconductor-GCMAPWEB', 'GCMAPWEB', 'GDAP', 'GC-PROFILE', 'GBIS', 'GBROWSE', 'GDTOOLS_NOT-EVIDENCE-IP', 'GDTOOLS_APPLY-IP', 'GDTOOLS_GD2GVF-IP', 'GCVIEW', 'GBIF_OCCURRENCE_WS', 'GDTOOLS_ANNOTATE-IP', 'GDTOOLS_MUTATIONS-IP', 'GATK2_VARIANT_EVAL-IP', 'GDTOOLS_UNION-IP', 'bioconductor-GCRMA', 'GCRMA', 'bioconductor-GARNIER', 'GANESH', 'GATK2_VARIANT_ANNOTATOR-IP', 'GATK2_INDEL_REALIGNER-IP', 'G2D', 'GALAXYWEB', 'GALAXY', 'GATK2_PRINT_READS-IP', 'GATK2_REDUCE_READS-IP', 'bioconductor-GARFIELD', 'GARFIELD', 'bioconductor-GAPREDICTION', 'GAPREDICTION', 'GASSST', 'GALACTOSEMIA_PROTEINS_DATABASE', 'GATK2_HAPLOTYPE_CALLER-IP', 'GATK2_VARIANT_COMBINE-IP', 'GATK2_VALIDATE_VARIANTS-IP', 'GATK2_UNIFIED_GENOTYPER-IP', 'GATK2_DEPTH_OF_COVERAGE-IP', 'GATK2_REALIGNER_TARGET_CREATOR-IP', 'bioconductor-GAGA', 'GAGA', 'GAT', 'GALAXY_IMAGE_ANALYSIS', 'G-SESAME', 'G-SQZ', 'GALIGN', 'GAMBIT', 'GASV', 'GALAXYREFINE', 'GABI-KAT', 'GATK2_VARIANT_APPLY_RECALIBRATION-IP', 'GATK2_BASE_RECALIBRATOR-IP', 'bioconductor-GAGE', 'GAGE', 'FUNSIP', 'FROGS_CLUSTERS_STAT-IP', 'FUSIONANALYSER', 'bioconductor-FUNCHIP', 'FUNCHIP', 'G-LANGUAGE_GAE', 'FUNTREE', 'FRST', 'FUNPAT', 'FUSIONCATCHER', 'FUSIONHUNTER', 'G-MO.R-SEQ', 'FROGS_BIOM_TO_STDBIOM-IP', 'FROGS_CLUSTERING-IP', 'G-LANGUAGE_WS', 'FROGS_AFFILIATIONS_STAT-IP', 'FROGS_PREPROCESS-IP', 'FROGS_BIOM_TO_TSV-IP', 'FUSIONMAP', 'FUZZYCLUST', 'bioconductor-FUZZPRO', 'FROGS_REMOVE_CHIMERA-IP', 'FULL-PARASITES', 'FT-COMAR', 'FUNCLUST', 'FUNGENES', 'bioconductor-FUZZNUC', 'bioconductor-FUZZTRAN', 'FUNCOUP', 'G-LANGUAGE_API', 'FUNEL', 'FROGS_NORMALISATION-IP', 'FROGS_FILTERS-IP', 'FROGS_UPLOAD_TAR-IP', 'FROGS_DEMULTIPLEX-IP', 'FUSIONSEQ', 'FUNTAXIS', 'FUNFOLD2', 'FUNSIMMAT', 'FTMAP', 'FORNA', 'bioconductor-FREAK', 'FQSEQNAME-IP', 'FROGS_AFFILIATION_OTU-IP', 'FQDUPLICATE-IP', 'FOXS', 'FPOCKET', 'FPOP', 'FREALIGN', 'FRAMEDP', 'FRECLU', 'FREEBAYES', 'FREEC', 'FQEXTRACT-IP', 'FRAGMENTATION_ANALYZER', 'FRAGMENTSTORE', 'FRAGHMMENT', 'FQQUALITY-IP', 'FOLD-RATE', 'FOLDMINER', 'FRAMED', 'FRAGGENESCAN', 'FOOTER', 'FOLDALIGN', 'FREESASA', 'FQSEQPAIR-IP', 'FQCONVERT-IP', 'FQCLEANER-IP', 'FRAGGENESCAN-IP', 'FROG2', 'FOLDX', 'FROG', 'FLUX', 'bioconductor-FLOWMEANS', 'FLOWMEANS', 'bioconductor-FLOWTYPE', 'FLOWTYPE', 'FLYFACTORSURVEY', 'FLYRNAI', 'FLOWSIM', 'FLOWSOM', 'bioconductor-FLOWREPOSITORYR', 'FLOWREPOSITORYR', 'bioconductor-FLOWWORKSPACE', 'FLOWWORKSPACE', 'FMM', 'FLYTF', 'FLYMINE', 'bioconductor-FOCALCALL', 'FOCALCALL', 'bioconductor-FLOWMAP', 'FLOWMAP', 'bioconductor-FLOWMERGE', 'FLOWMERGE', 'FLUGENOME', 'bioconductor-FLOWPLOIDY', 'FLOWPLOIDY', 'bioconductor-FLOWPEAKS', 'FLOWPEAKS', 'bioconductor-FLOWMATCH', 'FLOWMATCH', 'bioconductor-FLOWQB', 'FLOWQB', 'bioconductor-FLOWTRANS', 'FLOWTRANS', 'FLAD', 'FLOREMI', 'FLASH', 'FLEXBAR', 'bioconductor-FISHALYSER', 'FISHALYSER', 'bioconductor-FLOWCLUST', 'FLOWCLUST', 'FIRTH_TOOLS', 'FINGERPRINTSCAN', 'FIREDB_WS', 'FIREDB', 'FIRESTAR_WS', 'FIRESTAR', 'FISH_SEQUENCE_SEARCH', 'FLEXPORTAL', 'bioconductor-FITHIC', 'FITHIC', 'bioconductor-FLOWCL', 'FLOWCL', 'FLIPPER', 'FISH_PROFILE_SEARCH', 'FINDPROMOTER-IP', 'FIREDOCK', 'FLOWER', 'bioconductor-FLOWAI', 'FLOWAI', 'bioconductor-FLOWDENSITY', 'FLOWDENSITY', 'bioconductor-FLOWCLEAN', 'FLOWCLEAN', 'bioconductor-FLAGME', 'FLAGME', 'FISH', 'FINDGEO', 'bioconductor-FCCAC', 'FCCAC', 'bioconductor-FFPE', 'FFPE', 'FGAP', 'FFAS', 'FEATUREMAP3D', 'FEATURESCAN', 'FFGED', 'bioconductor-FINDKM', 'bioconductor-FEATCOPY', 'FILTER', 'FGENESH', 'FATCAT', 'FF', 'FEATUREP', 'bioconductor-FEATTEXT', 'bioconductor-FEATREPORT', 'FETCH-TAXONOMIC-RANKS-IP', 'FEATURECOUNTS', 'FIGARO', 'bioconductor-FINDMYFRIENDS', 'FINDMYFRIENDS', 'bioconductor-FDRAME', 'FDRAME', 'bioconductor-FEM', 'FEM', 'FETCHPRODOM_WS', 'FINDPEAKS', 'FILTERCONTROL-IP', 'bioconductor-FGSEA', 'FGSEA', 'FIGTREE', 'FATIGOPLUS', 'FFPRED', 'FIDEA', 'bioconductor-FEATMERGE', 'FHITINGS', 'bioconductor-FCI', 'FCI', 'FGF', 'FEATUREEXTRACT', 'FELLS', 'FIBERDOCK', 'FGDB', 'FASTQC_CLOUDIFB', 'FASTA', 'FASTA_REST', 'FASTME_ONECLICK-IP', 'FASTAFROMBED-IP', 'FALCON', 'FASTM_EBI', 'FASTME', 'FASTQVALIDATOR', 'FACS', 'FASTQC-IP', 'FASTA_TO_PHYLIP-IP', 'FASTQ_GROOMER_PARALLEL-IP', 'bioconductor-FASTSEG', 'FASTSEG', 'FASTTREE', 'bioconductor-FAMAGG', 'FAMAGG', 'FALC-LOOP', 'FASTM_API_EBI', 'bioconductor-FARMS', 'FARMS', 'FAF-DRUGS', 'FASTME-IP', 'FAST', 'FASTQC', 'FA-EXTRACT-SEQUENCE-IP', 'FASTSNP', 'FASTSCOP', 'FASTR3D', 'FASTEP', 'FASTQC_TOOLSHED-IP', 'FASTQ_SCREEN', 'bioconductor-FASTLIQUIDASSOCIATION', 'FASTLIQUIDASSOCIATION', 'bioconductor-FABIA', 'FABIA', 'bioconductor-FACTDESIGN', 'FACTDESIGN', 'FAST-LMM', 'FASTX-TOOLKIT', 'FANTOM', 'FASTML', 'FASTCONTACT', 'FABOX', 'EXPRESS_CLOUDIFB', 'EXTRACT_ARCHIVE_MERGE-IP', 'EXPRESS-IP', 'EXOMEPICKS', 'EXMS', 'EXPARNA-P', 'EVORHA', 'EXPORT_ONE_DATA-IP', 'bioconductor-EXPRESSIONVIEW', 'EXPRESSIONVIEW', 'bioconductor-EXPLORASE', 'EXPLORASE', 'EVOLVIEW', 'EVODESIGN', 'EXPARNA', 'bioconductor-EXTRACTFEAT', 'bioconductor-EXTRACTALIGN', 'EXPLORA', 'EXTRACTCENTRALREGIONS-IP', 'EXOMECNV', 'EXONERATE', 'bioconductor-EXOMEPEAK', 'EXOMEPEAK', 'bioconductor-EXPERIMENTHUBDATA', 'EXPERIMENTHUBDATA', 'EXALIGN', 'EVEX', 'EXPORT-IP', 'bioconductor-EXOMECOPY', 'EXOMECOPY', 'bioconductor-EXPERIMENTHUB', 'EXPERIMENTHUB', 'bioconductor-EXIMIR', 'EXIMIR', 'EXPRESSION_ATLAS', 'EXOME_VARIANT_SERVER', 'bioconductor-EXTRACTSEQ', 'EXCEMPLIFY', 'EXPORT_SBW-IP', 'bioconductor-EWCE', 'EWCE', 'EXPRESSION_PROFILER', 'EXPRESSYOURSELF', 'EXOCARTA', 'EXAC', 'EXOMISER', 'EUROPE_PMC_ORCID_TOOL', 'EUROPE_PMC_PLUS_GRANT_LINKING', 'EUROPE_PMC_PLUS_MANUSCRIPT_SUBMISSION_SYSTEM', 'EUHCVDB', 'bioconductor-ETANDEM', 'bioconductor-EST2GENOME', 'ERDS', 'EST2ASSEMBLY', 'ESIPROT', 'ESLPRED', 'EVACON', 'EUROPEPMC_API', 'ESCDB', 'EUROPEAN_VIRUS_ARCHIVE', 'ESIMSA', 'ERPIN', 'EUROPE_PMC_RESTFUL_WEB_SERVICE_FOR_ARTICLES', 'EVA', 'EUDYSBIOME', 'ETBLAST', 'EUGENE', 'EST2GENOME_WS_HUSAR', 'ERGO', 'ERNE', 'ERROR_CORRECTION_EVALUATION_TOOLKIT', 'bioconductor-ERCCDASHBOARD', 'ERCCDASHBOARD', 'bioconductor-ESETVIS', 'ESETVIS', 'ESPRITZ', 'EUROPE_PMC', 'ESTS_SOURCE', 'ERNWIN', 'ERANGE', 'ESIMSA2D', 'ESTEXPLORER', 'EUROPHENOME', 'ESPRIPT_3.0', 'EURISCO', 'ENSEMBL_GENOMES_PROTISTS_DATA_SLICER', 'EPD', 'ENSEMBLEQDB', 'bioconductor-EQUICKTANDEM', 'EP3', 'ENVIPAT', 'bioconductor-EPIGENOMIX', 'EPIGENOMIX', 'bioconductor-ENSEMBLDB', 'ENSEMBLDB', 'EPITOOLKIT', 'ENSEMBL_GENOMES_PROTISTS_BLAST', 'bioconductor-ENVISIONQUERY', 'ENVISIONQUERY', 'bioconductor-EPIVIZR', 'EPIVIZR', 'ENTERIX', 'EPROBALIGN', 'ENSEMBL_MOUSE_GENOME_BROWSER', 'ENSEMBL_ZEBRAFISH_GENOME_BROWSER', 'ENTEROTYPING', 'ENZYMEPORTAL_PROTEIN_SEQUENCE_SEARCH', 'ENZYME_PORTAL', 'bioconductor-ENTRET', 'EPIGRAPH', 'bioconductor-EPIVIZRDATA', 'EPIVIZRDATA', 'EPLANT', 'ENZYME', 'bioconductor-EPESTFIND', 'bioconductor-EPRIMER32', 'ENSEMBLDB3', 'bioconductor-EPIVIZRSTANDALONE', 'EPIVIZRSTANDALONE', 'ENSEMBL_RAT_GENOME_BROWSER', 'ENTREZ', 'ENSEMBL_GENOMES_PROTISTS_SEQUENCE_SEARCH', 'ENSEMBL_SEQUENCE_SEARCH_BLAST', 'ENSEMBL_GENOMES_PROTISTS_VEP', 'ENSEMBL_GENOMES_PROTISTS_ID_HISTORY_CONVERTER', 'ENTREZ_PROTEIN_CLUSTERS', 'EPSILON', 'bioconductor-EPRIMER3', 'bioconductor-EPIVIZRSERVER', 'EPIVIZRSERVER', 'ENTREZ_GENE', 'ENSEMBL_GENOMES_BACTERIA_ID_HISTORY_CONVERTER', 'ENSEMBL_GENOMES_FUNGI_ID_HISTORY_CONVERTER', 'ENSEMBL_GENOMES_METAZOA_ID_HISTORY_CONVERTER', 'ENSEMBL_GENOMES', 'ENRICHNET', 'bioconductor-ENRICHEDHEATMAP', 'ENRICHEDHEATMAP', 'bioconductor-ENCODEXPLORER', 'ENCODEXPLORER', 'ENSEMBL', 'ENSEMBL_GENOMES_BACTERIA_BLAST', 'ENSEMBL_GENOMES_BACTERIA_SEQUENCE_SEARCH', 'ENSEMBL_GENOMES_BACTERIA_VEP', 'ENSEMBL_GENOMES_FUNGI_VEP', 'ENSEMBL_GENOMES_METAZOA_DATA_SLICER', 'ENSEMBL_GENOMES_PLANTS_ASSEMBLY_CONVERTER', 'ENDSCRIPT_2.0', 'ENSEMBL_FUGU_GENOME_BROWSER', 'ENSEMBL_API', 'ENSEMBL_GENOMES_FUNGI_DATA_SLICER', 'ENSEMBL_GENOMES_FUNGI_ASSEMBLY_CONVERTER', 'ENSEMBL_GENOMES_METAZOA_ASSEMBLY_CONVERTER', 'ENSEMBL_GENOMES_PROTISTS_ASSEMBLY_CONVERTER', 'ENSEMBL_GENOMES_METAZOA_BLAST', 'ENSEMBL_GENOMES_PLANTS_SEQUENCE_SEARCH', 'ENSEMBL_GENOMES_METAZOA_VEP', 'ENIGMA', 'ENLY', 'ENSEMBL_GENOMES_FUNGI_SEQUENCE_SEARCH', 'ENSEMBL_GENOMES_PLANTS_BLAST', 'ENSEMBL_GENOMES_PLANTS_VEP', 'ENOLOGOS', 'ENDEAVOUR', 'ENCODE_DCC', 'ENSEMBL_C.ELEGANS_GENOME_BROWSER', 'ENSEMBL_GENOMES_METAZOA_SEQUENCE_SEARCH', 'ENSEMBL_GENOMES_FUNGI_BLAST', 'ENSEMBL_GENOMES_PLANTS_ID_HISTORY_CONVERTER', 'ENSEMBL_GENOMES_PLANTS_DATA_SLICER', 'bioconductor-ENRICHMENTBROWSER', 'ENRICHMENTBROWSER', 'ENSEMBL_DROSOPHILA_GENOME_BROWSER', 'EMAP', 'EMMA2', 'bioconductor-EMBOSSUPDATE', 'bioconductor-EGSEA', 'EGSEA', 'ELECTRONIC_PCR', 'EMAGE', 'ENA', 'bioconductor-EMBOSSVERSION', 'EGMRG', 'ELIXIR_ELEARNING_PORTAL', 'bioconductor-EINVERTED', 'bioconductor-EMPIRICALBROWNSMETHOD', 'EMPIRICALBROWNSMETHOD', 'ELNEMO', 'EMDATABANK.ORG', 'ELASTODB', 'EGGNOG', 'ELMMATCHER', 'ELMINSTANCEMAPPER', 'bioconductor-EMOWSE', 'EMAN', 'EMMA_WS_HUSAR', 'EMMA2_WS', 'bioconductor-EMBOSSDATA', 'bioconductor-EISA', 'EISA', 'ENA_SEQUENCE_SEARCH', 'EHECREGNET', 'bioconductor-EMMA', 'ELAND', 'EMBF', 'bioconductor-EDGE', 'EDGE', 'bioconductor-EDASEQ', 'EDASEQ', 'bioconductor-EEGC', 'EEGC', 'EDS', 'EFFECTIVECCBD', 'EFFECTIVEELD', 'EFFECTIVEDB', 'EDIA', 'ECOGENE', 'EFFECTIVES346', 'bioconductor-EDIALIGN', 'bioconductor-EDAMHASOUTPUT', 'EDENA', 'EF-SEEK', 'EFFECTIVET3', 'bioconductor-EDAMDEF', 'bioconductor-EDAMNAME', 'bioconductor-EDGER', 'EDGER', 'bioconductor-EDDA', 'EDDA', 'EGASSEMBLER', 'ECOCYC', 'EDULISS', 'EGA', 'bioconductor-EDAMISID', 'bioconductor-EDAMHASINPUT', 'bioconductor-EDAMISFORMAT', 'bioconductor-EGAD', 'EGAD', 'ECR_BROWSER', 'EFFECTIVE', 'EASYGENE', 'EB-EYE', 'EAGLEVIEW', 'bioconductor-EASYRNASEQ', 'EASYRNASEQ', 'E-LEA3D', 'EBI_METAGENOMICS', 'ECHO', 'bioconductor-DYEBIAS', 'DYEBIAS', 'DYNAMICPROTEOMICS_DB', 'ECHOBASE', 'ECID', 'EBIMAGE', 'EASEQ', 'EASYFIG', 'EASYPROT', 'bioconductor-DYNDOC', 'DYNDOC', 'bioconductor-EASYQPCR', 'EASYQPCR', 'bioconductor-DUPRADAR', 'DUPRADAR', 'EASYCLUSTER', 'EA-UTILS', 'bioconductor-EBSEA', 'EBSEA', 'bioconductor-EBSEQ', 'EBSEQ', 'E2G', 'E-RNAI', 'E3MINER', 'E1DS', 'EBI_TOOLS', 'EBI_SEARCH', 'DYNAMINE', 'EBCALL', 'bioconductor-EBARRAYS', 'EBARRAYS', 'DYNAVACS', 'EBI_PATENT_SEQUENCE_DATABASE', 'DRIP-PRED', 'bioconductor-DRFINDFORMAT', 'DRFAST', 'bioconductor-DSIMER', 'DSIMER', 'bioconductor-DRIVERNET', 'DRIVERNET', 'DROSOPHILA_DNASE_I_FOOTPRINT_DATABASE', 'DR.BIND', 'DSSP', 'DSAP', 'DSGSEQ', 'DSRC', 'bioconductor-DRUGVSDISEASE', 'DRUGVSDISEASE', 'DSHIFT', 'DSCHECK', 'DROID', 'DPVWEB', 'bioconductor-DRFINDRESOURCE', 'bioconductor-DUALKS', 'DUALKS', 'DSSPCONT', 'DRYGIN', 'DROPNET', 'DUCTAPE', 'DOTTUP_WS_HUSAR', 'bioconductor-DRFINDID', 'bioconductor-DREG', 'bioconductor-DRIMSEQ', 'DRIMSEQ', 'bioconductor-DSS', 'DSS', 'DRUGSCOREPPI', 'DSP', 'DRUGEBILITY_BLAST_SEARCH', 'DSK', 'bioconductor-DRTEXT', 'bioconductor-DRGET', 'bioconductor-DRFINDDATA', 'DTA_TO_MGF_FILE_CONVERTER', 'bioconductor-DTA', 'DTA', 'bioconductor-DUPCHECKER', 'DUPCHECKER', 'DRCANCER', 'DOUTFINDER', 'DR.VIS', 'DNA_CHROMATOGRAM_EXPLORER', 'bioconductor-DOPPELGANGR', 'DOPPELGANGR', 'DNABINDPROT', 'DOMMINO', 'DOE_JOINT_GENOME_INSTITUTE_GENOME_PORTAL', 'DOTTER', 'bioconductor-DOQTL', 'DOQTL', 'bioconductor-DNABARCODES', 'DNABARCODES', 'DNASHAPE', 'DORINA', 'DOTMATCHER_WS_HUSAR', 'bioconductor-DOTMATCHER', 'bioconductor-DOTPATH', 'DOGSITESCORER', 'DNAZIP', 'bioconductor-DOMAINSIGNATURES', 'DOMAINSIGNATURES', 'DNATOOLS', 'DNANNOTATOR', 'DOMINE', 'bioconductor-DOTTUP', 'DOMAINATION', 'DNAA', 'DNASER', 'bioconductor-DNASHAPER', 'DNASHAPER', 'bioconductor-DNACOPY', 'DNACOPY', 'DNAMATE', 'DOMAC', 'DNA_BASER', 'bioconductor-DMRCATE', 'DMRCATE', 'DNASU', 'DNATRAFFIC', 'DIGIT', 'DIMA_DOMAIN_INTERACTION_MAP', 'bioconductor-DISTMAT', 'DIG', 'bioconductor-DIFFLOOP', 'DIFFLOOP', 'DISCOVAR', 'DIPOLE', 'DIG2', 'DISPROT', 'DIGSEE', 'DIPRODB', 'DIFFSEQ_WS_HUSAR', 'DISTILD', 'DISGENET', 'bioconductor-DIFFSEQ', 'DINAMELT', 'DIRE', 'DILIMOT', 'DISCOSNP', 'DISCOTOPE', 'DISVIS', 'bioconductor-DKS', 'DKS', 'bioconductor-DIRECTOR', 'DIRECTOR', 'DISULFIND', 'DISCOVAR_DE_NOVO', 'DIMA_3.0', 'DISEASEMETH', 'DISLOCATE', 'DISTANCEP', 'DISTILLER', 'DIFFREPS', 'DINDEL', 'bioconductor-DIRICHLETMULTINOMIAL', 'DIRICHLETMULTINOMIAL', 'bioconductor-DIGGIT', 'DIGGIT', 'DISCRETE_DMD', 'DISEMBL_WS_JABAWS', 'DISEASECARD', 'DICTYOGLYC', 'DESEQ2', 'DIFFANY', 'DICHROMATCH', 'DICTYBASE', 'DETECTOR', 'bioconductor-DESCSEQ', 'DIANA_MIRPATH', 'DGD', 'DESEQ-IP', 'bioconductor-DEXSEQ', 'DEXSEQ', 'bioconductor-DERFINDERHELPER', 'DERFINDERHELPER', 'bioconductor-DEXUS', 'DEXUS', 'bioconductor-DIFFBIND', 'DIFFBIND', 'DIALIGN-TX', 'DHCL', 'DIANNA', 'DIDA', 'bioconductor-DERFINDERPLOT', 'DERFINDERPLOT', 'bioconductor-DESEQ', 'DESEQ', 'DEUTERATOR', 'bioconductor-DFP', 'DFP', 'DICHROWEB', 'DIFFERENTIAL_RNA-SEQ_PIPELINE', 'DIAL', 'bioconductor-DIFFGENEANALYSIS', 'DIFFGENEANALYSIS', 'DIARK', 'DELTAG_PREDICTION', 'bioconductor-DEBROWSER', 'DEBROWSER', 'DELTA_MASS', 'bioconductor-DEDS', 'DEDS', 'DECONSTRUCT', 'DECGPU', 'DECOY.PL', 'bioconductor-DEFORMATS', 'DEFORMATS', 'bioconductor-DEGREPORT', 'DEGREPORT', 'DECODE', 'DEEPBLUE', 'bioconductor-DENSITY', 'DENOVOGUI', 'DECONSEQ', 'DEEPTOOLS', 'bioconductor-DEEPBLUER', 'DEEPBLUER', 'DECONMSN', 'bioconductor-DEMAND', 'DEMAND', 'bioconductor-DELTAGSEG', 'DELTAGSEG', 'bioconductor-DECONRNASEQ', 'DECONRNASEQ', 'DENOVOID', 'DEEPBASE', 'DEPTH', 'DEFUSE', 'DEPIE', 'bioconductor-DEGAPSEQ', 'DECONTOOLS', 'bioconductor-DEGRAPH', 'DEGRAPH', 'bioconductor-DEGSEQ', 'DEGSEQ', 'DENDRIX', 'bioconductor-DEEPSNV', 'DEEPSNV', 'DEQOR', 'DEATHDOMAIN', 'DDPC', 'bioconductor-DBIBLAST', 'bioconductor-DBIGCG', 'DBNP', 'bioconductor-DBIFASTA', 'bioconductor-DBXCOMPRESS', 'DBTOOLKIT', 'DCODE.ORG', 'DBCRID', 'DBDEPC', 'DBETH', 'DBNSFP', 'DBSNP', 'DBEST', 'DCON', 'DBSTS', 'bioconductor-DBIFLAT', 'bioconductor-DBXUNCOMPRESS', 'DCLIP', 'bioconductor-DCHIPREP', 'DCHIPREP', 'bioconductor-DDGRAPH', 'DDGRAPH', 'DBSNP-Q', 'bioconductor-DBXRESOURCE', 'bioconductor-DBTELL', 'bioconductor-DBXOBO', 'bioconductor-DBXEDAM', 'bioconductor-DBXGCG', 'bioconductor-DDCT', 'DDCT', 'DBASS', 'DBTSS', 'DBCAN', 'DBD', 'bioconductor-DBXSTAT', 'bioconductor-DBXTAX', 'bioconductor-DBXREPORT', 'bioconductor-DBCHIP', 'DBCHIP', 'bioconductor-DCGSA', 'DCGSA', 'DBCP', 'DBDNV', 'bioconductor-DBXFASTA', 'bioconductor-DBXFLAT', 'bioconductor-DAN', 'CYCLUS3D', 'bioconductor-CYTOML', 'CYTOML', 'CYRFACE', 'bioconductor-CYCLE', 'CYCLE', 'DARC_SITE', 'DAVID_BIOINFORMATICS_RESOURCES', 'CYTOSCAPE.JS', 'CYP21', 'CYTOSCAPE', 'DATF', 'CYLOFOLD', 'CYBER-T', 'DASIR', 'DAVIDQUERY', 'CWORDS', 'bioconductor-DADA2', 'DADA2', 'bioconductor-DAPAR', 'DAPAR', 'D-OMIX', 'CYANOLYASE', 'CYGD', 'CYANA', 'DASTY', 'DAVID_WS', 'DATACUBE_EXPLORER', 'DATAANALYSIS2TPP', 'bioconductor-DAMA', 'DAMA', 'bioconductor-DAGLOGO', 'DAGLOGO', 'DASMIWEB', 'CYCLEBASE', 'CYANOBASE', 'DALILITE', 'CYP11B1', 'DAS', 'DATABASE_MANAGER', 'DANTE', 'DAT2MGF', 'DAMPD', 'CUNEIFORM', 'CUFFDIFF-IP', 'bioconductor-CTSGE', 'CTSGE', 'bioconductor-CVE', 'CVE', 'CTK', 'bioconductor-CUMMERBUND', 'CUMMERBUND', 'CUTADAPT_CLOUDIFB', 'bioconductor-CUSP', 'bioconductor-CUTGEXTRACT', 'CUDA-EC', 'CUFFLINKS', 'CUTADAPT', 'CSHL_FASTX_TRIMMER-IP', 'CSHL_FASTX_QUALITY_STATISTICS-IP', 'CUBE-DB', 'CUFFLINKS_CLOUDIFB', 'CUFFLINKS-IP', 'CVTREE', 'CSTMINER', 'CTD', 'CURVES', 'CSPRITZ', 'bioconductor-CUTSEQ', 'CSHL_FASTX_NUCLEOTIDES_DISTRIBUTION-IP', 'CUFFCOMPARE-IP', 'CSHL_PRINCETON_FASTX_BARCODE_SPLITTER-IP', 'CSHL_FASTX_CLIPPER-IP', 'bioconductor-CTC', 'CTC', 'CUPSAT', 'CSS', 'CSHL_FASTX_RENAMER-IP', 'CSHL_FASTX_COLLAPSER-IP', 'CUTADAPT-IP', 'CSHL_FASTX_REVERSE_COMPLEMENT-IP', 'CURTAIN', 'CUTICLEDB', 'CREATECONTROLSUBSET-IP', 'CRP', 'CS23D', 'CPNDB', 'CSA', 'CPRED', 'CSCAN', 'CRYSOL', 'CROPPAL', 'CRAVAT', 'bioconductor-CRIMAGE', 'CRIMAGE', 'CROSSLINK', 'CPPP', 'CRISPI', 'CSHL_FASTX_ARTIFACTS_FILTER-IP', 'bioconductor-CRLMM', 'CRLMM', 'CROSSTALKDB', 'CPMODULE', 'CRAC', 'CRISP', 'bioconductor-CSAR', 'CSAR', 'CS-ROSETTA3', 'CRASP', 'CRISPRCOMPAR', 'CRYPTODB', 'CRISPRMAP', 'CROP-IP', 'CPTRA', 'CROSSBOW', 'bioconductor-CROSSMETA', 'CROSSMETA', 'bioconductor-CRISPRSEEKPLUS', 'CRISPRSEEKPLUS', 'bioconductor-CPVSNP', 'CPVSNP', 'bioconductor-CRISPRSEEK', 'CRISPRSEEK', 'bioconductor-CRISPRVARIANTS', 'CRISPRVARIANTS', 'CRSD', 'CREME', 'CRISPRFINDER', 'CORYNEREGNET', 'CPDB', 'CORNET', 'CPA', 'CPHMODELS', 'CPMD', 'COUNTNUMBER_PARALLEL-IP', 'bioconductor-COUNTCLUST', 'COUNTCLUST', 'CPM', 'COV2HTML-IP', 'COUNTREADGCPERCENT-IP', 'bioconductor-COSNET', 'COSNET', 'bioconductor-COVRNA', 'COVRNA', 'bioconductor-CORMOTIF', 'CORMOTIF', 'CPC', 'COXPRESDB', 'COVALENTDOCK_CLOUD', 'CORUM', 'CORNET_MAIZE', 'bioconductor-CPGREPORT', 'COVERAGEGFF-IP', 'CPFP', 'CORRELOGO', 'CPGPLOT-EBI', 'CORNET_ARABIDOPSIS', 'CORSET', 'COSRANK', 'bioconductor-CPGPLOT', 'COUNTNUMBER-IP', 'CORTEX', 'bioconductor-COVEB', 'COVEB', 'bioconductor-CORREP', 'CORREP', 'bioconductor-CORMUT', 'CORMUT', 'bioconductor-COSMIQ', 'COSMIQ', 'CORRNA', 'CPLA', 'COSMIC', 'CONSURF-DB', 'CONVERT2ANNOVAR-IP', 'CONTRAIL', 'bioconductor-CONSENSUSCLUSTERPLUS', 'CONSENSUSCLUSTERPLUS', 'COPASI', 'CORKOAKDB', 'bioconductor-CONSAMBIG', 'COPYSEQ', 'CORAL', 'CONVERTTRANSCRIPTFILE-IP', 'bioconductor-COPYWRITER', 'COPYWRITER', 'CORGEN', 'COPRARNA', 'CONVERT-MATRIX', 'CONS_WS_HUSAR', 'CONVERTER_GENBANK_TO_FASTA-IP', 'bioconductor-CONSENSUSSEEKER', 'CONSENSUSSEEKER', 'bioconductor-CONTIBAIT', 'CONTIBAIT', 'CONSITE', 'COPAP', 'CONSENSUSPATHDB', 'COOT', 'CONSURF', 'COREMINE_MEDICAL', 'COORDINATESTOSEQUENCE-IP', 'CONTRA', 'CONVERTTRANSCRIPTFILE_FASTQTOFASTA-IP', 'CONSPRED', 'CONTRAV2', 'CONTEST', 'bioconductor-COREGNET', 'COREGNET', 'bioconductor-COPA', 'COPA', 'COPUB', 'COPS', 'bioconductor-COMPSEQ', 'COMPOMICS-UTILITIES', 'COMPARE_TAXOFILE-IP', 'CONAN-SNV', 'bioconductor-COMPLEXHEATMAP', 'COMPLEXHEATMAP', 'bioconductor-CONFESS', 'CONFESS', 'COMPASSXPORT', 'CONREAL', 'COMSIN', 'COMIR', 'COMPARATIVE_GENOMETRICS', 'CONDETRI', 'COMPAREOVERLAPPINGSMALLREF-IP', 'COMPUTECOVERAGE-IP', 'COMPOMICS-SIGPEP', 'COMSPARI', 'COMPAREMS2', 'CONPLEX', 'CONODICTOR', 'CONFAC', 'COMPSS', 'CONET', 'CONDITIONAL_QUANTILE_NORMALIZATION', 'COMODO', 'COMPAREOVERLAPPINGSMALLQUERY-IP', 'COMPREHENGSIVE', 'COMPAREPROSPECTOR', 'COMMET', 'COMPLUYEAST2DPAGE', 'CONDOP', 'COMPAREOVERLAPPING-IP', 'bioconductor-COMPEPITOOLS', 'COMPEPITOOLS', 'bioconductor-COMPASS', 'COMPASS', 'CONQUASS', 'COMPMERGE', 'bioconductor-CONS', 'COMPOMICS-ROVER', 'COMPAREOVERLAPPING_PARALLEL-IP', 'bioconductor-COMPGO', 'COMPGO', 'bioconductor-COMPCODER', 'COMPCODER', 'CONOSERVER', 'COANNEALING', 'COFACTORY', 'bioconductor-CODCOPY', 'bioconductor-COBINDR', 'COBINDR', 'COEUS', 'CODON_USAGE_DATABASE', 'COFACTOR', 'COASIM', 'COLOMBOS', 'COCOMAPS', 'bioconductor-CODERET', 'COLIMS', 'COGEME', 'CNVNATOR', 'COLIB_READ', 'COLORGFF-IP', 'bioconductor-CNVPANELIZER', 'CNVPANELIZER', 'bioconductor-CODEX', 'CODEX', 'bioconductor-CODELINK', 'CODELINK', 'bioconductor-COGPS', 'COGPS', 'CODONO', 'COMBFUNC', 'COLT-CANCER', 'COMET', 'COEXPNETVIZ', 'CNVKIT', 'COLLECT_MGF', 'bioconductor-COHCAP', 'COHCAP', 'CODEHOP', 'COLORADO-3D', 'bioconductor-CODCMP', 'COLLAPSEREADS-IP', 'bioconductor-COGENA', 'COGENA', 'bioconductor-CNVTOOLS', 'CNVTOOLS', 'COMBREX', 'CLUSTERING_RESUME-IP', 'CNV-SEQ', 'bioconductor-CLUSTEREXPERIMENT', 'CLUSTEREXPERIMENT', 'bioconductor-CN.MOPS', 'CN.MOPS', 'bioconductor-CNANORM', 'CNANORM', 'bioconductor-CLUSTCOMP', 'CLUSTCOMP', 'CLUSTER_BUSTER', 'CMWEB', 'CMD', 'CLUSTALW2_PHYLOGENY_REST', 'CLUSTALW2_API_EBI', 'CNOGPRO', 'CLUSTER_FLOW', 'CND', 'bioconductor-CLUSTERSTAB', 'CLUSTERSTAB', 'bioconductor-CNVGSA', 'CNVGSA', 'bioconductor-CN.FARMS', 'CN.FARMS', 'CMR', 'bioconductor-CLUSTERPROFILER', 'CLUSTERPROFILER', 'bioconductor-CLUSTERSIGNIFICANCE', 'CLUSTERSIGNIFICANCE', 'bioconductor-CMA', 'CMA', 'bioconductor-CNER', 'CNER', 'CLUSTVIS', 'CLUSTEVAL', 'CLUSTALW2_PHYLOGENY_EBI', 'CLUSTALW2_EBI', 'CLUSTERIZEBYSLIDINGWINDOWS-IP', 'bioconductor-CNORFUZZY', 'CNORFUZZY', 'CNANALYSIS', 'bioconductor-CNTOOLS', 'CNTOOLS', 'CMCOMPARE', 'CLC_MAPPER-IP', 'CLCBIO_GENOMICS_WORKBENCH', 'bioconductor-CLEAVER', 'CLEAVER', 'CLEANEX', 'CLEAN_READS', 'CLEAVELAND4', 'CLC_FIND_VARIATIONS-IP', 'CLC_MAPPER_LEGACY-IP', 'bioconductor-CLSTUTILS', 'CLSTUTILS', 'bioconductor-CLIPPDA', 'CLIPPDA', 'CLUSTALO_WS_JABAWS', 'CLUSTAL_OMEGA_EBI', 'CLINICAL_TRIALS_INFORMATION_MEDIATOR', 'CLIMA', 'CLEANTRANSCRIPTFILE-IP', 'CLIPCROP', 'CLOUDBURST', 'bioconductor-CLST', 'CLST', 'bioconductor-CLEANUPDTSEQ', 'CLEANUPDTSEQ', 'CLUSPRO', 'CLINVAR', 'CLIMA_2', 'CLIPPER', 'CLC_SEQUENCE_INFO-IP', 'CLC_FIND_VARIATIONS_TAB_CONVERTOR-IP', 'bioconductor-CLONOTYPER', 'CLONOTYPER', 'CLUSTAL_WS_JABAWS', 'CLOTBASE', 'CLUSTALO-IP', 'CLC_UNMAPPED_READS-IP', 'CLUSTALW-IP', 'CLC_MAPPING_INFO-IP', 'CLEAN_NGS-IP', 'CLOUDALIGNER', 'CLUSTALW', 'CLIPZ', 'CLIC', 'bioconductor-CINDEX', 'CINDEX', 'bioconductor-CHIPXPRESS', 'CHIPXPRESS', 'bioconductor-CLASSIFYR', 'CLASSIFYR', 'bioconductor-CHROMPLOT', 'CHROMPLOT', 'bioconductor-CISPATH', 'CISPATH', 'bioconductor-CHROGPS', 'CHROGPS', 'CITEXPLORE', 'CHOOCH', 'bioconductor-CIRDNA', 'CHROMASIG', 'CHROMHMM', 'CLC_ASSEMBLER-IP', 'CLC_CAS_TO_SAM-IP', 'CLANTOX', 'CIPRO_2.5', 'CIRCOS-IP', 'CIRCOS', 'CISGENOME', 'CLC_EXTRACT_CONSENSUS-IP', 'bioconductor-CHIPSIM', 'CHIPSIM', 'bioconductor-CHROMHEATMAP', 'CHROMHEATMAP', 'CLARK', 'bioconductor-CHROMSTAR', 'CHROMSTAR', 'bioconductor-CHIPSEQR', 'CHIPSEQR', 'bioconductor-CHOPSTICKS', 'CHOPSTICKS', 'CIRCBASE', 'CHOP', 'CIBEX', 'CISMOLS', 'CHLOROP', 'CHOPCHOP', 'CHIPSTER', 'CISTROME', 'CIF', 'CISRED', 'CHEMICAL_IDENTIFIER_RESOLVER', 'CHIPMUNK_CHIPHORDE-IP', 'CHIPMUNK', 'CHEMMINE_TOOLS', 'CHEMPROT', 'CHEMBL_NTD_SMALL_MOLECULE_SEARCH', 'CHIP-SEQ', 'CHEMBL_BIOLOGICALS_BLAST_SEARCH', 'CHEMBL_BLAST_SEARCH', 'CHEMHITS', 'bioconductor-CHIPS', 'CHIPMETA', 'bioconductor-CHIPCOMP', 'CHIPCOMP', 'bioconductor-CHIPSEQ', 'CHIPSEQ', 'CHEMGPS-NP', 'CHIMERASCAN', 'bioconductor-CHIPQC', 'CHIPQC', 'CHIP-ARRAY', 'CHEMBL', 'CHEMBL_SMALL_MOLECULE_SEARCH', 'CHIMERA', 'CHEMSPIDER', 'bioconductor-CHICAGO', 'CHICAGO', 'CHIPINFO', 'CHIMERDB', 'CHIPD', 'bioconductor-CHAOS', 'CHANGEGFFFEATURES-IP', 'CHANGETAGNAME-IP', 'bioconductor-CELLMAPPER', 'CELLMAPPER', 'bioconductor-CGEN', 'CGEN', 'bioconductor-CHAMP', 'CHAMP', 'CENTDIST', 'CHC_FIND', 'CELLNETANALYZER', 'CELLOSAURUS', 'CHEBI', 'bioconductor-CHARGE', 'CELLMISSY', 'bioconductor-CGHCALL', 'CGHCALL', 'CHARPROTDB', 'CENSOR', 'CG-CAT', 'bioconductor-CELLTREE', 'CELLTREE', 'bioconductor-CGHNORMALITER', 'CGHNORMALITER', 'CGD', 'bioconductor-CHECKTRANS', 'CHANGENAME-IP', 'CGA_TOOLS', 'bioconductor-CGHBASE', 'CGHBASE', 'bioconductor-CGHMCR', 'CGHMCR', 'bioconductor-CEXOR', 'CEXOR', 'CERMIT', 'CENTROIDHOMFOLD-LAST', 'CENTROIDFOLD', 'CENTROSOMEDB', 'CEQER', 'bioconductor-CGHREGIONS', 'CGHREGIONS', 'bioconductor-CHARM', 'CHARM', 'CGVIEW', 'CELLBASE', 'CCHARPPI', 'bioconductor-CCMAP', 'CCMAP', 'CDDB', 'CCDB', 'CAZY', 'CCFIND', 'CE-MC', 'CBS_DAS_PROTEIN_VIEWER', 'CCPEM', 'CELLCYCLEDB', 'CEGA', 'bioconductor-CELLITY', 'CELLITY', 'bioconductor-CCPROMISE', 'CCPROMISE', 'CEAS', 'CBARBEL', 'CCANCER', 'CCAP', 'CELL-MOTILITY', 'CDFREAD', 'CCRXP', 'CCHMMPROF', 'CCT', 'CD-SEARCH', 'CCP4_WIKI', 'CDK', 'CD-HIT-IP', 'CCDIGEST', 'CCPDB', 'CDD', 'CATCH', 'bioconductor-CANCERMUTATIONANALYSIS', 'CANCERMUTATIONANALYSIS', 'CAPWEB', 'CAVER_PYMOL_PLUGIN', 'bioconductor-CANCERINSILICO', 'CANCERINSILICO', 'bioconductor-CAOMICSV', 'CAOMICSV', 'CASTP', 'CAPS-DB', 'CAVER_ANALYST', 'CAVER_WEB', 'CANGS', 'bioconductor-CANCERSUBTYPES', 'CANCERSUBTYPES', 'bioconductor-CANCERCLASS', 'CANCERCLASS', 'CARRIE', 'CANPREDICT', 'CANCERRESOURCE', 'CARNA', 'CAVER', 'bioconductor-CATEGORYCOMPARE', 'CATEGORYCOMPARE', 'bioconductor-CASPER', 'CASPER', 'bioconductor-CAUSALR', 'CAUSALR', 'CARMAWEB', 'CARGO', 'CARNAC', 'CATDB', 'CATCHALL', 'bioconductor-CAND', 'CAND', 'CATH', 'CATALYTIC_SITE_IDENTIFICATION', 'CASNP', 'CASPR', 'BSMAP', 'bioconductor-BSSEQ', 'BSSEQ', 'bioconductor-BUFFEREDMATRIX', 'BUFFEREDMATRIX', 'bioconductor-BUFFEREDMATRIXMETHODS', 'BUFFEREDMATRIXMETHODS', 'BSQA', 'CADRE_GENOMES_BROWSER', 'bioconductor-CACHEEBEYESEARCH', 'bioconductor-CACHEDAS', 'BUSCO', 'bioconductor-BUMPHUNTER', 'BUMPHUNTER', 'CADGENE', 'CAMEO', 'BWA_CLOUDIFB', 'BUBASTIS', 'BWA-PSSM', 'BUJNICKI_LAB_TOOLS', 'bioconductor-CACHEDBFETCH', 'bioconductor-CAMERA', 'CAMERA', 'CAMP', 'CABS-FLEX', 'CABRI', 'bioconductor-CACHEENSEMBL', 'BTRIM', 'BWA-SW', 'CABOG', 'bioconductor-CALIB', 'CALIB', 'bioconductor-BUS', 'BUS', 'BTW', 'CABS-FOLD', 'BYKDB', 'BWA', 'bioconductor-BTWISTED', 'bioconductor-CAI', 'bioconductor-CAGER', 'CAGER', 'bioconductor-CAFE', 'CAFE', 'BWA_SAMTOOLS_BSC', 'CAMPO', 'BREAKWAY', 'bioconductor-BRAIN', 'BRAIN', 'BPROMPT', 'BSDD', 'BRCA_SHARE', 'BRCA-DIAGNOSTIC', 'BREEDB', 'BRAT', 'BREAKDANCER', 'BS_SEEKER', 'bioconductor-BROWSERVIZDEMO', 'BROWSERVIZDEMO', 'bioconductor-BRIDGE', 'BRIDGE', 'BOOGIE', 'BRENDA', 'BOWTIE2-IP', 'BRIG', 'BOW', 'bioconductor-BPRMETH', 'BPRMETH', 'BOTXMINER', 'BSDB', 'BONB', 'BRIDGEDB', 'BRISSKIT', 'BOWTIE', 'BREAKPOINTER', 'BREAKSEQ', 'BREAKTRANS', 'BPP', 'BRIX', 'BRASSICA_GENOME', 'BRESEQ_BAM2ALN-IP', 'BRESEQ-IP', 'BRESEQ_BAM2COV-IP', 'BOWTIE_WRAPPER-IP', 'bioconductor-BROWSERVIZ', 'BROWSERVIZ', 'bioconductor-BRAINSTARS', 'BRAINSTARS', 'BROP', 'BLUUES', 'BLASTPRODOM', 'bioconductor-BISEQ', 'BISEQ', 'BOMP', 'BLOSSOC', 'BIQ_ANALYZER', 'BISON', 'BLIXEM', 'bioconductor-BIOSVD', 'BIOSVD', 'bioconductor-BIOVIZBASE', 'BIOVIZBASE', 'BIOVERSE', 'BLSSPELLER', 'BMGE', 'BISMARK', 'BLACKFAN_ANEMIA', 'BMS_STANDARDS_REGISTRY', 'BLAST_EBI', 'BIPS', 'BLAST_API_EBI', 'BLAST', 'bioconductor-BLIMA', 'BLIMA', 'bioconductor-BIREWIRE', 'BIREWIRE', 'BLAST_VIEWER', 'BLASTO', 'BIOSYSTEMS', 'BLAST2SRS', 'BIPA', 'BIQ_ANALYZER_HT', 'BIS-SNP', 'BLAT', 'BLAST_BEST_SCORE-IP', 'bioconductor-BIRTA', 'BIRTA', 'BLAST2GO', 'BIP', 'BISC', 'BIOMAS', 'BIOMODELS_DATABASE_CONVERTERS', 'bioconductor-BIOSED', 'BIONUMERICS', 'BIOPYTHON', 'BIOPROFILING.DE', 'BIONODE-BWA', 'BIONODE-SAM', 'bioconductor-BIONET', 'BIONET', 'BIOPROSPECTOR', 'BIONMF', 'BIOSHELL_THREADER', 'BIOPHP', 'BIORUBY', 'bioconductor-BIOMVCCLASS', 'BIOMVCCLASS', 'BIOSMILE', 'BIOMART_CENTRAL', 'BIOSAMPLE', 'BIOMART_MART_SERVICE', 'BIOPIECES', 'BIOSMALLTALK', 'bioconductor-BIOSEQCLASS', 'BIOSEQCLASS', 'bioconductor-BIOSIGNER', 'BIOSIGNER', 'BIOPORTAL', 'BIONODE-NCBI', 'BIONIMBUS', 'BIOPERL', 'bioconductor-BIOQC', 'BIOQC', 'bioconductor-BIOMFORMAT', 'BIOMFORMAT', 'BIONUMBERS', 'BIOME', 'BIOPROJECT', 'BIONODE-TEMPLATE', 'BIONODE-SRA', 'BIONODE-FASTA', 'BIOJS-LEGACY', 'BIOJS-VIS-PMCCITATION', 'BIOJS-META-VIS', 'BIOJS-IO-SNIPSPECTOR', 'BIOJS-IO-ANNOTS', 'BIOJS-VIS-INCHLIB', 'BIOJS-IO-SAM', 'BIOJS-VIS-HGV', 'BIOJS-STAT-SEQS', 'BIOMAJ', 'BIOLCCC', 'BIOJS-IO-FASTA', 'BIOJS-IO-CLUSTAL', 'BIOJS-REST-ENSEMBL', 'BIOJS-VIS-SEQUENCE', 'BIOJS-VIS-EASY_FEATURES', 'BIOJS-VIS-INTERACTIONS-D3', 'BIOJS-VIS-FEATURE', 'BIOLIT', 'BIOJS-VIS-WIGEXPLORER', 'BIOJS-VIS-PDBVIEWER', 'BIOJS-VIS-CHROMOSOME', 'BIOJS-VIS-HEATMAP-D3', 'BIOJS-VIS-GPROFILER', 'BIOJS-IO-MATRIX', 'BIOJS-VIS-CIRCULARFV', 'BIOJS-ALG-SEQREGION', 'BIOJS-VIS-CIRCULARNET', 'BIOJS-VIS-DISEASEINFO', 'BIOJS-VIS-KEGGVIEWER', 'BIOJS-IO-GRADUATES', 'BIOLOGICALNETWORKS', 'BIOJS-VIS-SAM', 'BIOJS-META-PARSER', 'BIOJS-VIS-HPAFEATURE', 'BIOJS-IO-NEWICK', 'BIOJS-IO-GFF', 'BIOLABELER', 'BINGO', 'BIGR_XNAT', 'BIOALCIDAE', 'bioconductor-BIODIST', 'BIODIST', 'bioconductor-BIOBROOM', 'BIOBROOM', 'bioconductor-BIOCPARALLEL', 'BIOCPARALLEL', 'bioconductor-BIOCVIEWS', 'BIOCVIEWS', 'BIOINFO3D', 'BIOCATALOGUE_API', 'BIOCLIPSE', 'BIOJAVA', 'BIO-CIGAR', 'bioconductor-BIOBASE', 'BIOBASE', 'bioconductor-BIOCGENERICS', 'BIOCGENERICS', 'bioconductor-BIOCINSTALLER', 'BIOCINSTALLER', 'BIOGRID', 'BIGWIG_TO_BEDGRAPH-IP', 'bioconductor-BIOCGRAPH', 'BIOCGRAPH', 'bioconductor-BIOCCASESTUDIES', 'BIOCCASESTUDIES', 'BIOCATALOGUE', 'BIOEXTRACT', 'BIOBIKE', 'BIODRUGSCREEN', 'BINDINGDB', 'BIOISIS', 'BINCHE', 'BIGNASIM', 'bioconductor-BIOCANCER', 'BIOCANCER', 'BIOBAYESNET', 'BIOCONDUCTOR', 'BINDTRANSLATION', 'bioconductor-BIOCCHECK', 'BIOCCHECK', 'BINDN', 'BIOCYC', 'BEDTOOLS_BAMTOBED-IP', 'BEDTOOLS', 'bioconductor-BGEEDB', 'BGEEDB', 'BGEE', 'BEPIPRED', 'BEDTOOLS_MERGEBEDGRAPH-IP', 'BEDTOOLS_INTERSECTBED-IP', 'BEDGRAPHTOBIGWIG', 'BHAIRPRED', 'BEESPACE_NAVIGATOR', 'BGMUT', 'BEEP', 'BICYCLE', 'BIANA', 'BEDUTILS', 'BIGBWA', 'BEDTOOLS_INTERSECTBED_BAM-IP', 'BEDTOOLS_MULTIINTERSECTBED-IP', 'bioconductor-BIGGR', 'BIGGR', 'bioconductor-BIGMELON', 'BIGMELON', 'BIDAS', 'BICLUE', 'BFAST', 'BFCOUNTER', 'BEDTOOLS_GENOMECOVERAGEBED_HISTOGRAM-IP', 'bioconductor-BGAFUN', 'BGAFUN', 'bioconductor-BGX', 'BGX', 'bioconductor-BHC', 'BHC', 'bioconductor-BICARE', 'BICARE', 'BGD', 'BEETLEBASE', 'BETAWARE', 'BESST', 'BEDTOOLS_COVERAGEBED_COUNTS-IP', 'bioconductor-BIGMEMORYEXTRAS', 'BIGMEMORYEXTRAS', 'bioconductor-BGMIX', 'BGMIX', 'BIFORCE_TOOLBOX', 'BERKELEY_PHYLOGENOMICS_GROUP', 'BELVU', 'BAYESEMBLER', 'BBMRI-NL_CATALOG', 'BCFTOOLS', 'BEATSONLAB-MICROBIALGENOMICS', 'BEARR', 'BCOV', 'BEACONLAB_PSCAN', 'BDBM', 'BBMAP', 'BBSEQ', 'BCF-TOOLS-CAT-IP', 'bioconductor-BAYESKNOCKDOWN', 'BAYESKNOCKDOWN', 'BEAST', 'bioconductor-BEADARRAY', 'BEADARRAY', 'BEAP', 'BCF-TOOLS-INDEX-IP', 'bioconductor-BATCHQC', 'BATCHQC', 'bioconductor-BAYSEQ', 'BAYSEQ', 'BAYESMD', 'BBMRI-EU_CATALOG', 'BEAR', 'BCHECK_RNPB', 'BCFTOOLS_VIEW-IP', 'BATMAN', 'bioconductor-BEADDATAPACKR', 'BEADDATAPACKR', 'bioconductor-BCRANK', 'BCRANK', 'bioconductor-BEADARRAYSNP', 'BEADARRAYSNP', 'BBMRI-PORTAL', 'BAYESCALL', 'BCBIO-NEXTGEN', 'BEADS', 'BCF_VIEW_MPILEUP-IP', 'bioconductor-BAYESPEAK', 'BAYESPEAK', 'BEATMUSIC', 'BAMTOOLS', 'BARCRAWL_BARTAB', 'BALI-PHY', 'BASYS', 'BAGEL2', 'BAGEL3', 'BARRNAP', 'BANDAGE', 'BAMVIEW', 'BAM2ROI-IP', 'bioconductor-BADREGIONFINDER', 'BADREGIONFINDER', 'bioconductor-BALLGOWN', 'BALLGOWN', 'BASILISK', 'BAR', 'bioconductor-BANANA', 'BASE', 'BARRACUDA', 'BAGEL', 'BARNACLE', 'BARC', 'BAM_TO_SAM-IP', 'BAMBUS', 'BAMSEEK', 'BARCODE_GENERATOR', 'BAM_TO_SAM_PARALLEL-IP', 'BAM2FASTQ-IP', 'bioconductor-BASICSTARRSEQ', 'BASICSTARRSEQ', 'bioconductor-BASIC4CSEQ', 'BASIC4CSEQ', 'BACTGENESHOW', 'BALBES', 'BAMBINO', 'BAM_TO_BIGWIG-IP', 'BAMSTATS-IP', 'AUTOASSIGN', 'BACDIVE_WS', 'ASTALAVISTA', 'B6DB', 'bioconductor-BACKTRANSEQ', 'AUTOHD', 'BACELLO', 'AUTOSNPDB', 'BACMAP', 'ASTD', 'AZARA', 'ATAC', 'AUTO-RICKSHAW', 'BACKTRANSEQ-EBI', 'AUGUSTUS', 'ATPID', 'BACTERIOME.ORG', 'ATCOECIS', 'BACH', 'BABELOMICS', 'ATLAS-SNP2', 'bioconductor-BAC', 'BAC', 'BACKTRANAMBIG-EBI', 'ATHAMAP', 'ATLASGENETICSONCOLOGY', 'ATCC', 'ATLAS_SUITE', 'bioconductor-ATTRACT', 'ATTRACT', 'ASTERIAS', 'AVPPRED', 'ATIVS', 'BACDIVE', 'ASSIST', 'bioconductor-BACKTRANAMBIG', 'AUTOPHYLO', 'bioconductor-BACON', 'BACON', 'bioconductor-BAALCHIP', 'BAALCHIP', 'AXIMA2ANALYZE', 'bioconductor-ATLASRDF', 'ATLASRDF', 'bioconductor-ASSEMBLYGET', 'ASC', 'bioconductor-ASAFE', 'ASAFE', 'bioconductor-ARRAYTV', 'ARRAYTV', 'bioconductor-ARRAYQUALITY', 'ARRAYQUALITY', 'ARRAYPIPE', 'ARTEMIS', 'ASAP', 'ASSEMBLY_CONVERTER', 'ASP-G', 'ASAPRATIO', 'bioconductor-ARRAYMVOUT', 'ARRAYMVOUT', 'ARRAYXPATH', 'ASIDESIGNER', 'ARRAYMAP', 'ASSEMBLE2', 'ASATURA', 'bioconductor-ARRMNORMALIZATION', 'ARRMNORMALIZATION', 'bioconductor-ASGSCA', 'ASGSCA', 'ASMODELER', 'AS2TS', 'ASSEMBLY_PCR_OLIGO_MAKER', 'ASD', 'ASSAM', 'ASPICDB', 'ASCAT', 'ARRAYSTAR', 'bioconductor-ASPLI', 'ASPLI', 'bioconductor-ASEB', 'ASEB', 'ASEPCR', 'ARRAYPROSPECTOR', 'ASPIC-GENEID', 'bioconductor-ARRAYQUALITYMETRICS', 'ARRAYQUALITYMETRICS', 'ARTS', 'ASIAN', 'ASPGD', 'AQUARIA', 'ARCHDB', 'ARIA', 'AREM', 'APLTOMGFCONVERTER', 'bioconductor-ARRAYEXPRESSHTS', 'ARRAYEXPRESSHTS', 'AQUASAXS', 'ARABIDOPSIS_EFP_BROWSER', 'ARGOT2', 'APHIDBASE', 'APACHE_TAVERNA', 'ARDOCK', 'ARABIDOPSIS_GROWTH_REGULATORS_NETWORK', 'bioconductor-APCOMPLEX', 'APCOMPLEX', 'ARION_4_OMICS', 'ARISTO', 'APID', 'ARRAYEXPRESS', 'ARCHAEAFUN', 'ARF', 'ARCHPRED', 'ARGO', 'ARAMEMNON', 'ARACHNE', 'bioconductor-ANTIPROFILES', 'ANTIPROFILES', 'ARACHNOSERVER', 'ANTISMASH', 'ARP_WARP', 'ARRAY_SUITE', 'bioconductor-AROMA.LIGHT', 'AROMA.LIGHT', 'ARGOT_2.5', 'ANTARNA', 'bioconductor-ANNOTATIONHUBDATA', 'ANNOTATIONHUBDATA', 'ANTIHUNTER_2.0', 'ANTAR', 'ANNOVAR', 'ANNOVAR_ANNOTATE_VARIATION-IP', 'ANNOTATEPEAKS-IP', 'ANNOTATEGENES-IP', 'bioconductor-ANEUFINDER', 'ANEUFINDER', 'bioconductor-ANNOTATIONFUNCS', 'ANNOTATIONFUNCS', 'bioconductor-ANNOTATIONFORGE', 'ANNOTATIONFORGE', 'ANNIE', 'ANGSD', 'ANGELDUST', 'ANNOVAR_SUMMARIZE-IP', 'bioconductor-ANNOTATIONDBI', 'ANNOTATIONDBI', 'bioconductor-ANNOTATIONTOOLS', 'ANNOTATIONTOOLS', 'bioconductor-ANNOTATR', 'ANNOTATR', 'bioconductor-ANNOTATIONHUB', 'ANNOTATIONHUB', 'ANNOTQTL', 'bioconductor-ANTIGENIC', 'bioconductor-ANOTA', 'ANOTA', 'ANTIGENDB', 'ANNO-J', 'ANDREJ_SALI_LAB', 'AMPS-NMR', 'AMPLE', 'ALLERGOME', 'ALLPATHS', 'bioconductor-ANAQUIN', 'ANAQUIN', 'ALGPRED', 'AMIGENE', 'ALTANALYZE', 'ALIGN', 'ANCORR', 'bioconductor-ALLELICIMBALANCE', 'ALLELICIMBALANCE', 'bioconductor-AMOUNTAIN', 'AMOUNTAIN', 'ALEAVES', 'ALIGNMENTCOMPARATOR', 'ALLEN_BRAIN_ATLAS-API', 'bioconductor-ALIGNCOPY', 'ALKEMIO', 'ALTA-CYCLIC', 'ANCHOR', 'bioconductor-ALPINE', 'ALPINE', 'ALIAS', 'bioconductor-ALIGNCOPYPAIR', 'ALLOPRED', 'ALEXA-SEQ', 'bioconductor-ANAMIR', 'ANAMIR', 'AMIGO_2', 'ALGGEN', 'AMOEBADB', 'ALLELE_FREQUENCY_NET', 'ALIENTRIMMER', 'AMOS', 'bioconductor-ANALYSISPAGESERVER', 'ANALYSISPAGESERVER', 'bioconductor-AMPLIQUESO', 'AMPLIQUESO', 'ALTER', 'ALTERNATIVE_SPLICING_GALLERY', 'AGGLOINDEL', 'AGE', 'bioconductor-AFFYCOMPATIBLE', 'AFFYCOMPATIBLE', 'bioconductor-AFFY', 'AFFY', 'bioconductor-AFFYPLM', 'AFFYPLM', 'bioconductor-AFFYQCREPORT', 'AFFYQCREPORT', 'bioconductor-AGIMICRORNA', 'AGIMICRORNA', 'AGRIS', 'ALAMUT', 'AFTOL', 'ADVANCED_PDB_SEARCH', 'ADTEX', 'bioconductor-AGDEX', 'AGDEX', 'bioconductor-AFFXPARSER', 'AFFXPARSER', 'AGENDA', 'AHD2.0', 'AGBASE', 'AGRIGO', 'bioconductor-AFFYCONTAM', 'AFFYCONTAM', 'bioconductor-ADSPLIT', 'ADSPLIT', 'ALADYN', 'AFFYMETRIX_NETAFFX_ANALYSIS_CENTER', 'AFLPINSILICO', 'AGILE', 'AGGREGATE_SCORES_IN_INTERVALS2-IP', 'ADJUSTPKL', 'bioconductor-AFFYPDNN', 'AFFYPDNN', 'bioconductor-AFFYIO', 'AFFYIO', 'bioconductor-AFFYRNADEGRADATION', 'AFFYRNADEGRADATION', 'AHMII', 'bioconductor-AFFYCORETOOLS', 'AFFYCORETOOLS', 'bioconductor-AFFYCOMP', 'AFFYCOMP', 'bioconductor-AFFYILM', 'AFFYILM', 'bioconductor-AFFYPARA', 'AFFYPARA', 'ABACUS', 'ABIVIEW_WS_HUSAR', 'bioconductor-ACDE', 'ACDE', 'ACLAME', 'ADEPTH', 'ADENOSILAND', 'bioconductor-ABIVIEW', 'bioconductor-AAINDEXEXTRACT', 'ADD_FASTA_SEQUENCE_LENGTH-IP', 'bioconductor-ABAENRICHMENT', 'ABAENRICHMENT', 'bioconductor-ACGH', 'ACGH', 'ADGO', 'ACPYPE', 'bioconductor-ACDVALID', 'ABBA', 'bioconductor-ABARRAY', 'ABARRAY', 'ACDC_ESTIMATION', 'ADAPTER_REMOVAL', 'ACT', 'ABS', 'ACEVIEW', 'ABYSS', 'bioconductor-ACDTABLE', 'A5', 'bioconductor-A4PREPROC', 'A4PREPROC', 'bioconductor-ACME', 'ACME', 'bioconductor-ADACGH2', 'ADACGH2', 'bioconductor-ACDPRETTY', 'bioconductor-ACDTRACE', 'AACON', 'bioconductor-ABSSEQ', 'ABSSEQ', 'bioconductor-A4REPORTING', 'A4REPORTING', 'ACID', '3DEMBENCHMARK', '2DX', 'bioconductor-A4', 'A4', 'bioconductor-A4CORE', 'A4CORE', '3DLOGO', '3D-PARTNER', '3D-FUN', '3DNALANDSCAPES', '3DTF', '959_NEMATODE_GENOMES', '1000GENOMES_ID_HISTORY_CONVERTER', '1000GENOMES_VARIATION_PATTERN_FINDER', '3D-DART', '.NET_BIO', 'bioconductor-A4BASE', 'A4BASE', '3D-JURY', '4DXPRESS', 'A-MADMAN', '1000GENOMES_DATA_SLICER', '1000GENOMES_VCF2PED', '1000GENOMES_ASSEMBLY_CONVERTER', 'bioconductor-A4CLASSIF', 'A4CLASSIF', '3DSS', '3DLIGANDSITE', '3DBIONOTES', '1000GENOMES_VEP', '2D-PAGE', 'A_PURVA', '4PEAKS', '2BWT-BUILDER-IP', '3MATRIX', '3MOTIF', '3D-FOOTPRINT', '3DEM_LOUPE', '3V', 'MUTATIONINFO', 'ANDI', 'OCHEM', 'RNALIEN', 'EGGNOG-MAPPER', 'PROGENOMES', 'PROTEINSPLUS', 'PRED-TMBB2', 'EXPASY_MOLBIOSRVR', 'OMPDB', 'METAGEN', 'PRED-LIPO', 'PRED-SIGNAL', 'HMM-TM', 'PRED-TAT', 'GWAR', 'CALCHAS', 'BIOTRANSLATOR', 'ANASTASIA', 'BOWTIE2', 'EPGA', 'DRUGEVAR', 'DESUBS', 'CHRONOS', 'BIODATAOME', 'OPEN_PHACTS_API_SWAGGER', 'PROTEINS_API', 'WIKIPATHWAYS_API', 'BRIDGEDB_API', 'SEQUANIX', 'OPENTRIALS_API', 'ORCID_API', 'OPEN_TG-GATES_API', 'OSDB_API', 'OPEN_PHACTS_API', 'YARA', 'POLYPHEN-2', 'CADD_PHRED', 'DGVA', 'MUTATION_TASTER', 'GWAS_CATALOG', '1000GENOMES', 'MATCHMAKER_EXCHANGE', 'GTEX', 'MXM', 'SCENERY', 'TPP', 'BIOCWORKFLOWTOOLS', 'RHDF5LIB', 'RHDF5', 'CESAM', 'BUTLER', 'IHW', 'ISOFORMSWITCHANALYZER', 'SEQUENCESERVER', 'BIQ_ANALYZER_HIMOD', 'MEDECOM', 'HYDDB', 'SEQL-NRPS', 'PATBOX', 'CAW', 'GRNSIGHT', 'BIONODE', 'OSWITCH', 'ANTGENOMES.ORG', 'TGNET', 'GENEVALIDATOR', 'PHYLO.IO', 'RNBEADS', 'LINCS_CANVAS_BROWSER', 'MORPHIN', 'PROTOX', 'INSILICOSIMMOLBIOEXPT', 'WORMNET_V3', 'NEUROPID', 'TOPPMIR', 'PROWL', 'RASS', 'PTHGRN', 'NETVENN', 'RNAPDBEE', 'POCKETZEBRA', 'PROTPARAM', 'TEAM', 'TOGOTABLE', 'MRSFAST-ULTRA', 'RBPMAP', 'SOCKEYE', 'PIQMIE', 'POXVIRUSBIOINFRESCEN', 'SARA-COFFEE', 'VAP', 'TREEJUXTAPOSER', 'POSA', 'NETWORKANALYS', 'R._S._WEBTOOL', 'ONCOSEARCH', 'TSPRED', 'PEP-SITEFINDER', 'PHYLIP', 'PUBSERVER', 'PREDHS', 'NHGRI_MICORARRAY_PROJECT', 'YASARA', 'PRECISEPRIMER', 'SPINDEL_WORKBENCH', 'SWISSTARGETPREDICTION', 'VAST', 'MODEST', 'SPACED_WORDS', 'STARMIR', 'NRPSPREDICTOR2', 'R2_PLATFORM', 'VARMOD', 'SYMD', 'PERSONAL_GENOME_BROWSER', 'XTMS', 'PROBIS-LIGANDS', 'MULTI-FINGERPRINT_BROWSER', 'H_PLUS_PLUS_3.0', 'DDI-CPI', 'HAPEDIT', 'BCM_SRCHLAUSEQUTL', 'MET', 'BIOANALRES_BAR', 'MARBL', 'COMET-UNIVERSE', 'GENOMESCAN', 'CFM-ID', 'GCUA', 'INTEGRALL', 'DOTLET', 'CIPRES', 'E-RNA', 'ALIGNME', 'GALAXYSITE', 'DISEASECONNECT', 'CAD-SCORE', 'AIDA', 'INTEROPORC', 'IMODS', 'BIERAPP', 'BIOCARTA', 'CLIP', 'COGNAC', 'BLANNOTATOR', 'ALIGN-GVGD', 'DMINDA', 'DINIES', 'DUET', 'ALIGNMENT-ANNOTATOR', 'EVIDENCE_FINDER', 'GPS-SUMO', 'GENEGENIE', 'DISCOVERYSPACE', 'ALIGNACE', 'DRAR-CPI', 'ARRAYANALYSIS.ORG', 'BIOINFORMATICS_TOOLKIT', 'BANKIT', 'EVOCOR', 'BIOMET_TOOLBOX_2.0', 'DARIO', 'G-MO.R-SE', 'COUGER', '4PIPE4', 'NCBI_MASS_DOWNLOADER', 'DNASHAPEDTFBS', 'INTERVENE', 'STRUCTURE_THREADER', 'FUSION', 'LFPY', 'NEST', 'SCOARY', 'TRIFUSION', 'THESEQMANIPSUITE2', 'SOSUI_WWW_SERVER', 'TOPMATCH', 'RSLPRED', 'XWALK', 'PROTTEST', 'SUBSEQER', 'PROPRED1', 'RASMOL', 'SSAHA2', 'PRIMACLADE', 'PREDICTIVENETWORKS', 'WEIGHBOR', 'SIM4', 'SEWER', 'SCPSRM', 'TREESAAP', 'SVMTM', 'RNACONTEXT', 'SAM-SEQASSYMANAGER', 'SAGE_HOME_PAGE', 'STADEN_PACKAGE', 'STRAP', 'PSA', 'PUBGENE', 'TIGR_GENE_INDICES', 'POPMUSIC', 'SSTRUCTVIEW', 'POLYBAYES', 'TBBPRED', 'YEAST_METABOLINER', 'PROTOMAP', 'ZPICTURE_MULTI', 'SGRP_BLAST_SERVER', 'RNSC', 'SARPRED', 'TWINSCAN', 'TOPOGSA', 'VARID', 'TGGE-STAR', 'QUADRUPLEX', 'UCSC_SARS_GENBRWSRGW', 'RESOURCERER', 'THEOSPREYNETVISSYSTEM', 'ICE-INTERNCONTIGEXPLR', 'OLIGOARRAY', 'PEPTIDEMASS', 'NCI_FLICKER', 'NETMATCH', 'NLSTRADAMUS', 'NHLAPRED', 'MODIL', 'ORTHSEARCHSERVICE', 'JEVTRACE', 'PHYLOPAT', 'K2N', 'MATCHMINER', 'PATHWAY_MINER', 'MATRIX2PNG', 'NEMINE', 'NEMBASE4', 'MAPVIEW', 'PATHWAY_HUNTER_TOOL', 'MAXALIGN', 'GENMAPP', 'NETOGLYC_PREDSRVR', 'IPI-INTERNPROTINDX', 'HARVESTER', 'MICROBIAL_GENOME_VIEWER', 'NAVIGATOR', 'MLHKA', 'LAGAN', 'IDIOGRAPHICA', 'PEGASYS', 'MEDMINER', 'IMCB-FUGU_GENOME_PROJECT', 'MULAN', 'META-MEME', 'MODSUBNETBIOMIDENT', 'MCODE', 'PEPSTR', 'PHIRE', 'GENESPLICER', 'MCPROMOTER', 'GENOMENET', 'GENEZILLA', 'MONKEY', 'LIVEBENCH', 'LIGALIGN', 'SIDER', 'FACTA_PLUS', 'CELL_PP', 'COMA', 'EMBOSS', 'BTEVAL', 'CISREGTEST', 'FIRSTEF', 'ATSUBP', 'ACELA', 'GELSCAPE_AND_GELBANK', 'DETECT', 'GENEFISHER', 'FASTA_PROGRAMS', 'AD-ENM', 'GENE_SET_BUILDER', 'CANCERGENANATOMYPROJ', 'ALIGN-M', 'CTLPRED', 'GENE_ARRAY_ANALYZER', 'ENRICHMAPCYTOSCAPEPL', 'ESHADOW', 'EPONINE', 'BAGET', 'EPMV', 'ERMINEJ', 'COGS', 'EXON_ARRAY_ANALYZER', 'ARRAYTRACK', 'ARCHAEOPTERYX', 'CNVER', '3D-PSSM', 'ENET', 'APOLLOGENANNOTCURATTOOL', 'BOXSHADE', 'BETATPRED2', 'ARGONNENATLAB', 'AR_NHPRED', 'BIOMOBY', 'CHPREDICT', 'GENEEXPRDATAANALYZER', 'CHILIBOT', 'ALPHAPRED', 'BWT-SW', 'DECA', 'WEBSCIPIO', 'GENEVIEW', 'PKNOT-V2', 'GENEMANIA_CYTOSCAPE_PL', 'SA-SEARCH', 'PINT_CSB2', 'TAVERNA_COMPBIO', 'PETFOLD_AND_PETCOFOLD', 'GENIE_CBDM', 'EUGENE_HOM', 'PUBCRAWLER', 'SNPADOMAIN', 'CHAOS_DIALIGN_WWW_SERVER', 'IMGT_V-QUEST', 'ATOME-2', 'AMICA', 'AUTOCLASS_IJM', 'IBIS_NCBI', 'THE_CELL_IMAGLIB', 'SCDE_STEMCELLDISCENG', 'XANTHUSBASE', 'IMG_M', 'AUTISMKB', 'SUPERFAMILY', 'GENEXPRDB_GXD', 'SRA', 'NCBI_BOOKSHELF', 'PAZAR', 'E.COLI_INDEX', 'GENETESTS', 'DNA_METHYLATION_DATABASE', 'PLANTTFDB', 'DRUG2GENE', 'NON-CODING_RNA_DATABASE', 'EUROCARBDB', 'THE_CHR_7_ANNOT_PROJ', 'SIRNADB', 'MOUSE_ATLAS_GENEXPR', 'ALLEN_BRAIN_ATLAS', 'BUGASBASE', 'CANSAR', 'VFDB', 'ZFISHBOOK', 'YEAST_KID', 'ANIMAL_TFDB', 'PHOSPHOSITEPLUS', 'VIRSIRNADB', 'AH-DB', 'INTNUCLSEQDBCOLLAB', 'PRIDB', 'THE_BRENDA_TISONT', 'SMPDB', 'IMGT_3DSTRUCTURE', 'BIND__BIOMOLINTNETDB', 'IEDB-3D', 'REPAIRTOIRE', 'FINDBASE', 'STRGENHISTTAILREC', 'SOURCE', 'SCOR', 'SIGREANNOT-MART', 'FANTOM_FUNANNOTMOUSE', 'ENSEMBL_MOSQGENBRWSR', 'FUNSHIFT', 'EUGENES', 'GELBANK', 'GEMMA', 'THE_GRID', 'INTEROLOG_REGULOG_DB', 'IMGT_HLA_DB', 'STANFORDMICROARRAYDB', 'NEUROSCINFFRAMEWORK', 'UCSC_MSEGENBRWSRGW', 'ISLANDVIEWER', 'FPC', 'GFS', 'NJPLOT', 'DISEMBL', 'TREE-PUZZLE', 'GENEPATTERN_2.0', 'VECTORNTI', 'PROCKSI', 'CAP3SEQASYPROGRAM', 'TRAFAC', 'PROSPECTR', 'PROGNOSCAN', 'UCSC_CELGENBRWSRGW', 'COILS', 'NCBI_ASSEMBLY_ARCHIVE', 'HELIQUEST', 'INTERGR8', 'CHINOOK', 'ET_REPORT_MAKER', 'JOY', 'RB-PRED', 'STREAM', 'INT_HAPMAP_PROJECT', 'BETATPRED', 'PHYLOPRO', 'VEGA_ANNOTATION_BROWSER', 'VIRAL_BIOINFORMATICS', 'PROPRED', 'GENEEXPRESS2.1', 'VISCOSE', 'UCSC_FRFGENBRWSRGW', 'REPRO', 'ULYSSES', 'PEDRO', 'TALE-NT', 'BLOCKS_WWW_SERVER', 'UCSCRATGENBRWSRGW', 'BETATURNS', 'DBD2BS', 'GENOMETRAFAC', 'TREEVIEW', 'MIAMEXPRESS', 'CLUSTALX', 'WORDCLCYTOSCPLG', 'ENSEMBL_GENBRWSR', 'MARTVIEW', 'MREPS', 'CN3D', 'MEGA', 'CYTOSCAPE_WEB', 'UCSC_HUMGENBRWSRGW', 'ISLANDPATH', 'UCSC_CBGENBRWSRGW', 'CONSEQ', 'DEEP', 'LOGOBAR', 'PA-SUB', 'MINMOTIF_MINER', 'POLYPHRED', 'BIND_SOAP', 'UCSC_FUGENBRWSGW', 'SWISS-PDBVIEWER', 'EQUILIBRATOR', 'DOELAN', 'TRIMAL', 'NMRPROCFLOW', 'METFAMILY', 'SPECI', 'SIAMCAT', 'IPATH']\n"
     ]
    }
   ],
   "source": [
    "biotools_ids = [element['id'] for element in biotools_list]\n",
    "biotools_ids_processed = []\n",
    "for b in biotools_list:\n",
    "    if 'BioConductor' in b['collectionID']:\n",
    "        biotools_ids_processed.append('bioconductor-%s' % b['id'].upper())\n",
    "    if 'EMBOSS' in b['collectionID']:\n",
    "        biotools_ids_processed.append('bioconductor-%s' % b['id'].upper())\n",
    "    else:\n",
    "        biotools_ids_processed.append(b['id'].upper())\n",
    "biotools_ids = biotools_ids_processed\n",
    "print(biotools_ids)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## get list of bioconda entries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2017-10-10 11:55:40--  https://conda.anaconda.org/bioconda/linux-64/repodata.json\n",
      "Resolving conda.anaconda.org (conda.anaconda.org)... 104.16.193.233, 104.16.192.233\n",
      "Connecting to conda.anaconda.org (conda.anaconda.org)|104.16.193.233|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [application/json]\n",
      "Saving to: ‘repodata.json.4’\n",
      "\n",
      "repodata.json.4         [    <=>             ]   1,46M  1,79MB/s               "
     ]
    }
   ],
   "source": [
    "#repodata_url = 'https://conda.anaconda.org/bioconda/linux-64/repodata.json'\n",
    "#repodata = requests.get(repodata_url).json()\n",
    "!wget https://conda.anaconda.org/bioconda/linux-64/repodata.json\n",
    "import json\n",
    "bioconda_list = json.load(open('repodata.json'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "_and get the list of bioconda package names_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n"
     ]
    }
   ],
   "source": [
    "#bioconda_ids = [element['name'] for element in bioconda_list]\n",
    "#print(bioconda_ids))\n",
    "bioconda_ids = set([package['name'] for key, package in bioconda_list['packages'].items()])\n",
    "bioconda_ids_processed = []\n",
    "\"\"\"\n",
    "for bioconda_id in bioconda_ids:\n",
    "    if len(bioconda_id.split('-'))>1 and bioconda_id.split('-')[0]!='bioconductor':\n",
    "        bioconda_ids_processed.append(bioconda_id.split('-')[1].upper())\n",
    "    else:\n",
    "        bioconda_ids_processed.append(bioconda_id.upper())\n",
    "\"\"\"\n",
    "bioconda_ids = bioconda_ids_processed\n",
    "print(bioconda_ids)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "this is the intersection of biotools/biocondas entries, based on strict name equality"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "intersection = len(set([b.upper() for b in bioconda_ids]) & set([b.upper() for b in biotools_ids]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n"
     ]
    }
   ],
   "source": [
    "print([b.upper() for b in bioconda_ids])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [],
   "source": [
    "not_in_bioconda =  set([b.upper() for b in biotools_ids]) - set([b.upper() for b in bioconda_ids])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "set()\n"
     ]
    }
   ],
   "source": [
    "not_in_biotools = set([b.upper() for b in bioconda_ids]) - set([b.upper() for b in biotools_ids])\n",
    "print(not_in_biotools)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<matplotlib_venn._common.VennDiagram at 0x7f22a861deb8>"
      ]
     },
     "execution_count": 111,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAUwAAADuCAYAAABSzT4OAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMS4wLCBo\ndHRwOi8vbWF0cGxvdGxpYi5vcmcvpW3flQAAIABJREFUeJzt3XeUnPV97/H3d3tTr6ghgSQQIAkB\nAiGMEYimgIBjHAjBYEyOfcn1IbaDzc0xkGUSwA4hN/deHGLHAcxJDKYKEBgMsuhFAkuoI9S16m0l\nbW/zvX/8ZqXRasvs7sz8nmfm+zpnDoumPN/dnf3M8/yqqCrGGGO6luO7AGOMCQsLTGOMSZAFpjHG\nJMgC0xhjEmSBaYwxCbLANMaYBFlgGmNMgiwwjTEmQRaYxhiTIAtMY4xJkAWmMcYkyALTGGMSZIFp\njDEJssA0xpgEWWAaY0yCLDCNMSZBFpjGGJMgC0xjjEmQBaYxxiTIAtMYYxJkgWmMMQmywDTGmARZ\nYBpjTIIsMI0xJkEWmMYYkyALTGOMSZAFpjHGJCjPdwEmiURycL/TPCAfECAau7UAzag2+SvQmHCz\nwAwDkQKgT+xW1ubrIlw45pPIFYMIQBPQANQA1bFbVdzX1ag2Jvm7MCb0RFV912DiiRQBQ4Ehcbdi\nD5UcBvYBe2O3fRaiJttZYPomMhAYhQvJobizxqA6DOwBtgMVqNZ6rseYtLLATDeRPFxAnhj7b6nf\ngnplP7AVqAB2Y28mk+EsMNPBheRJsdtIINdvQSnRiAvODcBWVKOe6zEm6SwwU0lkGHAKcDKuUyZb\n1AMbga9Q3eO7GGOSxQIz2URKgImxW3/P1QTBAeBLYB2qDb6LMaY3LDCTxXXenIm77LYJAcdrwgXn\nclRrfBdjTE9YYPaWyFBgGq4Tx3QtCqwDvkD1kO9ijOkOC8yeEhmJC8oRvksJKQU244Jzr+dajEmI\nBWZ3iQwBzgeG+y4lg2wEPkW12nchxnTGAjNRrjPnXFxnjkm+FmA57ozT5rubQLLA7Ipb0GIycBbZ\nNTTIl1pgMapf+S7EmLYsMDsjMgZ3+d3PdylZaB/wPqr7fBdiTCsLzPaIFAIXAON9l5LlosBSYKnN\nHDJBYIHZljur/DpQ4rsUc8Q+4B1UK30XYrKbBWYrN9/7fGCS71JMu1qAz4AVtsiH8cUCE0BkEDAb\nm8oYBjtxZ5s2BMmknQWmyATcJXgmriCUqeqBBaju8F2IyS7ZG5giApwHTPFdiukRBRahutx3ISZ7\nZGdguj1yZgOjfZdiem0dbvhRi+9CTObLvsAU6QdcgbVXZpI9wB9QrfNdiMls2RWYIiOAy4EC36WY\npKsGXkP1sO9CTObKnsAUGY0LS+vcyVy1wOs2XtOkSnYEpshY4FJsYd9sUA/8PlOnVEpEBGjv1qzl\nNhsq1TI/MEVOBi7GwjKbNAJvorrLdyGJkogUAQOBAUCfuFtf3KIv0sVLKK5Z4nDsVhX39WEttz3l\nkyGzA1PkFNwYy67ebCbzNOM6grb7LqSt2FniUGAMMAQXlKmeiluP219pG1Ch5bo/xcfLSJkbmCKn\n4sLSZK8W3OX5Tt+FSOSY/ejHAMV+K6IWty1yBbDNzkATk5mBKXIiroPHzixNI/CKj44giUgJLhzH\n4rYyyUt3DQmK4oZmbQU2abnttdSRzAtMtynZ1QT3zWnSrwZ4OV27VUpERuF2EA3rfk/bgNXAFi3P\ntIDoncwKTJG+wHVAke9STOBU4s40U3LpGWuXHIcLysGpOIYHVbjgXGOX7E7mBKZIES4s+/ouxQTW\nTlybZtKmUUpEcnD7PE0lc1fmbwRWAiu1XOt9F+NTZgSmW8vyalzPozGd2Yjqgt6+iEQkH7d26hSy\nZ7HpZtwZ55JsPePMlMC8GJjguwwTGp+guqKnT5aIjAdmkr1NP7XAR1qum3wXkm7hD0yR04Cv+S7D\nhEoUeBXVPd15UqzX+0Lc0CADm3HBmZbOtCAId2CKDAGuweaHm+6rBl5CE2uTk4hMxJ1V2sItx2rE\nbR2yOht61MMbmCL5wPVYJ4/pua2ovtnZAyQiZbizSls7tXO7gfe1PLMXPglzYF6CbYNreu8zVJe2\nd4dEZBJuVX47q0xMFNchtMR3IakSzsB0+/Bc7LsMkxEUNz7zSHtmrAd8Nm6Wjum+jcA7Wp55q+CH\nbwUfkWJcW5IxySDALERy4cgl+LVYWPbGScDc2ApMGSV8Z5gis4GTfZdhMs4Xcj+bcduX+F4YI1NU\nAW9ouR70XUiyhOsM062abmFpkm77GCaOr+NyLCyTqQ9wnUQkrHPqjxOewHSzeWy8pUm6zeOpXTqT\n4r/dRa4oIbvkCrwC4M8kIqf4LiQZwhOYcA7uE8uYpFl3GrUrp1OCIKMbKfzGAWznyeTLAS6SiJzt\nu5DeCkdgigwCzvBdhsksm8dTu3bqsfPA/2I/hWUtZFzvbkCcLRE53XcRvRGOwIQZhKdWEwI7R1G3\n8uzj2yuLlNxv7yWrV+RJsZkSkdBOLQ1+CLm9xEf6LsNkjv1DqF8yk0Jy2l+R/9JDFA9poinddWUJ\nAWZLRIb4LqQngh+YMN13ASZzHO5H4+JZ5Gtux+/9PMj53h4LzBTKA66UiISuTyLYgen25hnmuwyT\nGeqLaf5kNjkteV0v1nJuNcXj6snKNR/TpBiYIxEp9F1IdwQ7MO3s0iSJgi7+Os1NhYnt9ZQDcsdu\n6/xJsf7AFRKR0Kw2FtzAFBmP26/ZmF5bcya1hwd2b8Hf0+rtLDMNhgMX+S4iUcENTJjmuwCTGfYN\npX7jqT3bRuIv99Oc7HrMccZLREKxY0IwA1NkJDDAdxkm/BoLaFlyAblIz/aon15N8eAmC800uEAi\nUuq7iK4EMzAh1INbTXB8/jUaG4vI7+nzc0Fu3E9DMmsy7SogBJfmwQtMkT7YnikmCbaeRO2BYb1f\nTOPiwxQVRYkmoybTqVESkVN9F9GZ4AUmnAY9u3wyplVzHtHV05KzUnqhkju30mb/pMl5QV5HM1iB\n6VYkCvQnjAmHNWdS11yQ2BCiRFx+yDbaS5NC3FToQApWYLo9ekI1kNUET00ZTVtPTu66lsObKLQh\nRmkzUSJygu8i2hO0wJzouwATfsvOpVlzkv/evvqgTZdMowt8F9Ce4ASmSBluEKsxPbZ7BHXJ6Ohp\nzwVVFObaAsPpMlAiErh9lYITmLb1hEmC1dNS954ujZJ3brV1/qTRVN8FtBWkwDzJdwEm3PacQH1N\n39S2gV9+yM4w0+gEichQ30XEC0ZgusvxUK6PZ4Jj7ZTUH+OMWgpt35+0CtRZZjACE8b6LsCE28EB\nNBzq5uIaPVGk5J5RZzN/0misRKSf7yJaWWCajLD+9PQtxfa1Klv2LY0ESMO1Q2L8B6YbrG6946bH\nGgpp2T0y9WeXrc6u7vncdNMjEyUigdgv3n9gwlCCUYcJqS3jqU/FuMuODGumwPb8SatcArIgTxCC\naoTvAky47Tgx/dMWZ1RbYKbZeN8FQDACM5BToEw41BXTXN0vfZfjrSbX2gIxadZXItLfdxF+A1Mk\nF3dJbkyPbBvnZ373hHpbjMMD7zN/fJ9hDgF745me2znGz3t4cDMFJS3WW55m3tfJ9R2Ydjlueqyh\niObD/f2tbnV6na1elGbDJCJJWeO0p3wH5iDPxzchtv1EGnu6V08ynFFrq7CnWQ4w2ncBPnlvxDXh\ntdfz6N1T67z//WQjr+2Y/n7hIoIFpumFwwP8DiA/oSl5K7qbhI2WiHi7qvD5CdnX8/FNiDUU0tJQ\n7Dcw+7WQZ+tjpl0RHhfq8RlYtu+46bEDQ/wPHM8BGd7ov44s5K3vw2dg2uW46bEDQ4IxpGdMYzDq\nyDLeVi+yM0wTSpWDg9GcM6rRLsk9yMrALPV4bBNy1X2DsWLQaBuJ6UNWBmZgN2s3wdaSQzSZe473\nxuAmm1PuQR9fPeUWmCZ06ouD025YFg1G00CWyQXKfBzYAtOETl1pcAKztMXOMD3xclnuJzBFCrwd\n24ReXWlwpiSW2BmmL1kUmHZ2aXqhtjQ4PdNFFpi+eAlMXw3nWRmY66D4Wrh1F4wQ4B/gqRdg2jKY\nkgstg2DvfPjNBKj7FYy9D24BUOA7MP9h+ALgL+CSBXChglwKHzwLf/T5faVbXTfGV+yooviB97m1\nst6t7P+tyTx11UQ2AvzsQy77uIJvPjqHu8b2p3pnFcX3v8ft1Y0MVCX3/NG8dee5fNzZ6+dBTmGU\naEMat8jo1GYGMI/v0EhfAE7iff6chSxjFAu4mRYKKWI/t/A4A6jnc8ay0L3PAJjGfC5z7zMAmhH+\nlXso5CB/wy/S/v10zMsqVb4CMxBDQtLtBrhxBqx6An51CHL3QEElFL0O80ogegl843/AnIXw0rWw\n4xZ4sASii6DfxXDf/bD8VRi+AC5cCT/rC82nww9eh+VXwV7f31+6NBUk3m74Tx9y4ymDWfWD8/hV\nTSO5hxooAFi5hwEbDnBacR4HWh/7n0uZNbiYnb+6mn/bVEnZXW/xj7efyaLSgs7bTPu0BCgw84gy\nixeYxlYOUsgvuZdVrOFtbmUmzzOTdbzKTF7jcm7hVU5lB1N5kHyibKMfv+E+ZrGc/Fizx0vMppSd\nNBOITcjieFlH19cvORhvrjTaAMUbYeKv4UOAftAyAeruhdUluDfnObBxX2xA/3BobP33Q3EfbIvh\nhLGwqfX+0+Grx2Gah2/Jm2iC756dVRTvrGbinee6n3lpAS0j+lAH8Nhn3HDzZF6Eo5f3AtrQQlFU\n4WA9hfm51BTmdd1eKgSniYBRHGIaWwHoTwOl7GQ//aljKDNYB8AU1rCDswAoo/FIODa0OYHaQn+2\nM5kp7ucXMF4yxNcZZtb1LH4Eg0qgajrctg1GjYEtr8KzIzi6CO3LcMGl8Hnr//8Sxt0D3z4MA38E\nT5RA9ALY/iRctwpKB0HTUjhjLGzx8k15ogn+qazZx6CiPKp+9Adu21/LqCGlbLn3Qp597SsmlRVw\n8OJxbPv3z48+/ntn887fLeD7NzzPw81Rir55Gr/Oy+k6DPOCE5fH2sggqhjNZDbxGTv5I2dyGV+w\nmLNpYOCRx33GOBbybRoYyPk8cSRA53MjF/EidYFsQsuqM8ysC8xGyN0NY74H7+2BB4qg8TtwZev9\nc2FODkQfhUWt/3YHbNoP9/8OHvovmLMP8q6HXd+ANy+BH06HvxkNFTkEp9c4HaIJLhrcHCX3YD1j\nrjyZ9/77GzyQn0PjI58w94+bmPOjGbza9vHz13L6kFK2Pffn3H3P1/nHV9dy0+7qrsMikJdLhyjk\nRe7gHJ5jAPX8GU+xiov4Z+6hkSKE5iOPnc4m/hf3800eYhlzqCWPhUymiCrOip2tBo+XH7uv33VQ\nP5NTZipUlkHlHbAJ4Hr408bYYqjfg/OXwJR34fH2Pjavh12F0PAyjAR4HD7aDQ9WwCNlUHsi7E7f\nd+JfToJLqo3rT2VRHpVzJrif+QWj+dOeGsbUNDH4B29y3w3P81B9MwN+8jb3rNtP30XbmTlzFEty\nBKaPYG9ZAfuW7qLLZYoD92nVQC5PcgcnsojLWQrAJHbxQ/4vP+FBprOY4nbavE9jF7k08CUjqWA8\nu5nKQzzEu3yXg5zCv3N7ur+VTnj5sfu6JM+6wJwOh/tB5csw7DrY/RZMGgU7I3D6i3DFQnhkeNzl\n+Vsw6GtQWQLRhTBwLww/D/YDLIU+06DqXRi4BKYtgp97+8Y8kGhiZ5gTBnG4JJ/KT7cxbMYodi/d\nxaShpWx9/Br+tfUxNzzPQw9fxkNj+1Pdp5ADS3cx6dpTWb/+AH2qGhh26mD2dXWc5iBdL0WBJ7mV\nvuzkBhYc+fed9OEEqmhBeIermMh7AGxgEGOoJJ8omxhILcMZyX7OYh4wD4CPmMifuJy/5gkP31FH\nvExesMBMowfhmTvgr74LeYNg7yvw1Hnw0xbIuwx+BDAONi6C386D8TfBnFxoEYjeCU9PhmqAa+CO\nWijNgZafwDMTcB0Z2UK6sWjvrVN55t8+468eXUxenwL23vt1nurosd89i9cf+Zjbbn6Rv1eQS8bx\n0tj+7mfeGQ1SE9NixrOLGZSwnYe5D4DpzGMfQ9nAxQCcwBKujg2XWsN4XmAOQgtClHN5mmFdf88B\n4OUMU1Q9ZJfIcOCa9B/YZIIvZlC7bRwlvutodeN4WmpzbbvoNFuv5bow3Qf11YZZ7+m4JgMU1Qbn\nCqUF1MLSCy8L61lgmtAprgnOJXB9TvD6fLLEIR8H9ROYqvVkaTum6b2SmuCM5KnLCc7KSVkmiwLT\nafB4bBNixQEKzOoEBrablMi6wMyqnl2TPEW1wVhtHaAm1wLTAwWqfBzYZ2BaO6bpkbwWcnKbg3Ep\nfDDX2jA9qNJy9fJz9xmYtR6PbUKutCoY+4FXFNoZpgdeLsfBb2Ae9HhsE3L99wfjDHNrQXDaU7NI\nVgZmpcdjm5AbtCcYQbUtILtXZpnDvg5sZ5gmlAbuCcYi1DsKbNC6Bwe6fkhq+A5Ma/8xPVJcR15e\nY9wSZR4cyqUpMCutZ48mYJevg/v7ZatG8XhqbcKvb6Xfjp89ecFoR80y2331kIP/tU+tHdP02KA9\nfof0fFVsQ4o88LqgsQWmCa1Rm92GZr6sKAnOnPYsktWB6a0twoRfaTX5JVX+ptiuKg5Gx1MW2afl\n6nX8dhAC0zp+TI+dUOGn4+dgLk0H82xIUZp53+zPb2CqNkHXWwAY05HRG/2c5W0sDMZMoyzjfUM2\n32eYADt9F2DCq6yKguLq9C8muzIw671njVot1+M3bkuzIATmDt8FmHAbvi39Z3ufltnleJpt8l0A\nBCMwrR3T9Mq4tRTQjY3ReutgLk0VhX576LOMAit8FwFBCEzVRqwd0/RCSS35A/ekb7nApaXWfplm\nm7RcAzHJxX9gOt57v0y4TVidvjGRH5XZ+Ms0W+a7gFZBCcxAtE+Y8Bqyi6KS6tSPyWyC6JJSClN9\nHHPEjiB09rQKRmCqVuJxjTuTGSasTP3c7nVFNDTZghvptNx3AfGC9Ivf4LsAE26jNlFcWJfa9sUF\n/ayDMo0OaLl6H3sZzwLTZAwBmbgidYHZILS835eiVL2+OU6gzi4hSIHpLsu9LQxqMsOJGygpO5Sa\ntszPS2mw9S/TphpY77uItoL2y//KdwEm/KYuSs3rvj4gcH8vmexTn+tediRob4C14HcVbRN+A/ZT\nOLwiubuSVubStKLELsfTZJuW60bfRbQnWIGp2kAAT8NN+Ez+nIKcJO5d/l7f9M9Xz1ItwIe+i+hI\nsALTWeW7ABN+hfXknbwmObN/miE6b6CNvUyTL4Iyq6c9wQtM1f3YwsImCSasSk4H0OIyGg7Y2pfp\nsB9Y6ruIzgQvMJ2Vvgsw4ZejyPT3yOntpfl/D7atdNMgCrwTxI6eeEENzM1Aje8iTPiV1pA/5bOe\nn2WuLqLOViZKi8+1XAM/rDCYgem24A3MhHsTbqM2UzJ8a896zZ8ZHNC/kcyyi5D8vQf5zbAGN3jV\nmF4781OKimq619NdUUDDF7bQRqodAt7Scg3FlNPgBqZqC7DEdxkmM+S1kHPue5DTkvhe4r8eavPG\nU6weeEPLNW1rmfZWcAPTWQsc9F2EyQx9D1Fwzgc0Eu06CL8som5pqQ1UT6Fm4M0gDyFqT7ADU1WB\nz32XYTLH0J0UnbmIus62tIiCPjbMesZTSIGFWq57fBfSXcEOTADVjdgWFiaJRm2m5NRl1HV0/+Iy\n6jYVWc94Cn2i5brZdxE9EfzAdFK0nILJVuPXUDJ27fFD15oh+h9D/ex1niVWaLmGdpx1OAJTdTs2\nx9wk2RlLKB2x+djhRm/2p25vvgVmimwAPvVdRG+EabrXJ8BosGEeJnnO+oSS/CZqtkygtDKXpieG\nUOy7pgy1TMs19FeK4TjDBFCtI+SfTiaYJn9O6YQV1Dw6nGbbryfposD7mRCWEKbABFBdC+zwXYbJ\nPKesZPtnZXwMqd9ILYs04oYOfem7kGQJV2A6H2BvapNctbie2w3AfOi4B90krBp4Rct1m+9Ckil8\ngal6CJsBZJLrw9ji1cTGBs7D9pfqjb3APC3XSt+FJFv4AtP5AtjpuwiTEdagx44J1HKtBl4GVoBN\nj+ym9cB8LdeMPEsXDcec9+OJlALXg01fMz22D3gltm5BuyQiw4CLgP5pqyqcaoAPtVy3+C4klcIb\nmAAiY4ArfZdhQqkReAntei6zRCQXOBuYCkiqCwsZBVYDi7VcU7YnfFCEOzABRM4FzvRdhgmdt1Hd\n1J0nSESG4M42B6ampNA5gBsyFLo54T2VCYEpwFXACN+lmNBYgeonPXmiRCQHOAv3IR3WPoDeal16\ncVnQt5RItvAHJoBIMXAd0Md3KSbwdgPzY6v695hEpB/uEn0i2RWcW3FDsA75LsSHzAhMAJF+wLVY\nJ5Dp2CFcJ0/SFqyViJQAU4BJkLFz0BXYhNsCN6tXDsucwAQQGYa7PA/THHmTHrW4sKxKxYtLRAqB\n04EzyJwP7RbcMKFlWq62kDeZFpgAImOBy7DeTHNUE+4yPOVnRxKRPOBU3FlnWaqPlyLVuJ7vL8O0\nfUQ6ZF5gAoicDlzguwwTCFHgjdgSgWkjERFgGDAWOBHol87j90AdUIG79N4alk3J0i0zAxNsuJFp\ntRBV72upSkT6czQ8h+L/CkiBPbiQ3JrtbZOJytzABAvN7KbAB2jwVsqRiBThgnMMMIT0XbrXAttw\nPd3btdzNnzeJy+zABBA5CzjHdxkmraLAu0E4s0yERKQANxh+IDAANzyu9dbdDswa4HDsVhX39WFr\nj+y9zA9MAJEpwAzfZZi0aAH+2HZBjbCSiBTjhitJF7dGoErLO54Xb3ovOwITrCMoOzQDb6GZtQaj\nCY7sCUwAkVOAC8mumRnZohF4E9VdvgsxmSu7AhNAZARwKZkzuNi4cYNvomqL/pqUyr7ABBDpC1yB\na2A34bYbdxmekQvWmmDJzsAEEMkHLsEN7zDhtBa3vYR1dJi0yN7AbCUyHZjmuwzTLVHgI1TX+C7E\nZBcLTACRccDXgULfpZguVeGGDWXNorUmOCwwW7k9gi7GFiIOsi+BT9DM3wrBBJMNr2mlWgO8DnyK\n7XseNLW4XvD3LSyPEpHNInJpO/9+oYis9VFTR0RERWS87zp6y9aNjOdOt5cjshWYhVskwfi1gbh9\nw03XVPUD4BTfdWQiC8z2qB5E5BVgMm7/lgLPFWWjOuBjVDf4LsSYVnZJ3hFVRXU58Cxu+IpJjxZg\nGfCshWVCpovIahGpFJEnRaRIRGaJyJHpoSIySUTeFZGDIrJKRK6Ju69YRP5FRLaIyCER+VDcHlmI\nyDWxxx+MPX9S3PM2i8iPRWR57HnPikhR3P0/EZGdIrJDRG6PL1hErhKRpSJyWEQqROT+lP6EksgC\nsyuqdai+B8zDDZI2qbMJeB7VRag2+i4mJG7GTcI4Gbch273xd4obbzwfeAvXxHQn8Ftx04QBHsHt\nuT4Tt1rS3UBURCYCzwA/xC1B93tgvojEX23dAFwJjMOtMH9b7JhXAj/G7XwwATezLl4NcCvQH7el\nzF+LyHW9+BmkjQVmolT3ovoKsBD3CzfJsx94DdW3UT3su5iQ+YWqVqibFvogcFOb+2fg1tv8uao2\nqupC4DXgJhHJAW4HfqCq21W1RVU/VtdefCPwuqq+ra6j7RGgGBesrf6fqu6IHXs+R9eevQF4UlVX\nqutMvT++IFV9V1VXqGpU3VXcM7j93gPP2jC7S3U9IptxjepTCe++LUFwAHf5vR4b39ZTFXFfb+H4\nYXEjgAo9dlvhLcBIYDBuTYX2mj5GxB4HgKpGRaQi9rxW8Qud1MYdewTwpzbHO0JEzgN+jtswrgA3\n/vn5dmoIHDvD7AnVZlRXAb8D3gVsR73u2Q38AdUXUF1nYdkro+O+HgPsaHP/DmB07Gwy/nHbgX1A\nPe5yvq0dxE0bFhGJHSuRvZF2tlNXvKeBV4HRqtoP+CX+t+xIiAVmb6hGUf0K9+m4APcGNB2rAF5F\n9RVUt3T5aJOI74vIKBEZCNyD66SMtwh39ne3iOSLyCxgLvC72FnnE8D/FpERIpIrIueLSCHwHHCV\niMyOtYPeBTQAHydQ03PAbSJymoiUAOVt7u8DHFDVenHbyPxlj75zDywwk8H1qG9E9SXcJ+dXuMVs\njVuncjXwAqpv2HqVSfc0rkNnI+7S+oH4O9V1ns0F5uA+0B8DbtWjex39GFgBfIZrIvknIEdV1wLf\nAh6NPW8uMFcT6IxT1TeA/4Nr718f+2+8/wn8g4hUAX+PC9hQsKmRqeI+lU/GtXUO81yNDztwHxwb\nUbUPD5MRLDDTQaQ/bsjHSUBfz9Wk0n7cWc56VKt9F2NMsllgppvIAFxjelD2p+6NRlwnQAVQEZuP\nb0zGssD0yTWujwRG4S7b+xPsAI3i2rlaQ3IXxw5XMSajWWAGiWv3HIybWdF683UJH8UNl9obu+0D\n9tvq5iabWWAGnTsL7YcbitEHN1A+/uveTD5owG0gVo1bmDf+60rrrDHmWBaYYSeShwvN/Ngt/v9z\ncItZRGP/bQaacG2PjRaIxnSPBaYxxiTIBq4bY0yCLDCNMSZBFpjGGJMgC0xjjEmQBaYxxiTIAtMY\nYxJkgWmMMQmywOxAbFe8tps3ISIXioj3XSRF5Dci8kDXjzTGJIsFZjep6geqekrXjzxeRyFsjAkH\nC0xjjEmQBWbnpovIahGpFJEnRaRIRGaJyLbWB4jIpNgm9wdjm95f094Lich/4TaDmi8i1SJyd+zf\nr4k972DsdSb14LUHi8hrsccdEJEP2mx6ZYxJAvuj6tzNwBW4rSYmAvfG3xnbHGo+bk+VocCdwG9F\n5LhLdlW9BdiK2xelTFUfFpGJuD2Zf4hbyu33uEAt6M5r4zao2hZ7jWHATwFbJMCYJLPA7NwvVLUi\ntlH9g8BNbe6fgVti7eeq2qiqC4HX2nlcR24EXlfVt1W1CXgEKAZmdvO1m4ATgBNVtSnWzmqBaUyS\nWWB2riLu6y0c3ai+1QigQo9ddXwLx25235kRxG1yH3uditjzu/Pa/4zbne8tEdkoIn+X4PGNMd1g\ngdm5tpvR72hz/w5gdJv2wjHwo+7zAAABDUlEQVR0vNl927O+Hbi9fQAQEYkdc3t3XltVq1T1LlU9\nCbgG+FsRmd3hd2WM6RELzM59X0RGichA4B7g2Tb3LwJqgbtFJF9EZuH2b/5dB6+3G7dzZKvngKtE\nZHaszfIu3CroH3fntUXkahEZHwvcQxxdNNgYk0QWmJ17GtfpshG3fewxA8Vjm9rPBebg9rx5DLhV\nVb8EEJGfisgbcU/5GXBvrDf7x6q6FvgW8Gjs+XNxnUKNXb12GxOABbjtJT4BHlPVd5LxAzDGHGUr\nrhtjTILsDNMYYxJkgWmMMQmywDTGmARZYBpjTIIsMI0xJkEWmMYYkyALTGOMSZAFpjHGJMgC0xhj\nEmSBaYwxCfr/tYsbMalxf3wAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7f22a87156d8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%matplotlib inline\n",
    "from matplotlib_venn import venn2\n",
    "venn2(subsets=(len(biotools_ids), len(bioconda_ids), intersection), set_labels = ('bio.tools', 'bioconda', 'overlap'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "questions:\n",
    "* non-bioinformatics conda packages (yamllint?)\n",
    "* packages that include a version number in biotools: ggplot2\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': '0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.1.3',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.19.52',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.7.11',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r >=2.8.0',\n",
       "   'r-dbi'],\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': '7e2c7475f6d59cdddbb995aa969b57f3',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 1980788,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.16.1'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': 'r3.3.1_0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.7.1',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.23.10',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.9.47',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r 3.3.1*',\n",
       "   'r-dbi'],\n",
       "  'has_prefix': False,\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': 'c43c37e78cc910a8e8f04a1f792fcb08',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 2386659,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.20.3'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': '0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.1.3',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.19.52',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.7.11',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r >=2.8.0',\n",
       "   'r-dbi'],\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': 'aae18d3c7c4e358d380ca3390f75680e',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 1982400,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.16.4'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': 'r3.3.2_0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.7.1',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.23.10',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.9.47',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r-base 3.3.2*',\n",
       "   'r-dbi'],\n",
       "  'has_prefix': False,\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': '57b392256753a0d525e9167453e7ab22',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 2387193,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.20.3'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': '0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.1.3',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.19.52',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.7.11',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r >=2.8.0',\n",
       "   'r-dbi'],\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': '67c4a7911b5068e6e2406846d89f5402',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 1979666,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.16.0'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': 'r3.3.1_0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.7.1',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.23.10',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.9.47',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r 3.3.1*',\n",
       "   'r-dbi'],\n",
       "  'has_prefix': False,\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': '0926840187963f318e435f7f8d160df8',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 2543421,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.18.7'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': 'r3.4.1_0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.7.1',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.23.10',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.9.47',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r-base 3.4.1*',\n",
       "   'r-dbi'],\n",
       "  'has_prefix': False,\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': '1a75be779ef523732e2a165819a5ecb2',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 2439208,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.20.3'},\n",
       " {'arch': 'x86_64',\n",
       "  'binstar': {'channel': 'main',\n",
       "   'owner_id': '55f35cc885a76554d7c66613',\n",
       "   'package_id': '562657e9d6f51f0b163323ba'},\n",
       "  'build': '0',\n",
       "  'build_number': 0,\n",
       "  'depends': ['bioconductor-annotationdbi >=1.27.9',\n",
       "   'bioconductor-biobase',\n",
       "   'bioconductor-biocgenerics >=0.15.3',\n",
       "   'bioconductor-biostrings >=2.33.5',\n",
       "   'bioconductor-bsgenome >=1.37.6',\n",
       "   'bioconductor-genomeinfodb >=1.1.3',\n",
       "   'bioconductor-genomicfeatures >=1.19.17',\n",
       "   'bioconductor-genomicranges >=1.19.47',\n",
       "   'bioconductor-iranges >=2.3.25',\n",
       "   'bioconductor-rsamtools >=1.19.52',\n",
       "   'bioconductor-rtracklayer >=1.25.16',\n",
       "   'bioconductor-s4vectors >=0.7.11',\n",
       "   'bioconductor-summarizedexperiment >=0.3.1',\n",
       "   'bioconductor-xvector >=0.5.6',\n",
       "   'bioconductor-zlibbioc',\n",
       "   'r >=2.8.0',\n",
       "   'r-dbi'],\n",
       "  'license': 'Artistic-2.0',\n",
       "  'machine': 'x86_64',\n",
       "  'md5': 'f9182e4d677a705b0cc11125f572e9ad',\n",
       "  'name': 'bioconductor-variantannotation',\n",
       "  'operatingsystem': 'linux',\n",
       "  'platform': 'linux',\n",
       "  'requires': [],\n",
       "  'size': 1983074,\n",
       "  'subdir': 'linux-64',\n",
       "  'target-triplet': 'x86_64-any-linux',\n",
       "  'version': '1.16.3'}]"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[package for key, package in bioconda_list['packages'].items() if package['name']=='bioconductor-variantannotation']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "253"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len([element for element in biotools_list if 'EMBOSS' in element['collectionID']])"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
